{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "GetDescriptions.ipynb",
      "version": "0.3.2",
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "metadata": {
        "id": "M-ky1Khq8DSa",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "import requests\n",
        "import urllib.request\n",
        "import time\n",
        "from bs4 import BeautifulSoup"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "leI7YP5bKpg9",
        "colab": {}
      },
      "source": [
        "base_url = 'https://ghr.nlm.nih.gov/gene/'\n",
        "\n",
        "def scrap_site(gene):\n",
        "  response = requests.get(base_url + gene)\n",
        "  soup = BeautifulSoup(response.text, \"html.parser\")\n",
        "  \n",
        "  descr = \"\"\n",
        "\n",
        "  for div_tag in soup.findAll('div',{'class': 'expandBoxContent'}):\n",
        "    descr += div_tag.text\n",
        "    \n",
        "  return descr.trim()\n",
        "  "
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "xtyjeAsJKaDX",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34768
        },
        "outputId": "a9f11ebd-f6d4-41a5-f979-8ec972cf7c40"
      },
      "source": [
        "with open(\"vep.genes.tsv\") as file:\n",
        "  \n",
        "  for line in file:\n",
        "    gene = line.split('\\t')[0]\n",
        "    gene_data = scrap_site(gene)\n",
        "    print(gene + '\\t' + gene_data)"
      ],
      "execution_count": 33,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "HPSE2\t   The HPSE2 gene provides instructions for making a protein called heparanase 2. Little is known about this protein, but its structure is similar to that of another protein called heparanase 1. Heparanase 1 is an enzyme that splits (cleaves) molecules called heparan sulfate proteoglycans (HSPGs) by removing the heparan sulfate portion (the side chain).HSPGs are important parts of the lattice of proteins and other molecules outside the cell (extracellular matrix) and of basement membranes, which are thin, sheet-like structures that separate and support cells in many tissues. Cleavage of HSPGs by heparanase 1 may lead to changes in the basement membrane or extracellular matrix that allow cell movement or release of substances from the cell. The specific function of the heparanase 2 enzyme is not well understood, but studies suggest that it may block the action of heparanase 1.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Migraine     Genetics Home Reference provides information about migraine. More About This Health Condition      Ochoa syndrome     At least nine HPSE2 gene mutations have been identified in people with Ochoa syndrome (also called urofacial syndrome), a disorder that causes urinary problems and unusual facial expressions. These mutations result in changes in the heparanase 2 protein that likely prevent it from functioning. The connection between HPSE2 gene mutations and the features of Ochoa syndrome are unclear. Because the areas of the brain that control facial expression and urination are in close proximity, some researchers have suggested that the genetic changes may lead to an abnormality in this brain region that may account for the symptoms of Ochoa syndrome. Other researchers believe that a defective heparanase 2 protein may lead to problems with the development of the urinary tract or with muscle function in the face and bladder. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about migraine. More About This Health Condition  At least nine HPSE2 gene mutations have been identified in people with Ochoa syndrome (also called urofacial syndrome), a disorder that causes urinary problems and unusual facial expressions. These mutations result in changes in the heparanase 2 protein that likely prevent it from functioning. The connection between HPSE2 gene mutations and the features of Ochoa syndrome are unclear. Because the areas of the brain that control facial expression and urination are in close proximity, some researchers have suggested that the genetic changes may lead to an abnormality in this brain region that may account for the symptoms of Ochoa syndrome. Other researchers believe that a defective heparanase 2 protein may lead to problems with the development of the urinary tract or with muscle function in the face and bladder. More About This Health Condition    Cytogenetic Location: 10q24.2, which is the long (q) arm of chromosome 10 at position 24.2Molecular Location: base pairs 98,457,077 to 99,235,875 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        heparanase 2heparanase-2HPA2HPR2HPSE2_HUMAN     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Urofacial Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HPSE2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     HEPARANASE 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Urofacial Syndrome  Tests of HPSE2  PubMed  HEPARANASE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Al Badr W, Al Bader S, Otto E, Hildebrandt F, Ackley T, Peng W, Xu J, Li J, Owens KM, Bloom D, Innis JW. Exome capture and massively parallel sequencing identifies a novel HPSE2 mutation in a Saudi Arabian child with Ochoa (urofacial) syndrome. J Pediatr Urol. 2011 Oct;7(5):569-73. doi: 10.1016/j.jpurol.2011.02.034. Epub 2011 Mar 29. Citation on PubMed or Free article on PubMed CentralDaly SB, Urquhart JE, Hilton E, McKenzie EA, Kammerer RA, Lewis M, Kerr B, Stuart H, Donnai D, Long DA, Burgu B, Aydogdu O, Derbent M, Garcia-Minaur S, Reardon W, Gener B, Shalev S, Smith R, Woolf AS, Black GC, Newman WG. Mutations in HPSE2 cause urofacial syndrome. Am J Hum Genet. 2010 Jun 11;86(6):963-9. Erratum in: Am J Hum Genet. 2010 Aug 13;87(2):309. Citation on PubMed or Free article on PubMed CentralOMIM: HEPARANASE 2Pang J, Zhang S, Yang P, Hawkins-Lee B, Zhong J, Zhang Y, Ochoa B, Agundez JA, Voelckel MA, Fisher RB, Gu W, Xiong WC, Mei L, She JX, Wang CY. Loss-of-function mutations in HPSE2 cause the autosomal recessive urofacial syndrome. Am J Hum Genet. 2010 Jun 11;86(6):957-62. Erratum in: Am J Hum Genet. 2010 Jul 9;87(1):161. Fisher, Richard B [added]. Citation on PubMed or Free article on PubMed Central \n",
            "NKX2-3\t\n",
            "SEC31B\t\n",
            "SLF2\t\n",
            "PDZD7\t   From NCBI Gene:This gene encodes a ciliary protein homologous to proteins which are mutated in Usher syndrome patients, and mutations and translocations involving this gene have been associated with two types of Usher syndrome. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2010]From UniProt:In cochlear developing hair cells, essential in organizing the USH2 complex at stereocilia ankle links. Blocks inhibition of adenylate cyclase activity mediated by ADGRV1.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Usher syndrome, type 2AUsher syndrome, type 2CDEAFNESS, AUTOSOMAL RECESSIVE 57From UniProt:Deafness, autosomal recessive, 57 (DFNB57): A form of non-syndromic, sensorineural deafness characterized by symmetric, bilateral hearing loss with onset in early childhood. Vestibular function is preserved. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. DFNB57 severity ranges from moderate to severe. [MIM:618003]Usher syndrome 2C (USH2C): USH is a genetically heterogeneous condition characterized by the association of retinitis pigmentosa with sensorineural deafness. Age at onset and differences in auditory and vestibular function distinguish Usher syndrome type 1 (USH1), Usher syndrome type 2 (USH2) and Usher syndrome type 3 (USH3). USH2 is characterized by congenital mild hearing impairment with normal vestibular responses. [MIM:605472]A chromosomal aberration disrupting PDZD7 has been found in patients with non-syndromic sensorineural deafness. Translocation t(10;11),t(10;11).Usher syndrome 2A (USH2A): USH is a genetically heterogeneous condition characterized by the association of retinitis pigmentosa with sensorineural deafness. Age at onset and differences in auditory and vestibular function distinguish Usher syndrome type 1 (USH1), Usher syndrome type 2 (USH2) and Usher syndrome type 3 (USH3). USH2 is characterized by congenital mild hearing impairment with normal vestibular responses. [MIM:276901]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q24.31, which is the long (q) arm of chromosome 10 at position 24.31Molecular Location: base pairs 101,007,683 to 101,031,157 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNB57PDZK7     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PDZD7      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 57OMIM: PDZ DOMAIN-CONTAINING 7OMIM: USHER SYNDROME, TYPE IIAOMIM: USHER SYNDROME, TYPE IIC      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PDZD7  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 57OMIM: PDZ DOMAIN-CONTAINING 7OMIM: USHER SYNDROME, TYPE IIAOMIM: USHER SYNDROME, TYPE IIC  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DPCD\t\n",
            "FBXW4\t   From NCBI Gene:This gene is a member of the F-box/WD-40 gene family, which recruit specific target proteins through their WD-40 protein-protein binding domains for ubiquitin mediated degradation. In mouse, a highly similar protein is thought to be responsible for maintaining the apical ectodermal ridge of developing limb buds; disruption of the mouse gene results in the absence of central digits, underdeveloped or absent metacarpal/metatarsal bones and syndactyly. This phenotype is remarkably similar to split hand-split foot malformation in humans, a clinically heterogeneous condition with a variety of modes of transmission. An autosomal recessive form has been mapped to the chromosomal region where this gene is located, and complex rearrangements involving duplications of this gene and others have been associated with the condition. A pseudogene of this locus has been mapped to one of the introns of the BCR gene on chromosome 22. [provided by RefSeq, Jul 2008]From UniProt:Probably recognizes and binds to some phosphorylated proteins and promotes their ubiquitination and degradation. Likely to be involved in key signaling pathways crucial for normal limb development. May participate in Wnt signaling.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Split-hand/foot malformation 3 (SHFM3): A limb malformation involving the central rays of the autopod and presenting with syndactyly, median clefts of the hands and feet, and aplasia and/or hypoplasia of the phalanges, metacarpals, and metatarsals. Some patients have been found to have mental retardation, ectodermal and craniofacial findings, and orofacial clefting. [MIM:246560]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q24.32, which is the long (q) arm of chromosome 10 at position 24.32Molecular Location: base pairs 101,610,664 to 101,695,295 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DACFBW4FBWD4SHFM3SHSF3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FBXW4      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: F-BOX AND WD40 DOMAIN PROTEIN 4      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FBXW4  OMIM: F-BOX AND WD40 DOMAIN PROTEIN 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PCGF6\t\n",
            "PDCD11\t\n",
            "RBM20\t   From NCBI Gene:This gene encodes a protein that binds RNA and regulates splicing. Mutations in this gene have been associated with familial dilated cardiomyopathy. [provided by RefSeq, Apr 2014]From UniProt:RNA-binding protein that acts as a regulator of mRNA splicing of a subset of genes involved in cardiac development. Regulates splicing of TTN (Titin).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Familial dilated cardiomyopathyFrom NCBI Gene:Dilated cardiomyopathy 1DDFrom UniProt:Cardiomyopathy, dilated 1DD (CMD1DD): A disorder characterized by ventricular dilation and impaired systolic function, resulting in congestive heart failure and arrhythmia. Patients are at risk of premature death. [MIM:613172]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q25.2, which is the long (q) arm of chromosome 10 at position 25.2Molecular Location: base pairs 110,641,933 to 110,839,471 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RBM20      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CARDIOMYOPATHY, DILATED, 1DDOMIM: RNA-BINDING MOTIF PROTEIN 20      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RBM20  OMIM: CARDIOMYOPATHY, DILATED, 1DDOMIM: RNA-BINDING MOTIF PROTEIN 20  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GPAM\t\n",
            "NRAP\t\n",
            "CASP7\t\n",
            "DCLRE1A\t\n",
            "USP6NL\t\n",
            "VWA2\t\n",
            "AFAP1L2\t\n",
            "ATRNL1\t\n",
            "PNLIPRP2\t\n",
            "PROSER2\t\n",
            "TACC2\t\n",
            "DMBT1\t   From NCBI Gene:Loss of sequences from human chromosome 10q has been associated with the progression of human cancers. This gene was originally isolated based on its deletion in a medulloblastoma cell line. This gene is expressed with transcripts of 6.0, 7.5, and 8.0 kb in fetal lung and with one transcript of 8.0 kb in adult lung, although the 7.5 kb transcript has not been characterized. The encoded protein precursor is a glycoprotein containing multiple scavenger receptor cysteine-rich (SRCR) domains separated by SRCR-interspersed domains (SID). Transcript variant 2 (8.0 kb) has been shown to bind surfactant protein D independently of carbohydrate recognition. This indicates that DMBT1 may not be a classical tumor suppressor gene, but rather play a role in the interaction of tumor cells and the immune system. [provided by RefSeq, Mar 2016]From UniProt:May be considered as a candidate tumor suppressor gene for brain, lung, esophageal, gastric, and colorectal cancers. May play roles in mucosal defense system, cellular immune defense and epithelial differentiation. May play a role as an opsonin receptor for SFTPD and SPAR in macrophage tissues throughout the body, including epithelial cells lining the gastrointestinal tract. May play a role in liver regeneration. May be an important factor in fate decision and differentiation of transit-amplifying ductular (oval) cells within the hepatic lineage. Required for terminal differentiation of columnar epithelial cells during early embryogenesis. May function as a binding protein in saliva for the regulation of taste sensation. Binds to HIV-1 envelope protein and has been shown to both inhibit and facilitate viral transmission. Displays a broad calcium-dependent binding spectrum against both Gram-positive and Gram-negative bacteria, suggesting a role in defense against bacterial pathogens. Binds to a range of poly-sulfated and poly-phosphorylated ligands which may explain its broad bacterial-binding specificity. Inhibits cytoinvasion of S.enterica. Associates with the actin cytoskeleton and is involved in its remodeling during regulated exocytosis. Interacts with pancreatic zymogens in a pH-dependent manner and may act as a Golgi cargo receptor in the regulated secretory pathway of the pancreatic acinar cell.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Glioma (GLM): Gliomas are benign or malignant central nervous system neoplasms derived from glial cells. They comprise astrocytomas and glioblastoma multiforme that are derived from astrocytes, oligodendrogliomas derived from oligodendrocytes and ependymomas derived from ependymocytes. [MIM:137800]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q26.13, which is the long (q) arm of chromosome 10 at position 26.13Molecular Location: base pairs 122,560,665 to 122,643,736 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GP340muclinSAGSALSA     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DMBT1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: DELETED IN MALIGNANT BRAIN TUMORS 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DMBT1  OMIM: DELETED IN MALIGNANT BRAIN TUMORS 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CHST15\t\n",
            "ZRANB1\t\n",
            "CTBP2\t\n",
            "NPS\t\n",
            "MKI67\t\n",
            "GLRX3\t\n",
            "CFAP46\t\n",
            "PRAP1\t\n",
            "CYP2E1\t\n",
            "FRG2B\t\n",
            "DCLRE1C\t   From NCBI Gene:This gene encodes a nuclear protein that is involved in V(D)J recombination and DNA repair. The encoded protein has single-strand-specific 5'-3' exonuclease activity; it also exhibits endonuclease activity on 5' and 3' overhangs and hairpins. The protein also functions in the regulation of the cell cycle in response to DNA damage. Mutations in this gene can cause Athabascan-type severe combined immunodeficiency (SCIDA) and Omenn syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]From UniProt:Required for V(D)J recombination, the process by which exons encoding the antigen-binding domains of immunoglobulins and T-cell receptor proteins are assembled from individual V, (D), and J gene segments. V(D)J recombination is initiated by the lymphoid specific RAG endonuclease complex, which generates site specific DNA double strand breaks (DSBs). These DSBs present two types of DNA end structures: hairpin sealed coding ends and phosphorylated blunt signal ends. These ends are independently repaired by the non homologous end joining (NHEJ) pathway to form coding and signal joints respectively. This protein exhibits single-strand specific 5'-3' exonuclease activity in isolation and acquires endonucleolytic activity on 5' and 3' hairpins and overhangs when in a complex with PRKDC. The latter activity is required specifically for the resolution of closed hairpins prior to the formation of the coding joint. May also be required for the repair of complex DSBs induced by ionizing radiation, which require substantial end-processing prior to religation by NHEJ.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Omenn syndromeFrom NCBI Gene:Histiocytic medullary reticulosisSevere combined immunodeficiency with sensitivity to ionizing radiationFrom UniProt:Severe combined immunodeficiency Athabaskan type (SCIDA): A variety of SCID with sensitivity to ionizing radiation. A founder mutation has been detected in Athabascan-speaking native Americans, being inherited as an autosomal recessive trait. Affected individuals exhibit clinical symptoms and defects in DNA repair comparable to those seen in RS-SCID. [MIM:602450]Severe combined immunodeficiency autosomal recessive T-cell-negative/B-cell-negative/NK-cell-positive with sensitivity to ionizing radiation (RSSCID): A form of severe combined immunodeficiency, a genetically and clinically heterogeneous group of rare congenital disorders characterized by impairment of both humoral and cell-mediated immunity, leukopenia, and low or absent antibody levels. Patients present in infancy with recurrent, persistent infections by opportunistic organisms. The common characteristic of all types of SCID is absence of T-cell-mediated cellular immunity due to a defect in T-cell development. Individuals affected by RS-SCID show defects in the DNA repair machinery necessary for coding joint formation and the completion of V(D)J recombination. A subset of cells from such patients show increased radiosensitivity. [MIM:602450]Omenn syndrome (OS): Severe immunodeficiency characterized by the presence of activated, anergic, oligoclonal T-cells, hypereosinophilia, and high IgE levels. [MIM:603554]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10p13, which is the short (p) arm of chromosome 10 at position 13Molecular Location: base pairs 14,897,564 to 14,954,432 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        A-SCIDDCLREC1CRS-SCIDSCIDASNM1C     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DCLRE1C      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DNA CROSS-LINK REPAIR PROTEIN 1COMIM: OMENN SYNDROMEOMIM: SEVERE COMBINED IMMUNODEFICIENCY WITH SENSITIVITY TO IONIZING RADIATION      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DCLRE1C  OMIM: DNA CROSS-LINK REPAIR PROTEIN 1COMIM: OMENN SYNDROMEOMIM: SEVERE COMBINED IMMUNODEFICIENCY WITH SENSITIVITY TO IONIZING RADIATION  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MALRD1\t\n",
            "SPAG6\t\n",
            "KIAA1217\t\n",
            "PTCHD3\t\n",
            "SVIL\t\n",
            "PITRM1\t\n",
            "ZNF239\t\n",
            "AGAP4\t\n",
            "GPRIN2\t\n",
            "NPY4R\t\n",
            "ANXA8L1\t\n",
            "AKR1C2\t   From NCBI Gene:This gene encodes a member of the aldo/keto reductase superfamily, which consists of more than 40 known enzymes and proteins. These enzymes catalyze the conversion of aldehydes and ketones to their corresponding alcohols using NADH and/or NADPH as cofactors. The enzymes display overlapping but distinct substrate specificity. This enzyme binds bile acid with high affinity, and shows minimal 3-alpha-hydroxysteroid dehydrogenase activity. This gene shares high sequence identity with three other gene members and is clustered with those three genes at chromosome 10p15-p14. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Dec 2011]From UniProt:Works in concert with the 5-alpha/5-beta-steroid reductases to convert steroid hormones into the 3-alpha/5-alpha and 3-alpha/5-beta-tetrahydrosteroids. Catalyzes the inactivation of the most potent androgen 5-alpha-dihydrotestosterone (5-alpha-DHT) to 5-alpha-androstane-3-alpha,17-beta-diol (3-alpha-diol). Has a high bile-binding ability.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:46,XY sex reversal 8From UniProt:46,XY sex reversal 8 (SRXY8): A disorder of sex development. Affected individuals have a 46,XY karyotype but present as phenotypically normal females. [MIM:614279]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10p15.1, which is the short (p) arm of chromosome 10 at position 15.1Molecular Location: base pairs 4,987,775 to 5,018,033 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AKR1C-pseudoBABPDDDD-2DD/BABPDD2DDH2HAKRDHBABMCDR2SRXY8TDD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AKR1C2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: 46,XY SEX REVERSAL 8OMIM: ALDO-KETO REDUCTASE FAMILY 1, MEMBER C2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AKR1C2  OMIM: 46,XY SEX REVERSAL 8OMIM: ALDO-KETO REDUCTASE FAMILY 1, MEMBER C2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "AKR1C3\t\n",
            "AGAP7P\t\n",
            "ASAH2B\t\n",
            "MBL2\t   The MBL2 gene provides instructions for making a protein that assembles into a protein complex called mannose-binding lectin. Functional mannose-binding lectins are made up of two to six protein groups called trimers, which are each composed of three of the protein pieces (subunits) produced from the MBL2 gene. This protein complex plays an important role in the immune system's response to foreign invaders (pathogens).Mannose-binding lectin recognizes and attaches (binds) to sugars, such as mannose, fucose, and glucose, that are found on the surface of bacteria, viruses, and yeast. This binding turns on (activates) the complement system, which is a group of immune system proteins that work together to destroy pathogens, trigger inflammation, and remove debris from cells and tissues. Attachment of mannose-binding lectin also targets the pathogen to be engulfed and broken down by special immune cells. Recognition of foreign invaders by mannose-binding lectin provides one of the body's first lines of defense against infection.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Mannose-binding lectin deficiency     Several common mutations of the MBL2 gene can lead to a condition called mannose-binding lectin deficiency. People with this condition have low levels of mannose-binding lectin and may be susceptible to recurrent infections. Several of the disease-associated mutations occur in a region of the MBL2 gene known as exon 1 and result in a change to single protein building blocks (amino acids) in the mannose-binding lectin subunit. Other mutations occur in an area of DNA near the MBL2 gene called the promoter region, which helps control the production of the mannose-binding lectin subunit.The change of a single amino acid in the mannose-binding lectin subunit eliminates its ability to assemble into the functional mannose-binding lectin. Similarly, certain mutations in the promoter region of the MBL2 gene reduce production of the mannose-binding lectin subunit, leading to a decreased number of subunits available for protein assembly and a reduction in the amount of functional protein. With decreased levels of mannose-binding lectin, the body does not recognize and fight foreign invaders efficiently. Consequently, infections can be more common in people with this condition. However, researchers believe that a number of factors, including other genetic and environmental factors, are involved in the development of mannose-binding lectin deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several common mutations of the MBL2 gene can lead to a condition called mannose-binding lectin deficiency. People with this condition have low levels of mannose-binding lectin and may be susceptible to recurrent infections. Several of the disease-associated mutations occur in a region of the MBL2 gene known as exon 1 and result in a change to single protein building blocks (amino acids) in the mannose-binding lectin subunit. Other mutations occur in an area of DNA near the MBL2 gene called the promoter region, which helps control the production of the mannose-binding lectin subunit.The change of a single amino acid in the mannose-binding lectin subunit eliminates its ability to assemble into the functional mannose-binding lectin. Similarly, certain mutations in the promoter region of the MBL2 gene reduce production of the mannose-binding lectin subunit, leading to a decreased number of subunits available for protein assembly and a reduction in the amount of functional protein. With decreased levels of mannose-binding lectin, the body does not recognize and fight foreign invaders efficiently. Consequently, infections can be more common in people with this condition. However, researchers believe that a number of factors, including other genetic and environmental factors, are involved in the development of mannose-binding lectin deficiency. More About This Health Condition    Cytogenetic Location: 10q21.1, which is the long (q) arm of chromosome 10 at position 21.1Molecular Location: base pairs 52,764,977 to 52,772,847 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        COLEC1collectin-1HSMBPCmannan-binding lectinmannose-binding lectin (protein C) 2, solublemannose-binding lectin (protein C) 2, soluble (opsonic defect)mannose-binding lectin 2, soluble (opsonic defect)mannose-binding protein Cmannose-binding protein C precursorMBLMBL2_HUMANMBL2DMBPMBP-CMBP1     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Immunobiology: The Immune System in Health and Disease (5th Edition, 2001): Mannan-Binding Lectin (MBL) Binds to Patterns of Carbohydrate Groups in the Correct Spatial OrientationMolecular Biology of the Cell (4th Edition, 2002): Complement Activation Targets Pathogens for Phagocytosis or Lysis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MBL2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     LECTIN, MANNOSE-BINDING, SOLUBLE, 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology: The Immune System in Health and Disease (5th Edition, 2001): Mannan-Binding Lectin (MBL) Binds to Patterns of Carbohydrate Groups in the Correct Spatial OrientationMolecular Biology of the Cell (4th Edition, 2002): Complement Activation Targets Pathogens for Phagocytosis or Lysis  Tests of MBL2  PubMed  LECTIN, MANNOSE-BINDING, SOLUBLE, 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Arora M, Munoz E, Tenner AJ. Identification of a site on mannan-binding lectin critical for enhancement of phagocytosis. J Biol Chem. 2001 Nov 16;276(46):43087-94. Epub 2001 Aug 30. Citation on PubMedBouwman LH, Roep BO, Roos A. Mannose-binding lectin: clinical implications for infection, transplantation, and autoimmunity. Hum Immunol. 2006 Apr-May;67(4-5):247-56. Epub 2006 Apr 17. Review. Citation on PubMedOMIM: LECTIN, MANNOSE-BINDING, SOLUBLE, 2Martin P, Lerner A, Johnson L, Lerner DL, Haraguchi S, Good RA, Day NK. Inherited mannose-binding lectin deficiency as evidenced by genetic and immunologic analyses: association with severe recurrent infections. Ann Allergy Asthma Immunol. 2003 Oct;91(4):386-92. Citation on PubMedWeis WI, Drickamer K, Hendrickson WA. Structure of a C-type mannose-binding protein complexed with an oligosaccharide. Nature. 1992 Nov 12;360(6400):127-34. Citation on PubMed \n",
            "FAM13C\t\n",
            "CCDC6\t   From NCBI Gene:This gene encodes a coiled-coil domain-containing protein. The encoded protein is ubiquitously expressed and may function as a tumor suppressor. A chromosomal rearrangement resulting in the expression of a fusion gene containing a portion of this gene and the intracellular kinase-encoding domain of the ret proto-oncogene is the cause of thyroid papillary carcinoma.[provided by RefSeq, Sep 2010]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving CCDC6 is found in papillary thyroid carcinomas (PTCs). Inversion inv(10)(q11.2;q21) generates the RET/CCDC6 (PTC1) oncogene.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q21.2, which is the long (q) arm of chromosome 10 at position 21.2Molecular Location: base pairs 59,788,747 to 59,906,656 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        D10S170H4PTCTPCTST1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CCDC6      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 6      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CCDC6  OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ANK3\t   From NCBI Gene:Ankyrins are a family of proteins that are believed to link the integral membrane proteins to the underlying spectrin-actin cytoskeleton and play key roles in activities such as cell motility, activation, proliferation, contact, and the maintenance of specialized membrane domains. Multiple isoforms of ankyrin with different affinities for various target proteins are expressed in a tissue-specific, developmentally regulated manner. Most ankyrins are typically composed of three structural domains: an amino-terminal domain containing multiple ankyrin repeats; a central region with a highly conserved spectrin binding domain; and a carboxy-terminal regulatory domain which is the least conserved and subject to variation. Ankyrin 3 is an immunologically distinct gene product from ankyrins 1 and 2, and was originally found at the axonal initial segment and nodes of Ranvier of neurons in the central and peripheral nervous systems. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Feb 2011]From UniProt:In skeletal muscle, required for costamere localization of DMD and betaDAG1 (By similarity). Membrane-cytoskeleton linker. May participate in the maintenance/targeting of ion channels and cell adhesion molecules at the nodes of Ranvier and axonal initial segments. Regulates KCNA1 channel activity in function of dietary Mg(2+) levels, and thereby contributes to the regulation of renal Mg(2+) reabsorption (PubMed:23903368).Isoform 5: May be part of a Golgi-specific membrane cytoskeleton in association with beta-spectrin.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mental retardation, autosomal recessive 37From UniProt:Genetic variations in ANK3 may be associated with autism spectrum disorders susceptibility.Mental retardation, autosomal recessive 37 (MRT37): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. MRT37 patients manifest delayed global development with speech delay, hypotonia, spasticity, and a sleep disorder. Severe behavioral abnormalities include aggression, hyperactivity, and grinding of the teeth. [MIM:615493]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q21.2, which is the long (q) arm of chromosome 10 at position 21.2Molecular Location: base pairs 60,026,298 to 60,733,526 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ANKYRIN-GMRT37     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ANK3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ANKYRIN 3OMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 37      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ANK3  OMIM: ANKYRIN 3OMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 37  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RTKN2\t\n",
            "ADO\t\n",
            "CTNNA3\t   From NCBI Gene:This gene encodes a protein that belongs to the vinculin/alpha-catenin family. The encoded protein plays a role in cell-cell adhesion in muscle cells. Mutations in this gene are associated with arrhythmogenic right ventricular dysplasia, familial 13. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2014]From UniProt:May be involved in formation of stretch-resistant cell-cell adhesion complexes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Arrhythmogenic right ventricular cardiomyopathyFrom NCBI Gene:Arrhythmogenic right ventricular dysplasia, familial, 13From UniProt:Arrhythmogenic right ventricular dysplasia, familial, 13 (ARVD13): A congenital heart disease characterized by infiltration of adipose and fibrous tissue into the right ventricle and loss of myocardial cells, resulting in ventricular and supraventricular arrhythmias. [MIM:615616]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q21.3, which is the long (q) arm of chromosome 10 at position 21.3Molecular Location: base pairs 65,912,518 to 67,696,217 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARVD13VR22     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CTNNA3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA, FAMILIAL, 13OMIM: CATENIN, ALPHA-3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CTNNA3  OMIM: ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA, FAMILIAL, 13OMIM: CATENIN, ALPHA-3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYPN\t   From NCBI Gene:Striated muscle in vertebrates comprises large proteins which must be organized properly to contract efficiently. Z-lines in striated muscle are a sign of this organization, representing the ends of actin thin filaments, titin, nebulin or nebulette and accessory proteins required for structure and function. This gene encodes a protein which interacts with nebulin in skeletal muscle or nebulette in cardiac muscle and alpha-actinin. In addition, this gene product can interact with a protein with the I-band indicating it has a regulatory as well as structural function. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2011]From UniProt:Component of the sarcomere that tethers together nebulin (skeletal muscle) and nebulette (cardiac muscle) to alpha-actinin, at the Z lines.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Familial dilated cardiomyopathyFrom NCBI Gene:Nemaline myopathy 11, autosomal recessiveDilated cardiomyopathy 1KKFrom UniProt:Cardiomyopathy, dilated 1KK (CMD1KK): A disorder characterized by ventricular dilation and impaired systolic function, resulting in congestive heart failure and arrhythmia. Patients are at risk of premature death. [MIM:615248]Cardiomyopathy, familial hypertrophic 22 (CMH22): A hereditary heart disorder characterized by ventricular hypertrophy, which is usually asymmetric and often involves the interventricular septum. The symptoms include dyspnea, syncope, collapse, palpitations, and chest pain. They can be readily provoked by exercise. The disorder has inter- and intrafamilial variability ranging from benign to malignant forms with high risk of cardiac failure and sudden cardiac death. [MIM:615248]Cardiomyopathy, familial restrictive 4 (RCM4): A heart disorder characterized by impaired filling of the ventricles with reduced diastolic volume, in the presence of normal or near normal wall thickness and systolic function. [MIM:615248]Nemaline myopathy 11 (NEM11): An autosomal recessive form of nemaline myopathy. Nemaline myopathies are muscular disorders characterized by muscle weakness of varying severity and onset, and abnormal thread-like or rod-shaped structures in muscle fibers on histologic examination. NEM11 is characterized by slowly progressive muscle weakness and atrophy, mainly affecting the lower limbs and neck. Some patients may have mild cardiac or respiratory involvement, but they do not have respiratory failure. [MIM:617336]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q21.3, which is the long (q) arm of chromosome 10 at position 21.3Molecular Location: base pairs 68,087,907 to 68,212,017 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMD1DDCMH22MYOPNEM11RCM4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYPN      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CARDIOMYOPATHY, DILATED, 1KKOMIM: MYOPALLADINOMIM: NEMALINE MYOPATHY 11, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MYPN  OMIM: CARDIOMYOPATHY, DILATED, 1KKOMIM: MYOPALLADINOMIM: NEMALINE MYOPATHY 11, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HKDC1\t\n",
            "CDH23\t   The CDH23 gene provides instructions for making cadherin 23, a type of protein that helps cells stick together. Different versions of this protein are made in different cell types, including a short version in the retina, which is the light-sensitive layer in the back of the eye, and a longer version in the inner ear. Cadherin 23 interacts with other proteins in the cell membrane as part of a protein complex that is involved in cell attachment.Research suggests that the cadherin 23 protein complex helps to shape structures in the inner ear called hair bundles. These structures are made of stereocilia, which are hairlike projections that bend in response to sound waves. This bending motion is critical for converting sound waves to nerve impulses, which are then transmitted to the brain. Stereocilia are also elements of the vestibular system, the part of the inner ear that helps maintain the body's balance and orientation in space. Bending of these stereocilia is needed to transmit signals from the vestibular system to the brain.In the retina, the role of the cadherin 23 protein complex is less well understood. Studies suggest that it plays a critical role in the function of photoreceptor cells, which are specialized cells that detect light and color.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Nonsyndromic hearing loss     Several dozen mutations in the CDH23 gene have been identified in people with nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene cause a form of hearing loss designated as DFNB12. This type of hearing loss begins before a child learns to speak (prelingual) and is severe to profound.Most of the mutations that cause DFNB12 change single protein building blocks (amino acids) in cadherin 23. These mutations reduce but do not eliminate the function of this protein. The altered protein disrupts development of stereocilia in the inner ear, which leads to hearing loss.Researchers speculate that some children with apparently nonsyndromic hearing loss caused by a CDH23 mutation may actually have Usher syndrome (described below). A few children thought to have DFNB12 have developed the vision disorder retinitis pigmentosa later in life, which is characteristic of Usher syndrome. More About This Health Condition      Usher syndrome     More than 60 mutations in the CDH23 gene can cause Usher syndrome type I, which is characterized by a combination of hearing loss, vision loss, and problems with balance and coordination. Specifically, CDH23 gene mutations cause a form of the disorder known as Usher syndrome type ID (USH1D).Most of these mutations disrupt protein production, resulting in an abnormally small, nonfunctional version of cadherin 23 or preventing the production of any of this protein. Less frequently, mutations change single amino acids in cadherin 23. All of the CDH23 gene mutations that cause Usher syndrome appear to eliminate the function of cadherin 23. A lack of this protein in the inner ear disrupts the normal development and function of stereocilia, which leads to hearing loss and difficulty with balance and coordination. A lack of this protein in the retina causes retinitis pigmentosa, a condition in which light-sensing cells of the retina gradually deteriorate, resulting in progressive vision loss. More About This Health Condition      Age-related hearing loss     Genetics Home Reference provides information about age-related hearing loss. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several dozen mutations in the CDH23 gene have been identified in people with nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene cause a form of hearing loss designated as DFNB12. This type of hearing loss begins before a child learns to speak (prelingual) and is severe to profound.Most of the mutations that cause DFNB12 change single protein building blocks (amino acids) in cadherin 23. These mutations reduce but do not eliminate the function of this protein. The altered protein disrupts development of stereocilia in the inner ear, which leads to hearing loss.Researchers speculate that some children with apparently nonsyndromic hearing loss caused by a CDH23 mutation may actually have Usher syndrome (described below). A few children thought to have DFNB12 have developed the vision disorder retinitis pigmentosa later in life, which is characteristic of Usher syndrome. More About This Health Condition  More than 60 mutations in the CDH23 gene can cause Usher syndrome type I, which is characterized by a combination of hearing loss, vision loss, and problems with balance and coordination. Specifically, CDH23 gene mutations cause a form of the disorder known as Usher syndrome type ID (USH1D).Most of these mutations disrupt protein production, resulting in an abnormally small, nonfunctional version of cadherin 23 or preventing the production of any of this protein. Less frequently, mutations change single amino acids in cadherin 23. All of the CDH23 gene mutations that cause Usher syndrome appear to eliminate the function of cadherin 23. A lack of this protein in the inner ear disrupts the normal development and function of stereocilia, which leads to hearing loss and difficulty with balance and coordination. A lack of this protein in the retina causes retinitis pigmentosa, a condition in which light-sensing cells of the retina gradually deteriorate, resulting in progressive vision loss. More About This Health Condition  Genetics Home Reference provides information about age-related hearing loss. More About This Health Condition    Cytogenetic Location: 10q22.1, which is the long (q) arm of chromosome 10 at position 22.1Molecular Location: base pairs 71,396,934 to 71,815,947 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAD23_HUMANcadherin-23cadherin-like 23cadherin-related 23CDHR23DFNB12KIAA1774KIAA1812otocadherinUSH1D     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Neuroscience (second edition, 2001): Hair Cells and the Mechanoelectrical Transduction of Sound WavesNeuroscience (second edition, 2001): The Retina      Clinical Information from GeneReviews (2 links)     Hereditary Hearing Loss and Deafness OverviewUsher Syndrome Type I      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CDH23      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CADHERIN 23      Research Resources (9 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHereditary Hearing Loss HomepageHGNC Gene Symbol ReportLeiden Open Variation Database: CDH23 Gene MutationsMonarch InitiativeNCBI GeneRetNet: Summaries of Genes and Loci Causing Retinal Diseases: CDH23UniProt      Neuroscience (second edition, 2001): Hair Cells and the Mechanoelectrical Transduction of Sound WavesNeuroscience (second edition, 2001): The Retina  Hereditary Hearing Loss and Deafness OverviewUsher Syndrome Type I  Tests of CDH23  PubMed  CADHERIN 23  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHereditary Hearing Loss HomepageHGNC Gene Symbol ReportLeiden Open Variation Database: CDH23 Gene MutationsMonarch InitiativeNCBI GeneRetNet: Summaries of Genes and Loci Causing Retinal Diseases: CDH23UniProt  Astuto LM, Bork JM, Weston MD, Askew JW, Fields RR, Orten DJ, Ohliger SJ, Riazuddin S, Morell RJ, Khan S, Riazuddin S, Kremer H, van Hauwe P, Moller CG, Cremers CW, Ayuso C, Heckenlively JR, Rohrschneider K, Spandau U, Greenberg J, Ramesar R, Reardon W, Bitoun P, Millan J, Legge R, Friedman TB, Kimberling WJ. CDH23 mutation and phenotype heterogeneity: a profile of 107 diverse families with Usher syndrome and nonsyndromic deafness. Am J Hum Genet. 2002 Aug;71(2):262-75. Epub 2002 Jun 19. Citation on PubMed or Free article on PubMed CentralBolz H, von Brederlow B, Ramrez A, Bryda EC, Kutsche K, Nothwang HG, Seeliger M, del C-Salced Cabrera M, Vila MC, Molina OP, Gal A, Kubisch C. Mutation of CDH23, encoding a new member of the cadherin gene family, causes Usher syndrome type 1D. Nat Genet. 2001 Jan;27(1):108-12. Citation on PubMedBork JM, Peters LM, Riazuddin S, Bernstein SL, Ahmed ZM, Ness SL, Polomeno R, Ramesh A, Schloss M, Srisailpathy CR, Wayne S, Bellman S, Desmukh D, Ahmed Z, Khan SN, Kaloustian VM, Li XC, Lalwani A, Riazuddin S, Bitner-Glindzicz M, Nance WE, Liu XZ, Wistow G, Smith RJ, Griffith AJ, Wilcox ER, Friedman TB, Morell RJ. Usher syndrome 1D and nonsyndromic autosomal recessive deafness DFNB12 are caused by allelic mutations of the novel cadherin-like gene CDH23. Am J Hum Genet. 2001 Jan;68(1):26-37. Epub 2000 Nov 21. Citation on PubMed or Free article on PubMed CentralMiyagawa M, Nishio SY, Usami S. Prevalence and clinical features of hearing loss patients with CDH23 mutations: a large cohort study. PLoS One. 2012;7(8):e40366. doi: 10.1371/journal.pone.0040366. Epub 2012 Aug 10. Citation on PubMed or Free article on PubMed CentralOshima A, Jaijo T, Aller E, Millan JM, Carney C, Usami S, Moller C, Kimberling WJ. Mutation profile of the CDH23 gene in 56 probands with Usher syndrome type I. Hum Mutat. 2008 Jun;29(6):E37-46. doi: 10.1002/humu.20761. Citation on PubMed or Free article on PubMed CentralSchultz JM, Bhatti R, Madeo AC, Turriff A, Muskett JA, Zalewski CK, King KA, Ahmed ZM, Riazuddin S, Ahmad N, Hussain Z, Qasim M, Kahn SN, Meltzer MR, Liu XZ, Munisamy M, Ghosh M, Rehm HL, Tsilou ET, Griffith AJ, Zein WM, Brewer CC, Riazuddin S, Friedman TB. Allelic hierarchy of CDH23 mutations causing non-syndromic deafness DFNB12 or Usher syndrome USH1D in compound heterozygotes. J Med Genet. 2011 Nov;48(11):767-75. doi: 10.1136/jmedgenet-2011-100262. Epub 2011 Sep 22. Citation on PubMedSiemens J, Lillo C, Dumont RA, Reynolds A, Williams DS, Gillespie PG, Mller U. Cadherin 23 is a component of the tip link in hair-cell stereocilia. Nature. 2004 Apr 29;428(6986):950-5. Epub 2004 Mar 31. Citation on PubMed \n",
            "CAMK2G\t\n",
            "PLAU\t   From NCBI Gene:This gene encodes a secreted serine protease that converts plasminogen to plasmin. The encoded preproprotein is proteolytically processed to generate A and B polypeptide chains. These chains associate via a single disulfide bond to form the catalytically inactive high molecular weight urokinase-type plasminogen activator (HMW-uPA). HMW-uPA can be further processed into the catalytically active low molecular weight urokinase-type plasminogen activator (LMW-uPA). This low molecular weight form does not bind to the urokinase-type plasminogen activator receptor. Mutations in this gene may be associated with Quebec platelet disorder and late-onset Alzheimer's disease. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016]From UniProt:Specifically cleaves the zymogen plasminogen to form the active enzyme plasmin.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Alzheimer's diseaseQuebec platelet disorderFrom UniProt:Quebec platelet disorder (QPD): An autosomal dominant bleeding disorder due to a gain-of-function defect in fibrinolysis. Although affected individuals do not exhibit systemic fibrinolysis, they show delayed onset bleeding after challenge, such as surgery. The hallmark of the disorder is markedly increased PLAU levels within platelets, which causes intraplatelet plasmin generation and secondary degradation of alpha-granule proteins. [MIM:601709]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q22.2, which is the long (q) arm of chromosome 10 at position 22.2Molecular Location: base pairs 73,909,182 to 73,917,501 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATFBDPLT5QPDu-PAUPAURK     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PLAU      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ALZHEIMER DISEASEOMIM: PLASMINOGEN ACTIVATOR, URINARYOMIM: QUEBEC PLATELET DISORDER      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PLAU  OMIM: ALZHEIMER DISEASEOMIM: PLASMINOGEN ACTIVATOR, URINARYOMIM: QUEBEC PLATELET DISORDER  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KAT6B\t   The KAT6B gene provides instructions for making a type of enzyme called a histone acetyltransferase. These enzymes modify histones, which are structural proteins that attach (bind) to DNA and give chromosomes their shape. By adding a small molecule called an acetyl group to particular locations on histones, histone acetyltransferases control the activity of certain genes.Little is known about the function of the histone acetyltransferase produced from the KAT6B gene. It is active in cells and tissues throughout the body, where it interacts with many other proteins. It appears to regulate genes that are important for early development, including development of the skeleton and nervous system.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Genitopatellar syndrome     At least eight mutations in the KAT6B gene have been identified in people with genitopatellar syndrome, a rare condition characterized by genital abnormalities, missing or underdeveloped kneecaps (patellae), intellectual disability, and abnormalities affecting other parts of the body. The mutations that cause genitopatellar syndrome occur near the end of the KAT6B gene in a region known as exon 18. These mutations lead to the production of a shortened histone acetyltransferase enzyme. Researchers suspect that the shortened enzyme may function differently than the full-length version, altering the regulation of various genes during early development. Because the altered enzyme takes on a different function, these mutations are described as \"gain-of-function.\" However, it is unclear how these changes lead to the specific features of genitopatellar syndrome. More About This Health Condition      Ohdo syndrome, Say-Barber-Biesecker-Young-Simpson variant     More than 10 mutations in the KAT6B gene have been found to cause the Say-Barber-Biesecker-Young-Simpson (SBBYS) variant of Ohdo syndrome. This condition has signs and symptoms that overlap with those of genitopatellar syndrome (described above), although some of the specific developmental abnormalities differ between the two conditions. Mutations that cause the SBBYS variant of Ohdo syndrome have been identified throughout the KAT6B gene, although  many of them occur in exon 18. Studies suggest that these mutations likely prevent the production of functional histone acetyltransferase from one copy of the KAT6B gene in each cell. A shortage of this enzyme impairs the regulation of various genes during early development. Because these mutations lead to a reduction in the enzyme, they are described as \"loss-of-function.\" However, it is unclear how these changes lead to the specific features of the condition. More About This Health Condition      Coloboma     Genetics Home Reference provides information about coloboma. More About This Health Condition      Cancers     Genetic changes involving the KAT6B gene have been associated with certain types of cancer. These mutations are somatic, which means they are acquired during a person's lifetime and are present only in certain cells. The genetic changes are chromosomal rearrangements (translocations) that disrupt the region of chromosome 10 containing the KAT6B gene. Researchers have found a translocation that attaches this region of chromosome 10 to part of chromosome 16 in some people with a cancer of blood-forming cells called acute myeloid leukemia (AML). This translocation has also been identified in some people with therapy-related myelodysplastic syndrome, a blood disorder that can occur after a person has undergone chemotherapy for another form of cancer.It is unclear how translocations involving the KAT6B gene are related to the development of cancer. These changes likely alter histone modification, which could prevent normal regulation of gene activity. Impaired gene regulation may contribute to the growth of cancers by allowing abnormal cells to grow and divide uncontrollably.      Tumors     Somatic changes involving the KAT6B gene have also been identified in some people with uterine leiomyomas, which are noncancerous growths in the uterus that are also known as uterine fibroids. Uterine leiomyomas are common in adult women. These growths can cause pelvic pain and abnormal bleeding, and, in some cases, lead to an inability to have biological children (infertility). The genetic change associated with uterine leiomyomas is a translocation between the region of chromosome 10 containing the KAT6B gene and a particular region of chromosome 17. It is unclear how this translocation is related to tumor development. Changes in histone modification that impair normal gene regulation may allow certain cells to divide in an uncontrolled way, leading to the growth of a tumor.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least eight mutations in the KAT6B gene have been identified in people with genitopatellar syndrome, a rare condition characterized by genital abnormalities, missing or underdeveloped kneecaps (patellae), intellectual disability, and abnormalities affecting other parts of the body. The mutations that cause genitopatellar syndrome occur near the end of the KAT6B gene in a region known as exon 18. These mutations lead to the production of a shortened histone acetyltransferase enzyme. Researchers suspect that the shortened enzyme may function differently than the full-length version, altering the regulation of various genes during early development. Because the altered enzyme takes on a different function, these mutations are described as \"gain-of-function.\" However, it is unclear how these changes lead to the specific features of genitopatellar syndrome. More About This Health Condition  More than 10 mutations in the KAT6B gene have been found to cause the Say-Barber-Biesecker-Young-Simpson (SBBYS) variant of Ohdo syndrome. This condition has signs and symptoms that overlap with those of genitopatellar syndrome (described above), although some of the specific developmental abnormalities differ between the two conditions. Mutations that cause the SBBYS variant of Ohdo syndrome have been identified throughout the KAT6B gene, although  many of them occur in exon 18. Studies suggest that these mutations likely prevent the production of functional histone acetyltransferase from one copy of the KAT6B gene in each cell. A shortage of this enzyme impairs the regulation of various genes during early development. Because these mutations lead to a reduction in the enzyme, they are described as \"loss-of-function.\" However, it is unclear how these changes lead to the specific features of the condition. More About This Health Condition  Genetics Home Reference provides information about coloboma. More About This Health Condition  Genetic changes involving the KAT6B gene have been associated with certain types of cancer. These mutations are somatic, which means they are acquired during a person's lifetime and are present only in certain cells. The genetic changes are chromosomal rearrangements (translocations) that disrupt the region of chromosome 10 containing the KAT6B gene. Researchers have found a translocation that attaches this region of chromosome 10 to part of chromosome 16 in some people with a cancer of blood-forming cells called acute myeloid leukemia (AML). This translocation has also been identified in some people with therapy-related myelodysplastic syndrome, a blood disorder that can occur after a person has undergone chemotherapy for another form of cancer.It is unclear how translocations involving the KAT6B gene are related to the development of cancer. These changes likely alter histone modification, which could prevent normal regulation of gene activity. Impaired gene regulation may contribute to the growth of cancers by allowing abnormal cells to grow and divide uncontrollably.  Somatic changes involving the KAT6B gene have also been identified in some people with uterine leiomyomas, which are noncancerous growths in the uterus that are also known as uterine fibroids. Uterine leiomyomas are common in adult women. These growths can cause pelvic pain and abnormal bleeding, and, in some cases, lead to an inability to have biological children (infertility). The genetic change associated with uterine leiomyomas is a translocation between the region of chromosome 10 containing the KAT6B gene and a particular region of chromosome 17. It is unclear how this translocation is related to tumor development. Changes in histone modification that impair normal gene regulation may allow certain cells to divide in an uncontrolled way, leading to the growth of a tumor.    Cytogenetic Location: 10q22.2, which is the long (q) arm of chromosome 10 at position 22.2Molecular Location: base pairs 74,824,927 to 75,032,623 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GTPTShistone acetyltransferase KAT6Bhistone acetyltransferase MORFhistone acetyltransferase MOZ2histone acetyltransferase MYST4K(lysine) acetyltransferase 6BKAT6B_HUMANmonocytic leukemia zinc finger protein-related factorMORFMOZ-related factorMOZ2MYST-4MYST histone acetyltransferase (monocytic leukemia) 4MYST4qkfquerkopfZC2HC6B     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Epigenomics Help (2010): Histone Modification \"Writers\"      Clinical Information from GeneReviews (1 link)     KAT6B-Related Disorders      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KAT6B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (4 links)     LEIOMYOMA, UTERINELEUKEMIA, ACUTE MYELOIDLYSINE ACETYLTRANSFERASE 6BMYELODYSPLASTIC SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Epigenomics Help (2010): Histone Modification \"Writers\"  KAT6B-Related Disorders  Tests of KAT6B  PubMed  LEIOMYOMA, UTERINELEUKEMIA, ACUTE MYELOIDLYSINE ACETYLTRANSFERASE 6BMYELODYSPLASTIC SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Campeau PM, Kim JC, Lu JT, Schwartzentruber JA, Abdul-Rahman OA, Schlaubitz S, Murdock DM, Jiang MM, Lammer EJ, Enns GM, Rhead WJ, Rowland J, Robertson SP, Cormier-Daire V, Bainbridge MN, Yang XJ, Gingras MC, Gibbs RA, Rosenblatt DS, Majewski J, Lee BH. Mutations in KAT6B, encoding a histone acetyltransferase, cause Genitopatellar syndrome. Am J Hum Genet. 2012 Feb 10;90(2):282-9. doi: 10.1016/j.ajhg.2011.11.023. Epub 2012 Jan 19. Citation on PubMed or Free article on PubMed CentralCampeau PM, Lu JT, Dawson BC, Fokkema IF, Robertson SP, Gibbs RA, Lee BH. The KAT6B-related disorders genitopatellar syndrome and Ohdo/SBBYS syndrome have distinct clinical features reflecting distinct molecular mechanisms. Hum Mutat. 2012 Nov;33(11):1520-5. doi: 10.1002/humu.22141. Epub 2012 Jul 12. Review. Citation on PubMed or Free article on PubMed CentralChampagne N, Bertos NR, Pelletier N, Wang AH, Vezmar M, Yang Y, Heng HH, Yang XJ. Identification of a human histone acetyltransferase related to monocytic leukemia zinc finger protein. J Biol Chem. 1999 Oct 1;274(40):28528-36. Citation on PubMedClayton-Smith J, O'Sullivan J, Daly S, Bhaskar S, Day R, Anderson B, Voss AK, Thomas T, Biesecker LG, Smith P, Fryer A, Chandler KE, Kerr B, Tassabehji M, Lynch SA, Krajewska-Walasek M, McKee S, Smith J, Sweeney E, Mansour S, Mohammed S, Donnai D, Black G. Whole-exome-sequencing identifies mutations in histone acetyltransferase gene KAT6B in individuals with the Say-Barber-Biesecker variant of Ohdo syndrome. Am J Hum Genet. 2011 Nov 11;89(5):675-81. doi: 10.1016/j.ajhg.2011.10.008. Citation on PubMed or Free article on PubMed CentralKojima K, Kaneda K, Yoshida C, Dansako H, Fujii N, Yano T, Shinagawa K, Yasukawa M, Fujita S, Tanimoto M. A novel fusion variant of the MORF and CBP genes detected in therapy-related myelodysplastic syndrome with t(10;16)(q22;p13). Br J Haematol. 2003 Jan;120(2):271-3. Citation on PubMedMoore SD, Herrick SR, Ince TA, Kleinman MS, Dal Cin P, Morton CC, Quade BJ. Uterine leiomyomata with t(10;17) disrupt the histone acetyltransferase MORF. Cancer Res. 2004 Aug 15;64(16):5570-7. Citation on PubMedPanagopoulos I, Fioretos T, Isaksson M, Samuelsson U, Billstrm R, Strmbeck B, Mitelman F, Johansson B. Fusion of the MORF and CBP genes in acute myeloid leukemia with the t(10;16)(q22;p13). Hum Mol Genet. 2001 Feb 15;10(4):395-404. Citation on PubMedSimpson MA, Deshpande C, Dafou D, Vissers LE, Woollard WJ, Holder SE, Gillessen-Kaesbach G, Derks R, White SM, Cohen-Snuijf R, Kant SG, Hoefsloot LH, Reardon W, Brunner HG, Bongers EM, Trembath RC. De novo mutations of the gene encoding the histone acetyltransferase KAT6B cause Genitopatellar syndrome. Am J Hum Genet. 2012 Feb 10;90(2):290-4. doi: 10.1016/j.ajhg.2011.11.024. Epub 2012 Jan 19. Citation on PubMed or Free article on PubMed CentralYang XJ, Ullah M. MOZ and MORF, two large MYSTic HATs in normal and cancer stem cells. Oncogene. 2007 Aug 13;26(37):5408-19. Review. Citation on PubMed \n",
            "DUSP13\t\n",
            "KCNMA1\t   From NCBI Gene:MaxiK channels are large conductance, voltage and calcium-sensitive potassium channels which are fundamental to the control of smooth muscle tone and neuronal excitability. MaxiK channels can be formed by 2 subunits: the pore-forming alpha subunit, which is the product of this gene, and the modulatory beta subunit. Intracellular calcium regulates the physical association between the alpha and beta subunits. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]From UniProt:Potassium channel activated by both membrane depolarization or increase in cytosolic Ca(2+) that mediates export of K(+). It is also activated by the concentration of cytosolic Mg(2+). Its activation dampens the excitatory events that elevate the cytosolic Ca(2+) concentration and/or depolarize the cell membrane. It therefore contributes to repolarization of the membrane potential. Plays a key role in controlling excitability in a number of systems, such as regulation of the contraction of smooth muscle, the tuning of hair cells in the cochlea, regulation of transmitter release, and innate immunity. In smooth muscles, its activation by high level of Ca(2+), caused by ryanodine receptors in the sarcoplasmic reticulum, regulates the membrane potential. In cochlea cells, its number and kinetic properties partly determine the characteristic frequency of each hair cell and thereby helps to establish a tonotopic map. Kinetics of KCNMA1 channels are determined by alternative splicing, phosphorylation status and its combination with modulating beta subunits. Highly sensitive to both iberiotoxin (IbTx) and charybdotoxin (CTX).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Generalized epilepsy and paroxysmal dyskinesiaCEREBELLAR ATROPHY, DEVELOPMENTAL DELAY, AND SEIZURESFrom UniProt:Paroxysmal nonkinesigenic dyskinesia, 3, with or without generalized epilepsy (PNKD3): An autosomal dominant neurologic disorder characterized by absence seizures, generalized tonic-clonic seizures, paroxysmal nonkinesigenic dyskinesia and involuntary dystonic or choreiform movements. Onset is usually in childhood. Patients may have seizures only, dyskinesia only, or both. [MIM:609446]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q22.3, which is the long (q) arm of chromosome 10 at position 22.3Molecular Location: base pairs 76,869,601 to 77,637,969 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA205K10.1BKTMCADEDShSloKCa1.1MaxiKmSLO1PNKD3SAKCASLOSLO-ALPHASLO1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KCNMA1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CEREBELLAR ATROPHY, DEVELOPMENTAL DELAY, AND SEIZURESOMIM: PAROXYSMAL NONKINESIGENIC DYSKINESIA, 3, WITH OR WITHOUT GENERALIZED EPILEPSYOMIM: POTASSIUM CHANNEL, CALCIUM-ACTIVATED, LARGE CONDUCTANCE, SUBFAMILY M, ALPHA MEMBER 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KCNMA1  OMIM: CEREBELLAR ATROPHY, DEVELOPMENTAL DELAY, AND SEIZURESOMIM: PAROXYSMAL NONKINESIGENIC DYSKINESIA, 3, WITH OR WITHOUT GENERALIZED EPILEPSYOMIM: POTASSIUM CHANNEL, CALCIUM-ACTIVATED, LARGE CONDUCTANCE, SUBFAMILY M, ALPHA MEMBER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SFTPA2\t   From NCBI Gene:This gene is one of several genes encoding pulmonary-surfactant associated proteins (SFTPA) located on chromosome 10. Mutations in this gene and a highly similar gene located nearby, which affect the highly conserved carbohydrate recognition domain, are associated with idiopathic pulmonary fibrosis. The current version of the assembly displays only a single centromeric SFTPA gene pair rather than the two gene pairs shown in the previous assembly which were thought to have resulted from a duplication. [provided by RefSeq, Sep 2009]From UniProt:In presence of calcium ions, it binds to surfactant phospholipids and contributes to lower the surface tension at the air-liquid interface in the alveoli of the mammalian lung and is essential for normal respiration.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Idiopathic pulmonary fibrosisFrom NCBI Gene:Idiopathic fibrosing alveolitis, chronic formFrom UniProt:Pulmonary fibrosis, idiopathic (IPF): A lung disease characterized by shortness of breath, radiographically evident diffuse pulmonary infiltrates, and varying degrees of inflammation and fibrosis on biopsy. In some cases, the disorder can be rapidly progressive and characterized by sequential acute lung injury with subsequent scarring and end-stage lung disease. [MIM:178500]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q22.3, which is the long (q) arm of chromosome 10 at position 22.3Molecular Location: base pairs 79,555,852 to 79,560,461 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        COLEC5PSAPPSP-APSPASFTP1SFTPA2BSP-2ASP-ASPA2SPAII     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SFTPA2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PULMONARY FIBROSIS, IDIOPATHICOMIM: SURFACTANT, PULMONARY-ASSOCIATED PROTEIN A2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SFTPA2  OMIM: PULMONARY FIBROSIS, IDIOPATHICOMIM: SURFACTANT, PULMONARY-ASSOCIATED PROTEIN A2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMEM254\t\n",
            "ANXA11\t   From NCBI Gene:This gene encodes a member of the annexin family, a group of calcium-dependent phospholipid-binding proteins. Annexins have unique N-terminal domains and conserved C-terminal domains, which contain calcium-dependent phospholipid-binding sites. The encoded protein is a 56-kD antigen recognized by sera from patients with various autoimmune diseases. Several transcript variants encoding two different isoforms have been identified. [provided by RefSeq, Dec 2015]From UniProt:Binds specifically to calcyclin in a calcium-dependent manner (By similarity). Required for midbody formation and completion of the terminal phase of cytokinesis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:AMYOTROPHIC LATERAL SCLEROSIS 23From UniProt:Amyotrophic lateral sclerosis 23 (ALS23): A form of amyotrophic lateral sclerosis, a neurodegenerative disorder affecting upper motor neurons in the brain and lower motor neurons in the brain stem and spinal cord, resulting in fatal paralysis. Sensory abnormalities are absent. The pathologic hallmarks of the disease include pallor of the corticospinal tract due to loss of motor neurons, presence of ubiquitin-positive inclusions within surviving motor neurons, and deposition of pathologic aggregates. The etiology of amyotrophic lateral sclerosis is likely to be multifactorial, involving both genetic and environmental factors. The disease is inherited in 5-10% of the cases. ALS23 is an autosomal dominant form with incomplete penetrance. [MIM:617839]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10q22.3, which is the long (q) arm of chromosome 10 at position 22.3Molecular Location: base pairs 80,153,953 to 80,205,677 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ALS23ANX11CAP-50CAP50     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ANXA11      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: AMYOTROPHIC LATERAL SCLEROSIS 23OMIM: ANNEXIN A11      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ANXA11  OMIM: AMYOTROPHIC LATERAL SCLEROSIS 23OMIM: ANNEXIN A11  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LRIT1\t\n",
            "KIF20B\t\n",
            "TUBB8\t   From NCBI Gene:The protein encoded by this gene represents the primary beta-tubulin subunit of oocytes and the early embryo. Defects in this gene, which is primate-specific, are a cause of oocyte maturation defect 2 and infertility. [provided by RefSeq, Mar 2016]From UniProt:Tubulin is the major constituent of microtubules. It binds two moles of GTP, one at an exchangeable site on the beta chain and one at a non-exchangeable site on the alpha chain (By similarity). TUBB8 has a key role in meiotic spindle assembly and oocyte maturation (PubMed:26789871).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Oocyte maturation defect 2From UniProt:Oocyte maturation defect 2 (OOMD2): An autosomal dominant infertility disorder caused by defective oocyte maturation. Oocytes are arrested at metaphase I, and have an abnormal or no detectable spindle on polarization microscopy. [MIM:616780]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 10p15.3, which is the short (p) arm of chromosome 10 at position 15.3Molecular Location: base pairs 46,437 to 49,504 on chromosome 10 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA631M21.2OOMDOOMD2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TUBB8      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: OOCYTE MATURATION DEFECT 2OMIM: TUBULIN, BETA-8      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TUBB8  OMIM: OOCYTE MATURATION DEFECT 2OMIM: TUBULIN, BETA-8  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TNKS2\t\n",
            "SORBS1\t\n",
            "ZNF518A\t\n",
            "FRAT2\t\n",
            "ANKRD2\t\n",
            "AVPI1\t\n",
            "TRPC6\t   From NCBI Gene:The protein encoded by this gene forms a receptor-activated calcium channel in the cell membrane. The channel is activated by diacylglycerol and is thought to be under the control of a phosphatidylinositol second messenger system. Activation of this channel occurs independently of protein kinase C and is not triggered by low levels of intracellular calcium. Defects in this gene are a cause of focal segmental glomerulosclerosis 2 (FSGS2). [provided by RefSeq, Mar 2009]From UniProt:Thought to form a receptor-activated non-selective calcium permeant cation channel (PubMed:19936226, PubMed:23291369). Probably is operated by a phosphatidylinositol second messenger system activated by receptor tyrosine kinases or G-protein coupled receptors. Activated by diacylglycerol (DAG) in a membrane-delimited fashion, independently of protein kinase C (PubMed:26892346). Seems not to be activated by intracellular calcium store depletion.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Focal segmental glomerulosclerosis 2From UniProt:Focal segmental glomerulosclerosis 2 (FSGS2): A renal pathology defined by the presence of segmental sclerosis in glomeruli and resulting in proteinuria, reduced glomerular filtration rate and progressive decline in renal function. Renal insufficiency often progresses to end-stage renal disease, a highly morbid state requiring either dialysis therapy or kidney transplantation. [MIM:603965]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q22.1, which is the long (q) arm of chromosome 11 at position 22.1Molecular Location: base pairs 101,451,470 to 101,583,928 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FSGS2TRP6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TRPC6      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 2OMIM: TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY C, MEMBER 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TRPC6  OMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 2OMIM: TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY C, MEMBER 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MUC6\t\n",
            "MMP12\t\n",
            "CASP12\t\n",
            "MRVI1\t   From NCBI Gene:This gene is similar to a putative mouse tumor suppressor gene (Mrvi1) that is frequently disrupted by mouse AIDS-related virus (MRV). The encoded protein, which is found in the membrane of the endoplasmic reticulum, is similar to Jaw1, a lymphoid-restricted protein whose expression is down-regulated during lymphoid differentiation. This protein is a substrate of cGMP-dependent kinase-1 (PKG1) that can function as a regulator of IP3-induced calcium release. Studies in mouse suggest that MRV integration at Mrvi1 induces myeloid leukemia by altering the expression of a gene important for myeloid cell growth and/or differentiation, and thus this gene may function as a myeloid leukemia tumor suppressor gene. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene, and alternative translation start sites, including a non-AUG (CUG) start site, are used. [provided by RefSeq, May 2011]From UniProt:Plays a role as NO/PRKG1-dependent regulator of IP3-induced calcium release; its phosphorylation by PRKG1 inhibits bradykinin and IP3-induced calcium release from intracellular stores. Recruits PRKG1 to the endoplasmic reticulum and may mediate the assembly of PRKG1 and ITPR1 in a macrocomplex. Involved in PRKG1 signaling cascade leading to inhibition of platelet activation and aggregation. Mediates also NO-dependent inhibition of calcium signaling in gastrointestinal smooth muscle contributing to NO-dependent relaxation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Migraine     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p15.4, which is the short (p) arm of chromosome 11 at position 15.4Molecular Location: base pairs 10,573,091 to 10,693,988 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IRAGJAW1L     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MRVI1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MURINE RETROVIRUS INTEGRATION SITE 1, HOMOLOG OF      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MRVI1  OMIM: MURINE RETROVIRUS INTEGRATION SITE 1, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NPAT\t\n",
            "ATM\t   The ATM gene provides instructions for making a protein that is located primarily in the nucleus of cells, where it helps control the rate at which cells grow and divide. This protein also plays an important role in the normal development and activity of several body systems, including the nervous system and the immune system.  Additionally, the ATM protein assists cells in recognizing damaged or broken DNA strands. DNA can be damaged by agents such as toxic chemicals or radiation. Breaks in DNA strands also occur naturally when chromosomes exchange genetic material during cell division. The ATM protein coordinates DNA repair by activating enzymes that fix the broken strands. Efficient repair of damaged DNA strands helps maintain the stability of the cell's genetic information.Because of its central role in cell division and DNA repair, the ATM protein is of great interest in cancer research.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ataxia-telangiectasia     Researchers have identified several hundred mutations in the ATM gene that cause ataxia-telangiectasia. People with this disorder have mutations in both copies of the ATM gene in each cell. Most of these mutations disrupt protein production, resulting in an abnormally small, nonfunctional version of the ATM protein. Cells without any functional ATM protein are hypersensitive to radiation and do not respond normally to DNA damage. Instead of activating DNA repair, the defective ATM protein allows mutations to accumulate in other genes, which may cause cells to grow and divide in an uncontrolled way.  This kind of unregulated cell growth can lead to the formation of cancerous tumors.  In addition, ATM mutations can allow cells to die inappropriately, particularly affecting cells in a part of the brain involved in coordinating movements (the cerebellum). This loss of brain cells causes the movement problems characteristic of ataxia-telangiectasia. More About This Health Condition      Breast cancer     Genetics Home Reference provides information about breast cancer. More About This Health Condition      Melanoma     Genetics Home Reference provides information about melanoma. More About This Health Condition      Other cancers     Research suggests that people who carry one mutated copy of the ATM gene in each cell may have an increased risk of developing several other types of cancer. In particular, some studies have shown that cancers of the breast, stomach, bladder, pancreas, lung, and ovaries occur more frequently in ATM mutation carriers than in people who do not carry these mutations.  The results of similar studies, however, have been conflicting.  Additional research is needed to clarify which other types of cancer, if any, are associated with ATM mutations.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Researchers have identified several hundred mutations in the ATM gene that cause ataxia-telangiectasia. People with this disorder have mutations in both copies of the ATM gene in each cell. Most of these mutations disrupt protein production, resulting in an abnormally small, nonfunctional version of the ATM protein. Cells without any functional ATM protein are hypersensitive to radiation and do not respond normally to DNA damage. Instead of activating DNA repair, the defective ATM protein allows mutations to accumulate in other genes, which may cause cells to grow and divide in an uncontrolled way.  This kind of unregulated cell growth can lead to the formation of cancerous tumors.  In addition, ATM mutations can allow cells to die inappropriately, particularly affecting cells in a part of the brain involved in coordinating movements (the cerebellum). This loss of brain cells causes the movement problems characteristic of ataxia-telangiectasia. More About This Health Condition  Genetics Home Reference provides information about breast cancer. More About This Health Condition  Genetics Home Reference provides information about melanoma. More About This Health Condition  Research suggests that people who carry one mutated copy of the ATM gene in each cell may have an increased risk of developing several other types of cancer. In particular, some studies have shown that cancers of the breast, stomach, bladder, pancreas, lung, and ovaries occur more frequently in ATM mutation carriers than in people who do not carry these mutations.  The results of similar studies, however, have been conflicting.  Additional research is needed to clarify which other types of cancer, if any, are associated with ATM mutations.    Cytogenetic Location: 11q22.3, which is the long (q) arm of chromosome 11 at position 22.3Molecular Location: base pairs 108,222,484 to 108,369,099 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AT mutatedAT proteinAT1ATAataxia telangiectasia mutatedataxia telangiectasia mutated (includes complementation groups A, C and D)ataxia telangiectasia mutated proteinATCATDATDCATEATM_HUMANhuman phosphatidylinositol 3-kinase homologserine-protein kinase ATMTEL1TELO1     Related Information   How are genetic conditions and genes named?          Educational Resources (4 links)     Cancer Medicine (sixth edition, 2003): Ataxia-TelangiectaisaCancer Medicine (sixth edition, 2003): Cell Cycle Checkpoints, Integration of Signalling, and DNA RepairEurekah Bioscience Collection: Replication-Independent Double-Strand Breaks (DSBs)National Cancer Institute: Genetics of Breast and Ovarian Cancer (PDQ)      Clinical Information from GeneReviews (1 link)     Ataxia-Telangiectasia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATM      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATAXIA-TELANGIECTASIA MUTATED GENE      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics Web: ATMClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Cancer Medicine (sixth edition, 2003): Ataxia-TelangiectaisaCancer Medicine (sixth edition, 2003): Cell Cycle Checkpoints, Integration of Signalling, and DNA RepairEurekah Bioscience Collection: Replication-Independent Double-Strand Breaks (DSBs)National Cancer Institute: Genetics of Breast and Ovarian Cancer (PDQ)  Ataxia-Telangiectasia  Tests of ATM  PubMed  ATAXIA-TELANGIECTASIA MUTATED GENE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics Web: ATMClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Ahmed M, Rahman N. ATM and breast cancer susceptibility. Oncogene. 2006 Sep 25;25(43):5906-11. Review. Citation on PubMedBradbury JM, Jackson SP. ATM and ATR. Curr Biol. 2003 Jun 17;13(12):R468. Citation on PubMedGeoffroy-Perez B, Janin N, Ossian K, Laug A, Croquette MF, Griscelli C, Debr M, Bressac-de-Paillerets B, Aurias A, Stoppa-Lyonnet D, Andrieu N. Cancer risk in heterozygotes for ataxia-telangiectasia. Int J Cancer. 2001 Jul 15;93(2):288-93. Citation on PubMedGumy-Pause F, Wacker P, Sappino AP. ATM gene and lymphoid malignancies. Leukemia. 2004 Feb;18(2):238-42. Review. Citation on PubMedHall J. The Ataxia-telangiectasia mutated gene and breast cancer: gene expression profiles and sequence variants. Cancer Lett. 2005 Sep 28;227(2):105-14. Epub 2005 Jan 8. Review. Citation on PubMedKim JH, Kim H, Lee KY, Choe KH, Ryu JS, Yoon HI, Sung SW, Yoo KY, Hong YC. Genetic polymorphisms of ataxia telangiectasia mutated affect lung cancer risk. Hum Mol Genet. 2006 Apr 1;15(7):1181-6. Epub 2006 Feb 23. Citation on PubMedKitagawa R, Kastan MB. The ATM-dependent DNA damage signaling pathway. Cold Spring Harb Symp Quant Biol. 2005;70:99-109. Review. Citation on PubMedKurz EU, Lees-Miller SP. DNA damage-induced activation of ATM and ATM-dependent signaling pathways. DNA Repair (Amst). 2004 Aug-Sep;3(8-9):889-900. Review. Citation on PubMedLavin MF, Birrell G, Chen P, Kozlov S, Scott S, Gueven N. ATM signaling and genomic stability in response to DNA damage. Mutat Res. 2005 Jan 6;569(1-2):123-32. Review. Citation on PubMedLavin MF, Kozlov S. DNA damage-induced signalling in ataxia-telangiectasia and related syndromes. Radiother Oncol. 2007 Jun;83(3):231-7. Epub 2007 May 23. Review. Citation on PubMedMcKinnon PJ. ATM and ataxia telangiectasia. EMBO Rep. 2004 Aug;5(8):772-6. Review. Citation on PubMed or Free article on PubMed CentralMotoyama N, Naka K. DNA damage tumor suppressor genes and genomic instability. Curr Opin Genet Dev. 2004 Feb;14(1):11-6. Review. Citation on PubMedProkopcova J, Kleibl Z, Banwell CM, Pohlreich P. The role of ATM in breast cancer development. Breast Cancer Res Treat. 2007 Aug;104(2):121-8. Epub 2006 Oct 24. Review. Citation on PubMedRenwick A, Thompson D, Seal S, Kelly P, Chagtai T, Ahmed M, North B, Jayatilake H, Barfoot R, Spanova K, McGuffog L, Evans DG, Eccles D; Breast Cancer Susceptibility Collaboration (UK), Easton DF, Stratton MR, Rahman N. ATM mutations that cause ataxia-telangiectasia are breast cancer susceptibility alleles. Nat Genet. 2006 Aug;38(8):873-5. Epub 2006 Jul 9. Citation on PubMedShiloh Y. ATM and related protein kinases: safeguarding genome integrity. Nat Rev Cancer. 2003 Mar;3(3):155-68. Review. Citation on PubMedThompson D, Duedal S, Kirner J, McGuffog L, Last J, Reiman A, Byrd P, Taylor M, Easton DF. Cancer risks and mortality in heterozygous ATM mutation carriers. J Natl Cancer Inst. 2005 Jun 1;97(11):813-22. Citation on PubMedYang J, Yu Y, Hamrick HE, Duerksen-Hughes PJ. ATM, ATR and DNA-PK: initiators of the cellular genotoxic stress responses. Carcinogenesis. 2003 Oct;24(10):1571-80. Epub 2003 Aug 14. Review. Citation on PubMed \n",
            "DIXDC1\t\n",
            "PLET1\t\n",
            "CSNK2A3\t\n",
            "CADM1\t\n",
            "ZPR1\t\n",
            "APOA5\t   From NCBI Gene:The protein encoded by this gene is an apolipoprotein that plays an important role in regulating the plasma triglyceride levels, a major risk factor for coronary artery disease. It is a component of high density lipoprotein and is highly similar to a rat protein that is upregulated in response to liver injury. Mutations in this gene have been associated with hypertriglyceridemia and hyperlipoproteinemia type 5. This gene is located proximal to the apolipoprotein gene cluster on chromosome 11q23. Alternatively spliced transcript variants encoding the same protein have been identified. [provided by RefSeq, Oct 2009]From UniProt:Minor apolipoprotein mainly associated with HDL and to a lesser extent with VLDL. May also be associated with chylomicrons. Important determinant of plasma triglyceride (TG) levels by both being a potent stimulator of apo-CII lipoprotein lipase (LPL) TG hydrolysis and an inhibitor of the hepatic VLDL-TG production rate (without affecting the VLDL-apoB production rate) (By similarity). Activates poorly lecithin:cholesterol acyltransferase (LCAT) and does not enhance efflux of cholesterol from macrophages.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Familial hypertriglyceridemiaFamilial type 5 hyperlipoproteinemiaFrom UniProt:Hypertriglyceridemia, familial (FHTR): A common inherited disorder in which the concentration of very low density lipoprotein (VLDL) is elevated in the plasma. This leads to increased risk of heart disease, obesity, and pancreatitis. [MIM:145750]Hyperlipoproteinemia 5 (HLPP5): Characterized by increased amounts of chylomicrons and very low density lipoprotein (VLDL) and decreased low density lipoprotein (LDL) and high density lipoprotein (HDL) in the plasma after a fast. Numerous conditions cause this phenotype, including insulin-dependent diabetes mellitus, contraceptive steroids, alcohol abuse, and glycogen storage disease type 1A (GSD1A). [MIM:144650]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q23.3, which is the long (q) arm of chromosome 11 at position 23.3Molecular Location: base pairs 116,789,370 to 116,792,420 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        APOAVRAP3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of APOA5      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: APOLIPOPROTEIN A-VOMIM: HYPERLIPOPROTEINEMIA, TYPE VOMIM: HYPERTRIGLYCERIDEMIA, FAMILIAL      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of APOA5  OMIM: APOLIPOPROTEIN A-VOMIM: HYPERLIPOPROTEINEMIA, TYPE VOMIM: HYPERTRIGLYCERIDEMIA, FAMILIAL  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMPRSS13\t\n",
            "TMPRSS4\t\n",
            "JAML\t\n",
            "TTC36\t\n",
            "TREH\t   From NCBI Gene:This gene encodes an enzyme that hydrolyses trehalose, a disaccharide formed from two glucose molecules found mainly in fungi, plants, and insects. A partial duplication of this gene is located adjacent to this locus on chromosome 11. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2014]From UniProt:Intestinal trehalase is probably involved in the hydrolysis of ingested trehalose.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:alpha, alpha-Trehalase deficiencyFrom UniProt:Trehalase deficiency (TREHD): An autosomal recessive condition characterized by the inability to digest trehalose, a disaccharide found in mushrooms, products containing baker's yeast, and dried food. Individuals with trehalase deficiency suffer from abdominal pain, increased rectal flatulence, and diarrhea due to osmotic water flow into the colon. [MIM:612119]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q23.3, which is the long (q) arm of chromosome 11 at position 23.3Molecular Location: base pairs 118,658,232 to 118,679,672 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        TRETREATREHD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TREH      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: TREHALASEOMIM: TREHALASE DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TREH  OMIM: TREHALASEOMIM: TREHALASE DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "VPS11\t   From NCBI Gene:Vesicle mediated protein sorting plays an important role in segregation of intracellular molecules into distinct organelles. Genetic studies in yeast have identified more than 40 vacuolar protein sorting (VPS) genes involved in vesicle transport to vacuoles. This gene encodes the human homolog of yeast class C Vps11 protein. The mammalian class C Vps proteins are predominantly associated with late endosomes/lysosomes, and like their yeast counterparts, may mediate vesicle trafficking steps in the endosome/lysosome pathway. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2014]From UniProt:Plays a role in vesicle-mediated protein trafficking to lysosomal compartments including the endocytic membrane transport and autophagic pathways. Believed to act as a core component of the putative HOPS and CORVET endosomal tethering complexes which are proposed to be involved in the Rab5-to-Rab7 endosome conversion probably implicating MON1A/B, and via binding SNAREs and SNARE complexes to mediate tethering and docking events during SNARE-mediated membrane fusion. The HOPS complex is proposed to be recruited to Rab7 on the late endosomal membrane and to regulate late endocytic, phagocytic and autophagic traffic towards lysosomes. The CORVET complex is proposed to function as a Rab5 effector to mediate early endosome fusion probably in specific endosome subpopulations (PubMed:11382755, PubMed:23351085, PubMed:24554770, PubMed:25266290, PubMed:25783203). Required for fusion of endosomes and autophagosomes with lysosomes (PubMed:25783203). Involved in cargo transport from early to late endosomes and required for the transition from early to late endosomes (PubMed:21148287). Involved in the retrograde Shiga toxin transport (PubMed:23593995).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Leukodystrophy, hypomyelinating, 12From UniProt:Leukodystrophy, hypomyelinating, 12 (HLD12): An autosomal recessive neurologic disorder characterized by developmental delay, spasticity, truncal hypotonia, acquired microcephaly, intellectual disability with variable seizure disorder, accompanied by thin corpus callosum, paucity of white matter and delayed myelination. [MIM:616683]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q23.3, which is the long (q) arm of chromosome 11 at position 23.3Molecular Location: base pairs 119,067,752 to 119,081,978 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        END1HLD12hVPS11PEP5RNF108     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of VPS11      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LEUKODYSTROPHY, HYPOMYELINATING, 12OMIM: VACUOLAR PROTEIN SORTING 11, YEAST, HOMOLOG OF      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of VPS11  OMIM: LEUKODYSTROPHY, HYPOMYELINATING, 12OMIM: VACUOLAR PROTEIN SORTING 11, YEAST, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MUC5AC\t\n",
            "MICALCL\t\n",
            "OR6M1\t\n",
            "OR8D4\t\n",
            "OR10G4\t\n",
            "OR8G5\t\n",
            "OR8B2\t\n",
            "OR8B4\t\n",
            "OR8A1\t\n",
            "MUC5B\t   From NCBI Gene:This gene encodes a member of the mucin family of proteins, which are highly glycosylated macromolecular components of mucus secretions. This family member is the major gel-forming mucin in mucus. It is a major contributor to the lubricating and viscoelastic properties of whole saliva, normal lung mucus and cervical mucus. This gene has been found to be up-regulated in some human diseases, including sinus mucosa of chronic rhinosinusitis (CRS), CRS with nasal polyposis, chronic obstructive pulmonary disease (COPD) and H. pylori-associated gastric disease, and it may be involved in the pathogenesis of these diseases. [provided by RefSeq, Jul 2010]From UniProt:Gel-forming mucin that is thought to contribute to the lubricating and viscoelastic properties of whole saliva and cervical mucus.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Idiopathic pulmonary fibrosisFrom NCBI Gene:Idiopathic fibrosing alveolitis, chronic formFrom UniProt:Pulmonary fibrosis, idiopathic (IPF): A lung disease characterized by shortness of breath, radiographically evident diffuse pulmonary infiltrates, and varying degrees of inflammation and fibrosis on biopsy. In some cases, the disorder can be rapidly progressive and characterized by sequential acute lung injury with subsequent scarring and end-stage lung disease. [MIM:178500]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p15.5, which is the short (p) arm of chromosome 11 at position 15.5Molecular Location: base pairs 1,223,065 to 1,262,176 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MG1MUC-5BMUC5MUC9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MUC5B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MUCIN 5, SUBTYPE B, TRACHEOBRONCHIALOMIM: PULMONARY FIBROSIS, IDIOPATHIC      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MUC5B  OMIM: MUCIN 5, SUBTYPE B, TRACHEOBRONCHIALOMIM: PULMONARY FIBROSIS, IDIOPATHIC  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PKNOX2\t\n",
            "EI24\t   From NCBI Gene:This gene encodes a putative tumor suppressor and has higher expression in p53-expressing cells than in control cells and is an immediate-early induction target of p53-mediated apoptosis. The encoded protein may suppress cell growth by inducing apoptotic cell death through the caspase 9 and mitochondrial pathways. This gene is located on human chromosome 11q24, a region frequently altered in cancers. Alternative splicing results in multiple transcript variants. Pseudogenes of this gene have been defined on chromosomes 1, 3, 7, and 8. [provided by RefSeq, Feb 2014]From UniProt:Acts as a negative growth regulator via p53-mediated apoptosis pathway. Regulates formation of degradative autolysosomes during autophagy (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:EI24 is on a chromosomal region frequently deleted in solid tumors, and it is thought to play a role in breast and cervical cancer. Particularly, expression analysis of EI24 in cancerous tissues shows that EI24 loss is associated with tumor invasiveness.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q24.2, which is the long (q) arm of chromosome 11 at position 24.2Molecular Location: base pairs 125,569,216 to 125,584,689 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        EPG4PIG8TP53I8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of EI24      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: ETOPOSIDE-INDUCED 1.4 mRNA      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of EI24  OMIM: ETOPOSIDE-INDUCED 1.4 mRNA  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PATE1\t\n",
            "PATE4\t\n",
            "C11orf45\t\n",
            "TP53AIP1\t\n",
            "ADAMTS8\t\n",
            "SNX19\t\n",
            "SPON1\t\n",
            "KRTAP5-1\t\n",
            "KRTAP5-2\t\n",
            "KRTAP5-5\t\n",
            "PLEKHA7\t\n",
            "OTOG\t   From NCBI Gene:The protein encoded by this gene is a component of the acellular membranes of the inner ear. Disruption of the orthologous mouse gene shows that it plays a role in auditory and vestibular functions. It is involved in fibrillar network organization, the anchoring of otoconial membranes and cupulae to the neuroepithelia, and likely in sound stimulation resistance. Mutations in this gene cause autosomal recessive nonsyndromic deafness, type 18B. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, May 2014]From UniProt:Glycoprotein specific to acellular membranes of the inner ear. May be required for the anchoring of the otoconial membranes and cupulae to the underlying neuroepithelia in the vestibule. May be involved in the organization and/or stabilization of the fibrillar network that compose the tectorial membrane in the cochlea. May play a role in mechanotransduction processes (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 18bFrom UniProt:Deafness, autosomal recessive, 18B (DFNB18B): A form of non-syndromic deafness characterized by a moderate hearing impairment, which can be associated with vestibular dysfunction, and a flat to shallow \"U\" or slightly downsloping shaped audiograms. [MIM:614945]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p15.1, which is the short (p) arm of chromosome 11 at position 15.1Molecular Location: base pairs 17,547,373 to 17,645,944 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNB18BMLEMPOTGN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of OTOG      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 18BOMIM: OTOGELIN      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of OTOG  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 18BOMIM: OTOGELIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MRGPRX2\t\n",
            "SCGB1C1\t\n",
            "ODF3\t\n",
            "RIC8A\t\n",
            "TH\t   The TH gene provides instructions for making the enzyme tyrosine hydroxylase, which is important for normal functioning of the nervous system. Tyrosine hydroxylase takes part in the first step of the pathway that produces a group of hormones called catecholamines. This enzyme helps convert the protein building block (amino acid) tyrosine to a catecholamine called dopamine. Dopamine is also known as a neurotransmitter because it transmits signals between nerve cells in the brain to help control physical movement and emotional behavior. Other catecholamines called norepinephrine and epinephrine are produced from dopamine. Norepinephrine and epinephrine are involved in the autonomic nervous system, which controls involuntary body processes such as the regulation of blood pressure and body temperature.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Dopa-responsive dystonia     More than two dozen mutations in the TH gene have been found to cause dopa-responsive dystonia. This condition is characterized by a pattern of involuntary muscle contractions (dystonia), tremors, and other uncontrolled movements and usually responds to treatment with a medication called L-Dopa. Most TH gene mutations that cause this condition change single protein building blocks (amino acids) in the tyrosine hydroxylase enzyme, resulting in a decrease in functional enzyme. A reduction in normal tyrosine hydroxylase enzyme leads to a decrease in the production of dopamine, which causes the movement problems characteristic of dopa-responsive dystonia. The amount of functional enzyme that is produced is associated with the severity of the signs and symptoms. Less functional enzyme leads to more severe symptoms. More About This Health Condition      Tyrosine hydroxylase deficiency     More than 20 mutations in the TH gene have been identified in people with tyrosine hydroxylase (TH) deficiency. These mutations result in reduced activity of the tyrosine hydroxylase enzyme. As a result, the body produces less dopamine, norepinephrine, and epinephrine. These catecholamines are necessary for normal nervous system function, and changes in their levels contribute to the abnormal movements, nervous system dysfunction, and other neurological problems seen in people with TH deficiency.Dopa-responsive dystonia is sometimes considered a mild form of tyrosine hydroxylase deficiency. It is uncertain whether they are two separate disorders or part of the same disease spectrum. More About This Health Condition      Other disorders     Certain common TH variations (polymorphisms) modify catecholamine production, which affects the risk of developing conditions associated with regulation of the autonomic nervous system. These TH gene polymorphisms affect the extent to which blood pressure increases with stress and may increase the risk of high blood pressure (hypertension).One TH gene polymorphism has been associated with sudden infant death syndrome (SIDS). SIDS is a major cause of death in babies younger than 1 year. It is characterized by sudden and unexplained death, usually during sleep. The polymorphism, called allele *9.3, is the most common TH gene polymorphism among people of European descent and has been identified in a larger percentage of babies who die from SIDS than in other babies. This version of the gene may affect the regulation of breathing or awakening in infants.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than two dozen mutations in the TH gene have been found to cause dopa-responsive dystonia. This condition is characterized by a pattern of involuntary muscle contractions (dystonia), tremors, and other uncontrolled movements and usually responds to treatment with a medication called L-Dopa. Most TH gene mutations that cause this condition change single protein building blocks (amino acids) in the tyrosine hydroxylase enzyme, resulting in a decrease in functional enzyme. A reduction in normal tyrosine hydroxylase enzyme leads to a decrease in the production of dopamine, which causes the movement problems characteristic of dopa-responsive dystonia. The amount of functional enzyme that is produced is associated with the severity of the signs and symptoms. Less functional enzyme leads to more severe symptoms. More About This Health Condition  More than 20 mutations in the TH gene have been identified in people with tyrosine hydroxylase (TH) deficiency. These mutations result in reduced activity of the tyrosine hydroxylase enzyme. As a result, the body produces less dopamine, norepinephrine, and epinephrine. These catecholamines are necessary for normal nervous system function, and changes in their levels contribute to the abnormal movements, nervous system dysfunction, and other neurological problems seen in people with TH deficiency.Dopa-responsive dystonia is sometimes considered a mild form of tyrosine hydroxylase deficiency. It is uncertain whether they are two separate disorders or part of the same disease spectrum. More About This Health Condition  Certain common TH variations (polymorphisms) modify catecholamine production, which affects the risk of developing conditions associated with regulation of the autonomic nervous system. These TH gene polymorphisms affect the extent to which blood pressure increases with stress and may increase the risk of high blood pressure (hypertension).One TH gene polymorphism has been associated with sudden infant death syndrome (SIDS). SIDS is a major cause of death in babies younger than 1 year. It is characterized by sudden and unexplained death, usually during sleep. The polymorphism, called allele *9.3, is the most common TH gene polymorphism among people of European descent and has been identified in a larger percentage of babies who die from SIDS than in other babies. This version of the gene may affect the regulation of breathing or awakening in infants.    Cytogenetic Location: 11p15.5, which is the short (p) arm of chromosome 11 at position 15.5Molecular Location: base pairs 2,163,929 to 2,174,081 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DYT5bTY3H_HUMANTYHtyrosine 3-monooxygenase     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Basic Neurochemistry (6th edition, 1999): Tyrosine hydroxylase is the rate-limiting enzyme for the biosynthesis of catecholaminesNeuroscience (second edition, 2001): Regulation of tyrosine hydroxylase by protein phosphorylation (image)      Clinical Information from GeneReviews (1 link)     Tyrosine Hydroxylase Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TH      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TYROSINE HYDROXYLASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (6th edition, 1999): Tyrosine hydroxylase is the rate-limiting enzyme for the biosynthesis of catecholaminesNeuroscience (second edition, 2001): Regulation of tyrosine hydroxylase by protein phosphorylation (image)  Tyrosine Hydroxylase Deficiency  Tests of TH  PubMed  TYROSINE HYDROXYLASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Furukawa Y, Kish S. Tyrosine Hydroxylase Deficiency. 2008 Feb 8 [updated 2017 May 11]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1437/ Citation on PubMedFurukawa Y, Kish SJ, Fahn S. Dopa-responsive dystonia due to mild tyrosine hydroxylase deficiency. Ann Neurol. 2004 Jan;55(1):147-8. Citation on PubMedHoffmann GF, Assmann B, Brutigam C, Dionisi-Vici C, Hussler M, de Klerk JB, Naumann M, Steenbergen-Spanjers GC, Strassburg HM, Wevers RA. Tyrosine hydroxylase deficiency causes progressive encephalopathy and dopa-nonresponsive dystonia. Ann Neurol. 2003;54 Suppl 6:S56-65. Citation on PubMedHunt SC. Tyrosine hydroxylase: another piece of the genetics of hypertension puzzle. Circulation. 2007 Aug 28;116(9):970-2. Citation on PubMedKlintschar M, Reichenpfader B, Saternus KS. A functional polymorphism in the tyrosine hydroxylase gene indicates a role of noradrenalinergic signaling in sudden infant death syndrome. J Pediatr. 2008 Aug;153(2):190-3. doi: 10.1016/j.jpeds.2008.02.032. Epub 2008 Apr 3. Citation on PubMedPearl PL, Capp PK, Novotny EJ, Gibson KM. Inherited disorders of neurotransmitters in children and adults. Clin Biochem. 2005 Dec;38(12):1051-8. Epub 2005 Nov 18. Review. Citation on PubMedPearl PL, Taylor JL, Trzcinski S, Sokohl A. The pediatric neurotransmitter disorders. J Child Neurol. 2007 May;22(5):606-16. Review. Citation on PubMedRao F, Zhang L, Wessel J, Zhang K, Wen G, Kennedy BP, Rana BK, Das M, Rodriguez-Flores JL, Smith DW, Cadman PE, Salem RM, Mahata SK, Schork NJ, Taupenot L, Ziegler MG, O'Connor DT. Adrenergic polymorphism and the human stress response. Ann N Y Acad Sci. 2008 Dec;1148:282-96. doi: 10.1196/annals.1410.085. Citation on PubMed or Free article on PubMed CentralRao F, Zhang L, Wessel J, Zhang K, Wen G, Kennedy BP, Rana BK, Das M, Rodriguez-Flores JL, Smith DW, Cadman PE, Salem RM, Mahata SK, Schork NJ, Taupenot L, Ziegler MG, O'Connor DT. Tyrosine hydroxylase, the rate-limiting enzyme in catecholamine biosynthesis: discovery of common human genetic variants governing transcription, autonomic activity, and blood pressure in vivo. Circulation. 2007 Aug 28;116(9):993-1006. Epub 2007 Aug 13. Citation on PubMedSchiller A, Wevers RA, Steenbergen GC, Blau N, Jung HH. Long-term course of L-dopa-responsive dystonia caused by tyrosine hydroxylase deficiency. Neurology. 2004 Oct 26;63(8):1524-6. Citation on PubMedOMIM: TYROSINE HYDROXYLASEVerbeek MM, Steenbergen-Spanjers GC, Willemsen MA, Hol FA, Smeitink J, Seeger J, Grattan-Smith P, Ryan MM, Hoffmann GF, Donati MA, Blau N, Wevers RA. Mutations in the cyclic adenosine monophosphate response element of the tyrosine hydroxylase gene. Ann Neurol. 2007 Oct;62(4):422-6. Citation on PubMedYeung WL, Wong VC, Chan KY, Hui J, Fung CW, Yau E, Ko CH, Lam CW, Mak CM, Siu S, Low L. Expanding phenotype and clinical analysis of tyrosine hydroxylase deficiency. J Child Neurol. 2011 Feb;26(2):179-87. doi: 10.1177/0883073810377014. Epub 2010 Sep 7. Erratum in: J Child Neurol. 2012 Jun;27(6):829-31. Citation on PubMedZhang L, Rao F, Wessel J, Kennedy BP, Rana BK, Taupenot L, Lillie EO, Cockburn M, Schork NJ, Ziegler MG, O'Connor DT. Functional allelic heterogeneity and pleiotropy of a repeat polymorphism in tyrosine hydroxylase: prediction of catecholamines and response to stress in twins. Physiol Genomics. 2004 Nov 17;19(3):277-91. Epub 2004 Sep 14. Citation on PubMed \n",
            "ANO5\t   The ANO5 gene provides instructions for making a protein called anoctamin-5. While the specific function of this protein is not well understood, it belongs to a family of proteins, called anoctamins, that act as chloride channels. Chloride channels, which transport negatively charged chlorine atoms (chloride ions) in and out of cells, play a key role in a cell's ability to generate and transmit electrical signals. Studies suggest that most anoctamin proteins function as chloride channels that are turned on (activated) in the presence of positively charged calcium atoms (calcium ions); these channels are known as calcium-activated chloride channels. The mechanism for this calcium activation is unclear. Anoctamin proteins are also involved in maintaining the membrane that surrounds cells and repairing the membrane if it gets damaged.The anoctamin-5 protein is most abundant in muscles used for movement (skeletal muscles). For the body to move normally, skeletal muscles must tense (contract) and relax in a coordinated way. The regulation of chloride flow within muscle cells plays a role in controlling muscle contraction and relaxation.The anoctamin-5 protein is also found in other cells including heart (cardiac) muscle cells and bone cells. Studies have suggested that the anoctamin-5 protein may be important for the development of muscle and bone before birth.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Gnathodiaphyseal dysplasia     At least three ANO5 gene mutations have been identified in people with a bone disorder called gnathodiaphyseal dysplasia, which leads to fragile bones, jaw problems, and other skeletal abnormalities. The ANO5 gene mutations that cause gnathodiaphyseal dysplasia change single protein building blocks (amino acids) in the anoctamin-5 protein. It is unclear how these mutations lead to the signs and symptoms of gnathodiaphyseal dysplasia, or why they primarily affect bones while other ANO5 gene mutations cause muscle disorders. Researchers suggest that the mutations may affect the way cells process calcium, an important mineral in bone development and growth. More About This Health Condition      Limb-girdle muscular dystrophy     More than 40 mutations in the ANO5 gene have been identified in people with limb-girdle muscular dystrophy type 2L. Limb-girdle muscular dystrophy is a group of related disorders characterized by muscle weakness and wasting (atrophy), particularly in the shoulders, hips, thighs, and upper arms.The ANO5 gene mutations identified in people with limb-girdle muscular dystrophy type 2L change single amino acids in the anoctamin-5 protein sequence, disrupt how genetic information is pieced together to make a blueprint for producing the protein, or result in a premature stop signal that leads to an abnormally short protein. One of the mutations adds an extra DNA building block (nucleotide) to the ANO5 gene (written as 191dupA) and is believed to be a relatively common cause of limb-girdle muscular dystrophy in people with northern European ancestry. This mutation alters the instructions used to make the anoctamin-5 protein, leading to a premature stop signal that would produce an abnormally short protein. Instead, a cellular error-catching mechanism called nonsense-mediated decay prevents the protein from being produced at all.ANO5 gene mutations that eliminate or impair the role of the anoctamin-5 protein as a chloride channel likely lead to impaired muscle function, resulting in the signs and symptoms of limb-girdle muscular dystrophy. More About This Health Condition      Miyoshi myopathy     At least 10 mutations in the ANO5 gene have been found to cause Miyoshi myopathy. When caused by mutations in this gene, the condition is also known as distal anoctaminopathy. Miyoshi myopathy is a muscle disorder that is characterized by progressive weakness and atrophy of muscles that are away from the center of the body (distal muscles), particularly those in the legs. The ANO5 gene mutations identified in people with Miyoshi myopathy change single amino acids in the anoctamin-5 protein or result in the production of an abnormally short protein that is quickly broken down.These mutations result in the production of little or no anoctamin-5 protein. The effects of the loss of anoctamin-5 are unclear. While chloride is necessary for normal muscle function, it is unknown how a lack of this chloride channel causes the signs and symptoms of Miyoshi myopathy.The 191dupA mutation that can cause limb-girdle muscular dystrophy (described above) is also a common cause of Miyoshi myopathy in individuals of northern European ancestry. It is not known why the 191dupA mutation can result in different patterns of signs and symptoms. Miyoshi myopathy caused by ANO5 gene mutations is likely a variation of limb-girdle muscular dystrophy because it is caused by mutations in the same gene, and in some cases even by the same mutation. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three ANO5 gene mutations have been identified in people with a bone disorder called gnathodiaphyseal dysplasia, which leads to fragile bones, jaw problems, and other skeletal abnormalities. The ANO5 gene mutations that cause gnathodiaphyseal dysplasia change single protein building blocks (amino acids) in the anoctamin-5 protein. It is unclear how these mutations lead to the signs and symptoms of gnathodiaphyseal dysplasia, or why they primarily affect bones while other ANO5 gene mutations cause muscle disorders. Researchers suggest that the mutations may affect the way cells process calcium, an important mineral in bone development and growth. More About This Health Condition  More than 40 mutations in the ANO5 gene have been identified in people with limb-girdle muscular dystrophy type 2L. Limb-girdle muscular dystrophy is a group of related disorders characterized by muscle weakness and wasting (atrophy), particularly in the shoulders, hips, thighs, and upper arms.The ANO5 gene mutations identified in people with limb-girdle muscular dystrophy type 2L change single amino acids in the anoctamin-5 protein sequence, disrupt how genetic information is pieced together to make a blueprint for producing the protein, or result in a premature stop signal that leads to an abnormally short protein. One of the mutations adds an extra DNA building block (nucleotide) to the ANO5 gene (written as 191dupA) and is believed to be a relatively common cause of limb-girdle muscular dystrophy in people with northern European ancestry. This mutation alters the instructions used to make the anoctamin-5 protein, leading to a premature stop signal that would produce an abnormally short protein. Instead, a cellular error-catching mechanism called nonsense-mediated decay prevents the protein from being produced at all.ANO5 gene mutations that eliminate or impair the role of the anoctamin-5 protein as a chloride channel likely lead to impaired muscle function, resulting in the signs and symptoms of limb-girdle muscular dystrophy. More About This Health Condition  At least 10 mutations in the ANO5 gene have been found to cause Miyoshi myopathy. When caused by mutations in this gene, the condition is also known as distal anoctaminopathy. Miyoshi myopathy is a muscle disorder that is characterized by progressive weakness and atrophy of muscles that are away from the center of the body (distal muscles), particularly those in the legs. The ANO5 gene mutations identified in people with Miyoshi myopathy change single amino acids in the anoctamin-5 protein or result in the production of an abnormally short protein that is quickly broken down.These mutations result in the production of little or no anoctamin-5 protein. The effects of the loss of anoctamin-5 are unclear. While chloride is necessary for normal muscle function, it is unknown how a lack of this chloride channel causes the signs and symptoms of Miyoshi myopathy.The 191dupA mutation that can cause limb-girdle muscular dystrophy (described above) is also a common cause of Miyoshi myopathy in individuals of northern European ancestry. It is not known why the 191dupA mutation can result in different patterns of signs and symptoms. Miyoshi myopathy caused by ANO5 gene mutations is likely a variation of limb-girdle muscular dystrophy because it is caused by mutations in the same gene, and in some cases even by the same mutation. More About This Health Condition    Cytogenetic Location: 11p14.3, which is the short (p) arm of chromosome 11 at position 14.3Molecular Location: base pairs 22,192,485 to 22,283,367 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ANO5_HUMANanoctamin-5GDD1gnathodiaphyseal dysplasia 1 proteinintegral membrane protein GDD1LGMD2LTMEM16Etransmembrane protein 16E     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     ANO5-Related Muscle Diseases      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ANO5      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ANOCTAMIN 5      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      ANO5-Related Muscle Diseases  Tests of ANO5  PubMed  ANOCTAMIN 5  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ANOCTAMIN 5Bolduc V, Marlow G, Boycott KM, Saleki K, Inoue H, Kroon J, Itakura M, Robitaille Y, Parent L, Baas F, Mizuta K, Kamata N, Richard I, Linssen WH, Mahjneh I, de Visser M, Bashir R, Brais B. Recessive mutations in the putative calcium-activated chloride channel Anoctamin 5 cause proximal LGMD2L and distal MMD3 muscular dystrophies. Am J Hum Genet. 2010 Feb 12;86(2):213-21. doi: 10.1016/j.ajhg.2009.12.013. Epub 2010 Jan 21. Citation on PubMed or Free article on PubMed CentralBouquet F, Cosse M, Bhin A, Deburgrave N, Romero N, Leturcq F, Eymard B. Miyoshi-like distal myopathy with mutations in anoctamin 5 gene. Rev Neurol (Paris). 2012 Feb;168(2):135-41. doi: 10.1016/j.neurol.2011.10.005. Epub 2012 Feb 13. Citation on PubMedHartzell HC, Yu K, Xiao Q, Chien LT, Qu Z. Anoctamin/TMEM16 family members are Ca2+-activated Cl- channels. J Physiol. 2009 May 15;587(Pt 10):2127-39. doi: 10.1113/jphysiol.2008.163709. Epub 2008 Nov 17. Review. Citation on PubMed or Free article on PubMed CentralHicks D, Sarkozy A, Muelas N, Koehler K, Huebner A, Hudson G, Chinnery PF, Barresi R, Eagle M, Polvikoski T, Bailey G, Miller J, Radunovic A, Hughes PJ, Roberts R, Krause S, Walter MC, Laval SH, Straub V, Lochmller H, Bushby K. A founder mutation in Anoctamin 5 is a major cause of limb-girdle muscular dystrophy. Brain. 2011 Jan;134(Pt 1):171-82. doi: 10.1093/brain/awq294. Citation on PubMed or Free article on PubMed CentralLiewluck T, Winder TL, Dimberg EL, Crum BA, Heppelmann CJ, Wang Y, Bergen HR 3rd, Milone M. ANO5-muscular dystrophy: clinical, pathological and molecular findings. Eur J Neurol. 2013 Oct;20(10):1383-9. doi: 10.1111/ene.12191. Epub 2013 May 12. Citation on PubMedMahjneh I, Jaiswal J, Lamminen A, Somer M, Marlow G, Kiuru-Enari S, Bashir R. A new distal myopathy with mutation in anoctamin 5. Neuromuscul Disord. 2010 Dec;20(12):791-5. doi: 10.1016/j.nmd.2010.07.270. Epub 2010 Aug 7. Citation on PubMed or Free article on PubMed CentralPenttil S, Palmio J, Suominen T, Raheem O, Evil A, Muelas Gomez N, Tasca G, Waddell LB, Clarke NF, Barboi A, Hackman P, Udd B. Eight new mutations and the expanding phenotype variability in muscular dystrophy caused by ANO5. Neurology. 2012 Mar 20;78(12):897-903. doi: 10.1212/WNL.0b013e31824c4682. Epub 2012 Mar 7. Erratum in: Neurology. 2013 Jan 8;80(2):226. Citation on PubMedSarkozy A, Hicks D, Hudson J, Laval SH, Barresi R, Hilton-Jones D, Deschauer M, Harris E, Rufibach L, Hwang E, Bashir R, Walter MC, Krause S, van den Bergh P, Illa I, Pnisson-Besnier I, De Waele L, Turnbull D, Guglieri M, Schrank B, Schoser B, Seeger J, Schreiber H, Glser D, Eagle M, Bailey G, Walters R, Longman C, Norwood F, Winer J, Muntoni F, Hanna M, Roberts M, Bindoff LA, Brierley C, Cooper RG, Cottrell DA, Davies NP, Gibson A, Gorman GS, Hammans S, Jackson AP, Khan A, Lane R, McConville J, McEntagart M, Al-Memar A, Nixon J, Panicker J, Parton M, Petty R, Price CJ, Rakowicz W, Ray P, Schapira AH, Swingler R, Turner C, Wagner KR, Maddison P, Shaw PJ, Straub V, Bushby K, Lochmller H. ANO5 gene analysis in a large cohort of patients with anoctaminopathy: confirmation of male prevalence and high occurrence of the common exon 5 gene mutation. Hum Mutat. 2013 Aug;34(8):1111-8. doi: 10.1002/humu.22342. Epub 2013 Jun 12. Citation on PubMedTsutsumi S, Kamata N, Vokes TJ, Maruoka Y, Nakakuki K, Enomoto S, Omura K, Amagasa T, Nagayama M, Saito-Ohara F, Inazawa J, Moritani M, Yamaoka T, Inoue H, Itakura M. The novel gene encoding a putative transmembrane protein is mutated in gnathodiaphyseal dysplasia (GDD). Am J Hum Genet. 2004 Jun;74(6):1255-61. Epub 2004 Apr 29. Citation on PubMed or Free article on PubMed Central \n",
            "TSPAN32\t\n",
            "CDKN1C\t   The CDKN1C gene provides instructions for making a protein that helps regulate growth. This protein acts as a tumor suppressor, which means that it keeps cells from growing and dividing too fast or in an uncontrolled way. It also is involved in controlling growth before birth, preventing the developing fetus from becoming too large.People inherit one copy of most genes from their mother and one copy from their father. Both copies are typically active, or \"turned on,\" in cells. However, the activity of the CDKN1C gene depends on which parent it was inherited from. In most tissues, the copy of the gene inherited from a person's mother (the maternally inherited copy) has much higher activity than the copy inherited from the father (the paternally inherited copy). This sort of parent-specific difference in gene activation is caused by a phenomenon called genomic imprinting.CDKN1C is part of a cluster of genes on the short (p) arm of chromosome 11 that undergo genomic imprinting. A nearby region of DNA known as imprinting center 2 (IC2) or KvDMR controls the parent-specific genomic imprinting of CDKN1C and several other genes thought to help regulate growth. The IC2 region undergoes a process called methylation, which is a chemical reaction that attaches small molecules called methyl groups to certain segments of DNA. Methylation, which occurs during the formation of an egg or sperm cell, is a way of marking or \"stamping\" the parent of origin. The IC2 region is normally methylated only on the maternally inherited copy of chromosome 11.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Beckwith-Wiedemann syndrome     Beckwith-Wiedemann syndrome is a condition that causes overgrowth and has other signs and symptoms that affect many parts of the body. At least half of all cases of Beckwith-Wiedemann syndrome result from changes in methylation of the IC2 region. Specifically, the maternally inherited copy of the IC2 region has too few methyl groups attached (hypomethylation). This abnormality disrupts the regulation of several genes that are normally controlled by IC2, including CDKN1C. Because this gene normally restrains cell growth and division, a reduction in its activity leads to overgrowth and the other features of Beckwith-Wiedemann syndrome.In a few cases, Beckwith-Wiedemann syndrome has been caused by deletions of a small amount of DNA from the maternally inherited copy of the IC2 region. Like abnormal methylation, these deletions disrupt the activity of several genes, including CDKN1C.Beckwith-Wiedemann syndrome can also result from mutations within the maternally inherited copy of the CDKN1C gene. More than two dozen such mutations have been identified. Some of these genetic changes lead to an abnormally short, nonfunctional version of the CDKN1C protein, while others alter single protein building blocks (amino acids) or delete a small number of amino acids from the protein. All of these mutations are described as \"loss-of-function\" because they alter the structure of the CDKN1C protein such that it can no longer control growth effectively. The resulting problems with growth regulation lead to overgrowth and the other features of Beckwith-Wiedemann syndrome. More About This Health Condition      Intrauterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita, and genital anomalies     Intrauterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita, and genital anomalies, commonly known by the acronym IMAGe, is a rare syndrome that affects the growth of many parts of the body. The condition is characterized by slow growth before and after birth, skeletal abnormalities, hormonal changes, and genital abnormalities in males. At least six mutations in the CDKN1C gene have been found to cause this condition. Because this gene is paternally imprinted, IMAGe syndrome results only when the mutation is present on the maternally inherited copy of the gene.The CDKN1C gene mutations that cause IMAGe syndrome replace single amino acids in a region known as the proliferating cell nuclear antigen (PCNA)-binding domain near the end of the gene. These mutations appear to increase the stability of the CDKN1C protein, preventing it from being broken down normally. These changes increase the amount of the protein that is available to restrain cell growth and division. Because these mutations enhance the protein's usual function, they are described as \"gain-of-function.\" The excess CDKN1C protein leads to IMAGe syndrome by impairing normal growth and development starting before birth. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Beckwith-Wiedemann syndrome is a condition that causes overgrowth and has other signs and symptoms that affect many parts of the body. At least half of all cases of Beckwith-Wiedemann syndrome result from changes in methylation of the IC2 region. Specifically, the maternally inherited copy of the IC2 region has too few methyl groups attached (hypomethylation). This abnormality disrupts the regulation of several genes that are normally controlled by IC2, including CDKN1C. Because this gene normally restrains cell growth and division, a reduction in its activity leads to overgrowth and the other features of Beckwith-Wiedemann syndrome.In a few cases, Beckwith-Wiedemann syndrome has been caused by deletions of a small amount of DNA from the maternally inherited copy of the IC2 region. Like abnormal methylation, these deletions disrupt the activity of several genes, including CDKN1C.Beckwith-Wiedemann syndrome can also result from mutations within the maternally inherited copy of the CDKN1C gene. More than two dozen such mutations have been identified. Some of these genetic changes lead to an abnormally short, nonfunctional version of the CDKN1C protein, while others alter single protein building blocks (amino acids) or delete a small number of amino acids from the protein. All of these mutations are described as \"loss-of-function\" because they alter the structure of the CDKN1C protein such that it can no longer control growth effectively. The resulting problems with growth regulation lead to overgrowth and the other features of Beckwith-Wiedemann syndrome. More About This Health Condition  Intrauterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita, and genital anomalies, commonly known by the acronym IMAGe, is a rare syndrome that affects the growth of many parts of the body. The condition is characterized by slow growth before and after birth, skeletal abnormalities, hormonal changes, and genital abnormalities in males. At least six mutations in the CDKN1C gene have been found to cause this condition. Because this gene is paternally imprinted, IMAGe syndrome results only when the mutation is present on the maternally inherited copy of the gene.The CDKN1C gene mutations that cause IMAGe syndrome replace single amino acids in a region known as the proliferating cell nuclear antigen (PCNA)-binding domain near the end of the gene. These mutations appear to increase the stability of the CDKN1C protein, preventing it from being broken down normally. These changes increase the amount of the protein that is available to restrain cell growth and division. Because these mutations enhance the protein's usual function, they are described as \"gain-of-function.\" The excess CDKN1C protein leads to IMAGe syndrome by impairing normal growth and development starting before birth. More About This Health Condition    Cytogenetic Location: 11p15.4, which is the short (p) arm of chromosome 11 at position 15.4Molecular Location: base pairs 2,883,218 to 2,885,804 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BWCRCDN1C_HUMANcyclin-dependent kinase inhibitor 1Ccyclin-dependent kinase inhibitor 1C (p57, Kip2)cyclin-dependent kinase inhibitor p57KIP2p57p57KIP2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Cell: A Molecular Approach (second edition, 2000): DNA Methylation      Clinical Information from GeneReviews (2 links)     Beckwith-Wiedemann SyndromeIMAGe Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CDKN1C      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CYCLIN-DEPENDENT KINASE INHIBITOR 1C      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics WebClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Cell: A Molecular Approach (second edition, 2000): DNA Methylation  Beckwith-Wiedemann SyndromeIMAGe Syndrome  Tests of CDKN1C  PubMed  CYCLIN-DEPENDENT KINASE INHIBITOR 1C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics WebClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Arboleda VA, Lee H, Parnaik R, Fleming A, Banerjee A, Ferraz-de-Souza B, Dlot EC, Rodriguez-Fernandez IA, Braslavsky D, Bergad I, Dell'Angelica EC, Nelson SF, Martinez-Agosto JA, Achermann JC, Vilain E. Mutations in the PCNA-binding domain of CDKN1C cause IMAGe syndrome. Nat Genet. 2012 May 27;44(7):788-92. doi: 10.1038/ng.2275. Citation on PubMed or Free article on PubMed CentralBorges KS, Arboleda VA, Vilain E. Mutations in the PCNA-binding site of CDKN1C inhibit cell proliferation by impairing the entry into S phase. Cell Div. 2015 Mar 28;10:2. doi: 10.1186/s13008-015-0008-8. eCollection 2015. Citation on PubMed or Free article on PubMed CentralEggermann T, Binder G, Brioude F, Maher ER, Lapunzina P, Cubellis MV, Bergad I, Prawitt D, Begemann M. CDKN1C mutations: two sides of the same coin. Trends Mol Med. 2014 Nov;20(11):614-22. doi: 10.1016/j.molmed.2014.09.001. Epub 2014 Sep 25. Review. Citation on PubMedGurrieri F, Zollino M, Oliva A, Pascali V, Orteschi D, Pietrobono R, Camporeale A, Coll Vidal M, Partemi S, Brugada R, Bellocci F, Neri G. Mild Beckwith-Wiedemann and severe long-QT syndrome due to deletion of the imprinting center 2 on chromosome 11p. Eur J Hum Genet. 2013 Sep;21(9):965-9. doi: 10.1038/ejhg.2012.280. Epub 2013 Mar 20. Citation on PubMed or Free article on PubMed CentralHamajima N, Johmura Y, Suzuki S, Nakanishi M, Saitoh S. Increased protein stability of CDKN1C causes a gain-of-function phenotype in patients with IMAGe syndrome. PLoS One. 2013 Sep 30;8(9):e75137. doi: 10.1371/journal.pone.0075137. eCollection 2013. Citation on PubMed or Free article on PubMed CentralMilani D, Pezzani L, Tabano S, Miozzo M. Beckwith-Wiedemann and IMAGe syndromes: two very different diseases caused by mutations on the same gene. Appl Clin Genet. 2014 Sep 16;7:169-75. doi: 10.2147/TACG.S35474. eCollection 2014. Review. Citation on PubMed or Free article on PubMed CentralRiccio A, Cubellis MV. Gain of function in CDKN1C. Nat Genet. 2012 Jun 27;44(7):737-8. doi: 10.1038/ng.2336. Citation on PubMedRomanelli V, Belinchn A, Benito-Sanz S, Martnez-Glez V, Gracia-Bouthelier R, Heath KE, Campos-Barros A, Garca-Miaur S, Fernandez L, Meneses H, Lpez-Siguero JP, Guilln-Navarro E, Gmez-Puertas P, Wesselink JJ, Mercado G, Esteban-Marfil V, Palomo R, Mena R, Snchez A, Del Campo M, Lapunzina P. CDKN1C (p57(Kip2)) analysis in Beckwith-Wiedemann syndrome (BWS) patients: Genotype-phenotype correlations, novel mutations, and polymorphisms. Am J Med Genet A. 2010 Jun;152A(6):1390-7. doi: 10.1002/ajmg.a.33453. Review. Citation on PubMed \n",
            "SLC22A18\t   From NCBI Gene:This gene is one of several tumor-suppressing subtransferable fragments located in the imprinted gene domain of 11p15.5, an important tumor-suppressor gene region. Alterations in this region have been associated with the Beckwith-Wiedemann syndrome, Wilms tumor, rhabdomyosarcoma, adrenocortical carcinoma, and lung, ovarian, and breast cancer. This gene is imprinted, with preferential expression from the maternal allele. Mutations in this gene have been found in Wilms' tumor and lung cancer. This protein may act as a transporter of organic cations, and have a role in the transport of chloroquine and quinidine-related compounds in kidney. Several alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Oct 2015]From UniProt:May act as a transporter of organic cations based on a proton efflux antiport mechanism. May play a role in the transport of chloroquine and quinidine-related compounds in kidney.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Lung cancerFrom NCBI Gene:Rhabdomyosarcoma 1Familial cancer of breastLung cancerFrom UniProt:Rhabdomyosarcoma, embryonal, 1 (RMSE1): A form of rhabdomyosarcoma, a highly malignant tumor of striated muscle derived from primitive mesenchymal cells and exhibiting differentiation along rhabdomyoblastic lines. Rhabdomyosarcoma is one of the most frequently occurring soft tissue sarcomas and the most common in children. It occurs in four forms: alveolar, pleomorphic, embryonal and botryoidal rhabdomyosarcomas. [MIM:268210]Lung cancer (LNCR): A common malignancy affecting tissues of the lung. The most common form of lung cancer is non-small cell lung cancer (NSCLC) that can be divided into 3 major histologic subtypes: squamous cell carcinoma, adenocarcinoma, and large cell lung cancer. NSCLC is often diagnosed at an advanced stage and has a poor prognosis. [MIM:211980]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p15.4, which is the short (p) arm of chromosome 11 at position 15.4Molecular Location: base pairs 2,899,721 to 2,925,246 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BWR1ABWSCR1AHETIMPT1ITMORCTL2p45-BWR1ASLC22A1LTSSC5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC22A18      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: BREAST CANCEROMIM: LUNG CANCEROMIM: RHABDOMYOSARCOMA, EMBRYONAL, 1OMIM: SOLUTE CARRIER FAMILY 22 (ORGANIC CATION TRANSPORTER), MEMBER 18      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SLC22A18  OMIM: BREAST CANCEROMIM: LUNG CANCEROMIM: RHABDOMYOSARCOMA, EMBRYONAL, 1OMIM: SOLUTE CARRIER FAMILY 22 (ORGANIC CATION TRANSPORTER), MEMBER 18  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PGGHG\t\n",
            "MRGPRE\t\n",
            "PRRG4\t\n",
            "PDHX\t   The PDHX gene provides instructions for making a protein called E3 binding protein, which is part of a large group of proteins known as the pyruvate dehydrogenase complex. This complex is made up of several enzymes, including one called E3, and other proteins. E3 binding protein attaches E3 to the complex and provides the correct structure for the complex to perform its function.The pyruvate dehydrogenase complex plays an important role in the pathways that convert the energy from food into a form that cells can use. This enzyme converts a molecule called pyruvate, which is formed from the breakdown of carbohydrates, into another molecule called acetyl-CoA. This conversion is essential to begin the series of chemical reactions that produces adenosine triphosphate (ATP), the cell's main energy source.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Pyruvate dehydrogenase deficiency     Mutations in the PDHX gene cause pyruvate dehydrogenase deficiency in a small number of people. This condition is characterized by a potentially life-threatening buildup of a chemical called lactic acid in the body (lactic acidosis), delayed development, and neurological problems. PDHX gene mutations associated with pyruvate dehydrogenase deficiency result in the complete absence of E3 binding protein. Loss of this protein impairs the binding of the E3 enzyme to the pyruvate dehydrogenase complex, which leads to a reduction of the complex's activity. With decreased function of this complex, pyruvate builds up and is converted, in another chemical reaction, to lactic acid, causing lactic acidosis. In addition, the production of cellular energy is diminished. The brain, which is especially dependent on this form of energy, is severely affected, resulting in the neurological problems associated with pyruvate dehydrogenase deficiency. More About This Health Condition      Leigh syndrome     Genetics Home Reference provides information about Leigh syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the PDHX gene cause pyruvate dehydrogenase deficiency in a small number of people. This condition is characterized by a potentially life-threatening buildup of a chemical called lactic acid in the body (lactic acidosis), delayed development, and neurological problems. PDHX gene mutations associated with pyruvate dehydrogenase deficiency result in the complete absence of E3 binding protein. Loss of this protein impairs the binding of the E3 enzyme to the pyruvate dehydrogenase complex, which leads to a reduction of the complex's activity. With decreased function of this complex, pyruvate builds up and is converted, in another chemical reaction, to lactic acid, causing lactic acidosis. In addition, the production of cellular energy is diminished. The brain, which is especially dependent on this form of energy, is severely affected, resulting in the neurological problems associated with pyruvate dehydrogenase deficiency. More About This Health Condition  Genetics Home Reference provides information about Leigh syndrome. More About This Health Condition    Cytogenetic Location: 11p13, which is the short (p) arm of chromosome 11 at position 13Molecular Location: base pairs 34,915,829 to 34,996,128 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        dihydrolipoamide dehydrogenase-binding protein of pyruvate dehydrogenase complexDLDBPE3BPlipoyl-containing pyruvate dehydrogenase complex component XODPX_HUMANOPDXproXpyruvate dehydrogenase complex, component Xpyruvate dehydrogenase complex, E3-binding protein subunitpyruvate dehydrogenase complex, lipoyl-containing component Xpyruvate dehydrogenase protein X component, mitochondrial     Related Information   How are genetic conditions and genes named?          Educational Resources (4 links)     Biochemistry (fifth edition, 2002): The Formation of Acetyl Coenzyme A from PyruvateBiochemistry (fifth edition, 2002): The Pyruvate Dehydrogenase Complex Is Regulated Allosterically and by Reversible PhosphorylationMolecular Biology of the Cell (fourth edition, 2002): Sugars and Fats Are Both Degraded to Acetyl CoA in MitochondriaMolecular Cell Biology (fourth edition, 2000): Mitochondrial Oxidation of Pyruvate Begins with the Formation of Acetyl CoA      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PDHX      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PYRUVATE DEHYDROGENASE COMPLEX, COMPONENT X      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): The Formation of Acetyl Coenzyme A from PyruvateBiochemistry (fifth edition, 2002): The Pyruvate Dehydrogenase Complex Is Regulated Allosterically and by Reversible PhosphorylationMolecular Biology of the Cell (fourth edition, 2002): Sugars and Fats Are Both Degraded to Acetyl CoA in MitochondriaMolecular Cell Biology (fourth edition, 2000): Mitochondrial Oxidation of Pyruvate Begins with the Formation of Acetyl CoA  Tests of PDHX  PubMed  PYRUVATE DEHYDROGENASE COMPLEX, COMPONENT X  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Biochemistry (fifth edition, 2002): The Formation of Acetyl Coenzyme A from PyruvateBrown RM, Head RA, Brown GK. Pyruvate dehydrogenase E3 binding protein deficiency. Hum Genet. 2002 Feb;110(2):187-91. Epub 2002 Jan 22. Citation on PubMedBrown RM, Head RA, Morris AA, Raiman JA, Walter JH, Whitehouse WP, Brown GK. Pyruvate dehydrogenase E3 binding protein (protein X) deficiency. Dev Med Child Neurol. 2006 Sep;48(9):756-60. Citation on PubMedOMIM: PYRUVATE DEHYDROGENASE COMPLEX, COMPONENT XPatel MS, Korotchkina LG, Sidhu S. Interaction of E1 and E3 components with the core proteins of the human pyruvate dehydrogenase complex. J Mol Catal B Enzym. 2009 Nov 1;61(1-2):2-6. Citation on PubMed or Free article on PubMed CentralSmolle M, Prior AE, Brown AE, Cooper A, Byron O, Lindsay JG. A new level of architectural complexity in the human pyruvate dehydrogenase complex. J Biol Chem. 2006 Jul 14;281(28):19772-80. Epub 2006 May 5. Citation on PubMed or Free article on PubMed Central \n",
            "ART5\t\n",
            "ALKBH3\t\n",
            "OR52I1\t\n",
            "CREB3L1\t   From NCBI Gene:The protein encoded by this gene is normally found in the membrane of the endoplasmic reticulum (ER). However, upon stress to the ER, the encoded protein is cleaved and the released cytoplasmic transcription factor domain translocates to the nucleus. There it activates the transcription of target genes by binding to box-B elements. [provided by RefSeq, Jun 2013]From UniProt:(Microbial infection) May play a role in limiting virus spread by inhibiting proliferation of virus-infected cells. Upon infection with diverse DNA and RNA viruses, inhibits cell-cycle progression by binding to promoters and activating transcription of genes encoding cell-cycle inhibitors, such as p21/CDKN1A (PubMed:21767813).Transcription factor involved in unfolded protein response (UPR). Binds the DNA consensus sequence 5'-GTGXGCXGC-3' (PubMed:21767813). In the absence of endoplasmic reticulum (ER) stress, inserted into ER membranes, with N-terminal DNA-binding and transcription activation domains oriented toward the cytosolic face of the membrane. In response to ER stress, transported to the Golgi, where it is cleaved in a site-specific manner by resident proteases S1P/MBTPS1 and S2P/MBTPS2. The released N-terminal cytosolic domain is translocated to the nucleus to effect transcription of specific target genes. Plays a critical role in bone formation through the transcription of COL1A1, and possibly COL1A2, and the secretion of bone matrix proteins. Directly binds to the UPR element (UPRE)-like sequence in an osteoblast-specific COL1A1 promoter region and induces its transcription. Does not regulate COL1A1 in other tissues, such as skin (By similarity). Required to protect astrocytes from ER stress-induced cell death. In astrocytes, binds to the cAMP response element (CRE) of the BiP/HSPA5 promoter and participate in its transcriptional activation (By similarity). Required for TGFB1 to activate genes involved in the assembly of collagen extracellular matrix (PubMed:25310401).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Osteogenesis imperfecta, type xviFrom UniProt:Osteogenesis imperfecta 16 (OI16): An autosomal recessive form of osteogenesis imperfecta, a connective tissue disorder characterized by low bone mass, bone fragility and susceptibility to fractures after minimal trauma. Disease severity ranges from very mild forms without fractures to intrauterine fractures and perinatal lethality. Extraskeletal manifestations, which affect a variable number of patients, are dentinogenesis imperfecta, hearing loss, and blue sclerae. OI16 is a severe form. [MIM:616229]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p11.2, which is the short (p) arm of chromosome 11 at position 11.2Molecular Location: base pairs 46,277,638 to 46,321,422 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        OASISOI16     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CREB3L1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: cAMP RESPONSE ELEMENT-BINDING PROTEIN 3-LIKE 1OMIM: OSTEOGENESIS IMPERFECTA, TYPE XVI      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CREB3L1  OMIM: cAMP RESPONSE ELEMENT-BINDING PROTEIN 3-LIKE 1OMIM: OSTEOGENESIS IMPERFECTA, TYPE XVI  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OR51D1\t\n",
            "ZNF408\t   From NCBI Gene:The protein encoded by this gene contains ten tandem zinc fingers and an N-terminal SET domain, so it is likely a DNA binding protein that interacts with other proteins. In adults, the encoded protein is expressed most highly in retina. Consequently, defects in this gene have been associated with familial exudative vitreoretinopathy (FEVR) and retinitis pigmentosa (RP). [provided by RefSeq, Dec 2016]From UniProt:May be involved in transcriptional regulation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Exudative vitreoretinopathy 6Retinitis pigmentosa 72From UniProt:Vitreoretinopathy, exudative 6 (EVR6): A form of exudative vitreoretinopathy, a disorder of the retinal vasculature characterized by an abrupt cessation of growth of peripheral capillaries, leading to an avascular peripheral retina. This may lead to compensatory retinal neovascularization, which is thought to be induced by hypoxia from the initial avascular insult. New vessels are prone to leakage and rupture causing exudates and bleeding, followed by scarring, retinal detachment and blindness. Clinical features can be highly variable, even within the same family. Patients with mild forms of the disease are asymptomatic, and their only disease related abnormality is an arc of avascular retina in the extreme temporal periphery. [MIM:616468]Retinitis pigmentosa 72 (RP72): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:616469]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p11.2, which is the short (p) arm of chromosome 11 at position 11.2Molecular Location: base pairs 46,700,767 to 46,705,916 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        EVR6RP72     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ZNF408      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: EXUDATIVE VITREORETINOPATHY 6OMIM: RETINITIS PIGMENTOSA 72OMIM: ZINC FINGER PROTEIN 408      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ZNF408  OMIM: EXUDATIVE VITREORETINOPATHY 6OMIM: RETINITIS PIGMENTOSA 72OMIM: ZINC FINGER PROTEIN 408  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RAPSN\t   The RAPSN gene provides instructions for making a protein called rapsyn that attaches (binds) to the different parts (subunits) of a protein found in the muscle cell membrane called acetylcholine receptor (AChR). This binding helps keep the receptor subunits together and anchors the AChR protein in the muscle cell membrane. The AChR protein plays a critical role in the normal function of the neuromuscular junction. The neuromuscular junction is the area between the ends of nerve cells and muscle cells where signals are relayed to trigger muscle movement.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Congenital myasthenic syndrome     At least 45 mutations in the RAPSN gene have been found to cause congenital myasthenic syndrome. Most of these mutations change single protein building blocks (amino acids) in the rapsyn protein. A common mutation replaces the amino acid asparagine with the amino acid lysine at position 88 in the rapsyn protein (written as Asn88Lys or N88K). Most mutations in the RAPSN gene result in a reduction in functional rapsyn protein. The lack of rapsyn protein results in decreased binding between rapsyn and the AChR protein, which leads to disorganization of the receptor protein in the muscle cell membrane and a reduction in the number of receptors. As a result, signaling at the neuromuscular junction is decreased, which leads to decreased muscle movement and the muscle weakness characteristic of congenital myasthenic syndrome. More About This Health Condition      Multiple pterygium syndrome     Genetics Home Reference provides information about multiple pterygium syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 45 mutations in the RAPSN gene have been found to cause congenital myasthenic syndrome. Most of these mutations change single protein building blocks (amino acids) in the rapsyn protein. A common mutation replaces the amino acid asparagine with the amino acid lysine at position 88 in the rapsyn protein (written as Asn88Lys or N88K). Most mutations in the RAPSN gene result in a reduction in functional rapsyn protein. The lack of rapsyn protein results in decreased binding between rapsyn and the AChR protein, which leads to disorganization of the receptor protein in the muscle cell membrane and a reduction in the number of receptors. As a result, signaling at the neuromuscular junction is decreased, which leads to decreased muscle movement and the muscle weakness characteristic of congenital myasthenic syndrome. More About This Health Condition  Genetics Home Reference provides information about multiple pterygium syndrome. More About This Health Condition    Cytogenetic Location: 11p11.2, which is the short (p) arm of chromosome 11 at position 11.2Molecular Location: base pairs 47,437,757 to 47,449,178 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        RAPSN_HUMANRAPSYNreceptor-associated protein of the synapseRING finger protein 205RNF205     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Basic Neurochemistry (6th edition, 1999): The neuromuscular junction is a well-defined structure that mediates the release and postsynaptic effects of acetylcholineWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Congenital Myasthenic Syndromes      Tests Listed in the Genetic Testing Registry (1 link)     Tests of RAPSN      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     RECEPTOR-ASSOCIATED PROTEIN OF THE SYNAPSE, 43-KD      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (6th edition, 1999): The neuromuscular junction is a well-defined structure that mediates the release and postsynaptic effects of acetylcholineWashington University, St. Louis: Neuromuscular Disease Center  Congenital Myasthenic Syndromes  Tests of RAPSN  PubMed  RECEPTOR-ASSOCIATED PROTEIN OF THE SYNAPSE, 43-KD  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Barii N, Chaouch A, Mller JS, Lochmller H. Genetic heterogeneity and pathophysiological mechanisms in congenital myasthenic syndromes. Eur J Paediatr Neurol. 2011 May;15(3):189-96. doi: 10.1016/j.ejpn.2011.03.006. Epub 2011 Apr 17. Review. Citation on PubMedBeeson D, Webster R, Cossins J, Lashley D, Spearman H, Maxwell S, Slater CR, Newsom-Davis J, Palace J, Vincent A. Congenital myasthenic syndromes and the formation of the neuromuscular junction. Ann N Y Acad Sci. 2008;1132:99-103. doi: 10.1196/annals.1405.049. Citation on PubMedEngel AG. Congenital myasthenic syndromes in 2012. Curr Neurol Neurosci Rep. 2012 Feb;12(1):92-101. doi: 10.1007/s11910-011-0234-7. Review. Citation on PubMed or Free article on PubMed CentralEngel AG. Current status of the congenital myasthenic syndromes. Neuromuscul Disord. 2012 Feb;22(2):99-111. doi: 10.1016/j.nmd.2011.10.009. Epub 2011 Nov 21. Review. Citation on PubMed or Free article on PubMed CentralKinali M, Beeson D, Pitt MC, Jungbluth H, Simonds AK, Aloysius A, Cockerill H, Davis T, Palace J, Manzur AY, Jimenez-Mallebrera C, Sewry C, Muntoni F, Robb SA. Congenital myasthenic syndromes in childhood: diagnostic and management challenges. J Neuroimmunol. 2008 Sep 15;201-202:6-12. doi: 10.1016/j.jneuroim.2008.06.026. Epub 2008 Aug 15. Citation on PubMedMilone M, Shen XM, Selcen D, Ohno K, Brengman J, Iannaccone ST, Harper CM, Engel AG. Myasthenic syndrome due to defects in rapsyn: Clinical and molecular findings in 39 patients. Neurology. 2009 Jul 21;73(3):228-35. doi: 10.1212/WNL.0b013e3181ae7cbc. Citation on PubMed or Free article on PubMed CentralOMIM: RECEPTOR-ASSOCIATED PROTEIN OF THE SYNAPSE, 43-KD \n",
            "OR4B1\t\n",
            "OR4X2\t\n",
            "OR4X1\t\n",
            "OR4C3\t\n",
            "OR51S1\t\n",
            "TRIM64C\t\n",
            "FOLH1\t\n",
            "RNH1\t   From NCBI Gene:This gene encodes an endonuclease that specifically degrades the RNA of RNA-DNA hybrids and plays a key role in DNA replication and repair. Alternate in-frame start codon initiation results in the production of alternate isoforms that are directed to the mitochondria or to the nucleus. The production of the mitochondrial isoform is modulated by an upstream open reading frame (uORF). Mutations in this gene have been found in individuals with progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 2. Alternative splicing results in additional coding and non-coding transcript variants. Pseudogenes of this gene have been defined on chromosomes 2 and 17. [provided by RefSeq, Jul 2017]From UniProt:Endonuclease that specifically degrades the RNA of RNA-DNA hybrids (PubMed:10497183). Plays a role in RNA polymerase II (RNAp II) transcription termination by degrading R-loop RNA-DNA hybrid formation at G-rich pause sites located downstream of the poly(A) site and behind the elongating RNAp II (PubMed:21700224).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Progressive external ophthalmoplegiaFrom NCBI Gene:Progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 2From UniProt:Progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 2 (PEOB2): A form of progressive external ophthalmoplegia, a mitochondrial myopathy characterized by progressive paralysis of the levator palpebrae, orbicularis oculi, and extraocular muscles. PEOB2 patients manifest exercise intolerance, muscle weakness, and signs and symptoms of spinocerebellar ataxia, such as impaired gait and dysarthria. Some patients may have respiratory insufficiency. [MIM:616479]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p25.3, which is the short (p) arm of chromosome 2 at position 25.3Molecular Location: base pairs 3,531,813 to 3,558,371 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        H1RNAPEOB2RNH1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RNASEH1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PROGRESSIVE EXTERNAL OPHTHALMOPLEGIA WITH MITOCHONDRIAL DNA DELETIONS, AUTOSOMAL RECESSIVE 2OMIM: RIBONUCLEASE H1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RNASEH1  OMIM: PROGRESSIVE EXTERNAL OPHTHALMOPLEGIA WITH MITOCHONDRIAL DNA DELETIONS, AUTOSOMAL RECESSIVE 2OMIM: RIBONUCLEASE H1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OR52J3\t\n",
            "OR52E2\t\n",
            "OR4C46\t\n",
            "OR51V1\t\n",
            "OR51B2\t\n",
            "OR51B5\t\n",
            "OR51B6\t\n",
            "OR51M1\t\n",
            "OR51Q1\t\n",
            "OR51I2\t\n",
            "OR52D1\t\n",
            "OR4C16\t\n",
            "TRIM51\t\n",
            "OR5F1\t\n",
            "OR8K3\t\n",
            "OR8K1\t\n",
            "OR5R1\t\n",
            "OR5M1\t\n",
            "OR5AR1\t\n",
            "OR9G1\t\n",
            "TRIM6-TRIM34\t\n",
            "PRG2\t\n",
            "SMTNL1\t\n",
            "OR1S1\t\n",
            "ZFP91-CNTF\t\n",
            "ZFP91\t\n",
            "OR52N2\t\n",
            "OR5AN1\t\n",
            "OR5A1\t\n",
            "OR52L1\t\n",
            "MS4A6E\t\n",
            "MS4A14\t\n",
            "MS4A12\t\n",
            "MS4A13\t\n",
            "MS4A15\t\n",
            "VPS37C\t\n",
            "TMEM216\t   From NCBI Gene:This locus encodes a transmembrane domain-containing protein. Mutations at this locus have been associated with Meckel-Gruber Syndrome Type 2, and Joubert Syndrome 2, also known as Cerebello-oculorenal Syndrome 2. [provided by RefSeq, Aug 2010]From UniProt:Part of the tectonic-like complex which is required for tissue-specific ciliogenesis and may regulate ciliary membrane composition.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Meckel syndromeJoubert syndromeFrom NCBI Gene:Meckel syndrome type 2Joubert syndrome 2From UniProt:Meckel syndrome 2 (MKS2): A disorder characterized by a combination of renal cysts and variably associated features including developmental anomalies of the central nervous system (typically encephalocele), hepatic ductal dysplasia and cysts, and polydactyly. [MIM:603194]Joubert syndrome 2 (JBTS2): A disorder presenting with cerebellar ataxia, oculomotor apraxia, hypotonia, neonatal breathing abnormalities and psychomotor delay. Neuroradiologically, it is characterized by cerebellar vermian hypoplasia/aplasia, thickened and reoriented superior cerebellar peduncles, and an abnormally large interpeduncular fossa, giving the appearance of a molar tooth on transaxial slices (molar tooth sign). Additional variable features include retinal dystrophy and renal disease. [MIM:608091]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q12.2, which is the long (q) arm of chromosome 11 at position 12.2Molecular Location: base pairs 61,391,982 to 61,398,863 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HSPC244     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TMEM216      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: JOUBERT SYNDROME 2OMIM: MECKEL SYNDROME, TYPE 2OMIM: TRANSMEMBRANE PROTEIN 216      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TMEM216  OMIM: JOUBERT SYNDROME 2OMIM: MECKEL SYNDROME, TYPE 2OMIM: TRANSMEMBRANE PROTEIN 216  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OR56B4\t\n",
            "OR52W1\t\n",
            "EML3\t\n",
            "SLC22A24\t\n",
            "SLC22A10\t\n",
            "RTN3\t\n",
            "MARK2\t\n",
            "DNAJC4\t\n",
            "PLCB3\t\n",
            "ESRRA\t\n",
            "SMPD1\t   The SMPD1 gene provides instructions for making an enzyme called acid sphingomyelinase. This enzyme is found in lysosomes, which are small compartments in the cell that digest and recycle molecules. Acid sphingomyelinase is responsible for the conversion of a fat (lipid) called sphingomyelin into another type of lipid called ceramide. Sphingomyelin also binds (attaches) to a fat called cholesterol and helps to form other lipids that play roles in various cell processes. The formations of these lipids is critical for the normal structure and function of cells and tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Niemann-Pick disease     At least 175 mutations in the SMPD1 gene have been found to cause Niemann-Pick disease types A and B. These types of Niemann-Pick disease are characterized by a buildup of fat within cells that leads to lung disease and enlargement of the liver and spleen (hepatosplenomegaly). Type A is more severe and is characterized by severe neurological impairment in early childhood.SMPD1 gene mutations that cause complete loss of enzyme function tend to cause Niemann-Pick disease type A. In the Ashkenazi (eastern and central European) Jewish population, three mutations are responsible for about 90 percent of all Niemann-Pick disease type A cases. Mutations that lead to the production of an enzyme that retains some activity often cause Niemann-Pick disease type B. A reduction in enzyme activity within cells allows sphingomyelin to accumulate in cells. The accumulation of this lipid causes cells to malfunction and eventually die. Over time, cell loss impairs function of tissues and organs including the brain, lungs, spleen, and liver in people with Niemann-Pick disease types A and B. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 175 mutations in the SMPD1 gene have been found to cause Niemann-Pick disease types A and B. These types of Niemann-Pick disease are characterized by a buildup of fat within cells that leads to lung disease and enlargement of the liver and spleen (hepatosplenomegaly). Type A is more severe and is characterized by severe neurological impairment in early childhood.SMPD1 gene mutations that cause complete loss of enzyme function tend to cause Niemann-Pick disease type A. In the Ashkenazi (eastern and central European) Jewish population, three mutations are responsible for about 90 percent of all Niemann-Pick disease type A cases. Mutations that lead to the production of an enzyme that retains some activity often cause Niemann-Pick disease type B. A reduction in enzyme activity within cells allows sphingomyelin to accumulate in cells. The accumulation of this lipid causes cells to malfunction and eventually die. Over time, cell loss impairs function of tissues and organs including the brain, lungs, spleen, and liver in people with Niemann-Pick disease types A and B. More About This Health Condition    Cytogenetic Location: 11p15.4, which is the short (p) arm of chromosome 11 at position 15.4Molecular Location: base pairs 6,390,414 to 6,394,998 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        acid sphingomyelinaseASMASM_HUMANsphingomyelin phosphodiesterase 1, acid lysosomalsphingomyelin phosphodiesterase 1, acid lysosomal (acid sphingomyelinase)     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Basic Neurochemistry (sixth edition, 1999): Sphingolipidoses are Caused by Genetic Defects in a Series of Lysosomal Enzymes and Other Proteins Essential for the Catabolism of SphingolipidsMadame Curie Bioscience Database: Defects in Lipid Degradation      Clinical Information from GeneReviews (1 link)     Acid Sphingomyelinase Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SMPD1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SPHINGOMYELIN PHOSPHODIESTERASE 1, ACID LYSOSOMAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Sphingolipidoses are Caused by Genetic Defects in a Series of Lysosomal Enzymes and Other Proteins Essential for the Catabolism of SphingolipidsMadame Curie Bioscience Database: Defects in Lipid Degradation  Acid Sphingomyelinase Deficiency  Tests of SMPD1  PubMed  SPHINGOMYELIN PHOSPHODIESTERASE 1, ACID LYSOSOMAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Hollak CE, de Sonnaville ES, Cassiman D, Linthorst GE, Groener JE, Morava E, Wevers RA, Mannens M, Aerts JM, Meersseman W, Akkerman E, Niezen-Koning KE, Mulder MF, Visser G, Wijburg FA, Lefeber D, Poorthuis BJ. Acid sphingomyelinase (Asm) deficiency patients in The Netherlands and Belgium: disease spectrum and natural course in attenuated patients. Mol Genet Metab. 2012 Nov;107(3):526-33. doi: 10.1016/j.ymgme.2012.06.015. Epub 2012 Jun 30. Citation on PubMedIrun P, Malln M, Dominguez C, Rodriguez-Sureda V, Alvarez-Sala LA, Arslan N, Bermejo N, Guerrero C, Perez de Soto I, Villaln L, Giraldo P, Pocovi M. Identification of seven novel SMPD1 mutations causing Niemann-Pick disease types A and B. Clin Genet. 2013 Oct;84(4):356-61. doi: 10.1111/cge.12076. Epub 2013 Jan 4. Citation on PubMedOMIM: SPHINGOMYELIN PHOSPHODIESTERASE 1, ACID LYSOSOMALSchuchman EH. The pathogenesis and treatment of acid sphingomyelinase-deficient Niemann-Pick disease. J Inherit Metab Dis. 2007 Oct;30(5):654-63. Epub 2007 Jul 12. Review. Citation on PubMedSimonaro CM, Desnick RJ, McGovern MM, Wasserstein MP, Schuchman EH. The demographics and distribution of type B Niemann-Pick disease: novel mutations lead to new genotype/phenotype correlations. Am J Hum Genet. 2002 Dec;71(6):1413-9. Epub 2002 Oct 4. Citation on PubMed or Free article on PubMed CentralWasserstein MP, Aron A, Brodie SE, Simonaro C, Desnick RJ, McGovern MM. Acid sphingomyelinase deficiency: prevalence and characterization of an intermediate phenotype of Niemann-Pick disease. J Pediatr. 2006 Oct;149(4):554-9. Citation on PubMedWasserstein MP, Desnick RJ, Schuchman EH, Hossain S, Wallenstein S, Lamm C, McGovern MM. The natural history of type B Niemann-Pick disease: results from a 10-year longitudinal study. Pediatrics. 2004 Dec;114(6):e672-7. Epub 2004 Nov 15. Citation on PubMed \n",
            "CAPN1\t   From NCBI Gene:The calpains, calcium-activated neutral proteases, are nonlysosomal, intracellular cysteine proteases. The mammalian calpains include ubiquitous, stomach-specific, and muscle-specific proteins. The ubiquitous enzymes consist of heterodimers with distinct large, catalytic subunits associated with a common small, regulatory subunit. This gene encodes the large subunit of the ubiquitous enzyme, calpain 1. Several transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Nov 2010]From UniProt:Calcium-regulated non-lysosomal thiol-protease which catalyzes limited proteolysis of substrates involved in cytoskeletal remodeling and signal transduction.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spastic paraplegia 76, autosomal recessiveFrom UniProt:Spastic paraplegia 76, autosomal recessive (SPG76): A form of spastic paraplegia, a neurodegenerative disorder characterized by a slow, gradual, progressive weakness and spasticity of the lower limbs. Rate of progression and the severity of symptoms are quite variable. Initial symptoms may include difficulty with balance, weakness and stiffness in the legs, muscle spasms, and dragging the toes when walking. In some forms of the disorder, bladder symptoms (such as incontinence) may appear, or the weakness and stiffness may spread to other parts of the body. [MIM:616907]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q13.1, which is the long (q) arm of chromosome 11 at position 13.1Molecular Location: base pairs 65,181,215 to 65,212,006 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CANPCANP1CANPL1muCANPmuCLSPG76     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CAPN1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CALPAIN 1OMIM: SPASTIC PARAPLEGIA 76, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CAPN1  OMIM: CALPAIN 1OMIM: SPASTIC PARAPLEGIA 76, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LTBP3\t   From NCBI Gene:The protein encoded by this gene forms a complex with transforming growth factor beta (TGF-beta) proteins and may be involved in their subcellular localization. Activation of this complex requires removal of the encoded binding protein. This protein also may play a structural role in the extracellular matrix. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2010]From UniProt:Key regulator of transforming growth factor beta (TGFB1, TGFB2 and TGFB3) that controls TGF-beta activation by maintaining it in a latent state during storage in extracellular space. Associates specifically via disulfide bonds with the Latency-associated peptide (LAP), which is the regulatory chain of TGF-beta, and regulates integrin-dependent activation of TGF-beta.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Verloes Bourguignon syndromeGELEOPHYSIC DYSPLASIA 3From UniProt:Geleophysic dysplasia 3 (GPHYSD3): A form of geleophysic dysplasia, a rare skeletal disease characterized by severe short stature, short hands and feet, and joint limitations. Radiologic features include delayed bone age, cone-shaped epiphyses, shortened long tubular bones, and ovoid vertebral bodies. Affected individuals have characteristic facial features including a 'happy' face with full cheeks, shortened nose, hypertelorism, long and flat philtrum, and thin upper lip. Other distinctive features include skin thickening, progressive cardiac valvular thickening often leading to an early death, toe walking, tracheal stenosis, respiratory insufficiency, and lysosomal-like storage vacuoles in various tissues. GPHYSD3 inheritance is autosomal dominant. [MIM:617809]Dental anomalies and short stature (DASS): A disorder characterized by hypoplastic amelogenesis imperfecta, significant short stature, brachyolmia-like anomalies including platyspondyly with short pedicles, narrow intervertebral and interpedicular distances, rectangular-shaped vertebrae with posterior scalloping and herniation of the nuclei, and broad femoral necks. Dental anomalies include widely spaced, small, yellow teeth, oligodontia, and severely reduced to absent enamel. [MIM:601216]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q13.1, which is the long (q) arm of chromosome 11 at position 13.1Molecular Location: base pairs 65,538,559 to 65,558,389 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DASSGPHYSD3LTBP-3LTBP2pp6425STHAG6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LTBP3      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DENTAL ANOMALIES AND SHORT STATUREOMIM: GELEOPHYSIC DYSPLASIA 3OMIM: LATENT TRANSFORMING GROWTH FACTOR-BETA-BINDING PROTEIN 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LTBP3  OMIM: DENTAL ANOMALIES AND SHORT STATUREOMIM: GELEOPHYSIC DYSPLASIA 3OMIM: LATENT TRANSFORMING GROWTH FACTOR-BETA-BINDING PROTEIN 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "EHBP1L1\t\n",
            "AP5B1\t\n",
            "DNHD1\t\n",
            "MUS81\t\n",
            "TBX10\t\n",
            "UNC93B1\t   From NCBI Gene:This gene encodes a protein that is involved in innate and adaptive immune response by regulating toll-like receptor signaling. The encoded protein traffics nucleotide sensing toll-like receptors to the endolysosome from the endoplasmic reticulum. Deficiency of the encoded protein has been associated with herpes simplex encephalitis. [provided by RefSeq, Feb 2014]From UniProt:Plays an important role in innate and adaptive immunity by regulating nucleotide-sensing Toll-like receptor (TLR) signaling. Required for the transport of a subset of TLRs (including TLR3, TLR7 and TLR9) from the endoplasmic reticulum to endolysosomes where they can engage pathogen nucleotides and activate signaling cascades. May play a role in autoreactive B-cells removal.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Herpes simplex encephalitis 1From UniProt:Encephalopathy, acute, infection-induced, Herpes-specific, 1 (IIAE1): A rare complication of human herpesvirus 1 (HHV-1) infection, occurring in only a small minority of HHV-1 infected individuals. It is characterized by hemorrhagic necrosis of parts of the temporal and frontal lobes. Onset is over several days and involves fever, headache, seizures, stupor, and often coma, frequently with a fatal outcome. [MIM:610551]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q13.2, which is the long (q) arm of chromosome 11 at position 13.2Molecular Location: base pairs 67,991,100 to 68,004,125 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IIAE1Unc-93B1UNC93UNC93B     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of UNC93B1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ENCEPHALOPATHY, ACUTE, INFECTION-INDUCED (HERPES-SPECIFIC), SUSCEPTIBILITY TO, 1OMIM: UNC93, C. ELEGANS, HOMOLOG OF, B1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of UNC93B1  OMIM: ENCEPHALOPATHY, ACUTE, INFECTION-INDUCED (HERPES-SPECIFIC), SUSCEPTIBILITY TO, 1OMIM: UNC93, C. ELEGANS, HOMOLOG OF, B1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ALDH3B1\t\n",
            "OR2AG2\t\n",
            "TESMIN\t\n",
            "OR10A4\t\n",
            "DEAF1\t   From NCBI Gene:This gene encodes a zinc finger domain-containing protein that functions as a regulator of transcription. The encoded proteins binds to its own promoter as well as to that of several target genes. Activity of this protein is important in the regulation of embryonic development. Mutations in this gene have been found in individuals with autosomal dominant cognitive disability. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2014]From UniProt:Transcription factor that binds to sequence with multiple copies of 5'-TTC[CG]G-3' present in its own promoter and that of the HNRPA2B1 gene. Down-regulates transcription of these genes. Binds to the retinoic acid response element (RARE) 5'-AGGGTTCACCGAAAGTTCA-3'. Activates the proenkephalin gene independently of promoter binding, probably through protein-protein interaction. When secreted, behaves as an inhibitor of cell proliferation, by arresting cells in the G0 or G1 phase. Required for neural tube closure and skeletal patterning. Regulates epithelial cell proliferation and side-branching in the mammary gland. Controls the expression of peripheral tissue antigens in pancreatic lymph nodes. Isoform 1 displays greater transcriptional activity than isoform 4. Isoform 4 may inhibit transcriptional activity of isoform 1 by interacting with isoform 1 and retaining it in the cytoplasm. Transcriptional activator of EIF4G3.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Dyskinesia, seizures, and intellectual developmental disorderMental retardation, autosomal dominant 24From UniProt:Mental retardation, autosomal dominant 24 (MRD24): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. [MIM:615828]Dyskinesia, seizures, and intellectual developmental disorder (DYSEIDD): A neurodevelopmental disorder characterized by psychomotor delay, epilepsy, intellectual disability, speech impairment and dyskinesia of the limbs. Patients also manifest autistic features and other behavioral abnormalities. DYSEIDD transmission pattern is consistent with autosomal recessive inheritance. [MIM:617171]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11p15.5, which is the short (p) arm of chromosome 11 at position 15.5Molecular Location: base pairs 644,220 to 695,754 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MRD24NUDRSPNZMYND5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DEAF1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DEFORMED EPIDERMAL AUTOREGULATORY FACTOR 1 HOMOLOGOMIM: DYSKINESIA, SEIZURES, AND INTELLECTUAL DEVELOPMENTAL DISORDEROMIM: MENTAL RETARDATION, AUTOSOMAL DOMINANT 24      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DEAF1  OMIM: DEFORMED EPIDERMAL AUTOREGULATORY FACTOR 1 HOMOLOGOMIM: DYSKINESIA, SEIZURES, AND INTELLECTUAL DEVELOPMENTAL DISORDEROMIM: MENTAL RETARDATION, AUTOSOMAL DOMINANT 24  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ANO1\t\n",
            "TMEM80\t\n",
            "NLRP14\t\n",
            "KRTAP5-7\t\n",
            "KRTAP5-8\t\n",
            "KRTAP5-10\t\n",
            "P2RY2\t\n",
            "ARHGEF17\t\n",
            "XRRA1\t\n",
            "MYO7A\t   The MYO7A gene provides instructions for making a protein called myosin VIIA, which is part of a group of proteins called unconventional myosins. These proteins, which have similar structures, help transport molecules within cells. Myosins interact with actin, a protein that is important for cell movement and shape. Researchers believe that myosins use long filaments of actin as tracks along which to transport other molecules.Myosin VIIA is made in the inner ear and in the retina, which is the light-sensitive tissue at the back of the eye. In the inner ear, myosin VIIA plays a role in the development and maintenance of hairlike projections called stereocilia. Stereocilia, which are rich in actin, line the inner ear and bend in response to sound waves. This bending motion is critical for converting sound waves to nerve impulses, which are then transmitted to the brain. Stereocilia are also elements of the vestibular system, the part of the inner ear that helps maintain the body's balance and orientation in space. Bending of these stereocilia is needed to transmit signals from the vestibular system to the brain.In the retina, myosin VIIA is found primarily in a thin layer of cells called the retinal pigment epithelium (RPE). Myosin VIIA probably plays a role in the development and maintenance of this tissue, which supports and nourishes the retina. Research suggests that one function of myosin VIIA is to carry small sacs of pigment (called melanosomes) within the RPE. This pigment is necessary for normal vision. Myosin VIIA is also found in other parts of the retina, where it likely carries additional proteins and molecules that are important for vision.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Nonsyndromic hearing loss     Researchers have identified several MYO7A gene mutations that can cause nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene are thought to cause two forms of nonsyndromic hearing loss: DFNA11 and DFNB2.DFNA11 is inherited in an autosomal dominant pattern, which means only one mutated copy of the MYO7A gene in each cell is sufficient to cause the condition. This form of hearing loss begins in childhood, after a child learns to speak (postlingual), and becomes more severe over time.Most of the mutations that cause DFNA11 alter a single protein building block (amino acid) in myosin VIIA, resulting in an abnormal protein that does not work properly. Other genetic changes delete a small amount of DNA from critical regions of the MYO7A gene, which probably changes the structure of the protein. Researchers suspect that the altered protein causes hearing loss by disrupting the growth and organization of stereocilia in the inner ear.DFNB2 is inherited in an autosomal recessive pattern, which means both copies of the MYO7A gene are mutated in each cell. The hearing loss can be postlingual or begin before a child learns to speak (prelingual). Some researchers have suggested that individuals with DFNB2 may actually have Usher syndrome (described below), because some individuals who were thought to have nonsyndromic hearing loss developed retinitis pigmentosa (a vision disorder characteristic of Usher syndrome) later in life. However, other individuals diagnosed with DFNB2 never develop retinitis pigmentosa, and recent studies indicate that DFNB2 and Usher syndrome probably result from different mutations in the MYO7A gene.The mutations that cause DFNB2 alter the structure and function of myosin VIIA, but they probably do not eliminate the protein's function completely. Recent studies found that the protein likely retains enough function in the retina to allow for normal vision, but not enough function in the inner ear to permit normal hearing. More About This Health Condition      Usher syndrome     More than 200 mutations in the MYO7A gene have been identified in people with Usher syndrome type I, which is characterized by a combination of hearing loss, vision loss, and problems with balance and coordination. Specifically, MYO7A gene mutations cause a form of the disorder known as Usher syndrome type IB (USH1B), which accounts for more than half of all cases of Usher syndrome type I.Many of these genetic changes alter a single protein building block (amino acid) in critical regions of the myosin VIIA protein. Other mutations introduce a premature stop signal in the instructions for making myosin VIIA. Still other mutations insert or delete small amounts of DNA in the MYO7A gene. All of these changes lead to the production of a nonfunctional version of myosin VIIA or prevent the production of any of this protein. A lack of myosin VIIA in the inner ear disrupts the normal development and function of stereocilia, which leads to hearing loss and difficulty with balance and coordination. A lack of myosin VIIA in the retina causes retinitis pigmentosa, a condition in which light-sensing cells of the retina gradually deteriorate, resulting in progressive vision loss. More About This Health Condition      Age-related hearing loss     Genetics Home Reference provides information about age-related hearing loss. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Researchers have identified several MYO7A gene mutations that can cause nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene are thought to cause two forms of nonsyndromic hearing loss: DFNA11 and DFNB2.DFNA11 is inherited in an autosomal dominant pattern, which means only one mutated copy of the MYO7A gene in each cell is sufficient to cause the condition. This form of hearing loss begins in childhood, after a child learns to speak (postlingual), and becomes more severe over time.Most of the mutations that cause DFNA11 alter a single protein building block (amino acid) in myosin VIIA, resulting in an abnormal protein that does not work properly. Other genetic changes delete a small amount of DNA from critical regions of the MYO7A gene, which probably changes the structure of the protein. Researchers suspect that the altered protein causes hearing loss by disrupting the growth and organization of stereocilia in the inner ear.DFNB2 is inherited in an autosomal recessive pattern, which means both copies of the MYO7A gene are mutated in each cell. The hearing loss can be postlingual or begin before a child learns to speak (prelingual). Some researchers have suggested that individuals with DFNB2 may actually have Usher syndrome (described below), because some individuals who were thought to have nonsyndromic hearing loss developed retinitis pigmentosa (a vision disorder characteristic of Usher syndrome) later in life. However, other individuals diagnosed with DFNB2 never develop retinitis pigmentosa, and recent studies indicate that DFNB2 and Usher syndrome probably result from different mutations in the MYO7A gene.The mutations that cause DFNB2 alter the structure and function of myosin VIIA, but they probably do not eliminate the protein's function completely. Recent studies found that the protein likely retains enough function in the retina to allow for normal vision, but not enough function in the inner ear to permit normal hearing. More About This Health Condition  More than 200 mutations in the MYO7A gene have been identified in people with Usher syndrome type I, which is characterized by a combination of hearing loss, vision loss, and problems with balance and coordination. Specifically, MYO7A gene mutations cause a form of the disorder known as Usher syndrome type IB (USH1B), which accounts for more than half of all cases of Usher syndrome type I.Many of these genetic changes alter a single protein building block (amino acid) in critical regions of the myosin VIIA protein. Other mutations introduce a premature stop signal in the instructions for making myosin VIIA. Still other mutations insert or delete small amounts of DNA in the MYO7A gene. All of these changes lead to the production of a nonfunctional version of myosin VIIA or prevent the production of any of this protein. A lack of myosin VIIA in the inner ear disrupts the normal development and function of stereocilia, which leads to hearing loss and difficulty with balance and coordination. A lack of myosin VIIA in the retina causes retinitis pigmentosa, a condition in which light-sensing cells of the retina gradually deteriorate, resulting in progressive vision loss. More About This Health Condition  Genetics Home Reference provides information about age-related hearing loss. More About This Health Condition    Cytogenetic Location: 11q13.5, which is the long (q) arm of chromosome 11 at position 13.5Molecular Location: base pairs 77,128,192 to 77,215,241 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNA11DFNB2MYO7A_HUMANmyosin VIIA (Usher syndrome 1B (autosomal recessive, severe))NSRD2USH1B     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Neuroscience (second edition, 2001): Hair Cells and the Mechanoelectrical Transduction of Sound WavesNeuroscience (second edition, 2001): The RetinaThe Cell: A Molecular Approach (second edition, 2000): Unconventional Myosins      Clinical Information from GeneReviews (2 links)     Hereditary Hearing Loss and Deafness OverviewUsher Syndrome Type I      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYO7A      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     MYOSIN VIIA      Research Resources (9 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportLeiden Open Variation Database: MYO7A Gene MutationsMonarch InitiativeNCBI GeneRetNet: Summaries of Genes and Loci Causing Retinal Diseases: MYO7AThe Hereditary Hearing Loss HomepageUniProt      Neuroscience (second edition, 2001): Hair Cells and the Mechanoelectrical Transduction of Sound WavesNeuroscience (second edition, 2001): The RetinaThe Cell: A Molecular Approach (second edition, 2000): Unconventional Myosins  Hereditary Hearing Loss and Deafness OverviewUsher Syndrome Type I  Tests of MYO7A  PubMed  MYOSIN VIIA  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportLeiden Open Variation Database: MYO7A Gene MutationsMonarch InitiativeNCBI GeneRetNet: Summaries of Genes and Loci Causing Retinal Diseases: MYO7AThe Hereditary Hearing Loss HomepageUniProt  Hildebrand MS, Thorne NP, Bromhead CJ, Kahrizi K, Webster JA, Fattahi Z, Bataejad M, Kimberling WJ, Stephan D, Najmabadi H, Bahlo M, Smith RJ. Variable hearing impairment in a DFNB2 family with a novel MYO7A missense mutation. Clin Genet. 2010 Jun;77(6):563-71. doi: 10.1111/j.1399-0004.2009.01344.x. Epub 2010 Feb 4. Citation on PubMed or Free article on PubMed CentralLenassi E, Saihan Z, Cipriani V, Le Quesne Stabej P, Moore AT, Luxon LM, Bitner-Glindzicz M, Webster AR. Natural history and retinal structure in patients with Usher syndrome type 1 owing to MYO7A mutation. Ophthalmology. 2014 Feb;121(2):580-7. doi: 10.1016/j.ophtha.2013.09.017. Epub 2013 Nov 5. Citation on PubMedLiu XZ, Walsh J, Mburu P, Kendrick-Jones J, Cope MJ, Steel KP, Brown SD. Mutations in the myosin VIIA gene cause non-syndromic recessive deafness. Nat Genet. 1997 Jun;16(2):188-90. Citation on PubMedOMIM: MYOSIN VIIARiazuddin S, Nazli S, Ahmed ZM, Yang Y, Zulfiqar F, Shaikh RS, Zafar AU, Khan SN, Sabar F, Javid FT, Wilcox ER, Tsilou E, Boger ET, Sellers JR, Belyantseva IA, Riazuddin S, Friedman TB. Mutation spectrum of MYO7A and evaluation of a novel nonsyndromic deafness DFNB2 allele with residual function. Hum Mutat. 2008 Apr;29(4):502-11. doi: 10.1002/humu.20677. Citation on PubMedRong W, Chen X, Zhao K, Liu Y, Liu X, Ha S, Liu W, Kang X, Sheng X, Zhao C. Novel and recurrent MYO7A mutations in Usher syndrome type 1 and type 2. PLoS One. 2014 May 15;9(5):e97808. doi: 10.1371/journal.pone.0097808. eCollection 2014. Citation on PubMed or Free article on PubMed CentralWeil D, Blanchard S, Kaplan J, Guilford P, Gibson F, Walsh J, Mburu P, Varela A, Levilliers J, Weston MD, et al. Defective myosin VIIA gene responsible for Usher syndrome type 1B. Nature. 1995 Mar 2;374(6517):60-1. Citation on PubMedWeil D, Kssel P, Blanchard S, Lvy G, Levi-Acobas F, Drira M, Ayadi H, Petit C. The autosomal recessive isolated deafness, DFNB2, and the Usher 1B syndrome are allelic defects of the myosin-VIIA gene. Nat Genet. 1997 Jun;16(2):191-3. Citation on PubMedWilliams DS, Lopes VS. The many different cellular functions of MYO7A in the retina. Biochem Soc Trans. 2011 Oct;39(5):1207-10. doi: 10.1042/BST0391207. Review. Citation on PubMed or Free article on PubMed Central \n",
            "GDPD4\t\n",
            "ALG8\t   From NCBI Gene:This gene encodes a member of the ALG6/ALG8 glucosyltransferase family. The encoded protein catalyzes the addition of the second glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation of proteins. Mutations in this gene have been associated with congenital disorder of glycosylation type Ih (CDG-Ih). Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]From UniProt:Adds the second glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation. Transfers glucose from dolichyl phosphate glucose (Dol-P-Glc) onto the lipid-linked oligosaccharide Glc(1)Man(9)GlcNAc(2)-PP-Dol before it is transferred to the nascent peptide (By similarity). Required for PKD1/Polycystin-1 maturation and localization to the plasma membrane of the primary cilia (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Congenital disorder of glycosylation type 1HPOLYCYSTIC LIVER DISEASE 3 WITH OR WITHOUT KIDNEY CYSTSFrom UniProt:Congenital disorder of glycosylation 1H (CDG1H): A form of congenital disorder of glycosylation, a multisystem disorder caused by a defect in glycoprotein biosynthesis and characterized by under-glycosylated serum glycoproteins. Congenital disorders of glycosylation result in a wide variety of clinical features, such as defects in the nervous system development, psychomotor retardation, dysmorphic features, hypotonia, coagulation disorders, and immunodeficiency. The broad spectrum of features reflects the critical role of N-glycoproteins during embryonic development, differentiation, and maintenance of cell functions. [MIM:608104]Polycystic liver disease 3 with or without kidney cysts (PCLD3): A form of polycystic liver disease, an autosomal dominant hepatobiliary disease characterized by overgrowth of biliary epithelium and supportive connective tissue, resulting in multiple liver cysts. PCLD3 patients may also develop kidney cysts that usually do not result in clinically significant renal disease. [MIM:617874]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q14.1, which is the long (q) arm of chromosome 11 at position 14.1Molecular Location: base pairs 78,100,942 to 78,139,653 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDG1HPCLD3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ALG8      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ALG8, S. CEREVISIAE, HOMOLOG OFOMIM: CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IhOMIM: POLYCYSTIC LIVER DISEASE 3 WITH OR WITHOUT KIDNEY CYSTS      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ALG8  OMIM: ALG8, S. CEREVISIAE, HOMOLOG OFOMIM: CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IhOMIM: POLYCYSTIC LIVER DISEASE 3 WITH OR WITHOUT KIDNEY CYSTS  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PNPLA2\t   The PNPLA2 gene provides instructions for making an enzyme called adipose triglyceride lipase (ATGL).  The ATGL enzyme plays a role in breaking down fats called triglycerides, which are the main source of stored energy in cells.  Triglycerides are the major component of cell structures called lipid droplets (also called adiposomes).  The ATGL enzyme is found on the surface of lipid droplets. When activated, the ATGL enzyme breaks down triglycerides to provide energy for the body.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Neutral lipid storage disease with myopathy     At least five mutations in the PNPLA2 gene have been found to cause neutral lipid storage disease with myopathy.  Some of these mutations cause the enzyme to function abnormally.  Other mutations prevent the enzyme from ever reaching lipid droplets, so it is unable to interact with triglycerides.  Any disruption in the breakdown of triglycerides leads to the accumulation of these fats in muscle and other tissues throughout the body, resulting in the signs and symptoms of neutral lipid storage disease with myopathy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least five mutations in the PNPLA2 gene have been found to cause neutral lipid storage disease with myopathy.  Some of these mutations cause the enzyme to function abnormally.  Other mutations prevent the enzyme from ever reaching lipid droplets, so it is unable to interact with triglycerides.  Any disruption in the breakdown of triglycerides leads to the accumulation of these fats in muscle and other tissues throughout the body, resulting in the signs and symptoms of neutral lipid storage disease with myopathy. More About This Health Condition    Cytogenetic Location: 11p15.5, which is the short (p) arm of chromosome 11 at position 15.5Molecular Location: base pairs 818,890 to 825,573 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        adipose triglyceride lipaseATGLdesnutrinFP17548patatin-like phospholipase domain containing 2PLPL2_HUMANtransport-secretion protein 2.2triglyceride hydrolaseTTS-2.2TTS2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): The Utilization of Fatty Acids as Fuel Requires Three Stages of Processing      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PNPLA2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PATATIN-LIKE PHOSPHOLIPASE DOMAIN-CONTAINING PROTEIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): The Utilization of Fatty Acids as Fuel Requires Three Stages of Processing  Tests of PNPLA2  PubMed  PATATIN-LIKE PHOSPHOLIPASE DOMAIN-CONTAINING PROTEIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Arner P, Langin D. The role of neutral lipases in human adipose tissue lipolysis. Curr Opin Lipidol. 2007 Jun;18(3):246-50. Review. Citation on PubMedFischer J, Lefvre C, Morava E, Mussini JM, Lafort P, Negre-Salvayre A, Lathrop M, Salvayre R. The gene encoding adipose triglyceride lipase (PNPLA2) is mutated in neutral lipid storage disease with myopathy. Nat Genet. 2007 Jan;39(1):28-30. Epub 2006 Dec 24. Citation on PubMedKobayashi K, Inoguchi T, Maeda Y, Nakashima N, Kuwano A, Eto E, Ueno N, Sasaki S, Sawada F, Fujii M, Matoba Y, Sumiyoshi S, Kawate H, Takayanagi R. The lack of the C-terminal domain of adipose triglyceride lipase causes neutral lipid storage disease through impaired interactions with lipid droplets. J Clin Endocrinol Metab. 2008 Jul;93(7):2877-84. doi: 10.1210/jc.2007-2247. Epub 2008 Apr 29. Citation on PubMedOMIM: PATATIN-LIKE PHOSPHOLIPASE DOMAIN-CONTAINING PROTEIN 2Schweiger M, Schoiswohl G, Lass A, Radner FP, Haemmerle G, Malli R, Graier W, Cornaciu I, Oberer M, Salvayre R, Fischer J, Zechner R, Zimmermann R. The C-terminal region of human adipose triglyceride lipase affects enzyme activity and lipid droplet binding. J Biol Chem. 2008 Jun 20;283(25):17211-20. doi: 10.1074/jbc.M710566200. Epub 2008 Apr 29. Citation on PubMedSmirnova E, Goldberg EB, Makarova KS, Lin L, Brown WJ, Jackson CL. ATGL has a key role in lipid droplet/adiposome degradation in mammalian cells. EMBO Rep. 2006 Jan;7(1):106-13. Citation on PubMed or Free article on PubMed Central \n",
            "SYTL2\t\n",
            "TRIM66\t\n",
            "TYR\t   The TYR gene provides instructions for making an enzyme called tyrosinase.  This enzyme is located in melanocytes, which are specialized cells that produce a pigment called melanin. Melanin is the substance that gives skin, hair, and eyes their color. Melanin is also found in the light-sensitive tissue at the back of the eye (the retina), where it plays a role in normal vision.Tyrosinase is responsible for the first step in melanin production.  It converts a protein building block (amino acid) called tyrosine to another compound called dopaquinone.  A series of additional chemical reactions convert dopaquinone to melanin in the skin, hair follicles, the colored part of the eye (the iris), and the retina.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Oculocutaneous albinism     More than 100 mutations in the TYR gene have been identified in people with oculocutaneous albinism type 1.  These mutations disrupt the normal production of melanin, which reduces coloring of the hair, skin, and eyes and causes problems with vision. Most TYR mutations eliminate the activity of tyrosinase, preventing melanocytes from producing any melanin throughout life.  These mutations cause a form of oculocutaneous albinism called type 1A (OCA1A).  People with this form of albinism have white hair, light-colored eyes, and very pale skin that does not tan. Other mutations in the TYR gene reduce but do not eliminate tyrosinase activity.  These mutations, which allow some melanin to be produced, cause oculocutaneous albinism type 1B (OCA1B).  People with type 1B are also born with white hair, light-colored eyes, and pale skin, but hair and eye color often darken over time and skin may tan. More About This Health Condition      Melanoma     Genetics Home Reference provides information about melanoma. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 100 mutations in the TYR gene have been identified in people with oculocutaneous albinism type 1.  These mutations disrupt the normal production of melanin, which reduces coloring of the hair, skin, and eyes and causes problems with vision. Most TYR mutations eliminate the activity of tyrosinase, preventing melanocytes from producing any melanin throughout life.  These mutations cause a form of oculocutaneous albinism called type 1A (OCA1A).  People with this form of albinism have white hair, light-colored eyes, and very pale skin that does not tan. Other mutations in the TYR gene reduce but do not eliminate tyrosinase activity.  These mutations, which allow some melanin to be produced, cause oculocutaneous albinism type 1B (OCA1B).  People with type 1B are also born with white hair, light-colored eyes, and pale skin, but hair and eye color often darken over time and skin may tan. More About This Health Condition  Genetics Home Reference provides information about melanoma. More About This Health Condition    Cytogenetic Location: 11q14.3, which is the long (q) arm of chromosome 11 at position 14.3Molecular Location: base pairs 89,177,565 to 89,295,759 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LB24-ABMonophenol monooxygenaseOCA1AOCAIASK29-ABTumor Rejection Antigen ABTYRO_HUMAN     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Oculocutaneous Albinism Type 1      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TYR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TYROSINASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Oculocutaneous Albinism Type 1  Tests of TYR  PubMed  TYROSINASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  King RA, Pietsch J, Fryer JP, Savage S, Brott MJ, Russell-Eggitt I, Summers CG, Oetting WS. Tyrosinase gene mutations in oculocutaneous albinism 1 (OCA1): definition of the phenotype. Hum Genet. 2003 Nov;113(6):502-13. Epub 2003 Sep 10. Citation on PubMedLewis RA. Oculocutaneous Albinism Type 1. 2000 Jan 19 [updated 2013 May 16]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1166/ Citation on PubMedMurisier F, Beermann F. Genetics of pigment cells: lessons from the tyrosinase gene family. Histol Histopathol. 2006 May;21(5):567-78. Review. Citation on PubMedOetting WS, Fryer JP, Shriram S, King RA. Oculocutaneous albinism type 1: the last 100 years. Pigment Cell Res. 2003 Jun;16(3):307-11. Review. Citation on PubMedOetting WS. The tyrosinase gene and oculocutaneous albinism type 1 (OCA1): A model for understanding the molecular biology of melanin formation. Pigment Cell Res. 2000 Oct;13(5):320-5. Review. Citation on PubMedOpitz S, Ksmann-Kellner B, Kaufmann M, Schwinger E, Zhlke C. Detection of 53 novel DNA variations within the tyrosinase gene and accumulation of mutations in 17 patients with albinism. Hum Mutat. 2004 Jun;23(6):630-1. Citation on PubMed \n",
            "NOX4\t\n",
            "AKIP1\t\n",
            "FOLH1B\t\n",
            "TRIM49\t\n",
            "TRIM64B\t\n",
            "NAALAD2\t\n",
            "SCUBE2\t\n",
            "MRE11\t   From NCBI Gene:This gene encodes a nuclear protein involved in homologous recombination, telomere length maintenance, and DNA double-strand break repair. By itself, the protein has 3' to 5' exonuclease activity and endonuclease activity. The protein forms a complex with the RAD50 homolog; this complex is required for nonhomologous joining of DNA ends and possesses increased single-stranded DNA endonuclease and 3' to 5' exonuclease activities. In conjunction with a DNA ligase, this protein promotes the joining of noncomplementary ends in vitro using short homologies near the ends of the DNA fragments. This gene has a pseudogene on chromosome 3. Alternative splicing of this gene results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]From UniProt:Component of the MRN complex, which plays a central role in double-strand break (DSB) repair, DNA recombination, maintenance of telomere integrity and meiosis (PubMed:9651580, PubMed:9590181, PubMed:9705271, PubMed:11741547, PubMed:29670289). The complex possesses single-strand endonuclease activity and double-strand-specific 3'-5' exonuclease activity, which are provided by MRE11 (PubMed:9651580, PubMed:9590181, PubMed:9705271, PubMed:11741547, PubMed:29670289). RAD50 may be required to bind DNA ends and hold them in close proximity (PubMed:9651580, PubMed:9590181, PubMed:9705271, PubMed:11741547, PubMed:29670289). This could facilitate searches for short or long regions of sequence homology in the recombining DNA templates, and may also stimulate the activity of DNA ligases and/or restrict the nuclease activity of MRE11 to prevent nucleolytic degradation past a given point (PubMed:9651580, PubMed:9590181, PubMed:9705271, PubMed:11741547, PubMed:29670289, PubMed:30612738). The complex may also be required for DNA damage signaling via activation of the ATM kinase (PubMed:15064416). In telomeres the MRN complex may modulate t-loop formation (PubMed:10888888).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Ovarian cancerBreast cancerFrom NCBI Gene:Ataxia-telangiectasia-like disorder 1From UniProt:Ataxia-telangiectasia-like disorder 1 (ATLD1): A rare disorder characterized by progressive cerebellar ataxia, dysarthria, abnormal eye movements, and absence of telangiectasia. ATLD patients show normal levels of total IgG, IgA and IgM, although there may be reduced levels of specific functional antibodies. At the cellular level, ATLD exhibits hypersensitivity to ionizing radiation and radioresistant DNA synthesis. [MIM:604391]Defects in MRE11 can be a cause of nephronophthisis-related ciliopathies (NPHP-RC), a group of recessive diseases that affect kidney, retina and brain. A homozygous truncating mutation MRE11 has been found in patients with cerebellar vermis hypoplasia, ataxia and dysarthria.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q21, which is the long (q) arm of chromosome 11 at position 21Molecular Location: base pairs 94,415,570 to 94,512,701 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATLDHNGS1MRE11AMRE11B     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MRE11      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ATAXIA-TELANGIECTASIA-LIKE DISORDER 1OMIM: MEIOTIC RECOMBINATION 11, S. CEREVISIAE, HOMOLOG OF, A      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MRE11  OMIM: ATAXIA-TELANGIECTASIA-LIKE DISORDER 1OMIM: MEIOTIC RECOMBINATION 11, S. CEREVISIAE, HOMOLOG OF, A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "C11orf97\t\n",
            "SRSF8\t\n",
            "ENDOD1\t\n",
            "MAML2\t   From NCBI Gene:The protein encoded by this gene is a member of the Mastermind-like family of proteins. All family members are proline and glutamine-rich, and contain a conserved basic domain that binds the ankyrin repeat domain of the intracellular domain of the Notch receptors (ICN1-4) in their N-terminus, and a transcriptional activation domain in their C-terminus. This protein binds to an extended groove that is formed by the interaction of CBF1, Suppressor of Hairless, LAG-1 (CSL) with ICN, and positively regulates Notch signaling. High levels of expression of this gene have been observed in several B cell-derived lymphomas. Translocations resulting in fusion proteins with both CRTC1 and CRTC3 have been implicated in the development of mucoepidermoid carcinomas, while a translocation event with CXCR4 has been linked with chronic lymphocytic leukemia (CLL). Copy number variation in the polyglutamine tract has been observed. [provided by RefSeq, Jan 2015]From UniProt:Acts as a transcriptional coactivator for NOTCH proteins. Has been shown to amplify NOTCH-induced transcription of HES1. Potentiates activation by NOTCH3 and NOTCH4 more efficiently than MAML1 or MAML3.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving MAML2 is found in mucoepidermoid carcinomas, benign Warthin tumors and clear cell hidradenomas. Translocation t(11;19)(q21;p13) with CRTC1. The fusion protein consists of the N-terminus of CRTC1 joined to the C-terminus of MAML2. The reciprocal fusion protein consisting of the N-terminus of MAML2 joined to the C-terminus of CRTC1 has been detected in a small number of mucoepidermoid carcinomas.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 11q21, which is the long (q) arm of chromosome 11 at position 21Molecular Location: base pairs 95,976,593 to 96,343,308 on chromosome 11 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MAM-3MAM2MAM3MLL-MAML2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MAML2      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MASTERMIND-LIKE 2      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MAML2  OMIM: MASTERMIND-LIKE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "WEE1\t\n",
            "SCYL2\t\n",
            "NR1H4\t   From NCBI Gene:This gene encodes a ligand-activated transcription factor that shares structural features in common with nuclear hormone receptor family members. This protein functions as a receptor for bile acids, and when bound to bile acids, binds to DNA and regulates the expression of genes involved in bile acid synthesis and transport. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Feb 2016]From UniProt:Isoform 1: Promotes transcriptional activation of target genes NR0B2/SHP (inducible by unconjugated CDCA), SLC51B/OSTB (inducible by unconjugated CDCA and DCA) and FABP6/IBAP; low activity for ABCB11/BSEP (inducible by unconjugated CDCA, DCA and ACA); not inducible by taurine- and glycine-amidated CDCA.Isoform 4: Promotes transcriptional activation of target genes ABCB11/BSEP (inducible by unconjugated CDCA, ACA and DCA), NR0B2/SHP (inducible by unconjugated CDCA, ACA and DCA), SLC51B/OSTB (inducible by unconjugated CDCA and DCA) and FABP6/IBAP; most efficient isoform compared to isoforms 1 to 3; not inducible by taurine- and glycine-amidated CDCA.Ligand-activated transcription factor. Receptor for bile acids (BAs) such as chenodeoxycholic acid (CDCA), lithocholic acid, deoxycholic acid (DCA) and allocholic acid (ACA). Plays a essential role in BA homeostasis through the regulation of genes involved in BA synthesis, conjugation and enterohepatic circulation. Also regulates lipid and glucose homeostasis and is involved innate immune response (PubMed:10334992, PubMed:10334993, PubMed:21383957, PubMed:22820415). The FXR-RXR heterodimer binds predominantly to farnesoid X receptor response elements (FXREs) containing two inverted repeats of the consensus sequence 5'-AGGTCA-3' in which the monomers are spaced by 1 nucleotide (IR-1) but also to tandem repeat DR1 sites with lower affinity, and can be activated by either FXR or RXR-specific ligands. It is proposed that monomeric nuclear receptors such as NR5A2/LRH-1 bound to coregulatory nuclear responsive element (NRE) halfsites located in close proximity to FXREs modulate transcriptional activity (By similarity). In the liver activates transcription of the corepressor NR0B2 thereby indirectly inhibiting CYP7A1 and CYP8B1 (involved in BA synthesis) implicating at least in part histone demethylase KDM1A resulting in epigenomic repression, and SLC10A1/NTCP (involved in hepatic uptake of conjugated BAs). Activates transcription of the repressor MAFG (involved in regulation of BA synthesis) (By similarity). Activates transcription of SLC27A5/BACS and BAAT (involved in BA conjugation), ABCB11/BSEP (involved in bile salt export) by directly recruiting histone methyltransferase CARM1, and ABCC2/MRP2 (involved in secretion of conjugated BAs) and ABCB4 (involved in secretion of phosphatidylcholine in the small intestine) (PubMed:12754200, PubMed:15471871, PubMed:17895379). Activates transcription of SLC27A5/BACS and BAAT (involved in BA conjugation), ABCB11/BSEP (involved in bile salt export) by directly recruiting histone methyltransferase CARM1, and ABCC2/MRP2 (involved in secretion of conjugated BAs) and ABCB4 (involved in secretion of phosphatidylcholine in the small intestine) (PubMed:10514450, PubMed:15239098, PubMed:16269519). In the intestine activates FGF19 expression and secretion leading to hepatic CYP7A1 repression (PubMed:12815072, PubMed:19085950). The function also involves the coordinated induction of hepatic KLB/beta-klotho expression (By similarity). Regulates transcription of liver UGT2B4 and SULT2A1 involved in BA detoxification; binding to the UGT2B4 promoter seems to imply a monomeric transactivation independent of RXRA (PubMed:12806625, PubMed:16946559). Modulates lipid homeostasis by activating liver NR0B2/SHP-mediated repression of SREBF1 (involved in de novo lipogenesis), expression of PLTP (involved in HDL formation), SCARB1 (involved in HDL hepatic uptake), APOE, APOC1, APOC4, PPARA (involved in beta-oxidation of fatty acids), VLDLR and SDC1 (involved in the hepatic uptake of LDL and IDL remnants), and inhibiting expression of MTTP (involved in VLDL assembly (PubMed:12660231, PubMed:12554753, PubMed:15337761). Increases expression of APOC2 (promoting lipoprotein lipase activity implicated in triglyceride clearance) (PubMed:11579204). Transrepresses APOA1 involving a monomeric competition with NR2A1 for binding to a DR1 element (PubMed:11927623, PubMed:21804189). Also reduces triglyceride clearance by inhibiting expression of ANGPTL3 and APOC3 (both involved in inhibition of lipoprotein lipase) (PubMed:12891557). Involved in glucose homeostasis by modulating hepatic gluconeogenesis through activation of NR0B2/SHP-mediated repression of respective genes. Modulates glycogen synthesis (inducing phosphorylation of glycogen synthase kinase-3) (By similarity). Modulates glucose-stimulated insulin secretion and is involved in insulin resistance (PubMed:20447400). Involved in intestinal innate immunity. Plays a role in protecting the distal small intestine against bacterial overgrowth and preservation of the epithelial barrier (By similarity). Down-regulates inflammatory cytokine expression in several types of immune cells including macrophages and mononuclear cells (PubMed:21242261). Mediates trans-repression of TLR4-induced cytokine expression; the function seems to require its sumoylation and prevents N-CoR nuclear receptor corepressor clearance from target genes such as IL1B and NOS2 (PubMed:19864602). Involved in the TLR9-mediated protective mechanism in intestinal inflammation. Plays an anti-inflammatory role in liver inflammation; proposed to inhibit proinflammatory (but not antiapoptotic) NF-kappa-B signaling) (By similarity).Isoform 2: Promotes transcriptional activation of target genes ABCB11/BSEP (inducible by unconjugated CDCA, DCA and ACA), NR0B2/SHP (inducible by unconjugated CDCA DCA and ACA), SLC51B/OSTB (inducible by unconjugated CDCA and DCA) and FABP6/IBAP; not inducible by taurine- and glycine-amidated CDCA.Isoform 3: Promotes transcriptional activation of target genes NR0B2/SHP (inducible by unconjugated CDCA), SLC51B/OSTB (inducible by unconjugated CDCA and DCA) and IBAP; low activity for ABCB11/BSEP (inducible by unconjugated CDCA, DCA and ACA); not inducible by taurine- and glycine-amidated CDCA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Cholestasis, progressive familial intrahepatic, 5From UniProt:May be involved in cholesterol cholelithiasis.May be involved in intrahepatic cholestasis of pregnancy.Cholestasis, progressive familial intrahepatic, 5 (PFIC5): A disorder characterized by early onset of cholestasis that progresses to hepatic fibrosis, cirrhosis, and end-stage liver disease before adulthood. PFIC5 is an autosomal recessive, severe form characterized by onset of intralobular cholestasis in the neonatal period. [MIM:617049]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q23.1, which is the long (q) arm of chromosome 12 at position 23.1Molecular Location: base pairs 100,473,773 to 100,563,867 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BARFXRHRR-1HRR1PFIC5RIP14     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NR1H4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CHOLESTASIS, PROGRESSIVE FAMILIAL INTRAHEPATIC, 5OMIM: NUCLEAR RECEPTOR SUBFAMILY 1, GROUP H, MEMBER 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NR1H4  OMIM: CHOLESTASIS, PROGRESSIVE FAMILIAL INTRAHEPATIC, 5OMIM: NUCLEAR RECEPTOR SUBFAMILY 1, GROUP H, MEMBER 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ANO4\t\n",
            "CLEC1B\t\n",
            "UTP20\t\n",
            "CHPT1\t\n",
            "ASCL1\t   From NCBI Gene:This gene encodes a member of the basic helix-loop-helix (BHLH) family of transcription factors. The protein activates transcription by binding to the E box (5'-CANNTG-3'). Dimerization with other BHLH proteins is required for efficient DNA binding. This protein plays a role in the neuronal commitment and differentiation and in the generation of olfactory and autonomic neurons. Mutations in this gene may contribute to the congenital central hypoventilation syndrome (CCHS) phenotype in rare cases. [provided by RefSeq, Jul 2008]From UniProt:Transcription factor that plays a key role in neuronal differentiation: acts as a pioneer transcription factor, accessing closed chromatin to allow other factors to bind and activate neural pathways. Directly binds the E box motif (5'-CANNTG-3') on promoters and promotes transcription of neuronal genes. The combination of three transcription factors, ASCL1, POU3F2/BRN2 and MYT1L, is sufficient to reprogram fibroblasts and other somatic cells into induced neuronal (iN) cells in vitro. Plays a role at early stages of development of specific neural lineages in most regions of the CNS, and of several lineages in the PNS. Essential for the generation of olfactory and autonomic neurons. Acts synergistically with FOXN4 to specify the identity of V2b neurons rather than V2a from bipotential p2 progenitors during spinal cord neurogenesis, probably through DLL4-NOTCH signaling activation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Congenital central hypoventilation     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q23.2, which is the long (q) arm of chromosome 12 at position 23.2Molecular Location: base pairs 102,957,674 to 102,960,516 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ASH1bHLHa46HASH1MASH1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ASCL1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ACHAETE-SCUTE COMPLEX, DROSOPHILA, HOMOLOG OF, 1OMIM: CENTRAL HYPOVENTILATION SYNDROME, CONGENITAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ASCL1  OMIM: ACHAETE-SCUTE COMPLEX, DROSOPHILA, HOMOLOG OF, 1OMIM: CENTRAL HYPOVENTILATION SYNDROME, CONGENITAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KLRC3\t\n",
            "KLRC2\t\n",
            "CKAP4\t\n",
            "ISCU\t   The ISCU gene provides instructions for making a protein called the iron-sulfur cluster assembly enzyme. As its name suggests, this enzyme is involved in the formation of clusters of iron and sulfur atoms (Fe-S clusters). Specifically, the enzyme acts as a platform, or scaffold, for the assembly of these clusters. Fe-S clusters are critical for the function of many different proteins, including those needed for DNA repair and the regulation of iron levels. Proteins containing Fe-S clusters are also necessary for energy production within mitochondria, which are the cell structures that convert the energy from food into a form that cells can use.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Myopathy with deficiency of iron-sulfur cluster assembly enzyme     At least two mutations in the ISCU gene have been found to cause myopathy with deficiency of iron-sulfur cluster assembly enzyme. The more common mutation, written as IVS5+382G>C, alters the way the gene's instructions are pieced together to produce the enzyme. Most affected individuals have this mutation in both copies of the ISCU gene in each cell. The other mutation, which has been identified in one family, replaces the amino acid glycine with the amino acid glutamate at position 50 in the enzyme (written as Gly50Glu or G50E). Affected individuals in this family have had the G50E mutation in one copy of the ISCU gene in each cell and the IVS5+382G>C mutation in the other copy of the gene. This combination of mutations causes a severe variant of the disorder characterized by progressive muscle weakness and wasting.Mutations in the ISCU gene severely limit the amount of iron-sulfur cluster assembly enzyme that is made in cells. A shortage of this enzyme prevents the normal production of proteins that contain Fe-S clusters, which disrupts a variety of cellular activities. A reduction in the amount of iron-sulfur cluster assembly enzyme is particularly damaging to skeletal muscle cells. Within the mitochondria of these cells, a lack of this enzyme causes problems with energy production and an overload of iron. These defects lead to muscle weakness, pain, and the other features of myopathy with deficiency of iron-sulfur cluster assembly enzyme. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two mutations in the ISCU gene have been found to cause myopathy with deficiency of iron-sulfur cluster assembly enzyme. The more common mutation, written as IVS5+382G>C, alters the way the gene's instructions are pieced together to produce the enzyme. Most affected individuals have this mutation in both copies of the ISCU gene in each cell. The other mutation, which has been identified in one family, replaces the amino acid glycine with the amino acid glutamate at position 50 in the enzyme (written as Gly50Glu or G50E). Affected individuals in this family have had the G50E mutation in one copy of the ISCU gene in each cell and the IVS5+382G>C mutation in the other copy of the gene. This combination of mutations causes a severe variant of the disorder characterized by progressive muscle weakness and wasting.Mutations in the ISCU gene severely limit the amount of iron-sulfur cluster assembly enzyme that is made in cells. A shortage of this enzyme prevents the normal production of proteins that contain Fe-S clusters, which disrupts a variety of cellular activities. A reduction in the amount of iron-sulfur cluster assembly enzyme is particularly damaging to skeletal muscle cells. Within the mitochondria of these cells, a lack of this enzyme causes problems with energy production and an overload of iron. These defects lead to muscle weakness, pain, and the other features of myopathy with deficiency of iron-sulfur cluster assembly enzyme. More About This Health Condition    Cytogenetic Location: 12q23.3, which is the long (q) arm of chromosome 12 at position 23.3Molecular Location: base pairs 108,561,463 to 108,569,384 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HMLhnifUiron-sulfur cluster scaffold homolog (E. coli)IscUIscU iron-sulfur cluster scaffold homologISCU_HUMANISU2MGC74517NIFUNifU-like N-terminal domain containingNIFUNnitrogen fixation cluster-like     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Cell Biology (fourth edition, 2000): Three-dimensional structures of some iron-sulfur clusters in electron-transporting proteins (figure)      Clinical Information from GeneReviews (1 link)     Myopathy with Deficiency of ISCU      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ISCU      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     IRON-SULFUR CLUSTER SCAFFOLD, E. COLI, HOMOLOG OF      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Three-dimensional structures of some iron-sulfur clusters in electron-transporting proteins (figure)  Myopathy with Deficiency of ISCU  Tests of ISCU  PubMed  IRON-SULFUR CLUSTER SCAFFOLD, E. COLI, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Kollberg G, Tulinius M, Melberg A, Darin N, Andersen O, Holmgren D, Oldfors A, Holme E. Clinical manifestation and a new ISCU mutation in iron-sulphur cluster deficiency myopathy. Brain. 2009 Aug;132(Pt 8):2170-9. doi: 10.1093/brain/awp152. Epub 2009 Jun 30. Citation on PubMedLi K, Tong WH, Hughes RM, Rouault TA. Roles of the mammalian cytosolic cysteine desulfurase, ISCS, and scaffold protein, ISCU, in iron-sulfur cluster assembly. J Biol Chem. 2006 May 5;281(18):12344-51. Epub 2006 Mar 9. Citation on PubMedMochel F, Knight MA, Tong WH, Hernandez D, Ayyad K, Taivassalo T, Andersen PM, Singleton A, Rouault TA, Fischbeck KH, Haller RG. Splice mutation in the iron-sulfur cluster scaffold protein ISCU causes myopathy with exercise intolerance. Am J Hum Genet. 2008 Mar;82(3):652-60. doi: 10.1016/j.ajhg.2007.12.012. Epub 2008 Feb 14. Citation on PubMed or Free article on PubMed CentralOlsson A, Lind L, Thornell LE, Holmberg M. Myopathy with lactic acidosis is linked to chromosome 12q23.3-24.11 and caused by an intron mutation in the ISCU gene resulting in a splicing defect. Hum Mol Genet. 2008 Jun 1;17(11):1666-72. doi: 10.1093/hmg/ddn057. Epub 2008 Feb 23. Citation on PubMedRouault TA, Tong WH. Iron-sulfur cluster biogenesis and human disease. Trends Genet. 2008 Aug;24(8):398-407. doi: 10.1016/j.tig.2008.05.008. Epub 2008 Jul 5. Review. Citation on PubMed or Free article on PubMed CentralTong WH, Rouault TA. Functions of mitochondrial ISCU and cytosolic ISCU in mammalian iron-sulfur cluster biogenesis and iron homeostasis. Cell Metab. 2006 Mar;3(3):199-210. Citation on PubMed \n",
            "ACACB\t\n",
            "TAS2R19\t\n",
            "TAS2R31\t\n",
            "ATXN2\t   The ATXN2 gene provides instructions for making a protein called ataxin-2. This protein is found throughout the body, but its function is unknown. Ataxin-2 is found in the fluid inside cells (cytoplasm) and seems to interact with a cell structure called the endoplasmic reticulum. The endoplasmic reticulum is involved in protein production, processing, and transport. Researchers believe that ataxin-2 may be involved in processing RNA, a chemical cousin of DNA. Ataxin-2 is also thought to play a role in the translation of genetic information to produce proteins.One region of the ATXN2 gene contains a DNA segment known as a CAG trinucleotide repeat. This segment is made up of a series of three DNA building blocks (cytosine, adenine, and guanine) that appear multiple times in a row. Normally, the CAG segment is repeated approximately 22 times within the gene.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Spinocerebellar ataxia type 2     Spinocerebellar ataxia type 2 (SCA2) is a condition characterized by progressive problems with movement. SCA2 results from a mutation in the ATXN2 gene known as a trinucleotide repeat expansion. This mutation increases the length of the repeated CAG segment in the ATXN2 gene. People with 32 or more repeats CAG repeats in the ATXN2 gene develop SCA2.It is unclear how the abnormally long CAG segment affects the function of the ataxin-2 protein. The abnormal protein apparently leads to cell death, as people with SCA2 show a loss of brain cells. Certain brain cells called Purkinje cells seem to be particularly sensitive to the presence of abnormal ataxin-2. Purkinje cells are located in the part of the brain that coordinates movement (cerebellum) and are involved in chemical signaling between nerve cells (neurons). It is unknown how the abnormal ataxin-2 protein leads to the death of Purkinje and other brain cells. Over time, the loss of these cells causes the movement problems characteristic of SCA2. More About This Health Condition      Amyotrophic lateral sclerosis     Genetics Home Reference provides information about amyotrophic lateral sclerosis. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Spinocerebellar ataxia type 2 (SCA2) is a condition characterized by progressive problems with movement. SCA2 results from a mutation in the ATXN2 gene known as a trinucleotide repeat expansion. This mutation increases the length of the repeated CAG segment in the ATXN2 gene. People with 32 or more repeats CAG repeats in the ATXN2 gene develop SCA2.It is unclear how the abnormally long CAG segment affects the function of the ataxin-2 protein. The abnormal protein apparently leads to cell death, as people with SCA2 show a loss of brain cells. Certain brain cells called Purkinje cells seem to be particularly sensitive to the presence of abnormal ataxin-2. Purkinje cells are located in the part of the brain that coordinates movement (cerebellum) and are involved in chemical signaling between nerve cells (neurons). It is unknown how the abnormal ataxin-2 protein leads to the death of Purkinje and other brain cells. Over time, the loss of these cells causes the movement problems characteristic of SCA2. More About This Health Condition  Genetics Home Reference provides information about amyotrophic lateral sclerosis. More About This Health Condition    Cytogenetic Location: 12q24.12, which is the long (q) arm of chromosome 12 at position 24.12Molecular Location: base pairs 111,452,214 to 111,599,676 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ataxin-2ATX2ATX2_HUMANSCA2     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Basic Neurochemistry (sixth edition, 1999): Trinucleotide expansion is one of the most important categories of mutation underlying neurodegenerative disordersWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Spinocerebellar Ataxia Type 2      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATXN2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATAXIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Trinucleotide expansion is one of the most important categories of mutation underlying neurodegenerative disordersWashington University, St. Louis: Neuromuscular Disease Center  Spinocerebellar Ataxia Type 2  Tests of ATXN2  PubMed  ATAXIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ATAXIN 2Lastres-Becker I, Rb U, Auburger G. Spinocerebellar ataxia 2 (SCA2). Cerebellum. 2008;7(2):115-24. doi: 10.1007/s12311-008-0019-y. Review. Citation on PubMedVelzquez Prez L, Cruz GS, Santos Falcn N, Enrique Almaguer Mederos L, Escalona Batallan K, Rodrguez Labrada R, Paneque Herrera M, Laffita Mesa JM, Rodrguez Daz JC, Rodrguez RA, Gonzlez Zaldivar Y, Coello Almarales D, Almaguer Gotay D, Jorge Cedeo H. Molecular epidemiology of spinocerebellar ataxias in Cuba: insights into SCA2 founder effect in Holguin. Neurosci Lett. 2009 Apr 24;454(2):157-60. doi: 10.1016/j.neulet.2009.03.015. Epub 2009 Mar 11. Citation on PubMedvan de Loo S, Eich F, Nonis D, Auburger G, Nowock J. Ataxin-2 associates with rough endoplasmic reticulum. Exp Neurol. 2009 Jan;215(1):110-8. doi: 10.1016/j.expneurol.2008.09.020. Epub 2008 Oct 11. Citation on PubMed \n",
            "TAS2R46\t\n",
            "ACAD10\t\n",
            "TAS2R43\t\n",
            "NAA25\t\n",
            "OAS1\t   From NCBI Gene:This gene is induced by interferons and encodes a protein that synthesizes 2',5'-oligoadenylates (2-5As). This protein activates latent RNase L, which results in viral RNA degradation and the inhibition of viral replication. Alternative splicing results in multiple transcript variants with different enzymatic activities. Polymorphisms in this gene have been associated with susceptibility to viral infection and diabetes mellitus, type 1. A disease-associated allele in a splice acceptor site influences the production of the p46 splice isoform. This gene is located in a cluster of related genes on chromosome 12. [provided by RefSeq, Feb 2016]From UniProt:Interferon-induced, dsRNA-activated antiviral enzyme which plays a critical role in cellular innate antiviral response. In addition, it may also play a role in other cellular processes such as apoptosis, cell growth, differentiation and gene regulation. Synthesizes higher oligomers of 2'-5'-oligoadenylates (2-5A) from ATP which then bind to the inactive monomeric form of ribonuclease L (RNase L) leading to its dimerization and subsequent activation. Activation of RNase L leads to degradation of cellular as well as viral RNA, resulting in the inhibition of protein synthesis, thus terminating viral replication. Can mediate the antiviral effect via the classical RNase L-dependent pathway or an alternative antiviral pathway independent of RNase L. The secreted form displays antiviral effect against vesicular stomatitis virus (VSV), herpes simplex virus type 2 (HSV-2), and encephalomyocarditis virus (EMCV) and stimulates the alternative antiviral pathway independent of RNase L.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Type 1 diabetes     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q24.13, which is the long (q) arm of chromosome 12 at position 24.13Molecular Location: base pairs 112,906,777 to 112,932,190 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        E18/E16IFI-4OIASOIASI     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of OAS1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: 2-PRIME,5-PRIME-OLIGOADENYLATE SYNTHETASE 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of OAS1  OMIM: 2-PRIME,5-PRIME-OLIGOADENYLATE SYNTHETASE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TAS2R42\t\n",
            "OAS2\t\n",
            "PLBD2\t\n",
            "SDSL\t\n",
            "PRB3\t\n",
            "PRB4\t\n",
            "MAP1LC3B2\t\n",
            "FBXW8\t\n",
            "CABP1\t\n",
            "ACADS\t   The ACADS gene provides instructions for making an enzyme called short-chain acyl-CoA dehydrogenase (SCAD). This enzyme functions within mitochondria, the energy-producing centers within cells. SCAD is essential for fatty acid oxidation, which is the multistep process that breaks down (metabolizes) fats and converts them to energy.SCAD is required to metabolize a group of fats called short-chain fatty acids. These fatty acids are found in some foods and are also produced when larger fatty acids are metabolized. Fatty acids are a major source of energy for the heart and muscles. During periods without food (fasting), fatty acids are also an important energy source for the liver and other tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Short-chain acyl-CoA dehydrogenase deficiency     More than 55 mutations in the ACADS gene have been found to cause short-chain acyl-CoA dehydrogenase (SCAD) deficiency. Almost all of these mutations change single protein building blocks (amino acids) in the SCAD enzyme. These mutations prevent the enzyme from properly metabolizing short-chain fatty acids. As a result, these fats are not converted into energy, which can lead to the characteristic signs and symptoms of this disorder, including lack of energy (lethargy), low blood sugar (hypoglycemia), poor muscle tone (hypotonia), and weakness.Researchers have also identified two common variations (polymorphisms) in the ACADS gene that each change one amino acid in the SCAD enzyme. Unlike other changes in the ACADS gene, these polymorphisms do not cause SCAD deficiency but may increase a person's risk of developing this disorder. One of these polymorphisms replaces the amino acid arginine with the amino acid tryptophan at protein position 147 (written as Arg147Trp or R147W). The other polymorphism switches the amino acid glycine with the amino acid serine at protein position 185 (written as Gly185Ser or G185S). Other genetic and environmental factors likely influence the risk of developing SCAD deficiency when a person carries either of these polymorphisms. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 55 mutations in the ACADS gene have been found to cause short-chain acyl-CoA dehydrogenase (SCAD) deficiency. Almost all of these mutations change single protein building blocks (amino acids) in the SCAD enzyme. These mutations prevent the enzyme from properly metabolizing short-chain fatty acids. As a result, these fats are not converted into energy, which can lead to the characteristic signs and symptoms of this disorder, including lack of energy (lethargy), low blood sugar (hypoglycemia), poor muscle tone (hypotonia), and weakness.Researchers have also identified two common variations (polymorphisms) in the ACADS gene that each change one amino acid in the SCAD enzyme. Unlike other changes in the ACADS gene, these polymorphisms do not cause SCAD deficiency but may increase a person's risk of developing this disorder. One of these polymorphisms replaces the amino acid arginine with the amino acid tryptophan at protein position 147 (written as Arg147Trp or R147W). The other polymorphism switches the amino acid glycine with the amino acid serine at protein position 185 (written as Gly185Ser or G185S). Other genetic and environmental factors likely influence the risk of developing SCAD deficiency when a person carries either of these polymorphisms. More About This Health Condition    Cytogenetic Location: 12q24.31, which is the long (q) arm of chromosome 12 at position 24.31Molecular Location: base pairs 120,725,738 to 120,740,008 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACAD3ACADS_HUMANacyl-CoA dehydrogenase, C-2 to C-3 short chainacyl-Coenzyme A dehydrogenase, C-2 to C-3 short chain precursorButyryl-CoA dehydrogenaseButyryl dehydrogenaseSCADUnsaturated acyl-CoA reductase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): The Utilization of Fatty Acids as Fuel Requires Three Stages of Processing      Clinical Information from GeneReviews (1 link)     Short-Chain Acyl-CoA Dehydrogenase Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ACADS      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ACYL-CoA DEHYDROGENASE, SHORT-CHAIN      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): The Utilization of Fatty Acids as Fuel Requires Three Stages of Processing  Short-Chain Acyl-CoA Dehydrogenase Deficiency  Tests of ACADS  PubMed  ACYL-CoA DEHYDROGENASE, SHORT-CHAIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ACYL-CoA DEHYDROGENASE, SHORT-CHAINGregersen N, Andresen BS, Corydon MJ, Corydon TJ, Olsen RK, Bolund L, Bross P. Mutation analysis in mitochondrial fatty acid oxidation defects: Exemplified by acyl-CoA dehydrogenase deficiencies, with special focus on genotype-phenotype relationship. Hum Mutat. 2001 Sep;18(3):169-89. Review. Citation on PubMedJethva R, Bennett MJ, Vockley J. Short-chain acyl-coenzyme A dehydrogenase deficiency. Mol Genet Metab. 2008 Dec;95(4):195-200. doi: 10.1016/j.ymgme.2008.09.007. Epub 2008 Nov 5. Review. Citation on PubMed or Free article on PubMed CentralKim SH, Park HD, Sohn YB, Park SW, Cho SY, Ji S, Kim SJ, Choi EW, Kim CH, Ko AR, Yeau S, Paik KH, Jin DK. Mutations of ACADS gene associated with short-chain acyl-coenzyme A dehydrogenase deficiency. Ann Clin Lab Sci. 2011 Fall;41(1):84-8. Citation on PubMedNagan N, Kruckeberg KE, Tauscher AL, Bailey KS, Rinaldo P, Matern D. The frequency of short-chain acyl-CoA dehydrogenase gene variants in the US population and correlation with the C(4)-acylcarnitine concentration in newborn blood spots. Mol Genet Metab. 2003 Apr;78(4):239-46. Citation on PubMedPedersen CB, Klvraa S, Klvraa A, Stenbroen V, Kjeldsen M, Ensenauer R, Tein I, Matern D, Rinaldo P, Vianey-Saban C, Ribes A, Lehnert W, Christensen E, Corydon TJ, Andresen BS, Vang S, Bolund L, Vockley J, Bross P, Gregersen N. The ACADS gene variation spectrum in 114 patients with short-chain acyl-CoA dehydrogenase (SCAD) deficiency is dominated by missense variations leading to protein misfolding at the cellular level. Hum Genet. 2008 Aug;124(1):43-56. doi: 10.1007/s00439-008-0521-9. Epub 2008 Jun 4. Citation on PubMedSchmidt SP, Corydon TJ, Pedersen CB, Bross P, Gregersen N. Misfolding of short-chain acyl-CoA dehydrogenase leads to mitochondrial fission and oxidative stress. Mol Genet Metab. 2010 Jun;100(2):155-62. doi: 10.1016/j.ymgme.2010.03.009. Epub 2010 Mar 19. Citation on PubMedSchmidt SP, Corydon TJ, Pedersen CB, Vang S, Palmfeldt J, Stenbroen V, Wanders RJ, Ruiter JP, Gregersen N. Toxic response caused by a misfolding variant of the mitochondrial protein short-chain acyl-CoA dehydrogenase. J Inherit Metab Dis. 2011 Apr;34(2):465-75. doi: 10.1007/s10545-010-9255-7. Epub 2010 Dec 18. Citation on PubMed or Free article on PubMed CentralShirao K, Okada S, Tajima G, Tsumura M, Hara K, Yasunaga S, Ohtsubo M, Hata I, Sakura N, Shigematsu Y, Takihara Y, Kobayashi M. Molecular pathogenesis of a novel mutation, G108D, in short-chain acyl-CoA dehydrogenase identified in subjects with short-chain acyl-CoA dehydrogenase deficiency. Hum Genet. 2010 Jun;127(6):619-28. doi: 10.1007/s00439-010-0822-7. Epub 2010 Apr 8. Review. Citation on PubMedYoung SP, Matern D, Gregersen N, Stevens RD, Bali D, Liu HM, Koeberl DD, Millington DS. A comparison of in vitro acylcarnitine profiling methods for the diagnosis of classical and variant short chain acyl-CoA dehydrogenase deficiency. Clin Chim Acta. 2003 Nov;337(1-2):103-13. Citation on PubMedvan Maldegem BT, Waterham HR, Duran M, van der Vlies M, van Woerden CS, Bobu LL, Wanders RJ, Wijburg FA. The 625G>A SCAD gene variant is common but not associated with increased C4-carnitine in newborn blood spots. J Inherit Metab Dis. 2005;28(4):557-62. Citation on PubMed \n",
            "HNF1A\t   From NCBI Gene:The protein encoded by this gene is a transcription factor required for the expression of several liver-specific genes. The encoded protein functions as a homodimer and binds to the inverted palindrome 5'-GTTAATNATTAAC-3'. Defects in this gene are a cause of maturity onset diabetes of the young type 3 (MODY3) and also can result in the appearance of hepatic adenomas. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Apr 2015]From UniProt:Transcriptional activator that regulates the tissue specific expression of multiple genes, especially in pancreatic islet cells and in liver (By similarity). Binds to the inverted palindrome 5'-GTTAATNATTAAC-3' (PubMed:12453420, PubMed:10966642). Activates the transcription of CYP1A2, CYP2E1 and CYP3A11 (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Congenital hyperinsulinismType 1 diabetesFrom NCBI Gene:Maturity-onset diabetes of the young, type 3Diabetes mellitus, insulin-dependent, 20Diabetes mellitus type 2Hepatic adenomas, familialRenal cell carcinoma, nonpapillaryDiabetes mellitus type 1From UniProt:Maturity-onset diabetes of the young 3 (MODY3): A form of diabetes that is characterized by an autosomal dominant mode of inheritance, onset in childhood or early adulthood (usually before 25 years of age), a primary defect in insulin secretion and frequent insulin-independence at the beginning of the disease. [MIM:600496]Hepatic adenomas familial (HEPAF): Rare benign liver tumors of presumable epithelial origin that develop in an otherwise normal liver. Hepatic adenomas may be single or multiple. They consist of sheets of well-differentiated hepatocytes that contain fat and glycogen and can produce bile. Bile ducts or portal areas are absent. Kupffer cells, if present, are reduced in number and are non-functional. Conditions associated with adenomas are insulin-dependent diabetes mellitus and glycogen storage diseases (types 1 and 3). [MIM:142330]Diabetes mellitus, insulin-dependent, 20 (IDDM20): A multifactorial disorder of glucose homeostasis that is characterized by susceptibility to ketoacidosis in the absence of insulin therapy. Clinical features are polydipsia, polyphagia and polyuria which result from hyperglycemia-induced osmotic diuresis and secondary thirst. These derangements result in long-term complications that affect the eyes, kidneys, nerves, and blood vessels. [MIM:612520]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q24.31, which is the long (q) arm of chromosome 12 at position 24.31Molecular Location: base pairs 120,977,683 to 121,002,512 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HNF-1AHNF1HNF4AIDDM20LFB1MODY3TCF-1TCF1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HNF1A      Catalog of Genes and Diseases from OMIM (7 links)     OMIM: DIABETES MELLITUS, INSULIN-DEPENDENTOMIM: DIABETES MELLITUS, INSULIN-DEPENDENT, 20OMIM: DIABETES MELLITUS, NONINSULIN-DEPENDENTOMIM: HEPATIC ADENOMAS, FAMILIALOMIM: HNF1 HOMEOBOX AOMIM: MATURITY-ONSET DIABETES OF THE YOUNG, TYPE 3OMIM: RENAL CELL CARCINOMA, NONPAPILLARY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HNF1A  OMIM: DIABETES MELLITUS, INSULIN-DEPENDENTOMIM: DIABETES MELLITUS, INSULIN-DEPENDENT, 20OMIM: DIABETES MELLITUS, NONINSULIN-DEPENDENTOMIM: HEPATIC ADENOMAS, FAMILIALOMIM: HNF1 HOMEOBOX AOMIM: MATURITY-ONSET DIABETES OF THE YOUNG, TYPE 3OMIM: RENAL CELL CARCINOMA, NONPAPILLARY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "P2RX7\t\n",
            "CAMKK2\t\n",
            "ORAI1\t   From NCBI Gene:The protein encoded by this gene is a membrane calcium channel subunit that is activated by the calcium sensor STIM1 when calcium stores are depleted. This type of channel is the primary way for calcium influx into T-cells. Defects in this gene are a cause of immune dysfunction with T-cell inactivation due to calcium entry defect type 1 (IDTICED1). [provided by RefSeq, Sep 2011]From UniProt:Ca(2+) release-activated Ca(2+) (CRAC) channel subunit which mediates Ca(2+) influx following depletion of intracellular Ca(2+) stores and channel activation by the Ca(2+) sensor, STIM1 (PubMed:16582901, PubMed:16645049, PubMed:16733527, PubMed:16766533, PubMed:16807233, PubMed:19249086, PubMed:23307288, PubMed:24351972, PubMed:24591628, PubMed:28219928). CRAC channels are the main pathway for Ca(2+) influx in T-cells and promote the immune response to pathogens by activating the transcription factor NFAT (PubMed:16582901).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Kidney stonesTubular aggregate myopathyFrom NCBI Gene:Myopathy, tubular aggregate, 2Immune dysfunction with T-cell inactivation due to calcium entry defect 1From UniProt:Myopathy, tubular aggregate, 2 (TAM2): A rare congenital myopathy characterized by regular arrays of membrane tubules on muscle biopsies without additional histopathological hallmarks. Tubular aggregates in muscle are structures of variable appearance consisting of an outer tubule containing either one or more microtubule-like structures or amorphous material. TAM2 patients have myopathy and pupillary abnormalities. [MIM:615883]Immunodeficiency 9 (IMD9): An immune disorder characterized by recurrent infections, impaired activation and proliferative response of T-cells, decreased T-cell production of cytokines, and normal lymphocytes counts and serum immunoglobulin levels. In surviving patients ectodermal dysplasia with anhidrosis and non-progressive myopathy may be observed. [MIM:612782]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q24.31, which is the long (q) arm of chromosome 12 at position 24.31Molecular Location: base pairs 121,626,550 to 121,642,040 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CRACM1IMD9ORAT1TAM2TMEM142A     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ORAI1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: IMMUNODEFICIENCY 9OMIM: MYOPATHY, TUBULAR AGGREGATE, 2OMIM: ORAI CALCIUM RELEASE-ACTIVATED CALCIUM MODULATOR 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ORAI1  OMIM: IMMUNODEFICIENCY 9OMIM: MYOPATHY, TUBULAR AGGREGATE, 2OMIM: ORAI CALCIUM RELEASE-ACTIVATED CALCIUM MODULATOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PSMD9\t\n",
            "WDR66\t   From NCBI Gene:This protein encoded by this gene belongs to the WD repeat-containing family of proteins, which function in the formation of protein-protein complexes in a variety of biological pathways. This family member appears to function in the determination of mean platelet volume (MPV), and polymorphisms in this gene have been associated with variance in MPV. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Sep 2011]From UniProt:Involved in spermatozoa motility (PubMed:30122540, PubMed:30122541). May also regulate cilium motility through its role in the assembly of the axonemal radial spokes (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:SPERMATOGENIC FAILURE 33From UniProt:Spermatogenic failure 33 (SPGF33): An autosomal recessive infertility disorder caused by spermatogenesis defects that result in multiple abnormalities of sperm flagellum, including short, bent, curled, thick, or absent flagella. [MIM:618152]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q24.31, which is the long (q) arm of chromosome 12 at position 24.31Molecular Location: base pairs 121,918,557 to 122,003,927 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CaM-IP4CFAP251SPGF33     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of WDR66      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SPERMATOGENIC FAILURE 33OMIM: WD REPEAT-CONTAINING PROTEIN 66      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of WDR66  OMIM: SPERMATOGENIC FAILURE 33OMIM: WD REPEAT-CONTAINING PROTEIN 66  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "BCL2L14\t\n",
            "KNTC1\t\n",
            "HIP1R\t\n",
            "ATP6V0A2\t   The ATP6V0A2 gene provides instructions for making one part, the a2 subunit, of a large protein complex (a group of proteins that work together). This protein complex is known as a vacuolar H+-ATPase (V-ATPase). A V-ATPase acts as a pump to move positively charged hydrogen atoms (protons) across cell membranes.V-ATPases are embedded in the membranes surrounding cells, where they transport protons into and out of cells. This movement of protons helps regulate the relative acidity (pH) of cells and their surrounding environment. Tight control of pH is necessary for most biological reactions to proceed properly.Within cells, V-ATPases help regulate the pH of particular cell compartments. These compartments include endosomes and lysosomes, which digest and recycle materials that the cell no longer needs. Studies suggest that V-ATPases are also involved in the movement (trafficking) of small sac-like structures called vesicles. Vesicles transport many types of molecules within cells.V-ATPases also play a key role in a complex process called glycosylation, in which proteins are modified by adding sugar molecules. Glycosylation is necessary for the normal function of many different kinds of proteins. V-ATPases regulate the pH of a cellular structure called the Golgi apparatus, where glycosylation occurs.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Cutis laxa     At least 18 mutations in the ATP6V0A2 gene have been identified in people with cutis laxa. ATP6V0A2 mutations cause a form of the disorder called autosomal recessive cutis laxa type II, which is characterized by loose, sagging skin and distinctive facial features. Some affected individuals also have delayed development, intellectual disability, seizures, and problems with movement that can worsen over time.Mutations in the ATP6V0A2 gene prevent the cell from producing a functional a2 subunit, which disrupts the normal function of V-ATPases. It is unclear how these genetic changes cause the signs and symptoms of cutis laxa. Researchers suspect that changes in V-ATPase function may disrupt the normal glycosylation of proteins, including several that are involved in the assembly and maintenance of elastic fibers. Elastic fibers are slender bundles of proteins that provide strength and flexibility to connective tissue (tissue that supports the body's joints and organs). People with cutis laxa have a reduced density of elastic fibers, which weakens connective tissue in the skin, lungs, and other organs. These defects in connective tissue underlie many of the major features of the disorder. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 18 mutations in the ATP6V0A2 gene have been identified in people with cutis laxa. ATP6V0A2 mutations cause a form of the disorder called autosomal recessive cutis laxa type II, which is characterized by loose, sagging skin and distinctive facial features. Some affected individuals also have delayed development, intellectual disability, seizures, and problems with movement that can worsen over time.Mutations in the ATP6V0A2 gene prevent the cell from producing a functional a2 subunit, which disrupts the normal function of V-ATPases. It is unclear how these genetic changes cause the signs and symptoms of cutis laxa. Researchers suspect that changes in V-ATPase function may disrupt the normal glycosylation of proteins, including several that are involved in the assembly and maintenance of elastic fibers. Elastic fibers are slender bundles of proteins that provide strength and flexibility to connective tissue (tissue that supports the body's joints and organs). People with cutis laxa have a reduced density of elastic fibers, which weakens connective tissue in the skin, lungs, and other organs. These defects in connective tissue underlie many of the major features of the disorder. More About This Health Condition    Cytogenetic Location: 12q24.31, which is the long (q) arm of chromosome 12 at position 24.31Molecular Location: base pairs 123,712,318 to 123,761,755 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        A2V-ATPaseATP6a2ATP6N1DATPase, H+ transporting, lysosomal V0 subunit a2J6B7Stv1TJ6TJ6MTJ6sVph1VPP2_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): What Is the Purpose of Glycosylation?Molecular Cell Biology (fourth edition, 2000): V-Class H+ ATPases Pump Protons across Lysosomal and Vacuolar Membranes      Clinical Information from GeneReviews (1 link)     ATP6V0A2-Related Cutis Laxa      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATP6V0A2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATPase, H+ TRANSPORTING, LYSOSOMAL, V0 SUBUNIT A2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): What Is the Purpose of Glycosylation?Molecular Cell Biology (fourth edition, 2000): V-Class H+ ATPases Pump Protons across Lysosomal and Vacuolar Membranes  ATP6V0A2-Related Cutis Laxa  Tests of ATP6V0A2  PubMed  ATPase, H+ TRANSPORTING, LYSOSOMAL, V0 SUBUNIT A2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Guillard M, Dimopoulou A, Fischer B, Morava E, Lefeber DJ, Kornak U, Wevers RA. Vacuolar H+-ATPase meets glycosylation in patients with cutis laxa. Biochim Biophys Acta. 2009 Sep;1792(9):903-14. doi: 10.1016/j.bbadis.2008.12.009. Epub 2009 Jan 8. Review. Citation on PubMedHucthagowder V, Morava E, Kornak U, Lefeber DJ, Fischer B, Dimopoulou A, Aldinger A, Choi J, Davis EC, Abuelo DN, Adamowicz M, Al-Aama J, Basel-Vanagaite L, Fernandez B, Greally MT, Gillessen-Kaesbach G, Kayserili H, Lemyre E, Tekin M, Trkmen S, Tuysuz B, Yksel-Konuk B, Mundlos S, Van Maldergem L, Wevers RA, Urban Z. Loss-of-function mutations in ATP6V0A2 impair vesicular trafficking, tropoelastin secretion and cell survival. Hum Mol Genet. 2009 Jun 15;18(12):2149-65. doi: 10.1093/hmg/ddp148. Epub 2009 Mar 25. Citation on PubMed or Free article on PubMed CentralKornak U, Reynders E, Dimopoulou A, van Reeuwijk J, Fischer B, Rajab A, Budde B, Nrnberg P, Foulquier F; ARCL Debr-type Study Group, Lefeber D, Urban Z, Gruenewald S, Annaert W, Brunner HG, van Bokhoven H, Wevers R, Morava E, Matthijs G, Van Maldergem L, Mundlos S. Impaired glycosylation and cutis laxa caused by mutations in the vesicular H+-ATPase subunit ATP6V0A2. Nat Genet. 2008 Jan;40(1):32-4. Epub 2007 Dec 23. Citation on PubMedMarshansky V, Futai M. The V-type H+-ATPase in vesicular trafficking: targeting, regulation and function. Curr Opin Cell Biol. 2008 Aug;20(4):415-26. doi: 10.1016/j.ceb.2008.03.015. Epub 2008 May 27. Review. Citation on PubMedMarshansky V. The V-ATPase a2-subunit as a putative endosomal pH-sensor. Biochem Soc Trans. 2007 Nov;35(Pt 5):1092-9. Review. Citation on PubMedMorava E, Lefeber DJ, Urban Z, de Meirleir L, Meinecke P, Gillessen Kaesbach G, Sykut-Cegielska J, Adamowicz M, Salafsky I, Ranells J, Lemyre E, van Reeuwijk J, Brunner HG, Wevers RA. Defining the phenotype in an autosomal recessive cutis laxa syndrome with a combined congenital defect of glycosylation. Eur J Hum Genet. 2008 Jan;16(1):28-35. Epub 2007 Oct 31. Citation on PubMed \n",
            "DNAH10\t\n",
            "CCDC92\t\n",
            "NCOR2\t\n",
            "SCARB1\t   From NCBI Gene:The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2019]From UniProt:(Microbial infection) Acts as a receptor for hepatitis C virus in hepatocytes and appears to facilitate its cell entry (PubMed:12356718, PubMed:12913001, PubMed:18000990). Binding between SCARB1 and the hepatitis C virus glycoprotein E2 is independent of the genotype of the viral isolate (PubMed:12356718). Mediates uptake of M.fortuitum, E.coli and S.aureus (PubMed:16020694).Receptor for different ligands such as phospholipids, cholesterol ester, lipoproteins, phosphatidylserine and apoptotic cells (PubMed:12016218, PubMed:12519372, PubMed:21226579). Receptor for HDL, mediating selective uptake of cholesteryl ether and HDL-dependent cholesterol efflux (PubMed:26965621). Also facilitates the flux of free and esterified cholesterol between the cell surface and apoB-containing lipoproteins and modified lipoproteins, although less efficiently than HDL. May be involved in the phagocytosis of apoptotic cells, via its phosphatidylserine binding activity (PubMed:12016218).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:High density lipoprotein cholesterol level quantitative trait locus 6     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q24.31, which is the long (q) arm of chromosome 12 at position 24.31Molecular Location: base pairs 124,777,628 to 124,863,973 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD36L1CLA-1CLA1HDLQTL6SR-BISRB1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SCARB1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HIGH DENSITY LIPOPROTEIN CHOLESTEROL LEVEL QUANTITATIVE TRAIT LOCUS 6OMIM: SCAVENGER RECEPTOR CLASS B, MEMBER 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SCARB1  OMIM: HIGH DENSITY LIPOPROTEIN CHOLESTEROL LEVEL QUANTITATIVE TRAIT LOCUS 6OMIM: SCAVENGER RECEPTOR CLASS B, MEMBER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMEM132B\t\n",
            "BORCS5\t\n",
            "HEBP1\t\n",
            "EP400\t\n",
            "DDX51\t\n",
            "FBRSL1\t\n",
            "ANKLE2\t   From NCBI Gene:This gene encodes a member of the LEM family of inner nuclear membrane proteins. The encoded protein functions as a mitotic regulator through postmitotic formation of the nuclear envelope. Mutations in this gene cause morphology defects in the nuclear envelope and BAF hyperphosphorylation. [provided by RefSeq, Mar 2014]From UniProt:Involved in mitotic nuclear envelope reassembly by promoting dephosphorylation of BAF/BANF1 during mitotic exit. Coordinates the control of BAF/BANF1 dephosphorylation by inhibiting VRK1 kinase and promoting dephosphorylation of BAF/BANF1 by protein phosphatase 2A (PP2A), thereby facilitating nuclear envelope assembly. It is unclear whether it acts as a real PP2A regulatory subunit or whether it is involved in recruitment of the PP2A complex. Involved in brain development (PubMed:25259927).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Microcephaly 16, primary, autosomal recessiveFrom UniProt:Microcephaly 16, primary, autosomal recessive (MCPH16): A form of microcephaly, a disease defined as a head circumference more than 3 standard deviations below the age, sex and ethnically matched mean. Brain weight is markedly reduced and the cerebral cortex is disproportionately small. [MIM:616681]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q24.33, which is the long (q) arm of chromosome 12 at position 24.33Molecular Location: base pairs 132,725,667 to 132,761,888 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA0692Lem4LEMD7MCPH16     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ANKLE2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ANKYRIN REPEAT- AND LEM DOMAIN-CONTAINING PROTEIN 2OMIM: MICROCEPHALY 16, PRIMARY, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ANKLE2  OMIM: ANKYRIN REPEAT- AND LEM DOMAIN-CONTAINING PROTEIN 2OMIM: MICROCEPHALY 16, PRIMARY, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ANHX\t\n",
            "ATF7IP\t\n",
            "PLBD1\t\n",
            "SLC15A5\t\n",
            "PLCZ1\t   From NCBI Gene:The protein encoded by this gene is a member of the phosphoinositide-specific phospholipase C family. Members in this family, classified into six isotypes that are tissue- and organ-specific, hydrolyze phosphatidylinositol 4,5-bisphosphate just before the phosphate group to yield diacylglycerol and inositol 1,4,5-trisphosphate. This protein localizes to the acrosome in spermatozoa and elicits Ca(2+) oscillations and egg activation during fertilization that leads to early embryonic development. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]From UniProt:The production of the second messenger molecules diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3) is mediated by activated phosphatidylinositol-specific phospholipase C enzymes. In vitro, hydrolyzes PtdIns(4,5)P2 in a Ca(2+)-dependent manner. Triggers intracellular Ca(2+) oscillations in oocytes solely during M phase and is involved in inducing oocyte activation and initiating embryonic development up to the blastocyst stage. Is therefore a strong candidate for the egg-activating soluble sperm factor that is transferred from the sperm into the egg cytoplasm following gamete membrane fusion. May exert an inhibitory effect on phospholipase-C-coupled processes that depend on calcium ions and protein kinase C, including CFTR trafficking and function.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spermatogenic failure 17From UniProt:Spermatogenic failure 17 (SPGF17): An autosomal recessive infertility disorder due to failure of oocyte activation and fertilization by sperm that otherwise exhibits normal morphology. [MIM:617214]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12p12.3, which is the short (p) arm of chromosome 12 at position 12.3Molecular Location: base pairs 18,645,529 to 18,738,059 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        NYD-SP27PLC-zeta-1PLCzetaSPGF17     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PLCZ1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PHOSPHOLIPASE C, ZETA-1OMIM: SPERMATOGENIC FAILURE 17      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PLCZ1  OMIM: PHOSPHOLIPASE C, ZETA-1OMIM: SPERMATOGENIC FAILURE 17  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DCP1B\t\n",
            "SLCO1C1\t\n",
            "SLCO1A2\t\n",
            "LRMP\t\n",
            "LMNTD1\t\n",
            "DDX11\t   The DDX11 gene provides instructions for making an enzyme called ChlR1, which functions as a helicase. Helicases are enzymes that attach (bind) to DNA and temporarily unwind the two spiral strands (double helix) of the DNA molecule so it can be copied (replicated) in preparation for cell division. ChlR1 is also involved in repairing any errors that are made when DNA is copied. In addition, ChlR1 is involved in other processes leading up to cell division. After replication, the DNA from each chromosome is arranged into two identical structures, called sister chromatids, which the ChlR1 enzyme helps to keep together until they are ready to separate into individual cells. This enzyme also ensures proper separation of chromatids during cell division. By helping repair errors in DNA and ensuring proper DNA replication, the ChlR1 enzyme plays a role in maintaining the stability of a cell's genetic information.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Warsaw breakage syndrome     At least three mutations in the DDX11 gene have been found to cause Warsaw breakage syndrome. This condition causes multiple abnormalities that may include impaired growth, distinctive facial features, hearing loss, and heart malformations. The mutations that cause Warsaw breakage syndrome severely reduce or completely eliminate ChlR1 enzyme activity. As a result, the enzyme cannot bind to DNA and cannot unwind the DNA strands to help with DNA replication and repair. A lack of functional ChlR1 impairs cell division and leads to an accumulation of DNA damage. This DNA damage can appear as breaks in the DNA, giving the condition its name. It is unclear how these problems in DNA maintenance lead to the specific abnormalities characteristic of Warsaw breakage syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three mutations in the DDX11 gene have been found to cause Warsaw breakage syndrome. This condition causes multiple abnormalities that may include impaired growth, distinctive facial features, hearing loss, and heart malformations. The mutations that cause Warsaw breakage syndrome severely reduce or completely eliminate ChlR1 enzyme activity. As a result, the enzyme cannot bind to DNA and cannot unwind the DNA strands to help with DNA replication and repair. A lack of functional ChlR1 impairs cell division and leads to an accumulation of DNA damage. This DNA damage can appear as breaks in the DNA, giving the condition its name. It is unclear how these problems in DNA maintenance lead to the specific abnormalities characteristic of Warsaw breakage syndrome. More About This Health Condition    Cytogenetic Location: 12p11.21, which is the short (p) arm of chromosome 12 at position 11.21Molecular Location: base pairs 31,073,845 to 31,104,799 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CHL1-related helicase gene-1CHL1-related protein 1CHLR1DDX11_HUMANDEAD/H (Asp-Glu-Ala-Asp/His) box helicase 11DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 11 (CHL1-like helicase homolog, S. cerevisiae)DEAD/H box protein 11hCHLR1keratinocyte growth factor-regulated gene 2 proteinKRG-2KRG2probable ATP-dependent RNA helicase DDX11     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Genomes (second edition, 2002): The Replication ProcessThe Cell: A Molecular Approach: Action of Helicases and Single-Stranded DNA-Binding Proteins (figure)      Tests Listed in the Genetic Testing Registry (1 link)     Tests of DDX11      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     DEAD/H BOX 11      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Genomes (second edition, 2002): The Replication ProcessThe Cell: A Molecular Approach: Action of Helicases and Single-Stranded DNA-Binding Proteins (figure)  Tests of DDX11  PubMed  DEAD/H BOX 11  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Capo-Chichi JM, Bharti SK, Sommers JA, Yammine T, Chouery E, Patry L, Rouleau GA, Samuels ME, Hamdan FF, Michaud JL, Brosh RM Jr, Mgarbane A, Kibar Z. Identification and biochemical characterization of a novel mutation in DDX11 causing Warsaw breakage syndrome. Hum Mutat. 2013 Jan;34(1):103-7. doi: 10.1002/humu.22226. Epub 2012 Oct 17. Citation on PubMed or Free article on PubMed CentralOMIM: DEAD/H BOX 11Inoue A, Hyle J, Lechner MS, Lahti JM. Mammalian ChlR1 has a role in heterochromatin organization. Exp Cell Res. 2011 Oct 15;317(17):2522-35. doi: 10.1016/j.yexcr.2011.08.006. Epub 2011 Aug 11. Citation on PubMed or Free article on PubMed CentralShah N, Inoue A, Woo Lee S, Beishline K, Lahti JM, Noguchi E. Roles of ChlR1 DNA helicase in replication recovery from DNA damage. Exp Cell Res. 2013 Aug 15;319(14):2244-53. doi: 10.1016/j.yexcr.2013.06.005. Epub 2013 Jun 22. Citation on PubMed or Free article on PubMed Centralvan der Lelij P, Chrzanowska KH, Godthelp BC, Rooimans MA, Oostra AB, Stumm M, Zdzienicka MZ, Joenje H, de Winter JP. Warsaw breakage syndrome, a cohesinopathy associated with mutations in the XPD helicase family member DDX11/ChlR1. Am J Hum Genet. 2010 Feb 12;86(2):262-6. doi: 10.1016/j.ajhg.2010.01.008. Epub 2010 Feb 4. Citation on PubMed or Free article on PubMed Central \n",
            "CRACR2A\t\n",
            "MUC19\t\n",
            "PDZRN4\t\n",
            "VDR\t   The VDR gene provides instructions for making a protein called vitamin D receptor (VDR), which allows the body to respond to vitamin D. This vitamin can be acquired from foods in the diet or made in the body with help from sunlight exposure. Vitamin D is involved in maintaining the proper balance of several minerals in the body, including calcium and phosphate, which are essential for the normal formation of bones and teeth. One of vitamin D's major roles is to control the absorption of calcium and phosphate from the intestines into the bloodstream. Vitamin D is also involved in several processes unrelated to bone and tooth formation.The VDR protein attaches (binds) to the active form of vitamin D, known as calcitriol. This interaction allows VDR to partner with another protein called retinoid X receptor (RXR). The resulting complex then binds to particular regions of DNA, known as vitamin D response elements, and regulates the activity of vitamin D-responsive genes. By turning these genes on or off, the complex helps control calcium and phosphate absorption and other processes.Although the mechanism is not completely understood, the VDR protein is also involved in hair growth. Studies suggest that this process does not require calcitriol binding.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Vitamin D-dependent rickets     Mutations in the VDR gene cause vitamin D-dependent rickets type 2A (VDDR2A), also known as hereditary vitamin D-resistant rickets (HVDRR). This disorder of bone development is characterized by low levels of calcium (hypocalcemia) and phosphate (hypophosphatemia) in the blood, which lead to soft, weak bones (rickets) that are prone to fracture. A common feature of this condition is abnormally curved (bowed) legs.The VDR gene mutations that cause this condition prevent the VDR protein from functioning properly. Some changes in the VDR gene lead to an abnormally short version of the VDR protein. Others result in the production of an abnormal receptor that cannot bind to calcitriol, to RXR, or to DNA. Although there is plenty of calcitriol in the body, the altered VDR protein cannot stimulate gene activity important for mineral absorption. The lack of calcium and phosphate absorption in the intestines slows deposition of these minerals into developing bone (bone mineralization), which leads to soft, weak bones and other features of VDDR2A. Hypocalcemia also causes muscle weakness and seizures in some affected individuals. Most VDR gene mutations impair hair growth, leading to hair loss (alopecia); however, mutations that block VDR's ability to interact with calcitriol do not cause alopecia, indicating that calcitriol is not necessary for the receptor's role in hair development. More About This Health Condition      Alopecia areata     Genetics Home Reference provides information about alopecia areata. More About This Health Condition      Intervertebral disc disease     Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition      Kidney stones     Genetics Home Reference provides information about kidney stones. More About This Health Condition      Leprosy     Genetics Home Reference provides information about leprosy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the VDR gene cause vitamin D-dependent rickets type 2A (VDDR2A), also known as hereditary vitamin D-resistant rickets (HVDRR). This disorder of bone development is characterized by low levels of calcium (hypocalcemia) and phosphate (hypophosphatemia) in the blood, which lead to soft, weak bones (rickets) that are prone to fracture. A common feature of this condition is abnormally curved (bowed) legs.The VDR gene mutations that cause this condition prevent the VDR protein from functioning properly. Some changes in the VDR gene lead to an abnormally short version of the VDR protein. Others result in the production of an abnormal receptor that cannot bind to calcitriol, to RXR, or to DNA. Although there is plenty of calcitriol in the body, the altered VDR protein cannot stimulate gene activity important for mineral absorption. The lack of calcium and phosphate absorption in the intestines slows deposition of these minerals into developing bone (bone mineralization), which leads to soft, weak bones and other features of VDDR2A. Hypocalcemia also causes muscle weakness and seizures in some affected individuals. Most VDR gene mutations impair hair growth, leading to hair loss (alopecia); however, mutations that block VDR's ability to interact with calcitriol do not cause alopecia, indicating that calcitriol is not necessary for the receptor's role in hair development. More About This Health Condition  Genetics Home Reference provides information about alopecia areata. More About This Health Condition  Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition  Genetics Home Reference provides information about kidney stones. More About This Health Condition  Genetics Home Reference provides information about leprosy. More About This Health Condition    Cytogenetic Location: 12q13.11, which is the long (q) arm of chromosome 12 at position 13.11Molecular Location: base pairs 47,841,537 to 47,905,031 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        1,25-dihydroxyvitamin D3 receptorNR1I1nuclear receptor subfamily 1 group I member 1vitamin D (1,25- dihydroxyvitamin D3) receptorvitamin D3 receptor     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Dietary Reference Intakes for Calcium, Phosphorus, Magnesium, Vitamin D, and Fluoride (1997): Vitamin DEndocrinology: An Integrated Approach (2001): Classical Actions of Vitamin D on Intestine and BoneMolecular Cell Biology (fourth edition, 2000): Lipid-Soluble Hormones Control the Activities of Nuclear Receptors      Tests Listed in the Genetic Testing Registry (1 link)     Tests of VDR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     VITAMIN D RECEPTOR      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Dietary Reference Intakes for Calcium, Phosphorus, Magnesium, Vitamin D, and Fluoride (1997): Vitamin DEndocrinology: An Integrated Approach (2001): Classical Actions of Vitamin D on Intestine and BoneMolecular Cell Biology (fourth edition, 2000): Lipid-Soluble Hormones Control the Activities of Nuclear Receptors  Tests of VDR  PubMed  VITAMIN D RECEPTOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Feldman D, J Malloy P. Mutations in the vitamin D receptor and hereditary vitamin D-resistant rickets. Bonekey Rep. 2014 Mar 5;3:510. doi: 10.1038/bonekey.2014.5. eCollection 2014. Review. Citation on PubMed or Free article on PubMed CentralMalloy PJ, Feldman D. Genetic disorders and defects in vitamin d action. Endocrinol Metab Clin North Am. 2010 Jun;39(2):333-46, table of contents. doi: 10.1016/j.ecl.2010.02.004. Review. Citation on PubMed or Free article on PubMed CentralMalloy PJ, Feldman D. The role of vitamin D receptor mutations in the development of alopecia. Mol Cell Endocrinol. 2011 Dec 5;347(1-2):90-6. doi: 10.1016/j.mce.2011.05.045. Epub 2011 Jun 13. Review. Citation on PubMed or Free article on PubMed CentralMalloy PJ, Tasic V, Taha D, Ttncler F, Ying GS, Yin LK, Wang J, Feldman D. Vitamin D receptor mutations in patients with hereditary 1,25-dihydroxyvitamin D-resistant rickets. Mol Genet Metab. 2014 Jan;111(1):33-40. doi: 10.1016/j.ymgme.2013.10.014. Epub 2013 Nov 4. Citation on PubMed or Free article on PubMed CentralRyan JW, Anderson PH, Turner AG, Morris HA. Vitamin D activities and metabolic bone disease. Clin Chim Acta. 2013 Oct 21;425:148-52. doi: 10.1016/j.cca.2013.07.024. Epub 2013 Jul 30. Review. Citation on PubMedTamura M, Ishizawa M, Isojima T, zen S, Oka A, Makishima M, Kitanaka S. Functional analyses of a novel missense and other mutations of the vitamin D receptor in association with alopecia. Sci Rep. 2017 Jul 11;7(1):5102. doi: 10.1038/s41598-017-05081-x. Citation on PubMed or Free article on PubMed CentralTiosano D, Hadad S, Chen Z, Nemirovsky A, Gepstein V, Militianu D, Weisman Y, Abrams SA. Calcium absorption, kinetics, bone density, and bone structure in patients with hereditary vitamin D-resistant rickets. J Clin Endocrinol Metab. 2011 Dec;96(12):3701-9. doi: 10.1210/jc.2011-1432. Epub 2011 Sep 14. Citation on PubMedOMIM: VITAMIN D RECEPTOR \n",
            "OR10AD1\t\n",
            "ASIC1\t\n",
            "CERS5\t\n",
            "CELA1\t\n",
            "ANKRD33\t\n",
            "KRT81\t   The KRT81 gene provides instructions for making the type II hair keratin K81 protein (K81). This protein belongs to a group of proteins known as keratins, which are tough, fibrous proteins that form the structural framework of cells that make up the hair, skin, and nails. Each keratin protein partners with another keratin protein to form molecules called intermediate filaments. These filaments assemble into strong networks that provide strength and resiliency to the tissues and protect them from being damaged by everyday physical stresses. The K81 protein is found in cells that make up the inner compartment of the hair shaft known as the cortex, and this protein helps give hair its strength and elasticity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Monilethrix     At least two mutations in the KRT81 gene have been identified in people with monilethrix, a hair condition characterized by strands of hair with a beaded appearance and short, brittle hair that breaks easily. Mutations associated with this condition change a single protein building block (amino acid) in the K81 protein. The amino acid changes usually occur in a region of the K81 protein thought to be important in intermediate filament formation. In people with monilethrix, the cortex of the affected hair shaft appears abnormal. However, it is unclear how mutations in the KRT81 gene are related to the abnormality in the cortex or the beaded appearance of the hair. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two mutations in the KRT81 gene have been identified in people with monilethrix, a hair condition characterized by strands of hair with a beaded appearance and short, brittle hair that breaks easily. Mutations associated with this condition change a single protein building block (amino acid) in the K81 protein. The amino acid changes usually occur in a region of the K81 protein thought to be important in intermediate filament formation. In people with monilethrix, the cortex of the affected hair shaft appears abnormal. However, it is unclear how mutations in the KRT81 gene are related to the abnormality in the cortex or the beaded appearance of the hair. More About This Health Condition    Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,285,913 to 52,291,515 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ghHb1hair keratin K2.9hard keratin, type II, 1Hb-1HB1hHAKB2-1hHb1keratin-81keratin 81, type IIkeratin, hair, basic, 1KRT81_HUMANKRTHB1type II hair keratin Hb1     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database: Development and Structure of the Hair Follicle      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT81      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KERATIN 81, TYPE II      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Development and Structure of the Hair Follicle  Tests of KRT81  PubMed  KERATIN 81, TYPE II  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: KERATIN 81, TYPE IIKorge BP, Hamm H, Jury CS, Traupe H, Irvine AD, Healy E, Birch-MacHin M, Rees JL, Messenger AG, Holmes SC, Parry DA, Munro CS. Identification of novel mutations in basic hair keratins hHb1 and hHb6 in monilethrix: implications for protein structure and clinical phenotype. J Invest Dermatol. 1999 Oct;113(4):607-12. Citation on PubMedLangbein L, Rogers MA, Winter H, Praetzel S, Schweizer J. The catalog of human hair keratins. II. Expression of the six type II members in the hair follicle and the combined catalog of human type I and II keratins. J Biol Chem. 2001 Sep 14;276(37):35123-32. Epub 2001 Jul 9. Citation on PubMedRogers MA, Langbein L, Praetzel S, Moll I, Krieg T, Winter H, Schweizer J. Sequences and differential expression of three novel human type-II hair keratins. Differentiation. 1997 Feb;61(3):187-94. Citation on PubMedSchweizer J, Bowden PE, Coulombe PA, Langbein L, Lane EB, Magin TM, Maltais L, Omary MB, Parry DA, Rogers MA, Wright MW. New consensus nomenclature for mammalian keratins. J Cell Biol. 2006 Jul 17;174(2):169-74. Epub 2006 Jul 10. Citation on PubMed or Free article on PubMed Central \n",
            "KRT84\t\n",
            "KRT75\t   From NCBI Gene:This gene is a member of the type II keratin family clustered on the long arm of chromosome 12. Type I and type II keratins heteropolymerize to form intermediate-sized filaments in the cytoplasm of epithelial cells. This gene is expressed in the companion layer, upper germinative matrix region of the hair follicle, and medulla of the hair shaft. The encoded protein plays an essential role in hair and nail formation. Variations in this gene have been associated with the hair disorders pseudofolliculitis barbae (PFB) and loose anagen hair syndrome (LAHS). [provided by RefSeq, Oct 2008]From UniProt:Plays a central role in hair and nail formation. Essential component of keratin intermediate filaments in the companion layer of the hair follicle.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Pseudofolliculitis barbaeFrom UniProt:Loose anagen hair syndrome (LAHS): In LAHS, anagen hairs are easily pulled from the scalp. The hair is relatively sparse and does not grow long. Hair of fair color and hair shafts of reduced caliber, and an early age of onset are features. Usually the hairs are not fragile and there are no areas of breakage. [MIM:600628]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,424,070 to 52,434,326 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        K6HFKB18PFB     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT75      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KERATIN 75, TYPE IIOMIM: PSEUDOFOLLICULITIS BARBAE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KRT75  OMIM: KERATIN 75, TYPE IIOMIM: PSEUDOFOLLICULITIS BARBAE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KRT6B\t   The KRT6B gene provides instructions for making a protein called keratin 6b or K6b. Keratins are a group of tough, fibrous proteins that form the structural framework of certain cells, particularly cells that make up the skin, hair, and nails. Keratin 6b is produced in the nails, the hair follicles, and the skin on the palms of the hands and soles of the feet. It is also found in the skin's sebaceous glands, which produce an oily substance called sebum.Keratin 6b partners with a similar protein, keratin 17, to form molecules called keratin intermediate filaments. These filaments assemble into dense networks that provide strength and resilience to the skin, nails, and other tissues. Networks of keratin intermediate filaments protect these tissues from being damaged by friction and other everyday physical stresses. Keratin 6b is also among several keratins involved in wound healing.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Pachyonychia congenita     At least four mutations in the KRT6B gene have been identified in people with pachyonychia congenita, a rare condition that primarily affects the nails and skin. In most cases, this condition becomes apparent within the first few months of life. These mutations either change single protein building blocks (amino acids) in keratin 6b or delete a small number of amino acids from the protein.The KRT6B gene mutations responsible for pachyonychia congenita change the structure of keratin 6b, preventing it from interacting effectively with keratin 17 and interfering with the assembly of the keratin intermediate filament network. Without this network, skin cells become fragile and are easily damaged, making the skin less resistant to friction and minor trauma. Even normal activities such as walking can cause skin cells to break down, resulting in the formation of painful blisters and calluses. In the sebaceous glands, abnormal keratin filaments lead to the development of sebum-filled cysts called steatocystomas. Defective keratin 6b also disrupts the growth and function of other tissues, such as the hair follicles and nails, which explains why the signs and symptoms of pachyonychia congenita can also affect these other parts of the body. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least four mutations in the KRT6B gene have been identified in people with pachyonychia congenita, a rare condition that primarily affects the nails and skin. In most cases, this condition becomes apparent within the first few months of life. These mutations either change single protein building blocks (amino acids) in keratin 6b or delete a small number of amino acids from the protein.The KRT6B gene mutations responsible for pachyonychia congenita change the structure of keratin 6b, preventing it from interacting effectively with keratin 17 and interfering with the assembly of the keratin intermediate filament network. Without this network, skin cells become fragile and are easily damaged, making the skin less resistant to friction and minor trauma. Even normal activities such as walking can cause skin cells to break down, resulting in the formation of painful blisters and calluses. In the sebaceous glands, abnormal keratin filaments lead to the development of sebum-filled cysts called steatocystomas. Defective keratin 6b also disrupts the growth and function of other tissues, such as the hair follicles and nails, which explains why the signs and symptoms of pachyonychia congenita can also affect these other parts of the body. More About This Health Condition    Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,446,651 to 52,452,126 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CK 6BCK6Bcytokeratin-6Bcytokeratin 6BK2C6B_HUMANK6BK6b keratinkeratin 6B, type IIkeratin, epidermal, type II, K6Bkeratin, type II cytoskeletal 6BKRTL1     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Keratin Filaments in Epithelial Cells (image)The Cell: A Molecular Approach (second edition, 2000): Intermediate Filaments      Clinical Information from GeneReviews (1 link)     Pachyonychia Congenita      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT6B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KERATIN 6B, TYPE II      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Keratin Filaments in Epithelial Cells (image)The Cell: A Molecular Approach (second edition, 2000): Intermediate Filaments  Pachyonychia Congenita  Tests of KRT6B  PubMed  KERATIN 6B, TYPE II  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  McLean WH, Hansen CD, Eliason MJ, Smith FJ. The phenotypic and molecular genetic features of pachyonychia congenita. J Invest Dermatol. 2011 May;131(5):1015-7. doi: 10.1038/jid.2011.59. Epub 2011 Mar 24. Review. Citation on PubMedSharma VM, Stein SL. A novel mutation in K6b in pachyonychia congenita type 2. J Invest Dermatol. 2007 Aug;127(8):2060-2. Epub 2007 Apr 12. Citation on PubMedSmith FJ, Jonkman MF, van Goor H, Coleman CM, Covello SP, Uitto J, McLean WH. A mutation in human keratin K6b produces a phenocopy of the K17 disorder pachyonychia congenita type 2. Hum Mol Genet. 1998 Jul;7(7):1143-8. Citation on PubMedWilson NJ, O'Toole EA, Milstone LM, Hansen CD, Shepherd AA, Al-Asadi E, Schwartz ME, McLean WH, Sprecher E, Smith FJ. The molecular genetic analysis of the expanding pachyonychia congenita case collection. Br J Dermatol. 2014 Aug;171(2):343-55. doi: 10.1111/bjd.12958. Epub 2014 Aug 6. Citation on PubMed or Free article on PubMed Central \n",
            "KRT6C\t   The KRT6C gene provides instructions for making a protein called keratin 6c or K6c. Keratins are a group of tough, fibrous proteins that form the structural framework of certain cells, particularly cells that make up the skin, hair, and nails. Keratin 6c is found in the skin, although it is unknown which other tissues may produce this protein.Keratin 6c is a component of molecules called keratin intermediate filaments. These filaments assemble into dense networks that provide strength and resilience to the skin, nails, and other tissues. Networks of keratin intermediate filaments protect these tissues from being damaged by friction and other everyday physical stresses.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Pachyonychia congenita     At least four mutations in the KRT6C gene have been found to cause pachyonychia congenita, a rare condition that primarily affects the nails and skin. In most cases, this condition becomes apparent within the first few months of life.One of the mutations associated with pachyonychia congenita changes a single protein building block (amino acid) in the keratin 6c protein. Specifically, this mutation replaces the amino acid glutamic acid with the amino acid lysine at protein position 472 (written as Glu472Lys or E472K). The other KRT6C gene mutations delete one or more amino acids from the keratin 6c protein.All of the known KRT6C gene mutations alter the structure of keratin 6c and interfere with the assembly of the keratin intermediate filament network. Without this network, skin cells become fragile and are easily damaged, making the skin less resistant to friction and minor trauma. Even normal activities such as walking can cause skin cells to break down, resulting in the formation of severe, painful blisters and calluses. Additionally, fragile skin cells may abnormally produce more keratin in response to damage, which makes the skin problems worse. Defective keratin 6c also disrupts the growth and function of other tissues, such as the hair follicles and nails, which explains why the signs and symptoms of pachyonychia congenita can also affect these other parts of the body. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least four mutations in the KRT6C gene have been found to cause pachyonychia congenita, a rare condition that primarily affects the nails and skin. In most cases, this condition becomes apparent within the first few months of life.One of the mutations associated with pachyonychia congenita changes a single protein building block (amino acid) in the keratin 6c protein. Specifically, this mutation replaces the amino acid glutamic acid with the amino acid lysine at protein position 472 (written as Glu472Lys or E472K). The other KRT6C gene mutations delete one or more amino acids from the keratin 6c protein.All of the known KRT6C gene mutations alter the structure of keratin 6c and interfere with the assembly of the keratin intermediate filament network. Without this network, skin cells become fragile and are easily damaged, making the skin less resistant to friction and minor trauma. Even normal activities such as walking can cause skin cells to break down, resulting in the formation of severe, painful blisters and calluses. Additionally, fragile skin cells may abnormally produce more keratin in response to damage, which makes the skin problems worse. Defective keratin 6c also disrupts the growth and function of other tissues, such as the hair follicles and nails, which explains why the signs and symptoms of pachyonychia congenita can also affect these other parts of the body. More About This Health Condition    Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,468,516 to 52,473,785 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CK-6CCK-6Ecytokeratin-6Ccytokeratin-6EK2C6C_HUMANK6CK6Ekeratin-6Ckeratin 6C, type IIkeratin 6Ekeratin K6hkeratin, type II cytoskeletal 6CKRT6Etype-II keratin Kb12     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Cell: A Molecular Approach (second edition, 2000): Intermediate Filaments      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT6C      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KERATIN 6C, TYPE II      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Cell: A Molecular Approach (second edition, 2000): Intermediate Filaments  Tests of KRT6C  PubMed  KERATIN 6C, TYPE II  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Akasaka E, Nakano H, Nakano A, Toyomaki Y, Takiyoshi N, Rokunohe D, Nishikawa Y, Korekawa A, Matsuzaki Y, Mitsuhashi Y, Sawamura D. Diffuse and focal palmoplantar keratoderma can be caused by a keratin 6c mutation. Br J Dermatol. 2011 Dec;165(6):1290-2. doi: 10.1111/j.1365-2133.2011.10552.x. Citation on PubMedBowden PE. Mutations in a keratin 6 isomer (K6c) cause a type of focal palmoplantar keratoderma. J Invest Dermatol. 2010 Feb;130(2):336-8. doi: 10.1038/jid.2009.395. Citation on PubMedTakahashi K, Paladini RD, Coulombe PA. Cloning and characterization of multiple human genes and cDNAs encoding highly related type II keratin 6 isoforms. J Biol Chem. 1995 Aug 4;270(31):18581-92. Citation on PubMedWilson NJ, Messenger AG, Leachman SA, O'Toole EA, Lane EB, McLean WH, Smith FJ. Keratin K6c mutations cause focal palmoplantar keratoderma. J Invest Dermatol. 2010 Feb;130(2):425-9. doi: 10.1038/jid.2009.215. Epub 2009 Jul 16. Citation on PubMedWilson NJ, O'Toole EA, Milstone LM, Hansen CD, Shepherd AA, Al-Asadi E, Schwartz ME, McLean WH, Sprecher E, Smith FJ. The molecular genetic analysis of the expanding pachyonychia congenita case collection. Br J Dermatol. 2014 Aug;171(2):343-55. doi: 10.1111/bjd.12958. Epub 2014 Aug 6. Citation on PubMed or Free article on PubMed Central \n",
            "KRT72\t\n",
            "KRT1\t   The KRT1 gene provides instructions for making a protein called keratin 1. Keratins are a group of tough, fibrous proteins that form the structural framework of cells called keratinocytes that make up the skin, hair, and nails. Keratin 1 is produced in keratinocytes in the outer layer of the skin (the epidermis), including the skin on the palms of the hands and soles of the feet.The keratin 1 protein partners with another keratin protein, either keratin 9 or keratin 10, to form molecules called keratin intermediate filaments. These filaments assemble into strong networks that provide strength and resiliency to the skin and protect it from being damaged by friction and other everyday physical stresses.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Epidermolytic hyperkeratosis     Dozens of mutations in the KRT1 gene have been found in people with epidermolytic hyperkeratosis. This condition is a skin disorder characterized by red, blistering skin at an early age and thick skin (hyperkeratosis) later in life. People with KRT1 gene mutations typically have PS-type epidermolytic hyperkeratosis, which features thick skin on the palms of the hands and soles of the feet (palmoplantar hyperkeratosis) in addition to other parts of the body.Most KRT1 gene mutations associated with epidermolytic hyperkeratosis change a single protein building block (amino acid) in the keratin 1 protein. These amino acid changes commonly occur in regions of the protein that play a role in intermediate filament formation. The mutations alter the keratin 1 protein and seem to affect how intermediate filaments interact with each other to form networks. The altered proteins still form intermediate filaments, but the intermediate filament networks are weaker and do not function normally. Without a strong network, skin cells become fragile and are easily damaged, which can lead to blistering in response to friction or mild trauma. It is unclear how these mutations cause the overgrowth of keratinocytes that results in hyperkeratotic skin. More About This Health Condition      Other disorders     KRT1 gene mutations are involved in many other skin disorders. In several of these conditions, there is palmoplantar hyperkeratosis, but the skin on other parts of the body is usually not affected. A condition called epidermolytic palmoplantar keratoderma caused by KRT1 gene mutations is relatively mild. Affected individuals typically have palmoplantar hyperkeratosis with detachment or loosening of the epidermis (epidermolysis), usually seen as blistering. People with nonepidermolytic palmoplantar keratoderma have palmoplantar hyperkeratosis with no evidence of epidermolysis. In striate palmoplantar keratoderma type 3, the skin thickening on the palms and soles follows a specific pattern.KRT1 gene mutations are also responsible for a skin disorder called Curth-Macklin ichthyosis hystrix. This condition involves severe hyperkeratosis on the palms and soles and sometimes the skin over large joints or on the torso. This condition is distinguished by specific changes in the keratinocytes.Another skin condition caused by genetic changes in the KRT1 gene called cyclic ichthyosis with epidermolytic hyperkeratosis is similar to epidermolytic hyperkeratosis, but the skin changes disappear for short periods, then recur. The recurrent skin changes can last for weeks or months.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Dozens of mutations in the KRT1 gene have been found in people with epidermolytic hyperkeratosis. This condition is a skin disorder characterized by red, blistering skin at an early age and thick skin (hyperkeratosis) later in life. People with KRT1 gene mutations typically have PS-type epidermolytic hyperkeratosis, which features thick skin on the palms of the hands and soles of the feet (palmoplantar hyperkeratosis) in addition to other parts of the body.Most KRT1 gene mutations associated with epidermolytic hyperkeratosis change a single protein building block (amino acid) in the keratin 1 protein. These amino acid changes commonly occur in regions of the protein that play a role in intermediate filament formation. The mutations alter the keratin 1 protein and seem to affect how intermediate filaments interact with each other to form networks. The altered proteins still form intermediate filaments, but the intermediate filament networks are weaker and do not function normally. Without a strong network, skin cells become fragile and are easily damaged, which can lead to blistering in response to friction or mild trauma. It is unclear how these mutations cause the overgrowth of keratinocytes that results in hyperkeratotic skin. More About This Health Condition  KRT1 gene mutations are involved in many other skin disorders. In several of these conditions, there is palmoplantar hyperkeratosis, but the skin on other parts of the body is usually not affected. A condition called epidermolytic palmoplantar keratoderma caused by KRT1 gene mutations is relatively mild. Affected individuals typically have palmoplantar hyperkeratosis with detachment or loosening of the epidermis (epidermolysis), usually seen as blistering. People with nonepidermolytic palmoplantar keratoderma have palmoplantar hyperkeratosis with no evidence of epidermolysis. In striate palmoplantar keratoderma type 3, the skin thickening on the palms and soles follows a specific pattern.KRT1 gene mutations are also responsible for a skin disorder called Curth-Macklin ichthyosis hystrix. This condition involves severe hyperkeratosis on the palms and soles and sometimes the skin over large joints or on the torso. This condition is distinguished by specific changes in the keratinocytes.Another skin condition caused by genetic changes in the KRT1 gene called cyclic ichthyosis with epidermolytic hyperkeratosis is similar to epidermolytic hyperkeratosis, but the skin changes disappear for short periods, then recur. The recurrent skin changes can last for weeks or months.    Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,674,736 to 52,680,407 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        67 kDa cytokeratinCK-1CK1cytokeratin 1cytokeratin-1EHK1hair alpha proteinK1K2C1_HUMANkeratin 1, type IIkeratin, type II cytoskeletal 1KRT1Atype-II keratin Kb1     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Human Intermediate Filament DatabaseMolecular Cell Biology (4th Edition, 2000): Intermediate Filaments      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (6 links)     ICHTHYOSIS HYSTRIX, CURTH-MACKLIN TYPEICHTHYOSIS, CYCLIC, WITH EPIDERMOLYTIC HYPERKERATOSISKERATIN 1, TYPE IIKERATOSIS PALMOPLANTARIS STRIATA IIIPALMOPLANTAR KERATODERMA, EPIDERMOLYTICPALMOPLANTAR KERATODERMA, NONEPIDERMOLYTIC      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Human Intermediate Filament DatabaseMolecular Cell Biology (4th Edition, 2000): Intermediate Filaments  Tests of KRT1  PubMed  ICHTHYOSIS HYSTRIX, CURTH-MACKLIN TYPEICHTHYOSIS, CYCLIC, WITH EPIDERMOLYTIC HYPERKERATOSISKERATIN 1, TYPE IIKERATOSIS PALMOPLANTARIS STRIATA IIIPALMOPLANTAR KERATODERMA, EPIDERMOLYTICPALMOPLANTAR KERATODERMA, NONEPIDERMOLYTIC  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Chamcheu JC, Siddiqui IA, Syed DN, Adhami VM, Liovic M, Mukhtar H. Keratin gene mutations in disorders of human skin and its appendages. Arch Biochem Biophys. 2011 Apr 15;508(2):123-37. doi: 10.1016/j.abb.2010.12.019. Epub 2010 Dec 19. Review. Citation on PubMed or Free article on PubMed CentralChipev CC, Korge BP, Markova N, Bale SJ, DiGiovanna JJ, Compton JG, Steinert PM. A leucine----proline mutation in the H1 subdomain of keratin 1 causes epidermolytic hyperkeratosis. Cell. 1992 Sep 4;70(5):821-8. Citation on PubMedDiGiovanna JJ, Bale SJ. Clinical heterogeneity in epidermolytic hyperkeratosis. Arch Dermatol. 1994 Aug;130(8):1026-35. Citation on PubMedOMIM: ICHTHYOSIS HYSTRIX, CURTH-MACKLIN TYPEOMIM: ICHTHYOSIS, CYCLIC, WITH EPIDERMOLYTIC HYPERKERATOSISOMIM: KERATIN 1, TYPE IIOMIM: KERATOSIS PALMOPLANTARIS STRIATA IIIOMIM: PALMOPLANTAR KERATODERMA, EPIDERMOLYTICOMIM: PALMOPLANTAR KERATODERMA, NONEPIDERMOLYTICRothnagel JA, Dominey AM, Dempsey LD, Longley MA, Greenhalgh DA, Gagne TA, Huber M, Frenk E, Hohl D, Roop DR. Mutations in the rod domains of keratins 1 and 10 in epidermolytic hyperkeratosis. Science. 1992 Aug 21;257(5073):1128-30. Citation on PubMedYang JM, Chipev CC, DiGiovanna JJ, Bale SJ, Marekov LN, Steinert PM, Compton JG. Mutations in the H1 and 1A domains in the keratin 1 gene in epidermolytic hyperkeratosis. J Invest Dermatol. 1994 Jan;102(1):17-23. Citation on PubMedYang JM, Nam K, Kim HC, Lee JH, Park JK, Wu K, Lee ES, Steinert PM. A novel glutamic acid to aspartic acid mutation near the end of the 2B rod domain in the keratin 1 chain in epidermolytic hyperkeratosis. J Invest Dermatol. 1999 Mar;112(3):376-9. Citation on PubMed \n",
            "KRT77\t\n",
            "KRT76\t\n",
            "KRT3\t   The KRT3 gene provides instructions for making a protein called keratin 3. Keratins are a group of tough, fibrous proteins that form the structural framework of epithelial cells, which are cells that line the surfaces and cavities of the body. Keratin 3 is produced in a tissue on the surface of the eye called the corneal epithelium. This tissue forms the outermost layer of the cornea, which is the clear front covering of the eye. The corneal epithelium acts as a barrier to help prevent foreign materials, such as dust and bacteria, from entering the eye.The keratin 3 protein partners with another keratin protein, keratin 12, to form molecules known as intermediate filaments. These filaments assemble into strong networks that provide strength and resilience to the corneal epithelium.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Meesmann corneal dystrophy     At least three mutations in the KRT3 gene have been found to cause Meesmann corneal dystrophy, an eye disease characterized by the formation of tiny cysts in the corneal epithelium.All of the identified KRT3 gene mutations associated with Meesmann corneal dystrophy change single protein building blocks (amino acids) in the keratin 3 protein. These changes occur in a region of the protein that is critical for the formation and stability of intermediate filaments. The altered keratin 3 protein interferes with the assembly of intermediate filaments, weakening the structural framework of the corneal epithelium. As a result, this outer layer of the cornea is abnormally fragile and develops the cysts that characterize Meesmann corneal dystrophy. The cysts likely contain clumps of abnormal keratin proteins and other cellular debris. When the cysts break open (rupture), they cause eye irritation, increased sensitivity to light (photophobia), and related symptoms. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three mutations in the KRT3 gene have been found to cause Meesmann corneal dystrophy, an eye disease characterized by the formation of tiny cysts in the corneal epithelium.All of the identified KRT3 gene mutations associated with Meesmann corneal dystrophy change single protein building blocks (amino acids) in the keratin 3 protein. These changes occur in a region of the protein that is critical for the formation and stability of intermediate filaments. The altered keratin 3 protein interferes with the assembly of intermediate filaments, weakening the structural framework of the corneal epithelium. As a result, this outer layer of the cornea is abnormally fragile and develops the cysts that characterize Meesmann corneal dystrophy. The cysts likely contain clumps of abnormal keratin proteins and other cellular debris. When the cysts break open (rupture), they cause eye irritation, increased sensitivity to light (photophobia), and related symptoms. More About This Health Condition    Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,789,685 to 52,805,735 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        65 kDa cytokeratinCK-3CK3cytokeratin 3cytokeratin-3K2C3_HUMANK3keratin 3, type IIkeratin, type II cytoskeletal 3type-II keratin Kb3     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): Intermediate FilamentsNational Eye Institute: Facts About the Cornea and Corneal Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KERATIN 3, TYPE II      Research Resources (6 links)     ClinVarHGNC Gene Symbol ReportHuman Intermediate Filament DatabaseMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Intermediate FilamentsNational Eye Institute: Facts About the Cornea and Corneal Disease  Tests of KRT3  PubMed  KERATIN 3, TYPE II  ClinVarHGNC Gene Symbol ReportHuman Intermediate Filament DatabaseMonarch InitiativeNCBI GeneUniProt  Chen YT, Tseng SH, Chao SC. Novel mutations in the helix termination motif of keratin 3 and keratin 12 in 2 Taiwanese families with Meesmann corneal dystrophy. Cornea. 2005 Nov;24(8):928-32. Citation on PubMedIrvine AD, Corden LD, Swensson O, Swensson B, Moore JE, Frazer DG, Smith FJ, Knowlton RG, Christophers E, Rochels R, Uitto J, McLean WH. Mutations in cornea-specific keratin K3 or K12 genes cause Meesmann's corneal dystrophy. Nat Genet. 1997 Jun;16(2):184-7. Citation on PubMedSzaflik JP, Odak M, Maksym RB, Kamiska A, Pollak A, Udziela M, Poski R, Szaflik J. Genetics of Meesmann corneal dystrophy: a novel mutation in the keratin 3 gene in an asymptomatic family suggests genotype-phenotype correlation. Mol Vis. 2008 Sep 15;14:1713-8. Citation on PubMed or Free article on PubMed Central \n",
            "KRT4\t   The KRT4 gene provides instructions for making a protein called keratin 4. Keratins are a group of tough, fibrous proteins that form the structural framework of epithelial cells, which are cells that line the surfaces and cavities of the body. Keratin 4 is found in the moist lining (mucosae) of the mouth, nose, esophagus, genitals, and anus.Keratin 4 partners with a similar protein, keratin 13 (produced from the KRT13 gene), to form molecules known as intermediate filaments. These filaments assemble into strong networks that provide strength and resilience to the different mucosae. Networks of intermediate filaments protect the mucosae from being damaged by friction or other everyday physical stresses.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           White sponge nevus     At least six mutations in the KRT4 gene have been found to cause white sponge nevus, a condition that results in the formation of white patches of tissue called nevi (singular: nevus) that appear as thickened, velvety, sponge-like tissue. These nevi most often occur on the mouth (oral) mucosa (plural: mucosae). Rarely, white sponge nevus occurs on the mucosae of the nose, esophagus, genitals, or anus.The KRT4 gene mutations that cause white sponge nevus disrupt the structure of keratin 4. As a result, keratin 4 does not fit together properly with keratin 13, leading to the formation of irregular intermediate filaments that are easily damaged with little friction or trauma. Fragile intermediate filaments in the oral mucosa might be damaged when eating or brushing one's teeth. Damage to intermediate filaments leads to inflammation and promotes the abnormal growth and division (proliferation) of epithelial cells, causing the mucosae to thicken and resulting in white sponge nevus. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least six mutations in the KRT4 gene have been found to cause white sponge nevus, a condition that results in the formation of white patches of tissue called nevi (singular: nevus) that appear as thickened, velvety, sponge-like tissue. These nevi most often occur on the mouth (oral) mucosa (plural: mucosae). Rarely, white sponge nevus occurs on the mucosae of the nose, esophagus, genitals, or anus.The KRT4 gene mutations that cause white sponge nevus disrupt the structure of keratin 4. As a result, keratin 4 does not fit together properly with keratin 13, leading to the formation of irregular intermediate filaments that are easily damaged with little friction or trauma. Fragile intermediate filaments in the oral mucosa might be damaged when eating or brushing one's teeth. Damage to intermediate filaments leads to inflammation and promotes the abnormal growth and division (proliferation) of epithelial cells, causing the mucosae to thicken and resulting in white sponge nevus. More About This Health Condition    Cytogenetic Location: 12q13.13, which is the long (q) arm of chromosome 12 at position 13.13Molecular Location: base pairs 52,806,543 to 52,814,116 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CK-4CK4CYK4cytokeratin 4K4keratin 4, type IIkeratin, type II cytoskeletal 4type-II keratin Kb4     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Madame Curie Bioscience Database: Keratin Intermediate Filaments and Diseases of the SkinThe Cell: A Molecular Approach (2nd edition, 2000): Intermediate Filaments      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KERATIN 4, TYPE II      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene      Madame Curie Bioscience Database: Keratin Intermediate Filaments and Diseases of the SkinThe Cell: A Molecular Approach (2nd edition, 2000): Intermediate Filaments  Tests of KRT4  PubMed  KERATIN 4, TYPE II  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene  OMIM: KERATIN 4, TYPE IIKimura M, Nagao T, Machida J, Warnakulasuriya S. Mutation of keratin 4 gene causing white sponge nevus in a Japanese family. Int J Oral Maxillofac Surg. 2013 May;42(5):615-8. doi: 10.1016/j.ijom.2012.10.030. Epub 2012 Nov 24. Citation on PubMedShimizu A, Yokoyama Y, Shimomura Y, Ishikawa O. White sponge nevus caused by a missense mutation in the keratin 4 gene. Eur J Dermatol. 2012 Jul-Aug;22(4):571-2. doi: 10.1684/ejd.2012.1765. Citation on PubMedZhang JM, Yang ZW, Chen RY, Gao P, Zhang YR, Zhang LF. Two new mutations in the keratin 4 gene causing oral white sponge nevus in Chinese family. Oral Dis. 2009 Jan;15(1):100-5. doi: 10.1111/j.1601-0825.2008.01498.x. Epub 2008 Oct 25. Citation on PubMed \n",
            "KRT78\t\n",
            "HOXC12\t\n",
            "OR9K2\t\n",
            "OR6C74\t\n",
            "OR6C1\t\n",
            "OR6C65\t\n",
            "OR6C2\t\n",
            "OR6C68\t\n",
            "OR2AP1\t\n",
            "OR10P1\t\n",
            "GLS2\t\n",
            "PTGES3\t\n",
            "SLC26A10\t\n",
            "MRPL51\t\n",
            "ZNF384\t\n",
            "P3H3\t\n",
            "ATN1\t   The ATN1 gene provides instructions for making a protein called atrophin 1.  Although the exact function of this protein is unknown, it appears to play an important role in nerve cells (neurons) in many areas of the brain.  Based on studies in other animals, researchers speculate that atrophin 1 may act as a transcriptional co-repressor.  A transcriptional co-repressor is a protein that interacts with other DNA-binding proteins to suppress the activity of certain genes, although it cannot attach (bind) to DNA by itself.One region of the ATN1 gene contains a particular DNA segment known as a CAG trinucleotide repeat.  This segment is made up of a series of three DNA building blocks (cytosine, adenine, and guanine) that appear multiple times in a row.  In most people, the number of CAG repeats in the ATN1 gene ranges from 6 to 35.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Dentatorubral-pallidoluysian atrophy     Dentatorubral-pallidoluysian atrophy (DRPLA) results from an increased number of copies (expansion) of the CAG trinucleotide repeat in the ATN1 gene. In people with this condition, the CAG segment is abnormally repeated at least 48 times, and the repeat region may be two or three times its usual length. Although the extended CAG region changes the structure of atrophin 1, it is unclear how the altered protein damages brain cells. Researchers believe that abnormal atrophin 1 accumulates in neurons and interferes with normal cell functions. The dysfunction and eventual death of neurons in many parts of the brain lead to involuntary movements, intellectual decline, and the other characteristic features of DRPLA. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Dentatorubral-pallidoluysian atrophy (DRPLA) results from an increased number of copies (expansion) of the CAG trinucleotide repeat in the ATN1 gene. In people with this condition, the CAG segment is abnormally repeated at least 48 times, and the repeat region may be two or three times its usual length. Although the extended CAG region changes the structure of atrophin 1, it is unclear how the altered protein damages brain cells. Researchers believe that abnormal atrophin 1 accumulates in neurons and interferes with normal cell functions. The dysfunction and eventual death of neurons in many parts of the brain lead to involuntary movements, intellectual decline, and the other characteristic features of DRPLA. More About This Health Condition    Cytogenetic Location: 12p13.31, which is the short (p) arm of chromosome 12 at position 13.31Molecular Location: base pairs 6,924,463 to 6,942,321 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATN1_HUMANatrophin-1B37D12S755Edentatorubral-pallidoluysian atrophy proteinDRPLANOD     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Some genetic diseases are caused by the expansion of repeats of three nucleotides      Clinical Information from GeneReviews (1 link)     DRPLA      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATN1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATROPHIN 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Some genetic diseases are caused by the expansion of repeats of three nucleotides  DRPLA  Tests of ATN1  PubMed  ATROPHIN 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Katsuno M, Banno H, Suzuki K, Takeuchi Y, Kawashima M, Tanaka F, Adachi H, Sobue G. Molecular genetics and biomarkers of polyglutamine diseases. Curr Mol Med. 2008 May;8(3):221-34. Review. Citation on PubMedNagafuchi S, Yanagisawa H, Ohsaki E, Shirayama T, Tadokoro K, Inoue T, Yamada M. Structure and expression of the gene responsible for the triplet repeat disorder, dentatorubral and pallidoluysian atrophy (DRPLA). Nat Genet. 1994 Oct;8(2):177-82. Citation on PubMedShen Y, Lee G, Choe Y, Zoltewicz JS, Peterson AS. Functional architecture of atrophins. J Biol Chem. 2007 Feb 16;282(7):5037-44. Epub 2006 Dec 6. Citation on PubMedTsuji S. Dentatorubral-pallidoluysian atrophy: clinical aspects and molecular genetics. Adv Neurol. 2002;89:231-9. Review. Citation on PubMedWood JD, Nucifora FC Jr, Duan K, Zhang C, Wang J, Kim Y, Schilling G, Sacchi N, Liu JM, Ross CA. Atrophin-1, the dentato-rubral and pallido-luysian atrophy gene product, interacts with ETO/MTG8 in the nuclear matrix and represses transcription. J Cell Biol. 2000 Sep 4;150(5):939-48. Citation on PubMed or Free article on PubMed CentralYamada M, Wood JD, Shimohata T, Hayashi S, Tsuji S, Ross CA, Takahashi H. Widespread occurrence of intranuclear atrophin-1 accumulation in the central nervous system neurons of patients with dentatorubral-pallidoluysian atrophy. Ann Neurol. 2001 Jan;49(1):14-23. Citation on PubMed \n",
            "EMG1\t   The EMG1 gene provides instructions for making a protein that is involved in the production of cellular structures called ribosomes, which process the cell's genetic instructions to create new proteins. Ribosomes are assembled in a cell compartment called the nucleolus.The EMG1 protein is involved in the assembly of a part of the ribosome called the small subunit (SSU). In this role, the EMG1 protein functions as part of a protein complex called the SSU processome. In addition to helping to assemble the SSU, the SSU processome is involved in the maturation of a molecule called 18S rRNA, which is a chemical cousin of DNA that is incorporated into the SSU.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Bowen-Conradi syndrome     An EMG1 gene mutation common in the Hutterite population of the United States and Canada causes Bowen-Conradi syndrome, a disorder that affects many parts of the body and is usually fatal in the first few months of life. The mutation that causes Bowen-Conradi syndrome, written as Asp86Gly or D86G, changes the protein building block (amino acid) aspartic acid to the amino acid glycine at position 86 in the EMG1 protein. Studies suggest that the mutation makes the protein unstable, resulting in a decrease in the amount of EMG1 protein that is available in the nucleolus. A shortage of this protein in the nucleolus would impair ribosome production, which may reduce cell growth and division (proliferation); however, it is unknown how EMG1 gene mutations lead to the particular signs and symptoms of Bowen-Conradi syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      An EMG1 gene mutation common in the Hutterite population of the United States and Canada causes Bowen-Conradi syndrome, a disorder that affects many parts of the body and is usually fatal in the first few months of life. The mutation that causes Bowen-Conradi syndrome, written as Asp86Gly or D86G, changes the protein building block (amino acid) aspartic acid to the amino acid glycine at position 86 in the EMG1 protein. Studies suggest that the mutation makes the protein unstable, resulting in a decrease in the amount of EMG1 protein that is available in the nucleolus. A shortage of this protein in the nucleolus would impair ribosome production, which may reduce cell growth and division (proliferation); however, it is unknown how EMG1 gene mutations lead to the particular signs and symptoms of Bowen-Conradi syndrome. More About This Health Condition    Cytogenetic Location: 12p13.31, which is the short (p) arm of chromosome 12 at position 13.31Molecular Location: base pairs 6,970,781 to 6,997,428 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        18S rRNA (pseudouridine-N1-)-methyltransferase NEP118S rRNA (pseudouridine(1248)-N1)-methyltransferase18S rRNA Psi1248 methyltransferaseC2FEMG1 N1-specific pseudouridine methyltransferaseEMG1 nucleolar protein homologessential for mitotic growth 1Grcc2fNEP1NEP1_HUMANribosomal RNA small subunit methyltransferase NEP1ribosome biogenesis protein NEP1     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database: Pre-Ribosomal RNA Processing in Multicellular Organisms      Tests Listed in the Genetic Testing Registry (1 link)     Tests of EMG1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ESSENTIAL FOR MITOTIC GROWTH 1, S. CEREVISIAE, HOMOLOG OF      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Pre-Ribosomal RNA Processing in Multicellular Organisms  Tests of EMG1  PubMed  ESSENTIAL FOR MITOTIC GROWTH 1, S. CEREVISIAE, HOMOLOG OF  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Armistead J, Khatkar S, Meyer B, Mark BL, Patel N, Coghlan G, Lamont RE, Liu S, Wiechert J, Cattini PA, Koetter P, Wrogemann K, Greenberg CR, Entian KD, Zelinski T, Triggs-Raine B. Mutation of a gene essential for ribosome biogenesis, EMG1, causes Bowen-Conradi syndrome. Am J Hum Genet. 2009 Jun;84(6):728-39. doi: 10.1016/j.ajhg.2009.04.017. Epub 2009 May 21. Citation on PubMed or Free article on PubMed CentralArmistead J, Patel N, Wu X, Hemming R, Chowdhury B, Basra GS, Del Bigio MR, Ding H, Triggs-Raine B. Growth arrest in the ribosomopathy, Bowen-Conradi syndrome, is due to dramatically reduced cell proliferation and a defect in mitotic progression. Biochim Biophys Acta. 2015 May;1852(5):1029-37. doi: 10.1016/j.bbadis.2015.02.007. Epub 2015 Feb 20. Citation on PubMedArmistead J, Triggs-Raine B. Diverse diseases from a ubiquitous process: the ribosomopathy paradox. FEBS Lett. 2014 May 2;588(9):1491-500. doi: 10.1016/j.febslet.2014.03.024. Epub 2014 Mar 19. Review. Citation on PubMedOMIM: ESSENTIAL FOR MITOTIC GROWTH 1, S. CEREVISIAE, HOMOLOG OFLamont RE, Loredo-Osti J, Roslin NM, Mauthe J, Coghlan G, Nylen E, Frappier D, Innes AM, Lemire EG, Lowry RB, Greenberg CR, Triggs-Raine BL, Morgan K, Wrogemann K, Fujiwara TM, Zelinski T. A locus for Bowen-Conradi syndrome maps to chromosome region 12p13.3. Am J Med Genet A. 2005 Jan 15;132A(2):136-43. Citation on PubMedSondalle SB, Baserga SJ. Human diseases of the SSU processome. Biochim Biophys Acta. 2014 Jun;1842(6):758-64. doi: 10.1016/j.bbadis.2013.11.004. Epub 2013 Nov 12. Review. Citation on PubMed or Free article on PubMed CentralWurm JP, Meyer B, Bahr U, Held M, Frolow O, Ktter P, Engels JW, Heckel A, Karas M, Entian KD, Whnert J. The ribosome assembly factor Nep1 responsible for Bowen-Conradi syndrome is a pseudouridine-N1-specific methyltransferase. Nucleic Acids Res. 2010 Apr;38(7):2387-98. doi: 10.1093/nar/gkp1189. Epub 2010 Jan 4. Citation on PubMed or Free article on PubMed Central \n",
            "RBP5\t\n",
            "CD163L1\t\n",
            "KCNC2\t   From NCBI Gene:The Shaker gene family of Drosophila encodes components of voltage-gated potassium channels and is comprised of four subfamilies. Based on sequence similarity, this gene is similar to one of these subfamilies, namely the Shaw subfamily. The protein encoded by this gene belongs to the delayed rectifier class of channel proteins and is an integral membrane protein that mediates the voltage-dependent potassium ion permeability of excitable membranes. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2012]From UniProt:Voltage-gated potassium channel that mediates transmembrane potassium transport in excitable membranes, primarily in the brain. Contributes to the regulation of the fast action potential repolarization and in sustained high-frequency firing in neurons of the central nervous system. Homotetramer channels mediate delayed-rectifier voltage-dependent potassium currents that activate rapidly at high-threshold voltages and inactivate slowly. Forms tetrameric channels through which potassium ions pass in accordance with their electrochemical gradient. The channel alternates between opened and closed conformations in response to the voltage difference across the membrane (PubMed:15709110). Can form functional homotetrameric and heterotetrameric channels that contain variable proportions of KCNC1, and possibly other family members as well; channel properties depend on the type of alpha subunits that are part of the channel. Channel properties may be modulated either by the association with ancillary subunits, such as KCNE1, KCNE2 or KCNE3 or indirectly by nitric oxide (NO) through a cGMP- and PKG-mediated signaling cascade, slowing channel activation and deactivation of delayed rectifier potassium channels (By similarity). Contributes to fire sustained trains of very brief action potentials at high frequency in retinal ganglion cells, thalamocortical and suprachiasmatic nucleus (SCN) neurons and in hippocampal and neocortical interneurons (PubMed:15709110). Sustained maximal action potential firing frequency in inhibitory hippocampal interneurons is negatively modulated by histamine H2 receptor activation in a cAMP- and protein kinase (PKA) phosphorylation-dependent manner. Plays a role in maintaining the fidelity of synaptic transmission in neocortical GABAergic interneurons by generating action potential (AP) repolarization at nerve terminals, thus reducing spike-evoked calcium influx and GABA neurotransmitter release. Required for long-range synchronization of gamma oscillations over distance in the neocortex. Contributes to the modulation of the circadian rhythm of spontaneous action potential firing in suprachiasmatic nucleus (SCN) neurons in a light-dependent manner (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving KCNC2 has been found in a mother and her two children with varying degrees of neurodevelopmental delay and cerebellar ataxia. One child also exhibits episodes of unresponsiveness suggestive of absence seizures and facial dysmorphism. Deletion at 12q21.1 deletes exons 3-5 of KCNC2.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q21.1, which is the long (q) arm of chromosome 12 at position 21.1Molecular Location: base pairs 75,040,078 to 75,209,748 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KV3.2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KCNC2      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: POTASSIUM CHANNEL, VOLTAGE-GATED, SHAW-RELATED SUBFAMILY, MEMBER 2      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KCNC2  OMIM: POTASSIUM CHANNEL, VOLTAGE-GATED, SHAW-RELATED SUBFAMILY, MEMBER 2  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GLIPR1L2\t\n",
            "PHLDA1\t\n",
            "OTOGL\t   From NCBI Gene:The protein encoded by this gene belongs to the otogelin family. This gene is expressed in the inner ear of vertebrates with the highest level of expression seen at the embryonic stage and lowest in adult. Knockdown studies in zebrafish suggest that this gene is essential for normal inner ear function. Mutations in this gene are associated with autosomal recessive deafness. [provided by RefSeq, Dec 2012]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 84b     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q21.31, which is the long (q) arm of chromosome 12 at position 21.31Molecular Location: base pairs 80,130,377 to 80,379,090 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C12orf64DFNB84B     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of OTOGL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 84BOMIM: OTOGELIN-LIKE PROTEIN      Research Resources (4 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene      Tests of OTOGL  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 84BOMIM: OTOGELIN-LIKE PROTEIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene \n",
            "PTPRQ\t   From NCBI Gene:This locus encodes a member of the type III receptor-like protein-tyrosine phosphatase family. The encoded protein catalyzes the dephosphorylation of phosphotyrosine and phosphatidylinositol and plays roles in cellular proliferation and differentiation. Mutations at this locus have been linked to autosomal recessive deafness. [provided by RefSeq, Mar 2014]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:DEAFNESS, AUTOSOMAL DOMINANT 73Deafness, autosomal recessive 84     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12q21.31, which is the long (q) arm of chromosome 12 at position 21.31Molecular Location: base pairs 80,424,261 to 80,680,273 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNA73DFNB84DFNB84APTPGMC1R-PTP-Q     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PTPRQ      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DEAFNESS, AUTOSOMAL DOMINANT 73OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 84AOMIM: PROTEIN-TYROSINE PHOSPHATASE, RECEPTOR-TYPE, Q      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene      Tests of PTPRQ  OMIM: DEAFNESS, AUTOSOMAL DOMINANT 73OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 84AOMIM: PROTEIN-TYROSINE PHOSPHATASE, RECEPTOR-TYPE, Q  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene \n",
            "CLEC4A\t\n",
            "ZNF705A\t\n",
            "FAM90A1\t\n",
            "A2ML1\t   From NCBI Gene:This gene encodes a member of the alpha-macroglobulin superfamily. The encoded protein is thought to be an N-glycosylated monomeric protein that acts as an inhibitor of several proteases. It has been shown to form covalent interactions with proteases, and has been reported as the p170 antigen recognized by autoantibodies in the autoimmune disease paraneoplastic pemphigus (PNP; PMID:20805888). Mutations in these gene have also been associated with some cases of Noonan syndrome (NS; PMID:24939586) as well as some cases of otitis media (PMID:26121085). Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Aug 2015]From UniProt:Is able to inhibit all four classes of proteinases by a unique 'trapping' mechanism. This protein has a peptide stretch, called the 'bait region' which contains specific cleavage sites for different proteinases. When a proteinase cleaves the bait region, a conformational change is induced in the protein which traps the proteinase. The entrapped enzyme remains active against low molecular weight substrates (activity against high molecular weight substrates is greatly reduced). Following cleavage in the bait region a thioester bond is hydrolyzed and mediates the covalent binding of the protein to the proteinase (By similarity). Displays inhibitory activity against chymotrypsin, papain, thermolysin, subtilisin A and, to a lesser extent, elastase but not trypsin. May play an important role during desquamation by inhibiting extracellular proteases.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Noonan syndromeFrom NCBI Gene:Otitis media, susceptibility toFrom UniProt:Otitis media (OM): An inflammation of the middle ear resulting in earache, fever, hearing disturbance, and vertigo. [MIM:166760]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12p13.31, which is the short (p) arm of chromosome 12 at position 13.31Molecular Location: base pairs 8,822,472 to 8,887,202 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CPAMD9OMSp170     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of A2ML1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ALPHA-2-MACROGLOBULIN-LIKE 1OMIM: OTITIS MEDIA, SUSCEPTIBILITY TO      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of A2ML1  OMIM: ALPHA-2-MACROGLOBULIN-LIKE 1OMIM: OTITIS MEDIA, SUSCEPTIBILITY TO  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PHC1\t   From NCBI Gene:This gene is a homolog of the Drosophila polyhomeotic gene, which is a member of the Polycomb group of genes. The gene product is a component of a multimeric protein complex that contains EDR2 and the vertebrate Polycomb protein BMH1. The gene product, the EDR2 protein, and the Drosophila polyhomeotic protein share 2 highly conserved domains, named homology domains I and II. These domains are involved in protein-protein interactions and may mediate heterodimerization of the protein encoded by this gene and the EDR2 protein. [provided by RefSeq, Jul 2008]From UniProt:Component of a Polycomb group (PcG) multiprotein PRC1-like complex, a complex class required to maintain the transcriptionally repressive state of many genes, including Hox genes, throughout development. PcG PRC1 complex acts via chromatin remodeling and modification of histones; it mediates monoubiquitination of histone H2A 'Lys-119', rendering chromatin heritably changed in its expressibility. Required for proper control of cellular levels of GMNN expression.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Primary autosomal recessive microcephaly 11From UniProt:Microcephaly 11, primary, autosomal recessive (MCPH11): A form of microcephaly, a disease defined as a head circumference more than 3 standard deviations below the age-related mean. Brain weight is markedly reduced and the cerebral cortex is disproportionately small. [MIM:615414]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12p13.31, which is the short (p) arm of chromosome 12 at position 13.31Molecular Location: base pairs 8,914,664 to 8,941,467 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        EDR1HPH1MCPH11RAE28     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PHC1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MICROCEPHALY 11, PRIMARY, AUTOSOMAL RECESSIVEOMIM: POLYHOMEOTIC-LIKE 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PHC1  OMIM: MICROCEPHALY 11, PRIMARY, AUTOSOMAL RECESSIVEOMIM: POLYHOMEOTIC-LIKE 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "A2M\t   From NCBI Gene:The protein encoded by this gene is a protease inhibitor and cytokine transporter. It uses a bait-and-trap mechanism to inhibit a broad spectrum of proteases, including trypsin, thrombin and collagenase. It can also inhibit inflammatory cytokines, and it thus disrupts inflammatory cascades. Mutations in this gene are a cause of alpha-2-macroglobulin deficiency. This gene is implicated in Alzheimer's disease (AD) due to its ability to mediate the clearance and degradation of A-beta, the major component of beta-amyloid deposits. A related pseudogene, which is also located on the p arm of chromosome 12, has been identified. [provided by RefSeq, Nov 2016]From UniProt:Is able to inhibit all four classes of proteinases by a unique 'trapping' mechanism. This protein has a peptide stretch, called the 'bait region' which contains specific cleavage sites for different proteinases. When a proteinase cleaves the bait region, a conformational change is induced in the protein which traps the proteinase. The entrapped enzyme remains active against low molecular weight substrates (activity against high molecular weight substrates is greatly reduced). Following cleavage in the bait region, a thioester bond is hydrolyzed and mediates the covalent binding of the protein to the proteinase.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Alzheimer's diseaseAlpha-2-macroglobulin deficiency     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 12p13.31, which is the short (p) arm of chromosome 12 at position 13.31Molecular Location: base pairs 9,067,708 to 9,116,229 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        A2MDCPAMD5FWP007S863-7     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of A2M      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ALPHA-2-MACROGLOBULINOMIM: ALPHA-2-MACROGLOBULIN DEFICIENCYOMIM: ALZHEIMER DISEASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of A2M  OMIM: ALPHA-2-MACROGLOBULINOMIM: ALPHA-2-MACROGLOBULIN DEFICIENCYOMIM: ALZHEIMER DISEASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PZP\t\n",
            "WNK1\t   The WNK1 gene provides instructions for making multiple versions (isoforms) of the WNK1 protein. The different WNK1 isoforms are important in several functions in the body, including blood pressure regulation and pain sensation.One isoform produced from the WNK1 gene is the full-length version, called the L-WNK1 protein, which is found in cells throughout the body. A different isoform, called the kidney-specific WNK1 protein or KS-WNK1, is found only in kidney cells. The L-WNK1 and KS-WNK1 proteins act as kinases, which are enzymes that change the activity of other proteins by adding a cluster of oxygen and phosphorus atoms (a phosphate group) at specific positions.The L-WNK1 and KS-WNK1 proteins regulate channels in the cell membrane that control the transport of sodium or potassium into and out of cells. In the kidneys, sodium channels help transport sodium into specialized cells, which then transfer it into the blood. This transfer helps keep sodium in the body through a process called reabsorption. Potassium channels handle excess potassium that has been transferred from the blood into kidney cells. The channels transport potassium out of the cells in a process called secretion, so that it can be removed from the body in urine.The L-WNK1 protein increases sodium reabsorption and decreases potassium secretion, whereas the KS-WNK1 protein has the opposite effect. Sodium and potassium are important for regulating blood pressure, and a balance of L-WNK1 protein and KS-WNK1 protein in the kidneys helps maintain the correct levels of sodium and potassium for healthy blood pressure.Another isoform produced from the WNK1 gene, called the WNK1/HSN2 protein, is found in the cells of the nervous system, including nerve cells that transmit the sensations of pain, temperature, and touch (sensory neurons). The WNK1/HSN2 protein appears to regulate channels in the cell membrane that can transport negatively charged chlorine atoms (chloride ions). These channels maintain the proper amount of chloride inside cells, which is important for controlling the activation (excitation) of the neurons.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hereditary sensory and autonomic neuropathy type II     Mutations in the WNK1 gene are responsible for one type of hereditary sensory and autonomic neuropathy type II (HSAN2) called HSAN2A. People with HSAN2A lose the ability to feel pain or sense hot and cold. More than a dozen mutations in the WNK1 gene have been identified in people with HSAN2A. All of these mutations lead to an abnormally shortened WNK1/HSN2 protein that is probably nonfunctional. People with HSAN2A have a reduction in the number of sensory neurons; however, the role that the abnormal WNK1/HSN2 protein plays in that loss is unclear. The loss of sensory neurons results in the signs and symptoms of HSAN2A.WNK1 gene mutations involved in HSAN2A do not appear to affect the L-WNK1 or KS-WNK1 isoforms. More About This Health Condition      Pseudohypoaldosteronism type 2     At least two mutations in the WNK1 gene have been found to cause pseudohypoaldosteronism type 2 (PHA2), a condition characterized by high blood pressure (hypertension) and high levels of potassium in the blood (hyperkalemia). The mutations involved in this condition delete large numbers of DNA building blocks (nucleotides) from the WNK1 gene. These deletions lead to increased activity of the WNK1 gene and excess L-WNK1 protein. An increase in L-WNK1 protein abnormally increases sodium reabsorption and blocks potassium secretion, resulting in hypertension and hyperkalemia.WNK1 gene mutations involved in PHA2 do not appear to affect the KS-WNK1 or WNK1/HSN2 isoforms. More About This Health Condition      Other disorders     Studies have associated normal variations in the WNK1 gene with an increased risk of high blood pressure (hypertension) in people without PHA2 (described above). A combination of genetic variations and environmental factors likely influence the development of this complex condition.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the WNK1 gene are responsible for one type of hereditary sensory and autonomic neuropathy type II (HSAN2) called HSAN2A. People with HSAN2A lose the ability to feel pain or sense hot and cold. More than a dozen mutations in the WNK1 gene have been identified in people with HSAN2A. All of these mutations lead to an abnormally shortened WNK1/HSN2 protein that is probably nonfunctional. People with HSAN2A have a reduction in the number of sensory neurons; however, the role that the abnormal WNK1/HSN2 protein plays in that loss is unclear. The loss of sensory neurons results in the signs and symptoms of HSAN2A.WNK1 gene mutations involved in HSAN2A do not appear to affect the L-WNK1 or KS-WNK1 isoforms. More About This Health Condition  At least two mutations in the WNK1 gene have been found to cause pseudohypoaldosteronism type 2 (PHA2), a condition characterized by high blood pressure (hypertension) and high levels of potassium in the blood (hyperkalemia). The mutations involved in this condition delete large numbers of DNA building blocks (nucleotides) from the WNK1 gene. These deletions lead to increased activity of the WNK1 gene and excess L-WNK1 protein. An increase in L-WNK1 protein abnormally increases sodium reabsorption and blocks potassium secretion, resulting in hypertension and hyperkalemia.WNK1 gene mutations involved in PHA2 do not appear to affect the KS-WNK1 or WNK1/HSN2 isoforms. More About This Health Condition  Studies have associated normal variations in the WNK1 gene with an increased risk of high blood pressure (hypertension) in people without PHA2 (described above). A combination of genetic variations and environmental factors likely influence the development of this complex condition.    Cytogenetic Location: 12p13.33, which is the short (p) arm of chromosome 12 at position 13.33Molecular Location: base pairs 752,593 to 911,452 on chromosome 12 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HSAN2HSN2KDPp65PPP1R167PRKWNK1prostate-derived sterile 20-like kinaseprotein kinase with no lysine 1protein kinase, lysine deficient 1PSKserine/threonine-protein kinase WNK1WNK1_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Ion Channels and the Electrical Properties of MembranesUndiagnosed Diseases Network: Gene Page      Clinical Information from GeneReviews (2 links)     Hereditary Sensory and Autonomic Neuropathy Type IIPseudohypoaldosteronism Type II      Tests Listed in the Genetic Testing Registry (1 link)     Tests of WNK1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PROTEIN KINASE, LYSINE-DEFICIENT 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Ion Channels and the Electrical Properties of MembranesUndiagnosed Diseases Network: Gene Page  Hereditary Sensory and Autonomic Neuropathy Type IIPseudohypoaldosteronism Type II  Tests of WNK1  PubMed  PROTEIN KINASE, LYSINE-DEFICIENT 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bercier V. WNK1/HSN2 isoform and the regulation of KCC2 activity. Rare Dis. 2013 Sep 19;1:e26537. doi: 10.4161/rdis.26537. eCollection 2013. Citation on PubMed or Free article on PubMed CentralChvez-Canales M, Zhang C, Soukaseum C, Moreno E, Pacheco-Alvarez D, Vidal-Petiot E, Castaeda-Bueno M, Vzquez N, Rojas-Vega L, Meermeier NP, Rogers S, Jeunemaitre X, Yang CL, Ellison DH, Gamba G, Hadchouel J. WNK-SPAK-NCC cascade revisited: WNK1 stimulates the activity of the Na-Cl cotransporter via SPAK, an effect antagonized by WNK4. Hypertension. 2014 Nov;64(5):1047-53. doi: 10.1161/HYPERTENSIONAHA.114.04036. Epub 2014 Aug 11. Citation on PubMedGamba G. Regulation of the renal Na+-Cl- cotransporter by phosphorylation and ubiquitylation. Am J Physiol Renal Physiol. 2012 Dec 15;303(12):F1573-83. doi: 10.1152/ajprenal.00508.2012. Epub 2012 Oct 3. Review. Citation on PubMed or Free article on PubMed CentralHuang CL, Kuo E. Mechanisms of disease: WNK-ing at the mechanism of salt-sensitive hypertension. Nat Clin Pract Nephrol. 2007 Nov;3(11):623-30. Review. Citation on PubMedLazrak A, Liu Z, Huang CL. Antagonistic regulation of ROMK by long and kidney-specific WNK1 isoforms. Proc Natl Acad Sci U S A. 2006 Jan 31;103(5):1615-20. Epub 2006 Jan 20. Citation on PubMed or Free article on PubMed CentralNewhouse S, Farrall M, Wallace C, Hoti M, Burke B, Howard P, Onipinla A, Lee K, Shaw-Hawkins S, Dobson R, Brown M, Samani NJ, Dominiczak AF, Connell JM, Lathrop GM, Kooner J, Chambers J, Elliott P, Clarke R, Collins R, Laan M, Org E, Juhanson P, Veldre G, Viigimaa M, Eyheramendy S, Cappuccio FP, Ji C, Iacone R, Strazzullo P, Kumari M, Marmot M, Brunner E, Caulfield M, Munroe PB. Polymorphisms in the WNK1 gene are associated with blood pressure variation and urinary potassium excretion. PLoS One. 2009;4(4):e5003. doi: 10.1371/journal.pone.0005003. Epub 2009 Apr 4. Citation on PubMed or Free article on PubMed CentralOMIM: PROTEIN KINASE, LYSINE-DEFICIENT 1Shekarabi M, Girard N, Rivire JB, Dion P, Houle M, Toulouse A, Lafrenire RG, Vercauteren F, Hince P, Laganiere J, Rochefort D, Faivre L, Samuels M, Rouleau GA. Mutations in the nervous system--specific HSN2 exon of WNK1 cause hereditary sensory neuropathy type II. J Clin Invest. 2008 Jul;118(7):2496-505. doi: 10.1172/JCI34088. Citation on PubMed or Free article on PubMed CentralVerpoorten N, De Jonghe P, Timmerman V. Disease mechanisms in hereditary sensory and autonomic neuropathies. Neurobiol Dis. 2006 Feb;21(2):247-55. Epub 2005 Sep 23. Review. Citation on PubMedVidal-Petiot E, Elvira-Matelot E, Mutig K, Soukaseum C, Baudrie V, Wu S, Cheval L, Huc E, Cambillau M, Bachmann S, Doucet A, Jeunemaitre X, Hadchouel J. WNK1-related Familial Hyperkalemic Hypertension results from an increased expression of L-WNK1 specifically in the distal nephron. Proc Natl Acad Sci U S A. 2013 Aug 27;110(35):14366-71. doi: 10.1073/pnas.1304230110. Epub 2013 Aug 12. Citation on PubMed or Free article on PubMed CentralWilson FH, Disse-Nicodme S, Choate KA, Ishikawa K, Nelson-Williams C, Desitter I, Gunel M, Milford DV, Lipkin GW, Achard JM, Feely MP, Dussol B, Berland Y, Unwin RJ, Mayan H, Simon DB, Farfel Z, Jeunemaitre X, Lifton RP. Human hypertension caused by mutations in WNK kinases. Science. 2001 Aug 10;293(5532):1107-12. Citation on PubMed \n",
            "CLECL1\t\n",
            "KLRF1\t\n",
            "CLYBL\t\n",
            "ZIC5\t\n",
            "MYO16\t\n",
            "SOX1\t\n",
            "TUBGCP3\t\n",
            "ZMYM2\t   The ZMYM2 gene (previously known as ZNF198) provides instructions for making a protein whose function is not clearly understood. It is a member of a family of zinc finger proteins, which contain one or more short regions called zinc finger domains. The zinc finger domains in the ZMYM2 protein are thought to allow it to regulate interactions between other proteins. ZMYM2 is found in the nucleus of the cell, where it likely associates with other proteins. Through these associations, the ZMYM2 protein may be involved in repairing DNA errors, controlling gene activity, or forming structures in the nucleus called PML nuclear bodies that block the growth and division of cells and promote their self-destruction (apoptosis).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           8p11 myeloproliferative syndrome     A genetic change involving the ZMYM2 gene causes most cases of 8p11 myeloproliferative syndrome. This condition is characterized by an increased number of white blood cells (myeloproliferative disorder) and the development of lymphoma, a blood-related cancer that causes tumor formation in the lymph nodes. The myeloproliferative disorder usually develops into another form of blood cancer called acute myeloid leukemia. 8p11 myeloproliferative syndrome most commonly results from a rearrangement (translocation) of genetic material between chromosome 13 and chromosome 8. This genetic change fuses part of the ZMYM2 gene on chromosome 13 with part of the FGFR1 gene on chromosome 8. The translocation is found only in cancer cells.The protein produced from the normal FGFR1 gene can turn on cellular signaling that helps the cell respond to its environment, for example by stimulating cell growth. The protein produced from the fused ZMYM2-FGFR1 gene leads to constant FGFR1 signaling. The uncontrolled signaling promotes continuous cell growth and division, leading to cancer. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A genetic change involving the ZMYM2 gene causes most cases of 8p11 myeloproliferative syndrome. This condition is characterized by an increased number of white blood cells (myeloproliferative disorder) and the development of lymphoma, a blood-related cancer that causes tumor formation in the lymph nodes. The myeloproliferative disorder usually develops into another form of blood cancer called acute myeloid leukemia. 8p11 myeloproliferative syndrome most commonly results from a rearrangement (translocation) of genetic material between chromosome 13 and chromosome 8. This genetic change fuses part of the ZMYM2 gene on chromosome 13 with part of the FGFR1 gene on chromosome 8. The translocation is found only in cancer cells.The protein produced from the normal FGFR1 gene can turn on cellular signaling that helps the cell respond to its environment, for example by stimulating cell growth. The protein produced from the fused ZMYM2-FGFR1 gene leads to constant FGFR1 signaling. The uncontrolled signaling promotes continuous cell growth and division, leading to cancer. More About This Health Condition    Cytogenetic Location: 13q12.11, which is the long (q) arm of chromosome 13 at position 12.11Molecular Location: base pairs 19,957,406 to 20,091,845 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FIMfused in myeloproliferative disorders proteinMYMRAMPrearranged in an atypical myeloproliferative disorderSCLLzinc finger MYM-type protein 2zinc finger protein 198zinc finger, MYM-type 2ZMYM2_HUMANZNF198     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Cell Biology (fourth edition, 2000): Zinc-Finger Proteins      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ZMYM2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ZINC FINGER, MYM-TYPE 2      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Zinc-Finger Proteins  Tests of ZMYM2  PubMed  ZINC FINGER, MYM-TYPE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Gocke CB, Yu H. ZNF198 stabilizes the LSD1-CoREST-HDAC1 complex on chromatin through its MYM-type zinc fingers. PLoS One. 2008 Sep 22;3(9):e3255. doi: 10.1371/journal.pone.0003255. Citation on PubMed or Free article on PubMed CentralJackson CC, Medeiros LJ, Miranda RN. 8p11 myeloproliferative syndrome: a review. Hum Pathol. 2010 Apr;41(4):461-76. doi: 10.1016/j.humpath.2009.11.003. Review. Citation on PubMedKunapuli P, Kasyapa CS, Chin SF, Caldas C, Cowell JK. ZNF198, a zinc finger protein rearranged in myeloproliferative disease, localizes to the PML nuclear bodies and interacts with SUMO-1 and PML. Exp Cell Res. 2006 Nov 15;312(19):3739-51. Epub 2006 Aug 14. Citation on PubMedKunapuli P, Somerville R, Still IH, Cowell JK. ZNF198 protein, involved in rearrangement in myeloproliferative disease, forms complexes with the DNA repair-associated HHR6A/6B and RAD18 proteins. Oncogene. 2003 May 29;22(22):3417-23. Citation on PubMedXiao S, McCarthy JG, Aster JC, Fletcher JA. ZNF198-FGFR1 transforming activity depends on a novel proline-rich ZNF198 oligomerization domain. Blood. 2000 Jul 15;96(2):699-704. Citation on PubMedXiao S, Nalabolu SR, Aster JC, Ma J, Abruzzo L, Jaffe ES, Stone R, Weissman SM, Hudson TJ, Fletcher JA. FGFR1 is fused with a novel zinc-finger gene, ZNF198, in the t(8;13) leukaemia/lymphoma syndrome. Nat Genet. 1998 Jan;18(1):84-7. Citation on PubMedOMIM: ZINC FINGER, MYM-TYPE 2 \n",
            "GJB2\t   The GJB2 gene provides instructions for making a protein called gap junction beta 2, more commonly known as connexin 26.  Connexin 26 is a member of the connexin protein family. Connexin proteins form channels called gap junctions that permit the transport of nutrients, charged atoms (ions), and signaling molecules between adjoining cells. The size of the gap junction and the types of particles that move through it are determined by the particular connexin proteins that make up the channel. Gap junctions made with connexin 26 transport potassium ions and certain small molecules.Connexin 26 is found in cells throughout the body, including the inner ear. Because of its presence in the inner ear, especially the snail-shaped structure called the cochlea, researchers are interested in this protein's role in hearing. Hearing requires the conversion of sound waves to electrical nerve impulses. This conversion involves many processes, including maintenance of the proper level of potassium ions in the inner ear. Some studies indicate that channels made with connexin 26 help to maintain the correct level of potassium ions. Other research suggests that connexin 26 is required for the maturation of certain cells in the cochlea.Connexin 26 is also found in the skin. It is thought to play a role in the growth, maturation, and stability of the skin's outermost layer, the epidermis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Bart-Pumphrey syndrome     At least two GJB2 gene mutations have been identified in people with Bart-Pumphrey syndrome. This condition is characterized by a white discoloration of the nails (leukonychia), thickened skin on the palms of the hands and soles of the feet (palmoplantar keratoderma), wart-like growths (knuckle pads) on the knuckles of the fingers and toes, and hearing loss.  The GJB2 gene mutations that cause Bart-Pumphrey syndrome replace the protein building block (amino acid) glycine with the amino acid serine at protein position 59 (Gly59Ser or G59S) or replace the amino acid asparagine with the amino acid lysine at protein position 54 (Asn54Lys or N54K). The altered protein probably disrupts the function of normal connexin 26 in cells. This disruption could affect skin growth and also impair hearing by disturbing the conversion of sound waves to nerve impulses. More About This Health Condition      Hystrix-like ichthyosis with deafness     At least one GJB2 gene mutation has been identified in people with hystrix-like ichthyosis with deafness (HID), a disorder characterized by dry, scaly skin (ichthyosis) and hearing loss that is usually profound. This mutation replaces the amino acid aspartic acid with the amino acid asparagine at protein position 50, written as Asp50Asn or D50N.  The mutation is thought to result in channels that constantly leak ions, which impairs the health of the cells and increases cell death. Death of cells in the skin and the inner ear may underlie the signs and symptoms of HID.Because the D50N mutation can also cause keratitis-ichthyosis-deafness (KID) syndrome (described below), many researchers categorize KID syndrome and HID as a single disorder, which they call KID/HID. It is not known why some people with this gene mutation have eye problems while others do not. More About This Health Condition      Keratitis-ichthyosis-deafness syndrome     At least nine GJB2 gene mutations have been identified in people with keratitis-ichthyosis-deafness (KID) syndrome, with the most common being the D50N mutation that also occurs in hystrix-like ichthyosis with deafness (described above). KID syndrome is characterized by keratitis, which is inflammation of the front surface of the eye (the cornea); thick, reddened patches of dry and scaly skin (ichthyosis); and deafness.The GJB2 gene mutations that cause KID syndrome change single amino acids in connexin 26. The mutations are thought to result in channels that constantly leak ions, which impairs the health of the cells and increases cell death. Death of cells in the skin and the inner ear may underlie the ichthyosis and deafness that occur in KID syndrome. It is unclear how GJB2 gene mutations affect the eye. More About This Health Condition      Nonsyndromic hearing loss     Researchers have identified more than 100 GJB2 gene mutations that can cause nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene can cause two forms of nonsyndromic hearing loss: DFNB1 and DFNA3.DFNB1 is inherited in an autosomal recessive pattern, which means both copies of the GJB2 gene are mutated in each cell. This form of the condition accounts for about half of all cases of autosomal recessive nonsyndromic hearing loss. It is characterized by mild to profound hearing loss that is present before a child learns to speak (prelingual) and does not become more severe over time.Some of the mutations that cause DFNB1 delete or insert DNA building blocks (base pairs) within or near the GJB2 gene. The most common mutation in many populations, particularly in people of northern European descent, deletes one base pair at position 35 in the GJB2 gene (written as 35delG). In Asian populations, a frequently reported mutation deletes a base pair at position 235 (235delC). Among people with eastern European (Ashkenazi) Jewish ancestry, the deletion of a single base pair at position 167 (167delT) is a common mutation. Less frequently, GJB2 gene mutations that cause DFNB1 replace one base pair with an incorrect one or delete a segment of DNA near the gene.The GJB2 gene mutations that result in DFNB1 are described as \"loss of function\" because they lead to an altered or nonfunctional version of connexin 26, which appears to disrupt the assembly or function of gap junctions. In the inner ear, the abnormal or missing gap junctions likely alter the levels of potassium ions, which may affect the function and survival of cells that are needed for hearing.DFNA3 is inherited in an autosomal dominant pattern, which means only one mutated copy of the GJB2 gene in each cell is sufficient to cause the condition. This form of hearing loss can be either prelingual or begin after a child learns to speak (postlingual). The hearing loss ranges from mild to profound, becomes more severe over time, and particularly affects the ability to hear high-frequency sounds.The GJB2 gene mutations that cause DFNA3 replace one amino acid in connexin 26 with an incorrect amino acid. These mutations are described as \"dominant negative,\" which means that they lead to an abnormal version of connexin 26 that prevents the formation of any functional gap junctions. An absence of these channels probably affects the function and survival of cells in the inner ear that are essential for hearing. More About This Health Condition      Palmoplantar keratoderma with deafness     At least nine GJB2 gene mutations have been identified in people with palmoplantar keratoderma with deafness, a condition characterized by hearing loss and unusually thick skin on the palms of the hands and soles of the feet.  The GJB2 gene mutations that cause this condition change single amino acids in connexin 26. The altered protein probably disrupts the function of normal connexin 26 in cells and may interfere with the function of other connexin proteins. This disruption could affect skin growth and also impair hearing by disturbing the conversion of sound waves to nerve impulses. More About This Health Condition      Vohwinkel syndrome     At least three GJB2 gene mutations have been identified in people with Vohwinkel syndrome, a condition characterized by hearing loss and skin abnormalities. In addition to abnormal patches of skin, affected individuals develop tight bands of abnormal fibrous tissue around their fingers and toes that may cut off the circulation to the digits and result in spontaneous amputation. The GJB2 gene mutations that cause Vohwinkel syndrome change single amino acids in connexin 26. The altered protein probably disrupts the function of normal connexin 26 in cells and may interfere with the function of other connexin proteins. These abnormalities could affect skin growth and also impair hearing by disturbing the conversion of sound waves to nerve impulses. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two GJB2 gene mutations have been identified in people with Bart-Pumphrey syndrome. This condition is characterized by a white discoloration of the nails (leukonychia), thickened skin on the palms of the hands and soles of the feet (palmoplantar keratoderma), wart-like growths (knuckle pads) on the knuckles of the fingers and toes, and hearing loss.  The GJB2 gene mutations that cause Bart-Pumphrey syndrome replace the protein building block (amino acid) glycine with the amino acid serine at protein position 59 (Gly59Ser or G59S) or replace the amino acid asparagine with the amino acid lysine at protein position 54 (Asn54Lys or N54K). The altered protein probably disrupts the function of normal connexin 26 in cells. This disruption could affect skin growth and also impair hearing by disturbing the conversion of sound waves to nerve impulses. More About This Health Condition  At least one GJB2 gene mutation has been identified in people with hystrix-like ichthyosis with deafness (HID), a disorder characterized by dry, scaly skin (ichthyosis) and hearing loss that is usually profound. This mutation replaces the amino acid aspartic acid with the amino acid asparagine at protein position 50, written as Asp50Asn or D50N.  The mutation is thought to result in channels that constantly leak ions, which impairs the health of the cells and increases cell death. Death of cells in the skin and the inner ear may underlie the signs and symptoms of HID.Because the D50N mutation can also cause keratitis-ichthyosis-deafness (KID) syndrome (described below), many researchers categorize KID syndrome and HID as a single disorder, which they call KID/HID. It is not known why some people with this gene mutation have eye problems while others do not. More About This Health Condition  At least nine GJB2 gene mutations have been identified in people with keratitis-ichthyosis-deafness (KID) syndrome, with the most common being the D50N mutation that also occurs in hystrix-like ichthyosis with deafness (described above). KID syndrome is characterized by keratitis, which is inflammation of the front surface of the eye (the cornea); thick, reddened patches of dry and scaly skin (ichthyosis); and deafness.The GJB2 gene mutations that cause KID syndrome change single amino acids in connexin 26. The mutations are thought to result in channels that constantly leak ions, which impairs the health of the cells and increases cell death. Death of cells in the skin and the inner ear may underlie the ichthyosis and deafness that occur in KID syndrome. It is unclear how GJB2 gene mutations affect the eye. More About This Health Condition  Researchers have identified more than 100 GJB2 gene mutations that can cause nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene can cause two forms of nonsyndromic hearing loss: DFNB1 and DFNA3.DFNB1 is inherited in an autosomal recessive pattern, which means both copies of the GJB2 gene are mutated in each cell. This form of the condition accounts for about half of all cases of autosomal recessive nonsyndromic hearing loss. It is characterized by mild to profound hearing loss that is present before a child learns to speak (prelingual) and does not become more severe over time.Some of the mutations that cause DFNB1 delete or insert DNA building blocks (base pairs) within or near the GJB2 gene. The most common mutation in many populations, particularly in people of northern European descent, deletes one base pair at position 35 in the GJB2 gene (written as 35delG). In Asian populations, a frequently reported mutation deletes a base pair at position 235 (235delC). Among people with eastern European (Ashkenazi) Jewish ancestry, the deletion of a single base pair at position 167 (167delT) is a common mutation. Less frequently, GJB2 gene mutations that cause DFNB1 replace one base pair with an incorrect one or delete a segment of DNA near the gene.The GJB2 gene mutations that result in DFNB1 are described as \"loss of function\" because they lead to an altered or nonfunctional version of connexin 26, which appears to disrupt the assembly or function of gap junctions. In the inner ear, the abnormal or missing gap junctions likely alter the levels of potassium ions, which may affect the function and survival of cells that are needed for hearing.DFNA3 is inherited in an autosomal dominant pattern, which means only one mutated copy of the GJB2 gene in each cell is sufficient to cause the condition. This form of hearing loss can be either prelingual or begin after a child learns to speak (postlingual). The hearing loss ranges from mild to profound, becomes more severe over time, and particularly affects the ability to hear high-frequency sounds.The GJB2 gene mutations that cause DFNA3 replace one amino acid in connexin 26 with an incorrect amino acid. These mutations are described as \"dominant negative,\" which means that they lead to an abnormal version of connexin 26 that prevents the formation of any functional gap junctions. An absence of these channels probably affects the function and survival of cells in the inner ear that are essential for hearing. More About This Health Condition  At least nine GJB2 gene mutations have been identified in people with palmoplantar keratoderma with deafness, a condition characterized by hearing loss and unusually thick skin on the palms of the hands and soles of the feet.  The GJB2 gene mutations that cause this condition change single amino acids in connexin 26. The altered protein probably disrupts the function of normal connexin 26 in cells and may interfere with the function of other connexin proteins. This disruption could affect skin growth and also impair hearing by disturbing the conversion of sound waves to nerve impulses. More About This Health Condition  At least three GJB2 gene mutations have been identified in people with Vohwinkel syndrome, a condition characterized by hearing loss and skin abnormalities. In addition to abnormal patches of skin, affected individuals develop tight bands of abnormal fibrous tissue around their fingers and toes that may cut off the circulation to the digits and result in spontaneous amputation. The GJB2 gene mutations that cause Vohwinkel syndrome change single amino acids in connexin 26. The altered protein probably disrupts the function of normal connexin 26 in cells and may interfere with the function of other connexin proteins. These abnormalities could affect skin growth and also impair hearing by disturbing the conversion of sound waves to nerve impulses. More About This Health Condition    Cytogenetic Location: 13q12.11, which is the long (q) arm of chromosome 13 at position 12.11Molecular Location: base pairs 20,187,463 to 20,192,975 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CX26CXB2_HUMANDFNA3DFNB1gap junction protein, beta 2, 26kDaNSRD1     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Biochemistry (fifth edition, 2002): Gap Junctions Allow Ions and Small Molecules to Flow between Communicating CellsMadame Curie Bioscience Database: Gap Junctions: Cell-Cell Channels in AnimalsMolecular Biology of the Cell (fourth edition, 2002): Gap Junctions Allow Small Molecules to Pass Directly from Cell to Cell      Clinical Information from GeneReviews (3 links)     Hereditary Hearing Loss and Deafness OverviewNonsyndromic Hearing Loss and Deafness, DFNA3Nonsyndromic Hearing Loss and Deafness, DFNB1      Tests Listed in the Genetic Testing Registry (1 link)     Tests of GJB2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     GAP JUNCTION PROTEIN, BETA-2      Research Resources (8 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarConnexin-Deafness HomepageHereditary Hearing Loss HomepageHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Gap Junctions Allow Ions and Small Molecules to Flow between Communicating CellsMadame Curie Bioscience Database: Gap Junctions: Cell-Cell Channels in AnimalsMolecular Biology of the Cell (fourth edition, 2002): Gap Junctions Allow Small Molecules to Pass Directly from Cell to Cell  Hereditary Hearing Loss and Deafness OverviewNonsyndromic Hearing Loss and Deafness, DFNA3Nonsyndromic Hearing Loss and Deafness, DFNB1  Tests of GJB2  PubMed  GAP JUNCTION PROTEIN, BETA-2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarConnexin-Deafness HomepageHereditary Hearing Loss HomepageHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: GAP JUNCTION PROTEIN, BETA-2Lee JR, White TW. Connexin-26 mutations in deafness and skin disease. Expert Rev Mol Med. 2009 Nov 19;11:e35. doi: 10.1017/S1462399409001276. Review. Citation on PubMedMarlin S, Feldmann D, Blons H, Loundon N, Rouillon I, Albert S, Chauvin P, Garabdian EN, Couderc R, Odent S, Joannard A, Schmerber S, Delobel B, Leman J, Journel H, Catros H, Lemarechal C, Dollfus H, Eliot MM, Delaunoy JL, David A, Calais C, Drouin-Garraud V, Obstoy MF, Goizet C, Duriez F, Fellmann F, Hlias J, Vigneron J, Montaut B, Matin-Coignard D, Faivre L, Baumann C, Lewin P, Petit C, Denoyelle F. GJB2 and GJB6 mutations: genotypic and phenotypic correlations in a large cohort of hearing-impaired patients. Arch Otolaryngol Head Neck Surg. 2005 Jun;131(6):481-7. Citation on PubMedNickel R, Forge A. Gap junctions and connexins in the inner ear: their roles in homeostasis and deafness. Curr Opin Otolaryngol Head Neck Surg. 2008 Oct;16(5):452-7. doi: 10.1097/MOO.0b013e32830e20b0. Review. Citation on PubMedSanchez HA, Verselis VK. Aberrant Cx26 hemichannels and keratitis-ichthyosis-deafness syndrome: insights into syndromic hearing loss. Front Cell Neurosci. 2014 Oct 27;8:354. doi: 10.3389/fncel.2014.00354. eCollection 2014. Review. Citation on PubMed or Free article on PubMed CentralSmith RJH, Jones MKN. Nonsyndromic Hearing Loss and Deafness, DFNB1. 1998 Sep 28 [updated 2016 Aug 18]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1272/ Citation on PubMedSmith RJH, Ranum PT. Nonsyndromic Hearing Loss and Deafness, DFNA3. 1998 Sep 28 [updated 2016 Dec 22]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1536/ Citation on PubMedTsukada K, Nishio SY, Hattori M, Usami S. Ethnic-specific spectrum of GJB2 and SLC26A4 mutations: their origin and a literature review. Ann Otol Rhinol Laryngol. 2015 May;124 Suppl 1:61S-76S. doi: 10.1177/0003489415575060. Review. Citation on PubMedXu J, Nicholson BJ. The role of connexins in ear and skin physiology - functional insights from disease-associated mutations. Biochim Biophys Acta. 2013 Jan;1828(1):167-78. doi: 10.1016/j.bbamem.2012.06.024. Epub 2012 Jul 13. Review. Citation on PubMed or Free article on PubMed Central \n",
            "SGCG\t   The SGCG gene provides instructions for making the gamma component (subunit) of a group of proteins called the sarcoglycan protein complex. The sarcoglycan protein complex is located in the membrane surrounding muscle cells. It helps maintain the structure of muscle tissue by attaching (binding) to and stabilizing the dystrophin complex, which is made up of proteins called dystrophins and dystroglycans. The large dystrophin complex strengthens muscle fibers and protects them from injury as muscles tense (contract) and relax. It acts as an anchor, connecting each muscle cell's structural framework (cytoskeleton) with the lattice of proteins and other molecules outside the cell (extracellular matrix).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Limb-girdle muscular dystrophy     Approximately 40 mutations in the SGCG gene have been identified in people with limb-girdle muscular dystrophy type 2C. Limb-girdle muscular dystrophy is a group of related disorders characterized by muscle weakness and wasting, particularly in the shoulders, hips, and limbs. Forms of limb-girdle muscular dystrophy caused by gene mutations that affect the sarcoglycan complex are called sarcoglycanopathies.SGCG gene mutations may prevent the sarcoglycan complex from forming or from binding to and stabilizing the dystrophin complex. Problems with these complexes reduce the strength and resilience of muscle fibers and result in the signs and symptoms of limb-girdle muscular dystrophy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Approximately 40 mutations in the SGCG gene have been identified in people with limb-girdle muscular dystrophy type 2C. Limb-girdle muscular dystrophy is a group of related disorders characterized by muscle weakness and wasting, particularly in the shoulders, hips, and limbs. Forms of limb-girdle muscular dystrophy caused by gene mutations that affect the sarcoglycan complex are called sarcoglycanopathies.SGCG gene mutations may prevent the sarcoglycan complex from forming or from binding to and stabilizing the dystrophin complex. Problems with these complexes reduce the strength and resilience of muscle fibers and result in the signs and symptoms of limb-girdle muscular dystrophy. More About This Health Condition    Cytogenetic Location: 13q12.12, which is the long (q) arm of chromosome 13 at position 12.12Molecular Location: base pairs 23,160,308 to 23,325,165 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        35 kDa dystrophin-associated glycoprotein35DAG35kD dystrophin-associated glycoproteinA4DAGA4DMDADMDA1gamma sarcoglycangamma-sarcoglycangamma-SGLGMD2CMAMMGC130048sarcoglycan, gamma (35kDa dystrophin-associated glycoprotein)SCARMD2SCG3SG-gammaSGCG_HUMANTYPE     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     University of Washington Neuromuscular Disease Center      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SGCG      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SARCOGLYCAN, GAMMA      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      University of Washington Neuromuscular Disease Center  Tests of SGCG  PubMed  SARCOGLYCAN, GAMMA  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Broglio L, Tentorio M, Cotelli MS, Mancuso M, Vielmi V, Gregorelli V, Padovani A, Filosto M. Limb-girdle muscular dystrophy-associated protein diseases. Neurologist. 2010 Nov;16(6):340-52. doi: 10.1097/NRL.0b013e3181d35b39. Review. Citation on PubMedBnnemann CG, Wong J, Jones KJ, Lidov HG, Feener CA, Shapiro F, Darras BT, Kunkel LM, North KN. Primary gamma-sarcoglycanopathy (LGMD 2C): broadening of the mutational spectrum guided by the immunohistochemical profile. Neuromuscul Disord. 2002 Mar;12(3):273-80. Citation on PubMedGeorgieva B, Todorova A, Tournev I, Mitev V, Kremensky I. C283Y gamma-sarcoglycan gene mutation in the Bulgarian Roma (Gypsy) population: prevalence study and carrier screening in a high-risk community. Clin Genet. 2004 Nov;66(5):467-72. Citation on PubMedGuglieri M, Straub V, Bushby K, Lochmller H. Limb-girdle muscular dystrophies. Curr Opin Neurol. 2008 Oct;21(5):576-84. doi: 10.1097/WCO.0b013e32830efdc2. Review. Citation on PubMedHack AA, Groh ME, McNally EM. Sarcoglycans in muscular dystrophy. Microsc Res Tech. 2000 Feb 1-15;48(3-4):167-80. Review. Citation on PubMedMeena AK, Sreenivas D, Sundaram C, Rajasekhar R, Sita JS, Borgohain R, Suvarna A, Kaul S. Sarcoglycanopathies: a clinico-pathological study. Neurol India. 2007 Apr-Jun;55(2):117-21. Citation on PubMedOzawa E, Mizuno Y, Hagiwara Y, Sasaoka T, Yoshida M. Molecular and cell biology of the sarcoglycan complex. Muscle Nerve. 2005 Nov;32(5):563-76. Review. Citation on PubMedOMIM: SARCOGLYCAN, GAMMASandon D, Betto R. Sarcoglycanopathies: molecular pathogenesis and therapeutic prospects. Expert Rev Mol Med. 2009 Sep 28;11:e28. doi: 10.1017/S1462399409001203. Review. Citation on PubMed or Free article on PubMed CentralStraub V, Bushby K. The childhood limb-girdle muscular dystrophies. Semin Pediatr Neurol. 2006 Jun;13(2):104-14. Review. Citation on PubMedTrabelsi M, Kavian N, Daoud F, Commere V, Deburgrave N, Beugnet C, Llense S, Barbot JC, Vasson A, Kaplan JC, Leturcq F, Chelly J. Revised spectrum of mutations in sarcoglycanopathies. Eur J Hum Genet. 2008 Jul;16(7):793-803. doi: 10.1038/ejhg.2008.9. Epub 2008 Feb 20. Citation on PubMed \n",
            "PARP4\t\n",
            "PABPC3\t\n",
            "FLT3\t   The FLT3 gene provides instructions for making a protein called fms-like tyrosine kinase 3 (FLT3), which is part of a family of proteins called receptor tyrosine kinases (RTKs). Receptor tyrosine kinases transmit signals from the cell surface into the cell through a process called signal transduction. The FLT3 protein is found in the outer membrane of certain cell types where a specific protein called FLT3 ligand, or FL, can attach (bind) to it. This binding turns on (activates) the FLT3 protein, which subsequently activates a series of proteins inside the cell that are part of multiple signaling pathways. The signaling pathways stimulated by the FLT3 protein control many important cellular processes such as the growth and division (proliferation) and survival of cells, particularly of early blood cells called hematopoietic progenitor cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Core binding factor acute myeloid leukemia     Genetics Home Reference provides information about core binding factor acute myeloid leukemia. More About This Health Condition      Cytogenetically normal acute myeloid leukemia     Changes in the FLT3 gene are involved in a form of blood cancer known as cytogenetically normal acute myeloid leukemia (CN-AML). While large chromosomal abnormalities can be involved in the development of acute myeloid leukemia, about half of cases do not have these abnormalities; these are classified as CN-AML.The FLT3 gene mutations involved in CN-AML are called somatic mutations; they are found only in cells that become cancerous and are not inherited. Two types of FLT3 gene mutations are found in CN-AML. The most common, which occurs in up to 34 percent of CN-AML cases, is called the FLT3 internal tandem duplication (FLT3-ITD). In this type of mutation, a short sequence of DNA is copied and inserted directly following the original sequence. The duplicated DNA sequence can vary in size, but all FLT3-ITD mutations result in alterations in the region of the protein that spans the cell membrane, known as the juxtamembrane domain. The altered juxtamembrane domain allows the FLT3 receptor to activate signaling pathways without binding of FL; the receptor is always turned on and is said to be constitutively activated. Constant signaling leads to uncontrolled proliferation of abnormal, immature white blood cells, a hallmark of acute myeloid leukemia.The other type of FLT3 gene mutation is found in about 14 percent of people with CN-AML. These mutations are referred to as FLT3-TKD mutations because they change single protein building blocks (amino acids) in a region of the protein known as the tyrosine kinase domain (TKD). The most commonly changed amino acid is asparagine at position 835; it is typically replaced by the amino acid tyrosine. This mutation is written as Asp835Tyr or D835Y. Like FLT3-ITD mutations, FLT3-TKD mutations result in a constitutively activated FLT3 receptor and constant signaling, leading to acute myeloid leukemia. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about core binding factor acute myeloid leukemia. More About This Health Condition  Changes in the FLT3 gene are involved in a form of blood cancer known as cytogenetically normal acute myeloid leukemia (CN-AML). While large chromosomal abnormalities can be involved in the development of acute myeloid leukemia, about half of cases do not have these abnormalities; these are classified as CN-AML.The FLT3 gene mutations involved in CN-AML are called somatic mutations; they are found only in cells that become cancerous and are not inherited. Two types of FLT3 gene mutations are found in CN-AML. The most common, which occurs in up to 34 percent of CN-AML cases, is called the FLT3 internal tandem duplication (FLT3-ITD). In this type of mutation, a short sequence of DNA is copied and inserted directly following the original sequence. The duplicated DNA sequence can vary in size, but all FLT3-ITD mutations result in alterations in the region of the protein that spans the cell membrane, known as the juxtamembrane domain. The altered juxtamembrane domain allows the FLT3 receptor to activate signaling pathways without binding of FL; the receptor is always turned on and is said to be constitutively activated. Constant signaling leads to uncontrolled proliferation of abnormal, immature white blood cells, a hallmark of acute myeloid leukemia.The other type of FLT3 gene mutation is found in about 14 percent of people with CN-AML. These mutations are referred to as FLT3-TKD mutations because they change single protein building blocks (amino acids) in a region of the protein known as the tyrosine kinase domain (TKD). The most commonly changed amino acid is asparagine at position 835; it is typically replaced by the amino acid tyrosine. This mutation is written as Asp835Tyr or D835Y. Like FLT3-ITD mutations, FLT3-TKD mutations result in a constitutively activated FLT3 receptor and constant signaling, leading to acute myeloid leukemia. More About This Health Condition    Cytogenetic Location: 13q12.2, which is the long (q) arm of chromosome 13 at position 12.2Molecular Location: base pairs 28,003,274 to 28,100,592 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD135CD135 antigenfetal liver kinase 2FL cytokine receptorFLK-2FLK2FLT3_HUMANfms-like tyrosine kinase 3fms-related tyrosine kinase 3growth factor receptor tyrosine kinase type IIIreceptor-type tyrosine-protein kinase FLT3stem cell tyrosine kinase 1STK-1STK1     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): Activated Receptor Tyrosine Kinases Phosphorylate ThemselvesThe Cell: A Molecular Approach (second edition, 2000): Receptor Protein-Tyrosine Kinases      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FLT3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FMS-RELATED TYROSINE KINASE 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Activated Receptor Tyrosine Kinases Phosphorylate ThemselvesThe Cell: A Molecular Approach (second edition, 2000): Receptor Protein-Tyrosine Kinases  Tests of FLT3  PubMed  FMS-RELATED TYROSINE KINASE 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Choudhary C, Schwble J, Brandts C, Tickenbrock L, Sargin B, Kindler T, Fischer T, Berdel WE, Mller-Tidow C, Serve H. AML-associated Flt3 kinase domain mutations show signal transduction differences compared with Flt3 ITD mutations. Blood. 2005 Jul 1;106(1):265-73. Epub 2005 Mar 15. Citation on PubMedOMIM: FMS-RELATED TYROSINE KINASE 3Gilliland DG, Griffin JD. The roles of FLT3 in hematopoiesis and leukemia. Blood. 2002 Sep 1;100(5):1532-42. Review. Citation on PubMedKiyoi H, Ohno R, Ueda R, Saito H, Naoe T. Mechanism of constitutive activation of FLT3 with internal tandem duplication in the juxtamembrane domain. Oncogene. 2002 Apr 11;21(16):2555-63. Citation on PubMedLi L, Piloto O, Kim KT, Ye Z, Nguyen HB, Yu X, Levis M, Cheng L, Small D. FLT3/ITD expression increases expansion, survival and entry into cell cycle of human haematopoietic stem/progenitor cells. Br J Haematol. 2007 Apr;137(1):64-75. Citation on PubMedYamamoto Y, Kiyoi H, Nakano Y, Suzuki R, Kodera Y, Miyawaki S, Asou N, Kuriyama K, Yagasaki F, Shimazaki C, Akiyama H, Saito K, Nishimura M, Motoji T, Shinagawa K, Takeshita A, Saito H, Ueda R, Ohno R, Naoe T. Activating mutation of D835 within the activation loop of FLT3 in human hematologic malignancies. Blood. 2001 Apr 15;97(8):2434-9. Citation on PubMed \n",
            "ALOX5AP\t   From NCBI Gene:This gene encodes a protein which, with 5-lipoxygenase, is required for leukotriene synthesis. Leukotrienes are arachidonic acid metabolites which have been implicated in various types of inflammatory responses, including asthma, arthritis and psoriasis. This protein localizes to the plasma membrane. Inhibitors of its function impede translocation of 5-lipoxygenase from the cytoplasm to the cell membrane and inhibit 5-lipoxygenase activation. Alternatively spliced transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Feb 2011]From UniProt:Required for leukotriene biosynthesis by ALOX5 (5-lipoxygenase). Anchors ALOX5 to the membrane. Binds arachidonic acid, and could play an essential role in the transfer of arachidonic acid to ALOX5. Binds to MK-886, a compound that blocks the biosynthesis of leukotrienes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Ischemic strokeFrom UniProt:Genetic variations in ALOX5AP may be associated with susceptibility to myocardial infarction. Involvement in myocardial infarction is however unclear: according to some authors (PubMed:14770184), a 4-SNP haplotype in ALOX5AP confers risk of myocardial infarction, while according to other (PubMed:17304054) ALOX5AP is not implicated in this condition.Ischemic stroke (ISCHSTR): A stroke is an acute neurologic event leading to death of neural tissue of the brain and resulting in loss of motor, sensory and/or cognitive function. Ischemic strokes, resulting from vascular occlusion, is considered to be a highly complex disease consisting of a group of heterogeneous disorders with multiple genetic and environmental risk factors. [MIM:601367]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 13q12.3, which is the long (q) arm of chromosome 13 at position 12.3Molecular Location: base pairs 30,713,478 to 30,764,428 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FLAP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ALOX5AP      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ARACHIDONATE 5-LIPOXYGENASE-ACTIVATING PROTEINOMIM: STROKE, ISCHEMIC      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ALOX5AP  OMIM: ARACHIDONATE 5-LIPOXYGENASE-ACTIVATING PROTEINOMIM: STROKE, ISCHEMIC  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "B3GLCT\t   The B3GLCT gene (formerly known as B3GALTL) provides instructions for making an enzyme called beta 3-glucosyltransferase (B3Glc-T), which is involved in the complex process of adding sugar molecules to proteins (glycosylation). Glycosylation modifies proteins so they can perform a wider variety of functions. The B3Glc-T enzyme is involved in a two-step glycosylation pathway that results in the formation of a sugar structure, made up of the sugars fucose and glucose, on a specific location of several different proteins. The B3Glc-T enzyme is responsible for the second step, which adds a glucose molecule to the fucose molecule already attached to the protein. The B3GLCT gene is normally turned on (active) in most cells of the body, which suggests that the B3Glc-T enzyme plays an important role across many cell types.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Peters plus syndrome     At least 10 mutations that cause Peters plus syndrome have been identified in the B3GLCT gene. Peters plus syndrome is characterized by eye abnormalities, short stature, intellectual disability, and distinctive facial features. The most common B3GLCT gene mutation replaces the DNA building block (nucleotide) guanine with the nucleotide adenine near an area of the gene called exon 8 (written as 660+1G>A). This mutation disrupts how genetic information is pieced together to produce the B3Glc-T enzyme. The resulting enzyme is abnormally short and nonfunctional. It is unclear how the loss of functional B3Glc-T enzyme leads to the signs and symptoms of Peters plus syndrome, but impaired glycosylation likely disrupts the function of many proteins, which may contribute to the variety of features. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 10 mutations that cause Peters plus syndrome have been identified in the B3GLCT gene. Peters plus syndrome is characterized by eye abnormalities, short stature, intellectual disability, and distinctive facial features. The most common B3GLCT gene mutation replaces the DNA building block (nucleotide) guanine with the nucleotide adenine near an area of the gene called exon 8 (written as 660+1G>A). This mutation disrupts how genetic information is pieced together to produce the B3Glc-T enzyme. The resulting enzyme is abnormally short and nonfunctional. It is unclear how the loss of functional B3Glc-T enzyme leads to the signs and symptoms of Peters plus syndrome, but impaired glycosylation likely disrupts the function of many proteins, which may contribute to the variety of features. More About This Health Condition    Cytogenetic Location: 13q12.3, which is the long (q) arm of chromosome 13 at position 12.3Molecular Location: base pairs 31,199,975 to 31,332,276 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        B3GALTLB3Glc-TB3GLT_HUMANB3GTLbeta 1,3-galactosyltransferase-likebeta-3-glycosyltransferase-likebeta3Glc-T     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Essentials of Glycobiology (second edition, 2009): Glycosyltransferases and Glycan-processing EnzymesMolecular Biology of the Cell (fourth edition, 2002): What is the Purpose of Glycosylation?      Clinical Information from GeneReviews (1 link)     Peters Plus Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of B3GLCT      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     BETA-3-GLUCOSYLTRANSFERASE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Essentials of Glycobiology (second edition, 2009): Glycosyltransferases and Glycan-processing EnzymesMolecular Biology of the Cell (fourth edition, 2002): What is the Purpose of Glycosylation?  Peters Plus Syndrome  Tests of B3GLCT  PubMed  BETA-3-GLUCOSYLTRANSFERASE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Hess D, Keusch JJ, Oberstein SA, Hennekam RC, Hofsteenge J. Peters Plus syndrome is a new congenital disorder of glycosylation and involves defective Omicron-glycosylation of thrombospondin type 1 repeats. J Biol Chem. 2008 Mar 21;283(12):7354-60. doi: 10.1074/jbc.M710251200. Epub 2008 Jan 16. Citation on PubMedKozma K, Keusch JJ, Hegemann B, Luther KB, Klein D, Hess D, Haltiwanger RS, Hofsteenge J. Identification and characterization of abeta1,3-glucosyltransferase that synthesizes the Glc-beta1,3-Fuc disaccharide on thrombospondin type 1 repeats. J Biol Chem. 2006 Dec 1;281(48):36742-51. Epub 2006 Oct 10. Citation on PubMedLesnik Oberstein SA, Kriek M, White SJ, Kalf ME, Szuhai K, den Dunnen JT, Breuning MH, Hennekam RC. Peters Plus syndrome is caused by mutations in B3GALTL, a putative glycosyltransferase. Am J Hum Genet. 2006 Sep;79(3):562-6. Epub 2006 Jul 19. Erratum in: Am J Hum Genet. 2006 Nov;79(5):985. Citation on PubMed or Free article on PubMed CentralLesnik Oberstein SAJ, Ruivenkamp CAL, Hennekam RC. Peters Plus Syndrome. 2007 Oct 8 [updated 2017 Aug 24]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1464/ Citation on PubMedReis LM, Tyler RC, Abdul-Rahman O, Trapane P, Wallerstein R, Broome D, Hoffman J, Khan A, Paradiso C, Ron N, Bergner A, Semina EV. Mutation analysis of B3GALTL in Peters Plus syndrome. Am J Med Genet A. 2008 Oct 15;146A(20):2603-10. doi: 10.1002/ajmg.a.32498. Citation on PubMed or Free article on PubMed CentralSato T, Sato M, Kiyohara K, Sogabe M, Shikanai T, Kikuchi N, Togayachi A, Ishida H, Ito H, Kameyama A, Gotoh M, Narimatsu H. Molecular cloning and characterization of a novel human beta1,3-glucosyltransferase, which is localized at the endoplasmic reticulum and glucosylates O-linked fucosylglycan on thrombospondin type 1 repeat domain. Glycobiology. 2006 Dec;16(12):1194-206. Epub 2006 Aug 9. Citation on PubMed \n",
            "KL\t   The KL gene provides instructions for making the protein alpha-klotho, which is found primarily in kidney cells. This protein plays a major role in regulating the phosphate levels within the body (phosphate homeostasis). Among its many functions, phosphate plays a critical role in the formation and growth of bones in childhood and helps maintain bone strength in adults. Phosphate levels are controlled in large part by the kidneys. The kidneys normally rid the body of excess phosphate by excreting it in urine, and they reabsorb this mineral into the bloodstream when more is needed.Alpha-klotho attaches (binds) to and turns on (activates) a protein called FGF receptor 1 that spans the membrane of many types of cells, including kidney cells. Once the receptor is active, another protein called fibroblast growth factor 23 can also bind to it. Binding of fibroblast growth factor 23 to its receptor stimulates signaling that stops phosphate reabsorption into the bloodstream.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hyperphosphatemic familial tumoral calcinosis     At least one mutation in the KL gene has been found to cause hyperphosphatemic familial tumoral calcinosis (HFTC), a condition characterized by an increase in the levels of phosphate in the blood (hyperphosphatemia) and abnormal deposits of phosphate and calcium (calcinosis) in the body's tissues. The KL gene mutation that causes HFTC replaces the protein building block (amino acid) histidine with the amino acid arginine at position 193 in the protein sequence (written as His193Arg or H193R). This mutation results in a shortage of functional alpha-klotho. As a result, FGF receptor 1 is not activated, making it unavailable for fibroblast growth factor 23 binding. The fibroblast growth factor 23 protein is normal, but it cannot bind to its receptor and cannot send out signals to stop phosphate reabsorption. As a result, too much phosphate is reabsorbed into the bloodstream, leading to hyperphosphatemia and subsequent calcinosis. More About This Health Condition      Kidney stones     Genetics Home Reference provides information about kidney stones. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least one mutation in the KL gene has been found to cause hyperphosphatemic familial tumoral calcinosis (HFTC), a condition characterized by an increase in the levels of phosphate in the blood (hyperphosphatemia) and abnormal deposits of phosphate and calcium (calcinosis) in the body's tissues. The KL gene mutation that causes HFTC replaces the protein building block (amino acid) histidine with the amino acid arginine at position 193 in the protein sequence (written as His193Arg or H193R). This mutation results in a shortage of functional alpha-klotho. As a result, FGF receptor 1 is not activated, making it unavailable for fibroblast growth factor 23 binding. The fibroblast growth factor 23 protein is normal, but it cannot bind to its receptor and cannot send out signals to stop phosphate reabsorption. As a result, too much phosphate is reabsorbed into the bloodstream, leading to hyperphosphatemia and subsequent calcinosis. More About This Health Condition  Genetics Home Reference provides information about kidney stones. More About This Health Condition    Cytogenetic Location: 13q13.1, which is the long (q) arm of chromosome 13 at position 13.1Molecular Location: base pairs 33,016,063 to 33,066,145 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        alpha-klothoKLOT_HUMANklotho precursor     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Hyperphosphatemic Familial Tumoral Calcinosis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KL      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KLOTHO      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Hyperphosphatemic Familial Tumoral Calcinosis  Tests of KL  PubMed  KLOTHO  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Farrow EG, Imel EA, White KE. Miscellaneous non-inflammatory musculoskeletal conditions. Hyperphosphatemic familial tumoral calcinosis (FGF23, GALNT3 and Klotho). Best Pract Res Clin Rheumatol. 2011 Oct;25(5):735-47. doi: 10.1016/j.berh.2011.10.020. Review. Citation on PubMed or Free article on PubMed CentralIchikawa S, Imel EA, Kreiter ML, Yu X, Mackenzie DS, Sorenson AH, Goetz R, Mohammadi M, White KE, Econs MJ. A homozygous missense mutation in human KLOTHO causes severe tumoral calcinosis. J Clin Invest. 2007 Sep;117(9):2684-91. Citation on PubMed or Free article on PubMed CentralOMIM: KLOTHORazzaque MS. The FGF23-Klotho axis: endocrine regulation of phosphate homeostasis. Nat Rev Endocrinol. 2009 Nov;5(11):611-9. doi: 10.1038/nrendo.2009.196. Review. Citation on PubMed or Free article on PubMed CentralSprecher E. Familial tumoral calcinosis: from characterization of a rare phenotype to the pathogenesis of ectopic calcification. J Invest Dermatol. 2010 Mar;130(3):652-60. doi: 10.1038/jid.2009.337. Epub 2009 Oct 29. Citation on PubMed or Free article on PubMed Central \n",
            "FREM2\t   The FREM2 gene provides instructions for making a protein that is part of a group of proteins called the FRAS/FREM complex; in addition to being part of the complex, FREM2 regulates the complex's formation. The FRAS/FREM complex is found in basement membranes, which are thin, sheet-like structures that separate and support cells in many tissues. The complex is particularly important during development before birth. One of its roles is to anchor the top layer of skin by connecting the basement membrane of the top layer to the layer of skin below. The FRAS/FREM complex is also involved in the proper development of certain other organs and tissues, including the kidneys, although the mechanism is unclear.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Fraser syndrome     At least two mutations in the FREM2 gene have been found to cause Fraser syndrome; these mutations are involved in a small percentage of cases of this condition. Fraser syndrome affects development before birth and is characterized by eyes that are completely covered by skin (cryptophthalmos), fusion of the skin between the fingers and toes (cutaneous syndactyly), and abnormalities of the kidneys and other organs and tissues.FREM2 gene mutations involved in Fraser syndrome lead to production of an abnormal FREM2 protein that likely does not function properly. As a result, the FRAS/FREM complex cannot form. Lack of the FRAS/FREM complex in the basement membrane of skin leads to detachment of the top layer of skin, causing blisters to form during development. These blisters likely prevent the proper formation of certain structures before birth, leading to cryptophthalmos and cutaneous syndactyly. It is unknown how lack of the FRAS/FREM complex leads to kidney abnormalities and other problems in Fraser syndrome. More About This Health Condition      Coloboma     Genetics Home Reference provides information about coloboma. More About This Health Condition      Congenital anomalies of kidney and urinary tract     Genetics Home Reference provides information about congenital anomalies of kidney and urinary tract. More About This Health Condition      Other disorders     Mutations in the FREM2 gene have also been found in people with abnormalities of the kidneys and urinary tract but no other signs and symptoms of Fraser syndrome (described above). Such abnormalities are grouped together as congenital anomalies of the kidney and urinary tract (CAKUT). The FREM2 gene mutations involved in CAKUT typically change single protein building blocks (amino acids) in the FREM2 protein. Researchers speculate that the effects of these mutations are milder than those of mutations that cause Fraser syndrome; some FREM2 protein function may still remain. How these gene mutations affect the FRAS/FREM complex or lead to renal agenesis and other CAKUT is unknown.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two mutations in the FREM2 gene have been found to cause Fraser syndrome; these mutations are involved in a small percentage of cases of this condition. Fraser syndrome affects development before birth and is characterized by eyes that are completely covered by skin (cryptophthalmos), fusion of the skin between the fingers and toes (cutaneous syndactyly), and abnormalities of the kidneys and other organs and tissues.FREM2 gene mutations involved in Fraser syndrome lead to production of an abnormal FREM2 protein that likely does not function properly. As a result, the FRAS/FREM complex cannot form. Lack of the FRAS/FREM complex in the basement membrane of skin leads to detachment of the top layer of skin, causing blisters to form during development. These blisters likely prevent the proper formation of certain structures before birth, leading to cryptophthalmos and cutaneous syndactyly. It is unknown how lack of the FRAS/FREM complex leads to kidney abnormalities and other problems in Fraser syndrome. More About This Health Condition  Genetics Home Reference provides information about coloboma. More About This Health Condition  Genetics Home Reference provides information about congenital anomalies of kidney and urinary tract. More About This Health Condition  Mutations in the FREM2 gene have also been found in people with abnormalities of the kidneys and urinary tract but no other signs and symptoms of Fraser syndrome (described above). Such abnormalities are grouped together as congenital anomalies of the kidney and urinary tract (CAKUT). The FREM2 gene mutations involved in CAKUT typically change single protein building blocks (amino acids) in the FREM2 protein. Researchers speculate that the effects of these mutations are milder than those of mutations that cause Fraser syndrome; some FREM2 protein function may still remain. How these gene mutations affect the FRAS/FREM complex or lead to renal agenesis and other CAKUT is unknown.    Cytogenetic Location: 13q13.3, which is the long (q) arm of chromosome 13 at position 13.3Molecular Location: base pairs 38,687,036 to 38,887,131 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DKFZp686J0811ECM3 homologFRAS1-related extracellular matrix protein 2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FREM2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     CONGENITAL ANOMALIES OF KIDNEY AND URINARY TRACT 1FRAS1-RELATED EXTRACELLULAR MATRIX PROTEIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FREM2  PubMed  CONGENITAL ANOMALIES OF KIDNEY AND URINARY TRACT 1FRAS1-RELATED EXTRACELLULAR MATRIX PROTEIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: FRAS1-RELATED EXTRACELLULAR MATRIX PROTEIN 2Jadeja S, Smyth I, Pitera JE, Taylor MS, van Haelst M, Bentley E, McGregor L, Hopkins J, Chalepakis G, Philip N, Perez Aytes A, Watt FM, Darling SM, Jackson I, Woolf AS, Scambler PJ. Identification of a new gene mutated in Fraser syndrome and mouse myelencephalic blebs. Nat Genet. 2005 May;37(5):520-5. Epub 2005 Apr 17. Citation on PubMedKohl S, Hwang DY, Dworschak GC, Hilger AC, Saisawat P, Vivante A, Stajic N, Bogdanovic R, Reutter HM, Kehinde EO, Tasic V, Hildebrandt F. Mild recessive mutations in six Fraser syndrome-related genes cause isolated congenital anomalies of the kidney and urinary tract. J Am Soc Nephrol. 2014 Sep;25(9):1917-22. doi: 10.1681/ASN.2013101103. Epub 2014 Apr 3. Citation on PubMed or Free article on PubMed CentralPavlakis E, Chiotaki R, Chalepakis G. The role of Fras1/Frem proteins in the structure and function of basement membrane. Int J Biochem Cell Biol. 2011 Apr;43(4):487-95. doi: 10.1016/j.biocel.2010.12.016. Epub 2010 Dec 21. Review. Citation on PubMedPetrou P, Makrygiannis AK, Chalepakis G. The Fras1/Frem family of extracellular matrix proteins: structure, function, and association with Fraser syndrome and the mouse bleb phenotype. Connect Tissue Res. 2008;49(3):277-82. doi: 10.1080/03008200802148025. Citation on PubMedShort K, Wiradjaja F, Smyth I. Let's stick together: the role of the Fras1 and Frem proteins in epidermal adhesion. IUBMB Life. 2007 Jul;59(7):427-35. Review. Citation on PubMedvan Haelst MM, Maiburg M, Baujat G, Jadeja S, Monti E, Bland E, Pearce K; Fraser Syndrome Collaboration Group, Hennekam RC, Scambler PJ. Molecular study of 33 families with Fraser syndrome new data and mutation review. Am J Med Genet A. 2008 Sep 1;146A(17):2252-7. doi: 10.1002/ajmg.a.32440. Citation on PubMed \n",
            "MRPS31\t\n",
            "VWA8\t\n",
            "ZC3H13\t\n",
            "RCBTB1\t   From NCBI Gene:This gene encodes a protein with an N-terminal RCC1 domain and a C-terminal BTB (broad complex, tramtrack and bric-a-brac) domain. In rat, over-expression of this gene in vascular smooth muscle cells induced cellular hypertrophy. In rat, the C-terminus of RCBTB1 interacts with the angiotensin II receptor-1A. In humans, this gene maps to a region of chromosome 13q that is frequently deleted in B-cell chronic lymphocytic leukemia and other lymphoid malignancies. [provided by RefSeq, Jul 2008]From UniProt:May be involved in cell cycle regulation by chromatin remodeling.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Retinal dystrophy with or without extraocular anomaliesFrom UniProt:Retinal dystrophy with or without extraocular anomalies (RDEOA): An autosomal recessive disease characterized by progressive retinal dystrophy, chorioretinal macular atrophy, reduced cone and rod responses on ERG, and decrease visual acuity. Extraocular anomalies are variably present in some patients and include pulmonary fibrosis, sensorineural hearing loss, and endocrine features such as goiter and primary ovarian insufficiency. [MIM:617175]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 13q14.2, which is the long (q) arm of chromosome 13 at position 14.2Molecular Location: base pairs 49,531,944 to 49,585,587 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CLLD7CLLL7GLPRDEOA     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RCBTB1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: RCC1 DOMAIN- AND BTB DOMAIN-CONTAINING PROTEIN 1OMIM: RETINAL DYSTROPHY WITH OR WITHOUT EXTRAOCULAR ANOMALIES      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RCBTB1  OMIM: RCC1 DOMAIN- AND BTB DOMAIN-CONTAINING PROTEIN 1OMIM: RETINAL DYSTROPHY WITH OR WITHOUT EXTRAOCULAR ANOMALIES  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DHRS12\t\n",
            "ATP7B\t   The ATP7B gene provides instructions for making a protein called copper-transporting ATPase 2.  This protein is part of the P-type ATPase family, a group of proteins that transport metals into and out of cells by using energy stored in the molecule adenosine triphosphate (ATP).  Copper-transporting ATPase 2 is found primarily in the liver, with smaller amounts in the kidneys and brain. It plays a role in the transport of copper from the liver to other parts of the body.  Copper is an important part of certain enzymes that maintain normal cell functions.  Copper-transporting ATPase 2 is also important for the removal of excess copper from the body.Within liver cells, copper-transporting ATPase 2 is found in a structure called the Golgi apparatus, which modifies newly produced enzymes and other proteins.  Here, copper-transporting ATPase 2 supplies copper to a protein  called ceruloplasmin, which transports copper to other parts of the body via the blood.  If copper levels in the liver get too high, copper-transporting ATPase 2 leaves the Golgi and transfers copper to small sacs (vesicles) for elimination through bile.  Bile is a substance produced by the liver that is important for digestion and the removal of waste products.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Wilson disease     Researchers have identified more than 250 ATP7B gene mutations that cause Wilson disease.  About half the mutations change one of the protein building blocks (amino acids) used to make copper-transporting ATPase 2.  This type of mutation alters the 3-dimensional structure of the protein or its stability, preventing copper-transporting ATPase 2 from functioning properly.  A common amino acid substitution replaces the amino acid histidine with the amino acid glutamine at position 1069 in the protein (written as His1069Gln or H1069Q).  This particular mutation occurs in nearly 40 percent of affected individuals with a Northern or Eastern European ancestry.  Approximately one-third of Asians with Wilson disease have a mutation that replaces the amino acid arginine with the amino acid leucine at position 778 (written as Arg778Leu or R778L).  In the Costa Rican population, more than 60 percent of affected individuals have a mutation that replaces the amino acid aspartic acid with the amino acid serine at position 1270 (written as Asp1270Ser or D1270S).Other types of mutations delete or insert small segments of DNA within the ATP7B gene or introduce a stop signal in the gene's instructions for making copper-transporting ATPase 2.  As a result, no protein is produced, or an abnormally small protein is made.  These types of mutations usually result in  symptoms that are more severe than those caused by mutations that change a single amino acid.With a shortage of functional protein, removal of excess copper from the body is impaired.  As a result, copper accumulates to toxic levels that can damage tissues and organs, particularly the liver and brain. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Researchers have identified more than 250 ATP7B gene mutations that cause Wilson disease.  About half the mutations change one of the protein building blocks (amino acids) used to make copper-transporting ATPase 2.  This type of mutation alters the 3-dimensional structure of the protein or its stability, preventing copper-transporting ATPase 2 from functioning properly.  A common amino acid substitution replaces the amino acid histidine with the amino acid glutamine at position 1069 in the protein (written as His1069Gln or H1069Q).  This particular mutation occurs in nearly 40 percent of affected individuals with a Northern or Eastern European ancestry.  Approximately one-third of Asians with Wilson disease have a mutation that replaces the amino acid arginine with the amino acid leucine at position 778 (written as Arg778Leu or R778L).  In the Costa Rican population, more than 60 percent of affected individuals have a mutation that replaces the amino acid aspartic acid with the amino acid serine at position 1270 (written as Asp1270Ser or D1270S).Other types of mutations delete or insert small segments of DNA within the ATP7B gene or introduce a stop signal in the gene's instructions for making copper-transporting ATPase 2.  As a result, no protein is produced, or an abnormally small protein is made.  These types of mutations usually result in  symptoms that are more severe than those caused by mutations that change a single amino acid.With a shortage of functional protein, removal of excess copper from the body is impaired.  As a result, copper accumulates to toxic levels that can damage tissues and organs, particularly the liver and brain. More About This Health Condition    Cytogenetic Location: 13q14.3, which is the long (q) arm of chromosome 13 at position 14.3Molecular Location: base pairs 51,932,669 to 52,012,130 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATP7B_HUMANATPase, Cu++ transporting, beta polypeptideATPase, Cu++ transporting, beta polypeptide (Wilson disease)Copper pump 2PWDWC1Wilson disease-associated proteinWND     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Wilson Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATP7B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATPase, Cu(2+)-TRANSPORTING, BETA POLYPEPTIDE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Wilson Disease  Tests of ATP7B  PubMed  ATPase, Cu(2+)-TRANSPORTING, BETA POLYPEPTIDE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cater MA, Forbes J, La Fontaine S, Cox D, Mercer JF. Intracellular trafficking of the human Wilson protein: the role of the six N-terminal metal-binding sites. Biochem J. 2004 Jun 15;380(Pt 3):805-13. Citation on PubMed or Free article on PubMed CentralFatemi N, Sarkar B. Molecular mechanism of copper transport in Wilson disease. Environ Health Perspect. 2002 Oct;110 Suppl 5:695-8. Review. Citation on PubMed or Free article on PubMed CentralFerenci P. Regional distribution of mutations of the ATP7B gene in patients with Wilson disease: impact on genetic testing. Hum Genet. 2006 Sep;120(2):151-9. Epub 2006 Jun 22. Review. Citation on PubMedHuster D, Hoppert M, Lutsenko S, Zinke J, Lehmann C, Mssner J, Berr F, Caca K. Defective cellular localization of mutant ATP7B in Wilson's disease patients and hepatoma cell lines. Gastroenterology. 2003 Feb;124(2):335-45. Citation on PubMedKitzberger R, Madl C, Ferenci P. Wilson disease. Metab Brain Dis. 2005 Dec;20(4):295-302. Review. Citation on PubMedLangner C, Denk H. Wilson disease. Virchows Arch. 2004 Aug;445(2):111-8. Epub 2004 Jun 17. Review. Citation on PubMedPanagiotakaki E, Tzetis M, Manolaki N, Loudianos G, Papatheodorou A, Manesis E, Nousia-Arvanitakis S, Syriopoulou V, Kanavakis E. Genotype-phenotype correlations for a wide spectrum of mutations in the Wilson disease gene (ATP7B). Am J Med Genet A. 2004 Dec 1;131(2):168-73. Citation on PubMed \n",
            "UTP14C\t\n",
            "NEK3\t\n",
            "DACH1\t\n",
            "UCHL3\t\n",
            "SLAIN1\t\n",
            "SPRY2\t   From NCBI Gene:This gene encodes a protein belonging to the sprouty family. The encoded protein contains a carboxyl-terminal cysteine-rich domain essential for the inhibitory activity on receptor tyrosine kinase signaling proteins and is required for growth factor stimulated translocation of the protein to membrane ruffles. In primary dermal endothelial cells this gene is transiently upregulated in response to fibroblast growth factor two. This protein is indirectly involved in the non-cell autonomous inhibitory effect on fibroblast growth factor two signaling. The protein interacts with Cas-Br-M (murine) ectropic retroviral transforming sequence, and can function as a bimodal regulator of epidermal growth factor receptor/mitogen-activated protein kinase signaling. This protein may play a role in alveoli branching during lung development as shown by a similar mouse protein. [provided by RefSeq, Jul 2008]From UniProt:May function as an antagonist of fibroblast growth factor (FGF) pathways and may negatively modulate respiratory organogenesis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:IgA nephropathy, susceptibility to, 3From UniProt:IgA nephropathy 3 (IGAN3): A form of IgA nephropathy, a common primary glomerulonephritis characterized by glomerular sclerosis, interstitial fibrosis, and mesangial glomerular deposits of immunoglobulin A and immunoglobulin G visible on renal biopsies. IgA nephropathy is associated with renal insufficiency that can progress to end-stage renal disease. Proteinuria and hematuria are characteristic clinical presentations. [MIM:616818]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 13q31.1, which is the long (q) arm of chromosome 13 at position 31.1Molecular Location: base pairs 80,335,976 to 80,341,115 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        hSPRY2IGAN3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SPRY2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: IgA NEPHROPATHY, SUSCEPTIBILITY TO, 3OMIM: SPROUTY, DROSOPHILA, HOMOLOG OF, 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SPRY2  OMIM: IgA NEPHROPATHY, SUSCEPTIBILITY TO, 3OMIM: SPROUTY, DROSOPHILA, HOMOLOG OF, 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SOX21\t\n",
            "RNF113B\t\n",
            "DOCK9\t   From UniProt:Guanine nucleotide-exchange factor (GEF) that activates CDC42 by exchanging bound GDP for free GTP. Overexpression induces filopodia formation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Keratoconus     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 13q32.3, which is the long (q) arm of chromosome 13 at position 32.3Molecular Location: base pairs 98,793,487 to 99,086,406 on chromosome 13 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ZIZ1ZIZIMIN1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DOCK9      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: DEDICATOR OF CYTOKINESIS 9      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DOCK9  OMIM: DEDICATOR OF CYTOKINESIS 9  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLC25A47\t\n",
            "RTL1\t\n",
            "EXOC3L4\t\n",
            "TNFAIP2\t\n",
            "ASPG\t\n",
            "INF2\t   From NCBI Gene:This gene represents a member of the formin family of proteins. It is considered a diaphanous formin due to the presence of a diaphanous inhibitory domain located at the N-terminus of the encoded protein. Studies of a similar mouse protein indicate that the protein encoded by this locus may function in polymerization and depolymerization of actin filaments. Mutations at this locus have been associated with focal segmental glomerulosclerosis 5.[provided by RefSeq, Aug 2010]From UniProt:Severs actin filaments and accelerates their polymerization and depolymerization.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Charcot-Marie-Tooth diseaseFrom NCBI Gene:Focal segmental glomerulosclerosis 5Charcot-Marie-Tooth disease, dominant intermediate EFrom UniProt:Charcot-Marie-Tooth disease, dominant, intermediate type, E (CMTDIE): A form of Charcot-Marie-Tooth disease, a disorder of the peripheral nervous system, characterized by progressive weakness and atrophy, initially of the peroneal muscles and later of the distal muscles of the arms. The dominant intermediate type E is characterized by clinical and pathologic features intermediate between demyelinating and axonal peripheral neuropathies, and motor median nerve conduction velocities ranging from 25 to 45 m/sec. Patients additionally manifest focal segmental glomerulonephritis, proteinuria, progression to end-stage renal disease, and a characteristic histologic pattern on renal biopsy. [MIM:614455]Focal segmental glomerulosclerosis 5 (FSGS5): A renal pathology defined by the presence of segmental sclerosis in glomeruli and resulting in proteinuria, reduced glomerular filtration rate and progressive decline in renal function. Renal insufficiency often progresses to end-stage renal disease, a highly morbid state requiring either dialysis therapy or kidney transplantation. [MIM:613237]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q32.33, which is the long (q) arm of chromosome 14 at position 32.33Molecular Location: base pairs 104,689,606 to 104,719,610 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C14orf151C14orf173CMTDIEFSGS5pp9484     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of INF2      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CHARCOT-MARIE-TOOTH DISEASE, DOMINANT INTERMEDIATE EOMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 5OMIM: INVERTED FORMIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of INF2  OMIM: CHARCOT-MARIE-TOOTH DISEASE, DOMINANT INTERMEDIATE EOMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 5OMIM: INVERTED FORMIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ADSSL1\t   From NCBI Gene:This gene encodes a member of the adenylosuccinate synthase family of proteins. The encoded muscle-specific enzyme plays a role in the purine nucleotide cycle by catalyzing the first step in the conversion of inosine monophosphate (IMP) to adenosine monophosphate (AMP). Mutations in this gene may cause adolescent onset distal myopathy. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]From UniProt:Component of the purine nucleotide cycle (PNC), which interconverts IMP and AMP to regulate the nucleotide levels in various tissues, and which contributes to glycolysis and ammoniagenesis. Catalyzes the first committed step in the biosynthesis of AMP from IMP.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Myopathy, distal, 5From UniProt:Myopathy, distal, 5 (MPD5): A form of distal myopathy, a group of muscular disorders characterized by progressive muscular weakness and muscle atrophy beginning in the hands, the legs or the feet. MPD5 is an autosomal recessive form, predominantly affecting the lower limbs. [MIM:617030]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q32.33, which is the long (q) arm of chromosome 14 at position 32.33Molecular Location: base pairs 104,724,197 to 104,747,325 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        Adss1MPD5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADSSL1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ADENYLOSUCCINATE SYNTHASE-LIKE 1OMIM: MYOPATHY, DISTAL, 5      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADSSL1  OMIM: ADENYLOSUCCINATE SYNTHASE-LIKE 1OMIM: MYOPATHY, DISTAL, 5  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZBTB42\t   From NCBI Gene:The protein encoded by this gene is a member of the C2H2 zinc finger protein family. This protein is predicted to have a pox virus and zinc finger (POZ) domain at the N-terminus and four zinc finger domains at the C-terminus. In human and mouse, the protein localizes to the nuclei of skeletal muscle cells. Knockdown of this gene in zebrafish results in abnormal skeletal muscle development and myofibrillar disorganization. A novel homozygous variant of the human gene has been associated with lethal congenital contracture syndrome, an autosomal recessive disorder that results in muscle wasting. [provided by RefSeq, Mar 2015]From UniProt:Transcriptional repressor. Specifically binds DNA and probably acts by recruiting chromatin remodeling multiprotein complexes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Lethal congenital contracture syndrome 6From UniProt:Lethal congenital contracture syndrome 6 (LCCS6): A form of lethal congenital contracture syndrome, an autosomal recessive disorder characterized by degeneration of anterior horn neurons, extreme skeletal muscle atrophy and congenital non-progressive joint contractures. The contractures can involve the upper or lower limbs and/or the vertebral column, leading to various degrees of flexion or extension limitations evident at birth. LCCS6 features include severe polyhydramnios and absent stomach, in addition to multiple contracture deformities. [MIM:616248]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q32.33, which is the long (q) arm of chromosome 14 at position 32.33Molecular Location: base pairs 104,800,597 to 104,804,712 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LCCS6ZNF925     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ZBTB42      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LETHAL CONGENITAL CONTRACTURE SYNDROME 6OMIM: ZINC FINGER- AND BTB DOMAIN-CONTAINING PROTEIN 42: ZBTB42      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ZBTB42  OMIM: LETHAL CONGENITAL CONTRACTURE SYNDROME 6OMIM: ZINC FINGER- AND BTB DOMAIN-CONTAINING PROTEIN 42: ZBTB42  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PACS2\t\n",
            "CRIP2\t\n",
            "CRIP1\t\n",
            "TMEM121\t\n",
            "IGHA2\t\n",
            "IGHV1-2\t\n",
            "IGHV3-20\t\n",
            "IGHV3-38\t\n",
            "IGHV4-39\t\n",
            "IGHV3-53\t\n",
            "IGHV3-64\t\n",
            "IGHV7-81\t\n",
            "OR11H12\t\n",
            "OR4K15\t\n",
            "OR11G2\t\n",
            "OR11H6\t\n",
            "TEP1\t\n",
            "OR6S1\t\n",
            "OR5AU1\t\n",
            "TRAC\t   From NCBI Gene:T cell receptors recognize foreign antigens which have been processed as small peptides and bound to major histocompatibility complex (MHC) molecules at the surface of antigen presenting cells (APC). Each T cell receptor is a dimer consisting of one alpha and one beta chain or one delta and one gamma chain. In a single cell, the T cell receptor loci are rearranged and expressed in the order delta, gamma, beta, and alpha. If both delta and gamma rearrangements produce functional chains, the cell expresses delta and gamma. If not, the cell proceeds to rearrange the beta and alpha loci. This region represents the germline organization of the T cell receptor alpha and delta loci. Both the alpha and delta loci include V (variable), J (joining), and C (constant) segments and the delta locus also includes diversity (D) segments. The delta locus is situated within the alpha locus, between the alpha V and J segments. During T cell development, the delta chain is synthesized by a recombination event at the DNA level joining a D segment with a J segment; a V segment is then joined to the D-J gene. The alpha chain is synthesized by recombination joining a single V segment with a J segment. For both chains, the C segment is later joined by splicing at the RNA level. Recombination of many different V segments with several J segments provides a wide range of antigen recognition. Additional diversity is attained by junctional diversity, resulting from the random additional of nucleotides by terminal deoxynucleotidyltransferase. Five variable segments can be used in either alpha or delta chains and are described by TRAV/DV symbols. Several V and J segments of the alpha locus are known to be incapable of encoding a protein and are considered pseudogenes. [provided by RefSeq, Aug 2016]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:T-cell receptor alpha/beta deficiency     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q11.2, which is the long (q) arm of chromosome 14 at position 11.2Molecular Location: base pairs 22,547,506 to 22,552,132 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IMD7TCRATRATRCA     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TRAC      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: T-CELL RECEPTOR ALPHA CHAIN CONSTANT REGIONOMIM: T-CELL RECEPTOR-ALPHA/BETA DEFICIENCY      Research Resources (4 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene      Tests of TRAC  OMIM: T-CELL RECEPTOR ALPHA CHAIN CONSTANT REGIONOMIM: T-CELL RECEPTOR-ALPHA/BETA DEFICIENCY  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene \n",
            "OXA1L\t\n",
            "MRPL52\t\n",
            "RBM23\t\n",
            "PSMB11\t\n",
            "SLC7A8\t\n",
            "HOMEZ\t\n",
            "MYH6\t   The MYH6 gene provides instructions for making a protein known as the cardiac alpha ()-myosin heavy chain. This protein is found in heart (cardiac) muscle cells, where it forms part of a larger protein called type II myosin. Type II myosin helps generate the mechanical force that is needed for cardiac muscle to contract, allowing the heart to pump blood to the rest of the body.Type II myosin is one of the major components of cell structures called sarcomeres. These structures are the basic units of muscle contraction. Sarcomeres are composed of thick filaments made up of type II myosin and thin filaments made up of another protein called actin. The overlapping thick and thin filaments attach to each other and release, which allows the filaments to move relative to one another so that muscles can contract. Sarcomeres also appear to have an important role in the early development of structures in the heart.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Sick sinus syndrome     At least one variation of the MYH6 gene has been associated with an increased risk of developing sick sinus syndrome. This condition affects the function of the sino-atrial (SA) node, which is an area of specialized cells in the heart that functions as a natural pacemaker. The variation, which was identified in the Icelandic population, changes a single protein building block (amino acid) in the -myosin heavy chain. Specifically, it replaces the amino acid arginine with the amino acid tryptophan at protein position 721 (written as Arg721Trp). Researchers found that about half of the people in their sample who had the Arg721Trp variation developed sick sinus syndrome during their lifetime, compared with about 6 percent of people who did not carry the variation. They speculated that the variation may alter the structure of the -myosin heavy chain and disrupt its usual role in cardiac muscle contraction. These changes could alter the way the heart beats in some people, leading to an abnormally slow heartbeat (bradycardia) and related symptoms such as dizziness, light-headedness, and fainting (syncope). More About This Health Condition      Familial dilated cardiomyopathy     Genetics Home Reference provides information about familial dilated cardiomyopathy. More About This Health Condition      Other disorders     Mutations in the MYH6 gene have been found to cause several additional heart conditions. These include congenital heart defects, particularly atrial-septal defect (ASD), which is a hole in the wall (septum) that separates the two upper chambers of the heart (the atria). Most of the MYH6 gene mutations associated with ASD affect a part of the -myosin heavy chain known as the head domain. This part of the protein is involved in attaching (binding) type II myosin to actin. Abnormal interaction between these two proteins may disrupt early heart development, leading to heart defects such as ASD.MYH6 gene mutations can also cause two diseases of the cardiac muscle, dilated cardiomyopathy (DCM) and hypertrophic cardiomyopathy (HCM). DCM weakens and enlarges the heart, while HCM is characterized by thickening (hypertrophy) of the cardiac muscle. Both of these conditions prevent the heart from pumping blood efficiently and increase the risk of heart failure and sudden death. Mutations associated with DCM and HCM can affect any part of the -myosin heavy chain, and most change single amino acids in the protein. These changes likely alter the -myosin heavy chain in ways that affect the structure and function of cardiac muscle.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least one variation of the MYH6 gene has been associated with an increased risk of developing sick sinus syndrome. This condition affects the function of the sino-atrial (SA) node, which is an area of specialized cells in the heart that functions as a natural pacemaker. The variation, which was identified in the Icelandic population, changes a single protein building block (amino acid) in the -myosin heavy chain. Specifically, it replaces the amino acid arginine with the amino acid tryptophan at protein position 721 (written as Arg721Trp). Researchers found that about half of the people in their sample who had the Arg721Trp variation developed sick sinus syndrome during their lifetime, compared with about 6 percent of people who did not carry the variation. They speculated that the variation may alter the structure of the -myosin heavy chain and disrupt its usual role in cardiac muscle contraction. These changes could alter the way the heart beats in some people, leading to an abnormally slow heartbeat (bradycardia) and related symptoms such as dizziness, light-headedness, and fainting (syncope). More About This Health Condition  Genetics Home Reference provides information about familial dilated cardiomyopathy. More About This Health Condition  Mutations in the MYH6 gene have been found to cause several additional heart conditions. These include congenital heart defects, particularly atrial-septal defect (ASD), which is a hole in the wall (septum) that separates the two upper chambers of the heart (the atria). Most of the MYH6 gene mutations associated with ASD affect a part of the -myosin heavy chain known as the head domain. This part of the protein is involved in attaching (binding) type II myosin to actin. Abnormal interaction between these two proteins may disrupt early heart development, leading to heart defects such as ASD.MYH6 gene mutations can also cause two diseases of the cardiac muscle, dilated cardiomyopathy (DCM) and hypertrophic cardiomyopathy (HCM). DCM weakens and enlarges the heart, while HCM is characterized by thickening (hypertrophy) of the cardiac muscle. Both of these conditions prevent the heart from pumping blood efficiently and increase the risk of heart failure and sudden death. Mutations associated with DCM and HCM can affect any part of the -myosin heavy chain, and most change single amino acids in the protein. These changes likely alter the -myosin heavy chain in ways that affect the structure and function of cardiac muscle.    Cytogenetic Location: 14q11.2, which is the long (q) arm of chromosome 14 at position 11.2Molecular Location: base pairs 23,381,990 to 23,408,277 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        alpha-MHCASD3CMD1EECMH14MYH6_HUMANMYHCmyHC-alphaMYHCAmyosin-6myosin heavy chain, cardiac muscle alpha isoformmyosin, heavy chain 6, cardiac muscle, alphamyosin, heavy polypeptide 6, cardiac muscle, alpha (cardiomyopathy, hypertrophic 1)SSS3     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): Myosin: The Actin Motor ProteinNeuromuscular Disease Center, Washington University: Myosin and Associated Muscle Proteins      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYH6      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (4 links)     ATRIAL SEPTAL DEFECT 3CARDIOMYOPATHY, DILATED, 1EECARDIOMYOPATHY, FAMILIAL HYPERTROPHIC, 14MYOSIN, HEAVY CHAIN 6, CARDIAC MUSCLE, ALPHA      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Myosin: The Actin Motor ProteinNeuromuscular Disease Center, Washington University: Myosin and Associated Muscle Proteins  Tests of MYH6  PubMed  ATRIAL SEPTAL DEFECT 3CARDIOMYOPATHY, DILATED, 1EECARDIOMYOPATHY, FAMILIAL HYPERTROPHIC, 14MYOSIN, HEAVY CHAIN 6, CARDIAC MUSCLE, ALPHA  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Carniel E, Taylor MR, Sinagra G, Di Lenarda A, Ku L, Fain PR, Boucek MM, Cavanaugh J, Miocic S, Slavov D, Graw SL, Feiger J, Zhu XZ, Dao D, Ferguson DA, Bristow MR, Mestroni L. Alpha-myosin heavy chain: a sarcomeric gene associated with dilated and hypertrophic phenotypes of cardiomyopathy. Circulation. 2005 Jul 5;112(1):54-9. Citation on PubMedGranados-Riveron JT, Ghosh TK, Pope M, Bu'Lock F, Thornborough C, Eason J, Kirk EP, Fatkin D, Feneley MP, Harvey RP, Armour JA, David Brook J. Alpha-cardiac myosin heavy chain (MYH6) mutations affecting myofibril formation are associated with congenital heart defects. Hum Mol Genet. 2010 Oct 15;19(20):4007-16. doi: 10.1093/hmg/ddq315. Epub 2010 Jul 23. Citation on PubMedHershberger RE, Norton N, Morales A, Li D, Siegfried JD, Gonzalez-Quintana J. Coding sequence rare variants identified in MYBPC3, MYH6, TPM1, TNNC1, and TNNI3 from 312 patients with familial or idiopathic dilated cardiomyopathy. Circ Cardiovasc Genet. 2010 Apr;3(2):155-61. doi: 10.1161/CIRCGENETICS.109.912345. Epub 2010 Mar 9. Citation on PubMed or Free article on PubMed CentralHolm H, Gudbjartsson DF, Sulem P, Masson G, Helgadottir HT, Zanon C, Magnusson OT, Helgason A, Saemundsdottir J, Gylfason A, Stefansdottir H, Gretarsdottir S, Matthiasson SE, Thorgeirsson GM, Jonasdottir A, Sigurdsson A, Stefansson H, Werge T, Rafnar T, Kiemeney LA, Parvez B, Muhammad R, Roden DM, Darbar D, Thorleifsson G, Walters GB, Kong A, Thorsteinsdottir U, Arnar DO, Stefansson K. A rare variant in MYH6 is associated with high risk of sick sinus syndrome. Nat Genet. 2011 Mar 6;43(4):316-20. doi: 10.1038/ng.781. Citation on PubMed or Free article on PubMed CentralPosch MG, Waldmuller S, Mller M, Scheffold T, Fournier D, Andrade-Navarro MA, De Geeter B, Guillaumont S, Dauphin C, Yousseff D, Schmitt KR, Perrot A, Berger F, Hetzer R, Bouvagnet P, zcelik C. Cardiac alpha-myosin (MYH6) is the predominant sarcomeric disease gene for familial atrial septal defects. PLoS One. 2011;6(12):e28872. doi: 10.1371/journal.pone.0028872. Epub 2011 Dec 14. Citation on PubMed or Free article on PubMed Central \n",
            "DHRS4\t\n",
            "DHRS4L2\t\n",
            "DHRS4L1\t\n",
            "REC8\t\n",
            "IPO4\t\n",
            "CHMP4A\t   From NCBI Gene:This gene encodes a member of the chromatin-modifying protein/charged multivesicular body protein (CHMP) protein family. The protein is part of the endosomal sorting complex required for transport (ESCRT) complex III (ESCRT-III), which functions in the sorting of endocytosed cell-surface receptors into multivesicular endosomes. The ESCRT machinery also functions in the final abscisson stage of cytokinesis and in the budding of enveloped viruses such as HIV-1. The three proteins of the CHMP4 subfamily interact with programmed cell death 6 interacting protein (PDCD6IP, also known as ALIX), which also functions in the ESCRT pathway. The CHMP4 proteins assemble into membrane-attached 5-nm filaments that form circular scaffolds and promote or stabilize outward budding. These polymers are proposed to help generate the luminal vesicles of multivesicular bodies. Mutations in this gene result in autosomal dominant posterior polar cataracts.[provided by RefSeq, Oct 2009]From UniProt:(Microbial infection) The ESCRT machinery also functions in topologically equivalent membrane fission events, such as the budding of enveloped viruses (HIV-1 and other lentiviruses). Via its interaction with PDCD6IP involved in HIV-1 p6- and p9-dependent virus release.Probable core component of the endosomal sorting required for transport complex III (ESCRT-III) which is involved in multivesicular bodies (MVBs) formation and sorting of endosomal cargo proteins into MVBs. MVBs contain intraluminal vesicles (ILVs) that are generated by invagination and scission from the limiting membrane of the endosome and mostly are delivered to lysosomes enabling degradation of membrane proteins, such as stimulated growth factor receptors, lysosomal enzymes and lipids. The MVB pathway appears to require the sequential function of ESCRT-O, -I,-II and -III complexes. ESCRT-III proteins mostly dissociate from the invaginating membrane before the ILV is released (PubMed:12860994, PubMed:18209100). The ESCRT machinery also functions in topologically equivalent membrane fission events, such as the terminal stages of cytokinesis (PubMed:21310966). Together with SPAST, the ESCRT-III complex promotes nuclear envelope sealing and mitotic spindle disassembly during late anaphase (PubMed:26040712). Plays a role in the endosomal sorting pathway. ESCRT-III proteins are believed to mediate the necessary vesicle extrusion and/or membrane fission activities, possibly in conjunction with the AAA ATPase VPS4. When overexpressed, membrane-assembled circular arrays of CHMP4B filaments can promote or stabilize negative curvature and outward budding. CHMP4A/B/C are required for the exosomal release of SDCBP, CD63 and syndecan (PubMed:22660413).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Cataract, posterior polar, 3From UniProt:Cataract 31, multiple types (CTRCT31): An opacification of the crystalline lens of the eye that frequently results in visual impairment or blindness. Opacities vary in morphology, are often confined to a portion of the lens, and may be static or progressive. In general, the more posteriorly located and dense an opacity, the greater the impact on visual function. CTRCT31 includes posterior polar, progressive posterior subcapsular, nuclear, and anterior subcapsular cataracts. [MIM:605387]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20q11.22, which is the long (q) arm of chromosome 20 at position 11.22Molecular Location: base pairs 33,811,304 to 33,854,367 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C20orf178CHMP4ACTPP3CTRCT31dJ553F4.4Shax1SNF7SNF7-2Vps32-2VPS32B     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CHMP4B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CATARACT 31, MULTIPLE TYPESOMIM: CHMP FAMILY, MEMBER 4B      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CHMP4B  OMIM: CATARACT 31, MULTIPLE TYPESOMIM: CHMP FAMILY, MEMBER 4B  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TGM1\t   The TGM1 gene provides instructions for making an enzyme called transglutaminase 1. This enzyme is found in cells that make up the outermost layer of the skin (the epidermis). Transglutaminase 1 is involved in the formation of the cornified cell envelope, which is a structure that surrounds skin cells and helps form a protective barrier between the body and its environment. Specifically, transglutaminase 1 forms strong bonds, called cross-links, between the structural proteins that make up the cornified cell envelope. This cross-linking provides strength and stability to the epidermis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Lamellar ichthyosis     Many mutations in the TGM1 gene have been found to cause lamellar ichthyosis, which is a condition that causes scaly skin that covers much of the body, and other skin abnormalities. Some TGM1 gene mutations that cause this condition change single DNA building blocks (nucleotides) in the transglutaminase 1 enzyme. The most frequently occurring mutation (written as 877-2A>G) affects the way the gene's instructions are pieced together to form the enzyme and results in an abnormally shortened, nonfunctional enzyme. Other TGM1 gene mutations result in a transglutaminase 1 enzyme that cannot function normally, is abnormally short, or is not produced. A lack of functional transglutaminase 1 prevents the formation of the cornified cell envelope, causing the skin abnormalities of lamellar ichthyosis. More About This Health Condition      Other disorders     In addition to lamellar ichthyosis (described above), TGM1 gene mutations have been found to cause other forms of ichthyosis. In one type, called self-healing collodion baby, affected infants are born with a tight, clear sheath covering their skin called a collodion membrane. This membrane usually dries and peels off during the first few weeks of life, and affected infants often show near normal skin within a few months. Another type of ichthyosis, called bathing suit ichthyosis, is characterized by scaly skin that is limited to the trunk.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Many mutations in the TGM1 gene have been found to cause lamellar ichthyosis, which is a condition that causes scaly skin that covers much of the body, and other skin abnormalities. Some TGM1 gene mutations that cause this condition change single DNA building blocks (nucleotides) in the transglutaminase 1 enzyme. The most frequently occurring mutation (written as 877-2A>G) affects the way the gene's instructions are pieced together to form the enzyme and results in an abnormally shortened, nonfunctional enzyme. Other TGM1 gene mutations result in a transglutaminase 1 enzyme that cannot function normally, is abnormally short, or is not produced. A lack of functional transglutaminase 1 prevents the formation of the cornified cell envelope, causing the skin abnormalities of lamellar ichthyosis. More About This Health Condition  In addition to lamellar ichthyosis (described above), TGM1 gene mutations have been found to cause other forms of ichthyosis. In one type, called self-healing collodion baby, affected infants are born with a tight, clear sheath covering their skin called a collodion membrane. This membrane usually dries and peels off during the first few weeks of life, and affected infants often show near normal skin within a few months. Another type of ichthyosis, called bathing suit ichthyosis, is characterized by scaly skin that is limited to the trunk.    Cytogenetic Location: 14q12, which is the long (q) arm of chromosome 14 at position 12Molecular Location: base pairs 24,249,114 to 24,263,210 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        epidermal TGaseICR2protein-glutamine gamma-glutamyltransferase KTGASETGase-1TGase KTGKTGM1_HUMANtransglutaminase-1transglutaminase 1 (K polypeptide epidermal type I, protein-glutamine-gamma-glutamyltransferase)transglutaminase Ktransglutaminase, keratinocyte     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (fourth edition, 2002): Epidermal Cells Form a Multilayered Waterproof Barrier      Clinical Information from GeneReviews (1 link)     Autosomal Recessive Congenital Ichthyosis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TGM1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSGLUTAMINASE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Epidermal Cells Form a Multilayered Waterproof Barrier  Autosomal Recessive Congenital Ichthyosis  Tests of TGM1  PubMed  TRANSGLUTAMINASE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Farasat S, Wei MH, Herman M, Liewehr DJ, Steinberg SM, Bale SJ, Fleckman P, Toro JR. Novel transglutaminase-1 mutations and genotype-phenotype investigations of 104 patients with autosomal recessive congenital ichthyosis in the USA. J Med Genet. 2009 Feb;46(2):103-11. doi: 10.1136/jmg.2008.060905. Epub 2008 Oct 23. Citation on PubMed or Free article on PubMed CentralHerman ML, Farasat S, Steinbach PJ, Wei MH, Toure O, Fleckman P, Blake P, Bale SJ, Toro JR. Transglutaminase-1 gene mutations in autosomal recessive congenital ichthyosis: summary of mutations (including 23 novel) and modeling of TGase-1. Hum Mutat. 2009 Apr;30(4):537-47. doi: 10.1002/humu.20952. Review. Citation on PubMed or Free article on PubMed CentralIsraeli S, Goldberg I, Fuchs-Telem D, Bergman R, Indelman M, Bitterman-Deutsch O, Harel A, Mashiach Y, Sarig O, Sprecher E. Non-syndromic autosomal recessive congenital ichthyosis in the Israeli population. Clin Exp Dermatol. 2013 Dec;38(8):911-6. doi: 10.1111/ced.12148. Epub 2013 Apr 26. Citation on PubMedOji V, Tadini G, Akiyama M, Blanchet Bardon C, Bodemer C, Bourrat E, Coudiere P, DiGiovanna JJ, Elias P, Fischer J, Fleckman P, Gina M, Harper J, Hashimoto T, Hausser I, Hennies HC, Hohl D, Hovnanian A, Ishida-Yamamoto A, Jacyk WK, Leachman S, Leigh I, Mazereeuw-Hautier J, Milstone L, Morice-Picard F, Paller AS, Richard G, Schmuth M, Shimizu H, Sprecher E, Van Steensel M, Taeb A, Toro JR, Vabres P, Vahlquist A, Williams M, Traupe H. Revised nomenclature and classification of inherited ichthyoses: results of the First Ichthyosis Consensus Conference in Sorze 2009. J Am Acad Dermatol. 2010 Oct;63(4):607-41. doi: 10.1016/j.jaad.2009.11.020. Review. Citation on PubMedRodrguez-Pazos L, Ginarte M, Vega A, Toribio J. Autosomal recessive congenital ichthyosis. Actas Dermosifiliogr. 2013 May;104(4):270-84. doi: 10.1016/j.adengl.2011.11.021. Epub 2013 Apr 3. Review. English, Spanish. Citation on PubMedOMIM: TRANSGLUTAMINASE 1Terrinoni A, Serra V, Codispoti A, Talamonti E, Bui L, Palombo R, Sette M, Campione E, Didona B, Annicchiarico-Petruzzelli M, Zambruno G, Melino G, Candi E. Novel transglutaminase 1 mutations in patients affected by lamellar ichthyosis. Cell Death Dis. 2012 Oct 25;3:e416. doi: 10.1038/cddis.2012.152. Citation on PubMed or Free article on PubMed Central \n",
            "NOP9\t\n",
            "NFATC4\t\n",
            "GZMB\t\n",
            "HEATR5A\t\n",
            "SSTR1\t\n",
            "MIA2\t   From NCBI Gene:This gene encodes s receptor in the endoplasmic reticulum, which plays a role in the export of large pre-chylomicrons and pre-very low density lipoproteins (pre-VLDLs). Three major classes of transcripts are generated from this gene- melanoma inhibitory activity 2-specific transcripts, cTAGE family member 5-specific transcripts and transcripts that include exons from both these transcript species (TANGO1-like or TALI). Additionally, alternative splicing in these transcripts results in multiple transcript variants encoding multiple isoforms. [provided by RefSeq, Sep 2016]From UniProt:Plays a role in the transport of cargos that are too large to fit into COPII-coated vesicles and require specific mechanisms to be incorporated into membrane-bound carriers and exported from the endoplasmic reticulum (PubMed:27138255, PubMed:21525241, PubMed:25202031, PubMed:27170179). Plays a role in the secretion of lipoproteins, pre-chylomicrons and pre-VLDLs, by participating in their export from the endoplasmic reticulum (PubMed:27138255). Thereby, may play a role in cholesterol and triglyceride homeostasis (By similarity). Required for collagen VII (COL7A1) secretion by loading COL7A1 into transport carriers and recruiting PREB/SEC12 at the endoplasmic reticulum exit sites (PubMed:21525241, PubMed:25202031, PubMed:27170179).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Autoantibodies against MIA2 are present in several cancer types, including benign meningioma and cutaneous T-cell lymphoma (CTCL).     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q21.1, which is the long (q) arm of chromosome 14 at position 21.1Molecular Location: base pairs 39,233,916 to 39,388,513 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CTAGE5MEA6MGEAMGEA6MGEA11TALI     Related Information   How are genetic conditions and genes named?          Catalog of Genes and Diseases from OMIM (1 link)     OMIM: CUTANEOUS T-CELL LYMPHOMA-ASSOCIATED ANTIGEN 5      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      OMIM: CUTANEOUS T-CELL LYMPHOMA-ASSOCIATED ANTIGEN 5  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FSCB\t\n",
            "DNAAF2\t   From NCBI Gene:This gene encodes a highly conserved protein involved in the preassembly of dynein arm complexes which power cilia. These complexes are found in some cilia and are assembled in the cytoplasm prior to transport for cilia formation. Mutations in this gene have been associated with primary ciliary dyskinesia. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Oct 2009]From UniProt:Required for cytoplasmic pre-assembly of axonemal dyneins, thereby playing a central role in motility in cilia and flagella. Involved in pre-assembly of dynein arm complexes in the cytoplasm before intraflagellar transport loads them for the ciliary compartment.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary ciliary dyskinesiaFrom NCBI Gene:Ciliary dyskinesia, primary, 10From UniProt:Ciliary dyskinesia, primary, 10 (CILD10): A disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia; reduced fertility is often observed in male patients due to abnormalities of sperm tails. Half of the patients exhibit randomization of left-right body asymmetry and situs inversus, due to dysfunction of monocilia at the embryonic node. Primary ciliary dyskinesia associated with situs inversus is referred to as Kartagener syndrome. [MIM:612518]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q21.3, which is the long (q) arm of chromosome 14 at position 21.3Molecular Location: base pairs 49,625,174 to 49,635,230 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C14orf104CILD10KTUPF13     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DNAAF2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 10OMIM: DYNEIN, AXONEMAL, ASSEMBLY FACTOR 2      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DNAAF2  OMIM: CILIARY DYSKINESIA, PRIMARY, 10OMIM: DYNEIN, AXONEMAL, ASSEMBLY FACTOR 2  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NEMF\t\n",
            "CDKL1\t\n",
            "MAP4K5\t\n",
            "NID2\t\n",
            "DDHD1\t   From NCBI Gene:This gene is a member of the intracellular phospholipase A1 gene family. The protein encoded by this gene preferentially hydrolyzes phosphatidic acid. It is a cytosolic protein with some mitochondrial localization, and is thought to be involved in the regulation of mitochondrial dynamics. Overexpression of this gene causes fragmentation of the tubular structures in mitochondria, while depletion of the gene results in mitochondrial tubule elongation. Deletion of this gene in male mice caused fertility defects, resulting from disruption in the organization of the mitochondria during spermiogenesis. In humans, mutations in this gene have been associated with hereditary spastic paraplegia (HSP), also known as Strumpell-Lorrain disease, or, familial spastic paraparesis (FSP). This inherited disorder is characterized by progressive weakness and spasticity of the legs. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Aug 2015]From UniProt:Phospholipase that hydrolyzes phosphatidic acid, including 1,2-dioleoyl-sn-phosphatidic acid. The different isoforms may change the substrate specificity (PubMed:22922100). Required for the organization of the endoplasmic reticulum exit sites (ERES), also known as transitional endoplasmic reticulum (tER) (PubMed:17428803).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spastic paraplegia 28, autosomal recessiveFrom UniProt:Spastic paraplegia 28, autosomal recessive (SPG28): A form of spastic paraplegia, a neurodegenerative disorder characterized by a slow, gradual, progressive weakness and spasticity of the lower limbs. Rate of progression and the severity of symptoms are quite variable. Initial symptoms may include difficulty with balance, weakness and stiffness in the legs, muscle spasms, and dragging the toes when walking. In some forms of the disorder, bladder symptoms (such as incontinence) may appear, or the weakness and stiffness may spread to other parts of the body. Some SPG28 patients also have distal sensory impairment. [MIM:609340]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q22.1, which is the long (q) arm of chromosome 14 at position 22.1Molecular Location: base pairs 53,036,740 to 53,153,328 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PA-PLA1PAPLA1SPG28     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DDHD1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DDHD DOMAIN-CONTAINING PROTEIN 1OMIM: SPASTIC PARAPLEGIA 28, AUTOSOMAL RECESSIVE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DDHD1  OMIM: DDHD DOMAIN-CONTAINING PROTEIN 1OMIM: SPASTIC PARAPLEGIA 28, AUTOSOMAL RECESSIVE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PLEKHG3\t\n",
            "RAD51B\t   From NCBI Gene:The protein encoded by this gene is a member of the RAD51 protein family. RAD51 family members are evolutionarily conserved proteins essential for DNA repair by homologous recombination. This protein has been shown to form a stable heterodimer with the family member RAD51C, which further interacts with the other family members, such as RAD51, XRCC2, and XRCC3. Overexpression of this gene was found to cause cell cycle G1 delay and cell apoptosis, which suggested a role of this protein in sensing DNA damage. Rearrangements between this locus and high mobility group AT-hook 2 (HMGA2, GeneID 8091) have been observed in uterine leiomyomata. [provided by RefSeq, Mar 2016]From UniProt:Involved in the homologous recombination repair (HRR) pathway of double-stranded DNA breaks arising during DNA replication or induced by DNA-damaging agents. May promote the assembly of presynaptic RAD51 nucleoprotein filaments. Binds single-stranded DNA and double-stranded DNA and has DNA-dependent ATPase activity. Part of the RAD21 paralog protein complex BCDX2 which acts in the BRCA1-BRCA2-dependent HR pathway. Upon DNA damage, BCDX2 acts downstream of BRCA2 recruitment and upstream of RAD51 recruitment. BCDX2 binds predominantly to the intersection of the four duplex arms of the Holliday junction and to junction of replication forks. The BCDX2 complex was originally reported to bind single-stranded DNA, single-stranded gaps in duplex DNA and specifically to nicks in duplex DNA. The BCDX2 subcomplex RAD51B:RAD51C exhibits single-stranded DNA-dependent ATPase activity suggesting an involvement in early stages of the HR pathway.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving RAD51B is found in pulmonary chondroid hamartoma. Translocation t(6;14)(p21;q23-24) with HMGA1.A chromosomal aberration involving RAD51B is found in uterine leiomyoma. Translocation t(12;14)(q15;q23-24) with HMGA2.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q24.1, which is the long (q) arm of chromosome 14 at position 24.1Molecular Location: base pairs 67,819,779 to 68,683,118 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        R51H2RAD51L1REC2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RAD51B      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: RAD51, S. CEREVISIAE, HOMOLOG OF, B      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RAD51B  OMIM: RAD51, S. CEREVISIAE, HOMOLOG OF, B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MAP3K9\t\n",
            "DCAF4\t\n",
            "RBM25\t\n",
            "PAPLN\t\n",
            "RIOX1\t\n",
            "ACOT4\t\n",
            "ANGEL1\t\n",
            "IRF2BPL\t   From NCBI Gene:This gene encodes a transcription factor that may play a role in regulating female reproductive function. [provided by RefSeq, Jun 2012]From UniProt:Probable E3 ubiquitin protein ligase involved in the proteasome-mediated ubiquitin-dependent degradation of target proteins (PubMed:29374064). Through the degradation of CTNNB1, functions downstream of FOXF2 to negatively regulate the Wnt signaling pathway (PubMed:29374064). Probably plays a role in the development of the central nervous system and in neuronal maintenance (Probable). Also acts as a transcriptional regulator of genes controlling female reproductive function. May play a role in gene transcription by transactivating GNRH1 promoter and repressing PENK promoter (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:NEURODEVELOPMENTAL DISORDER WITH REGRESSION, ABNORMAL MOVEMENTS, LOSS OF SPEECH, AND SEIZURESFrom UniProt:Neurodevelopmental disorder with regression, abnormal movements, loss of speech, and seizures (NEDAMSS): An autosomal dominant disorder characterized by global developmental delay or neurodevelopmental regression, hypotonia, progressive ataxia, intellectual disability, seizures, and abnormal movements. [MIM:618088]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q24.3, which is the long (q) arm of chromosome 14 at position 24.3Molecular Location: base pairs 77,024,543 to 77,028,699 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C14orf4EAP1NEDAMSS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of IRF2BPL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: INTERFERON REGULATORY FACTOR 2-BINDING PROTEIN LIKEOMIM: NEURODEVELOPMENTAL DISORDER WITH REGRESSION, ABNORMAL MOVEMENTS, LOSS OF SPEECH, AND SEIZURES      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of IRF2BPL  OMIM: INTERFERON REGULATORY FACTOR 2-BINDING PROTEIN LIKEOMIM: NEURODEVELOPMENTAL DISORDER WITH REGRESSION, ABNORMAL MOVEMENTS, LOSS OF SPEECH, AND SEIZURES  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CEP128\t\n",
            "STON2\t\n",
            "ATXN3\t   The ATXN3 gene provides instructions for making an enzyme called ataxin-3, which is found in cells throughout the body. Ataxin-3 is involved in a mechanism called the ubiquitin-proteasome system that destroys and gets rid of excess or damaged proteins. The molecule ubiquitin attaches (binds) to unneeded proteins and tags them to be broken down (degraded) within cells. Ataxin-3 removes (cleaves) the ubiquitin from these unwanted proteins just before they are degraded so that the ubiquitin can be used again. Due to its role in cleaving ubiquitin from proteins, ataxin-3 is known as a deubiquitinating enzyme.Researchers believe that ataxin-3 also may be involved in regulating the first stage of protein production (transcription).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Spinocerebellar ataxia type 3     Spinocerebellar ataxia type 3 (SCA3) is a condition characterized by progressive problems with movement. SCA3 results from a mutation in the ATXN3 gene known as a trinucleotide repeat expansion. This mutation increases the length of the repeated CAG segment in the ATXN3 gene. People with SCA3 have more than 50 CAG repeats.The expanded CAG segment leads to the production of an abnormally long version of the ataxin-3 protein that folds into the wrong 3-dimensional shape. This nonfunctional ataxin-3 protein cannot remove ubiquitin from proteins that are no longer needed. As a result, these unwanted proteins, along with ubiquitin and ataxin-3, cluster together to form clumps (aggregates) within the nucleus of the cells. It is unclear how these aggregates affect cell function, because they are found in healthy cells as well as those that die.Nerve cells (neurons) and other types of brain cells are most affected by mutations in the ATXN3 gene. SCA3 is associated with cell death in the part of the brain that is connected to the spinal cord (the brainstem), the part of the brain involved in coordinating movements (the cerebellum), and other areas of the brain. This condition is also associated with the death of neurons in the spinal cord. Over time, the loss of cells in the brain and spinal cord cells cause the signs and symptoms characteristic of SCA3. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Spinocerebellar ataxia type 3 (SCA3) is a condition characterized by progressive problems with movement. SCA3 results from a mutation in the ATXN3 gene known as a trinucleotide repeat expansion. This mutation increases the length of the repeated CAG segment in the ATXN3 gene. People with SCA3 have more than 50 CAG repeats.The expanded CAG segment leads to the production of an abnormally long version of the ataxin-3 protein that folds into the wrong 3-dimensional shape. This nonfunctional ataxin-3 protein cannot remove ubiquitin from proteins that are no longer needed. As a result, these unwanted proteins, along with ubiquitin and ataxin-3, cluster together to form clumps (aggregates) within the nucleus of the cells. It is unclear how these aggregates affect cell function, because they are found in healthy cells as well as those that die.Nerve cells (neurons) and other types of brain cells are most affected by mutations in the ATXN3 gene. SCA3 is associated with cell death in the part of the brain that is connected to the spinal cord (the brainstem), the part of the brain involved in coordinating movements (the cerebellum), and other areas of the brain. This condition is also associated with the death of neurons in the spinal cord. Over time, the loss of cells in the brain and spinal cord cells cause the signs and symptoms characteristic of SCA3. More About This Health Condition    Cytogenetic Location: 14q32.12, which is the long (q) arm of chromosome 14 at position 32.12Molecular Location: base pairs 92,058,552 to 92,106,621 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AT3ataxin-3ATX3ATX3_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Basic Neurochemistry (sixth edition, 1999): Trinucleotide expansion is one of the most important categories of mutation underlying neurodegenerative disordersMadame Curie Bioscience Database: Ubiquitination/De-UbiquitinationWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Spinocerebellar Ataxia Type 3      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATXN3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATAXIN 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Trinucleotide expansion is one of the most important categories of mutation underlying neurodegenerative disordersMadame Curie Bioscience Database: Ubiquitination/De-UbiquitinationWashington University, St. Louis: Neuromuscular Disease Center  Spinocerebellar Ataxia Type 3  Tests of ATXN3  PubMed  ATAXIN 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ATAXIN 3D'Abreu A, Frana MC Jr, Paulson HL, Lopes-Cendes I. Caring for Machado-Joseph disease: current understanding and how to help patients. Parkinsonism Relat Disord. 2010 Jan;16(1):2-7. doi: 10.1016/j.parkreldis.2009.08.012. Epub 2009 Oct 6. Review. Citation on PubMed or Free article on PubMed CentralHaacke A, Broadley SA, Boteva R, Tzvetkov N, Hartl FU, Breuer P. Proteolytic cleavage of polyglutamine-expanded ataxin-3 is critical for aggregation and sequestration of non-expanded ataxin-3. Hum Mol Genet. 2006 Feb 15;15(4):555-68. Epub 2006 Jan 11. Citation on PubMedNicastro G, Masino L, Esposito V, Menon RP, De Simone A, Fraternali F, Pastore A. Josephin domain of ataxin-3 contains two distinct ubiquitin-binding sites. Biopolymers. 2009 Dec;91(12):1203-14. doi: 10.1002/bip.21210. Citation on PubMed \n",
            "RIN3\t\n",
            "IFI27\t\n",
            "SERPINA9\t\n",
            "SYNE3\t\n",
            "BCL11B\t   From NCBI Gene:This gene encodes a C2H2-type zinc finger protein and is closely related to BCL11A, a gene whose translocation may be associated with B-cell malignancies. Although the specific function of this gene has not been determined, the encoded protein is known to be a transcriptional repressor, and is regulated by the NURD nucleosome remodeling and histone deacetylase complex. Four alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Aug 2013]From UniProt:Key regulator of both differentiation and survival of T-lymphocytes during thymocyte development in mammals. Essential in controlling the responsiveness of hematopoietic stem cells to chemotactic signals by modulating the expression of the receptors CCR7 and CCR9, which direct the movement of progenitor cells from the bone marrow to the thymus (PubMed:27959755). Is a regulator of IL2 promoter and enhances IL2 expression in activated CD4(+) T-lymphocytes (PubMed:16809611). Tumor-suppressor that represses transcription through direct, TFCOUP2-independent binding to a GC-rich response element (By similarity). May also function in the P53-signaling pathway (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:INTELLECTUAL DEVELOPMENTAL DISORDER WITH SPEECH DELAY, DYSMORPHIC FACIES, AND T-CELL ABNORMALITIESImmunodeficiency 49From UniProt:Intellectual developmental disorder with speech delay, dysmorphic facies, and T-cell abnormalities (IDDSFTA): An autosomal dominant developmental disorder with onset in first months of life, and characterized by delayed psychomotor development with intellectual disability and speech delay. Additional features include autistic features, attention deficit-hyperactivity disorder, anxiety, and other behavioral abnormalities. Some patients suffer from recurrent infections, asthma and allergies. [MIM:618092]Immunodeficiency 49 (IMD49): A form of severe combined immunodeficiency characterized by severe T-cell lymphopenia, no detectable T-cell receptor excision circles, no naive helper CD4+ T-cells, and impaired T-cell proliferative response. In addition to primary immunodeficiency, affected individuals manifest multiple abnormal systemic features, including severe delayed psychomotor development, intellectual disability, spastic quadriplegia, and craniofacial abnormalities. [MIM:617237]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 14q32.2, which is the long (q) arm of chromosome 14 at position 32.2Molecular Location: base pairs 99,169,287 to 99,271,713 on chromosome 14 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATL1ATL1-alphaATL1-betaATL1-deltaATL1-gammaCTIP-2CTIP2hRIT1-alphaIDDFSTAIMD49RIT1ZNF856B     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of BCL11B      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: B-CELL CLL/LYMPHOMA 11BOMIM: IMMUNODEFICIENCY 49OMIM: INTELLECTUAL DEVELOPMENTAL DISORDER WITH SPEECH DELAY, DYSMORPHIC FACIES, AND T-CELL ABNORMALITIES      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of BCL11B  OMIM: B-CELL CLL/LYMPHOMA 11BOMIM: IMMUNODEFICIENCY 49OMIM: INTELLECTUAL DEVELOPMENTAL DISORDER WITH SPEECH DELAY, DYSMORPHIC FACIES, AND T-CELL ABNORMALITIES  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MEF2A\t   From NCBI Gene:The protein encoded by this gene is a DNA-binding transcription factor that activates many muscle-specific, growth factor-induced, and stress-induced genes. The encoded protein can act as a homodimer or as a heterodimer and is involved in several cellular processes, including muscle development, neuronal differentiation, cell growth control, and apoptosis. Defects in this gene could be a cause of autosomal dominant coronary artery disease 1 with myocardial infarction (ADCAD1). Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2010]From UniProt:Transcriptional activator which binds specifically to the MEF2 element, 5'-YTA[AT](4)TAR-3', found in numerous muscle-specific genes. Also involved in the activation of numerous growth factor- and stress-induced genes. Mediates cellular functions not only in skeletal and cardiac muscle development, but also in neuronal differentiation and survival. Plays diverse roles in the control of cell growth, survival and apoptosis via p38 MAPK signaling in muscle-specific and/or growth factor-related transcription. In cerebellar granule neurons, phosphorylated and sumoylated MEF2A represses transcription of NUR77 promoting synaptic differentiation. Associates with chromatin to the ZNF16 promoter.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Coronary artery disease, autosomal dominant, 1From UniProt:Coronary artery disease, autosomal dominant, 1 (ADCAD1): A common heart disease characterized by reduced or absent blood flow in one or more of the arteries that encircle and supply the heart. Its most important complication is acute myocardial infarction. [MIM:608320]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q26.3, which is the long (q) arm of chromosome 15 at position 26.3Molecular Location: base pairs 99,565,547 to 99,716,488 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADCAD1mef2RSRFC4RSRFC9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MEF2A      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CORONARY ARTERY DISEASE, AUTOSOMAL DOMINANT, 1OMIM: MADS BOX TRANSCRIPTION ENHANCER FACTOR 2, POLYPEPTIDE A      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MEF2A  OMIM: CORONARY ARTERY DISEASE, AUTOSOMAL DOMINANT, 1OMIM: MADS BOX TRANSCRIPTION ENHANCER FACTOR 2, POLYPEPTIDE A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LYSMD4\t\n",
            "LINS1\t   From NCBI Gene:The Drosophila segment polarity gene lin encodes a protein, lines, which plays important roles in development of the epidermis and hindgut. This gene encodes a protein containing a lines-like domain. This gene is located on chromosome 15 and clustered with the gene encoding ankyrin repeat and SOCS box-containing protein 7. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2017]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mental retardation, autosomal recessive 27From UniProt:Mental retardation, autosomal recessive 27 (MRT27): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. [MIM:614340]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q26.3, which is the long (q) arm of chromosome 15 at position 26.3Molecular Location: base pairs 100,566,914 to 100,603,032 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LINSMRT27WINS1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LINS1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LINES, DROSOPHILA, HOMOLOG OFOMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 27      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LINS1  OMIM: LINES, DROSOPHILA, HOMOLOG OFOMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 27  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PCSK6\t\n",
            "OR4M2\t\n",
            "OR4N4\t\n",
            "OCA2\t   The OCA2 gene (formerly called the P gene) provides instructions for making a protein called the P protein. This protein is located in melanocytes, which are specialized cells that produce a pigment called melanin. Melanin is the substance that gives skin, hair, and eyes their color. Melanin is also found in the light-sensitive tissue at the back of the eye (the retina), where it plays a role in normal vision.Although the exact function of the P protein is unknown, it is essential for normal pigmentation and is likely involved in the production of melanin. Within melanocytes, the P protein may transport molecules into and out of structures called melanosomes (where melanin is produced).  Researchers believe that this protein may also help regulate the relative acidity (pH) of melanosomes.  Tight control of pH is necessary for most biological processes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Oculocutaneous albinism     More than 80 mutations in the OCA2 gene have been identified in people with oculocutaneous albinism type 2. People with this form of albinism often have light yellow, blond, or light brown hair; creamy white skin; light-colored eyes; and problems with vision. The most common OCA2 mutation is a large deletion in the gene, which is found in many affected individuals of sub-Saharan African heritage. Other OCA2 gene mutations, including changes in single DNA building blocks (base pairs) and small deletions, are more common in other populations. Mutations in the OCA2 gene disrupt the normal production of melanin, which reduces coloring of the hair, skin, and eyes and affects vision. More About This Health Condition      Angelman syndrome     The OCA2 gene is located in a region of chromosome 15 that is often deleted in individuals with Angelman syndrome. A loss of this gene does not cause the characteristic neurologic features of Angelman syndrome; however, people with this condition who are missing one copy of the OCA2 gene tend to have unusually light-colored hair and fair skin. Cells with only one copy of the OCA2 gene make a reduced amount of P protein compared with cells with two functional copies of this gene, which affects the coloring of the hair and skin.A small percentage of people with Angelman syndrome also have oculocutaneous albinism type 2. This condition occurs when people have two nonfunctional copies of the OCA2 gene in each cell. In addition to a deletion in chromosome 15 that removes one copy of the OCA2 gene, these individuals have a mutation in the OCA2 gene on the other copy of chromosome 15. As a result, cells make little or no functional P protein. A lack of P protein disrupts the production of melanin, leading to the characteristic features of albinism. More About This Health Condition      Melanoma     Genetics Home Reference provides information about melanoma. More About This Health Condition      Prader-Willi syndrome     The region of chromosome 15 containing the OCA2 gene is often deleted in individuals with Prader-Willi syndrome. A loss of this gene does not cause intellectual disability and the other characteristic features of Prader-Willi syndrome; however, people with this condition who are missing one copy of the OCA2 gene tend to have unusually light-colored hair and fair skin. Cells missing a copy of the OCA2 gene make less P protein than cells with two functional copies of the gene, which affects the coloring of the hair and skin.Oculocutaneous albinism type 2 also occurs in a small number of people with Prader-Willi syndrome. This condition occurs when people have two nonfunctional copies of the OCA2 gene in each cell. In addition to a deletion in chromosome 15 that removes one copy of the OCA2 gene, these individuals have a mutation in the OCA2 gene on the other copy of chromosome 15. As a result, cells make little or no functional P protein. A lack of P protein disrupts the production of melanin, leading to the characteristic features of albinism. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 80 mutations in the OCA2 gene have been identified in people with oculocutaneous albinism type 2. People with this form of albinism often have light yellow, blond, or light brown hair; creamy white skin; light-colored eyes; and problems with vision. The most common OCA2 mutation is a large deletion in the gene, which is found in many affected individuals of sub-Saharan African heritage. Other OCA2 gene mutations, including changes in single DNA building blocks (base pairs) and small deletions, are more common in other populations. Mutations in the OCA2 gene disrupt the normal production of melanin, which reduces coloring of the hair, skin, and eyes and affects vision. More About This Health Condition  The OCA2 gene is located in a region of chromosome 15 that is often deleted in individuals with Angelman syndrome. A loss of this gene does not cause the characteristic neurologic features of Angelman syndrome; however, people with this condition who are missing one copy of the OCA2 gene tend to have unusually light-colored hair and fair skin. Cells with only one copy of the OCA2 gene make a reduced amount of P protein compared with cells with two functional copies of this gene, which affects the coloring of the hair and skin.A small percentage of people with Angelman syndrome also have oculocutaneous albinism type 2. This condition occurs when people have two nonfunctional copies of the OCA2 gene in each cell. In addition to a deletion in chromosome 15 that removes one copy of the OCA2 gene, these individuals have a mutation in the OCA2 gene on the other copy of chromosome 15. As a result, cells make little or no functional P protein. A lack of P protein disrupts the production of melanin, leading to the characteristic features of albinism. More About This Health Condition  Genetics Home Reference provides information about melanoma. More About This Health Condition  The region of chromosome 15 containing the OCA2 gene is often deleted in individuals with Prader-Willi syndrome. A loss of this gene does not cause intellectual disability and the other characteristic features of Prader-Willi syndrome; however, people with this condition who are missing one copy of the OCA2 gene tend to have unusually light-colored hair and fair skin. Cells missing a copy of the OCA2 gene make less P protein than cells with two functional copies of the gene, which affects the coloring of the hair and skin.Oculocutaneous albinism type 2 also occurs in a small number of people with Prader-Willi syndrome. This condition occurs when people have two nonfunctional copies of the OCA2 gene in each cell. In addition to a deletion in chromosome 15 that removes one copy of the OCA2 gene, these individuals have a mutation in the OCA2 gene on the other copy of chromosome 15. As a result, cells make little or no functional P protein. A lack of P protein disrupts the production of melanin, leading to the characteristic features of albinism. More About This Health Condition    Cytogenetic Location: 15q12-q13.1, which is the long (q) arm of chromosome 15 between positions 12 and 13.1Molecular Location: base pairs 27,719,008 to 28,099,342 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BOCAMelanocyte-specific transporter proteinoculocutaneous albinism IIoculocutaneous albinism II (pink-eye dilution homolog, mouse)P geneP_HUMANPEDPink-eyed dilution protein homolog     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Protein Spotlight, Swiss Institute of Bioinformatics      Clinical Information from GeneReviews (3 links)     Angelman SyndromeOculocutaneous Albinism Type 2Prader-Willi Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of OCA2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     OCA2 MELANOSOMAL TRANSMEMBRANE PROTEIN      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportInternational Albinism Center, University of Minnesota: Albinism DatabaseMonarch InitiativeNCBI GeneUniProt      Protein Spotlight, Swiss Institute of Bioinformatics  Angelman SyndromeOculocutaneous Albinism Type 2Prader-Willi Syndrome  Tests of OCA2  PubMed  OCA2 MELANOSOMAL TRANSMEMBRANE PROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportInternational Albinism Center, University of Minnesota: Albinism DatabaseMonarch InitiativeNCBI GeneUniProt  Bittel DC, Kibiryeva N, Talebizadeh Z, Butler MG. Microarray analysis of gene/transcript expression in Prader-Willi syndrome: deletion versus UPD. J Med Genet. 2003 Aug;40(8):568-74. Citation on PubMed or Free article on PubMed CentralBittel DC, Kibiryeva N, Talebizadeh Z, Driscoll DJ, Butler MG. Microarray analysis of gene/transcript expression in Angelman syndrome: deletion versus UPD. Genomics. 2005 Jan;85(1):85-91. Citation on PubMedBrilliant MH. The mouse p (pink-eyed dilution) and human P genes, oculocutaneous albinism type 2 (OCA2), and melanosomal pH. Pigment Cell Res. 2001 Apr;14(2):86-93. Review. Citation on PubMedDuffy DL, Montgomery GW, Chen W, Zhao ZZ, Le L, James MR, Hayward NK, Martin NG, Sturm RA. A three-single-nucleotide polymorphism haplotype in intron 1 of OCA2 explains most human eye-color variation. Am J Hum Genet. 2007 Feb;80(2):241-52. Epub 2006 Dec 20. Citation on PubMed or Free article on PubMed CentralFridman C, Hosomi N, Varela MC, Souza AH, Fukai K, Koiffmann CP. Angelman syndrome associated with oculocutaneous albinism due to an intragenic deletion of the P gene. Am J Med Genet A. 2003 Jun 1;119A(2):180-3. Citation on PubMedKerr R, Stevens G, Manga P, Salm S, John P, Haw T, Ramsay M. Identification of P gene mutations in individuals with oculocutaneous albinism in sub-Saharan Africa. Hum Mutat. 2000;15(2):166-72. Erratum in: Hum Mutat 2000;16(1):following 86. Citation on PubMedLewis RA. Oculocutaneous Albinism Type 2. 2003 Jul 17 [updated 2012 Aug 16]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1232/ Citation on PubMedOetting WS, Garrett SS, Brott M, King RA. P gene mutations associated with oculocutaneous albinism type II (OCA2). Hum Mutat. 2005 Mar;25(3):323. Citation on PubMedSaitoh S, Oiso N, Wada T, Narazaki O, Fukai K. Oculocutaneous albinism type 2 with a P gene missense mutation in a patient with Angelman syndrome. J Med Genet. 2000 May;37(5):392-4. Citation on PubMed or Free article on PubMed CentralSpritz RA, Bailin T, Nicholls RD, Lee ST, Park SK, Mascari MJ, Butler MG. Hypopigmentation in the Prader-Willi syndrome correlates with P gene deletion but not with haplotype of the hemizygous P allele. Am J Med Genet. 1997 Jul 11;71(1):57-62. Citation on PubMedSuzuki T, Miyamura Y, Matsunaga J, Shimizu H, Kawachi Y, Ohyama N, Ishikawa O, Ishikawa T, Terao H, Tomita Y. Six novel P gene mutations and oculocutaneous albinism type 2 frequency in Japanese albino patients. J Invest Dermatol. 2003 May;120(5):781-3. Citation on PubMedToyofuku K, Valencia JC, Kushimoto T, Costin GE, Virador VM, Vieira WD, Ferrans VJ, Hearing VJ. The etiology of oculocutaneous albinism (OCA) type II: the pink protein modulates the processing and transport of tyrosinase. Pigment Cell Res. 2002 Jun;15(3):217-24. Erratum in: Pigment Cell Res. 2002 Oct;15(5):400.. Citation on PubMed \n",
            "HERC2\t   From NCBI Gene:This gene belongs to the HERC gene family that encodes a group of unusually large proteins, which contain multiple structural domains. All members have at least 1 copy of an N-terminal region showing homology to the cell cycle regulator RCC1 and a C-terminal HECT (homologous to E6-AP C terminus) domain found in a number of E3 ubiquitin protein ligases. Genetic variations in this gene are associated with skin/hair/eye pigmentation variability. Multiple pseudogenes of this gene are located on chromosomes 15 and 16. [provided by RefSeq, Mar 2012]From UniProt:E3 ubiquitin-protein ligase that regulates ubiquitin-dependent retention of repair proteins on damaged chromosomes. Recruited to sites of DNA damage in response to ionizing radiation (IR) and facilitates the assembly of UBE2N and RNF8 promoting DNA damage-induced formation of 'Lys-63'-linked ubiquitin chains. Acts as a mediator of binding specificity between UBE2N and RNF8. Involved in the maintenance of RNF168 levels. E3 ubiquitin-protein ligase that promotes the ubiquitination and proteasomal degradation of XPA which influences the circadian oscillation of DNA excision repair activity. By controlling the steady-state expression of the IGF1R receptor, indirectly regulates the insulin-like growth factor receptor signaling pathway (PubMed:26692333).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:MelanomaFrom NCBI Gene:Skin/hair/eye pigmentation, variation in, 1Prader-Willi syndromeMental retardation, autosomal recessive 38From UniProt:Mental retardation, autosomal recessive 38 (MRT38): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. MRT38 is characterized by global developmental delay affecting motor, speech, adaptive, and social development. Patients manifest autistic features, aggression, self-injury, impulsivity, and distractibility. [MIM:615516]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q13.1, which is the long (q) arm of chromosome 15 at position 13.1Molecular Location: base pairs 28,111,037 to 28,322,173 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        D15F37S1jdf2MRT38p528SHEP1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HERC2      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: HECT DOMAIN AND RCC1-LIKE DOMAIN 2OMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 38OMIM: PRADER-WILLI SYNDROMEOMIM: SKIN/HAIR/EYE PIGMENTATION, VARIATION IN, 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HERC2  OMIM: HECT DOMAIN AND RCC1-LIKE DOMAIN 2OMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 38OMIM: PRADER-WILLI SYNDROMEOMIM: SKIN/HAIR/EYE PIGMENTATION, VARIATION IN, 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GOLGA8F\t\n",
            "FAM189A1\t\n",
            "GOLGA8R\t\n",
            "GOLGA8H\t\n",
            "TRPM1\t   The TRPM1 gene provides instructions for making a protein called transient receptor potential cation channel subfamily M member 1 (TRPM1). This protein acts as a channel, transporting positively charged atoms (cations) into cells. The TRPM1 channel is found on the surface of two types of cells: pigment-producing cells called melanocytes and specialized bipolar cells in the light-sensitive tissue at the back of the eye (the retina).In melanocytes, the TRPM1 channel is thought to play a role in the production of a pigment called melanin, which is the substance that gives skin, hair, and eyes their color (pigmentation). It is unclear what role the channel plays, but increased channel activity is associated with greater melanin production and darker pigmentation.In bipolar cells, TRPM1 channels are involved in the pathway that receives visual signals from cells called rods, which are used to see in low light. This signaling is an essential step in the transmission of visual information from the eyes to the brain. In low-light conditions, visual signals from rod cells trigger the TRPM1 channels to close, which causes visual signals to be transmitted. In bright-light conditions, the TRPM1 channel is open, allowing cations to flow in and out of bipolar cells and preventing visual signals from being sent.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Autosomal recessive congenital stationary night blindness     More than 35 mutations in the TRPM1 gene have been found to cause autosomal recessive congenital stationary night blindness, which is characterized by the inability to see in low light and other vision problems such as nearsightedness (myopia). Mutations in the TRPM1 gene are found in approximately half of all people with this condition.Most TRPM1 gene mutations change single protein building blocks (amino acids) in the TRPM1 channel and either alter the structure of the channel or prevent the channel from reaching the bipolar cell membrane. As a result, the TRPM1 channel is nonfunctional and prevents bipolar cells from relaying visual signals. The brain does not receive the visual information sent by rods, leading to difficulty seeing in low light. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 35 mutations in the TRPM1 gene have been found to cause autosomal recessive congenital stationary night blindness, which is characterized by the inability to see in low light and other vision problems such as nearsightedness (myopia). Mutations in the TRPM1 gene are found in approximately half of all people with this condition.Most TRPM1 gene mutations change single protein building blocks (amino acids) in the TRPM1 channel and either alter the structure of the channel or prevent the channel from reaching the bipolar cell membrane. As a result, the TRPM1 channel is nonfunctional and prevents bipolar cells from relaying visual signals. The brain does not receive the visual information sent by rods, leading to difficulty seeing in low light. More About This Health Condition    Cytogenetic Location: 15q13.3, which is the long (q) arm of chromosome 15 at position 13.3Molecular Location: base pairs 31,001,061 to 31,161,273 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        long transient receptor potential channel 1LTRPC1melastatin-1MLSN1transient receptor potential cation channel, subfamily M, member 1transient receptor potential melastatin familyTRPM1_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Madame Curie Bioscience Database: The TRP Family of Channel ProteinsTRP Ion Channel Function in Sensory Transduction and Cellular Signaling Cascades (2007): TRP Superfamily Genes and Their Gene ProductsWebvision: The Organization of the Retina and Visual System: Retinal Processing: Bipolar Cells and the Photopic and Scotopic Pathways      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TRPM1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY M, MEMBER 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: The TRP Family of Channel ProteinsTRP Ion Channel Function in Sensory Transduction and Cellular Signaling Cascades (2007): TRP Superfamily Genes and Their Gene ProductsWebvision: The Organization of the Retina and Visual System: Retinal Processing: Bipolar Cells and the Photopic and Scotopic Pathways  Tests of TRPM1  PubMed  TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY M, MEMBER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Audo I, Kohl S, Leroy BP, Munier FL, Guillonneau X, Mohand-Sad S, Bujakowska K, Nandrot EF, Lorenz B, Preising M, Kellner U, Renner AB, Bernd A, Antonio A, Moskova-Doumanova V, Lancelot ME, Poloschek CM, Drumare I, Defoort-Dhellemmes S, Wissinger B, Lveillard T, Hamel CP, Schorderet DF, De Baere E, Berger W, Jacobson SG, Zrenner E, Sahel JA, Bhattacharya SS, Zeitz C. TRPM1 is mutated in patients with autosomal-recessive complete congenital stationary night blindness. Am J Hum Genet. 2009 Nov;85(5):720-9. doi: 10.1016/j.ajhg.2009.10.013. Epub 2009 Nov 5. Citation on PubMed or Free article on PubMed CentralDevi S, Markandeya Y, Maddodi N, Dhingra A, Vardi N, Balijepalli RC, Setaluri V. Metabotropic glutamate receptor 6 signaling enhances TRPM1 calcium channel function and increases melanin content in human melanocytes. Pigment Cell Melanoma Res. 2013 May;26(3):348-56. doi: 10.1111/pcmr.12083. Epub 2013 Mar 27. Citation on PubMed or Free article on PubMed CentralGuo H, Carlson JA, Slominski A. Role of TRPM in melanocytes and melanoma. Exp Dermatol. 2012 Sep;21(9):650-4. doi: 10.1111/j.1600-0625.2012.01565.x. Citation on PubMed or Free article on PubMed CentralKoike C, Numata T, Ueda H, Mori Y, Furukawa T. TRPM1: a vertebrate TRP channel responsible for retinal ON bipolar function. Cell Calcium. 2010 Aug-Sep;48(2-3):95-101. doi: 10.1016/j.ceca.2010.08.004. Epub 2010 Sep 16. Review. Citation on PubMedLi Z, Sergouniotis PI, Michaelides M, Mackay DS, Wright GA, Devery S, Moore AT, Holder GE, Robson AG, Webster AR. Recessive mutations of the gene TRPM1 abrogate ON bipolar cell function and cause complete congenital stationary night blindness in humans. Am J Hum Genet. 2009 Nov;85(5):711-9. doi: 10.1016/j.ajhg.2009.10.003. Epub 2009 Oct 29. Citation on PubMed or Free article on PubMed CentralNakamura M, Sanuki R, Yasuma TR, Onishi A, Nishiguchi KM, Koike C, Kadowaki M, Kondo M, Miyake Y, Furukawa T. TRPM1 mutations are associated with the complete form of congenital stationary night blindness. Mol Vis. 2010 Mar 12;16:425-37. Citation on PubMed or Free article on PubMed CentralOMIM: TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY M, MEMBER 1van Genderen MM, Bijveld MM, Claassen YB, Florijn RJ, Pearring JN, Meire FM, McCall MA, Riemslag FC, Gregg RG, Bergen AA, Kamermans M. Mutations in TRPM1 are a common cause of complete congenital stationary night blindness. Am J Hum Genet. 2009 Nov;85(5):730-6. doi: 10.1016/j.ajhg.2009.10.012. Epub 2009 Nov 5. Citation on PubMed or Free article on PubMed Central \n",
            "OTUD7A\t\n",
            "GOLGA8N\t\n",
            "FMN1\t\n",
            "RYR3\t\n",
            "NUTM1\t\n",
            "GOLGA8B\t\n",
            "EIF2AK4\t   The EIF2AK4 gene provides instructions for making a protein that helps direct a cell's response to changes that could damage the cell. This protein is found in several tissues throughout the body, including blood vessel walls. The EIF2AK4 protein can turn on (activate) another protein called eIF2 alpha (eIF2), which helps control protein production. When cells are under stress, for example when the level of protein building blocks (amino acids) is too low, EIF2AK4 activates eIF2. When turned on, eIF2 stimulates processes that reduce protein production, which helps conserve amino acids. In addition, activated eIF2 can trigger production of certain proteins called transcription factors, which control gene activity. The transcription factors regulated by eIF2 control the activity of genes involved in processes that help reduce the stress on the cell.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Pulmonary veno-occlusive disease     Mutations in the EIF2AK4 gene are the primary genetic cause of a condition called pulmonary veno-occlusive disease (PVOD); at least 22 mutations in this gene have been found in affected individuals. In PVOD, excess fibrous tissue builds up in the small vessels in the lungs that carry oxygenated blood from the lungs to the heart (the pulmonary veins). This buildup narrows (occludes) the vessels and impairs blood flow. Because blood flow through the lungs is difficult, pressure rises in the vessels that carry blood that needs to be oxygenated to the lungs from the heart (pulmonary arteries). Increased pressure in these vessels is known as pulmonary arterial hypertension.The EIF2AK4 gene mutations involved in PVOD likely lead to a complete loss of functional protein. It is not known how absence of EIF2AK4 protein function leads to the pulmonary vein abnormalities characteristic of PVOD. More About This Health Condition      Pulmonary arterial hypertension     Genetics Home Reference provides information about pulmonary arterial hypertension. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the EIF2AK4 gene are the primary genetic cause of a condition called pulmonary veno-occlusive disease (PVOD); at least 22 mutations in this gene have been found in affected individuals. In PVOD, excess fibrous tissue builds up in the small vessels in the lungs that carry oxygenated blood from the lungs to the heart (the pulmonary veins). This buildup narrows (occludes) the vessels and impairs blood flow. Because blood flow through the lungs is difficult, pressure rises in the vessels that carry blood that needs to be oxygenated to the lungs from the heart (pulmonary arteries). Increased pressure in these vessels is known as pulmonary arterial hypertension.The EIF2AK4 gene mutations involved in PVOD likely lead to a complete loss of functional protein. It is not known how absence of EIF2AK4 protein function leads to the pulmonary vein abnormalities characteristic of PVOD. More About This Health Condition  Genetics Home Reference provides information about pulmonary arterial hypertension. More About This Health Condition    Cytogenetic Location: 15q15.1, which is the long (q) arm of chromosome 15 at position 15.1Molecular Location: base pairs 39,934,101 to 40,036,153 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        E2AK4_HUMANeukaryotic translation initiation factor 2-alpha kinase 4GCN2GCN2 eIF2alpha kinaseGCN2-like proteingeneral control nonderepressible 2KIAA1338PVOD2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Biosciences Database (2000): Mechanism of Translation Initiation in Eukaryotes      Tests Listed in the Genetic Testing Registry (1 link)     Tests of EIF2AK4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     EUKARYOTIC TRANSLATION INITIATION FACTOR 2-ALPHA KINASE 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Biosciences Database (2000): Mechanism of Translation Initiation in Eukaryotes  Tests of EIF2AK4  PubMed  EUKARYOTIC TRANSLATION INITIATION FACTOR 2-ALPHA KINASE 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Deval C, Chaveroux C, Maurin AC, Cherasse Y, Parry L, Carraro V, Milenkovic D, Ferrara M, Bruhat A, Jousse C, Fafournoux P. Amino acid limitation regulates the expression of genes involved in several specific biological processes through GCN2-dependent and GCN2-independent pathways. FEBS J. 2009 Feb;276(3):707-18. doi: 10.1111/j.1742-4658.2008.06818.x. Epub 2008 Dec 19. Citation on PubMedDonnelly N, Gorman AM, Gupta S, Samali A. The eIF2 kinases: their structures and functions. Cell Mol Life Sci. 2013 Oct;70(19):3493-511. doi: 10.1007/s00018-012-1252-6. Epub 2013 Jan 26. Review. Citation on PubMedOMIM: EUKARYOTIC TRANSLATION INITIATION FACTOR 2-ALPHA KINASE 4Eyries M, Montani D, Girerd B, Perret C, Leroy A, Lonjou C, Chelghoum N, Coulet F, Bonnet D, Dorfmller P, Fadel E, Sitbon O, Simonneau G, Tregout DA, Humbert M, Soubrier F. EIF2AK4 mutations cause pulmonary veno-occlusive disease, a recessive form of pulmonary hypertension. Nat Genet. 2014 Jan;46(1):65-9. doi: 10.1038/ng.2844. Epub 2013 Dec 1. Citation on PubMedMurgua JR, Serrano R. New functions of protein kinase Gcn2 in yeast and mammals. IUBMB Life. 2012 Dec;64(12):971-4. doi: 10.1002/iub.1090. Epub 2012 Nov 5. Review. Citation on PubMed \n",
            "KNL1\t   From NCBI Gene:The protein encoded by this gene is a component of the multiprotein assembly that is required for creation of kinetochore-microtubule attachments and chromosome segregation. The encoded protein functions as a scaffold for proteins that influence the spindle assembly checkpoint during the eukaryotic cell cycle and it interacts with at least five different kinetochore proteins and two checkpoint kinases. In adults, this gene is predominantly expressed in normal testes, various cancer cell lines and primary tumors from other tissues and is ubiquitously expressed in fetal tissues. This gene was originally identified as a fusion partner with the mixed-lineage leukemia (MLL) gene in t(11;15)(q23;q14). Mutations in this gene cause autosomal recessive primary microcephaly-4 (MCPH4). Alternative splicing results in multiple transcript variants encoding different isoforms. Additional splice variants have been described but their biological validity has not been confirmed. [provided by RefSeq, Jan 2013]From UniProt:Performs two crucial functions during mitosis: it is essential for spindle-assembly checkpoint signaling and for correct chromosome alignment. Required for attachment of the kinetochores to the spindle microtubules. Directly links BUB1 and BUB1B to kinetochores. Part of the MIS12 complex, which may be fundamental for kinetochore formation and proper chromosome segregation during mitosis. Acts in coordination with CENPK to recruit the NDC80 complex to the outer kinetochore.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autosomal recessive primary microcephalyFrom NCBI Gene:Primary autosomal recessive microcephaly 4From UniProt:Microcephaly 4, primary, autosomal recessive (MCPH4): A disease defined as a head circumference more than 3 standard deviations below the age-related mean. Brain weight is markedly reduced and the cerebral cortex is disproportionately small. Despite this marked reduction in size, the gyral pattern is relatively well preserved, with no major abnormality in cortical architecture. Affected individuals are mentally retarded. Primary microcephaly is further defined by the absence of other syndromic features or significant neurological deficits due to degenerative brain disorder. [MIM:604321]A chromosomal aberration involving KNL1 is associated with acute myeloblastic leukemia (AML). Translocation t(11;15)(q23;q14) with KMT2A. May give rise to a KMT2A-KNL1 fusion protein.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q15.1, which is the long (q) arm of chromosome 15 at position 15.1Molecular Location: base pairs 40,594,012 to 40,664,342 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AF15Q14CASC5CT29D40hKNL-1hSpc105MCPH4PPP1R55Spc7     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KNL1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KINETOCHORE SCAFFOLD 1OMIM: MICROCEPHALY 4, PRIMARY, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KNL1  OMIM: KINETOCHORE SCAFFOLD 1OMIM: MICROCEPHALY 4, PRIMARY, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NDUFAF1\t   From NCBI Gene:This gene encodes a complex I assembly factor protein. Complex I (NADH-ubiquinone oxidoreductase) catalyzes the transfer of electrons from NADH to ubiquinone (coenzyme Q) in the first step of the mitochondrial respiratory chain, resulting in the translocation of protons across the inner mitochondrial membrane. The encoded protein is required for assembly of complex I, and mutations in this gene are a cause of mitochondrial complex I deficiency. Alternatively spliced transcript variants have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 19. [provided by RefSeq, Dec 2011]From UniProt:Chaperone protein involved in early stages of the assembly of mitochondrial NADH:ubiquinone oxidoreductase complex (complex I).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Mitochondrial complex I deficiencyFrom NCBI Gene:MITOCHONDRIAL COMPLEX I DEFICIENCY, NUCLEAR TYPE 11From UniProt:Mitochondrial complex I deficiency, nuclear type 11 (MC1DN11): A form of mitochondrial complex I deficiency, the most common biochemical signature of mitochondrial disorders, a group of highly heterogeneous conditions characterized by defective oxidative phosphorylation, which collectively affects 1 in 5-10000 live births. Clinical disorders have variable severity, ranging from lethal neonatal disease to adult-onset neurodegenerative disorders. Phenotypes include macrocephaly with progressive leukodystrophy, non-specific encephalopathy, cardiomyopathy, myopathy, liver disease, Leigh syndrome, Leber hereditary optic neuropathy, and some forms of Parkinson disease. MC1DN11 transmission pattern is consistent with autosomal recessive inheritance. [MIM:618234]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q15.1, which is the long (q) arm of chromosome 15 at position 15.1Molecular Location: base pairs 41,387,349 to 41,402,501 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CGI-65CGI65CIA30MC1DN11     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NDUFAF1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MITOCHONDRIAL COMPLEX I DEFICIENCY, NUCLEAR TYPE 11OMIM: NADH DEHYDROGENASE (UBIQUINONE) COMPLEX I, ASSEMBLY FACTOR 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NDUFAF1  OMIM: MITOCHONDRIAL COMPLEX I DEFICIENCY, NUCLEAR TYPE 11OMIM: NADH DEHYDROGENASE (UBIQUINONE) COMPLEX I, ASSEMBLY FACTOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LTK\t   From NCBI Gene:The protein encoded by this gene is a member of the ros/insulin receptor family of tyrosine kinases. Tyrosine-specific phosphorylation of proteins is a key to the control of diverse pathways leading to cell growth and differentiation. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2008]From UniProt:Receptor with a tyrosine-protein kinase activity. The exact function of this protein is not known. Studies with chimeric proteins (replacing its extracellular region with that of several known growth factor receptors, such as EGFR and CSFIR) demonstrate its ability to promote growth and specifically neurite outgrowth, and cell survival. Signaling appears to involve the PI3 kinase pathway. Involved in regulation of the secretory pathway involving endoplasmic reticulum (ER) export sites (ERESs) and ER to Golgi transport.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Systemic lupus erythematosusFrom UniProt:Genetic variations in LTK that cause up-regulation of the PI3K pathway may possibly contribute to susceptibility to abnormal proliferation of self-reactive B-cells and, therefore, to systemic lupus erythematosus (SLE). SLE is a chronic, inflammatory and often febrile multisystemic disorder of connective tissue, thought to represent a failure of the regulatory mechanisms of the autoimmune system.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q15.1, which is the long (q) arm of chromosome 15 at position 15.1Molecular Location: base pairs 41,503,639 to 41,513,887 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        TYK1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LTK      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: LEUKOCYTE TYROSINE KINASE      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LTK  OMIM: LEUKOCYTE TYROSINE KINASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RPAP1\t\n",
            "MGA\t\n",
            "SPTBN5\t\n",
            "PLA2G4E\t\n",
            "MAP1A\t\n",
            "CATSPER2\t   The CATSPER2 gene provides instructions for producing a protein that is found in the tail (flagellum) of sperm cells. The CATSPER2 protein plays a role in sperm cell movement (motility) and is required for sperm cells to push through the outside membrane of the egg cell during fertilization. The CATSPER2 protein is embedded in the membrane of the sperm tail and is necessary in order for positively charged calcium atoms (calcium cations) to enter the cell. Calcium cations are needed for a type of sperm motility called hyperactivation. Hyperactivation is characterized by vigorous movements of the sperm tail, which are required for the sperm to push through the membrane of the egg cell during fertilization.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Sensorineural deafness and male infertility     The symptoms of sensorineural deafness and male infertility are caused by a deletion of genetic material on the long (q) arm of chromosome 15. The chromosomal region that is typically deleted contains multiple genes, including the CATSPER2 gene. People with this condition have the deletion in both copies of chromosome 15 in each cell. As a result of the deletion, affected individuals are missing both copies of the CATSPER2 gene, and no CATSPER2 protein is produced. A lack of CATSPER2 protein impairs calcium entry into the sperm cell, which decreases motility and prevents hyperactivation. Lack of hyperactivation results in sperm that are unable to push through the membrane of the egg cell and achieve fertilization. These sperm abnormalities are the cause of infertility in affected males. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      The symptoms of sensorineural deafness and male infertility are caused by a deletion of genetic material on the long (q) arm of chromosome 15. The chromosomal region that is typically deleted contains multiple genes, including the CATSPER2 gene. People with this condition have the deletion in both copies of chromosome 15 in each cell. As a result of the deletion, affected individuals are missing both copies of the CATSPER2 gene, and no CATSPER2 protein is produced. A lack of CATSPER2 protein impairs calcium entry into the sperm cell, which decreases motility and prevents hyperactivation. Lack of hyperactivation results in sperm that are unable to push through the membrane of the egg cell and achieve fertilization. These sperm abnormalities are the cause of infertility in affected males. More About This Health Condition    Cytogenetic Location: 15q15.3, which is the long (q) arm of chromosome 15 at position 15.3Molecular Location: base pairs 43,630,562 to 43,648,884 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        cation channel sperm-associated protein 2cation channel, sperm associated 2CTSR2_HUMANsperm ion channel     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Developmental Biology (sixth edition, 2000): Action at a Distance: Mammalian Gametes      Clinical Information from GeneReviews (1 link)     CATSPER-Related Male Infertility      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CATSPER2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CATION CHANNEL, SPERM-ASSOCIATED, 2      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): Action at a Distance: Mammalian Gametes  CATSPER-Related Male Infertility  Tests of CATSPER2  PubMed  CATION CHANNEL, SPERM-ASSOCIATED, 2  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Avidan N, Tamary H, Dgany O, Cattan D, Pariente A, Thulliez M, Borot N, Moati L, Barthelme A, Shalmon L, Krasnov T, Ben-Asher E, Olender T, Khen M, Yaniv I, Zaizov R, Shalev H, Delaunay J, Fellous M, Lancet D, Beckmann JS. CATSPER2, a human autosomal nonsyndromic male infertility gene. Eur J Hum Genet. 2003 Jul;11(7):497-502. Citation on PubMedOMIM: CATION CHANNEL, SPERM-ASSOCIATED, 2Quill TA, Sugden SA, Rossi KL, Doolittle LK, Hammer RE, Garbers DL. Hyperactivated sperm motility driven by CatSper2 is required for fertilization. Proc Natl Acad Sci U S A. 2003 Dec 9;100(25):14869-74. Epub 2003 Dec 1. Citation on PubMed or Free article on PubMed CentralZhang Y, Malekpour M, Al-Madani N, Kahrizi K, Zanganeh M, Lohr NJ, Mohseni M, Mojahedi F, Daneshi A, Najmabadi H, Smith RJ. Sensorineural deafness and male infertility: a contiguous gene deletion syndrome. J Med Genet. 2007 Apr;44(4):233-40. Epub 2006 Nov 10. Erratum in: J Med Genet. 2007 Aug;44(8):544. Lohr, Naomi J [added]. Citation on PubMed or Free article on PubMed Central \n",
            "SORD\t\n",
            "SPPL2A\t\n",
            "MYO5C\t\n",
            "FAM214A\t\n",
            "WDR72\t   From NCBI Gene:This gene encodes a protein with eight WD-40 repeats. Mutations in this gene have been associated with amelogenesis imperfecta hypomaturation type 2A3. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2013]From UniProt:Plays a major role in formation of tooth enamel (PubMed:19853237, PubMed:25008349). Specifically required during the maturation phase of amelogenesis for normal formation of the enamel matrix and clearance of enamel proteins. May be involved in localization of the calcium transporter SLC24A4 to the ameloblast cell membrane.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Amelogenesis imperfectaFrom NCBI Gene:Amelogenesis imperfecta, hypomaturation type, IIA3From UniProt:Amelogenesis imperfecta, hypomaturation type, 2A3 (AI2A3): A defect of enamel formation. The disorder involves both primary and secondary dentitions. The teeth have a shiny agar jelly appearance and the enamel is softer than normal. Brown pigment is present in middle layers of enamel. [MIM:613211]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q21.3, which is the long (q) arm of chromosome 15 at position 21.3Molecular Location: base pairs 53,513,741 to 53,762,878 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AI2A3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of WDR72      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: AMELOGENESIS IMPERFECTA, HYPOMATURATION TYPE, IIA3OMIM: WD REPEAT-CONTAINING PROTEIN 72      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of WDR72  OMIM: AMELOGENESIS IMPERFECTA, HYPOMATURATION TYPE, IIA3OMIM: WD REPEAT-CONTAINING PROTEIN 72  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLTM\t\n",
            "LDHAL6B\t\n",
            "VPS13C\t   From NCBI Gene:This gene encodes a member of the vacuolar protein sorting-associated 13 gene family. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Oct 2010]From UniProt:Necessary for proper mitochondrial function and maintenance of mitochondrial transmembrane potential. Involved in the regulation of PINK1/PRKN-mediated mitophagy in response to mitochondrial depolarization.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Parkinson disease 23, autosomal recessive early-onsetFrom UniProt:Parkinson disease 23, autosomal recessive, early onset (PARK23): An autosomal recessive, early-onset form of Parkinson disease, a complex neurodegenerative disorder characterized by bradykinesia, resting tremor, muscular rigidity and postural instability, as well as by a clinically significant response to treatment with levodopa. The pathology involves the loss of dopaminergic neurons in the substantia nigra and the presence of Lewy bodies (intraneuronal accumulations of aggregated proteins), in surviving neurons in various areas of the brain. [MIM:616840]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q22.2, which is the long (q) arm of chromosome 15 at position 22.2Molecular Location: base pairs 61,852,389 to 62,060,465 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PARK23     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of VPS13C      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PARKINSON DISEASE 23, AUTOSOMAL RECESSIVE EARLY-ONSETOMIM: VACUOLAR PROTEIN SORTING 13, YEAST, HOMOLOG OF, C      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of VPS13C  OMIM: PARKINSON DISEASE 23, AUTOSOMAL RECESSIVE EARLY-ONSETOMIM: VACUOLAR PROTEIN SORTING 13, YEAST, HOMOLOG OF, C  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FBXL22\t\n",
            "HERC1\t   From NCBI Gene:This gen encodes a member of the HERC protein family. This protein stimulates guanine nucleotide exchange on ARF1 and Rab proteins. This protein may be involved in membrane transport processes. [provided by RefSeq, Mar 2012]From UniProt:Involved in membrane trafficking via some guanine nucleotide exchange factor (GEF) activity and its ability to bind clathrin. Acts as a GEF for Arf and Rab, by exchanging bound GDP for free GTP. Binds phosphatidylinositol 4,5-bisphosphate, which is required for GEF activity. May also act as a E3 ubiquitin-protein ligase which accepts ubiquitin from an E2 ubiquitin-conjugating enzyme in the form of a thioester and then directly transfers the ubiquitin to targeted substrates.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Macrocephaly, dysmorphic facies, and psychomotor retardationFrom UniProt:Macrocephaly, dysmorphic facies, and psychomotor retardation (MDFPMR): An autosomal recessive syndrome characterized by large head and somatic overgrowth, intellectual disability, and facial dysmorphism. Seizures, hypotonia and ataxic gait are observed in some patients. [MIM:617011]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q22.31, which is the long (q) arm of chromosome 15 at position 22.31Molecular Location: base pairs 63,608,618 to 63,833,948 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MDFPMRp532p619     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HERC1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HECT DOMAIN AND RCC1-LIKE DOMAIN 1OMIM: MACROCEPHALY, DYSMORPHIC FACIES, AND PSYCHOMOTOR RETARDATION      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HERC1  OMIM: HECT DOMAIN AND RCC1-LIKE DOMAIN 1OMIM: MACROCEPHALY, DYSMORPHIC FACIES, AND PSYCHOMOTOR RETARDATION  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KBTBD13\t   From NCBI Gene:The gene belongs to a family of genes encoding proteins containing a BTB domain and several kelch repeats. The BTB domain functions as a protein-protein interaction module, which includes an ability to self-associate or to interact with non-BTB domain-containing proteins. The kelch motif typically occurs in groups of five to seven repeats, and has been found in proteins with diverse functions. Known functions of these family members include transcription regulation, ion channel tetramerization and gating, protein ubiquitination or degradation, and cytoskeleton regulation. The exact function of this family member has yet to be determined. [provided by RefSeq, Jun 2010]From UniProt:Substrate-specific adapter of a BCR (BTB-CUL3-RBX1) E3 ubiquitin ligase complex.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nemaline myopathyFrom NCBI Gene:Nemaline myopathy 6From UniProt:Nemaline myopathy 6 (NEM6): A form of nemaline myopathy characterized by childhood onset of slowly progressive proximal muscle weakness, exercise intolerance, and slow movements with stiff muscles. Patients are unable to run or correct themselves from falling over. [MIM:609273]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q22.31, which is the long (q) arm of chromosome 15 at position 22.31Molecular Location: base pairs 65,076,816 to 65,079,938 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HCG1645727NEM6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KBTBD13      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KELCH REPEAT- AND BTB/POZ DOMAIN-CONTAINING PROTEIN 13OMIM: NEMALINE MYOPATHY 6      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KBTBD13  OMIM: KELCH REPEAT- AND BTB/POZ DOMAIN-CONTAINING PROTEIN 13OMIM: NEMALINE MYOPATHY 6  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IGDCC4\t\n",
            "NOX5\t\n",
            "LRRC49\t\n",
            "NR2E3\t   From NCBI Gene:This protein is part of a large family of nuclear receptor transcription factors involved in signaling pathways. Nuclear receptors have been shown to regulate pathways involved in embryonic development, as well as in maintenance of proper cell function in adults. Members of this family are characterized by discrete domains that function in DNA and ligand binding. This gene encodes a retinal nuclear receptor that is a ligand-dependent transcription factor. Defects in this gene are a cause of enhanced S cone syndrome. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]From UniProt:Orphan nuclear receptor of retinal photoreceptor cells. Transcriptional factor that is an activator of rod development and repressor of cone development. Binds the promoter region of a number of rod- and cone-specific genes, including rhodopsin, M- and S-opsin and rod-specific phosphodiesterase beta subunit. Enhances rhodopsin expression. Represses M- and S-cone opsin expression.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Retinitis pigmentosaFrom NCBI Gene:Enhanced s-cone syndromeRetinitis pigmentosa 37From UniProt:Retinitis pigmentosa 37 (RP37): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:611131]Enhanced S cone syndrome (ESCS): Autosomal recessive retinopathy in which patients have increased sensitivity to blue light; perception of blue light is mediated by what is normally the least populous cone photoreceptor subtype, the S (short wavelength, blue) cones. ESCS is also associated with visual loss, with night blindness occurring from early in life, varying degrees of L (long, red)- and M (middle, green)-cone vision, and retinal degeneration. [MIM:268100]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q23, which is the long (q) arm of chromosome 15 at position 23Molecular Location: base pairs 71,810,548 to 71,818,259 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ESCSPNRrd7RNRRP37     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NR2E3      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ENHANCED S-CONE SYNDROMEOMIM: NUCLEAR RECEPTOR SUBFAMILY 2, GROUP E, MEMBER 3OMIM: RETINITIS PIGMENTOSA 37      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NR2E3  OMIM: ENHANCED S-CONE SYNDROMEOMIM: NUCLEAR RECEPTOR SUBFAMILY 2, GROUP E, MEMBER 3OMIM: RETINITIS PIGMENTOSA 37  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CD276\t\n",
            "GOLGA6A\t\n",
            "CCDC33\t\n",
            "ULK3\t\n",
            "SCAPER\t\n",
            "CHRNA5\t   From NCBI Gene:The protein encoded by this gene is a nicotinic acetylcholine receptor subunit and a member of a superfamily of ligand-gated ion channels that mediate fast signal transmission at synapses. These receptors are thought to be heteropentamers composed of separate but similar subunits. Defects in this gene have been linked to susceptibility to lung cancer type 2 (LNCR2).[provided by RefSeq, Jun 2010]From UniProt:After binding acetylcholine, the AChR responds by an extensive change in conformation that affects all subunits and leads to opening of an ion-conducting channel across the plasma membrane.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Smoking as a quantitative trait locus 3     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q25.1, which is the long (q) arm of chromosome 15 at position 25.1Molecular Location: base pairs 78,565,520 to 78,595,269 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LNCR2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CHRNA5      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CHOLINERGIC RECEPTOR, NEURONAL NICOTINIC, ALPHA POLYPEPTIDE 5OMIM: SMOKING AS A QUANTITATIVE TRAIT LOCUS 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CHRNA5  OMIM: CHOLINERGIC RECEPTOR, NEURONAL NICOTINIC, ALPHA POLYPEPTIDE 5OMIM: SMOKING AS A QUANTITATIVE TRAIT LOCUS 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ADAMTS7\t\n",
            "IL16\t   From NCBI Gene:The protein encoded by this gene is a pleiotropic cytokine that functions as a chemoattractant, a modulator of T cell activation, and an inhibitor of HIV replication. The signaling process of this cytokine is mediated by CD4. The product of this gene undergoes proteolytic processing, which is found to yield two functional proteins. The cytokine function is exclusively attributed to the secreted C-terminal peptide, while the N-terminal product may play a role in cell cycle control. Caspase 3 is reported to be involved in the proteolytic processing of this protein. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]From UniProt:Isoform 1 may act as a scaffolding protein that anchors ion channels in the membrane.Interleukin-16 stimulates a migratory response in CD4+ lymphocytes, monocytes, and eosinophils. Primes CD4+ T-cells for IL-2 and IL-15 responsiveness. Also induces T-lymphocyte expression of interleukin 2 receptor. Ligand for CD4.Isoform 3 is involved in cell cycle progression in T-cells. Appears to be involved in transcriptional regulation of SKP2 and is probably part of a transcriptional repression complex on the core promoter of the SKP2 gene. May act as a scaffold for GABPB1 (the DNA-binding subunit the GABP transcription factor complex) and HDAC3 thus maintaining transcriptional repression and blocking cell cycle progression in resting T-cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Alopecia areata     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q25.1, which is the long (q) arm of chromosome 15 at position 25.1Molecular Location: base pairs 81,196,878 to 81,312,763 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LCFNIL16prIL-16PRIL16     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of IL16      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: INTERLEUKIN 16      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of IL16  OMIM: INTERLEUKIN 16  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TM6SF1\t\n",
            "ADAMTSL3\t\n",
            "WDR73\t   From NCBI Gene:The protein encoded by this gene is thought to contain multiple WD40 repeats. WD40 repeats are motifs that contain 40-60 amino acids, and usually end with Trp-Asp (WD). This protein is found in the cytoplasm during interphase, but accumulates at the spindle poles and astral microtubules during mitosis. Reduced expression of this gene results in abnormalities in the size and morphology of the nucleus. Mutations in this gene have been associated with Galloway-Mowat syndrome PMID: 25466283), which is a rare autosomal recessive disorder that affects both the central nervous system and kidneys. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2015]From UniProt:May play a role in the regulation of microtubule organization and dynamics (PubMed:25466283).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Galloway-Mowat syndrome 1From UniProt:Galloway-Mowat syndrome 1 (GAMOS1): A form of Galloway-Mowat syndrome, a severe renal-neurological disease characterized by early-onset nephrotic syndrome associated with microcephaly, central nervous system abnormalities, developmental delays, and a propensity for seizures. Brain anomalies include gyration defects ranging from lissencephaly to pachygyria and polymicrogyria, and cerebellar hypoplasia. Most patients show facial dysmorphism characterized by a small, narrow forehead, large/floppy ears, deep-set eyes, hypertelorism and micrognathia. Additional variable features are visual impairment and arachnodactyly. Patients may die in early childhood. GAMOS1 inheritance is autosomal recessive. [MIM:251300]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q25.2, which is the long (q) arm of chromosome 15 at position 25.2Molecular Location: base pairs 84,641,495 to 84,654,343 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GAMOSGAMOS1HSPC264     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of WDR73      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GALLOWAY-MOWAT SYNDROME 1OMIM: WD REPEAT-CONTAINING PROTEIN 73      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of WDR73  OMIM: GALLOWAY-MOWAT SYNDROME 1OMIM: WD REPEAT-CONTAINING PROTEIN 73  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ALPK3\t\n",
            "SLC28A1\t\n",
            "GOLGA6L3\t\n",
            "AKAP13\t\n",
            "AGBL1\t   From NCBI Gene:Polyglutamylation is a reversible posttranslational modification catalyzed by polyglutamylases that results in the addition of glutamate side chains on the modified protein. This gene encodes a glutamate decarboxylase that catalyzes the deglutamylation of polyglutamylated proteins. Mutations in this gene result in dominant late-onset Fuchs corneal dystrophy. [provided by RefSeq, Nov 2013]From UniProt:Metallocarboxypeptidase that mediates deglutamylation of target proteins. Catalyzes the deglutamylation of polyglutamate side chains generated by post-translational polyglutamylation in proteins such as tubulins. Also removes gene-encoded polyglutamates from the carboxy-terminus of target proteins such as MYLK. Acts as a long-chain deglutamylase and specifically shortens long polyglutamate chains, while it is not able to remove the branching point glutamate, a process catalyzed by AGBL5/CCP5.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Fuchs endothelial dystrophyFrom NCBI Gene:Corneal dystrophy, Fuchs endothelial, 8From UniProt:Corneal dystrophy, Fuchs endothelial, 8 (FECD8): A corneal disease caused by loss of endothelium of the central cornea. It is characterized by focal wart-like guttata that arise from Descemet membrane and develop in the central cornea, epithelial blisters, reduced vision and pain. Descemet membrane is thickened by abnormal collagenous deposition. [MIM:615523]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q25.3, which is the long (q) arm of chromosome 15 at position 25.3Molecular Location: base pairs 86,079,639 to 87,031,476 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CCP4FECD8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGBL1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ATP/GTP-BINDING PROTEIN-LIKE 1OMIM: CORNEAL DYSTROPHY, FUCHS ENDOTHELIAL, 8      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AGBL1  OMIM: ATP/GTP-BINDING PROTEIN-LIKE 1OMIM: CORNEAL DYSTROPHY, FUCHS ENDOTHELIAL, 8  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ACAN\t   The ACAN gene provides instructions for making the aggrecan protein. Aggrecan is a type of protein known as a proteoglycan, which means it has several sugar molecules attached to it. It is the most abundant proteoglycan in cartilage, a tough, flexible tissue that makes up much of the skeleton during early development. Most cartilage is later converted to bone (a process called ossification), except for the cartilage that continues to cover and protect the ends of bones and is present in the nose, airways, and external ears.Aggrecan attaches to the other components of cartilage, organizing the network of molecules that gives cartilage its strength. These interactions occur at a specific region of the aggrecan protein called the C-type lectin domain (CLD). Because of the attached sugars, aggrecan attracts water molecules and gives cartilage its gel-like structure. This feature enables the cartilage to resist compression, protecting bones and joints. Although its role is unclear, aggrecan affects bone development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Familial osteochondritis dissecans     At least one mutation in the ACAN gene has been found to cause familial osteochondritis dissecans. This condition is characterized by areas of bone damage (lesions) caused by the detachment of cartilage and some of the underlying bone from the end of the bone at a joint. People with familial osteochondritis dissecans have multiple lesions that affect the knees, elbows, hips, or ankles. Other common features are short stature and early development of a painful joint disorder called osteoarthritis.The ACAN gene mutation associated with this condition changes a single protein building block (amino acid) in the CLD of the aggrecan protein. Specifically, the amino acid valine is replaced by the amino acid methionine at protein position 2303 (written as Val2303Met or V2303M). The abnormal aggrecan protein is unable to attach to other components of cartilage. As a result, the cartilage is disorganized and weak. It is unclear how the abnormal cartilage is involved in the development of the lesions and osteoarthritis characteristic of familial osteochondritis dissecans. Researchers have suggested that a disorganized cartilage network in growing bones impairs their growth, leading to short stature. More About This Health Condition      Intervertebral disc disease     Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition      Other disorders     Two other conditions associated with short stature, called spondyloepimetaphyseal dysplasia, aggrecan type and spondyloepiphyseal dysplasia, Kimberley type, are caused by mutations in the ACAN gene. People with spondyloepimetaphyseal dysplasia, aggrecan type have extremely short stature, short fingers and toes, and distinctive facial features. This condition is caused by a mutation that changes the amino acid at position 2267 in the aggrecan protein from aspartic acid to asparagine (written as Asp2267Asn or D2267N). The amino acid change, which occurs in the CLD, alters aggrecan's interaction with at least one component of the cartilage network. It is unclear how this change leads to the particular signs and symptoms of spondyloepimetaphyseal dysplasia, aggrecan type.Spondyloepiphyseal dysplasia, Kimberley type is characterized by short stature and early development of osteoarthritis, particularly in the knees, ankles, and hips. This condition is caused by a mutation in which a single DNA building block is inserted into the ACAN gene, which could disrupt the gene's instructions and lead to the production of an abnormally short aggrecan protein that is missing the CLD. It is unknown if the abnormal protein is produced or what effects it might have. It is unclear what role this gene mutation plays in the development of the specific features of spondyloepiphyseal dysplasia, Kimberley type.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least one mutation in the ACAN gene has been found to cause familial osteochondritis dissecans. This condition is characterized by areas of bone damage (lesions) caused by the detachment of cartilage and some of the underlying bone from the end of the bone at a joint. People with familial osteochondritis dissecans have multiple lesions that affect the knees, elbows, hips, or ankles. Other common features are short stature and early development of a painful joint disorder called osteoarthritis.The ACAN gene mutation associated with this condition changes a single protein building block (amino acid) in the CLD of the aggrecan protein. Specifically, the amino acid valine is replaced by the amino acid methionine at protein position 2303 (written as Val2303Met or V2303M). The abnormal aggrecan protein is unable to attach to other components of cartilage. As a result, the cartilage is disorganized and weak. It is unclear how the abnormal cartilage is involved in the development of the lesions and osteoarthritis characteristic of familial osteochondritis dissecans. Researchers have suggested that a disorganized cartilage network in growing bones impairs their growth, leading to short stature. More About This Health Condition  Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition  Two other conditions associated with short stature, called spondyloepimetaphyseal dysplasia, aggrecan type and spondyloepiphyseal dysplasia, Kimberley type, are caused by mutations in the ACAN gene. People with spondyloepimetaphyseal dysplasia, aggrecan type have extremely short stature, short fingers and toes, and distinctive facial features. This condition is caused by a mutation that changes the amino acid at position 2267 in the aggrecan protein from aspartic acid to asparagine (written as Asp2267Asn or D2267N). The amino acid change, which occurs in the CLD, alters aggrecan's interaction with at least one component of the cartilage network. It is unclear how this change leads to the particular signs and symptoms of spondyloepimetaphyseal dysplasia, aggrecan type.Spondyloepiphyseal dysplasia, Kimberley type is characterized by short stature and early development of osteoarthritis, particularly in the knees, ankles, and hips. This condition is caused by a mutation in which a single DNA building block is inserted into the ACAN gene, which could disrupt the gene's instructions and lead to the production of an abnormally short aggrecan protein that is missing the CLD. It is unknown if the abnormal protein is produced or what effects it might have. It is unclear what role this gene mutation plays in the development of the specific features of spondyloepiphyseal dysplasia, Kimberley type.    Cytogenetic Location: 15q26.1, which is the long (q) arm of chromosome 15 at position 26.1Molecular Location: base pairs 88,803,442 to 88,875,354 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AGC1AGCANaggrecan core proteincartilage-specific proteoglycan core proteinchondroitin sulfate proteoglycan core protein 1CSPG1CSPGCPlarge aggregating proteoglycanMSK16SEDK     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (fourth Edition, 2002): Noncollagen Components of the Extracellular Matrix      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ACAN      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (3 links)     AGGRECANSPONDYLOEPIMETAPHYSEAL DYSPLASIA, AGGRECAN TYPESPONDYLOEPIPHYSEAL DYSPLASIA, KIMBERLEY TYPE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth Edition, 2002): Noncollagen Components of the Extracellular Matrix  Tests of ACAN  PubMed  AGGRECANSPONDYLOEPIMETAPHYSEAL DYSPLASIA, AGGRECAN TYPESPONDYLOEPIPHYSEAL DYSPLASIA, KIMBERLEY TYPE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: AGGRECANAspberg A, Adam S, Kostka G, Timpl R, Heinegrd D. Fibulin-1 is a ligand for the C-type lectin domains of aggrecan and versican. J Biol Chem. 1999 Jul 16;274(29):20444-9. Citation on PubMedGentili C, Cancedda R. Cartilage and bone extracellular matrix. Curr Pharm Des. 2009;15(12):1334-48. Review. Citation on PubMedGleghorn L, Ramesar R, Beighton P, Wallis G. A mutation in the variable repeat region of the aggrecan gene (AGC1) causes a form of spondyloepiphyseal dysplasia associated with severe, premature osteoarthritis. Am J Hum Genet. 2005 Sep;77(3):484-90. Epub 2005 Jul 22. Citation on PubMed or Free article on PubMed CentralOlin AI, Mrgelin M, Sasaki T, Timpl R, Heinegrd D, Aspberg A. The proteoglycans aggrecan and Versican form networks with fibulin-2 through their lectin domain binding. J Biol Chem. 2001 Jan 12;276(2):1253-61. Citation on PubMedStattin EL, Wiklund F, Lindblom K, Onnerfjord P, Jonsson BA, Tegner Y, Sasaki T, Struglics A, Lohmander S, Dahl N, Heinegrd D, Aspberg A. A missense mutation in the aggrecan C-type lectin domain disrupts extracellular matrix interactions and causes dominant familial osteochondritis dissecans. Am J Hum Genet. 2010 Feb 12;86(2):126-37. doi: 10.1016/j.ajhg.2009.12.018. Epub 2010 Feb 4. Citation on PubMed or Free article on PubMed CentralTompson SW, Merriman B, Funari VA, Fresquet M, Lachman RS, Rimoin DL, Nelson SF, Briggs MD, Cohn DH, Krakow D. A recessive skeletal dysplasia, SEMD aggrecan type, results from a missense mutation affecting the C-type lectin domain of aggrecan. Am J Hum Genet. 2009 Jan;84(1):72-9. doi: 10.1016/j.ajhg.2008.12.001. Epub 2008 Dec 24. Citation on PubMed or Free article on PubMed Central \n",
            "MFGE8\t\n",
            "FANCI\t   From NCBI Gene:The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group I. Alternative splicing results in two transcript variants encoding different isoforms. [provided by RefSeq, Jul 2008]From UniProt:Plays an essential role in the repair of DNA double-strand breaks by homologous recombination and in the repair of interstrand DNA cross-links (ICLs) by promoting FANCD2 monoubiquitination by FANCL and participating in recruitment to DNA repair sites. Required for maintenance of chromosomal stability. Specifically binds branched DNA: binds both single-stranded DNA (ssDNA) and double-stranded DNA (dsDNA). Participates in S phase and G2 phase checkpoint activation upon DNA damage.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Fanconi anemiaFrom NCBI Gene:Fanconi anemia, complementation group IFrom UniProt:Fanconi anemia complementation group I (FANCI): A disorder affecting all bone marrow elements and resulting in anemia, leukopenia and thrombopenia. It is associated with cardiac, renal and limb malformations, dermal pigmentary changes, and a predisposition to the development of malignancies. At the cellular level it is associated with hypersensitivity to DNA-damaging agents, chromosomal instability (increased chromosome breakage) and defective DNA repair. [MIM:609053]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 15q26.1, which is the long (q) arm of chromosome 15 at position 26.1Molecular Location: base pairs 89,243,954 to 89,317,132 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA1794     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FANCI      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FANCI GENEOMIM: FANCONI ANEMIA, COMPLEMENTATION GROUP I      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FANCI  OMIM: FANCI GENEOMIM: FANCONI ANEMIA, COMPLEMENTATION GROUP I  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "POLG\t   The POLG gene provides instructions for making the active piece, called the alpha subunit, of a protein called polymerase gamma (pol ). To be most effective, the alpha subunit attaches to two copies of another protein called the beta subunit to form pol . Pol  is a DNA polymerase, which is a type of enzyme that \"reads\" sequences of DNA and uses them as templates to produce new DNA. These enzymes are important for copying (replicating) cells' genetic material. DNA polymerases also play critical roles in DNA repair.Pol  functions in mitochondria. Mitochondria are structures within cells in which a process called oxidative phosphorylation converts the energy from food into a form that cells can use. Mitochondria each contain a small amount of DNA, known as mitochondrial DNA (mtDNA), which is essential for the normal function of these structures. Pol  is the only DNA polymerase that is active in mitochondria and that can replicate mtDNA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alpers-Huttenlocher syndrome     There are many mutations in the POLG gene that cause Alpers-Huttenlocher syndrome. Alpers-Huttenlocher syndrome is part of a group of conditions called POLG-related disorders that have overlapping signs and symptoms affecting muscle-, nerve-, and brain-related functions. Alpers-Huttenlocher syndrome is characterized by seizures, loss of mental and movement abilities (psychomotor regression), and liver disease. The liver disease in Alpers-Huttenlocher syndrome can be brought on or made worse by valproic acid, a common treatment for seizures.Most POLG gene mutations change single protein building blocks (amino acids) in the alpha subunit of pol . The mutations can have several effects on the function of pol . The alpha subunit may lose the ability to attach to the beta subunits to form pol . Alternately, mutated pol  may be unable to bind the existing mtDNA strand to use as a template. Or, it may have a reduced ability to attract the DNA building blocks (nucleotides) that it uses to form new DNA. These effects impair DNA synthesis and may lead to insertion of the wrong nucleotide and decreased ability to fix the error.The most common POLG gene mutation in Alpers-Huttenlocher syndrome replaces the amino acid alanine with the amino acid threonine at position 467 (written as Ala467Thr or A467T). This mutation blocks the ability of the alpha subunit to attach to the beta subunits and reduces pol 's ability to synthesize DNA. The Ala467Thr mutation is also common in other POLG-related disorders. The different conditions may be determined, in part, by the mutation in the other copy of POLG, but there are still some mutation combinations that can cause more than one of the disorders. It is unclear how the same mutation can lead to different conditions.Although the mechanism is unknown, many people with Alpers-Huttenlocher syndrome have fewer copies of mtDNA (mtDNA depletion). This abnormality is seen only in the tissues affected by the disease. MtDNA depletion leads to impaired oxidative phosphorylation and a decrease in cellular energy. These impairments affect tissues whose cells do not divide continually, such as brain, muscle, and liver. These tissues are most affected because they are more dependent on oxidative phosphorylation for energy, and impaired cells in these tissues are not generally replaced by new cells. The lack of energy supplies in these tissues could account for the signs and symptoms of Alpers-Huttenlocher syndrome. More About This Health Condition      Ataxia neuropathy spectrum     Another condition caused by mutations in the POLG gene is ataxia neuropathy spectrum, a POLG-related disorder that is characterized by problems with coordination and balance (ataxia) and disturbances in nerve function (neuropathy). The conditions previously named mitochondrial recessive ataxia syndrome (MIRAS) and sensory ataxia neuropathy dysarthria and ophthalmoplegia (SANDO) are now included in the ataxia neuropathy spectrum.As mentioned above, most mutations in the POLG gene change single amino acids in the alpha subunit of pol , and these mutations can have several effects on the function of pol . As a result, pol  has a reduced ability to replicate mtDNA. The most common POLG gene mutation in ataxia neuropathy spectrum is the same as that in Alpers-Huttenlocher syndrome, Ala467Thr. It is unclear how the same mutation can lead to different disorders.As in other POLG-related disorders, people with ataxia neuropathy spectrum typically have mtDNA depletion in the tissues affected by the condition, such as the brain. MtDNA depletion decreases the amount of energy available to the cell due to reduced oxidative phosphorylation, which may account for the signs and symptoms of ataxia neuropathy spectrum. More About This Health Condition      Childhood myocerebrohepatopathy spectrum     Childhood myocerebrohepatopathy syndrome (MCHS) is also caused by mutations in the POLG gene. MCHS is a POLG-related disorder that affects the muscles (myo-), brain (cerebro-), and liver (hepato-).Many mutations in the POLG gene can cause MCHS. Most of these mutations change single amino acids in the alpha subunit of pol . These mutations reduce the activity of pol , decreasing mtDNA replication.As in other POLG-related disorders, people with MCHS typically have mtDNA depletion in muscle, brain, or liver tissue. MtDNA depletion impairs oxidative phosphorylation in these tissues and decreases the energy available to the cells, which may cause the signs and symptoms of MCHS. More About This Health Condition      Myoclonic epilepsy myopathy sensory ataxia     Mutations in the POLG gene cause another POLG-related disorder called myoclonic epilepsy myopathy sensory ataxia (MEMSA), which is characterized by recurrent seizures (epilepsy), muscle weakness (myopathy), and problems with coordination and balance (ataxia).Most of the POLG gene mutations involved in MEMSA change single amino acids in the alpha subunit of pol . These mutations result in a less active form of pol  that has a reduced ability to replicate mtDNA.As in other POLG-related disorders, people with MEMSA typically have mtDNA depletion in affected tissues, such as muscle or brain. MtDNA depletion leads to impaired oxidative phosphorylation and decreased energy reserves in affected tissues, which may cause the signs and symptoms of MEMSA. More About This Health Condition      Progressive external ophthalmoplegia     Mutations in the POLG gene are frequently responsible for an eye condition called progressive external ophthalmoplegia, another POLG-related disorder. This condition weakens the muscles that control eye movement and causes the eyelids to droop (ptosis).There are at least 67 POLG gene mutations that cause progressive external ophthalmoplegia. Most POLG gene mutations change single amino acids in the alpha subunit of pol , which decreases the efficiency of mtDNA replication. As in another POLG-related disorder, Alpers-Huttenlocher syndrome, the most common POLG gene mutation in progressive external ophthalmoplegia is Ala467Thr. It is unclear how the same mutation can lead to different disorders.In progressive external ophthalmoplegia, mutations in the POLG gene result in large deletions of genetic material from mtDNA in muscle tissue, rather than the overall mtDNA depletion seen in other POLG-related disorders. The reason for this difference is unknown. Researchers have not determined how deletions of mtDNA lead to the specific signs and symptoms of progressive external ophthalmoplegia, although the features of the condition are probably related to impaired oxidative phosphorylation. It has been suggested that eye muscles are commonly affected by mitochondrial defects because they are especially dependent on oxidative phosphorylation for energy. More About This Health Condition      Leigh syndrome     Genetics Home Reference provides information about Leigh syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      There are many mutations in the POLG gene that cause Alpers-Huttenlocher syndrome. Alpers-Huttenlocher syndrome is part of a group of conditions called POLG-related disorders that have overlapping signs and symptoms affecting muscle-, nerve-, and brain-related functions. Alpers-Huttenlocher syndrome is characterized by seizures, loss of mental and movement abilities (psychomotor regression), and liver disease. The liver disease in Alpers-Huttenlocher syndrome can be brought on or made worse by valproic acid, a common treatment for seizures.Most POLG gene mutations change single protein building blocks (amino acids) in the alpha subunit of pol . The mutations can have several effects on the function of pol . The alpha subunit may lose the ability to attach to the beta subunits to form pol . Alternately, mutated pol  may be unable to bind the existing mtDNA strand to use as a template. Or, it may have a reduced ability to attract the DNA building blocks (nucleotides) that it uses to form new DNA. These effects impair DNA synthesis and may lead to insertion of the wrong nucleotide and decreased ability to fix the error.The most common POLG gene mutation in Alpers-Huttenlocher syndrome replaces the amino acid alanine with the amino acid threonine at position 467 (written as Ala467Thr or A467T). This mutation blocks the ability of the alpha subunit to attach to the beta subunits and reduces pol 's ability to synthesize DNA. The Ala467Thr mutation is also common in other POLG-related disorders. The different conditions may be determined, in part, by the mutation in the other copy of POLG, but there are still some mutation combinations that can cause more than one of the disorders. It is unclear how the same mutation can lead to different conditions.Although the mechanism is unknown, many people with Alpers-Huttenlocher syndrome have fewer copies of mtDNA (mtDNA depletion). This abnormality is seen only in the tissues affected by the disease. MtDNA depletion leads to impaired oxidative phosphorylation and a decrease in cellular energy. These impairments affect tissues whose cells do not divide continually, such as brain, muscle, and liver. These tissues are most affected because they are more dependent on oxidative phosphorylation for energy, and impaired cells in these tissues are not generally replaced by new cells. The lack of energy supplies in these tissues could account for the signs and symptoms of Alpers-Huttenlocher syndrome. More About This Health Condition  Another condition caused by mutations in the POLG gene is ataxia neuropathy spectrum, a POLG-related disorder that is characterized by problems with coordination and balance (ataxia) and disturbances in nerve function (neuropathy). The conditions previously named mitochondrial recessive ataxia syndrome (MIRAS) and sensory ataxia neuropathy dysarthria and ophthalmoplegia (SANDO) are now included in the ataxia neuropathy spectrum.As mentioned above, most mutations in the POLG gene change single amino acids in the alpha subunit of pol , and these mutations can have several effects on the function of pol . As a result, pol  has a reduced ability to replicate mtDNA. The most common POLG gene mutation in ataxia neuropathy spectrum is the same as that in Alpers-Huttenlocher syndrome, Ala467Thr. It is unclear how the same mutation can lead to different disorders.As in other POLG-related disorders, people with ataxia neuropathy spectrum typically have mtDNA depletion in the tissues affected by the condition, such as the brain. MtDNA depletion decreases the amount of energy available to the cell due to reduced oxidative phosphorylation, which may account for the signs and symptoms of ataxia neuropathy spectrum. More About This Health Condition  Childhood myocerebrohepatopathy syndrome (MCHS) is also caused by mutations in the POLG gene. MCHS is a POLG-related disorder that affects the muscles (myo-), brain (cerebro-), and liver (hepato-).Many mutations in the POLG gene can cause MCHS. Most of these mutations change single amino acids in the alpha subunit of pol . These mutations reduce the activity of pol , decreasing mtDNA replication.As in other POLG-related disorders, people with MCHS typically have mtDNA depletion in muscle, brain, or liver tissue. MtDNA depletion impairs oxidative phosphorylation in these tissues and decreases the energy available to the cells, which may cause the signs and symptoms of MCHS. More About This Health Condition  Mutations in the POLG gene cause another POLG-related disorder called myoclonic epilepsy myopathy sensory ataxia (MEMSA), which is characterized by recurrent seizures (epilepsy), muscle weakness (myopathy), and problems with coordination and balance (ataxia).Most of the POLG gene mutations involved in MEMSA change single amino acids in the alpha subunit of pol . These mutations result in a less active form of pol  that has a reduced ability to replicate mtDNA.As in other POLG-related disorders, people with MEMSA typically have mtDNA depletion in affected tissues, such as muscle or brain. MtDNA depletion leads to impaired oxidative phosphorylation and decreased energy reserves in affected tissues, which may cause the signs and symptoms of MEMSA. More About This Health Condition  Mutations in the POLG gene are frequently responsible for an eye condition called progressive external ophthalmoplegia, another POLG-related disorder. This condition weakens the muscles that control eye movement and causes the eyelids to droop (ptosis).There are at least 67 POLG gene mutations that cause progressive external ophthalmoplegia. Most POLG gene mutations change single amino acids in the alpha subunit of pol , which decreases the efficiency of mtDNA replication. As in another POLG-related disorder, Alpers-Huttenlocher syndrome, the most common POLG gene mutation in progressive external ophthalmoplegia is Ala467Thr. It is unclear how the same mutation can lead to different disorders.In progressive external ophthalmoplegia, mutations in the POLG gene result in large deletions of genetic material from mtDNA in muscle tissue, rather than the overall mtDNA depletion seen in other POLG-related disorders. The reason for this difference is unknown. Researchers have not determined how deletions of mtDNA lead to the specific signs and symptoms of progressive external ophthalmoplegia, although the features of the condition are probably related to impaired oxidative phosphorylation. It has been suggested that eye muscles are commonly affected by mitochondrial defects because they are especially dependent on oxidative phosphorylation for energy. More About This Health Condition  Genetics Home Reference provides information about Leigh syndrome. More About This Health Condition    Cytogenetic Location: 15q26.1, which is the long (q) arm of chromosome 15 at position 26.1Molecular Location: base pairs 89,316,305 to 89,334,795 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DNA polymerase subunit gamma-1mitochondrial DNA polymerase catalytic subunitPolG-alphaPolG, catalytic subunitPOLG1POLGApolymerase (DNA directed), gammapolymerase (DNA) gamma, catalytic subunit     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): How Cells Obtain Energy from FoodMolecular Cell Biology (fourth edition, 2000): Mitochondria are the Principal Sites of ATP Production in Aerobic Cells      Clinical Information from GeneReviews (3 links)     Mitochondrial DNA Deletion SyndromesMitochondrial DNA Maintenance Defects OverviewPOLG-Related Disorders      Tests Listed in the Genetic Testing Registry (1 link)     Tests of POLG      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     POLYMERASE, DNA, GAMMA      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): How Cells Obtain Energy from FoodMolecular Cell Biology (fourth edition, 2000): Mitochondria are the Principal Sites of ATP Production in Aerobic Cells  Mitochondrial DNA Deletion SyndromesMitochondrial DNA Maintenance Defects OverviewPOLG-Related Disorders  Tests of POLG  PubMed  POLYMERASE, DNA, GAMMA  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Chan SS, Longley MJ, Copeland WC. The common A467T mutation in the human mitochondrial DNA polymerase (POLG) compromises catalytic efficiency and interaction with the accessory subunit. J Biol Chem. 2005 Sep 9;280(36):31341-6. Epub 2005 Jul 16. Citation on PubMedMilone M, Massie R. Polymerase gamma 1 mutations: clinical correlations. Neurologist. 2010 Mar;16(2):84-91. doi: 10.1097/NRL.0b013e3181c78a89. Review. Citation on PubMedMoraes CT, Shanske S, Tritschler HJ, Aprille JR, Andreetta F, Bonilla E, Schon EA, DiMauro S. mtDNA depletion with variable tissue expression: a novel genetic abnormality in mitochondrial diseases. Am J Hum Genet. 1991 Mar;48(3):492-501. Citation on PubMed or Free article on PubMed CentralOMIM: POLYMERASE, DNA, GAMMARocher C, Taanman JW, Pierron D, Faustin B, Benard G, Rossignol R, Malgat M, Pedespan L, Letellier T. Influence of mitochondrial DNA level on cellular energy metabolism: implications for mitochondrial diseases. J Bioenerg Biomembr. 2008 Apr;40(2):59-67. doi: 10.1007/s10863-008-9130-5. Epub 2008 Apr 16. Citation on PubMedSohl CD, Kasiviswanathan R, Copeland WC, Anderson KS. Mutations in human DNA polymerase  confer unique mechanisms of catalytic deficiency that mirror the disease severity in mitochondrial disorder patients. Hum Mol Genet. 2013 Mar 15;22(6):1074-85. doi: 10.1093/hmg/dds509. Epub 2012 Dec 3. Citation on PubMed or Free article on PubMed CentralStumpf JD, Copeland WC. Mitochondrial DNA replication and disease: insights from DNA polymerase  mutations. Cell Mol Life Sci. 2011 Jan;68(2):219-33. doi: 10.1007/s00018-010-0530-4. Epub 2010 Oct 8. Review. Citation on PubMed or Free article on PubMed CentralVan Goethem G, Dermaut B, Lfgren A, Martin JJ, Van Broeckhoven C. Mutation of POLG is associated with progressive external ophthalmoplegia characterized by mtDNA deletions. Nat Genet. 2001 Jul;28(3):211-2. Citation on PubMedVan Goethem G, Martin JJ, Dermaut B, Lfgren A, Wibail A, Ververken D, Tack P, Dehaene I, Van Zandijcke M, Moonen M, Ceuterick C, De Jonghe P, Van Broeckhoven C. Recessive POLG mutations presenting with sensory and ataxic neuropathy in compound heterozygote patients with progressive external ophthalmoplegia. Neuromuscul Disord. 2003 Feb;13(2):133-42. Citation on PubMedVan Goethem G, Martin JJ, Van Broeckhoven C. Progressive external ophthalmoplegia characterized by multiple deletions of mitochondrial DNA: unraveling the pathogenesis of human mitochondrial DNA instability and the initiation of a genetic classification. Neuromolecular Med. 2003;3(3):129-46. Review. Citation on PubMedYu Wai Man CY, Chinnery PF, Griffiths PG. Extraocular muscles have fundamentally distinct properties that make them selectively vulnerable to certain disorders. Neuromuscul Disord. 2005 Jan;15(1):17-23. Epub 2004 Nov 26. Review. Citation on PubMed \n",
            "MESP1\t\n",
            "MESP2\t   The MESP2 gene provides instructions for making a transcription factor, which is a protein that attaches (binds) to specific regions of DNA and helps control the activity of particular genes.  The MESP2 protein controls the activity of genes in the Notch pathway, an important pathway in embryonic development. The Notch pathway plays a critical role in the development of the bones of the spine (vertebrae). Specifically, the MESP2 protein and the Notch pathway are involved in separating future vertebrae from one another during early development, in a complex process called somite segmentation. Although the exact mechanism of somite segmentation is unclear, it appears to require the activity of several proteins in the Notch pathway, including the NOTCH1 protein and the MESP2 protein, to be turned on and off (oscillate) in a specific pattern.The MESP2 protein regulates Notch activity by turning on (activating) genes in the Notch pathway, which ultimately block (repress) the activity of the NOTCH1 protein. Additionally, through unknown mechanisms, the MESP2 protein seems to mark the boundary separating future vertebrae from one another.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Spondylocostal dysostosis     Genetics Home Reference provides information about spondylocostal dysostosis. More About This Health Condition      Spondylothoracic dysostosis     At least three mutations in the MESP2 gene have been found to cause spondylothoracic dysostosis, a condition characterized by abnormal development of bones in the spine and ribs.  All of the known mutations replace one protein building block (amino acid) in the protein sequence. The most common mutation replaces the amino acid glutamate with a premature stop signal at position 103 (written as Glu103Ter or E103X). A similar mutation occurs at amino acid position 230 (written as Glu230Ter or E230X). The third mutation replaces the amino acid leucine with the amino acid valine at position 125 (written as Leu125Val or L125V). Most affected individuals have the Glu103Ter mutation in both copies of the MESP2 gene.  However, a few people with spondylothoracic dysostosis have the Glu103Ter mutation in one copy of the MESP2 gene and either the Leu125Val or the Glu230Ter mutation in the other copy.Mutations in the MESP2 gene prevent the production of any protein or lead to the production of an abnormally short, nonfunctional protein. When the MESP2 protein is nonfunctional or absent, the NOTCH1 protein is abnormally active and the boundary separating future vertebrae from one another does not form. This results in the malformation and fusion of the bones of the spine and ribs seen in spondylothoracic dysostosis. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about spondylocostal dysostosis. More About This Health Condition  At least three mutations in the MESP2 gene have been found to cause spondylothoracic dysostosis, a condition characterized by abnormal development of bones in the spine and ribs.  All of the known mutations replace one protein building block (amino acid) in the protein sequence. The most common mutation replaces the amino acid glutamate with a premature stop signal at position 103 (written as Glu103Ter or E103X). A similar mutation occurs at amino acid position 230 (written as Glu230Ter or E230X). The third mutation replaces the amino acid leucine with the amino acid valine at position 125 (written as Leu125Val or L125V). Most affected individuals have the Glu103Ter mutation in both copies of the MESP2 gene.  However, a few people with spondylothoracic dysostosis have the Glu103Ter mutation in one copy of the MESP2 gene and either the Leu125Val or the Glu230Ter mutation in the other copy.Mutations in the MESP2 gene prevent the production of any protein or lead to the production of an abnormally short, nonfunctional protein. When the MESP2 protein is nonfunctional or absent, the NOTCH1 protein is abnormally active and the boundary separating future vertebrae from one another does not form. This results in the malformation and fusion of the bones of the spine and ribs seen in spondylothoracic dysostosis. More About This Health Condition    Cytogenetic Location: 15q26.1, which is the long (q) arm of chromosome 15 at position 26.1Molecular Location: base pairs 89,776,358 to 89,778,754 on chromosome 15 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bHLHc6class C basic helix-loop-helix protein 6mesoderm posterior 2 homolog (mouse)mesoderm posterior basic helix-loop-helix transcription factor 2mesoderm posterior protein 2MESP2_HUMANSCDO2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (4th Edition, 2002): Paraxial Mesoderm: The Somites and Their Derivatives      Clinical Information from GeneReviews (1 link)     Spondylocostal Dysostosis, Autosomal Recessive      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MESP2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     MESODERM POSTERIOR BASIC HELIX-LOOP-HELIX TRANSCRIPTION FACTOR 2      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarGENATLASHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (4th Edition, 2002): Paraxial Mesoderm: The Somites and Their Derivatives  Spondylocostal Dysostosis, Autosomal Recessive  Tests of MESP2  PubMed  MESODERM POSTERIOR BASIC HELIX-LOOP-HELIX TRANSCRIPTION FACTOR 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarGENATLASHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cornier AS, Staehling-Hampton K, Delventhal KM, Saga Y, Caubet JF, Sasaki N, Ellard S, Young E, Ramirez N, Carlo SE, Torres J, Emans JB, Turnpenny PD, Pourqui O. Mutations in the MESP2 gene cause spondylothoracic dysostosis/Jarcho-Levin syndrome. Am J Hum Genet. 2008 Jun;82(6):1334-41. doi: 10.1016/j.ajhg.2008.04.014. Epub 2008 May 15. Citation on PubMed or Free article on PubMed CentralFerjentsik Z, Hayashi S, Dale JK, Bessho Y, Herreman A, De Strooper B, del Monte G, de la Pompa JL, Maroto M. Notch is a critical component of the mouse somitogenesis oscillator and is essential for the formation of the somites. PLoS Genet. 2009 Sep;5(9):e1000662. doi: 10.1371/journal.pgen.1000662. Epub 2009 Sep 25. Citation on PubMed or Free article on PubMed CentralGibb S, Maroto M, Dale JK. The segmentation clock mechanism moves up a notch. Trends Cell Biol. 2010 Oct;20(10):593-600. doi: 10.1016/j.tcb.2010.07.001. Epub 2010 Aug 18. Review. Citation on PubMed or Free article on PubMed CentralMorimoto M, Takahashi Y, Endo M, Saga Y. The Mesp2 transcription factor establishes segmental borders by suppressing Notch activity. Nature. 2005 May 19;435(7040):354-9. Citation on PubMedOginuma M, Takahashi Y, Kitajima S, Kiso M, Kanno J, Kimura A, Saga Y. The oscillation of Notch activation, but not its boundary, is required for somite border formation and rostral-caudal patterning within a somite. Development. 2010 May;137(9):1515-22. doi: 10.1242/dev.044545. Epub 2010 Mar 24. Citation on PubMedSasaki N, Kiso M, Kitagawa M, Saga Y. The repression of Notch signaling occurs via the destabilization of mastermind-like 1 by Mesp2 and is essential for somitogenesis. Development. 2011 Jan;138(1):55-64. doi: 10.1242/dev.055533. Epub 2010 Nov 23. Citation on PubMedSparrow DB, Chapman G, Turnpenny PD, Dunwoodie SL. Disruption of the somitic molecular clock causes abnormal vertebral segmentation. Birth Defects Res C Embryo Today. 2007 Jun;81(2):93-110. Review. Citation on PubMedTurnpenny PD, Young E; ICVS (International Consortium for Vertebral Anomalies and Scoliosis). Spondylocostal Dysostosis, Autosomal Recessive. 2009 Aug 25 [updated 2013 Jan 17]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK8828/ Citation on PubMedWhittock NV, Sparrow DB, Wouters MA, Sillence D, Ellard S, Dunwoodie SL, Turnpenny PD. Mutated MESP2 causes spondylocostal dysostosis in humans. Am J Hum Genet. 2004 Jun;74(6):1249-54. Epub 2004 Apr 30. Citation on PubMed or Free article on PubMed Central \n",
            "ANPEP\t\n",
            "CRTC3\t\n",
            "UNC45A\t\n",
            "FAM174B\t\n",
            "ARRDC4\t\n",
            "PGPEP1L\t\n",
            "SYNM\t\n",
            "ATF7IP2\t\n",
            "PRM3\t\n",
            "GSPT1\t\n",
            "TPSG1\t\n",
            "TPSB2\t\n",
            "TPSAB1\t   From NCBI Gene:Tryptases comprise a family of trypsin-like serine proteases, the peptidase family S1. Tryptases are enzymatically active only as heparin-stabilized tetramers, and they are resistant to all known endogenous proteinase inhibitors. Several tryptase genes are clustered on chromosome 16p13.3. These genes are characterized by several distinct features. They have a highly conserved 3' UTR and contain tandem repeat sequences at the 5' flank and 3' UTR which are thought to play a role in regulation of the mRNA stability. These genes have an intron immediately upstream of the initiator Met codon, which separates the site of transcription initiation from protein coding sequence. This feature is characteristic of tryptases but is unusual in other genes. The alleles of this gene exhibit an unusual amount of sequence variation, such that the alleles were once thought to represent two separate genes, alpha and beta 1. Beta tryptases appear to be the main isoenzymes expressed in mast cells; whereas in basophils, alpha tryptases predominate. Tryptases have been implicated as mediators in the pathogenesis of asthma and other allergic and inflammatory disorders. [provided by RefSeq, Jul 2008]From UniProt: TRYB2_HUMAN:Tryptase is the major neutral protease present in mast cells and is secreted upon the coupled activation-degranulation response of this cell type. May play a role in innate immunity.From UniProt: TRYB1_HUMAN:Tryptase is the major neutral protease present in mast cells and is secreted upon the coupled activation-degranulation response of this cell type. May play a role in innate immunity. Isoform 2 cleaves large substrates, such as fibronectin, more efficiently than isoform 1, but seems less efficient toward small substrates (PubMed:18854315).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt: TRYB1_HUMAN:Hereditary alpha tryptasemia is caused by an increase in the copy number (usually between two and three copies) of the alpha allele. Affected individuals have elevated basal serum tryptase levels that are associated with cutaneous flushing and pruritus, dysautonomia, functional gastrointestinal symptoms, chronic pain, and connective tissue abnormalities. It is not clear if the associated multisystem complaints might be due to the coinheritance of a second functional genetic variant.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16p13.3, which is the short (p) arm of chromosome 16 at position 13.3Molecular Location: base pairs 1,240,677 to 1,242,554 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        TPS1TPS2TPSB1TPSB2Tryptase-2     Related Information   How are genetic conditions and genes named?          Catalog of Genes and Diseases from OMIM (1 link)     OMIM: TRYPTASE, ALPHA/BETA-1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt: TRYB1_HUMANUniProt: TRYB2_HUMAN      OMIM: TRYPTASE, ALPHA/BETA-1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt: TRYB1_HUMANUniProt: TRYB2_HUMAN \n",
            "TPSD1\t\n",
            "UNKL\t\n",
            "NPIPA1\t\n",
            "PTX4\t\n",
            "NPIPA5\t\n",
            "NPRL3\t   The NPRL3 gene provides instructions for making a protein that is one piece of a group of proteins (complex) called GATOR1. This complex is found in cells throughout the body, where it regulates a signaling pathway called the mTOR pathway. The mTOR pathway is involved in cell growth and division (proliferation), the survival of cells, and the creation (synthesis) of new proteins. The role of the GATOR1 complex is to block this pathway by inhibiting (stopping) the activity of a complex called mTOR complex 1 (mTORC1) that is integral to the mTOR pathway.In the brain, the mTOR pathway regulates many processes, including the growth and development of nerve cells and their ability to change and adapt over time (plasticity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Familial focal epilepsy with variable foci     At least ten NPRL3 gene mutations have been found to cause familial focal epilepsy with variable foci (FFEVF), which is an uncommon form of recurrent seizures (epilepsy) that runs in families. Most of these mutations lead to the production of an abnormally short, nonfunctional protein. As a result, formation of normal GATOR1 complex is reduced, leading to overactivity of mTORC1 and excessive signaling of the mTOR pathway. It is not clear how an abnormally active mTOR pathway leads to the seizures of FFEVF. Research suggests that increased mTOR pathway signaling in the brain leads to changes in the connections between nerve cells (synapses) and increased activation (excitation) of nerve cells, which can cause seizures. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least ten NPRL3 gene mutations have been found to cause familial focal epilepsy with variable foci (FFEVF), which is an uncommon form of recurrent seizures (epilepsy) that runs in families. Most of these mutations lead to the production of an abnormally short, nonfunctional protein. As a result, formation of normal GATOR1 complex is reduced, leading to overactivity of mTORC1 and excessive signaling of the mTOR pathway. It is not clear how an abnormally active mTOR pathway leads to the seizures of FFEVF. Research suggests that increased mTOR pathway signaling in the brain leads to changes in the connections between nerve cells (synapses) and increased activation (excitation) of nerve cells, which can cause seizures. More About This Health Condition    Cytogenetic Location: 16p13.3, which is the short (p) arm of chromosome 16 at position 13.3Molecular Location: base pairs 85,801 to 138,698 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        alpha-globin regulatory element-containing gene proteinC16orf35CGTHBAconserved gene telomeric to alpha globin clusterHS-40MARENPR3RMD11     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Jasper's Basic Mechanisms of the Epilepsies (fourth edition, 2012): mTOR and Epileptogenesis in Developmental Brain Malformations      Tests Listed in the Genetic Testing Registry (1 link)     Tests of NPRL3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     NITROGEN PERMEASE REGULATOR-LIKE 3      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Jasper's Basic Mechanisms of the Epilepsies (fourth edition, 2012): mTOR and Epileptogenesis in Developmental Brain Malformations  Tests of NPRL3  PubMed  NITROGEN PERMEASE REGULATOR-LIKE 3  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Baldassari S, Licchetta L, Tinuper P, Bisulli F, Pippucci T. GATOR1 complex: the common genetic actor in focal epilepsies. J Med Genet. 2016 Aug;53(8):503-10. doi: 10.1136/jmedgenet-2016-103883. Review. Citation on PubMedBaulac S. mTOR signaling pathway genes in focal epilepsies. Prog Brain Res. 2016;226:61-79. doi: 10.1016/bs.pbr.2016.04.013. Citation on PubMedOMIM: NITROGEN PERMEASE REGULATOR-LIKE 3Sim JC, Scerri T, Fanjul-Fernndez M, Riseley JR, Gillies G, Pope K, van Roozendaal H, Heng JI, Mandelstam SA, McGillivray G, MacGregor D, Kannan L, Maixner W, Harvey AS, Amor DJ, Delatycki MB, Crino PB, Bahlo M, Lockhart PJ, Leventer RJ. Familial cortical dysplasia caused by mutation in the mammalian target of rapamycin regulator NPRL3. Ann Neurol. 2016 Jan;79(1):132-7. doi: 10.1002/ana.24502. Citation on PubMedWeckhuysen S, Marsan E, Lambrecq V, Marchal C, Morin-Brureau M, An-Gourfinkel I, Baulac M, Fohlen M, Kallay Zetchi C, Seeck M, de la Grange P, Dermaut B, Meurs A, Thomas P, Chassoux F, Leguern E, Picard F, Baulac S. Involvement of GATOR complex genes in familial focal epilepsies and focal cortical dysplasia. Epilepsia. 2016 Jun;57(6):994-1003. doi: 10.1111/epi.13391. Citation on PubMed \n",
            "COQ7\t   From NCBI Gene:The protein encoded by this gene is similar to a mitochondrial di-iron containing hydroxylase in Saccharomyces cerevisiae that is involved with ubiquinone biosynthesis. Mutations in the yeast gene lead to slower development and longer life span. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Jul 2010]From UniProt:Catalyzes the hydroxylation of 2-polyprenyl-3-methyl-6-methoxy-1,4-benzoquinol (DMQH2) during ubiquinone biosynthesis. Has also a structural role in the COQ enzyme complex, stabilizing other COQ polypeptides. Involved in lifespan determination in a ubiquinone-independent manner.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary coenzyme Q10 deficiencyFrom NCBI Gene:Coenzyme Q10 deficiency, primary, 8From UniProt:Coenzyme Q10 deficiency, primary, 8 (COQ10D8): An autosomal recessive disorder resulting from mitochondrial dysfunction and characterized by decreased levels of coenzyme Q10. Patients manifest neonatal lung hypoplasia, contractures, early infantile hypertension and cardiac hypertrophy, secondary to prenatal kidney dysplasia, with neonatal and infantile renal dysfunction. Clinical features also include progressive peripheral neuropathy, muscular hypotonia and atrophy, and mild psychomotor delay with hearing and visual impairment. [MIM:616733]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16p12.3, which is the short (p) arm of chromosome 16 at position 12.3Molecular Location: base pairs 19,067,595 to 19,080,095 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAT5CLK-1CLK1COQ10D8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COQ7      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: COENZYME Q10 DEFICIENCY, PRIMARY, 8OMIM: COQ7, S. CEREVISIAE, HOMOLOG OF      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COQ7  OMIM: COENZYME Q10 DEFICIENCY, PRIMARY, 8OMIM: COQ7, S. CEREVISIAE, HOMOLOG OF  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ACSM5\t\n",
            "ACSM2A\t\n",
            "ZNF598\t\n",
            "ACSM1\t\n",
            "NPW\t\n",
            "ERI2\t\n",
            "SLC9A3R2\t\n",
            "PKD1\t   The PKD1 gene provides instructions for making a protein called polycystin-1.  This protein is most active in kidney cells before birth; much less of the protein is made in normal adult kidneys.  Although its exact function is not well understood, polycystin-1 appears to interact with a smaller, somewhat similar protein called polycystin-2.Polycystin-1 spans the cell membrane of kidney cells, so that one end of the protein remains inside the cell and the other end projects from the outer surface of the cell. This positioning of the protein allows it to interact with other proteins, carbohydrates, and fat molecules (lipids) outside the cell and to receive signals that help the cell respond to its environment. When a molecule binds to polycystin-1 on the surface of the cell, the protein interacts with polycystin-2 to trigger a cascade of chemical reactions inside the cell. These chemical reactions instruct the cell to undergo certain changes, such as maturing to take on specialized functions.  Polycystin-1 and polycystin-2 likely work together to help regulate cell growth and division (proliferation), cell movement (migration), and interactions with other cells.Polycystin-1 is also found in cell structures called primary cilia.  Primary cilia are tiny, fingerlike projections that line the small tubes where urine is formed (renal tubules).  Researchers believe that primary cilia sense the movement of fluid through these tubules, which appears to help maintain the tubules' size and structure.  The interaction of polycystin-1 and polycystin-2 in renal tubules promotes the normal development and function of the kidneys.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Polycystic kidney disease     More than 250 mutations in the PKD1 gene have been identified in people with polycystic kidney disease.  These mutations are responsible for about 85 percent of cases of autosomal dominant polycystic kidney disease (ADPKD), which is the most common type of this disorder. Mutations in the PKD1 gene include deletions or insertions of DNA building blocks (base pairs) and alterations of one or more base pairs.  Most PKD1 mutations are predicted to produce an abnormally small, nonfunctional version of the polycystin-1 protein. Although researchers are uncertain how a lack of polycystin-1 leads to the formation of cysts, it probably disrupts the protein's signaling function within the cell and in primary cilia.  As a result, cells lining the renal tubules may grow and divide abnormally, leading to the growth of numerous cysts characteristic of polycystic kidney disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 250 mutations in the PKD1 gene have been identified in people with polycystic kidney disease.  These mutations are responsible for about 85 percent of cases of autosomal dominant polycystic kidney disease (ADPKD), which is the most common type of this disorder. Mutations in the PKD1 gene include deletions or insertions of DNA building blocks (base pairs) and alterations of one or more base pairs.  Most PKD1 mutations are predicted to produce an abnormally small, nonfunctional version of the polycystin-1 protein. Although researchers are uncertain how a lack of polycystin-1 leads to the formation of cysts, it probably disrupts the protein's signaling function within the cell and in primary cilia.  As a result, cells lining the renal tubules may grow and divide abnormally, leading to the growth of numerous cysts characteristic of polycystic kidney disease. More About This Health Condition    Cytogenetic Location: 16p13.3, which is the short (p) arm of chromosome 16 at position 13.3Molecular Location: base pairs 2,088,708 to 2,135,898 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        Lov-1PBPPc-1PC1PKD1_HUMANpolycystic kidney disease 1 (autosomal dominant)polycystin-1TRPP1     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Polycystic Kidney Disease, Autosomal Dominant      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PKD1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     POLYCYSTIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Polycystic Kidney Disease, Autosomal Dominant  Tests of PKD1  PubMed  POLYCYSTIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Al-Bhalal L, Akhtar M. Molecular basis of autosomal dominant polycystic kidney disease. Adv Anat Pathol. 2005 May;12(3):126-33. Review. Citation on PubMedBissler JJ, Dixon BP. A mechanistic approach to inherited polycystic kidney disease. Pediatr Nephrol. 2005 May;20(5):558-66. Epub 2005 Feb 18. Review. Citation on PubMedBoucher C, Sandford R. Autosomal dominant polycystic kidney disease (ADPKD, MIM 173900, PKD1 and PKD2 genes, protein products known as polycystin-1 and polycystin-2). Eur J Hum Genet. 2004 May;12(5):347-54. Review. Citation on PubMedHarris PC, Torres VE. Polycystic Kidney Disease, Autosomal Dominant. 2002 Jan 10 [updated 2015 Jun 11]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1246/ Citation on PubMedHorie S. ADPKD: molecular characterization and quest for treatment. Clin Exp Nephrol. 2005 Dec;9(4):282-91. Review. Citation on PubMedLina F, Satlinb LM. Polycystic kidney disease: the cilium as a common pathway in cystogenesis. Curr Opin Pediatr. 2004 Apr;16(2):171-6. Review. Citation on PubMedNauli SM, Alenghat FJ, Luo Y, Williams E, Vassilev P, Li X, Elia AE, Lu W, Brown EM, Quinn SJ, Ingber DE, Zhou J. Polycystins 1 and 2 mediate mechanosensation in the primary cilium of kidney cells. Nat Genet. 2003 Feb;33(2):129-37. Epub 2003 Jan 6. Citation on PubMedNauli SM, Rossetti S, Kolb RJ, Alenghat FJ, Consugar MB, Harris PC, Ingber DE, Loghman-Adham M, Zhou J. Loss of polycystin-1 in human cyst-lining epithelia leads to ciliary dysfunction. J Am Soc Nephrol. 2006 Apr;17(4):1015-25. Citation on PubMedOng AC, Harris PC. Molecular pathogenesis of ADPKD: the polycystin complex gets complex. Kidney Int. 2005 Apr;67(4):1234-47. Citation on PubMedWilson PD. Polycystic kidney disease. N Engl J Med. 2004 Jan 8;350(2):151-64. Review. Citation on PubMed \n",
            "NPIPB5\t\n",
            "MLST8\t\n",
            "USP31\t\n",
            "RBBP6\t\n",
            "TNRC6A\t   From NCBI Gene:This gene encodes a member of the trinucleotide repeat containing 6 protein family. The protein functions in post-transcriptional gene silencing through the RNA interference (RNAi) and microRNA pathways. The protein associates with messenger RNAs and Argonaute proteins in cytoplasmic bodies known as GW-bodies or P-bodies. Inhibiting expression of this gene delocalizes other GW-body proteins and impairs RNAi and microRNA-induced gene silencing. [provided by RefSeq, Jul 2008]From UniProt:Plays a role in RNA-mediated gene silencing by both micro-RNAs (miRNAs) and short interfering RNAs (siRNAs). Required for miRNA-dependent repression of translation and for siRNA-dependent endonucleolytic cleavage of complementary mRNAs by argonaute family proteins. As a scaffolding protein, associates with argonaute proteins bound to partially complementary mRNAs, and can simultaneously recruit CCR4-NOT and PAN deadenylase complexes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:EPILEPSY, FAMILIAL ADULT MYOCLONIC, 6From UniProt:Epilepsy, familial adult myoclonic, 6 (FAME6): A form of familial myoclonic epilepsy, a neurologic disorder characterized by cortical hand tremors, myoclonic jerks and occasional generalized or focal seizures with a non-progressive or very slowly progressive disease course. Usually, myoclonic tremor is the presenting symptom, characterized by tremulous finger movements and myoclonic jerks of the limbs increased by action and posture. In a minority of patients, seizures are the presenting symptom. Some patients exhibit mild cognitive impairment. FAME6 inheritance is autosomal dominant. [MIM:618074]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16p12.1, which is the short (p) arm of chromosome 16 at position 12.1Molecular Location: base pairs 24,610,209 to 24,826,227 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAGH26FAME6GW1GW182TNRC6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TNRC6A      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: EPILEPSY, FAMILIAL ADULT MYOCLONIC, 6OMIM: TRINUCLEOTIDE REPEAT-CONTAINING GENE 6A      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TNRC6A  OMIM: EPILEPSY, FAMILIAL ADULT MYOCLONIC, 6OMIM: TRINUCLEOTIDE REPEAT-CONTAINING GENE 6A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LCMT1\t\n",
            "NPIPB6\t\n",
            "ASPHD1\t\n",
            "NPIPB13\t\n",
            "BICDL2\t\n",
            "IL32\t\n",
            "ITGAD\t\n",
            "ZNF720\t\n",
            "RGS11\t\n",
            "PDIA2\t\n",
            "NLRC3\t\n",
            "CORO7\t\n",
            "NME4\t\n",
            "PPL\t\n",
            "RBL2\t\n",
            "RPGRIP1L\t   From NCBI Gene:The protein encoded by this gene can localize to the basal body-centrosome complex or to primary cilia and centrosomes in ciliated cells. The encoded protein has been found to interact with nephrocystin-4. Defects in this gene are a cause of Joubert syndrome type 7 (JBTS7) and Meckel syndrome type 5 (MKS5). [provided by RefSeq, Jun 2016]From UniProt:Negatively regulates signaling through the G-protein coupled thromboxane A2 receptor (TBXA2R) (PubMed:19464661). May be involved in mechanisms like programmed cell death, craniofacial development, patterning of the limbs, and formation of the left-right axis (By similarity). Involved in the organization of apical junctions; the function is proposed to implicate a NPHP1-4-8 module. Does not seem to be strictly required for ciliogenesis (PubMed:19464661). Involved in establishment of planar cell polarity such as in cochlear sensory epithelium and is proposed to implicate stabilization of disheveled proteins (By similarity). Involved in regulation of proteasomal activity at the primary cilium probably implicating association with PSDM2 (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Meckel syndromeJoubert syndromeColobomaFrom NCBI Gene:COACH syndromeMeckel syndrome type 5Joubert syndrome 7From UniProt:Joubert syndrome 7 (JBTS7): A disorder presenting with cerebellar ataxia, oculomotor apraxia, hypotonia, neonatal breathing abnormalities and psychomotor delay. Neuroradiologically, it is characterized by cerebellar vermian hypoplasia/aplasia, thickened and reoriented superior cerebellar peduncles, and an abnormally large interpeduncular fossa, giving the appearance of a molar tooth on transaxial slices (molar tooth sign). Additional variable features include retinal dystrophy and renal disease. [MIM:611560]Ciliary dysfunction leads to a broad spectrum of disorders, collectively termed ciliopathies. Overlapping clinical features include retinal degeneration, renal cystic disease, skeletal abnormalities, fibrosis of various organ, and a complex range of anatomical and functional defects of the central and peripheral nervous system. The ciliopathy range of diseases includes Meckel-Gruber syndrome, Bardet-Biedl syndrome, Joubert syndrome, nephronophtisis, Senior-Loken syndrome, and Jeune asphyxiating thoracic dystrophy among others. Single-locus allelism is insufficient to explain the variable penetrance and expressivity of such disorders, leading to the suggestion that variations across multiple sites of the ciliary proteome, including RPGRIP1L, influence the clinical outcome.Meckel syndrome 5 (MKS5): A disorder characterized by a combination of renal cysts and variably associated features including developmental anomalies of the central nervous system (typically encephalocele), hepatic ductal dysplasia and cysts, and polydactyly. [MIM:611561]COACH syndrome (COACHS): A disorder characterized by mental retardation, ataxia due to cerebellar hypoplasia, and hepatic fibrosis. Patients present the molar tooth sign, a midbrain-hindbrain malformation pathognomonic for Joubert syndrome and related disorders. Other features, such as coloboma and renal cysts, may be variable. [MIM:216360]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q12.2, which is the long (q) arm of chromosome 16 at position 12.2Molecular Location: base pairs 53,599,239 to 53,703,938 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CORS3FTMJBTS7MKS5NPHP8PPP1R134     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RPGRIP1L      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: COACH SYNDROMEOMIM: JOUBERT SYNDROME 7OMIM: MECKEL SYNDROME, TYPE 5OMIM: RPGRIP1-LIKE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RPGRIP1L  OMIM: COACH SYNDROMEOMIM: JOUBERT SYNDROME 7OMIM: MECKEL SYNDROME, TYPE 5OMIM: RPGRIP1-LIKE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IRX6\t\n",
            "CES1\t   From NCBI Gene:This gene encodes a member of the carboxylesterase large family. The family members are responsible for the hydrolysis or transesterification of various xenobiotics, such as cocaine and heroin, and endogenous substrates with ester, thioester, or amide bonds. They may participate in fatty acyl and cholesterol ester metabolism, and may play a role in the blood-brain barrier system. This enzyme is the major liver enzyme and functions in liver drug clearance. Mutations of this gene cause carboxylesterase 1 deficiency. Three transcript variants encoding three different isoforms have been found for this gene. [provided by RefSeq, Jun 2010]From UniProt:Involved in the detoxification of xenobiotics and in the activation of ester and amide prodrugs. Hydrolyzes aromatic and aliphatic esters, but has no catalytic activity toward amides or a fatty acyl-CoA ester. Hydrolyzes the methyl ester group of cocaine to form benzoylecgonine. Catalyzes the transesterification of cocaine to form cocaethylene. Displays fatty acid ethyl ester synthase activity, catalyzing the ethyl esterification of oleic acid to ethyloleate.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:DRUG METABOLISM, ALTERED, CES1-RELATED     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q12.2, which is the long (q) arm of chromosome 16 at position 12.2Molecular Location: base pairs 55,802,851 to 55,833,163 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACATCE-1CEHCES2hCE-1HMSEHMSE1PCE-1REHSES1TGH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CES1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CARBOXYLESTERASE 1OMIM: DRUG METABOLISM, ALTERED, CES1-RELATED      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CES1  OMIM: CARBOXYLESTERASE 1OMIM: DRUG METABOLISM, ALTERED, CES1-RELATED  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CES5A\t\n",
            "MT1A\t\n",
            "DRC7\t\n",
            "CNGB1\t   From NCBI Gene:In humans, the rod photoreceptor cGMP-gated cation channel helps regulate ion flow into the rod photoreceptor outer segment in response to light-induced alteration of the levels of intracellular cGMP. This channel consists of two subunits, alpha and beta, with the protein encoded by this gene representing the beta subunit. Defects in this gene are a cause of cause of retinitis pigmentosa type 45. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2013]From UniProt:Isoform GARP2 is a high affinity rod photoreceptor phosphodiesterase (PDE6)-binding protein that modulates its catalytic properties: it is a regulator of spontaneous activation of rod PDE6, thereby serving to lower rod photoreceptor 'dark noise' and allowing these sensory cells to operate at the single photon detection limit.Subunit of cyclic nucleotide-gated (CNG) channels, nonselective cation channels, which play important roles in both visual and olfactory signal transduction. When associated with CNGA1, it is involved in the regulation of ion flow into the rod photoreceptor outer segment (ROS), in response to light-induced alteration of the levels of intracellular cGMP.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Retinitis pigmentosaFrom NCBI Gene:Retinitis pigmentosa 45From UniProt:Retinitis pigmentosa 45 (RP45): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:613767]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q21, which is the long (q) arm of chromosome 16 at position 21Molecular Location: base pairs 57,882,340 to 57,971,116 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CNCG2CNCG3LCNCG4CNG4CNGB1BGAR1GARPGARP2RCNC2RCNCbRCNCbetaRP45     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CNGB1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CYCLIC NUCLEOTIDE-GATED CHANNEL, BETA-1OMIM: RETINITIS PIGMENTOSA 45      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CNGB1  OMIM: CYCLIC NUCLEOTIDE-GATED CHANNEL, BETA-1OMIM: RETINITIS PIGMENTOSA 45  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CNOT1\t\n",
            "BEAN1\t   From NCBI Gene:The protein encoded by this gene is one of several proteins that interact with NEDD4, a member of a family of ubiquitin-protein ligases. These proteins have PY motifs in common that bind to the WW domains of NEDD4. NEDD4 is developmentally regulated, and is highly expressed in embryonic tissues. Mutations in this gene (i.e., intronic insertions of >100 copies of pentanucleotide repeats including a (TGGAA)n sequence) are associated with spinocerebellar ataxia type 31. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2010]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spinocerebellar ataxia 31From UniProt:Spinocerebellar ataxia 31 (SCA31): A form of spinocerebellar ataxia, a clinically and genetically heterogeneous group of cerebellar disorders. Patients show progressive incoordination of gait and often poor coordination of hands, speech and eye movements, due to degeneration of the cerebellum with variable involvement of the brainstem and spinal cord. SCA31 belongs to the autosomal dominant cerebellar ataxias type III (ADCA III) which are characterized by pure cerebellar ataxia without additional signs. [MIM:117210]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q21, which is the long (q) arm of chromosome 16 at position 21Molecular Location: base pairs 66,427,297 to 66,495,288 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BEANSCA31     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of BEAN1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: BRAIN-EXPRESSED, ASSOCIATED WITH NEDD4OMIM: SPINOCEREBELLAR ATAXIA 31      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of BEAN1  OMIM: BRAIN-EXPRESSED, ASSOCIATED WITH NEDD4OMIM: SPINOCEREBELLAR ATAXIA 31  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "E2F4\t\n",
            "THAP11\t\n",
            "LCAT\t   The LCAT gene provides instructions for making an enzyme called lecithin-cholesterol acyltransferase (LCAT). This enzyme plays a role in removing cholesterol from the blood and tissues. Cholesterol is a waxy, fat-like substance that is produced in the body and obtained from foods that come from animals (particularly egg yolks, meat, poultry, fish, and dairy products). The body needs this substance to build cell membranes, make certain hormones, and produce compounds that aid in fat digestion. Too much cholesterol, however, increases a person's risk of developing heart disease, and can also lead to buildup of cholesterol in other tissues.The LCAT enzyme helps transport cholesterol out of the blood and tissues by a process called cholesterol esterification. This process results in a form of cholesterol that is more efficiently carried by molecules called lipoproteins, which transport the cholesterol to the liver. Once in the liver, the cholesterol is redistributed to other tissues or removed from the body. The enzyme has two major functions, called  alpha- and beta-LCAT activity. Alpha-LCAT activity helps attach cholesterol to a lipoprotein called high-density lipoprotein (HDL). Beta-LCAT activity helps attach cholesterol to other lipoproteins called very low-density lipoprotein (VLDL) and low-density lipoprotein (LDL).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Complete LCAT deficiency     More than 70 mutations in the LCAT gene have been identified in people with complete LCAT deficiency, a disorder that primarily causes corneal clouding and kidney problems. Individuals with complete LCAT deficiency have mutations in both copies of the LCAT gene in each cell. Most of these mutations change single amino acids in the LCAT enzyme sequence. Others add or delete small amounts of genetic material in the LCAT gene. The mutations either prevent the production of LCAT or impair both alpha-LCAT and beta-LCAT activity, reducing the enzyme's ability to attach cholesterol to lipoproteins. Impairment of this mechanism for reducing cholesterol in the body leads to cholesterol deposits in the corneas, kidneys, and other tissues and organs. These deposits cause the signs and symptoms of complete LCAT deficiency. More About This Health Condition      Fish-eye disease     At least 18 mutations in the LCAT gene have been identified in people with fish-eye disease, also called partial LCAT deficiency. This disorder causes clouding of the clear covering of the eyes (corneas). Individuals with fish-eye disease have mutations in both copies of the LCAT gene in each cell. Most of these mutations change single protein building blocks (amino acids) in the LCAT sequence. The mutations impair alpha-LCAT activity, reducing the enzyme's ability to attach cholesterol to HDL. Impairment of this mechanism for reducing cholesterol in the body leads to cholesterol deposits in the corneas that gradually cause them to become cloudy. More About This Health Condition      Other disorders     People with one LCAT gene mutation in each cell may have an increased risk of atherosclerosis, which is an accumulation of cholesterol-rich fatty deposits and scar-like tissue in the lining of the arteries that can impede blood flow and lead to heart attacks, strokes, and other health problems. A single LCAT gene mutation likely reduces alpha-LCAT activity and binding of cholesterol to HDL, leading to less efficient transport of cholesterol from the blood and resulting in the development of atherosclerosis.Atherosclerosis also occurs in some people with fish-eye disease or complete LCAT deficiency. However, because atherosclerosis is very common in the general population and both fish-eye disease and complete LCAT deficiency are very rare, it is difficult to determine whether affected individuals are at increased risk of atherosclerosis.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 70 mutations in the LCAT gene have been identified in people with complete LCAT deficiency, a disorder that primarily causes corneal clouding and kidney problems. Individuals with complete LCAT deficiency have mutations in both copies of the LCAT gene in each cell. Most of these mutations change single amino acids in the LCAT enzyme sequence. Others add or delete small amounts of genetic material in the LCAT gene. The mutations either prevent the production of LCAT or impair both alpha-LCAT and beta-LCAT activity, reducing the enzyme's ability to attach cholesterol to lipoproteins. Impairment of this mechanism for reducing cholesterol in the body leads to cholesterol deposits in the corneas, kidneys, and other tissues and organs. These deposits cause the signs and symptoms of complete LCAT deficiency. More About This Health Condition  At least 18 mutations in the LCAT gene have been identified in people with fish-eye disease, also called partial LCAT deficiency. This disorder causes clouding of the clear covering of the eyes (corneas). Individuals with fish-eye disease have mutations in both copies of the LCAT gene in each cell. Most of these mutations change single protein building blocks (amino acids) in the LCAT sequence. The mutations impair alpha-LCAT activity, reducing the enzyme's ability to attach cholesterol to HDL. Impairment of this mechanism for reducing cholesterol in the body leads to cholesterol deposits in the corneas that gradually cause them to become cloudy. More About This Health Condition  People with one LCAT gene mutation in each cell may have an increased risk of atherosclerosis, which is an accumulation of cholesterol-rich fatty deposits and scar-like tissue in the lining of the arteries that can impede blood flow and lead to heart attacks, strokes, and other health problems. A single LCAT gene mutation likely reduces alpha-LCAT activity and binding of cholesterol to HDL, leading to less efficient transport of cholesterol from the blood and resulting in the development of atherosclerosis.Atherosclerosis also occurs in some people with fish-eye disease or complete LCAT deficiency. However, because atherosclerosis is very common in the general population and both fish-eye disease and complete LCAT deficiency are very rare, it is difficult to determine whether affected individuals are at increased risk of atherosclerosis.    Cytogenetic Location: 16q22.1, which is the long (q) arm of chromosome 16 at position 22.1Molecular Location: base pairs 67,939,884 to 67,944,112 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LCAT_HUMANphosphatidylcholine-sterol acyltransferasephosphatidylcholine-sterol acyltransferase precursorphospholipid-cholesterol acyltransferase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database: Fat Absorption and Lipid Metabolism in Cholestasis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of LCAT      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     ATHEROSCLEROSIS SUSCEPTIBILITYLECITHIN:CHOLESTEROL ACYLTRANSFERASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Fat Absorption and Lipid Metabolism in Cholestasis  Tests of LCAT  PubMed  ATHEROSCLEROSIS SUSCEPTIBILITYLECITHIN:CHOLESTEROL ACYLTRANSFERASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Calabresi L, Pisciotta L, Costantin A, Frigerio I, Eberini I, Alessandrini P, Arca M, Bon GB, Boscutti G, Busnach G, Frasc G, Gesualdo L, Gigante M, Lupattelli G, Montali A, Pizzolitto S, Rabbone I, Rolleri M, Ruotolo G, Sampietro T, Sessa A, Vaudo G, Cantafora A, Veglia F, Calandra S, Bertolini S, Franceschini G. The molecular basis of lecithin:cholesterol acyltransferase deficiency syndromes: a comprehensive study of molecular and biochemical findings in 13 unrelated Italian families. Arterioscler Thromb Vasc Biol. 2005 Sep;25(9):1972-8. Epub 2005 Jun 30. Citation on PubMedDuivenvoorden R, Holleboom AG, van den Bogaard B, Nederveen AJ, de Groot E, Hutten BA, Schimmel AW, Hovingh GK, Kastelein JJ, Kuivenhoven JA, Stroes ES. Carriers of lecithin cholesterol acyltransferase gene mutations have accelerated atherogenesis as assessed by carotid 3.0-T magnetic resonance imaging [corrected]. J Am Coll Cardiol. 2011 Dec 6;58(24):2481-7. doi: 10.1016/j.jacc.2010.11.092. Erratum in: J Am Coll Cardiol. 2012 Jan 10;59(2):196. Citation on PubMedOMIM: LECITHIN:CHOLESTEROL ACYLTRANSFERASEReshetnyak Y, Tchedre KT, Nair MP, Pritchard PH, Lacko AG. Structural differences between wild-type and fish eye disease mutant of lecithin:cholesterol acyltransferase. J Biomol Struct Dyn. 2006 Aug;24(1):75-82. Citation on PubMedRoshan B, Ganda OP, Desilva R, Ganim RB, Ward E, Haessler SD, Polisecki EY, Asztalos BF, Schaefer EJ. Homozygous lecithin:cholesterol acyltransferase (LCAT) deficiency due to a new loss of function mutation and review of the literature. J Clin Lipidol. 2011 Nov-Dec;5(6):493-9. doi: 10.1016/j.jacl.2011.07.002. Epub 2011 Aug 23. Review. Citation on PubMed or Free article on PubMed CentralSavel J, Lafitte M, Pucheu Y, Pradeau V, Tabarin A, Couffinhal T. Very low levels of HDL cholesterol and atherosclerosis, a variable relationship--a review of LCAT deficiency. Vasc Health Risk Manag. 2012;8:357-61. doi: 10.2147/VHRM.S29985. Epub 2012 Jun 5. Review. Citation on PubMed or Free article on PubMed Centralvan den Bogaard B, Holleboom AG, Duivenvoorden R, Hutten BA, Kastelein JJ, Hovingh GK, Kuivenhoven JA, Stroes ES, van den Born BJ. Patients with low HDL-cholesterol caused by mutations in LCAT have increased arterial stiffness. Atherosclerosis. 2012 Dec;225(2):481-5. doi: 10.1016/j.atherosclerosis.2012.09.022. Epub 2012 Sep 27. Citation on PubMed \n",
            "CLEC18A\t\n",
            "PDPR\t\n",
            "WDR90\t\n",
            "HYDIN\t   From NCBI Gene:This gene encodes a protein that may be involved in cilia motility. Mutations in this gene cause of autosomal recessive primary ciliary dyskinesia-5, a disorder characterized by the accumulation of cerebrospinal fluid within the ventricles of the brain. A duplicate copy of this gene has been found in humans on chromosome 1. [provided by RefSeq, Jan 2013]From UniProt:Required for ciliary motility.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:1q21.1 microdeletionPrimary ciliary dyskinesiaFrom NCBI Gene:Ciliary dyskinesia, primary, 5From UniProt:Ciliary dyskinesia, primary, 5 (CILD5): An autosomal recessive form of primary dyskinesia, a disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia; reduced fertility is often observed in male patients due to abnormalities of sperm tails. Half of the patients exhibit randomization of left-right body asymmetry and situs inversus, due to dysfunction of monocilia at the embryonic node. Primary ciliary dyskinesia associated with situs inversus is referred to as Kartagener syndrome. CILD5 is characterized by early onset of a progressive decline in lung function due to an inability to clear mucus and particles from the airways. Affected individuals have recurrent infections of the sinuses, ears, airways, and lungs. Sperm motility is also decreased. Individuals with CILD5 do not have situs inversus. [MIM:608647]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q22.2, which is the long (q) arm of chromosome 16 at position 22.2Molecular Location: base pairs 70,802,084 to 71,230,722 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CILD5HYDIN1HYDIN2PPP1R31     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HYDIN      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 5OMIM: HYDROCEPHALUS-INDUCING, MOUSE, HOMOLOG OF      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HYDIN  OMIM: CILIARY DYSKINESIA, PRIMARY, 5OMIM: HYDROCEPHALUS-INDUCING, MOUSE, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF19\t\n",
            "PKD1L3\t\n",
            "PMFBP1\t\n",
            "RHOT2\t\n",
            "NPIPB15\t\n",
            "CLEC18B\t\n",
            "ZFP1\t\n",
            "CTRB1\t\n",
            "CNTNAP4\t\n",
            "ADAMTS18\t   From NCBI Gene:This gene encodes a member of the ADAMTS (a disintegrin and metalloproteinase with thrombospondin motifs) protein family. ADAMTS family members share several distinct protein modules, including a propeptide region, a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. Individual members of this family differ in the number of C-terminal TS motifs, and some have unique C-terminal domains. The encoded preproprotein is proteolytically processed to generate the mature protein, which may regulate hemostatic balance and function as a tumor suppressor. Mutations in this gene may be associated with microcornea, myopic chorioretinal atrophy, and telecanthus (MMCAT) and cone-rod dystrophy in human patients. [provided by RefSeq, May 2016]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Microcornea, myopic chorioretinal atrophy, and telecanthusFrom UniProt:Microcornea, myopic chorioretinal atrophy, and telecanthus (MMCAT): A ocular syndrome characterized by microcornea and myopic chorioretinal atrophy. Microcornea is defined by a corneal diameter inferior to 10 mm in both meridians in an otherwise normal eye. In addition to ocular findings, some patients have telecanthus and posteriorly rotated ears. [MIM:615458]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q23.1, which is the long (q) arm of chromosome 16 at position 23.1Molecular Location: base pairs 77,282,128 to 77,435,114 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADAMTS21KNO2MMCAT     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADAMTS18      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: A DISINTEGRIN-LIKE AND METALLOPROTEINASE WITH THROMBOSPONDIN TYPE 1 MOTIF, 18OMIM: MICROCORNEA, MYOPIC CHORIORETINAL ATROPHY, AND TELECANTHUS      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADAMTS18  OMIM: A DISINTEGRIN-LIKE AND METALLOPROTEINASE WITH THROMBOSPONDIN TYPE 1 MOTIF, 18OMIM: MICROCORNEA, MYOPIC CHORIORETINAL ATROPHY, AND TELECANTHUS  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "WWOX\t   From NCBI Gene:This gene encodes a member of the short-chain dehydrogenases/reductases (SDR) protein family. This gene spans the FRA16D common chromosomal fragile site and appears to function as a tumor suppressor gene. Expression of the encoded protein is able to induce apoptosis, while defects in this gene are associated with multiple types of cancer. Disruption of this gene is also associated with autosomal recessive spinocerebellar ataxia 12. Disruption of a similar gene in mouse results in impaired steroidogenesis, additionally suggesting a metabolic function for the protein. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2014]From UniProt:Putative oxidoreductase. Acts as a tumor suppressor and plays a role in apoptosis. Required for normal bone development (By similarity). May function synergistically with p53/TP53 to control genotoxic stress-induced cell death. Plays a role in TGFB1 signaling and TGFB1-mediated cell death. May also play a role in tumor necrosis factor (TNF)-mediated cell death. Inhibits Wnt signaling, probably by sequestering DVL2 in the cytoplasm.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spinocerebellar ataxia, autosomal recessive 12Epileptic encephalopathy, early infantile, 28Malignant tumor of esophagusFrom UniProt:Spinocerebellar ataxia, autosomal recessive, 12 (SCAR12): Spinocerebellar ataxia is a clinically and genetically heterogeneous group of cerebellar disorders. Patients show progressive incoordination of gait and often poor coordination of hands, speech and eye movements, due to degeneration of the cerebellum with variable involvement of the brainstem and spinal cord. SCAR12 is additionally characterized by onset of generalized seizures in infancy, and delayed psychomotor development with mental retardation. Some patients may also show spasticity. [MIM:614322]Epileptic encephalopathy, early infantile, 28 (EIEE28): A form of epileptic encephalopathy, a heterogeneous group of severe childhood onset epilepsies characterized by refractory seizures, neurodevelopmental impairment, and poor prognosis. Development is normal prior to seizure onset, after which cognitive and motor delays become apparent. [MIM:616211]Defects in WWOX may be involved in several cancer types. The gene spans the second most common chromosomal fragile site (FRA16D) which is frequently altered in cancers (PubMed:10861292). Alteration of the expression and expression of some isoforms is associated with cancers. However, it is still unclear if alteration of WWOX is directly implicated in cancerogenesis or if it corresponds to a secondary effect (PubMed:10861292, PubMed:11572989, PubMed:15266310, PubMed:15073125, PubMed:15131042).Esophageal cancer (ESCR): A malignancy of the esophagus. The most common types are esophageal squamous cell carcinoma and adenocarcinoma. Cancer of the esophagus remains a devastating disease because it is usually not detected until it has progressed to an advanced incurable stage. [MIM:133239]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q23.1-q23.2, which is the long (q) arm of chromosome 16 between positions 23.1 and 23.2Molecular Location: base pairs 78,099,413 to 79,212,667 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        D16S432EEIEE28FORFRA16DHHCMA56PRO0128SCAR12SDR41C1WOX1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of WWOX      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: EPILEPTIC ENCEPHALOPATHY, EARLY INFANTILE, 28OMIM: ESOPHAGEAL CANCEROMIM: SPINOCEREBELLAR ATAXIA, AUTOSOMAL RECESSIVE 12OMIM: WW DOMAIN-CONTAINING OXIDOREDUCTASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of WWOX  OMIM: EPILEPTIC ENCEPHALOPATHY, EARLY INFANTILE, 28OMIM: ESOPHAGEAL CANCEROMIM: SPINOCEREBELLAR ATAXIA, AUTOSOMAL RECESSIVE 12OMIM: WW DOMAIN-CONTAINING OXIDOREDUCTASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PKD1L2\t\n",
            "SDR42E1\t\n",
            "TAF1C\t\n",
            "KCNG4\t\n",
            "METTL22\t\n",
            "ZCCHC14\t\n",
            "JPH3\t   The JPH3 gene provides instructions for making a protein called junctophilin-3, which is found primarily in the brain.  Although the exact function of this protein is unclear, researchers believe that it plays a role in the formation of a structure called the junctional membrane complex.  This complex connects certain channels inside cells with other channels at the cell surface.   The junctional membrane complex appears to be involved in the release of charged calcium atoms (calcium ions), which are critical for transmitting signals within cells.  As part of the junctional membrane complex, junctophilin-3 is probably involved in signaling within and between nerve cells (neurons) in the brain.One region of the JPH3 gene contains a particular DNA segment known as a CAG/CTG trinucleotide repeat. This segment is made up of a series of three DNA building blocks (nucleotides) that appear multiple times in a row. Normally, the CAG/CTG segment is repeated 6 to 28 times within the gene.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Huntington disease-like syndrome     A particular type of mutation in the JPH3 gene has been found to cause signs and symptoms that resemble those of Huntington disease, including uncontrolled movements, emotional problems, and loss of thinking ability. Researchers have named this condition Huntington disease-like 2 (HDL2).The mutation associated with HDL2 increases the size of the CAG/CTG trinucleotide repeat in the JPH3 gene.  People with this condition have 44 to 59 CAG/CTG repeats.  People with 29 to about 43 CAG/CTG repeats  may or may not develop the signs and symptoms of HDL2.Researchers are working to determine the effects of the abnormally large CAG/CTG segment.  They believe that the mutated JPH3 gene produces an altered version of messenger RNA, which is a molecular blueprint of the gene that is normally used for protein production.  The abnormal messenger RNA forms clumps inside neurons that interfere with the normal functions of these cells.  The dysfunction and eventual death of neurons in certain areas of the brain underlie the signs and symptoms of HDL2. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A particular type of mutation in the JPH3 gene has been found to cause signs and symptoms that resemble those of Huntington disease, including uncontrolled movements, emotional problems, and loss of thinking ability. Researchers have named this condition Huntington disease-like 2 (HDL2).The mutation associated with HDL2 increases the size of the CAG/CTG trinucleotide repeat in the JPH3 gene.  People with this condition have 44 to 59 CAG/CTG repeats.  People with 29 to about 43 CAG/CTG repeats  may or may not develop the signs and symptoms of HDL2.Researchers are working to determine the effects of the abnormally large CAG/CTG segment.  They believe that the mutated JPH3 gene produces an altered version of messenger RNA, which is a molecular blueprint of the gene that is normally used for protein production.  The abnormal messenger RNA forms clumps inside neurons that interfere with the normal functions of these cells.  The dysfunction and eventual death of neurons in certain areas of the brain underlie the signs and symptoms of HDL2. More About This Health Condition    Cytogenetic Location: 16q24.2, which is the long (q) arm of chromosome 16 at position 24.2Molecular Location: base pairs 87,601,835 to 87,698,156 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAGL237FLJ44707HDL2JP-3JP3JPH3_HUMANjunctophilin-3junctophilin type 3TNRC22trinucleotide repeat containing 22     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Huntington Disease-Like 2      Tests Listed in the Genetic Testing Registry (1 link)     Tests of JPH3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     JUNCTOPHILIN 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Huntington Disease-Like 2  Tests of JPH3  PubMed  JUNCTOPHILIN 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Greenstein PE, Vonsattel JP, Margolis RL, Joseph JT. Huntington's disease like-2 neuropathology. Mov Disord. 2007 Jul 30;22(10):1416-23. Citation on PubMedMargolis RL, O'Hearn E, Rosenblatt A, Willour V, Holmes SE, Franz ML, Callahan C, Hwang HS, Troncoso JC, Ross CA. A disorder similar to Huntington's disease is associated with a novel CAG repeat expansion. Ann Neurol. 2001 Dec;50(6):373-80. Citation on PubMedNishi M, Mizushima A, Nakagawara Ki, Takeshima H. Characterization of human junctophilin subtype genes. Biochem Biophys Res Commun. 2000 Jul 14;273(3):920-7. Citation on PubMedRudnicki DD, Pletnikova O, Vonsattel JP, Ross CA, Margolis RL. A comparison of huntington disease and huntington disease-like 2 neuropathology. J Neuropathol Exp Neurol. 2008 Apr;67(4):366-74. doi: 10.1097/NEN.0b013e31816b4aee. Citation on PubMedTakeshima H, Komazaki S, Nishi M, Iino M, Kangawa K. Junctophilins: a novel family of junctional membrane complex proteins. Mol Cell. 2000 Jul;6(1):11-22. Citation on PubMedWalker RH, Jankovic J, O'Hearn E, Margolis RL. Phenotypic features of Huntington's disease-like 2. Mov Disord. 2003 Dec;18(12):1527-30. Citation on PubMed \n",
            "ZFPM1\t\n",
            "CYBA\t   The CYBA gene provides instructions for making a protein called the cytochrome b-245 alpha chain (also known as p22-phox). This protein is one part (subunit) of a group of proteins that forms an enzyme complex called NADPH oxidase, which plays an essential role in the immune system. Within this complex, the cytochrome b-245 alpha chain has a beta chain partner (produced from the CYBB gene). Both alpha and beta chains are required for either to function, and the NADPH oxidase complex requires both chains in order to be functional. NADPH oxidase is primarily active in immune system cells called phagocytes. These cells catch and destroy foreign invaders such as bacteria and fungi. NADPH oxidase is also thought to regulate the activity of immune cells called neutrophils. These cells play a role in adjusting the inflammatory response to optimize healing and reduce injury to the body.The presence of foreign invaders stimulates phagocytes and triggers the assembly of NADPH oxidase. This enzyme participates in a chemical reaction that converts oxygen to a toxic molecule called superoxide. Superoxide is used to generate several other compounds, including hydrogen peroxide (a strong disinfectant) and hypochlorous acid (the active ingredient in bleach). These highly reactive, toxic substances are known as reactive oxygen species. Phagocytes use these substances to kill foreign invaders, preventing them from reproducing in the body and causing illness.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Chronic granulomatous disease     More than 40 mutations in the CYBA gene have been found to cause chronic granulomatous disease. People with this disorder are at increased risk of developing recurrent episodes of infection and inflammation due to a weakened immune system. Mutations in the CYBA gene cause less than 5 percent of all cases of this condition. Most of these mutations change single building blocks of protein (amino acids) in the cytochrome b-245 alpha chain or cause it to be abnormally short and nonfunctional. An altered protein not only diminishes the function of the alpha chain, but the function of its beta chain partner as well. Without these subunits, NADPH oxidase cannot assemble or function properly. As a result, phagocytes are unable to produce reactive oxygen species to kill foreign invaders, and neutrophil activity is not regulated. A lack of NADPH oxidase leaves affected individuals vulnerable to many types of infection and excessive inflammation. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 40 mutations in the CYBA gene have been found to cause chronic granulomatous disease. People with this disorder are at increased risk of developing recurrent episodes of infection and inflammation due to a weakened immune system. Mutations in the CYBA gene cause less than 5 percent of all cases of this condition. Most of these mutations change single building blocks of protein (amino acids) in the cytochrome b-245 alpha chain or cause it to be abnormally short and nonfunctional. An altered protein not only diminishes the function of the alpha chain, but the function of its beta chain partner as well. Without these subunits, NADPH oxidase cannot assemble or function properly. As a result, phagocytes are unable to produce reactive oxygen species to kill foreign invaders, and neutrophil activity is not regulated. A lack of NADPH oxidase leaves affected individuals vulnerable to many types of infection and excessive inflammation. More About This Health Condition    Cytogenetic Location: 16q24.2, which is the long (q) arm of chromosome 16 at position 24.2Molecular Location: base pairs 88,643,289 to 88,651,084 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CY24A_HUMANcytochrome b-245 light chaincytochrome b-245, alpha polypeptidecytochrome b light chaincytochrome b(558) alpha chaincytochrome b, alpha polypeptidecytochrome b558 subunit alphaflavocytochrome b-558 alpha polypeptideneutrophil cytochrome b 22 kDa polypeptidep22phoxsuperoxide-generating NADPH oxidase light chain subunit     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Emory University NADPH Oxidase PageImmunobiology: The Immune System in Health and Disease (2001, fifth edition): After entering tissues, many pathogens are recognized, ingested, and killed by phagocytesImmunobiology: The Immune System in Health and Disease (2001, fifth edition): Defects in phagocytic cells permit widespread bacterial infections      Clinical Information from GeneReviews (1 link)     Chronic Granulomatous Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CYBA      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CYTOCHROME b(-245), ALPHA SUBUNIT      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportID Bases: CYBA Gene Mutation DatabaseMonarch InitiativeNCBI GeneUniProt      Emory University NADPH Oxidase PageImmunobiology: The Immune System in Health and Disease (2001, fifth edition): After entering tissues, many pathogens are recognized, ingested, and killed by phagocytesImmunobiology: The Immune System in Health and Disease (2001, fifth edition): Defects in phagocytic cells permit widespread bacterial infections  Chronic Granulomatous Disease  Tests of CYBA  PubMed  CYTOCHROME b(-245), ALPHA SUBUNIT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportID Bases: CYBA Gene Mutation DatabaseMonarch InitiativeNCBI GeneUniProt  OMIM: CYTOCHROME b(-245), ALPHA SUBUNITKannengiesser C, Grard B, El Benna J, Henri D, Kroviarski Y, Chollet-Martin S, Gougerot-Pocidalo MA, Elbim C, Grandchamp B. Molecular epidemiology of chronic granulomatous disease in a series of 80 kindreds: identification of 31 novel mutations. Hum Mutat. 2008 Sep;29(9):E132-49. doi: 10.1002/humu.20820. Citation on PubMedRoos D, Kuhns DB, Maddalena A, Bustamante J, Kannengiesser C, de Boer M, van Leeuwen K, Kker MY, Wolach B, Roesler J, Malech HL, Holland SM, Gallin JI, Stasia MJ. Hematologically important mutations: the autosomal recessive forms of chronic granulomatous disease (second update). Blood Cells Mol Dis. 2010 Apr 15;44(4):291-9. doi: 10.1016/j.bcmd.2010.01.009. Epub 2010 Feb 18. Review. Citation on PubMed or Free article on PubMed CentralStasia MJ, Li XJ. Genetics and immunopathology of chronic granulomatous disease. Semin Immunopathol. 2008 Jul;30(3):209-35. doi: 10.1007/s00281-008-0121-8. Epub 2008 May 29. Review. Citation on PubMedSumimoto H. Structure, regulation and evolution of Nox-family NADPH oxidases that produce reactive oxygen species. FEBS J. 2008 Jul;275(13):3249-77. doi: 10.1111/j.1742-4658.2008.06488.x. Epub 2008 May 30. Review. Erratum in: FEBS J. 2008 Aug;275(15):3984. Citation on PubMed \n",
            "CTU2\t   From NCBI Gene:This gene encodes a protein which is involved in the post-transcriptional modification of transfer RNAs (tRNAs). The encoded protein plays a role in thiolation of uridine residue present at the wobble position in a subset of tRNAs, resulting in enhanced codon reading accuracy. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jan 2016]From UniProt:Plays a central role in 2-thiolation of mcm(5)S(2)U at tRNA wobble positions of tRNA(Lys), tRNA(Glu) and tRNA(Gln). May act by forming a heterodimer with CTU1/ATPBD3 that ligates sulfur from thiocarboxylated URM1 onto the uridine of tRNAs at wobble position.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:MICROCEPHALY, FACIAL DYSMORPHISM, RENAL AGENESIS, AND AMBIGUOUS GENITALIA SYNDROME     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q24.3, which is the long (q) arm of chromosome 16 at position 24.3Molecular Location: base pairs 88,706,463 to 88,715,386 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C16orf84MFRGNCS2UPF0432     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CTU2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CYTOSOLIC THIOURIDYLASE, SUBUNIT 2OMIM: MICROCEPHALY, FACIAL DYSMORPHISM, RENAL AGENESIS, AND AMBIGUOUS GENITALIA SYNDROME      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CTU2  OMIM: CYTOSOLIC THIOURIDYLASE, SUBUNIT 2OMIM: MICROCEPHALY, FACIAL DYSMORPHISM, RENAL AGENESIS, AND AMBIGUOUS GENITALIA SYNDROME  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PIEZO1\t   From NCBI Gene:The protein encoded by this gene is a mechanically-activated ion channel that links mechanical forces to biological signals. The encoded protein contains 36 transmembrane domains and functions as a homotetramer. Defects in this gene have been associated with dehydrated hereditary stomatocytosis. [provided by RefSeq, Jul 2015]From UniProt:Pore-forming subunit of a mechanosensitive non-specific cation channel (PubMed:23479567, PubMed:23695678). Generates currents characterized by a linear current-voltage relationship that are sensitive to ruthenium red and gadolinium. Plays a key role in epithelial cell adhesion by maintaining integrin activation through R-Ras recruitment to the ER, most probably in its activated state, and subsequent stimulation of calpain signaling (PubMed:20016066). In the kidney, may contribute to the detection of intraluminal pressure changes and to urine flow sensing. Acts as shear-stress sensor that promotes endothelial cell organization and alignment in the direction of blood flow through calpain activation (PubMed:25119035). Plays a key role in blood vessel formation and vascular structure in both development and adult physiology (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:XerocytosisLymphedema, hereditary, IIIFrom UniProt:Dehydrated hereditary stomatocytosis 1 with or without pseudohyperkalemia and/or perinatal edema (DHS1): An autosomal dominant hemolytic anemia characterized by primary erythrocyte dehydration. DHS erythrocytes exhibit decreased total cation and potassium content that are not accompanied by a proportional net gain of sodium and water. DHS patients typically exhibit mild to moderate compensated hemolytic anemia, with an increased erythrocyte mean corpuscular hemoglobin concentration and a decreased osmotic fragility, both of which reflect cellular dehydration. Patients may also show perinatal edema and pseudohyperkalemia due to loss of potassium from red cells stored at room temperature. A minor proportion of red cells appear as stomatocytes on blood films. Complications such as splenomegaly and cholelithiasis, resulting from increased red cell trapping in the spleen and elevated bilirubin levels, respectively, may occur. The course of DHS is frequently associated with iron overload, which may lead to hepatosiderosis. [MIM:194380]Lymphedema, hereditary, 3 (LMPH3): A severe form of lymphedema, a chronic disabling condition which results in swelling of the extremities due to altered lymphatic flow. Patients with lymphedema suffer from recurrent local infections, and physical impairment. LMPH3 manifests as generalized lymphatic dysplasia, characterized by uniform, widespread lymphedema affecting all segments of the body, with systemic involvement such as intestinal and/or pulmonary lymphangiectasia, pleural effusions, chylothoraces and/or pericardial effusions, and with a high incidence of non-immune hydrops fetalis. [MIM:616843]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 16q24.3, which is the long (q) arm of chromosome 16 at position 24.3Molecular Location: base pairs 88,715,338 to 88,785,220 on chromosome 16 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DHSFAM38ALMPH3LMPHM6Mib     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PIEZO1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DEHYDRATED HEREDITARY STOMATOCYTOSIS 1 WITH OR WITHOUT PSEUDOHYPERKALEMIA AND/OR PERINATAL EDEMAOMIM: LYMPHATIC MALFORMATION 6OMIM: PIEZO-TYPE MECHANOSENSITIVE ION CHANNEL COMPONENT 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PIEZO1  OMIM: DEHYDRATED HEREDITARY STOMATOCYTOSIS 1 WITH OR WITHOUT PSEUDOHYPERKALEMIA AND/OR PERINATAL EDEMAOMIM: LYMPHATIC MALFORMATION 6OMIM: PIEZO-TYPE MECHANOSENSITIVE ION CHANNEL COMPONENT 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF778\t\n",
            "SPIRE2\t\n",
            "PRDM7\t\n",
            "MYH8\t   From NCBI Gene:Myosins are actin-based motor proteins that function in the generation of mechanical force in eukaryotic cells. Muscle myosins are heterohexamers composed of 2 myosin heavy chains and 2 pairs of nonidentical myosin light chains. This gene encodes a member of the class II or conventional myosin heavy chains, and functions in skeletal muscle contraction. This gene is predominantly expressed in fetal skeletal muscle. This gene is found in a cluster of myosin heavy chain genes on chromosome 17. A mutation in this gene results in trismus-pseudocamptodactyly syndrome. [provided by RefSeq, Sep 2009]From UniProt:Muscle contraction.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Hecht syndromeCarney complex variantFrom UniProt:Carney complex variant (CACOV): Carney complex is a multiple neoplasia syndrome characterized by spotty skin pigmentation, cardiac and other myxomas, endocrine tumors, and psammomatous melanotic schwannomas. Familial cardiac myxomas are associated with spotty pigmentation of the skin and other phenotypes, including primary pigmented nodular adrenocortical dysplasia, extracardiac (frequently cutaneous) myxomas, schwannomas, and pituitary, thyroid, testicular, bone, ovarian, and breast tumors. Cardiac myxomas do not develop in all patients with the Carney complex, but affected patients have at least two features of the complex or one feature and a clinically significant family history. [MIM:608837]Arthrogryposis, distal, 7 (DA7): A form of distal arthrogryposis, a disease characterized by congenital joint contractures that mainly involve two or more distal parts of the limbs, in the absence of a primary neurological or muscle disease. DA7 is characterized by an inability to open the mouth fully (trismus) and pseudocamptodactyly in which wrist dorsiflexion, but not volarflexion, produces involuntary flexion contracture of distal and proximal interphalangeal joints. Additional features include shortened hamstring muscles and short stature. [MIM:158300]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.1, which is the short (p) arm of chromosome 17 at position 13.1Molecular Location: base pairs 10,390,325 to 10,421,950 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DA7gtMHC-FMyHC-periMyHC-pn     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYH8      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ARTHROGRYPOSIS, DISTAL, TYPE 7OMIM: CARNEY COMPLEX VARIANTOMIM: MYOSIN, HEAVY CHAIN 8, SKELETAL MUSCLE, PERINATAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MYH8  OMIM: ARTHROGRYPOSIS, DISTAL, TYPE 7OMIM: CARNEY COMPLEX VARIANTOMIM: MYOSIN, HEAVY CHAIN 8, SKELETAL MUSCLE, PERINATAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYH4\t\n",
            "DNAH9\t   From NCBI Gene:This gene encodes the heavy chain subunit of axonemal dynein, a large multi-subunit molecular motor. Axonemal dynein attaches to microtubules and hydrolyzes ATP to mediate the movement of cilia and flagella. The gene expresses at least two transcript variants; additional variants have been described, but their full length nature has not been determined. [provided by RefSeq, Jul 2008]From UniProt:Force generating protein of respiratory cilia. Produces force towards the minus ends of microtubules. Dynein has ATPase activity; the force-producing power stroke is thought to occur on release of ADP.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:CILIARY DYSKINESIA, PRIMARY, 40     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p12, which is the short (p) arm of chromosome 17 at position 12Molecular Location: base pairs 11,598,431 to 11,970,168 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CILD40DNAH17LDnahc9DNEL1DYH9HL-20HL20     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DNAH9      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 40OMIM: DYNEIN, AXONEMAL, HEAVY CHAIN 9      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DNAH9  OMIM: CILIARY DYSKINESIA, PRIMARY, 40OMIM: DYNEIN, AXONEMAL, HEAVY CHAIN 9  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CDRT1\t\n",
            "TBC1D26\t\n",
            "NCOR1\t\n",
            "CENPV\t\n",
            "WDR81\t   From NCBI Gene:This gene encodes a multi-domain transmembrane protein which is predominantly expressed in the brain and is thought to play a role in endolysosomal trafficking. Mutations in this gene are associated with an autosomal recessive form of a syndrome exhibiting cerebellar ataxia, cognitive disability, and disequilibrium (CAMRQ2). Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2017]From UniProt:Functions as a negative regulator of the PI3 kinase/PI3K activity associated with endosomal membranes via BECN1, a core subunit of the PI3K complex. By modifying the phosphatidylinositol 3-phosphate/PtdInsP3 content of endosomal membranes may regulate endosome fusion, recycling, sorting and early to late endosome transport (PubMed:26783301). It is for instance, required for the delivery of cargos like BST2/tetherin from early to late endosome and thereby participates indirectly to their degradation by the lysosome (PubMed:27126989). May also play a role in aggrephagy, the macroautophagic degradation of ubiquitinated protein aggregates. In this process, may regulate the interaction of SQSTM1 with ubiquitinated proteins and also recruit MAP1LC3C (PubMed:28404643). May also be involved in maintenance of normal mitochondrial structure and organization (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Cerebellar ataxia, mental retardation, and dysequilibrium syndrome 2HYDROCEPHALUS, CONGENITAL, 3, WITH BRAIN ANOMALIESFrom UniProt:Cerebellar ataxia, mental retardation, and dysequilibrium syndrome 2 (CAMRQ2): A congenital cerebellar ataxia associated with cerebellar hypoplasia, mental retardation, and inability to walk bipedally, resulting in quadrupedal locomotion as a functional adaptation. Additional findings include generalized brain atrophy and mild hypoplasia of the corpus callosum. [MIM:610185]Hydrocephalus, congenital, 3, with brain anomalies (HYC3): A form of congenital hydrocephalus, a disease characterized by onset in utero of enlarged ventricles due to accumulation of ventricular cerebrospinal fluid. HYC3 features include enlarged ventricles, hypoplastic or absent cerebellum, holoprosencephaly and Dandy-Walker malformation. Most patients die in utero or shortly after birth. HYC3 inheritance is autosomal recessive. [MIM:617967]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.3, which is the short (p) arm of chromosome 17 at position 13.3Molecular Location: base pairs 1,716,523 to 1,738,599 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAMRQ2HYC3PPP1R166SORF-2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of WDR81      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CEREBELLAR ATAXIA, MENTAL RETARDATION, AND DYSEQUILIBRIUM SYNDROME 2OMIM: HYDROCEPHALUS, CONGENITAL, 3, WITH BRAIN ANOMALIESOMIM: WD REPEAT-CONTAINING PROTEIN 81      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of WDR81  OMIM: CEREBELLAR ATAXIA, MENTAL RETARDATION, AND DYSEQUILIBRIUM SYNDROME 2OMIM: HYDROCEPHALUS, CONGENITAL, 3, WITH BRAIN ANOMALIESOMIM: WD REPEAT-CONTAINING PROTEIN 81  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SMYD4\t\n",
            "MPRIP\t\n",
            "PEMT\t\n",
            "RAI1\t   The RAI1 gene provides instructions for making a protein that is active in cells throughout the body, particularly nerve cells (neurons) in the brain. Located in the nucleus of the cell, the RAI1 protein helps control the activity (expression) of certain genes. Most of the genes regulated by RAI1 have not been identified. However, studies suggest that this protein controls the expression of several genes involved in daily (circadian) rhythms, such as the sleep-wake cycle. The RAI1 protein also appears to play a role in development of the brain and of bones in the head and face (craniofacial bones).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Potocki-Lupski syndrome     Having an extra copy of the RAI1 gene in each cell is thought to underlie many of the major features of Potocki-Lupski syndrome. This condition is characterized by delayed development, mild to moderate intellectual disability, behavioral problems including autism spectrum disorder (which affects social interaction and communication), sleep disturbances, and other health problems. The condition results from abnormal copying (duplication) of a small piece of the short (p) arm of chromosome 17 at position p11.2. In about two-thirds of affected individuals, the duplicated segment includes approximately 3.7 million DNA building blocks (base pairs), also written as 3.7 megabases (Mb). (A deletion of this segment causes Smith-Magenis syndrome, described below.) In the remaining one-third of cases, the duplication is larger or smaller, ranging from less than 1 Mb to almost 20 Mb. All of these duplications affect one of the two copies of chromosome 17 in each cell.All of the duplications known to cause Potocki-Lupski syndrome contain the RAI1 gene. Studies suggest that the duplication increases the amount of RAI1 protein, which disrupts the expression of genes that influence circadian rhythms. These changes may account for the sleep disturbances that occur with Potocki-Lupski syndrome. Too much RAI1 protein may also disrupt brain development, which could account for delayed development, intellectual disability, behavioral problems, and other neurological features of this condition. Development of the bones in the head and face may also be affected, leading to subtle facial differences in people with Potocki-Lupski syndrome. More About This Health Condition      Smith-Magenis syndrome     Researchers believe that a partial or total loss of function of the RAI1 gene accounts for most of the signs and symptoms of Smith-Magenis syndrome. The major features of this condition include mild to moderate intellectual disability, delayed speech and language skills, distinctive facial features, sleep disturbances, behavioral problems, and other abnormalities affecting many parts of the body.In most people with Smith-Magenis syndrome, the condition results from a deletion of a small piece of chromosome 17 in each cell. Most often, this chromosome segment, located at 17p11.2, is the same one that is duplicated in Potocki-Lupski syndrome (described above). Occasionally the deletion is larger or smaller. All of the deletions affect one of the two copies of chromosome 17 in each cell.All of the deletions known to cause Smith-Magenis syndrome contain the RAI1 gene. Studies suggest that the deletion leads to a reduced amount of RAI1 protein in cells, which disrupts the expression of genes involved in circadian rhythms. These changes may account for the sleep disturbances that occur with Smith-Magenis syndrome. It is unclear how a loss of one copy of the RAI1 gene leads to the other physical, mental, and behavioral problems associated with this condition.A small percentage of people with Smith-Magenis syndrome have a mutation in the RAI1 gene instead of a chromosomal deletion. Although these individuals have many of the major features of the condition, they are less likely than people with a deletion to have certain other features, including short stature, hearing loss, and heart or kidney abnormalities. It is likely that, in people with a deletion, the loss of other genes in the deleted region accounts for these additional signs and symptoms; the role of these genes is under study. More About This Health Condition      Yuan-Harel-Lupski syndrome     Having an extra copy of the RAI1 gene in each cell is thought to underlie many of the major features of Yuan-Harel-Lupski (YUHAL) syndrome, which is characterized by multiple neurological problems. This condition results from duplication of a small piece of chromosome 17 in a region designated p12-p11.2. In YUHAL syndrome, the duplicated segments can range in size from 3.2 Mb to 19.7 Mb. These duplications affect one of the two copies of chromosome 17 in each cell.The duplications that cause YUHAL syndrome all contain the RAI1 gene and a nearby gene called PMP22; the segments may also contain additional genes. Some features of YUHAL syndrome, such as delayed development, behavioral and sleep problems, and unusual facial features, are similar to those of Potocki-Lupski syndrome (described above) and are likely caused by an extra copy of the RAI1 gene. Other features of YUHAL syndrome, such as muscle weakness and decreased sensitivity to touch, heat, and cold in the lower legs and feet, likely result from an extra copy of the PMP22 gene. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Having an extra copy of the RAI1 gene in each cell is thought to underlie many of the major features of Potocki-Lupski syndrome. This condition is characterized by delayed development, mild to moderate intellectual disability, behavioral problems including autism spectrum disorder (which affects social interaction and communication), sleep disturbances, and other health problems. The condition results from abnormal copying (duplication) of a small piece of the short (p) arm of chromosome 17 at position p11.2. In about two-thirds of affected individuals, the duplicated segment includes approximately 3.7 million DNA building blocks (base pairs), also written as 3.7 megabases (Mb). (A deletion of this segment causes Smith-Magenis syndrome, described below.) In the remaining one-third of cases, the duplication is larger or smaller, ranging from less than 1 Mb to almost 20 Mb. All of these duplications affect one of the two copies of chromosome 17 in each cell.All of the duplications known to cause Potocki-Lupski syndrome contain the RAI1 gene. Studies suggest that the duplication increases the amount of RAI1 protein, which disrupts the expression of genes that influence circadian rhythms. These changes may account for the sleep disturbances that occur with Potocki-Lupski syndrome. Too much RAI1 protein may also disrupt brain development, which could account for delayed development, intellectual disability, behavioral problems, and other neurological features of this condition. Development of the bones in the head and face may also be affected, leading to subtle facial differences in people with Potocki-Lupski syndrome. More About This Health Condition  Researchers believe that a partial or total loss of function of the RAI1 gene accounts for most of the signs and symptoms of Smith-Magenis syndrome. The major features of this condition include mild to moderate intellectual disability, delayed speech and language skills, distinctive facial features, sleep disturbances, behavioral problems, and other abnormalities affecting many parts of the body.In most people with Smith-Magenis syndrome, the condition results from a deletion of a small piece of chromosome 17 in each cell. Most often, this chromosome segment, located at 17p11.2, is the same one that is duplicated in Potocki-Lupski syndrome (described above). Occasionally the deletion is larger or smaller. All of the deletions affect one of the two copies of chromosome 17 in each cell.All of the deletions known to cause Smith-Magenis syndrome contain the RAI1 gene. Studies suggest that the deletion leads to a reduced amount of RAI1 protein in cells, which disrupts the expression of genes involved in circadian rhythms. These changes may account for the sleep disturbances that occur with Smith-Magenis syndrome. It is unclear how a loss of one copy of the RAI1 gene leads to the other physical, mental, and behavioral problems associated with this condition.A small percentage of people with Smith-Magenis syndrome have a mutation in the RAI1 gene instead of a chromosomal deletion. Although these individuals have many of the major features of the condition, they are less likely than people with a deletion to have certain other features, including short stature, hearing loss, and heart or kidney abnormalities. It is likely that, in people with a deletion, the loss of other genes in the deleted region accounts for these additional signs and symptoms; the role of these genes is under study. More About This Health Condition  Having an extra copy of the RAI1 gene in each cell is thought to underlie many of the major features of Yuan-Harel-Lupski (YUHAL) syndrome, which is characterized by multiple neurological problems. This condition results from duplication of a small piece of chromosome 17 in a region designated p12-p11.2. In YUHAL syndrome, the duplicated segments can range in size from 3.2 Mb to 19.7 Mb. These duplications affect one of the two copies of chromosome 17 in each cell.The duplications that cause YUHAL syndrome all contain the RAI1 gene and a nearby gene called PMP22; the segments may also contain additional genes. Some features of YUHAL syndrome, such as delayed development, behavioral and sleep problems, and unusual facial features, are similar to those of Potocki-Lupski syndrome (described above) and are likely caused by an extra copy of the RAI1 gene. Other features of YUHAL syndrome, such as muscle weakness and decreased sensitivity to touch, heat, and cold in the lower legs and feet, likely result from an extra copy of the PMP22 gene. More About This Health Condition    Cytogenetic Location: 17p11.2, which is the short (p) arm of chromosome 17 at position 11.2Molecular Location: base pairs 17,681,376 to 17,811,453 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA1820RAI1_HUMANSMCRSMS     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Neuroscience (second edition, 2001): The Circadian Cycle of Sleep and WakefulnessSleep Disorders and Sleep Deprivation: An Unmet Public Health Problem (2006): Circadian Rhythms, The 24-Hour Clock      Clinical Information from GeneReviews (2 links)     Potocki-Lupski SyndromeSmith-Magenis Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of RAI1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     RETINOIC ACID-INDUCED GENE 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Neuroscience (second edition, 2001): The Circadian Cycle of Sleep and WakefulnessSleep Disorders and Sleep Deprivation: An Unmet Public Health Problem (2006): Circadian Rhythms, The 24-Hour Clock  Potocki-Lupski SyndromeSmith-Magenis Syndrome  Tests of RAI1  PubMed  RETINOIC ACID-INDUCED GENE 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bi W, Saifi GM, Girirajan S, Shi X, Szomju B, Firth H, Magenis RE, Potocki L, Elsea SH, Lupski JR. RAI1 point mutations, CAG repeat variation, and SNP analysis in non-deletion Smith-Magenis syndrome. Am J Med Genet A. 2006 Nov 15;140(22):2454-63. Citation on PubMedBi W, Saifi GM, Shaw CJ, Walz K, Fonseca P, Wilson M, Potocki L, Lupski JR. Mutations of RAI1, a PHD-containing protein, in nondeletion patients with Smith-Magenis syndrome. Hum Genet. 2004 Nov;115(6):515-24. Epub 2004 Sep 30. Citation on PubMedCarmona-Mora P, Encina CA, Canales CP, Cao L, Molina J, Kairath P, Young JI, Walz K. Functional and cellular characterization of human Retinoic Acid Induced 1 (RAI1) mutations associated with Smith-Magenis Syndrome. BMC Mol Biol. 2010 Aug 25;11:63. doi: 10.1186/1471-2199-11-63. Citation on PubMed or Free article on PubMed CentralGirirajan S, Elsas LJ 2nd, Devriendt K, Elsea SH. RAI1 variations in Smith-Magenis syndrome patients without 17p11.2 deletions. J Med Genet. 2005 Nov;42(11):820-8. Epub 2005 Mar 23. Citation on PubMed or Free article on PubMed CentralGirirajan S, Vlangos CN, Szomju BB, Edelman E, Trevors CD, Dupuis L, Nezarati M, Bunyan DJ, Elsea SH. Genotype-phenotype correlation in Smith-Magenis syndrome: evidence that multiple genes in 17p11.2 contribute to the clinical spectrum. Genet Med. 2006 Jul;8(7):417-27. Citation on PubMedHuang WH, Guenthner CJ, Xu J, Nguyen T, Schwarz LA, Wilkinson AW, Gozani O, Chang HY, Shamloo M, Luo L. Molecular and Neural Functions of Rai1, the Causal Gene for Smith-Magenis Syndrome. Neuron. 2016 Oct 19;92(2):392-406. doi: 10.1016/j.neuron.2016.09.019. Epub 2016 Sep 29. Citation on PubMed or Free article on PubMed CentralMullegama SV, Alaimo JT, Fountain MD, Burns B, Balog AH, Chen L, Elsea SH. RAI1 Overexpression Promotes Altered Circadian Gene Expression and Dyssomnia in Potocki-Lupski Syndrome. J Pediatr Genet. 2017 Sep;6(3):155-164. doi: 10.1055/s-0037-1599147. Epub 2017 Mar 7. Citation on PubMed or Free article on PubMed CentralPotocki L, Bi W, Treadwell-Deering D, Carvalho CM, Eifert A, Friedman EM, Glaze D, Krull K, Lee JA, Lewis RA, Mendoza-Londono R, Robbins-Furman P, Shaw C, Shi X, Weissenberger G, Withers M, Yatsenko SA, Zackai EH, Stankiewicz P, Lupski JR. Characterization of Potocki-Lupski syndrome (dup(17)(p11.2p11.2)) and delineation of a dosage-sensitive critical interval that can convey an autism phenotype. Am J Hum Genet. 2007 Apr;80(4):633-49. Epub 2007 Feb 26. Citation on PubMed or Free article on PubMed CentralPotocki L, Chen KS, Park SS, Osterholm DE, Withers MA, Kimonis V, Summers AM, Meschino WS, Anyane-Yeboa K, Kashork CD, Shaffer LG, Lupski JR. Molecular mechanism for duplication 17p11.2- the homologous recombination reciprocal of the Smith-Magenis microdeletion. Nat Genet. 2000 Jan;24(1):84-7. Citation on PubMedPotocki L, Neira-Fresneda J, Yuan B. Potocki-Lupski Syndrome. 2017 Aug 24. In: Adam MP, Ardinger HH, Pagon RA, Wallace SE, Bean LJH, Mefford HC, Stephens K, Amemiya A, Ledbetter N, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK447920/ Citation on PubMedSlager RE, Newton TL, Vlangos CN, Finucane B, Elsea SH. Mutations in RAI1 associated with Smith-Magenis syndrome. Nat Genet. 2003 Apr;33(4):466-8. Epub 2003 Mar 24. Citation on PubMedSmith ACM, Boyd KE, Elsea SH, Finucane BM, Haas-Givler B, Gropman A, Laje G, Magenis E, Potocki L. Smith-Magenis Syndrome. 2001 Oct 22 [updated 2012 Jun 28]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1310/ Citation on PubMedWilliams SR, Zies D, Mullegama SV, Grotewiel MS, Elsea SH. Smith-Magenis syndrome results in disruption of CLOCK gene transcription and reveals an integral role for RAI1 in the maintenance of circadian rhythmicity. Am J Hum Genet. 2012 Jun 8;90(6):941-9. doi: 10.1016/j.ajhg.2012.04.013. Epub 2012 May 10. Citation on PubMed or Free article on PubMed CentralYuan B, Harel T, Gu S, Liu P, Burglen L, Chantot-Bastaraud S, Gelowani V, Beck CR, Carvalho CM, Cheung SW, Coe A, Malan V, Munnich A, Magoulas PL, Potocki L, Lupski JR. Nonrecurrent 17p11.2p12 Rearrangement Events that Result in Two Concomitant Genomic Disorders: The PMP22-RAI1 Contiguous Gene Duplication Syndrome. Am J Hum Genet. 2015 Nov 5;97(5):691-707. doi: 10.1016/j.ajhg.2015.10.003. Citation on PubMed or Free article on PubMed Central \n",
            "DRC3\t\n",
            "MYO15A\t   From NCBI Gene:This gene encodes an unconventional myosin. This protein differs from other myosins in that it has a long N-terminal extension preceding the conserved motor domain. Studies in mice suggest that this protein is necessary for actin organization in the hair cells of the cochlea. Mutations in this gene have been associated with profound, congenital, neurosensory, nonsyndromal deafness. This gene is located within the Smith-Magenis syndrome region on chromosome 17. Read-through transcripts containing an upstream gene and this gene have been identified, but they are not thought to encode a fusion protein. Several alternatively spliced transcript variants have been described, but their full length sequences have not been determined. [provided by RefSeq, Jul 2008]From UniProt:Myosins are actin-based motor molecules with ATPase activity. Unconventional myosins serve in intracellular movements. Their highly divergent tails are presumed to bind to membranous compartments, which would be moved relative to actin filaments. Required for the arrangement of stereocilia in mature hair bundles (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 3From UniProt:Deafness, autosomal recessive, 3 (DFNB3): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. [MIM:600316]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p11.2, which is the short (p) arm of chromosome 17 at position 11.2Molecular Location: base pairs 18,108,706 to 18,180,244 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNB3MYO15     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYO15A      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 3OMIM: MYOSIN XVA      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MYO15A  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 3OMIM: MYOSIN XVA  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SMCR8\t\n",
            "ZNF286B\t\n",
            "FBXW10\t\n",
            "FAM83G\t\n",
            "SLC5A10\t\n",
            "DPH1\t   From NCBI Gene:The protein encoded by this gene is an enzyme involved in the biosynthesis of diphthamide, a modified histidine found only in elongation factor-2 (EEF2). Diphthamide residues in EEF2 are targeted for ADP-ribosylation by diphtheria toxin and Pseudomonas exotoxin A. Defects in this gene have been associated with both ovarian cancer and autosomal recessive intellectual disability with short stature, craniofacial, and ectodermal anomalies. [provided by RefSeq, Oct 2016]From UniProt:Required for the first step in the synthesis of diphthamide, a post-translational modification of histidine which occurs in translation elongation factor 2 (EEF2). When overexpressed, suppresses colony formation ability and growth rate of ovarian cancer cells. Acts also as a tumor suppressor in lung and breast cancers (By similarity). Plays a role in embryonic growth, organogenesis and postnatal survival (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Developmental delay with short stature, dysmorphic features, and  sparse hairFrom UniProt:Developmental delay with short stature, dysmorphic features, and sparse hair (DEDSSH): An autosomal recessive syndrome characterized by intellectual disability, short stature, and craniofacial and ectodermal anomalies including scaphocephaly with or without craniosynostosis, prominent forehead, sparse eyebrows and hair, hypoplastic toenails and, in some cases, dental anomalies. [MIM:616901]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.3, which is the short (p) arm of chromosome 17 at position 13.3Molecular Location: base pairs 2,030,112 to 2,043,431 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DEDSSHDPH2LDPH2L1OVCA1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DPH1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEVELOPMENTAL DELAY WITH SHORT STATURE, DYSMORPHIC FEATURES, AND SPARSE HAIROMIM: DIPHTHAMIDE BIOSYNTHESIS PROTEIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DPH1  OMIM: DEVELOPMENTAL DELAY WITH SHORT STATURE, DYSMORPHIC FEATURES, AND SPARSE HAIROMIM: DIPHTHAMIDE BIOSYNTHESIS PROTEIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MAP2K3\t   From NCBI Gene:The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase is activated by mitogenic and environmental stress, and participates in the MAP kinase-mediated signaling cascade. It phosphorylates and thus activates MAPK14/p38-MAPK. This kinase can be activated by insulin, and is necessary for the expression of glucose transporter. Expression of RAS oncogene is found to result in the accumulation of the active form of this kinase, which thus leads to the constitutive activation of MAPK14, and confers oncogenic transformation of primary cells. The inhibition of this kinase is involved in the pathogenesis of Yersina pseudotuberculosis. Multiple alternatively spliced transcript variants that encode distinct isoforms have been reported for this gene. [provided by RefSeq, Jul 2008]From UniProt:Dual specificity kinase. Is activated by cytokines and environmental stress in vivo. Catalyzes the concomitant phosphorylation of a threonine and a tyrosine residue in the MAP kinase p38. Part of a signaling cascade that begins with the activation of the adrenergic receptor ADRA1B and leads to the activation of MAPK14.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Defects in MAP2K3 may be involved in colon cancer.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p11.2, which is the short (p) arm of chromosome 17 at position 11.2Molecular Location: base pairs 21,284,656 to 21,315,240 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MAPKK3MEK3MKK3PRKMK3SAPKK-2SAPKK2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MAP2K3      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MITOGEN-ACTIVATED PROTEIN KINASE KINASE 3      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MAP2K3  OMIM: MITOGEN-ACTIVATED PROTEIN KINASE KINASE 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KCNJ12\t\n",
            "SMG6\t\n",
            "POLDIP2\t\n",
            "SEBOX\t\n",
            "SARM1\t\n",
            "SLC46A1\t   The SLC46A1 gene provides instructions for making a protein called the proton-coupled folate transporter (PCFT). PCFT is important for normal functioning of intestinal epithelial cells, which are cells that line the walls of the intestine. These cells have fingerlike projections called microvilli that absorb nutrients from food as it passes through the intestine. Based on their appearance, groups of these microvilli are known collectively as the brush border. PCFT is involved in the process of using energy to move certain B vitamins called folates across the brush border membrane for absorption, a mechanism called active transport. It is also involved in the transport of folates between the brain and the fluid that surrounds it (cerebrospinal fluid). Folates are important for many cell functions, including the production of DNA and its chemical cousin, RNA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hereditary folate malabsorption     More than 10 mutations in the SLC46A1 gene have been identified in people with hereditary folate malabsorption. These mutations cause the substitution of one protein building block (amino acid) for another amino acid in the PCFT protein, or result in a PCFT protein that is shorter than normal. The mutated PCFT protein has little or no activity. In some cases the abnormal protein is not transported to the cell membrane, and so it is unable to perform its function. PCFT inactivity impairs the body's ability to absorb folates from food, leading to the signs and symptoms of hereditary folate malabsorption. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 10 mutations in the SLC46A1 gene have been identified in people with hereditary folate malabsorption. These mutations cause the substitution of one protein building block (amino acid) for another amino acid in the PCFT protein, or result in a PCFT protein that is shorter than normal. The mutated PCFT protein has little or no activity. In some cases the abnormal protein is not transported to the cell membrane, and so it is unable to perform its function. PCFT inactivity impairs the body's ability to absorb folates from food, leading to the signs and symptoms of hereditary folate malabsorption. More About This Health Condition    Cytogenetic Location: 17q11.2, which is the long (q) arm of chromosome 17 at position 11.2Molecular Location: base pairs 28,394,642 to 28,406,630 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HCP1heme carrier protein 1MGC9564PCFTproton-coupled folate transportersolute carrier family 46 (folate transporter), member 1solute carrier family 46, member 1     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Hereditary Folate Malabsorption      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC46A1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SOLUTE CARRIER FAMILY 46 (FOLATE TRANSPORTER), MEMBER 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Hereditary Folate Malabsorption  Tests of SLC46A1  PubMed  SOLUTE CARRIER FAMILY 46 (FOLATE TRANSPORTER), MEMBER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Andrews NC. When is a heme transporter not a heme transporter? When it's a folate transporter. Cell Metab. 2007 Jan;5(1):5-6. Citation on PubMedGonen N, Bram EE, Assaraf YG. PCFT/SLC46A1 promoter methylation and restoration of gene expression in human leukemia cells. Biochem Biophys Res Commun. 2008 Nov 28;376(4):787-92. doi: 10.1016/j.bbrc.2008.09.074. Epub 2008 Sep 24. Citation on PubMedIfergan I, Assaraf YG. Molecular mechanisms of adaptation to folate deficiency. Vitam Horm. 2008;79:99-143. doi: 10.1016/S0083-6729(08)00404-4. Citation on PubMedKronn D, Goldman ID. Hereditary Folate Malabsorption. 2008 Jun 17 [updated 2017 Apr 27]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1673/ Citation on PubMedLasry I, Berman B, Straussberg R, Sofer Y, Bessler H, Sharkia M, Glaser F, Jansen G, Drori S, Assaraf YG. A novel loss-of-function mutation in the proton-coupled folate transporter from a patient with hereditary folate malabsorption reveals that Arg 113 is crucial for function. Blood. 2008 Sep 1;112(5):2055-61. doi: 10.1182/blood-2008-04-150276. Epub 2008 Jun 17. Citation on PubMedMin SH, Oh SY, Karp GI, Poncz M, Zhao R, Goldman ID. The clinical course and genetic defect in the PCFT gene in a 27-year-old woman with hereditary folate malabsorption. J Pediatr. 2008 Sep;153(3):435-7. doi: 10.1016/j.jpeds.2008.04.009. Citation on PubMed or Free article on PubMed CentralNakai Y, Inoue K, Abe N, Hatakeyama M, Ohta KY, Otagiri M, Hayashi Y, Yuasa H. Functional characterization of human proton-coupled folate transporter/heme carrier protein 1 heterologously expressed in mammalian cells as a folate transporter. J Pharmacol Exp Ther. 2007 Aug;322(2):469-76. Epub 2007 May 2. Citation on PubMedQiu A, Jansen M, Sakaris A, Min SH, Chattopadhyay S, Tsai E, Sandoval C, Zhao R, Akabas MH, Goldman ID. Identification of an intestinal folate transporter and the molecular basis for hereditary folate malabsorption. Cell. 2006 Dec 1;127(5):917-28. Citation on PubMedOMIM: SOLUTE CARRIER FAMILY 46 (FOLATE TRANSPORTER), MEMBER 1Unal ES, Zhao R, Qiu A, Goldman ID. N-linked glycosylation and its impact on the electrophoretic mobility and function of the human proton-coupled folate transporter (HsPCFT). Biochim Biophys Acta. 2008 Jun;1778(6):1407-14. doi: 10.1016/j.bbamem.2008.03.009. Epub 2008 Mar 20. Citation on PubMed or Free article on PubMed CentralWolf G. Identification of proton-coupled high-affinity human intestinal folate transporter mutated in human hereditary familial folate malabsorption. Nutr Rev. 2007 Dec;65(12 Pt 1):554-7. Review. Citation on PubMedZhao R, Goldman ID. The molecular identity and characterization of a Proton-coupled Folate Transporter--PCFT; biological ramifications and impact on the activity of pemetrexed. Cancer Metastasis Rev. 2007 Mar;26(1):129-39. Review. Citation on PubMedZhao R, Matherly LH, Goldman ID. Membrane transporters and folate homeostasis: intestinal absorption and transport into systemic compartments and tissues. Expert Rev Mol Med. 2009 Jan 28;11:e4. doi: 10.1017/S1462399409000969. Review. Citation on PubMed or Free article on PubMed CentralZhao R, Min SH, Qiu A, Sakaris A, Goldberg GL, Sandoval C, Malatack JJ, Rosenblatt DS, Goldman ID. The spectrum of mutations in the PCFT gene, coding for an intestinal folate transporter, that are the basis for hereditary folate malabsorption. Blood. 2007 Aug 15;110(4):1147-52. Epub 2007 Apr 19. Citation on PubMed or Free article on PubMed CentralZhao R, Qiu A, Tsai E, Jansen M, Akabas MH, Goldman ID. The proton-coupled folate transporter: impact on pemetrexed transport and on antifolates activities compared with the reduced folate carrier. Mol Pharmacol. 2008 Sep;74(3):854-62. doi: 10.1124/mol.108.045443. Epub 2008 Jun 4. Citation on PubMed or Free article on PubMed Central \n",
            "TP53I13\t\n",
            "SSH2\t\n",
            "EFCAB5\t\n",
            "CRLF3\t\n",
            "OR1D5\t\n",
            "OR1A2\t\n",
            "OR1A1\t\n",
            "OR3A1\t\n",
            "CCL8\t\n",
            "CCT6B\t\n",
            "SPATA22\t\n",
            "MMP28\t\n",
            "TRPV3\t   From NCBI Gene:This gene product belongs to a family of nonselective cation channels that function in a variety of processes, including temperature sensation and vasoregulation. The thermosensitive members of this family are expressed in subsets of sensory neurons that terminate in the skin, and are activated at distinct physiological temperatures. This channel is activated at temperatures between 22 and 40 degrees C. This gene lies in close proximity to another family member gene on chromosome 17, and the two encoded proteins are thought to associate with each other to form heteromeric channels. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Apr 2012]From UniProt:Putative receptor-activated non-selective calcium permeant cation channel. It is activated by innocuous (warm) temperatures and shows an increased response at noxious temperatures greater than 39 degrees Celsius. Activation exhibits an outward rectification. May associate with TRPV1 and may modulate its activity. Is a negative regulator of hair growth and cycling: TRPV3-coupled signaling suppresses keratinocyte proliferation in hair follicles and induces apoptosis and premature hair follicle regression (catagen).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Palmoplantar keratoderma, mutilating, with periorificial keratotic plaquesPalmoplantar keratoderma, nonepidermolytic, focal 2From UniProt:Palmoplantar keratoderma, non-epidermolytic, focal 2 (FNEPPK2): A dermatological disorder characterized by non-epidermolytic, abnormal thickening of the skin on the palms and soles. Focal palmoplantar keratoderma consists of localized areas of hyperkeratosis located mainly on pressure points and sites of recurrent friction. [MIM:616400]Olmsted syndrome (OLMS): A rare congenital disorder characterized by bilateral mutilating palmoplantar keratoderma and periorificial keratotic plaques with severe itching at all lesions. Diffuse alopecia, constriction of digits, and onychodystrophy have also been reported. Infections and squamous cell carcinomas can arise on the keratotic areas. The digital constriction may progress to autoamputation of fingers and toes. [MIM:614594]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.2, which is the short (p) arm of chromosome 17 at position 13.2Molecular Location: base pairs 3,510,502 to 3,557,995 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FNEPPK2OLMSVRL3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TRPV3      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: PALMOPLANTAR KERATODERMA, MUTILATING, WITH PERIORIFICIAL KERATOTIC PLAQUESOMIM: PALMOPLANTAR KERATODERMA, NONEPIDERMOLYTIC, FOCAL 2OMIM: TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY V, MEMBER 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TRPV3  OMIM: PALMOPLANTAR KERATODERMA, MUTILATING, WITH PERIORIFICIAL KERATOTIC PLAQUESOMIM: PALMOPLANTAR KERATODERMA, NONEPIDERMOLYTIC, FOCAL 2OMIM: TRANSIENT RECEPTOR POTENTIAL CATION CHANNEL, SUBFAMILY V, MEMBER 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "P2RX5\t\n",
            "ITGAE\t\n",
            "STARD3\t\n",
            "ERBB2\t   From NCBI Gene:This gene encodes a member of the epidermal growth factor (EGF) receptor family of receptor tyrosine kinases. This protein has no ligand binding domain of its own and therefore cannot bind growth factors. However, it does bind tightly to other ligand-bound EGF receptor family members to form a heterodimer, stabilizing ligand binding and enhancing kinase-mediated activation of downstream signalling pathways, such as those involving mitogen-activated protein kinase and phosphatidylinositol-3 kinase. Allelic variations at amino acid positions 654 and 655 of isoform a (positions 624 and 625 of isoform b) have been reported, with the most common allele, Ile654/Ile655, shown here. Amplification and/or overexpression of this gene has been reported in numerous cancers, including breast and ovarian tumors. Alternative splicing results in several additional transcript variants, some encoding different isoforms and others that have not been fully characterized. [provided by RefSeq, Jul 2008]From UniProt:Protein tyrosine kinase that is part of several cell surface receptor complexes, but that apparently needs a coreceptor for ligand binding. Essential component of a neuregulin-receptor complex, although neuregulins do not interact with it alone. GP30 is a potential ligand for this receptor. Regulates outgrowth and stabilization of peripheral microtubules (MTs). Upon ERBB2 activation, the MEMO1-RHOA-DIAPH1 signaling pathway elicits the phosphorylation and thus the inhibition of GSK3B at cell membrane. This prevents the phosphorylation of APC and CLASP2, allowing its association with the cell membrane. In turn, membrane-bound APC allows the localization of MACF1 to the cell membrane, which is required for microtubule capture and stabilization.In the nucleus is involved in transcriptional regulation. Associates with the 5'-TCAAATTC-3' sequence in the PTGS2/COX-2 promoter and activates its transcription. Implicated in transcriptional activation of CDKN1A; the function involves STAT3 and SRC. Involved in the transcription of rRNA genes by RNA Pol I and enhances protein synthesis and cell growth.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:NeuroblastomaLung cancerFrom NCBI Gene:Lung cancerNeoplasm of stomachGlioma susceptibility 1From UniProt:Chromosomal aberrations involving ERBB2 may be a cause gastric cancer. Deletions within 17q12 region producing fusion transcripts with CDK12, leading to CDK12-ERBB2 fusion leading to truncated CDK12 protein not in-frame with ERBB2.Ovarian cancer (OC): The term ovarian cancer defines malignancies originating from ovarian tissue. Although many histologic types of ovarian tumors have been described, epithelial ovarian carcinoma is the most common form. Ovarian cancers are often asymptomatic and the recognized signs and symptoms, even of late-stage disease, are vague. Consequently, most patients are diagnosed with advanced disease. [MIM:167000]Gastric cancer (GASC): A malignant disease which starts in the stomach, can spread to the esophagus or the small intestine, and can extend through the stomach wall to nearby lymph nodes and organs. It also can metastasize to other parts of the body. The term gastric cancer or gastric carcinoma refers to adenocarcinoma of the stomach that accounts for most of all gastric malignant tumors. Two main histologic types are recognized, diffuse type and intestinal type carcinomas. Diffuse tumors are poorly differentiated infiltrating lesions, resulting in thickening of the stomach. In contrast, intestinal tumors are usually exophytic, often ulcerating, and associated with intestinal metaplasia of the stomach, most often observed in sporadic disease. [MIM:613659]Glioma (GLM): Gliomas are benign or malignant central nervous system neoplasms derived from glial cells. They comprise astrocytomas and glioblastoma multiforme that are derived from astrocytes, oligodendrogliomas derived from oligodendrocytes and ependymomas derived from ependymocytes. [MIM:137800]Lung cancer (LNCR): A common malignancy affecting tissues of the lung. The most common form of lung cancer is non-small cell lung cancer (NSCLC) that can be divided into 3 major histologic subtypes: squamous cell carcinoma, adenocarcinoma, and large cell lung cancer. NSCLC is often diagnosed at an advanced stage and has a poor prognosis. [MIM:211980]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q12, which is the long (q) arm of chromosome 17 at position 12Molecular Location: base pairs 39,688,084 to 39,728,662 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD340HER-2HER-2/neuHER2MLN 19NEUNGLTKR1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ERBB2      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: GASTRIC CANCEROMIM: GLIOMA SUSCEPTIBILITY 1OMIM: LUNG CANCEROMIM: V-ERB-B2 AVIAN ERYTHROBLASTIC LEUKEMIA VIRAL ONCOGENE HOMOLOG 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ERBB2  OMIM: GASTRIC CANCEROMIM: GLIOMA SUSCEPTIBILITY 1OMIM: LUNG CANCEROMIM: V-ERB-B2 AVIAN ERYTHROBLASTIC LEUKEMIA VIRAL ONCOGENE HOMOLOG 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GSDMB\t\n",
            "GSDMA\t\n",
            "KRT24\t\n",
            "KRT10\t   The KRT10 gene provides instructions for making a protein called keratin 10. Keratins are a group of tough, fibrous proteins that form the structural framework of cells called keratinocytes that make up the skin, hair, and nails. Keratin 10 is produced in keratinocytes in the outer layer of the skin (the epidermis).In the fluid-filled space inside these cells (the cytoplasm), the keratin 10 protein partners with a similar protein, keratin 1 (produced from the KRT1 gene), to form molecules called keratin intermediate filaments. These filaments assemble into strong networks that provide strength and resiliency to the skin and protect it from being damaged by friction and other everyday physical stresses.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Epidermolytic hyperkeratosis     Dozens of mutations in the KRT10 gene have been found in people with epidermolytic hyperkeratosis. This condition is characterized by red, blistering skin at an early age and thick skin (hyperkeratosis) later in life. People with KRT10 gene mutations typically have NPS-type epidermolytic hyperkeratosis, which features thick skin on many parts of the body but not the palms of the hands or soles of the feet.Most KRT10 gene mutations associated with epidermolytic hyperkeratosis change a single protein building block (amino acid) in the keratin 10 protein. These amino acid changes commonly occur in regions of the protein that play a role in intermediate filament formation. The mutations alter the keratin 10 protein and seem to affect how intermediate filaments interact with each other to form networks. The altered proteins still form intermediate filaments, but the intermediate filament networks are disorganized and do not function normally. Without a strong network, skin cells become fragile and are easily damaged, which can lead to blistering in response to friction or mild trauma. It is unclear how these mutations cause the overgrowth of keratinocytes that results in hyperkeratotic skin. More About This Health Condition      Ichthyosis with confetti     Mutations in the KRT10 gene can cause another skin disorder known as ichthyosis with confetti (also called congenital reticular ichthyosiform erythroderma), which is characterized by red, scaly skin all over the body with small patches of normal skin that look like confetti. The patches of normal skin increase in number and size with age. The KRT10 gene mutations involved in this condition, which are initially found in every cell of the body, alter the genetic sequence that is used as a blueprint for protein production, leading to production of abnormal keratin 10 protein. The abnormal protein includes a region at the end with an excess of the amino acid arginine; this arginine-rich region is not found in the normal keratin 10 protein. Researchers believe that this abnormal amino acid sequence directs the protein into the nucleus of the cell, where it cannot form the strong network of intermediate filaments. Loss of this network disrupts the outer layer of skin, contributing to the development of red, scaly skin.In some abnormal cells, the mutated gene corrects itself through a complex process by which genetic material is exchanged between chromosomes. As a result, normal keratin 10 protein is produced and remains in the cytoplasm. The cell becomes normal and, as it continues to grow and divide, forms patches of normal skin in people with ichthyosis with confetti. More About This Health Condition      Other disorders     Cyclic ichthyosis with epidermolytic hyperkeratosis is another skin disorder caused by mutations in the KRT10 gene. This condition is similar to epidermolytic hyperkeratosis (described above), but the skin changes disappear for short periods, and then recur. The recurrent skin changes can last for weeks or months.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Dozens of mutations in the KRT10 gene have been found in people with epidermolytic hyperkeratosis. This condition is characterized by red, blistering skin at an early age and thick skin (hyperkeratosis) later in life. People with KRT10 gene mutations typically have NPS-type epidermolytic hyperkeratosis, which features thick skin on many parts of the body but not the palms of the hands or soles of the feet.Most KRT10 gene mutations associated with epidermolytic hyperkeratosis change a single protein building block (amino acid) in the keratin 10 protein. These amino acid changes commonly occur in regions of the protein that play a role in intermediate filament formation. The mutations alter the keratin 10 protein and seem to affect how intermediate filaments interact with each other to form networks. The altered proteins still form intermediate filaments, but the intermediate filament networks are disorganized and do not function normally. Without a strong network, skin cells become fragile and are easily damaged, which can lead to blistering in response to friction or mild trauma. It is unclear how these mutations cause the overgrowth of keratinocytes that results in hyperkeratotic skin. More About This Health Condition  Mutations in the KRT10 gene can cause another skin disorder known as ichthyosis with confetti (also called congenital reticular ichthyosiform erythroderma), which is characterized by red, scaly skin all over the body with small patches of normal skin that look like confetti. The patches of normal skin increase in number and size with age. The KRT10 gene mutations involved in this condition, which are initially found in every cell of the body, alter the genetic sequence that is used as a blueprint for protein production, leading to production of abnormal keratin 10 protein. The abnormal protein includes a region at the end with an excess of the amino acid arginine; this arginine-rich region is not found in the normal keratin 10 protein. Researchers believe that this abnormal amino acid sequence directs the protein into the nucleus of the cell, where it cannot form the strong network of intermediate filaments. Loss of this network disrupts the outer layer of skin, contributing to the development of red, scaly skin.In some abnormal cells, the mutated gene corrects itself through a complex process by which genetic material is exchanged between chromosomes. As a result, normal keratin 10 protein is produced and remains in the cytoplasm. The cell becomes normal and, as it continues to grow and divide, forms patches of normal skin in people with ichthyosis with confetti. More About This Health Condition  Cyclic ichthyosis with epidermolytic hyperkeratosis is another skin disorder caused by mutations in the KRT10 gene. This condition is similar to epidermolytic hyperkeratosis (described above), but the skin changes disappear for short periods, and then recur. The recurrent skin changes can last for weeks or months.    Cytogenetic Location: 17q21.2, which is the long (q) arm of chromosome 17 at position 21.2Molecular Location: base pairs 40,818,117 to 40,822,621 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CK-10CK10cytokeratin 10K1C10_HUMANK10keratin-10keratin 10, type Ikeratin, type I cytoskeletal 10     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Human Intermediate Filament DatabaseMolecular Cell Biology (fourth edition, 2000): Intermediate Filaments      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT10      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     ICHTHYOSIS, CYCLIC, WITH EPIDERMOLYTIC HYPERKERATOSISKERATIN 10, TYPE I      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Human Intermediate Filament DatabaseMolecular Cell Biology (fourth edition, 2000): Intermediate Filaments  Tests of KRT10  PubMed  ICHTHYOSIS, CYCLIC, WITH EPIDERMOLYTIC HYPERKERATOSISKERATIN 10, TYPE I  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Chamcheu JC, Siddiqui IA, Syed DN, Adhami VM, Liovic M, Mukhtar H. Keratin gene mutations in disorders of human skin and its appendages. Arch Biochem Biophys. 2011 Apr 15;508(2):123-37. doi: 10.1016/j.abb.2010.12.019. Epub 2010 Dec 19. Review. Citation on PubMed or Free article on PubMed CentralChipev CC, Yang JM, DiGiovanna JJ, Steinert PM, Marekov L, Compton JG, Bale SJ. Preferential sites in keratin 10 that are mutated in epidermolytic hyperkeratosis. Am J Hum Genet. 1994 Feb;54(2):179-90. Citation on PubMed or Free article on PubMed CentralChoate KA, Lu Y, Zhou J, Choi M, Elias PM, Farhi A, Nelson-Williams C, Crumrine D, Williams ML, Nopper AJ, Bree A, Milstone LM, Lifton RP. Mitotic recombination in patients with ichthyosis causes reversion of dominant mutations in KRT10. Science. 2010 Oct 1;330(6000):94-7. doi: 10.1126/science.1192280. Epub 2010 Aug 26. Citation on PubMed or Free article on PubMed CentralDiGiovanna JJ, Bale SJ. Clinical heterogeneity in epidermolytic hyperkeratosis. Arch Dermatol. 1994 Aug;130(8):1026-35. Citation on PubMedHuber M, Scaletta C, Benathan M, Frenk E, Greenhalgh DA, Rothnagel JA, Roop DR, Hohl D. Abnormal keratin 1 and 10 cytoskeleton in cultured keratinocytes from epidermolytic hyperkeratosis caused by keratin 10 mutations. J Invest Dermatol. 1994 May;102(5):691-4. Citation on PubMedOMIM: ICHTHYOSIS, CYCLIC, WITH EPIDERMOLYTIC HYPERKERATOSISOMIM: KERATIN 10, TYPE IRothnagel JA, Dominey AM, Dempsey LD, Longley MA, Greenhalgh DA, Gagne TA, Huber M, Frenk E, Hohl D, Roop DR. Mutations in the rod domains of keratins 1 and 10 in epidermolytic hyperkeratosis. Science. 1992 Aug 21;257(5073):1128-30. Citation on PubMed \n",
            "KRT40\t\n",
            "KRTAP3-2\t\n",
            "KRTAP1-3\t\n",
            "KRTAP1-1\t\n",
            "KRTAP2-1\t\n",
            "KRTAP4-1\t\n",
            "KRTAP9-1\t\n",
            "KRTAP9-9\t\n",
            "KRTAP29-1\t\n",
            "KRTAP17-1\t\n",
            "KRT37\t\n",
            "KRT32\t\n",
            "KRT35\t\n",
            "KRT36\t\n",
            "KRT13\t   The KRT13 gene provides instructions for making a protein called keratin 13. Keratins are a group of tough, fibrous proteins that form the structural framework of epithelial cells, which are cells that line the surfaces and cavities of the body. Keratin 13 is found in the moist lining (mucosae) of the mouth, nose, esophagus, genitals, and anus.Keratin 13 partners with a similar protein, keratin 4 (produced from the KRT4 gene), to form molecules known as intermediate filaments. These filaments assemble into strong networks that provide strength and resilience to the different mucosae. Networks of intermediate filaments protect the mucosae from being damaged by friction or other everyday physical stresses.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           White sponge nevus     At least five KRT13 gene mutations have been found to cause white sponge nevus, a condition that results in the formation of white patches of tissue called nevi (singular: nevus) that appear as thickened, velvety, sponge-like tissue. These nevi most often occur on the mouth (oral) mucosa (plural: mucosae). Rarely, white sponge nevus occurs on the mucosae of the nose, esophagus, genitals, or anus.The KRT13 gene mutations that cause white sponge nevus disrupt the structure of keratin 13. As a result, keratin 13 does not fit together properly with keratin 4, leading to the formation of irregular intermediate filaments that are easily damaged with little friction or trauma. Fragile intermediate filaments in the oral mucosa might be damaged when eating or brushing one's teeth. Damage to intermediate filaments leads to inflammation and promotes the abnormal growth and division (proliferation) of epithelial cells, causing the mucosae to thicken and resulting in white sponge nevus. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least five KRT13 gene mutations have been found to cause white sponge nevus, a condition that results in the formation of white patches of tissue called nevi (singular: nevus) that appear as thickened, velvety, sponge-like tissue. These nevi most often occur on the mouth (oral) mucosa (plural: mucosae). Rarely, white sponge nevus occurs on the mucosae of the nose, esophagus, genitals, or anus.The KRT13 gene mutations that cause white sponge nevus disrupt the structure of keratin 13. As a result, keratin 13 does not fit together properly with keratin 4, leading to the formation of irregular intermediate filaments that are easily damaged with little friction or trauma. Fragile intermediate filaments in the oral mucosa might be damaged when eating or brushing one's teeth. Damage to intermediate filaments leads to inflammation and promotes the abnormal growth and division (proliferation) of epithelial cells, causing the mucosae to thicken and resulting in white sponge nevus. More About This Health Condition    Cytogenetic Location: 17q21.2, which is the long (q) arm of chromosome 17 at position 21.2Molecular Location: base pairs 41,500,981 to 41,505,613 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CK-13CK13cytokeratin-13cytokeratin 13K1C13_HUMANK13keratin-13keratin 13, type Ikeratin, type I cytoskeletal 13     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Madame Curie Bioscience Database: Keratin Intermediate Filaments and Diseases of the SkinThe Cell: A Molecular Approach (2nd edition, 2000): Intermediate Filaments      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KRT13      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KERATIN 13, TYPE I      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Keratin Intermediate Filaments and Diseases of the SkinThe Cell: A Molecular Approach (2nd edition, 2000): Intermediate Filaments  Tests of KRT13  PubMed  KERATIN 13, TYPE I  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: KERATIN 13, TYPE INishizawa A, Nakajima R, Nakano H, Sawamura D, Takayama K, Satoh T, Yokozeki H. A de novo missense mutation in the keratin 13 gene in oral white sponge naevus. Br J Dermatol. 2008 Sep;159(4):974-5. doi: 10.1111/j.1365-2133.2008.08716.x. Epub 2008 Jul 4. Citation on PubMedRugg E, Magee G, Wilson N, Brandrup F, Hamburger J, Lane E. Identification of two novel mutations in keratin 13 as the cause of white sponge naevus. Oral Dis. 1999 Oct;5(4):321-4. Citation on PubMedShibuya Y, Zhang J, Yokoo S, Umeda M, Komori T. Constitutional mutation of keratin 13 gene in familial white sponge nevus. Oral Surg Oral Med Oral Pathol Oral Radiol Endod. 2003 Nov;96(5):561-5. Citation on PubMed \n",
            "HAP1\t\n",
            "JUP\t   The JUP gene provides instructions for making a protein called plakoglobin. This protein is found primarily in cells of the heart and skin, where it is part of two specialized structures called adherens junctions and desmosomes. Both of these structures help hold neighboring cells together, which provides strength and stability to tissues. Desmosomes may also be involved in other critical cell functions, including chemical signaling pathways, the process by which cells mature to perform specific functions (differentiation), and the self-destruction of cells (apoptosis).Studies suggest that plakoglobin also plays a role in signaling within cells as part of the Wnt pathway. Wnt signaling controls the activity of certain genes and regulates the interactions between cells. This signaling pathway is involved in many aspects of development, including the normal development of the heart, skin, and hair.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Arrhythmogenic right ventricular cardiomyopathy     Genetics Home Reference provides information about arrhythmogenic right ventricular cardiomyopathy. More About This Health Condition      Keratoderma with woolly hair     Several mutations in the JUP gene have been found to cause a form of keratoderma with woolly hair classified as type I. This form of the condition is also known as Naxos disease. It is characterized by thick, calloused skin on the palms of the hands and soles of the feet (palmoplantar keratoderma); coarse, dry, fine, tightly curled, and sometimes sparse hair; and a potentially life-threatening form of heart disease called arrhythmogenic right ventricular cardiomyopathy (ARVC).The JUP gene mutations that cause keratoderma with woolly hair type I lead to the production of an abnormally short version of the plakoglobin protein. The abnormal plakoglobin does not interact appropriately with other desmosomal proteins, which alters the structure of desmosomes and prevents cells from attaching to one another effectively. Researchers suspect that the impaired connections between cells make the skin, hair, and heart muscle more fragile. Over time, as these tissues are exposed to mechanical stress (for example, friction on the surface of the skin or the constant contraction and relaxation of the heart muscle), they become damaged and can no longer function normally. This mechanism probably underlies the skin, hair, and heart problems that occur in keratoderma with woolly hair type I.Studies suggest that the abnormally short plakoglobin also impairs Wnt signaling, which appears to cause heart muscle cells to be replaced with fat cells over time. This abnormal signaling may influence the development of ARVC in affected individuals. More About This Health Condition      Other disorders     JUP gene mutations have also been found to cause a spectrum of signs and symptoms that overlap with those of keratoderma with woolly hair type I (described above). Some affected families have had similar skin and hair abnormalities but no apparent heart problems, while others have had ARVC only, with normal skin and hair. Although these conditions are related to impaired function of plakoglobin and abnormal desmosomes, it is unclear how mutations in this gene lead to these different patterns of features.At least one mutation in the JUP gene can cause a disorder known as lethal congenital epidermolysis bullosa (LCEB). Features of this condition include very fragile skin that blisters and detaches easily, a complete absence of hair (alopecia), and abnormal fingernails. The skin abnormalities lead to a severe loss of fluids and death in early infancy. The mutation found to cause LCEB, written as Gln539Ter or Q539X, prevents the production of any functional plakoglobin. As a result, the skin has no stable desmosomes, so skin cells are unable to attach to one another effectively. Because affected individuals die before heart disease may become apparent, it is unknown whether a complete loss of plakoglobin also impairs desmosomes in the heart.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about arrhythmogenic right ventricular cardiomyopathy. More About This Health Condition  Several mutations in the JUP gene have been found to cause a form of keratoderma with woolly hair classified as type I. This form of the condition is also known as Naxos disease. It is characterized by thick, calloused skin on the palms of the hands and soles of the feet (palmoplantar keratoderma); coarse, dry, fine, tightly curled, and sometimes sparse hair; and a potentially life-threatening form of heart disease called arrhythmogenic right ventricular cardiomyopathy (ARVC).The JUP gene mutations that cause keratoderma with woolly hair type I lead to the production of an abnormally short version of the plakoglobin protein. The abnormal plakoglobin does not interact appropriately with other desmosomal proteins, which alters the structure of desmosomes and prevents cells from attaching to one another effectively. Researchers suspect that the impaired connections between cells make the skin, hair, and heart muscle more fragile. Over time, as these tissues are exposed to mechanical stress (for example, friction on the surface of the skin or the constant contraction and relaxation of the heart muscle), they become damaged and can no longer function normally. This mechanism probably underlies the skin, hair, and heart problems that occur in keratoderma with woolly hair type I.Studies suggest that the abnormally short plakoglobin also impairs Wnt signaling, which appears to cause heart muscle cells to be replaced with fat cells over time. This abnormal signaling may influence the development of ARVC in affected individuals. More About This Health Condition  JUP gene mutations have also been found to cause a spectrum of signs and symptoms that overlap with those of keratoderma with woolly hair type I (described above). Some affected families have had similar skin and hair abnormalities but no apparent heart problems, while others have had ARVC only, with normal skin and hair. Although these conditions are related to impaired function of plakoglobin and abnormal desmosomes, it is unclear how mutations in this gene lead to these different patterns of features.At least one mutation in the JUP gene can cause a disorder known as lethal congenital epidermolysis bullosa (LCEB). Features of this condition include very fragile skin that blisters and detaches easily, a complete absence of hair (alopecia), and abnormal fingernails. The skin abnormalities lead to a severe loss of fluids and death in early infancy. The mutation found to cause LCEB, written as Gln539Ter or Q539X, prevents the production of any functional plakoglobin. As a result, the skin has no stable desmosomes, so skin cells are unable to attach to one another effectively. Because affected individuals die before heart disease may become apparent, it is unknown whether a complete loss of plakoglobin also impairs desmosomes in the heart.    Cytogenetic Location: 17q21.2, which is the long (q) arm of chromosome 17 at position 21.2Molecular Location: base pairs 41,754,607 to 41,786,768 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARVD12catenin (cadherin-associated protein), gamma 80kDaCTNNGdesmoplakin-3desmoplakin IIIDP3DPIIIPDGBPKGBplakoglobin     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Desmosomes Connect Intermediate Filaments from Cell to CellMolecular Cell Biology (first edition, 2000): Desmosomes      Clinical Information from GeneReviews (1 link)     Arrhythmogenic Right Ventricular Cardiomyopathy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of JUP      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA, FAMILIAL, 12JUNCTION PLAKOGLOBIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Desmosomes Connect Intermediate Filaments from Cell to CellMolecular Cell Biology (first edition, 2000): Desmosomes  Arrhythmogenic Right Ventricular Cardiomyopathy  Tests of JUP  PubMed  ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA, FAMILIAL, 12JUNCTION PLAKOGLOBIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Asimaki A, Syrris P, Wichter T, Matthias P, Saffitz JE, McKenna WJ. A novel dominant mutation in plakoglobin causes arrhythmogenic right ventricular cardiomyopathy. Am J Hum Genet. 2007 Nov;81(5):964-73. Epub 2007 Sep 28. Citation on PubMed or Free article on PubMed CentralCabral RM, Liu L, Hogan C, Dopping-Hepenstal PJ, Winik BC, Asial RA, Dobson R, Mein CA, Baselaga PA, Mellerio JE, Nanda A, Boente Mdel C, Kelsell DP, McGrath JA, South AP. Homozygous mutations in the 5' region of the JUP gene result in cutaneous disease but normal heart development in children. J Invest Dermatol. 2010 Jun;130(6):1543-50. doi: 10.1038/jid.2010.7. Epub 2010 Feb 4. Citation on PubMedErken H, Yariz KO, Duman D, Kaya CT, Sayin T, Heper AO, Tekin M. Cardiomyopathy with alopecia and palmoplantar keratoderma (CAPK) is caused by a JUP mutation. Br J Dermatol. 2011 Oct;165(4):917-21. doi: 10.1111/j.1365-2133.2011.10455.x. Epub 2011 Aug 9. Citation on PubMedOMIM: JUNCTION PLAKOGLOBINMcKoy G, Protonotarios N, Crosby A, Tsatsopoulou A, Anastasakis A, Coonar A, Norman M, Baboonian C, Jeffery S, McKenna WJ. Identification of a deletion in plakoglobin in arrhythmogenic right ventricular cardiomyopathy with palmoplantar keratoderma and woolly hair (Naxos disease). Lancet. 2000 Jun 17;355(9221):2119-24. Citation on PubMedPigors M, Kiritsi D, Krmpelmann S, Wagner N, He Y, Podda M, Kohlhase J, Hausser I, Bruckner-Tuderman L, Has C. Lack of plakoglobin leads to lethal congenital epidermolysis bullosa: a novel clinico-genetic entity. Hum Mol Genet. 2011 May 1;20(9):1811-9. doi: 10.1093/hmg/ddr064. Epub 2011 Feb 14. Citation on PubMedProtonotarios N, Tsatsopoulou A, Anastasakis A, Sevdalis E, McKoy G, Stratos K, Gatzoulis K, Tentolouris K, Spiliopoulou C, Panagiotakos D, McKenna W, Toutouzas P. Genotype-phenotype assessment in autosomal recessive arrhythmogenic right ventricular cardiomyopathy (Naxos disease) caused by a deletion in plakoglobin. J Am Coll Cardiol. 2001 Nov 1;38(5):1477-84. Citation on PubMed \n",
            "CYB5D2\t\n",
            "COASY\t   From NCBI Gene:Coenzyme A (CoA) functions as a carrier of acetyl and acyl groups in cells and thus plays an important role in numerous synthetic and degradative metabolic pathways in all organisms. In eukaryotes, CoA and its derivatives are also involved in membrane trafficking and signal transduction. This gene encodes the bifunctional protein coenzyme A synthase (CoAsy) which carries out the last two steps in the biosynthesis of CoA from pantothenic acid (vitamin B5). The phosphopantetheine adenylyltransferase domain of this bifunctional protein catalyzes the conversion of 4'-phosphopantetheine into dephospho-coenzyme A (dpCoA) while its dephospho-CoA kinase domain completes the final step by phosphorylating dpCoA to form CoA. Mutations in this gene are associated with neurodegeneration with brain iron accumulation (NBIA). Alternative splicing results in multiple isoforms. [provided by RefSeq, Apr 2014]From UniProt:Bifunctional enzyme that catalyzes the fourth and fifth sequential steps of CoA biosynthetic pathway. The fourth reaction is catalyzed by the phosphopantetheine adenylyltransferase, coded by the coaD domain; the fifth reaction is catalyzed by the dephospho-CoA kinase, coded by the coaE domain. May act as a point of CoA biosynthesis regulation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:PONTOCEREBELLAR HYPOPLASIA, TYPE 12Neurodegeneration with brain iron accumulation 6From UniProt:Neurodegeneration with brain iron accumulation 6 (NBIA6): A neurodegenerative disorder associated with iron accumulation in the brain, primarily in the basal ganglia. It is characterized by progressive motor and cognitive dysfunction beginning in childhood or young adulthood. Patients show extrapyramidal motor signs, such as spasticity, dystonia, and parkinsonism. [MIM:615643]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q21.2, which is the long (q) arm of chromosome 17 at position 21.2Molecular Location: base pairs 42,562,074 to 42,566,281 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DPCKNBIA6NBPPCH12pOV-2PPATUKR1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COASY      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: COENZYME A SYNTHASEOMIM: NEURODEGENERATION WITH BRAIN IRON ACCUMULATION 6OMIM: PONTOCEREBELLAR HYPOPLASIA, TYPE 12      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COASY  OMIM: COENZYME A SYNTHASEOMIM: NEURODEGENERATION WITH BRAIN IRON ACCUMULATION 6OMIM: PONTOCEREBELLAR HYPOPLASIA, TYPE 12  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "BRCA1\t   The BRCA1 gene provides instructions for making a protein that acts as a tumor suppressor. Tumor suppressor proteins help prevent cells from growing and dividing too rapidly or in an uncontrolled way.The BRCA1 protein is involved in repairing damaged DNA. In the nucleus of many types of normal cells, the BRCA1 protein interacts with several other proteins to mend breaks in DNA. These breaks can be caused by natural and medical radiation or other environmental exposures, and they also occur when chromosomes exchange genetic material in preparation for cell division. By helping to repair DNA, the BRCA1 protein plays a critical role in maintaining the stability of a cell's genetic information.Research suggests that the BRCA1 protein also regulates the activity of other genes and plays an essential role in embryonic development. To carry out these functions, the BRCA1 protein interacts with many other proteins, including other tumor suppressors and proteins that regulate cell division.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Breast cancer     Researchers have identified more than 1,800 mutations in the BRCA1 gene. Many of these mutations are associated with an increased risk of breast cancer in both men and women, as well as several other types of cancer. These mutations are present in every cell in the body and can be passed from one generation to the next. As a result, they are associated with cancers that cluster in families. However, not everyone who inherits a mutation in the BRCA1 gene will develop cancer. Other genetic, environmental, and lifestyle factors also contribute to a person's cancer risk.Most BRCA1 gene mutations lead to the production of an abnormally short version of the BRCA1 protein or prevent any protein from being made from one copy of the gene. As a result, less of this protein is available to help repair damaged DNA or fix mutations that occur in other genes. As these defects accumulate, they can trigger cells to grow and divide uncontrollably to form a tumor. More About This Health Condition      Ovarian cancer     Many of the same BRCA1 gene mutations that increase the risk of breast cancer (described above) also increase the risk of ovarian cancer. Families with these mutations are often said to be affected by hereditary breast and ovarian cancer syndrome. Women with BRCA1 gene mutations have a 35 to 60 percent chance of developing ovarian cancer in their lifetimes, as compared with 1.6 percent in the general population. More About This Health Condition      Prostate cancer     At least five inherited BRCA1 gene mutations have been found to increase the risk of prostate cancer. These mutations likely reduce the BRCA1 protein's ability to repair DNA, allowing potentially damaging mutations to persist in various other genes. The accumulation of damaging mutations can lead to the out-of-control cell growth and division that can cause a tumor to develop. Men who carry a BRCA1 gene mutation that increases the risk of prostate cancer may also be at increased risk for other cancers. More About This Health Condition      Other cancers     Inherited mutations in the BRCA1 gene also increase the risk of several other types of cancer, including pancreatic cancer and colon cancer. These mutations impair the ability of the BRCA1 protein to help repair damaged DNA. As defects accumulate in DNA, they can trigger cells to grow and divide without order to form a tumor. It is not clear why different individuals with BRCA1 mutations develop cancers in different organs. Environmental factors that affect specific organs may contribute to the development of cancers at particular sites.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Researchers have identified more than 1,800 mutations in the BRCA1 gene. Many of these mutations are associated with an increased risk of breast cancer in both men and women, as well as several other types of cancer. These mutations are present in every cell in the body and can be passed from one generation to the next. As a result, they are associated with cancers that cluster in families. However, not everyone who inherits a mutation in the BRCA1 gene will develop cancer. Other genetic, environmental, and lifestyle factors also contribute to a person's cancer risk.Most BRCA1 gene mutations lead to the production of an abnormally short version of the BRCA1 protein or prevent any protein from being made from one copy of the gene. As a result, less of this protein is available to help repair damaged DNA or fix mutations that occur in other genes. As these defects accumulate, they can trigger cells to grow and divide uncontrollably to form a tumor. More About This Health Condition  Many of the same BRCA1 gene mutations that increase the risk of breast cancer (described above) also increase the risk of ovarian cancer. Families with these mutations are often said to be affected by hereditary breast and ovarian cancer syndrome. Women with BRCA1 gene mutations have a 35 to 60 percent chance of developing ovarian cancer in their lifetimes, as compared with 1.6 percent in the general population. More About This Health Condition  At least five inherited BRCA1 gene mutations have been found to increase the risk of prostate cancer. These mutations likely reduce the BRCA1 protein's ability to repair DNA, allowing potentially damaging mutations to persist in various other genes. The accumulation of damaging mutations can lead to the out-of-control cell growth and division that can cause a tumor to develop. Men who carry a BRCA1 gene mutation that increases the risk of prostate cancer may also be at increased risk for other cancers. More About This Health Condition  Inherited mutations in the BRCA1 gene also increase the risk of several other types of cancer, including pancreatic cancer and colon cancer. These mutations impair the ability of the BRCA1 protein to help repair damaged DNA. As defects accumulate in DNA, they can trigger cells to grow and divide without order to form a tumor. It is not clear why different individuals with BRCA1 mutations develop cancers in different organs. Environmental factors that affect specific organs may contribute to the development of cancers at particular sites.    Cytogenetic Location: 17q21.31, which is the long (q) arm of chromosome 17 at position 21.31Molecular Location: base pairs 43,044,295 to 43,125,483 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BRCA1 geneBRCA1_HUMANBRCC1breast cancer 1breast cancer 1 genebreast cancer 1, early onsetbreast cancer 1, early onset genebreast cancer type 1 susceptibility genebreast cancer type 1 susceptibility proteinIRISPPP1R53PSCPRNF53     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Cancer Medicine (sixth edition, 2003): BRCA1 and BRCA2: Gene Structure and FunctionMadame Curie Bioscience Database: Origin, Recognition, Signaling and Repair of DNA Double-Strand Breaks in Mammalian CellsNational Cancer Institute: Genetics of Breast and Gynecologic Cancers (PDQ)      Clinical Information from GeneReviews (1 link)     BRCA1- and BRCA2-Associated Hereditary Breast and Ovarian Cancer      Tests Listed in the Genetic Testing Registry (1 link)     Tests of BRCA1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     BREAST CANCER 1 GENE      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics Web: BRCA1ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Cancer Medicine (sixth edition, 2003): BRCA1 and BRCA2: Gene Structure and FunctionMadame Curie Bioscience Database: Origin, Recognition, Signaling and Repair of DNA Double-Strand Breaks in Mammalian CellsNational Cancer Institute: Genetics of Breast and Gynecologic Cancers (PDQ)  BRCA1- and BRCA2-Associated Hereditary Breast and Ovarian Cancer  Tests of BRCA1  PubMed  BREAST CANCER 1 GENE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics Web: BRCA1ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Antoniou A, Pharoah PD, Narod S, Risch HA, Eyfjord JE, Hopper JL, Loman N, Olsson H, Johannsson O, Borg A, Pasini B, Radice P, Manoukian S, Eccles DM, Tang N, Olah E, Anton-Culver H, Warner E, Lubinski J, Gronwald J, Gorski B, Tulinius H, Thorlacius S, Eerola H, Nevanlinna H, Syrjkoski K, Kallioniemi OP, Thompson D, Evans C, Peto J, Lalloo F, Evans DG, Easton DF. Average risks of breast and ovarian cancer associated with BRCA1 or BRCA2 mutations detected in case Series unselected for family history: a combined analysis of 22 studies. Am J Hum Genet. 2003 May;72(5):1117-30. Epub 2003 Apr 3. Erratum in: Am J Hum Genet. 2003 Sep;73(3):709. Citation on PubMed or Free article on PubMed CentralChen S, Parmigiani G. Meta-analysis of BRCA1 and BRCA2 penetrance. J Clin Oncol. 2007 Apr 10;25(11):1329-33. Citation on PubMed or Free article on PubMed CentralFoulkes WD, Shuen AY. In brief: BRCA1 and BRCA2. J Pathol. 2013 Aug;230(4):347-9. doi: 10.1002/path.4205. Review. Citation on PubMedFoulkes WD. BRCA1 and BRCA2 - update and implications on the genetics of breast cancer: a clinical perspective. Clin Genet. 2014 Jan;85(1):1-4. doi: 10.1111/cge.12291. Epub 2013 Oct 25. Review. Erratum in: Clin Genet. 2014 Mar;85(3):302. Citation on PubMedFriebel TM, Domchek SM, Rebbeck TR. Modifiers of cancer risk in BRCA1 and BRCA2 mutation carriers: systematic review and meta-analysis. J Natl Cancer Inst. 2014 Jun;106(6):dju091. doi: 10.1093/jnci/dju091. Review. Erratum in: J Natl Cancer Inst. 2014 Aug;106(8):dju235 doi:10.1093/jnci/dju235. Citation on PubMed or Free article on PubMed CentralKobayashi H, Ohno S, Sasaki Y, Matsuura M. Hereditary breast and ovarian cancer susceptibility genes (review). Oncol Rep. 2013 Sep;30(3):1019-29. doi: 10.3892/or.2013.2541. Epub 2013 Jun 19. Review. Citation on PubMedNational Cancer Institute: Genetics of Breast and Gynecologic Cancers (PDQ)Nelson HD, Fu R, Goddard K, Mitchell JP, Okinaka-Hu L, Pappas M, Zakher B. Risk Assessment, Genetic Counseling, and Genetic Testing for BRCA-Related Cancer: Systematic Review to Update the U.S. Preventive Services Task Force Recommendation [Internet]. Rockville (MD): Agency for Healthcare Research and Quality (US); 2013 Dec. Available from http://www.ncbi.nlm.nih.gov/books/NBK179201/ Citation on PubMedPennington KP, Swisher EM. Hereditary ovarian cancer: beyond the usual suspects. Gynecol Oncol. 2012 Feb;124(2):347-53. doi: 10.1016/j.ygyno.2011.12.415. Review. Citation on PubMedPetrucelli N, Daly MB, Pal T. BRCA1- and BRCA2-Associated Hereditary Breast and Ovarian Cancer. 1998 Sep 4 [updated 2016 Dec 15]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1247/ Citation on PubMedShulman LP. Hereditary breast and ovarian cancer (HBOC): clinical features and counseling for BRCA1 and BRCA2, Lynch syndrome, Cowden syndrome, and Li-Fraumeni syndrome. Obstet Gynecol Clin North Am. 2010 Mar;37(1):109-33, Table of Contents. doi: 10.1016/j.ogc.2010.03.003. Review. Citation on PubMedToss A, Tomasello C, Razzaboni E, Contu G, Grandi G, Cagnacci A, Schilder RJ, Cortesi L. Hereditary ovarian cancer: not only BRCA 1 and 2 genes. Biomed Res Int. 2015;2015:341723. doi: 10.1155/2015/341723. Epub 2015 May 17. Review. Citation on PubMed or Free article on PubMed CentralWalsh T, Casadei S, Coats KH, Swisher E, Stray SM, Higgins J, Roach KC, Mandell J, Lee MK, Ciernikova S, Foretova L, Soucek P, King MC. Spectrum of mutations in BRCA1, BRCA2, CHEK2, and TP53 in families at high risk of breast cancer. JAMA. 2006 Mar 22;295(12):1379-88. Citation on PubMed \n",
            "MPP2\t\n",
            "CCDC103\t   From NCBI Gene:This gene encodes a protein that contains a coiled-coil domain. [provided by RefSeq, Apr 2012]From UniProt:Dynein-attachment factor required for cilia motility.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary ciliary dyskinesiaFrom NCBI Gene:Ciliary dyskinesia, primary, 17From UniProt:Ciliary dyskinesia, primary, 17 (CILD17): A disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia; reduced fertility is often observed in male patients due to abnormalities of sperm tails. Half of the patients exhibit randomization of left-right body asymmetry and situs inversus, due to dysfunction of monocilia at the embryonic node. Primary ciliary dyskinesia associated with situs inversus is referred to as Kartagener syndrome. [MIM:614679]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q21.31, which is the long (q) arm of chromosome 17 at position 21.31Molecular Location: base pairs 44,899,712 to 44,903,679 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CILD17PR46bSMH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CCDC103      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 17OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 103      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CCDC103  OMIM: CILIARY DYSKINESIA, PRIMARY, 17OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 103  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PLCD3\t\n",
            "MAP3K14\t\n",
            "LRRC37A\t\n",
            "LRRC37A2\t\n",
            "GGT6\t\n",
            "WNT3\t   The WNT3 gene is part of a large family of WNT genes, which play critical roles in development before birth.  WNT genes provide instructions for making proteins that participate in chemical signaling pathways in the body.  These pathways control the activity of certain genes and regulate the interactions between cells during embryonic development.Research in animals indicates that the protein produced from the WNT3 gene is critical for the outgrowth of the limbs in the developing embryo.  The WNT3 protein also appears to play an important role in determining the anterior-posterior axis (the imaginary line that runs from head to tail in animals) during the earliest stages of embryonic development.  Additionally, the effects of mutations in the human WNT3 gene suggest that the protein may be involved in the normal formation of the facial features, head, heart, lungs, nervous system, skeleton, and genitalia.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Tetra-amelia syndrome     A mutation in the WNT3 gene has been shown to cause tetra-amelia syndrome among members of one large family from Turkey.  This mutation, which occurs in both copies of the WNT3 gene in each cell, replaces one protein building block (amino acid) with a premature stop signal in the instructions for making the WNT3 protein.  This mutation is written as Gln83Ter or Q83X.Researchers believe that the Gln83Ter mutation results in the production of an abnormally short, nonfunctional version of the WNT3 protein.  Loss of the WNT3 protein disrupts normal limb formation before birth and leads to the other serious birth defects associated with tetra-amelia syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A mutation in the WNT3 gene has been shown to cause tetra-amelia syndrome among members of one large family from Turkey.  This mutation, which occurs in both copies of the WNT3 gene in each cell, replaces one protein building block (amino acid) with a premature stop signal in the instructions for making the WNT3 protein.  This mutation is written as Gln83Ter or Q83X.Researchers believe that the Gln83Ter mutation results in the production of an abnormally short, nonfunctional version of the WNT3 protein.  Loss of the WNT3 protein disrupts normal limb formation before birth and leads to the other serious birth defects associated with tetra-amelia syndrome. More About This Health Condition    Cytogenetic Location: 17q21.31-q21.32, which is the long (q) arm of chromosome 17 between positions 21.31 and 21.32Molecular Location: base pairs 46,762,506 to 46,818,760 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        INT4Oncogene INT4Proto-oncogene protein Wnt-3wingless-type MMTV integration site family member 3wingless-type MMTV integration site family, member 3WNT-3 proto-oncogene proteinWNT3_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Developmental Biology (sixth edition, 2000): The Wnt Family      Tests Listed in the Genetic Testing Registry (1 link)     Tests of WNT3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     WINGLESS-TYPE MMTV INTEGRATION SITE FAMILY, MEMBER 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): The Wnt Family  Tests of WNT3  PubMed  WINGLESS-TYPE MMTV INTEGRATION SITE FAMILY, MEMBER 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Barrow JR, Howell WD, Rule M, Hayashi S, Thomas KR, Capecchi MR, McMahon AP. Wnt3 signaling in the epiblast is required for proper orientation of the anteroposterior axis. Dev Biol. 2007 Dec 1;312(1):312-20. Epub 2007 Sep 26. Citation on PubMedBarrow JR, Thomas KR, Boussadia-Zahui O, Moore R, Kemler R, Capecchi MR, McMahon AP. Ectodermal Wnt3/beta-catenin signaling is required for the establishment and maintenance of the apical ectodermal ridge. Genes Dev. 2003 Feb 1;17(3):394-409. Citation on PubMed or Free article on PubMed CentralKatoh M. Molecular cloning and characterization of human WNT3. Int J Oncol. 2001 Nov;19(5):977-82. Citation on PubMedLiu P, Wakamiya M, Shea MJ, Albrecht U, Behringer RR, Bradley A. Requirement for Wnt3 in vertebrate axis formation. Nat Genet. 1999 Aug;22(4):361-5. Citation on PubMedNiemann S, Zhao C, Pascu F, Stahl U, Aulepp U, Niswander L, Weber JL, Mller U. Homozygous WNT3 mutation causes tetra-amelia in a large consanguineous family. Am J Hum Genet. 2004 Mar;74(3):558-63. Epub 2004 Feb 5. Citation on PubMed or Free article on PubMed CentralRoelink H, Wang J, Black DM, Solomon E, Nusse R. Molecular cloning and chromosomal localization to 17q21 of the human WNT3 gene. Genomics. 1993 Sep;17(3):790-2. Citation on PubMed \n",
            "COPZ2\t\n",
            "HOXB1\t   From NCBI Gene:This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, located on different chromosomes, consisting of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXB genes located in a cluster on chromosome 17. [provided by RefSeq, Jul 2008]From UniProt:Sequence-specific transcription factor which is part of a developmental regulatory system that provides cells with specific positional identities on the anterior-posterior axis. Acts on the anterior body structures.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Hereditary congenital facial paresis 3From UniProt:Facial paresis, hereditary congenital, 3 (HCFP3): A form of facial paresis, a disease characterized by isolated dysfunction of the facial nerve (CN VII). HCFP3 patients are affected by bilateral facial palsy, facial muscle weakness of muscles innervated by CN VII, hearing loss, and strabismus. [MIM:614744]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q21.32, which is the long (q) arm of chromosome 17 at position 21.32Molecular Location: base pairs 48,529,445 to 48,530,910 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HCFP3Hox-2.9HOX2HOX2I     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HOXB1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FACIAL PARESIS, HEREDITARY CONGENITAL, 3OMIM: HOMEOBOX B1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HOXB1  OMIM: FACIAL PARESIS, HEREDITARY CONGENITAL, 3OMIM: HOMEOBOX B1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TTLL6\t\n",
            "VMO1\t\n",
            "PPP1R9B\t\n",
            "GP1BA\t   The GP1BA gene provides instructions for making a protein called glycoprotein Ib-alpha (GPIb). This protein is one piece (subunit) of a protein complex called GPIb-IX-V, which plays a role in blood clotting. GPIb-IX-V is found on the surface of small cell fragments called platelets, which circulate in blood and are an essential component of blood clots. The complex can attach (bind) to a protein called von Willebrand factor, fitting together like a lock and its key. Von Willebrand factor is found on the inside surface of blood vessels, particularly when there is an injury. Binding of the GPIb-IX-V complex to von Willebrand factor allows platelets to stick to the blood vessel wall at the site of the injury. These platelets form clots, plugging holes in the blood vessels to help stop bleeding.To form the GPIb-IX-V complex, GPIb interacts with other protein subunits called GPIb-beta, GPIX, and GPV, each of which is produced from a different gene. GPIb is essential for assembly of the complex at the platelet surface. It is the piece of the complex that interacts with von Willebrand factor to trigger blood clotting. GPIb also interacts with other blood clotting proteins to aid in other steps of the clotting process.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Bernard-Soulier syndrome     At least 54 GP1BA gene mutations have been found to cause Bernard-Soulier syndrome, a condition characterized by a reduced number of platelets that are larger than normal (macrothrombocytopenia) and excessive bleeding. Some of these mutations lead to production of an altered GPIb subunit that is likely broken down too soon or that cannot get to the platelet surface. Lack of this subunit on the surface of platelets prevents formation of the GPIb-IX-V complex. Without GPIb-IX-V, platelets cannot come together at the site of an injury to form a clot, leading to the bleeding problems associated with Bernard-Soulier syndrome. Other mutations lead to production of a subunit that can form GPIb-IX-V complexes but cannot interact with von Willebrand factor, which also impairs the accumulation of platelets necessary for clotting. More About This Health Condition      Other disorders     At least six mutations in the GP1BA gene can cause another bleeding disorder called platelet-type von Willebrand disease (also known as pseudo-von Willebrand disease). This disorder is characterized by a reduced number of platelets in the blood (thrombocytopenia) and mild bleeding abnormalities. In contrast to mutations that cause Bernard-Soulier syndrome (described above), mutations involved in platelet-type von Willebrand disease lead to excessive binding of the GPIb-IX-V complex to von Willebrand factor. Because platelets containing the altered complex attach to von Willebrand factor without an injury to the blood vessel, fewer platelets are available for clot formation when an injury occurs, which leads to excessive bleeding in people with platelet-type von Willebrand disease.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 54 GP1BA gene mutations have been found to cause Bernard-Soulier syndrome, a condition characterized by a reduced number of platelets that are larger than normal (macrothrombocytopenia) and excessive bleeding. Some of these mutations lead to production of an altered GPIb subunit that is likely broken down too soon or that cannot get to the platelet surface. Lack of this subunit on the surface of platelets prevents formation of the GPIb-IX-V complex. Without GPIb-IX-V, platelets cannot come together at the site of an injury to form a clot, leading to the bleeding problems associated with Bernard-Soulier syndrome. Other mutations lead to production of a subunit that can form GPIb-IX-V complexes but cannot interact with von Willebrand factor, which also impairs the accumulation of platelets necessary for clotting. More About This Health Condition  At least six mutations in the GP1BA gene can cause another bleeding disorder called platelet-type von Willebrand disease (also known as pseudo-von Willebrand disease). This disorder is characterized by a reduced number of platelets in the blood (thrombocytopenia) and mild bleeding abnormalities. In contrast to mutations that cause Bernard-Soulier syndrome (described above), mutations involved in platelet-type von Willebrand disease lead to excessive binding of the GPIb-IX-V complex to von Willebrand factor. Because platelets containing the altered complex attach to von Willebrand factor without an injury to the blood vessel, fewer platelets are available for clot formation when an injury occurs, which leads to excessive bleeding in people with platelet-type von Willebrand disease.    Cytogenetic Location: 17p13.2, which is the short (p) arm of chromosome 17 at position 13.2Molecular Location: base pairs 4,932,275 to 4,935,030 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        antigen CD42b-alphaBDPLT1BDPLT3BSSCD42BCD42b-alphaDBPLT3glycoprotein Ib (platelet), alpha polypeptideglycoprotein Ib platelet alpha subunitGP-Ib alphaGP1BGPIbAGPIbalphaplatelet glycoprotein Ib alpha chain precursorplatelet membrane glycoprotein 1b-alpha subunitVWDP     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Clinical Methods: The History, Physical, and Laboratory Examinations (third edition, 1990): Excessive Bleeding and Bruising, Basic SciencePlatelet-Vessel Wall Interactions in Hemostasis and Thrombosis (2010): Platelet Adhesion to Vascular Walls      Tests Listed in the Genetic Testing Registry (1 link)     Tests of GP1BA      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     GLYCOPROTEIN Ib, PLATELET, ALPHA POLYPEPTIDEPSEUDO-VON WILLEBRAND DISEASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Clinical Methods: The History, Physical, and Laboratory Examinations (third edition, 1990): Excessive Bleeding and Bruising, Basic SciencePlatelet-Vessel Wall Interactions in Hemostasis and Thrombosis (2010): Platelet Adhesion to Vascular Walls  Tests of GP1BA  PubMed  GLYCOPROTEIN Ib, PLATELET, ALPHA POLYPEPTIDEPSEUDO-VON WILLEBRAND DISEASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cauwenberghs N, Vanhoorelbeke K, Vauterin S, Deckmyn H. Structural determinants within platelet glycoprotein Ibalpha involved in its binding to von Willebrand factor. Platelets. 2000 Nov;11(7):373-8. Review. Citation on PubMedOMIM: GLYCOPROTEIN Ib, PLATELET, ALPHA POLYPEPTIDELi R, Emsley J. The organizing principle of the platelet glycoprotein Ib-IX-V complex. J Thromb Haemost. 2013 Apr;11(4):605-14. doi: 10.1111/jth.12144. Review. Citation on PubMed or Free article on PubMed CentralOthman M, Kaur H, Emsley J. Platelet-type von Willebrand disease: new insights into the molecular pathophysiology of a unique platelet defect. Semin Thromb Hemost. 2013 Sep;39(6):663-73. doi: 10.1055/s-0033-1353442. Epub 2013 Aug 11. Review. Citation on PubMedSavoia A, Kunishima S, De Rocco D, Zieger B, Rand ML, Pujol-Moix N, Caliskan U, Tokgoz H, Pecci A, Noris P, Srivastava A, Ward C, Morel-Kopp MC, Alessi MC, Bellucci S, Beurrier P, de Maistre E, Favier R, Hzard N, Hurtaud-Roux MF, Latger-Cannard V, Lavenu-Bombled C, Proulle V, Meunier S, Ngrier C, Nurden A, Randrianaivo H, Fabris F, Platokouki H, Rosenberg N, HadjKacem B, Heller PG, Karimi M, Balduini CL, Pastore A, Lanza F. Spectrum of the mutations in Bernard-Soulier syndrome. Hum Mutat. 2014 Sep;35(9):1033-45. doi: 10.1002/humu.22607. Epub 2014 Jul 15. Review. Citation on PubMedSchade AJ, Arya M, Gao S, Diz-Kkkaya R, Anvari B, McIntire LV, Lpez JA, Dong JF. Cytoplasmic truncation of glycoprotein Ib alpha weakens its interaction with von Willebrand factor and impairs cell adhesion. Biochemistry. 2003 Feb 25;42(7):2245-51. Citation on PubMed \n",
            "EME1\t\n",
            "USP6\t\n",
            "STXBP4\t\n",
            "NLRP1\t   The NLRP1 gene provides instructions for making a member of a family of proteins called nucleotide-binding domain and leucine-rich repeat containing (NLR) proteins. These proteins are involved in the immune system, helping to regulate the process of inflammation. Inflammation occurs when the immune system sends signaling molecules and white blood cells to a site of injury or disease to fight microbial invaders and facilitate tissue repair. The body then stops (inhibits) the inflammatory response to prevent damage to its own cells and tissues.The NLRP1 protein is involved in the assembly of a molecular complex called an inflammasome, which helps trigger the inflammatory process in response to the presence of bacteria or viruses. Researchers believe that the NLRP1 protein may also play a role in the self-destruction of cells (apoptosis).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Vitiligo     Studies have associated variations in the NLRP1 gene with an increased risk of vitiligo, an autoimmune condition that results in patchy changes in skin coloring (pigmentation).One of the NLRP1 gene variations associated with vitiligo changes the protein building block (amino acid) leucine to the amino acid histidine at position 155 in the NLRP1 protein sequence, written as Leu155His or L155H. This and other variations likely affect the activity of the NLRP1 protein, making it more difficult for the body to control inflammation and prevent the immune system from attacking its own tissues. While the pigment loss associated with vitiligo results from the immune system attacking pigment-producing cells (melanocytes) in the skin, it is unclear what specific circumstances trigger the immune system to do so. The condition probably results from a combination of genetic and environmental factors, most of which have not been identified. More About This Health Condition      Autoimmune Addison disease     Genetics Home Reference provides information about autoimmune Addison disease. More About This Health Condition      Autoimmune disorders     NLRP1 gene variations have been associated with an increased risk of autoimmune disorders. Autoimmune disorders occur when the immune system malfunctions and attacks the body's tissues and organs. These disorders include type 1 diabetes, in which insulin-producing cells in the pancreas are destroyed, and Addison disease, which is caused by autoimmune damage to the small hormone-producing glands on top of each kidney (adrenal glands). Certain NLRP1 gene variations seem to make affected individuals more prone to overactivity of the immune system, resulting in damage to the body's own tissues and organs.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Studies have associated variations in the NLRP1 gene with an increased risk of vitiligo, an autoimmune condition that results in patchy changes in skin coloring (pigmentation).One of the NLRP1 gene variations associated with vitiligo changes the protein building block (amino acid) leucine to the amino acid histidine at position 155 in the NLRP1 protein sequence, written as Leu155His or L155H. This and other variations likely affect the activity of the NLRP1 protein, making it more difficult for the body to control inflammation and prevent the immune system from attacking its own tissues. While the pigment loss associated with vitiligo results from the immune system attacking pigment-producing cells (melanocytes) in the skin, it is unclear what specific circumstances trigger the immune system to do so. The condition probably results from a combination of genetic and environmental factors, most of which have not been identified. More About This Health Condition  Genetics Home Reference provides information about autoimmune Addison disease. More About This Health Condition  NLRP1 gene variations have been associated with an increased risk of autoimmune disorders. Autoimmune disorders occur when the immune system malfunctions and attacks the body's tissues and organs. These disorders include type 1 diabetes, in which insulin-producing cells in the pancreas are destroyed, and Addison disease, which is caused by autoimmune damage to the small hormone-producing glands on top of each kidney (adrenal glands). Certain NLRP1 gene variations seem to make affected individuals more prone to overactivity of the immune system, resulting in damage to the body's own tissues and organs.    Cytogenetic Location: 17p13.2, which is the short (p) arm of chromosome 17 at position 13.2Molecular Location: base pairs 5,501,399 to 5,584,512 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CARD7CLR17.1DEFCAPDKFZp586O1822KIAA0926NACNALP1NALP1_HUMANNLR family, pyrin domain containing 1SLEV1VAMAS1     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     National Institutes of Health: Scientists Implicate Gene in Vitiligo and Other Autoimmune Diseases      Tests Listed in the Genetic Testing Registry (1 link)     Tests of NLRP1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     NLR FAMILY, PYRIN DOMAIN-CONTAINING 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      National Institutes of Health: Scientists Implicate Gene in Vitiligo and Other Autoimmune Diseases  Tests of NLRP1  PubMed  NLR FAMILY, PYRIN DOMAIN-CONTAINING 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Jin Y, Birlea SA, Fain PR, Spritz RA. Genetic variations in NALP1 are associated with generalized vitiligo in a Romanian population. J Invest Dermatol. 2007 Nov;127(11):2558-62. Epub 2007 Jul 19. Citation on PubMedJin Y, Mailloux CM, Gowan K, Riccardi SL, LaBerge G, Bennett DC, Fain PR, Spritz RA. NALP1 in vitiligo-associated multiple autoimmune disease. N Engl J Med. 2007 Mar 22;356(12):1216-25. Citation on PubMedJin Y, Riccardi SL, Gowan K, Fain PR, Spritz RA. Fine-mapping of vitiligo susceptibility loci on chromosomes 7 and 9 and interactions with NLRP1 (NALP1). J Invest Dermatol. 2010 Mar;130(3):774-83. doi: 10.1038/jid.2009.273. Epub 2009 Sep 3. Citation on PubMed or Free article on PubMed CentralOMIM: NLR FAMILY, PYRIN DOMAIN-CONTAINING 1Pontillo A, Brandao L, Guimaraes R, Segat L, Araujo J, Crovella S. Two SNPs in NLRP3 gene are involved in the predisposition to type-1 diabetes and celiac disease in a pediatric population from northeast Brazil. Autoimmunity. 2010 Dec;43(8):583-9. doi: 10.3109/08916930903540432. Epub 2010 Apr 7. Citation on PubMedSmith AG, Sturm RA. Multiple genes and locus interactions in susceptibility to vitiligo. J Invest Dermatol. 2010 Mar;130(3):643-5. doi: 10.1038/jid.2009.403. Citation on PubMedSpritz RA. The genetics of generalized vitiligo and associated autoimmune diseases. Pigment Cell Res. 2007 Aug;20(4):271-8. Review. Citation on PubMedTaeb A. NALP1 and the inflammasomes: challenging our perception of vitiligo and vitiligo-related autoimmune disorders. Pigment Cell Res. 2007 Aug;20(4):260-2. Citation on PubMedZurawek M, Fichna M, Januszkiewicz-Lewandowska D, Gryczyska M, Fichna P, Nowak J. A coding variant in NLRP1 is associated with autoimmune Addison's disease. Hum Immunol. 2010 May;71(5):530-4. doi: 10.1016/j.humimm.2010.02.004. Epub 2010 Mar 1. Citation on PubMed \n",
            "AKAP1\t\n",
            "RNF43\t   From NCBI Gene:The protein encoded by this gene is a RING-type E3 ubiquitin ligase and is predicted to contain a transmembrane domain, a protease-associated domain, an ectodomain, and a cytoplasmic RING domain. This protein is thought to negatively regulate Wnt signaling, and expression of this gene results in an increase in ubiquitination of frizzled receptors, an alteration in their subcellular distribution, resulting in reduced surface levels of these receptors. Mutations in this gene have been reported in multiple tumor cells, including colorectal and endometrial cancers. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2015]From UniProt:E3 ubiquitin-protein ligase that acts as a negative regulator of the Wnt signaling pathway by mediating the ubiquitination, endocytosis and subsequent degradation of Wnt receptor complex components Frizzled. Acts on both canonical and non-canonical Wnt signaling pathway (PubMed:18313049, PubMed:22575959, PubMed:22895187). Along with RSPO2 and ZNRF3, constitutes a master switch that governs limb specification (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:CholangiocarcinomaFrom NCBI Gene:Sessile serrated polyposis cancer syndromeFrom UniProt:Sessile serrated polyposis cancer syndrome (SSPCS): A rare disease characterized by multiple and/or large serrated polyps developing in the colon, and an increased personal and familial risk of colorectal cancer. A patient is diagnosed with SSPCS if at least one of the following criteria is met: the presence of at least five sessile serrated polyps proximal to the sigmoid colon, two of which are greater than 10 mm in diameter; the presence of any number of serrated polyps occurring proximal to the sigmoid colon in an individual who has a first-degree relative with serrated polyposis; the presence of more than 20 serrated polyps of any size distributed throughout the colon. Sessile serrated polyps are also known as sessile serrated adenomas (SSA) and are estimated to be responsible for 20 to 35% of all colon cancers. Individuals with SSPCS may have a strong personal or family history of extracolonic cancers. [MIM:617108]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q22, which is the long (q) arm of chromosome 17 at position 22Molecular Location: base pairs 58,352,500 to 58,418,894 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        RNF124SSPCSURCC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RNF43      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: RING FINGER PROTEIN 43OMIM: SESSILE SERRATED POLYPOSIS CANCER SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RNF43  OMIM: RING FINGER PROTEIN 43OMIM: SESSILE SERRATED POLYPOSIS CANCER SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PPM1E\t\n",
            "BCAS3\t\n",
            "WSCD1\t\n",
            "MARCH10\t\n",
            "DCAF7\t\n",
            "CSHL1\t\n",
            "LRRC37A3\t\n",
            "PITPNM3\t   From NCBI Gene:This gene encodes a member of a family of membrane-associated phosphatidylinositol transfer domain-containing proteins. The calcium-binding protein has phosphatidylinositol (PI) transfer activity and interacts with the protein tyrosine kinase PTK2B (also known as PYK2). The protein is homologous to a Drosophila protein that is implicated in the visual transduction pathway in flies. Mutations in this gene result in autosomal dominant cone dystrophy. Multiple transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Sep 2009]From UniProt:Catalyzes the transfer of phosphatidylinositol and phosphatidylcholine between membranes (in vitro) (By similarity). Binds calcium ions.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Cone-rod dystrophyFrom NCBI Gene:Cone-rod dystrophy 5From UniProt:Cone-rod dystrophy 5 (CORD5): An inherited retinal dystrophy characterized by retinal pigment deposits visible on fundus examination, predominantly in the macular region, and initial loss of cone photoreceptors followed by rod degeneration. This leads to decreased visual acuity and sensitivity in the central visual field, followed by loss of peripheral vision. Severe loss of vision occurs earlier than in retinitis pigmentosa, due to cone photoreceptors degenerating at a higher rate than rod photoreceptors. [MIM:600977]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.2-p13.1, which is the short (p) arm of chromosome 17 between positions 13.2 and 13.1Molecular Location: base pairs 6,451,263 to 6,556,557 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACKR6CORD5NIR1RDGBA3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PITPNM3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CONE-ROD DYSTROPHY 5OMIM: PHOSPHATIDYLINOSITOL TRANSFER PROTEIN, MEMBRANE-ASSOCIATED, 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PITPNM3  OMIM: CONE-ROD DYSTROPHY 5OMIM: PHOSPHATIDYLINOSITOL TRANSFER PROTEIN, MEMBRANE-ASSOCIATED, 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GEMIN4\t   From NCBI Gene:The product of this gene is part of a large complex localized to the cytoplasm, nucleoli, and to discrete nuclear bodies called Gemini bodies (gems). The complex functions in spliceosomal snRNP assembly in the cytoplasm, and regenerates spliceosomes required for pre-mRNA splicing in the nucleus. The encoded protein directly interacts with a DEAD box protein and several spliceosome core proteins. Alternatively spliced transcript variants have been described, but their biological validity has not been determined. [provided by RefSeq, Jul 2008]From UniProt:The SMN complex plays a catalyst role in the assembly of small nuclear ribonucleoproteins (snRNPs), the building blocks of the spliceosome. Thereby, plays an important role in the splicing of cellular pre-mRNAs. Most spliceosomal snRNPs contain a common set of Sm proteins SNRPB, SNRPD1, SNRPD2, SNRPD3, SNRPE, SNRPF and SNRPG that assemble in a heptameric protein ring on the Sm site of the small nuclear RNA to form the core snRNP. In the cytosol, the Sm proteins SNRPD1, SNRPD2, SNRPE, SNRPF and SNRPG are trapped in an inactive 6S pICln-Sm complex by the chaperone CLNS1A that controls the assembly of the core snRNP. Dissociation by the SMN complex of CLNS1A from the trapped Sm proteins and their transfer to an SMN-Sm complex triggers the assembly of core snRNPs and their transport to the nucleus.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:NEURODEVELOPMENTAL DISORDER WITH MICROCEPHALY, CATARACTS, AND RENAL ABNORMALITIESFrom UniProt:Neurodevelopmental disorder with microcephaly, cataracts, and renal abnormalities (NEDMCR): An autosomal recessive, severe neurodevelopmental disorder characterized by global developmental delay since infancy, microcephaly, poor or absent speech, and inability to walk or spasticity. Additional features include renal abnormalities, congenital cataracts, gastroesophageal reflux disease, seizures with onset in infancy or childhood, hyporeflexia, and non-specific white matter abnormalities on brain imaging. [MIM:617913]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.3, which is the short (p) arm of chromosome 17 at position 13.3Molecular Location: base pairs 744,421 to 752,868 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HC56HCAP1HHRF-1NEDMCRp97     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GEMIN4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GEM-ASSOCIATED PROTEIN 4OMIM: NEURODEVELOPMENTAL DISORDER WITH MICROCEPHALY, CATARACTS, AND RENAL ABNORMALITIES      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GEMIN4  OMIM: GEM-ASSOCIATED PROTEIN 4OMIM: NEURODEVELOPMENTAL DISORDER WITH MICROCEPHALY, CATARACTS, AND RENAL ABNORMALITIES  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KIAA0753\t   From NCBI Gene:This gene encodes a subunit of a protein complex that regulates ciliogenesis and cilia maintenance. The encoded protein has also been shown to regulate centriolar duplication. Mutations in this gene cause an orofaciodigital syndrome and a form of Joubert syndrome in human patients. [provided by RefSeq, May 2017]From UniProt:Involved in centriole duplication. Positively regulates CEP63 centrosomal localization. Required for WDR62 centrosomal localization and promotes the centrosomal localization of CDK2 (PubMed:24613305, PubMed:26297806).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Orofaciodigital syndrome XVFrom UniProt:Orofaciodigital syndrome 15 (OFD15): A form of orofaciodigital syndrome, a group of heterogeneous disorders characterized by malformations of the oral cavity, face and digits, and associated phenotypic abnormalities that lead to the delineation of various subtypes. OFD15 features include facial dysmorphism, lobulated tongue, clefting of the alveolar ridges, left hand postaxial polydactyly, broad right hallux and left hallux duplication, and intermittent respiratory difficulty. Brain anomalies include vermis hypoplasia with molar tooth sign, agenesis of corpus callosum, and ventricular dilation. OFD15 inheritance is autosomal recessive. [MIM:617127]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17p13.1, which is the short (p) arm of chromosome 17 at position 13.1Molecular Location: base pairs 6,578,151 to 6,640,927 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MNROFIP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KIAA0753      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KIAA0753 GENEOMIM: OROFACIODIGITAL SYNDROME XV      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KIAA0753  OMIM: KIAA0753 GENEOMIM: OROFACIODIGITAL SYNDROME XV  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FBXO39\t\n",
            "ABCA9\t\n",
            "ABCA6\t\n",
            "BCL6B\t\n",
            "BTBD17\t\n",
            "C17orf77\t\n",
            "GRIN2C\t\n",
            "FADS6\t\n",
            "OTOP2\t\n",
            "FGF11\t\n",
            "TSEN54\t   The TSEN54 gene provides instructions for making one part (subunit) of an enzyme called the tRNA splicing endonuclease complex. This complex helps process several types of RNA molecules, which are chemical cousins of DNA.The tRNA splicing endonuclease complex is particularly important for the normal processing of a form of RNA known as transfer RNA (tRNA). tRNA molecules help assemble protein building blocks called amino acids into full-length proteins. However, before they can assemble proteins, tRNAs must be processed into mature molecules. In particular, regions called introns need to be removed from some tRNAs for the molecules to be functional. The tRNA splicing endonuclease complex recognizes and then removes introns to help produce mature tRNA molecules.Studies suggest that the tRNA splicing endonuclease complex may also be involved in processing another form of RNA known as messenger RNA (mRNA). mRNA serves as a genetic blueprint for making proteins. Researchers suspect that the tRNA splicing endonuclease complex cuts (cleaves) one end of mRNA molecules so a string of adenines (one of the building blocks of RNA) can be added. This process is known as polyadenylation, and the string of adenines is known as a poly(A) tail. The poly(A) tail signals the stopping point for protein production and protects mRNA from being broken down before protein production occurs.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Pontocerebellar hypoplasia     Several mutations in the TSEN54 gene have been identified in people with a disorder of brain development called pontocerebellar hypoplasia. The major features of this condition include delayed development, problems with movement, and intellectual disability. TSEN54 gene mutations are the most frequent cause of a form of the disorder designated pontocerebellar hypoplasia type 2 (PCH2). When PCH2 results from TSEN54 gene mutations, it is sometimes categorized more specifically as PCH2A. Mutations in the TSEN54 gene also cause pontocerebellar hypoplasia type 4 (PCH4) and appear to be a rare cause of pontocerebellar hypoplasia type 1 (PCH1).The most common mutation in the TSEN54 gene replaces the amino acid alanine with the amino acid serine at position 307 in the TSEN54 protein (written as Ala307Ser or A307S). About 90 percent of all people with PCH2 have this mutation in both copies of the TSEN54 gene in each cell. At least one person diagnosed with PCH1 also had the mutation in both copies of the gene. Most individuals with PCH4 have the common Ala307Ser mutation in one copy of the TSEN54 gene in each cell and a different mutation in the other copy of the gene.The TSEN54 gene mutations that cause pontocerebellar hypoplasia impair the function of the tRNA splicing endonuclease complex, which likely disrupts the processing of RNA molecules and affects the production of many types of proteins. Before birth, these changes appear to have the most severe impact on fast-growing tissues, such as those in the brain. However, it is unknown exactly how reduced function of the tRNA splicing endonuclease complex leads to abnormal brain development in people with pontocerebellar hypoplasia. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several mutations in the TSEN54 gene have been identified in people with a disorder of brain development called pontocerebellar hypoplasia. The major features of this condition include delayed development, problems with movement, and intellectual disability. TSEN54 gene mutations are the most frequent cause of a form of the disorder designated pontocerebellar hypoplasia type 2 (PCH2). When PCH2 results from TSEN54 gene mutations, it is sometimes categorized more specifically as PCH2A. Mutations in the TSEN54 gene also cause pontocerebellar hypoplasia type 4 (PCH4) and appear to be a rare cause of pontocerebellar hypoplasia type 1 (PCH1).The most common mutation in the TSEN54 gene replaces the amino acid alanine with the amino acid serine at position 307 in the TSEN54 protein (written as Ala307Ser or A307S). About 90 percent of all people with PCH2 have this mutation in both copies of the TSEN54 gene in each cell. At least one person diagnosed with PCH1 also had the mutation in both copies of the gene. Most individuals with PCH4 have the common Ala307Ser mutation in one copy of the TSEN54 gene in each cell and a different mutation in the other copy of the gene.The TSEN54 gene mutations that cause pontocerebellar hypoplasia impair the function of the tRNA splicing endonuclease complex, which likely disrupts the processing of RNA molecules and affects the production of many types of proteins. Before birth, these changes appear to have the most severe impact on fast-growing tissues, such as those in the brain. However, it is unknown exactly how reduced function of the tRNA splicing endonuclease complex leads to abnormal brain development in people with pontocerebellar hypoplasia. More About This Health Condition    Cytogenetic Location: 17q25.1, which is the long (q) arm of chromosome 17 at position 25.1Molecular Location: base pairs 75,516,508 to 75,524,739 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FLJ37147SEN54SEN54_HUMANSEN54LtRNA-intron endonuclease Sen54tRNA splicing endonuclease 54 homologtRNA splicing endonuclease 54 homolog (S. cerevisiae)TSEN54 tRNA splicing endonuclease subunit     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Biology of the Cell (fourth edition, 2002): From RNA to ProteinMolecular Biology of the Cell (fourth edition, 2002): Structure of a tRNA-splicing endonuclease docked to a precursor tRNA (figure)The Cell: A Molecular Approach (second edition, 2000): Processing of mRNA in Eukaryotes      Clinical Information from GeneReviews (1 link)     TSEN54-Related Pontocerebellar Hypoplasia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TSEN54      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     tRNA SPLICING ENDONUCLEASE 54, S. CEREVISIAE, HOMOLOG OF      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): From RNA to ProteinMolecular Biology of the Cell (fourth edition, 2002): Structure of a tRNA-splicing endonuclease docked to a precursor tRNA (figure)The Cell: A Molecular Approach (second edition, 2000): Processing of mRNA in Eukaryotes  TSEN54-Related Pontocerebellar Hypoplasia  Tests of TSEN54  PubMed  tRNA SPLICING ENDONUCLEASE 54, S. CEREVISIAE, HOMOLOG OF  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bailey KA, Aldinger KA. Mutations in the tRNA splicing endonuclease complex cause pontocerebellar hypoplasia. Clin Genet. 2009 May;75(5):427-8. doi: 10.1111/j.1399-0004.2009.01186_3.x. Citation on PubMedBattini R, D'Arrigo S, Cassandrini D, Guzzetta A, Fiorillo C, Pantaleoni C, Romano A, Alfei E, Cioni G, Santorelli FM. Novel mutations in TSEN54 in pontocerebellar hypoplasia type 2. J Child Neurol. 2014 Apr;29(4):520-5. doi: 10.1177/0883073812470002. Epub 2013 Jan 9. Citation on PubMedBudde BS, Namavar Y, Barth PG, Poll-The BT, Nrnberg G, Becker C, van Ruissen F, Weterman MA, Fluiter K, te Beek ET, Aronica E, van der Knaap MS, Hhne W, Toliat MR, Crow YJ, Steinling M, Voit T, Roelenso F, Brussel W, Brockmann K, Kyllerman M, Boltshauser E, Hammersen G, Willemsen M, Basel-Vanagaite L, Krgeloh-Mann I, de Vries LS, Sztriha L, Muntoni F, Ferrie CD, Battini R, Hennekam RC, Grillo E, Beemer FA, Stoets LM, Wollnik B, Nrnberg P, Baas F. tRNA splicing endonuclease mutations cause pontocerebellar hypoplasia. Nat Genet. 2008 Sep;40(9):1113-8. doi: 10.1038/ng.204. Citation on PubMedNamavar Y, Barth PG, Kasher PR, van Ruissen F, Brockmann K, Bernert G, Writzl K, Ventura K, Cheng EY, Ferriero DM, Basel-Vanagaite L, Eggens VR, Krgeloh-Mann I, De Meirleir L, King M, Graham JM Jr, von Moers A, Knoers N, Sztriha L, Korinthenberg R; PCH Consortium, Dobyns WB, Baas F, Poll-The BT. Clinical, neuroradiological and genetic findings in pontocerebellar hypoplasia. Brain. 2011 Jan;134(Pt 1):143-56. doi: 10.1093/brain/awq287. Epub 2010 Oct 15. Citation on PubMedNamavar Y, Eggens VRC, Barth PG, Baas F. TSEN54-Related Pontocerebellar Hypoplasia. 2009 Sep 8 [updated 2016 Jul 14]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK9673/ Citation on PubMedRudaks LI, Moore L, Shand KL, Wilkinson C, Barnett CP. Novel TSEN54 mutation causing pontocerebellar hypoplasia type 4. Pediatr Neurol. 2011 Sep;45(3):185-8. doi: 10.1016/j.pediatrneurol.2011.05.009. Citation on PubMedSimonati A, Cassandrini D, Bazan D, Santorelli FM. TSEN54 mutation in a child with pontocerebellar hypoplasia type 1. Acta Neuropathol. 2011 May;121(5):671-3. doi: 10.1007/s00401-011-0823-1. Epub 2011 Apr 6. Citation on PubMed \n",
            "RECQL5\t\n",
            "ITGB4\t   The ITGB4 gene provides instructions for making one part (the 4 subunit) of a protein known as an integrin. Integrins are a group of proteins that regulate the attachment of cells to one another (cell-cell adhesion) and to the surrounding network of proteins and other molecules (cell-matrix adhesion). Integrins also transmit chemical signals that regulate cell growth and the activity of certain genes.The integrin protein made with the 4 subunit is known as 64 integrin. This protein is found primarily in epithelial cells, which are cells that line the surfaces and cavities of the body. The 64 integrin protein plays a particularly important role in strengthening and stabilizing the skin. It is a component of hemidesmosomes, which are microscopic structures that anchor the outer layer of the skin (the epidermis) to underlying layers. As part of a complex network of proteins in hemidesmosomes, 64 integrin helps to hold the layers of skin together.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Epidermolysis bullosa with pyloric atresia     At least 60 mutations in the ITGB4 gene have been found to cause epidermolysis bullosa with pyloric atresia (EB-PA). In addition to skin blistering, people with EB-PA are born with a life-threatening obstruction of the digestive tract called pyloric atresia. Mutations in the ITGB4 gene account for about 80 percent of all cases of EB-PA.ITGB4 gene mutations alter the normal structure and function of the 4 integrin subunit or prevent cells from producing enough of this subunit. As a result, 64 integrin is defective or missing. Mutations that lead to a complete or near-complete loss of 64 integrin tend to cause more severe signs and symptoms of EB-PA, while mutations that alter the structure or stability of this protein usually cause milder signs and symptoms. A shortage of functional 64 integrin causes cells in the epidermis to be fragile and easily damaged. Friction or other minor trauma can cause the skin layers to separate, leading to the widespread formation of blisters. It is less clear how mutations in the ITGB4 gene are related to pyloric atresia. More About This Health Condition      Cancers     Researchers believe that 64 integrin may play a critical role in the progression of cancerous tumors called carcinomas. These cancers arise in epithelial cells and can affect many tissues and organs, including the breast, lung, liver, colon, and skin.Changes in the location and activity of 64 integrin within cancer cells are associated with the progression of carcinomas. The integrin protein activates key signaling molecules, which trigger cancer cells to migrate through the body and invade other tissues. These signals also make cancer cells more resistant to self-destruction (apoptosis).Recent studies suggest that, in addition to its role in the progression of existing carcinomas, 64 integrin may be involved in the initial formation of these tumors.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 60 mutations in the ITGB4 gene have been found to cause epidermolysis bullosa with pyloric atresia (EB-PA). In addition to skin blistering, people with EB-PA are born with a life-threatening obstruction of the digestive tract called pyloric atresia. Mutations in the ITGB4 gene account for about 80 percent of all cases of EB-PA.ITGB4 gene mutations alter the normal structure and function of the 4 integrin subunit or prevent cells from producing enough of this subunit. As a result, 64 integrin is defective or missing. Mutations that lead to a complete or near-complete loss of 64 integrin tend to cause more severe signs and symptoms of EB-PA, while mutations that alter the structure or stability of this protein usually cause milder signs and symptoms. A shortage of functional 64 integrin causes cells in the epidermis to be fragile and easily damaged. Friction or other minor trauma can cause the skin layers to separate, leading to the widespread formation of blisters. It is less clear how mutations in the ITGB4 gene are related to pyloric atresia. More About This Health Condition  Researchers believe that 64 integrin may play a critical role in the progression of cancerous tumors called carcinomas. These cancers arise in epithelial cells and can affect many tissues and organs, including the breast, lung, liver, colon, and skin.Changes in the location and activity of 64 integrin within cancer cells are associated with the progression of carcinomas. The integrin protein activates key signaling molecules, which trigger cancer cells to migrate through the body and invade other tissues. These signals also make cancer cells more resistant to self-destruction (apoptosis).Recent studies suggest that, in addition to its role in the progression of existing carcinomas, 64 integrin may be involved in the initial formation of these tumors.    Cytogenetic Location: 17q25.1, which is the long (q) arm of chromosome 17 at position 25.1Molecular Location: base pairs 75,721,328 to 75,757,819 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD104CD104 antigenGP150integrin beta 4integrin beta-4 subunitIntegrin beta(4)Integrin beta4integrin, beta-4integrin, beta 4ITB4_HUMANLymphocyte antigen CD104     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Madame Curie Bioscience Database: Integrins in Cancer Cell InvasionMadame Curie Bioscience Database: Integrins: An Overview of Structural and Functional AspectsMolecular Biology of the Cell (fourth edition, 2002): Integrins      Clinical Information from GeneReviews (1 link)     Epidermolysis Bullosa with Pyloric Atresia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ITGB4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     INTEGRIN, BETA-4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Integrins in Cancer Cell InvasionMadame Curie Bioscience Database: Integrins: An Overview of Structural and Functional AspectsMolecular Biology of the Cell (fourth edition, 2002): Integrins  Epidermolysis Bullosa with Pyloric Atresia  Tests of ITGB4  PubMed  INTEGRIN, BETA-4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Ashton GH, Sorelli P, Mellerio JE, Keane FM, Eady RA, McGrath JA. Alpha 6 beta 4 integrin abnormalities in junctional epidermolysis bullosa with pyloric atresia. Br J Dermatol. 2001 Feb;144(2):408-14. Citation on PubMedDang N, Klingberg S, Rubin AI, Edwards M, Borelli S, Relic J, Marr P, Tran K, Turner A, Smith N, Murrell DF. Differential expression of pyloric atresia in junctional epidermolysis bullosa with ITGB4 mutations suggests that pyloric atresia is due to factors other than the mutations and not predictive of a poor outcome: three novel mutations and a review of the literature. Acta Derm Venereol. 2008;88(5):438-48. doi: 10.2340/00015555-0484. Review. Citation on PubMedGuo W, Pylayeva Y, Pepe A, Yoshioka T, Muller WJ, Inghirami G, Giancotti FG. Beta 4 integrin amplifies ErbB2 signaling to promote mammary tumorigenesis. Cell. 2006 Aug 11;126(3):489-502. Citation on PubMedLipscomb EA, Mercurio AM. Mobilization and activation of a signaling competent alpha6beta4integrin underlies its contribution to carcinoma progression. Cancer Metastasis Rev. 2005 Sep;24(3):413-23. Review. Citation on PubMedNakano A, Pulkkinen L, Murrell D, Rico J, Lucky AW, Garzon M, Stevens CA, Robertson S, Pfendner E, Uitto J. Epidermolysis bullosa with congenital pyloric atresia: novel mutations in the beta 4 integrin gene (ITGB4) and genotype/phenotype correlations. Pediatr Res. 2001 May;49(5):618-26. Citation on PubMedPulkkinen L, Kim DU, Uitto J. Epidermolysis bullosa with pyloric atresia: novel mutations in the beta4 integrin gene (ITGB4). Am J Pathol. 1998 Jan;152(1):157-66. Citation on PubMed or Free article on PubMed CentralPulkkinen L, Rouan F, Bruckner-Tuderman L, Wallerstein R, Garzon M, Brown T, Smith L, Carter W, Uitto J. Novel ITGB4 mutations in lethal and nonlethal variants of epidermolysis bullosa with pyloric atresia: missense versus nonsense. Am J Hum Genet. 1998 Nov;63(5):1376-87. Citation on PubMed or Free article on PubMed CentralYoon SO, Shin S, Lipscomb EA. A novel mechanism for integrin-mediated ras activation in breast carcinoma cells: the alpha6beta4 integrin regulates ErbB2 translation and transactivates epidermal growth factor receptor/ErbB2 signaling. Cancer Res. 2006 Mar 1;66(5):2732-9. Citation on PubMed \n",
            "TRIM65\t\n",
            "SENP3\t\n",
            "TNRC6C\t\n",
            "KDM6B\t\n",
            "CCDC40\t   From NCBI Gene:This gene encodes a protein that is necessary for motile cilia function. It functions in correct left-right axis formation by regulating the assembly of the inner dynein arm and the dynein regulatory complexes, which control ciliary beat. Mutations in this gene cause ciliary dyskinesia type 15, a disorder due to defects in cilia motility. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2011]From UniProt:Required for assembly of dynein regulatory complex (DRC) and inner dynein arm (IDA) complexes, which are responsible for ciliary beat regulation, thereby playing a central role in motility in cilia and flagella (PubMed:21131974). Probably acts together with CCDC39 to form a molecular ruler that determines the 96 nanometer (nm) repeat length and arrangements of components in cilia and flagella (By similarity). Not required for outer dynein arm complexes assembly. Required for axonemal recruitment of CCDC39 (PubMed:21131974).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary ciliary dyskinesiaFrom NCBI Gene:Ciliary dyskinesia, primary, 15From UniProt:Ciliary dyskinesia, primary, 15 (CILD15): A disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia; reduced fertility is often observed in male patients due to abnormalities of sperm tails. Half of the patients exhibit randomization of left-right body asymmetry and situs inversus, due to dysfunction of monocilia at the embryonic node. Primary ciliary dyskinesia associated with situs inversus is referred to as Kartagener syndrome. [MIM:613808]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q25.3, which is the long (q) arm of chromosome 17 at position 25.3Molecular Location: base pairs 80,036,632 to 80,100,613 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CILD15FAP172     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CCDC40      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 15OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 40      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CCDC40  OMIM: CILIARY DYSKINESIA, PRIMARY, 15OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 40  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RNF213\t   The RNF213 gene provides instructions for making a protein whose role is unknown. The RNF213 protein, which is found in tissues throughout the body, contains specific regions (domains) that hint at possible functions. One domain, known as a RING finger, is found in proteins that have an enzyme activity known as E3 ubiquitin-protein ligase. Proteins with this activity target other proteins to be broken down (degraded) within cells. Protein degradation is a normal process that removes damaged or unnecessary proteins and helps maintain the normal functions of cells. Proteins with a RING finger domain are involved in many different cellular functions, including cell growth and division, the transmission of chemical signals (signal transduction), and the self-destruction of cells (apoptosis).The RNF213 protein also contains two regions called AAA+ ATPase domains. Proteins with these domains typically regulate mechanical processes in the cell, such as protein unfolding, DNA unwinding, or transporting molecules.Although the function of the RNF213 protein is unknown, studies suggest that it plays a role in the proper development of blood vessels.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Migraine     Genetics Home Reference provides information about migraine. More About This Health Condition      Moyamoya disease     At least 24 genetic changes in the RNF213 gene have been associated with moyamoya disease. This condition is characterized by narrowing of blood vessels in the brain and subsequent growth of networks of small, fragile blood vessels. The resulting lack of blood flow in the brain leads to strokes and other features of the condition.Changes in the RNF213 gene involved in moyamoya disease replace single protein building blocks (amino acids) in the RNF213 protein. The most common change is a normal variation (polymorphism) of the gene that replaces the amino acid arginine with the amino acid lysine at protein position 4810 (written as Arg4810Lys or R4810K). This genetic change is found in a majority of affected individuals in Asian populations and is relatively common in unaffected individuals in these populations, occurring in about 1 percent. (This phenomenon, in which people with an altered copy of the gene never develop the condition, is known as reduced penetrance.) However, the Arg4810Lys change is not found in affected individuals of European descent. These individuals have rarer RNF213 gene mutations that change single amino acids in the protein.Little is known about the effect of these changes on the function of the RNF213 protein, and researchers are unsure how the changes contribute to the characteristic blood vessel abnormalities of moyamoya disease. It is unclear if RNF213 gene mutations are involved in the narrowing of blood vessels in the brain. For unknown reasons, people with moyamoya disease have elevated levels of proteins involved in cell and tissue growth, including the growth of blood vessels (angiogenesis). An excess of these proteins could account for the growth of new blood vessels characteristic of moyamoya disease. It is not clear if changes in the RNF213 gene are involved in the overproduction of these proteins.Little is known about the cause of moyamoya disease. Studies suggest that other genes could be involved in development of the condition. It has been suggested that other factors, such as infection or inflammation, in combination with genetic changes may be necessary for development of moyamoya disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about migraine. More About This Health Condition  At least 24 genetic changes in the RNF213 gene have been associated with moyamoya disease. This condition is characterized by narrowing of blood vessels in the brain and subsequent growth of networks of small, fragile blood vessels. The resulting lack of blood flow in the brain leads to strokes and other features of the condition.Changes in the RNF213 gene involved in moyamoya disease replace single protein building blocks (amino acids) in the RNF213 protein. The most common change is a normal variation (polymorphism) of the gene that replaces the amino acid arginine with the amino acid lysine at protein position 4810 (written as Arg4810Lys or R4810K). This genetic change is found in a majority of affected individuals in Asian populations and is relatively common in unaffected individuals in these populations, occurring in about 1 percent. (This phenomenon, in which people with an altered copy of the gene never develop the condition, is known as reduced penetrance.) However, the Arg4810Lys change is not found in affected individuals of European descent. These individuals have rarer RNF213 gene mutations that change single amino acids in the protein.Little is known about the effect of these changes on the function of the RNF213 protein, and researchers are unsure how the changes contribute to the characteristic blood vessel abnormalities of moyamoya disease. It is unclear if RNF213 gene mutations are involved in the narrowing of blood vessels in the brain. For unknown reasons, people with moyamoya disease have elevated levels of proteins involved in cell and tissue growth, including the growth of blood vessels (angiogenesis). An excess of these proteins could account for the growth of new blood vessels characteristic of moyamoya disease. It is not clear if changes in the RNF213 gene are involved in the overproduction of these proteins.Little is known about the cause of moyamoya disease. Studies suggest that other genes could be involved in development of the condition. It has been suggested that other factors, such as infection or inflammation, in combination with genetic changes may be necessary for development of moyamoya disease. More About This Health Condition    Cytogenetic Location: 17q25.3, which is the long (q) arm of chromosome 17 at position 25.3Molecular Location: base pairs 80,260,820 to 80,398,786 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ALK lymphoma oligomerization partner on chromosome 17ALO17C17orf27E3 ubiquitin-protein ligase RNF213KIAA1554KIAA1618MYMY2mysterinMYSTRNET57RN213_HUMAN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RNF213      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     RING FINGER PROTEIN 213      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RNF213  PubMed  RING FINGER PROTEIN 213  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cecchi AC, Guo D, Ren Z, Flynn K, Santos-Cortez RL, Leal SM, Wang GT, Regalado ES, Steinberg GK, Shendure J, Bamshad MJ; University of Washington Center for Mendelian Genomics, Grotta JC, Nickerson DA, Pannu H, Milewicz DM. RNF213 rare variants in an ethnically diverse population with Moyamoya disease. Stroke. 2014 Nov;45(11):3200-7. doi: 10.1161/STROKEAHA.114.006244. Epub 2014 Oct 2. Citation on PubMed or Free article on PubMed CentralFujimura M, Sonobe S, Nishijima Y, Niizuma K, Sakata H, Kure S, Tominaga T. Genetics and Biomarkers of Moyamoya Disease: Significance of RNF213 as a Susceptibility Gene. J Stroke. 2014 May;16(2):65-72. doi: 10.5853/jos.2014.16.2.65. Epub 2014 May 30. Review. Citation on PubMed or Free article on PubMed CentralKamada F, Aoki Y, Narisawa A, Abe Y, Komatsuzaki S, Kikuchi A, Kanno J, Niihori T, Ono M, Ishii N, Owada Y, Fujimura M, Mashimo Y, Suzuki Y, Hata A, Tsuchiya S, Tominaga T, Matsubara Y, Kure S. A genome-wide association study identifies RNF213 as the first Moyamoya disease gene. J Hum Genet. 2011 Jan;56(1):34-40. doi: 10.1038/jhg.2010.132. Epub 2010 Nov 4. Citation on PubMedLiu W, Morito D, Takashima S, Mineharu Y, Kobayashi H, Hitomi T, Hashikata H, Matsuura N, Yamazaki S, Toyoda A, Kikuta K, Takagi Y, Harada KH, Fujiyama A, Herzig R, Krischek B, Zou L, Kim JE, Kitakaze M, Miyamoto S, Nagata K, Hashimoto N, Koizumi A. Identification of RNF213 as a susceptibility gene for moyamoya disease and its possible role in vascular development. PLoS One. 2011;6(7):e22542. doi: 10.1371/journal.pone.0022542. Epub 2011 Jul 20. Citation on PubMed or Free article on PubMed CentralMorito D, Nishikawa K, Hoseki J, Kitamura A, Kotani Y, Kiso K, Kinjo M, Fujiyoshi Y, Nagata K. Moyamoya disease-associated protein mysterin/RNF213 is a novel AAA+ ATPase, which dynamically changes its oligomeric state. Sci Rep. 2014 Mar 24;4:4442. doi: 10.1038/srep04442. Citation on PubMed or Free article on PubMed CentralOMIM: RING FINGER PROTEIN 213 \n",
            "CNTROB\t\n",
            "BAIAP2\t\n",
            "TSPAN10\t\n",
            "MYADML2\t\n",
            "ASPSCR1\t   From NCBI Gene:The protein encoded by this gene contains a UBX domain and interacts with glucose transporter type 4 (GLUT4). This protein is a tether, which sequesters the GLUT4 in intracellular vesicles in muscle and fat cells in the absence of insulin, and redistributes the GLUT4 to the plasma membrane within minutes of insulin stimulation. Translocation t(X;17)(p11;q25) of this gene with transcription factor TFE3 gene results in a ASPSCR1-TFE3 fusion protein in alveolar soft part sarcoma and in renal cell carcinomas. Multiple alternatively spliced transcript variants have been found. [provided by RefSeq, Oct 2011]From UniProt:Tethering protein that sequesters GLUT4-containing vesicles in the cytoplasm in the absence of insulin. Modulates the amount of GLUT4 that is available at the cell surface (By similarity). Enhances VCP methylation catalyzed by VCPKMT.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Alveolar soft part sarcomaFrom UniProt:A chromosomal aberration involving ASPSCR1 is found in patients with alveolar soft part sarcoma. Translocation t(X;17)(p11;q25) with TFE3 forms a ASPSCR1-TFE3 fusion protein.A chromosomal aberration involving ASPSCR1 has been found in two patients with of papillary renal cell carcinoma. Translocation t(X;17)(p11.2;q25).     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q25.3, which is the long (q) arm of chromosome 17 at position 25.3Molecular Location: base pairs 81,977,546 to 82,017,669 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ASPCR1ASPLASPSRCC17TUGUBXD9UBXN9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ASPSCR1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ALVEOLAR SOFT PART SARCOMAOMIM: ALVEOLAR SOFT PART SARCOMA CHROMOSOME REGION, CANDIDATE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ASPSCR1  OMIM: ALVEOLAR SOFT PART SARCOMAOMIM: ALVEOLAR SOFT PART SARCOMA CHROMOSOME REGION, CANDIDATE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF750\t   From NCBI Gene:This gene encodes a protein with a nuclear localization site and a C2H2 zinc finger domain. Mutations in this gene have been associated with seborrhea-like dermatitis with psoriasiform elements. [provided by RefSeq, Jul 2008]From UniProt:Transcription factor involved in epidermis differentiation. Required for terminal epidermal differentiation: acts downstream of p63/TP63 and activates expression of late epidermal differentiation genes. Specifically binds to the promoter of KLF4 and promotes its expression.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Seborrhea-like dermatitis with psoriasiform elementsFrom UniProt:Seborrhea-like dermatitis with psoriasiform elements (SLDP): Characterized by a chronic fine diffuse scaly erythematous rash on the face, particularly on the chin, nasolabial folds and eyebrows, around earlobes and over the scalp. The rash exacerbate in the winter, with emotional stress and after strenuous physical activity. Hyperkeratosis of skin over the elbows, knees, palms, soles and metacarpophalangeal joints is evident. There is no arthralgia, arthritis or neurological disorders. [MIM:610227]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 17q25.3, which is the long (q) arm of chromosome 17 at position 25.3Molecular Location: base pairs 82,829,434 to 82,840,055 on chromosome 17 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ZFP750     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ZNF750      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SEBORRHEA-LIKE DERMATITIS WITH PSORIASIFORM ELEMENTSOMIM: ZINC FINGER PROTEIN 750      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ZNF750  OMIM: SEBORRHEA-LIKE DERMATITIS WITH PSORIASIFORM ELEMENTSOMIM: ZINC FINGER PROTEIN 750  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ODF4\t\n",
            "CCDC42\t\n",
            "MFSD6L\t\n",
            "PIK3R6\t\n",
            "ABR\t\n",
            "PIEZO2\t\n",
            "SLC35G4\t\n",
            "GNAL\t   From NCBI Gene:This gene encodes a stimulatory G protein alpha subunit which mediates odorant signaling in the olfactory epithelium. This protein couples dopamine type 1 receptors and adenosine A2A receptors and is widely expressed in the central nervous system. Mutations in this gene have been associated with dystonia 25 and this gene is located in a susceptibility region for bipolar disorder and schizophrenia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2013]From UniProt:Guanine nucleotide-binding proteins (G proteins) are involved as modulators or transducers in various transmembrane signaling systems. G(olf) alpha mediates signal transduction within the olfactory neuroepithelium and the basal ganglia. May be involved in some aspect of visual transduction, and in mediating the effect of one or more hormones/neurotransmitters.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Dystonia 25From UniProt:Dystonia 25 (DYT25): A form of dystonia, a disorder defined by the presence of sustained involuntary muscle contraction, often leading to abnormal postures. DYT25 is an autosomal dominant neurologic disorder characterized by adult onset of focal dystonia, usually involving the neck. The dystonia most often progresses to involve other regions, particularly the face and laryngeal muscles, and less commonly the trunk and limbs. [MIM:615073]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18p11.21, which is the short (p) arm of chromosome 18 at position 11.21Molecular Location: base pairs 11,689,015 to 11,885,685 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DYT25     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GNAL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DYSTONIA 25OMIM: GUANINE NUCLEOTIDE-BINDING PROTEIN, ALPHA-ACTIVATING ACTIVITY POLYPEPTIDE, OLFACTORY TYPE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GNAL  OMIM: DYSTONIA 25OMIM: GUANINE NUCLEOTIDE-BINDING PROTEIN, ALPHA-ACTIVATING ACTIVITY POLYPEPTIDE, OLFACTORY TYPE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CEP192\t\n",
            "GREB1L\t   From UniProt:Plays a major role in early metanephros and genital development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Congenital anomalies of kidney and urinary tractFrom NCBI Gene:RENAL HYPODYSPLASIA/APLASIA 3From UniProt:Renal hypodysplasia/aplasia 3 (RHDA3): A severe, autosomal dominant disease encompassing a spectrum of kidney development defects. Clinical manifestations are highly variable and include bilateral or unilateral renal agenesis, renal aplasia, hypoplasia, (cystic) dysplasia, severe obstructive uropathy, and vesicoureteral reflux. Bilateral renal agenesis is almost invariably fatal in utero or in the perinatal period. Unilateral renal agenesis can lead to future health issues including end-stage renal disease. [MIM:617805]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q11.1-q11.2, which is the long (q) arm of chromosome 18 between positions 11.1 and 11.2Molecular Location: base pairs 21,242,225 to 21,528,503 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C18orf6KIAA1772RHDA3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GREB1L      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GREB1-LIKE PROTEINOMIM: RENAL HYPODYSPLASIA/APLASIA 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GREB1L  OMIM: GREB1-LIKE PROTEINOMIM: RENAL HYPODYSPLASIA/APLASIA 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DSC3\t   From NCBI Gene:The protein encoded by this gene is a calcium-dependent glycoprotein that is a member of the desmocollin subfamily of the cadherin superfamily. These desmosomal family members, along with the desmogleins, are found primarily in epithelial cells where they constitute the adhesive proteins of the desmosome cell-cell junction and are required for cell adhesion and desmosome formation. The desmosomal family members are arranged in two clusters on chromosome 18, occupying less than 650 kb combined. Mutations in this gene are a cause of hypotrichosis and recurrent skin vesicles disorder. The protein can act as an autoantigen in pemphigus diseases, and it is also considered to be a biomarker for some cancers. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Apr 2014]From UniProt:Component of intercellular desmosome junctions. Involved in the interaction of plaque proteins and intermediate filaments mediating cell-cell adhesion. May contribute to epidermal cell positioning (stratification) by mediating differential adhesiveness between cells that express different isoforms.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Hypotrichosis and recurrent skin vesiclesFrom UniProt:Hypotrichosis and recurrent skin vesicles (HRSV): A disorder characterized by hypotrichosis and the appearance of recurrent skin vesicle formation. Affected individuals show sparse and fragile hair on scalp, as well as absent eyebrows and eyelashes. Vesicles filled with thin, watery fluid are observed on the scalp and skin of most of the body. Mucosal vesicles are absent. [MIM:613102]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q12.1, which is the long (q) arm of chromosome 18 at position 12.1Molecular Location: base pairs 30,989,365 to 31,042,815 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDHF3DSCDSC1DSC2DSC4HT-CP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DSC3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DESMOCOLLIN 3OMIM: HYPOTRICHOSIS AND RECURRENT SKIN VESICLES      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DSC3  OMIM: DESMOCOLLIN 3OMIM: HYPOTRICHOSIS AND RECURRENT SKIN VESICLES  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DSG2\t   From NCBI Gene:This gene encodes a member of the desmoglein family and cadherin cell adhesion molecule superfamily of proteins. Desmogleins are calcium-binding transmembrane glycoprotein components of desmosomes, cell-cell junctions between epithelial, myocardial, and other cell types. The encoded preproprotein is proteolytically processed to generate the mature glycoprotein. This gene is present in a gene cluster with other desmoglein gene family members on chromosome 18. Mutations in this gene have been associated with arrhythmogenic right ventricular dysplasia, familial, 10. [provided by RefSeq, Jan 2016]From UniProt:Component of intercellular desmosome junctions. Involved in the interaction of plaque proteins and intermediate filaments mediating cell-cell adhesion.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Arrhythmogenic right ventricular cardiomyopathyFamilial dilated cardiomyopathyFrom NCBI Gene:Arrhythmogenic right ventricular cardiomyopathy, type 10Dilated cardiomyopathy 1BBFrom UniProt:Cardiomyopathy, dilated 1BB (CMD1BB): A disorder characterized by ventricular dilation and impaired systolic function, resulting in congestive heart failure and arrhythmia. Patients are at risk of premature death. [MIM:612877]Arrhythmogenic right ventricular dysplasia, familial, 10 (ARVD10): A congenital heart disease characterized by infiltration of adipose and fibrous tissue into the right ventricle and loss of myocardial cells, resulting in ventricular and supraventricular arrhythmias. [MIM:610193]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q12.1, which is the long (q) arm of chromosome 18 at position 12.1Molecular Location: base pairs 31,498,004 to 31,549,008 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDHF5HDGC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DSG2      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA, FAMILIAL, 10OMIM: CARDIOMYOPATHY, DILATED, 1BBOMIM: DESMOGLEIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DSG2  OMIM: ARRHYTHMOGENIC RIGHT VENTRICULAR DYSPLASIA, FAMILIAL, 10OMIM: CARDIOMYOPATHY, DILATED, 1BBOMIM: DESMOGLEIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYOM1\t\n",
            "DTNA\t   From NCBI Gene:The protein encoded by this gene belongs to the dystrobrevin subfamily of the dystrophin family. This protein is a component of the dystrophin-associated protein complex (DPC), which consists of dystrophin and several integral and peripheral membrane proteins, including dystroglycans, sarcoglycans, syntrophins and alpha- and beta-dystrobrevin. The DPC localizes to the sarcolemma and its disruption is associated with various forms of muscular dystrophy. Mutations in this gene are associated with left ventricular noncompaction with congenital heart defects. Multiple alternatively spliced transcript variants encoding different isoforms have been identified for this gene. [provided by RefSeq, Jul 2008]From UniProt:May be involved in the formation and stability of synapses as well as being involved in the clustering of nicotinic acetylcholine receptors.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Left ventricular noncompactionFrom NCBI Gene:Left ventricular noncompaction 1From UniProt:Left ventricular non-compaction 1 (LVNC1): A form of left ventricular non-compaction, a cardiomyopathy due to myocardial morphogenesis arrest and characterized by a hypertrophic left ventricle, a severely thickened 2-layered myocardium, numerous prominent trabeculations, deep intertrabecular recesses, and poor systolic function. Clinical manifestations are variable. Some affected individuals experience no symptoms at all, others develop heart failure. In some cases, left ventricular non-compaction is associated with other congenital heart anomalies. LVNC1 is an autosomal dominant condition. [MIM:604169]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q12.1, which is the long (q) arm of chromosome 18 at position 12.1Molecular Location: base pairs 34,493,290 to 34,891,844 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        D18S892EDRP3DTNDTN-ALVNC1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DTNA      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DYSTROBREVIN, ALPHAOMIM: LEFT VENTRICULAR NONCOMPACTION 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DTNA  OMIM: DYSTROBREVIN, ALPHAOMIM: LEFT VENTRICULAR NONCOMPACTION 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "COLEC12\t\n",
            "MOCOS\t   The MOCOS gene provides instructions for making an enzyme called molybdenum cofactor sulfurase. This enzyme is necessary for the function of two other enzymes, xanthine dehydrogenase and aldehyde oxidase. Xanthine dehydrogenase is involved in the normal breakdown of purines, which are building blocks of DNA and its chemical cousin, RNA. Specifically, it carries out the final two steps in the process: the conversion of a molecule called hypoxanthine to another molecule called xanthine, and the conversion of xanthine to uric acid, a waste product that is normally excreted in urine and feces. Less is known about the function of aldehyde oxidase, although it appears to play a role in the breakdown (metabolism) of many different compounds.Molybdenum cofactor sulfurase carries out a chemical reaction that adds sulfur to a molecule called the molybdenum cofactor. This molecule is required for xanthine dehydrogenase and aldehyde oxidase to be turned on (activated) and carry out their functions.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hereditary xanthinuria     At least four mutations in the MOCOS gene have been found to cause hereditary xanthinuria type II, a condition that most often affects the kidneys. Most of these mutations change a single protein building block (amino acid) in molybdenum cofactor sulfurase. The effects of these mutations are not fully understood, but they likely alter the shape and function of the enzyme. If molybdenum cofactor sulfurase is unable to add sulfur to the molybdenum cofactor, xanthine dehydrogenase and aldehyde oxidase are not activated. The loss of aldehyde oxidase activity does not appear to cause any signs or symptoms. However, the loss of xanthine dehydrogenase activity prevents the conversion of xanthine to uric acid, leading to an accumulation of xanthine in the kidneys and other tissues. The excess xanthine can form tiny crystals that accumulate in the kidneys, occasionally leading to the formation of stones that can impair kidney function and ultimately cause kidney failure. Less commonly, xanthine crystals build up in the muscles, causing pain and cramping. In some people with hereditary xanthinuria type II, the condition does not cause any health problems. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least four mutations in the MOCOS gene have been found to cause hereditary xanthinuria type II, a condition that most often affects the kidneys. Most of these mutations change a single protein building block (amino acid) in molybdenum cofactor sulfurase. The effects of these mutations are not fully understood, but they likely alter the shape and function of the enzyme. If molybdenum cofactor sulfurase is unable to add sulfur to the molybdenum cofactor, xanthine dehydrogenase and aldehyde oxidase are not activated. The loss of aldehyde oxidase activity does not appear to cause any signs or symptoms. However, the loss of xanthine dehydrogenase activity prevents the conversion of xanthine to uric acid, leading to an accumulation of xanthine in the kidneys and other tissues. The excess xanthine can form tiny crystals that accumulate in the kidneys, occasionally leading to the formation of stones that can impair kidney function and ultimately cause kidney failure. Less commonly, xanthine crystals build up in the muscles, causing pain and cramping. In some people with hereditary xanthinuria type II, the condition does not cause any health problems. More About This Health Condition    Cytogenetic Location: 18q12.2, which is the long (q) arm of chromosome 18 at position 12.2Molecular Location: base pairs 36,187,210 to 36,268,722 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FLJ20733HMCSMCSMOS     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Basic Neurochemistry (sixth edition, 1999): Purine Release and Metabolism      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MOCOS      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     MOLYBDENUM COFACTOR SULFURASE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Purine Release and Metabolism  Tests of MOCOS  PubMed  MOLYBDENUM COFACTOR SULFURASE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Ichida K, Amaya Y, Okamoto K, Nishino T. Mutations associated with functional disorder of xanthine oxidoreductase and hereditary xanthinuria in humans. Int J Mol Sci. 2012 Nov 21;13(11):15475-95. doi: 10.3390/ijms131115475. Review. Citation on PubMed or Free article on PubMed CentralIchida K, Matsumura T, Sakuma R, Hosoya T, Nishino T. Mutation of human molybdenum cofactor sulfurase gene is responsible for classical xanthinuria type II. Biochem Biophys Res Commun. 2001 Apr 20;282(5):1194-200. Citation on PubMedOMIM: MOLYBDENUM COFACTOR SULFURASEPeretz H, Naamati MS, Levartovsky D, Lagziel A, Shani E, Horn I, Shalev H, Landau D. Identification and characterization of the first mutation (Arg776Cys) in the C-terminal domain of the Human Molybdenum Cofactor Sulfurase (HMCS) associated with type II classical xanthinuria. Mol Genet Metab. 2007 May;91(1):23-9. Epub 2007 Mar 23. Citation on PubMedYamamoto T, Moriwaki Y, Takahashi S, Tsutsumi Z, Tuneyoshi K, Matsui K, Cheng J, Hada T. Identification of a new point mutation in the human molybdenum cofactor sulferase gene that is responsible for xanthinuria type II. Metabolism. 2003 Nov;52(11):1501-4. Citation on PubMed \n",
            "FHOD3\t\n",
            "TGIF1\t   The TGIF1 gene provides instructions for making a protein called TG-interacting factor. This protein is important for normal development of the front part of the brain (forebrain). TG-interacting factor is a transcription factor, which means that it regulates the activity of certain genes. This protein turns off genes by attaching (binding) to specific regions of DNA or by interacting with other DNA-binding proteins.TG-interacting factor regulates signaling pathways that are important for embryonic development. This protein blocks the signals of the transforming growth factor beta (TGF-) pathway. This signaling pathway transmits chemical signals from the cell surface to the nucleus, which allows the environment outside the cell to affect how the cell produces other proteins. TG-interacting factor also blocks a molecule called retinoic acid from regulating gene activity. Retinoic acid, a form of vitamin A, binds to a group of transcription factors that regulate a number of genes important for early development. By blocking these signaling pathways, TG-interacting factor ensures that certain genes are turned off at the proper time.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Nonsyndromic holoprosencephaly     At least 13 mutations in the TGIF1 gene have been found to cause nonsyndromic holoprosencephaly. This condition occurs when the brain fails to divide into two halves (hemispheres) during early development. TGIF1 gene mutations are the fourth most common cause of nonsyndromic holoprosencephaly. These mutations disrupt the protein's ability to bind with DNA or interact with other proteins. As a result, TG-interacting factor cannot block the signals of the TGF- pathway and retinoic acid. If the signals involved in forebrain development are not properly regulated, the brain does not separate into two hemispheres. The signs and symptoms of nonsyndromic holoprosencephaly are caused by abnormal development of the brain and face. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 13 mutations in the TGIF1 gene have been found to cause nonsyndromic holoprosencephaly. This condition occurs when the brain fails to divide into two halves (hemispheres) during early development. TGIF1 gene mutations are the fourth most common cause of nonsyndromic holoprosencephaly. These mutations disrupt the protein's ability to bind with DNA or interact with other proteins. As a result, TG-interacting factor cannot block the signals of the TGF- pathway and retinoic acid. If the signals involved in forebrain development are not properly regulated, the brain does not separate into two hemispheres. The signs and symptoms of nonsyndromic holoprosencephaly are caused by abnormal development of the brain and face. More About This Health Condition    Cytogenetic Location: 18p11.31, which is the short (p) arm of chromosome 18 at position 11.31Molecular Location: base pairs 3,411,927 to 3,458,412 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        5'-TG-3' interacting factor5'-TG-3'-interacting factor 1homeobox protein TGIFhomeobox protein TGIF1HPE4MGC5066MGC39747TALE homeobox TG-interacting factorTGFB-induced factor homeobox 1TGIFTGIF1_HUMANtransforming growth factor-beta-induced factor     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Neuroscience (second edition, 2001): The Molecular Basis of Neural Induction      Clinical Information from GeneReviews (1 link)     Holoprosencephaly Overview      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TGIF1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSFORMING GROWTH FACTOR-BETA-INDUCED FACTOR      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Neuroscience (second edition, 2001): The Molecular Basis of Neural Induction  Holoprosencephaly Overview  Tests of TGIF1  PubMed  TRANSFORMING GROWTH FACTOR-BETA-INDUCED FACTOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Aguilella C, Dubourg C, Attia-Sobol J, Vigneron J, Blayau M, Pasquier L, Lazaro L, Odent S, David V. Molecular screening of the TGIF gene in holoprosencephaly: identification of two novel mutations. Hum Genet. 2003 Feb;112(2):131-4. Epub 2002 Nov 21. Citation on PubMedDubourg C, Bendavid C, Pasquier L, Henry C, Odent S, David V. Holoprosencephaly. Orphanet J Rare Dis. 2007 Feb 2;2:8. Review. Citation on PubMed or Free article on PubMed CentralEl-Jaick KB, Powers SE, Bartholin L, Myers KR, Hahn J, Orioli IM, Ouspenskaia M, Lacbawan F, Roessler E, Wotton D, Muenke M. Functional analysis of mutations in TGIF associated with holoprosencephaly. Mol Genet Metab. 2007 Jan;90(1):97-111. Epub 2006 Sep 7. Review. Citation on PubMed or Free article on PubMed CentralGripp KW, Wotton D, Edwards MC, Roessler E, Ades L, Meinecke P, Richieri-Costa A, Zackai EH, Massagu J, Muenke M, Elledge SJ. Mutations in TGIF cause holoprosencephaly and link NODAL signalling to human neural axis determination. Nat Genet. 2000 Jun;25(2):205-8. Citation on PubMedRoessler E, Muenke M. The molecular genetics of holoprosencephaly. Am J Med Genet C Semin Med Genet. 2010 Feb 15;154C(1):52-61. doi: 10.1002/ajmg.c.30236. Review. Citation on PubMed or Free article on PubMed CentralSolomon BD, Gropman A, Muenke M. Holoprosencephaly Overview. 2000 Dec 27 [updated 2013 Aug 29]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1530/ Citation on PubMedSolomon BD, Mercier S, Vlez JI, Pineda-Alvarez DE, Wyllie A, Zhou N, Dubourg C, David V, Odent S, Roessler E, Muenke M. Analysis of genotype-phenotype correlations in human holoprosencephaly. Am J Med Genet C Semin Med Genet. 2010 Feb 15;154C(1):133-41. doi: 10.1002/ajmg.c.30240. Review. Citation on PubMed or Free article on PubMed CentralOMIM: TRANSFORMING GROWTH FACTOR-BETA-INDUCED FACTOR \n",
            "SETBP1\t   The SETBP1 gene provides instructions for making a protein called SET binding protein 1 (SETBP1), which is found in cells throughout the body. The SETBP1 protein is known to attach (bind) to another protein called SET. However, the function of the SETBP1 protein and the effect of its binding to the SET protein are unknown.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Schinzel-Giedion syndrome     At least five mutations in the SETBP1 gene have been identified in children with Schinzel-Giedion syndrome, a severe condition apparent at birth that is characterized by distinctive facial features and abnormalities in many body systems. The gene mutations in affected individuals cluster in one region of the gene known as exon 4. However, the effects of the mutations on the function of the gene or the protein are unknown. Researchers are working to understand how mutations in the SETBP1 gene cause the signs and symptoms of Schinzel-Giedion syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least five mutations in the SETBP1 gene have been identified in children with Schinzel-Giedion syndrome, a severe condition apparent at birth that is characterized by distinctive facial features and abnormalities in many body systems. The gene mutations in affected individuals cluster in one region of the gene known as exon 4. However, the effects of the mutations on the function of the gene or the protein are unknown. Researchers are working to understand how mutations in the SETBP1 gene cause the signs and symptoms of Schinzel-Giedion syndrome. More About This Health Condition    Cytogenetic Location: 18q12.3, which is the long (q) arm of chromosome 18 at position 12.3Molecular Location: base pairs 44,680,072 to 45,068,510 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA0437SEBSET-binding proteinSET-binding protein isoform aSET-binding protein isoform bSETBP_HUMAN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SETBP1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SET-BINDING PROTEIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SETBP1  PubMed  SET-BINDING PROTEIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Hoischen A, van Bon BW, Gilissen C, Arts P, van Lier B, Steehouwer M, de Vries P, de Reuver R, Wieskamp N, Mortier G, Devriendt K, Amorim MZ, Revencu N, Kidd A, Barbosa M, Turner A, Smith J, Oley C, Henderson A, Hayes IM, Thompson EM, Brunner HG, de Vries BB, Veltman JA. De novo mutations of SETBP1 cause Schinzel-Giedion syndrome. Nat Genet. 2010 Jun;42(6):483-5. doi: 10.1038/ng.581. Epub 2010 May 2. Citation on PubMedMinakuchi M, Kakazu N, Gorrin-Rivas MJ, Abe T, Copeland TD, Ueda K, Adachi Y. Identification and characterization of SEB, a novel protein that binds to the acute undifferentiated leukemia-associated protein SET. Eur J Biochem. 2001 Mar;268(5):1340-51. Citation on PubMedOMIM: SET-BINDING PROTEIN 1Suphapeetiporn K, Srichomthong C, Shotelersuk V. SETBP1 mutations in two Thai patients with Schinzel-Giedion syndrome. Clin Genet. 2011 Apr;79(4):391-3. doi: 10.1111/j.1399-0004.2010.01552.x. Citation on PubMed \n",
            "SLC14A1\t   From NCBI Gene:The protein encoded by this gene is a membrane transporter that mediates urea transport in erythrocytes. This gene forms the basis for the Kidd blood group system. [provided by RefSeq, Mar 2009]From UniProt:Urea channel that facilitates transmembrane urea transport down a concentration gradient. A constriction of the transmembrane channel functions as selectivity filter through which urea is expected to pass in dehydrated form. The rate of urea conduction is increased by hypotonic stress. Plays an important role in the kidney medulla collecting ducts, where it allows rapid equilibration between the lumen of the collecting ducts and the interstitium, and thereby prevents water loss driven by the high concentration of urea in the urine. Facilitates urea transport across erythrocyte membranes. May also play a role in transmembrane water transport, possibly by indirect means.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:BLOOD GROUP--KIDD SYSTEM     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q12.3, which is the long (q) arm of chromosome 18 at position 12.3Molecular Location: base pairs 45,724,123 to 45,752,520 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HsT1341HUT11JKRACH1RACH2UT-B1UT1UTE     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC14A1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: BLOOD GROUP, KIDD SYSTEMOMIM: SOLUTE CARRIER FAMILY 14 (UREA TRANSPORTER), MEMBER 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SLC14A1  OMIM: BLOOD GROUP, KIDD SYSTEMOMIM: SOLUTE CARRIER FAMILY 14 (UREA TRANSPORTER), MEMBER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "EPG5\t   The EPG5 gene provides instructions for making a protein that is involved in a cellular process called autophagy. This process recycles worn-out or unnecessary cell parts and breaks down certain proteins when they are no longer needed. Autophagy also helps cells use materials most efficiently when energy demands are high. During autophagy, materials to be recycled or removed are isolated in compartments called autophagosomes. The autophagosomes are then transported to cell structures called lysosomes that break down the materials. The EPG5 protein is important for the interaction between autophagosomes and lysosomes that allows the transfer of materials.In addition to its role in autophagy, the EPG5 protein aids in the cell's ability to recognize infection from foreign invaders such as bacteria and viruses. The protein transports molecules from these invaders within cells so they can interact with immune system proteins that trigger reactions to fight the infection.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Vici syndrome     At least 60 mutations in the EPG5 gene have been found to cause Vici syndrome, a severe condition that primarily affects the brain, immune system, heart, skin, and eyes. Most of these mutations lead to an abnormally short EPG5 protein that does not function. Without EPG5 protein activity, foreign invaders cannot trigger immune reactions, which leads to recurrent infections. In addition, autophagy is impaired. Researchers speculate that problems with autophagy disrupt the normal development and survival of cells in the brain and other organs and tissues that require large amounts of energy; however, they do not fully understand how the impairment leads to signs and symptoms of Vici syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 60 mutations in the EPG5 gene have been found to cause Vici syndrome, a severe condition that primarily affects the brain, immune system, heart, skin, and eyes. Most of these mutations lead to an abnormally short EPG5 protein that does not function. Without EPG5 protein activity, foreign invaders cannot trigger immune reactions, which leads to recurrent infections. In addition, autophagy is impaired. Researchers speculate that problems with autophagy disrupt the normal development and survival of cells in the brain and other organs and tissues that require large amounts of energy; however, they do not fully understand how the impairment leads to signs and symptoms of Vici syndrome. More About This Health Condition    Cytogenetic Location: 18q12.3-q21.1, which is the long (q) arm of chromosome 18 between positions 12.3 and 21.1Molecular Location: base pairs 45,800,586 to 45,967,339 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ectopic P granules protein 5 homologHEEW1hEPG5KIAA1632VICIS     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database (2000): Macroautophagy in Mammalian Cells      Tests Listed in the Genetic Testing Registry (1 link)     Tests of EPG5      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ECTOPIC P-GRANULES AUTOPHAGY PROTEIN 5, C. ELEGANS, HOMOLOG OF      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database (2000): Macroautophagy in Mammalian Cells  Tests of EPG5  PubMed  ECTOPIC P-GRANULES AUTOPHAGY PROTEIN 5, C. ELEGANS, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Baron O, Boudi A, Dias C, Schilling M, Nlle A, Vizcay-Barrena G, Rattray I, Jungbluth H, Scheper W, Fleck RA, Bates GP, Fanto M. Stall in Canonical Autophagy-Lysosome Pathways Prompts Nucleophagy-Based Nuclear Breakdown in Neurodegeneration. Curr Biol. 2017 Dec 4;27(23):3626-3642.e6. doi: 10.1016/j.cub.2017.10.054. Epub 2017 Nov 22. Citation on PubMed or Free article on PubMed CentralCullup T, Kho AL, Dionisi-Vici C, Brandmeier B, Smith F, Urry Z, Simpson MA, Yau S, Bertini E, McClelland V, Al-Owain M, Koelker S, Koerner C, Hoffmann GF, Wijburg FA, ten Hoedt AE, Rogers RC, Manchester D, Miyata R, Hayashi M, Said E, Soler D, Kroisel PM, Windpassinger C, Filloux FM, Al-Kaabi S, Hertecant J, Del Campo M, Buk S, Bodi I, Goebel HH, Sewry CA, Abbs S, Mohammed S, Josifova D, Gautel M, Jungbluth H. Recessive mutations in EPG5 cause Vici syndrome, a multisystem disorder with defective autophagy. Nat Genet. 2013 Jan;45(1):83-7. doi: 10.1038/ng.2497. Epub 2012 Dec 9. Citation on PubMed or Free article on PubMed CentralOMIM: ECTOPIC P-GRANULES AUTOPHAGY PROTEIN 5, C. ELEGANS, HOMOLOG OFHori I, Otomo T, Nakashima M, Miya F, Negishi Y, Shiraishi H, Nonoda Y, Magara S, Tohyama J, Okamoto N, Kumagai T, Shimoda K, Yukitake Y, Kajikawa D, Morio T, Hattori A, Nakagawa M, Ando N, Nishino I, Kato M, Tsunoda T, Saitsu H, Kanemura Y, Yamasaki M, Kosaki K, Matsumoto N, Yoshimori T, Saitoh S. Defects in autophagosome-lysosome fusion underlie Vici syndrome, a neurodevelopmental disorder with multisystem involvement. Sci Rep. 2017 Jun 14;7(1):3552. doi: 10.1038/s41598-017-02840-8. Citation on PubMed or Free article on PubMed CentralPiano Mortari E, Folgiero V, Marcellini V, Romania P, Bellacchio E, D'Alicandro V, Bocci C, Carrozzo R, Martinelli D, Petrini S, Axiotis E, Farroni C, Locatelli F, Schara U, Pilz DT, Jungbluth H, Dionisi-Vici C, Carsetti R. The Vici syndrome protein EPG5 regulates intracellular nucleic acid trafficking linking autophagy to innate and adaptive immunity. Autophagy. 2018;14(1):22-37. doi: 10.1080/15548627.2017.1389356. Epub 2018 Jan 2. Citation on PubMed or Free article on PubMed CentralWang Z, Miao G, Xue X, Guo X, Yuan C, Wang Z, Zhang G, Chen Y, Feng D, Hu J, Zhang H. The Vici Syndrome Protein EPG5 Is a Rab7 Effector that Determines the Fusion Specificity of Autophagosomes with Late Endosomes/Lysosomes. Mol Cell. 2016 Sep 1;63(5):781-95. doi: 10.1016/j.molcel.2016.08.021. Citation on PubMed \n",
            "C18orf25\t\n",
            "LOXHD1\t   From NCBI Gene:This gene encodes a highly conserved protein consisting entirely of PLAT (polycystin/lipoxygenase/alpha-toxin) domains, thought to be involved in targeting proteins to the plasma membrane. Studies in mice show that this gene is expressed in the mechanosensory hair cells in the inner ear, and mutations in this gene lead to auditory defects, indicating that this gene is essential for normal hair cell function. Screening of human families segregating deafness identified a mutation in this gene which causes DFNB77, a progressive form of autosomal-recessive nonsyndromic hearing loss (ARNSHL). Alternatively spliced transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Mar 2010]From UniProt:Involved in hearing. Required for normal function of hair cells in the inner ear (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 77From UniProt:Deafness, autosomal recessive, 77 (DFNB77): A form of non-syndromic deafness characterized by preserved low-frequency hearing, and a trend toward mild to moderate mid-frequency and high-frequency hearing loss during childhood and adolescence. Hearing loss progresses to become moderate to severe at mid and high frequencies during adulthood. [MIM:613079]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q21.1, which is the long (q) arm of chromosome 18 at position 21.1Molecular Location: base pairs 46,476,972 to 46,657,115 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNB77LH2D1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LOXHD1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 77OMIM: LIPOXYGENASE HOMOLOGY DOMAIN-CONTAINING 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LOXHD1  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 77OMIM: LIPOXYGENASE HOMOLOGY DOMAIN-CONTAINING 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KATNAL2\t   From UniProt:Severs microtubules in vitro in an ATP-dependent manner. This activity may promote rapid reorganization of cellular microtubule arrays.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autism spectrum disorder     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q21.1, which is the long (q) arm of chromosome 18 at position 21.1Molecular Location: base pairs 46,917,550 to 47,104,795 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KATNAL2      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: KATANIN, p60 SUBUNIT, A-LIKE PROTEIN 2      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KATNAL2  OMIM: KATANIN, p60 SUBUNIT, A-LIKE PROTEIN 2  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MBD1\t\n",
            "MAPK4\t\n",
            "MEX3C\t   From NCBI Gene:This gene encodes a member of a family of proteins with two K homology (KH) RNA-binding domains and a C-terminal RING-finger domain. The protein interacts with mRNA via the KH domains, and the protein shuttles between the nucleus and cytoplasm. Polymorphisms in this gene may contribute to hypertension. [provided by RefSeq, Oct 2009]From UniProt:E3 ubiquitin ligase responsible for the post-transcriptional regulation of common HLA-A allotypes. Binds to the 3' UTR of HLA-A2 mRNA, and regulates its levels by promoting mRNA decay. RNA binding is sufficient to prevent translation, but ubiquitin ligase activity is required for mRNA degradation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Genetic variations in MEX3C may be associated with susceptibility to essential hypertension.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q21.2, which is the long (q) arm of chromosome 18 at position 21.2Molecular Location: base pairs 51,174,550 to 51,197,681 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BM-013MEX-3CRKHD2RNF194     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MEX3C      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MEX3, C. ELEGANS, HOMOLOG OF, C      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MEX3C  OMIM: MEX3, C. ELEGANS, HOMOLOG OF, C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "POLI\t\n",
            "ALPK2\t\n",
            "PIGN\t   From NCBI Gene:This gene encodes a protein that is involved in glycosylphosphatidylinositol (GPI)-anchor biosynthesis. The GPI-anchor is a glycolipid found on many blood cells and serves to anchor proteins to the cell surface. This protein is expressed in the endoplasmic reticulum and transfers phosphoethanolamine (EtNP) to the first mannose of the GPI anchor. Two alternatively spliced variants, which encode an identical isoform, have been reported. [provided by RefSeq, Jul 2008]From UniProt:Ethanolamine phosphate transferase involved in glycosylphosphatidylinositol-anchor biosynthesis. Transfers ethanolamine phosphate to the first alpha-1,4-linked mannose of the glycosylphosphatidylinositol precursor of GPI-anchor (By similarity). May act as suppressor of replication stress and chromosome missegregation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Multiple congenital anomalies-hypotonia-seizures syndrome 1From UniProt:Multiple congenital anomalies-hypotonia-seizures syndrome 1 (MCAHS1): An autosomal recessive disorder characterized by neonatal hypotonia, lack of psychomotor development, seizures, dysmorphic features, and variable congenital anomalies involving the cardiac, urinary, and gastrointestinal systems. Most affected individuals die before 3 years of age. [MIM:614080]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q21.33, which is the long (q) arm of chromosome 18 at position 21.33Molecular Location: base pairs 62,044,224 to 62,187,118 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MCAHSMCAHS1MCD4MDC4PIG-N     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PIGN      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MULTIPLE CONGENITAL ANOMALIES-HYPOTONIA-SEIZURES SYNDROME 1OMIM: PHOSPHATIDYLINOSITOL GLYCAN ANCHOR BIOSYNTHESIS CLASS N PROTEIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PIGN  OMIM: MULTIPLE CONGENITAL ANOMALIES-HYPOTONIA-SEIZURES SYNDROME 1OMIM: PHOSPHATIDYLINOSITOL GLYCAN ANCHOR BIOSYNTHESIS CLASS N PROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TNFRSF11A\t   The TNFRSF11A gene provides instructions for making a protein called receptor activator of NF-B (RANK). This protein plays an important role in bone remodeling, a normal process in which old bone is broken down and new bone is created to replace it. During bone remodeling, RANK helps direct the formation and function of specialized cells called osteoclasts, which break down bone tissue.  RANK is located on the surface of immature osteoclasts, where it receives signals that trigger these cells to mature and become fully functional.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Osteopetrosis     Genetics Home Reference provides information about osteopetrosis. More About This Health Condition      Paget disease of bone     At least two very similar mutations in the TNFRSF11A gene have been found to cause the rare, early-onset form of Paget disease of bone. Both mutations are duplications, which means that they abnormally copy a segment of genetic material within the gene. Each of these mutations results in the production of a RANK protein that contains several extra protein building blocks (amino acids).Through a mechanism that is not well understood, duplication mutations in the TNFRSF11A gene appear to overactivate the chemical signaling pathway that promotes osteoclast formation. The increased signaling stimulates the production of too many osteoclasts and triggers these cells to break down bone abnormally. In people with early-onset Paget disease of bone, affected bone is broken down and replaced much faster than usual. When the new bone tissue grows, it is weaker and less organized than normal bone. These problems with bone remodeling cause certain bones to become unusually large, misshapen, and easily broken (fractured). More About This Health Condition      Other disorders     Mutations in the TNFRSF11A gene are responsible for several other rare bone diseases, including two very similar disorders called familial expansile osteolysis (FEO) and expansile skeletal hyperphosphatasia (ESH). These disorders have signs and symptoms that overlap with those of early-onset Paget disease of bone. In fact, some researchers believe that FEO, ESH, and early-onset Paget disease of bone actually may be slightly different forms of a single condition. FEO and ESH both appear early in life and are characterized by skeletal abnormalities, tooth loss, and progressive hearing loss.Like early-onset Paget disease of bone, FEO and ESH result from duplication mutations in the TNFRSF11A gene. Studies suggest that these mutations overactivate RANK, leading it to stimulate the production of too many osteoclasts and trigger these cells to break down bone abnormally. The resulting imbalance in bone remodeling causes the major features of these disorders. It is unclear why duplication mutations in the TNFRSF11A gene can cause several different bone diseases.TNFRSF11A gene mutations also cause a bone disease called autosomal recessive osteopetrosis (ARO). This disorder appears in infancy and is characterized by abnormally dense bones. The increased bone density leads to a variety of complications, including an increased risk of fractures, vision impairment, hearing loss, and problems with the immune system related to defective bone marrow. Mutations in the TNFRSF11A gene appear to be a very rare cause of ARO; fewer than 10 mutations have been found in affected individuals. Most of these mutations change single amino acids in the RANK protein, which prevents it from receiving signals on the surface of immature osteoclasts. As a result, people with this condition have a total absence of mature, functional osteoclasts. Without these specialized cells to break down bone tissue, excess bone is formed throughout the skeleton.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about osteopetrosis. More About This Health Condition  At least two very similar mutations in the TNFRSF11A gene have been found to cause the rare, early-onset form of Paget disease of bone. Both mutations are duplications, which means that they abnormally copy a segment of genetic material within the gene. Each of these mutations results in the production of a RANK protein that contains several extra protein building blocks (amino acids).Through a mechanism that is not well understood, duplication mutations in the TNFRSF11A gene appear to overactivate the chemical signaling pathway that promotes osteoclast formation. The increased signaling stimulates the production of too many osteoclasts and triggers these cells to break down bone abnormally. In people with early-onset Paget disease of bone, affected bone is broken down and replaced much faster than usual. When the new bone tissue grows, it is weaker and less organized than normal bone. These problems with bone remodeling cause certain bones to become unusually large, misshapen, and easily broken (fractured). More About This Health Condition  Mutations in the TNFRSF11A gene are responsible for several other rare bone diseases, including two very similar disorders called familial expansile osteolysis (FEO) and expansile skeletal hyperphosphatasia (ESH). These disorders have signs and symptoms that overlap with those of early-onset Paget disease of bone. In fact, some researchers believe that FEO, ESH, and early-onset Paget disease of bone actually may be slightly different forms of a single condition. FEO and ESH both appear early in life and are characterized by skeletal abnormalities, tooth loss, and progressive hearing loss.Like early-onset Paget disease of bone, FEO and ESH result from duplication mutations in the TNFRSF11A gene. Studies suggest that these mutations overactivate RANK, leading it to stimulate the production of too many osteoclasts and trigger these cells to break down bone abnormally. The resulting imbalance in bone remodeling causes the major features of these disorders. It is unclear why duplication mutations in the TNFRSF11A gene can cause several different bone diseases.TNFRSF11A gene mutations also cause a bone disease called autosomal recessive osteopetrosis (ARO). This disorder appears in infancy and is characterized by abnormally dense bones. The increased bone density leads to a variety of complications, including an increased risk of fractures, vision impairment, hearing loss, and problems with the immune system related to defective bone marrow. Mutations in the TNFRSF11A gene appear to be a very rare cause of ARO; fewer than 10 mutations have been found in affected individuals. Most of these mutations change single amino acids in the RANK protein, which prevents it from receiving signals on the surface of immature osteoclasts. As a result, people with this condition have a total absence of mature, functional osteoclasts. Without these specialized cells to break down bone tissue, excess bone is formed throughout the skeleton.    Cytogenetic Location: 18q21.33, which is the long (q) arm of chromosome 18 at position 21.33Molecular Location: base pairs 62,325,287 to 62,388,096 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD265FEOODFROFEOPTB7osteoclast differentiation factor receptorOSTSPDB2RANKreceptor activator of NF-kappa-Breceptor activator of nuclear factor-kappa BTNR11_HUMANTRANCERtumor necrosis factor receptor superfamily member 11atumor necrosis factor receptor superfamily member 11a, NFKB activatortumor necrosis factor receptor superfamily, member 11atumor necrosis factor receptor superfamily, member 11a, activator of NFKBtumor necrosis factor receptor superfamily, member 11a, NFKB activator     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (fourth edition, 2002): Bone Is Continually Remodeled by the Cells Within It      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TNFRSF11A      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (3 links)     FAMILIAL EXPANSILE OSTEOLYSISOSTEOPETROSIS, AUTOSOMAL RECESSIVE 7TUMOR NECROSIS FACTOR RECEPTOR SUPERFAMILY, MEMBER 11A      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Bone Is Continually Remodeled by the Cells Within It  Tests of TNFRSF11A  PubMed  FAMILIAL EXPANSILE OSTEOLYSISOSTEOPETROSIS, AUTOSOMAL RECESSIVE 7TUMOR NECROSIS FACTOR RECEPTOR SUPERFAMILY, MEMBER 11A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Elahi E, Shafaghati Y, Asadi S, Absalan F, Goodarzi H, Gharaii N, Karimi-Nejad MH, Shahram F, Hughes AE. Intragenic SNP haplotypes associated with 84dup18 mutation in TNFRSF11A in four FEO pedigrees suggest three independent origins for this mutation. J Bone Miner Metab. 2007;25(3):159-64. Epub 2007 Apr 20. Citation on PubMedGuerrini MM, Sobacchi C, Cassani B, Abinun M, Kilic SS, Pangrazio A, Moratto D, Mazzolari E, Clayton-Smith J, Orchard P, Coxon FP, Helfrich MH, Crockett JC, Mellis D, Vellodi A, Tezcan I, Notarangelo LD, Rogers MJ, Vezzoni P, Villa A, Frattini A. Human osteoclast-poor osteopetrosis with hypogammaglobulinemia due to TNFRSF11A (RANK) mutations. Am J Hum Genet. 2008 Jul;83(1):64-76. doi: 10.1016/j.ajhg.2008.06.015. Citation on PubMed or Free article on PubMed CentralHughes AE, Ralston SH, Marken J, Bell C, MacPherson H, Wallace RG, van Hul W, Whyte MP, Nakatsuka K, Hovy L, Anderson DM. Mutations in TNFRSF11A, affecting the signal peptide of RANK, cause familial expansile osteolysis. Nat Genet. 2000 Jan;24(1):45-8. Citation on PubMedJohnson-Pais TL, Singer FR, Bone HG, McMurray CT, Hansen MF, Leach RJ. Identification of a novel tandem duplication in exon 1 of the TNFRSF11A gene in two unrelated patients with familial expansile osteolysis. J Bone Miner Res. 2003 Feb;18(2):376-80. Citation on PubMedKe YH, Yue H, He JW, Liu YJ, Zhang ZL. Early onset Paget's disease of bone caused by a novel mutation (78dup27) of the TNFRSF11A gene in a Chinese family. Acta Pharmacol Sin. 2009 Aug;30(8):1204-10. doi: 10.1038/aps.2009.90. Epub 2009 Jul 6. Citation on PubMed or Free article on PubMed CentralLi J, Sarosi I, Yan XQ, Morony S, Capparelli C, Tan HL, McCabe S, Elliott R, Scully S, Van G, Kaufman S, Juan SC, Sun Y, Tarpley J, Martin L, Christensen K, McCabe J, Kostenuik P, Hsu H, Fletcher F, Dunstan CR, Lacey DL, Boyle WJ. RANK is the intrinsic hematopoietic cell surface receptor that controls osteoclastogenesis and regulation of bone mass and calcium metabolism. Proc Natl Acad Sci U S A. 2000 Feb 15;97(4):1566-71. Citation on PubMed or Free article on PubMed CentralNakatsuka K, Nishizawa Y, Ralston SH. Phenotypic characterization of early onset Paget's disease of bone caused by a 27-bp duplication in the TNFRSF11A gene. J Bone Miner Res. 2003 Aug;18(8):1381-5. Citation on PubMedPalenzuela L, Vives-Bauza C, Fernndez-Cadenas I, Meseguer A, Font N, Sarret E, Schwartz S, Andreu AL. Familial expansile osteolysis in a large Spanish kindred resulting from an insertion mutation in the TNFRSF11A gene. J Med Genet. 2002 Oct;39(10):E67. Citation on PubMed or Free article on PubMed CentralWhyte MP, Hughes AE. Expansile skeletal hyperphosphatasia is caused by a 15-base pair tandem duplication in TNFRSF11A encoding RANK and is allelic to familial expansile osteolysis. J Bone Miner Res. 2002 Jan;17(1):26-9. Citation on PubMed \n",
            "ZCCHC2\t\n",
            "SERPINB3\t\n",
            "SERPINB11\t\n",
            "SERPINB8\t   From NCBI Gene:The protein encoded by this gene is a member of the ov-serpin family of serine protease inhibitors. The encoded protein is produced by platelets and can bind to and inhibit the function of furin, a serine protease involved in platelet functions. In addition, this protein has been found to enhance the mechanical stability of cell-cell adhesion in the skin, and defects in this gene have been associated with an autosomal-recessive form of exfoliative ichthyosis. [provided by RefSeq, Jan 2017]From UniProt:Has an important role in epithelial desmosome-mediated cell-cell adhesion.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Peeling skin syndrome 5From UniProt:Peeling skin syndrome 5 (PSS5): A form of peeling skin syndrome, a genodermatosis characterized by generalized, continuous shedding of the outer layers of the epidermis. Two main PSS subtypes have been suggested. Patients with non-inflammatory PSS (type A) manifest white scaling, with painless and easy removal of the skin, irritation when in contact with water, dust and sand, and no history of erythema, pruritis or atopy. Inflammatory PSS (type B) is associated with generalized erythema, pruritus and atopy. It is an ichthyosiform erythroderma characterized by lifelong patchy peeling of the entire skin with onset at birth or shortly after. Several patients have been reported with high IgE levels. PSS5 patients manifest hyperkeratosis and superficial peeling of areas of the palmar and dorsal faces of hands and feet. Additional variable features include erythema, superficial scaling of forearms and legs and diffuse yellowish hyperkeratotic palmoplantar plaques. PSS5 inheritance is autosomal recessive. [MIM:617115]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q22.1, which is the long (q) arm of chromosome 18 at position 22.1Molecular Location: base pairs 63,970,028 to 64,019,779 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C18orf53CAP2PI-8PI8PSS5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SERPINB8      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PEELING SKIN SYNDROME 5OMIM: SERPIN FAMILY B, MEMBER 8      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SERPINB8  OMIM: PEELING SKIN SYNDROME 5OMIM: SERPIN FAMILY B, MEMBER 8  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RTTN\t   From NCBI Gene:This gene encodes a large protein whose specific function is unknown. Absence of the orthologous protein in mouse results in embryonic lethality with deficient axial rotation, abnormal differentiation of the neural tube, and randomized looping of the heart tube during development. In human, mutations in this gene are associated with polymicrogyria with seizures. In human fibroblasts this protein localizes at the ciliary basal bodies. Given the intracellular localization of this protein and the phenotypic effects of mutations, this gene is suspected of playing a role in the maintenance of normal ciliary structure which in turn effects the developmental process of left-right organ specification, axial rotation, and perhaps notochord development. [provided by RefSeq, Jan 2013]From UniProt:Involved in the genetic cascade that governs left-right specification. Plays a role in the maintenance of a normal ciliary structure. Required for correct asymmetric expression of NODAL, LEFTY and PITX2.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Microcephaly, short stature, and polymicrogyria with or without seizuresFrom UniProt:Microcephaly, short stature, and polymicrogyria with or without seizures (MSSP): A disease characterized by many irregular small gyri in the brain surface and fusion of the molecular layer over multiple small gyri, which gives a festooned appearance to the cortical surface, without abnormal neuronal migration. Polymicrogyria is a heterogeneous disorder, considered to be the result of postmigratory abnormal cortical organization. MSSP patients have moderate to severe mental retardation, poor speech, dysarthria and seizures. [MIM:614833]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18q22.2, which is the long (q) arm of chromosome 18 at position 22.2Molecular Location: base pairs 70,003,804 to 70,205,726 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MSSP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RTTN      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MICROCEPHALY, SHORT STATURE, AND POLYMICROGYRIA WITH OR WITHOUT SEIZURESOMIM: ROTATIN      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RTTN  OMIM: MICROCEPHALY, SHORT STATURE, AND POLYMICROGYRIA WITH OR WITHOUT SEIZURESOMIM: ROTATIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ENOSF1\t\n",
            "LAMA1\t   From NCBI Gene:This gene encodes one of the alpha 1 subunits of laminin. The laminins are a family of extracellular matrix glycoproteins that have a heterotrimeric structure consisting of an alpha, beta and gamma chain. These proteins make up a major component of the basement membrane and have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Mutations in this gene may be associated with Poretti-Boltshauser syndrome. [provided by RefSeq, Sep 2014]From UniProt:Binding to cells via a high affinity receptor, laminin is thought to mediate the attachment, migration and organization of cells into tissues during embryonic development by interacting with other extracellular matrix components.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Poretti-Boltshauser syndromeFrom UniProt:Poretti-Boltshauser syndrome (PTBHS): An autosomal recessive disorder characterized by cerebellar dysplasia, cerebellar vermis atrophy, cerebellar cysts in most patients, high myopia, variable retinal dystrophy, and eye movement abnormalities including strabismus, ocular apraxia, nystagmus. Affected individuals have ataxia, delayed motor development, language impairment, and intellectual disability with variable severity. [MIM:615960]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 18p11.31, which is the short (p) arm of chromosome 18 at position 11.31Molecular Location: base pairs 6,941,744 to 7,117,814 on chromosome 18 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LAMAPTBHSS-LAM-alpha     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LAMA1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LAMININ, ALPHA-1OMIM: PORETTI-BOLTSHAUSER SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LAMA1  OMIM: LAMININ, ALPHA-1OMIM: PORETTI-BOLTSHAUSER SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NETO1\t\n",
            "FBXO15\t\n",
            "CNDP2\t\n",
            "CNDP1\t\n",
            "SALL3\t\n",
            "KCNG2\t\n",
            "TXNDC2\t\n",
            "COL5A3\t\n",
            "GRIN3B\t\n",
            "OR4F17\t\n",
            "DOCK6\t   The DOCK6 gene provides instructions for making a protein known as a guanine nucleotide exchange factor (GEF). GEFs turn on (activate) proteins called GTPases, which play an important role in chemical signaling within cells. Often referred to as molecular switches, GTPases can be turned on and off. GTPases are turned off (inactivated) when they are attached (bound) to a molecule called GDP and are activated when they are bound to another molecule called GTP. The DOCK6 protein activates GTPases known as Cdc42 and Rac1 by exchanging GTP for the attached GDP. Once Cdc42 and Rac1 are active, they transmit signals that are critical for various aspects of embryonic development. The DOCK6 protein appears to regulate these GTPases specifically during development of the limbs, skull, and heart. DOCK6 also plays a role in the development of fibers (axons) that extend from nerve cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Adams-Oliver syndrome     Mutations in the DOCK6 gene cause Adams-Oliver syndrome, a condition characterized by areas of missing skin (aplasia cutis congenita), usually on the scalp, and malformations of the hands and feet. Neurological abnormalities, such as brain or eye malformations and intellectual disability, are more common in DOCK6-related Adams-Oliver syndrome than in cases associated with other genes. Most DOCK6 gene mutations involved in this condition lead to production of an abnormally short DOCK6 protein that is likely unable to function. Other mutations change single protein building blocks (amino acids) in the DOCK6 protein, which impairs the protein's normal function. The inability of DOCK6 to turn on Cdc42 or Rac1 leads to a reduction in their signaling, which impairs proper development of certain tissues, including the skin on the top of the head and the bones in the hands and feet. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the DOCK6 gene cause Adams-Oliver syndrome, a condition characterized by areas of missing skin (aplasia cutis congenita), usually on the scalp, and malformations of the hands and feet. Neurological abnormalities, such as brain or eye malformations and intellectual disability, are more common in DOCK6-related Adams-Oliver syndrome than in cases associated with other genes. Most DOCK6 gene mutations involved in this condition lead to production of an abnormally short DOCK6 protein that is likely unable to function. Other mutations change single protein building blocks (amino acids) in the DOCK6 protein, which impairs the protein's normal function. The inability of DOCK6 to turn on Cdc42 or Rac1 leads to a reduction in their signaling, which impairs proper development of certain tissues, including the skin on the top of the head and the bones in the hands and feet. More About This Health Condition    Cytogenetic Location: 19p13.2, which is the short (p) arm of chromosome 19 at position 13.2Molecular Location: base pairs 11,199,293 to 11,262,501 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AOS2dedicator of cytokinesis protein 6DOCK6_HUMANKIAA1395ZIR1     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database (2000): Rho GTPases Function as Membrane-Associated GDP/GTP-Regulated Molecular Switches      Tests Listed in the Genetic Testing Registry (1 link)     Tests of DOCK6      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     DEDICATOR OF CYTOKINESIS 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database (2000): Rho GTPases Function as Membrane-Associated GDP/GTP-Regulated Molecular Switches  Tests of DOCK6  PubMed  DEDICATOR OF CYTOKINESIS 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: DEDICATOR OF CYTOKINESIS 6Miyamoto Y, Torii T, Yamamori N, Ogata T, Tanoue A, Yamauchi J. Akt and PP2A reciprocally regulate the guanine nucleotide exchange factor Dock6 to control axon growth of sensory neurons. Sci Signal. 2013 Mar 5;6(265):ra15. doi: 10.1126/scisignal.2003661. Citation on PubMedMiyamoto Y, Yamauchi J, Sanbe A, Tanoue A. Dock6, a Dock-C subfamily guanine nucleotide exchanger, has the dual specificity for Rac1 and Cdc42 and regulates neurite outgrowth. Exp Cell Res. 2007 Feb 15;313(4):791-804. Epub 2006 Dec 6. Citation on PubMedRaftopoulou M, Hall A. Cell migration: Rho GTPases lead the way. Dev Biol. 2004 Jan 1;265(1):23-32. Review. Citation on PubMedShaheen R, Faqeih E, Sunker A, Morsy H, Al-Sheddi T, Shamseldin HE, Adly N, Hashem M, Alkuraya FS. Recessive mutations in DOCK6, encoding the guanidine nucleotide exchange factor DOCK6, lead to abnormal actin cytoskeleton organization and Adams-Oliver syndrome. Am J Hum Genet. 2011 Aug 12;89(2):328-33. doi: 10.1016/j.ajhg.2011.07.009. Epub 2011 Aug 4. Citation on PubMed or Free article on PubMed CentralSukalo M, Tilsen F, Kayserili H, Mller D, Tysz B, Ruddy DM, Wakeling E, rstavik KH, Snape KM, Trembath R, De Smedt M, van der Aa N, Skalej M, Mundlos S, Wuyts W, Southgate L, Zenker M. DOCK6 mutations are responsible for a distinct autosomal-recessive variant of Adams-Oliver syndrome associated with brain and eye anomalies. Hum Mutat. 2015 Jun;36(6):593-8. doi: 10.1002/humu.22795. Epub 2015 Apr 21. Erratum in: Hum Mutat. 2015 Nov;36(11):1112. Citation on PubMed \n",
            "RGL3\t\n",
            "PRKCSH\t   From NCBI Gene:This gene encodes the beta-subunit of glucosidase II, an N-linked glycan-processing enzyme in the endoplasmic reticulum. The encoded protein is an acidic phosphoprotein known to be a substrate for protein kinase C. Mutations in this gene have been associated with the autosomal dominant polycystic liver disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2014]From UniProt:Regulatory subunit of glucosidase II that cleaves sequentially the 2 innermost alpha-1,3-linked glucose residues from the Glc(2)Man(9)GlcNAc(2) oligosaccharide precursor of immature glycoproteins (PubMed:10929008). Required for efficient PKD1/Polycystin-1 biogenesis and trafficking to the plasma membrane of the primary cilia (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Polycystic liver disease 1From UniProt:Polycystic liver disease 1 with or without kidney cysts (PCLD1): An autosomal dominant hepatobiliary disease characterized by overgrowth of biliary epithelium and supportive connective tissue, resulting in multiple liver cysts. A subset of patients may develop kidney cysts that usually do not result in clinically significant renal disease. [MIM:174050]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19p13.2, which is the short (p) arm of chromosome 19 at position 13.2Molecular Location: base pairs 11,435,257 to 11,450,968 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AGE-R2G19P1GIIBPCLDPCLD1PKCSHPLD1VASAP-60     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PRKCSH      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: POLYCYSTIC LIVER DISEASE 1 WITH OR WITHOUT KIDNEY CYSTSOMIM: PROTEIN KINASE C SUBSTRATE, 80-KD, HEAVY CHAIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PRKCSH  OMIM: POLYCYSTIC LIVER DISEASE 1 WITH OR WITHOUT KIDNEY CYSTSOMIM: PROTEIN KINASE C SUBSTRATE, 80-KD, HEAVY CHAIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF653\t\n",
            "ACP5\t   The ACP5 gene provides instructions for making an enzyme called tartrate-resistant acid phosphatase type 5 (TRAP). The TRAP enzyme primarily regulates the activity of a protein called osteopontin, which is produced in bone cells called osteoclasts and in immune cells. Osteopontin performs a variety of functions in these cells. Two versions (isoforms) of the TRAP enzyme are produced: TRAP5a is found primarily in immune cells and TRAP5b is found primarily in bone cells called osteoclasts.Osteoclasts are specialized cells that break down and remove (resorb) bone tissue that is no longer needed. These cells are involved in bone remodeling, which is a normal process that replaces old bone tissue with new bone. During bone remodeling, osteopontin is turned on (activated), allowing osteoclasts to attach (bind) to bones. When the breakdown of bone is complete, TRAP5b turns off (inactivates) osteopontin, causing the osteoclasts to release themselves from bone.In the immune system, osteopontin is found primarily in cells called macrophages and dendritic cells. The protein helps fight infection by promoting inflammation, regulating immune cell activity, and turning on various immune system cells that are necessary to fight off foreign invaders such as bacteria and viruses. Like TRAP5b in bone cells, the TRAP5a enzyme inactivates osteopontin in macrophages and dendritic cells when it is no longer needed.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Spondyloenchondrodysplasia with immune dysregulation     At least 10 mutations in the ACP5 gene have been found to cause spondyloenchondrodysplasia with immune dysregulation (SPENCDI). This condition is characterized by abnormalities in bone growth and immune system function. The ACP5 gene mutations that cause SPENCDI typically change single protein building blocks (amino acids) in the TRAP enzyme or result in the production of an abnormally short enzyme. These mutations affect both TRAP isoforms and impair or eliminate TRAP's ability to inactivate osteopontin. As a result, osteopontin is abnormally active, prolonging bone breakdown by osteoclasts and triggering abnormal inflammation and immune responses by immune cells.In people with SPENCDI, increased bone breakdown contributes to the skeletal abnormalities, including irregularly shaped bones and short stature. An overactive immune system leads to increased susceptibility to autoimmune disorders, which occur when the immune system malfunctions and attacks the body's own tissues and organs. Immune system abnormalities also impair the body's normal response to harmful invaders, resulting in frequent infections. SPENCDI has several additional features, including movement disorders and intellectual disability, but it is unknown how changes in TRAP enzyme function lead to these other signs and symptoms. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 10 mutations in the ACP5 gene have been found to cause spondyloenchondrodysplasia with immune dysregulation (SPENCDI). This condition is characterized by abnormalities in bone growth and immune system function. The ACP5 gene mutations that cause SPENCDI typically change single protein building blocks (amino acids) in the TRAP enzyme or result in the production of an abnormally short enzyme. These mutations affect both TRAP isoforms and impair or eliminate TRAP's ability to inactivate osteopontin. As a result, osteopontin is abnormally active, prolonging bone breakdown by osteoclasts and triggering abnormal inflammation and immune responses by immune cells.In people with SPENCDI, increased bone breakdown contributes to the skeletal abnormalities, including irregularly shaped bones and short stature. An overactive immune system leads to increased susceptibility to autoimmune disorders, which occur when the immune system malfunctions and attacks the body's own tissues and organs. Immune system abnormalities also impair the body's normal response to harmful invaders, resulting in frequent infections. SPENCDI has several additional features, including movement disorders and intellectual disability, but it is unknown how changes in TRAP enzyme function lead to these other signs and symptoms. More About This Health Condition    Cytogenetic Location: 19p13.2, which is the short (p) arm of chromosome 19 at position 13.2Molecular Location: base pairs 11,574,660 to 11,579,008 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PPA5_HUMANtartrate-resistant acid ATPasetartrate-resistant acid phosphatase type 5TRAPTrATPase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Bone Health and Osteoporosis: A Report of the Surgeon General (2004): How Bones Change Throughout Life      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ACP5      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PHOSPHATASE, ACID, TYPE 5, TARTRATE-RESISTANT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Bone Health and Osteoporosis: A Report of the Surgeon General (2004): How Bones Change Throughout Life  Tests of ACP5  PubMed  PHOSPHATASE, ACID, TYPE 5, TARTRATE-RESISTANT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Behrens TW, Graham RR. TRAPing a new gene for autoimmunity. Nat Genet. 2011 Feb;43(2):90-1. doi: 10.1038/ng0211-90. Citation on PubMedBriggs TA, Rice GI, Daly S, Urquhart J, Gornall H, Bader-Meunier B, Baskar K, Baskar S, Baudouin V, Beresford MW, Black GC, Dearman RJ, de Zegher F, Foster ES, Francs C, Hayman AR, Hilton E, Job-Deslandre C, Kulkarni ML, Le Merrer M, Linglart A, Lovell SC, Maurer K, Musset L, Navarro V, Picard C, Puel A, Rieux-Laucat F, Roifman CM, Scholl-Brgi S, Smith N, Szynkiewicz M, Wiedeman A, Wouters C, Zeef LA, Casanova JL, Elkon KB, Janckila A, Lebon P, Crow YJ. Tartrate-resistant acid phosphatase deficiency causes a bone dysplasia with autoimmunity and a type I interferon expression signature. Nat Genet. 2011 Feb;43(2):127-31. doi: 10.1038/ng.748. Epub 2011 Jan 9. Citation on PubMed or Free article on PubMed CentralHayman AR. Tartrate-resistant acid phosphatase (TRAP) and the osteoclast/immune cell dichotomy. Autoimmunity. 2008 Apr;41(3):218-23. doi: 10.1080/08916930701694667. Review. Citation on PubMedLausch E, Janecke A, Bros M, Trojandt S, Alanay Y, De Laet C, Hbner CA, Meinecke P, Nishimura G, Matsuo M, Hirano Y, Tenoutasse S, Kiss A, Rosa RF, Unger SL, Renella R, Bonaf L, Spranger J, Unger S, Zabel B, Superti-Furga A. Genetic deficiency of tartrate-resistant acid phosphatase associated with skeletal dysplasia, cerebral calcifications and autoimmunity. Nat Genet. 2011 Feb;43(2):132-7. doi: 10.1038/ng.749. Epub 2011 Jan 9. Citation on PubMedOMIM: PHOSPHATASE, ACID, TYPE 5, TARTRATE-RESISTANT \n",
            "ZNF440\t\n",
            "ZNF878\t\n",
            "STK11\t   The STK11 gene (also called LKB1) provides instructions for making an enzyme called serine/threonine kinase 11. This enzyme is a tumor suppressor, which means that it helps keep cells from growing and dividing too fast or in an uncontrolled way. This enzyme helps certain types of cells correctly orient themselves within tissues (polarization) and assists in determining the amount of energy a cell uses. This kinase also promotes a type of programmed cell death known as apoptosis. In addition to its role as a tumor suppressor, serine/threonine kinase 11 function appears to be required for normal development before birth.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Peutz-Jeghers syndrome     Inherited mutations in the STK11 gene cause Peutz-Jeghers syndrome, a condition characterized by the development of noncancerous growths called hamartomatous polyps in the gastrointestinal tract and a greatly increased risk of developing several types of cancer. More than 340 STK11 gene mutations have been identified in people with this condition. Many of these mutations result in the production of an abnormally short, nonfunctional version of the serine/threonine kinase 11 enzyme. Other mutations change single protein building blocks (amino acids) used to build the enzyme. Mutations in the STK11 gene impair the enzyme's tumor suppressor function, allowing cells to grow and divide without control or order. This uncontrolled cell growth can lead to the formation of hamartomatous polyps and cancerous tumors. More About This Health Condition      Breast cancer     Inherited changes in the STK11 gene greatly increase the risk of developing breast cancer, as well as other types of cancer, as part of Peutz-Jeghers syndrome (described above). These mutations are thought to account for only a small fraction of all breast cancer cases. More About This Health Condition      Lung cancer     Genetics Home Reference provides information about lung cancer. More About This Health Condition      Ovarian cancer     Genetics Home Reference provides information about ovarian cancer. More About This Health Condition      Other cancers     Noninherited (somatic) mutations in the STK11 gene have been found in various forms of cancer. Somatic mutations are acquired during a person's lifetime and are present only in certain cells. They do not occur as part of a cancer syndrome. Somatic STK11 gene mutations have been identified in a form of lung cancer called non-small cell lung carcinoma, cervical cancer, colorectal cancer, an aggressive type of skin cancer called melanoma, and pancreatic cancer. These mutations impair the function of serine/threonine kinase 11, which can allow cells to grow and divide uncontrollably and contribute to the formation of a cancerous tumor.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Inherited mutations in the STK11 gene cause Peutz-Jeghers syndrome, a condition characterized by the development of noncancerous growths called hamartomatous polyps in the gastrointestinal tract and a greatly increased risk of developing several types of cancer. More than 340 STK11 gene mutations have been identified in people with this condition. Many of these mutations result in the production of an abnormally short, nonfunctional version of the serine/threonine kinase 11 enzyme. Other mutations change single protein building blocks (amino acids) used to build the enzyme. Mutations in the STK11 gene impair the enzyme's tumor suppressor function, allowing cells to grow and divide without control or order. This uncontrolled cell growth can lead to the formation of hamartomatous polyps and cancerous tumors. More About This Health Condition  Inherited changes in the STK11 gene greatly increase the risk of developing breast cancer, as well as other types of cancer, as part of Peutz-Jeghers syndrome (described above). These mutations are thought to account for only a small fraction of all breast cancer cases. More About This Health Condition  Genetics Home Reference provides information about lung cancer. More About This Health Condition  Genetics Home Reference provides information about ovarian cancer. More About This Health Condition  Noninherited (somatic) mutations in the STK11 gene have been found in various forms of cancer. Somatic mutations are acquired during a person's lifetime and are present only in certain cells. They do not occur as part of a cancer syndrome. Somatic STK11 gene mutations have been identified in a form of lung cancer called non-small cell lung carcinoma, cervical cancer, colorectal cancer, an aggressive type of skin cancer called melanoma, and pancreatic cancer. These mutations impair the function of serine/threonine kinase 11, which can allow cells to grow and divide uncontrollably and contribute to the formation of a cancerous tumor.    Cytogenetic Location: 19p13.3, which is the short (p) arm of chromosome 19 at position 13.3Molecular Location: base pairs 1,205,799 to 1,228,435 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LKB1PJSserine/threonine kinase 11 (Peutz-Jeghers syndrome)Serine/threonine-protein kinase 11STK11_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Cancer Syndromes (2009): Peutz-Jeghers SyndromeNational Cancer Institute: Genetics of Breast and Gynecologic Cancers (PDQ)      Clinical Information from GeneReviews (1 link)     Peutz-Jeghers Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of STK11      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SERINE/THREONINE PROTEIN KINASE 11      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics Web: STK11ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Cancer Syndromes (2009): Peutz-Jeghers SyndromeNational Cancer Institute: Genetics of Breast and Gynecologic Cancers (PDQ)  Peutz-Jeghers Syndrome  Tests of STK11  PubMed  SERINE/THREONINE PROTEIN KINASE 11  Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics Web: STK11ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Hardie DG, Alessi DR. LKB1 and AMPK and the cancer-metabolism link - ten years after. BMC Biol. 2013 Apr 15;11:36. doi: 10.1186/1741-7007-11-36. Review. Citation on PubMed or Free article on PubMed CentralHemminki A, Markie D, Tomlinson I, Avizienyte E, Roth S, Loukola A, Bignell G, Warren W, Aminoff M, Hglund P, Jrvinen H, Kristo P, Pelin K, Ridanp M, Salovaara R, Toro T, Bodmer W, Olschwang S, Olsen AS, Stratton MR, de la Chapelle A, Aaltonen LA. A serine/threonine kinase gene defective in Peutz-Jeghers syndrome. Nature. 1998 Jan 8;391(6663):184-7. Citation on PubMedMcGarrity TJ, Amos CI, Baker MJ. Peutz-Jeghers Syndrome. 2001 Feb 23 [updated 2016 Jul 14]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1266/ Citation on PubMedNational Cancer Institute: Genetics of Breast and Gynecologic Cancers (PDQ)Vaahtomeri K, Mkel TP. Molecular mechanisms of tumor suppression by LKB1. FEBS Lett. 2011 Apr 6;585(7):944-51. doi: 10.1016/j.febslet.2010.12.034. Epub 2010 Dec 27. Review. Citation on PubMedWeissman SM, Weiss SM, Newlin AC. Genetic testing by cancer site: ovary. Cancer J. 2012 Jul-Aug;18(4):320-7. doi: 10.1097/PPO.0b013e31826246c2. Review. Citation on PubMedWingo SN, Gallardo TD, Akbay EA, Liang MC, Contreras CM, Boren T, Shimamura T, Miller DS, Sharpless NE, Bardeesy N, Kwiatkowski DJ, Schorge JO, Wong KK, Castrillon DH. Somatic LKB1 mutations promote cervical cancer progression. PLoS One. 2009;4(4):e5137. doi: 10.1371/journal.pone.0005137. Epub 2009 Apr 2. Citation on PubMed or Free article on PubMed CentralZhuang ZG, Di GH, Shen ZZ, Ding J, Shao ZM. Enhanced expression of LKB1 in breast cancer cells attenuates angiogenesis, invasion, and metastatic potential. Mol Cancer Res. 2006 Nov;4(11):843-9. Citation on PubMed \n",
            "ZNF443\t\n",
            "ZNF564\t\n",
            "RTBDN\t\n",
            "DNASE2\t\n",
            "CACNA1A\t   The CACNA1A gene belongs to a family of genes that provide instructions for making calcium channels. These channels, which transport positively charged calcium atoms (calcium ions) across cell membranes, play a key role in a cell's ability to generate and transmit electrical signals. Calcium ions are involved in many different cellular functions, including cell-to-cell communication, the tensing of muscle fibers (muscle contraction), and the regulation of certain genes.The CACNA1A gene provides instructions for making one part (the alpha-1 subunit) of a calcium channel called CaV2.1. This subunit forms the hole (pore) through which calcium ions can flow. CaV2.1 channels play an essential role in communication between nerve cells (neurons) in the brain. These channels help control the release of neurotransmitters, which are chemicals that relay signals from one neuron to another. Researchers believe that CaV2.1 channels are also involved in the survival of neurons and the ability of these cells to change and adapt over time (plasticity).Near one end of the CACNA1A gene, a segment of three DNA building blocks (nucleotides) is repeated multiple times. This sequence, which is written as CAG, is called a triplet or trinucleotide repeat. In most people, the number of CAG repeats in this gene ranges from 4 to 18.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Episodic ataxia     More than 80 mutations in the CACNA1A gene have been found to cause episodic ataxia type 2 (EA2), the most common form of episodic ataxia. In addition to problems with coordination and balance (ataxia), EA2 is associated with involuntary eye movements called nystagmus. The CACNA1A mutations responsible for EA2 reduce the production of functional CaV2.1 channels or prevent these channels from reaching the cell membrane, where they are needed to transport calcium ions. A decrease in the number of these channels reduces the total flow of calcium ions into neurons, which disrupts the release of neurotransmitters in the brain. Although changes in signaling between neurons underlie the episodes of uncoordinated movement seen in people with episodic ataxia, it is unclear how altered calcium ion transport causes the specific features of the condition. More About This Health Condition      Familial hemiplegic migraine     At least 20 mutations in the CACNA1A gene have been identified in people with familial hemiplegic migraine type 1 (FHM1). This condition is characterized by migraine headaches with a pattern of neurological symptoms known as aura. In FHM1, the aura includes temporary numbness or weakness on one side of the body (hemiparesis). Like EA2 (described above), FHM1 is commonly associated with ataxia and nystagmus. Most of the mutations that cause FHM1 change single protein building blocks (amino acids) in the CaV2.1 channel. The most common mutation, which has been found in more than a dozen affected families, replaces the amino acid threonine with the amino acid methionine at protein position 666 (written as Thr666Met or T666M).The CACNA1A mutations responsible for familial hemiplegic migraine change the structure of the CaV2.1 channel. The altered channels open more easily than usual, which increases the inward flow of calcium ions. A greater influx of calcium ions through CaV2.1 channels increases the cell's release of neurotransmitters. The resulting changes in signaling between neurons lead to development of these severe headaches in people with familial hemiplegic migraine. More About This Health Condition      Spinocerebellar ataxia type 6     Spinocerebellar ataxia type 6 (SCA6) is another disorder caused by CACNA1A gene mutations. The major features of this condition include progressive ataxia, nystagmus, and impaired speech (dysarthria), most often beginning in a person's forties or fifties. SCA6 results from an increased number of copies (expansion) of the CAG trinucleotide repeat in the CACNA1A gene. In people with this condition, the CAG segment is repeated from 20 to more than 30 times.An increase in the length of the CAG segment leads to the production of an abnormally long version of the alpha-1 subunit. The abnormal subunit is found in the cell membrane as well as in the fluid inside cells (cytoplasm), where it forms clumps (aggregates). The effect these aggregates have on cell functioning is unknown. The lack of normal calcium channels impairs the cells' ability to transport calcium ions. These changes alter the release of neurotransmitters in the brain and eventually lead to the death of neurons. Certain neurons called Purkinje cells seem to be particularly sensitive to a disruption in calcium transport. Purkinje cells are located in the part of the brain that coordinates movement (cerebellum). Over time, the loss of Purkinje cells and other cells of the cerebellum causes the movement problems characteristic of SCA6. More About This Health Condition      Sporadic hemiplegic migraine     At least nine mutations in the CACNA1A gene have been found to cause sporadic hemiplegic migraine. The signs and symptoms of this condition are identical to those of FHM1 (described above); however, sporadic hemiplegic migraine occurs in people with no family history of the condition. As in FHM1, sporadic hemiplegic migraine caused by CACNA1A gene mutations is commonly associated with ataxia and nystagmus in addition to migraine headaches and auras.CACNA1A gene mutations that cause sporadic hemiplegic migraine change single amino acids in the CaV2.1 channel. Many of these mutations are also found in families with FHM1. The altered channels are more active than usual, which increases the release of neurotransmitters. The abnormal signaling between neurons caused by these changes lead to the headaches and auras characteristic of sporadic hemiplegic migraine. More About This Health Condition      19p13.13 deletion syndrome     The CACNA1A gene is located in a region of chromosome 19 that is missing in most people with 19p13.13 deletion syndrome. As a result of this deletion, many affected individuals are missing one copy of CACNA1A and several other genes in each cell. Features associated with 19p13.13 deletion syndrome include an unusually large head size (macrocephaly), tall stature, intellectual disability, seizures, ataxia, and other health problems. Researchers are working to determine which missing genes contribute to the specific features of the disorder. Studies suggest that the loss of one copy of the CACNA1A gene may underlie the seizures and ataxia in affected individuals. The deletion reduces the amount of CaV2.1 channels produced within cells, although it is unclear exactly how a shortage of these channels is related to seizures and ataxia in people with 19p13.13 deletion syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 80 mutations in the CACNA1A gene have been found to cause episodic ataxia type 2 (EA2), the most common form of episodic ataxia. In addition to problems with coordination and balance (ataxia), EA2 is associated with involuntary eye movements called nystagmus. The CACNA1A mutations responsible for EA2 reduce the production of functional CaV2.1 channels or prevent these channels from reaching the cell membrane, where they are needed to transport calcium ions. A decrease in the number of these channels reduces the total flow of calcium ions into neurons, which disrupts the release of neurotransmitters in the brain. Although changes in signaling between neurons underlie the episodes of uncoordinated movement seen in people with episodic ataxia, it is unclear how altered calcium ion transport causes the specific features of the condition. More About This Health Condition  At least 20 mutations in the CACNA1A gene have been identified in people with familial hemiplegic migraine type 1 (FHM1). This condition is characterized by migraine headaches with a pattern of neurological symptoms known as aura. In FHM1, the aura includes temporary numbness or weakness on one side of the body (hemiparesis). Like EA2 (described above), FHM1 is commonly associated with ataxia and nystagmus. Most of the mutations that cause FHM1 change single protein building blocks (amino acids) in the CaV2.1 channel. The most common mutation, which has been found in more than a dozen affected families, replaces the amino acid threonine with the amino acid methionine at protein position 666 (written as Thr666Met or T666M).The CACNA1A mutations responsible for familial hemiplegic migraine change the structure of the CaV2.1 channel. The altered channels open more easily than usual, which increases the inward flow of calcium ions. A greater influx of calcium ions through CaV2.1 channels increases the cell's release of neurotransmitters. The resulting changes in signaling between neurons lead to development of these severe headaches in people with familial hemiplegic migraine. More About This Health Condition  Spinocerebellar ataxia type 6 (SCA6) is another disorder caused by CACNA1A gene mutations. The major features of this condition include progressive ataxia, nystagmus, and impaired speech (dysarthria), most often beginning in a person's forties or fifties. SCA6 results from an increased number of copies (expansion) of the CAG trinucleotide repeat in the CACNA1A gene. In people with this condition, the CAG segment is repeated from 20 to more than 30 times.An increase in the length of the CAG segment leads to the production of an abnormally long version of the alpha-1 subunit. The abnormal subunit is found in the cell membrane as well as in the fluid inside cells (cytoplasm), where it forms clumps (aggregates). The effect these aggregates have on cell functioning is unknown. The lack of normal calcium channels impairs the cells' ability to transport calcium ions. These changes alter the release of neurotransmitters in the brain and eventually lead to the death of neurons. Certain neurons called Purkinje cells seem to be particularly sensitive to a disruption in calcium transport. Purkinje cells are located in the part of the brain that coordinates movement (cerebellum). Over time, the loss of Purkinje cells and other cells of the cerebellum causes the movement problems characteristic of SCA6. More About This Health Condition  At least nine mutations in the CACNA1A gene have been found to cause sporadic hemiplegic migraine. The signs and symptoms of this condition are identical to those of FHM1 (described above); however, sporadic hemiplegic migraine occurs in people with no family history of the condition. As in FHM1, sporadic hemiplegic migraine caused by CACNA1A gene mutations is commonly associated with ataxia and nystagmus in addition to migraine headaches and auras.CACNA1A gene mutations that cause sporadic hemiplegic migraine change single amino acids in the CaV2.1 channel. Many of these mutations are also found in families with FHM1. The altered channels are more active than usual, which increases the release of neurotransmitters. The abnormal signaling between neurons caused by these changes lead to the headaches and auras characteristic of sporadic hemiplegic migraine. More About This Health Condition  The CACNA1A gene is located in a region of chromosome 19 that is missing in most people with 19p13.13 deletion syndrome. As a result of this deletion, many affected individuals are missing one copy of CACNA1A and several other genes in each cell. Features associated with 19p13.13 deletion syndrome include an unusually large head size (macrocephaly), tall stature, intellectual disability, seizures, ataxia, and other health problems. Researchers are working to determine which missing genes contribute to the specific features of the disorder. Studies suggest that the loss of one copy of the CACNA1A gene may underlie the seizures and ataxia in affected individuals. The deletion reduces the amount of CaV2.1 channels produced within cells, although it is unclear exactly how a shortage of these channels is related to seizures and ataxia in people with 19p13.13 deletion syndrome. More About This Health Condition    Cytogenetic Location: 19p13.13, which is the short (p) arm of chromosome 19 at position 13.13Molecular Location: base pairs 13,206,442 to 13,506,460 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        APCAbrain calcium channel 1CAC1A_HUMANCACNL1A4calcium channel, alpha 1A subunitcalcium channel, L type, alpha-1 polypeptide, isoform 4calcium channel, voltage-dependent, P/Q type, alpha 1A subunitCAV2.1HPCASCA6Voltage-gated calcium channel subunit alpha Cav2.1     Related Information   How are genetic conditions and genes named?          Educational Resources (4 links)     Eurekah Bioscience Collection: Biochemical Properties of the Cav2 Family of Ca2+ ChannelsNeuromuscular Disease Center, Washington University: Calcium channelsUndiagnosed Diseases Network: Gene PageWashington University, St. Louis: Neuromuscular Disease Center: SCA6      Clinical Information from GeneReviews (3 links)     Episodic Ataxia Type 2Familial Hemiplegic MigraineSpinocerebellar Ataxia Type 6      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CACNA1A      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CALCIUM CHANNEL, VOLTAGE-DEPENDENT, P/Q TYPE, ALPHA-1A SUBUNIT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Eurekah Bioscience Collection: Biochemical Properties of the Cav2 Family of Ca2+ ChannelsNeuromuscular Disease Center, Washington University: Calcium channelsUndiagnosed Diseases Network: Gene PageWashington University, St. Louis: Neuromuscular Disease Center: SCA6  Episodic Ataxia Type 2Familial Hemiplegic MigraineSpinocerebellar Ataxia Type 6  Tests of CACNA1A  PubMed  CALCIUM CHANNEL, VOLTAGE-DEPENDENT, P/Q TYPE, ALPHA-1A SUBUNIT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Auvin S, Holder-Espinasse M, Lamblin MD, Andrieux J. Array-CGH detection of a de novo 0.7-Mb deletion in 19p13.13 including CACNA1A associated with mental retardation and epilepsy with infantile spasms. Epilepsia. 2009 Nov;50(11):2501-3. doi: 10.1111/j.1528-1167.2009.02189.x. Citation on PubMedJeng CJ, Sun MC, Chen YW, Tang CY. Dominant-negative effects of episodic ataxia type 2 mutations involve disruption of membrane trafficking of human P/Q-type Ca2+ channels. J Cell Physiol. 2008 Feb;214(2):422-33. Citation on PubMedKors EE, Haan J, Giffin NJ, Pazdera L, Schnittger C, Lennox GG, Terwindt GM, Vermeulen FL, Van den Maagdenberg AM, Frants RR, Ferrari MD. Expanding the phenotypic spectrum of the CACNA1A gene T666M mutation: a description of 5 families with familial hemiplegic migraine. Arch Neurol. 2003 May;60(5):684-8. Citation on PubMedLuo X, Rosenfeld JA, Yamamoto S, Harel T, Zuo Z, Hall M, Wierenga KJ, Pastore MT, Bartholomew D, Delgado MR, Rotenberg J, Lewis RA, Emrick L, Bacino CA, Eldomery MK, Coban Akdemir Z, Xia F, Yang Y, Lalani SR, Lotze T, Lupski JR, Lee B, Bellen HJ, Wangler MF; Members of the UDN. Clinically severe CACNA1A alleles affect synaptic function and neurodegeneration differentially. PLoS Genet. 2017 Jul 24;13(7):e1006905. doi: 10.1371/journal.pgen.1006905. eCollection 2017 Jul. Citation on PubMed or Free article on PubMed CentralNimmakayalu M, Horton VK, Darbro B, Patil SR, Alsayouf H, Keppler-Noreuil K, Shchelochkov OA. Apparent germline mosaicism for a novel 19p13.13 deletion disrupting NFIX and CACNA1A. Am J Med Genet A. 2013 May;161A(5):1105-9. doi: 10.1002/ajmg.a.35790. Epub 2013 Mar 13. Citation on PubMedRajakulendran S, Schorge S, Kullmann DM, Hanna MG. Dysfunction of the Ca(V)2.1 calcium channel in cerebellar ataxias. F1000 Biol Rep. 2010 Jan 18;2. pii: 4. doi: 10.3410/B2-4. Citation on PubMed or Free article on PubMed CentralRiant F, Ducros A, Ploton C, Barbance C, Depienne C, Tournier-Lasserve E. De novo mutations in ATP1A2 and CACNA1A are frequent in early-onset sporadic hemiplegic migraine. Neurology. 2010 Sep 14;75(11):967-72. doi: 10.1212/WNL.0b013e3181f25e8f. Citation on PubMedTerwindt G, Kors E, Haan J, Vermeulen F, Van den Maagdenberg A, Frants R, Ferrari M. Mutation analysis of the CACNA1A calcium channel subunit gene in 27 patients with sporadic hemiplegic migraine. Arch Neurol. 2002 Jun;59(6):1016-8. Review. Citation on PubMedTottene A, Fellin T, Pagnutti S, Luvisetto S, Striessnig J, Fletcher C, Pietrobon D. Familial hemiplegic migraine mutations increase Ca(2+) influx through single human CaV2.1 channels and decrease maximal CaV2.1 current density in neurons. Proc Natl Acad Sci U S A. 2002 Oct 1;99(20):13284-9. Epub 2002 Sep 16. Citation on PubMed or Free article on PubMed CentralWan J, Khanna R, Sandusky M, Papazian DM, Jen JC, Baloh RW. CACNA1A mutations causing episodic and progressive ataxia alter channel trafficking and kinetics. Neurology. 2005 Jun 28;64(12):2090-7. Citation on PubMedZhuchenko O, Bailey J, Bonnen P, Ashizawa T, Stockton DW, Amos C, Dobyns WB, Subramony SH, Zoghbi HY, Lee CC. Autosomal dominant cerebellar ataxia (SCA6) associated with small polyglutamine expansions in the alpha 1A-voltage-dependent calcium channel. Nat Genet. 1997 Jan;15(1):62-9. Citation on PubMedde Vries B, Freilinger T, Vanmolkot KR, Koenderink JB, Stam AH, Terwindt GM, Babini E, van den Boogerd EH, van den Heuvel JJ, Frants RR, Haan J, Pusch M, van den Maagdenberg AM, Ferrari MD, Dichgans M. Systematic analysis of three FHM genes in 39 sporadic patients with hemiplegic migraine. Neurology. 2007 Dec 4;69(23):2170-6. Citation on PubMed \n",
            "C19orf57\t\n",
            "DCAF15\t\n",
            "PALM3\t\n",
            "OR7A10\t\n",
            "OR7C2\t\n",
            "CCDC105\t\n",
            "OR1I1\t\n",
            "PLK5\t\n",
            "CYP4F12\t\n",
            "OR10H3\t\n",
            "OR10H1\t\n",
            "HSH2D\t\n",
            "CHERP\t\n",
            "CPAMD8\t   From NCBI Gene:This gene encodes a member of the protease inhibitor I39 (alpha-2-macroglobulin) family of proteins. These proteins are important in innate and acquired immunity. The encoded protein is membrane-associated and proteolytically processed to generate two chains. Mutations in this gene cause a form of anterior segment dysgenesis, a developmental disorder of the eye. [provided by RefSeq, May 2017]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Anterior segment dysgenesis 8     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19p13.11, which is the short (p) arm of chromosome 19 at position 13.11Molecular Location: base pairs 16,892,947 to 17,026,818 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ASGD8K-CAPVIP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CPAMD8      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ANTERIOR SEGMENT DYSGENESIS 8OMIM: COMPLEMENT COMPONENT 3- AND PREGNANCY ZONE PROTEIN-LIKE ALPHA-2-MACROGLOBULIN DOMAIN-CONTAINING PROTEIN 8      Research Resources (4 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene      Tests of CPAMD8  OMIM: ANTERIOR SEGMENT DYSGENESIS 8OMIM: COMPLEMENT COMPONENT 3- AND PREGNANCY ZONE PROTEIN-LIKE ALPHA-2-MACROGLOBULIN DOMAIN-CONTAINING PROTEIN 8  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene \n",
            "ANKLE1\t\n",
            "PLVAP\t\n",
            "NXNL1\t\n",
            "MAP1S\t\n",
            "ATP8B3\t\n",
            "ARRDC2\t\n",
            "REXO1\t\n",
            "IFI30\t\n",
            "MPV17L2\t\n",
            "SSBP4\t\n",
            "GDF1\t   From NCBI Gene:This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer. Studies in rodents suggest that this protein is involved in the establishment of left-right asymmetry in early embryogenesis and in neural development in later embryogenesis. The encoded protein is translated from a bicistronic mRNA that also encodes ceramide synthase 1. Mutations in this gene are associated with several congenital cardiovascular malformations. [provided by RefSeq, Jul 2016]From UniProt:May mediate cell differentiation events during embryonic development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Heterotaxy syndromeCritical congenital heart diseaseFrom NCBI Gene:Bilateral right-sidedness sequenceTransposition of the great arteries, dextro-looped 3From UniProt:Congenital heart defects, multiple types, 6 (CHTD6): An autosomal dominant disorder characterized by congenital developmental abnormalities involving structures of the heart. Common defects include tetralogy of Fallot, transposition of the great arteries, double-outlet right ventricle, total anomalous pulmonary venous return, pulmonary stenosis or atresia, atrioventricular canal, ventricular septal defect, and hypoplastic left or right ventricle. [MIM:613854]Tetralogy of Fallot (TOF): A congenital heart anomaly which consists of pulmonary stenosis, ventricular septal defect, dextroposition of the aorta (aorta is on the right side instead of the left) and hypertrophy of the right ventricle. In this condition, blood from both ventricles (oxygen-rich and oxygen-poor) is pumped into the body often causing cyanosis. [MIM:187500]Conotruncal heart malformations (CTHM): A group of congenital heart defects involving the outflow tracts. Examples include truncus arteriosus communis, double-outlet right ventricle and transposition of great arteries. Truncus arteriosus communis is characterized by a single outflow tract instead of a separate aorta and pulmonary artery. In transposition of the great arteries, the aorta arises from the right ventricle and the pulmonary artery from the left ventricle. In double outlet of the right ventricle, both the pulmonary artery and aorta arise from the right ventricle. [MIM:217095]Right atrial isomerism (RAI): A severe complex congenital heart defect resulting from embryonic disruption of proper left-right axis determination. RAI is usually characterized by complete atrioventricular septal defect with a common atrium and univentricular AV connection, total anomalous pulmonary drainage, and transposition or malposition of the great arteries. Affected individuals present at birth with severe cardiac failure. Other associated abnormalities include bilateral trilobed lungs, midline liver, and asplenia, as well as situs inversus affecting other organs. [MIM:208530]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19p13.11, which is the short (p) arm of chromosome 19 at position 13.11Molecular Location: base pairs 18,868,546 to 18,896,144 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CERS1CHTD6DORVDTGA3LAG1LASS1RAIUOG1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GDF1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CONGENITAL HEART DEFECTS, MULTIPLE TYPES, 6OMIM: GROWTH/DIFFERENTIATION FACTOR 1OMIM: RIGHT ATRIAL ISOMERISM      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GDF1  OMIM: CONGENITAL HEART DEFECTS, MULTIPLE TYPES, 6OMIM: GROWTH/DIFFERENTIATION FACTOR 1OMIM: RIGHT ATRIAL ISOMERISM  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NCAN\t\n",
            "BTBD2\t\n",
            "ZNF90\t\n",
            "ZNF626\t\n",
            "ZNF714\t\n",
            "ZNF708\t\n",
            "ZNF493\t\n",
            "ZNF208\t\n",
            "DOT1L\t   From NCBI Gene:The protein encoded by this gene is a histone methyltransferase that methylates lysine-79 of histone H3. It is inactive against free core histones, but shows significant histone methyltransferase activity against nucleosomes. [provided by RefSeq, Aug 2011]From UniProt:Histone methyltransferase. Methylates 'Lys-79' of histone H3. Nucleosomes are preferred as substrate compared to free histones (PubMed:12123582). Binds to DNA (PubMed:12628190).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Osteoarthritis     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19p13.3, which is the short (p) arm of chromosome 19 at position 13.3Molecular Location: base pairs 2,163,963 to 2,232,578 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DOT1KMT4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DOT1L      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: DOT1-LIKE      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DOT1L  OMIM: DOT1-LIKE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF98\t\n",
            "SPPL2B\t\n",
            "ZNF91\t\n",
            "SLC39A3\t\n",
            "URI1\t\n",
            "RGS9BP\t   The RGS9BP gene (which is also known as R9AP) provides instructions for making a protein called RGS9 binding protein. This protein is found in the retina, which is the specialized tissue at the back of the eye that detects light and color. Within the retina, the protein is associated with light-detecting cells called photoreceptors. As its name suggests, RGS9 binding protein interacts with a protein called RGS9 (which is produced from the RGS9 gene). It anchors the RGS9 protein to photoreceptors and stimulates RGS9's activity.When light enters the eye, it stimulates specialized pigments in photoreceptor cells. This stimulation triggers a series of chemical reactions that produce an electrical signal, which is interpreted by the brain as vision. (This process is known as phototransduction.) Once photoreceptors have been stimulated by light, they must return to their resting state before they can be stimulated again. RGS9 and RGS9 binding protein are involved in a chemical reaction that helps return photoreceptors to their resting state quickly after light exposure.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Bradyopsia     At least six mutations in the RGS9BP gene have been found to cause bradyopsia, a rare condition that affects vision. In people with bradyopsia, the eyes adapt more slowly than usual to changing light conditions (for example, walking out of a darkened movie theater into daylight or driving into a dark tunnel on a sunny day). Some affected individuals also have difficulty seeing small moving objects, such as a tennis ball.The RGS9BP gene mutations that cause bradyopsia prevent RGS9 binding protein from anchoring the RGS9 protein to photoreceptor cells. The resulting loss of RGS9 protein function prevents photoreceptors from recovering quickly after responding to light. Normally they return to their resting state in a fraction of a second, but in people with RGS9BP gene mutations, it can take ten seconds or longer. During that time, the photoreceptors cannot respond to light. This delay causes temporary blindness in response to changing light conditions and may interfere with seeing small objects when they are in motion. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least six mutations in the RGS9BP gene have been found to cause bradyopsia, a rare condition that affects vision. In people with bradyopsia, the eyes adapt more slowly than usual to changing light conditions (for example, walking out of a darkened movie theater into daylight or driving into a dark tunnel on a sunny day). Some affected individuals also have difficulty seeing small moving objects, such as a tennis ball.The RGS9BP gene mutations that cause bradyopsia prevent RGS9 binding protein from anchoring the RGS9 protein to photoreceptor cells. The resulting loss of RGS9 protein function prevents photoreceptors from recovering quickly after responding to light. Normally they return to their resting state in a fraction of a second, but in people with RGS9BP gene mutations, it can take ten seconds or longer. During that time, the photoreceptors cannot respond to light. This delay causes temporary blindness in response to changing light conditions and may interfere with seeing small objects when they are in motion. More About This Health Condition    Cytogenetic Location: 19q13.11, which is the long (q) arm of chromosome 19 at position 13.11Molecular Location: base pairs 32,675,407 to 32,678,300 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FLJ45744R9APregulator of G-protein signaling 9-binding proteinregulator of G-protein signaling 9 binding proteinRGS9 anchor proteinRGS9-anchoring protein     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Webvision: The Organization of the Retina and Visual System (2010): Phototransduction in Rods and ConesWebvision: The Organization of the Retina and Visual System (updated 2012): Photoreceptors      Tests Listed in the Genetic Testing Registry (1 link)     Tests of RGS9BP      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     REGULATOR OF G PROTEIN SIGNALING 9-BINDING PROTEIN      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Webvision: The Organization of the Retina and Visual System (2010): Phototransduction in Rods and ConesWebvision: The Organization of the Retina and Visual System (updated 2012): Photoreceptors  Tests of RGS9BP  PubMed  REGULATOR OF G PROTEIN SIGNALING 9-BINDING PROTEIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cheng JY, Luu CD, Yong VH, Mathur R, Aung T, Vithana EN. Bradyopsia in an Asian man. Arch Ophthalmol. 2007 Aug;125(8):1138-40. Citation on PubMedMichaelides M, Li Z, Rana NA, Richardson EC, Hykin PG, Moore AT, Holder GE, Webster AR. Novel mutations and electrophysiologic findings in RGS9- and R9AP-associated retinal dysfunction (Bradyopsia). Ophthalmology. 2010 Jan;117(1):120-127.e1. doi: 10.1016/j.ophtha.2009.06.011. Epub 2009 Oct 8. Citation on PubMedNishiguchi KM, Sandberg MA, Kooijman AC, Martemyanov KA, Pott JW, Hagstrom SA, Arshavsky VY, Berson EL, Dryja TP. Defects in RGS9 or its anchor protein R9AP in patients with slow photoreceptor deactivation. Nature. 2004 Jan 1;427(6969):75-8. Citation on PubMedOMIM: REGULATOR OF G PROTEIN SIGNALING 9-BINDING PROTEIN \n",
            "SLC7A9\t   The SLC7A9 gene provides instructions for producing one part (subunit) of a protein made primarily in the kidneys.  This subunit joins with another protein subunit, produced from the SLC3A1 gene, to form a transporter protein complex. During the process of urine formation in the kidneys, this protein complex absorbs particular protein building blocks (amino acids) back into the blood.  In particular, the amino acids cystine, ornithine, arginine, and lysine are absorbed back into the blood through this mechanism.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Cystinuria     At least 95 mutations in the SLC7A9 gene have been found to cause cystinuria.  Many of these mutations alter a single DNA building block (nucleotide) or insert or delete a small number of nucleotides in the SLC7A9 gene. These changes lead to an abnormally functioning transporter protein complex, which causes certain amino acids to become concentrated in the urine.  Cystine is the only amino acid that forms crystals and stones in the bladder or kidneys, leading to the signs and symptoms of cystinuria. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 95 mutations in the SLC7A9 gene have been found to cause cystinuria.  Many of these mutations alter a single DNA building block (nucleotide) or insert or delete a small number of nucleotides in the SLC7A9 gene. These changes lead to an abnormally functioning transporter protein complex, which causes certain amino acids to become concentrated in the urine.  Cystine is the only amino acid that forms crystals and stones in the bladder or kidneys, leading to the signs and symptoms of cystinuria. More About This Health Condition    Cytogenetic Location: 19q13.11, which is the long (q) arm of chromosome 19 at position 13.11Molecular Location: base pairs 32,830,511 to 32,870,957 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        b0,+ATBAT1_HUMANCSNU3solute carrier family 7 (amino acid transporter light chain, bo,+ system), member 9solute carrier family 7 (glycoprotein-associated amino acid transporter light chain, bo,+ system), member 9solute carrier family 7, member 9     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     National Institute of Diabetes and Digestive and Kidney Diseases: Your Kidneys and How They Work      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC7A9      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SOLUTE CARRIER FAMILY 7 (CATIONIC AMINO ACID TRANSPORTER, y+ SYSTEM), MEMBER 9      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      National Institute of Diabetes and Digestive and Kidney Diseases: Your Kidneys and How They Work  Tests of SLC7A9  PubMed  SOLUTE CARRIER FAMILY 7 (CATIONIC AMINO ACID TRANSPORTER, y+ SYSTEM), MEMBER 9  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Dello Strologo L, Pras E, Pontesilli C, Beccia E, Ricci-Barbini V, de Sanctis L, Ponzone A, Gallucci M, Bisceglia L, Zelante L, Jimenez-Vidal M, Font M, Zorzano A, Rousaud F, Nunes V, Gasparini P, Palacn M, Rizzoni G. Comparison between SLC3A1 and SLC7A9 cystinuria patients and carriers: a need for a new classification. J Am Soc Nephrol. 2002 Oct;13(10):2547-53. Citation on PubMedFernndez E, Carrascal M, Rousaud F, Abin J, Zorzano A, Palacn M, Chillarn J. rBAT-b(0,+)AT heterodimer is the main apical reabsorption system for cystine in the kidney. Am J Physiol Renal Physiol. 2002 Sep;283(3):F540-8. Citation on PubMedFont MA, Feliubadal L, Estivill X, Nunes V, Golomb E, Kreiss Y, Pras E, Bisceglia L, d'Adamo AP, Zelante L, Gasparini P, Bassi MT, George AL Jr, Manzoni M, Riboni M, Ballabio A, Borsani G, Reig N, Fernndez E, Zorzano A, Bertran J, Palacn M; International Cystinuria Consortium. Functional analysis of mutations in SLC7A9, and genotype-phenotype correlation in non-Type I cystinuria. Hum Mol Genet. 2001 Feb 15;10(4):305-16. Citation on PubMedFont-Llitjs M, Jimnez-Vidal M, Bisceglia L, Di Perna M, de Sanctis L, Rousaud F, Zelante L, Palacn M, Nunes V. New insights into cystinuria: 40 new mutations, genotype-phenotype correlation, and digenic inheritance causing partial phenotype. J Med Genet. 2005 Jan;42(1):58-68. Citation on PubMed or Free article on PubMed CentralGoodyer P. The molecular basis of cystinuria. Nephron Exp Nephrol. 2004;98(2):e45-9. Review. Citation on PubMedLangman CB. The molecular basis of kidney stones. Curr Opin Pediatr. 2004 Apr;16(2):188-93. Review. Citation on PubMedOMIM: SOLUTE CARRIER FAMILY 7 (CATIONIC AMINO ACID TRANSPORTER, y+ SYSTEM), MEMBER 9Schmidt C, Vester U, Wagner CA, Lahme S, Hesse A, Hoyer P, Lang F, Zerres K, Eggermann T; Arbeitsgemeinschaft fr Pdiatrische Nephrologie. Significant contribution of genomic rearrangements in SLC3A1 and SLC7A9 to the etiology of cystinuria. Kidney Int. 2003 Nov;64(5):1564-72. Citation on PubMed \n",
            "ZNF302\t\n",
            "GRAMD1A\t\n",
            "FAM187B\t\n",
            "LSR\t\n",
            "KMT2B\t   From NCBI Gene:This gene encodes a protein which contains multiple domains including a CXXC zinc finger, three PHD zinc fingers, two FY-rich domains, and a SET (suppressor of variegation, enhancer of zeste, and trithorax) domain. The SET domain is a conserved C-terminal domain that characterizes proteins of the MLL (mixed-lineage leukemia) family. This gene is ubiquitously expressed in adult tissues. It is also amplified in solid tumor cell lines, and may be involved in human cancer. Two alternatively spliced transcript variants encoding distinct isoforms have been reported for this gene, however, the full length nature of the shorter transcript is not known. [provided by RefSeq, Jul 2008]From UniProt:Histone methyltransferase. Methylates 'Lys-4' of histone H3. H3 'Lys-4' methylation represents a specific tag for epigenetic transcriptional activation. Plays a central role in beta-globin locus transcription regulation by being recruited by NFE2. Plays an important role in controlling bulk H3K4me during oocyte growth and preimplantation development. Required during the transcriptionally active period of oocyte growth for the establishment and/or maintenance of bulk H3K4 trimethylation (H3K4me3), global transcriptional silencing that preceeds resumption of meiosis, oocyte survival and normal zygotic genome activation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Dystonia 28, childhood-onsetFrom UniProt:Dystonia 28, childhood-onset (DYT28): A form of dystonia, a disorder defined by the presence of sustained involuntary muscle contraction, often leading to abnormal postures. DYT28 is an autosomal dominant, progressive form characterized by onset in the first decade of life and variable severity. Dystonia begins focally in the lower limbs, resulting in gait difficulties, with later progression to other body regions, including the upper limbs, neck, and orofacial region. [MIM:617284]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19q13.12, which is the long (q) arm of chromosome 19 at position 13.12Molecular Location: base pairs 35,717,818 to 35,738,880 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CXXC10DYT28HRX2MLL1BMLL2MLL4TRX2WBP-7WBP7     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KMT2B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DYSTONIA 28, CHILDHOOD-ONSETOMIM: LYSINE-SPECIFIC METHYLTRANSFERASE 2B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KMT2B  OMIM: DYSTONIA 28, CHILDHOOD-ONSETOMIM: LYSINE-SPECIFIC METHYLTRANSFERASE 2B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LIN37\t\n",
            "NPHS1\t   The NPHS1 gene provides instructions for making a protein called nephrin. Nephrin is primarily found in the kidneys, which are organs that filter waste products from the blood and remove them in urine. Specifically, nephrin is found in cells called podocytes, which are located in specialized kidney structures called glomeruli. Nephrin is located at the cell surface in the area between two podocytes called the slit diaphragm. The slit diaphragm is known as a filtration barrier because it captures proteins in blood so that they remain in the body while allowing other molecules like sugars and salts to be excreted in urine. Nephrin proteins on one cell interact with nephrin proteins on adjacent podocytes, forming a zipper-like structure. This structure allows the passage of small molecules through the slit diaphragm while preventing larger molecules like proteins from passing through. Nephrin proteins are essential for forming the slit diaphragm, anchoring the slit diaphragm to podocytes, and filtering blood.Nephrin is also involved in cell signaling. It relays signals from outside the cell to inside the cell. Additionally, nephrin proteins on the surface of adjacent cells send and receive signals, allowing podocytes to communicate with one another.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Congenital nephrotic syndrome     At least 250 mutations in the NPHS1 gene have been found to cause congenital nephrotic syndrome. This condition is a kidney disorder that begins in infancy and typically leads to irreversible kidney failure (end-stage renal disease) by early childhood.NPHS1 gene mutations account for all cases of congenital nephrotic syndrome of the Finnish type. This form of the condition is found in people of Finnish ancestry. Two specific mutations, both of which result in an abnormally short, nonfunctional nephrin protein, account for nearly all cases. The first mutation, known as Finnmajor, is written as L41fsX90 and is responsible for 78 percent of cases. The second mutation, known as Finnminor, is written as R1109X and is responsible for 16 percent of cases.NPHS1 gene mutations can also cause congenital nephrotic syndrome in non-Finnish individuals. Most of these mutations result in an abnormal nephrin protein that is trapped inside the cell and cannot get to the podocyte cell surface. A shortage of functional nephrin at the podocyte cell surface impairs the formation of normal slit diaphragms. Without a functional slit diaphragm, molecules pass through the kidneys abnormally and are excreted in urine. The filtering ability of the kidneys worsens from birth, eventually leading to end-stage renal disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 250 mutations in the NPHS1 gene have been found to cause congenital nephrotic syndrome. This condition is a kidney disorder that begins in infancy and typically leads to irreversible kidney failure (end-stage renal disease) by early childhood.NPHS1 gene mutations account for all cases of congenital nephrotic syndrome of the Finnish type. This form of the condition is found in people of Finnish ancestry. Two specific mutations, both of which result in an abnormally short, nonfunctional nephrin protein, account for nearly all cases. The first mutation, known as Finnmajor, is written as L41fsX90 and is responsible for 78 percent of cases. The second mutation, known as Finnminor, is written as R1109X and is responsible for 16 percent of cases.NPHS1 gene mutations can also cause congenital nephrotic syndrome in non-Finnish individuals. Most of these mutations result in an abnormal nephrin protein that is trapped inside the cell and cannot get to the podocyte cell surface. A shortage of functional nephrin at the podocyte cell surface impairs the formation of normal slit diaphragms. Without a functional slit diaphragm, molecules pass through the kidneys abnormally and are excreted in urine. The filtering ability of the kidneys worsens from birth, eventually leading to end-stage renal disease. More About This Health Condition    Cytogenetic Location: 19q13.12, which is the long (q) arm of chromosome 19 at position 13.12Molecular Location: base pairs 35,825,372 to 35,851,993 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CNFnephrinnephrin precursornephrosis 1, congenital, Finnish type (nephrin)NPHNNPHS1 nephrinrenal glomerulus-specific cell adhesion receptor     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     National Institute of Diabetes and Digestive and Kidney Diseases: The Kidneys and How They Work      Tests Listed in the Genetic Testing Registry (1 link)     Tests of NPHS1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     NEPHRIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      National Institute of Diabetes and Digestive and Kidney Diseases: The Kidneys and How They Work  Tests of NPHS1  PubMed  NEPHRIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Fogo AB, Lusco MA, Najafian B, Alpers CE. AJKD Atlas of Renal Pathology: Congenital Nephrotic Syndrome of Finnish Type. Am J Kidney Dis. 2015 Sep;66(3):e11-2. doi: 10.1053/j.ajkd.2015.07.008. Citation on PubMedGrahammer F, Schell C, Huber TB. The podocyte slit diaphragm--from a thin grey line to a complex signalling hub. Nat Rev Nephrol. 2013 Oct;9(10):587-98. doi: 10.1038/nrneph.2013.169. Epub 2013 Sep 3. Review. Citation on PubMedOMIM: NEPHRINOvunc B, Ashraf S, Vega-Warner V, Bockenhauer D, Elshakhs NA, Joseph M, Hildebrandt F; Gesellschaft fr Pdiatrische Nephrologie (GPN) Study Group. Mutation analysis of NPHS1 in a worldwide cohort of congenital nephrotic syndrome patients. Nephron Clin Pract. 2012;120(3):c139-46. doi: 10.1159/000337379. Epub 2012 May 11. Citation on PubMedSchoeb DS, Chernin G, Heeringa SF, Matejas V, Held S, Vega-Warner V, Bockenhauer D, Vlangos CN, Moorani KN, Neuhaus TJ, Kari JA, MacDonald J, Saisawat P, Ashraf S, Ovunc B, Zenker M, Hildebrandt F; Gesselschaft fr Paediatrische Nephrologie (GPN) Study Group. Nineteen novel NPHS1 mutations in a worldwide cohort of patients with congenital nephrotic syndrome (CNS). Nephrol Dial Transplant. 2010 Sep;25(9):2970-6. doi: 10.1093/ndt/gfq088. Epub 2010 Feb 18. Citation on PubMed or Free article on PubMed CentralWang JJ, Mao JH. The etiology of congenital nephrotic syndrome: current status and challenges. World J Pediatr. 2016 May;12(2):149-58. doi: 10.1007/s12519-016-0009-y. Epub 2016 Mar 9. Citation on PubMed \n",
            "CAPNS1\t\n",
            "ZNF461\t\n",
            "TJP3\t\n",
            "ZNF568\t\n",
            "ZNF607\t\n",
            "SPRED3\t\n",
            "ECH1\t\n",
            "CCER2\t\n",
            "PLEKHG2\t   From NCBI Gene:The protein encoded by this gene is a RhoGTPase that can activate CDC42 by promoting exchange of GDP for GTP on CDC42. The encoded protein is activated by binding to the beta and gamma subunits of heterotrimeric guanine nucleotide-binding protein. Defects in this gene have been associated with leukodystrophy and acquired microcephaly with or without dystonia. [provided by RefSeq, May 2017]From UniProt:May be a transforming oncogene with exchange activity for CDC42 (By similarity). May be a guanine-nucleotide exchange factor (GEF) for RAC1 and CDC42. Activated by the binding to subunits beta and gamma of the heterotrimeric guanine nucleotide-binding protein (G protein) (PubMed:18045877). Involved in the regulation of actin polymerization (PubMed:26573021).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Leukodystrophy and acquired microcephaly with or without dystoniaFrom UniProt:Leukodystrophy and acquired microcephaly with or without dystonia (LDAMD): An autosomal recessive neurologic disorder characterized by profound mental retardation, dystonia, postnatal microcephaly, and white matter abnormalities consistent with leukodystrophy. [MIM:616763]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19q13.2, which is the long (q) arm of chromosome 19 at position 13.2Molecular Location: base pairs 39,412,582 to 39,428,415 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARHGEF42CLGCTB-60E11.4LDAMD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PLEKHG2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LEUKODYSTROPHY AND ACQUIRED MICROCEPHALY WITH OR WITHOUT DYSTONIAOMIM: PLECKSTRIN HOMOLOGY DOMAIN- AND RhoGEF DOMAIN-CONTAINING PROTEIN G2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PLEKHG2  OMIM: LEUKODYSTROPHY AND ACQUIRED MICROCEPHALY WITH OR WITHOUT DYSTONIAOMIM: PLECKSTRIN HOMOLOGY DOMAIN- AND RhoGEF DOMAIN-CONTAINING PROTEIN G2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PRX\t   From NCBI Gene:This gene encodes a protein involved in peripheral nerve myelin upkeep. The encoded protein contains 2 PDZ domains which were named after PSD95 (post synaptic density protein), DlgA (Drosophila disc large tumor suppressor), and ZO1 (a mammalian tight junction protein). Two alternatively spliced transcript variants have been described for this gene which encode different protein isoforms and which are targeted differently in the Schwann cell. Mutations in this gene cause Charcot-Marie-Tooth neuoropathy, type 4F and Dejerine-Sottas neuropathy. [provided by RefSeq, Jul 2008]From UniProt:Scaffolding protein that functions as part of a dystroglycan complex in Schwann cells, and as part of EZR and AHNAK-containing complexes in eye lens fiber cells. Required for the maintenance of the peripheral myelin sheath that is essential for normal transmission of nerve impulses and normal perception of sensory stimuli. Required for normal transport of MBP mRNA from the perinuclear to the paranodal regions. Required for normal remyelination after nerve injury. Required for normal elongation of Schwann cells and normal length of the internodes between the nodes of Ranvier. The demyelinated nodes of Ranvier permit saltatory transmission of nerve impulses; shorter internodes cause slower transmission of nerve impulses. Required for the formation of appositions between the abaxonal surface of the myelin sheath and the Schwann cell plasma membrane; the Schwann cell cytoplasm is restricted to regions between these appositions. Required for the formation of Cajal bands and of Schmidt-Lanterman incisures that correspond to short, cytoplasm-filled regions on myelinated nerves. Recruits DRP2 to the Schwann cell plasma membrane. Required for normal protein composition of the eye lens fiber cell plasma membrane and normal eye lens fiber cell morphology.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Charcot-Marie-Tooth diseaseFrom NCBI Gene:Dejerine-Sottas diseaseCharcot-Marie-Tooth disease, demyelinating, type 4fFrom UniProt:Dejerine-Sottas syndrome (DSS): A severe degenerating neuropathy of the demyelinating Charcot-Marie-Tooth disease category, with onset by age 2 years. Characterized by motor and sensory neuropathy with very slow nerve conduction velocities, increased cerebrospinal fluid protein concentrations, hypertrophic nerve changes, delayed age of walking as well as areflexia. There are both autosomal dominant and autosomal recessive forms of Dejerine-Sottas syndrome. [MIM:145900]Charcot-Marie-Tooth disease 4F (CMT4F): A recessive demyelinating form of Charcot-Marie-Tooth disease, a disorder of the peripheral nervous system, characterized by progressive weakness and atrophy, initially of the peroneal muscles and later of the distal muscles of the arms. Charcot-Marie-Tooth disease is classified in two main groups on the basis of electrophysiologic properties and histopathology: primary peripheral demyelinating neuropathies (designated CMT1 when they are dominantly inherited) and primary peripheral axonal neuropathies (CMT2). Demyelinating neuropathies are characterized by severely reduced nerve conduction velocities (less than 38 m/sec), segmental demyelination and remyelination with onion bulb formations on nerve biopsy, slowly progressive distal muscle atrophy and weakness, absent deep tendon reflexes, and hollow feet. By convention autosomal recessive forms of demyelinating Charcot-Marie-Tooth disease are designated CMT4. CMT4F is characterized by distal sensory impairment and distal muscle weakness and atrophy affecting the lower more than the upper limbs. The age at onset is variable and can range from childhood to adult years. When the onset is in infancy, the phenotype is characterized as Dejerine-Sottas syndrome. [MIM:614895]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19q13.2, which is the long (q) arm of chromosome 19 at position 13.2Molecular Location: base pairs 40,393,762 to 40,414,718 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMT4F     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PRX      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CHARCOT-MARIE-TOOTH DISEASE, DEMYELINATING, TYPE 4FOMIM: HYPERTROPHIC NEUROPATHY OF DEJERINE-SOTTASOMIM: PERIAXIN      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PRX  OMIM: CHARCOT-MARIE-TOOTH DISEASE, DEMYELINATING, TYPE 4FOMIM: HYPERTROPHIC NEUROPATHY OF DEJERINE-SOTTASOMIM: PERIAXIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LTBP4\t   From NCBI Gene:The protein encoded by this gene binds transforming growth factor beta (TGFB) as it is secreted and targeted to the extracellular matrix. TGFB is biologically latent after secretion and insertion into the extracellular matrix, and sheds TGFB and other proteins upon activation. Defects in this gene may be a cause of cutis laxa and severe pulmonary, gastrointestinal, and urinary abnormalities. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, May 2010]From UniProt:Key regulator of transforming growth factor beta (TGFB1, TGFB2 and TGFB3) that controls TGF-beta activation by maintaining it in a latent state during storage in extracellular space. Associates specifically via disulfide bonds with the Latency-associated peptide (LAP), which is the regulatory chain of TGF-beta, and regulates integrin-dependent activation of TGF-beta.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Cutis laxaFrom NCBI Gene:Cutis laxa with severe pulmonary, gastrointestinal, and urinary abnormalitiesFrom UniProt:Duchenne muscular dystrophy (DMD): Most common form of muscular dystrophy; a sex-linked recessive disorder. It typically presents in boys aged 3 to 7 year as proximal muscle weakness causing waddling gait, toe-walking, lordosis, frequent falls, and difficulty in standing up and climbing up stairs. The pelvic girdle is affected first, then the shoulder girdle. Progression is steady and most patients are confined to a wheelchair by age of 10 or 12. Flexion contractures and scoliosis ultimately occur. About 50% of patients have a lower IQ than their genetic expectations would suggest. There is no treatment. [MIM:310200]Urban-Rifkin-Davis syndrome (URDS): A syndrome characterized by disrupted pulmonary, gastrointestinal, urinary, musculoskeletal, craniofacial and dermal development. Clinical features include cutis laxa, mild cardiovascular lesions, respiratory distress with cystic and atelectatic changes in the lungs, and diverticulosis, tortuosity and stenosis at various levels of the intestinal tract. Craniofacial features include microretrognathia, flat midface, receding forehead and wide fontanelles. [MIM:613177]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19q13.2, which is the long (q) arm of chromosome 19 at position 13.2Molecular Location: base pairs 40,592,888 to 40,629,820 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARCL1CLTBP-4LTBP4LLTBP4S     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LTBP4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CUTIS LAXA, AUTOSOMAL RECESSIVE, TYPE ICOMIM: LATENT TRANSFORMING GROWTH FACTOR-BETA-BINDING PROTEIN 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LTBP4  OMIM: CUTIS LAXA, AUTOSOMAL RECESSIVE, TYPE ICOMIM: LATENT TRANSFORMING GROWTH FACTOR-BETA-BINDING PROTEIN 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NUMBL\t\n",
            "CYP2A7\t\n",
            "CYP2F1\t\n",
            "BCKDHA\t   The BCKDHA gene provides instructions for making one part, the alpha subunit, of a group of enzymes called the branched-chain alpha-keto acid dehydrogenase (BCKD) enzyme complex. Two alpha subunits connect with two beta subunits, which are produced from the BCKDHB gene, to form a critical piece of the enzyme complex called the E1 component.The BCKD enzyme complex is responsible for one step in the normal breakdown of three protein building blocks (amino acids). These amino acidsleucine, isoleucine, and valineare obtained from the diet. They are present in many kinds of food, particularly protein-rich foods such as milk, meat, and eggs. The BCKD enzyme complex is active in mitochondria, which are specialized structures inside cells that serve as energy-producing centers. The breakdown of leucine, isoleucine, and valine produces molecules that can be used for energy.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Maple syrup urine disease     More than 80 mutations in the BCKDHA gene have been identified in people with maple syrup urine disease. These mutations most often cause the severe, classic form of the disorder, which becomes apparent soon after birth. Maple syrup urine disease gets its name from the distinctive sweet odor of affected infants' urine. It is also characterized by poor feeding, vomiting, lack of energy (lethargy), abnormal movements, and delayed development.Most BCKDHA mutations change single amino acids in the alpha subunit of the BCKD enzyme complex. In the Old Order Mennonite population, where maple syrup urine disease occurs frequently, the most common mutation replaces the amino acid tyrosine with the amino acid asparagine at position 438 (written as Tyr438Asn or Y438N).Mutations in the BCKDHA gene disrupt the normal function of the BCKD enzyme complex, preventing it from effectively breaking down leucine, isoleucine, and valine. As a result, these amino acids and their byproducts build up in the body. This accumulation is toxic to cells and tissues, particularly in the nervous system. The buildup of these substances can lead to seizures, developmental delay, and the other health problems associated with maple syrup urine disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 80 mutations in the BCKDHA gene have been identified in people with maple syrup urine disease. These mutations most often cause the severe, classic form of the disorder, which becomes apparent soon after birth. Maple syrup urine disease gets its name from the distinctive sweet odor of affected infants' urine. It is also characterized by poor feeding, vomiting, lack of energy (lethargy), abnormal movements, and delayed development.Most BCKDHA mutations change single amino acids in the alpha subunit of the BCKD enzyme complex. In the Old Order Mennonite population, where maple syrup urine disease occurs frequently, the most common mutation replaces the amino acid tyrosine with the amino acid asparagine at position 438 (written as Tyr438Asn or Y438N).Mutations in the BCKDHA gene disrupt the normal function of the BCKD enzyme complex, preventing it from effectively breaking down leucine, isoleucine, and valine. As a result, these amino acids and their byproducts build up in the body. This accumulation is toxic to cells and tissues, particularly in the nervous system. The buildup of these substances can lead to seizures, developmental delay, and the other health problems associated with maple syrup urine disease. More About This Health Condition    Cytogenetic Location: 19q13.2, which is the long (q) arm of chromosome 19 at position 13.2Molecular Location: base pairs 41,397,789 to 41,425,005 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BCKDE1ABCKDH E1-alphabranched chain keto acid dehydrogenase E1, alpha polypeptide (maple syrup urine disease)MSUD1ODBA_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Basic Neurochemistry (sixth edition, 1998): Major pathways of branched-chain amino acid metabolism (figure)Basic Neurochemistry (sixth edition, 1998): Maple syrup urine disease was the first congenital defect of branched-chain amino acid catabolism to be described      Clinical Information from GeneReviews (1 link)     Maple Syrup Urine Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of BCKDHA      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     BRANCHED-CHAIN KETO ACID DEHYDROGENASE E1, ALPHA POLYPEPTIDE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1998): Major pathways of branched-chain amino acid metabolism (figure)Basic Neurochemistry (sixth edition, 1998): Maple syrup urine disease was the first congenital defect of branched-chain amino acid catabolism to be described  Maple Syrup Urine Disease  Tests of BCKDHA  PubMed  BRANCHED-CHAIN KETO ACID DEHYDROGENASE E1, ALPHA POLYPEPTIDE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Flaschker N, Feyen O, Fend S, Simon E, Schadewaldt P, Wendel U. Description of the mutations in 15 subjects with variant forms of maple syrup urine disease. J Inherit Metab Dis. 2007 Nov;30(6):903-9. Epub 2007 Oct 8. Citation on PubMedHenneke M, Flaschker N, Helbling C, Mller M, Schadewaldt P, Grtner J, Wendel U. Identification of twelve novel mutations in patients with classic and variant forms of maple syrup urine disease. Hum Mutat. 2003 Nov;22(5):417. Citation on PubMedPuffenberger EG. Genetic heritage of the Old Order Mennonites of southeastern Pennsylvania. Am J Med Genet C Semin Med Genet. 2003 Aug 15;121C(1):18-31. Citation on PubMedQuental S, Macedo-Ribeiro S, Matos R, Vilarinho L, Martins E, Teles EL, Rodrigues E, Diogo L, Garcia P, Eusbio F, Gaspar A, Sequeira S, Furtado F, Lana I, Amorim A, Prata MJ. Molecular and structural analyses of maple syrup urine disease and identification of a founder mutation in a Portuguese Gypsy community. Mol Genet Metab. 2008 Jun;94(2):148-56. doi: 10.1016/j.ymgme.2008.02.008. Epub 2008 Apr 2. Citation on PubMedRodrguez-Pombo P, Navarrete R, Merinero B, Gmez-Puertas P, Ugarte M. Mutational spectrum of maple syrup urine disease in Spain. Hum Mutat. 2006 Jul;27(7):715. Citation on PubMedStrauss KA, Puffenberger EG, Morton DH. Maple Syrup Urine Disease. 2006 Jan 30 [updated 2013 May 9]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1319/ Citation on PubMed \n",
            "CEACAM21\t\n",
            "CEACAM5\t\n",
            "TMIGD2\t\n",
            "PSG6\t\n",
            "PSG2\t\n",
            "CD177\t\n",
            "PHLDB3\t\n",
            "ZNF45\t\n",
            "ZNF221\t\n",
            "ZNF284\t\n",
            "ZNF229\t\n",
            "CEACAM20\t\n",
            "PLIN4\t\n",
            "PLIN5\t\n",
            "KLC3\t\n",
            "CD3EAP\t\n",
            "FBXO46\t\n",
            "BHMG1\t\n",
            "DMPK\t   The DMPK gene provides instructions for making a protein called myotonic dystrophy protein kinase. Although the specific function of this protein is unknown, it appears to play an important role in muscle, heart, and brain cells. This protein may be involved in communication within cells. It also appears to regulate the production and function of important structures inside muscle cells by interacting with other proteins. For example, myotonic dystrophy protein kinase has been shown to turn off (inhibit) part of a muscle protein called myosin phosphatase. Myosin phosphatase is an enzyme that plays a role in muscle tensing (contraction) and relaxation.One region of the DMPK gene contains a segment of three DNA building blocks (nucleotides) that is repeated multiple times. This sequence, which is written as CTG, is called a triplet or trinucleotide repeat. In most people, the number of CTG repeats in this gene ranges from 5 to 34.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Myotonic dystrophy     Type 1 myotonic dystrophy results from a mutation in the DMPK gene known as a trinucleotide repeat expansion. This mutation increases the size of the repeated CTG segment in the DMPK gene. People with type 1 myotonic dystrophy have from 50 to 5,000 CTG repeats in most cells. The number of repeats may be even greater in certain types of cells, such as muscle cells.The mutated DMPK gene produces an altered version of messenger RNA, which is a molecular blueprint of the gene that is normally used to guide the production of proteins. Researchers have found that the altered messenger RNA traps proteins to form clumps within the cell. The clumps interfere with the production of many other proteins. These changes prevent muscle cells and cells in other tissues from functioning properly, leading to muscle weakness and the other features of type 1 myotonic dystrophy.The size of the trinucleotide repeat expansion is associated with the severity of signs and symptoms. People with the classic features of type 1 myotonic dystrophy, including muscle weakness and wasting beginning in adulthood, usually have between 100 and 1,000 CTG repeats. People born with the more severe congenital form of type 1 myotonic dystrophy tend to have a larger number of CTG repeats, often more than 2,000. This form of the condition is apparent in infancy and may involve life-threatening health problems.As the altered DMPK gene is passed from one generation to the next, the size of the CTG repeat expansion often increases in size. People with 35 to 49 CTG repeats have not been reported to develop type 1 myotonic dystrophy, but their children are at risk of having the disorder if the number of CTG repeats increases. Repeat lengths from 35 to 49 are called premutations. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Type 1 myotonic dystrophy results from a mutation in the DMPK gene known as a trinucleotide repeat expansion. This mutation increases the size of the repeated CTG segment in the DMPK gene. People with type 1 myotonic dystrophy have from 50 to 5,000 CTG repeats in most cells. The number of repeats may be even greater in certain types of cells, such as muscle cells.The mutated DMPK gene produces an altered version of messenger RNA, which is a molecular blueprint of the gene that is normally used to guide the production of proteins. Researchers have found that the altered messenger RNA traps proteins to form clumps within the cell. The clumps interfere with the production of many other proteins. These changes prevent muscle cells and cells in other tissues from functioning properly, leading to muscle weakness and the other features of type 1 myotonic dystrophy.The size of the trinucleotide repeat expansion is associated with the severity of signs and symptoms. People with the classic features of type 1 myotonic dystrophy, including muscle weakness and wasting beginning in adulthood, usually have between 100 and 1,000 CTG repeats. People born with the more severe congenital form of type 1 myotonic dystrophy tend to have a larger number of CTG repeats, often more than 2,000. This form of the condition is apparent in infancy and may involve life-threatening health problems.As the altered DMPK gene is passed from one generation to the next, the size of the CTG repeat expansion often increases in size. People with 35 to 49 CTG repeats have not been reported to develop type 1 myotonic dystrophy, but their children are at risk of having the disorder if the number of CTG repeats increases. Repeat lengths from 35 to 49 are called premutations. More About This Health Condition    Cytogenetic Location: 19q13.32, which is the long (q) arm of chromosome 19 at position 13.32Molecular Location: base pairs 45,769,709 to 45,782,557 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DM-kinaseDM-PKDM protein kinaseDM1DM1PKDm15DMKDMPK_HUMANdystrophia myotonica 1dystrophia myotonica kinase, B15dystrophia myotonica protein kinasedystrophia myotonica-protein kinaseMDPKMT-PKmyotonic dystrophy protein kinasemyotoninmyotonin-protein kinase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database: DM1 Mutation Is an Expansion of CTG Trinucleotide Repeats      Clinical Information from GeneReviews (1 link)     Myotonic Dystrophy Type 1      Tests Listed in the Genetic Testing Registry (1 link)     Tests of DMPK      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     DYSTROPHIA MYOTONICA PROTEIN KINASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: DM1 Mutation Is an Expansion of CTG Trinucleotide Repeats  Myotonic Dystrophy Type 1  Tests of DMPK  PubMed  DYSTROPHIA MYOTONICA PROTEIN KINASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bird TD. Myotonic Dystrophy Type 1. 1999 Sep 17 [updated 2015 Oct 22]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1165/ Citation on PubMedBotta A, Rinaldi F, Catalli C, Vergani L, Bonifazi E, Romeo V, Loro E, Viola A, Angelini C, Novelli G. The CTG repeat expansion size correlates with the splicing defects observed in muscles from myotonic dystrophy type 1 patients. J Med Genet. 2008 Oct;45(10):639-46. doi: 10.1136/jmg.2008.058909. Epub 2008 Jul 8. Citation on PubMedCho DH, Tapscott SJ. Myotonic dystrophy: emerging mechanisms for DM1 and DM2. Biochim Biophys Acta. 2007 Feb;1772(2):195-204. Epub 2006 Jun 20. Review. Citation on PubMedDay JW, Ranum LP. RNA pathogenesis of the myotonic dystrophies. Neuromuscul Disord. 2005 Jan;15(1):5-16. Epub 2004 Nov 26. Review. Citation on PubMedEbralidze A, Wang Y, Petkova V, Ebralidse K, Junghans RP. RNA leaching of transcription factors disrupts transcription in myotonic dystrophy. Science. 2004 Jan 16;303(5656):383-7. Epub 2003 Dec 4. Citation on PubMedFardaei M, Rogers MT, Thorpe HM, Larkin K, Hamshere MG, Harper PS, Brook JD. Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells. Hum Mol Genet. 2002 Apr 1;11(7):805-14. Citation on PubMedKaliman P, Llagostera E. Myotonic dystrophy protein kinase (DMPK) and its role in the pathogenesis of myotonic dystrophy 1. Cell Signal. 2008 Nov;20(11):1935-41. doi: 10.1016/j.cellsig.2008.05.005. Epub 2008 May 18. Review. Citation on PubMedMachuca-Tzili L, Brook D, Hilton-Jones D. Clinical and molecular aspects of the myotonic dystrophies: a review. Muscle Nerve. 2005 Jul;32(1):1-18. Review. Citation on PubMedRanum LP, Day JW. Myotonic dystrophy: RNA pathogenesis comes into focus. Am J Hum Genet. 2004 May;74(5):793-804. Epub 2004 Apr 2. Review. Citation on PubMed or Free article on PubMed CentralSalvatori S, Fanin M, Trevisan CP, Furlan S, Reddy S, Nagy JI, Angelini C. Decreased expression of DMPK: correlation with CTG repeat expansion and fibre type composition in myotonic dystrophy type 1. Neurol Sci. 2005 Oct;26(4):235-42. Citation on PubMedUeda H, Ohno S, Kobayashi T. Myotonic dystrophy and myotonic dystrophy protein kinase. Prog Histochem Cytochem. 2000;35(3):187-251. Review. Citation on PubMed \n",
            "DMWD\t\n",
            "IGFL1\t\n",
            "ELSPBP1\t\n",
            "LIG1\t\n",
            "CCDC114\t   From NCBI Gene:This gene encodes a coiled-coil domain-containing protein that is a component of the outer dynein arm docking complex in cilia cells. Mutations in this gene may cause primary ciliary dyskinesia 20. [provided by RefSeq, May 2013]From UniProt:Probable component of the outer dynein arm complex required along the entire axoneme for tethering of outer dynein arms.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary ciliary dyskinesiaFrom NCBI Gene:Ciliary dyskinesia, primary, 20From UniProt:Ciliary dyskinesia, primary, 20 (CILD20): A disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia. Patients may exhibit randomization of left-right body asymmetry and situs inversus, due to dysfunction of monocilia at the embryonic node. Primary ciliary dyskinesia associated with situs inversus is referred to as Kartagener syndrome. Unlike other forms of CILD characterized by reduced fertility, patients with CILD20 do not appear to be infertile. [MIM:615067]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19q13.33, which is the long (q) arm of chromosome 19 at position 13.33Molecular Location: base pairs 48,296,452 to 48,322,793 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CILD20     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CCDC114      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 20OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 114: CCDC114      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CCDC114  OMIM: CILIARY DYSKINESIA, PRIMARY, 20OMIM: COILED-COIL DOMAIN-CONTAINING PROTEIN 114: CCDC114  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FUT2\t   From NCBI Gene:The protein encoded by this gene is a Golgi stack membrane protein that is involved in the creation of a precursor of the H antigen, which is required for the final step in the soluble A and B antigen synthesis pathway. This gene is one of two encoding the galactoside 2-L-fucosyltransferase enzyme. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Mediates the transfer of fucose to the terminal galactose on glycan chains of cell surface glycoproteins and glycolipids (PubMed:7876235). The resulting epitope plays a role in cell-cell interaction including host-microbe interaction (PubMed:12692541, PubMed:8018146). Mediates interaction with intestinal microbiota influencing its composition (PubMed:21625510, PubMed:24733310, PubMed:22068912). Creates a soluble precursor oligosaccharide FuC-alpha ((1,2)Galbeta-) called the H antigen which is an essential substrate for the final step in the soluble ABO blood group antigen synthesis pathway (PubMed:7876235).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Bombay phenotypeVitamin b12 plasma level quantitative trait locus 1     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19q13.33, which is the long (q) arm of chromosome 19 at position 13.33Molecular Location: base pairs 48,695,971 to 48,705,934 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        B12QTL1SESe2SEC2sej     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FUT2      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BOMBAY PHENOTYPEOMIM: FUCOSYLTRANSFERASE 2OMIM: VITAMIN B12 PLASMA LEVEL QUANTITATIVE TRAIT LOCUS 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FUT2  OMIM: BOMBAY PHENOTYPEOMIM: FUCOSYLTRANSFERASE 2OMIM: VITAMIN B12 PLASMA LEVEL QUANTITATIVE TRAIT LOCUS 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RASIP1\t\n",
            "DHDH\t\n",
            "UHRF1\t   From NCBI Gene:This gene encodes a member of a subfamily of RING-finger type E3 ubiquitin ligases. The protein binds to specific DNA sequences, and recruits a histone deacetylase to regulate gene expression. Its expression peaks at late G1 phase and continues during G2 and M phases of the cell cycle. It plays a major role in the G1/S transition by regulating topoisomerase IIalpha and retinoblastoma gene expression, and functions in the p53-dependent DNA damage checkpoint. It is regarded as a hub protein for the integration of epigenetic information. This gene is up-regulated in various cancers, and it is therefore considered to be a therapeutic target. Multiple transcript variants encoding different isoforms have been found for this gene. A related pseudogene exists on chromosome 12. [provided by RefSeq, Feb 2014]From UniProt:Multidomain protein that acts as a key epigenetic regulator by bridging DNA methylation and chromatin modification. Specifically recognizes and binds hemimethylated DNA at replication forks via its YDG domain and recruits DNMT1 methyltransferase to ensure faithful propagation of the DNA methylation patterns through DNA replication. In addition to its role in maintenance of DNA methylation, also plays a key role in chromatin modification: through its tudor-like regions and PHD-type zinc fingers, specifically recognizes and binds histone H3 trimethylated at 'Lys-9' (H3K9me3) and unmethylated at 'Arg-2' (H3R2me0), respectively, and recruits chromatin proteins. Enriched in pericentric heterochromatin where it recruits different chromatin modifiers required for this chromatin replication. Also localizes to euchromatic regions where it negatively regulates transcription possibly by impacting DNA methylation and histone modifications. Has E3 ubiquitin-protein ligase activity by mediating the ubiquitination of target proteins such as histone H3 and PML. It is still unclear how E3 ubiquitin-protein ligase activity is related to its role in chromatin in vivo. May be involved in DNA repair.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Defects in UHRF1 may be a cause of cancers. Overexpressed in many different forms of human cancers, including bladder, breast, cervical, colorectal and prostate cancers, as well as pancreatic adenocarcinomas, rhabdomyosarcomas and gliomas. Plays an important role in the correlation of histone modification and gene silencing in cancer progression. Expression is associated with a poor prognosis in patients with various cancers, suggesting that it participates in cancer progression.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19p13.3, which is the short (p) arm of chromosome 19 at position 13.3Molecular Location: base pairs 4,903,080 to 4,962,154 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        hNP95hUHRF1huNp95ICBP90Np95RNF106TDRD22     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of UHRF1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: UBIQUITIN-LIKE PROTEIN CONTAINING PHD AND RING FINGER DOMAINS 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of UHRF1  OMIM: UBIQUITIN-LIKE PROTEIN CONTAINING PHD AND RING FINGER DOMAINS 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CGB7\t\n",
            "HRC\t\n",
            "PTH2\t\n",
            "SCAF1\t\n",
            "MADCAM1\t\n",
            "NR1H2\t\n",
            "KDM4B\t\n",
            "KLK7\t\n",
            "KLK12\t\n",
            "VSIG10L\t\n",
            "SIGLEC12\t\n",
            "SIGLEC6\t\n",
            "ZNF480\t\n",
            "ZNF880\t\n",
            "ZNF534\t\n",
            "ZNF28\t\n",
            "ZNF816\t\n",
            "ZNF415\t\n",
            "VN1R4\t\n",
            "ZNF761\t\n",
            "CACNG7\t\n",
            "ZNRF4\t\n",
            "NLRP2\t\n",
            "GP6\t   The GP6 gene provides instructions for making a protein called glycoprotein VI (GPVI). GPVI is a receptor protein that is embedded in the outer membrane of blood cell fragments called platelets, which are an essential component of blood clots. Normally, blood clots protect the body after an injury by sealing off damaged blood vessels and preventing further blood loss.Receptor proteins, like GPVI, have specific sites into which certain other proteins, called ligands, fit like keys into locks. Together, ligands and their receptors trigger signals that affect cell function. The main ligand for GPVI is a protein called collagen, which is found on blood vessel walls. In response to an injury that causes bleeding, the GPVI protein attaches (binds) to collagen, which begins clot formation and signals additional platelets to come together to increase the size of the clot. The GPVI protein can also bind to a protein called fibrin, which is the main protein that forms blood clots.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Glycoprotein VI deficiency     At least five mutations in the GP6 gene have been found to cause glycoprotein VI deficiency, a bleeding disorder associated with a decreased ability to form blood clots. The mutations lead to the production of no GPVI protein; an abnormally short, nonfunctional GPVI protein; or a protein that is less able to bind to collagen. Without GPVI binding to collagen, platelets cannot come together efficiently at the site of an injury to form a clot, leading to an increased risk of nosebleeds, abnormally heavy or prolonged bleeding following minor injury or surgery, or other bleeding problems associated with glycoprotein VI deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least five mutations in the GP6 gene have been found to cause glycoprotein VI deficiency, a bleeding disorder associated with a decreased ability to form blood clots. The mutations lead to the production of no GPVI protein; an abnormally short, nonfunctional GPVI protein; or a protein that is less able to bind to collagen. Without GPVI binding to collagen, platelets cannot come together efficiently at the site of an injury to form a clot, leading to an increased risk of nosebleeds, abnormally heavy or prolonged bleeding following minor injury or surgery, or other bleeding problems associated with glycoprotein VI deficiency. More About This Health Condition    Cytogenetic Location: 19q13.42, which is the long (q) arm of chromosome 19 at position 13.42Molecular Location: base pairs 55,013,705 to 55,038,264 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        glycoprotein 6GPVIGPVI collagen receptorplatelet collagen receptorplatelet glycoprotein VIplatelet membrane glycoprotein VI     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Platelet-Vessel Wall Interactions in Hemostasis and Thrombosis (2010): Platelet Adhesion Molecules      Tests Listed in the Genetic Testing Registry (1 link)     Tests of GP6      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     GLYCOPROTEIN VI, PLATELET      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Platelet-Vessel Wall Interactions in Hemostasis and Thrombosis (2010): Platelet Adhesion Molecules  Tests of GP6  PubMed  GLYCOPROTEIN VI, PLATELET  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Arthur JF, Dunkley S, Andrews RK. Platelet glycoprotein VI-related clinical defects. Br J Haematol. 2007 Nov;139(3):363-72. Review. Citation on PubMedDumont B, Lasne D, Rothschild C, Bouabdelli M, Ollivier V, Oudin C, Ajzenberg N, Grandchamp B, Jandrot-Perrus M. Absence of collagen-induced platelet activation caused by compound heterozygous GPVI mutations. Blood. 2009 Aug 27;114(9):1900-3. doi: 10.1182/blood-2009-03-213504. Citation on PubMedOMIM: GLYCOPROTEIN VI, PLATELETHermans C, Wittevrongel C, Thys C, Smethurst PA, Van Geet C, Freson K. A compound heterozygous mutation in glycoprotein VI in a patient with a bleeding disorder. J Thromb Haemost. 2009 Aug;7(8):1356-63. doi: 10.1111/j.1538-7836.2009.03520.x. Citation on PubMedMatus V, Valenzuela G, Sez CG, Hidalgo P, Lagos M, Aranda E, Panes O, Pereira J, Pillois X, Nurden AT, Mezzano D. An adenine insertion in exon 6 of human GP6 generates a truncated protein associated with a bleeding disorder in four Chilean families. J Thromb Haemost. 2013 Sep;11(9):1751-9. doi: 10.1111/jth.12334. Citation on PubMed \n",
            "EPS8L1\t\n",
            "HSPBP1\t\n",
            "ZNF579\t\n",
            "NLRP8\t\n",
            "NLRP5\t\n",
            "ZNF787\t\n",
            "GALP\t\n",
            "ZSCAN5A\t\n",
            "USP29\t\n",
            "ZIM3\t\n",
            "AURKC\t   The AURKC gene provides instructions for making a protein called aurora kinase C. This protein regulates certain events during cell division. Cell division occurs when a cell replicates and splits its contents, separating into two new cells. Aurora kinase C helps the two dividing cells separate from each other and ensures that these cells each contain a complete set of chromosomes. Aurora kinase C is most abundant in male testes, which are the male reproductive organs in which sperm are produced and stored. In the testes, this protein regulates the division of sperm cells, ensuring that every new sperm cell divides properly and contains one copy of each chromosome.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Macrozoospermia     At least four mutations in the AURKC gene have been found to cause macrozoospermia. This condition causes males to have abnormal sperm, which leads to an inability to father biological children (infertility). AURKC gene mutations that cause macrozoospermia lead to the production of a nonfunctional protein or a protein that is quickly broken down. One mutation that is a frequent cause of this condition in men of North African descent deletes a single DNA building block (nucleotide) from the AURKC gene (written 144delC). The protein produced from the altered gene is abnormally short and nonfunctional.A lack of aurora kinase C blocks cell division in sperm cells. As a result, sperm cells have extra chromosomes, usually four copies of each instead of the usual one. This increase in chromosome number enlarges the sperm cell head and leads to the presence of multiple tails (flagella). Because of the additional genetic material, if one of these abnormal sperm cells combines with an egg cell, the embryo will not develop or the pregnancy will result in miscarriage. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least four mutations in the AURKC gene have been found to cause macrozoospermia. This condition causes males to have abnormal sperm, which leads to an inability to father biological children (infertility). AURKC gene mutations that cause macrozoospermia lead to the production of a nonfunctional protein or a protein that is quickly broken down. One mutation that is a frequent cause of this condition in men of North African descent deletes a single DNA building block (nucleotide) from the AURKC gene (written 144delC). The protein produced from the altered gene is abnormally short and nonfunctional.A lack of aurora kinase C blocks cell division in sperm cells. As a result, sperm cells have extra chromosomes, usually four copies of each instead of the usual one. This increase in chromosome number enlarges the sperm cell head and leads to the presence of multiple tails (flagella). Because of the additional genetic material, if one of these abnormal sperm cells combines with an egg cell, the embryo will not develop or the pregnancy will result in miscarriage. More About This Health Condition    Cytogenetic Location: 19q13.43, which is the long (q) arm of chromosome 19 at position 13.43Molecular Location: base pairs 57,230,317 to 57,235,550 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AIK3AIK3 protein kinaseARK-3ARK3AurCaurora-related kinase 3aurora/IPL1-like kinase 3aurora/IPL1-related kinase 3serine-threonine-protein kinase 13serine/threonine-protein kinase aurora-CSTK13     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Developmental Biology (sixth edition, 2000): SpermatogenesisMadame Curie Bioscience Database: Cytogenetics of Human SpermMolecular Biology of the Cell (fourth edition, 2002): Sperm      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AURKC      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     AURORA KINASE C      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): SpermatogenesisMadame Curie Bioscience Database: Cytogenetics of Human SpermMolecular Biology of the Cell (fourth edition, 2002): Sperm  Tests of AURKC  PubMed  AURORA KINASE C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: AURORA KINASE CBen Khelifa M, Coutton C, Blum MG, Abada F, Harbuz R, Zouari R, Guichet A, May-Panloup P, Mitchell V, Rollet J, Triki C, Merdassi G, Vialard F, Koscinski I, Viville S, Keskes L, Soulie JP, Rives N, Dorphin B, Lestrade F, Hesters L, Poirot C, Benzacken B, Jouk PS, Satre V, Hennebicq S, Arnoult C, Lunardi J, Ray PF. Identification of a new recurrent aurora kinase C mutation in both European and African men with macrozoospermia. Hum Reprod. 2012 Nov;27(11):3337-46. doi: 10.1093/humrep/des296. Epub 2012 Aug 11. Citation on PubMedBen Khelifa M, Zouari R, Harbuz R, Halouani L, Arnoult C, Lunardi J, Ray PF. A new AURKC mutation causing macrozoospermia: implications for human spermatogenesis and clinical diagnosis. Mol Hum Reprod. 2011 Dec;17(12):762-8. doi: 10.1093/molehr/gar050. Epub 2011 Jul 6. Citation on PubMed or Free article on PubMed CentralDieterich K, Soto Rifo R, Faure AK, Hennebicq S, Ben Amar B, Zahi M, Perrin J, Martinez D, Sle B, Jouk PS, Ohlmann T, Rousseaux S, Lunardi J, Ray PF. Homozygous mutation of AURKC yields large-headed polyploid spermatozoa and causes male infertility. Nat Genet. 2007 May;39(5):661-5. Epub 2007 Apr 15. Citation on PubMedDieterich K, Zouari R, Harbuz R, Vialard F, Martinez D, Bellayou H, Prisant N, Zoghmar A, Guichaoua MR, Koscinski I, Kharouf M, Noruzinia M, Nadifi S, Sefiani A, Lornage J, Zahi M, Viville S, Sle B, Jouk PS, Jacob MC, Escalier D, Nikas Y, Hennebicq S, Lunardi J, Ray PF. The Aurora Kinase C c.144delC mutation causes meiosis I arrest in men and is frequent in the North African population. Hum Mol Genet. 2009 Apr 1;18(7):1301-9. doi: 10.1093/hmg/ddp029. Epub 2009 Jan 15. Citation on PubMed \n",
            "ZNF419\t\n",
            "ZNF211\t\n",
            "FUT6\t   From NCBI Gene:The protein encoded by this gene is a Golgi stack membrane protein that is involved in the creation of sialyl-Lewis X, an E-selectin ligand. Mutations in this gene are a cause of fucosyltransferase-6 deficiency. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Enzyme involved in the biosynthesis of the E-Selectin ligand, sialyl-Lewis X. Catalyzes the transfer of fucose from GDP-beta-fucose to alpha-2,3 sialylated substrates.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Fucosyltransferase 6 deficiency     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 19p13.3, which is the short (p) arm of chromosome 19 at position 13.3Molecular Location: base pairs 5,830,626 to 5,840,019 on chromosome 19 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FCT3AFT1AFuc-TVIFucT-VI     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FUT6      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FUCOSYLTRANSFERASE 6OMIM: FUCOSYLTRANSFERASE 6 DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FUT6  OMIM: FUCOSYLTRANSFERASE 6OMIM: FUCOSYLTRANSFERASE 6 DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF814\t\n",
            "FUT5\t\n",
            "ZNF274\t\n",
            "POLRMT\t\n",
            "ADGRE1\t\n",
            "MBD3L3\t\n",
            "SNAPC2\t\n",
            "FBN3\t\n",
            "NDUFA7\t\n",
            "KANK3\t\n",
            "ACTL9\t\n",
            "MUC16\t\n",
            "AGL\t   The AGL gene provides instructions for making the glycogen debranching enzyme. This enzyme is involved in the breakdown of a complex sugar called glycogen, which is a major source of stored energy in the body. Glycogen is made up of several molecules of a simple sugar called glucose. Some glucose molecules are linked together in a straight line, while others branch off and form side chains. The glycogen debranching enzyme is involved in the breakdown of these side chains. The branched structure of glycogen makes it more compact for storage and allows it to break down more easily when it is needed for fuel.The AGL gene provides instructions for making several different versions (isoforms) of the glycogen debranching enzyme. These isoforms vary by size and are active (expressed) in different tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Glycogen storage disease type III     Approximately 100 mutations in the AGL gene have been found to cause glycogen storage disease type III (also called GSDIII or Cori disease). Most of these mutations lead to a premature stop signal in the instructions for making the glycogen debranching enzyme, resulting in a nonfunctional enzyme. As a result, the side chains of glycogen molecules cannot be removed and abnormal, partially broken down glycogen molecules are stored within cells. A buildup of abnormal glycogen damages organs and tissues throughout the body, particularly the liver and muscles, leading to the signs and symptoms of GSDIII.Mutations in the AGL gene can affect different isoforms of the enzyme, depending on where the mutations are located in the gene. For example, mutations that occur in a part of the AGL gene called exon 3 affect the isoform that is primarily expressed in the liver. These mutations almost always lead to GSD type IIIb, which is characterized by liver problems. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Approximately 100 mutations in the AGL gene have been found to cause glycogen storage disease type III (also called GSDIII or Cori disease). Most of these mutations lead to a premature stop signal in the instructions for making the glycogen debranching enzyme, resulting in a nonfunctional enzyme. As a result, the side chains of glycogen molecules cannot be removed and abnormal, partially broken down glycogen molecules are stored within cells. A buildup of abnormal glycogen damages organs and tissues throughout the body, particularly the liver and muscles, leading to the signs and symptoms of GSDIII.Mutations in the AGL gene can affect different isoforms of the enzyme, depending on where the mutations are located in the gene. For example, mutations that occur in a part of the AGL gene called exon 3 affect the isoform that is primarily expressed in the liver. These mutations almost always lead to GSD type IIIb, which is characterized by liver problems. More About This Health Condition    Cytogenetic Location: 1p21.2, which is the short (p) arm of chromosome 1 at position 21.2Molecular Location: base pairs 99,850,077 to 99,924,023 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        amylo-1, 6-glucosidase, 4-alpha-glucanotransferaseGDEGDE_HUMANglycogen debrancherglycogen debranching enzyme     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): A Debranching Enzyme Also Is Needed for the Breakdown of GlycogenWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Glycogen Storage Disease Type III      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGL      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     AMYLO-1,6-GLUCOSIDASE, 4-ALPHA-GLUCANOTRANSFERASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): A Debranching Enzyme Also Is Needed for the Breakdown of GlycogenWashington University, St. Louis: Neuromuscular Disease Center  Glycogen Storage Disease Type III  Tests of AGL  PubMed  AMYLO-1,6-GLUCOSIDASE, 4-ALPHA-GLUCANOTRANSFERASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: AMYLO-1,6-GLUCOSIDASE, 4-ALPHA-GLUCANOTRANSFERASECheng A, Zhang M, Okubo M, Omichi K, Saltiel AR. Distinct mutations in the glycogen debranching enzyme found in glycogen storage disease type III lead to impairment in diverse cellular functions. Hum Mol Genet. 2009 Jun 1;18(11):2045-52. doi: 10.1093/hmg/ddp128. Epub 2009 Mar 19. Citation on PubMed or Free article on PubMed CentralEndo Y, Horinishi A, Vorgerd M, Aoyama Y, Ebara T, Murase T, Odawara M, Podskarbi T, Shin YS, Okubo M. Molecular analysis of the AGL gene: heterogeneity of mutations in patients with glycogen storage disease type III from Germany, Canada, Afghanistan, Iran, and Turkey. J Hum Genet. 2006;51(11):958-63. Epub 2006 Sep 19. Citation on PubMedFrisbie JH, O'Connell DJ, Tow DE, Sasahara AA, Belko JS. Autologous radioiodinated fibrinogen, simplified. J Nucl Med. 1975 May;16(5):393-401. Citation on PubMedLucchiari S, Pagliarani S, Salani S, Filocamo M, Di Rocco M, Melis D, Rodolico C, Musumeci O, Toscano A, Bresolin N, Comi GP. Hepatic and neuromuscular forms of glycogenosis type III: nine mutations in AGL. Hum Mutat. 2006 Jun;27(6):600-1. Citation on PubMed \n",
            "COL11A1\t   The COL11A1 gene provides instructions for making a component of type XI collagen called the pro-alpha1(XI) chain. Collagens are molecules that provide structure and strength to the connective tissues that support the body's muscles, joints, organs, and skin. Type XI collagen is normally found in cartilage, a tough but flexible tissue that makes up much of the skeleton during early development. Most cartilage is later converted to bone, except for the cartilage that continues to cover and protect the ends of bones and is present in the nose and external ears. Type XI collagen is also part of the inner ear; the vitreous, which is the clear gel that fills the eyeball; and the nucleus pulposus, which is the center portion of the discs between the bones of the spine (vertebrae).Collagens begin as rope-like procollagen molecules that are each made up of three chains. The pro-alpha1(XI) chain combines with two other collagen chains, pro-alpha2(XI) and pro-alpha1(II), to form a triple-stranded procollagen molecule. Then the ropelike procollagen is processed by enzymes to create mature collagen. Mature collagen molecules arrange themselves into long, thin fibrils that form stable interactions (cross-links) with one another in the spaces between cells (the extracellular matrix). The cross-links result in the formation of very strong type XI collagen fibers.Type XI collagen also helps maintain the spacing and width (diameter) of another type of collagen molecule, type II collagen. Type II collagen is an important component of the vitreous and cartilage. The arrangement and size of type II collagen fibrils is essential for the normal structure of these tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Fibrochondrogenesis     At least seven mutations in the COL11A1 gene have been identified in people with fibrochondrogenesis type 1, a disorder of bone growth characterized by severe skeletal abnormalities, hearing loss, and vision loss. Infants with fibrochondrogenesis type 1 have a very narrow chest that prevents the lungs from developing normally. Most children with this condition are stillborn or die shortly after birth from respiratory failure, although some have lived into childhood.Some cases of fibrochondrogenesis type 1 result from a combination of COL11A1 gene mutations. Specifically, one copy of the gene has a mutation that prevents the production of any functional pro-alpha1(XI) chain, and the other copy has a mutation that results in an abnormal version of the pro-alpha1(XI) chain. When the abnormal chain is incorporated into collagen molecules, it creates defective type XI collagen. The abnormal collagen weakens connective tissues, impairing the formation of bones throughout the skeleton and causing changes in the eye and inner ear that lead to vision and hearing problems.In at least two reported cases, fibrochondrogenesis type 1 has been caused by combinations of COL11A1 gene mutations that completely eliminate the production of the pro-alpha1(XI) chain. Researchers speculate that a loss of this chain changes the structure of type XI collagen molecules and disrupts its ability to form cross-links. However, the effects of these mutations are still under study. More About This Health Condition      Stickler syndrome     Mutations in the COL11A1 gene account for 10 to 20 percent of all cases of Stickler syndrome. When Stickler syndrome results from COL11A1 gene mutations, it is classified as type II. Signs and symptoms of this condition include a distinctive facial appearance, eye abnormalities, hearing loss, and joint problems. These signs and symptoms tend to be less severe than those of fibrochondrogenesis (described above). However, they vary widely among affected individuals.More than two dozen COL11A1 gene mutations have been found in people with Stickler syndrome. Some of these mutations change single protein building blocks (amino acids) or delete a small number of amino acids from the pro-alpha1(XI) chain. Other mutations cause segments of DNA to be skipped when the protein is made, resulting in an abnormally short pro-alpha1(XI) chain. All of these changes impair the production or assembly of type XI collagen molecules. Defective collagen disrupts the normal development of connective tissues in many different parts of the body, which leads to the varied signs and symptoms of Stickler syndrome.Mutations in the COL11A1 gene can also cause Marshall syndrome, a condition that is very similar to Stickler syndrome. Some researchers have classified Marshall syndrome as a variant of Stickler syndrome, while others consider it to be a separate disorder. Most of the mutations associated with Marshall syndrome cause a segment of DNA in the COL11A1 gene to be skipped when the protein is made, resulting in an abnormally short pro-alpha1(XI) chain. This shortened protein impairs the formation of mature type XI collagen, which leads to the abnormal development of connective tissues and the signs and symptoms of Marshall syndrome. More About This Health Condition      Carpal tunnel syndrome     Genetics Home Reference provides information about carpal tunnel syndrome. More About This Health Condition      Intervertebral disc disease     Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition      Osteoarthritis     Genetics Home Reference provides information about osteoarthritis. More About This Health Condition      Other disorders     Several variations in the COL11A1 gene may increase the risk of developing osteoarthritis, a common disease of the joints. Osteoarthritis is characterized by the breakdown of joint cartilage, which causes pain, stiffness, and restricted movement. The genetic changes associated with osteoarthritis are differences in single amino acids in the pro-alpha1(XI) chain of type XI collagen. Studies suggest that the altered pro-alpha1(XI) chain may weaken collagen fibers, which could contribute to the erosion of joint cartilage. Variations in the COL11A1 gene are among many genetic and environmental factors that likely influence this complex disease.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least seven mutations in the COL11A1 gene have been identified in people with fibrochondrogenesis type 1, a disorder of bone growth characterized by severe skeletal abnormalities, hearing loss, and vision loss. Infants with fibrochondrogenesis type 1 have a very narrow chest that prevents the lungs from developing normally. Most children with this condition are stillborn or die shortly after birth from respiratory failure, although some have lived into childhood.Some cases of fibrochondrogenesis type 1 result from a combination of COL11A1 gene mutations. Specifically, one copy of the gene has a mutation that prevents the production of any functional pro-alpha1(XI) chain, and the other copy has a mutation that results in an abnormal version of the pro-alpha1(XI) chain. When the abnormal chain is incorporated into collagen molecules, it creates defective type XI collagen. The abnormal collagen weakens connective tissues, impairing the formation of bones throughout the skeleton and causing changes in the eye and inner ear that lead to vision and hearing problems.In at least two reported cases, fibrochondrogenesis type 1 has been caused by combinations of COL11A1 gene mutations that completely eliminate the production of the pro-alpha1(XI) chain. Researchers speculate that a loss of this chain changes the structure of type XI collagen molecules and disrupts its ability to form cross-links. However, the effects of these mutations are still under study. More About This Health Condition  Mutations in the COL11A1 gene account for 10 to 20 percent of all cases of Stickler syndrome. When Stickler syndrome results from COL11A1 gene mutations, it is classified as type II. Signs and symptoms of this condition include a distinctive facial appearance, eye abnormalities, hearing loss, and joint problems. These signs and symptoms tend to be less severe than those of fibrochondrogenesis (described above). However, they vary widely among affected individuals.More than two dozen COL11A1 gene mutations have been found in people with Stickler syndrome. Some of these mutations change single protein building blocks (amino acids) or delete a small number of amino acids from the pro-alpha1(XI) chain. Other mutations cause segments of DNA to be skipped when the protein is made, resulting in an abnormally short pro-alpha1(XI) chain. All of these changes impair the production or assembly of type XI collagen molecules. Defective collagen disrupts the normal development of connective tissues in many different parts of the body, which leads to the varied signs and symptoms of Stickler syndrome.Mutations in the COL11A1 gene can also cause Marshall syndrome, a condition that is very similar to Stickler syndrome. Some researchers have classified Marshall syndrome as a variant of Stickler syndrome, while others consider it to be a separate disorder. Most of the mutations associated with Marshall syndrome cause a segment of DNA in the COL11A1 gene to be skipped when the protein is made, resulting in an abnormally short pro-alpha1(XI) chain. This shortened protein impairs the formation of mature type XI collagen, which leads to the abnormal development of connective tissues and the signs and symptoms of Marshall syndrome. More About This Health Condition  Genetics Home Reference provides information about carpal tunnel syndrome. More About This Health Condition  Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition  Genetics Home Reference provides information about osteoarthritis. More About This Health Condition  Several variations in the COL11A1 gene may increase the risk of developing osteoarthritis, a common disease of the joints. Osteoarthritis is characterized by the breakdown of joint cartilage, which causes pain, stiffness, and restricted movement. The genetic changes associated with osteoarthritis are differences in single amino acids in the pro-alpha1(XI) chain of type XI collagen. Studies suggest that the altered pro-alpha1(XI) chain may weaken collagen fibers, which could contribute to the erosion of joint cartilage. Variations in the COL11A1 gene are among many genetic and environmental factors that likely influence this complex disease.    Cytogenetic Location: 1p21.1, which is the short (p) arm of chromosome 1 at position 21.1Molecular Location: base pairs 102,876,467 to 103,108,580 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        COBA1_HUMANCOLL6collagen type XI alpha 1collagen XI, alpha-1 polypeptidecollagen, type XI, alpha 1STL2     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Biology of the Cell (fourth edition, 2002): Collagens Are the Major Proteins of the Extracellular MatrixMolecular Biology of the Cell (fourth edition, 2002): Some Types of Collagen and Their PropertiesMolecular Cell Biology (fourth edition, 2000): Collagen: The Fibrous Proteins of the Matrix      Clinical Information from GeneReviews (1 link)     Stickler Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL11A1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     COLLAGEN, TYPE XI, ALPHA-1OSTEOARTHRITIS SUSCEPTIBILITY 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Collagens Are the Major Proteins of the Extracellular MatrixMolecular Biology of the Cell (fourth edition, 2002): Some Types of Collagen and Their PropertiesMolecular Cell Biology (fourth edition, 2000): Collagen: The Fibrous Proteins of the Matrix  Stickler Syndrome  Tests of COL11A1  PubMed  COLLAGEN, TYPE XI, ALPHA-1OSTEOARTHRITIS SUSCEPTIBILITY 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Akawi NA, Al-Gazali L, Ali BR. Clinical and molecular analysis of UAE fibrochondrogenesis patients expands the phenotype and reveals two COL11A1 homozygous null mutations. Clin Genet. 2012 Aug;82(2):147-56. doi: 10.1111/j.1399-0004.2011.01734.x. Epub 2011 Jul 18. Citation on PubMedAkawi NA, Ali BR, Al-Gazali L. A response to Dr. Alzahrani's letter to the editor regarding the mechanism underlying fibrochondrogenesis. Gene. 2013 Oct 10;528(2):367-8. doi: 10.1016/j.gene.2013.07.038. Epub 2013 Jul 29. Citation on PubMedAlzahrani F, Alshammari MJ, Alkuraya FS. Molecular pathogenesis of fibrochondrogenesis: is it really simple COL11A1 deficiency? Gene. 2012 Dec 15;511(2):480-1. doi: 10.1016/j.gene.2012.09.069. Epub 2012 Sep 28. Citation on PubMedAnnunen S, Krkk J, Czarny M, Warman ML, Brunner HG, Kriinen H, Mulliken JB, Tranebjaerg L, Brooks DG, Cox GF, Cruysberg JR, Curtis MA, Davenport SL, Friedrich CA, Kaitila I, Krawczynski MR, Latos-Bielenska A, Mukai S, Olsen BR, Shinno N, Somer M, Vikkula M, Zlotogora J, Prockop DJ, Ala-Kokko L. Splicing mutations of 54-bp exons in the COL11A1 gene cause Marshall syndrome, but other mutations cause overlapping Marshall/Stickler phenotypes. Am J Hum Genet. 1999 Oct;65(4):974-83. Citation on PubMed or Free article on PubMed CentralOMIM: COLLAGEN, TYPE XI, ALPHA-1Majava M, Hoornaert KP, Bartholdi D, Bouma MC, Bouman K, Carrera M, Devriendt K, Hurst J, Kitsos G, Niedrist D, Petersen MB, Shears D, Stolte-Dijkstra I, Van Hagen JM, Ala-Kokko L, Mnnikk M, Mortier GR. A report on 10 new patients with heterozygous mutations in the COL11A1 gene and a review of genotype-phenotype correlations in type XI collagenopathies. Am J Med Genet A. 2007 Feb 1;143A(3):258-64. Citation on PubMedRodriguez-Fontenla C, Calaza M, Evangelou E, Valdes AM, Arden N, Blanco FJ, Carr A, Chapman K, Deloukas P, Doherty M, Esko T, Garcs Alet CM, Gomez-Reino Carnota JJ, Helgadottir H, Hofman A, Jonsdottir I, Kerkhof HJ, Kloppenburg M, McCaskie A, Ntzani EE, Ollier WE, Oreiro N, Panoutsopoulou K, Ralston SH, Ramos YF, Riancho JA, Rivadeneira F, Slagboom PE, Styrkarsdottir U, Thorsteinsdottir U, Thorleifsson G, Tsezou A, Uitterlinden AG, Wallis GA, Wilkinson JM, Zhai G, Zhu Y; arcOGEN Consortium, Felson DT, Ioannidis JP, Loughlin J, Metspalu A, Meulenbelt I, Stefansson K, van Meurs JB, Zeggini E, Spector TD, Gonzalez A. Assessment of osteoarthritis candidate genes in a meta-analysis of nine genome-wide association studies. Arthritis Rheumatol. 2014 Apr;66(4):940-9. doi: 10.1002/art.38300. Citation on PubMed or Free article on PubMed CentralTompson SW, Bacino CA, Safina NP, Bober MB, Proud VK, Funari T, Wangler MF, Nevarez L, Ala-Kokko L, Wilcox WR, Eyre DR, Krakow D, Cohn DH. Fibrochondrogenesis results from mutations in the COL11A1 type XI collagen gene. Am J Hum Genet. 2010 Nov 12;87(5):708-12. doi: 10.1016/j.ajhg.2010.10.009. Epub 2010 Oct 28. Citation on PubMed or Free article on PubMed Central \n",
            "GPSM2\t   From NCBI Gene:The protein encoded by this gene belongs to a family of proteins that modulate activation of G proteins, which transduce extracellular signals received by cell surface receptors into integrated cellular responses. The N-terminal half of this protein contains 10 copies of leu-gly-asn (LGN) repeat, and the C-terminal half contains 4 GoLoco motifs, which are involved in guanine nucleotide exchange. This protein may play a role in neuroblast division and in the development of normal hearing. Mutations in this gene are associated with autosomal recessive nonsyndromic deafness (DFNB82). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Mar 2016]From UniProt:Plays an important role in mitotic spindle pole organization via its interaction with NUMA1 (PubMed:11781568, PubMed:15632202, PubMed:21816348). Required for cortical dynein-dynactin complex recruitment during metaphase (PubMed:22327364). Plays a role in metaphase spindle orientation (PubMed:22327364). Plays also an important role in asymmetric cell divisions (PubMed:21816348). Has guanine nucleotide dissociation inhibitor (GDI) activity towards G(i) alpha proteins, such as GNAI1 and GNAI3, and thereby regulates their activity (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Chudley-McCullough syndromeFrom UniProt:Chudley-McCullough syndrome (CMCS): An autosomal recessive neurologic disorder characterized by early-onset sensorineural deafness and specific brain anomalies on MRI, including hypoplasia of the corpus callosum, enlarged cysterna magna with mild focal cerebellar dysplasia, and nodular heterotopia. Some patients have hydrocephalus. Psychomotor development is normal. [MIM:604213]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p13.3, which is the short (p) arm of chromosome 1 at position 13.3Molecular Location: base pairs 108,875,350 to 108,934,335 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMCSDFNB82LGNPINS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GPSM2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CHUDLEY-MCCULLOUGH SYNDROMEOMIM: G PROTEIN SIGNALING MODULATOR 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GPSM2  OMIM: CHUDLEY-MCCULLOUGH SYNDROMEOMIM: G PROTEIN SIGNALING MODULATOR 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CELSR2\t\n",
            "PSRC1\t\n",
            "MASP2\t   From NCBI Gene:This gene encodes a member of the peptidase S1 family of serine proteases. The encoded preproprotein is proteolytically processed to generate A and B chains that heterodimerize to form the mature protease. This protease cleaves complement components C2 and C4 in order to generate C3 convertase in the lectin pathway of the complement system. The encoded protease also plays a role in the coagulation cascade through cleavage of prothrombin to form thrombin. Myocardial infarction and acute stroke patients exhibit reduced serum concentrations of the encoded protein. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Feb 2016]From UniProt:Serum protease that plays an important role in the activation of the complement system via mannose-binding lectin. After activation by auto-catalytic cleavage it cleaves C2 and C4, leading to their activation and to the formation of C3 convertase.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:MASP2 deficiencyFrom UniProt:MASP2 deficiency (MASPD): A disorder that results in autoimmune manifestations, recurrent severe infections, and chronic inflammatory disease. [MIM:613791]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.22, which is the short (p) arm of chromosome 1 at position 36.22Molecular Location: base pairs 11,026,523 to 11,047,239 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MAP19MASP-2MASP1P1sMAP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MASP2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MANNAN-BINDING LECTIN SERINE PROTEASE 2OMIM: MASP2 DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MASP2  OMIM: MANNAN-BINDING LECTIN SERINE PROTEASE 2OMIM: MASP2 DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DDX20\t\n",
            "PPM1J\t\n",
            "AMPD1\t   The AMPD1 gene provides instructions for producing an enzyme called adenosine monophosphate (AMP) deaminase. This enzyme is found in the muscles used for movement (skeletal muscles), where it plays a role in producing energy. Specifically, during physical activity, this enzyme converts a molecule called adenosine monophosphate (AMP) to a molecule called inosine monophosphate (IMP) as part of a process called the purine nucleotide cycle. This cycle reuses molecules called purines, which are a group of building blocks of DNA (nucleotides), its chemical cousin RNA, and molecules such as AMP that serve as energy sources in the cell. As part of the purine nucleotide cycle, AMP deaminase converts AMP to IMP, and as the cycle continues, molecules are produced that the muscle cells can use for energy. Skeletal muscle cells need energy to function and move the body.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Adenosine monophosphate deaminase deficiency     At least nine mutations in the AMPD1 gene have been found to cause AMP deaminase deficiency. This condition is characterized by skeletal muscle pain or weakness after exercise or prolonged physical activity (exercise intolerance). Most cases are caused by a mutation that results in a premature stop signal in the instructions for making AMP deaminase (written as Gly12Ter or Q12X). The resulting enzyme is abnormally short and nonfunctional and cannot participate in the purine nucleotide cycle. As a result, the process stalls and energy production in skeletal muscle cells is decreased. Skeletal muscles are particularly sensitive to decreases in energy during periods of exercise or increased activity when energy demands increase. The lack of AMP deaminase as a source of energy production can result in fatigue and muscle weakness or pain in some people with AMP deaminase deficiency.It is not known why some people with this condition do not experience symptoms. Researchers speculate that additional factors, both genetic and environmental, may determine whether a person develops the signs and symptoms of AMP deaminase deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least nine mutations in the AMPD1 gene have been found to cause AMP deaminase deficiency. This condition is characterized by skeletal muscle pain or weakness after exercise or prolonged physical activity (exercise intolerance). Most cases are caused by a mutation that results in a premature stop signal in the instructions for making AMP deaminase (written as Gly12Ter or Q12X). The resulting enzyme is abnormally short and nonfunctional and cannot participate in the purine nucleotide cycle. As a result, the process stalls and energy production in skeletal muscle cells is decreased. Skeletal muscles are particularly sensitive to decreases in energy during periods of exercise or increased activity when energy demands increase. The lack of AMP deaminase as a source of energy production can result in fatigue and muscle weakness or pain in some people with AMP deaminase deficiency.It is not known why some people with this condition do not experience symptoms. Researchers speculate that additional factors, both genetic and environmental, may determine whether a person develops the signs and symptoms of AMP deaminase deficiency. More About This Health Condition    Cytogenetic Location: 1p13.2, which is the short (p) arm of chromosome 1 at position 13.2Molecular Location: base pairs 114,673,098 to 114,695,618 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        Adenosine monophosphate deaminase-1 (muscle)AMP deaminaseAMPD1_HUMANMADMADA     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): Generating AMP and GMPBiochemistry (fifth edition, 2002): Structures of ATP, ADP, and AMP      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AMPD1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ADENOSINE MONOPHOSPHATE DEAMINASE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Generating AMP and GMPBiochemistry (fifth edition, 2002): Structures of ATP, ADP, and AMP  Tests of AMPD1  PubMed  ADENOSINE MONOPHOSPHATE DEAMINASE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ADENOSINE MONOPHOSPHATE DEAMINASE 1Hancock CR, Brault JJ, Terjung RL. Protecting the cellular energy state during contractions: role of AMP deaminase. J Physiol Pharmacol. 2006 Nov;57 Suppl 10:17-29. Review. Citation on PubMedIsackson PJ, Bujnicki H, Harding CO, Vladutiu GD. Myoadenylate deaminase deficiency caused by alternative splicing due to a novel intronic mutation in the AMPD1 gene. Mol Genet Metab. 2005 Sep-Oct;86(1-2):250-6. Epub 2005 Jul 22. Citation on PubMedTeijeira S, San Milln B, Fernndez JM, Rivas E, Viitez I, Miranda S, Gonzlez F, Navarro C. Myoadenylate deaminase deficiency: clinico-pathological and molecular study of a series of 27 Spanish cases. Clin Neuropathol. 2009 Mar-Apr;28(2):136-42. Citation on PubMedToyama K, Morisaki H, Kitamura Y, Gross M, Tamura T, Nakahori Y, Vance JM, Speer M, Kamatani N, Morisaki T. Haplotype analysis of human AMPD1 gene: origin of common mutant allele. J Med Genet. 2004 Jun;41(6):e74. Erratum in: J Med Genet. 2005 Mar;42(3):288. Citation on PubMed or Free article on PubMed Central \n",
            "CSDE1\t\n",
            "FBXO2\t\n",
            "IGSF3\t   From NCBI Gene:The protein encoded by this gene is an immunoglobulin-like membrane protein containing several V-type Ig-like domains. A mutation in this gene has been associated with bilateral nasolacrimal duct obstruction (LCDD). [provided by RefSeq, Jun 2016]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Lacrimal duct defectFrom UniProt:Lacrimal duct defect (LCDD): A condition resulting in the imbalance between tear production and tear drainage. Infants typically manifest persistent epiphora and/or recurrent infections of the lacrimal pathway, such as conjunctivitis. LCDD is caused by failure of the nasolacrimal duct to open into the inferior meatus. [MIM:149700]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p13.1, which is the short (p) arm of chromosome 1 at position 13.1Molecular Location: base pairs 116,574,398 to 116,667,755 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        EWI-3LCDDV8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of IGSF3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: IMMUNOGLOBULIN SUPERFAMILY, MEMBER 3OMIM: LACRIMAL DUCT DEFECT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of IGSF3  OMIM: IMMUNOGLOBULIN SUPERFAMILY, MEMBER 3OMIM: LACRIMAL DUCT DEFECT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CD101\t\n",
            "SPAG17\t\n",
            "MTHFR\t   The MTHFR gene provides instructions for making an enzyme called methylenetetrahydrofolate reductase. This enzyme plays a role in processing amino acids, the building blocks of proteins. Methylenetetrahydrofolate reductase is important for a chemical reaction involving forms of the vitamin folate (also called vitamin B9). Specifically, this enzyme converts a molecule called 5,10-methylenetetrahydrofolate to a molecule called 5-methyltetrahydrofolate. This reaction is required for the multistep process that converts the amino acid homocysteine to another amino acid, methionine. The body uses methionine to make proteins and other important compounds.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Homocystinuria     At least 40 mutations in the MTHFR gene have been identified in people with homocystinuria, a disorder in which the body is unable to process homocysteine and methionine properly. People with this condition often develop eye problems, abnormal blood clotting, skeletal abnormalities, and cognitive problems. Most of the mutations that cause homocystinuria change single amino acids in methylenetetrahydrofolate reductase. These changes impair the function of the enzyme, and some cause the enzyme to be turned off (inactivated). Other mutations lead to the production of an abnormally small, nonfunctional version of the enzyme. Without functional methylenetetrahydrofolate reductase, homocysteine cannot be converted to methionine. As a result, homocysteine builds up in the bloodstream, and the amount of methionine is reduced. Some of the excess homocysteine is excreted in urine (homocystinuria). Researchers have not determined how altered levels of homocysteine and methionine lead to the various health problems affecting multiple parts of the body in people with homocystinuria. More About This Health Condition      Age-related hearing loss     Genetics Home Reference provides information about age-related hearing loss. More About This Health Condition      Alopecia areata     Genetics Home Reference provides information about alopecia areata. More About This Health Condition      Anencephaly     Several variations (polymorphisms) in the MTHFR gene have been associated with an increased risk of neural tube defects, a group of birth defects that occur during the development of the brain and spinal cord. Anencephaly is one of the most common types of neural tube defect. Affected individuals are missing large parts of the brain and have missing or incompletely formed skull bones.The most well-studied polymorphism related to neural tube defects changes a single DNA building block (nucleotide) in the MTHFR gene. Specifically, it replaces the nucleotide cytosine with the nucleotide thymine at position 677 (written as 677C>T). This common variant results in a form of methylenetetrahydrofolate reductase that has reduced activity at higher temperatures (thermolabile). People with the 677C>T polymorphism, particularly those with two copies of the genetic change, have elevated levels of homocysteine in their blood resulting from the reduced activity of methylenetetrahydrofolate reductase.Researchers have studied MTHFR gene polymorphisms in individuals with neural tube defects and in their mothers, but it remains unclear how these variations could affect the developing brain and spinal cord. The increased risk of neural tube defects may be related to differences in the ability of methylenetetrahydrofolate reductase to process folate; a shortage of this vitamin is an established risk factor for neural tube defects.Although MTHFR gene polymorphisms are associated with an increased risk of neural tube defects, these variations are common in many populations worldwide. Most people with MTHFR gene polymorphisms do not have neural tube defects, and their children are also typically unaffected. Changes in the MTHFR gene are only one of many genetic and environmental factors that are thought to contribute to these complex conditions. More About This Health Condition      Spina bifida     Polymorphisms in the MTHFR gene are also associated with an increased risk of spina bifida, another common type of neural tube defect. In people with this condition, when the spine forms, the bones of the spinal column do not close completely around the developing nerves of the spinal cord. As a result, part of the spinal cord may protrude through an opening in the spine, leading to permanent nerve damage.As described above, variations in the MTHFR gene may increase the risk of neural tube defects by changing the ability of methylenetetrahydrofolate reductase to process folate. However, these variations are common in many populations worldwide. Most people with MTHFR gene polymorphisms do not have neural tube defects, nor do their children. More About This Health Condition      Other disorders     Polymorphisms in the MTHFR gene have also been studied as possible risk factors for a variety of common conditions. These include heart disease, stroke, high blood pressure (hypertension), high blood pressure during pregnancy (preeclampsia), an eye disorder called glaucoma, psychiatric disorders, and certain types of cancer. Research indicates that individuals who have the 677C>T polymorphism on both copies of the MTHFR gene have an increased risk of developing vascular disease, including heart disease and stroke. The 677C>T polymorphism has also been suggested as a risk factor for cleft lip and palate, a birth defect in which there is a split in the upper lip and an opening in the roof of the mouth. Many of the MTHFR gene polymorphisms alter or decrease the activity of methylenetetrahydrofolate reductase, leading to an increase of homocysteine in the blood. This increase in homocysteine levels may contribute to the development of many of these conditions.Studies of MTHFR gene variations in people with these disorders have had mixed results, with associations found in some studies but not in others. Therefore, it remains unclear what role changes in the MTHFR gene play in these disorders. It is likely that additional factors influence the processing of homocysteine and that variations in homocysteine levels play a role in whether a person develops any of these conditions. A large number of genetic and environmental factors, most of which remain unknown, likely determine the risk of developing most common, complex conditions.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 40 mutations in the MTHFR gene have been identified in people with homocystinuria, a disorder in which the body is unable to process homocysteine and methionine properly. People with this condition often develop eye problems, abnormal blood clotting, skeletal abnormalities, and cognitive problems. Most of the mutations that cause homocystinuria change single amino acids in methylenetetrahydrofolate reductase. These changes impair the function of the enzyme, and some cause the enzyme to be turned off (inactivated). Other mutations lead to the production of an abnormally small, nonfunctional version of the enzyme. Without functional methylenetetrahydrofolate reductase, homocysteine cannot be converted to methionine. As a result, homocysteine builds up in the bloodstream, and the amount of methionine is reduced. Some of the excess homocysteine is excreted in urine (homocystinuria). Researchers have not determined how altered levels of homocysteine and methionine lead to the various health problems affecting multiple parts of the body in people with homocystinuria. More About This Health Condition  Genetics Home Reference provides information about age-related hearing loss. More About This Health Condition  Genetics Home Reference provides information about alopecia areata. More About This Health Condition  Several variations (polymorphisms) in the MTHFR gene have been associated with an increased risk of neural tube defects, a group of birth defects that occur during the development of the brain and spinal cord. Anencephaly is one of the most common types of neural tube defect. Affected individuals are missing large parts of the brain and have missing or incompletely formed skull bones.The most well-studied polymorphism related to neural tube defects changes a single DNA building block (nucleotide) in the MTHFR gene. Specifically, it replaces the nucleotide cytosine with the nucleotide thymine at position 677 (written as 677C>T). This common variant results in a form of methylenetetrahydrofolate reductase that has reduced activity at higher temperatures (thermolabile). People with the 677C>T polymorphism, particularly those with two copies of the genetic change, have elevated levels of homocysteine in their blood resulting from the reduced activity of methylenetetrahydrofolate reductase.Researchers have studied MTHFR gene polymorphisms in individuals with neural tube defects and in their mothers, but it remains unclear how these variations could affect the developing brain and spinal cord. The increased risk of neural tube defects may be related to differences in the ability of methylenetetrahydrofolate reductase to process folate; a shortage of this vitamin is an established risk factor for neural tube defects.Although MTHFR gene polymorphisms are associated with an increased risk of neural tube defects, these variations are common in many populations worldwide. Most people with MTHFR gene polymorphisms do not have neural tube defects, and their children are also typically unaffected. Changes in the MTHFR gene are only one of many genetic and environmental factors that are thought to contribute to these complex conditions. More About This Health Condition  Polymorphisms in the MTHFR gene are also associated with an increased risk of spina bifida, another common type of neural tube defect. In people with this condition, when the spine forms, the bones of the spinal column do not close completely around the developing nerves of the spinal cord. As a result, part of the spinal cord may protrude through an opening in the spine, leading to permanent nerve damage.As described above, variations in the MTHFR gene may increase the risk of neural tube defects by changing the ability of methylenetetrahydrofolate reductase to process folate. However, these variations are common in many populations worldwide. Most people with MTHFR gene polymorphisms do not have neural tube defects, nor do their children. More About This Health Condition  Polymorphisms in the MTHFR gene have also been studied as possible risk factors for a variety of common conditions. These include heart disease, stroke, high blood pressure (hypertension), high blood pressure during pregnancy (preeclampsia), an eye disorder called glaucoma, psychiatric disorders, and certain types of cancer. Research indicates that individuals who have the 677C>T polymorphism on both copies of the MTHFR gene have an increased risk of developing vascular disease, including heart disease and stroke. The 677C>T polymorphism has also been suggested as a risk factor for cleft lip and palate, a birth defect in which there is a split in the upper lip and an opening in the roof of the mouth. Many of the MTHFR gene polymorphisms alter or decrease the activity of methylenetetrahydrofolate reductase, leading to an increase of homocysteine in the blood. This increase in homocysteine levels may contribute to the development of many of these conditions.Studies of MTHFR gene variations in people with these disorders have had mixed results, with associations found in some studies but not in others. Therefore, it remains unclear what role changes in the MTHFR gene play in these disorders. It is likely that additional factors influence the processing of homocysteine and that variations in homocysteine levels play a role in whether a person develops any of these conditions. A large number of genetic and environmental factors, most of which remain unknown, likely determine the risk of developing most common, complex conditions.    Cytogenetic Location: 1p36.22, which is the short (p) arm of chromosome 1 at position 36.22Molecular Location: base pairs 11,785,730 to 11,806,103 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        5,10-methylenetetrahydrofolate reductase5,10-methylenetetrahydrofolate reductase (NADPH)methylenetetrahydrofolate reductase (NAD(P)H)MTHR_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Madame Curie Bioscience Database: Molecular Biology of Methylenetetrahydrofolate Reductase (MTHFR) and Overview of Mutations/PolymorphismsMedical Genetics Summaries from the National Center for Biotechnology Information: Methylenetetrahydrofolate Reductase Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MTHFR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     5,10-METHYLENETETRAHYDROFOLATE REDUCTASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Molecular Biology of Methylenetetrahydrofolate Reductase (MTHFR) and Overview of Mutations/PolymorphismsMedical Genetics Summaries from the National Center for Biotechnology Information: Methylenetetrahydrofolate Reductase Deficiency  Tests of MTHFR  PubMed  5,10-METHYLENETETRAHYDROFOLATE REDUCTASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bhargava S, Ali A, Parakh R, Saxena R, Srivastava LM. Higher incidence of C677T polymorphism of the MTHFR gene in North Indian patients with vascular disease. Vascular. 2012 Apr;20(2):88-95. doi: 10.1258/vasc.2011.oa0320. Epub 2012 Feb 28. Citation on PubMedBotto LD, Yang Q. 5,10-Methylenetetrahydrofolate reductase gene variants and congenital anomalies: a HuGE review. Am J Epidemiol. 2000 May 1;151(9):862-77. Review. Citation on PubMedKhandanpour N, Willis G, Meyer FJ, Armon MP, Loke YK, Wright AJ, Finglas PM, Jennings BA. Peripheral arterial disease and methylenetetrahydrofolate reductase (MTHFR) C677T mutations: A case-control study and meta-analysis. J Vasc Surg. 2009 Mar;49(3):711-8. doi: 10.1016/j.jvs.2008.10.004. Epub 2009 Jan 21. Review. Citation on PubMedKumar A, Kumar P, Prasad M, Sagar R, Yadav AK, Pandit AK, Jali VP, Pathak A. Association of C677T polymorphism in the methylenetetrahydrofolate reductase gene (MTHFR gene) with ischemic stroke: a meta-analysis. Neurol Res. 2015 Jul;37(7):568-77. doi: 10.1179/1743132815Y.0000000008. Epub 2015 Jan 16. Citation on PubMedMoll S, Varga EA. Homocysteine and MTHFR Mutations. Circulation. 2015 Jul 7;132(1):e6-9. doi: 10.1161/CIRCULATIONAHA.114.013311. Review. Citation on PubMedSibani S, Christensen B, O'Ferrall E, Saadi I, Hiou-Tim F, Rosenblatt DS, Rozen R. Characterization of six novel mutations in the methylenetetrahydrofolate reductase (MTHFR) gene in patients with homocystinuria. Hum Mutat. 2000;15(3):280-7. Citation on PubMedTrabetti E. Homocysteine, MTHFR gene polymorphisms, and cardio-cerebrovascular risk. J Appl Genet. 2008;49(3):267-82. doi: 10.1007/BF03195624. Review. Citation on PubMedTrimmer EE. Methylenetetrahydrofolate reductase: biochemical characterization and medical significance. Curr Pharm Des. 2013;19(14):2574-93. Review. Citation on PubMedUrreizti R, Moya-Garca AA, Pino-ngeles A, Cozar M, Langkilde A, Fanhoe U, Esteves C, Arribas J, Vilaseca MA, Prez-Dueas B, Pineda M, Gonzlez V, Artuch R, Baldellou A, Vilarinho L, Fowler B, Ribes A, Snchez-Jimnez F, Grinberg D, Balcells S. Molecular characterization of five patients with homocystinuria due to severe methylenetetrahydrofolate reductase deficiency. Clin Genet. 2010 Nov;78(5):441-8. doi: 10.1111/j.1399-0004.2010.01391.x. Citation on PubMedXie SZ, Liu ZZ, Yu JH, Liu L, Wang W, Xie DL, Qin JB. Association between the MTHFR C677T polymorphism and risk of cancer: evidence from 446 case-control studies. Tumour Biol. 2015 Nov;36(11):8953-72. doi: 10.1007/s13277-015-3648-z. Epub 2015 Jun 17. Citation on PubMedYadav U, Kumar P, Yadav SK, Mishra OP, Rai V. \"Polymorphisms in folate metabolism genes as maternal risk factor for neural tube defects: an updated meta-analysis\". Metab Brain Dis. 2015 Feb;30(1):7-24. doi: 10.1007/s11011-014-9575-7. Epub 2014 Jul 9. Review. Citation on PubMedYan L, Zhao L, Long Y, Zou P, Ji G, Gu A, Zhao P. Association of the maternal MTHFR C677T polymorphism with susceptibility to neural tube defects in offsprings: evidence from 25 case-control studies. PLoS One. 2012;7(10):e41689. doi: 10.1371/journal.pone.0041689. Epub 2012 Oct 3. Citation on PubMed or Free article on PubMed CentralZhang T, Lou J, Zhong R, Wu J, Zou L, Sun Y, Lu X, Liu L, Miao X, Xiong G. Genetic variants in the folate pathway and the risk of neural tube defects: a meta-analysis of the published literature. PLoS One. 2013 Apr 4;8(4):e59570. doi: 10.1371/journal.pone.0059570. Print 2013. Citation on PubMed or Free article on PubMed Central \n",
            "SRGAP2C\t\n",
            "AADACL3\t\n",
            "PRAMEF1\t\n",
            "PRAMEF11\t\n",
            "HNRNPCL1\t\n",
            "PRAMEF2\t\n",
            "PRAMEF7\t\n",
            "PRAMEF5\t\n",
            "FAM72D\t\n",
            "NBPF9\t\n",
            "PDE4DIP\t   From NCBI Gene:The protein encoded by this gene serves to anchor phosphodiesterase 4D to the Golgi/centrosome region of the cell. Defects in this gene may be a cause of myeloproliferative disorder (MBD) associated with eosinophilia. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2010]From UniProt:Functions as an anchor sequestering components of the cAMP-dependent pathway to Golgi and/or centrosomes (By similarity).Isoform 13: Participates in microtubule dynamics, promoting microtubule assembly. Depending upon the cell context, may act at the level of the Golgi apparatus or that of the centrosome (PubMed:25217626, PubMed:27666745, PubMed:28814570, PubMed:29162697). In complex with AKAP9, recruits CAMSAP2 to the Golgi apparatus and tethers non-centrosomal minus-end microtubules to the Golgi, an important step for polarized cell movement (PubMed:27666745, PubMed:28814570). In complex with AKAP9, EB1/MAPRE1 and CDK5RAP2, contributes to microtubules nucleation and extension from the centrosome to the cell periphery, a crucial process for directed cell migration, mitotic spindle orientation and cell-cycle progression (PubMed:29162697).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving PDE4DIP may be the cause of a myeloproliferative disorder (MBD) associated with eosinophilia. Translocation t(1;5)(q23;q33) that forms a PDE4DIP-PDGFRB fusion protein.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q21.2, which is the long (q) arm of chromosome 1 at position 21.2Molecular Location: base pairs 148,808,181 to 149,051,277 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMYA2MMGL     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PDE4DIP      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: PHOSPHODIESTERASE 4D-INTERACTING PROTEIN      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PDE4DIP  OMIM: PHOSPHODIESTERASE 4D-INTERACTING PROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NBPF12\t\n",
            "NBPF11\t\n",
            "NBPF15\t\n",
            "CTSS\t\n",
            "MRPL9\t\n",
            "TCHH\t   The TCHH gene provides instructions for making a protein called trichohyalin. This protein is primarily found in hair follicles, which are specialized structures in the skin where hair growth occurs. Trichohyalin can also be found in the hair strand (shaft). Once trichohyalin is produced, it is modified by other proteins so that it can attach (bind) to other trichohyalin proteins and to molecules called keratin intermediate filaments to create organized cross-links. These links form dense networks that give the hair shaft its cylindrical shape.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Uncombable hair syndrome     At least one mutation in the TCHH gene has been found to cause uncombable hair syndrome. This condition is characterized by dry, frizzy, blond scalp hair that cannot be combed flat. This condition usually improves over time, and by adolescence individuals with uncombable hair syndrome have hair that lies flat and has normal or nearly normal texture.The TCHH gene mutation that has been identified leads to a premature stop signal in the instructions used to make trichohyalin, resulting in an abnormally short protein with severely reduced activity. A shortage (deficiency) of functional protein reduces the cross-links that are formed between trichohyalin proteins and keratin intermediate filaments. As a result, the cross-section of the hair shaft becomes triangular, heart-shaped, or flat. These angular hair shafts result in frizzy hair that will not lie flat, which is typical of uncombable hair syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least one mutation in the TCHH gene has been found to cause uncombable hair syndrome. This condition is characterized by dry, frizzy, blond scalp hair that cannot be combed flat. This condition usually improves over time, and by adolescence individuals with uncombable hair syndrome have hair that lies flat and has normal or nearly normal texture.The TCHH gene mutation that has been identified leads to a premature stop signal in the instructions used to make trichohyalin, resulting in an abnormally short protein with severely reduced activity. A shortage (deficiency) of functional protein reduces the cross-links that are formed between trichohyalin proteins and keratin intermediate filaments. As a result, the cross-section of the hair shaft becomes triangular, heart-shaped, or flat. These angular hair shafts result in frizzy hair that will not lie flat, which is typical of uncombable hair syndrome. More About This Health Condition    Cytogenetic Location: 1q21.3, which is the long (q) arm of chromosome 1 at position 21.3Molecular Location: base pairs 152,106,317 to 152,115,454 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AHFTHHTHLTRHY     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Developmental Biology (sixth edition, 2000): Cutaneous AppendagesMadame Curie Bioscience: Molecular Mechanisms of Embryonic and Adult Hair Development      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TCHH      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRICHOHYALIN      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): Cutaneous AppendagesMadame Curie Bioscience: Molecular Mechanisms of Embryonic and Adult Hair Development  Tests of TCHH  PubMed  TRICHOHYALIN  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Calderon P, Otberg N, Shapiro J. Uncombable hair syndrome. J Am Acad Dermatol. 2009 Sep;61(3):512-5. doi: 10.1016/j.jaad.2009.01.006. Citation on PubMedOMIM: TRICHOHYALINTakase T, Hirai Y. Identification of the C-terminal tail domain of AHF/trichohyalin as the critical site for modulation of the keratin filamentous meshwork in the keratinocyte. J Dermatol Sci. 2012 Feb;65(2):141-8. doi: 10.1016/j.jdermsci.2011.12.014. Citation on PubMed Basmanav FB, Cau L, Tafazzoli A, Mchin MC, Wolf S, Romano MT, Valentin F, Wiegmann H, Huchenq A, Kandil R, Garcia Bartels N, Kilic A, George S, Ralser DJ, Bergner S, Ferguson DJ, Oprisoreanu AM, Wehner M, Thiele H, Altmller J, Nrnberg P, Swan D, Houniet D, Bchner A, Weibel L, Wagner N, Grimalt R, Bygum A, Serre G, Blume-Peytavi U, Sprecher E, Schoch S, Oji V, Hamm H, Farrant P, Simon M, Betz RC. Mutations in Three Genes Encoding Proteins Involved in Hair Shaft Formation Cause Uncombable Hair Syndrome. Am J Hum Genet. 2016 Dec 1;99(6):1292-1304. doi: 10.1016/j.ajhg.2016.10.004. Citation on PubMed or Free article on PubMed Central \n",
            "RPTN\t\n",
            "HRNR\t\n",
            "FLG\t   The FLG gene provides instructions for making a large protein called profilaggrin, which is found in cells that make up the outermost layer of skin (the epidermis). Profilaggrin is cut (cleaved) to produce multiple copies  of the filaggrin protein, which is important for the structure of the epidermis. The profilaggrin molecule can contain 10, 11, or 12 copies of the filaggrin protein, depending on the genetics of the individual. Further processing of the filaggrin protein produces other molecules that play a role in hydration of the skin.The epidermis acts as a barrier to help minimize water loss and protect the body from foreign substances, including toxins, bacteria, and substances that can cause allergic reactions (allergens), such as pollen and dust mites. Filaggrin plays an important role in the skin's barrier function. It brings together structural proteins in the outermost skin cells to form tight bundles, flattening and strengthening the cells to create a strong barrier. In addition, processing of filaggrin proteins leads to production of molecules that are part of the skin's \"natural moisturizing factor,\" which helps maintain hydration of the skin. These molecules also maintain the correct acidity (pH) of the skin, which is another important aspect of the barrier.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alopecia areata     Genetics Home Reference provides information about alopecia areata. More About This Health Condition      Atopic dermatitis     Mutations in the FLG gene are strongly associated with a skin disorder called atopic dermatitis (also known as atopic eczema). This condition is characterized by dry, itchy skin and red rashes. Twenty to 30 percent of people with atopic dermatitis have an FLG gene mutation compared with 8 to 10 percent of the general population without atopic dermatitis. Having a mutation in this gene only increases the risk of developing atopic dermatitis. Not everyone with an FLG gene mutation will develop the disorder. Other genes and environmental factors contribute to atopic dermatitis. Individuals with a mutation in both copies of the FLG gene are at greater risk of developing atopic dermatitis than those with a mutation in only one copy of the gene. The condition is typically more severe in people with two mutated copies.In addition to FLG gene mutations, the number of filaggrin proteins produced from each profilaggrin molecule is associated with development of atopic dermatitis; individuals with genetic instructions for producing 10 copies are at greater risk of developing the disorder than those with instructions for producing 11 or 12 copies.Approximately 40 mutations in the FLG gene have been identified in people with atopic dermatitis. These mutations lead to production of an abnormally short profilaggrin molecule that cannot be cleaved to produce filaggrin proteins. The resulting shortage of filaggrin can impair the barrier function of the skin. Further, a lack of filaggrin leads to a shortage of natural moisturizing factor, which allows excess water to be lost through the skin and causes dry skin in affected individuals.Research suggests that the impairment of the skin's barrier function caused by a shortage of filaggrin contributes to development of allergic disorders. Atopic dermatitis is not initially caused by an allergic reaction, although sometimes substances that can cause allergic reactions (allergens) are thought to contribute to flare-ups of the rashes. In addition, people with atopic dermatitis have an increased likelihood of developing other allergic disorders, including asthma, hay fever, and food allergies. (The tendency to develop allergic disorders is known as atopy.) More About This Health Condition      Other disorders     More than 40 FLG gene mutations have been found to cause a relatively common skin disorder called ichthyosis vulgaris, which is characterized by dry, scaly skin, particularly during the winter when humidity in the air is low. Affected individuals can also have unusually prominent creases on the palms of their hands and soles of their feet (palmar and plantar hyperlinearity) and an increased risk of allergic disorders, including atopic dermatitis (described above). Typically, individuals with a mutation in one copy of the FLG gene are mildly affected; they may have symptoms only seasonally, or they may never have obvious skin problems. Individuals with two mutated copies of the gene have more severe symptoms that may be present year-round.Some of the mutations associated with atopic dermatitis are also found in people with ichthyosis vulgaris. As in atopic dermatitis, the mutations lead to production of an abnormally short profilaggrin molecule that cannot be cleaved to produce filaggrin proteins. A lack of filaggrin and the molecules produced from its processing lead to the dry, scaly skin characteristic of ichthyosis vulgaris. Impairment of the skin's barrier function likely contributes to the development of allergic diseases in some people with ichthyosis vulgaris. Individuals with FLG gene mutations have an increased risk of developing asthma, but only if they also have atopic dermatitis. FLG gene mutations also increase the risk of hay fever, food allergies, and skin sensitivity to nickel, regardless of whether the person has atopic dermatitis. Additional factors, such as living with a cat and smoking tobacco products, may increase the risk of atopic dermatitis and asthma, respectively, in people with ichthyosis vulgaris.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about alopecia areata. More About This Health Condition  Mutations in the FLG gene are strongly associated with a skin disorder called atopic dermatitis (also known as atopic eczema). This condition is characterized by dry, itchy skin and red rashes. Twenty to 30 percent of people with atopic dermatitis have an FLG gene mutation compared with 8 to 10 percent of the general population without atopic dermatitis. Having a mutation in this gene only increases the risk of developing atopic dermatitis. Not everyone with an FLG gene mutation will develop the disorder. Other genes and environmental factors contribute to atopic dermatitis. Individuals with a mutation in both copies of the FLG gene are at greater risk of developing atopic dermatitis than those with a mutation in only one copy of the gene. The condition is typically more severe in people with two mutated copies.In addition to FLG gene mutations, the number of filaggrin proteins produced from each profilaggrin molecule is associated with development of atopic dermatitis; individuals with genetic instructions for producing 10 copies are at greater risk of developing the disorder than those with instructions for producing 11 or 12 copies.Approximately 40 mutations in the FLG gene have been identified in people with atopic dermatitis. These mutations lead to production of an abnormally short profilaggrin molecule that cannot be cleaved to produce filaggrin proteins. The resulting shortage of filaggrin can impair the barrier function of the skin. Further, a lack of filaggrin leads to a shortage of natural moisturizing factor, which allows excess water to be lost through the skin and causes dry skin in affected individuals.Research suggests that the impairment of the skin's barrier function caused by a shortage of filaggrin contributes to development of allergic disorders. Atopic dermatitis is not initially caused by an allergic reaction, although sometimes substances that can cause allergic reactions (allergens) are thought to contribute to flare-ups of the rashes. In addition, people with atopic dermatitis have an increased likelihood of developing other allergic disorders, including asthma, hay fever, and food allergies. (The tendency to develop allergic disorders is known as atopy.) More About This Health Condition  More than 40 FLG gene mutations have been found to cause a relatively common skin disorder called ichthyosis vulgaris, which is characterized by dry, scaly skin, particularly during the winter when humidity in the air is low. Affected individuals can also have unusually prominent creases on the palms of their hands and soles of their feet (palmar and plantar hyperlinearity) and an increased risk of allergic disorders, including atopic dermatitis (described above). Typically, individuals with a mutation in one copy of the FLG gene are mildly affected; they may have symptoms only seasonally, or they may never have obvious skin problems. Individuals with two mutated copies of the gene have more severe symptoms that may be present year-round.Some of the mutations associated with atopic dermatitis are also found in people with ichthyosis vulgaris. As in atopic dermatitis, the mutations lead to production of an abnormally short profilaggrin molecule that cannot be cleaved to produce filaggrin proteins. A lack of filaggrin and the molecules produced from its processing lead to the dry, scaly skin characteristic of ichthyosis vulgaris. Impairment of the skin's barrier function likely contributes to the development of allergic diseases in some people with ichthyosis vulgaris. Individuals with FLG gene mutations have an increased risk of developing asthma, but only if they also have atopic dermatitis. FLG gene mutations also increase the risk of hay fever, food allergies, and skin sensitivity to nickel, regardless of whether the person has atopic dermatitis. Additional factors, such as living with a cat and smoking tobacco products, may increase the risk of atopic dermatitis and asthma, respectively, in people with ichthyosis vulgaris.    Cytogenetic Location: 1q21.3, which is the long (q) arm of chromosome 1 at position 21.3Molecular Location: base pairs 152,302,175 to 152,325,203 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATOD2epidermal filaggrin     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Immunobiology:The Immune System in Health and Disease (fifth edition, 2001): Effector Mechanisms in Allergic ReactionsImmunobiology:The Immune System in Health and Disease (fifth edition, 2001): The Production of IgE      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FLG      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     FILAGGRINICHTHYOSIS VULGARIS      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology:The Immune System in Health and Disease (fifth edition, 2001): Effector Mechanisms in Allergic ReactionsImmunobiology:The Immune System in Health and Disease (fifth edition, 2001): The Production of IgE  Tests of FLG  PubMed  FILAGGRINICHTHYOSIS VULGARIS  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Brown SJ, McLean WH. One remarkable molecule: filaggrin. J Invest Dermatol. 2012 Mar;132(3 Pt 2):751-62. doi: 10.1038/jid.2011.393. Epub 2011 Dec 8. Review. Citation on PubMed or Free article on PubMed CentralOMIM: FILAGGRINIrvine AD, McLean WH, Leung DY. Filaggrin mutations associated with skin and allergic diseases. N Engl J Med. 2011 Oct 6;365(14):1315-27. doi: 10.1056/NEJMra1011040. Review. Citation on PubMedLiang Y, Chang C, Lu Q. The Genetics and Epigenetics of Atopic Dermatitis-Filaggrin and Other Polymorphisms. Clin Rev Allergy Immunol. 2015 Sep 18. [Epub ahead of print] Citation on PubMedPalmer CN, Irvine AD, Terron-Kwiatkowski A, Zhao Y, Liao H, Lee SP, Goudie DR, Sandilands A, Campbell LE, Smith FJ, O'Regan GM, Watson RM, Cecil JE, Bale SJ, Compton JG, DiGiovanna JJ, Fleckman P, Lewis-Jones S, Arseculeratne G, Sergeant A, Munro CS, El Houate B, McElreavey K, Halkjaer LB, Bisgaard H, Mukhopadhyay S, McLean WH. Common loss-of-function variants of the epidermal barrier protein filaggrin are a major predisposing factor for atopic dermatitis. Nat Genet. 2006 Apr;38(4):441-6. Epub 2006 Mar 19. Citation on PubMedRodrguez E, Baurecht H, Herberich E, Wagenpfeil S, Brown SJ, Cordell HJ, Irvine AD, Weidinger S. Meta-analysis of filaggrin polymorphisms in eczema and asthma: robust risk factors in atopic disease. J Allergy Clin Immunol. 2009 Jun;123(6):1361-70.e7. doi: 10.1016/j.jaci.2009.03.036. Citation on PubMedSmith FJ, Irvine AD, Terron-Kwiatkowski A, Sandilands A, Campbell LE, Zhao Y, Liao H, Evans AT, Goudie DR, Lewis-Jones S, Arseculeratne G, Munro CS, Sergeant A, O'Regan G, Bale SJ, Compton JG, DiGiovanna JJ, Presland RB, Fleckman P, McLean WH. Loss-of-function mutations in the gene encoding filaggrin cause ichthyosis vulgaris. Nat Genet. 2006 Mar;38(3):337-42. Epub 2006 Jan 29. Citation on PubMedThyssen JP, Godoy-Gijon E, Elias PM. Ichthyosis vulgaris: the filaggrin mutation disease. Br J Dermatol. 2013 Jun;168(6):1155-66. doi: 10.1111/bjd.12219. Epub 2013 May 6. Review. Citation on PubMed \n",
            "FLG2\t   From NCBI Gene:The filaggrin-like protein encoded by this gene is upregulated by calcium, proteolyzed by calpain 1, and is involved in epithelial homeostasis. The encoded protein is required for proper cornification in skin, with defects in this gene being associated with skin diseases. This protein also has a function in skin barrier protection. In fact, in addition to providing a physical barrier, C-terminal fragments of this protein display antimicrobial activity against P. aeruginosa. [provided by RefSeq, Dec 2015]From UniProt:Essential for normal cell-cell adhesion in the cornified cell layers (PubMed:29758285). Important for proper integrity and mechanical strength of the stratum corneum of the epidermis (PubMed:29505760).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:PEELING SKIN SYNDROME 6From UniProt:Peeling skin syndrome 6 (PSS6): A form of peeling skin syndrome, a genodermatosis characterized by generalized, continuous shedding of the outer layers of the epidermis. Two main PSS subtypes have been suggested. Patients with non-inflammatory PSS (type A) manifest white scaling, with painless and easy removal of the skin, irritation when in contact with water, dust and sand, and no history of erythema, pruritis or atopy. Inflammatory PSS (type B) is associated with generalized erythema, pruritus and atopy. It is an ichthyosiform erythroderma characterized by lifelong patchy peeling of the entire skin with onset at birth or shortly after. Several patients have been reported with high IgE levels. PSS6 patients manifest generalized ichthyotic dry skin, and bullous peeling lesions on the trunk and limbs at sites of minor trauma. Skin symptoms are exacerbated by warmth and humidity. PSS6 inheritance is autosomal recessive. [MIM:618084]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q21.3, which is the long (q) arm of chromosome 1 at position 21.3Molecular Location: base pairs 152,348,735 to 152,360,028 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IFPSPSS6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FLG2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FILAGGRIN FAMILY MEMBER 2OMIM: PEELING SKIN SYNDROME 6      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FLG2  OMIM: FILAGGRIN FAMILY MEMBER 2OMIM: PEELING SKIN SYNDROME 6  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LCE4A\t\n",
            "SPRR1A\t\n",
            "LOR\t   The LOR gene is part of a cluster of genes on chromosome 1 called the epidermal differentiation complex. These genes are involved in the formation and maintenance of the outer layer of skin (the epidermis), particularly its tough outer surface (the stratum corneum). The stratum corneum, which is formed in a process known as cornification, provides a sturdy barrier between the body and its environment. Each cell of the stratum corneum, called a corneocyte, is surrounded by a protein shell called a cornified envelope.The LOR gene provides instructions for making a protein called loricrin, which is a major component of the cornified envelope. Links between loricrin and other components of the envelopes hold the corneocytes together and help give the stratum corneum its strength.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Vohwinkel syndrome     At least two mutations in the LOR gene have been identified in people with the variant form of Vohwinkel syndrome, sometimes called loricrin keratoderma. This disorder is characterized by skin abnormalities including widespread dry, scaly skin (ichthyosis), especially on the limbs. The mutations that cause the variant form of Vohwinkel syndrome change the structure of the loricrin protein; the altered protein is trapped inside the cell and cannot reach the cornified envelope. While other proteins can partially compensate for the missing loricrin, the envelope of some of the corneocytes is thinner than normal, resulting in the dry, scaly skin (ichthyosis) and other skin abnormalities associated with the variant form of Vohwinkel syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two mutations in the LOR gene have been identified in people with the variant form of Vohwinkel syndrome, sometimes called loricrin keratoderma. This disorder is characterized by skin abnormalities including widespread dry, scaly skin (ichthyosis), especially on the limbs. The mutations that cause the variant form of Vohwinkel syndrome change the structure of the loricrin protein; the altered protein is trapped inside the cell and cannot reach the cornified envelope. While other proteins can partially compensate for the missing loricrin, the envelope of some of the corneocytes is thinner than normal, resulting in the dry, scaly skin (ichthyosis) and other skin abnormalities associated with the variant form of Vohwinkel syndrome. More About This Health Condition    Cytogenetic Location: 1q21.3, which is the long (q) arm of chromosome 1 at position 21.3Molecular Location: base pairs 153,259,635 to 153,262,125 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LORI_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Developmental Biology (sixth edition, 2000): The Epidermis and the Origin of Cutaneous Structures      Tests Listed in the Genetic Testing Registry (1 link)     Tests of LOR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     LORICRIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): The Epidermis and the Origin of Cutaneous Structures  Tests of LOR  PubMed  LORICRIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Drera B, Tadini G, Balbo F, Marchese L, Barlati S, Colombi M. De novo occurrence of the 730insG recurrent mutation in an Italian family with the ichthyotic variant of Vohwinkel syndrome, loricrin keratoderma. Clin Genet. 2008 Jan;73(1):85-8. Epub 2007 Oct 22. Citation on PubMedElias PM, Williams ML, Crumrine D, Schmuth M. Inherited disorders of corneocyte proteins. Curr Probl Dermatol. 2010;39:98-131. doi: 10.1159/000321086. Epub 2010 Sep 14. Review. Citation on PubMedGedicke MM, Traupe H, Fischer B, Tinschert S, Hennies HC. Towards characterization of palmoplantar keratoderma caused by gain-of-function mutation in loricrin: analysis of a family and review of the literature. Br J Dermatol. 2006 Jan;154(1):167-71. Review. Citation on PubMedHenry J, Toulza E, Hsu CY, Pellerin L, Balica S, Mazereeuw-Hautier J, Paul C, Serre G, Jonca N, Simon M. Update on the epidermal differentiation complex. Front Biosci (Landmark Ed). 2012 Jan 1;17:1517-32. Review. Citation on PubMedOMIM: LORICRINMaestrini E, Monaco AP, McGrath JA, Ishida-Yamamoto A, Camisa C, Hovnanian A, Weeks DE, Lathrop M, Uitto J, Christiano AM. A molecular defect in loricrin, the major component of the cornified cell envelope, underlies Vohwinkel's syndrome. Nat Genet. 1996 May;13(1):70-7. Citation on PubMedO'Driscoll J, Muston GC, McGrath JA, Lam HM, Ashworth J, Christiano AM. A recurrent mutation in the loricrin gene underlies the ichthyotic variant of Vohwinkel syndrome. Clin Exp Dermatol. 2002 May;27(3):243-6. Citation on PubMedSchmuth M, Fluhr JW, Crumrine DC, Uchida Y, Hachem JP, Behne M, Moskowitz DG, Christiano AM, Feingold KR, Elias PM. Structural and functional consequences of loricrin mutations in human loricrin keratoderma (Vohwinkel syndrome with ichthyosis). J Invest Dermatol. 2004 Apr;122(4):909-22. Citation on PubMedSchmuth M, Gruber R, Elias PM, Williams ML. Ichthyosis update: towards a function-driven model of pathogenesis of the disorders of cornification and the role of corneocyte proteins in these disorders. Adv Dermatol. 2007;23:231-56. Review. Citation on PubMed or Free article on PubMed Central \n",
            "PGLYRP3\t\n",
            "KCNN3\t\n",
            "THBS3\t\n",
            "GBA\t   The GBA gene provides instructions for making an enzyme called beta-glucocerebrosidase. This enzyme is active in lysosomes, which are structures inside cells that act as recycling centers. Lysosomes use digestive enzymes to break down toxic substances, digest bacteria that invade the cell, and recycle worn-out cell components. Based on these functions, enzymes in the lysosome are sometimes called housekeeping enzymes. Beta-glucocerebrosidase is a housekeeping enzyme that helps break down a large molecule called glucocerebroside into a sugar (glucose) and a simpler fat molecule (ceramide). Glucocerebroside is a component of the membrane that surrounds cells. It gets broken down by beta-glucocerebrosidase when cells die, and the components are reused as new cells are formed.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Gaucher disease     More than 380 mutations in the GBA gene have been identified in people with Gaucher disease, a disorder with varied features that affect many parts of the body. Affected individuals can have enlargement of the liver and spleen (hepatosplenomegaly), blood cell abnormalities, and rarely, severe neurological problems. The mutations occur in both copies of the gene in each cell. Most of the GBA gene mutations responsible for Gaucher disease change single protein building blocks (amino acids) in beta-glucocerebrosidase, altering the structure of the enzyme and preventing it from working normally. Other mutations delete or insert genetic material in the GBA gene or lead to the production of an abnormally short, nonfunctional version of the enzyme.Mutations in the GBA gene greatly reduce or eliminate the activity of beta-glucocerebrosidase in cells. As a result, glucocerebroside is not broken down properly. This molecule and related substances can build up in white blood cells called macrophages in the spleen, liver, bone marrow, and other organs. The abnormal accumulation and storage of these substances damages tissues and organs, causing the characteristic features of Gaucher disease. More About This Health Condition      Parkinson disease     Changes in the GBA gene are also associated with Parkinson disease and parkinsonism, which are similar disorders that affect movement. Characteristic features include tremors and impaired balance and coordination (postural instability). People with Gaucher disease (described above) have mutations in both copies of the GBA gene in each cell, while those with a mutation in just one copy of the gene are called carriers. People with Gaucher disease and people who are carriers of a GBA gene mutation have an increased risk of developing Parkinson disease or parkinsonism.Symptoms of Parkinson disease and parkinsonism result from the loss of nerve cells (neurons) that produce dopamine. Dopamine is a chemical messenger that transmits signals within the brain to produce smooth physical movements. It remains unclear how GBA gene mutations are related to these disorders. Studies suggest that changes in this gene may contribute to the faulty breakdown of toxic substances in neurons by impairing the function of lysosomes. Alternatively, the changes may increase the formation of abnormal protein deposits. As a result, toxic substances or protein deposits could accumulate and kill dopamine-producing neurons, leading to abnormal movements and balance problems. More About This Health Condition      Dementia with Lewy bodies     GBA gene mutations can increase the risk of developing dementia with Lewy bodies; however, some people with a mutation in the GBA gene never develop this condition. Dementia with Lewy bodies is characterized by intellectual decline (dementia); visual hallucinations; sudden changes in attention and mood; and movement problems characteristic of Parkinson disease (described above) such as rigidity of limbs, tremors, and impaired balance and coordination.Mutations in one copy of the GBA gene increase the risk of developing dementia with Lewy bodies. These mutations result in the production of an altered beta-glucocerebrosidase enzyme. This abnormal enzyme may interfere with the function of lysosomes and the normal breakdown of a protein called alpha-synuclein, which increases the risk that these proteins accumulate and form Lewy bodies. Accumulation of these protein clusters throughout the brain impairs neuron function and ultimately causes cell death. Over time, the loss of neurons increasingly impairs intellectual and motor function and the regulation of emotions, resulting in the signs and symptoms of dementia with Lewy bodies. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 380 mutations in the GBA gene have been identified in people with Gaucher disease, a disorder with varied features that affect many parts of the body. Affected individuals can have enlargement of the liver and spleen (hepatosplenomegaly), blood cell abnormalities, and rarely, severe neurological problems. The mutations occur in both copies of the gene in each cell. Most of the GBA gene mutations responsible for Gaucher disease change single protein building blocks (amino acids) in beta-glucocerebrosidase, altering the structure of the enzyme and preventing it from working normally. Other mutations delete or insert genetic material in the GBA gene or lead to the production of an abnormally short, nonfunctional version of the enzyme.Mutations in the GBA gene greatly reduce or eliminate the activity of beta-glucocerebrosidase in cells. As a result, glucocerebroside is not broken down properly. This molecule and related substances can build up in white blood cells called macrophages in the spleen, liver, bone marrow, and other organs. The abnormal accumulation and storage of these substances damages tissues and organs, causing the characteristic features of Gaucher disease. More About This Health Condition  Changes in the GBA gene are also associated with Parkinson disease and parkinsonism, which are similar disorders that affect movement. Characteristic features include tremors and impaired balance and coordination (postural instability). People with Gaucher disease (described above) have mutations in both copies of the GBA gene in each cell, while those with a mutation in just one copy of the gene are called carriers. People with Gaucher disease and people who are carriers of a GBA gene mutation have an increased risk of developing Parkinson disease or parkinsonism.Symptoms of Parkinson disease and parkinsonism result from the loss of nerve cells (neurons) that produce dopamine. Dopamine is a chemical messenger that transmits signals within the brain to produce smooth physical movements. It remains unclear how GBA gene mutations are related to these disorders. Studies suggest that changes in this gene may contribute to the faulty breakdown of toxic substances in neurons by impairing the function of lysosomes. Alternatively, the changes may increase the formation of abnormal protein deposits. As a result, toxic substances or protein deposits could accumulate and kill dopamine-producing neurons, leading to abnormal movements and balance problems. More About This Health Condition  GBA gene mutations can increase the risk of developing dementia with Lewy bodies; however, some people with a mutation in the GBA gene never develop this condition. Dementia with Lewy bodies is characterized by intellectual decline (dementia); visual hallucinations; sudden changes in attention and mood; and movement problems characteristic of Parkinson disease (described above) such as rigidity of limbs, tremors, and impaired balance and coordination.Mutations in one copy of the GBA gene increase the risk of developing dementia with Lewy bodies. These mutations result in the production of an altered beta-glucocerebrosidase enzyme. This abnormal enzyme may interfere with the function of lysosomes and the normal breakdown of a protein called alpha-synuclein, which increases the risk that these proteins accumulate and form Lewy bodies. Accumulation of these protein clusters throughout the brain impairs neuron function and ultimately causes cell death. Over time, the loss of neurons increasingly impairs intellectual and motor function and the regulation of emotions, resulting in the signs and symptoms of dementia with Lewy bodies. More About This Health Condition    Cytogenetic Location: 1q22, which is the long (q) arm of chromosome 1 at position 22Molecular Location: base pairs 155,234,448 to 155,244,862 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        acid beta-glucosidasealgucerasebeta-D-glucosyl-N-acylsphingosine glucohydrolasebeta-glucocerebrosidaseGBA1GLCM_HUMANGLUCglucocerebrosidaseglucocerebroside beta-glucosidaseglucosidase, beta, acidglucosidase, beta; acid (includes glucosylceramidase)glucosphingosine glucosylhydrolaseglucosylceramide beta-glucosidaseimiglucerase     Related Information   How are genetic conditions and genes named?          Educational Resources (4 links)     Dementia: A NICE-SCIE Guideline on Supporting People With Dementia and Their Carers in Health and Social Care (2007): DementiaDementia: A NICE-SCIE Guideline on Supporting People With Dementia and Their Carers in Health and Social Care (2007): Diagnosis and AssessmentHuGENet Case Study: Glucocerebrosidase Gene Mutations and Parkinson's DiseaseNIH News Release: Researchers Uncover Genetic Clues to a Common Form of Age-Related Dementia (July 17, 2006)      Clinical Information from GeneReviews (1 link)     Gaucher Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of GBA      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     GLUCOSIDASE, BETA, ACID      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GenePDGeneUniProt      Dementia: A NICE-SCIE Guideline on Supporting People With Dementia and Their Carers in Health and Social Care (2007): DementiaDementia: A NICE-SCIE Guideline on Supporting People With Dementia and Their Carers in Health and Social Care (2007): Diagnosis and AssessmentHuGENet Case Study: Glucocerebrosidase Gene Mutations and Parkinson's DiseaseNIH News Release: Researchers Uncover Genetic Clues to a Common Form of Age-Related Dementia (July 17, 2006)  Gaucher Disease  Tests of GBA  PubMed  GLUCOSIDASE, BETA, ACID  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GenePDGeneUniProt  Aharon-Peretz J, Rosenbaum H, Gershoni-Baruch R. Mutations in the glucocerebrosidase gene and Parkinson's disease in Ashkenazi Jews. N Engl J Med. 2004 Nov 4;351(19):1972-7. Citation on PubMedCharrow J, Andersson HC, Kaplan P, Kolodny EH, Mistry P, Pastores G, Prakash-Cheng A, Rosenbloom BE, Scott CR, Wappner RS, Weinreb NJ. Enzyme replacement therapy and monitoring for children with type 1 Gaucher disease: consensus recommendations. J Pediatr. 2004 Jan;144(1):112-20. Review. Citation on PubMedClark LN, Kartsaklis LA, Wolf Gilbert R, Dorado B, Ross BM, Kisselev S, Verbitsky M, Mejia-Santana H, Cote LJ, Andrews H, Vonsattel JP, Fahn S, Mayeux R, Honig LS, Marder K. Association of glucocerebrosidase mutations with dementia with lewy bodies. Arch Neurol. 2009 May;66(5):578-83. doi: 10.1001/archneurol.2009.54. Citation on PubMed or Free article on PubMed CentralClark LN, Ross BM, Wang Y, Mejia-Santana H, Harris J, Louis ED, Cote LJ, Andrews H, Fahn S, Waters C, Ford B, Frucht S, Ottman R, Marder K. Mutations in the glucocerebrosidase gene are associated with early-onset Parkinson disease. Neurology. 2007 Sep 18;69(12):1270-7. Citation on PubMed or Free article on PubMed CentralCox TM. Gaucher disease: understanding the molecular pathogenesis of sphingolipidoses. J Inherit Metab Dis. 2001;24 Suppl 2:106-21; discussion 87-8. Review. Citation on PubMedGermain DP. Gaucher's disease: a paradigm for interventional genetics. Clin Genet. 2004 Feb;65(2):77-86. Review. Citation on PubMedGoker-Alpan O, Giasson BI, Eblan MJ, Nguyen J, Hurtig HI, Lee VM, Trojanowski JQ, Sidransky E. Glucocerebrosidase mutations are an important risk factor for Lewy body disorders. Neurology. 2006 Sep 12;67(5):908-10. Epub 2006 Jun 21. Citation on PubMedJmoudiak M, Futerman AH. Gaucher disease: pathological mechanisms and modern management. Br J Haematol. 2005 Apr;129(2):178-88. Review. Citation on PubMedMata IF, Samii A, Schneer SH, Roberts JW, Griffith A, Leis BC, Schellenberg GD, Sidransky E, Bird TD, Leverenz JB, Tsuang D, Zabetian CP. Glucocerebrosidase gene mutations: a risk factor for Lewy body disorders. Arch Neurol. 2008 Mar;65(3):379-82. doi: 10.1001/archneurol.2007.68. Citation on PubMed or Free article on PubMed CentralNalls MA, Duran R, Lopez G, Kurzawa-Akanbi M, McKeith IG, Chinnery PF, Morris CM, Theuns J, Crosiers D, Cras P, Engelborghs S, De Deyn PP, Van Broeckhoven C, Mann DM, Snowden J, Pickering-Brown S, Halliwell N, Davidson Y, Gibbons L, Harris J, Sheerin UM, Bras J, Hardy J, Clark L, Marder K, Honig LS, Berg D, Maetzler W, Brockmann K, Gasser T, Novellino F, Quattrone A, Annesi G, De Marco EV, Rogaeva E, Masellis M, Black SE, Bilbao JM, Foroud T, Ghetti B, Nichols WC, Pankratz N, Halliday G, Lesage S, Klebe S, Durr A, Duyckaerts C, Brice A, Giasson BI, Trojanowski JQ, Hurtig HI, Tayebi N, Landazabal C, Knight MA, Keller M, Singleton AB, Wolfsberg TG, Sidransky E. A multicenter study of glucocerebrosidase mutations in dementia with Lewy bodies. JAMA Neurol. 2013 Jun;70(6):727-35. doi: 10.1001/jamaneurol.2013.1925. Citation on PubMed or Free article on PubMed CentralOrvisky E, Park JK, Parker A, Walker JM, Martin BM, Stubblefield BK, Uyama E, Tayebi N, Sidransky E. The identification of eight novel glucocerebrosidase (GBA) mutations in patients with Gaucher disease. Hum Mutat. 2002 Apr;19(4):458-9. Citation on PubMedPelled D, Trajkovic-Bodennec S, Lloyd-Evans E, Sidransky E, Schiffmann R, Futerman AH. Enhanced calcium release in the acute neuronopathic form of Gaucher disease. Neurobiol Dis. 2005 Feb;18(1):83-8. Citation on PubMedSidransky E, Nalls MA, Aasly JO, Aharon-Peretz J, Annesi G, Barbosa ER, Bar-Shira A, Berg D, Bras J, Brice A, Chen CM, Clark LN, Condroyer C, De Marco EV, Drr A, Eblan MJ, Fahn S, Farrer MJ, Fung HC, Gan-Or Z, Gasser T, Gershoni-Baruch R, Giladi N, Griffith A, Gurevich T, Januario C, Kropp P, Lang AE, Lee-Chen GJ, Lesage S, Marder K, Mata IF, Mirelman A, Mitsui J, Mizuta I, Nicoletti G, Oliveira C, Ottman R, Orr-Urtreger A, Pereira LV, Quattrone A, Rogaeva E, Rolfs A, Rosenbaum H, Rozenberg R, Samii A, Samaddar T, Schulte C, Sharma M, Singleton A, Spitz M, Tan EK, Tayebi N, Toda T, Troiano AR, Tsuji S, Wittstock M, Wolfsberg TG, Wu YR, Zabetian CP, Zhao Y, Ziegler SG. Multicenter analysis of glucocerebrosidase mutations in Parkinson's disease. N Engl J Med. 2009 Oct 22;361(17):1651-61. doi: 10.1056/NEJMoa0901281. Citation on PubMed or Free article on PubMed CentralSidransky E. Gaucher disease: complexity in a \"simple\" disorder. Mol Genet Metab. 2004 Sep-Oct;83(1-2):6-15. Review. Citation on PubMedSidransky E. Heterozygosity for a Mendelian disorder as a risk factor for complex disease. Clin Genet. 2006 Oct;70(4):275-82. Review. Citation on PubMedTsuang D, Leverenz JB, Lopez OL, Hamilton RL, Bennett DA, Schneider JA, Buchman AS, Larson EB, Crane PK, Kaye JA, Kramer P, Woltjer R, Kukull W, Nelson PT, Jicha GA, Neltner JH, Galasko D, Masliah E, Trojanowski JQ, Schellenberg GD, Yearout D, Huston H, Fritts-Penniman A, Mata IF, Wan JY, Edwards KL, Montine TJ, Zabetian CP. GBA mutations increase risk for Lewy body disease with and without Alzheimer disease pathology. Neurology. 2012 Nov 6;79(19):1944-50. doi: 10.1212/WNL.0b013e3182735e9a. Epub 2012 Oct 3. Citation on PubMed or Free article on PubMed CentralVelayati A, Yu WH, Sidransky E. The role of glucocerebrosidase mutations in Parkinson disease and Lewy body disorders. Curr Neurol Neurosci Rep. 2010 May;10(3):190-8. doi: 10.1007/s11910-010-0102-x. Review. Citation on PubMed or Free article on PubMed Central \n",
            "GLMP\t\n",
            "RHBG\t\n",
            "IQGAP3\t\n",
            "GPATCH4\t\n",
            "HDGF\t\n",
            "CDK11B\t\n",
            "CASP9\t\n",
            "OR6Y1\t\n",
            "OR10X1\t\n",
            "OR10Z1\t\n",
            "OR10J1\t\n",
            "SLAMF8\t\n",
            "SLC35E2B\t\n",
            "FCGR2A\t   From NCBI Gene:This gene encodes one member of a family of immunoglobulin Fc receptor genes found on the surface of many immune response cells. The protein encoded by this gene is a cell surface receptor found on phagocytic cells such as macrophages and neutrophils, and is involved in the process of phagocytosis and clearing of immune complexes. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2008]From UniProt:Binds to the Fc region of immunoglobulins gamma. Low affinity receptor. By binding to IgG it initiates cellular responses against pathogens and soluble antigens. Promotes phagocytosis of opsonized antigens.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Rheumatoid arthritisFrom NCBI Gene:Cystic fibrosisSystemic lupus erythematosusSusceptibility to malaria     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q23.3, which is the long (q) arm of chromosome 1 at position 23.3Molecular Location: base pairs 161,505,415 to 161,524,048 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD32CD32ACDw32FCG2FcGRFCGR2FCGR2A1IGFR2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FCGR2A      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: CYSTIC FIBROSISOMIM: Fc FRAGMENT OF IgG, LOW AFFINITY IIa, RECEPTOR FOROMIM: MALARIA, SUSCEPTIBILITY TOOMIM: SYSTEMIC LUPUS ERYTHEMATOSUS      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FCGR2A  OMIM: CYSTIC FIBROSISOMIM: Fc FRAGMENT OF IgG, LOW AFFINITY IIa, RECEPTOR FOROMIM: MALARIA, SUSCEPTIBILITY TOOMIM: SYSTEMIC LUPUS ERYTHEMATOSUS  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HSPA6\t\n",
            "FCGR3A\t   From NCBI Gene:This gene encodes a receptor for the Fc portion of immunoglobulin G, and it is involved in the removal of antigen-antibody complexes from the circulation, as well as other other antibody-dependent responses. This gene (FCGR3A) is highly similar to another nearby gene (FCGR3B) located on chromosome 1. The receptor encoded by this gene is expressed on natural killer (NK) cells as an integral membrane glycoprotein anchored through a transmembrane peptide, whereas FCGR3B is expressed on polymorphonuclear neutrophils (PMN) where the receptor is anchored through a phosphatidylinositol (PI) linkage. Mutations in this gene have been linked to susceptibility to recurrent viral infections, susceptibility to systemic lupus erythematosus, and alloimmune neonatal neutropenia. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Receptor for the Fc region of IgG. Binds complexed or aggregated IgG and also monomeric IgG. Mediates antibody-dependent cellular cytotoxicity (ADCC) and other antibody-dependent responses, such as phagocytosis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Immunodeficiency 20From UniProt:Immunodeficiency 20 (IMD20): A rare autosomal recessive primary immunodeficiency characterized by functional deficiency of NK cells. Affected individuals typically present with severe herpes viral infections, particularly Epstein Barr virus (EBV), and human papillomavirus (HPV). [MIM:615707]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q23.3, which is the long (q) arm of chromosome 1 at position 23.3Molecular Location: base pairs 161,541,759 to 161,550,737 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD16CD16AFCG3FCGR3FCGRIIIFCR-10FCRIIIFCRIIIAIGFR3IMD20     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FCGR3A      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: Fc FRAGMENT OF IgG, LOW AFFINITY IIIa, RECEPTOR FOROMIM: IMMUNODEFICIENCY 20      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FCGR3A  OMIM: Fc FRAGMENT OF IgG, LOW AFFINITY IIIa, RECEPTOR FOROMIM: IMMUNODEFICIENCY 20  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FCGR2C\t   From NCBI Gene:This gene encodes one of three members of a family of low-affinity immunoglobulin gamma Fc receptors found on the surface of many immune response cells. The encoded protein is a transmembrane glycoprotein and may be involved in phagocytosis and clearing of immune complexes. An allelic polymorphism in this gene results in both coding and non-coding variants. [provided by RefSeq, Apr 2012]From UniProt:Receptor for the Fc region of complexed immunoglobulins gamma. Low affinity receptor. Involved in a variety of effector and regulatory functions such as phagocytosis of immune complexes and modulation of antibody production by B-cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Autoimmune thrombocytopenia     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q23.3, which is the long (q) arm of chromosome 1 at position 23.3Molecular Location: base pairs 161,581,339 to 161,601,220 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD32CD32CCDW32FCG2FCRIICIGFR2     Related Information   How are genetic conditions and genes named?          Catalog of Genes and Diseases from OMIM (2 links)     OMIM: Fc FRAGMENT OF IgG, LOW AFFINITY IIc, RECEPTOR FOROMIM: THROMBOCYTOPENIC PURPURA, AUTOIMMUNE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      OMIM: Fc FRAGMENT OF IgG, LOW AFFINITY IIc, RECEPTOR FOROMIM: THROMBOCYTOPENIC PURPURA, AUTOIMMUNE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SPEN\t\n",
            "FAM131C\t\n",
            "CDK11A\t\n",
            "ARHGEF19\t\n",
            "SPATA21\t\n",
            "NBPF1\t\n",
            "SELP\t   From NCBI Gene:This gene encodes a 140 kDa protein that is stored in the alpha-granules of platelets and Weibel-Palade bodies of endothelial cells. This protein redistributes to the plasma membrane during platelet activation and degranulation and mediates the interaction of activated endothelial cells or platelets with leukocytes. The membrane protein is a calcium-dependent receptor that binds to sialylated forms of Lewis blood group carbohydrate antigens on neutrophils and monocytes. Alternative splice variants may occur but are not well documented. [provided by RefSeq, Jul 2008]From UniProt:Ca(2+)-dependent receptor for myeloid cells that binds to carbohydrates on neutrophils and monocytes. Mediates the interaction of activated endothelial cells or platelets with leukocytes. The ligand recognized is sialyl-Lewis X. Mediates rapid rolling of leukocyte rolling over vascular surfaces during the initial steps in inflammation through interaction with SELPLG.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Ischemic stroke (ISCHSTR): A stroke is an acute neurologic event leading to death of neural tissue of the brain and resulting in loss of motor, sensory and/or cognitive function. Ischemic strokes, resulting from vascular occlusion, is considered to be a highly complex disease consisting of a group of heterogeneous disorders with multiple genetic and environmental risk factors. [MIM:601367]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q24.2, which is the long (q) arm of chromosome 1 at position 24.2Molecular Location: base pairs 169,588,849 to 169,630,139 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD62CD62PGMP140GRMPLECAM3PADGEMPSEL     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SELP      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: SELECTIN P      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SELP  OMIM: SELECTIN P  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MST1L\t\n",
            "FMO1\t\n",
            "PIGC\t   From NCBI Gene:This gene encodes an endoplasmic reticulum associated protein that is involved in glycosylphosphatidylinositol (GPI) lipid anchor biosynthesis. The GPI lipid anchor is a glycolipid found on many blood cells and serves to anchor proteins to the cell surface. The encoded protein is one subunit of the GPI N-acetylglucosaminyl (GlcNAc) transferase that transfers GlcNAc to phosphatidylinositol (PI) on the cytoplasmic side of the endoplasmic reticulum. Two alternatively spliced transcripts that encode the same protein have been found for this gene. A pseudogene on chromosome 11 has also been characterized. [provided by RefSeq, Jul 2008]From UniProt:Involved in GPI anchor biosynthesis (PubMed:8806613, PubMed:27694521). Part of the complex catalyzing the transfer of N-acetylglucosamine from UDP-N-acetylglucosamine to phosphatidylinositol, the first step of GPI biosynthesis (ECO:0000269|PubMed:27694521).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:GLYCOSYLPHOSPHATIDYLINOSITOL BIOSYNTHESIS DEFECT 16From UniProt:Glycosylphosphatidylinositol biosynthesis defect 16 (GPIBD16): An autosomal recessive disorder characterized by delayed psychomotor development, intellectual disability, and seizures. [MIM:617816]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q24.3, which is the long (q) arm of chromosome 1 at position 24.3Molecular Location: base pairs 172,441,457 to 172,444,090 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GPI2GPIBD16MRT62     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PIGC      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GLYCOSYLPHOSPHATIDYLINOSITOL BIOSYNTHESIS DEFECT 16OMIM: PHOSPHATIDYLINOSITOL GLYCAN ANCHOR BIOSYNTHESIS CLASS C PROTEIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PIGC  OMIM: GLYCOSYLPHOSPHATIDYLINOSITOL BIOSYNTHESIS DEFECT 16OMIM: PHOSPHATIDYLINOSITOL GLYCAN ANCHOR BIOSYNTHESIS CLASS C PROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CROCC\t\n",
            "KIAA0040\t\n",
            "PADI4\t   From NCBI Gene:This gene is a member of a gene family which encodes enzymes responsible for the conversion of arginine residues to citrulline residues. This gene may play a role in granulocyte and macrophage development leading to inflammation and immune response. [provided by RefSeq, Jul 2008]From UniProt:Catalyzes the citrullination/deimination of arginine residues of proteins such as histones, thereby playing a key role in histone code and regulation of stem cell maintenance. Citrullinates histone H1 at 'Arg-54' (to form H1R54ci), histone H3 at 'Arg-2', 'Arg-8', 'Arg-17' and/or 'Arg-26' (to form H3R2ci, H3R8ci, H3R17ci, H3R26ci, respectively) and histone H4 at 'Arg-3' (to form H4R3ci). Acts as a key regulator of stem cell maintenance by mediating citrullination of histone H1: citrullination of 'Arg-54' of histone H1 (H1R54ci) results in H1 displacement from chromatin and global chromatin decondensation, thereby promoting pluripotency and stem cell maintenance. Promotes profound chromatin decondensation during the innate immune response to infection in neutrophils by mediating formation of H1R54ci. Citrullination of histone H3 prevents their methylation by CARM1 and HRMT1L2/PRMT1 and represses transcription. Citrullinates EP300/P300 at 'Arg-2142', which favors its interaction with NCOA2/GRIP1.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Rheumatoid arthritisFrom NCBI Gene:Rheumatoid arthritisFrom UniProt:Rheumatoid arthritis (RA): An inflammatory disease with autoimmune features and a complex genetic component. It primarily affects the joints and is characterized by inflammatory changes in the synovial membranes and articular structures, widespread fibrinoid degeneration of the collagen fibers in mesenchymal tissues, and by atrophy and rarefaction of bony structures. [MIM:180300]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.13, which is the short (p) arm of chromosome 1 at position 36.13Molecular Location: base pairs 17,308,195 to 17,364,004 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PADPAD4PADI5PDI4PDI5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PADI4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PEPTIDYLARGININE DEIMINASE, TYPE IVOMIM: RHEUMATOID ARTHRITIS      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PADI4  OMIM: PEPTIDYLARGININE DEIMINASE, TYPE IVOMIM: RHEUMATOID ARTHRITIS  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PADI6\t   From NCBI Gene:This gene encodes a member of the peptidyl arginine deiminase family of enzymes, which catalyze the post-translational deimination of proteins by converting arginine residues into citrullines in the presence of calcium ions. The family members have distinct substrate specificities and tissue-specific expression patterns. This protein may play a role in cytoskeletal reorganization in the egg and in early embryo development. [provided by RefSeq, Sep 2012]From UniProt:Catalyzes the deimination of arginine residues of proteins (By similarity). May be involved in cytoskeletal reorganization in the egg and early embryo (PubMed:27545678).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Preimplantation embryonic lethality 2From UniProt:Preimplantation embryonic lethality 2 (PREMBL2): A rare cause of female primary infertility. In affected women, ovulation and fertilization proceed normally but embryos are arrested at early stages of development. Inheritance is autosomal recessive. [MIM:617234]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.13, which is the short (p) arm of chromosome 1 at position 36.13Molecular Location: base pairs 17,372,196 to 17,401,699 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        hPADVIPREMBL2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PADI6      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PEPTIDYLARGININE DEIMINASE, TYPE VIOMIM: PREIMPLANTATION EMBRYONIC LETHALITY 2      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PADI6  OMIM: PEPTIDYLARGININE DEIMINASE, TYPE VIOMIM: PREIMPLANTATION EMBRYONIC LETHALITY 2  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SEC16B\t\n",
            "TEX35\t\n",
            "ARHGEF10L\t\n",
            "TOR1AIP1\t   From NCBI Gene:This gene encodes a type 2 integral membrane protein that binds A- and B-type lamins. The encoded protein localizes to the inner nuclear membrane and may be involved in maintaining the attachment of the nuclear membrane to the nuclear lamina during cell division. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Apr 2016]From UniProt:Required for nuclear membrane integrity. Induces TOR1A and TOR1B ATPase activity and is required for their location on the nuclear membrane. Binds to A- and B-type lamins. Possible role in membrane attachment and assembly of the nuclear lamina.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Muscular dystrophy, limb-girdle, type 2yFrom UniProt:Myopathy, autosomal recessive, with rigid spine and distal joint contractures (MRRSDC): An autosomal recessive degenerative myopathy characterized by muscle weakness initially involving the proximal lower limbs, followed by distal upper and lower limb muscle weakness and atrophy. Other features include joint contractures, rigid spine, and restricted pulmonary function. Cardiac involvement has been observed in some patients. Disease onset is in the first or second decades of life. [MIM:617072]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q25.2, which is the long (q) arm of chromosome 1 at position 25.2Molecular Location: base pairs 179,882,042 to 179,920,077 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LAP1LAP1BLGMD2Y     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TOR1AIP1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MYOPATHY, AUTOSOMAL RECESSIVE, WITH RIGID SPINE AND DISTAL JOINT CONTRACTURESOMIM: TORSIN A-INTERACTING PROTEIN 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TOR1AIP1  OMIM: MYOPATHY, AUTOSOMAL RECESSIVE, WITH RIGID SPINE AND DISTAL JOINT CONTRACTURESOMIM: TORSIN A-INTERACTING PROTEIN 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RNASEL\t   From NCBI Gene:This gene encodes a component of the interferon-regulated 2-5A system that functions in the antiviral and antiproliferative roles of interferons. Mutations in this gene have been associated with predisposition to prostate cancer and this gene is a candidate for the hereditary prostate cancer 1 (HPC1) allele. [provided by RefSeq, Jul 2008]From UniProt:Endoribonuclease that functions in the interferon (IFN) antiviral response. In INF treated and virus infected cells, RNASEL probably mediates its antiviral effects through a combination of direct cleavage of single-stranded viral RNAs, inhibition of protein synthesis through the degradation of rRNA, induction of apoptosis, and induction of other antiviral genes. RNASEL mediated apoptosis is the result of a JNK-dependent stress-response pathway leading to cytochrome c release from mitochondria and caspase-dependent apoptosis. Therefore, activation of RNASEL could lead to elimination of virus infected cells under some circumstances. In the crosstalk between autophagy and apoptosis proposed to induce autophagy as an early stress response to small double-stranded RNA and at later stages of prolonged stress to activate caspase-dependent proteolytic cleavage of BECN1 to terminate autophagy and promote apoptosis (PubMed:26263979). Might play a central role in the regulation of mRNA turnover (PubMed:11585831). Cleaves 3' of UpNp dimers, with preference for UU and UA sequences, to sets of discrete products ranging from between 4 and 22 nucleotides in length.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Prostate cancerFrom NCBI Gene:Prostate cancer, hereditary, 1From UniProt:Prostate cancer, hereditary, 1 (HPC1): A condition associated with familial predisposition to cancer of the prostate. Most prostate cancers are adenocarcinomas that develop in the acini of the prostatic ducts. Other rare histopathologic types of prostate cancer that occur in approximately 5% of patients include small cell carcinoma, mucinous carcinoma, prostatic ductal carcinoma, transitional cell carcinoma, squamous cell carcinoma, basal cell carcinoma, adenoid cystic carcinoma (basaloid), signet-ring cell carcinoma and neuroendocrine carcinoma. [MIM:601518]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q25.3, which is the long (q) arm of chromosome 1 at position 25.3Molecular Location: base pairs 182,573,634 to 182,589,285 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PRCA1RNS4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RNASEL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PROSTATE CANCER, HEREDITARY, 1OMIM: RIBONUCLEASE L      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RNASEL  OMIM: PROSTATE CANCER, HEREDITARY, 1OMIM: RIBONUCLEASE L  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMEM52\t\n",
            "HMCN1\t   From NCBI Gene:This gene encodes a large extracellular member of the immunoglobulin superfamily. A similar protein in C. elegans forms long, fine tracks at specific extracellular sites that are involved in many processes such as stabilization of the germline syncytium, anchorage of mechanosensory neurons to the epidermis, and organization of hemidesmosomes in the epidermis. Mutations in this gene may be associated with age-related macular degeneration. [provided by RefSeq, Jul 2008]From UniProt:Promotes cleavage furrow maturation during cytokinesis in preimplantation embryos. May play a role in the architecture of adhesive and flexible epithelial cell junctions. May play a role during myocardial remodeling by imparting an effect on cardiac fibroblast migration.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Age-related macular degenerationFrom NCBI Gene:Age-related macular degeneration 1From UniProt:Macular degeneration, age-related, 1 (ARMD1): A form of age-related macular degeneration, a multifactorial eye disease and the most common cause of irreversible vision loss in the developed world. In most patients, the disease is manifest as ophthalmoscopically visible yellowish accumulations of protein and lipid that lie beneath the retinal pigment epithelium and within an elastin-containing structure known as Bruch membrane. [MIM:603075]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q25.3-q31.1, which is the long (q) arm of chromosome 1 between positions 25.3 and 31.1Molecular Location: base pairs 185,734,525 to 186,190,953 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARMD1FBLN6FIBL-6FIBL6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HMCN1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HEMICENTINOMIM: MACULAR DEGENERATION, AGE-RELATED, 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HMCN1  OMIM: HEMICENTINOMIM: MACULAR DEGENERATION, AGE-RELATED, 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CFAP74\t\n",
            "PRG4\t   From NCBI Gene:The protein encoded by this gene is a large proteoglycan that is synthesized by chondrocytes located at the surface of articular cartilage and by some synovial lining cells. This protein contains both chondroitin sulfate and keratan sulfate glycosaminoglycans. It functions as a boundary lubricant at the cartilage surface and contributes to the elastic absorption and energy dissipation of synovial fluid. Mutations in this gene result in camptodactyly-arthropathy-coxa vara-pericarditis syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]From UniProt:Isoform F plays a role as a growth factor acting on the primitive cells of both hematopoietic and endothelial cell lineages.Plays a role in boundary lubrication within articulating joints. Prevents protein deposition onto cartilage from synovial fluid by controlling adhesion-dependent synovial growth and inhibiting the adhesion of synovial cells to the cartilage surface.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Camptodactyly arthropathy coxa vara pericarditis syndromeFrom UniProt:Camptodactyly-arthropathy-coxa vara-pericarditis syndrome (CACP): An autosomal recessive disorder characterized by the association of congenital or early-onset camptodactyly and non-inflammatory arthropathy with synovial hyperplasia. Individuals with CACP have normal appearing joints at birth but with advancing age develop joint failure, non-inflammatory synoviocyte hyperplasia and subintimal fibrosis of the synovial capsule. Some patients also manifest progressive coxa vara deformity and/or non-inflammatory pericardial or pleural effusions. [MIM:208250]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q31.1, which is the long (q) arm of chromosome 1 at position 31.1Molecular Location: base pairs 186,296,273 to 186,314,562 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CACPHAPOJCAPMSFSZP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PRG4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CAMPTODACTYLY-ARTHROPATHY-COXA VARA-PERICARDITIS SYNDROMEOMIM: PROTEOGLYCAN 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PRG4  OMIM: CAMPTODACTYLY-ARTHROPATHY-COXA VARA-PERICARDITIS SYNDROMEOMIM: PROTEOGLYCAN 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TPR\t   From NCBI Gene:This gene encodes a large coiled-coil protein that forms intranuclear filaments attached to the inner surface of nuclear pore complexes (NPCs). The protein directly interacts with several components of the NPC. It is required for the nuclear export of mRNAs and some proteins. Oncogenic fusions of the 5' end of this gene with several different kinase genes occur in some neoplasias. [provided by RefSeq, Jul 2008]From UniProt:Component of the nuclear pore complex (NPC), a complex required for the trafficking across the nuclear envelope. Functions as a scaffolding element in the nuclear phase of the NPC essential for normal nucleocytoplasmic transport of proteins and mRNAs, plays a role in the establishment of nuclear-peripheral chromatin compartmentalization in interphase, and in the mitotic spindle checkpoint signaling during mitosis. Involved in the quality control and retention of unspliced mRNAs in the nucleus; in association with NUP153, regulates the nuclear export of unspliced mRNA species bearing constitutive transport element (CTE) in a NXF1- and KHDRBS1-independent manner. Negatively regulates both the association of CTE-containing mRNA with large polyribosomes and translation initiation. Does not play any role in Rev response element (RRE)-mediated export of unspliced mRNAs. Implicated in nuclear export of mRNAs transcribed from heat shock gene promoters; associates both with chromatin in the HSP70 promoter and with mRNAs transcribed from this promoter under stress-induced conditions. Modulates the nucleocytoplasmic transport of activated MAPK1/ERK2 and huntingtin/HTT and may serve as a docking site for the XPO1/CRM1-mediated nuclear export complex. According to some authors, plays a limited role in the regulation of nuclear protein export (PubMed:22253824 and PubMed:11952838). Plays also a role as a structural and functional element of the perinuclear chromatin distribution; involved in the formation and/or maintenance of NPC-associated perinuclear heterochromatin exclusion zones (HEZs). Finally, acts as a spatial regulator of the spindle-assembly checkpoint (SAC) response ensuring a timely and effective recruitment of spindle checkpoint proteins like MAD1L1 and MAD2L1 to unattached kinetochore during the metaphase-anaphase transition before chromosome congression. Its N-terminus is involved in activation of oncogenic kinases.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Involved in tumorigenic rearrangements with the MET.A chromosomal aberration involving TPR has been found in papillary thyroid carcinomas (PTCs). Intrachromosomal rearrangement that links the 5'-end of the TPR gene to the protein kinase domain of NTRK1 forms the fusion protein TRK-T1. TRK-T1 is a 55 kDa protein reacting with antibodies against the carboxy terminus of the NTRK1 protein.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q31.1, which is the long (q) arm of chromosome 1 at position 31.1Molecular Location: base pairs 186,311,652 to 186,375,325 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TPR      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: TRANSLOCATED PROMOTER REGION      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TPR  OMIM: TRANSLOCATED PROMOTER REGION  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KLHDC7A\t\n",
            "CFHR4\t   From NCBI Gene:This gene is a member of the complement factor H (CFH) gene family, and encodes one of the 5 CFH-related (CFHR) proteins. These 5 genes are closely linked to the CFH gene on chromosome 1q31-q32. The CFHRs are secreted plasma proteins synthesized primarily by the hepatocytes, and composed of highly-related short consensus repeats (SCRs). This protein enhances the cofactor activity of CFH, and is involved in complement regulation. It can associate with lipoproteins and may play a role in lipid metabolism. Alternatively spliced transcript variants encoding different isoforms (varying in the number of SCRs) have been described for this gene. [provided by RefSeq, Jan 2011]From UniProt:Involved in complement regulation. Can associate with lipoproteins and may play a role in lipid metabolism.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Age-related macular degeneration     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q31.3, which is the long (q) arm of chromosome 1 at position 31.3Molecular Location: base pairs 196,888,014 to 196,918,911 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CFHL4FHR-4FHR4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CFHR4      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: COMPLEMENT FACTOR H-RELATED 4      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CFHR4  OMIM: COMPLEMENT FACTOR H-RELATED 4  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PTPRC\t   From NCBI Gene:The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitosis, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus is classified as a receptor type PTP. This PTP has been shown to be an essential regulator of T- and B-cell antigen receptor signaling. It functions through either direct interaction with components of the antigen receptor complexes, or by activating various Src family kinases required for the antigen receptor signaling. This PTP also suppresses JAK kinases, and thus functions as a regulator of cytokine receptor signaling. Alternatively spliced transcripts variants of this gene, which encode distinct isoforms, have been reported. [provided by RefSeq, Jun 2012]From UniProt:(Microbial infection) Acts as a receptor for human cytomegalovirus protein UL11 and mediates binding of UL11 to T-cells, leading to reduced induction of tyrosine phosphorylation of multiple signaling proteins upon T-cell receptor stimulation and impaired T-cell proliferation.Protein tyrosine-protein phosphatase required for T-cell activation through the antigen receptor. Acts as a positive regulator of T-cell coactivation upon binding to DPP4. The first PTPase domain has enzymatic activity, while the second one seems to affect the substrate specificity of the first one. Upon T-cell activation, recruits and dephosphorylates SKAP1 and FYN. Dephosphorylates LYN, and thereby modulates LYN activity (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Rheumatoid arthritisFrom NCBI Gene:Severe combined immunodeficiency, autosomal recessive, T cell-negative, B cell-positive, NK cell-positiveHepatitis c virus, susceptibility toFrom UniProt:Severe combined immunodeficiency autosomal recessive T-cell-negative/B-cell-positive/NK-cell-positive (T(-)B(+)NK(+) SCID): A form of severe combined immunodeficiency (SCID), a genetically and clinically heterogeneous group of rare congenital disorders characterized by impairment of both humoral and cell-mediated immunity, leukopenia, and low or absent antibody levels. Patients present in infancy recurrent, persistent infections by opportunistic organisms. The common characteristic of all types of SCID is absence of T-cell-mediated cellular immunity due to a defect in T-cell development. [MIM:608971]Multiple sclerosis (MS): A multifactorial, inflammatory, demyelinating disease of the central nervous system. Sclerotic lesions are characterized by perivascular infiltration of monocytes and lymphocytes and appear as indurated areas in pathologic specimens (sclerosis in plaques). The pathological mechanism is regarded as an autoimmune attack of the myelin sheath, mediated by both cellular and humoral immunity. Clinical manifestations include visual loss, extra-ocular movement disorders, paresthesias, loss of sensation, weakness, dysarthria, spasticity, ataxia and bladder dysfunction. Genetic and environmental factors influence susceptibility to the disease. [MIM:126200]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q31.3-q32.1, which is the long (q) arm of chromosome 1 between positions 31.3 and 32.1Molecular Location: base pairs 198,638,968 to 198,757,476 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        B220CD45CD45RGP180L-CALCALY5T200     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PTPRC      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: HEPATITIS C VIRUS, SUSCEPTIBILITY TOOMIM: PROTEIN-TYROSINE PHOSPHATASE, RECEPTOR-TYPE, COMIM: SEVERE COMBINED IMMUNODEFICIENCY, AUTOSOMAL RECESSIVE, T CELL-NEGATIVE, B CELL-POSITIVE, NK CELL-POSITIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PTPRC  OMIM: HEPATITIS C VIRUS, SUSCEPTIBILITY TOOMIM: PROTEIN-TYROSINE PHOSPHATASE, RECEPTOR-TYPE, COMIM: SEVERE COMBINED IMMUNODEFICIENCY, AUTOSOMAL RECESSIVE, T CELL-NEGATIVE, B CELL-POSITIVE, NK CELL-POSITIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CACNA1S\t   The CACNA1S gene belongs to a family of genes that provide instructions for making calcium channels. These channels, which transport positively charged calcium atoms (ions) into cells, play a key role in a cell's ability to generate and transmit electrical signals.The CACNA1S gene provides instructions for making calcium channels that are abundant in muscles used for movement (skeletal muscles). For the body to move normally, these muscles must tense (contract) and relax in a coordinated way. Muscle contractions are triggered by the flow of certain ions into muscle cells. Channels made with the CACNA1S protein control the transport of calcium ions into muscle cells and are part of the cellular machinery that initiates muscle contractions.Channels made with the CACNA1S protein also activate a protein called ryanodine receptor 1 (produced by the RYR1 gene). Ryanodine receptor 1 forms a channel that releases stored calcium ions within muscle cells in response to certain signals.  The resulting increase in calcium ion concentration stimulates muscle fibers to contract, allowing the body to move.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hypokalemic periodic paralysis     At least four mutations in the CACNA1S gene have been identified in people with hypokalemic periodic paralysis. These mutations cause up to 70 percent of all cases of this disorder.Mutations in the CACNA1S gene change single protein building blocks (amino acids) used to make the CACNA1S protein, which alters the structure and function of calcium channels in skeletal muscle cells. The altered channels open more slowly than usual, reducing the flow of calcium ions into these cells. This disruption in calcium ion transport prevents muscles from contracting normally. It is unclear precisely how these changes lead to episodes of muscle weakness in people with hypokalemic periodic paralysis. More About This Health Condition      Malignant hyperthermia     CACNA1S mutations account for a very small percentage of all cases of malignant hyperthermia.  Researchers have identified two mutations in the CACNA1S gene that are associated with an increased risk of this condition.  One of these mutations replaces the amino acid arginine with the amino acid cysteine at protein position 1086 (written as Arg1086Cys); the other mutation replaces the amino acid arginine with the amino acid histidine at the same protein position (Arg1086His).Genetic mutations likely cause channels made with the CACNA1S protein to activate the RYR1 channel improperly in response to certain drugs (particularly some anesthetics and a type of muscle relaxant used during surgery).  As a result, large amounts of calcium ions are released from storage within muscle cells. An overabundance of available calcium ions causes skeletal muscles to contract abnormally, which leads to muscle rigidity in people with malignant hyperthermia. An increase in calcium ion concentration also activates processes that generate heat (leading to increased body temperature) and produce excess acid (leading to acidosis). More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least four mutations in the CACNA1S gene have been identified in people with hypokalemic periodic paralysis. These mutations cause up to 70 percent of all cases of this disorder.Mutations in the CACNA1S gene change single protein building blocks (amino acids) used to make the CACNA1S protein, which alters the structure and function of calcium channels in skeletal muscle cells. The altered channels open more slowly than usual, reducing the flow of calcium ions into these cells. This disruption in calcium ion transport prevents muscles from contracting normally. It is unclear precisely how these changes lead to episodes of muscle weakness in people with hypokalemic periodic paralysis. More About This Health Condition  CACNA1S mutations account for a very small percentage of all cases of malignant hyperthermia.  Researchers have identified two mutations in the CACNA1S gene that are associated with an increased risk of this condition.  One of these mutations replaces the amino acid arginine with the amino acid cysteine at protein position 1086 (written as Arg1086Cys); the other mutation replaces the amino acid arginine with the amino acid histidine at the same protein position (Arg1086His).Genetic mutations likely cause channels made with the CACNA1S protein to activate the RYR1 channel improperly in response to certain drugs (particularly some anesthetics and a type of muscle relaxant used during surgery).  As a result, large amounts of calcium ions are released from storage within muscle cells. An overabundance of available calcium ions causes skeletal muscles to contract abnormally, which leads to muscle rigidity in people with malignant hyperthermia. An increase in calcium ion concentration also activates processes that generate heat (leading to increased body temperature) and produce excess acid (leading to acidosis). More About This Health Condition    Cytogenetic Location: 1q32.1, which is the long (q) arm of chromosome 1 at position 32.1Molecular Location: base pairs 201,039,509 to 201,112,566 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAC1S_HUMANCACH1CACN1CACNL1A3calcium channel, voltage-dependent, L type, alpha 1S subunitCav1.1CCHL1A3DHPRdihydropyridine receptordihydropyridine-sensitive L-type calcium channel alpha-1 subunitHOKPPHypoKPPhypoPPMHS5Voltage-dependent L-type calcium channel subunit alpha-1Svoltage-gated calcium channel subunit alpha Cav1.1     Related Information   How are genetic conditions and genes named?          Educational Resources (4 links)     Basic Neurochemistry (sixth edition, 1999): Ca2+ channel mutations produce hypokalemic periodic paralysisEurekah Bioscience Collection: High Voltage-Activated Ca2+ ChannelsMolecular Cell Biology (fourth edition, 2000): Muscle: A Specialized Contractile MachineNeuromuscular Disease Center, Washington University      Clinical Information from GeneReviews (2 links)     Hypokalemic Periodic ParalysisMalignant Hyperthermia Susceptibility      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CACNA1S      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CALCIUM CHANNEL, VOLTAGE-DEPENDENT, L TYPE, ALPHA-1S SUBUNIT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Ca2+ channel mutations produce hypokalemic periodic paralysisEurekah Bioscience Collection: High Voltage-Activated Ca2+ ChannelsMolecular Cell Biology (fourth edition, 2000): Muscle: A Specialized Contractile MachineNeuromuscular Disease Center, Washington University  Hypokalemic Periodic ParalysisMalignant Hyperthermia Susceptibility  Tests of CACNA1S  PubMed  CALCIUM CHANNEL, VOLTAGE-DEPENDENT, L TYPE, ALPHA-1S SUBUNIT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Fouad G, Dalakas M, Servidei S, Mendell JR, Van den Bergh P, Angelini C, Alderson K, Griggs RC, Tawil R, Gregg R, Hogan K, Powers PA, Weinberg N, Malonee W, Ptcek LJ. Genotype-phenotype correlations of DHP receptor alpha 1-subunit gene mutations causing hypokalemic periodic paralysis. Neuromuscul Disord. 1997 Jan;7(1):33-8. Citation on PubMedLehmann-Horn F, Jurkat-Rott K, Rdel R. Periodic paralysis: understanding channelopathies. Curr Neurol Neurosci Rep. 2002 Jan;2(1):61-9. Review. Citation on PubMedMiller TM, Dias da Silva MR, Miller HA, Kwiecinski H, Mendell JR, Tawil R, McManis P, Griggs RC, Angelini C, Servidei S, Petajan J, Dalakas MC, Ranum LP, Fu YH, Ptcek LJ. Correlating phenotype and genotype in the periodic paralyses. Neurology. 2004 Nov 9;63(9):1647-55. Citation on PubMedMonnier N, Procaccio V, Stieglitz P, Lunardi J. Malignant-hyperthermia susceptibility is associated with a mutation of the alpha 1-subunit of the human dihydropyridine-sensitive L-type voltage-dependent calcium-channel receptor in skeletal muscle. Am J Hum Genet. 1997 Jun;60(6):1316-25. Citation on PubMed or Free article on PubMed CentralRosenberg H, Sambuughin N, Riazi S, Dirksen R. Malignant Hyperthermia Susceptibility. 2003 Dec 19 [updated 2013 Jan 31]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1146/ Citation on PubMedStewart SL, Hogan K, Rosenberg H, Fletcher JE. Identification of the Arg1086His mutation in the alpha subunit of the voltage-dependent calcium channel (CACNA1S) in a North American family with malignant hyperthermia. Clin Genet. 2001 Mar;59(3):178-84. Citation on PubMedVenance SL, Cannon SC, Fialho D, Fontaine B, Hanna MG, Ptacek LJ, Tristani-Firouzi M, Tawil R, Griggs RC; CINCH investigators. The primary periodic paralyses: diagnosis, pathogenesis and treatment. Brain. 2006 Jan;129(Pt 1):8-17. Epub 2005 Sep 29. Review. Citation on PubMedVicart S, Sternberg D, Arzel-Hzode M, Franques J, Bendahhou S, Lory P, Hainque B, Fournier E, Nicole S, Fontaine B. Hypokalemic Periodic Paralysis. 2002 Apr 30 [updated 2014 Jul 31]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1338/ Citation on PubMedWang Q, Liu M, Xu C, Tang Z, Liao Y, Du R, Li W, Wu X, Wang X, Liu P, Zhang X, Zhu J, Ren X, Ke T, Wang Q, Yang J. Novel CACNA1S mutation causes autosomal dominant hypokalemic periodic paralysis in a Chinese family. J Mol Med (Berl). 2005 Mar;83(3):203-8. Epub 2005 Feb 22. Citation on PubMed or Free article on PubMed CentralYamakage M, Namiki A. Calcium channels--basic aspects of their structure, function and gene encoding; anesthetic action on the channels--a review. Can J Anaesth. 2002 Feb;49(2):151-64. Review. Citation on PubMed \n",
            "IGFN1\t\n",
            "OTUD3\t\n",
            "LGR6\t\n",
            "PPFIA4\t\n",
            "CHIT1\t   From NCBI Gene:Chitotriosidase is secreted by activated human macrophages and is markedly elevated in plasma of Gaucher disease patients. The expression of chitotriosidase occurs only at a late stage of differentiation of monocytes to activated macrophages in culture. Human macrophages can synthesize a functional chitotriosidase, a highly conserved enzyme with a strongly regulated expression. This enzyme may play a role in the degradation of chitin-containing pathogens. Several alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Jan 2012]From UniProt:Degrades chitin, chitotriose and chitobiose. May participate in the defense against nematodes and other pathogens. Isoform 3 has no enzymatic activity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Chitotriosidase deficiency     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q32.1, which is the long (q) arm of chromosome 1 at position 32.1Molecular Location: base pairs 203,216,079 to 203,229,732 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CHI3CHITCHITD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CHIT1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CHITINASE 1OMIM: CHITOTRIOSIDASE DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CHIT1  OMIM: CHITINASE 1OMIM: CHITOTRIOSIDASE DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KISS1\t   From NCBI Gene:This gene is a metastasis suppressor gene that suppresses metastases of melanomas and breast carcinomas without affecting tumorigenicity. The encoded protein may inhibit chemotaxis and invasion and thereby attenuate metastasis in malignant melanomas. Studies suggest a putative role in the regulation of events downstream of cell-matrix adhesion, perhaps involving cytoskeletal reorganization. A protein product of this gene, kisspeptin, stimulates gonadotropin-releasing hormone (GnRH)-induced gonadotropin secretion and regulates the pubertal activation of GnRH nuerons. A polymorphism in the terminal exon of this mRNA results in two protein isoforms. An adenosine present at the polymorphic site represents the third position in a stop codon. When the adenosine is absent, a downstream stop codon is utilized and the encoded protein extends for an additional seven amino acid residues. [provided by RefSeq, Mar 2012]From UniProt:Metastasis suppressor protein in malignant melanomas and in some breast cancers. May regulate events downstream of cell-matrix adhesion, perhaps involving cytoskeletal reorganization. Generates a C-terminally amidated peptide, metastin which functions as the endogenous ligand of the G-protein coupled receptor GPR54. Activation of the receptor inhibits cell proliferation and cell migration, key characteristics of tumor metastasis. Kp-10 is a decapeptide derived from the primary translation product, isolated in conditioned medium of first trimester trophoblast. Kp-10, but not other kisspeptins, increased intracellular Ca(2+) levels in isolated first trimester trophoblasts. Kp-10 is a paracrine/endocrine regulator in fine-tuning trophoblast invasion generated by the trophoblast itself. The receptor is also essential for normal gonadotropin-released hormone physiology and for puberty. The hypothalamic KiSS1/GPR54 system is a pivotal factor in central regulation of the gonadotropic axis at puberty and in adulthood.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Central precocious pubertyFrom NCBI Gene:Hypogonadotropic hypogonadism 13 with or without anosmiaFrom UniProt:Hypogonadotropic hypogonadism 13 with or without anosmia (HH13): A disorder characterized by absent or incomplete sexual maturation by the age of 18 years, in conjunction with low levels of circulating gonadotropins and testosterone and no other abnormalities of the hypothalamic-pituitary axis. In some cases, it is associated with non-reproductive phenotypes, such as anosmia, cleft palate, and sensorineural hearing loss. Anosmia or hyposmia is related to the absence or hypoplasia of the olfactory bulbs and tracts. Hypogonadism is due to deficiency in gonadotropin-releasing hormone and probably results from a failure of embryonic migration of gonadotropin-releasing hormone-synthesizing neurons. In the presence of anosmia, idiopathic hypogonadotropic hypogonadism is referred to as Kallmann syndrome, whereas in the presence of a normal sense of smell, it has been termed normosmic idiopathic hypogonadotropic hypogonadism (nIHH). [MIM:614842]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q32.1, which is the long (q) arm of chromosome 1 at position 32.1Molecular Location: base pairs 204,190,341 to 204,196,491 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HH13KiSS-1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KISS1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HYPOGONADOTROPIC HYPOGONADISM 13 WITH OR WITHOUT ANOSMIAOMIM: KISS1 METASTASIS SUPPRESSOR      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KISS1  OMIM: HYPOGONADOTROPIC HYPOGONADISM 13 WITH OR WITHOUT ANOSMIAOMIM: KISS1 METASTASIS SUPPRESSOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PLA2G2C\t\n",
            "PM20D1\t\n",
            "SRGAP2\t   From NCBI Gene:This locus encodes a member of the SLIT-ROBO Rho GTPase activating protein family. The encoded protein stimulates GTPase activity of Rac1, and plays a role in cortical neuron development. This locus has several paralogs on human chromosome 1 resulting from segmental duplication. While this locus itself is conserved among various species, the paralogs are found only in the genus Homo, and not in the genomes of non-human great apes. Alternatively spliced transcript variants have been described for this locus. [provided by RefSeq, Jul 2014]From UniProt:RAC1 GTPase activating protein (GAP) that binds and deforms membranes, and regulates actin dynamics to regulate cell migration and differentiation. Plays an important role in different aspects of neuronal morphogenesis and migration mainly during development of the cerebral cortex. This includes the biogenesis of neurites, where it is required for both axons and dendrites outgrowth, and the maturation of the dendritic spines. Also stimulates the branching of the leading process and negatively regulates neuron radial migration in the cerebral cortex. Its interaction and inhibition by SRGAP2C reduces the rate of spine maturation, alters dendritic spine morphology and density and indirectly increases neuronal migration. It may have implications for cognition, learning and memory. In non-neuronal cells, it may also play a role in cell migration by regulating the formation of lamellipodia and filopodia.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration disrupting SRGAP2 has been found in a patient with early infantile epileptic encephalopathy. Balanced translocation t(1;9)(q32;q13) (PubMed:22106086).     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q32.1, which is the long (q) arm of chromosome 1 at position 32.1Molecular Location: base pairs 206,203,556 to 206,464,443 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARHGAP34FNBP2SRGAP2ASRGAP3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SRGAP2      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: SLIT-ROBO RHO GTPase-ACTIVATING PROTEIN 2      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SRGAP2  OMIM: SLIT-ROBO RHO GTPase-ACTIVATING PROTEIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "VWA5B1\t\n",
            "CR2\t   From NCBI Gene:This gene encodes a membrane protein, which functions as a receptor for Epstein-Barr virus (EBV) binding on B and T lymphocytes. Genetic variations in this gene are associated with susceptibility to systemic lupus erythematosus type 9 (SLEB9). Alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Sep 2009]From UniProt:(Microbial infection) Acts as a receptor for Epstein-Barr virus.Receptor for complement C3, for the Epstein-Barr virus on human B-cells and T-cells and for HNRNPU (PubMed:7753047). Participates in B lymphocytes activation (PubMed:7753047).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Systemic lupus erythematosusCommon variable immune deficiencyFrom NCBI Gene:Common variable immunodeficiency 7Systemic lupus erythematosus 9Common variable immunodeficiency 2From UniProt:Systemic lupus erythematosus 9 (SLEB9): A chronic, relapsing, inflammatory, and often febrile multisystemic disorder of connective tissue, characterized principally by involvement of the skin, joints, kidneys and serosal membranes. It is of unknown etiology, but is thought to represent a failure of the regulatory mechanisms of the autoimmune system. The disease is marked by a wide range of system dysfunctions, an elevated erythrocyte sedimentation rate, and the formation of LE cells in the blood or bone marrow. [MIM:610927]Immunodeficiency, common variable, 7 (CVID7): A primary immunodeficiency characterized by antibody deficiency, hypogammaglobulinemia, recurrent bacterial infections and an inability to mount an antibody response to antigen. The defect results from a failure of B-cell differentiation and impaired secretion of immunoglobulins; the numbers of circulating B-cells is usually in the normal range, but can be low. [MIM:614699]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q32.2, which is the long (q) arm of chromosome 1 at position 32.2Molecular Location: base pairs 207,454,300 to 207,489,895 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C3DRCD21CRCVID7SLEB9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CR2      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: COMPLEMENT COMPONENT RECEPTOR 2OMIM: IMMUNODEFICIENCY, COMMON VARIABLE, 2OMIM: IMMUNODEFICIENCY, COMMON VARIABLE, 7OMIM: SYSTEMIC LUPUS ERYTHEMATOSUS, SUSCEPTIBILITY TO, 9      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CR2  OMIM: COMPLEMENT COMPONENT RECEPTOR 2OMIM: IMMUNODEFICIENCY, COMMON VARIABLE, 2OMIM: IMMUNODEFICIENCY, COMMON VARIABLE, 7OMIM: SYSTEMIC LUPUS ERYTHEMATOSUS, SUSCEPTIBILITY TO, 9  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CR1\t   From NCBI Gene:This gene is a member of the receptors of complement activation (RCA) family and is located in the 'cluster RCA' region of chromosome 1. The gene encodes a monomeric single-pass type I membrane glycoprotein found on erythrocytes, leukocytes, glomerular podocytes, and splenic follicular dendritic cells. The Knops blood group system is a system of antigens located on this protein. The protein mediates cellular binding to particles and immune complexes that have activated complement. Decreases in expression of this protein and/or mutations in its gene have been associated with gallbladder carcinomas, mesangiocapillary glomerulonephritis, systemic lupus erythematosus and sarcoidosis. Mutations in this gene have also been associated with a reduction in Plasmodium falciparum rosetting, conferring protection against severe malaria. Alternate allele-specific splice variants, encoding different isoforms, have been characterized. Additional allele specific isoforms, including a secreted form, have been described but have not been fully characterized. [provided by RefSeq, Jul 2008]From UniProt:(Microbial infection) Acts as a receptor for Epstein-Barr virus.Membrane immune adherence receptor that plays a critical role in the capture and clearance of complement-opsonized pathogens by erythrocytes and monocytes/macrophages (PubMed:2963069). Mediates the binding by these cells of particles and immune complexes that have activated complement to eliminate them from the circulation (PubMed:2963069). Acts also in the inhibition of spontaneous complement activation by impairing the formation and function of the alternative and classical pathway C3/C5 convertases, and by serving as a cofactor for the cleavage by factor I of C3b to iC3b, C3c and C3d,g, and of C4b to C4c and C4d (PubMed:2972794, PubMed:8175757). Plays also a role in immune regulation by contributing, upon ligand binding, to the generation of regulatory T cells from activated helper T cells (PubMed:25742728).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:C3 glomerulopathyFrom NCBI Gene:Susceptibility to malariaKNOPS BLOOD GROUP SYSTEM     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q32.2, which is the long (q) arm of chromosome 1 at position 32.2Molecular Location: base pairs 207,496,128 to 207,641,765 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C3BRC4BRCD35KN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CR1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: COMPLEMENT COMPONENT RECEPTOR 1OMIM: KNOPS BLOOD GROUP SYSTEMOMIM: MALARIA, SUSCEPTIBILITY TO      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CR1  OMIM: COMPLEMENT COMPONENT RECEPTOR 1OMIM: KNOPS BLOOD GROUP SYSTEMOMIM: MALARIA, SUSCEPTIBILITY TO  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CR1L\t\n",
            "CD34\t\n",
            "SYT14\t   From NCBI Gene:This gene is a member of the synaptotagmin gene family and encodes a protein similar to other family members that mediate membrane trafficking in synaptic transmission. The encoded protein is a calcium-independent synaptotagmin. Mutations in this gene are a cause of autosomal recessive spinocerebellar ataxia-11 (SCAR11), and a t(1;3) translocation of this gene has been associated with neurodevelopmental abnormalities. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 4. [provided by RefSeq, Dec 2011]From UniProt:May be involved in the trafficking and exocytosis of secretory vesicles in non-neuronal tissues. Is Ca(2+)-independent.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spinocerebellar ataxia, autosomal recessive 11From UniProt:Spinocerebellar ataxia, autosomal recessive, 11 (SCAR11): Spinocerebellar ataxia is a clinically and genetically heterogeneous group of cerebellar disorders. Patients show progressive incoordination of gait and often poor coordination of hands, speech and eye movements, due to degeneration of the cerebellum with variable involvement of the brainstem and spinal cord. SCAR11 is associated with psychomotor retardation. [MIM:614229]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q32.2, which is the long (q) arm of chromosome 1 at position 32.2Molecular Location: base pairs 209,938,165 to 210,171,315 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        SCAR11sytXIV     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SYT14      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SPINOCEREBELLAR ATAXIA, AUTOSOMAL RECESSIVE 11OMIM: SYNAPTOTAGMIN 14      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SYT14  OMIM: SPINOCEREBELLAR ATAXIA, AUTOSOMAL RECESSIVE 11OMIM: SYNAPTOTAGMIN 14  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HHAT\t\n",
            "NBPF3\t\n",
            "USP48\t\n",
            "HLX\t\n",
            "HSPG2\t   The HSPG2 gene provides instructions for making a protein called perlecan. This protein is found in the extracellular matrix, which is the intricate lattice of proteins and other molecules that forms in the spaces between cells. Specifically, it is found in part of the extracellular matrix called the basement membrane, which is a thin, sheet-like structure that separates and supports cells in many tissues. Perlecan is also found in cartilage, a tough, flexible tissue that makes up much of the skeleton during early development. Most cartilage is later converted to bone, except for the cartilage that continues to cover and protect the ends of bones and is present in the nose and external ears.Perlecan is a heparan sulfate proteoglycan, which is a type of protein that interacts with many other proteins and has a variety of functions. In particular, perlecan is involved in cell signaling, the sticking (adhesion) of cells to one another, the formation of new blood vessels (angiogenesis), and the maintenance of basement membranes and cartilage throughout life. The protein also plays a critical role at the neuromuscular junction, which is the area between the ends of nerve cells and muscle cells where signals are relayed to trigger muscle contraction.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Schwartz-Jampel syndrome     More than 30 mutations in the HSPG2 gene have been found to cause Schwartz-Jampel syndrome. This rare condition is characterized by continuous muscle contraction (myotonia) that restricts movement, as well as bone abnormalities known as chondrodysplasia. Most of the mutations reduce the amount of perlecan that is produced. Other mutations lead to a version of perlecan that is only partially functional. A reduction in the amount or function of this protein disrupts the normal development of cartilage and bone tissue, which underlies chondrodysplasia in affected individuals. A reduced amount of functional perlecan at the neuromuscular junction likely alters the balance of other molecules that signal when muscles should contract and when they should relax. As a result, muscle contraction is triggered continuously, leading to myotonia. More About This Health Condition      Other disorders     Mutations in the HSPG2 gene can also cause another, more severe form of chondrodysplasia called dyssegmental dysplasia, Silverman-Handmaker type. Because of the very severe abnormalities associated with this rare condition, most affected individuals die before birth, are stillborn, or live only into early infancy. At least seven HSPG2 gene mutations have been identified in people with dyssegmental dysplasia, Silverman-Handmaker type. These mutations prevent the production or transport of any functional perlecan. A total lack of this protein significantly disrupts the development of cartilage and bone tissue, causing this very severe type of chondrodysplasia.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 30 mutations in the HSPG2 gene have been found to cause Schwartz-Jampel syndrome. This rare condition is characterized by continuous muscle contraction (myotonia) that restricts movement, as well as bone abnormalities known as chondrodysplasia. Most of the mutations reduce the amount of perlecan that is produced. Other mutations lead to a version of perlecan that is only partially functional. A reduction in the amount or function of this protein disrupts the normal development of cartilage and bone tissue, which underlies chondrodysplasia in affected individuals. A reduced amount of functional perlecan at the neuromuscular junction likely alters the balance of other molecules that signal when muscles should contract and when they should relax. As a result, muscle contraction is triggered continuously, leading to myotonia. More About This Health Condition  Mutations in the HSPG2 gene can also cause another, more severe form of chondrodysplasia called dyssegmental dysplasia, Silverman-Handmaker type. Because of the very severe abnormalities associated with this rare condition, most affected individuals die before birth, are stillborn, or live only into early infancy. At least seven HSPG2 gene mutations have been identified in people with dyssegmental dysplasia, Silverman-Handmaker type. These mutations prevent the production or transport of any functional perlecan. A total lack of this protein significantly disrupts the development of cartilage and bone tissue, causing this very severe type of chondrodysplasia.    Cytogenetic Location: 1p36.12, which is the short (p) arm of chromosome 1 at position 36.12Molecular Location: base pairs 21,822,232 to 21,937,297 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        endorepellin (domain V region)perlecanperlecan proteoglycanPLCPRCAN     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Essentials of Glycobiology (second edition, 2009): Proteoglycans and Sulfated Glycosaminoglycans      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HSPG2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     DYSSEGMENTAL DYSPLASIA, SILVERMAN-HANDMAKER TYPEHEPARAN SULFATE PROTEOGLYCAN OF BASEMENT MEMBRANE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Essentials of Glycobiology (second edition, 2009): Proteoglycans and Sulfated Glycosaminoglycans  Tests of HSPG2  PubMed  DYSSEGMENTAL DYSPLASIA, SILVERMAN-HANDMAKER TYPEHEPARAN SULFATE PROTEOGLYCAN OF BASEMENT MEMBRANE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Arikawa-Hirasawa E, Le AH, Nishino I, Nonaka I, Ho NC, Francomano CA, Govindraj P, Hassell JR, Devaney JM, Spranger J, Stevenson RE, Iannaccone S, Dalakas MC, Yamada Y. Structural and functional mutations of the perlecan gene cause Schwartz-Jampel syndrome, with myotonic myopathy and chondrodysplasia. Am J Hum Genet. 2002 May;70(5):1368-75. Epub 2002 Apr 8. Citation on PubMed or Free article on PubMed CentralArikawa-Hirasawa E, Wilcox WR, Le AH, Silverman N, Govindraj P, Hassell JR, Yamada Y. Dyssegmental dysplasia, Silverman-Handmaker type, is caused by functional null mutations of the perlecan gene. Nat Genet. 2001 Apr;27(4):431-4. Citation on PubMedArikawa-Hirasawa E, Wilcox WR, Yamada Y. Dyssegmental dysplasia, Silverman-Handmaker type: unexpected role of perlecan in cartilage development. Am J Med Genet. 2001 Winter;106(4):254-7. Review. Citation on PubMedOMIM: HEPARAN SULFATE PROTEOGLYCAN OF BASEMENT MEMBRANELadhani NN, Chitayat D, Nezarati MM, Laureane MC, Keating S, Silver RJ, Unger S, Velsher L, Sirkin W, Toi A, Glanc P. Dyssegmental dysplasia, Silverman-Handmaker type: prenatal ultrasound findings and molecular analysis. Prenat Diagn. 2013 Nov;33(11):1039-43. doi: 10.1002/pd.4193. Epub 2013 Aug 4. Citation on PubMedMelrose J, Hayes AJ, Whitelock JM, Little CB. Perlecan, the \"jack of all trades\" proteoglycan of cartilaginous weight-bearing connective tissues. Bioessays. 2008 May;30(5):457-69. doi: 10.1002/bies.20748. Review. Citation on PubMedNicole S, Davoine CS, Topaloglu H, Cattolico L, Barral D, Beighton P, Hamida CB, Hammouda H, Cruaud C, White PS, Samson D, Urtizberea JA, Lehmann-Horn F, Weissenbach J, Hentati F, Fontaine B. Perlecan, the major proteoglycan of basement membranes, is altered in patients with Schwartz-Jampel syndrome (chondrodystrophic myotonia). Nat Genet. 2000 Dec;26(4):480-3. Citation on PubMedStum M, Davoine CS, Fontaine B, Nicole S. Schwartz-Jampel syndrome and perlecan deficiency. Acta Myol. 2005 Oct;24(2):89-92. Review. Citation on PubMedStum M, Davoine CS, Vicart S, Guillot-Nol L, Topaloglu H, Carod-Artal FJ, Kayserili H, Hentati F, Merlini L, Urtizberea JA, Hammouda el-H, Quan PC, Fontaine B, Nicole S. Spectrum of HSPG2 (Perlecan) mutations in patients with Schwartz-Jampel syndrome. Hum Mutat. 2006 Nov;27(11):1082-91. Citation on PubMed \n",
            "CELA3B\t\n",
            "CAPN8\t\n",
            "DNAH14\t\n",
            "EPHX1\t   From NCBI Gene:Epoxide hydrolase is a critical biotransformation enzyme that converts epoxides from the degradation of aromatic compounds to trans-dihydrodiols which can be conjugated and excreted from the body. Epoxide hydrolase functions in both the activation and detoxification of epoxides. Mutations in this gene cause preeclampsia, epoxide hydrolase deficiency or increased epoxide hydrolase activity. Alternatively spliced transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Dec 2008]From UniProt:Biotransformation enzyme that catalyzes the hydrolysis of arene and aliphatic epoxides to less reactive and more water soluble dihydrodiols by the trans addition of water (By similarity). May play a role in the metabolism of endogenous lipids such as epoxide-containing fatty acids (PubMed:22798687).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Hypercholanemia, familialFrom UniProt:Familial hypercholanemia (FHCA): A disorder characterized by elevated serum bile acid concentrations, itching, and fat malabsorption. [MIM:607748]In some populations, the high activity haplotype tyr113/his139 is overrepresented among women suffering from pregnancy-induced hypertension (pre-eclampsia) when compared with healthy controls.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q42.12, which is the long (q) arm of chromosome 1 at position 42.12Molecular Location: base pairs 225,810,074 to 225,845,563 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        EPHXEPOXHYL1MEH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of EPHX1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: EPOXIDE HYDROLASE 1, MICROSOMALOMIM: HYPERCHOLANEMIA, FAMILIAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of EPHX1  OMIM: EPOXIDE HYDROLASE 1, MICROSOMALOMIM: HYPERCHOLANEMIA, FAMILIAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMEM63A\t\n",
            "PARP1\t\n",
            "ITPKB\t\n",
            "CDC42BPA\t\n",
            "PRSS38\t\n",
            "OBSCN\t   From NCBI Gene:The obscurin gene spans more than 150 kb, contains over 80 exons and encodes a protein of approximately 720 kDa. The encoded protein contains 68 Ig domains, 2 fibronectin domains, 1 calcium/calmodulin-binding domain, 1 RhoGEF domain with an associated PH domain, and 2 serine-threonine kinase domains. This protein belongs to the family of giant sacromeric signaling proteins that includes titin and nebulin, and may have a role in the organization of myofibrils during assembly and may mediate interactions between the sarcoplasmic reticulum and myofibrils. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]From UniProt:Structural component of striated muscles which plays a role in myofibrillogenesis. Probably involved in the assembly of myosin into sarcomeric A bands in striated muscle (PubMed:11448995, PubMed:16205939). Has serine/threonine protein kinase activity and phosphorylates N-cadherin CDH2 and sodium/potassium-transporting ATPase subunit ATP1B1 (By similarity). Binds (via the PH domain) strongly to phosphatidylinositol 3,4-bisphosphate (PtdIns(3,4)P2) and phosphatidylinositol 4,5-bisphosphate (PtdIns(4,5)P2), and to a lesser extent to phosphatidylinositol 3-phosphate (PtdIns(3)P), phosphatidylinositol 4-phosphate (PtdIns(4)P), phosphatidylinositol 5-phosphate (PtdIns(5)P) and phosphatidylinositol 3,4,5-trisphosphate (PtdIns(3,4,5)P3) (PubMed:28826662).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving OBSCN has been found in Wilms tumor. Translocation t(1;7)(q42;p15) with PTHB1.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q42.13, which is the long (q) arm of chromosome 1 at position 42.13Molecular Location: base pairs 228,201,321 to 228,381,431 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARHGEF30UNC89     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of OBSCN      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: OBSCURIN      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of OBSCN  OMIM: OBSCURIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "COG2\t   From NCBI Gene:This gene encodes a subunit of the conserved oligomeric Golgi complex that is required for maintaining normal structure and activity of the Golgi complex. The encoded protein specifically interacts with the USO1 vesicle docking protein and may be necessary for normal Golgi ribbon formation and trafficking of Golgi enzymes. Mutations of this gene are associated with abnormal glycosylation within the Golgi apparatus. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Feb 2009]From UniProt:Required for normal Golgi morphology and function.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IIqFrom UniProt:Congenital disorder of glycosylation 2Q (CDG2Q): A form of congenital disorder of glycosylation, a genetically heterogeneous group of autosomal recessive, multisystem disorders caused by a defect in glycoprotein biosynthesis and characterized by under-glycosylated serum glycoproteins. Congenital disorders of glycosylation result in a wide variety of clinical features, such as defects in the nervous system development, psychomotor retardation, dysmorphic features, hypotonia, coagulation disorders, and immunodeficiency. The broad spectrum of features reflects the critical role of N-glycoproteins during embryonic development, differentiation, and maintenance of cell functions. The transmission pattern of CDG2Q is consistent with autosomal recessive inheritance. [MIM:617395]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q42.2, which is the long (q) arm of chromosome 1 at position 42.2Molecular Location: base pairs 230,642,456 to 230,693,985 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDG2QLDLC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COG2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: COMPONENT OF OLIGOMERIC GOLGI COMPLEX 2OMIM: CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IIq      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COG2  OMIM: COMPONENT OF OLIGOMERIC GOLGI COMPLEX 2OMIM: CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IIq  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "AGT\t   The AGT gene provides instructions for making a protein called angiotensinogen. This protein is part of the renin-angiotensin system, which regulates blood pressure and the balance of fluids and salts in the body. In the first step of this process, angiotensinogen is converted to angiotensin I. Through an additional step, angiotensin I is converted to angiotensin II. Angiotensin II causes blood vessels to narrow (constrict), which results in increased blood pressure. This molecule also stimulates production of the hormone aldosterone, which triggers the absorption of salt and water by the kidneys. The increased amount of fluid in the body also increases blood pressure. Proper blood pressure during fetal growth, which delivers oxygen to the developing tissues, is required for normal development of the kidneys, particularly of structures called the proximal tubules, and other tissues. In addition, angiotensin II may play a more direct role in kidney development, perhaps by affecting growth factors involved in the development of kidney structures.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Renal tubular dysgenesis     At least six mutations in the AGT gene have been found to cause a severe kidney disorder called renal tubular dysgenesis. This condition is characterized by abnormal kidney development before birth, the inability to produce urine (anuria), and severe low blood pressure (hypotension). These problems result in a reduction of amniotic fluid (oligohydramnios), which leads to a set of birth defects known as the Potter sequence.Renal tubular dysgenesis can be caused by mutations in both copies of any of the genes involved in the renin-angiotensin system. Most of the mutations in the AGT gene that cause this disorder change single protein building blocks (amino acids) in the angiotensinogen protein. These changes occur in a region of the protein that is necessary for its conversion to angiotensin I. It is thought that the altered angiotensinogen cannot be converted, leading to a nonfunctional renin-angiotensin system. Without this system, the kidneys cannot control blood pressure. Because of low blood pressure, the flow of blood is reduced (hypoperfusion), and the body does not get enough oxygen during fetal development. As a result, kidney development is impaired, leading to the features of renal tubular dysgenesis. More About This Health Condition      Hypertension     Genetics Home Reference provides information about hypertension. More About This Health Condition      Other disorders     Variations in the AGT gene are associated with susceptibility to a form of high blood pressure (hypertension) called essential hypertension. Essential hypertension is a complex disorder associated with many genetic and environmental factors. The AGT gene variations associated with this condition affect single DNA building blocks (nucleotides) and likely lead to higher levels of the angiotensinogen protein.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least six mutations in the AGT gene have been found to cause a severe kidney disorder called renal tubular dysgenesis. This condition is characterized by abnormal kidney development before birth, the inability to produce urine (anuria), and severe low blood pressure (hypotension). These problems result in a reduction of amniotic fluid (oligohydramnios), which leads to a set of birth defects known as the Potter sequence.Renal tubular dysgenesis can be caused by mutations in both copies of any of the genes involved in the renin-angiotensin system. Most of the mutations in the AGT gene that cause this disorder change single protein building blocks (amino acids) in the angiotensinogen protein. These changes occur in a region of the protein that is necessary for its conversion to angiotensin I. It is thought that the altered angiotensinogen cannot be converted, leading to a nonfunctional renin-angiotensin system. Without this system, the kidneys cannot control blood pressure. Because of low blood pressure, the flow of blood is reduced (hypoperfusion), and the body does not get enough oxygen during fetal development. As a result, kidney development is impaired, leading to the features of renal tubular dysgenesis. More About This Health Condition  Genetics Home Reference provides information about hypertension. More About This Health Condition  Variations in the AGT gene are associated with susceptibility to a form of high blood pressure (hypertension) called essential hypertension. Essential hypertension is a complex disorder associated with many genetic and environmental factors. The AGT gene variations associated with this condition affect single DNA building blocks (nucleotides) and likely lead to higher levels of the angiotensinogen protein.    Cytogenetic Location: 1q42.2, which is the long (q) arm of chromosome 1 at position 42.2Molecular Location: base pairs 230,702,523 to 230,714,590 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        angiotensinogen (serpin peptidase inhibitor, clade A, member 8)angiotensinogen preproproteinANGT_HUMANANHUpre-angiotensinogenserpin A8SERPINA8     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Merck Manual Consumer Version: The Body's Control of Blood Pressure      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGT      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     ANGIOTENSINOGENHYPERTENSION, ESSENTIAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Merck Manual Consumer Version: The Body's Control of Blood Pressure  Tests of AGT  PubMed  ANGIOTENSINOGENHYPERTENSION, ESSENTIAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ANGIOTENSINOGENGribouval O, Gonzales M, Neuhaus T, Aziza J, Bieth E, Laurent N, Bouton JM, Feuillet F, Makni S, Ben Amar H, Laube G, Delezoide AL, Bouvier R, Dijoud F, Ollagnon-Roman E, Roume J, Joubert M, Antignac C, Gubler MC. Mutations in genes in the renin-angiotensin system are associated with autosomal recessive renal tubular dysgenesis. Nat Genet. 2005 Sep;37(9):964-8. Epub 2005 Aug 14. Citation on PubMedGribouval O, Morinire V, Pawtowski A, Arrondel C, Sallinen SL, Saloranta C, Clericuzio C, Viot G, Tantau J, Blesson S, Cloarec S, Machet MC, Chitayat D, Thauvin C, Laurent N, Sampson JR, Bernstein JA, Clemenson A, Prieur F, Daniel L, Levy-Mozziconacci A, Lachlan K, Alessandri JL, Cartault F, Rivire JP, Picard N, Baumann C, Delezoide AL, Belar Ortega M, Chassaing N, Labrune P, Yu S, Firth H, Wellesley D, Bitzan M, Alfares A, Braverman N, Krogh L, Tolmie J, Gaspar H, Doray B, Majore S, Bonneau D, Triau S, Loirat C, David A, Bartholdi D, Peleg A, Brackman D, Stone R, DeBerardinis R, Corvol P, Michaud A, Antignac C, Gubler MC. Spectrum of mutations in the renin-angiotensin system genes in autosomal recessive renal tubular dysgenesis. Hum Mutat. 2012 Feb;33(2):316-26. doi: 10.1002/humu.21661. Epub 2011 Dec 22. Review. Citation on PubMedGubler MC, Antignac C. Renin-angiotensin system in kidney development: renal tubular dysgenesis. Kidney Int. 2010 Mar;77(5):400-6. doi: 10.1038/ki.2009.423. Epub 2009 Nov 18. Review. Citation on PubMedWatkins WS, Hunt SC, Williams GH, Tolpinrud W, Jeunemaitre X, Lalouel JM, Jorde LB. Genotype-phenotype analysis of angiotensinogen polymorphisms and essential hypertension: the importance of haplotypes. J Hypertens. 2010 Jan;28(1):65-75. doi: 10.1097/HJH.0b013e328332031a. Citation on PubMed or Free article on PubMed CentralWolf G. Angiotensin II and tubular development. Nephrol Dial Transplant. 2002;17 Suppl 9:48-51. Review. Citation on PubMed \n",
            "CAPN9\t\n",
            "DISC1\t   From NCBI Gene:This gene encodes a protein with multiple coiled coil motifs which is located in the nucleus, cytoplasm and mitochondria. The protein is involved in neurite outgrowth and cortical development through its interaction with other proteins. This gene is disrupted in a t(1;11)(q42.1;q14.3) translocation which segregates with schizophrenia and related psychiatric disorders in a large Scottish family. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Jul 2008]From UniProt:Involved in the regulation of multiple aspects of embryonic and adult neurogenesis. Required for neural progenitor proliferation in the ventrical/subventrical zone during embryonic brain development and in the adult dentate gyrus of the hippocampus. Participates in the Wnt-mediated neural progenitor proliferation as a positive regulator by modulating GSK3B activity and CTNNB1 abundance. Plays a role as a modulator of the AKT-mTOR signaling pathway controlling the tempo of the process of newborn neurons integration during adult neurogenesis, including neuron positioning, dendritic development and synapse formation. Inhibits the activation of AKT-mTOR signaling upon interaction with CCDC88A. Regulates the migration of early-born granule cell precursors toward the dentate gyrus during the hippocampal development. Plays a role, together with PCNT, in the microtubule network formation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Schizophrenia 9From UniProt:A chromosomal aberration involving DISC1 segregates with schizophrenia and related psychiatric disorders in a large Scottish family. Translocation t(1;11)(q42.1;q14.3). The truncated DISC1 protein produced by this translocation is unable to interact with ATF4, ATF5 and NDEL1.Schizophrenia 9 (SCZD9): A complex, multifactorial psychotic disorder or group of disorders characterized by disturbances in the form and content of thought (e.g. delusions, hallucinations), in mood (e.g. inappropriate affect), in sense of self and relationship to the external world (e.g. loss of ego boundaries, withdrawal), and in behavior (e.g bizarre or apparently purposeless behavior). Although it affects emotions, it is distinguished from mood disorders in which such disturbances are primary. Similarly, there may be mild impairment of cognitive function, and it is distinguished from the dementias in which disturbed cognitive function is considered primary. Some patients manifest schizophrenic as well as bipolar disorder symptoms and are often given the diagnosis of schizoaffective disorder. [MIM:604906]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q42.2, which is the long (q) arm of chromosome 1 at position 42.2Molecular Location: base pairs 231,626,815 to 232,041,272 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C1orf136SCZD9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DISC1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DISRUPTED IN SCHIZOPHRENIA 1OMIM: SCHIZOPHRENIA 9      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DISC1  OMIM: DISRUPTED IN SCHIZOPHRENIA 1OMIM: SCHIZOPHRENIA 9  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HEATR1\t\n",
            "ACTN2\t   From NCBI Gene:Alpha actinins belong to the spectrin gene superfamily which represents a diverse group of cytoskeletal proteins, including the alpha and beta spectrins and dystrophins. Alpha actinin is an actin-binding protein with multiple roles in different cell types. In nonmuscle cells, the cytoskeletal isoform is found along microfilament bundles and adherens-type junctions, where it is involved in binding actin to the membrane. In contrast, skeletal, cardiac, and smooth muscle isoforms are localized to the Z-disc and analogous dense bodies, where they help anchor the myofibrillar actin filaments. This gene encodes a muscle-specific, alpha actinin isoform that is expressed in both skeletal and cardiac muscles. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2013]From UniProt:F-actin cross-linking protein which is thought to anchor actin to a variety of intracellular structures. This is a bundling protein.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Familial hypertrophic cardiomyopathyFamilial dilated cardiomyopathyFrom NCBI Gene:Dilated cardiomyopathy 1AAFrom UniProt:Cardiomyopathy, dilated 1AA, with or without left ventricular non-compaction (CMD1AA): A disorder characterized by ventricular dilation and impaired systolic function, resulting in congestive heart failure and arrhythmia. Patients are at risk of premature death. [MIM:612158]Cardiomyopathy, familial hypertrophic 23, with or without left ventricular non-compaction (CMH23): A hereditary heart disorder characterized by ventricular hypertrophy, which is usually asymmetric and often involves the interventricular septum. The symptoms include dyspnea, syncope, collapse, palpitations, and chest pain. They can be readily provoked by exercise. The disorder has inter- and intrafamilial variability ranging from benign to malignant forms with high risk of cardiac failure and sudden cardiac death. [MIM:612158]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q43, which is the long (q) arm of chromosome 1 at position 43Molecular Location: base pairs 236,686,454 to 236,764,631 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMD1AACMH23     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ACTN2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ACTININ, ALPHA-2OMIM: CARDIOMYOPATHY, DILATED, 1AA, WITH OR WITHOUT LEFT VENTRICULAR NONCOMPACTION      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ACTN2  OMIM: ACTININ, ALPHA-2OMIM: CARDIOMYOPATHY, DILATED, 1AA, WITH OR WITHOUT LEFT VENTRICULAR NONCOMPACTION  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ASAP3\t\n",
            "E2F2\t\n",
            "FMN2\t   From NCBI Gene:This gene is a member of the formin homology protein family. The encoded protein is thought to have essential roles in organization of the actin cytoskeleton and in cell polarity. This protein mediates the formation of an actin mesh that positions the spindle during oogenesis and also regulates the formation of actin filaments in the nucleus. This protein also forms a perinuclear actin/focal-adhesion system that regulates the shape and position of the nucleus during cell migration. Mutations in this gene have been associated with infertility and also with an autosomal recessive form of intellectual disability (MRT47). Alternatively spliced transcript variants have been identified. [provided by RefSeq, Jul 2017]From UniProt:Actin-binding protein that is involved in actin cytoskeleton assembly and reorganization (PubMed:22330775, PubMed:21730168). Acts as an actin nucleation factor and promotes assembly of actin filaments together with SPIRE1 and SPIRE2 (PubMed:22330775, PubMed:21730168). Involved in intracellular vesicle transport along actin fibers, providing a novel link between actin cytoskeleton dynamics and intracellular transport (By similarity). Required for asymmetric spindle positioning, asymmetric oocyte division and polar body extrusion during female germ cell meiosis (By similarity). Plays a role in responses to DNA damage, cellular stress and hypoxia by protecting CDKN1A against degradation, and thereby plays a role in stress-induced cell cycle arrest (PubMed:23375502). Also acts in the nucleus: together with SPIRE1 and SPIRE2, promotes assembly of nuclear actin filaments in response to DNA damage in order to facilitate movement of chromatin and repair factors after DNA damage (PubMed:26287480). Protects cells against apoptosis by protecting CDKN1A against degradation (PubMed:23375502).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mental retardation, autosomal recessive 47From UniProt:Mental retardation, autosomal recessive 47 (MRT47): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. MRT47 patients show delayed development, with cognition and speech more affected than motor skills. [MIM:616193]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1q43, which is the long (q) arm of chromosome 1 at position 43Molecular Location: base pairs 240,091,885 to 240,475,189 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FMN2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FORMIN 2OMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 47      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FMN2  OMIM: FORMIN 2OMIM: MENTAL RETARDATION, AUTOSOMAL RECESSIVE 47  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "EXO1\t\n",
            "MYOM3\t\n",
            "GRHL3\t   From NCBI Gene:This gene encodes a member of the grainyhead family of transcription factors. The encoded protein may function as a transcription factor during development, and has been shown to stimulate migration of endothelial cells. Multiple transcript variants encoding distinct isoforms have been identified for this gene.[provided by RefSeq, Aug 2010]From UniProt:Transcription factor playing important roles in primary neurulation and in the differentiation of stratified epithelia of both ectodermal and endodermal origin (By similarity). Binds directly to the consensus DNA sequence 5'-AACCGGTT-3' acting as an activator and repressor on distinct target genes (PubMed:21081122, PubMed:25347468). xhibits functional redundancy with GRHL2 in epidermal morphogenetic events and epidermal wound repair (By similarity). Exhibits functional redundancy with GRHL2 in epidermal morphogenetic events and epidermal wound repair but is essential to form the epidermal barrier with TGM3 as critical direct target gene among others. Despite being dispensable during normal epidermal homeostasis in the adulthood, is again required for barrier repair after immune-mediated epidermal damage, regulates distinct gene batteries in embryonic epidermal differentiation and adult epidermal barrier reformation after injury. Plays unique and cooperative roles with GRHL2 in establishing distinct zones of primary neurulation. Essential for spinal closure, functions cooperatively with GRHL2 in closure 2 (forebrain/midbrain boundary) and posterior neuropore closure (By similarity). Also required for proper development of the oral periderm (PubMed:24360809). No genetic interaction with GRHL3, no functional cooperativity due to diverse target gene selectivity (PubMed:21081122).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Van der Woude syndrome 2From UniProt:Van der Woude syndrome 2 (VWS2): An autosomal dominant developmental disorder characterized by lower lip pits, cleft lip and/or cleft palate. [MIM:606713]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.11, which is the short (p) arm of chromosome 1 at position 36.11Molecular Location: base pairs 24,319,322 to 24,364,482 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        SOMTFCP2L4VWS2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GRHL3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GRAINYHEAD-LIKE 3OMIM: VAN DER WOUDE SYNDROME 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GRHL3  OMIM: GRAINYHEAD-LIKE 3OMIM: VAN DER WOUDE SYNDROME 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OR2B11\t\n",
            "OR2G2\t\n",
            "OR2G3\t\n",
            "OR14A16\t\n",
            "OR11L1\t\n",
            "OR2W3\t\n",
            "OR2T8\t\n",
            "OR2L8\t\n",
            "OR2AK2\t\n",
            "OR2L3\t\n",
            "OR2M2\t\n",
            "OR2T33\t\n",
            "OR2M7\t\n",
            "OR14C36\t\n",
            "OR2T4\t\n",
            "OR2T2\t\n",
            "OR2G6\t\n",
            "OR2T35\t\n",
            "OR2T27\t\n",
            "OR14I1\t\n",
            "UBXN11\t\n",
            "ZNF683\t\n",
            "SMPDL3B\t\n",
            "SERINC2\t\n",
            "EPHA10\t\n",
            "RHBDL2\t\n",
            "MACF1\t\n",
            "FOXJ3\t\n",
            "ERMAP\t   From NCBI Gene:The protein encoded by this gene is a cell surface transmembrane protein that may act as an erythroid cell receptor, possibly as a mediator of cell adhesion. Polymorphisms in this gene are responsible for the Scianna/Radin blood group system. Two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Possible role as a cell-adhesion or receptor molecule of erythroid cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Antigen in Scianna blood group systemRadin blood group     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p34.2, which is the short (p) arm of chromosome 1 at position 34.2Molecular Location: base pairs 42,817,105 to 42,844,989 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BTN5PRO2801RDSC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ERMAP      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BLOOD GROUP--SCIANNA SYSTEMOMIM: ERYTHROBLAST MEMBRANE-ASSOCIATED PROTEINOMIM: RADIN BLOOD GROUP ANTIGEN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ERMAP  OMIM: BLOOD GROUP--SCIANNA SYSTEMOMIM: ERYTHROBLAST MEMBRANE-ASSOCIATED PROTEINOMIM: RADIN BLOOD GROUP ANTIGEN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KLF17\t\n",
            "MAST2\t\n",
            "MOB3C\t\n",
            "AJAP1\t\n",
            "STIL\t   From NCBI Gene:This gene encodes a cytoplasmic protein implicated in regulation of the mitotic spindle checkpoint, a regulatory pathway that monitors chromosome segregation during cell division to ensure the proper distribution of chromosomes to daughter cells. The protein is phosphorylated in mitosis and in response to activation of the spindle checkpoint, and disappears when cells transition to G1 phase. It interacts with a mitotic regulator, and its expression is required to efficiently activate the spindle checkpoint. It is proposed to regulate Cdc2 kinase activity during spindle checkpoint arrest. Chromosomal deletions that fuse this gene and the adjacent locus commonly occur in T cell leukemias, and are thought to arise through illegitimate V-(D)-J recombination events. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Immediate-early gene. Plays an important role in embryonic development as well as in cellular growth and proliferation; its long-term silencing affects cell survival and cell cycle distribution as well as decreases CDK1 activity correlated with reduced phosphorylation of CDK1. Plays a role as a positive regulator of the sonic hedgehog pathway, acting downstream of PTCH1 (PubMed:16024801, PubMed:9372240). Plays an important role in the regulation of centriole duplication. Required for the onset of procentriole formation and proper mitotic progression. During procentriole formation, is essential for the correct loading of SASS6 and CENPJ to the base of the procentriole to initiate procentriole assembly (PubMed:22020124).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autosomal recessive primary microcephalyFrom NCBI Gene:Primary autosomal recessive microcephaly 7From UniProt:Microcephaly 7, primary, autosomal recessive (MCPH7): A disease defined as a head circumference more than 3 standard deviations below the age-related mean. Brain weight is markedly reduced and the cerebral cortex is disproportionately small. Despite this marked reduction in size, the gyral pattern is relatively well preserved, with no major abnormality in cortical architecture. Affected individuals are mentally retarded. Primary microcephaly is further defined by the absence of other syndromic features or significant neurological deficits due to degenerative brain disorder. [MIM:612703]A chromosomal aberration involving STIL may be a cause of some T-cell acute lymphoblastic leukemias (T-ALL). A deletion at 1p32 between STIL and TAL1 genes leads to STIL/TAL1 fusion mRNA with STIL exon 1 splicing to TAL1 exon 3. As both STIL exon 1 and TAL1 exon 3 are 5'-untranslated exons, STIL/TAL1 fusion mRNA predicts a full-length TAL1 protein under the control of the STIL promoter, leading to inappropriate TAL1 expression. In childhood T-cell malignancies (T-ALL), a type of defect such as STIL/TAL1 fusion is associated with a good prognosis. In cultured lymphocytes from healthy adults, STIL/TAL1 fusion mRNA may be detected after 7 days of culture.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p33, which is the short (p) arm of chromosome 1 at position 33Molecular Location: base pairs 47,250,139 to 47,314,787 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MCPH7SIL     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of STIL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MICROCEPHALY 7, PRIMARY, AUTOSOMAL RECESSIVEOMIM: SCL/TAL1-INTERRUPTING LOCUS      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of STIL  OMIM: MICROCEPHALY 7, PRIMARY, AUTOSOMAL RECESSIVEOMIM: SCL/TAL1-INTERRUPTING LOCUS  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FOXD2\t\n",
            "SPATA6\t\n",
            "NRDC\t\n",
            "LRP8\t   From NCBI Gene:This gene encodes a member of the low density lipoprotein receptor (LDLR) family. Low density lipoprotein receptors are cell surface proteins that play roles in both signal transduction and receptor-mediated endocytosis of specific ligands for lysosomal degradation. The encoded protein plays a critical role in the migration of neurons during development by mediating Reelin signaling, and also functions as a receptor for the cholesterol transport protein apolipoprotein E. Expression of this gene may be a marker for major depressive disorder. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Jun 2011]From UniProt:Cell surface receptor for Reelin (RELN) and apolipoprotein E (apoE)-containing ligands. LRP8 participates in transmitting the extracellular Reelin signal to intracellular signaling processes, by binding to DAB1 on its cytoplasmic tail. Reelin acts via both the VLDL receptor (VLDLR) and LRP8 to regulate DAB1 tyrosine phosphorylation and microtubule function in neurons. LRP8 has higher affinity for Reelin than VLDLR. LRP8 is thus a key component of the Reelin pathway which governs neuronal layering of the forebrain during embryonic brain development. Binds the endoplasmic reticulum resident receptor-associated protein (RAP). Binds dimers of beta 2-glycoprotein I and may be involved in the suppression of platelet aggregation in the vasculature. Highly expressed in the initial segment of the epididymis, where it affects the functional expression of clusterin and phospholipid hydroperoxide glutathione peroxidase (PHGPx), two proteins required for sperm maturation. May also function as an endocytic receptor. Not required for endocytic uptake of SEPP1 in the kidney which is mediated by LRP2 (By similarity). Together with its ligand, apolipoprotein E (apoE), may indirectly play a role in the suppression of the innate immune response by controlling the survival of myeloid-derived suppressor cells (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Myocardial infarction 1From UniProt:Myocardial infarction 1 (MCI1): A condition defined by the irreversible necrosis of heart muscle secondary to prolonged ischemia. [MIM:608446]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p32.3, which is the short (p) arm of chromosome 1 at position 32.3Molecular Location: base pairs 53,242,364 to 53,328,149 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        APOER2HSZ75190LRP-8MCI1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LRP8      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 8OMIM: MYOCARDIAL INFARCTION, SUSCEPTIBILITY TO      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LRP8  OMIM: LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 8OMIM: MYOCARDIAL INFARCTION, SUSCEPTIBILITY TO  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GLIS1\t\n",
            "CDCP2\t\n",
            "MROH7\t\n",
            "NPHP4\t   From NCBI Gene:This gene encodes a protein involved in renal tubular development and function. This protein interacts with nephrocystin, and belongs to a multifunctional complex that is localized to actin- and microtubule-based structures. Mutations in this gene are associated with nephronophthisis type 4, a renal disease, and with Senior-Loken syndrome type 4, a combination of nephronophthisis and retinitis pigmentosa. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2014]From UniProt:Involved in the organization of apical junctions; the function is proposed to implicate a NPHP1-4-8 module (PubMed:19755384, PubMed:21565611). Does not seem to be strictly required for ciliogenesis (PubMed:21565611). Required for building functional cilia. Involved in the organization of the subapical actin network in multiciliated epithelial cells. Seems to recruit INT to basal bodies of motile cilia which subsequently interacts with actin-modifying proteins such as DAAM1 (By similarity). In cooperation with INVS may downregulate the canonical Wnt pathway and promote the Wnt-PCP pathway by regulating expression and subcellular location of disheveled proteins. Stabilizes protein levels of JADE1 and promotes its translocation to the nucleus leading to cooperative inhibition of canonical Wnt signaling (PubMed:21498478, PubMed:22654112). Acts as negative regulator of the hippo pathway by association with LATS1 and modifying LATS1-dependent phosphorylation and localization of WWTR1/TAZ (PubMed:21555462).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:NephronophthisisSenior-Lken syndromeFrom NCBI Gene:Nephronophthisis 4Senior-Loken syndrome 4From UniProt:Nephronophthisis 4 (NPHP4): An autosomal recessive inherited disease resulting in end-stage renal disease at age ranging between 6 and 35 years. It is a progressive tubulo-interstitial kidney disorder characterized by polydipsia, polyuria, anemia and growth retardation. The most prominent histological features are modifications of the tubules with thickening of the basement membrane, interstitial fibrosis and, in the advanced stages, medullary cysts. [MIM:606966]Ciliary dysfunction leads to a broad spectrum of disorders, collectively termed ciliopathies. Overlapping clinical features include retinal degeneration, renal cystic disease, skeletal abnormalities, fibrosis of various organ, and a complex range of anatomical and functional defects of the central and peripheral nervous system. The ciliopathy range of diseases includes Meckel-Gruber syndrome, Bardet-Biedl syndrome, Joubert syndrome, nephronophtisis, Senior-Loken syndrome, and Jeune asphyxiating thoracic dystrophy among others. Single-locus allelism is insufficient to explain the variable penetrance and expressivity of such disorders, leading to the suggestion that variations across multiple sites of the ciliary proteome, including NPHP4, influence the clinical outcome (PubMed:21258341).May be involved in male infertility. Homozygosity for a frameshift truncating mutation are associated with markedly abnormal sperm morphology.May be involved in cardiac laterality defects and heterotaxy. Homozygosity for a frameshift truncating mutation are associated with markedly abnormal sperm morphology.Senior-Loken syndrome 4 (SLSN4): A renal-retinal disorder characterized by progressive wasting of the filtering unit of the kidney (nephronophthisis), with or without medullary cystic renal disease, and progressive eye disease. Typically this disorder becomes apparent during the first year of life. [MIM:606996]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.31, which is the short (p) arm of chromosome 1 at position 36.31Molecular Location: base pairs 5,862,808 to 5,992,473 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        POC10SLSN4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NPHP4      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: NEPHROCYSTIN 4OMIM: NEPHRONOPHTHISIS 4OMIM: SENIOR-LOKEN SYNDROME 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NPHP4  OMIM: NEPHROCYSTIN 4OMIM: NEPHRONOPHTHISIS 4OMIM: SENIOR-LOKEN SYNDROME 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ROR1\t   From NCBI Gene:This gene encodes a receptor tyrosine kinase-like orphan receptor that modulates neurite growth in the central nervous system. The encoded protein is a glycosylated type I membrane protein that belongs to the ROR subfamily of cell surface receptors. It is a pseudokinase that lacks catalytic activity and may interact with the non-canonical Wnt signalling pathway. This gene is highly expressed during early embryonic development but expressed at very low levels in adult tissues. Increased expression of this gene is associated with B-cell chronic lymphocytic leukaemia. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Jun 2012]From UniProt:Has very low kinase activity in vitro and is unlikely to function as a tyrosine kinase in vivo (PubMed:25029443). Receptor for ligand WNT5A which activate downstream NFkB signaling pathway and may result in the inhibition of WNT3A-mediated signaling (PubMed:25029443, PubMed:27162350). In inner ear, crucial for spiral ganglion neurons to innervate auditory hair cells (PubMed:27162350).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:DEAFNESS, AUTOSOMAL RECESSIVE 108From UniProt:Deafness, autosomal recessive, 108 (DFNB108): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. [MIM:617654]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p31.3, which is the short (p) arm of chromosome 1 at position 31.3Molecular Location: base pairs 63,774,019 to 64,181,498 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        dJ537F10.1NTRKR1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ROR1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 108OMIM: RECEPTOR TYROSINE KINASE-LIKE ORPHAN RECEPTOR 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ROR1  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 108OMIM: RECEPTOR TYROSINE KINASE-LIKE ORPHAN RECEPTOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NOL9\t\n",
            "LEPR\t   The LEPR gene  provides instructions for making a protein called the leptin receptor, which is involved in the regulation of body weight. The leptin receptor protein is found on the surface of cells in many organs and tissues of the body, including a part of the brain called the hypothalamus. The hypothalamus controls hunger and thirst as well as other functions such as sleep, moods, and body temperature. It also regulates the release of many hormones that have functions throughout the body.The leptin receptor is turned on (activated) by a hormone called leptin that attaches (binds) to the receptor, fitting into it like a key into a lock. Normally, the body's fat cells release leptin in proportion to their size. As fat cells become larger, they produce more leptin. This rise in leptin indicates that fat stores are increasing. In the hypothalamus, the binding of leptin to its receptor triggers a series of chemical signals that affect hunger and help produce a feeling of fullness (satiety).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Leptin receptor deficiency     At least 18 LEPR gene mutations that cause leptin receptor deficiency have been identified; this disorder is associated with excessive hunger, massive weight gain, and reduced production of hormones that direct sexual development (hypogonadotropic hypogonadism). Some of the mutations result in less receptor protein getting to the cell surface where leptin binding takes place. The receptors that get to the cell surface may bind to leptin, but their signaling function is impaired. The resulting shortage of leptin signaling disrupts normal feelings of hunger and satiety, leading to extreme weight gain.Because hypogonadotropic hypogonadism occurs in leptin receptor deficiency, researchers suggest that leptin receptor signaling is also involved in regulating the body's response to hormones that control sexual development, and that this response is affected by LEPR gene mutations. However, the mechanism of this effect is unknown. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 18 LEPR gene mutations that cause leptin receptor deficiency have been identified; this disorder is associated with excessive hunger, massive weight gain, and reduced production of hormones that direct sexual development (hypogonadotropic hypogonadism). Some of the mutations result in less receptor protein getting to the cell surface where leptin binding takes place. The receptors that get to the cell surface may bind to leptin, but their signaling function is impaired. The resulting shortage of leptin signaling disrupts normal feelings of hunger and satiety, leading to extreme weight gain.Because hypogonadotropic hypogonadism occurs in leptin receptor deficiency, researchers suggest that leptin receptor signaling is also involved in regulating the body's response to hormones that control sexual development, and that this response is affected by LEPR gene mutations. However, the mechanism of this effect is unknown. More About This Health Condition    Cytogenetic Location: 1p31.3, which is the short (p) arm of chromosome 1 at position 31.3Molecular Location: base pairs 65,420,652 to 65,637,493 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LEP-RLEPR_HUMANOB-ROB receptorOBR     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Basic Neurochemistry: Molecular, Cellular and Medical Aspects (sixth edition, 1999): Neuronal Control of Food Intake      Tests Listed in the Genetic Testing Registry (1 link)     Tests of LEPR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     LEPTIN RECEPTOR      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry: Molecular, Cellular and Medical Aspects (sixth edition, 1999): Neuronal Control of Food Intake  Tests of LEPR  PubMed  LEPTIN RECEPTOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Clment K, Vaisse C, Lahlou N, Cabrol S, Pelloux V, Cassuto D, Gourmelen M, Dina C, Chambaz J, Lacorte JM, Basdevant A, Bougnres P, Lebouc Y, Froguel P, Guy-Grand B. A mutation in the human leptin receptor gene causes obesity and pituitary dysfunction. Nature. 1998 Mar 26;392(6674):398-401. Citation on PubMedDubern B, Clement K. Leptin and leptin receptor-related monogenic obesity. Biochimie. 2012 Oct;94(10):2111-5. doi: 10.1016/j.biochi.2012.05.010. Epub 2012 May 22. Review. Citation on PubMedFarooqi IS, Wangensteen T, Collins S, Kimber W, Matarese G, Keogh JM, Lank E, Bottomley B, Lopez-Fernandez J, Ferraz-Amaro I, Dattani MT, Ercan O, Myhre AG, Retterstol L, Stanhope R, Edge JA, McKenzie S, Lessan N, Ghodsi M, De Rosa V, Perna F, Fontana S, Barroso I, Undlien DE, O'Rahilly S. Clinical and molecular genetic spectrum of congenital deficiency of the leptin receptor. N Engl J Med. 2007 Jan 18;356(3):237-47. Citation on PubMed or Free article on PubMed CentralKimber W, Peelman F, Prieur X, Wangensteen T, O'Rahilly S, Tavernier J, Farooqi IS. Functional characterization of naturally occurring pathogenic mutations in the human leptin receptor. Endocrinology. 2008 Dec;149(12):6043-52. doi: 10.1210/en.2008-0544. Epub 2008 Aug 14. Citation on PubMedOMIM: LEPTIN RECEPTORLee YS. The role of leptin-melanocortin system and human weight regulation: lessons from experiments of nature. Ann Acad Med Singapore. 2009 Jan;38(1):34-11. Review. Citation on PubMedWasim M, Awan FR, Najam SS, Khan AR, Khan HN. Role of Leptin Deficiency, Inefficiency, and Leptin Receptors in Obesity. Biochem Genet. 2016 Jun 16. [Epub ahead of print] Review. Citation on PubMed \n",
            "LEPROT\t\n",
            "THAP3\t\n",
            "SGIP1\t\n",
            "AK5\t\n",
            "PER3\t   From NCBI Gene:This gene is a member of the Period family of genes and is expressed in a circadian pattern in the suprachiasmatic nucleus, the primary circadian pacemaker in the mammalian brain. Genes in this family encode components of the circadian rhythms of locomotor activity, metabolism, and behavior. This gene is upregulated by CLOCK/ARNTL heterodimers but then represses this upregulation in a feedback loop using PER/CRY heterodimers to interact with CLOCK/ARNTL. Polymorphisms in this gene have been linked to sleep disorders. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2014]From UniProt:Originally described as a core component of the circadian clock. The circadian clock, an internal time-keeping system, regulates various physiological processes through the generation of approximately 24 hour circadian rhythms in gene expression, which are translated into rhythms in metabolism and behavior. It is derived from the Latin roots 'circa' (about) and 'diem' (day) and acts as an important regulator of a wide array of physiological functions including metabolism, sleep, body temperature, blood pressure, endocrine, immune, cardiovascular, and renal function. Consists of two major components: the central clock, residing in the suprachiasmatic nucleus (SCN) of the brain, and the peripheral clocks that are present in nearly every tissue and organ system. Both the central and peripheral clocks can be reset by environmental cues, also known as Zeitgebers (German for 'timegivers'). The predominant Zeitgeber for the central clock is light, which is sensed by retina and signals directly to the SCN. The central clock entrains the peripheral clocks through neuronal and hormonal signals, body temperature and feeding-related cues, aligning all clocks with the external light/dark cycle. Circadian rhythms allow an organism to achieve temporal homeostasis with its environment at the molecular level by regulating gene expression to create a peak of protein expression once every 24 hours to control when a particular physiological process is most active with respect to the solar day. Transcription and translation of core clock components (CLOCK, NPAS2, ARNTL/BMAL1, ARNTL2/BMAL2, PER1, PER2, PER3, CRY1 and CRY2) plays a critical role in rhythm generation, whereas delays imposed by post-translational modifications (PTMs) are important for determining the period (tau) of the rhythms (tau refers to the period of a rhythm and is the length, in time, of one complete cycle). A diurnal rhythm is synchronized with the day/night cycle, while the ultradian and infradian rhythms have a period shorter and longer than 24 hours, respectively. Disruptions in the circadian rhythms contribute to the pathology of cardiovascular diseases, cancer, metabolic syndromes and aging. A transcription/translation feedback loop (TTFL) forms the core of the molecular circadian clock mechanism. Transcription factors, CLOCK or NPAS2 and ARNTL/BMAL1 or ARNTL2/BMAL2, form the positive limb of the feedback loop, act in the form of a heterodimer and activate the transcription of core clock genes and clock-controlled genes (involved in key metabolic processes), harboring E-box elements (5'-CACGTG-3') within their promoters. The core clock genes: PER1/2/3 and CRY1/2 which are transcriptional repressors form the negative limb of the feedback loop and interact with the CLOCK|NPAS2-ARNTL/BMAL1|ARNTL2/BMAL2 heterodimer inhibiting its activity and thereby negatively regulating their own expression. This heterodimer also activates nuclear receptors NR1D1, NR1D2, RORA, RORB and RORG, which form a second feedback loop and which activate and repress ARNTL/BMAL1 transcription, respectively. Has a redundant role with the other PER proteins PER1 and PER2 and is not essential for the circadian rhythms maintenance. In contrast, plays an important role in sleep-wake timing and sleep homeostasis probably through the transcriptional regulation of sleep homeostasis-related genes, without influencing circadian parameters. Can bind heme.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Advanced sleep phase syndrome, familial, 3From UniProt:Advanced sleep phase syndrome, familial, 3 (FASPS3): A disorder characterized by very early sleep onset and offset. Individuals are 'morning larks' with a 4 hours advance of the sleep, temperature and melatonin rhythms. [MIM:616882]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.23, which is the short (p) arm of chromosome 1 at position 36.23Molecular Location: base pairs 7,784,285 to 7,845,181 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FASPS3GIG13     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PER3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ADVANCED SLEEP PHASE SYNDROME, FAMILIAL, 3OMIM: PERIOD, DROSOPHILA, HOMOLOG OF, 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PER3  OMIM: ADVANCED SLEEP PHASE SYNDROME, FAMILIAL, 3OMIM: PERIOD, DROSOPHILA, HOMOLOG OF, 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IFI44\t\n",
            "ADGRL4\t\n",
            "CTBS\t\n",
            "CLCA4\t\n",
            "RBMXL1\t\n",
            "GBP3\t\n",
            "GBP1\t\n",
            "GBP6\t\n",
            "CA6\t\n",
            "HFM1\t   From NCBI Gene:The protein encoded by this gene is thought to be an ATP-dependent DNA helicase and is expressed mainly in germ-line cells. Defects in this gene are a cause of premature ovarian failure 9 (POF9). [provided by RefSeq, Apr 2014]From UniProt:Required for crossover formation and complete synapsis of homologous chromosomes during meiosis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Premature ovarian failure 9From UniProt:Premature ovarian failure 9 (POF9): An ovarian disorder defined as the cessation of ovarian function under the age of 40 years. It is characterized by oligomenorrhea or amenorrhea, in the presence of elevated levels of serum gonadotropins and low estradiol. [MIM:615724]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p22.2, which is the short (p) arm of chromosome 1 at position 22.2Molecular Location: base pairs 91,260,766 to 91,408,008 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        helicaseMER3POF9SEC63D1Si-11Si-11-6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HFM1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HFM1, ATP-DEPENDENT DNA HELICASE, S. CEREVISIAE, HOMOLOG OFOMIM: PREMATURE OVARIAN FAILURE 9      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HFM1  OMIM: HFM1, ATP-DEPENDENT DNA HELICASE, S. CEREVISIAE, HOMOLOG OFOMIM: PREMATURE OVARIAN FAILURE 9  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "H6PD\t   From NCBI Gene:There are 2 forms of glucose-6-phosphate dehydrogenase. G form is X-linked and H form, encoded by this gene, is autosomally linked. This H form shows activity with other hexose-6-phosphates, especially galactose-6-phosphate, whereas the G form is specific for glucose-6-phosphate. Both forms are present in most tissues, but H form is not found in red cells. [provided by RefSeq, Jul 2008]From UniProt:Oxidizes glucose-6-phosphate and glucose, as well as other hexose-6-phosphates.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Cortisone reductase deficiency 1From UniProt:Cortisone reductase deficiency 1 (CORTRD1): An autosomal recessive error of cortisone metabolism characterized by a failure to regenerate cortisol from cortisone, resulting in increased cortisol clearance, activation of the hypothalamic- pituitary axis and ACTH-mediated adrenal androgen excess. Clinical features include hyperandrogenism resulting in hirsutism, oligo- amenorrhea, and infertility in females and premature pseudopuberty in males. [MIM:604931]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.22, which is the short (p) arm of chromosome 1 at position 36.22Molecular Location: base pairs 9,234,767 to 9,271,337 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CORTRD1G6PDHGDH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of H6PD      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CORTISONE REDUCTASE DEFICIENCY 1OMIM: HEXOSE-6-PHOSPHATE DEHYDROGENASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of H6PD  OMIM: CORTISONE REDUCTASE DEFICIENCY 1OMIM: HEXOSE-6-PHOSPHATE DEHYDROGENASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMED5\t\n",
            "ABCA4\t   The ABCA4 gene provides instructions for making a protein that is found in the retina, the specialized light-sensitive tissue that lines the back of the eye. Specifically, the ABCA4 protein is produced in the retina's light receptor cells (photoreceptors). The ABCA4 protein is active following phototransduction, the process by which light entering the eye is converted into electrical signals that are transmitted to the brain. Phototransduction leads to the formation of potentially toxic substances that can damage photoreceptor cells. The ABCA4 protein removes one of these substances, called N-retinylidene-PE, from photoreceptor cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Cone-rod dystrophy     More than 40 mutations in the ABCA4 gene have been found to cause a vision disorder called cone-rod dystrophy. The problems associated with this condition include a loss of visual sharpness (acuity), an increased sensitivity to light (photophobia), and impaired color vision. These vision problems worsen over time. It is estimated that ABCA4 gene mutations account for 30 to 60 percent of cases of cone-rod dystrophy that are inherited in an autosomal recessive pattern, which means that both copies of the gene in each cell have mutations.Most of the ABCA4 gene mutations that cause cone-rod dystrophy change single protein building blocks (amino acids) in the ABCA4 protein. The altered protein cannot remove N-retinylidene-PE from photoreceptor cells. As a result, N-retinylidene-PE combines with another substance to produce a molecule called N-retinylidene-N-retinylethanolamine (A2E), which builds up in these cells. The buildup of A2E is toxic to photoreceptor cells and leads to their deterioration, causing progressive vision loss in people with cone-rod dystrophy. Cone-rod dystrophy caused by ABCA4 gene mutations tends to be associated with more severe vision problems than cone-rod dystrophy caused by other genetic mutations. More About This Health Condition      Stargardt macular degeneration     More than 640 mutations in the ABCA4 gene have been found to cause Stargardt macular degeneration. Mutations in this gene are the most common cause of this eye disease. Stargardt macular degeneration is characterized by vision loss that worsens over time, particularly affecting central and night vision. Most of these mutations change amino acids in the ABCA4 protein. A malfunctioning ABCA4 protein cannot remove N-retinylidene-PE from photoreceptor cells. As a result, N-retinylidene-PE combines with another substance to produce a fatty yellow pigment called lipofuscin, which builds up in retinal cells. The buildup of lipofuscin is toxic to the cells of the retina and causes progressive vision loss in people with Stargardt macular degeneration.Several ABCA4 gene mutations have been found to cause different forms of vision loss in different individuals. It is unclear how mutations in the ABCA4 gene can cause different eye disorders. More About This Health Condition      Age-related macular degeneration     Genetics Home Reference provides information about age-related macular degeneration. More About This Health Condition      Retinitis pigmentosa     Genetics Home Reference provides information about retinitis pigmentosa. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 40 mutations in the ABCA4 gene have been found to cause a vision disorder called cone-rod dystrophy. The problems associated with this condition include a loss of visual sharpness (acuity), an increased sensitivity to light (photophobia), and impaired color vision. These vision problems worsen over time. It is estimated that ABCA4 gene mutations account for 30 to 60 percent of cases of cone-rod dystrophy that are inherited in an autosomal recessive pattern, which means that both copies of the gene in each cell have mutations.Most of the ABCA4 gene mutations that cause cone-rod dystrophy change single protein building blocks (amino acids) in the ABCA4 protein. The altered protein cannot remove N-retinylidene-PE from photoreceptor cells. As a result, N-retinylidene-PE combines with another substance to produce a molecule called N-retinylidene-N-retinylethanolamine (A2E), which builds up in these cells. The buildup of A2E is toxic to photoreceptor cells and leads to their deterioration, causing progressive vision loss in people with cone-rod dystrophy. Cone-rod dystrophy caused by ABCA4 gene mutations tends to be associated with more severe vision problems than cone-rod dystrophy caused by other genetic mutations. More About This Health Condition  More than 640 mutations in the ABCA4 gene have been found to cause Stargardt macular degeneration. Mutations in this gene are the most common cause of this eye disease. Stargardt macular degeneration is characterized by vision loss that worsens over time, particularly affecting central and night vision. Most of these mutations change amino acids in the ABCA4 protein. A malfunctioning ABCA4 protein cannot remove N-retinylidene-PE from photoreceptor cells. As a result, N-retinylidene-PE combines with another substance to produce a fatty yellow pigment called lipofuscin, which builds up in retinal cells. The buildup of lipofuscin is toxic to the cells of the retina and causes progressive vision loss in people with Stargardt macular degeneration.Several ABCA4 gene mutations have been found to cause different forms of vision loss in different individuals. It is unclear how mutations in the ABCA4 gene can cause different eye disorders. More About This Health Condition  Genetics Home Reference provides information about age-related macular degeneration. More About This Health Condition  Genetics Home Reference provides information about retinitis pigmentosa. More About This Health Condition    Cytogenetic Location: 1p22.1, which is the short (p) arm of chromosome 1 at position 22.1Molecular Location: base pairs 93,992,837 to 94,121,149 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABCA4_HUMANABCRATP-binding cassette sub-family A member 4ATP-binding cassette transporter, retinal-specificATP-binding cassette, sub-family A (ABC1), member 4photoreceptor rim proteinretina-specific ABC transporterretinal-specific ATP-binding cassette transporterRIM ABC transporterRIM proteinRMP     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Neuroscience (second edition, 2001): Functional Specialization of the Rod and Cone SystemsThe Human ATP-Binding Cassette (ABC) Transporter Superfamily: ABCA GenesWebvision: The Organization of the Retina and Visual System (2007): Photoreceptors      Clinical Information from GeneReviews (1 link)     Nonsyndromic Retinitis Pigmentosa Overview      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ABCA4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (3 links)     ATP-BINDING CASSETTE, SUBFAMILY A, MEMBER 4MACULAR DEGENERATION, AGE-RELATED, 2RETINITIS PIGMENTOSA 19      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Neuroscience (second edition, 2001): Functional Specialization of the Rod and Cone SystemsThe Human ATP-Binding Cassette (ABC) Transporter Superfamily: ABCA GenesWebvision: The Organization of the Retina and Visual System (2007): Photoreceptors  Nonsyndromic Retinitis Pigmentosa Overview  Tests of ABCA4  PubMed  ATP-BINDING CASSETTE, SUBFAMILY A, MEMBER 4MACULAR DEGENERATION, AGE-RELATED, 2RETINITIS PIGMENTOSA 19  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ATP-BINDING CASSETTE, SUBFAMILY A, MEMBER 4Molday RS, Zhang K. Defective lipid transport and biosynthesis in recessive and dominant Stargardt macular degeneration. Prog Lipid Res. 2010 Oct;49(4):476-92. doi: 10.1016/j.plipres.2010.07.002. Epub 2010 Jul 13. Review. Citation on PubMed or Free article on PubMed CentralMolday RS, Zhong M, Quazi F. The role of the photoreceptor ABC transporter ABCA4 in lipid transport and Stargardt macular degeneration. Biochim Biophys Acta. 2009 Jul;1791(7):573-83. doi: 10.1016/j.bbalip.2009.02.004. Epub 2009 Feb 20. Review. Citation on PubMed or Free article on PubMed CentralThiadens AA, Phan TM, Zekveld-Vroon RC, Leroy BP, van den Born LI, Hoyng CB, Klaver CC; Writing Committee for the Cone Disorders Study Group Consortium, Roosing S, Pott JW, van Schooneveld MJ, van Moll-Ramirez N, van Genderen MM, Boon CJ, den Hollander AI, Bergen AA, De Baere E, Cremers FP, Lotery AJ. Clinical course, genetic etiology, and visual outcome in cone and cone-rod dystrophy. Ophthalmology. 2012 Apr;119(4):819-26. doi: 10.1016/j.ophtha.2011.10.011. Epub 2012 Jan 20. Citation on PubMedTsybovsky Y, Molday RS, Palczewski K. The ATP-binding cassette transporter ABCA4: structural and functional properties and role in retinal disease. Adv Exp Med Biol. 2010;703:105-25. doi: 10.1007/978-1-4419-5635-4_8. Review. Citation on PubMed or Free article on PubMed Central \n",
            "AGRN\t   From NCBI Gene:This gene encodes one of several proteins that are critical in the development of the neuromuscular junction (NMJ), as identified in mouse knock-out studies. The encoded protein contains several laminin G, Kazal type serine protease inhibitor, and epidermal growth factor domains. Additional post-translational modifications occur to add glycosaminoglycans and disulfide bonds. In one family with congenital myasthenic syndrome affecting limb-girdle muscles, a mutation in this gene was found. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2015]From UniProt:Isoform 2: transmembrane form that is the predominate form in neurons of the brain, induces dendritic filopodia and synapse formation in mature hippocampal neurons in large part due to the attached glycosaminoglycan chains and the action of Rho-family GTPases.Isoform 1: heparan sulfate basal lamina glycoprotein that plays a central role in the formation and the maintenance of the neuromuscular junction (NMJ) and directs key events in postsynaptic differentiation. Component of the AGRN-LRP4 receptor complex that induces the phosphorylation and activation of MUSK. The activation of MUSK in myotubes induces the formation of NMJ by regulating different processes including the transcription of specific genes and the clustering of AChR in the postsynaptic membrane. Calcium ions are required for maximal AChR clustering. AGRN function in neurons is highly regulated by alternative splicing, glycan binding and proteolytic processing. Modulates calcium ion homeostasis in neurons, specifically by inducing an increase in cytoplasmic calcium ions. Functions differentially in the central nervous system (CNS) by inhibiting the alpha(3)-subtype of Na+/K+-ATPase and evoking depolarization at CNS synapses. This secreted isoform forms a bridge, after release from motor neurons, to basal lamina through binding laminin via the NtA domain.Isoform 3 and isoform 6: lack any 'z' insert, are muscle-specific and may be involved in endothelial cell differentiation.Agrin N-terminal 110 kDa subunit: is involved in regulation of neurite outgrowth probably due to the presence of the glycosaminoglcan (GAG) side chains of heparan and chondroitin sulfate attached to the Ser/Thr- and Gly/Ser-rich regions. Also involved in modulation of growth factor signaling (By similarity).Isoform 1, isoform 4 and isoform 5: neuron-specific (z+) isoforms that contain C-terminal insertions of 8-19 AA are potent activators of AChR clustering. Isoform 5, agrin (z+8), containing the 8-AA insert, forms a receptor complex in myotubules containing the neuronal AGRN, the muscle-specific kinase MUSK and LRP4, a member of the LDL receptor family. The splicing factors, NOVA1 and NOVA2, regulate AGRN splicing and production of the 'z' isoforms.Agrin C-terminal 22 kDa fragment: this released fragment is important for agrin signaling and to exert a maximal dendritic filopodia-inducing effect. All 'z' splice variants (z+) of this fragment also show an increase in the number of filopodia.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Congenital myasthenic syndromeFrom NCBI Gene:Myasthenic syndrome, congenital, 8From UniProt:Myasthenic syndrome, congenital, 8 (CMS8): A form of congenital myasthenic syndrome, a group of disorders characterized by failure of neuromuscular transmission, including pre-synaptic, synaptic, and post-synaptic disorders that are not of autoimmune origin. Clinical features are easy fatigability and muscle weakness. CMS8 is an autosomal recessive disease characterized by prominent defects of both the pre- and postsynaptic regions. Affected individuals have onset of muscle weakness in early childhood; the severity of the weakness and muscles affected is variable. [MIM:615120]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 1p36.33, which is the short (p) arm of chromosome 1 at position 36.33Molecular Location: base pairs 1,020,102 to 1,056,119 on chromosome 1 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AGRINCMS8CMSPPD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGRN      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: AGRINOMIM: MYASTHENIC SYNDROME, CONGENITAL, 8      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AGRN  OMIM: AGRINOMIM: MYASTHENIC SYNDROME, CONGENITAL, 8  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DEFB126\t   From NCBI Gene:Defensins are cysteine-rich cationic polypeptides that are important in the immunologic response to invading microorganisms. The antimicrobial protein encoded by this gene is secreted and is a member of the beta defensin protein family. Beta defensin genes are found in several clusters throughout the genome, with this gene mapping to a cluster at 20p13. The encoded protein is highly similar to an epididymal-specific secretory protein (ESP13.2) from cynomolgus monkey. Mutation of this gene is associated with impaired sperm function. [provided by RefSeq, Nov 2014]From UniProt:Highly glycosylated atypical beta-defensin involved in several aspects of sperm function. Facilitates sperm transport in the female reproductive tract and contributes to sperm protection against immunodetection; both functions are probably implicating the negative surface charge provided by its O-linked oligosaccharides in the sperm glycocalyx. Involved in binding of sperm to oviductal epithelial cells to form a sperm reservoir until ovulation. Release from the sperm surface during capacitation and ovaluation by an elevation of oviductal fluid pH is unmasking other surface components and allows sperm to penetrate the cumulus matrix and bind to the zona pellucida of the oocyte (By similarity). In vitro has antimicrobial activity and may inhibit LPS-mediated inflammation (PubMed:19373462, PubMed:23229569).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:May be involved in infertility. Homozygosity for frameshift truncating mutations are associated with reduced sperm O-linked glycan content, impaired sperm mobility and a reduced live birth rate (PubMed:21775668, PubMed:25721098). However, for one common mutation the change in sperm sialic acid levels has been challenged (PubMed:26832966).     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20p13, which is the short (p) arm of chromosome 20 at position 13Molecular Location: base pairs 142,553 to 145,751 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA530N10.1C20orf8DEFB-26DEFB26hBD-26HBD26     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DEFB126      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DEFB126  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SDCBP2\t\n",
            "DEFB127\t\n",
            "SIRPB2\t\n",
            "SIRPB1\t\n",
            "RRBP1\t\n",
            "MGME1\t   From NCBI Gene:The protein encoded by this gene is a nuclear-encoded mitochondrial protein necessary for the maintenance of mitochondrial genome synthesis. The encoded protein is a RecB-type exonuclease and primarily cleaves single-stranded DNA. Defects in this gene have been associated with mitochondrial DNA depletion syndrome-11. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2015]From UniProt:Metal-dependent single-stranded DNA (ssDNA) exonuclease involved in mitochondrial genome maintenance. Has preference for 5'-3' exonuclease activity but is also capable of endoduclease activity on linear substrates. Necessary for maintenance of proper 7S DNA levels. Probably involved in mitochondrial DNA (mtDNA) repair, possibly via the processing of displaced DNA containing Okazaki fragments during RNA-primed DNA synthesis on the lagging strand or via processing of DNA flaps during long-patch base excision repair. Specifically binds 5-hydroxymethylcytosine (5hmC)-containing DNA in stem cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mitochondrial DNA depletion syndrome 11From UniProt:Mitochondrial DNA depletion syndrome 11 (MTDPS11): An autosomal recessive mitochondrial disorder characterized by onset in childhood or adulthood of progressive external ophthalmoplegia, muscle weakness and atrophy, exercise intolerance, and respiratory insufficiency due to muscle weakness. More variable features include spinal deformity, emaciation, and cardiac abnormalities. Skeletal muscle biopsies show deletion and depletion of mitochondrial DNA (mtDNA) with variable defects in respiratory chain enzyme activities. [MIM:615084]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20p11.23, which is the short (p) arm of chromosome 20 at position 11.23Molecular Location: base pairs 17,968,594 to 17,991,122 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA504H3.4C20orf72DDK1MTDPS11     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MGME1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MITOCHONDRIAL DNA DEPLETION SYNDROME 11OMIM: MITOCHONDRIAL GENOME MAINTENANCE EXONUCLEASE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MGME1  OMIM: MITOCHONDRIAL DNA DEPLETION SYNDROME 11OMIM: MITOCHONDRIAL GENOME MAINTENANCE EXONUCLEASE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF133\t\n",
            "DZANK1\t\n",
            "SEC23B\t   The SEC23B gene provides instructions for making one component of a large group of interacting proteins called coat protein complex II (COPII). COPII is involved in the formation of vesicles, which are small sac-like structures that transport proteins and other materials within cells. Specifically, COPII triggers the formation of vesicles in a cellular structure called the endoplasmic reticulum (ER), which is involved in protein processing and transport. These COPII vesicles carry proteins that are destined to be exported out of cells (secreted).The SEC23B protein is very similar to the protein produced from a related gene, SEC23A. These proteins are both components of COPII, and they appear to have overlapping functions. In most types of cells, if one of these proteins is missing, the other may be able to compensate for the loss. However, research indicates that the SEC23B protein may have a unique function in developing red blood cells (erythroblasts).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Congenital dyserythropoietic anemia     At least 20 mutations in the SEC23B gene have been identified in people with congenital dyserythropoietic anemia (CDA) type II. Most of these mutations change single protein building blocks (amino acids) in the SEC23B protein. Other mutations delete genetic material from the SEC23B gene or alter the way the gene's instructions are used to make the SEC23B protein. The mutations responsible for CDA type II likely disrupt the function of the SEC23B protein. However, researchers suspect that these mutations do not completely eliminate the function of the protein, which appears to be essential for life.It is unclear how SEC23B mutations cause the characteristic features of CDA type II. The abnormal SEC23B protein leads to the production of erythroblasts that are unusually shaped and may have extra nuclei. These defective erythroblasts cannot develop into functional mature red blood cells. The resulting shortage of healthy red blood cells leads to the characteristic signs and symptoms of anemia, as well as complications including an enlarged liver and spleen (hepatosplenomegaly) and an abnormal buildup of iron that can damage the body's organs. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 20 mutations in the SEC23B gene have been identified in people with congenital dyserythropoietic anemia (CDA) type II. Most of these mutations change single protein building blocks (amino acids) in the SEC23B protein. Other mutations delete genetic material from the SEC23B gene or alter the way the gene's instructions are used to make the SEC23B protein. The mutations responsible for CDA type II likely disrupt the function of the SEC23B protein. However, researchers suspect that these mutations do not completely eliminate the function of the protein, which appears to be essential for life.It is unclear how SEC23B mutations cause the characteristic features of CDA type II. The abnormal SEC23B protein leads to the production of erythroblasts that are unusually shaped and may have extra nuclei. These defective erythroblasts cannot develop into functional mature red blood cells. The resulting shortage of healthy red blood cells leads to the characteristic signs and symptoms of anemia, as well as complications including an enlarged liver and spleen (hepatosplenomegaly) and an abnormal buildup of iron that can damage the body's organs. More About This Health Condition    Cytogenetic Location: 20p11.23, which is the short (p) arm of chromosome 20 at position 11.23Molecular Location: base pairs 18,507,482 to 18,561,415 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDA-IICDAIIHEMPASSC23B_HUMANSec23 homolog BSec23 homolog B (S. cerevisiae)Sec23 homolog B, COPII coat complex componentSEC23-like protein BSEC23-related protein Btransport protein SEC23B     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Madame Curie Bioscience Database: Coat Protein Complexes: Cellular Machineries Driving Vesicle Formation/FissionMolecular Biology of the Cell (fourth edition, 2002): Proteins Leave the ER in COPII-coated Transport Vesicles      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SEC23B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SEC23, S. CEREVISIAE, HOMOLOG OF, B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Coat Protein Complexes: Cellular Machineries Driving Vesicle Formation/FissionMolecular Biology of the Cell (fourth edition, 2002): Proteins Leave the ER in COPII-coated Transport Vesicles  Tests of SEC23B  PubMed  SEC23, S. CEREVISIAE, HOMOLOG OF, B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Denecke J, Marquardt T. Congenital dyserythropoietic anemia type II (CDAII/HEMPAS): where are we now? Biochim Biophys Acta. 2009 Sep;1792(9):915-20. doi: 10.1016/j.bbadis.2008.12.005. Epub 2008 Dec 25. Review. Citation on PubMedFromme JC, Orci L, Schekman R. Coordination of COPII vesicle trafficking by Sec23. Trends Cell Biol. 2008 Jul;18(7):330-6. doi: 10.1016/j.tcb.2008.04.006. Epub 2008 Jun 3. Review. Citation on PubMedHughes H, Stephens DJ. Assembly, organization, and function of the COPII coat. Histochem Cell Biol. 2008 Feb;129(2):129-51. Epub 2007 Dec 4. Review. Citation on PubMed or Free article on PubMed CentralSchwarz K, Iolascon A, Verissimo F, Trede NS, Horsley W, Chen W, Paw BH, Hopfner KP, Holzmann K, Russo R, Esposito MR, Spano D, De Falco L, Heinrich K, Joggerst B, Rojewski MT, Perrotta S, Denecke J, Pannicke U, Delaunay J, Pepperkok R, Heimpel H. Mutations affecting the secretory COPII coat component SEC23B cause congenital dyserythropoietic anemia type II. Nat Genet. 2009 Aug;41(8):936-40. doi: 10.1038/ng.405. Epub 2009 Jun 28. Citation on PubMed \n",
            "SIRPA\t\n",
            "CRNKL1\t\n",
            "KIZ\t   From NCBI Gene:The protein encoded by this gene localizes to centrosomes, strengthening and stabilizing the pericentriolar region prior to spindle formation. The encoded protein usually remains with the mother centrosome after centrosomal duplication. Sevral transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2013]From UniProt:Centrosomal protein required for establishing a robust mitotic centrosome architecture that can endure the forces that converge on the centrosomes during spindle formation. Required for stabilizing the expanded pericentriolar material around the centriole.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Retinitis pigmentosa 69From UniProt:Retinitis pigmentosa 69 (RP69): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:615780]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20p11.23, which is the short (p) arm of chromosome 20 at position 11.23Molecular Location: base pairs 21,125,975 to 21,246,622 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C20orf19HT013KizunaNCRNA00153PLK1S1RP69     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KIZ      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KIZUNA CENTROSOMAL PROTEINOMIM: RETINITIS PIGMENTOSA 69      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KIZ  OMIM: KIZUNA CENTROSOMAL PROTEINOMIM: RETINITIS PIGMENTOSA 69  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CST9\t\n",
            "DEFB132\t\n",
            "ABHD12\t   From NCBI Gene:This gene encodes an enzyme that catalyzes the hydrolysis of 2-arachidonoyl glycerol (2-AG), the main endocannabinoid lipid transmitter that acts on cannabinoid receptors, CB1 and CB2. The endocannabinoid system is involved in a wide range of physiological processes, including neurotransmission, mood, appetite, pain appreciation, addiction behavior, and inflammation. Mutations in this gene are associated with the neurodegenerative disease, PHARC (polyneuropathy, hearing loss, ataxia, retinitis pigmentosa, and cataract), resulting from an inborn error of endocannabinoid metabolism. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Jan 2011]From UniProt:Lysophosphatidylserine (LPS) lipase that mediates the hydrolysis of lysophosphatidylserine, a class of signaling lipids that regulates immunological and neurological processes (PubMed:25290914, PubMed:30237167, PubMed:30420694, PubMed:30720278, PubMed:30643283). Represents a major lysophosphatidylserine lipase in the brain, thereby playing a key role in the central nervous system (By similarity). Also able to hydrolyze oxidized phosphatidylserine; oxidized phosphatidylserine is produced in response to severe inflammatory stress and constitutes a proapoptotic 'eat me' signal (PubMed:30643283). Also has monoacylglycerol (MAG) lipase activity: hydrolyzes 2-arachidonoylglycerol (2-AG), thereby acting as a regulator of endocannabinoid signaling pathways (PubMed:22969151, PubMed:24027063). Has a strong preference for very-long-chain lipid substrates; substrate specificity is likely due to improved catalysis and not improved substrate binding (PubMed:30237167).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Charcot-Marie-Tooth diseaseFrom NCBI Gene:Polyneuropathy, hearing loss, ataxia, retinitis pigmentosa, and cataractFrom UniProt:Polyneuropathy, hearing loss, ataxia, retinitis pigmentosa, and cataract (PHARC): A slowly progressive neurologic disorder with a variable phenotype resembling Refsum disease. Clinical features include sensorineural hearing loss, visual problems related to cataracts, retinitis pigmentosa, pes cavus, ataxic and/or spastic gait disturbances with a progressive sensorimotor peripheral neuropathy. Other features include hyporeflexia, hyperreflexia, extensor plantar responses. [MIM:612674]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20p11.21, which is the short (p) arm of chromosome 20 at position 11.21Molecular Location: base pairs 25,294,743 to 25,390,982 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABHD12ABEM46L2C20orf22dJ965G21.2PHARC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ABHD12      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ABHYDROLASE DOMAIN-CONTAINING PROTEIN 12OMIM: POLYNEUROPATHY, HEARING LOSS, ATAXIA, RETINITIS PIGMENTOSA, AND CATARACT      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ABHD12  OMIM: ABHYDROLASE DOMAIN-CONTAINING PROTEIN 12OMIM: POLYNEUROPATHY, HEARING LOSS, ATAXIA, RETINITIS PIGMENTOSA, AND CATARACT  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NINL\t\n",
            "GNRH2\t\n",
            "BPIFB4\t\n",
            "RALY\t\n",
            "MYH7B\t   From NCBI Gene:The myosin II molecule is a multi-subunit complex consisting of two heavy chains and four light chains. This gene encodes a heavy chain of myosin II, which is a member of the motor-domain superfamily. The heavy chain includes a globular motor domain, which catalyzes ATP hydrolysis and interacts with actin, and a tail domain in which heptad repeat sequences promote dimerization by interacting to form a rod-like alpha-helical coiled coil. This heavy chain subunit is a slow-twitch myosin. Alternatively spliced transcript variants have been found, but the full-length nature of these variants is not determined. [provided by RefSeq, Mar 2010]From UniProt:Involved in muscle contraction.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Melanoma     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20q11.22, which is the long (q) arm of chromosome 20 at position 11.22Molecular Location: base pairs 34,955,835 to 35,002,437 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MHC14MYH14     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYH7B      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MYOSIN, HEAVY CHAIN 7B, CARDIAC MUSCLE, BETA      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MYH7B  OMIM: MYOSIN, HEAVY CHAIN 7B, CARDIAC MUSCLE, BETA  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CPNE1\t\n",
            "FAM83D\t\n",
            "CENPB\t\n",
            "MAVS\t\n",
            "PTPRT\t\n",
            "MYBL2\t\n",
            "TP53TG5\t\n",
            "SPINT4\t\n",
            "SNX21\t\n",
            "CDH22\t\n",
            "NCOA3\t   From NCBI Gene:The protein encoded by this gene is a nuclear receptor coactivator that interacts with nuclear hormone receptors to enhance their transcriptional activator functions. The encoded protein has histone acetyltransferase activity and recruits p300/CBP-associated factor and CREB binding protein as part of a multisubunit coactivation complex. This protein is initially found in the cytoplasm but is translocated into the nucleus upon phosphorylation. Several transcript variants encoding different isoforms have been found for this gene. In addition, a polymorphic repeat region is found in the C-terminus of the encoded protein. [provided by RefSeq, Mar 2010]From UniProt:Nuclear receptor coactivator that directly binds nuclear receptors and stimulates the transcriptional activities in a hormone-dependent fashion. Plays a central role in creating a multisubunit coactivator complex, which probably acts via remodeling of chromatin. Involved in the coactivation of different nuclear receptors, such as for steroids (GR and ER), retinoids (RARs and RXRs), thyroid hormone (TRs), vitamin D3 (VDR) and prostanoids (PPARs). Displays histone acetyltransferase activity. Also involved in the coactivation of the NF-kappa-B pathway via its interaction with the NFKB1 subunit.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Osteoarthritis     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20q13.12, which is the long (q) arm of chromosome 20 at position 13.12Molecular Location: base pairs 47,501,857 to 47,656,877 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACTRAIB-1AIB1bHLHe42CAGH16CTG26KAT13BpCIPRAC3SRC-3SRC3TNRC14TNRC16TRAM-1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NCOA3      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: NUCLEAR RECEPTOR COACTIVATOR 3      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NCOA3  OMIM: NUCLEAR RECEPTOR COACTIVATOR 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "BCAS4\t\n",
            "LRRN4\t\n",
            "TAF4\t\n",
            "LAMA5\t\n",
            "SLC17A9\t   From NCBI Gene:This gene encodes a member of a family of transmembrane proteins that are involved in the transport of small molecules. The encoded protein participates in the vesicular uptake, storage, and secretion of adenoside triphosphate (ATP) and other nucleotides. A mutation in this gene was found in individuals with autosomal dominant disseminated superficial actinic porokeratosis-8. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]From UniProt:Involved in vesicular storage and exocytosis of ATP. May accumulate ATP and other nucleotides in secretory vesicles such as adrenal chromaffin granules and synaptic vesicles.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Porokeratosis 8, disseminated superficial actinic typeFrom UniProt:Porokeratosis 8, disseminated superficial actinic type (POROK8): A form of porokeratosis, a disorder of faulty keratinization characterized by one or more atrophic patches surrounded by a distinctive hyperkeratotic ridgelike border called the cornoid lamella. The keratotic lesions can progress to overt cutaneous neoplasms, typically squamous cell carcinomas. Multiple clinical variants of porokeratosis are recognized, including porokeratosis of Mibelli, linear porokeratosis, disseminated superficial actinic porokeratosis, palmoplantar porokeratosis, and punctate porokeratosis. Disseminated superficial actinic porokeratosis (DSAP) is the most common subtype. It is characterized by multiple small, annular, anhidrotic, keratotic lesions that are located predominantly on sun-exposed areas of the skin, such as the face, neck, and distal limbs. The lesions typically begin to develop in adolescence and reach near-complete penetrance by the third or fourth decade of life. [MIM:616063]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20q13.33, which is the long (q) arm of chromosome 20 at position 13.33Molecular Location: base pairs 62,952,647 to 62,968,597 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C20orf59POROK8VNUT     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC17A9      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: POROKERATOSIS 8, DISSEMINATED SUPERFICIAL ACTINIC TYPEOMIM: SOLUTE CARRIER FAMILY 17 (VESICULAR NUCLEOTIDE TRANSPORTER), MEMBER 9      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SLC17A9  OMIM: POROKERATOSIS 8, DISSEMINATED SUPERFICIAL ACTINIC TYPEOMIM: SOLUTE CARRIER FAMILY 17 (VESICULAR NUCLEOTIDE TRANSPORTER), MEMBER 9  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NKAIN4\t\n",
            "HELZ2\t\n",
            "BMP2\t   From NCBI Gene:This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer, which plays a role in bone and cartilage development. Duplication of a regulatory region downstream of this gene causes a form of brachydactyly characterized by a malformed index finger and second toe in human patients. [provided by RefSeq, Jul 2016]From UniProt:Induces cartilage and bone formation (PubMed:3201241). Stimulates the differentiation of myoblasts into osteoblasts via the EIF2AK3-EIF2A- ATF4 pathway. BMP2 activation of EIF2AK3 stimulates phosphorylation of EIF2A which leads to increased expression of ATF4 which plays a central role in osteoblast differentiation. In addition stimulates TMEM119, which upregulates the expression of ATF4 (PubMed:24362451).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:SHORT STATURE, FACIAL DYSMORPHISM, AND SKELETAL ANOMALIES WITH OR WITHOUT CARDIAC ANOMALIESHemochromatosis type 1Type A2 brachydactylyFrom UniProt:Short stature, facial dysmorphism, and skeletal anomalies with or without cardiac anomalies (SSFSC): An autosomal dominant disorder characterized by short stature, facial dysmorphism, skeletal anomalies, and variable cardiac defects. Distinctive facial features include midface retrusion, short upturned nose, long philtrum, high-arched or cleft palate, and variable degrees of micrognathia and dental crowding. Skeletal anomalies include patterning defects of the axial skeleton, characterized by 11 pairs of ribs and brachydactyly of the fifth ray. Congenital heart defects are variably observed and appear to involve primarily the cardiac outflow tract. [MIM:617877]Brachydactyly A2 (BDA2): A form of brachydactyly. Brachydactyly defines a group of inherited malformations characterized by shortening of the digits due to abnormal development of the phalanges and/or the metacarpals. In brachydactyly type A2 shortening of the middle phalanges is confined to the index finger and the second toe, all other digits being more or less normal. Because of a rhomboid or triangular shape of the affected middle phalanx, the end of the second finger usually deviates radially. [MIM:112600]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 20p12.3, which is the short (p) arm of chromosome 20 at position 12.3Molecular Location: base pairs 6,768,098 to 6,780,280 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BDA2BMP2ASSFSC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of BMP2      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: BONE MORPHOGENETIC PROTEIN 2OMIM: BRACHYDACTYLY, TYPE A2OMIM: HEMOCHROMATOSIS, TYPE 1OMIM: SHORT STATURE, FACIAL DYSMORPHISM, AND SKELETAL ANOMALIES WITH OR WITHOUT CARDIAC ANOMALIES      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of BMP2  OMIM: BONE MORPHOGENETIC PROTEIN 2OMIM: BRACHYDACTYLY, TYPE A2OMIM: HEMOCHROMATOSIS, TYPE 1OMIM: SHORT STATURE, FACIAL DYSMORPHISM, AND SKELETAL ANOMALIES WITH OR WITHOUT CARDIAC ANOMALIES  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLC52A3\t   The SLC52A3 gene (previously called the C20orf54 gene) provides instructions for making a riboflavin transporter protein called RFVT3 (formerly known as RFT2). This protein moves (transports) a vitamin called riboflavin (also called vitamin B2) across the cell membrane. Riboflavin cannot be made by the body, so it must be obtained from the food a person eats. The RFVT3 protein is found at especially high levels in cells of the small intestine and is important for absorbing riboflavin during digestion so that the vitamin can be used in the body.In the cells of the body, riboflavin is the core component of molecules called flavin adenine dinucleotide (FAD) and flavin mononucleotide (FMN). These molecules function as coenzymes, which means they help enzymes carry out chemical reactions. FAD and FMN are involved in many different chemical reactions and are required for a variety of cellular processes. One important role of these coenzymes is in the production of energy for cells. FAD and FMN are also involved in the breakdown (metabolism) of carbohydrates, fats, and proteins.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Riboflavin transporter deficiency neuronopathy     More than two dozen mutations in the SLC52A3 gene have been found to cause riboflavin transporter deficiency neuronopathy. This neurological condition encompasses two disorders that were previously considered to be separate: Brown-Vialetto-Van Laere syndrome and Fazio-Londe disease. The gene mutations involved in this condition lead to production of abnormal RFVT3 proteins. Some mutations lead to the production of an altered protein that cannot get to the cell membrane, so it is unable to transport riboflavin into the cell. Other mutations lead to a version of the protein that can get to the cell membrane, but its function as a transporter is impaired. These changes impair the absorption of riboflavin in the small intestine. The resulting shortage of riboflavin leads to a reduction of FAD and FMN. However, it is unclear how these changes lead to the nerve problems that cause hearing loss, muscle weakness in the face and limbs, and breathing problems in people with the disorder. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than two dozen mutations in the SLC52A3 gene have been found to cause riboflavin transporter deficiency neuronopathy. This neurological condition encompasses two disorders that were previously considered to be separate: Brown-Vialetto-Van Laere syndrome and Fazio-Londe disease. The gene mutations involved in this condition lead to production of abnormal RFVT3 proteins. Some mutations lead to the production of an altered protein that cannot get to the cell membrane, so it is unable to transport riboflavin into the cell. Other mutations lead to a version of the protein that can get to the cell membrane, but its function as a transporter is impaired. These changes impair the absorption of riboflavin in the small intestine. The resulting shortage of riboflavin leads to a reduction of FAD and FMN. However, it is unclear how these changes lead to the nerve problems that cause hearing loss, muscle weakness in the face and limbs, and breathing problems in people with the disorder. More About This Health Condition    Cytogenetic Location: 20p13, which is the short (p) arm of chromosome 20 at position 13Molecular Location: base pairs 760,080 to 776,015 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA371L19.1BVVLSC20orf54hRFT2MGC10698RFT2RFT2_HUMANRFVT3riboflavin transporter 2solute carrier family 52 (riboflavin transporter), member 3solute carrier family 52, riboflavin transporter, member 3     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Riboflavin Transporter Deficiency Neuronopathy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC52A3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SOLUTE CARRIER FAMILY 52 (RIBOFLAVIN TRANSPORTER), MEMBER 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Riboflavin Transporter Deficiency Neuronopathy  Tests of SLC52A3  PubMed  SOLUTE CARRIER FAMILY 52 (RIBOFLAVIN TRANSPORTER), MEMBER 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bosch AM, Abeling NG, Ijlst L, Knoester H, van der Pol WL, Stroomer AE, Wanders RJ, Visser G, Wijburg FA, Duran M, Waterham HR. Brown-Vialetto-Van Laere and Fazio Londe syndrome is associated with a riboflavin transporter defect mimicking mild MADD: a new inborn error of metabolism with potential treatment. J Inherit Metab Dis. 2011 Feb;34(1):159-64. doi: 10.1007/s10545-010-9242-z. Epub 2010 Nov 26. Citation on PubMed or Free article on PubMed CentralDipti S, Childs AM, Livingston JH, Aggarwal AK, Miller M, Williams C, Crow YJ. Brown-Vialetto-Van Laere syndrome; variability in age at onset and disease progression highlighting the phenotypic overlap with Fazio-Londe disease. Brain Dev. 2005 Sep;27(6):443-6. Epub 2004 Dec 15. Citation on PubMedGreen P, Wiseman M, Crow YJ, Houlden H, Riphagen S, Lin JP, Raymond FL, Childs AM, Sheridan E, Edwards S, Josifova DJ. Brown-Vialetto-Van Laere syndrome, a ponto-bulbar palsy with deafness, is caused by mutations in c20orf54. Am J Hum Genet. 2010 Mar 12;86(3):485-9. doi: 10.1016/j.ajhg.2010.02.006. Epub 2010 Mar 4. Citation on PubMed or Free article on PubMed CentralManole A, Houlden H. Riboflavin Transporter Deficiency Neuronopathy. 2015 Jun 11. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK299312/ Citation on PubMedMcShane MA, Boyd S, Harding B, Brett EM, Wilson J. Progressive bulbar paralysis of childhood. A reappraisal of Fazio-Londe disease. Brain. 1992 Dec;115 ( Pt 6):1889-900. Citation on PubMedNabokina SM, Subramanian VS, Said HM. Effect of clinical mutations on functionality of the human riboflavin transporter-2 (hRFT-2). Mol Genet Metab. 2012 Apr;105(4):652-7. doi: 10.1016/j.ymgme.2011.12.021. Epub 2012 Jan 5. Citation on PubMed or Free article on PubMed CentralOMIM: SOLUTE CARRIER FAMILY 52 (RIBOFLAVIN TRANSPORTER), MEMBER 3Subramanian VS, Subramanya SB, Rapp L, Marchant JS, Ma TY, Said HM. Differential expression of human riboflavin transporters -1, -2, and -3 in polarized epithelia: a key role for hRFT-2 in intestinal riboflavin uptake. Biochim Biophys Acta. 2011 Dec;1808(12):3016-21. doi: 10.1016/j.bbamem.2011.08.004. Epub 2011 Aug 11. Citation on PubMed or Free article on PubMed CentralYonezawa A, Inui K. Novel riboflavin transporter family RFVT/SLC52: identification, nomenclature, functional characterization and genetic diseases of RFVT/SLC52. Mol Aspects Med. 2013 Apr-Jun;34(2-3):693-701. doi: 10.1016/j.mam.2012.07.014. Review. Citation on PubMed \n",
            "PLCB4\t   The PLCB4 gene provides instructions for making one form (the beta 4 isoform) of a protein called phospholipase C. This protein is involved in a signaling pathway within cells known as the phosphoinositide cycle, which helps transmit information from outside the cell to inside the cell. Phospholipase C carries out one particular step in the phosphoinositide cycle: the conversion of a molecule called phosphatidylinositol 4,5-bisphosphate (PIP2) to two smaller molecules, inositol 1,4,5-trisphosphate (IP3) and 1,2-diacylglycerol. These smaller molecules relay messages into the cell that ultimately influence many cell activities.Studies suggest that the beta 4 isoform of phospholipase C contributes to the development of the first and second pharyngeal arches. These embryonic structures ultimately develop into the jawbones, facial muscles, middle ear bones, ear canals, outer ears, and related tissues. This protein is also thought to play a role in vision, particularly in the function of the retina, which is a specialized tissue at the back of the eye that detects light and color.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Auriculo-condylar syndrome     At least nine mutations in the PLCB4 gene have been found to cause auriculo-condylar syndrome, a disorder that primarily affects the development of the ears and lower jaw (mandible). The identified mutations change single protein building blocks (amino acids) in the the beta 4 isoform of phospholipase C. These changes likely alter the structure of the protein and impair the phosphoinositide cycle. Abnormal signaling alters the formation of the lower jaw: instead of developing normally, the lower jaw becomes shaped more like the smaller upper jaw (maxilla). The abnormal shape leads to an unusually small chin (micrognathia) and problems with jaw function. Researchers are working to determine how mutations in this gene lead to the other developmental abnormalities associated with auriculo-condylar syndrome. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least nine mutations in the PLCB4 gene have been found to cause auriculo-condylar syndrome, a disorder that primarily affects the development of the ears and lower jaw (mandible). The identified mutations change single protein building blocks (amino acids) in the the beta 4 isoform of phospholipase C. These changes likely alter the structure of the protein and impair the phosphoinositide cycle. Abnormal signaling alters the formation of the lower jaw: instead of developing normally, the lower jaw becomes shaped more like the smaller upper jaw (maxilla). The abnormal shape leads to an unusually small chin (micrognathia) and problems with jaw function. Researchers are working to determine how mutations in this gene lead to the other developmental abnormalities associated with auriculo-condylar syndrome. More About This Health Condition    Cytogenetic Location: 20p12.3-p12.2, which is the short (p) arm of chromosome 20 between positions 12.3 and 12.2Molecular Location: base pairs 9,068,710 to 9,480,816 on chromosome 20 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        1-phosphatidyl-D-myo-inositol-4,5-bisphosphate1-phosphatidylinositol 4,5-bisphosphate phosphodiesterase beta-4ARCND2dJ1119D9.2 (Phopholipase C, beta 4 (1-Phosphatidylinositol-4,5-Bisphosphate Phosphodiesterase Beta 4))inositoltrisphosphohydrolasemonophosphatidylinositol phosphodiesterasephosphoinositidase Cphosphoinositide phospholipase C-beta-4phospholipase C, beta 4PI-PLCPLC-beta-4PLCB4_HUMANtriphosphoinositide phosphodiesterase     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): The Hydrolysis of Phosphatidyl Inositol Bisphosphate by Phospholipase C Generates Two MessengersMadame Curie Bioscience Database: Phospholipase C      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PLCB4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PHOSPHOLIPASE C, BETA-4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): The Hydrolysis of Phosphatidyl Inositol Bisphosphate by Phospholipase C Generates Two MessengersMadame Curie Bioscience Database: Phospholipase C  Tests of PLCB4  PubMed  PHOSPHOLIPASE C, BETA-4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Alvarez RA, Ghalayini AJ, Xu P, Hardcastle A, Bhattacharya S, Rao PN, Pettenati MJ, Anderson RE, Baehr W. cDNA sequence and gene locus of the human retinal phosphoinositide-specific phospholipase-C beta 4 (PLCB4). Genomics. 1995 Sep 1;29(1):53-61. Citation on PubMedGordon CT, Vuillot A, Marlin S, Gerkes E, Henderson A, AlKindy A, Holder-Espinasse M, Park SS, Omarjee A, Sanchis-Borja M, Bdira EB, Oufadem M, Sikkema-Raddatz B, Stewart A, Palmer R, McGowan R, Petit F, Delobel B, Speicher MR, Aurora P, Kilner D, Pellerin P, Simon M, Bonnefont JP, Tobias ES, Garca-Miar S, Bitner-Glindzicz M, Lindholm P, Meijer BA, Abadie V, Denoyelle F, Vazquez MP, Rotky-Fast C, Couloigner V, Pierrot S, Manach Y, Breton S, Hendriks YM, Munnich A, Jakobsen L, Kroisel P, Lin A, Kaban LB, Basel-Vanagaite L, Wilson L, Cunningham ML, Lyonnet S, Amiel J. Heterogeneity of mutational mechanisms and modes of inheritance in auriculocondylar syndrome. J Med Genet. 2013 Mar;50(3):174-86. doi: 10.1136/jmedgenet-2012-101331. Epub 2013 Jan 12. Citation on PubMedOMIM: PHOSPHOLIPASE C, BETA-4Rieder MJ, Green GE, Park SS, Stamper BD, Gordon CT, Johnson JM, Cunniff CM, Smith JD, Emery SB, Lyonnet S, Amiel J, Holder M, Heggie AA, Bamshad MJ, Nickerson DA, Cox TC, Hing AV, Horst JA, Cunningham ML. A human homeotic transformation resulting from mutations in PLCB4 and GNAI3 causes auriculocondylar syndrome. Am J Hum Genet. 2012 May 4;90(5):907-14. doi: 10.1016/j.ajhg.2012.04.002. Erratum in: Am J Hum Genet. 2012 Jun 8;90(6):1116. Am J Hum Genet. 2012 Aug 10;91(2):397. Citation on PubMed or Free article on PubMed CentralVines CM. Phospholipase C. Adv Exp Med Biol. 2012;740:235-54. doi: 10.1007/978-94-007-2888-2_10. Review. Citation on PubMed \n",
            "BAGE2\t\n",
            "LIPI\t   From NCBI Gene:The protein encoded by this gene is a phospholipase that hydrolyzes phosphatidic acid to produce lysophosphatidic acid. Defects in this gene are a cause of susceptibility to familial hypertrigliceridemia. This gene is also expressed at high levels in Ewing family tumor cells. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Dec 2014]From UniProt:Hydrolyzes specifically phosphatidic acid (PA) to produce 2-acyl lysophosphatidic acid (LPA; a potent bioactive lipid mediator) and fatty acid. Does not hydrolyze other phospholipids, like phosphatidylserine (PS), phosphatidylcholine (PC) and phosphatidylethanolamine (PE) or triacylglycerol (TG).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Familial hypertriglyceridemiaFrom UniProt:Hypertriglyceridemia, familial (FHTR): A common inherited disorder in which the concentration of very low density lipoprotein (VLDL) is elevated in the plasma. This leads to increased risk of heart disease, obesity, and pancreatitis. [MIM:145750]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 21q11.2, which is the long (q) arm of chromosome 21 at position 11.2Molecular Location: base pairs 14,108,813 to 14,210,953 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CT17LPDLmPA-PLA1 betaPLA1CPRED5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LIPI      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HYPERTRIGLYCERIDEMIA, FAMILIALOMIM: LIPASE I      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LIPI  OMIM: HYPERTRIGLYCERIDEMIA, FAMILIALOMIM: LIPASE I  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMPRSS15\t   From NCBI Gene:This gene encodes an enzyme that converts the pancreatic proenzyme trypsinogen to trypsin, which activates other proenzymes including chymotrypsinogen and procarboxypeptidases. The precursor protein is cleaved into two chains that form a heterodimer linked by a disulfide bond. This protein is a member of the trypsin family of peptidases. Mutations in this gene cause enterokinase deficiency, a malabsorption disorder characterized by diarrhea and failure to thrive. [provided by RefSeq, Jul 2008]From UniProt:Responsible for initiating activation of pancreatic proteolytic proenzymes (trypsin, chymotrypsin and carboxypeptidase A). It catalyzes the conversion of trypsinogen to trypsin which in turn activates other proenzymes including chymotrypsinogen, procarboxypeptidases, and proelastases.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Enterokinase deficiencyFrom UniProt:Enterokinase deficiency (ENTKD): Life-threatening intestinal malabsorption disorder characterized by diarrhea and failure to thrive. [MIM:226200]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 21q21.1, which is the long (q) arm of chromosome 21 at position 21.1Molecular Location: base pairs 18,269,107 to 18,403,806 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ENTKPRSS7     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TMPRSS15      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ENTEROKINASE DEFICIENCYOMIM: TRANSMEMBRANE PROTEASE, SERINE 15      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TMPRSS15  OMIM: ENTEROKINASE DEFICIENCYOMIM: TRANSMEMBRANE PROTEASE, SERINE 15  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ADAMTS5\t\n",
            "LTN1\t\n",
            "KRTAP13-2\t\n",
            "KRTAP15-1\t\n",
            "KRTAP19-6\t\n",
            "KRTAP6-3\t\n",
            "KRTAP6-1\t\n",
            "KRTAP7-1\t\n",
            "HUNK\t\n",
            "OLIG1\t\n",
            "DNAJC28\t\n",
            "GART\t\n",
            "SON\t   From NCBI Gene:This gene encodes a protein that contains multiple simple repeats. The encoded protein binds RNA and promotes pre-mRNA splicing, particularly of transcripts with poor splice sites. The protein also recognizes a specific DNA sequence found in the human hepatitis B virus (HBV) and represses HBV core promoter activity. There is a pseudogene for this gene on chromosome 1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]From UniProt:RNA-binding protein that acts as a mRNA splicing cofactor by promoting efficient splicing of transcripts that possess weak splice sites. Specifically promotes splicing of many cell-cycle and DNA-repair transcripts that possess weak splice sites, such as TUBG1, KATNB1, TUBGCP2, AURKB, PCNT, AKT1, RAD23A, and FANCG. Probably acts by facilitating the interaction between Serine/arginine-rich proteins such as SRSF2 and the RNA polymerase II. Also binds to DNA; binds to the consensus DNA sequence: 5'-GA[GT]AN[CG][AG]CC-3'. May indirectly repress hepatitis B virus (HBV) core promoter activity and transcription of HBV genes and production of HBV virions. Essential for correct RNA splicing of multiple genes critical for brain development, neuronal migration and metabolism, including TUBG1, FLNA, PNKP, WDR62, PSMD3, PCK2, PFKL, IDH2, and ACY1 (PubMed:27545680).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:ZTTK syndromeFrom UniProt:ZTTK syndrome (ZTTKS): An autosomal dominant syndrome characterized by intellectual disability, developmental delay, malformations of the cerebral cortex, epilepsy, vision problems, musculo-skeletal abnormalities, and congenital malformations. [MIM:617140]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 21q22.11, which is the long (q) arm of chromosome 21 at position 22.11Molecular Location: base pairs 33,543,038 to 33,577,514 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BASS1C21orf50DBP-5NREBPSON3TOKIMS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SON      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SON DNA-BINDING PROTEINOMIM: ZTTK SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SON  OMIM: SON DNA-BINDING PROTEINOMIM: ZTTK SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CLIC6\t\n",
            "TTC3\t\n",
            "BRWD1\t\n",
            "WRB-SH3BGR\t\n",
            "SH3BGR\t\n",
            "DSCAM\t   From NCBI Gene:This gene is a member of the immunoglobulin superfamily of cell adhesion molecules (Ig-CAMs), and is involved in human central and peripheral nervous system development. This gene is a candidate for Down syndrome and congenital heart disease (DSCHD). A gene encoding a similar Ig-CAM protein is located on chromosome 11. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Oct 2012]From UniProt:Cell adhesion molecule that plays a role in neuronal self-avoidance. Promotes repulsion between specific neuronal processes of either the same cell or the same subtype of cells. Mediates within retinal amacrine and ganglion cell subtypes both isoneuronal self-avoidance for creating an orderly dendritic arborization and heteroneuronal self-avoidance to maintain the mosaic spacing between amacrine and ganglion cell bodies (PubMed:10925149). Receptor for netrin required for axon guidance independently of and in collaboration with the receptor DCC. Might also collaborate with UNC5C in NTN1-mediated axon repulsion independently of DCC (By similarity). In spinal cord development plays a role in guiding commissural axons projection and pathfinding across the ventral midline to reach the floor plate upon ligand binding (PubMed:18585357, PubMed:19196994). Enhances netrin-induced phosphorylation of PAK1 and FYN (PubMed:15169762). Mediates intracellular signaling by stimulating the activation of MAPK8 and MAP kinase p38 (PubMed:18585357, PubMed:19196994). Adhesion molecule that promotes lamina-specific synaptic connections in the retina: expressed in specific subsets of interneurons and retinal ganglion cells (RGCs) and promotes synaptic connectivity via homophilic interactions (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autism spectrum disorder     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 21q22.2, which is the long (q) arm of chromosome 21 at position 22.2Molecular Location: base pairs 40,010,999 to 40,847,139 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CHD2CHD2-42CHD2-52     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DSCAM      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: DOWN SYNDROME CELL ADHESION MOLECULE      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DSCAM  OMIM: DOWN SYNDROME CELL ADHESION MOLECULE  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PRDM15\t\n",
            "UBASH3A\t\n",
            "KRTAP10-1\t\n",
            "KRTAP10-6\t\n",
            "KRTAP10-7\t\n",
            "KRTAP10-10\t\n",
            "KRTAP10-11\t\n",
            "ITGB2\t   The ITGB2 gene provides instructions for making one part (the 2 subunit) of at least four different proteins known as 2 integrins. The other subunit can be one of a variety of alpha () subunits that are produced from different genes. Integrins are a group of proteins that regulate the attachment of cells to one another (cell-cell adhesion) and to the surrounding network of proteins and other molecules (cell-matrix adhesion). Integrins also transmit signals that regulate cell growth and the activity of certain genes.Integrins that contain the 2 subunit are found embedded in the membrane that surrounds white blood cells (leukocytes). 2 integrins help leukocytes gather at sites of infection or injury, where they are needed to contribute to the immune response. 2 integrins recognize signs of inflammation and attach (bind) to proteins called ligands on the lining of blood vessels. This binding leads to linkage (adhesion) of the leukocyte to the blood vessel wall. Signaling through the 2 integrins triggers the transport of the attached leukocyte across the blood vessel wall to the site of infection or injury.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Leukocyte adhesion deficiency type 1     At least 90 mutations in the ITGB2 gene have been found to cause leukocyte adhesion deficiency type 1. This condition is characterized by severe, recurrent infections and delayed healing after injury. Many of the mutations that cause this condition change single protein building blocks (amino acids) in the 2 subunit and lead to the production of a 2 subunit that cannot bind with other subunits to form integrins. Rarely, defective 2 subunits are able to form integrins, but the integrins cannot bind ligands. Leukocytes that lack these integrins cannot attach to the blood vessel wall or cross the vessel wall to contribute to the immune response. As a result, there is a decreased response to injury and foreign invaders, such as bacteria and fungi, resulting in frequent infections, delayed wound healing, and other signs and symptoms of leukocyte adhesion deficiency type 1. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 90 mutations in the ITGB2 gene have been found to cause leukocyte adhesion deficiency type 1. This condition is characterized by severe, recurrent infections and delayed healing after injury. Many of the mutations that cause this condition change single protein building blocks (amino acids) in the 2 subunit and lead to the production of a 2 subunit that cannot bind with other subunits to form integrins. Rarely, defective 2 subunits are able to form integrins, but the integrins cannot bind ligands. Leukocytes that lack these integrins cannot attach to the blood vessel wall or cross the vessel wall to contribute to the immune response. As a result, there is a decreased response to injury and foreign invaders, such as bacteria and fungi, resulting in frequent infections, delayed wound healing, and other signs and symptoms of leukocyte adhesion deficiency type 1. More About This Health Condition    Cytogenetic Location: 21q22.3, which is the long (q) arm of chromosome 21 at position 22.3Molecular Location: base pairs 44,885,949 to 44,928,873 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD11b/ CD18CD11c/CD18CD11d/CD18CD18complement receptor C3 beta-subunitcomplement receptor C3 subunit betaCR3integrin beta-2integrin beta 2integrin beta chain, beta 2integrin, beta 2 (complement component 3 receptor 3 and 4 subunit)LFA-1 (L2)Mac-1 (M2)p150/95 (X2)D2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (fourth edition, 2002): Integrins      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ITGB2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     INTEGRIN, BETA-2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Integrins  Tests of ITGB2  PubMed  INTEGRIN, BETA-2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: INTEGRIN, BETA-2Parvaneh N, Mamishi S, Rezaei A, Rezaei N, Tamizifar B, Parvaneh L, Sherkat R, Ghalehbaghi B, Kashef S, Chavoshzadeh Z, Isaeian A, Ashrafi F, Aghamohammadi A. Characterization of 11 new cases of leukocyte adhesion deficiency type 1 with seven novel mutations in the ITGB2 gene. J Clin Immunol. 2010 Sep;30(5):756-60. doi: 10.1007/s10875-010-9433-2. Epub 2010 Jun 12. Citation on PubMedSchmidt S, Moser M, Sperandio M. The molecular basis of leukocyte recruitment and its deficiencies. Mol Immunol. 2013 Aug;55(1):49-58. doi: 10.1016/j.molimm.2012.11.006. Epub 2012 Dec 17. Review. Citation on PubMedvan de Vijver E, Maddalena A, Sanal , Holland SM, Uzel G, Madkaikar M, de Boer M, van Leeuwen K, Kker MY, Parvaneh N, Fischer A, Law SK, Klein N, Tezcan FI, Unal E, Patiroglu T, Belohradsky BH, Schwartz K, Somech R, Kuijpers TW, Roos D. Hematologically important mutations: leukocyte adhesion deficiency (first update). Blood Cells Mol Dis. 2012 Jan 15;48(1):53-61. doi: 10.1016/j.bcmd.2011.10.004. Epub 2011 Nov 30. Review. Citation on PubMed or Free article on PubMed Central \n",
            "COL18A1\t   The COL18A1 gene provides instructions for making a protein that forms collagen XVIII. Three COL18A1 proteins, called alpha 1 subunits, attach to each other to form collagen XVIII. Collagen XVIII is found in the basement membranes of tissues throughout the body. Basement membranes are thin, sheet-like structures that separate and support cells in these tissues.There are three versions (isoforms) of the alpha 1 subunit of collagen XVIII, which form three different lengths of the collagen XVIII protein. The short version of collagen XVIII is found in basement membranes throughout the body, including several parts of the eye. The two longer versions are found primarily in the liver. In addition, a piece of collagen XVIII can be cut off to form the protein endostatin. Endostatin is able to block the formation of blood vessels (angiogenesis) and is known as an anti-angiogenic factor.Little is known about the function of collagen XVIII, but it appears that all of the isoforms are involved in normal development of the eye.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Knobloch syndrome     At least a dozen mutations in the COL18A1 gene have been identified in people with Knobloch syndrome, a condition characterized by severe vision problems and a skull defect called an occipital encephalocele. Most COL18A1 gene mutations lead to an abnormally short version of the genetic blueprint used to make the alpha 1 subunit of collagen XVIII. There are a few other mutations that replace single protein building blocks (amino acids) in the protein sequence. Although the process is unclear, the COL18A1 gene mutations result in the loss of collagen XVIII. Most COL18A1 gene mutations affect all isoforms of collagen XVIII, but at least one mutation affects only the short isoform. The loss of one or all isoforms of the collagen XVIII protein likely causes the signs and symptoms of Knobloch syndrome. It is unclear whether endostatin is involved in this condition. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least a dozen mutations in the COL18A1 gene have been identified in people with Knobloch syndrome, a condition characterized by severe vision problems and a skull defect called an occipital encephalocele. Most COL18A1 gene mutations lead to an abnormally short version of the genetic blueprint used to make the alpha 1 subunit of collagen XVIII. There are a few other mutations that replace single protein building blocks (amino acids) in the protein sequence. Although the process is unclear, the COL18A1 gene mutations result in the loss of collagen XVIII. Most COL18A1 gene mutations affect all isoforms of collagen XVIII, but at least one mutation affects only the short isoform. The loss of one or all isoforms of the collagen XVIII protein likely causes the signs and symptoms of Knobloch syndrome. It is unclear whether endostatin is involved in this condition. More About This Health Condition    Cytogenetic Location: 21q22.3, which is the long (q) arm of chromosome 21 at position 22.3Molecular Location: base pairs 45,405,137 to 45,513,720 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        antiangiogenic agentcollagen alpha-1(XVIII) chaincollagen type XVIII alpha 1collagen, type XVIII, alpha 1endostatinFLJ27325FLJ34914KNOKNO1KSMGC74745multi-functional protein MFP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL18A1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     COLLAGEN, TYPE XVIII, ALPHA-1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COL18A1  PubMed  COLLAGEN, TYPE XVIII, ALPHA-1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: COLLAGEN, TYPE XVIII, ALPHA-1Edwards Y. Structure and expression of mammalian carbonic anhydrases. Biochem Soc Trans. 1990 Apr;18(2):171-5. Review. Citation on PubMedFukai N, Eklund L, Marneros AG, Oh SP, Keene DR, Tamarkin L, Niemel M, Ilves M, Li E, Pihlajaniemi T, Olsen BR. Lack of collagen XVIII/endostatin results in eye abnormalities. EMBO J. 2002 Apr 2;21(7):1535-44. Citation on PubMed or Free article on PubMed CentralPassos-Bueno MR, Suzuki OT, Armelin-Correa LM, Serti AL, Errera FI, Bagatini K, Kok F, Leite KR. Mutations in collagen 18A1 and their relevance to the human phenotype. An Acad Bras Cienc. 2006 Mar;78(1):123-31. Epub 2006 Mar 8. Review. Citation on PubMedSerti AL, Sossi V, Camargo AA, Zatz M, Brahe C, Passos-Bueno MR. Collagen XVIII, containing an endogenous inhibitor of angiogenesis and tumor growth, plays a critical role in the maintenance of retinal structure and in neural tube closure (Knobloch syndrome). Hum Mol Genet. 2000 Aug 12;9(13):2051-8. Citation on PubMedSuzuki O, Kague E, Bagatini K, Tu H, Heljasvaara R, Carvalhaes L, Gava E, de Oliveira G, Godoi P, Oliva G, Kitten G, Pihlajaniemi T, Passos-Bueno MR. Novel pathogenic mutations and skin biopsy analysis in Knobloch syndrome. Mol Vis. 2009;15:801-9. Epub 2009 Apr 23. Citation on PubMed or Free article on PubMed CentralSuzuki OT, Serti AL, Der Kaloustian VM, Kok F, Carpenter M, Murray J, Czeizel AE, Kliemann SE, Rosemberg S, Monteiro M, Olsen BR, Passos-Bueno MR. Molecular analysis of collagen XVIII reveals novel mutations, presence of a third isoform, and possible genetic heterogeneity in Knobloch syndrome. Am J Hum Genet. 2002 Dec;71(6):1320-9. Epub 2002 Nov 1. Citation on PubMed or Free article on PubMed Central \n",
            "COL6A2\t   The COL6A2 gene provides instructions for making one component of type VI collagen, which is a flexible protein found in the space that surrounds cells. Specifically, the protein produced from the COL6A2 gene is the alpha()2(VI) chain of type VI collagen. This chain combines with chains produced from other genes to produce a complete type VI collagen molecule.Collagens are found in the extracellular matrix, which is an intricate lattice that forms in the space between cells and provides structural support. Type VI collagen is located in the extracellular matrix surrounding cells that make up the muscles used for movement (skeletal muscle cells) and cells that make up connective tissue, which provides strength and flexibility to structures throughout the body, including skin and joints. The extracellular matrix is necessary for cell stability and growth. Research suggests that type VI collagen links basement membranes, which are thin, sheet-like structures that are part of the extracellular matrix, to nearby cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Collagen VI-related myopathy     Mutations in the COL6A2 gene have been found to cause some cases of collagen VI-related myopathy, which is a group of disorders that vary in severity but generally result in muscle weakness and joint deformities called contractures. These mutations often change single protein building blocks (amino acids) in the 2(VI) chain. The most frequently affected amino acid is glycine; changes to this building block disrupt the structure of the 2(VI) chain. Other mutations can also disrupt the structure of the 2(VI) chain.Mutations in the COL6A2 gene affect type VI collagen in different ways. Some mutations lead to altered 2(VI) chains that can be incorporated into the mature type VI collagen molecule, although they disrupt the molecule's structure and function. Other mutations result in an altered chain that cannot be incorporated at all. Still other mutations prevent the production of any functional 2(VI) chain, which impedes formation of type VI collagen. All of these COL6A2 gene mutations lead to a reduction or absence of functional collagen VI molecules. While it is difficult to predict the severity of collagen VI-related myopathy based on the type of mutation, in general, lower amounts of type VI collagen lead to more severe signs and symptoms that begin earlier in life.Changes in 2(VI) chain structure or production lead to an unstable extracellular matrix that is no longer attached to cells through the basement membrane. As a result, the stability of muscle cells and connective tissue progressively declines, which leads to the muscle weakness, contractures, and other signs and symptoms of collagen VI-related myopathy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the COL6A2 gene have been found to cause some cases of collagen VI-related myopathy, which is a group of disorders that vary in severity but generally result in muscle weakness and joint deformities called contractures. These mutations often change single protein building blocks (amino acids) in the 2(VI) chain. The most frequently affected amino acid is glycine; changes to this building block disrupt the structure of the 2(VI) chain. Other mutations can also disrupt the structure of the 2(VI) chain.Mutations in the COL6A2 gene affect type VI collagen in different ways. Some mutations lead to altered 2(VI) chains that can be incorporated into the mature type VI collagen molecule, although they disrupt the molecule's structure and function. Other mutations result in an altered chain that cannot be incorporated at all. Still other mutations prevent the production of any functional 2(VI) chain, which impedes formation of type VI collagen. All of these COL6A2 gene mutations lead to a reduction or absence of functional collagen VI molecules. While it is difficult to predict the severity of collagen VI-related myopathy based on the type of mutation, in general, lower amounts of type VI collagen lead to more severe signs and symptoms that begin earlier in life.Changes in 2(VI) chain structure or production lead to an unstable extracellular matrix that is no longer attached to cells through the basement membrane. As a result, the stability of muscle cells and connective tissue progressively declines, which leads to the muscle weakness, contractures, and other signs and symptoms of collagen VI-related myopathy. More About This Health Condition    Cytogenetic Location: 21q22.3, which is the long (q) arm of chromosome 21 at position 22.3Molecular Location: base pairs 46,098,071 to 46,132,849 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CO6A2_HUMANcollagen alpha-2(VI) chaincollagen type VI alpha 2collagen VI, alpha-2 polypeptidecollagen, type VI, alpha 2     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Cell Biology (fourth edition, 2000): Collagens Form Diverse StructuresThe Cell: A Molecular Approach (second edition, 2000): The Extracellular MatrixWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Collagen Type VI-Related Disorders      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL6A2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     COLLAGEN, TYPE VI, ALPHA-2      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Collagens Form Diverse StructuresThe Cell: A Molecular Approach (second edition, 2000): The Extracellular MatrixWashington University, St. Louis: Neuromuscular Disease Center  Collagen Type VI-Related Disorders  Tests of COL6A2  PubMed  COLLAGEN, TYPE VI, ALPHA-2  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Allamand V, Brias L, Richard P, Stojkovic T, Quijano-Roy S, Bonne G. ColVI myopathies: where do we stand, where do we go? Skelet Muscle. 2011 Sep 23;1:30. doi: 10.1186/2044-5040-1-30. Citation on PubMed or Free article on PubMed CentralBaker NL, Mrgelin M, Pace RA, Peat RA, Adams NE, Gardner RJ, Rowland LP, Miller G, De Jonghe P, Ceulemans B, Hannibal MC, Edwards M, Thompson EM, Jacobson R, Quinlivan RC, Aftimos S, Kornberg AJ, North KN, Bateman JF, Lamand SR. Molecular consequences of dominant Bethlem myopathy collagen VI mutations. Ann Neurol. 2007 Oct;62(4):390-405. Citation on PubMedBaker NL, Mrgelin M, Peat R, Goemans N, North KN, Bateman JF, Lamand SR. Dominant collagen VI mutations are a common cause of Ullrich congenital muscular dystrophy. Hum Mol Genet. 2005 Jan 15;14(2):279-93. Epub 2004 Nov 24. Citation on PubMedBushby KM, Collins J, Hicks D. Collagen type VI myopathies. Adv Exp Med Biol. 2014;802:185-99. doi: 10.1007/978-94-007-7893-1_12. Review. Citation on PubMedBnnemann CG. The collagen VI-related myopathies: muscle meets its matrix. Nat Rev Neurol. 2011 Jun 21;7(7):379-90. doi: 10.1038/nrneurol.2011.81. Review. Citation on PubMedOMIM: COLLAGEN, TYPE VI, ALPHA-2 \n",
            "LSS\t   From NCBI Gene:The protein encoded by this gene catalyzes the conversion of (S)-2,3 oxidosqualene to lanosterol. The encoded protein is a member of the terpene cyclase/mutase family and catalyzes the first step in the biosynthesis of cholesterol, steroid hormones, and vitamin D. Alternative splicing results in multiple transcript variants encoding different isoforms.[provided by RefSeq, Feb 2009]From UniProt:Catalyzes the cyclization of (S)-2,3 oxidosqualene to lanosterol, a reaction that forms the sterol nucleus. Through the production of lanosterol may regulate lens protein aggregation and increase transparency.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:HYPOTRICHOSIS 14Cataract 44From UniProt:Cataract 44 (CTRCT44): An opacification of the crystalline lens of the eye that frequently results in visual impairment or blindness. Opacities vary in morphology, are often confined to a portion of the lens, and may be static or progressive. In general, the more posteriorly located and dense an opacity, the greater the impact on visual function. [MIM:616509]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 21q22.3, which is the long (q) arm of chromosome 21 at position 22.3Molecular Location: base pairs 46,188,446 to 46,228,824 on chromosome 21 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CTRCT44HYPT14OSC     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LSS      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CATARACT 44OMIM: HYPOTRICHOSIS 14OMIM: LANOSTEROL SYNTHASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LSS  OMIM: CATARACT 44OMIM: HYPOTRICHOSIS 14OMIM: LANOSTEROL SYNTHASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CECR2\t\n",
            "CLTCL1\t\n",
            "CLDN5\t\n",
            "TXNRD2\t   From NCBI Gene:The protein encoded by this gene belongs to the pyridine nucleotide-disulfide oxidoreductase family, and is a member of the thioredoxin (Trx) system. Three thioredoxin reductase (TrxR) isozymes are found in mammals. TrxRs are selenocysteine-containing flavoenzymes, which reduce thioredoxins, as well as other substrates, and play a key role in redox homoeostasis. This gene encodes a mitochondrial form important for scavenging reactive oxygen species in mitochondria. It functions as a homodimer containing FAD, and selenocysteine (Sec) at the active site. Sec is encoded by UGA codon that normally signals translation termination. The 3' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, the Sec insertion sequence (SECIS) element, which is necessary for the recognition of UGA as a Sec codon rather than as a stop signal. Alternatively spliced transcript variants encoding different isoforms, including a few localized in the cytosol and some lacking the C-terminal Sec residue, have been found for this gene. [provided by RefSeq, Jun 2017]From UniProt:Involved in the control of reactive oxygen species levels and the regulation of mitochondrial redox homeostasis (PubMed:24601690). Maintains thioredoxin in a reduced state. May play a role in redox-regulated cell signaling.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Familial glucocorticoid deficiencyFrom NCBI Gene:GLUCOCORTICOID DEFICIENCY 5From UniProt:Glucocorticoid deficiency 5 (GCCD5): A form of glucocorticoid deficiency, a rare autosomal recessive disorder characterized by resistance to ACTH action on the adrenal cortex, adrenal insufficiency and an inability of the adrenal cortex to produce cortisol. It usually presents in the neonatal period or in early childhood with episodes of hypoglycemia and other symptoms related to cortisol deficiency, including failure to thrive, recurrent illnesses or infections, convulsions, and shock. In a small number of patients hypoglycemia can be sufficiently severe and persistent that it leads to serious long-term neurological damage or death. The diagnosis is readily confirmed with a low plasma cortisol measurement in the presence of an elevated ACTH level, and normal aldosterone and plasma renin measurements. [MIM:617825]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q11.21, which is the long (q) arm of chromosome 22 at position 11.21Molecular Location: base pairs 19,875,518 to 19,941,992 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GCCD5SELZTRTR-BETATR3TRXR2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TXNRD2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GLUCOCORTICOID DEFICIENCY 5OMIM: THIOREDOXIN REDUCTASE 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TXNRD2  OMIM: GLUCOCORTICOID DEFICIENCY 5OMIM: THIOREDOXIN REDUCTASE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RIMBP3\t\n",
            "SCARF2\t   From NCBI Gene:The protein encoded by this gene is similar to  SCARF1/SREC-I, a scavenger receptor protein that mediates the binding and degradation of acetylated low density lipoprotein (Ac-LDL). This protein has only little activity of internalizing modified low density lipoproteins (LDL), but it can interact with SCARF1 through its extracellular domain. The association of this protein with SCARF1 is suppressed by the presence of scavenger ligands. Alternatively spliced transcript variants encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008]From UniProt:Probable adhesion protein, which mediates homophilic and heterophilic interactions. In contrast to SCARF1, it poorly mediates the binding and degradation of acetylated low density lipoprotein (Ac-LDL) (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Marden Walker like syndromeFrom UniProt:Van den Ende-Gupta syndrome (VDEGS): A syndrome characterized by craniofacial and skeletal abnormalities that include blepharophimosis, a flat and wide nasal bridge, narrow and beaked nose, hypoplastic maxilla with or without cleft palate and everted lower lip, prominent ears, down-slanting eyes, arachnodactyly, and camptodactyly. Patients present congenital joint contractures that improve without intervention, and normal growth and development. Intelligence is normal. Rarely, enlarged cerebella can be present. Some patients experience respiratory problems due to laryngeal abnormalities. [MIM:600920]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q11.21, which is the long (q) arm of chromosome 22 at position 11.21Molecular Location: base pairs 20,424,584 to 20,437,859 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        NSR1SREC-IISREC2SRECRP-1VDEGS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SCARF2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SCAVENGER RECEPTOR CLASS F, MEMBER 2OMIM: VAN DEN ENDE-GUPTA SYNDROME      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SCARF2  OMIM: SCAVENGER RECEPTOR CLASS F, MEMBER 2OMIM: VAN DEN ENDE-GUPTA SYNDROME  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MED15\t\n",
            "RIMBP3B\t\n",
            "IGLV10-54\t\n",
            "IGLV5-48\t\n",
            "IGLV3-16\t\n",
            "IGLV3-12\t\n",
            "IGLV3-10\t\n",
            "SLC2A11\t\n",
            "PIWIL3\t\n",
            "SGSM1\t\n",
            "MYO18B\t   From NCBI Gene:The protein encoded by this gene may regulate muscle-specific genes when in the nucleus and may influence intracellular trafficking when in the cytoplasm. The encoded protein functions as a homodimer and may interact with F actin. Mutations in this gene are associated with lung cancer. [provided by RefSeq, Jul 2008]From UniProt:May be involved in intracellular trafficking of the muscle cell when in the cytoplasm, whereas entering the nucleus, may be involved in the regulation of muscle specific genes. May play a role in the control of tumor development and progression; restored MYO18B expression in lung cancer cells suppresses anchorage-independent growth.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Klippel-feil syndrome 4, autosomal recessive, with nemaline myopathy and facial dysmorphismFrom UniProt:Klippel-Feil syndrome 4, autosomal recessive, with nemaline myopathy and facial dysmorphism (KFS4): A form of Klippel-Feil syndrome, a skeletal disorder characterized by congenital fusion of cervical vertebrae. It is due to a failure in the normal segmentation of vertebrae during the early weeks of fetal development. The clinical triad consists of short neck, low posterior hairline, and limited neck movement. KFS4 features additionally include myopathy, mild short stature, microcephaly, and distinctive facies. [MIM:616549]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q12.1, which is the long (q) arm of chromosome 22 at position 12.1Molecular Location: base pairs 25,742,133 to 26,063,847 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KFS4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYO18B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KLIPPEL-FEIL SYNDROME 4, AUTOSOMAL RECESSIVE, WITH NEMALINE MYOPATHY AND FACIAL DYSMORPHISMOMIM: MYOSIN XVIIIB      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MYO18B  OMIM: KLIPPEL-FEIL SYNDROME 4, AUTOSOMAL RECESSIVE, WITH NEMALINE MYOPATHY AND FACIAL DYSMORPHISMOMIM: MYOSIN XVIIIB  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HPS4\t   From NCBI Gene:This gene encodes a protein component of biogenesis of lysosome-related organelles complexes (BLOC). BLOC complexes are important for the formation of endosomal-lysosomal organelles such as melanosomes and platelet dense granules. Mutations in this gene result in subtype 4 of Hermansky-Pudlak syndrome, a form of albinism. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2012]From UniProt:Component of the BLOC-3 complex, a complex that acts as a guanine exchange factor (GEF) for RAB32 and RAB38, promotes the exchange of GDP to GTP, converting them from an inactive GDP-bound form into an active GTP-bound form. The BLOC-3 complex plays an important role in the control of melanin production and melanosome biogenesis and promotes the membrane localization of RAB32 and RAB38 (PubMed:23084991).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Hermansky-Pudlak syndromeFrom NCBI Gene:Hermansky-Pudlak syndrome 4From UniProt:Hermansky-Pudlak syndrome 4 (HPS4): A form of Hermansky-Pudlak syndrome, a genetically heterogeneous autosomal recessive disorder characterized by oculocutaneous albinism, bleeding due to platelet storage pool deficiency, and lysosomal storage defects. This syndrome results from defects of diverse cytoplasmic organelles including melanosomes, platelet dense granules and lysosomes. Ceroid storage in the lungs is associated with pulmonary fibrosis, a common cause of premature death in individuals with HPS. [MIM:614073]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q12.1, which is the long (q) arm of chromosome 22 at position 12.1Molecular Location: base pairs 26,443,109 to 26,483,863 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BLOC3S2LE     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HPS4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HERMANSKY-PUDLAK SYNDROME 4OMIM: HPS4 GENE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HPS4  OMIM: HERMANSKY-PUDLAK SYNDROME 4OMIM: HPS4 GENE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SRRD\t\n",
            "ZNRF3\t\n",
            "GAS2L1\t\n",
            "NEFH\t   From NCBI Gene:Neurofilaments are type IV intermediate filament heteropolymers composed of light, medium, and heavy chains. Neurofilaments comprise the axoskeleton and functionally maintain neuronal caliber. They may also play a role in intracellular transport to axons and dendrites. This gene encodes the heavy neurofilament protein. This protein is commonly used as a biomarker of neuronal damage and susceptibility to amyotrophic lateral sclerosis (ALS) has been associated with mutations in this gene. [provided by RefSeq, Oct 2008]From UniProt:Neurofilaments usually contain three intermediate filament proteins: L, M, and H which are involved in the maintenance of neuronal caliber. NF-H has an important function in mature axons that is not subserved by the two smaller NF proteins.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Amyotrophic lateral sclerosisCharcot-Marie-Tooth diseaseFrom NCBI Gene:Amyotrophic lateral sclerosis type 1Charcot-Marie-Tooth disease, axonal, type 2CCFrom UniProt:Charcot-Marie-Tooth disease 2CC (CMT2CC): An axonal form of Charcot-Marie-Tooth disease, a disorder of the peripheral nervous system, characterized by progressive weakness and atrophy, initially of the peroneal muscles and later of the distal muscles of the arms. Charcot-Marie-Tooth disease is classified in two main groups on the basis of electrophysiologic properties and histopathology: primary peripheral demyelinating neuropathies (designated CMT1 when they are dominantly inherited) and primary peripheral axonal neuropathies (CMT2). Neuropathies of the CMT2 group are characterized by signs of axonal degeneration in the absence of obvious myelin alterations, normal or slightly reduced nerve conduction velocities, and progressive distal muscle weakness and atrophy. [MIM:616924]Amyotrophic lateral sclerosis (ALS): A neurodegenerative disorder affecting upper motor neurons in the brain and lower motor neurons in the brain stem and spinal cord, resulting in fatal paralysis. Sensory abnormalities are absent. The pathologic hallmarks of the disease include pallor of the corticospinal tract due to loss of motor neurons, presence of ubiquitin-positive inclusions within surviving motor neurons, and deposition of pathologic aggregates. The etiology of amyotrophic lateral sclerosis is likely to be multifactorial, involving both genetic and environmental factors. The disease is inherited in 5-10% of the cases. [MIM:105400]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q12.2, which is the long (q) arm of chromosome 22 at position 12.2Molecular Location: base pairs 29,480,192 to 29,491,290 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMT2CCNFH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NEFH      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: AMYOTROPHIC LATERAL SCLEROSIS 1OMIM: CHARCOT-MARIE-TOOTH DISEASE, AXONAL, TYPE 2CCOMIM: NEUROFILAMENT PROTEIN, HEAVY POLYPEPTIDE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NEFH  OMIM: AMYOTROPHIC LATERAL SCLEROSIS 1OMIM: CHARCOT-MARIE-TOOTH DISEASE, AXONAL, TYPE 2CCOMIM: NEUROFILAMENT PROTEIN, HEAVY POLYPEPTIDE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLC35E4\t\n",
            "PATZ1\t   From NCBI Gene:The protein encoded by this gene contains an A-T hook DNA binding motif which usually binds to other DNA binding structures to play an important role in chromatin modeling and transcription regulation. Its Poz domain is thought to function as a site for protein-protein interaction and is required for transcriptional repression, and the zinc-fingers comprise the DNA binding domain. Since the encoded protein has typical features of a transcription factor, it is postulated to be a repressor of gene expression. In small round cell sarcoma, this gene is fused to EWS by a small inversion of 22q, then the hybrid is thought to be translocated (t(1;22)(p36.1;q12). The rearrangement of chromosome 22 involves intron 8 of EWS and exon 1 of this gene creating a chimeric sequence containing the transactivation domain of EWS fused to zinc finger domain of this protein. This is a distinct example of an intra-chromosomal rearrangement of chromosome 22. Four alternatively spliced transcript variants are described for this gene. [provided by RefSeq, Jul 2008]From UniProt:Transcriptional repressor.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving PATZ1 is associated with small round cell sarcoma. Translocation t(1;22)(p36.1;q12) with EWSR1.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q12.2, which is the long (q) arm of chromosome 22 at position 12.2Molecular Location: base pairs 31,325,804 to 31,346,263 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        dJ400N23MAZRPATZRIAZZBTB19ZNF278ZSG     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PATZ1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: ZINC FINGER PROTEIN 278      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PATZ1  OMIM: ZINC FINGER PROTEIN 278  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RFPL2\t\n",
            "APOL5\t\n",
            "APOL3\t\n",
            "APOL1\t   From NCBI Gene:This gene encodes a secreted high density lipoprotein which binds to apolipoprotein A-I. Apolipoprotein A-I is a relatively abundant plasma protein and is the major apoprotein of HDL. It is involved in the formation of most cholesteryl esters in plasma and also promotes efflux of cholesterol from cells. This apolipoprotein L family member may play a role in lipid exchange and transport throughout the body, as well as in reverse cholesterol transport from peripheral cells to the liver. Several different transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2008]From UniProt:May play a role in lipid exchange and transport throughout the body. May participate in reverse cholesterol transport from peripheral cells to the liver.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Focal segmental glomerulosclerosis 4, susceptibility toFrom UniProt:Focal segmental glomerulosclerosis 4 (FSGS4): A renal pathology defined by the presence of segmental sclerosis in glomeruli and resulting in proteinuria, reduced glomerular filtration rate and progressive decline in renal function. Renal insufficiency often progresses to end-stage renal disease, a highly morbid state requiring either dialysis therapy or kidney transplantation. [MIM:612551]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q12.3, which is the long (q) arm of chromosome 22 at position 12.3Molecular Location: base pairs 36,253,071 to 36,267,531 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        APO-LAPOLAPOL-IFSGS4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of APOL1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: APOLIPOPROTEIN L-IOMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 4, SUSCEPTIBILITY TO      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of APOL1  OMIM: APOLIPOPROTEIN L-IOMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 4, SUSCEPTIBILITY TO  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TXN2\t   From NCBI Gene:This nuclear gene encodes a mitochondrial member of the thioredoxin family, a group of small multifunctional redox-active proteins. The encoded protein may play important roles in the regulation of the mitochondrial membrane potential and in protection against oxidant-induced apoptosis. [provided by RefSeq, Jul 2008]From UniProt:Important for the control of mitochondrial reactive oxygen species homeostasis, apoptosis regulation and cell viability. Possesses a dithiol-reducing activity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Combined oxidative phosphorylation deficiency 29From UniProt:Combined oxidative phosphorylation deficiency 29 (COXPD29): An autosomal recessive, infantile-onset, neurodegenerative disorder characterized by decreased activities of mitochondrial respiratory complexes I and III, severe cerebellar atrophy, epilepsy, dystonia, optic atrophy, and peripheral neuropathy. [MIM:616811]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q12.3, which is the long (q) arm of chromosome 22 at position 12.3Molecular Location: base pairs 36,467,036 to 36,482,025 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        COXPD29MT-TRXMTRXTRX2TXN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TXN2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: COMBINED OXIDATIVE PHOSPHORYLATION DEFICIENCY 29OMIM: THIOREDOXIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TXN2  OMIM: COMBINED OXIDATIVE PHOSPHORYLATION DEFICIENCY 29OMIM: THIOREDOXIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMPRSS6\t   The TMPRSS6 gene provides instructions for making a protein called matriptase-2. This protein is part of a signaling pathway that controls the levels of another protein called hepcidin, which is a key regulator of iron balance in the body. When blood iron levels are low, this signaling pathway reduces hepcidin production, allowing more iron from the diet to be absorbed through the intestines and transported out of storage sites (particularly in the liver and spleen) into the bloodstream. Iron is an essential component of hemoglobin, which is the molecule in red blood cells that carries oxygen.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Iron-refractory iron deficiency anemia     At least 40 mutations in the TMPRSS6 gene have been found to cause an inherited form of anemia called iron-refractory iron deficiency anemia. This condition is characterized by a shortage (deficiency) of iron in the bloodstream that is resistant (refractory) to treatment with iron.TMPRSS6 gene mutations greatly reduce the amount of functional matriptase-2, preventing it from controlling hepcidin levels. The resulting elevation in hepcidin activity blocks the absorption of iron through the intestines and the release of iron from storage. When not enough iron is available in the bloodstream, less hemoglobin is produced and red blood cells cannot carry oxygen to the body's cells and tissues effectively. The shortage of oxygen causes the signs and symptoms of anemia, which can include  tiredness (fatigue), weakness, and pale skin. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 40 mutations in the TMPRSS6 gene have been found to cause an inherited form of anemia called iron-refractory iron deficiency anemia. This condition is characterized by a shortage (deficiency) of iron in the bloodstream that is resistant (refractory) to treatment with iron.TMPRSS6 gene mutations greatly reduce the amount of functional matriptase-2, preventing it from controlling hepcidin levels. The resulting elevation in hepcidin activity blocks the absorption of iron through the intestines and the release of iron from storage. When not enough iron is available in the bloodstream, less hemoglobin is produced and red blood cells cannot carry oxygen to the body's cells and tissues effectively. The shortage of oxygen causes the signs and symptoms of anemia, which can include  tiredness (fatigue), weakness, and pale skin. More About This Health Condition    Cytogenetic Location: 22q12.3, which is the long (q) arm of chromosome 22 at position 12.3Molecular Location: base pairs 37,065,436 to 37,110,625 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        matriptase 2matriptase-2membrane-bound mosaic serine proteinase matriptase-2transmembrane protease serine 6transmembrane protease, serine 6type II transmembrane serine protease 6     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Hemoglobin Transports Oxygen Efficiently by Binding Oxygen Cooperatively      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TMPRSS6      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSMEMBRANE PROTEASE, SERINE 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Hemoglobin Transports Oxygen Efficiently by Binding Oxygen Cooperatively  Tests of TMPRSS6  PubMed  TRANSMEMBRANE PROTEASE, SERINE 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Camaschella C, Silvestri L. Molecular mechanisms regulating hepcidin revealed by hepcidin disorders. ScientificWorldJournal. 2011 Jul 7;11:1357-66. doi: 10.1100/tsw.2011.130. Review. Citation on PubMedDe Falco L, Totaro F, Nai A, Pagani A, Girelli D, Silvestri L, Piscopo C, Campostrini N, Dufour C, Al Manjomi F, Minkov M, Van Vuurden DG, Feliu A, Kattamis A, Camaschella C, Iolascon A. Novel TMPRSS6 mutations associated with iron-refractory iron deficiency anemia (IRIDA). Hum Mutat. 2010 May;31(5):E1390-405. doi: 10.1002/humu.21243. Citation on PubMedDu X, She E, Gelbart T, Truksa J, Lee P, Xia Y, Khovananth K, Mudd S, Mann N, Moresco EM, Beutler E, Beutler B. The serine protease TMPRSS6 is required to sense iron deficiency. Science. 2008 May 23;320(5879):1088-92. doi: 10.1126/science.1157121. Epub 2008 May 1. Citation on PubMed or Free article on PubMed CentralMeynard D, Vaja V, Sun CC, Corradini E, Chen S, Lpez-Otn C, Grgurevic L, Hong CC, Stirnberg M, Gtschow M, Vukicevic S, Babitt JL, Lin HY. Regulation of TMPRSS6 by BMP6 and iron in human cells and mice. Blood. 2011 Jul 21;118(3):747-56. doi: 10.1182/blood-2011-04-348698. Epub 2011 May 26. Citation on PubMed or Free article on PubMed CentralRamsay AJ, Hooper JD, Folgueras AR, Velasco G, Lpez-Otn C. Matriptase-2 (TMPRSS6): a proteolytic regulator of iron homeostasis. Haematologica. 2009 Jun;94(6):840-9. doi: 10.3324/haematol.2008.001867. Epub 2009 Apr 18. Review. Citation on PubMed or Free article on PubMed CentralStirnberg M, Gtschow M. Matriptase-2, a regulatory protease of iron homeostasis: possible substrates, cleavage sites and inhibitors. Curr Pharm Des. 2013;19(6):1052-61. Review. Citation on PubMedZhao N, Zhang AS, Enns CA. Iron regulation by hepcidin. J Clin Invest. 2013 Jun;123(6):2337-43. doi: 10.1172/JCI67225. Epub 2013 Jun 3. Citation on PubMed or Free article on PubMed Central \n",
            "CDC42EP1\t\n",
            "BAIAP2L2\t\n",
            "KDELR3\t\n",
            "APOBEC3B\t\n",
            "APOBEC3F\t\n",
            "APOBEC3H\t\n",
            "SYNGR1\t\n",
            "TNRC6B\t\n",
            "CENPM\t\n",
            "TTLL12\t\n",
            "PNPLA3\t   The PNPLA3 gene provides instructions for making a protein called adiponutrin, which is found in fat cells (adipocytes) and liver cells (hepatocytes). The liver is a part of the digestive system that helps break down food, store energy, and remove waste products, including toxins. The function of the adiponutrin protein is not well understood, but it is thought to help regulate the development of adipocytes and the production and breakdown of fats (lipogenesis and lipolysis) in hepatocytes and adipocytes. Studies indicate that the activity (expression) of the PNPLA3 gene decreases during periods without food (fasting) and increases after eating, suggesting that the amount of adiponutrin protein produced is regulated as needed to help process and store fats in the diet.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Non-alcoholic fatty liver disease     A particular variation in the PNPLA3 gene has been associated with an increased risk of developing non-alcoholic fatty liver disease (NAFLD). NAFLD is a buildup of excessive fat in the liver that can lead to liver damage resembling the damage caused by alcohol abuse, but that occurs in people who do not drink heavily. In some cases NAFLD leads to inflammation of the liver (non-alcoholic steatohepatitis, also known as NASH) and permanent liver damage (cirrhosis).The PNPLA3 gene variation associated with NAFLD changes the protein building block (amino acid) isoleucine to the amino acid methionine at protein position 148, written as Ile148Met or I148M. Research suggests that the altered protein leads to increased production and decreased breakdown of fats in the liver. Studies are ongoing to determine how this and other genetic changes contribute to the development of NAFLD and its complications. More About This Health Condition      Other disorders     The I148M variation of the PNPLA3 gene has also been associated with the worsening of other liver diseases, such as a viral infection called hepatitis C. The variation also increases the risk of liver damage in people with alcoholism. The mechanism of this effect is not well understood, but the altered protein appears to increase fibrosis of the liver in people with these conditions.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A particular variation in the PNPLA3 gene has been associated with an increased risk of developing non-alcoholic fatty liver disease (NAFLD). NAFLD is a buildup of excessive fat in the liver that can lead to liver damage resembling the damage caused by alcohol abuse, but that occurs in people who do not drink heavily. In some cases NAFLD leads to inflammation of the liver (non-alcoholic steatohepatitis, also known as NASH) and permanent liver damage (cirrhosis).The PNPLA3 gene variation associated with NAFLD changes the protein building block (amino acid) isoleucine to the amino acid methionine at protein position 148, written as Ile148Met or I148M. Research suggests that the altered protein leads to increased production and decreased breakdown of fats in the liver. Studies are ongoing to determine how this and other genetic changes contribute to the development of NAFLD and its complications. More About This Health Condition  The I148M variation of the PNPLA3 gene has also been associated with the worsening of other liver diseases, such as a viral infection called hepatitis C. The variation also increases the risk of liver damage in people with alcoholism. The mechanism of this effect is not well understood, but the altered protein appears to increase fibrosis of the liver in people with these conditions.    Cytogenetic Location: 22q13.31, which is the long (q) arm of chromosome 22 at position 13.31Molecular Location: base pairs 43,923,739 to 43,947,568 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        acylglycerol O-acyltransferaseadiponutrinADPNC22orf20calcium-independent phospholipase A2-epsilondJ796I17.1FLJ22012iPLA(2)epsiloniPLA2-epsiloniPLA2epsilonpatatin-like phospholipase domain-containing protein 3     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Endotext: Lipid and Lipoprotein Metabolism in Liver Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PNPLA3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PATATIN-LIKE PHOSPHOLIPASE DOMAIN-CONTAINING PROTEIN 3      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Endotext: Lipid and Lipoprotein Metabolism in Liver Disease  Tests of PNPLA3  PubMed  PATATIN-LIKE PHOSPHOLIPASE DOMAIN-CONTAINING PROTEIN 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Baulande S, Lasnier F, Lucas M, Pairault J. Adiponutrin, a transmembrane protein corresponding to a novel dietary- and obesity-linked mRNA specifically expressed in the adipose lineage. J Biol Chem. 2001 Sep 7;276(36):33336-44. Epub 2001 Jun 28. Citation on PubMedCaligiuri A, Gentilini A, Marra F. Molecular Pathogenesis of NASH. Int J Mol Sci. 2016 Sep 20;17(9). pii: E1575. Review. Citation on PubMed or Free article on PubMed CentralChen LZ, Xin YN, Geng N, Jiang M, Zhang DD, Xuan SY. PNPLA3 I148M variant in nonalcoholic fatty liver disease: demographic and ethnic characteristics and the role of the variant in nonalcoholic fatty liver fibrosis. World J Gastroenterol. 2015 Jan 21;21(3):794-802. doi: 10.3748/wjg.v21.i3.794. Review. Citation on PubMed or Free article on PubMed CentralDongiovanni P, Donati B, Fares R, Lombardi R, Mancina RM, Romeo S, Valenti L. PNPLA3 I148M polymorphism and progressive liver disease. World J Gastroenterol. 2013 Nov 7;19(41):6969-78. doi: 10.3748/wjg.v19.i41.6969. Review. Citation on PubMed or Free article on PubMed CentralKrawczyk M, Portincasa P, Lammert F. PNPLA3-associated steatohepatitis: toward a gene-based classification of fatty liver disease. Semin Liver Dis. 2013 Nov;33(4):369-79. doi: 10.1055/s-0033-1358525. Epub 2013 Nov 12. Review. Citation on PubMedOMIM: PATATIN-LIKE PHOSPHOLIPASE DOMAIN-CONTAINING PROTEIN 3Romeo S, Kozlitina J, Xing C, Pertsemlidis A, Cox D, Pennacchio LA, Boerwinkle E, Cohen JC, Hobbs HH. Genetic variation in PNPLA3 confers susceptibility to nonalcoholic fatty liver disease. Nat Genet. 2008 Dec;40(12):1461-5. doi: 10.1038/ng.257. Citation on PubMed or Free article on PubMed CentralSeverson TJ, Besur S, Bonkovsky HL. Genetic factors that affect nonalcoholic fatty liver disease: A systematic clinical review. World J Gastroenterol. 2016 Aug 7;22(29):6742-56. doi: 10.3748/wjg.v22.i29.6742. Citation on PubMed or Free article on PubMed CentralYu J, Marsh S, Hu J, Feng W, Wu C. The Pathogenesis of Nonalcoholic Fatty Liver Disease: Interplay between Diet, Gut Microbiota, and Genetic Background. Gastroenterol Res Pract. 2016;2016:2862173. doi: 10.1155/2016/2862173. Epub 2016 May 9. Review. Citation on PubMed or Free article on PubMed Central \n",
            "RIBC2\t\n",
            "PKDREJ\t\n",
            "GTSE1\t\n",
            "CELSR1\t   From NCBI Gene:The protein encoded by this gene is a member of the flamingo subfamily, part of the cadherin superfamily. The flamingo subfamily consists of nonclassic-type cadherins; a subpopulation that does not interact with catenins. The flamingo cadherins are located at the plasma membrane and have nine cadherin domains, seven epidermal growth factor-like repeats and two laminin A G-type repeats in their ectodomain. They also have seven transmembrane domains, a characteristic unique to this subfamily. It is postulated that these proteins are receptors involved in contact-mediated communication, with cadherin domains acting as homophilic binding regions and the EGF-like domains involved in cell adhesion and receptor-ligand interactions. This particular member is a developmentally regulated, neural-specific gene which plays an unspecified role in early embryogenesis. [provided by RefSeq, Jul 2008]From UniProt:Receptor that may have an important role in cell/cell signaling during nervous system formation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Neural tube defects (NTD): Congenital malformations of the central nervous system and adjacent structures related to defective neural tube closure during the first trimester of pregnancy. Failure of neural tube closure can occur at any level of the embryonic axis. Common NTD forms include anencephaly, myelomeningocele and spina bifida, which result from the failure of fusion in the cranial and spinal region of the neural tube. NTDs have a multifactorial etiology encompassing both genetic and environmental components. [MIM:182940]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q13.31, which is the long (q) arm of chromosome 22 at position 13.31Molecular Location: base pairs 46,360,834 to 46,537,502 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADGRC1CDHF9FMI2HFMI2ME2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CELSR1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: CADHERIN EGF LAG SEVEN-PASS G-TYPE RECEPTOR 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CELSR1  OMIM: CADHERIN EGF LAG SEVEN-PASS G-TYPE RECEPTOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MLC1\t   The MLC1 gene provides instructions for making a protein that is found primarily in the brain but also in the spleen and white blood cells (leukocytes). Within the brain, the MLC1 protein is found in astroglial cells, which are a specialized form of brain cells called glial cells. Glial cells protect and maintain other nerve cells (neurons). The MLC1 protein functions at junctions that connect neighboring astroglial cells. The role of the MLC1 protein at the cell junction is unknown, but research suggests that it may control the flow of fluids into cells or the strength of cells' attachment to one another (cell adhesion). Studies indicate that the MLC1 protein may be involved in transporting molecules across the blood-brain barrier and the brain-cerebrospinal fluid barrier. These barriers protect the brain's delicate nerve tissue by allowing only certain substances to pass into the brain.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Megalencephalic leukoencephalopathy with subcortical cysts     More than 80 mutations in the MLC1 gene have been found to cause megalencephalic leukoencephalopathy with subcortical cysts type 1; this type accounts for 75 percent of all cases. This condition affects brain development and function, resulting in problems with movement and recurrent seizures. Most of the MLC1 gene mutations that cause this condition change single protein building blocks (amino acids) in the MLC1 protein. These changes alter the structure of the MLC1 protein or prevent the cell from producing any protein. It is unknown how a lack of MLC1 protein at astroglial cell junctions impairs brain development and function, causing the signs and symptoms of megalencephalic leukoencephalopathy with subcortical cysts type 1. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 80 mutations in the MLC1 gene have been found to cause megalencephalic leukoencephalopathy with subcortical cysts type 1; this type accounts for 75 percent of all cases. This condition affects brain development and function, resulting in problems with movement and recurrent seizures. Most of the MLC1 gene mutations that cause this condition change single protein building blocks (amino acids) in the MLC1 protein. These changes alter the structure of the MLC1 protein or prevent the cell from producing any protein. It is unknown how a lack of MLC1 protein at astroglial cell junctions impairs brain development and function, causing the signs and symptoms of megalencephalic leukoencephalopathy with subcortical cysts type 1. More About This Health Condition    Cytogenetic Location: 22q13.33, which is the long (q) arm of chromosome 22 at position 13.33Molecular Location: base pairs 50,059,391 to 50,085,929 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA0027LVMmegalencephalic leukoencephalopathy with subcortical cysts 1 gene productMLCMLC1_HUMANVL     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Basic Neurochemistry (sixth edition, 1999): The Blood-Brain Barrier Undergoes DevelopmentBasic Neurochemistry (sixth edition, 1999): Virtually Nothing Can Enter or Leave the Central Nervous System Parenchyma Without Passing Through an Astrocytic InterphaseMolecular Biology of the Cell (fourth edition, 2002): Cell Junctions      Clinical Information from GeneReviews (1 link)     Megalencephalic Leukoencephalopathy with Subcortical Cysts      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MLC1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     MLC1 GENE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): The Blood-Brain Barrier Undergoes DevelopmentBasic Neurochemistry (sixth edition, 1999): Virtually Nothing Can Enter or Leave the Central Nervous System Parenchyma Without Passing Through an Astrocytic InterphaseMolecular Biology of the Cell (fourth edition, 2002): Cell Junctions  Megalencephalic Leukoencephalopathy with Subcortical Cysts  Tests of MLC1  PubMed  MLC1 GENE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Boor PK, de Groot K, Waisfisz Q, Kamphorst W, Oudejans CB, Powers JM, Pronk JC, Scheper GC, van der Knaap MS. MLC1: a novel protein in distal astroglial processes. J Neuropathol Exp Neurol. 2005 May;64(5):412-9. Citation on PubMedCapdevila-Nortes X, Lpez-Hernndez T, Apaja PM, Lpez de Heredia M, Sirisi S, Callejo G, Arnedo T, Nunes V, Lukacs GL, Gasull X, Estvez R. Insights into MLC pathogenesis: GlialCAM is an MLC1 chaperone required for proper activation of volume-regulated anion currents. Hum Mol Genet. 2013 Nov 1;22(21):4405-16. doi: 10.1093/hmg/ddt290. Epub 2013 Jun 20. Citation on PubMedIlja Boor PK, de Groot K, Mejaski-Bosnjak V, Brenner C, van der Knaap MS, Scheper GC, Pronk JC. Megalencephalic leukoencephalopathy with subcortical cysts: an update and extended mutation analysis of MLC1. Hum Mutat. 2006 Jun;27(6):505-12. Citation on PubMedLpez-Hernndez T, Sirisi S, Capdevila-Nortes X, Montolio M, Fernndez-Dueas V, Scheper GC, van der Knaap MS, Casquero P, Ciruela F, Ferrer I, Nunes V, Estvez R. Molecular mechanisms of MLC1 and GLIALCAM mutations in megalencephalic leukoencephalopathy with subcortical cysts. Hum Mol Genet. 2011 Aug 15;20(16):3266-77. doi: 10.1093/hmg/ddr238. Epub 2011 May 30. Citation on PubMedOMIM: MLC1 GENEMontagna G, Teijido O, Eymard-Pierre E, Muraki K, Cohen B, Loizzo A, Grosso P, Tedeschi G, Palacn M, Boespflug-Tanguy O, Bertini E, Santorelli FM, Estvez R. Vacuolating megalencephalic leukoencephalopathy with subcortical cysts: functional studies of novel variants in MLC1. Hum Mutat. 2006 Mar;27(3):292. Citation on PubMed \n",
            "MOV10L1\t\n",
            "PLXNB2\t\n",
            "SBF1\t   From NCBI Gene:This gene encodes a member of the protein-tyrosine phosphatase family. However, the encoded protein does not appear to be a catalytically active phosphatase because it lacks several amino acids in the catalytic pocket. This protein contains a Guanine nucleotide exchange factor (GEF) domain which is necessary for its role in growth and differentiation. Mutations in this gene have been associated with Charcot-Marie-Tooth disease 4B3. Pseudogenes of this gene have been defined on chromosomes 1 and 8. [provided by RefSeq, Dec 2014]From UniProt:Acts as an adapter for the phosphatase MTMR2 to regulate MTMR2 catalytic activity and subcellular location (PubMed:12668758). May function as a guanine nucleotide exchange factor (GEF) activating RAB28 (PubMed:20937701). Promotes the exchange of GDP to GTP, converting inactive GDP-bound Rab proteins into their active GTP-bound form (PubMed:20937701). Inhibits myoblast differentiation in vitro and induces oncogenic transformation in fibroblasts (PubMed:9537414).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Charcot-Marie-Tooth diseaseFrom NCBI Gene:Charcot-Marie-Tooth disease, type 4B3From UniProt:Charcot-Marie-Tooth disease 4B3 (CMT4B3): A recessive demyelinating form of Charcot-Marie-Tooth disease, a disorder of the peripheral nervous system, characterized by progressive weakness and atrophy, initially of the peroneal muscles and later of the distal muscles of the arms. Charcot-Marie-Tooth disease is classified in two main groups on the basis of electrophysiologic properties and histopathology: primary peripheral demyelinating neuropathies (designated CMT1 when they are dominantly inherited) and primary peripheral axonal neuropathies (CMT2). Demyelinating neuropathies are characterized by severely reduced nerve conduction velocities (less than 38 m/sec), segmental demyelination and remyelination with onion bulb formations on nerve biopsy, slowly progressive distal muscle atrophy and weakness, absent deep tendon reflexes, and hollow feet. By convention autosomal recessive forms of demyelinating Charcot-Marie-Tooth disease are designated CMT4. [MIM:615284]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q13.33, which is the long (q) arm of chromosome 22 at position 13.33Molecular Location: base pairs 50,445,000 to 50,475,071 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMT4B3DENND7AMTMR5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SBF1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CHARCOT-MARIE-TOOTH DISEASE, TYPE 4B3OMIM: SET-BINDING FACTOR 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SBF1  OMIM: CHARCOT-MARIE-TOOTH DISEASE, TYPE 4B3OMIM: SET-BINDING FACTOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CPT1B\t   From NCBI Gene:The protein encoded by this gene, a member of the carnitine/choline acetyltransferase family, is the rate-controlling enzyme of the long-chain fatty acid beta-oxidation pathway in muscle mitochondria. This enzyme is required for the net transport of long-chain fatty acyl-CoAs from the cytoplasm into the mitochondria. Multiple transcript variants encoding different isoforms have been found for this gene, and read-through transcripts are expressed from the upstream locus that include exons from this gene. [provided by RefSeq, Jun 2009]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Narcolepsy     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 22q13.33, which is the long (q) arm of chromosome 22 at position 13.33Molecular Location: base pairs 50,568,861 to 50,578,667 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CPT1-MCPT1MCPTICPTI-MM-CPT1MCCPT1MCPT1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CPT1B      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: CARNITINE PALMITOYLTRANSFERASE I, MUSCLE      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CPT1B  OMIM: CARNITINE PALMITOYLTRANSFERASE I, MUSCLE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MAPK8IP2\t\n",
            "ARSA\t   The ARSA gene provides instructions for making the enzyme arylsulfatase A. This enzyme is located in cellular structures called lysosomes, which are the cell's recycling centers. Within lysosomes, arylsulfatase A helps process substances known as sulfatides. Sulfatides are a subgroup of sphingolipids, a category of fats that are important components of cell membranes. Sulfatides are abundant in the nervous system's white matter, consisting of nerve fibers covered by myelin. Myelin, made up of multiple layers of membranes, insulates and protects nerves.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Metachromatic leukodystrophy     More than 110 mutations that cause metachromatic leukodystrophy, a disorder that causes deterioration of nervous system functions, have been identified in the ARSA gene. These mutations greatly reduce the activity of arylsulfatase A. Severe disruption in arylsulfatase A activity interferes with the breakdown of sulfatides. As a result, these substances can accumulate to toxic levels in the nervous system. The buildup of sulfatides gradually destroys the cells that produce myelin, the covering that protects nerves and promotes the efficient transmission of nerve impulses. Destruction of myelin leads to the loss of white matter (leukodystrophy) and impairment of nervous system function, resulting in the signs and symptoms of metachromatic leukodystrophy.In some cases, individuals with very low arylsulfatase A activity show no signs or symptoms of metachromatic leukodystrophy. This condition, called pseudoarylsulfatase deficiency, seems to be caused by specific variations of the ARSA gene. These variations are present in as many as 5 to 10 percent of Europeans and North Americans. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 110 mutations that cause metachromatic leukodystrophy, a disorder that causes deterioration of nervous system functions, have been identified in the ARSA gene. These mutations greatly reduce the activity of arylsulfatase A. Severe disruption in arylsulfatase A activity interferes with the breakdown of sulfatides. As a result, these substances can accumulate to toxic levels in the nervous system. The buildup of sulfatides gradually destroys the cells that produce myelin, the covering that protects nerves and promotes the efficient transmission of nerve impulses. Destruction of myelin leads to the loss of white matter (leukodystrophy) and impairment of nervous system function, resulting in the signs and symptoms of metachromatic leukodystrophy.In some cases, individuals with very low arylsulfatase A activity show no signs or symptoms of metachromatic leukodystrophy. This condition, called pseudoarylsulfatase deficiency, seems to be caused by specific variations of the ARSA gene. These variations are present in as many as 5 to 10 percent of Europeans and North Americans. More About This Health Condition    Cytogenetic Location: 22q13.33, which is the long (q) arm of chromosome 22 at position 13.33Molecular Location: base pairs 50,622,754 to 50,628,173 on chromosome 22 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARSA_HUMANcerebroside 3-sulfataseCerebroside-3-sulfate 3-sulfohydrolaseCerebroside-SulfataseMLDsulfatidase     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Basic Neurochemistry (sixth edition, 1999): Lysosomal DiseaseEssentials of Glycobiology (first edition, 1999): Glycosphingolipid Degradation      Clinical Information from GeneReviews (1 link)     Arylsulfatase A Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ARSA      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ARYLSULFATASE A      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Lysosomal DiseaseEssentials of Glycobiology (first edition, 1999): Glycosphingolipid Degradation  Arylsulfatase A Deficiency  Tests of ARSA  PubMed  ARYLSULFATASE A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ARYLSULFATASE ABasic Neurochemistry (sixth edition, 1999): Lysosomal DiseaseCesani M, Capotondo A, Plati T, Sergi LS, Fumagalli F, Roncarolo MG, Naldini L, Comi G, Sessa M, Biffi A. Characterization of new arylsulfatase A gene mutations reinforces genotype-phenotype correlation in metachromatic leukodystrophy. Hum Mutat. 2009 Oct;30(10):E936-45. doi: 10.1002/humu.21093. Citation on PubMedEssentials of Glycobiology (first edition, 1999): Glycosphingolipid DegradationGieselmann V, Krgeloh-Mann I. Metachromatic leukodystrophy--an update. Neuropediatrics. 2010 Feb;41(1):1-6. doi: 10.1055/s-0030-1253412. Epub 2010 Jun 22. Review. Citation on PubMedGrossi S, Regis S, Rosano C, Corsolini F, Uziel G, Sessa M, Di Rocco M, Parenti G, Deodato F, Leuzzi V, Biancheri R, Filocamo M. Molecular analysis of ARSA and PSAP genes in twenty-one Italian patients with metachromatic leukodystrophy: identification and functional characterization of 11 novel ARSA alleles. Hum Mutat. 2008 Nov;29(11):E220-30. doi: 10.1002/humu.20851. Citation on PubMedLugowska A, Poski R, Wodarski P, Tylki-Szymaska A. Molecular bases of metachromatic leukodystrophy in Polish patients. J Hum Genet. 2010 Jun;55(6):394-6. doi: 10.1038/jhg.2010.25. Epub 2010 Mar 26. Citation on PubMedPoeppel P, Habetha M, Marco A, Bssow H, Berna L, Gieselmann V. Missense mutations as a cause of metachromatic leukodystrophy. Degradation of arylsulfatase A in the endoplasmic reticulum. FEBS J. 2005 Mar;272(5):1179-88. Citation on PubMedRauschka H, Colsch B, Baumann N, Wevers R, Schmidbauer M, Krammer M, Turpin JC, Lefevre M, Olivier C, Tardieu S, Krivit W, Moser H, Moser A, Gieselmann V, Zalc B, Cox T, Reuner U, Tylki-Szymanska A, Aboul-Enein F, LeGuern E, Bernheimer H, Berger J. Late-onset metachromatic leukodystrophy: genotype strongly influences phenotype. Neurology. 2006 Sep 12;67(5):859-63. Citation on PubMedShukla P, Vasisht S, Srivastava R, Gupta N, Ghosh M, Kumar M, Sharma R, Gupta AK, Kaur P, Kamate M, Gulati S, Kalra V, Phadke S, Singhi P, Dherai AJ, Kabra M. Molecular and structural analysis of metachromatic leukodystrophy patients in Indian population. J Neurol Sci. 2011 Feb 15;301(1-2):38-45. doi: 10.1016/j.jns.2010.11.007. Epub 2010 Dec 16. Citation on PubMed \n",
            "AFF3\t   From NCBI Gene:This gene encodes a tissue-restricted nuclear transcriptional activator that is preferentially expressed in lymphoid tissue. Isolation of this protein initially defined a highly conserved LAF4/MLLT2 gene family of nuclear transcription factors that may function in lymphoid development and oncogenesis. In some ALL patients, this gene has been found fused to the gene for MLL. Multiple alternatively spliced transcript variants that encode different proteins have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Putative transcription activator that may function in lymphoid development and oncogenesis. Binds, in vitro, to double-stranded DNA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Rheumatoid arthritis     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q11.2, which is the long (q) arm of chromosome 2 at position 11.2Molecular Location: base pairs 99,545,861 to 100,142,583 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LAF4MLLT2-like     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AFF3      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: AF4/FMR2 FAMILY, MEMBER 3      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AFF3  OMIM: AF4/FMR2 FAMILY, MEMBER 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MRPS9\t\n",
            "GPR45\t\n",
            "TGFBRAP1\t\n",
            "C2orf49\t\n",
            "RGPD3\t\n",
            "ST6GAL2\t\n",
            "RGPD4\t\n",
            "SULT1C3\t\n",
            "GCC2\t\n",
            "LIMS1\t\n",
            "SH3RF3\t\n",
            "RGPD8\t\n",
            "CKAP2L\t   From NCBI Gene:The protein encoded by this gene is thought to be a mitotic spindle protein important to neural stem or progenitor cells. Mutations in this gene have been associated with spindle organization defects, including mitotic spindle defects, lagging chromosomes, and chromatin bridges. There is evidence that mutations in this gene are associated with Filippi syndrome, characterized by growth defects, microcephaly, intellectual disability, facial feature defects, and syndactyly. There is a pseudogene of this gene on chromosome 20. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2015]From UniProt:Microtubule-associated protein required for mitotic spindle formation and cell-cycle progression in neural progenitor cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Filippi syndromeFrom UniProt:Filippi syndrome (FLPIS): A rare disorder characterized by microcephaly, pre- and postnatal growth failure, syndactyly, and distinctive facial features, including a broad nasal bridge and underdeveloped alae nasi. Some affected individuals have intellectual disability, seizures, undescended testicles in males, and teeth and hair abnormalities. [MIM:272440]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q14.1, which is the long (q) arm of chromosome 2 at position 14.1Molecular Location: base pairs 112,736,349 to 112,764,677 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CKAP2L      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CYTOSKELETON-ASSOCIATED PROTEIN 2-LIKEOMIM: FILIPPI SYNDROME      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CKAP2L  OMIM: CYTOSKELETON-ASSOCIATED PROTEIN 2-LIKEOMIM: FILIPPI SYNDROME  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IL1A\t   The IL1A gene provides instructions for making a protein called interleukin-1 alpha. Interleukins are a group of proteins that are made primarily in immune system cells.  They are involved in cell-to-cell communication and have a wide variety of functions within the immune system. Interleukin-1 alpha is described as \"pro-inflammatory\" because it stimulates the activity of genes involved in inflammation and immunity. This protein plays a critical role in protecting the body from foreign invaders such as bacteria and viruses.  It is also involved in bone resorption, the breakdown and removal of bone tissue that is no longer needed.Interleukin-1 alpha is initially produced as a relatively long protein that is trapped within cells. Another protein, called calpain, cuts (cleaves) this precursor protein to create a shorter, mature version of interleukin-1 alpha. The shorter form of this protein is secreted by immune system cells to influence the functions of other cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ankylosing spondylitis     Several variations (polymorphisms) in the IL1A gene have been found to influence the risk of ankylosing spondylitis.  Each of these variations changes a single protein building block (amino acid) in interleukin-1 alpha. It is unclear how these variations alter the protein's function. Studies suggest that the effects of IL1A variations are probably related to the role of interleukin-1 alpha in promoting inflammation. Other genetic and environmental factors, many of which are unknown, also affect the chance of developing ankylosing spondylitis. More About This Health Condition      Idiopathic inflammatory myopathy     Genetics Home Reference provides information about idiopathic inflammatory myopathy. More About This Health Condition      Intervertebral disc disease     Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition      Keratoconus     Genetics Home Reference provides information about keratoconus. More About This Health Condition      Other disorders     Variations in the IL1A gene have been studied as potential risk factors for several other disorders associated with abnormal inflammation. These include chronic gum (periodontal) disease, a progressive bone infection known as chronic osteomyelitis, and an eye disease called open-angle glaucoma.The most well-studied variation affecting the IL1A gene is a change in a single DNA building block (nucleotide) in a region of regulatory DNA near the start of the gene (written as IL1A-889 C>T). This variation affects the production of interleukin-1 alpha within cells. Researchers have also identified a second common variation in the IL1A gene, written as IL1A+4845 G>T, which changes a single nucleotide in the gene. This variation likely affects the sensitivity of interleukin-1 alpha to cleavage by calpain.It is unclear how changes in the IL1A gene influence the risk of inflammatory disorders. Studies suggest that the effects of IL1A variations are probably related to the role of interleukin-1 alpha in promoting inflammation. Other genetic and environmental factors also likely affect the chance of developing these complex disorders.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several variations (polymorphisms) in the IL1A gene have been found to influence the risk of ankylosing spondylitis.  Each of these variations changes a single protein building block (amino acid) in interleukin-1 alpha. It is unclear how these variations alter the protein's function. Studies suggest that the effects of IL1A variations are probably related to the role of interleukin-1 alpha in promoting inflammation. Other genetic and environmental factors, many of which are unknown, also affect the chance of developing ankylosing spondylitis. More About This Health Condition  Genetics Home Reference provides information about idiopathic inflammatory myopathy. More About This Health Condition  Genetics Home Reference provides information about intervertebral disc disease. More About This Health Condition  Genetics Home Reference provides information about keratoconus. More About This Health Condition  Variations in the IL1A gene have been studied as potential risk factors for several other disorders associated with abnormal inflammation. These include chronic gum (periodontal) disease, a progressive bone infection known as chronic osteomyelitis, and an eye disease called open-angle glaucoma.The most well-studied variation affecting the IL1A gene is a change in a single DNA building block (nucleotide) in a region of regulatory DNA near the start of the gene (written as IL1A-889 C>T). This variation affects the production of interleukin-1 alpha within cells. Researchers have also identified a second common variation in the IL1A gene, written as IL1A+4845 G>T, which changes a single nucleotide in the gene. This variation likely affects the sensitivity of interleukin-1 alpha to cleavage by calpain.It is unclear how changes in the IL1A gene influence the risk of inflammatory disorders. Studies suggest that the effects of IL1A variations are probably related to the role of interleukin-1 alpha in promoting inflammation. Other genetic and environmental factors also likely affect the chance of developing these complex disorders.    Cytogenetic Location: 2q14.1, which is the long (q) arm of chromosome 2 at position 14.1Molecular Location: base pairs 112,773,915 to 112,785,398 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        hematopoietin-1IL-1 alphaIL-1AIL1IL1-ALPHAIL1A_HUMANIL1F1Interleukin-1 alphainterleukin 1, alphapreinterleukin 1 alphapro-interleukin-1-alpha     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     National Eye Institute: GlaucomaNational Institute of Arthritis and Musculoskeletal and Skin Diseases: Rheumatoid Arthritis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of IL1A      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     INTERLEUKIN 1-ALPHA      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      National Eye Institute: GlaucomaNational Institute of Arthritis and Musculoskeletal and Skin Diseases: Rheumatoid Arthritis  Tests of IL1A  PubMed  INTERLEUKIN 1-ALPHA  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Asensi V, Alvarez V, Valle E, Meana A, Fierer J, Coto E, Carton JA, Maradona JA, Paz J, Dieguez MA, de la Fuente B, Moreno A, Rubio S, Tuya MJ, Sarasa J, Llames S, Arribas JM. IL-1 alpha (-889) promoter polymorphism is a risk factor for osteomyelitis. Am J Med Genet A. 2003 Jun 1;119A(2):132-6. Citation on PubMedBrionez TF, Reveille JD. The contribution of genes outside the major histocompatibility complex to susceptibility to ankylosing spondylitis. Curr Opin Rheumatol. 2008 Jul;20(4):384-91. doi: 10.1097/BOR.0b013e32830460fe. Review. Citation on PubMedDinarello CA. Biologic basis for interleukin-1 in disease. Blood. 1996 Mar 15;87(6):2095-147. Review. Citation on PubMedGraves DT, Cochran D. The contribution of interleukin-1 and tumor necrosis factor to periodontal tissue destruction. J Periodontol. 2003 Mar;74(3):391-401. Review. Citation on PubMedKawaguchi Y, Tochimoto A, Hara M, Kawamoto M, Sugiura T, Saito S, Kamatani N. Contribution of single nucleotide polymorphisms of the IL1A gene to the cleavage of precursor IL-1alpha and its transcription activity. Immunogenetics. 2007 Jun;59(6):441-8. Epub 2007 Apr 18. Citation on PubMedLee S, Temple S, Roberts S, Price P. Complex effects of IL1A polymorphism and calpain inhibitors on interleukin 1 alpha (IL-1 alpha) mRNA levels and secretion of IL-1 alpha protein. Tissue Antigens. 2008 Jul;72(1):67-71. doi: 10.1111/j.1399-0039.2008.01052.x. Epub 2008 May 20. Citation on PubMedSims AM, Timms AE, Bruges-Armas J, Burgos-Vargas R, Chou CT, Doan T, Dowling A, Fialho RN, Gergely P, Gladman DD, Inman R, Kauppi M, Kaarela K, Laiho K, Maksymowych W, Pointon JJ, Rahman P, Reveille JD, Sorrentino R, Tuomilehto J, Vargas-Alarcon G, Wordsworth BP, Xu H, Brown MA; International Genetics of Ankylosing Spondylitis. Prospective meta-analysis of interleukin 1 gene complex polymorphisms confirms associations with ankylosing spondylitis. Ann Rheum Dis. 2008 Sep;67(9):1305-9. Epub 2007 Dec 6. Citation on PubMedTsezou A, Poultsides L, Kostopoulou F, Zintzaras E, Satra M, Kitsiou-Tzeli S, Malizos KN. Influence of interleukin 1alpha (IL-1alpha), IL-4, and IL-6 polymorphisms on genetic susceptibility to chronic osteomyelitis. Clin Vaccine Immunol. 2008 Dec;15(12):1888-90. doi: 10.1128/CVI.00209-08. Epub 2008 Oct 29. Citation on PubMed or Free article on PubMed Central \n",
            "IL37\t\n",
            "DPP10\t\n",
            "SNTG2\t\n",
            "LPIN1\t   From NCBI Gene:This gene encodes a magnesium-ion-dependent phosphatidic acid phosphohydrolase enzyme that catalyzes the penultimate step in triglyceride synthesis including the dephosphorylation of phosphatidic acid to yield diacylglycerol. Expression of this gene is required for adipocyte differentiation and it also functions as a nuclear transcriptional coactivator with some peroxisome proliferator-activated receptors to modulate expression of other genes involved in lipid metabolism. Mutations in this gene are associated with metabolic syndrome, type 2 diabetes, acute recurrent rhabdomyolysis, and autosomal recessive acute recurrent myoglobinuria (ARARM). This gene is also a candidate for several human lipodystrophy syndromes. [provided by RefSeq, Mar 2017]From UniProt:Plays important roles in controlling the metabolism of fatty acids at different levels. Acts as a magnesium-dependent phosphatidate phosphatase enzyme which catalyzes the conversion of phosphatidic acid to diacylglycerol during triglyceride, phosphatidylcholine and phosphatidylethanolamine biosynthesis in the reticulum endoplasmic membrane. Acts also as a nuclear transcriptional coactivator for PPARGC1A/PPARA to modulate lipid metabolism gene expression (By similarity). Is involved in adipocyte differentiation. May also be involved in mitochondrial fission by converting phosphatidic acid to diacylglycerol (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Myoglobinuria, acute recurrent, autosomal recessiveFrom UniProt:Myoglobinuria, acute recurrent, autosomal recessive (ARARM): Recurrent myoglobinuria is characterized by recurrent attacks of rhabdomyolysis (necrosis or disintegration of skeletal muscle) associated with muscle pain and weakness and followed by excretion of myoglobin in the urine. Renal failure may occasionally occur. [MIM:268200]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p25.1, which is the short (p) arm of chromosome 2 at position 25.1Molecular Location: base pairs 11,677,544 to 11,827,409 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PAP1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LPIN1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LIPIN 1OMIM: MYOGLOBINURIA, ACUTE RECURRENT, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LPIN1  OMIM: LIPIN 1OMIM: MYOGLOBINURIA, ACUTE RECURRENT, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CCDC93\t\n",
            "TMEM37\t\n",
            "GYPC\t   From NCBI Gene:Glycophorin C (GYPC) is an integral membrane glycoprotein. It is a minor species carried by human erythrocytes, but plays an important role in regulating the mechanical stability of red cells. A number of glycophorin C mutations have been described. The Gerbich and Yus phenotypes are due to deletion of exon 3 and 2, respectively. The Webb and Duch antigens, also known as glycophorin D, result from single point mutations of the glycophorin C gene. The glycophorin C protein has very little homology with glycophorins A and B. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]From UniProt:This protein is a minor sialoglycoprotein in human erythrocyte membranes. The blood group Gerbich antigens and receptors for Plasmodium falciparum merozoites are most likely located within the extracellular domain. Glycophorin-C plays an important role in regulating the stability of red cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Blood group, Gerbich systemSusceptibility to malaria     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q14.3, which is the long (q) arm of chromosome 2 at position 14.3Molecular Location: base pairs 126,655,935 to 126,696,675 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD236CD236RGEGE:GPC:GPD:GYPDGPCGPDGYPDPAS-2PAS-2'     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GYPC      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BLOOD GROUP, GERBICH SYSTEMOMIM: GLYCOPHORIN COMIM: MALARIA, SUSCEPTIBILITY TO      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GYPC  OMIM: BLOOD GROUP, GERBICH SYSTEMOMIM: GLYCOPHORIN COMIM: MALARIA, SUSCEPTIBILITY TO  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYO7B\t\n",
            "POTEF\t\n",
            "POTEI\t\n",
            "POTEJ\t\n",
            "POTEE\t\n",
            "ZNF806\t\n",
            "NCKAP5\t\n",
            "MAP3K19\t\n",
            "R3HDM1\t\n",
            "TPO\t   The TPO gene provides instructions for making an enzyme called thyroid peroxidase. This enzyme plays a central role in the function of the thyroid gland, a butterfly-shaped tissue in the lower neck. Thyroid peroxidase assists the chemical reaction that adds iodine to a protein called thyroglobulin, a critical step in generating thyroid hormones. Thyroid hormones play an important role in regulating growth, brain development, and the rate of chemical reactions in the body (metabolism).To function properly, thyroid peroxidase must be located in the cell membrane of certain thyroid cells, called follicular cells. Thyroid peroxidase has several different versions (isoforms), which vary by size and location within the cell. Some versions do not function because they are not located in the cell membrane.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Congenital hypothyroidism     TPO gene mutations can cause congenital hypothyroidism, a condition characterized by abnormally low levels of thyroid hormones starting from birth. The TPO gene mutations involved in this condition delete, add, or change DNA building blocks (base pairs) in the TPO gene. Some mutations lead to an abnormally small thyroid peroxidase enzyme that breaks apart before it can be inserted into the cell membrane. Other mutations change the enzyme's 3-dimensional shape, preventing it from functioning properly within the cell membrane. Without functional thyroid peroxidase, iodine taken up by the thyroid gland is not added to thyroglobulin. As a result, the production of thyroid hormones is absent or reduced, leading to the features of congenital hypothyroidism. In most affected individuals, the thyroid gland is enlarged (goiter) in an attempt to compensate for reduced hormone production. Because cases caused by TPO gene mutations result from a disruption of thyroid hormone synthesis, they are classified as thyroid dyshormonogenesis. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      TPO gene mutations can cause congenital hypothyroidism, a condition characterized by abnormally low levels of thyroid hormones starting from birth. The TPO gene mutations involved in this condition delete, add, or change DNA building blocks (base pairs) in the TPO gene. Some mutations lead to an abnormally small thyroid peroxidase enzyme that breaks apart before it can be inserted into the cell membrane. Other mutations change the enzyme's 3-dimensional shape, preventing it from functioning properly within the cell membrane. Without functional thyroid peroxidase, iodine taken up by the thyroid gland is not added to thyroglobulin. As a result, the production of thyroid hormones is absent or reduced, leading to the features of congenital hypothyroidism. In most affected individuals, the thyroid gland is enlarged (goiter) in an attempt to compensate for reduced hormone production. Because cases caused by TPO gene mutations result from a disruption of thyroid hormone synthesis, they are classified as thyroid dyshormonogenesis. More About This Health Condition    Cytogenetic Location: 2p25.3, which is the short (p) arm of chromosome 2 at position 25.3Molecular Location: base pairs 1,413,288 to 1,542,727 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MSAPERT_HUMANthyroid microsomal antigenthyroperoxidaseTPX     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TPO      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     THYROID PEROXIDASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TPO  PubMed  THYROID PEROXIDASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bakker B, Bikker H, Vulsma T, de Randamie JS, Wiedijk BM, De Vijlder JJ. Two decades of screening for congenital hypothyroidism in The Netherlands: TPO gene mutations in total iodide organification defects (an update). J Clin Endocrinol Metab. 2000 Oct;85(10):3708-12. Citation on PubMedCangul H, Aycan Z, Olivera-Nappa A, Saglam H, Schoenmakers NA, Boelaert K, Cetinkaya S, Tarim O, Bober E, Darendeliler F, Bas V, Demir K, Aydin BK, Kendall M, Cole T, Hgler W, Chatterjee VK, Barrett TG, Maher ER. Thyroid dyshormonogenesis is mainly caused by TPO mutations in consanguineous community. Clin Endocrinol (Oxf). 2013 Aug;79(2):275-81. doi: 10.1111/cen.12127. Epub 2013 May 6. Citation on PubMedChards T, Chapal N, Bresson D, Bs C, Giudicelli V, Lefranc MP, Praldi-Roux S. The human anti-thyroid peroxidase autoantibody repertoire in Graves' and Hashimoto's autoimmune thyroid diseases. Immunogenetics. 2002 Jun;54(3):141-57. Epub 2002 May 3. Review. Citation on PubMedKotani T, Umeki K, Kawano J, Suganuma T, Hishinuma A, Ieiri T, Harada S. Partial iodide organification defect caused by a novel mutation of the thyroid peroxidase gene in three siblings. Clin Endocrinol (Oxf). 2003 Aug;59(2):198-206. Citation on PubMedLee CC, Harun F, Jalaludin MY, Lim CY, Ng KL, Mat Junit S. Functional analyses of C.2268dup in thyroid peroxidase gene associated with goitrous congenital hypothyroidism. Biomed Res Int. 2014;2014:370538. doi: 10.1155/2014/370538. Epub 2014 Mar 17. Citation on PubMed or Free article on PubMed CentralNascimento AC, Guedes DR, Santos CS, Knobel M, Rubio IG, Medeiros-Neto G. Thyroperoxidase gene mutations in congenital goitrous hypothyroidism with total and partial iodide organification defect. Thyroid. 2003 Dec;13(12):1145-51. Citation on PubMedRivolta CM, Esperante SA, Grueiro-Papendieck L, Chiesa A, Moya CM, Domen S, Varela V, Targovnik HM. Five novel inactivating mutations in the thyroid peroxidase gene responsible for congenital goiter and iodide organification defect. Hum Mutat. 2003 Sep;22(3):259. Citation on PubMedOMIM: THYROID PEROXIDASE \n",
            "TNFAIP6\t\n",
            "NEB\t   The NEB gene provides instructions for making a protein called nebulin.  This protein plays an important role in skeletal muscles, which are muscles used for movement. Within skeletal muscle cells, nebulin is found in structures called sarcomeres.  Sarcomeres are necessary for muscles to tense (contract). Nebulin is one of several proteins that interact to generate the mechanical force needed for muscle contraction.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Nemaline myopathy     More than 60 mutations in the NEB gene have been found to cause nemaline myopathy. Some of these mutations change single protein building blocks (amino acids) in the nebulin protein or result in the production of an abnormally short protein. Other mutations insert or delete genetic material in the NEB gene. Most NEB gene mutations alter the structure or reduce the production of nebulin. A lack of functional nebulin impairs the muscle cells' ability to contract. Inefficient muscle contraction leads to muscle weakness and the other features of nemaline myopathy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 60 mutations in the NEB gene have been found to cause nemaline myopathy. Some of these mutations change single protein building blocks (amino acids) in the nebulin protein or result in the production of an abnormally short protein. Other mutations insert or delete genetic material in the NEB gene. Most NEB gene mutations alter the structure or reduce the production of nebulin. A lack of functional nebulin impairs the muscle cells' ability to contract. Inefficient muscle contraction leads to muscle weakness and the other features of nemaline myopathy. More About This Health Condition    Cytogenetic Location: 2q23.3, which is the long (q) arm of chromosome 2 at position 23.3Molecular Location: base pairs 151,485,334 to 151,734,487 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        NEBU_HUMANNEM2     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): Muscle ContractionMolecular Cell Biology (fourth edition, 2000): Titin and Nebulin Filaments Organize the Sarcomere      Clinical Information from GeneReviews (1 link)     Nemaline Myopathy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of NEB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     NEBULIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Muscle ContractionMolecular Cell Biology (fourth edition, 2000): Titin and Nebulin Filaments Organize the Sarcomere  Nemaline Myopathy  Tests of NEB  PubMed  NEBULIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Lehtokari VL, Pelin K, Sandbacka M, Ranta S, Donner K, Muntoni F, Sewry C, Angelini C, Bushby K, Van den Bergh P, Iannaccone S, Laing NG, Wallgren-Pettersson C. Identification of 45 novel mutations in the nebulin gene associated with autosomal recessive nemaline myopathy. Hum Mutat. 2006 Sep;27(9):946-56. Citation on PubMedOttenheijm CA, Hooijman P, DeChene ET, Stienen GJ, Beggs AH, Granzier H. Altered myofilament function depresses force generation in patients with nebulin-based nemaline myopathy (NEM2). J Struct Biol. 2010 May;170(2):334-43. doi: 10.1016/j.jsb.2009.11.013. Epub 2009 Nov 26. Citation on PubMed or Free article on PubMed CentralRyan MM, Schnell C, Strickland CD, Shield LK, Morgan G, Iannaccone ST, Laing NG, Beggs AH, North KN. Nemaline myopathy: a clinical study of 143 cases. Ann Neurol. 2001 Sep;50(3):312-20. Review. Citation on PubMed \n",
            "FMNL2\t\n",
            "DAPL1\t\n",
            "IFIH1\t   The IFIH1 gene provides instructions for making the MDA5 protein, which plays an important role in innate immunity, the body's early, nonspecific response to foreign invaders (pathogens) such as viruses and bacteria. In particular, the MDA5 protein recognizes a molecule called double-stranded RNA (a chemical cousin of DNA), which certain viruses, including rhinovirus (the virus that causes the common cold), respiratory syncytial virus (RSV), and the influenza (flu) virus, have as their genetic material or produce when they infect cells and copy (replicate) themselves. (Another subset of viruses has DNA as the genetic material.) When pieces of viral RNA are present inside a cell, multiple MDA5 proteins attach to it, one after another, forming a filament. Filament formation stimulates signals that turn on the production of immune system proteins called interferons. Interferons control the activity of genes that help block the viruses from replicating themselves and stimulate the activity of certain immune system cells to fight infection. Interferons also help regulate inflammation, which is another part of the body's innate immune response.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Aicardi-Goutires syndrome     At least 12 IFIH1 gene mutations have been found in people with Aicardi-Goutires syndrome. This disorder is characterized by abnormalities of the immune system, skin, and brain, including calcium deposits in parts of the brain.The IFIH1 gene mutations involved in Aicardi-Goutires syndrome are described as \"gain-of-function\" because they lead to production of an MDA5 protein with enhanced activity. The altered protein may more readily attach to RNA, even pieces of RNA that are not from viruses, or to other MDA5 proteins to form filaments. Alternatively, filaments containing the altered protein may not be broken down when immune signaling is no longer needed. As a result of these changes, interferon production is abnormally turned on, leading to excessive immune system activity and inflammation.Constant inflammation is thought to disrupt the way calcium is handled in the body, leading to calcium deposits in people with Aicardi-Goutires syndrome. Excessive inflammation is also thought to damage cells in the brain and skin, leading to the abnormalities in these tissues characteristic of this disorder. More About This Health Condition      MDA5 deficiency     At least four genetic changes in the IFIH1 gene have been found to cause MDA5 deficiency, an immune system disorder (immunodeficiency) that leads to recurrent, severe viral infections in the lungs and airways (respiratory tract). Infections are most commonly caused by rhinovirus, RSV, and the flu virus. The changes in the IFIH1 gene that cause this condition are described as \"loss-of-function\" mutations because they lead to an altered version of the MDA5 protein that cannot function. Studies suggest that the altered protein is unable to attach to viral RNA or to other MDA5 proteins to form filaments. As a result, interferon production is not turned on. A lack of the important early immune response stimulated by interferons leads to severe viral infections in infants with MDA5 deficiency. More About This Health Condition      Other disorders     IFIH1 gene mutations have also been found to cause Singleton-Merten syndrome. A feature of Singleton-Merten syndrome is calcium deposits in the large vessel that carries blood from the heart to the rest of the body (the aorta) and in certain valves in the heart. Other signs and symptoms include tooth abnormalities, low bone density (osteopenia), and other bone problems. Some people with IFIH1 gene mutations have signs and symptoms of both Singleton-Merten syndrome and Aicardi-Goutires syndrome (described above), suggesting that these two conditions may be part of a spectrum caused by IFIH1 gene mutations.As in Aicardi-Goutires syndrome, the IFIH1 gene mutations involved in Singleton-Merten syndrome are described as \"gain-of-function\" and lead to excessive immune system activity and inflammation, disrupting calcium handling in the body. It is unclear why people with gain-of-function mutations in the IFIH1 gene develop signs and symptoms of one condition or the other.Singleton-Merten syndrome and Aicardi-Goutires syndrome both have autoimmune features, which occur when the immune system malfunctions and damages the body's own tissues and organs. Common variations (polymorphisms) in the IFIH1 gene have been associated with other autoimmune disorders. It is thought that polymorphisms that enhance the activity of the MDA5 protein increase the risk of certain autoimmune disorders, while polymorphisms that reduce the activity of the MDA5 protein help protect against others.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 12 IFIH1 gene mutations have been found in people with Aicardi-Goutires syndrome. This disorder is characterized by abnormalities of the immune system, skin, and brain, including calcium deposits in parts of the brain.The IFIH1 gene mutations involved in Aicardi-Goutires syndrome are described as \"gain-of-function\" because they lead to production of an MDA5 protein with enhanced activity. The altered protein may more readily attach to RNA, even pieces of RNA that are not from viruses, or to other MDA5 proteins to form filaments. Alternatively, filaments containing the altered protein may not be broken down when immune signaling is no longer needed. As a result of these changes, interferon production is abnormally turned on, leading to excessive immune system activity and inflammation.Constant inflammation is thought to disrupt the way calcium is handled in the body, leading to calcium deposits in people with Aicardi-Goutires syndrome. Excessive inflammation is also thought to damage cells in the brain and skin, leading to the abnormalities in these tissues characteristic of this disorder. More About This Health Condition  At least four genetic changes in the IFIH1 gene have been found to cause MDA5 deficiency, an immune system disorder (immunodeficiency) that leads to recurrent, severe viral infections in the lungs and airways (respiratory tract). Infections are most commonly caused by rhinovirus, RSV, and the flu virus. The changes in the IFIH1 gene that cause this condition are described as \"loss-of-function\" mutations because they lead to an altered version of the MDA5 protein that cannot function. Studies suggest that the altered protein is unable to attach to viral RNA or to other MDA5 proteins to form filaments. As a result, interferon production is not turned on. A lack of the important early immune response stimulated by interferons leads to severe viral infections in infants with MDA5 deficiency. More About This Health Condition  IFIH1 gene mutations have also been found to cause Singleton-Merten syndrome. A feature of Singleton-Merten syndrome is calcium deposits in the large vessel that carries blood from the heart to the rest of the body (the aorta) and in certain valves in the heart. Other signs and symptoms include tooth abnormalities, low bone density (osteopenia), and other bone problems. Some people with IFIH1 gene mutations have signs and symptoms of both Singleton-Merten syndrome and Aicardi-Goutires syndrome (described above), suggesting that these two conditions may be part of a spectrum caused by IFIH1 gene mutations.As in Aicardi-Goutires syndrome, the IFIH1 gene mutations involved in Singleton-Merten syndrome are described as \"gain-of-function\" and lead to excessive immune system activity and inflammation, disrupting calcium handling in the body. It is unclear why people with gain-of-function mutations in the IFIH1 gene develop signs and symptoms of one condition or the other.Singleton-Merten syndrome and Aicardi-Goutires syndrome both have autoimmune features, which occur when the immune system malfunctions and damages the body's own tissues and organs. Common variations (polymorphisms) in the IFIH1 gene have been associated with other autoimmune disorders. It is thought that polymorphisms that enhance the activity of the MDA5 protein increase the risk of certain autoimmune disorders, while polymorphisms that reduce the activity of the MDA5 protein help protect against others.    Cytogenetic Location: 2q24.2, which is the long (q) arm of chromosome 2 at position 24.2Molecular Location: base pairs 162,267,079 to 162,318,708 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AGS7CADM-140 autoantigenclinically amyopathic dermatomyositis autoantigen 140 kDaDEAD/H (Asp-Glu-Ala-Asp/His) box polypeptidehelicardhelicase with 2 CARD domainsHlcdIDDM19interferon-induced helicase C domain-containing protein 1MDA-5MDA5melanoma differentiation-associated gene 5melanoma differentiation-associated protein 5murabutide down-regulated proteinRIG-I-like receptor 2RLR-2RNA helicase-DEAD box protein 116SGMRT1     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Immunobiology: The Immune System in Health and Disease (fifth edition, 2001): Interferons Induced by Viral Infection Make Several Contributions to Host DefenseMolecular Biology of the Cell (fourth edition, 2002): Innate ImmunityUndiagnosed Diseases Network      Tests Listed in the Genetic Testing Registry (1 link)     Tests of IFIH1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     INTERFERON-INDUCED HELICASE C DOMAIN-CONTAINING PROTEIN 1SINGLETON-MERTEN SYNDROME 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology: The Immune System in Health and Disease (fifth edition, 2001): Interferons Induced by Viral Infection Make Several Contributions to Host DefenseMolecular Biology of the Cell (fourth edition, 2002): Innate ImmunityUndiagnosed Diseases Network  Tests of IFIH1  PubMed  INTERFERON-INDUCED HELICASE C DOMAIN-CONTAINING PROTEIN 1SINGLETON-MERTEN SYNDROME 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Asgari S, Schlapbach LJ, Anchisi S, Hammer C, Bartha I, Junier T, Mottet-Osman G, Posfay-Barbe KM, Longchamp D, Stocker M, Cordey S, Kaiser L, Riedel T, Kenna T, Long D, Schibler A, Telenti A, Tapparel C, McLaren PJ, Garcin D, Fellay J. Severe viral respiratory infections in children with IFIH1 loss-of-function mutations. Proc Natl Acad Sci U S A. 2017 Aug 1;114(31):8342-8347. doi: 10.1073/pnas.1704259114. Epub 2017 Jul 17. Citation on PubMed or Free article on PubMed CentralBerke IC, Yu X, Modis Y, Egelman EH. MDA5 assembles into a polar helical filament on dsRNA. Proc Natl Acad Sci U S A. 2012 Nov 6;109(45):18437-41. doi: 10.1073/pnas.1212186109. Epub 2012 Oct 22. Citation on PubMed or Free article on PubMed CentralBursztejn AC, Briggs TA, del Toro Duany Y, Anderson BH, O'Sullivan J, Williams SG, Bodemer C, Fraitag S, Gebhard F, Leheup B, Lemelle I, Oojageer A, Raffo E, Schmitt E, Rice GI, Hur S, Crow YJ. Unusual cutaneous features associated with a heterozygous gain-of-function mutation in IFIH1: overlap between Aicardi-Goutires and Singleton-Merten syndromes. Br J Dermatol. 2015 Dec;173(6):1505-13. doi: 10.1111/bjd.14073. Epub 2015 Oct 29. Citation on PubMed or Free article on PubMed CentralOMIM: INTERFERON-INDUCED HELICASE C DOMAIN-CONTAINING PROTEIN 1Lamborn IT, Jing H, Zhang Y, Drutman SB, Abbott JK, Munir S, Bade S, Murdock HM, Santos CP, Brock LG, Masutani E, Fordjour EY, McElwee JJ, Hughes JD, Nichols DP, Belkadi A, Oler AJ, Happel CS, Matthews HF, Abel L, Collins PL, Subbarao K, Gelfand EW, Ciancanelli MJ, Casanova JL, Su HC. Recurrent rhinovirus infections in a child with inherited MDA5 deficiency. J Exp Med. 2017 Jul 3;214(7):1949-1972. doi: 10.1084/jem.20161759. Epub 2017 Jun 12. Citation on PubMed or Free article on PubMed CentralLoo YM, Gale M Jr. Immune signaling by RIG-I-like receptors. Immunity. 2011 May 27;34(5):680-92. doi: 10.1016/j.immuni.2011.05.003. Review. Citation on PubMed or Free article on PubMed CentralLooney BM, Xia CQ, Concannon P, Ostrov DA, Clare-Salzler MJ. Effects of type 1 diabetes-associated IFIH1 polymorphisms on MDA5 function and expression. Curr Diab Rep. 2015 Nov;15(11):96. doi: 10.1007/s11892-015-0656-8. Review. Citation on PubMedOda H, Nakagawa K, Abe J, Awaya T, Funabiki M, Hijikata A, Nishikomori R, Funatsuka M, Ohshima Y, Sugawara Y, Yasumi T, Kato H, Shirai T, Ohara O, Fujita T, Heike T. Aicardi-Goutires syndrome is caused by IFIH1 mutations. Am J Hum Genet. 2014 Jul 3;95(1):121-5. doi: 10.1016/j.ajhg.2014.06.007. Citation on PubMed or Free article on PubMed CentralRice GI, Del Toro Duany Y, Jenkinson EM, Forte GM, Anderson BH, Ariaudo G, Bader-Meunier B, Baildam EM, Battini R, Beresford MW, Casarano M, Chouchane M, Cimaz R, Collins AE, Cordeiro NJ, Dale RC, Davidson JE, De Waele L, Desguerre I, Faivre L, Fazzi E, Isidor B, Lagae L, Latchman AR, Lebon P, Li C, Livingston JH, Loureno CM, Mancardi MM, Masurel-Paulet A, McInnes IB, Menezes MP, Mignot C, O'Sullivan J, Orcesi S, Picco PP, Riva E, Robinson RA, Rodriguez D, Salvatici E, Scott C, Szybowska M, Tolmie JL, Vanderver A, Vanhulle C, Vieira JP, Webb K, Whitney RN, Williams SG, Wolfe LA, Zuberi SM, Hur S, Crow YJ. Gain-of-function mutations in IFIH1 cause a spectrum of human disease phenotypes associated with upregulated type I interferon signaling. Nat Genet. 2014 May;46(5):503-509. doi: 10.1038/ng.2933. Epub 2014 Mar 30. Citation on PubMed or Free article on PubMed CentralRutsch F, MacDougall M, Lu C, Buers I, Mamaeva O, Nitschke Y, Rice GI, Erlandsen H, Kehl HG, Thiele H, Nrnberg P, Hhne W, Crow YJ, Feigenbaum A, Hennekam RC. A specific IFIH1 gain-of-function mutation causes Singleton-Merten syndrome. Am J Hum Genet. 2015 Feb 5;96(2):275-82. doi: 10.1016/j.ajhg.2014.12.014. Epub 2015 Jan 22. Citation on PubMed or Free article on PubMed CentralVan Eyck L, De Somer L, Pombal D, Bornschein S, Frans G, Humblet-Baron S, Moens L, de Zegher F, Bossuyt X, Wouters C, Liston A. Brief Report: IFIH1 Mutation Causes Systemic Lupus Erythematosus With Selective IgA Deficiency. Arthritis Rheumatol. 2015 Jun;67(6):1592-7. doi: 10.1002/art.39110. Citation on PubMed \n",
            "TTC21B\t   From NCBI Gene:This gene encodes a member of TTC21 family, containing several tetratricopeptide repeat (TPR) domains. This protein is localized to the cilium axoneme, and may play a role in retrograde intraflagellar transport in cilia. Mutations in this gene are associated with various ciliopathies, nephronophthisis 12, and asphyxiating thoracic dystrophy 4. [provided by RefSeq, Oct 2011]From UniProt:Component of the IFT complex A (IFT-A), a complex required for retrograde ciliary transport and entry into cilia of G protein-coupled receptors (GPCRs). Essential for retrograde trafficking of IFT-1, IFT-B and GPCRs (PubMed:27932497). Negatively modulates the SHH signal transduction (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Joubert syndromeAsphyxiating thoracic dystrophyNephronophthisisFrom NCBI Gene:Nephronophthisis 12Asphyxiating thoracic dystrophy 4From UniProt:Ciliary dysfunction leads to a broad spectrum of disorders, collectively termed ciliopathies. Overlapping clinical features include retinal degeneration, renal cystic disease, skeletal abnormalities, fibrosis of various organ, and a complex range of anatomical and functional defects of the central and peripheral nervous system. The ciliopathy range of diseases includes Meckel-Gruber syndrome, Bardet-Biedl syndrome, Joubert syndrome, nephronophtisis, Senior-Loken syndrome, and Jeune asphyxiating thoracic dystrophy among others. TTC21B is causally associated with diverse ciliopathies. It also acts as a modifier gene across the ciliopathy spectrum, interacting in trans with mutations in other ciliopathy-causing genes and contributing to disease manifestation and severity.Joubert syndrome 11 (JBTS11): A disorder presenting with cerebellar ataxia, oculomotor apraxia, hypotonia, neonatal breathing abnormalities and psychomotor delay. Neuroradiologically, it is characterized by cerebellar vermian hypoplasia/aplasia, thickened and reoriented superior cerebellar peduncles, and an abnormally large interpeduncular fossa, giving the appearance of a molar tooth on transaxial slices (molar tooth sign). Additional variable features include retinal dystrophy and renal disease. [MIM:613820]Nephronophthisis 12 (NPHP12): An autosomal recessive disorder resulting in end-stage renal disease. It is a progressive tubulo-interstitial kidney disorder histologically characterized by modifications of the tubules with thickening of the basement membrane, interstitial fibrosis and, in the advanced stages, medullary cysts. Some patients manifest extra-renal features including retinal, skeletal and central nervous system defects. [MIM:613820]Short-rib thoracic dysplasia 4 with or without polydactyly (SRTD4): A form of short-rib thoracic dysplasia, a group of autosomal recessive ciliopathies that are characterized by a constricted thoracic cage, short ribs, shortened tubular bones, and a 'trident' appearance of the acetabular roof. Polydactyly is variably present. Non-skeletal involvement can include cleft lip/palate as well as anomalies of major organs such as the brain, eye, heart, kidneys, liver, pancreas, intestines, and genitalia. Some forms of the disease are lethal in the neonatal period due to respiratory insufficiency secondary to a severely restricted thoracic cage, whereas others are compatible with life. Disease spectrum encompasses Ellis-van Creveld syndrome, asphyxiating thoracic dystrophy (Jeune syndrome), Mainzer-Saldino syndrome, and short rib-polydactyly syndrome. [MIM:613819]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q24.3, which is the long (q) arm of chromosome 2 at position 24.3Molecular Location: base pairs 165,873,362 to 165,953,838 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATD4IFT139IFT139BJBTS11Nbla10696NPHP12SRTD4THM1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TTC21B      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: NEPHRONOPHTHISIS 12OMIM: SHORT-RIB THORACIC DYSPLASIA 4 WITH OR WITHOUT POLYDACTYLYOMIM: TETRATRICOPEPTIDE REPEAT DOMAIN-CONTAINING PROTEIN 21B      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TTC21B  OMIM: NEPHRONOPHTHISIS 12OMIM: SHORT-RIB THORACIC DYSPLASIA 4 WITH OR WITHOUT POLYDACTYLYOMIM: TETRATRICOPEPTIDE REPEAT DOMAIN-CONTAINING PROTEIN 21B  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "STK39\t\n",
            "LRP2\t   The LRP2 gene provides instructions for making a protein called megalin, which functions as a receptor. Receptor proteins have specific sites into which certain other proteins, called ligands, fit like keys into locks. Together, ligands and their receptors trigger signals that affect cell development and function. Megalin has many ligands involved in various body processes, including the absorption of vitamins A and D, immune functioning, stress response, and the transport of fats in the bloodstream.Megalin is embedded in the membrane of cells that line the surfaces and cavities of the body (epithelial cells). The receptor helps move its ligands from the cell surface into the cell (endocytosis), and is also involved in transporting the ligands of a related receptor called cubulin. Megalin is active in the development and function of many parts of the body, including the brain and spinal cord (central nervous system), eyes, ears, lungs, intestine, reproductive system, and the small tubes in the kidneys where urine is formed (renal tubules).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Donnai-Barrow syndrome     At least twelve LRP2 gene mutations have been identified in people with Donnai-Barrow syndrome. These mutations are believed to result in the absence of functional megalin protein. The lack of functional megalin in the renal tubules causes megalin's various ligands to be excreted in the urine rather than being absorbed back into the bloodstream. The features of Donnai-Barrow syndrome are probably caused by the inability of megalin to help absorb these ligands, disruption of biochemical signaling pathways, or other effects of the nonfunctional megalin protein. However, it is unclear how these abnormalities result in the specific signs and symptoms of the disorder.A condition previously classified as a separate disorder called facio-oculo-acoustico-renal (FOAR) syndrome has also been found to be caused by LRP2 gene mutations. FOAR syndrome is now considered to be the same disorder as Donnai-Barrow syndrome. More About This Health Condition      Coloboma     Genetics Home Reference provides information about coloboma. More About This Health Condition      Prostate cancer     Genetics Home Reference provides information about prostate cancer. More About This Health Condition      Cancers     Certain common genetic variations (polymorphisms) in the LRP2 gene may be associated with differences in the progression, recurrence, and severity of prostate cancer in affected men. Androgens, which are male sex hormones that contribute to the growth of cancerous (malignant) prostate tumors, are among megalin's ligands. A recent study suggests that prostate tumor cells may produce increased amounts of megalin and thereby absorb more androgens. LRP2 gene polymorphisms that increase the activity of megalin may encourage more aggressive tumor growth, while polymorphisms that decrease the activity of megalin may tend to slow tumor growth.      Other disorders     Certain LRP2 gene polymorphisms have also been associated with increased levels of low-density lipoproteins (LDLs) and cholesterol in the blood. LDLs are the primary carriers of cholesterol in the blood. Cholesterol is a waxy, fat-like substance that is produced in the body and obtained from foods that come from animals. Increased LDL levels may result in excess cholesterol circulating through the bloodstream and accumulating on the walls of the blood vessels, increasing a person's risk of cardiovascular disease.LDLs are among megalin's ligands, and researchers believe that variations in megalin function resulting from LRP2 gene changes may influence LDL levels.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least twelve LRP2 gene mutations have been identified in people with Donnai-Barrow syndrome. These mutations are believed to result in the absence of functional megalin protein. The lack of functional megalin in the renal tubules causes megalin's various ligands to be excreted in the urine rather than being absorbed back into the bloodstream. The features of Donnai-Barrow syndrome are probably caused by the inability of megalin to help absorb these ligands, disruption of biochemical signaling pathways, or other effects of the nonfunctional megalin protein. However, it is unclear how these abnormalities result in the specific signs and symptoms of the disorder.A condition previously classified as a separate disorder called facio-oculo-acoustico-renal (FOAR) syndrome has also been found to be caused by LRP2 gene mutations. FOAR syndrome is now considered to be the same disorder as Donnai-Barrow syndrome. More About This Health Condition  Genetics Home Reference provides information about coloboma. More About This Health Condition  Genetics Home Reference provides information about prostate cancer. More About This Health Condition  Certain common genetic variations (polymorphisms) in the LRP2 gene may be associated with differences in the progression, recurrence, and severity of prostate cancer in affected men. Androgens, which are male sex hormones that contribute to the growth of cancerous (malignant) prostate tumors, are among megalin's ligands. A recent study suggests that prostate tumor cells may produce increased amounts of megalin and thereby absorb more androgens. LRP2 gene polymorphisms that increase the activity of megalin may encourage more aggressive tumor growth, while polymorphisms that decrease the activity of megalin may tend to slow tumor growth.  Certain LRP2 gene polymorphisms have also been associated with increased levels of low-density lipoproteins (LDLs) and cholesterol in the blood. LDLs are the primary carriers of cholesterol in the blood. Cholesterol is a waxy, fat-like substance that is produced in the body and obtained from foods that come from animals. Increased LDL levels may result in excess cholesterol circulating through the bloodstream and accumulating on the walls of the blood vessels, increasing a person's risk of cardiovascular disease.LDLs are among megalin's ligands, and researchers believe that variations in megalin function resulting from LRP2 gene changes may influence LDL levels.    Cytogenetic Location: 2q31.1, which is the long (q) arm of chromosome 2 at position 31.1Molecular Location: base pairs 169,127,109 to 169,362,612 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        calcium sensor proteinDBSglycoprotein 330 (gp330)gp330Heymann nephritis antigen homologlow density lipoprotein receptor-related protein 2LRP2_HUMANmegalin     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Donnai-Barrow Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of LRP2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Donnai-Barrow Syndrome  Tests of LRP2  PubMed  LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Christensen EI, Birn H. Megalin and cubilin: multifunctional endocytic receptors. Nat Rev Mol Cell Biol. 2002 Apr;3(4):256-66. Review. Citation on PubMedChristensen EI, Nielsen R. Role of megalin and cubilin in renal physiology and pathophysiology. Rev Physiol Biochem Pharmacol. 2007;158:1-22. Citation on PubMedFisher CE, Howie SE. The role of megalin (LRP-2/Gp330) during development. Dev Biol. 2006 Aug 15;296(2):279-97. Epub 2006 Jun 8. Review. Citation on PubMedHolt SK, Karyadi DM, Kwon EM, Stanford JL, Nelson PS, Ostrander EA. Association of megalin genetic polymorphisms with prostate cancer risk and prognosis. Clin Cancer Res. 2008 Jun 15;14(12):3823-31. doi: 10.1158/1078-0432.CCR-07-4566. Citation on PubMed or Free article on PubMed CentralKantarci S, Al-Gazali L, Hill RS, Donnai D, Black GC, Bieth E, Chassaing N, Lacombe D, Devriendt K, Teebi A, Loscertales M, Robson C, Liu T, MacLaughlin DT, Noonan KM, Russell MK, Walsh CA, Donahoe PK, Pober BR. Mutations in LRP2, which encodes the multiligand receptor megalin, cause Donnai-Barrow and facio-oculo-acoustico-renal syndromes. Nat Genet. 2007 Aug;39(8):957-9. Epub 2007 Jul 15. Citation on PubMed or Free article on PubMed CentralKantarci S, Ragge NK, Thomas NS, Robinson DO, Noonan KM, Russell MK, Donnai D, Raymond FL, Walsh CA, Donahoe PK, Pober BR. Donnai-Barrow syndrome (DBS/FOAR) in a child with a homozygous LRP2 mutation due to complete chromosome 2 paternal isodisomy. Am J Med Genet A. 2008 Jul 15;146A(14):1842-7. doi: 10.1002/ajmg.a.32381. Citation on PubMed or Free article on PubMed CentralOMIM: LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 2Mii A, Nakajima T, Fujita Y, Iino Y, Kamimura K, Bujo H, Saito Y, Emi M, Katayama Y. Genetic association of low-density lipoprotein receptor-related protein 2 (LRP2) with plasma lipid levels. J Atheroscler Thromb. 2007 Dec;14(6):310-6. Epub 2007 Dec 17. Citation on PubMed \n",
            "METTL8\t\n",
            "ITGA6\t   The ITGA6 gene provides instructions for making one part (the 6 subunit) of two proteins known as 64 integrin and 61 integrin. Integrins are a group of proteins that regulate the attachment of cells to one another (cell-cell adhesion) and to the surrounding network of proteins and other molecules (cell-matrix adhesion). Integrins also transmit chemical signals that regulate cell growth and the activity of certain genes.The 64 integrin protein is found primarily in epithelial cells, which are cells that line the surfaces and cavities of the body. This protein plays a particularly important role in strengthening and stabilizing the skin. It is a component of hemidesmosomes, which are microscopic structures that anchor the outer layer of the skin (the epidermis) to underlying layers. As part of a complex network of proteins in hemidesmosomes, 64 integrin helps to hold the layers of skin together.The other integrin made with the 6 subunit, 61 integrin, functions during the formation of organs and tissues before birth. The 61 integrin protein has not been as well studied as 64 integrin.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Epidermolysis bullosa with pyloric atresia     At least five mutations in the ITGA6 gene have been found to cause epidermolysis bullosa with pyloric atresia (EB-PA). In addition to skin blistering, people with EB-PA are born with a life-threatening obstruction of the digestive tract called pyloric atresia. Mutations in the ITGA6 gene account for about 5 percent of all cases of EB-PA.The ITGA6 gene mutations responsible for EB-PA lead to a loss of functional 64 integrin. These mutations alter the normal structure and function of the 6 integrin subunit or prevent cells from producing enough of this subunit.  The resulting shortage of functional 64 integrin causes cells in the epidermis to be fragile and easily damaged. Friction or other minor trauma can cause the skin layers to separate, leading to the widespread formation of blisters. It is less clear how mutations in the ITGA6 gene are related to pyloric atresia. More About This Health Condition      Prostate cancer     Genetics Home Reference provides information about prostate cancer. More About This Health Condition      Cancers     Researchers believe that both 61 integrin and 64 integrin may play critical roles in the progression of cancerous tumors called carcinomas. These cancers arise in epithelial cells and can affect many tissues and organs, including the breast, lung, liver, prostate, and skin.Changes in the location and activity of 61 integrin and 64 integrin within cancer cells are associated with the progression of carcinomas. The integrin proteins activate key signaling molecules, which trigger cancer cells to migrate through the body and invade other tissues. These signals also make cancer cells more resistant to self-destruction (apoptosis).Recent studies suggest that, in addition to their roles in the progression of existing carcinomas, 61 integrin and 64 integrin may be involved in the initial formation of these tumors.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least five mutations in the ITGA6 gene have been found to cause epidermolysis bullosa with pyloric atresia (EB-PA). In addition to skin blistering, people with EB-PA are born with a life-threatening obstruction of the digestive tract called pyloric atresia. Mutations in the ITGA6 gene account for about 5 percent of all cases of EB-PA.The ITGA6 gene mutations responsible for EB-PA lead to a loss of functional 64 integrin. These mutations alter the normal structure and function of the 6 integrin subunit or prevent cells from producing enough of this subunit.  The resulting shortage of functional 64 integrin causes cells in the epidermis to be fragile and easily damaged. Friction or other minor trauma can cause the skin layers to separate, leading to the widespread formation of blisters. It is less clear how mutations in the ITGA6 gene are related to pyloric atresia. More About This Health Condition  Genetics Home Reference provides information about prostate cancer. More About This Health Condition  Researchers believe that both 61 integrin and 64 integrin may play critical roles in the progression of cancerous tumors called carcinomas. These cancers arise in epithelial cells and can affect many tissues and organs, including the breast, lung, liver, prostate, and skin.Changes in the location and activity of 61 integrin and 64 integrin within cancer cells are associated with the progression of carcinomas. The integrin proteins activate key signaling molecules, which trigger cancer cells to migrate through the body and invade other tissues. These signals also make cancer cells more resistant to self-destruction (apoptosis).Recent studies suggest that, in addition to their roles in the progression of existing carcinomas, 61 integrin and 64 integrin may be involved in the initial formation of these tumors.    Cytogenetic Location: 2q31.1, which is the long (q) arm of chromosome 2 at position 31.1Molecular Location: base pairs 172,427,354 to 172,506,455 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD49fCD49f AntigensCluster of differentiation antigen 49fFLJ18737integrin alpha 6integrin alpha chain, alpha 6Integrin alpha6integrin, alpha-6integrin, alpha 6ITA6_HUMANLymphocyte antigen CD49FVLA-6     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Madame Curie Bioscience Database: Integrins in Cancer Cell InvasionMadame Curie Bioscience Database: Integrins: An Overview of Structural and Functional AspectsMolecular Biology of the Cell (fourth edition, 2002): Integrins      Clinical Information from GeneReviews (1 link)     Epidermolysis Bullosa with Pyloric Atresia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ITGA6      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     INTEGRIN, ALPHA-6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Integrins in Cancer Cell InvasionMadame Curie Bioscience Database: Integrins: An Overview of Structural and Functional AspectsMolecular Biology of the Cell (fourth edition, 2002): Integrins  Epidermolysis Bullosa with Pyloric Atresia  Tests of ITGA6  PubMed  INTEGRIN, ALPHA-6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Allegra M, Gagnoux-Palacios L, Gache Y, Roques S, Lestringant G, Ortonne JP, Meneguzzi G. Rapid decay of alpha6 integrin caused by a mis-sense mutation in the propeller domain results in severe junctional epidermolysis bullosa with pyloric atresia. J Invest Dermatol. 2003 Dec;121(6):1336-43. Citation on PubMedAshton GH, Sorelli P, Mellerio JE, Keane FM, Eady RA, McGrath JA. Alpha 6 beta 4 integrin abnormalities in junctional epidermolysis bullosa with pyloric atresia. Br J Dermatol. 2001 Feb;144(2):408-14. Citation on PubMedChung J, Mercurio AM. Contributions of the alpha6 integrins to breast carcinoma survival and progression. Mol Cells. 2004 Apr 30;17(2):203-9. Review. Citation on PubMedChung J, Yoon S, Datta K, Bachelder RE, Mercurio AM. Hypoxia-induced vascular endothelial growth factor transcription and protection from apoptosis are dependent on alpha6beta1 integrin in breast carcinoma cells. Cancer Res. 2004 Jul 15;64(14):4711-6. Citation on PubMedGache Y, Romero-Graillet C, Spadafora A, Lpinard C, Descamps P, Bardon CB, Ortonne JP, Meneguzzi G. A novel homozygous mutation affecting integrin alpha6 in a case of junctional epidermolysis bullosa with pyloric atresia detected in utero by ultrasound examination. J Invest Dermatol. 1998 Nov;111(5):914-6. Citation on PubMedLipscomb EA, Mercurio AM. Mobilization and activation of a signaling competent alpha6beta4integrin underlies its contribution to carcinoma progression. Cancer Metastasis Rev. 2005 Sep;24(3):413-23. Review. Citation on PubMedNejjari M, Hafdi Z, Dumortier J, Bringuier AF, Feldmann G, Scoazec JY. alpha6beta1 integrin expression in hepatocarcinoma cells: regulation and role in cell adhesion and migration. Int J Cancer. 1999 Nov 12;83(4):518-25. Citation on PubMedPulkkinen L, Kimonis VE, Xu Y, Spanou EN, McLean WH, Uitto J. Homozygous alpha6 integrin mutation in junctional epidermolysis bullosa with congenital duodenal atresia. Hum Mol Genet. 1997 May;6(5):669-74. Citation on PubMedRuzzi L, Gagnoux-Palacios L, Pinola M, Belli S, Meneguzzi G, D'Alessio M, Zambruno G. A homozygous mutation in the integrin alpha6 gene in junctional epidermolysis bullosa with pyloric atresia. J Clin Invest. 1997 Jun 15;99(12):2826-31. Citation on PubMed or Free article on PubMed Central \n",
            "SCRN3\t\n",
            "HOXD9\t\n",
            "PDE11A\t   From NCBI Gene:The 3',5'-cyclic nucleotides cAMP and cGMP function as second messengers in a wide variety of signal transduction pathways. 3',5'-cyclic nucleotide phosphodiesterases (PDEs) catalyze the hydrolysis of cAMP and cGMP to the corresponding 5'-monophosphates and provide a mechanism to downregulate cAMP and cGMP signaling. This gene encodes a member of the PDE protein superfamily. Mutations in this gene are a cause of Cushing disease and adrenocortical hyperplasia. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Plays a role in signal transduction by regulating the intracellular concentration of cyclic nucleotides cAMP and cGMP. Catalyzes the hydrolysis of both cAMP and cGMP to 5'-AMP and 5'-GMP, respectively.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary macronodular adrenal hyperplasiaFrom NCBI Gene:Pigmented nodular adrenocortical disease, primary, 2From UniProt:Primary pigmented nodular adrenocortical disease 2 (PPNAD2): A rare bilateral adrenal defect causing ACTH-independent Cushing syndrome. Macroscopic appearance of the adrenals is characteristic with small pigmented micronodules observed in the cortex. Adrenal glands show overall normal size and weight, and multiple small yellow-to-dark brown nodules surrounded by a cortex with a uniform appearance. Microscopically, there are moderate diffuse cortical hyperplasia with mostly nonpigmented nodules, multiple capsular deficits and massive circumscribed and infiltrating extra-adrenal cortical excrescences with micronodules. Clinical manifestations of Cushing syndrome include facial and truncal obesity, abdominal striae, muscular weakness, osteoporosis, arterial hypertension, diabetes. [MIM:610475]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q31.2, which is the long (q) arm of chromosome 2 at position 31.2Molecular Location: base pairs 177,623,249 to 178,108,339 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PPNAD2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PDE11A      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PHOSPHODIESTERASE 11AOMIM: PIGMENTED NODULAR ADRENOCORTICAL DISEASE, PRIMARY, 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PDE11A  OMIM: PHOSPHODIESTERASE 11AOMIM: PIGMENTED NODULAR ADRENOCORTICAL DISEASE, PRIMARY, 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TTN\t   The TTN gene provides instructions for making a very large protein called titin. This protein plays an important role in muscles the body uses for movement (skeletal muscles) and in heart (cardiac) muscle. Slightly different versions (called isoforms) of titin are made in different muscles.Within muscle cells, titin is an essential component of structures called sarcomeres. Sarcomeres are the basic units of muscle contraction; they are made of proteins that generate the mechanical force needed for muscles to contract. Titin has several functions within sarcomeres. One of the protein's main jobs is to provide structure, flexibility, and stability to these cell structures. Titin interacts with other muscle proteins, including actin and myosin, to keep the components of sarcomeres in place as muscles contract and relax. Titin also contains a spring-like region that allows muscles to stretch. Additionally, researchers have found that titin plays a role in chemical signaling and in assembling new sarcomeres.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Centronuclear myopathy     At least 12 mutations in the TTN gene have been found to cause centronuclear myopathy, a condition that is characterized by muscle weakness (myopathy) in the skeletal muscles. Most of these mutations alter the way the gene's instructions are used to produce titin, resulting in production of an abnormal protein with reduced or altered activity in muscle cells. Other mutations prevent the production of titin protein. It is unclear how TTN gene mutations cause centronuclear myopathy, but it is likely that a shortage of normal titin protein leads to dysfunction of the sarcomere. Abnormal sarcomeres prevent muscle cells from contracting and relaxing normally, resulting in the muscle weakness that is characteristic of centronuclear myopathy. More About This Health Condition      Early-onset myopathy with fatal cardiomyopathy     At least two mutations in the TTN gene have been identified in people with early-onset myopathy with fatal cardiomyopathy (EOMFC), an inherited disease that affects both skeletal and cardiac muscle. These genetic changes occur near the end of the TTN gene and lead to the production of an abnormally short version of the titin protein. The defective protein disrupts the function of sarcomeres, preventing skeletal and cardiac muscle from developing and working normally. These muscle abnormalities underlie the characteristic features of EOMFC, including skeletal muscle weakness and a form of heart disease called dilated cardiomyopathy. More About This Health Condition      Familial dilated cardiomyopathy     More than 100 mutations in the TTN gene have been found to cause familial dilated cardiomyopathy, a condition that weakens and enlarges the heart, preventing it from pumping blood efficiently. Signs and symptoms of familial dilated cardiomyopathy typically begin in mid-adulthood and result in heart failure. TTN gene mutations account for approximately one-quarter of all cases of familial dilated cardiomyopathy. These mutations result in the production of an abnormal titin protein, particularly isoforms that are found in cardiac muscle. It is unclear how the altered protein causes familial dilated cardiomyopathy, but it likely impairs sarcomere function and disrupts chemical signaling. Changes in sarcomere function reduce the heart's ability to contract, weakening cardiac muscle and leading to the signs and symptoms of familial dilated cardiomyopathy. More About This Health Condition      Hereditary myopathy with early respiratory failure     Several mutations in the TTN gene have been found to cause hereditary myopathy with early respiratory failure (HMERF), an inherited disease that affects muscles used for movement (skeletal muscles) and muscles that are needed for breathing (respiratory muscles). These mutations change single DNA building blocks (nucleotides) in a region of the TTN gene called exon 344. These changes alter a region of the titin protein called the FN3 119 domain and are thought to impair the folding of the titin protein into its normal 3-dimensional shape. Researchers are studying how abnormally folded titin contributes to the muscle damage underlying the signs and symptoms of HMERF. It is unclear why these effects are usually limited to certain skeletal muscles and respiratory muscles, and do not involve cardiac muscle. More About This Health Condition      Limb-girdle muscular dystrophy     At least two TTN gene mutations have been found to cause limb-girdle muscular dystrophy type 2J (LGMD2J). Limb-girdle muscular dystrophy is a group of related disorders characterized by weakness and wasting of skeletal muscles, particularly in the shoulders, hips, and limbs. LGMD2J is a type of limb-girdle muscular dystrophy that has been identified primarily in the Finnish population. The genetic change found in this population deletes several amino acids and replaces them with other amino acids at the end of the titin protein. This complex mutation, known as FINmaj, causes LGMD2J when it occurs in both copies of the TTN gene. The FINmaj mutation may disrupt titin's interactions with other proteins that are needed to maintain muscle fibers. Loss of muscle fibers causes muscles to weaken and waste away over time, resulting in the signs and symptoms of limb-girdle muscular dystrophy. More About This Health Condition      Tibial muscular dystrophy     Several mutations in the TTN gene have been identified in people with tibial muscular dystrophy, a condition that primarily affects the muscles at the front of the lower leg. The FINmaj mutation (described above) has been found to cause tibial muscular dystrophy in all affected people of Finnish descent. Other TTN gene mutations cause tibial muscular dystrophy in non-Finnish European populations. This condition is caused by mutations that occur in one copy of the TTN gene.Researchers predict that the TTN gene mutations responsible for tibial muscular dystrophy, including FINmaj, alter the ability of the titin protein to interact with other proteins within sarcomeres. These alterations likely impair  muscle fiber maintenance or muscle contraction, causing muscles to weaken and waste away over time. It is unclear why the resulting weakness is usually limited to muscles in the lower legs in tibial muscular dystrophy.Researchers are working to determine why some conditions resulting from TTN gene mutations predominantly affect cardiac muscle, some predominantly affect skeletal muscle, and some affect both. They suspect that these differences may be related to the location of mutations in the TTN gene and how they affect the many versions of titin that are produced in different muscles. More About This Health Condition      Arrhythmogenic right ventricular cardiomyopathy     Genetics Home Reference provides information about arrhythmogenic right ventricular cardiomyopathy. More About This Health Condition      Familial hypertrophic cardiomyopathy     Genetics Home Reference provides information about familial hypertrophic cardiomyopathy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 12 mutations in the TTN gene have been found to cause centronuclear myopathy, a condition that is characterized by muscle weakness (myopathy) in the skeletal muscles. Most of these mutations alter the way the gene's instructions are used to produce titin, resulting in production of an abnormal protein with reduced or altered activity in muscle cells. Other mutations prevent the production of titin protein. It is unclear how TTN gene mutations cause centronuclear myopathy, but it is likely that a shortage of normal titin protein leads to dysfunction of the sarcomere. Abnormal sarcomeres prevent muscle cells from contracting and relaxing normally, resulting in the muscle weakness that is characteristic of centronuclear myopathy. More About This Health Condition  At least two mutations in the TTN gene have been identified in people with early-onset myopathy with fatal cardiomyopathy (EOMFC), an inherited disease that affects both skeletal and cardiac muscle. These genetic changes occur near the end of the TTN gene and lead to the production of an abnormally short version of the titin protein. The defective protein disrupts the function of sarcomeres, preventing skeletal and cardiac muscle from developing and working normally. These muscle abnormalities underlie the characteristic features of EOMFC, including skeletal muscle weakness and a form of heart disease called dilated cardiomyopathy. More About This Health Condition  More than 100 mutations in the TTN gene have been found to cause familial dilated cardiomyopathy, a condition that weakens and enlarges the heart, preventing it from pumping blood efficiently. Signs and symptoms of familial dilated cardiomyopathy typically begin in mid-adulthood and result in heart failure. TTN gene mutations account for approximately one-quarter of all cases of familial dilated cardiomyopathy. These mutations result in the production of an abnormal titin protein, particularly isoforms that are found in cardiac muscle. It is unclear how the altered protein causes familial dilated cardiomyopathy, but it likely impairs sarcomere function and disrupts chemical signaling. Changes in sarcomere function reduce the heart's ability to contract, weakening cardiac muscle and leading to the signs and symptoms of familial dilated cardiomyopathy. More About This Health Condition  Several mutations in the TTN gene have been found to cause hereditary myopathy with early respiratory failure (HMERF), an inherited disease that affects muscles used for movement (skeletal muscles) and muscles that are needed for breathing (respiratory muscles). These mutations change single DNA building blocks (nucleotides) in a region of the TTN gene called exon 344. These changes alter a region of the titin protein called the FN3 119 domain and are thought to impair the folding of the titin protein into its normal 3-dimensional shape. Researchers are studying how abnormally folded titin contributes to the muscle damage underlying the signs and symptoms of HMERF. It is unclear why these effects are usually limited to certain skeletal muscles and respiratory muscles, and do not involve cardiac muscle. More About This Health Condition  At least two TTN gene mutations have been found to cause limb-girdle muscular dystrophy type 2J (LGMD2J). Limb-girdle muscular dystrophy is a group of related disorders characterized by weakness and wasting of skeletal muscles, particularly in the shoulders, hips, and limbs. LGMD2J is a type of limb-girdle muscular dystrophy that has been identified primarily in the Finnish population. The genetic change found in this population deletes several amino acids and replaces them with other amino acids at the end of the titin protein. This complex mutation, known as FINmaj, causes LGMD2J when it occurs in both copies of the TTN gene. The FINmaj mutation may disrupt titin's interactions with other proteins that are needed to maintain muscle fibers. Loss of muscle fibers causes muscles to weaken and waste away over time, resulting in the signs and symptoms of limb-girdle muscular dystrophy. More About This Health Condition  Several mutations in the TTN gene have been identified in people with tibial muscular dystrophy, a condition that primarily affects the muscles at the front of the lower leg. The FINmaj mutation (described above) has been found to cause tibial muscular dystrophy in all affected people of Finnish descent. Other TTN gene mutations cause tibial muscular dystrophy in non-Finnish European populations. This condition is caused by mutations that occur in one copy of the TTN gene.Researchers predict that the TTN gene mutations responsible for tibial muscular dystrophy, including FINmaj, alter the ability of the titin protein to interact with other proteins within sarcomeres. These alterations likely impair  muscle fiber maintenance or muscle contraction, causing muscles to weaken and waste away over time. It is unclear why the resulting weakness is usually limited to muscles in the lower legs in tibial muscular dystrophy.Researchers are working to determine why some conditions resulting from TTN gene mutations predominantly affect cardiac muscle, some predominantly affect skeletal muscle, and some affect both. They suspect that these differences may be related to the location of mutations in the TTN gene and how they affect the many versions of titin that are produced in different muscles. More About This Health Condition  Genetics Home Reference provides information about arrhythmogenic right ventricular cardiomyopathy. More About This Health Condition  Genetics Home Reference provides information about familial hypertrophic cardiomyopathy. More About This Health Condition    Cytogenetic Location: 2q31.2, which is the long (q) arm of chromosome 2 at position 31.2Molecular Location: base pairs 178,525,989 to 178,807,423 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMH9CMPD4CONNECTINEOMFCLGMD2JTITIN_HUMANTMD     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Organization of Accessory Proteins in a Sarcomere (Figure)Molecular Cell Biology (fourth edition, 2000): Titin and Nebulin Filaments Organize the Sarcomere      Clinical Information from GeneReviews (5 links)     Dilated Cardiomyopathy OverviewHereditary Myopathy with Early Respiratory Failure (HMERF)Hypertrophic Cardiomyopathy OverviewSalih MyopathyUdd Distal Myopathy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TTN      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TITIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Organization of Accessory Proteins in a Sarcomere (Figure)Molecular Cell Biology (fourth edition, 2000): Titin and Nebulin Filaments Organize the Sarcomere  Dilated Cardiomyopathy OverviewHereditary Myopathy with Early Respiratory Failure (HMERF)Hypertrophic Cardiomyopathy OverviewSalih MyopathyUdd Distal Myopathy  Tests of TTN  PubMed  TITIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Carmignac V, Salih MA, Quijano-Roy S, Marchand S, Al Rayess MM, Mukhtar MM, Urtizberea JA, Labeit S, Guicheney P, Leturcq F, Gautel M, Fardeau M, Campbell KP, Richard I, Estournet B, Ferreiro A. C-terminal titin deletions cause a novel early-onset myopathy with fatal cardiomyopathy. Ann Neurol. 2007 Apr;61(4):340-51. Erratum in: Ann Neurol. 2012 May;71(5):728. Citation on PubMedChauveau C, Rowell J, Ferreiro A. A rising titan: TTN review and mutation update. Hum Mutat. 2014 Sep;35(9):1046-59. doi: 10.1002/humu.22611. Epub 2014 Jul 21. Review. Citation on PubMedFukuzawa A, Lange S, Holt M, Vihola A, Carmignac V, Ferreiro A, Udd B, Gautel M. Interactions with titin and myomesin target obscurin and obscurin-like 1 to the M-band: implications for hereditary myopathies. J Cell Sci. 2008 Jun 1;121(11):1841-51. doi: 10.1242/jcs.028019. Epub 2008 May 13. Citation on PubMedGerull B, Gramlich M, Atherton J, McNabb M, Trombits K, Sasse-Klaassen S, Seidman JG, Seidman C, Granzier H, Labeit S, Frenneaux M, Thierfelder L. Mutations of TTN, encoding the giant muscle filament titin, cause familial dilated cardiomyopathy. Nat Genet. 2002 Feb;30(2):201-4. Epub 2002 Jan 14. Citation on PubMedGerull B. The Rapidly Evolving Role of Titin in Cardiac Physiology and Cardiomyopathy. Can J Cardiol. 2015 Nov;31(11):1351-9. doi: 10.1016/j.cjca.2015.08.016. Epub 2015 Aug 28. Review. Citation on PubMedHackman JP, Vihola AK, Udd AB. The role of titin in muscular disorders. Ann Med. 2003;35(6):434-41. Citation on PubMedHackman P, Marchand S, Sarparanta J, Vihola A, Pnisson-Besnier I, Eymard B, Pardal-Fernndez JM, Hammouda el-H, Richard I, Illa I, Udd B. Truncating mutations in C-terminal titin may cause more severe tibial muscular dystrophy (TMD). Neuromuscul Disord. 2008 Dec;18(12):922-8. doi: 10.1016/j.nmd.2008.07.010. Epub 2008 Oct 22. Citation on PubMedHackman P, Vihola A, Haravuori H, Marchand S, Sarparanta J, De Seze J, Labeit S, Witt C, Peltonen L, Richard I, Udd B. Tibial muscular dystrophy is a titinopathy caused by mutations in TTN, the gene encoding the giant skeletal-muscle protein titin. Am J Hum Genet. 2002 Sep;71(3):492-500. Epub 2002 Jul 26. Citation on PubMed or Free article on PubMed CentralHerman DS, Lam L, Taylor MR, Wang L, Teekakirikul P, Christodoulou D, Conner L, DePalma SR, McDonough B, Sparks E, Teodorescu DL, Cirino AL, Banner NR, Pennell DJ, Graw S, Merlo M, Di Lenarda A, Sinagra G, Bos JM, Ackerman MJ, Mitchell RN, Murry CE, Lakdawala NK, Ho CY, Barton PJ, Cook SA, Mestroni L, Seidman JG, Seidman CE. Truncations of titin causing dilated cardiomyopathy. N Engl J Med. 2012 Feb 16;366(7):619-28. doi: 10.1056/NEJMoa1110186. Citation on PubMed or Free article on PubMed CentralLange S, Xiang F, Yakovenko A, Vihola A, Hackman P, Rostkova E, Kristensen J, Brandmeier B, Franzen G, Hedberg B, Gunnarsson LG, Hughes SM, Marchand S, Sejersen T, Richard I, Edstrm L, Ehler E, Udd B, Gautel M. The kinase domain of titin controls muscle gene expression and protein turnover. Science. 2005 Jun 10;308(5728):1599-603. Epub 2005 Mar 31. Citation on PubMedLinke WA. Sense and stretchability: the role of titin and titin-associated proteins in myocardial stress-sensing and mechanical dysfunction. Cardiovasc Res. 2008 Mar 1;77(4):637-48. Review. Citation on PubMedNorton N, Li D, Rampersaud E, Morales A, Martin ER, Zuchner S, Guo S, Gonzalez M, Hedges DJ, Robertson PD, Krumm N, Nickerson DA, Hershberger RE; National Heart, Lung, and Blood Institute GO Exome Sequencing Project and the Exome Sequencing Project Family Studies Project Team. Exome sequencing and genome-wide linkage analysis in 17 families illustrate the complex contribution of TTN truncating variants to dilated cardiomyopathy. Circ Cardiovasc Genet. 2013 Apr;6(2):144-53. doi: 10.1161/CIRCGENETICS.111.000062. Epub 2013 Feb 15. Citation on PubMed or Free article on PubMed CentralPalmio J, Evil A, Chapon F, Tasca G, Xiang F, Brdvik B, Eymard B, Echaniz-Laguna A, Laporte J, Krpp M, Mahjneh I, Quinlivan R, Lafort P, Damian M, Berardo A, Taratuto AL, Bueri JA, Tommiska J, Raivio T, Tuerk M, Glitz P, Chevessier F, Sewry C, Norwood F, Hedberg C, Schrder R, Edstrm L, Oldfors A, Hackman P, Udd B. Hereditary myopathy with early respiratory failure: occurrence in various populations. J Neurol Neurosurg Psychiatry. 2014 Mar;85(3):345-53. doi: 10.1136/jnnp-2013-304965. Epub 2013 Apr 19. Citation on PubMedPnisson-Besnier I, Hackman P, Suominen T, Sarparanta J, Huovinen S, Richard-Crmieux I, Udd B. Myopathies caused by homozygous titin mutations: limb-girdle muscular dystrophy 2J and variations of phenotype. J Neurol Neurosurg Psychiatry. 2010 Nov;81(11):1200-2. doi: 10.1136/jnnp.2009.178434. Epub 2010 Jun 22. Citation on PubMedSavarese M, Sarparanta J, Vihola A, Udd B, Hackman P. Increasing Role of Titin Mutations in Neuromuscular Disorders. J Neuromuscul Dis. 2016 Aug 30;3(3):293-308. Citation on PubMed or Free article on PubMed CentralUdd B, Vihola A, Sarparanta J, Richard I, Hackman P. Titinopathies and extension of the M-line mutation phenotype beyond distal myopathy and LGMD2J. Neurology. 2005 Feb 22;64(4):636-42. Citation on PubMedUdd B. Distal myopathies. Handb Clin Neurol. 2007;86:215-41. doi: 10.1016/S0072-9752(07)86011-8. Citation on PubMed \n",
            "FRZB\t   From NCBI Gene:The protein encoded by this gene is a secreted protein that is involved in the regulation of bone development. Defects in this gene are a cause of female-specific osteoarthritis (OA) susceptibility. [provided by RefSeq, Apr 2010]From UniProt:Soluble frizzled-related proteins (sFRPS) function as modulators of Wnt signaling through direct interaction with Wnts. They have a role in regulating cell growth and differentiation in specific cell types. SFRP3/FRZB appears to be involved in limb skeletogenesis. Antagonist of Wnt8 signaling. Regulates chondrocyte maturation and long bone development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:OsteoarthritisFrom UniProt:Osteoarthritis 1 (OS1): A degenerative disease of the joints characterized by degradation of the hyaline articular cartilage and remodeling of the subchondral bone with sclerosis. Clinical symptoms include pain and joint stiffness often leading to significant disability and joint replacement. [MIM:165720]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q32.1, which is the long (q) arm of chromosome 2 at position 32.1Molecular Location: base pairs 182,833,275 to 182,866,770 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FREFRITZFRP-3FRZB-1FRZB-PENFRZB1FZRBhFIZOS1SFRP3SRFP3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FRZB      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FRIZZLED-RELATED PROTEINOMIM: OSTEOARTHRITIS SUSCEPTIBILITY 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FRZB  OMIM: FRIZZLED-RELATED PROTEINOMIM: OSTEOARTHRITIS SUSCEPTIBILITY 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF804A\t\n",
            "FAM171B\t\n",
            "NT5C1B-RDH14\t\n",
            "NT5C1B\t\n",
            "PMS1\t   From NCBI Gene:This gene encodes a protein belonging to the DNA mismatch repair mutL/hexB family. This protein is thought to be involved in the repair of DNA mismatches, and it can form heterodimers with MLH1, a known DNA mismatch repair protein. Mutations in this gene cause hereditary nonpolyposis colorectal cancer type 3 (HNPCC3) either alone or in combination with mutations in other genes involved in the HNPCC phenotype, which is also known as Lynch syndrome. [provided by RefSeq, Jul 2008]From UniProt:Probably involved in the repair of mismatches in DNA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Lynch syndrome I     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q32.2, which is the long (q) arm of chromosome 2 at position 32.2Molecular Location: base pairs 189,784,085 to 189,877,629 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HNPCC3hPMS1MLH2PMSL1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PMS1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LYNCH SYNDROME IOMIM: POSTMEIOTIC SEGREGATION INCREASED, S. CEREVISIAE, 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PMS1  OMIM: LYNCH SYNDROME IOMIM: POSTMEIOTIC SEGREGATION INCREASED, S. CEREVISIAE, 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HIBCH\t   From NCBI Gene:This gene encodes the enzyme responsible for hydrolysis of both HIBYL-CoA and beta-hydroxypropionyl-CoA. Mutations in this gene have been associated with 3-hyroxyisobutyryl-CoA hydrolase deficiency. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010]From UniProt:Hydrolyzes 3-hydroxyisobutyryl-CoA (HIBYL-CoA), a saline catabolite. Has high activity toward isobutyryl-CoA. Could be an isobutyryl-CoA dehydrogenase that functions in valine catabolism. Also hydrolyzes 3-hydroxypropanoyl-CoA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Leigh syndromeFrom NCBI Gene:Beta-hydroxyisobutyryl-CoA deacylase deficiencyFrom UniProt:3-hydroxyisobutryl-CoA hydrolase deficiency (HIBCHD): An autosomal recessive inborn error of valine metabolism. It causes severely delayed psychomotor development, neurodegeneration, increased lactic acid, and brain lesions in the basal ganglia. [MIM:250620]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q32.2, which is the long (q) arm of chromosome 2 at position 32.2Molecular Location: base pairs 190,204,634 to 190,320,045 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HIBYLCOAH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HIBCH      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: 3-HYDROXYISOBUTYRYL-CoA HYDROLASEOMIM: 3-HYDROXYISOBUTYRYL-CoA HYDROLASE DEFICIENCY      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HIBCH  OMIM: 3-HYDROXYISOBUTYRYL-CoA HYDROLASEOMIM: 3-HYDROXYISOBUTYRYL-CoA HYDROLASE DEFICIENCY  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CCDC150\t\n",
            "ANKRD44\t\n",
            "CFLAR\t\n",
            "DYTN\t\n",
            "GDF7\t\n",
            "PIKFYVE\t   From NCBI Gene:Phosphorylated derivatives of phosphatidylinositol (PtdIns) regulate cytoskeletal functions, membrane trafficking, and receptor signaling by recruiting protein complexes to cell- and endosomal-membranes. Humans have multiple PtdIns proteins that differ by the degree and position of phosphorylation of the inositol ring. This gene encodes an enzyme (PIKfyve; also known as phosphatidylinositol-3-phosphate 5-kinase type III or PIPKIII) that phosphorylates the D-5 position in PtdIns and phosphatidylinositol-3-phosphate (PtdIns3P) to make PtdIns5P and PtdIns(3,5)biphosphate. The D-5 position also can be phosphorylated by type I PtdIns4P-5-kinases (PIP5Ks) that are encoded by distinct genes and preferentially phosphorylate D-4 phosphorylated PtdIns. In contrast, PIKfyve preferentially phosphorylates D-3 phosphorylated PtdIns. In addition to being a lipid kinase, PIKfyve also has protein kinase activity. PIKfyve regulates endomembrane homeostasis and plays a role in the biogenesis of endosome carrier vesicles from early endosomes. Mutations in this gene cause corneal fleck dystrophy (CFD); an autosomal dominant disorder characterized by numerous small white flecks present in all layers of the corneal stroma. Histologically, these flecks appear to be keratocytes distended with lipid and mucopolysaccharide filled intracytoplasmic vacuoles. Alternative splicing results in multiple transcript variants encoding distinct isoforms.[provided by RefSeq, May 2010]From UniProt:The PI(3,5)P2 regulatory complex regulates both the synthesis and turnover of phosphatidylinositol 3,5-bisphosphate (PtdIns(3,5)P2). Catalyzes the phosphorylation of phosphatidylinositol 3-phosphate on the fifth hydroxyl of the myo-inositol ring, to form phosphatidylinositol 3,5-bisphosphate. Required for endocytic-vacuolar pathway and nuclear migration. Plays a role in the biogenesis of endosome carrier vesicles (ECV)/ multivesicular bodies (MVB) transport intermediates from early endosomes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Fleck corneal dystrophyFrom UniProt:Corneal dystrophy, fleck (CFD): A form of stromal corneal dystrophy characterized by numerous small white flecks scattered in all levels of the stroma, with configurations varying from semicircular to wreath-like, curvilinear, or punctate. Although CFD may occasionally cause mild photophobia, patients are typically asymptomatic and have normal vision. [MIM:121850]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q34, which is the long (q) arm of chromosome 2 at position 34Molecular Location: base pairs 208,266,178 to 208,358,751 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CFDFAB1HEL37PIP5KPIP5K3ZFYVE29     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PIKFYVE      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CORNEAL DYSTROPHY, FLECKOMIM: PHOSPHOINOSITIDE KINASE, FYVE FINGER-CONTAINING      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PIKFYVE  OMIM: CORNEAL DYSTROPHY, FLECKOMIM: PHOSPHOINOSITIDE KINASE, FYVE FINGER-CONTAINING  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYL1\t\n",
            "CPS1\t   The CPS1 gene provides instructions for making the enzyme carbamoyl phosphate synthetase I. This enzyme participates in the urea cycle, a series of reactions that occurs in liver cells. The urea cycle processes excess nitrogen, generated when protein is used by the body, into a compound called urea that is excreted by the kidneys. Excreting the excess nitrogen prevents it from accumulating in the form of ammonia, which is toxic.The specific role of carbamoyl phosphate synthetase I is to control the first step of the urea cycle, a reaction in which excess nitrogen compounds are incorporated into the cycle to be processed.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Carbamoyl phosphate synthetase I deficiency     Approximately 10 mutations that cause carbamoyl phosphate synthetase I deficiency have been identified in the CPS1 gene. A mutated CPS1 gene may result in a carbamoyl phosphate synthetase I enzyme that is shorter than normal or the wrong shape, or may prevent the enzyme from being produced at all.The shape of an enzyme affects its ability to control a chemical reaction. If the carbamoyl phosphate synthetase enzyme is misshapen or missing, it cannot fulfill its role in the urea cycle. Excess nitrogen is not converted to urea for excretion, and ammonia accumulates in the body. Ammonia is toxic, especially to the nervous system, so this accumulation causes neurological problems and other signs and symptoms of carbamoyl phosphate synthetase I deficiency. More About This Health Condition      Other disorders     One common alteration (polymorphism) in the CPS1 gene has been associated with increased risk of circulatory problems in newborns and in individuals who have received bone marrow transplants. This genetic change results in the amino acid (protein building block) asparagine being substituted for the amino acid threonine at position 1405 in the protein sequence (written as Thr1405Asn or T1405N).Researchers believe that this polymorphism in the CPS1 gene may reduce the production of a compound called nitric oxide (NO). Normally, nitric oxide causes blood vessels to expand (dilate), which reduces blood pressure. A reduced amount of nitric oxide could lead to circulatory problems.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Approximately 10 mutations that cause carbamoyl phosphate synthetase I deficiency have been identified in the CPS1 gene. A mutated CPS1 gene may result in a carbamoyl phosphate synthetase I enzyme that is shorter than normal or the wrong shape, or may prevent the enzyme from being produced at all.The shape of an enzyme affects its ability to control a chemical reaction. If the carbamoyl phosphate synthetase enzyme is misshapen or missing, it cannot fulfill its role in the urea cycle. Excess nitrogen is not converted to urea for excretion, and ammonia accumulates in the body. Ammonia is toxic, especially to the nervous system, so this accumulation causes neurological problems and other signs and symptoms of carbamoyl phosphate synthetase I deficiency. More About This Health Condition  One common alteration (polymorphism) in the CPS1 gene has been associated with increased risk of circulatory problems in newborns and in individuals who have received bone marrow transplants. This genetic change results in the amino acid (protein building block) asparagine being substituted for the amino acid threonine at position 1405 in the protein sequence (written as Thr1405Asn or T1405N).Researchers believe that this polymorphism in the CPS1 gene may reduce the production of a compound called nitric oxide (NO). Normally, nitric oxide causes blood vessels to expand (dilate), which reduces blood pressure. A reduced amount of nitric oxide could lead to circulatory problems.    Cytogenetic Location: 2q34, which is the long (q) arm of chromosome 2 at position 34Molecular Location: base pairs 210,477,682 to 210,679,107 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        carbamoyl-phosphate synthase 1, mitochondrialcarbamoyl phosphate synthase Icarbamoyl-phosphate synthase, mitochondrial precursorcarbamoylphosphate synthetase ICPSase ICPSM_HUMAN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CPS1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CARBAMOYL PHOSPHATE SYNTHETASE I      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CPS1  PubMed  CARBAMOYL PHOSPHATE SYNTHETASE I  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Aoshima T, Kajita M, Sekido Y, Mimura S, Itakura A, Yasuda I, Saheki T, Watanabe K, Shimokata K, Niwa T. Carbamoyl phosphate synthetase I deficiency: molecular genetic findings and prenatal diagnosis. Prenat Diagn. 2001 Aug;21(8):634-7. Citation on PubMedOMIM: CARBAMOYL PHOSPHATE SYNTHETASE IFinckh U, Kohlschtter A, Schfer H, Sperhake K, Colombo JP, Gal A. Prenatal diagnosis of carbamoyl phosphate synthetase I deficiency by identification of a missense mutation in CPS1. Hum Mutat. 1998;12(3):206-11. Citation on PubMedFunghini S, Donati MA, Pasquini E, Zammarchi E, Morrone A. Structural organization of the human carbamyl phosphate synthetase I gene (CPS1) and identification of two novel genetic lesions. Hum Mutat. 2003 Oct;22(4):340-1. Citation on PubMedHberle J, Schmidt E, Pauli S, Rapp B, Christensen E, Wermuth B, Koch HG. Gene structure of human carbamylphosphate synthetase 1 and novel mutations in patients with neonatal onset. Hum Mutat. 2003 Apr;21(4):444. Citation on PubMedPearson DL, Dawling S, Walsh WF, Haines JL, Christman BW, Bazyk A, Scott N, Summar ML. Neonatal pulmonary hypertension--urea-cycle intermediates, nitric oxide production, and carbamoyl-phosphate synthetase function. N Engl J Med. 2001 Jun 14;344(24):1832-8. Citation on PubMedRapp B, Hberle J, Linnebank M, Wermuth B, Marquardt T, Harms E, Koch HG. Genetic analysis of carbamoylphosphate synthetase I and ornithine transcarbamylase deficiency using fibroblasts. Eur J Pediatr. 2001 May;160(5):283-7. Citation on PubMedSummar ML, Gainer JV, Pretorius M, Malave H, Harris S, Hall LD, Weisberg A, Vaughan DE, Christman BW, Brown NJ. Relationship between carbamoyl-phosphate synthetase genotype and systemic vascular function. Hypertension. 2004 Feb;43(2):186-91. Epub 2004 Jan 12. Citation on PubMedSummar ML, Hall LD, Eeds AM, Hutcheson HB, Kuo AN, Willis AS, Rubio V, Arvin MK, Schofield JP, Dawson EP. Characterization of genomic structure and polymorphisms in the human carbamyl phosphate synthetase I gene. Gene. 2003 Jun 5;311:51-7. Citation on PubMedWakutani Y, Nakayasu H, Takeshima T, Adachi M, Kawataki M, Kihira K, Sawada H, Bonno M, Yamamoto H, Nakashima K. Mutational analysis of carbamoylphosphate synthetase I deficiency in three Japanese patients. J Inherit Metab Dis. 2004;27(6):787-8. Citation on PubMed \n",
            "ERBB4\t   From NCBI Gene:This gene is a member of the Tyr protein kinase family and the epidermal growth factor receptor subfamily. It encodes a single-pass type I membrane protein with multiple cysteine rich domains, a transmembrane domain, a tyrosine kinase domain, a phosphotidylinositol-3 kinase binding site and a PDZ domain binding motif. The protein binds to and is activated by neuregulins and other factors and induces a variety of cellular responses including mitogenesis and differentiation. Multiple proteolytic events allow for the release of a cytoplasmic fragment and an extracellular fragment. Mutations in this gene have been associated with cancer. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized. [provided by RefSeq, Jul 2008]From UniProt:Tyrosine-protein kinase that plays an essential role as cell surface receptor for neuregulins and EGF family members and regulates development of the heart, the central nervous system and the mammary gland, gene transcription, cell proliferation, differentiation, migration and apoptosis. Required for normal cardiac muscle differentiation during embryonic development, and for postnatal cardiomyocyte proliferation. Required for normal development of the embryonic central nervous system, especially for normal neural crest cell migration and normal axon guidance. Required for mammary gland differentiation, induction of milk proteins and lactation. Acts as cell-surface receptor for the neuregulins NRG1, NRG2, NRG3 and NRG4 and the EGF family members BTC, EREG and HBEGF. Ligand binding triggers receptor dimerization and autophosphorylation at specific tyrosine residues that then serve as binding sites for scaffold proteins and effectors. Ligand specificity and signaling is modulated by alternative splicing, proteolytic processing, and by the formation of heterodimers with other ERBB family members, thereby creating multiple combinations of intracellular phosphotyrosines that trigger ligand- and context-specific cellular responses. Mediates phosphorylation of SHC1 and activation of the MAP kinases MAPK1/ERK2 and MAPK3/ERK1. Isoform JM-A CYT-1 and isoform JM-B CYT-1 phosphorylate PIK3R1, leading to the activation of phosphatidylinositol 3-kinase and AKT1 and protect cells against apoptosis. Isoform JM-A CYT-1 and isoform JM-B CYT-1 mediate reorganization of the actin cytoskeleton and promote cell migration in response to NRG1. Isoform JM-A CYT-2 and isoform JM-B CYT-2 lack the phosphotyrosine that mediates interaction with PIK3R1, and hence do not phosphorylate PIK3R1, do not protect cells against apoptosis, and do not promote reorganization of the actin cytoskeleton and cell migration. Proteolytic processing of isoform JM-A CYT-1 and isoform JM-A CYT-2 gives rise to the corresponding soluble intracellular domains (4ICD) that translocate to the nucleus, promote nuclear import of STAT5A, activation of STAT5A, mammary epithelium differentiation, cell proliferation and activation of gene expression. The ERBB4 soluble intracellular domains (4ICD) colocalize with STAT5A at the CSN2 promoter to regulate transcription of milk proteins during lactation. The ERBB4 soluble intracellular domains can also translocate to mitochondria and promote apoptosis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Amyotrophic lateral sclerosisPolycystic ovary syndromeFrom NCBI Gene:Amyotrophic lateral sclerosis 19From UniProt:Amyotrophic lateral sclerosis 19 (ALS19): A neurodegenerative disorder affecting upper motor neurons in the brain and lower motor neurons in the brain stem and spinal cord, resulting in fatal paralysis. Sensory abnormalities are absent. The pathologic hallmarks of the disease include pallor of the corticospinal tract due to loss of motor neurons, presence of ubiquitin-positive inclusions within surviving motor neurons, and deposition of pathologic aggregates. The etiology of amyotrophic lateral sclerosis is likely to be multifactorial, involving both genetic and environmental factors. The disease is inherited in 5-10% of the cases. [MIM:615515]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q34, which is the long (q) arm of chromosome 2 at position 34Molecular Location: base pairs 211,375,717 to 212,538,628 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ALS19HER4p180erbB4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ERBB4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: AMYOTROPHIC LATERAL SCLEROSIS 19OMIM: V-ERB-B2 AVIAN ERYTHROBLASTIC LEUKEMIA VIRAL ONCOGENE HOMOLOG 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ERBB4  OMIM: AMYOTROPHIC LATERAL SCLEROSIS 19OMIM: V-ERB-B2 AVIAN ERYTHROBLASTIC LEUKEMIA VIRAL ONCOGENE HOMOLOG 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "APOB\t   The APOB gene provides instructions for making two versions of the apolipoprotein B protein, a short version called apolipoprotein B-48 and a longer version known as apolipoprotein B-100. Both of these proteins are components of lipoproteins, which are particles that carry fats and fat-like substances (such as cholesterol) in the blood.Apolipoprotein B-48 is produced in the intestine, where it is a building block of a type of lipoprotein called a chylomicron. As food is digested after a meal, chylomicrons are formed to carry fat and cholesterol from the intestine into the bloodstream.  Chylomicrons are also necessary for the absorption of certain fat-soluble vitamins such as vitamin E and vitamin A.Apolipoprotein B-100, which is produced in the liver, is a component of several other types of lipoproteins. Specifically, this protein is a building block of very low-density lipoproteins (VLDLs), intermediate-density lipoproteins (IDLs), and low-density lipoproteins (LDLs). These related molecules all transport fats and cholesterol in the bloodstream.Low-density lipoproteins are the primary carriers of cholesterol in the blood.  Apolipoprotein B-100 allows these particles to attach to specific receptors on the surface of cells, particularly in the liver. The receptors transport low-density lipoproteins into the cell, where they are broken down to release cholesterol. The cholesterol is then used by the cell, stored, or removed from the body.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Familial hypobetalipoproteinemia     More than 90 mutations in the APOB gene have been found to cause familial hypobetalipoproteinemia (FHBL), a disorder that impairs the body's ability to absorb and transport fat. Most APOB gene mutations that cause FHBL lead to the production of apolipoprotein B that is abnormally short.The severity of the condition largely depends on the length of the abnormal apolipoprotein B. Some mutations in the APOB gene lead to the production of a protein that is shorter than apolipoprotein B-100, but longer than apolipoprotein B-48. In these cases, normal apolipoprotein B-48 is still made in the intestine. The normal-length apolipoprotein B-48 can form chylomicrons normally, but the abnormally short apolipoprotein B-100 produced in the liver is less able to produce lipoproteins. Other mutations result in a protein that is shorter than both apolipoprotein B-48 and apolipoprotein B-100. In these cases, no normal-length apolipoprotein B protein is produced. The severely shortened protein is not able to form lipoproteins in the liver or the intestine. Generally, if both versions of the protein are shorter than apolipoprotein B-48, the signs and symptoms are more severe than if some normal length apolipoprotein B-48 is produced. All of these protein changes lead to a reduction of functional apolipoprotein B. As a result, the transportation of dietary fats and cholesterol is decreased or absent. A decrease in fat transport reduces the body's ability to absorb fats and fat-soluble vitamins from the diet, leading to the signs and symptoms of FHBL. More About This Health Condition      Hypercholesterolemia     At least five mutations in the APOB gene are known to cause a form of inherited hypercholesterolemia called familial defective apolipoprotein B-100 (FDB). This condition is characterized by very high levels of cholesterol in the blood and an increased risk of developing heart disease. Each mutation that causes this condition changes a single protein building block (amino acid) in a critical region of apolipoprotein B-100. The altered protein prevents low-density lipoproteins from effectively binding to their receptors on the surface of cells. As a result, fewer low-density lipoproteins are removed from the blood, and cholesterol levels are much higher than normal.  As the excess cholesterol circulates through the bloodstream, it is deposited abnormally in tissues such as the skin, tendons, and arteries that supply blood to the heart (coronary arteries).  A buildup of cholesterol in the walls of coronary arteries greatly increases a person's risk of having a heart attack. More About This Health Condition      Other disorders     Researchers are studying other variations (polymorphisms) in the APOB gene that may influence heart disease risk in people without inherited cholesterol disorders.  Some studies have found that certain polymorphisms are associated with higher levels of low-density lipoproteins in the blood and an increased chance of developing or dying of heart disease. Other studies, however, have not shown such an association.  It is clear that a large number of genetic and lifestyle factors, many of which remain unknown, determine the risk of developing this complex condition.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 90 mutations in the APOB gene have been found to cause familial hypobetalipoproteinemia (FHBL), a disorder that impairs the body's ability to absorb and transport fat. Most APOB gene mutations that cause FHBL lead to the production of apolipoprotein B that is abnormally short.The severity of the condition largely depends on the length of the abnormal apolipoprotein B. Some mutations in the APOB gene lead to the production of a protein that is shorter than apolipoprotein B-100, but longer than apolipoprotein B-48. In these cases, normal apolipoprotein B-48 is still made in the intestine. The normal-length apolipoprotein B-48 can form chylomicrons normally, but the abnormally short apolipoprotein B-100 produced in the liver is less able to produce lipoproteins. Other mutations result in a protein that is shorter than both apolipoprotein B-48 and apolipoprotein B-100. In these cases, no normal-length apolipoprotein B protein is produced. The severely shortened protein is not able to form lipoproteins in the liver or the intestine. Generally, if both versions of the protein are shorter than apolipoprotein B-48, the signs and symptoms are more severe than if some normal length apolipoprotein B-48 is produced. All of these protein changes lead to a reduction of functional apolipoprotein B. As a result, the transportation of dietary fats and cholesterol is decreased or absent. A decrease in fat transport reduces the body's ability to absorb fats and fat-soluble vitamins from the diet, leading to the signs and symptoms of FHBL. More About This Health Condition  At least five mutations in the APOB gene are known to cause a form of inherited hypercholesterolemia called familial defective apolipoprotein B-100 (FDB). This condition is characterized by very high levels of cholesterol in the blood and an increased risk of developing heart disease. Each mutation that causes this condition changes a single protein building block (amino acid) in a critical region of apolipoprotein B-100. The altered protein prevents low-density lipoproteins from effectively binding to their receptors on the surface of cells. As a result, fewer low-density lipoproteins are removed from the blood, and cholesterol levels are much higher than normal.  As the excess cholesterol circulates through the bloodstream, it is deposited abnormally in tissues such as the skin, tendons, and arteries that supply blood to the heart (coronary arteries).  A buildup of cholesterol in the walls of coronary arteries greatly increases a person's risk of having a heart attack. More About This Health Condition  Researchers are studying other variations (polymorphisms) in the APOB gene that may influence heart disease risk in people without inherited cholesterol disorders.  Some studies have found that certain polymorphisms are associated with higher levels of low-density lipoproteins in the blood and an increased chance of developing or dying of heart disease. Other studies, however, have not shown such an association.  It is clear that a large number of genetic and lifestyle factors, many of which remain unknown, determine the risk of developing this complex condition.    Cytogenetic Location: 2p24.1, which is the short (p) arm of chromosome 2 at position 24.1Molecular Location: base pairs 21,001,429 to 21,044,073 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        apoB-48apoB-100APOB_HUMANapolipoprotein B (including Ag(x) antigen)     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Lipoproteins Transport Cholesterol and Triacylglycerols Throughout the Organism      Clinical Information from GeneReviews (1 link)     Familial Hypercholesterolemia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of APOB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     APOLIPOPROTEIN B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Lipoproteins Transport Cholesterol and Triacylglycerols Throughout the Organism  Familial Hypercholesterolemia  Tests of APOB  PubMed  APOLIPOPROTEIN B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Benn M, Nordestgaard BG, Jensen JS, Grande P, Sillesen H, Tybjaerg-Hansen A. Polymorphism in APOB associated with increased low-density lipoprotein levels in both genders in the general population. J Clin Endocrinol Metab. 2005 Oct;90(10):5797-803. Epub 2005 Jul 19. Citation on PubMedFouchier SW, Sankatsing RR, Peter J, Castillo S, Pocovi M, Alonso R, Kastelein JJ, Defesche JC. High frequency of APOB gene mutations causing familial hypobetalipoproteinaemia in patients of Dutch and Spanish descent. J Med Genet. 2005 Apr;42(4):e23. Citation on PubMed or Free article on PubMed CentralHooper AJ, van Bockxmeer FM, Burnett JR. Monogenic hypocholesterolaemic lipid disorders and apolipoprotein B metabolism. Crit Rev Clin Lab Sci. 2005;42(5-6):515-45. Review. Citation on PubMedOlofsson SO, Born J. Apolipoprotein B: a clinically important apolipoprotein which assembles atherogenic lipoproteins and promotes the development of atherosclerosis. J Intern Med. 2005 Nov;258(5):395-410. Review. Citation on PubMedSchonfeld G, Lin X, Yue P. Familial hypobetalipoproteinemia: genetics and metabolism. Cell Mol Life Sci. 2005 Jun;62(12):1372-8. Review. Citation on PubMedSoufi M, Sattler AM, Maerz W, Starke A, Herzum M, Maisch B, Schaefer JR. A new but frequent mutation of apoB-100-apoB His3543Tyr. Atherosclerosis. 2004 May;174(1):11-6. Citation on PubMedTarugi P, Averna M, Di Leo E, Cefal AB, Noto D, Magnolo L, Cattin L, Bertolini S, Calandra S. Molecular diagnosis of hypobetalipoproteinemia: an ENID review. Atherosclerosis. 2007 Dec;195(2):e19-27. Epub 2007 Jun 14. Review. Citation on PubMedTarugi P, Averna M. Hypobetalipoproteinemia: genetics, biochemistry, and clinical spectrum. Adv Clin Chem. 2011;54:81-107. Review. Citation on PubMedVrablk M, Ceska R, Hornek A. Major apolipoprotein B-100 mutations in lipoprotein metabolism and atherosclerosis. Physiol Res. 2001;50(4):337-43. Review. Citation on PubMedvan Aalst-Cohen ES, Jansen AC, de Jongh S, de Sauvage Nolting PR, Kastelein JJ. Clinical, diagnostic, and therapeutic aspects of familial hypercholesterolemia. Semin Vasc Med. 2004 Feb;4(1):31-41. Review. Citation on PubMed \n",
            "BARD1\t   From NCBI Gene:This gene encodes a protein which interacts with the N-terminal region of BRCA1. In addition to its ability to bind BRCA1 in vivo and in vitro, it shares homology with the 2 most conserved regions of BRCA1: the N-terminal RING motif and the C-terminal BRCT domain. The RING motif is a cysteine-rich sequence found in a variety of proteins that regulate cell growth, including the products of tumor suppressor genes and dominant protooncogenes. This protein also contains 3 tandem ankyrin repeats. The BARD1/BRCA1 interaction is disrupted by tumorigenic amino acid substitutions in BRCA1, implying that the formation of a stable complex between these proteins may be an essential aspect of BRCA1 tumor suppression. This protein may be the target of oncogenic mutations in breast or ovarian cancer. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2013]From UniProt:E3 ubiquitin-protein ligase. The BRCA1-BARD1 heterodimer specifically mediates the formation of 'Lys-6'-linked polyubiquitin chains and coordinates a diverse range of cellular pathways such as DNA damage repair, ubiquitination and transcriptional regulation to maintain genomic stability. Plays a central role in the control of the cell cycle in response to DNA damage. Acts by mediating ubiquitin E3 ligase activity that is required for its tumor suppressor function. Also forms a heterodimer with CSTF1/CSTF-50 to modulate mRNA processing and RNAP II stability by inhibiting pre-mRNA 3' cleavage.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:NeuroblastomaOvarian cancerBreast cancerFrom NCBI Gene:Familial cancer of breast     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q35, which is the long (q) arm of chromosome 2 at position 35Molecular Location: base pairs 214,725,645 to 214,809,711 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of BARD1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: BRCA1-ASSOCIATED RING DOMAIN 1OMIM: BREAST CANCER      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of BARD1  OMIM: BRCA1-ASSOCIATED RING DOMAIN 1OMIM: BREAST CANCER  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ABCA12\t   The ABCA12 gene provides instructions for making a protein known as an ATP-binding cassette (ABC) transporter.  ABC transporter proteins carry many types of molecules across cell membranes.  In particular, the ABCA12 protein plays a major role in transporting fats (lipids) in cells that make up the outermost layer of skin (the epidermis). This lipid transport is needed to maintain the layers of lipids within the epidermis that are necessary to prevent water loss (dehydration) and for normal development of the skin. The ABCA12 protein is also found in several other tissues, including the testes, placenta, lungs, stomach, fetal brain, and liver.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Harlequin ichthyosis     More than 65 mutations in the ABCA12 gene have been identified in people with harlequin ichthyosis. This skin condition is characterized by hard, thick scales that are present at birth; excessive dehydration; and increased risk of infections. Most of the mutations lead to an absence of ABCA12 protein or the production of an extremely small version of the protein that cannot transport lipids properly. A lack of lipid transport causes numerous problems with the development of the epidermis before and after birth. Specifically, it prevents the skin from forming an effective barrier against dehydration, and leads to the skin problems characteristic of harlequin ichthyosis. More About This Health Condition      Nonbullous congenital ichthyosiform erythroderma     At least 20 mutations in the ABCA12 gene have been found to cause nonbullous congenital ichthyosiform erythroderma (NBCIE). This condition affects the skin and causes redness; the development of fine, white scales; an increased risk of infections; and dehydration. These skin abnormalities tend to be less severe than those in harlequin ichthyosis (described above). Most of the mutations that cause NBCIE change single amino acids in the ABCA12 protein. These mutations likely lead to the production of a protein with reduced function, which impairs lipid transport and the formation of the lipid layers within the epidermis. Problems with this protective barrier underlie the skin abnormalities and other features of NBCIE. More About This Health Condition      Lamellar ichthyosis     Genetics Home Reference provides information about lamellar ichthyosis. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 65 mutations in the ABCA12 gene have been identified in people with harlequin ichthyosis. This skin condition is characterized by hard, thick scales that are present at birth; excessive dehydration; and increased risk of infections. Most of the mutations lead to an absence of ABCA12 protein or the production of an extremely small version of the protein that cannot transport lipids properly. A lack of lipid transport causes numerous problems with the development of the epidermis before and after birth. Specifically, it prevents the skin from forming an effective barrier against dehydration, and leads to the skin problems characteristic of harlequin ichthyosis. More About This Health Condition  At least 20 mutations in the ABCA12 gene have been found to cause nonbullous congenital ichthyosiform erythroderma (NBCIE). This condition affects the skin and causes redness; the development of fine, white scales; an increased risk of infections; and dehydration. These skin abnormalities tend to be less severe than those in harlequin ichthyosis (described above). Most of the mutations that cause NBCIE change single amino acids in the ABCA12 protein. These mutations likely lead to the production of a protein with reduced function, which impairs lipid transport and the formation of the lipid layers within the epidermis. Problems with this protective barrier underlie the skin abnormalities and other features of NBCIE. More About This Health Condition  Genetics Home Reference provides information about lamellar ichthyosis. More About This Health Condition    Cytogenetic Location: 2q35, which is the long (q) arm of chromosome 2 at position 35Molecular Location: base pairs 214,931,542 to 215,138,591 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABCAC_HUMANATP-binding cassette 12ATP-binding cassette transporter 12ATP-binding cassette, sub-family A (ABC1), member 12ATP-binding cassette, sub-family A, member 12ICR2B     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Human ATP-Binding Cassette (ABC) Transporter Superfamily: ABCA Genes      Clinical Information from GeneReviews (1 link)     Autosomal Recessive Congenital Ichthyosis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ABCA12      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATP-BINDING CASSETTE, SUBFAMILY A, MEMBER 12      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Human ATP-Binding Cassette (ABC) Transporter Superfamily: ABCA Genes  Autosomal Recessive Congenital Ichthyosis  Tests of ABCA12  PubMed  ATP-BINDING CASSETTE, SUBFAMILY A, MEMBER 12  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Akiyama M, Sugiyama-Nakagiri Y, Sakai K, McMillan JR, Goto M, Arita K, Tsuji-Abe Y, Tabata N, Matsuoka K, Sasaki R, Sawamura D, Shimizu H. Mutations in lipid transporter ABCA12 in harlequin ichthyosis and functional recovery by corrective gene transfer. J Clin Invest. 2005 Jul;115(7):1777-84. Citation on PubMed or Free article on PubMed CentralAkiyama M. Pathomechanisms of harlequin ichthyosis and ABCA transporters in human diseases. Arch Dermatol. 2006 Jul;142(7):914-8. Review. Citation on PubMedHovnanian A. Harlequin ichthyosis unmasked: a defect of lipid transport. J Clin Invest. 2005 Jul;115(7):1708-10. Review. Citation on PubMed or Free article on PubMed CentralKelsell DP, Norgett EE, Unsworth H, Teh MT, Cullup T, Mein CA, Dopping-Hepenstal PJ, Dale BA, Tadini G, Fleckman P, Stephens KG, Sybert VP, Mallory SB, North BV, Witt DR, Sprecher E, Taylor AE, Ilchyshyn A, Kennedy CT, Goodyear H, Moss C, Paige D, Harper JI, Young BD, Leigh IM, Eady RA, O'Toole EA. Mutations in ABCA12 underlie the severe congenital skin disease harlequin ichthyosis. Am J Hum Genet. 2005 May;76(5):794-803. Epub 2005 Mar 8. Citation on PubMed or Free article on PubMed CentralLefvre C, Audebert S, Jobard F, Bouadjar B, Lakhdar H, Boughdene-Stambouli O, Blanchet-Bardon C, Heilig R, Foglio M, Weissenbach J, Lathrop M, Prud'homme JF, Fischer J. Mutations in the transporter ABCA12 are associated with lamellar ichthyosis type 2. Hum Mol Genet. 2003 Sep 15;12(18):2369-78. Epub 2003 Jul 15. Citation on PubMedNawaz S, Tariq M, Ahmad I, Malik NA, Baig SM, Dahl N, Klar J. Non-bullous congentital ichthyosiform erythroderma associated with homozygosity for a novel missense mutation in an ATP binding domain of ABCA12. Eur J Dermatol. 2012 Mar-Apr;22(2):178-81. doi: 10.1684/ejd.2011.1638. Citation on PubMedSakai K, Akiyama M, Yanagi T, McMillan JR, Suzuki T, Tsukamoto K, Sugiyama H, Hatano Y, Hayashitani M, Takamori K, Nakashima K, Shimizu H. ABCA12 is a major causative gene for non-bullous congenital ichthyosiform erythroderma. J Invest Dermatol. 2009 Sep;129(9):2306-9. doi: 10.1038/jid.2009.23. Epub 2009 Mar 5. Citation on PubMedThomas AC, Cullup T, Norgett EE, Hill T, Barton S, Dale BA, Sprecher E, Sheridan E, Taylor AE, Wilroy RS, DeLozier C, Burrows N, Goodyear H, Fleckman P, Stephens KG, Mehta L, Watson RM, Graham R, Wolf R, Slavotinek A, Martin M, Bourn D, Mein CA, O'Toole EA, Kelsell DP. ABCA12 is the major harlequin ichthyosis gene. J Invest Dermatol. 2006 Nov;126(11):2408-13. Epub 2006 Aug 10. Citation on PubMedThomas AC, Sinclair C, Mahmud N, Cullup T, Mellerio JE, Harper J, Dale BA, Turc-Carel C, Hohl D, McGrath JA, Vahlquist A, Hellstrom-Pigg M, Ganemo A, Metcalfe K, Mein CA, O'Toole EA, Kelsell DP. Novel and recurring ABCA12 mutations associated with harlequin ichthyosis: implications for prenatal diagnosis. Br J Dermatol. 2008 Mar;158(3):611-3. Epub 2007 Nov 6. Citation on PubMedUitto J. The gene family of ABC transporters--novel mutations, new phenotypes. Trends Mol Med. 2005 Aug;11(8):341-3. Review. Citation on PubMed \n",
            "FN1\t   The FN1 gene provides instructions for making two types of the fibronectin-1 protein: soluble plasma fibronectin-1 and insoluble cellular fibronectin-1. Liver cells produce soluble plasma fibronectin-1 and release it into the bloodstream, where it is mainly involved in blood clotting and wound healing. Soluble plasma fibronectin-1 functions outside of cells (in the extracellular spaces), attaching (binding) to the surface of cells and binding to proteins, including other fibronectin-1 proteins. The attachment of these proteins form fibers that assist with tissue repair after an injury. Fibronectin-1 binding also helps with the continual formation of the extracellular matrix, which is an intricate lattice of proteins and other molecules that is made in the spaces between cells. This matrix provides structure and strength to tissues that support the body's organs. Many other cell types produce insoluble cellular fibronectin-1, which is released into the extracellular space and contributes to the creation of fibers and extracellular matrix. Both types of fibronectin-1 help individual cells expand (spread) and move (migrate) to cover more space and also influence cell shape and maturation (differentiation).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Fibronectin glomerulopathy     At least three mutations in the FN1 gene have been found to cause fibronectin glomerulopathy, a progressive kidney disease that usually begins in adulthood and results in irreversible kidney failure (end-stage renal disease). FN1 gene mutations account for about 40 percent of cases of fibronectin glomerulopathy. The FN1 gene mutations change single protein building blocks (amino acids) in the fibronectin-1 protein. One mutation that occurs in multiple families replaces the amino acid tyrosine with the amino acid cysteine at position 973 in the fibronectin-1 protein (written as Tyr973Cys or Y973C). FN1 gene mutations impair the protein's ability to bind to cells and proteins. The unbound fibronectin-1 protein, specifically soluble plasma fibronectin-1, is deposited in the glomeruli of the kidneys. These structures are clusters of tiny blood vessels in the kidneys that filter waste products from blood, which are then released in urine. Even though there is an abundance of fibronectin-1 in the glomeruli, the extracellular matrix that supports the blood vessels is weak because the altered fibronectin-1 cannot assist in the matrix's continual formation. Without a strong cellular support network, the glomeruli are less able to filter waste. As a result, products that normally are retained by the body, such as protein and blood, get released in the urine, and acids are not properly filtered from the blood. Over time, the kidneys' ability to filter waste decreases until the kidneys can no longer function, resulting in end-stage renal disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three mutations in the FN1 gene have been found to cause fibronectin glomerulopathy, a progressive kidney disease that usually begins in adulthood and results in irreversible kidney failure (end-stage renal disease). FN1 gene mutations account for about 40 percent of cases of fibronectin glomerulopathy. The FN1 gene mutations change single protein building blocks (amino acids) in the fibronectin-1 protein. One mutation that occurs in multiple families replaces the amino acid tyrosine with the amino acid cysteine at position 973 in the fibronectin-1 protein (written as Tyr973Cys or Y973C). FN1 gene mutations impair the protein's ability to bind to cells and proteins. The unbound fibronectin-1 protein, specifically soluble plasma fibronectin-1, is deposited in the glomeruli of the kidneys. These structures are clusters of tiny blood vessels in the kidneys that filter waste products from blood, which are then released in urine. Even though there is an abundance of fibronectin-1 in the glomeruli, the extracellular matrix that supports the blood vessels is weak because the altered fibronectin-1 cannot assist in the matrix's continual formation. Without a strong cellular support network, the glomeruli are less able to filter waste. As a result, products that normally are retained by the body, such as protein and blood, get released in the urine, and acids are not properly filtered from the blood. Over time, the kidneys' ability to filter waste decreases until the kidneys can no longer function, resulting in end-stage renal disease. More About This Health Condition    Cytogenetic Location: 2q35, which is the long (q) arm of chromosome 2 at position 35Molecular Location: base pairs 215,360,440 to 215,436,167 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CIGcold-insoluble globulinED-BfibronectinFINCFINC_HUMANFNFNZLETSmigration-stimulating factorMSF     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Madame Curie Bioscience Database: Fibronectin Matrix Assembly and Cross-Linking to FibrinMolecular Biology of the Cell (fourth edition, 2002): Fibronectin Is an Extracellular Protein That Helps Cells Attach to the MatrixMolecular Cell Biology (fourth edition, 2000): Fibronectins Bind Many Cells to Fibrous Collagens and Other Matrix Components      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FN1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FIBRONECTIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Fibronectin Matrix Assembly and Cross-Linking to FibrinMolecular Biology of the Cell (fourth edition, 2002): Fibronectin Is an Extracellular Protein That Helps Cells Attach to the MatrixMolecular Cell Biology (fourth edition, 2000): Fibronectins Bind Many Cells to Fibrous Collagens and Other Matrix Components  Tests of FN1  PubMed  FIBRONECTIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Castelletti F, Donadelli R, Banterla F, Hildebrandt F, Zipfel PF, Bresin E, Otto E, Skerka C, Renieri A, Todeschini M, Caprioli J, Caruso RM, Artuso R, Remuzzi G, Noris M. Mutations in FN1 cause glomerulopathy with fibronectin deposits. Proc Natl Acad Sci U S A. 2008 Feb 19;105(7):2538-43. doi: 10.1073/pnas.0707730105. Epub 2008 Feb 11. Citation on PubMed or Free article on PubMed CentralOMIM: FIBRONECTIN 1Satoskar AA, Shapiro JP, Bott CN, Song H, Nadasdy GM, Brodsky SV, Hebert LA, Birmingham DJ, Nadasdy T, Freitas MA, Rovin BH. Characterization of glomerular diseases using proteomic analysis of laser capture microdissected glomeruli. Mod Pathol. 2012 May;25(5):709-21. doi: 10.1038/modpathol.2011.205. Epub 2012 Jan 27. Citation on PubMed or Free article on PubMed CentralTo WS, Midwood KS. Plasma and cellular fibronectin: distinct and independent functions during tissue repair. Fibrogenesis Tissue Repair. 2011 Sep 16;4:21. doi: 10.1186/1755-1536-4-21. Citation on PubMed or Free article on PubMed Central \n",
            "XRCC5\t\n",
            "IGFBP2\t\n",
            "IGFBP5\t\n",
            "TNS1\t\n",
            "SLC11A1\t   From NCBI Gene:This gene is a member of the solute carrier family 11 (proton-coupled divalent metal ion transporters) family and encodes a multi-pass membrane protein. The protein functions as a divalent transition metal (iron and manganese) transporter involved in iron metabolism and host resistance to certain pathogens. Mutations in this gene have been associated with susceptibility to infectious diseases such as tuberculosis and leprosy, and inflammatory diseases such as rheumatoid arthritis and Crohn disease. Alternatively spliced variants that encode different protein isoforms have been described but the full-length nature of only one has been determined. [provided by RefSeq, Jul 2008]From UniProt:Divalent transition metal (iron and manganese) transporter involved in iron metabolism and host resistance to certain pathogens. Macrophage-specific membrane transport function. Controls natural resistance to infection with intracellular parasites. Pathogen resistance involves sequestration of Fe(2+) and Mn(2+), cofactors of both prokaryotic and eukaryotic catalases and superoxide dismutases, not only to protect the macrophage against its own generation of reactive oxygen species, but to deny the cations to the pathogen for synthesis of its protective enzymes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Juvenile idiopathic arthritisFrom NCBI Gene:Buruli ulcer, susceptibility toMycobacterium tuberculosis, susceptibility to     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q35, which is the long (q) arm of chromosome 2 at position 35Molecular Location: base pairs 218,381,766 to 218,396,894 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LSHNRAMPNRAMP1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC11A1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BURULI ULCER, SUSCEPTIBILITY TOOMIM: MYCOBACTERIUM TUBERCULOSIS, SUSCEPTIBILITY TOOMIM: SOLUTE CARRIER FAMILY 11 (PROTON-COUPLED DIVALENT METAL ION TRANSPORTER), MEMBER 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SLC11A1  OMIM: BURULI ULCER, SUSCEPTIBILITY TOOMIM: MYCOBACTERIUM TUBERCULOSIS, SUSCEPTIBILITY TOOMIM: SOLUTE CARRIER FAMILY 11 (PROTON-COUPLED DIVALENT METAL ION TRANSPORTER), MEMBER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PLCD4\t\n",
            "CFAP65\t\n",
            "OBSL1\t   The OBSL1 gene provides instructions for making a protein that is thought to help maintain normal levels of another protein called cullin-7, which is produced from the CUL7 gene. The cullin-7 protein plays a role in the ubiquitin-proteasome system, which is the cell machinery that breaks down (degrades) unwanted proteins.Cullin-7 helps assemble a complex known as an E3 ubiquitin ligase. This complex tags damaged and excess proteins with molecules called ubiquitin. Ubiquitin serves as a signal to specialized cell structures known as proteasomes, which attach (bind) to the tagged proteins and degrade them. The ubiquitin-proteasome system acts as the cell's quality control system by disposing of damaged, misshapen, and excess proteins. This system also regulates the level of proteins involved in several critical cell activities such as the timing of cell division and growth. In particular, the OBSL1 protein and cullin-7 are thought to help regulate proteins involved in the body's response to growth hormones, although their specific role in this process is unknown.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           3-M syndrome     At least 29 mutations in the OBSL1 gene have been identified in people with 3-M syndrome, a disorder that causes skeletal abnormalities including short stature (dwarfism) and unusual facial features. Most of these mutations substitute one protein building block (amino acid) for another amino acid in the OBSL1 protein or result in an OBSL1 protein that is abnormally short and nonfunctional.Mutations in the OBSL1 gene likely lead to reduced cullin-7 protein levels, preventing cullin-7 from bringing together the components of the E3 ubiquitin ligase complex and interfering with the process of tagging unneeded proteins for degradation. The body's response to growth hormones may be impaired as a result. However, the specific relationship between OBSL1 gene mutations and the signs and symptoms of 3-M syndrome are unknown. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 29 mutations in the OBSL1 gene have been identified in people with 3-M syndrome, a disorder that causes skeletal abnormalities including short stature (dwarfism) and unusual facial features. Most of these mutations substitute one protein building block (amino acid) for another amino acid in the OBSL1 protein or result in an OBSL1 protein that is abnormally short and nonfunctional.Mutations in the OBSL1 gene likely lead to reduced cullin-7 protein levels, preventing cullin-7 from bringing together the components of the E3 ubiquitin ligase complex and interfering with the process of tagging unneeded proteins for degradation. The body's response to growth hormones may be impaired as a result. However, the specific relationship between OBSL1 gene mutations and the signs and symptoms of 3-M syndrome are unknown. More About This Health Condition    Cytogenetic Location: 2q35, which is the long (q) arm of chromosome 2 at position 35Molecular Location: base pairs 219,549,408 to 219,571,573 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA0657obscurin-like protein 1 isoform 1 precursorobscurin-like protein 1 isoform 2 precursorobscurin-like protein 1 isoform 3 precursor     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Cell -- A Molecular Approach (second edition, 2000): The Ubiquitin-Proteasome Pathway      Clinical Information from GeneReviews (1 link)     Three M Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of OBSL1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     OBSCURIN-LIKE 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Cell -- A Molecular Approach (second edition, 2000): The Ubiquitin-Proteasome Pathway  Three M Syndrome  Tests of OBSL1  PubMed  OBSCURIN-LIKE 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Clayton PE, Hanson D, Magee L, Murray PG, Saunders E, Abu-Amero SN, Moore GE, Black GC. Exploring the spectrum of 3-M syndrome, a primordial short stature disorder of disrupted ubiquitination. Clin Endocrinol (Oxf). 2012 Sep;77(3):335-42. doi: 10.1111/j.1365-2265.2012.04428.x. Review. Citation on PubMedHanson D, Murray PG, Black GC, Clayton PE. The genetics of 3-M syndrome: unravelling a potential new regulatory growth pathway. Horm Res Paediatr. 2011;76(6):369-78. doi: 10.1159/000334392. Epub 2011 Nov 29. Review. Citation on PubMedHanson D, Murray PG, Coulson T, Sud A, Omokanye A, Stratta E, Sakhinia F, Bonshek C, Wilson LC, Wakeling E, Temtamy SA, Aglan M, Rosser EM, Mansour S, Carcavilla A, Nampoothiri S, Khan WI, Banerjee I, Chandler KE, Black GC, Clayton PE. Mutations in CUL7, OBSL1 and CCDC8 in 3-M syndrome lead to disordered growth factor signalling. J Mol Endocrinol. 2012 Oct 30;49(3):267-75. doi: 10.1530/JME-12-0034. Print 2012 Dec. Citation on PubMedHanson D, Murray PG, Sud A, Temtamy SA, Aglan M, Superti-Furga A, Holder SE, Urquhart J, Hilton E, Manson FD, Scambler P, Black GC, Clayton PE. The primordial growth disorder 3-M syndrome connects ubiquitination to the cytoskeletal adaptor OBSL1. Am J Hum Genet. 2009 Jun;84(6):801-6. doi: 10.1016/j.ajhg.2009.04.021. Epub 2009 May 28. Citation on PubMed or Free article on PubMed CentralHolder-Espinasse M. 3-M Syndrome. 2002 Mar 25 [updated 2012 Jan 26]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1481/ Citation on PubMedHuber C, Fradin M, Edouard T, Le Merrer M, Alanay Y, Da Silva DB, David A, Hamamy H, van Hest L, Lund AM, Michaud J, Oley C, Patel C, Rajab A, Skidmore DL, Stewart H, Tauber M, Munnich A, Cormier-Daire V. OBSL1 mutations in 3-M syndrome are associated with a modulation of IGFBP2 and IGFBP5 expression levels. Hum Mutat. 2010 Jan;31(1):20-6. doi: 10.1002/humu.21150. Citation on PubMedOMIM: OBSCURIN-LIKE 1 \n",
            "STK11IP\t\n",
            "COL4A3\t   The COL4A3 gene provides instructions for making one component of type IV collagen, which is a flexible protein. Specifically, this gene makes the alpha3(IV) chain of type IV collagen. This chain combines with two other types of alpha (IV) chains (the alpha4 and alpha5 chains) to make a complete type IV collagen molecule. Type IV collagen molecules attach to each other to form complex protein networks. These networks make up a large portion of basement membranes, which are thin sheet-like structures that separate and support cells in many tissues. Type IV collagen alpha3-4-5 networks play an especially important role in the basement membranes of the kidney, inner ear, and eye.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alport syndrome     More than 40 mutations in the COL4A3 gene have been found to cause Alport syndrome. Most of these mutations change single protein building blocks (amino acids) in a region where the alpha3(IV) collagen chain combines with other type IV collagen chains. Other mutations in the COL4A3 gene severely decrease or prevent the production of alpha3(IV) chains. As a result, there is a serious deficiency of the type IV collagen alpha3-4-5 network in the basement membranes of the kidney, inner ear, and eye. In the kidney, other types of collagen accumulate in the basement membranes, eventually leading to scarring of the kidneys and kidney failure. Mutations in this gene can also lead to abnormal function in the inner ear, resulting in hearing loss. More About This Health Condition      Keratoconus     Genetics Home Reference provides information about keratoconus. More About This Health Condition      Other disorders     Mutations in the COL4A3 gene have been found to cause thin basement membrane nephropathy. This condition typically causes people to have blood in their urine (hematuria) but no other signs or symptoms of kidney disease. In the past, this condition was often called benign familial hematuria. Thin basement membrane nephropathy rarely progresses to kidney failure.Goodpasture syndrome is a severe disease of the lungs and the kidneys caused by antibodies to the alpha3(IV) collagen chains. Antibodies are immune system proteins that normally attack foreign substances such as bacteria or viruses, but in Goodpasture syndrome, they target alpha3(IV) collagen chains. It remains unclear why some people make antibodies to their own collagen chains. The antibodies cause inflammation when they attach (bind) to the basement membranes of blood vessels in the air sacs (alveoli) of the lungs and filtering units (glomeruli) of the kidneys. As a result, people with Goodpasture syndrome can develop kidney failure and bleeding in the lungs, which causes them to cough up blood. In some people, antibodies attack only the kidneys. These people are said to have anti-glomerular basement membrane nephritis.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 40 mutations in the COL4A3 gene have been found to cause Alport syndrome. Most of these mutations change single protein building blocks (amino acids) in a region where the alpha3(IV) collagen chain combines with other type IV collagen chains. Other mutations in the COL4A3 gene severely decrease or prevent the production of alpha3(IV) chains. As a result, there is a serious deficiency of the type IV collagen alpha3-4-5 network in the basement membranes of the kidney, inner ear, and eye. In the kidney, other types of collagen accumulate in the basement membranes, eventually leading to scarring of the kidneys and kidney failure. Mutations in this gene can also lead to abnormal function in the inner ear, resulting in hearing loss. More About This Health Condition  Genetics Home Reference provides information about keratoconus. More About This Health Condition  Mutations in the COL4A3 gene have been found to cause thin basement membrane nephropathy. This condition typically causes people to have blood in their urine (hematuria) but no other signs or symptoms of kidney disease. In the past, this condition was often called benign familial hematuria. Thin basement membrane nephropathy rarely progresses to kidney failure.Goodpasture syndrome is a severe disease of the lungs and the kidneys caused by antibodies to the alpha3(IV) collagen chains. Antibodies are immune system proteins that normally attack foreign substances such as bacteria or viruses, but in Goodpasture syndrome, they target alpha3(IV) collagen chains. It remains unclear why some people make antibodies to their own collagen chains. The antibodies cause inflammation when they attach (bind) to the basement membranes of blood vessels in the air sacs (alveoli) of the lungs and filtering units (glomeruli) of the kidneys. As a result, people with Goodpasture syndrome can develop kidney failure and bleeding in the lungs, which causes them to cough up blood. In some people, antibodies attack only the kidneys. These people are said to have anti-glomerular basement membrane nephritis.    Cytogenetic Location: 2q36.3, which is the long (q) arm of chromosome 2 at position 36.3Molecular Location: base pairs 227,164,565 to 227,314,792 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CO4A3_HUMANcollagen IV, alpha-3 polypeptidecollagen type IV alpha 3collagen, type IV, alpha 3 (Goodpasture antigen)Goodpasture antigenTUMSTATIN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): A model of the molecular structure of a basal laminaMolecular Biology of the Cell (fourth edition, 2002): Basal Laminae Perform Diverse Functions      Clinical Information from GeneReviews (1 link)     Alport Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL4A3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (3 links)     COLLAGEN, TYPE IV, ALPHA-3GOODPASTURE SYNDROMEHEMATURIA, BENIGN FAMILIAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): A model of the molecular structure of a basal laminaMolecular Biology of the Cell (fourth edition, 2002): Basal Laminae Perform Diverse Functions  Alport Syndrome  Tests of COL4A3  PubMed  COLLAGEN, TYPE IV, ALPHA-3GOODPASTURE SYNDROMEHEMATURIA, BENIGN FAMILIAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: COLLAGEN, TYPE IV, ALPHA-3Frasc GM, Onetti-Muda A, Mari F, Longo I, Scala E, Pescucci C, Roccatello D, Alpa M, Coppo R, Li Volti G, Feriozzi S, Bergesio F, Schena FP, Renieri A; Italian Renal Immunopathology Group. Thin glomerular basement membrane disease: clinical significance of a morphological diagnosis--a collaborative study of the Italian Renal Immunopathology Group. Nephrol Dial Transplant. 2005 Mar;20(3):545-51. Epub 2004 Dec 23. Citation on PubMedGregory MC. The clinical features of thin basement membrane nephropathy. Semin Nephrol. 2005 May;25(3):140-5. Review. Citation on PubMedKashtan CE. Alport Syndrome and Thin Basement Membrane Nephropathy. 2001 Aug 28 [updated 2015 Nov 25]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1207/ Citation on PubMedKashtan CE. Alport syndrome. An inherited disorder of renal, ocular, and cochlear basement membranes. Medicine (Baltimore). 1999 Sep;78(5):338-60. Review. Citation on PubMedPescucci C, Mari F, Longo I, Vogiatzi P, Caselli R, Scala E, Abaterusso C, Gusmano R, Seri M, Miglietti N, Bresin E, Renieri A. Autosomal-dominant Alport syndrome: natural history of a disease due to COL4A3 or COL4A4 gene. Kidney Int. 2004 May;65(5):1598-603. Citation on PubMedRana K, Wang YY, Buzza M, Tonna S, Zhang KW, Lin T, Sin L, Padavarat S, Savige J. The genetics of thin basement membrane nephropathy. Semin Nephrol. 2005 May;25(3):163-70. Review. Citation on PubMedSlajpah M, Gorinsek B, Berginc G, Vizjak A, Ferluga D, Hvala A, Meglic A, Jaksa I, Furlan P, Gregoric A, Kaplan-Pavlovcic S, Ravnik-Glavac M, Glavac D. Sixteen novel mutations identified in COL4A3, COL4A4, and COL4A5 genes in Slovenian families with Alport syndrome and benign familial hematuria. Kidney Int. 2007 Jun;71(12):1287-95. Epub 2007 Mar 28. Citation on PubMedTazn Vega B, Badenas C, Ars E, Lens X, Mil M, Darnell A, Torra R. Autosomal recessive Alport's syndrome and benign familial hematuria are collagen type IV diseases. Am J Kidney Dis. 2003 Nov;42(5):952-9. Citation on PubMedWang YY, Rana K, Tonna S, Lin T, Sin L, Savige J. COL4A3 mutations and their clinical consequences in thin basement membrane nephropathy (TBMN). Kidney Int. 2004 Mar;65(3):786-90. Citation on PubMed \n",
            "C2orf83\t\n",
            "DAW1\t\n",
            "SP140\t\n",
            "SP140L\t\n",
            "SPATA3\t\n",
            "ARMC9\t\n",
            "GIGYF2\t   From NCBI Gene:This gene contains CAG trinucleotide repeats and encodes a protein containing several stretches of polyglutamine residues. The encoded protein may be involved in the regulation of tyrosine kinase receptor signaling. This gene is located in a chromosomal region that was genetically linked to Parkinson disease type 11, and mutations in this gene were thought to be causative for this disease. However, more recent studies in different populations have been unable to replicate this association. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2013]From UniProt:Key component of the 4EHP-GYF2 complex, a multiprotein complex that acts as a repressor of translation initiation (PubMed:22751931). In 4EHP-GYF2 the complex, acts as a factor that bridges EIF4E2 to ZFP36/TTP, linking translation repression with mRNA decay (By similarity). May act cooperatively with GRB10 to regulate tyrosine kinase receptor signaling, including IGF1 and insulin receptors (PubMed:12771153).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Parkinson disease 11From UniProt:Parkinson disease 11 (PARK11): A complex neurodegenerative disorder characterized by bradykinesia, resting tremor, muscular rigidity and postural instability, as well as by a clinically significant response to treatment with levodopa. The pathology involves the loss of dopaminergic neurons in the substantia nigra and the presence of Lewy bodies (intraneuronal accumulations of aggregated proteins), in surviving neurons in various areas of the brain. [MIM:607688]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q37.1, which is the long (q) arm of chromosome 2 at position 37.1Molecular Location: base pairs 232,697,305 to 232,860,577 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GYF2PARK11PERQ2PERQ3TNRC15     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GIGYF2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GRB10-INTERACTING GYF PROTEIN 2OMIM: PARKINSON DISEASE 11, AUTOSOMAL DOMINANT, SUSCEPTIBILITY TO      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GIGYF2  OMIM: GRB10-INTERACTING GYF PROTEIN 2OMIM: PARKINSON DISEASE 11, AUTOSOMAL DOMINANT, SUSCEPTIBILITY TO  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "USP40\t\n",
            "COL6A3\t   The COL6A3 gene provides instructions for making one component of type VI collagen, which is a flexible protein found in the space that surrounds cells. Specifically, the protein produced from the COL6A3 gene is the alpha()3(VI) chain of type VI collagen. This chain combines with chains produced from other genes to produce a complete type VI collagen molecule.Collagens are found in the extracellular matrix, which is an intricate lattice that forms in the space between cells and provides structural support. Type VI collagen is located in the extracellular matrix surrounding cells that make up the muscles used for movement (skeletal muscle cells) and cells that make up connective tissue, which provides strength and flexibility to structures throughout the body, including skin and joints. The extracellular matrix is necessary for cell stability and growth. Research suggests that type VI collagen links basement membranes, which are thin, sheet-like structures that are part of the extracellular matrix, to nearby cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Collagen VI-related myopathy     Mutations in the COL6A3 gene have been found to cause some cases of collagen VI-related myopathy, which is a group of disorders that vary in severity but generally result in muscle weakness and joint deformities called contractures. These mutations often change single protein building blocks (amino acids) in the 3(VI) chain. The most frequently affected amino acid is glycine; changes to this building block disrupt the structure of the 3(VI) chain. Other mutations can also disrupt the structure of the 3(VI) chain.Mutations in the COL6A3 gene affect type VI collagen in different ways. Some mutations lead to altered 3(VI) chains that can be incorporated into the mature type VI collagen molecule, although they disrupt the molecule's structure and function. Other mutations result in an altered chain that cannot be incorporated at all. Still other mutations prevent the production of any functional 3(VI) chain, which impedes formation of type VI collagen. All of these COL6A3 gene mutations lead to a reduction or absence of functional collagen VI molecules. While it is difficult to predict the severity of collagen VI-related myopathy based on the type of mutation, in general, lower amounts of type VI collagen lead to more severe signs and symptoms that begin earlier in life.Changes in 3(VI) chain structure or production lead to an unstable extracellular matrix that is no longer attached to cells through the basement membrane. As a result, the stability of muscle cells and connective tissue progressively declines, which leads to the muscle weakness, contractures, and other signs and symptoms of collagen VI-related myopathy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the COL6A3 gene have been found to cause some cases of collagen VI-related myopathy, which is a group of disorders that vary in severity but generally result in muscle weakness and joint deformities called contractures. These mutations often change single protein building blocks (amino acids) in the 3(VI) chain. The most frequently affected amino acid is glycine; changes to this building block disrupt the structure of the 3(VI) chain. Other mutations can also disrupt the structure of the 3(VI) chain.Mutations in the COL6A3 gene affect type VI collagen in different ways. Some mutations lead to altered 3(VI) chains that can be incorporated into the mature type VI collagen molecule, although they disrupt the molecule's structure and function. Other mutations result in an altered chain that cannot be incorporated at all. Still other mutations prevent the production of any functional 3(VI) chain, which impedes formation of type VI collagen. All of these COL6A3 gene mutations lead to a reduction or absence of functional collagen VI molecules. While it is difficult to predict the severity of collagen VI-related myopathy based on the type of mutation, in general, lower amounts of type VI collagen lead to more severe signs and symptoms that begin earlier in life.Changes in 3(VI) chain structure or production lead to an unstable extracellular matrix that is no longer attached to cells through the basement membrane. As a result, the stability of muscle cells and connective tissue progressively declines, which leads to the muscle weakness, contractures, and other signs and symptoms of collagen VI-related myopathy. More About This Health Condition    Cytogenetic Location: 2q37.3, which is the long (q) arm of chromosome 2 at position 37.3Molecular Location: base pairs 237,324,012 to 237,434,822 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CO6A3_HUMANcollagen alpha-3(VI) chaincollagen type VI alpha 3collagen VI, alpha-3 polypeptidecollagen, type VI, alpha 3     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Cell Biology (fourth edition, 2000): Collagens Form Diverse StructuresThe Cell: A Molecular Approach (second edition, 2000): The Extracellular MatrixWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Collagen Type VI-Related Disorders      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL6A3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     COLLAGEN, TYPE VI, ALPHA-3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Collagens Form Diverse StructuresThe Cell: A Molecular Approach (second edition, 2000): The Extracellular MatrixWashington University, St. Louis: Neuromuscular Disease Center  Collagen Type VI-Related Disorders  Tests of COL6A3  PubMed  COLLAGEN, TYPE VI, ALPHA-3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Allamand V, Brias L, Richard P, Stojkovic T, Quijano-Roy S, Bonne G. ColVI myopathies: where do we stand, where do we go? Skelet Muscle. 2011 Sep 23;1:30. doi: 10.1186/2044-5040-1-30. Citation on PubMed or Free article on PubMed CentralBaker NL, Mrgelin M, Pace RA, Peat RA, Adams NE, Gardner RJ, Rowland LP, Miller G, De Jonghe P, Ceulemans B, Hannibal MC, Edwards M, Thompson EM, Jacobson R, Quinlivan RC, Aftimos S, Kornberg AJ, North KN, Bateman JF, Lamand SR. Molecular consequences of dominant Bethlem myopathy collagen VI mutations. Ann Neurol. 2007 Oct;62(4):390-405. Citation on PubMedBaker NL, Mrgelin M, Peat R, Goemans N, North KN, Bateman JF, Lamand SR. Dominant collagen VI mutations are a common cause of Ullrich congenital muscular dystrophy. Hum Mol Genet. 2005 Jan 15;14(2):279-93. Epub 2004 Nov 24. Citation on PubMedBushby KM, Collins J, Hicks D. Collagen type VI myopathies. Adv Exp Med Biol. 2014;802:185-99. doi: 10.1007/978-94-007-7893-1_12. Review. Citation on PubMedBnnemann CG. The collagen VI-related myopathies: muscle meets its matrix. Nat Rev Neurol. 2011 Jun 21;7(7):379-90. doi: 10.1038/nrneurol.2011.81. Review. Citation on PubMedOMIM: COLLAGEN, TYPE VI, ALPHA-3 \n",
            "SCLY\t\n",
            "ESPNL\t\n",
            "TRAF3IP1\t   From NCBI Gene:The protein encoded by this gene interacts with TNF receptor-associated factor 3, tethering it to cytoskeletal microtubules. The encoded protein is also an inhibitor of the innate type I IFN response. Defects in this gene are a cause of Senior-Loken syndrome 9. [provided by RefSeq, Mar 2017]From UniProt:Plays an inhibitory role on IL13 signaling by binding to IL13RA1. Involved in suppression of IL13-induced STAT6 phosphorylation, transcriptional activity and DNA-binding. Recruits TRAF3 and DISC1 to the microtubules. Involved in kidney development and epithelial morphogenesis. Involved in the regulation of microtubule cytoskeleton organization. Is a negative regulator of microtubule stability, acting through the control of MAP4 levels (PubMed:26487268). Involved in ciliogenesis (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Senior-Loken syndrome 9From UniProt:Senior-Loken syndrome 9 (SLSN9): A renal-retinal disorder characterized by progressive wasting of the filtering unit of the kidney (nephronophthisis), with or without medullary cystic renal disease, and progressive eye disease. Typically this disorder becomes apparent during the first year of life. [MIM:616629]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q37.3, which is the long (q) arm of chromosome 2 at position 37.3Molecular Location: base pairs 238,320,489 to 238,400,900 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IFT54MIP-T3MIPT3SLSN9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TRAF3IP1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SENIOR-LOKEN SYNDROME 9OMIM: TNF RECEPTOR-ASSOCIATED FACTOR 3-INTERACTING PROTEIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TRAF3IP1  OMIM: SENIOR-LOKEN SYNDROME 9OMIM: TNF RECEPTOR-ASSOCIATED FACTOR 3-INTERACTING PROTEIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OR6B2\t\n",
            "OR6B3\t\n",
            "GPR35\t\n",
            "KIF1A\t   From NCBI Gene:The protein encoded by this gene is a member of the kinesin family and functions as an anterograde motor protein that transports membranous organelles along axonal microtubules. Mutations at this locus have been associated with spastic paraplegia-30 and hereditary sensory neuropathy IIC. Alternatively spliced transcript variants encoding distinct isoforms have been described. [provided by RefSeq, Apr 2012]From UniProt:Motor for anterograde axonal transport of synaptic vesicle precursors.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Hereditary sensory and autonomic neuropathy type IIFrom NCBI Gene:Hereditary sensory and autonomic neuropathy type IIAMental retardation, autosomal dominant 9Hereditary sensory and autonomic neuropathy type IICSpastic paraplegia 30, autosomal recessiveFrom UniProt:Mental retardation, autosomal dominant 9 (MRD9): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. [MIM:614255]Defects in KIF1A are a cause autosomal dominant spastic paraplegia. Spastic paraplegia is a neurodegenerative disorder characterized by a slow, gradual, progressive weakness and spasticity of the lower limbs.Neuropathy, hereditary sensory, 2C (HSN2C): A neurodegenerative disorder characterized by onset in the first decade of progressive distal sensory loss leading to ulceration and amputation of the fingers and toes. Affected individuals also develop distal muscle weakness, primarily affecting the lower limbs. [MIM:614213]Spastic paraplegia 30, autosomal recessive (SPG30): A form of spastic paraplegia, a neurodegenerative disorder characterized by a slow, gradual, progressive weakness and spasticity of the lower limbs. Rate of progression and the severity of symptoms are quite variable. Initial symptoms may include difficulty with balance, weakness and stiffness in the legs, muscle spasms, and dragging the toes when walking. In some forms of the disorder, bladder symptoms (such as incontinence) may appear, or the weakness and stiffness may spread to other parts of the body. SPG30 is characterized by onset in the first or second decades of unsteady spastic gait and hyperreflexia of the lower limbs. [MIM:610357]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q37.3, which is the long (q) arm of chromosome 2 at position 37.3Molecular Location: base pairs 240,713,764 to 240,820,308 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATSVC2orf20HSN2CMRD9SPG30UNC104     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KIF1A      Catalog of Genes and Diseases from OMIM (5 links)     OMIM: KINESIN FAMILY MEMBER 1AOMIM: MENTAL RETARDATION, AUTOSOMAL DOMINANT 9OMIM: NEUROPATHY, HEREDITARY SENSORY AND AUTONOMIC, TYPE IIAOMIM: NEUROPATHY, HEREDITARY SENSORY, TYPE IICOMIM: SPASTIC PARAPLEGIA 30, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KIF1A  OMIM: KINESIN FAMILY MEMBER 1AOMIM: MENTAL RETARDATION, AUTOSOMAL DOMINANT 9OMIM: NEUROPATHY, HEREDITARY SENSORY AND AUTONOMIC, TYPE IIAOMIM: NEUROPATHY, HEREDITARY SENSORY, TYPE IICOMIM: SPASTIC PARAPLEGIA 30, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "AGXT\t   The AGXT gene provides instructions for making an enzyme called serine-pyruvate aminotransferase. This enzyme is found in liver cells, specifically within cell structures called peroxisomes. These structures are important for several cellular activities, such as ridding the cell of toxic substances and helping to break down certain fats. In the peroxisome, serine-pyruvate aminotransferase converts a compound called glyoxylate to the protein building block (amino acid) glycine.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Primary hyperoxaluria     More than 175 mutations in the AGXT gene have been found to cause primary hyperoxaluria type 1. This condition is caused by the overproduction of a substance called oxalate. Excess amounts of this substance lead to kidney and bladder stones, which can begin anytime from childhood to early adulthood with kidney disease developing at any age. Deposition of oxalate in multiple other tissues throughout the body (systemic oxalosis) can cause additional health problems.Most of the AGXT gene mutations decrease or eliminate serine-pyruvate aminotransferase activity, which impairs the conversion of glyoxylate to glycine. Other mutations cause the enzyme to be misplaced in cells, transporting it to structures called mitochondria instead of to peroxisomes. While the enzyme in the mitochondria retains activity, it cannot access glyoxylate, which is in peroxisomes. All AGXT gene mutations result in the accumulation of glyoxylate, which is converted to oxalate instead of glycine. The oxalate is filtered through the kidneys and is either excreted in urine as a waste product or combines with calcium to form calcium oxalate, a hard compound that is the main component of kidney and bladder stones. Increased oxalate levels in the blood can lead to systemic oxalosis, particularly affecting bones and the walls of blood vessels in people with primary hyperoxaluria type 1. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 175 mutations in the AGXT gene have been found to cause primary hyperoxaluria type 1. This condition is caused by the overproduction of a substance called oxalate. Excess amounts of this substance lead to kidney and bladder stones, which can begin anytime from childhood to early adulthood with kidney disease developing at any age. Deposition of oxalate in multiple other tissues throughout the body (systemic oxalosis) can cause additional health problems.Most of the AGXT gene mutations decrease or eliminate serine-pyruvate aminotransferase activity, which impairs the conversion of glyoxylate to glycine. Other mutations cause the enzyme to be misplaced in cells, transporting it to structures called mitochondria instead of to peroxisomes. While the enzyme in the mitochondria retains activity, it cannot access glyoxylate, which is in peroxisomes. All AGXT gene mutations result in the accumulation of glyoxylate, which is converted to oxalate instead of glycine. The oxalate is filtered through the kidneys and is either excreted in urine as a waste product or combines with calcium to form calcium oxalate, a hard compound that is the main component of kidney and bladder stones. Increased oxalate levels in the blood can lead to systemic oxalosis, particularly affecting bones and the walls of blood vessels in people with primary hyperoxaluria type 1. More About This Health Condition    Cytogenetic Location: 2q37.3, which is the long (q) arm of chromosome 2 at position 37.3Molecular Location: base pairs 240,868,745 to 240,879,119 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AGTAGT1AGXT1alanine glyoxylate aminotransferasealanine-glyoxylate aminotransferasealanine-glyoxylate aminotransferase (oxalosis I; hyperoxaluria I; glycolicaciduria; serine-pyruvate aminotransferase)alanine-glyoxylate transaminaseL-alanine: glyoxylate aminotransferase 1pyruvate (glyoxylate) aminotransferaseserine-pyruvate aminotransferaseserine:pyruvate aminotransferaseSPATSPT     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Primary Hyperoxaluria Type 1      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGXT      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ALANINE-GLYOXYLATE AMINOTRANSFERASE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Primary Hyperoxaluria Type 1  Tests of AGXT  PubMed  ALANINE-GLYOXYLATE AMINOTRANSFERASE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cochat P, Rumsby G. Primary hyperoxaluria. N Engl J Med. 2013 Aug 15;369(7):649-58. doi: 10.1056/NEJMra1301564. Review. Erratum in: N Engl J Med. 2013 Nov 28;369(22):2168. Citation on PubMedCoulter-Mackie MB, White CT, Lange D, Chew BH. Primary Hyperoxaluria Type 1. 2002 Jun 19 [updated 2014 Jul 17]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1283/ Citation on PubMedHopp K, Cogal AG, Bergstralh EJ, Seide BM, Olson JB, Meek AM, Lieske JC, Milliner DS, Harris PC; Rare Kidney Stone Consortium. Phenotype-Genotype Correlations and Estimated Carrier Frequencies of Primary Hyperoxaluria. J Am Soc Nephrol. 2015 Oct;26(10):2559-70. doi: 10.1681/ASN.2014070698. Epub 2015 Feb 2. Citation on PubMed or Free article on PubMed CentralWilliams EL, Acquaviva C, Amoroso A, Chevalier F, Coulter-Mackie M, Monico CG, Giachino D, Owen T, Robbiano A, Salido E, Waterham H, Rumsby G. Primary hyperoxaluria type 1: update and additional mutation analysis of the AGXT gene. Hum Mutat. 2009 Jun;30(6):910-7. doi: 10.1002/humu.21021. Review. Citation on PubMed \n",
            "SNED1\t\n",
            "MTERF4\t\n",
            "FARP2\t\n",
            "SH3YL1\t\n",
            "RTP5\t\n",
            "ITSN2\t\n",
            "HADHA\t   The HADHA gene provides instructions for making part of an enzyme complex called mitochondrial trifunctional protein. This enzyme complex functions in mitochondria, the energy-producing centers within cells. Mitochondrial trifunctional protein is made of eight parts (subunits). Four alpha subunits are produced from the HADHA gene, and four beta subunits are produced from the HADHB gene. As the name suggests, mitochondrial trifunctional protein contains three enzymes that each perform a different function. The alpha subunits contain two of the enzymes, known as long-chain 3-hydroxyacyl-CoA dehydrogenase and long-chain 2-enoyl-CoA hydratase. The beta subunits contain the third enzyme. These enzymes are essential for fatty acid oxidation, which is the multistep process that breaks down (metabolizes) fats and converts them to energy.Mitochondrial trifunctional protein is required to metabolize a group of fats called long-chain fatty acids. Long-chain fatty acids are found in foods such as milk and certain oils. These fatty acids are stored in the body's fat tissues. Fatty acids are a major source of energy for the heart and muscles. During periods of fasting, fatty acids are also an important energy source for the liver and other tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Long-chain 3-hydroxyacyl-CoA dehydrogenase deficiency     Researchers have identified several HADHA gene mutations that cause long-chain 3-hydroxyacyl-CoA dehydrogenase (LCHAD) deficiency. These mutations decrease the long-chain 3-hydroxyacyl-CoA dehydrogenase enzyme activity of mitochondrial trifunctional protein.  (The protein's other enzyme activities remain normal or nearly normal.)  Many of the HADHA mutations change one of the protein building blocks (amino acids) used to make the alpha subunit.  The most common mutation replaces the amino acid glutamic acid with the amino acid glutamine at position 474 in the alpha subunit.  This mutation is written as Glu474Gln or E474Q.  The Glu474Gln mutation and other amino acid replacements probably alter the structure of the alpha subunit, preventing it from functioning normally.  Other HADHA mutations result in an abnormally short, nonfunctional version of the alpha subunit.With a shortage (deficiency) of functional alpha subunits, long-chain fatty acids cannot be metabolized and processed.  As a result, these fatty acids are not converted to energy, which can lead to some features of LCHAD deficiency, such as lack of energy (lethargy) and low blood sugar (hypoglycemia). Long-chain fatty acids or partially metabolized fatty acids may also build up and damage the liver, heart, muscles, and light-sensitive tissue at the back of the eye (retina). This abnormal buildup causes the other signs and symptoms of LCHAD deficiency. More About This Health Condition      Mitochondrial trifunctional protein deficiency     Researchers have identified several HADHA gene mutations that cause mitochondrial trifunctional protein deficiency. These mutations reduce all three enzyme activities of mitochondrial trifunctional protein. Some of these mutations result in abnormally short, nonfunctional alpha subunits and lead to decreased levels of mitochondrial trifunctional protein. Other mutations replace one amino acid with another amino acid in the alpha subunit, which probably alters the subunit's structure and disrupts all three functions of the enzyme complex.When mitochondrial trifunctional protein activity is lost, long-chain fatty acids cannot be metabolized and processed. As a result, these fatty acids are not converted to energy, which can lead to some features of this disorder, such as lethargy and hypoglycemia. Long-chain fatty acids or partially metabolized fatty acids may build up in tissues and damage the liver, heart, and muscles. This abnormal buildup causes the other signs and symptoms of mitochondrial trifunctional protein deficiency. More About This Health Condition      Other disorders     In a small number of cases, HADHA mutations appear to increase a woman's risk of developing two serious liver disorders during pregnancy, known as acute fatty liver of pregnancy (AFLP) and HELLP syndrome. AFLP begins with abdominal pain and can rapidly progress to liver failure. HELLP stands for hemolysis (the breakdown of red blood cells), elevated liver enzyme levels, and low platelets (cell fragments involved with blood clotting).A woman is more likely to have AFLP or HELLP syndrome if she has a mutation in one copy of the HADHA gene and the fetus she carries has two copies of a HADHA mutation, particularly the Glu474Gln mutation.  Little is known about the relationship between HADHA mutations and liver problems in the mother during pregnancy.  One possibility is that partially metabolized long-chain fatty acids produced by the fetus or placenta accumulate in the mother and are toxic to her liver.  In very rare cases of maternal liver disease, the mother has one copy of an altered HADHA gene and the fetus is not affected.  In these cases, the role of the mother's HADHA mutation in liver disease is unclear.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Researchers have identified several HADHA gene mutations that cause long-chain 3-hydroxyacyl-CoA dehydrogenase (LCHAD) deficiency. These mutations decrease the long-chain 3-hydroxyacyl-CoA dehydrogenase enzyme activity of mitochondrial trifunctional protein.  (The protein's other enzyme activities remain normal or nearly normal.)  Many of the HADHA mutations change one of the protein building blocks (amino acids) used to make the alpha subunit.  The most common mutation replaces the amino acid glutamic acid with the amino acid glutamine at position 474 in the alpha subunit.  This mutation is written as Glu474Gln or E474Q.  The Glu474Gln mutation and other amino acid replacements probably alter the structure of the alpha subunit, preventing it from functioning normally.  Other HADHA mutations result in an abnormally short, nonfunctional version of the alpha subunit.With a shortage (deficiency) of functional alpha subunits, long-chain fatty acids cannot be metabolized and processed.  As a result, these fatty acids are not converted to energy, which can lead to some features of LCHAD deficiency, such as lack of energy (lethargy) and low blood sugar (hypoglycemia). Long-chain fatty acids or partially metabolized fatty acids may also build up and damage the liver, heart, muscles, and light-sensitive tissue at the back of the eye (retina). This abnormal buildup causes the other signs and symptoms of LCHAD deficiency. More About This Health Condition  Researchers have identified several HADHA gene mutations that cause mitochondrial trifunctional protein deficiency. These mutations reduce all three enzyme activities of mitochondrial trifunctional protein. Some of these mutations result in abnormally short, nonfunctional alpha subunits and lead to decreased levels of mitochondrial trifunctional protein. Other mutations replace one amino acid with another amino acid in the alpha subunit, which probably alters the subunit's structure and disrupts all three functions of the enzyme complex.When mitochondrial trifunctional protein activity is lost, long-chain fatty acids cannot be metabolized and processed. As a result, these fatty acids are not converted to energy, which can lead to some features of this disorder, such as lethargy and hypoglycemia. Long-chain fatty acids or partially metabolized fatty acids may build up in tissues and damage the liver, heart, and muscles. This abnormal buildup causes the other signs and symptoms of mitochondrial trifunctional protein deficiency. More About This Health Condition  In a small number of cases, HADHA mutations appear to increase a woman's risk of developing two serious liver disorders during pregnancy, known as acute fatty liver of pregnancy (AFLP) and HELLP syndrome. AFLP begins with abdominal pain and can rapidly progress to liver failure. HELLP stands for hemolysis (the breakdown of red blood cells), elevated liver enzyme levels, and low platelets (cell fragments involved with blood clotting).A woman is more likely to have AFLP or HELLP syndrome if she has a mutation in one copy of the HADHA gene and the fetus she carries has two copies of a HADHA mutation, particularly the Glu474Gln mutation.  Little is known about the relationship between HADHA mutations and liver problems in the mother during pregnancy.  One possibility is that partially metabolized long-chain fatty acids produced by the fetus or placenta accumulate in the mother and are toxic to her liver.  In very rare cases of maternal liver disease, the mother has one copy of an altered HADHA gene and the fetus is not affected.  In these cases, the role of the mother's HADHA mutation in liver disease is unclear.    Cytogenetic Location: 2p23.3, which is the short (p) arm of chromosome 2 at position 23.3Molecular Location: base pairs 26,190,635 to 26,244,726 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ECHA_HUMANGBPhydroxyacyl-CoA dehydrogenase/3-ketoacyl-CoA thiolase/enoyl-CoA hydratase (trifunctional protein), alpha subunithydroxyacyl-Coenzyme A dehydrogenase/3-ketoacyl-Coenzyme A thiolase/enoyl-Coenzyme A hydratase (trifunctional protein), alpha subunithydroxyacyl dehydrogenase, subunit ALCEHLCHADlong-chain hydroxyacyl-CoA dehydrogenasemitochondrial trifunctional protein, alpha subunitMTPA     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): Fatty Acid MetabolismEmory University School of Medicine: HELLP, AFLP, and FAODs      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HADHA      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     HYDROXYACYL-CoA DEHYDROGENASE/3-KETOACYL-CoA THIOLASE/ENOYL-CoA HYDRATASE, ALPHA SUBUNIT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Fatty Acid MetabolismEmory University School of Medicine: HELLP, AFLP, and FAODs  Tests of HADHA  PubMed  HYDROXYACYL-CoA DEHYDROGENASE/3-KETOACYL-CoA THIOLASE/ENOYL-CoA HYDRATASE, ALPHA SUBUNIT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Angdisen J, Moore VD, Cline JM, Payne RM, Ibdah JA. Mitochondrial trifunctional protein defects: molecular basis and novel therapeutic approaches. Curr Drug Targets Immune Endocr Metabol Disord. 2005 Mar;5(1):27-40. Review. Citation on PubMedBlish KR, Ibdah JA. Maternal heterozygosity for a mitochondrial trifunctional protein mutation as a cause for liver disease in pregnancy. Med Hypotheses. 2005;64(1):96-100. Citation on PubMedChoi JH, Yoon HR, Kim GH, Park SJ, Shin YL, Yoo HW. Identification of novel mutations of the HADHA and HADHB genes in patients with mitochondrial trifunctional protein deficiency. Int J Mol Med. 2007 Jan;19(1):81-7. Citation on PubMedGutirrez Junquera C, Balmaseda E, Gil E, Martnez A, Sorli M, Cuartero I, Merinero B, Ugarte M. Acute fatty liver of pregnancy and neonatal long-chain 3-hydroxyacyl-coenzyme A dehydrogenase (LCHAD) deficiency. Eur J Pediatr. 2009 Jan;168(1):103-6. doi: 10.1007/s00431-008-0696-z. Epub 2008 Apr 12. Citation on PubMedOMIM: HYDROXYACYL-CoA DEHYDROGENASE/3-KETOACYL-CoA THIOLASE/ENOYL-CoA HYDRATASE, ALPHA SUBUNITIbdah JA, Bennett MJ, Rinaldo P, Zhao Y, Gibson B, Sims HF, Strauss AW. A fetal fatty-acid oxidation disorder as a cause of liver disease in pregnant women. N Engl J Med. 1999 Jun 3;340(22):1723-31. Citation on PubMedIbdah JA, Yang Z, Bennett MJ. Liver disease in pregnancy and fetal fatty acid oxidation defects. Mol Genet Metab. 2000 Sep-Oct;71(1-2):182-9. Review. Citation on PubMedOey NA, den Boer ME, Wijburg FA, Vekemans M, Aug J, Steiner C, Wanders RJ, Waterham HR, Ruiter JP, Atti-Bitach T. Long-chain fatty acid oxidation during early human development. Pediatr Res. 2005 Jun;57(6):755-9. Epub 2005 Apr 21. Citation on PubMedShekhawat PS, Matern D, Strauss AW. Fetal fatty acid oxidation disorders, their effect on maternal health and neonatal outcome: impact of expanded newborn screening on their diagnosis and management. Pediatr Res. 2005 May;57(5 Pt 2):78R-86R. Epub 2005 Apr 6. Review. Citation on PubMed or Free article on PubMed CentralSims HF, Brackett JC, Powell CK, Treem WR, Hale DE, Bennett MJ, Gibson B, Shapiro S, Strauss AW. The molecular basis of pediatric long chain 3-hydroxyacyl-CoA dehydrogenase deficiency associated with maternal acute fatty liver of pregnancy. Proc Natl Acad Sci U S A. 1995 Jan 31;92(3):841-5. Citation on PubMed or Free article on PubMed CentralSpiekerkoetter U, Khuchua Z, Yue Z, Bennett MJ, Strauss AW. General mitochondrial trifunctional protein (TFP) deficiency as a result of either alpha- or beta-subunit mutations exhibits similar phenotypes because mutations in either subunit alter TFP complex expression and subunit turnover. Pediatr Res. 2004 Feb;55(2):190-6. Epub 2003 Nov 19. Citation on PubMedSpiekerkoetter U, Mueller M, Cloppenburg E, Motz R, Mayatepek E, Bueltmann B, Korenke C. Intrauterine cardiomyopathy and cardiac mitochondrial proliferation in mitochondrial trifunctional protein (TFP) deficiency. Mol Genet Metab. 2008 Aug;94(4):428-30. doi: 10.1016/j.ymgme.2008.04.002. Epub 2008 May 15. Citation on PubMedTyni T, Pihko H. Long-chain 3-hydroxyacyl-CoA dehydrogenase deficiency. Acta Paediatr. 1999 Mar;88(3):237-45. Review. Citation on PubMedYang Z, Yamada J, Zhao Y, Strauss AW, Ibdah JA. Prospective screening for pediatric mitochondrial trifunctional protein defects in pregnancies complicated by liver disease. JAMA. 2002 Nov 6;288(17):2163-6. Citation on PubMedYang Z, Zhao Y, Bennett MJ, Strauss AW, Ibdah JA. Fetal genotypes and pregnancy outcomes in 35 families with mitochondrial trifunctional protein mutations. Am J Obstet Gynecol. 2002 Sep;187(3):715-20. Citation on PubMedden Boer ME, Wanders RJ, Morris AA, IJlst L, Heymans HS, Wijburg FA. Long-chain 3-hydroxyacyl-CoA dehydrogenase deficiency: clinical presentation and follow-up of 50 patients. Pediatrics. 2002 Jan;109(1):99-104. Citation on PubMed \n",
            "HADHB\t   The HADHB gene provides instructions for making part of an enzyme complex called mitochondrial trifunctional protein. This enzyme complex functions in mitochondria, the energy-producing centers within cells. Mitochondrial trifunctional protein is made of eight parts (subunits). Four alpha subunits are produced from the HADHA gene, and four beta subunits are produced from the HADHB gene. As the name suggests, mitochondrial trifunctional protein contains three enzymes that each perform a different function.  The beta subunits contain one of the enzymes, known as long-chain 3-keto-acyl-CoA thiolase. The alpha subunits contain the other two enzymes. These enzymes are essential for fatty acid oxidation, which is the multistep process that breaks down (metabolizes) fats and converts them to energy.Mitochondrial trifunctional protein is required to metabolize a group of fats called long-chain fatty acids. Long-chain fatty acids are found in foods such as milk and certain oils. These fatty acids are stored in the body's fat tissues. Fatty acids are a major source of energy for the heart and muscles. During periods of fasting, fatty acids are also an important energy source for the liver and other tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Mitochondrial trifunctional protein deficiency     Researchers have identified at least 26 mutations in the HADHB gene that cause mitochondrial trifunctional protein deficiency. These mutations reduce all three enzyme activities of mitochondrial trifunctional protein.  Most mutations change one of the protein building blocks (amino acids) used to make the beta subunit.  A change in amino acids probably alters the subunit's structure, which disrupts all three activities of the enzyme complex. Some mutations produce abnormally short, nonfunctional beta subunits and lead to decreased levels of mitochondrial trifunctional protein.With a loss of mitochondrial trifunctional protein activity, long-chain fatty acids cannot be metabolized and processed. As a result, these fatty acids are not converted to energy, which can lead to some features of this disorder, such as a lack of energy (lethargy) and low blood sugar (hypoglycemia). Long-chain fatty acids or partially metabolized fatty acids may also build up and damage the liver, heart, and muscles. This abnormal buildup causes the other signs and symptoms of mitochondrial trifunctional protein deficiency. More About This Health Condition      Other disorders     A few mutations in the HADHB gene have been found to decrease only the long-chain 3-keto-acyl-CoA thiolase enzyme activity of mitochondrial trifunctional protein. These mutations change single amino acids used to make the beta subunit. The signs and symptoms of isolated long-chain 3-keto-acyl-CoA thiolase deficiency are similar to those of mitochondrial trifunctional protein deficiency. These features include feeding difficulties, lethargy, hypoglycemia, weak muscle tone (hypotonia), and liver problems. Infants with this disorder are also at high risk for serious heart problems, breathing difficulties, coma, and sudden death.HADHB mutations appear to increase a woman's risk of developing two serious liver disorders during pregnancy, known as acute fatty liver of pregnancy (AFLP) and HELLP syndrome. AFLP begins with abdominal pain and can rapidly progress to liver failure.  HELLP stands for hemolysis (the breakdown of red blood cells), elevated liver enzyme levels, and low platelets (cell fragments involved with blood clotting). A small percentage of women who have a mutation in one copy of the HADHB gene and carry a fetus with mutations in both copies of the HADHB gene develop one of these maternal liver diseases. Little is known about the relationship between HADHB mutations and liver problems in the mother during pregnancy. One possibility is that partially metabolized long-chain fatty acids produced by the fetus or placenta accumulate in the mother and are toxic to the liver.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Researchers have identified at least 26 mutations in the HADHB gene that cause mitochondrial trifunctional protein deficiency. These mutations reduce all three enzyme activities of mitochondrial trifunctional protein.  Most mutations change one of the protein building blocks (amino acids) used to make the beta subunit.  A change in amino acids probably alters the subunit's structure, which disrupts all three activities of the enzyme complex. Some mutations produce abnormally short, nonfunctional beta subunits and lead to decreased levels of mitochondrial trifunctional protein.With a loss of mitochondrial trifunctional protein activity, long-chain fatty acids cannot be metabolized and processed. As a result, these fatty acids are not converted to energy, which can lead to some features of this disorder, such as a lack of energy (lethargy) and low blood sugar (hypoglycemia). Long-chain fatty acids or partially metabolized fatty acids may also build up and damage the liver, heart, and muscles. This abnormal buildup causes the other signs and symptoms of mitochondrial trifunctional protein deficiency. More About This Health Condition  A few mutations in the HADHB gene have been found to decrease only the long-chain 3-keto-acyl-CoA thiolase enzyme activity of mitochondrial trifunctional protein. These mutations change single amino acids used to make the beta subunit. The signs and symptoms of isolated long-chain 3-keto-acyl-CoA thiolase deficiency are similar to those of mitochondrial trifunctional protein deficiency. These features include feeding difficulties, lethargy, hypoglycemia, weak muscle tone (hypotonia), and liver problems. Infants with this disorder are also at high risk for serious heart problems, breathing difficulties, coma, and sudden death.HADHB mutations appear to increase a woman's risk of developing two serious liver disorders during pregnancy, known as acute fatty liver of pregnancy (AFLP) and HELLP syndrome. AFLP begins with abdominal pain and can rapidly progress to liver failure.  HELLP stands for hemolysis (the breakdown of red blood cells), elevated liver enzyme levels, and low platelets (cell fragments involved with blood clotting). A small percentage of women who have a mutation in one copy of the HADHB gene and carry a fetus with mutations in both copies of the HADHB gene develop one of these maternal liver diseases. Little is known about the relationship between HADHB mutations and liver problems in the mother during pregnancy. One possibility is that partially metabolized long-chain fatty acids produced by the fetus or placenta accumulate in the mother and are toxic to the liver.    Cytogenetic Location: 2p23.3, which is the short (p) arm of chromosome 2 at position 23.3Molecular Location: base pairs 26,244,748 to 26,290,465 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ECHB_HUMANHADHhydroxyacyl-CoA dehydrogenase/3-ketoacyl-CoA thiolase/enoyl-CoA hydratase (trifunctional protein), beta subunithydroxyacyl-Coenzyme A dehydrogenase/3-ketoacyl-Coenzyme A thiolase/enoyl-Coenzyme A hydratase (trifunctional protein), beta subunithydroxyacyl dehydrogenase, subunit BMTPBTFPBTP-beta     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): Fatty Acid MetabolismEmory University School of Medicine: HELLP, AFLP, and FAODs      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HADHB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     HYDROXYACYL-CoA DEHYDROGENASE/3-KETOACYL-CoA THIOLASE/ENOYL-CoA HYDRATASE, BETA SUBUNIT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Fatty Acid MetabolismEmory University School of Medicine: HELLP, AFLP, and FAODs  Tests of HADHB  PubMed  HYDROXYACYL-CoA DEHYDROGENASE/3-KETOACYL-CoA THIOLASE/ENOYL-CoA HYDRATASE, BETA SUBUNIT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Angdisen J, Moore VD, Cline JM, Payne RM, Ibdah JA. Mitochondrial trifunctional protein defects: molecular basis and novel therapeutic approaches. Curr Drug Targets Immune Endocr Metabol Disord. 2005 Mar;5(1):27-40. Review. Citation on PubMedChoi JH, Yoon HR, Kim GH, Park SJ, Shin YL, Yoo HW. Identification of novel mutations of the HADHA and HADHB genes in patients with mitochondrial trifunctional protein deficiency. Int J Mol Med. 2007 Jan;19(1):81-7. Citation on PubMedDas AM, Illsinger S, Lcke T, Hartmann H, Ruiter JP, Steuerwald U, Waterham HR, Duran M, Wanders RJ. Isolated mitochondrial long-chain ketoacyl-CoA thiolase deficiency resulting from mutations in the HADHB gene. Clin Chem. 2006 Mar;52(3):530-4. Epub 2006 Jan 19. Citation on PubMedEaton S, Bursby T, Middleton B, Pourfarzam M, Mills K, Johnson AW, Bartlett K. The mitochondrial trifunctional protein: centre of a beta-oxidation metabolon? Biochem Soc Trans. 2000 Feb;28(2):177-82. Review. Citation on PubMedOMIM: HYDROXYACYL-CoA DEHYDROGENASE/3-KETOACYL-CoA THIOLASE/ENOYL-CoA HYDRATASE, BETA SUBUNITOey NA, den Boer ME, Wijburg FA, Vekemans M, Aug J, Steiner C, Wanders RJ, Waterham HR, Ruiter JP, Atti-Bitach T. Long-chain fatty acid oxidation during early human development. Pediatr Res. 2005 Jun;57(6):755-9. Epub 2005 Apr 21. Citation on PubMedOrii KE, Aoyama T, Wakui K, Fukushima Y, Miyajima H, Yamaguchi S, Orii T, Kondo N, Hashimoto T. Genomic and mutational analysis of the mitochondrial trifunctional protein beta-subunit (HADHB) gene in patients with trifunctional protein deficiency. Hum Mol Genet. 1997 Aug;6(8):1215-24. Citation on PubMedSander J, Sander S, Steuerwald U, Janzen N, Peter M, Wanders RJ, Marquardt I, Korenke GC, Das AM. Neonatal screening for defects of the mitochondrial trifunctional protein. Mol Genet Metab. 2005 Jun;85(2):108-14. Epub 2005 Mar 24. Citation on PubMedShekhawat PS, Matern D, Strauss AW. Fetal fatty acid oxidation disorders, their effect on maternal health and neonatal outcome: impact of expanded newborn screening on their diagnosis and management. Pediatr Res. 2005 May;57(5 Pt 2):78R-86R. Epub 2005 Apr 6. Review. Citation on PubMed or Free article on PubMed CentralSpiekerkoetter U, Khuchua Z, Yue Z, Bennett MJ, Strauss AW. General mitochondrial trifunctional protein (TFP) deficiency as a result of either alpha- or beta-subunit mutations exhibits similar phenotypes because mutations in either subunit alter TFP complex expression and subunit turnover. Pediatr Res. 2004 Feb;55(2):190-6. Epub 2003 Nov 19. Citation on PubMedSpiekerkoetter U, Lindner M, Santer R, Grotzke M, Baumgartner MR, Boehles H, Das A, Haase C, Hennermann JB, Karall D, de Klerk H, Knerr I, Koch HG, Plecko B, Rschinger W, Schwab KO, Scheible D, Wijburg FA, Zschocke J, Mayatepek E, Wendel U. Management and outcome in 75 individuals with long-chain fatty acid oxidation defects: results from a workshop. J Inherit Metab Dis. 2009 Aug;32(4):488-97. doi: 10.1007/s10545-009-1125-9. Epub 2009 Apr 29. Citation on PubMedSpiekerkoetter U, Sun B, Khuchua Z, Bennett MJ, Strauss AW. Molecular and phenotypic heterogeneity in mitochondrial trifunctional protein deficiency due to beta-subunit mutations. Hum Mutat. 2003 Jun;21(6):598-607. Citation on PubMedSpierkerkoetter U, Khuchua Z, Yue Z, Strauss AW. The early-onset phenotype of mitochondrial trifunctional protein deficiency: a lethal disorder with multiple tissue involvement. J Inherit Metab Dis. 2004;27(2):294-6. Citation on PubMed \n",
            "ADGRF3\t\n",
            "DRC1\t   From NCBI Gene:This gene encodes a central component of the nexin-dynein complex (N-DRC), which regulates the assembly of ciliary dynein. Mutations in this gene can cause ciliary dyskinesia. [provided by RefSeq, Aug 2015]From UniProt:Component of the nexin-dynein regulatory complex (N-DRC) a key regulator of ciliary/flagellar motility which maintains the alignment and integrity of the distal axoneme and regulates microtubule sliding in motile axonemes (By similarity). Plays a critical role in the assembly of N-DRC and also stabilizes the assembly of multiple inner dynein arms and radial spokes. Coassembles with CCDC65/DRC2 to form a central scaffold needed for assembly of the N-DRC and its attachment to the outer doublet microtubules (PubMed:23354437).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Primary ciliary dyskinesiaFrom NCBI Gene:Ciliary dyskinesia, primary, 21From UniProt:Ciliary dyskinesia, primary, 21 (CILD21): A disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia. Patients may exhibit randomization of left-right body asymmetry and situs inversus, due to dysfunction of monocilia at the embryonic node. Primary ciliary dyskinesia associated with situs inversus is referred to as Kartagener syndrome. [MIM:615294]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p23.3, which is the short (p) arm of chromosome 2 at position 23.3Molecular Location: base pairs 26,401,912 to 26,456,713 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C2orf39CCDC164CILD21     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DRC1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 21OMIM: DYNEIN REGULATORY COMPLEX, SUBUNIT 1, CHLAMYDOMONAS, HOMOLOG OF      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DRC1  OMIM: CILIARY DYSKINESIA, PRIMARY, 21OMIM: DYNEIN REGULATORY COMPLEX, SUBUNIT 1, CHLAMYDOMONAS, HOMOLOG OF  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OTOF\t   From NCBI Gene:Mutations in this gene are a cause of neurosensory nonsyndromic recessive deafness, DFNB9. The short form of the encoded protein has 3 C2 domains, a single carboxy-terminal transmembrane domain found also in the C. elegans spermatogenesis factor FER-1 and human dysferlin, while the long form has 6 C2 domains. The homology suggests that this protein may be involved in vesicle membrane fusion. Several transcript variants encoding multiple isoforms have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Key calcium ion sensor involved in the Ca(2+)-triggered synaptic vesicle-plasma membrane fusion and in the control of neurotransmitter release at these output synapses. Interacts in a calcium-dependent manner to the presynaptic SNARE proteins at ribbon synapses of cochlear inner hair cells (IHCs) to trigger exocytosis of neurotransmitter. Also essential to synaptic exocytosis in immature outer hair cells (OHCs). May also play a role within the recycling of endosomes (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 9From UniProt:Auditory neuropathy, autosomal recessive, 1 (AUNB1): A form of sensorineural hearing loss with absent or severely abnormal auditory brainstem response, in the presence of normal cochlear outer hair cell function and normal otoacoustic emissions. Auditory neuropathies result from a lesion in the area including the inner hair cells, connections between the inner hair cells and the cochlear branch of the auditory nerve, the auditory nerve itself and auditory pathways of the brainstem. In some cases AUNB1 phenotype can be temperature sensitive. [MIM:601071]Deafness, autosomal recessive, 9 (DFNB9): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. [MIM:601071]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p23.3, which is the short (p) arm of chromosome 2 at position 23.3Molecular Location: base pairs 26,457,203 to 26,558,698 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AUNB1DFNB6DFNB9FER1L2NSRD9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of OTOF      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 9OMIM: OTOFERLIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of OTOF  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 9OMIM: OTOFERLIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "C2orf70\t\n",
            "TMEM214\t\n",
            "PLB1\t\n",
            "GALNT14\t\n",
            "TRAPPC12\t\n",
            "CYP1B1\t   The CYP1B1 gene provides instructions for producing an enzyme that is a member of the cytochrome P450 family of enzymes. These enzymes are involved in many processes in the body, such as assisting with reactions that break down drugs and produce certain fats (lipids). The CYP1B1 enzyme participates in biochemical reactions in which an oxygen atom is added to other molecules.The CYP1B1 enzyme is active in many tissues, including structures of the eye. The function of the CYP1B1 enzyme in the development of the eye is unclear, but it may play a role in forming structures at the front of the eye and may also be involved in a process that regulates the secretion of fluid inside the eye.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Early-onset glaucoma     More than 140 CYP1B1 gene mutations have been identified to cause early-onset glaucoma. People with this condition experience increased pressure within the eye before the age of 40. This pressure causes damage to the optic nerves connecting the eyes and the brain. Between 20 percent and 40 percent of people with glaucoma before the age of 5 (primary congenital glaucoma) have mutations in the CYP1B1 gene. The mutations that cause early-onset glaucoma may result in an enzyme that is unstable or the wrong shape.It is not well understood how defects in the CYP1B1 enzyme cause signs and symptoms of glaucoma. Recent studies suggest that the defects may interfere with the early development of the trabecular meshwork, which is a network of mesh-like canals that helps drain excess fluid from the eye. If fluid cannot be drained, pressure inside the eye may increase, which is characteristic of glaucoma.The CYP1B1 enzyme may interact with another protein called myocilin, which is produced from the MYOC gene. Individuals with mutations in both the MYOC and CYP1B1 genes may develop glaucoma at an earlier age and have more severe symptoms than do those with mutations in only one of the genes. More About This Health Condition      Peters anomaly     A few mutations in the CYP1B1 gene have been found to cause Peters anomaly. This condition is characterized by abnormal development of certain structures at the front of the eye and clouding of the clear front surface of the eye (cornea). The mutations that cause Peters anomaly likely impair the normal function of the CYP1B1 enzyme or disrupt enzyme production. As a result, there is a shortage (deficiency) of normal enzyme. A lack of the CYP1B1 enzyme likely disrupts normal development of the eye, although it is unclear exactly how this deficiency leads to the features of Peters anomaly. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 140 CYP1B1 gene mutations have been identified to cause early-onset glaucoma. People with this condition experience increased pressure within the eye before the age of 40. This pressure causes damage to the optic nerves connecting the eyes and the brain. Between 20 percent and 40 percent of people with glaucoma before the age of 5 (primary congenital glaucoma) have mutations in the CYP1B1 gene. The mutations that cause early-onset glaucoma may result in an enzyme that is unstable or the wrong shape.It is not well understood how defects in the CYP1B1 enzyme cause signs and symptoms of glaucoma. Recent studies suggest that the defects may interfere with the early development of the trabecular meshwork, which is a network of mesh-like canals that helps drain excess fluid from the eye. If fluid cannot be drained, pressure inside the eye may increase, which is characteristic of glaucoma.The CYP1B1 enzyme may interact with another protein called myocilin, which is produced from the MYOC gene. Individuals with mutations in both the MYOC and CYP1B1 genes may develop glaucoma at an earlier age and have more severe symptoms than do those with mutations in only one of the genes. More About This Health Condition  A few mutations in the CYP1B1 gene have been found to cause Peters anomaly. This condition is characterized by abnormal development of certain structures at the front of the eye and clouding of the clear front surface of the eye (cornea). The mutations that cause Peters anomaly likely impair the normal function of the CYP1B1 enzyme or disrupt enzyme production. As a result, there is a shortage (deficiency) of normal enzyme. A lack of the CYP1B1 enzyme likely disrupts normal development of the eye, although it is unclear exactly how this deficiency leads to the features of Peters anomaly. More About This Health Condition    Cytogenetic Location: 2p22.2, which is the short (p) arm of chromosome 2 at position 22.2Molecular Location: base pairs 38,067,603 to 38,076,181 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        aryl hydrocarbon hydroxylaseCP1BCP1B1_HUMANcytochrome P450, family 1, subfamily B, polypeptide 1cytochrome P450, subfamily I (dioxin-inducible), polypeptide 1 (glaucoma 3, primary infantile)flavoprotein-linked monooxygenaseGLC3Amicrosomal monooxygenasexenobiotic monooxygenase     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Primary Congenital Glaucoma      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CYP1B1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CYTOCHROME P450, SUBFAMILY I, POLYPEPTIDE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Primary Congenital Glaucoma  Tests of CYP1B1  PubMed  CYTOCHROME P450, SUBFAMILY I, POLYPEPTIDE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Achary MS, Reddy AB, Chakrabarti S, Panicker SG, Mandal AK, Ahmed N, Balasubramanian D, Hasnain SE, Nagarajaram HA. Disease-causing mutations in proteins: structural analysis of the CYP1B1 mutations causing primary congenital glaucoma in humans. Biophys J. 2006 Dec 15;91(12):4329-39. Epub 2006 Sep 8. Citation on PubMed or Free article on PubMed CentralBayat B, Yazdani S, Alavi A, Chiani M, Chitsazian F, Tusi BK, Suri F, Narooie-Nejhad M, Sanati MH, Elahi E. Contributions of MYOC and CYP1B1 mutations to JOAG. Mol Vis. 2008 Mar 13;14:508-17. Citation on PubMed or Free article on PubMed CentralOMIM: CYTOCHROME P450, SUBFAMILY I, POLYPEPTIDE 1Chavarria-Soley G, Sticht H, Aklillu E, Ingelman-Sundberg M, Pasutto F, Reis A, Rautenstrauss B. Mutations in CYP1B1 cause primary congenital glaucoma by reduction of either activity or abundance of the enzyme. Hum Mutat. 2008 Sep;29(9):1147-53. doi: 10.1002/humu.20786. Citation on PubMedChen Y, Jiang D, Yu L, Katz B, Zhang K, Wan B, Sun X. CYP1B1 and MYOC mutations in 116 Chinese patients with primary congenital glaucoma. Arch Ophthalmol. 2008 Oct;126(10):1443-7. doi: 10.1001/archopht.126.10.1443. Citation on PubMedEdward D, Al Rajhi A, Lewis RA, Curry S, Wang Z, Bejjani B. Molecular basis of Peters anomaly in Saudi Arabia. Ophthalmic Genet. 2004 Dec;25(4):257-70. Citation on PubMedMelki R, Colomb E, Lefort N, Brzin AP, Garchon HJ. CYP1B1 mutations in French patients with early-onset primary open-angle glaucoma. J Med Genet. 2004 Sep;41(9):647-51. Citation on PubMed or Free article on PubMed CentralRay K, Mukhopadhyay A, Acharya M. Recent advances in molecular genetics of glaucoma. Mol Cell Biochem. 2003 Nov;253(1-2):223-31. Review. Citation on PubMedVasiliou V, Gonzalez FJ. Role of CYP1B1 in glaucoma. Annu Rev Pharmacol Toxicol. 2008;48:333-58. Review. Citation on PubMedVincent A, Billingsley G, Priston M, Glaser T, Oliver E, Walter M, Ritch R, Levin A, Heon E. Further support of the role of CYP1B1 in patients with Peters anomaly. Mol Vis. 2006 May 16;12:506-10. Citation on PubMedVincent A, Billingsley G, Priston M, Williams-Lyn D, Sutherland J, Glaser T, Oliver E, Walter MA, Heathcote G, Levin A, Hon E. Phenotypic heterogeneity of CYP1B1: mutations in a patient with Peters' anomaly. J Med Genet. 2001 May;38(5):324-6. Citation on PubMed or Free article on PubMed CentralVincent AL, Billingsley G, Buys Y, Levin AV, Priston M, Trope G, Williams-Lyn D, Hon E. Digenic inheritance of early-onset glaucoma: CYP1B1, a potential modifier gene. Am J Hum Genet. 2002 Feb;70(2):448-60. Epub 2002 Jan 3. Citation on PubMed or Free article on PubMed CentralWeisschuh N, Schiefer U. Progress in the genetics of glaucoma. Dev Ophthalmol. 2003;37:83-93. Review. Citation on PubMed \n",
            "DHX57\t\n",
            "MORN2\t\n",
            "TMEM247\t\n",
            "VRK2\t\n",
            "SOX11\t   From NCBI Gene:This intronless gene encodes a member of the SOX (SRY-related HMG-box) family of transcription factors involved in the regulation of embryonic development and in the determination of the cell fate. The encoded protein may act as a transcriptional regulator after forming a protein complex with other proteins. The protein may function in the developing nervous system and play a role in tumorigenesis. [provided by RefSeq, Jul 2008]From UniProt:Transcriptional factor involved in the embryonic neurogenesis. May also have a role in tissue modeling during development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mental retardation, autosomal dominant 27From UniProt:Mental retardation, autosomal dominant 27 (MRD27): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. MRD27 patients show dysmorphic facial features, microcephaly, growth deficiency, hypoplastic fifth toenails, and mild intellectual disability. [MIM:615866]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p25.2, which is the short (p) arm of chromosome 2 at position 25.2Molecular Location: base pairs 5,692,667 to 5,701,385 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MRD27     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SOX11      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MENTAL RETARDATION, AUTOSOMAL DOMINANT 27OMIM: SRY-BOX 11      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SOX11  OMIM: MENTAL RETARDATION, AUTOSOMAL DOMINANT 27OMIM: SRY-BOX 11  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "C2orf74\t\n",
            "FAM161A\t   From NCBI Gene:This gene belongs to the FAM161 family. It is expressed mainly in the retina. Mouse studies suggested that this gene is involved in development of retinal progenitors during embryogenesis, and that its activity is restricted to mature photoreceptors after birth. Mutations in this gene cause autosomal recessive retinitis pigmentosa-28. Alternatively spliced transcript variants have been identified.[provided by RefSeq, Jan 2011]From UniProt:Involved in ciliogenesis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Retinitis pigmentosaFrom NCBI Gene:Retinitis pigmentosa 28From UniProt:Retinitis pigmentosa 28 (RP28): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:606068]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p15, which is the short (p) arm of chromosome 2 at position 15Molecular Location: base pairs 61,800,240 to 61,854,143 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        RP28     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FAM161A      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FAMILY WITH SEQUENCE SIMILARITY 161, MEMBER AOMIM: RETINITIS PIGMENTOSA 28      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FAM161A  OMIM: FAMILY WITH SEQUENCE SIMILARITY 161, MEMBER AOMIM: RETINITIS PIGMENTOSA 28  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "EHBP1\t   From NCBI Gene:This gene encodes an Eps15 homology domain binding protein. The encoded protein may play a role in endocytic trafficking. A single nucleotide polymorphism in this gene is associated with an aggressive form of prostate cancer. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]From UniProt:May play a role in actin reorganization. Links clathrin-mediated endocytosis to the actin cytoskeleton. May act as Rab effector protein and play a role in vesicle trafficking (PubMed:14676205, PubMed:27552051). Required for perinuclear sorting and insulin-regulated recycling of SLC2A4/GLUT4 in adipocytes (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Prostate cancerFrom NCBI Gene:Prostate cancer, hereditary, 12From UniProt:Prostate cancer, hereditary, 12 (HPC12): A condition associated with familial predisposition to cancer of the prostate. Most prostate cancers are adenocarcinomas that develop in the acini of the prostatic ducts. Other rare histopathologic types of prostate cancer that occur in approximately 5% of patients include small cell carcinoma, mucinous carcinoma, prostatic ductal carcinoma, transitional cell carcinoma, squamous cell carcinoma, basal cell carcinoma, adenoid cystic carcinoma (basaloid), signet-ring cell carcinoma and neuroendocrine carcinoma. [MIM:611868]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p15, which is the short (p) arm of chromosome 2 at position 15Molecular Location: base pairs 62,673,851 to 63,046,487 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HPC12NACSIN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of EHBP1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: EH DOMAIN-BINDING PROTEIN 1OMIM: PROSTATE CANCER, HEREDITARY, 12      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of EHBP1  OMIM: EH DOMAIN-BINDING PROTEIN 1OMIM: PROSTATE CANCER, HEREDITARY, 12  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "APLF\t\n",
            "AAK1\t\n",
            "ATP6V1B1\t   The ATP6V1B1 gene provides instructions for making a part (subunit) of a large protein complex known as vacuolar H+-ATPase (V-ATPase). V-ATPases are a group of similar complexes that act as pumps to move positively charged hydrogen atoms (protons) across membranes. Because acids are substances that can \"donate\" protons to other molecules, this movement of protons helps regulate the relative acidity (pH) of cells and their surrounding environment. Tight control of pH is necessary for most biological reactions to proceed properly.The V-ATPase that includes the subunit produced from the ATP6V1B1 gene is found in the inner ear and in nephrons, which are the functional structures within the kidneys. The kidneys filter waste products from the blood and remove them in urine. They also reabsorb needed nutrients and release them back into the blood. Each nephron consists of two parts: a renal corpuscle (also known as a glomerulus) that filters the blood, and a renal tubule that reabsorbs substances that are needed and eliminates unneeded substances in urine. The V-ATPase is involved in regulating the amount of acid that is removed from the blood into the urine, and also in maintaining the proper pH of the fluid in the inner ear (endolymph).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Renal tubular acidosis with deafness     More than 25 ATP6V1B1 gene mutations have been identified in people with renal tubular acidosis with deafness, a disorder involving excess acid in the blood (metabolic acidosis), bone weakness, and hearing loss caused by changes in the inner ear (sensorineural hearing loss).Mutations in the ATP6V1B1 gene impair the function of the V-ATPase proton pump. As a result, the kidneys are less able to control the acidity of the blood, which leads to bone weakness caused by loss of bone minerals (demineralization) and other consequences of metabolic acidosis. The body's capability to control the pH of the fluid in the inner ear is also impaired, resulting in sensorineural hearing loss. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 25 ATP6V1B1 gene mutations have been identified in people with renal tubular acidosis with deafness, a disorder involving excess acid in the blood (metabolic acidosis), bone weakness, and hearing loss caused by changes in the inner ear (sensorineural hearing loss).Mutations in the ATP6V1B1 gene impair the function of the V-ATPase proton pump. As a result, the kidneys are less able to control the acidity of the blood, which leads to bone weakness caused by loss of bone minerals (demineralization) and other consequences of metabolic acidosis. The body's capability to control the pH of the fluid in the inner ear is also impaired, resulting in sensorineural hearing loss. More About This Health Condition    Cytogenetic Location: 2p13.3, which is the short (p) arm of chromosome 2 at position 13.3Molecular Location: base pairs 70,935,868 to 70,965,431 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATP6B1ATPase, H+ transporting, lysosomal 56/58kDa, V1 subunit B1endomembrane proton pump 58 kDa subunitH(+)-transporting two-sector ATPase, 58kD subunitH+-ATPase beta 1 subunitRTA1BV-ATPase B1 subunitV-ATPase subunit B 1V-type proton ATPase subunit B, kidney isoformvacuolar proton pump 3vacuolar proton pump subunit B 1vacuolar proton pump, subunit 3VATBVMA2VPP3     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Cell Biology (fourth edition, 2000): Active Transport by ATP-Powered Pumps      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATP6V1B1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATPase, H+ TRANSPORTING, LYSOSOMAL, 56/58-KD, V1 SUBUNIT B, ISOFORM 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Active Transport by ATP-Powered Pumps  Tests of ATP6V1B1  PubMed  ATPase, H+ TRANSPORTING, LYSOSOMAL, 56/58-KD, V1 SUBUNIT B, ISOFORM 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ATPase, H+ TRANSPORTING, LYSOSOMAL, 56/58-KD, V1 SUBUNIT B, ISOFORM 1Alper SL. Familial renal tubular acidosis. J Nephrol. 2010 Nov-Dec;23 Suppl 16:S57-76. Review. Citation on PubMedAndreucci E, Bianchi B, Carboni I, Lavoratti G, Mortilla M, Fonda C, Bigozzi M, Genuardi M, Giglio S, Pela I. Inner ear abnormalities in four patients with dRTA and SNHL: clinical and genetic heterogeneity. Pediatr Nephrol. 2009 Nov;24(11):2147-53. doi: 10.1007/s00467-009-1261-3. Epub 2009 Jul 29. Citation on PubMedBatlle D, Haque SK. Genetic causes and mechanisms of distal renal tubular acidosis. Nephrol Dial Transplant. 2012 Oct;27(10):3691-704. doi: 10.1093/ndt/gfs442. Review. Citation on PubMedGil H, Santos F, Garca E, Alvarez MV, Ordez FA, Mlaga S, Coto E. Distal RTA with nerve deafness: clinical spectrum and mutational analysis in five children. Pediatr Nephrol. 2007 Jun;22(6):825-8. Epub 2007 Jan 11. Citation on PubMedMohebbi N, Vargas-Poussou R, Hegemann SC, Schuknecht B, Kistler AD, Wthrich RP, Wagner CA. Homozygous and compound heterozygous mutations in the ATP6V1B1 gene in patients with renal tubular acidosis and sensorineural hearing loss. Clin Genet. 2013 Mar;83(3):274-8. doi: 10.1111/j.1399-0004.2012.01891.x. Epub 2012 May 11. Citation on PubMedNikali K, Vanegas JJ, Burley MW, Martinez J, Lopez LM, Bedoya G, Wrong OM, Povey S, Unwin RJ, Ruiz-Linares A. Extensive founder effect for distal renal tubular acidosis (dRTA) with sensorineural deafness in an isolated South American population. Am J Med Genet A. 2008 Oct 15;146A(20):2709-12. doi: 10.1002/ajmg.a.32495. Citation on PubMedSethi SK, Singh N, Gil H, Bagga A. Genetic studies in a family with distal renal tubular acidosis and sensorineural deafness. Indian Pediatr. 2009 May;46(5):425-7. Citation on PubMedStover EH, Borthwick KJ, Bavalia C, Eady N, Fritz DM, Rungroj N, Giersch AB, Morton CC, Axon PR, Akil I, Al-Sabban EA, Baguley DM, Bianca S, Bakkaloglu A, Bircan Z, Chauveau D, Clermont MJ, Guala A, Hulton SA, Kroes H, Li Volti G, Mir S, Mocan H, Nayir A, Ozen S, Rodriguez Soriano J, Sanjad SA, Tasic V, Taylor CM, Topaloglu R, Smith AN, Karet FE. Novel ATP6V1B1 and ATP6V0A4 mutations in autosomal recessive distal renal tubular acidosis with new evidence for hearing loss. J Med Genet. 2002 Nov;39(11):796-803. Citation on PubMed or Free article on PubMed Central \n",
            "ANKRD53\t\n",
            "RAB11FIP5\t\n",
            "ALMS1\t   The ALMS1 gene provides instructions for making a protein whose function is unknown. Researchers believe that the protein may play a role in hearing, vision, regulation of body weight, and functions of the heart, kidney, lungs, and liver. It may also affect how the pancreas regulates insulin, a hormone that helps control blood sugar levels.The ALMS1 protein is present in most of the body's tissues, usually at low levels. Within cells, this protein is located in structures called centrosomes. Centrosomes play a role in cell division and the assembly of microtubules, which are proteins that transport materials in cells and help the cell maintain its shape. The ALMS1 protein is also found at the base of cilia, which are finger-like projections that stick out from the surface of cells.  Almost all cells have cilia at some stage of their life cycle.  Cilia are involved in cell movement and many different chemical signaling pathways. Based on its location within cells, researchers suggest that the ALMS1 protein might be involved in the organization of microtubules, the transport of various materials, and the normal function of cilia.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alstrm syndrome     More than 80 mutations in the ALMS1 gene have been identified in people with Alstrm syndrome. Most of these mutations lead to the production of an abnormally small version of the ALMS1 protein that does not function properly. Researchers propose that a lack of normally functioning ALMS1 protein in the brain could lead to overeating. A loss of this protein in the pancreas may cause insulin resistance, a condition in which the body cannot use insulin properly. The combined effects of overeating and insulin resistance impair the body's ability to handle excess sugar, leading to diabetes and obesity (two common features of Alstrm syndrome). It is unclear how ALMS1 mutations cause the other signs and symptoms of Alstrm syndrome.  Researchers suspect that this condition is associated with malfunctioning cilia in many of the body's tissues and organs. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 80 mutations in the ALMS1 gene have been identified in people with Alstrm syndrome. Most of these mutations lead to the production of an abnormally small version of the ALMS1 protein that does not function properly. Researchers propose that a lack of normally functioning ALMS1 protein in the brain could lead to overeating. A loss of this protein in the pancreas may cause insulin resistance, a condition in which the body cannot use insulin properly. The combined effects of overeating and insulin resistance impair the body's ability to handle excess sugar, leading to diabetes and obesity (two common features of Alstrm syndrome). It is unclear how ALMS1 mutations cause the other signs and symptoms of Alstrm syndrome.  Researchers suspect that this condition is associated with malfunctioning cilia in many of the body's tissues and organs. More About This Health Condition    Cytogenetic Location: 2p13.1, which is the short (p) arm of chromosome 2 at position 13.1Molecular Location: base pairs 73,385,758 to 73,609,919 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ALMS1_HUMANAlstrom syndrome 1Alstrom syndrome protein 1KIAA0328     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Cilia and Flagella Are Motile Structures Built from Microtubules and DyneinsThe Cell: A Molecular Approach (second edition, 2000): Structure, Assembly, and Dynamic Instability of Microtubules      Clinical Information from GeneReviews (1 link)     Alstrom Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ALMS1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ALMS1 GENE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Cilia and Flagella Are Motile Structures Built from Microtubules and DyneinsThe Cell: A Molecular Approach (second edition, 2000): Structure, Assembly, and Dynamic Instability of Microtubules  Alstrom Syndrome  Tests of ALMS1  PubMed  ALMS1 GENE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Collin GB, Marshall JD, Ikeda A, So WV, Russell-Eggitt I, Maffei P, Beck S, Boerkoel CF, Sicolo N, Martin M, Nishina PM, Naggert JK. Mutations in ALMS1 cause obesity, type 2 diabetes and neurosensory degeneration in Alstrm syndrome. Nat Genet. 2002 May;31(1):74-8. Epub 2002 Apr 8. Citation on PubMedHearn T, Renforth GL, Spalluto C, Hanley NA, Piper K, Brickwood S, White C, Connolly V, Taylor JF, Russell-Eggitt I, Bonneau D, Walker M, Wilson DI. Mutation of ALMS1, a large gene with a tandem repeat encoding 47 amino acids, causes Alstrm syndrome. Nat Genet. 2002 May;31(1):79-83. Epub 2002 Apr 8. Citation on PubMedHearn T, Spalluto C, Phillips VJ, Renforth GL, Copin N, Hanley NA, Wilson DI. Subcellular localization of ALMS1 supports involvement of centrosome and basal body dysfunction in the pathogenesis of obesity, insulin resistance, and type 2 diabetes. Diabetes. 2005 May;54(5):1581-7. Citation on PubMedJoy T, Cao H, Black G, Malik R, Charlton-Menys V, Hegele RA, Durrington PN. Alstrom syndrome (OMIM 203800): a case report and literature review. Orphanet J Rare Dis. 2007 Dec 21;2:49. Review. Citation on PubMed or Free article on PubMed CentralLi G, Vega R, Nelms K, Gekakis N, Goodnow C, McNamara P, Wu H, Hong NA, Glynne R. A role for Alstrm syndrome protein, alms1, in kidney ciliogenesis and cellular quiescence. PLoS Genet. 2007 Jan 5;3(1):e8. Epub 2006 Nov 30. Citation on PubMed or Free article on PubMed CentralMarshall JD, Bronson RT, Collin GB, Nordstrom AD, Maffei P, Paisey RB, Carey C, Macdermott S, Russell-Eggitt I, Shea SE, Davis J, Beck S, Shatirishvili G, Mihai CM, Hoeltzenbein M, Pozzan GB, Hopkinson I, Sicolo N, Naggert JK, Nishina PM. New Alstrm syndrome phenotypes based on the evaluation of 182 cases. Arch Intern Med. 2005 Mar 28;165(6):675-83. Citation on PubMedMarshall JD, Hinman EG, Collin GB, Beck S, Cerqueira R, Maffei P, Milan G, Zhang W, Wilson DI, Hearn T, Tavares P, Vettor R, Veronese C, Martin M, So WV, Nishina PM, Naggert JK. Spectrum of ALMS1 variants and evaluation of genotype-phenotype correlations in Alstrm syndrome. Hum Mutat. 2007 Nov;28(11):1114-23. Citation on PubMedMarshall JD, Paisey RB, Carey C, Macdermott S. Alstrm Syndrome. 2003 Feb 7 [updated 2012 May 31]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1267/ Citation on PubMedMinton JA, Owen KR, Ricketts CJ, Crabtree N, Shaikh G, Ehtisham S, Porter JR, Carey C, Hodge D, Paisey R, Walker M, Barrett TG. Syndromic obesity and diabetes: changes in body composition with age and mutation analysis of ALMS1 in 12 United Kingdom kindreds with Alstrom syndrome. J Clin Endocrinol Metab. 2006 Aug;91(8):3110-6. Epub 2006 May 23. Citation on PubMed \n",
            "NAT8B\t\n",
            "TET3\t\n",
            "GCFC2\t\n",
            "DNAH6\t\n",
            "TRABD2A\t\n",
            "TGOLN2\t\n",
            "RNF103\t\n",
            "EIF2AK3\t   From NCBI Gene:The protein encoded by this gene phosphorylates the alpha subunit of eukaryotic translation-initiation factor 2, leading to its inactivation, and thus to a rapid reduction of translational initiation and repression of global protein synthesis. This protein is thought to modulate mitochondrial function. It is a type I membrane protein located in the endoplasmic reticulum (ER), where it is induced by ER stress caused by malfolded proteins. Mutations in this gene are associated with Wolcott-Rallison syndrome. [provided by RefSeq, Sep 2015]From UniProt:Metabolic-stress sensing protein kinase that phosphorylates the alpha subunit of eukaryotic translation initiation factor 2 (eIF-2-alpha/EIF2S1) on 'Ser-52' during the unfolded protein response (UPR) and in response to low amino acid availability. Converts phosphorylated eIF-2-alpha/EIF2S1 either in a global protein synthesis inhibitor, leading to a reduced overall utilization of amino acids, or to a translation initiation activator of specific mRNAs, such as the transcriptional activator ATF4, and hence allowing ATF4-mediated reprogramming of amino acid biosynthetic gene expression to alleviate nutrient depletion. Serves as a critical effector of unfolded protein response (UPR)-induced G1 growth arrest due to the loss of cyclin-D1 (CCND1). Involved in control of mitochondrial morphology and function.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Wolcott-Rallison dysplasiaFrom UniProt:Wolcott-Rallison syndrome (WRS): A rare autosomal recessive disorder, characterized by permanent neonatal or early infancy insulin-dependent diabetes and, at a later age, epiphyseal dysplasia, osteoporosis, growth retardation and other multisystem manifestations, such as hepatic and renal dysfunctions, mental retardation and cardiovascular abnormalities. [MIM:226980]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p11.2, which is the short (p) arm of chromosome 2 at position 11.2Molecular Location: base pairs 88,556,740 to 88,627,576 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PEKPERKWRS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of EIF2AK3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: EPIPHYSEAL DYSPLASIA, MULTIPLE, WITH EARLY-ONSET DIABETES MELLITUSOMIM: EUKARYOTIC TRANSLATION INITIATION FACTOR 2-ALPHA KINASE 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of EIF2AK3  OMIM: EPIPHYSEAL DYSPLASIA, MULTIPLE, WITH EARLY-ONSET DIABETES MELLITUSOMIM: EUKARYOTIC TRANSLATION INITIATION FACTOR 2-ALPHA KINASE 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "KIDINS220\t   From NCBI Gene:This gene encodes a transmembrane protein that is preferentially expressed in the nervous system where it controls neuronal cell survival, differentiation into exons and dendrites, and synaptic plasticity. The encoded protein interacts with membrane receptors, cytosolic signaling components, and cytoskeletal proteins, serving as a scaffold that mediates crosstalk between the neurotrophin pathway and several other intracellular signaling pathways. Aberrant expression of this gene is associated with the onset of various neuropsychiatric disorders and neurodegenerative diseases, including Alzheimer's disease. Naturally occurring mutations in this gene are associated with a syndrome characterized by spastic paraplegia, intellectual disability, nystagmus and obesity. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2017]From UniProt:Promotes a prolonged MAP-kinase signaling by neurotrophins through activation of a Rap1-dependent mechanism. Provides a docking site for the CRKL-C3G complex, resulting in Rap1-dependent sustained ERK activation. May play an important role in regulating postsynaptic signal transduction through the syntrophin-mediated localization of receptor tyrosine kinases such as EPHA4. In cooperation with SNTA1 can enhance EPHA4-induced JAK/STAT activation. Plays a role in nerve growth factor (NGF)-induced recruitment of RAPGEF2 to late endosomes and neurite outgrowth. May play a role in neurotrophin- and ephrin-mediated neuronal outgrowth and in axon guidance during neural development and in neuronal regeneration (By similarity). Modulates stress-induced apoptosis of melanoma cells via regulation of the MEK/ERK signaling pathway.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spastic paraplegia, intellectual disability, nystagmus, and obesityFrom UniProt:Spastic paraplegia, intellectual disability, nystagmus, and obesity (SINO): An autosomal dominant syndrome characterized by rapid growth in infancy, obesity, global developmental delay, intellectual disability, spastic paraplegia, ocular defects, and dysmorphic facial features. [MIM:617296]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2p25.1, which is the short (p) arm of chromosome 2 at position 25.1Molecular Location: base pairs 8,721,134 to 8,837,639 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARMSSINO     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KIDINS220      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KINASE D-INTERACTING SUBSTRATE, 220-KDOMIM: SPASTIC PARAPLEGIA, INTELLECTUAL DISABILITY, NYSTAGMUS, AND OBESITY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KIDINS220  OMIM: KINASE D-INTERACTING SUBSTRATE, 220-KDOMIM: SPASTIC PARAPLEGIA, INTELLECTUAL DISABILITY, NYSTAGMUS, AND OBESITY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IGKV3D-11\t\n",
            "ZNF2\t\n",
            "ANKRD36C\t\n",
            "ADRA2B\t   From NCBI Gene:This intronless gene encodes a seven-pass transmembrane protein. This protein is a member of a subfamily of G protein-coupled receptors that regulate neurotransmitter release from sympathetic nerves and from adrenergic neurons in the central nervous system. [provided by RefSeq, Apr 2014]From UniProt:Alpha-2 adrenergic receptors mediate the catecholamine-induced inhibition of adenylate cyclase through the action of G proteins. The rank order of potency for agonists of this receptor is clonidine > norepinephrine > epinephrine = oxymetazoline > dopamine > p-tyramine = phenylephrine > serotonin > p-synephrine / p-octopamine. For antagonists, the rank order is yohimbine > chlorpromazine > phentolamine > mianserine > spiperone > prazosin > alprenolol > propanolol > pindolol.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Epilepsy, familial adult myoclonic 2From UniProt:Epilepsy, familial adult myoclonic, 2 (FAME2): A form of familial myoclonic epilepsy, a neurologic disorder characterized by cortical hand tremors, myoclonic jerks and occasional generalized or focal seizures with a non-progressive or very slowly progressive disease course. Usually, myoclonic tremor is the presenting symptom, characterized by tremulous finger movements and myoclonic jerks of the limbs increased by action and posture. In a minority of patients, seizures are the presenting symptom. Some patients exhibit mild cognitive impairment. FAME2 inheritance is autosomal dominant. [MIM:607876]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 2q11.2, which is the long (q) arm of chromosome 2 at position 11.2Molecular Location: base pairs 96,112,875 to 96,116,245 on chromosome 2 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADRA2L1ADRA2RL1ADRARL1alpha-2BARALPHA2BARFAME2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADRA2B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ALPHA-2B-ADRENERGIC RECEPTOROMIM: EPILEPSY, FAMILIAL ADULT MYOCLONIC, 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADRA2B  OMIM: ALPHA-2B-ADRENERGIC RECEPTOROMIM: EPILEPSY, FAMILIAL ADULT MYOCLONIC, 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ITPRIPL1\t\n",
            "FANCD2\t   From NCBI Gene:The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group D2. This protein is monoubiquinated in response to DNA damage, resulting in its localization to nuclear foci with other proteins (BRCA1 AND BRCA2) involved in homology-directed DNA repair. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2016]From UniProt:Required for maintenance of chromosomal stability. Promotes accurate and efficient pairing of homologs during meiosis. Involved in the repair of DNA double-strand breaks, both by homologous recombination and single-strand annealing. May participate in S phase and G2 phase checkpoint activation upon DNA damage. Plays a role in preventing breakage and loss of missegregating chromatin at the end of cell division, particularly after replication stress. Required for the targeting, or stabilization, of BLM to non-centromeric abnormal structures induced by replicative stress. Promotes BRCA2/FANCD1 loading onto damaged chromatin. May also be involved in B-cell immunoglobulin isotype switching.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Fanconi anemiaFrom NCBI Gene:Fanconi anemia, complementation group D2From UniProt:Fanconi anemia complementation group D2 (FANCD2): A disorder affecting all bone marrow elements and resulting in anemia, leukopenia and thrombopenia. It is associated with cardiac, renal and limb malformations, dermal pigmentary changes, and a predisposition to the development of malignancies. At the cellular level it is associated with hypersensitivity to DNA-damaging agents, chromosomal instability (increased chromosome breakage) and defective DNA repair. [MIM:227646]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p25.3, which is the short (p) arm of chromosome 3 at position 25.3Molecular Location: base pairs 10,026,387 to 10,101,937 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FA-D2FA4FACDFADFAD2FANCD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FANCD2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FANCD2 GENEOMIM: FANCONI ANEMIA, COMPLEMENTATION GROUP D2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FANCD2  OMIM: FANCD2 GENEOMIM: FANCONI ANEMIA, COMPLEMENTATION GROUP D2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IRAK2\t\n",
            "MYH15\t\n",
            "RETNLB\t\n",
            "PHLDB2\t\n",
            "SLC9C1\t\n",
            "ATG3\t\n",
            "CFAP44\t\n",
            "SIDT1\t\n",
            "USF3\t\n",
            "SYN2\t   From NCBI Gene:This gene is a member of the synapsin gene family. Synapsins encode neuronal phosphoproteins which associate with the cytoplasmic surface of synaptic vesicles. Family members are characterized by common protein domains, and they are implicated in synaptogenesis and the modulation of neurotransmitter release, suggesting a potential role in several neuropsychiatric diseases. This member of the synapsin family encodes a neuron-specific phosphoprotein that selectively binds to small synaptic vesicles in the presynaptic nerve terminal. Polymorphisms in this gene are associated with abnormal presynaptic function and related neuronal disorders, including autism, epilepsy, bipolar disorder and schizophrenia. Alternative splicing of this gene results in multiple transcript variants. The tissue inhibitor of metalloproteinase 4 gene is located within an intron of this gene and is transcribed in the opposite direction. [provided by RefSeq, Feb 2014]From UniProt:Neuronal phosphoprotein that coats synaptic vesicles, binds to the cytoskeleton, and is believed to function in the regulation of neurotransmitter release. May play a role in noradrenaline secretion by sympathetic neurons (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:SchizophreniaFrom NCBI Gene:SchizophreniaFrom UniProt:Schizophrenia (SCZD): A complex, multifactorial psychotic disorder or group of disorders characterized by disturbances in the form and content of thought (e.g. delusions, hallucinations), in mood (e.g. inappropriate affect), in sense of self and relationship to the external world (e.g. loss of ego boundaries, withdrawal), and in behavior (e.g bizarre or apparently purposeless behavior). Although it affects emotions, it is distinguished from mood disorders in which such disturbances are primary. Similarly, there may be mild impairment of cognitive function, and it is distinguished from the dementias in which disturbed cognitive function is considered primary. Some patients manifest schizophrenic as well as bipolar disorder symptoms and are often given the diagnosis of schizoaffective disorder. [MIM:181500]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p25.2, which is the short (p) arm of chromosome 3 at position 25.2Molecular Location: base pairs 12,004,360 to 12,192,032 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        SYNII     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SYN2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SCHIZOPHRENIAOMIM: SYNAPSIN II      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SYN2  OMIM: SCHIZOPHRENIAOMIM: SYNAPSIN II  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "POLQ\t\n",
            "SLC15A2\t\n",
            "KALRN\t   From NCBI Gene:Huntington's disease (HD), a neurodegenerative disorder characterized by loss of striatal neurons, is caused by an expansion of a polyglutamine tract in the HD protein huntingtin. This gene encodes a protein that interacts with the huntingtin-associated protein 1, which is a huntingtin binding protein that may function in vesicle trafficking. [provided by RefSeq, Apr 2016]From UniProt:Promotes the exchange of GDP by GTP. Activates specific Rho GTPase family members, thereby inducing various signaling mechanisms that regulate neuronal shape, growth, and plasticity, through their effects on the actin cytoskeleton. Induces lamellipodia independent of its GEF activity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Coronary heart disease 5From UniProt:Coronary heart disease 5 (CHDS5): A multifactorial disease characterized by an imbalance between myocardial functional requirements and the capacity of the coronary vessels to supply sufficient blood flow. Decreased capacity of the coronary vessels is often associated with thickening and loss of elasticity of the coronary arteries. [MIM:608901]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3q21.1-q21.2, which is the long (q) arm of chromosome 3 between positions 21.1 and 21.2Molecular Location: base pairs 124,033,341 to 124,726,325 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARHGEF24CHD5CHDS5DUETDUOHAPIPTRAD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KALRN      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CORONARY HEART DISEASE, SUSCEPTIBILITY TO, 5OMIM: KALIRIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KALRN  OMIM: CORONARY HEART DISEASE, SUSCEPTIBILITY TO, 5OMIM: KALIRIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MUC13\t\n",
            "HEG1\t\n",
            "SLC41A3\t\n",
            "ALDH1L1\t\n",
            "CHST13\t\n",
            "TMEM40\t\n",
            "MBD4\t\n",
            "IQSEC1\t\n",
            "COL6A5\t   From NCBI Gene:This gene encodes a member of the collagen superfamily of proteins. The encoded protein contains multiple von Willebrand factor A-like domains and may interact with the alpha 1 and alpha 2 chains of collagen VI to form the complete collagen VI trimer. Polymorphisms in this gene may be linked to dermal phenotypes, such as eczema. Alternative splicing results in multiple transcript variants. [provided by RefSeq, May 2013]From UniProt:Collagen VI acts as a cell-binding protein.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Patients affected by atopic dermatitis display an abnormal distribution of COL29A1 mRNA and protein in skin suggesting that COL29A1 may be involved in the pathogenesis of the disease.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3q22.1, which is the long (q) arm of chromosome 3 at position 22.1Molecular Location: base pairs 130,345,516 to 130,484,846 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        COL29A1VWA4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL6A5      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: COLLAGEN, TYPE VI, ALPHA-5      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COL6A5  OMIM: COLLAGEN, TYPE VI, ALPHA-5  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "COL6A6\t\n",
            "ASTE1\t\n",
            "NEK11\t\n",
            "DZIP1L\t\n",
            "A4GNT\t\n",
            "XRN1\t\n",
            "SLC6A6\t\n",
            "GRIP2\t\n",
            "FGD5\t\n",
            "GYG1\t   From NCBI Gene:This gene encodes a member of the glycogenin family. Glycogenin is a glycosyltransferase that catalyzes the formation of a short glucose polymer from uridine diphosphate glucose in an autoglucosylation reaction. This reaction is followed by elongation and branching of the polymer, catalyzed by glycogen synthase and branching enzyme, to form glycogen. This gene is expressed in muscle and other tissues. Mutations in this gene result in glycogen storage disease XV. This gene has pseudogenes on chromosomes 1, 8 and 13 respectively. Alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Sep 2010]From UniProt:Self-glucosylates, via an inter-subunit mechanism, to form an oligosaccharide primer that serves as substrate for glycogen synthase.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Glycogen storage disease XVPolyglucosan body myopathy 2From UniProt:Glycogen storage disease 15 (GSD15): A metabolic disorder resulting in muscle weakness, associated with the glycogen depletion in skeletal muscle, and cardiac arrhythmia, associated with the accumulation of abnormal storage material in the heart. The skeletal muscle shows a marked predominance of slow-twitch, oxidative muscle fibers and mitochondrial proliferation. [MIM:613507]Polyglucosan body myopathy 2 (PGBM2): A glycogen storage disease characterized by polyglucosan accumulation in muscle, and skeletal myopathy without cardiac involvement. Most patients manifest slowly progressive, hip girdle, shoulder girdle, and/or hand and leg muscle weakness. Polyglucosan contains abnormally long and poorly branched glucosyl chains and is variably resistant to digestion by alpha-amylase. [MIM:616199]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3q24, which is the long (q) arm of chromosome 3 at position 24Molecular Location: base pairs 148,991,408 to 149,027,669 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GSD15GYG     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GYG1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: GLYCOGEN STORAGE DISEASE XVOMIM: GLYCOGENIN 1OMIM: POLYGLUCOSAN BODY MYOPATHY 2      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GYG1  OMIM: GLYCOGEN STORAGE DISEASE XVOMIM: GLYCOGENIN 1OMIM: POLYGLUCOSAN BODY MYOPATHY 2  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IGSF10\t\n",
            "TMEM14EP\t\n",
            "OTOL1\t\n",
            "SI\t   The SI gene provides instructions for producing the enzyme sucrase-isomaltase.  This enzyme is found in the intestinal tract, where it is involved in breaking down the sugars sucrose (a sugar found in fruits, and also known as table sugar) and maltose (the sugar found in grains).  Sucrose and maltose are called disaccharides because they are each made up of two simple sugar molecules.   Disaccharides must be broken down into simple sugar molecules to be digested properly.The sucrase-isomaltase enzyme is found on the surface of the intestinal epithelial cells, which are cells that line the walls of the intestine. These cells have fingerlike projections called microvilli that absorb nutrients from food as it passes through the intestine. Based on their appearance, groups of these microvilli are known collectively as the brush border.  The role of the sucrase-isomaltase enzyme is to break down sucrose and maltose into simple sugars so that they can be absorbed by microvilli into intestinal epithelial cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Congenital sucrase-isomaltase deficiency     At least 10 mutations in the SI gene have been found to cause congenital sucrase-isomaltase deficiency.  These mutations disrupt the folding and processing of the sucrose-isomaltase enzyme, transportation of the enzyme within the intestinal epithelial cells, the orientation of the enzyme to the cell surface, or its normal functioning.  An impairment in any of these cell processes results in a sucrase-isomaltase enzyme that cannot effectively break down sucrose, maltose, or other compounds made from these sugar molecules (carbohydrates).  The inability to digest these sugars causes the intestinal discomfort seen in people with congenital sucrase-isomaltase deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 10 mutations in the SI gene have been found to cause congenital sucrase-isomaltase deficiency.  These mutations disrupt the folding and processing of the sucrose-isomaltase enzyme, transportation of the enzyme within the intestinal epithelial cells, the orientation of the enzyme to the cell surface, or its normal functioning.  An impairment in any of these cell processes results in a sucrase-isomaltase enzyme that cannot effectively break down sucrose, maltose, or other compounds made from these sugar molecules (carbohydrates).  The inability to digest these sugars causes the intestinal discomfort seen in people with congenital sucrase-isomaltase deficiency. More About This Health Condition    Cytogenetic Location: 3q26.1, which is the long (q) arm of chromosome 3 at position 26.1Molecular Location: base pairs 164,978,898 to 165,078,495 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MGC131621MGC131622Oligosaccharide alpha-1,6-glucosidasesucrase-isomaltase (alpha-glucosidase)SUIS_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Biochemistry (fifith edition, 2002): Sucrose, Lactose, and Maltose Are the Common DisaccharidesMolecular Cell Biology (fourth edition, 2000): The Intestinal Epithelium Is Highly PolarizedThe Cell: A Molecular Approach (second edition, 2000): Glucose transport by intestinal epithelial cells      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SI      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SUCRASE-ISOMALTASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifith edition, 2002): Sucrose, Lactose, and Maltose Are the Common DisaccharidesMolecular Cell Biology (fourth edition, 2000): The Intestinal Epithelium Is Highly PolarizedThe Cell: A Molecular Approach (second edition, 2000): Glucose transport by intestinal epithelial cells  Tests of SI  PubMed  SUCRASE-ISOMALTASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Jacob R, Zimmer KP, Schmitz J, Naim HY. Congenital sucrase-isomaltase deficiency arising from cleavage and secretion of a mutant form of the enzyme. J Clin Invest. 2000 Jul;106(2):281-7. Citation on PubMed or Free article on PubMed CentralKeiser M, Alfalah M, Prpsting MJ, Castelletti D, Naim HY. Altered folding, turnover, and polarized sorting act in concert to define a novel pathomechanism of congenital sucrase-isomaltase deficiency. J Biol Chem. 2006 May 19;281(20):14393-9. Epub 2006 Mar 16. Citation on PubMedRitz V, Alfalah M, Zimmer KP, Schmitz J, Jacob R, Naim HY. Congenital sucrase-isomaltase deficiency because of an accumulation of the mutant enzyme in the endoplasmic reticulum. Gastroenterology. 2003 Dec;125(6):1678-85. Citation on PubMedOMIM: SUCRASE-ISOMALTASESander P, Alfalah M, Keiser M, Korponay-Szabo I, Kovcs JB, Leeb T, Naim HY. Novel mutations in the human sucrase-isomaltase gene (SI) that cause congenital carbohydrate malabsorption. Hum Mutat. 2006 Jan;27(1):119. Citation on PubMed \n",
            "ZBBX\t\n",
            "LRRC34\t\n",
            "SAMD7\t\n",
            "NAALADL2\t\n",
            "KCNMB3\t\n",
            "HTR3D\t\n",
            "HTR3E\t\n",
            "MAGEF1\t\n",
            "KNG1\t   From NCBI Gene:This gene uses alternative splicing to generate two different proteins- high molecular weight kininogen (HMWK) and low molecular weight kininogen (LMWK). HMWK is essential for blood coagulation and assembly of the kallikrein-kinin system. Also, bradykinin, a peptide causing numerous physiological effects, is released from HMWK. Bradykinin also functions as an antimicrobial peptide with antibacterial and antifungal activity. In contrast to HMWK, LMWK is not involved in blood coagulation. Three transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Nov 2014]From UniProt:(1) Kininogens are inhibitors of thiol proteases; (2) HMW-kininogen plays an important role in blood coagulation by helping to position optimally prekallikrein and factor XI next to factor XII; (3) HMW-kininogen inhibits the thrombin- and plasmin-induced aggregation of thrombocytes; (4) the active peptide bradykinin that is released from HMW-kininogen shows a variety of physiological effects: (4A) influence in smooth muscle contraction, (4B) induction of hypotension, (4C) natriuresis and diuresis, (4D) decrease in blood glucose level, (4E) it is a mediator of inflammation and causes (4E1) increase in vascular permeability, (4E2) stimulation of nociceptors (4E3) release of other mediators of inflammation (e.g. prostaglandins), (4F) it has a cardioprotective effect (directly via bradykinin action, indirectly via endothelium-derived relaxing factor action); (5) LMW-kininogen inhibits the aggregation of thrombocytes; (6) LMW-kininogen is in contrast to HMW-kininogen not involved in blood clotting.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:High molecular weight kininogen deficiencyFrom UniProt:High molecular weight kininogen deficiency (HMWK deficiency): Autosomal recessive coagulation defect. Patients with HWMK deficiency do not have a hemorrhagic tendency, but they exhibit abnormal surface-mediated activation of fibrinolysis. [MIM:228960]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3q27.3, which is the long (q) arm of chromosome 3 at position 27.3Molecular Location: base pairs 186,717,309 to 186,744,410 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BDKBKHMWKKNG     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KNG1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HIGH MOLECULAR WEIGHT KININOGEN DEFICIENCYOMIM: KININOGEN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KNG1  OMIM: HIGH MOLECULAR WEIGHT KININOGEN DEFICIENCYOMIM: KININOGEN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GMNC\t\n",
            "ATP13A4\t\n",
            "TMEM44\t\n",
            "MUC20\t\n",
            "MUC4\t\n",
            "DLG1\t\n",
            "ZNF860\t\n",
            "PLCD1\t   From NCBI Gene:This gene encodes a member of the phospholipase C family. Phospholipase C isozymes play critical roles in intracellular signal transduction by catalyzing the hydrolysis of phosphatidylinositol 4,5-bisphosphate (PIP2) into the second messengers diacylglycerol (DAG) and inositol triphosphate (IP3). The encoded protein functions as a tumor suppressor in several types of cancer, and mutations in this gene are a cause of hereditary leukonychia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene. [provided by RefSeq, Dec 2011]From UniProt:The production of the second messenger molecules diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3) is mediated by activated phosphatidylinositol-specific phospholipase C enzymes. Essential for trophoblast and placental development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Leukonychia totalisFrom UniProt:Nail disorder, non-syndromic congenital, 3 (NDNC3): A nail disorder characterized by a white appearance of the nail plate (true leukonychia), the nail bed (pseudoleukonychia), or neither (apparent leukonychia). Leukonychia may involve all of the nail (leukonychia totalis) or only part of the nail (leukonychia partialis), or can appear as one or more transverse bands (leukonychia striata) or white spots (leukonychia punctata). [MIM:151600]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p22.2, which is the short (p) arm of chromosome 3 at position 22.2Molecular Location: base pairs 38,007,496 to 38,029,663 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        NDNC3PLC-III     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PLCD1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: NAIL DISORDER, NONSYNDROMIC CONGENITAL, 3OMIM: PHOSPHOLIPASE C, DELTA-1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PLCD1  OMIM: NAIL DISORDER, NONSYNDROMIC CONGENITAL, 3OMIM: PHOSPHOLIPASE C, DELTA-1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DLEC1\t   From NCBI Gene:The cytogenetic location of this gene is 3p21.3, and it is located in a region that is commonly deleted in a variety of malignancies. Down-regulation of this gene has been observed in several human cancers including lung, esophageal, renal tumors, and head and neck squamous cell carcinoma. In some cases, reduced expression of this gene in tumor cells is a result of aberrant promoter methylation. Several alternatively spliced transcripts have been observed that contain disrupted coding regions and likely encode nonfunctional proteins.[provided by RefSeq, Mar 2016]From UniProt:May act as a tumor suppressor by inhibiting cell proliferation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Lung cancerFrom UniProt:DLEC1 silencing due to promoter methylation and aberrant transcription are implicated in the development of different cancers, including esophageal (ESCR), renal and lung cancers (LNCR).Esophageal cancer (ESCR): A malignancy of the esophagus. The most common types are esophageal squamous cell carcinoma and adenocarcinoma. Cancer of the esophagus remains a devastating disease because it is usually not detected until it has progressed to an advanced incurable stage. [MIM:133239]Lung cancer (LNCR): A common malignancy affecting tissues of the lung. The most common form of lung cancer is non-small cell lung cancer (NSCLC) that can be divided into 3 major histologic subtypes: squamous cell carcinoma, adenocarcinoma, and large cell lung cancer. NSCLC is often diagnosed at an advanced stage and has a poor prognosis. [MIM:211980]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p22.2, which is the short (p) arm of chromosome 3 at position 22.2Molecular Location: base pairs 38,038,595 to 38,122,741 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CFAP81DLC-1DLC1F56     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DLEC1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: DELETED IN LUNG AND ESOPHAGEAL CANCER 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DLEC1  OMIM: DELETED IN LUNG AND ESOPHAGEAL CANCER 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TTC21A\t\n",
            "RPL14\t\n",
            "TRAK1\t\n",
            "CCDC13\t\n",
            "ZNF852\t\n",
            "SUMF1\t   The SUMF1 gene provides instructions for making an enzyme called formylglycine-generating enzyme (FGE). This enzyme is found in a cell structure called the endoplasmic reticulum, which is involved in protein processing and transport. The FGE enzyme modifies other enzymes called sulfatases, which aid in breaking down substances that contain chemical groups known as sulfates. These substances include a variety of sugars, fats, and hormones. Specifically, FGE converts a protein building block (amino acid) within sulfatases called cysteine into a molecule called C-alpha-formylglycine.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Multiple sulfatase deficiency     At least 35 mutations in the SUMF1 gene have been found to cause multiple sulfatase deficiency. This condition is apparent at birth or early childhood and is characterized by neurological decline, scaly skin (ichthyosis), and skeletal abnormalities. Most SUMF1 gene mutations that cause multiple sulfatase deficiency change single amino acids in the FGE enzyme. These changes severely reduce enzyme function or produce an unstable enzyme that is quickly broken down. The activity of multiple sulfatases is impaired because the FGE enzyme modifies all known sulfatase enzymes. Sulfate-containing molecules that are not broken down build up in cells, often resulting in cell death. The death of cells in particular tissues, specifically the brain, skeleton, and skin, cause many of the signs and symptoms of multiple sulfatase deficiency. Research indicates that mutations that lead to reduced FGE enzyme function are associated with the less severe cases of the condition, whereas mutations that lead to the production an of unstable FGE enzyme tend to be associated with the more severe cases of multiple sulfatase deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 35 mutations in the SUMF1 gene have been found to cause multiple sulfatase deficiency. This condition is apparent at birth or early childhood and is characterized by neurological decline, scaly skin (ichthyosis), and skeletal abnormalities. Most SUMF1 gene mutations that cause multiple sulfatase deficiency change single amino acids in the FGE enzyme. These changes severely reduce enzyme function or produce an unstable enzyme that is quickly broken down. The activity of multiple sulfatases is impaired because the FGE enzyme modifies all known sulfatase enzymes. Sulfate-containing molecules that are not broken down build up in cells, often resulting in cell death. The death of cells in particular tissues, specifically the brain, skeleton, and skin, cause many of the signs and symptoms of multiple sulfatase deficiency. Research indicates that mutations that lead to reduced FGE enzyme function are associated with the less severe cases of the condition, whereas mutations that lead to the production an of unstable FGE enzyme tend to be associated with the more severe cases of multiple sulfatase deficiency. More About This Health Condition    Cytogenetic Location: 3p26.1, which is the short (p) arm of chromosome 3 at position 26.1Molecular Location: base pairs 4,034,714 to 4,467,282 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AAPA3037C-alpha-formylglycine-generating enzyme 1FGEFGly-generating enzymesulfatase-modifying factor 1UNQ3037     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Essentials of Glycobiology (second edition, 2009): Proteoglycans and Sulfated Glycosaminoglycans      Clinical Information from GeneReviews (1 link)     Multiple Sulfatase Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SUMF1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SULFATASE-MODIFYING FACTOR 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Essentials of Glycobiology (second edition, 2009): Proteoglycans and Sulfated Glycosaminoglycans  Multiple Sulfatase Deficiency  Tests of SUMF1  PubMed  SULFATASE-MODIFYING FACTOR 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Annunziata I, Bouch V, Lombardi A, Settembre C, Ballabio A. Multiple sulfatase deficiency is due to hypomorphic mutations of the SUMF1 gene. Hum Mutat. 2007 Sep;28(9):928. Citation on PubMedCosma MP, Pepe S, Annunziata I, Newbold RF, Grompe M, Parenti G, Ballabio A. The multiple sulfatase deficiency gene encodes an essential and limiting factor for the activity of sulfatases. Cell. 2003 May 16;113(4):445-56. Citation on PubMedCosma MP, Pepe S, Parenti G, Settembre C, Annunziata I, Wade-Martins R, Di Domenico C, Di Natale P, Mankad A, Cox B, Uziel G, Mancini GM, Zammarchi E, Donati MA, Kleijer WJ, Filocamo M, Carrozzo R, Carella M, Ballabio A. Molecular and functional analysis of SUMF1 mutations in multiple sulfatase deficiency. Hum Mutat. 2004 Jun;23(6):576-81. Citation on PubMedDierks T, Dickmanns A, Preusser-Kunze A, Schmidt B, Mariappan M, von Figura K, Ficner R, Rudolph MG. Molecular basis for multiple sulfatase deficiency and mechanism for formylglycine generation of the human formylglycine-generating enzyme. Cell. 2005 May 20;121(4):541-52. Citation on PubMedDierks T, Schmidt B, Borissenko LV, Peng J, Preusser A, Mariappan M, von Figura K. Multiple sulfatase deficiency is caused by mutations in the gene encoding the human C(alpha)-formylglycine generating enzyme. Cell. 2003 May 16;113(4):435-44. Citation on PubMedOMIM: SULFATASE-MODIFYING FACTOR 1Schlotawa L, Ennemann EC, Radhakrishnan K, Schmidt B, Chakrapani A, Christen HJ, Moser H, Steinmann B, Dierks T, Grtner J. SUMF1 mutations affecting stability and activity of formylglycine generating enzyme predict clinical outcome in multiple sulfatase deficiency. Eur J Hum Genet. 2011 Mar;19(3):253-61. doi: 10.1038/ejhg.2010.219. Epub 2011 Jan 12. Citation on PubMed or Free article on PubMed CentralSchlotawa L, Radhakrishnan K, Baumgartner M, Schmid R, Schmidt B, Dierks T, Grtner J. Rapid degradation of an active formylglycine generating enzyme variant leads to a late infantile severe form of multiple sulfatase deficiency. Eur J Hum Genet. 2013 Sep;21(9):1020-3. doi: 10.1038/ejhg.2012.291. Epub 2013 Jan 16. Citation on PubMed or Free article on PubMed CentralSchlotawa L, Steinfeld R, von Figura K, Dierks T, Grtner J. Molecular analysis of SUMF1 mutations: stability and residual activity of mutant formylglycine-generating enzyme determine disease severity in multiple sulfatase deficiency. Hum Mutat. 2008 Jan;29(1):205. Citation on PubMed \n",
            "LTF\t\n",
            "TMIE\t   From NCBI Gene:This gene encodes a transmembrane inner ear protein. Studies in mouse suggest that this gene is required for normal postnatal maturation of sensory hair cells in the cochlea, including correct development of stereocilia bundles. This gene is one of multiple genes responsible for recessive non-syndromic deafness (DFNB), also known as autosomal recessive nonsyndromic hearing loss (ARNSHL), the most common form of congenitally acquired inherited hearing impairment. [provided by RefSeq, Mar 2009]From UniProt:Unknown. The protein may play some role in a cellular membrane location. May reside within an internal membrane compartment and function in pathways such as those involved in protein and/or vesicle trafficking. Alternatively, the mature protein may be localized in the plasma membrane and serve as a site of interaction for other molecules through its highly charged C-terminal domain.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 6From UniProt:Deafness, autosomal recessive, 6 (DFNB6): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. [MIM:600971]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p21.31, which is the short (p) arm of chromosome 3 at position 21.31Molecular Location: base pairs 46,693,778 to 46,710,923 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNB6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TMIE      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 6OMIM: TRANSMEMBRANE INNER EAR-EXPRESSED GENE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TMIE  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 6OMIM: TRANSMEMBRANE INNER EAR-EXPRESSED GENE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NBEAL2\t   The NBEAL2 gene provides instructions for making a protein whose function is unclear. The protein appears to be critical for the normal development of platelets, which are blood cell fragments involved in blood clotting.Platelets are produced in bone marrow, the spongy tissue in the center of long bones that produces most of the blood cells the body needs. Platelets are formed from large precursor cells known as megakaryocytes. Within these cells, the NBEAL2 protein is thought to play a role in the development of sacs called alpha-granules, which are the most abundant components of platelets. Alpha-granules contain growth factors and other proteins that are important for blood clotting and wound healing. In response to an injury that causes bleeding, the proteins stored in alpha-granules help platelets stick to one another to form a plug that seals off damaged blood vessels and prevents further blood loss.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Gray platelet syndrome     At least 35 mutations in the NBEAL2 gene have been found to cause gray platelet syndrome, a disorder associated with abnormal bleeding. Most people with gray platelet syndrome also develop a condition called myelofibrosis, which is characterized by the buildup of scar tissue (fibrosis) in the bone marrow that prevents it from making enough normal blood cells.Mutations in the NBEAL2 gene disrupt the normal production of alpha-granules in megakaryocytes. Without alpha-granules, platelets are abnormally large and fewer in number than usual (macrothrombocytopenia). The abnormal platelets also appear gray when viewed under a microscope, which gives this condition its name. A lack of alpha-granules impairs the clumping of platelets in response to injury, increasing the risk of abnormal bleeding. Myelofibrosis is thought to occur because the growth factors and other proteins that are normally packaged into alpha-granules leak out into the bone marrow. The proteins lead to fibrosis that affects the bone marrow's ability to make new blood cells. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 35 mutations in the NBEAL2 gene have been found to cause gray platelet syndrome, a disorder associated with abnormal bleeding. Most people with gray platelet syndrome also develop a condition called myelofibrosis, which is characterized by the buildup of scar tissue (fibrosis) in the bone marrow that prevents it from making enough normal blood cells.Mutations in the NBEAL2 gene disrupt the normal production of alpha-granules in megakaryocytes. Without alpha-granules, platelets are abnormally large and fewer in number than usual (macrothrombocytopenia). The abnormal platelets also appear gray when viewed under a microscope, which gives this condition its name. A lack of alpha-granules impairs the clumping of platelets in response to injury, increasing the risk of abnormal bleeding. Myelofibrosis is thought to occur because the growth factors and other proteins that are normally packaged into alpha-granules leak out into the bone marrow. The proteins lead to fibrosis that affects the bone marrow's ability to make new blood cells. More About This Health Condition    Cytogenetic Location: 3p21.31, which is the short (p) arm of chromosome 3 at position 21.31Molecular Location: base pairs 46,979,683 to 47,009,704 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BDPLT4GPSKIAA0540neurobeachin-like 2neurobeachin-like protein 2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     University of Washington: Platelet Activation      Tests Listed in the Genetic Testing Registry (1 link)     Tests of NBEAL2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     NEUROBEACHIN-LIKE 2      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      University of Washington: Platelet Activation  Tests of NBEAL2  PubMed  NEUROBEACHIN-LIKE 2  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Albers CA, Cvejic A, Favier R, Bouwmans EE, Alessi MC, Bertone P, Jordan G, Kettleborough RN, Kiddle G, Kostadima M, Read RJ, Sipos B, Sivapalaratnam S, Smethurst PA, Stephens J, Voss K, Nurden A, Rendon A, Nurden P, Ouwehand WH. Exome sequencing identifies NBEAL2 as the causative gene for gray platelet syndrome. Nat Genet. 2011 Jul 17;43(8):735-7. doi: 10.1038/ng.885. Citation on PubMed or Free article on PubMed CentralBottega R, Pecci A, De Candia E, Pujol-Moix N, Heller PG, Noris P, De Rocco D, Podda GM, Glembotsky AC, Cattaneo M, Balduini CL, Savoia A. Correlation between platelet phenotype and NBEAL2 genotype in patients with congenital thrombocytopenia and -granule deficiency. Haematologica. 2013 Jun;98(6):868-74. doi: 10.3324/haematol.2012.075861. Epub 2012 Oct 25. Citation on PubMed or Free article on PubMed CentralGunay-Aygun M, Falik-Zaccai TC, Vilboux T, Zivony-Elboum Y, Gumruk F, Cetin M, Khayat M, Boerkoel CF, Kfir N, Huang Y, Maynard D, Dorward H, Berger K, Kleta R, Anikster Y, Arat M, Freiberg AS, Kehrel BE, Jurk K, Cruz P, Mullikin JC, White JG, Huizing M, Gahl WA. NBEAL2 is mutated in gray platelet syndrome and is required for biogenesis of platelet -granules. Nat Genet. 2011 Jul 17;43(8):732-4. doi: 10.1038/ng.883. Citation on PubMed or Free article on PubMed CentralKahr WH, Hinckley J, Li L, Schwertz H, Christensen H, Rowley JW, Pluthero FG, Urban D, Fabbro S, Nixon B, Gadzinski R, Storck M, Wang K, Ryu GY, Jobe SM, Schutte BC, Moseley J, Loughran NB, Parkinson J, Weyrich AS, Di Paola J. Mutations in NBEAL2, encoding a BEACH protein, cause gray platelet syndrome. Nat Genet. 2011 Jul 17;43(8):738-40. doi: 10.1038/ng.884. Citation on PubMedOMIM: NEUROBEACHIN-LIKE 2 \n",
            "KIF9\t\n",
            "SMARCC1\t\n",
            "MAP4\t\n",
            "NME6\t\n",
            "COL7A1\t   The COL7A1 gene provides instructions for making proteins that are used to assemble type VII collagen.  Collagens are a family of proteins that strengthen and support connective tissues, such as skin, bone, tendons, and ligaments, throughout the body.  In particular, type VII collagen plays an essential role in strengthening and stabilizing the skin.The proteins produced from the COL7A1 gene, called pro-1(VII) chains, are the components of type VII collagen. Three pro-1(VII) chains twist together to form a triple-stranded, ropelike molecule known as a procollagen. Procollagen molecules are secreted by the cell and processed by enzymes to remove extra protein segments from the ends. Once these molecules are processed, they arrange themselves into long, thin bundles of mature type VII collagen.Type VII collagen is the major component of structures in the skin called anchoring fibrils.  These fibrils are found in a region known as the epidermal basement membrane zone, which is a two-layer membrane located between the top layer of skin, called the epidermis, and an underlying layer called the dermis.  Anchoring fibrils hold the two layers of skin together by connecting the epidermal basement membrane to the dermis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Dystrophic epidermolysis bullosa     More than 400 mutations in the COL7A1 gene have been identified in people with dystrophic epidermolysis bullosa.  These mutations alter the structure or disrupt the production of type VII collagen, which impairs the ability of anchoring fibrils to connect the epidermis to the dermis.  When type VII collagen is abnormal or missing, anchoring fibrils cannot form properly.  As a result, friction or other minor trauma can cause the two skin layers to separate.  This separation leads to the formation of blisters, which can result in extensive scarring as they heal.The autosomal recessive types of dystrophic epidermolysis bullosa (RDEB) result from mutations in both copies of the COL7A1 gene in each cell.  The most severe, classic form of this disorder is known as the Hallopeau-Siemens type (RDEB-HS).  Most of the COL7A1 mutations responsible for RDEB-HS significantly reduce or eliminate the production of type VII collagen.  As a result, few or no anchoring fibrils are present to connect the epidermis with the dermis.  This lack of anchoring fibrils causes the severe signs and symptoms of RDEB-HS.  A somewhat less severe form of autosomal recessive dystrophic epidermolysis bullosa, known as the non-Hallopeau-Siemens type (Non-HS RDEB), is caused by other types of mutations.  These genetic changes allow a small amount of normal or partially functional type VII collagen to be produced.A milder, autosomal dominant form of dystrophic epidermolysis bullosa (DDEB) results from mutations in one copy of the COL7A1 gene in each cell.  In many cases, these mutations occur in a part of type VII collagen known as the collagenous region.  This region gives type VII collagen its usual triple-stranded structure.  It is made up of a pattern of protein building blocks (amino acids) in which every third amino acid is a glycine.  Mutations that substitute other amino acids for glycine in this region can disrupt the triple-stranded structure of type VII collagen.  When the abnormally shaped collagen molecules are incorporated into anchoring fibrils, they interfere with the fibrils' normal function and prevent them from effectively connecting the epidermis and the dermis.  DDEB can also be caused by other types of mutations, particularly changes that affect the folding of type VII collagen.It is unclear how COL7A1 mutations are associated with an increased risk of certain cancers, particularly squamous cell carcinoma, in people with severe types of dystrophic epidermolysis bullosa.  Some research has suggested that abnormal forms of type VII collagen that retain a fragment called the NC1 domain may increase the risk of tumor formation.  Other studies, however, have not found this association. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 400 mutations in the COL7A1 gene have been identified in people with dystrophic epidermolysis bullosa.  These mutations alter the structure or disrupt the production of type VII collagen, which impairs the ability of anchoring fibrils to connect the epidermis to the dermis.  When type VII collagen is abnormal or missing, anchoring fibrils cannot form properly.  As a result, friction or other minor trauma can cause the two skin layers to separate.  This separation leads to the formation of blisters, which can result in extensive scarring as they heal.The autosomal recessive types of dystrophic epidermolysis bullosa (RDEB) result from mutations in both copies of the COL7A1 gene in each cell.  The most severe, classic form of this disorder is known as the Hallopeau-Siemens type (RDEB-HS).  Most of the COL7A1 mutations responsible for RDEB-HS significantly reduce or eliminate the production of type VII collagen.  As a result, few or no anchoring fibrils are present to connect the epidermis with the dermis.  This lack of anchoring fibrils causes the severe signs and symptoms of RDEB-HS.  A somewhat less severe form of autosomal recessive dystrophic epidermolysis bullosa, known as the non-Hallopeau-Siemens type (Non-HS RDEB), is caused by other types of mutations.  These genetic changes allow a small amount of normal or partially functional type VII collagen to be produced.A milder, autosomal dominant form of dystrophic epidermolysis bullosa (DDEB) results from mutations in one copy of the COL7A1 gene in each cell.  In many cases, these mutations occur in a part of type VII collagen known as the collagenous region.  This region gives type VII collagen its usual triple-stranded structure.  It is made up of a pattern of protein building blocks (amino acids) in which every third amino acid is a glycine.  Mutations that substitute other amino acids for glycine in this region can disrupt the triple-stranded structure of type VII collagen.  When the abnormally shaped collagen molecules are incorporated into anchoring fibrils, they interfere with the fibrils' normal function and prevent them from effectively connecting the epidermis and the dermis.  DDEB can also be caused by other types of mutations, particularly changes that affect the folding of type VII collagen.It is unclear how COL7A1 mutations are associated with an increased risk of certain cancers, particularly squamous cell carcinoma, in people with severe types of dystrophic epidermolysis bullosa.  Some research has suggested that abnormal forms of type VII collagen that retain a fragment called the NC1 domain may increase the risk of tumor formation.  Other studies, however, have not found this association. More About This Health Condition    Cytogenetic Location: 3p21.31, which is the short (p) arm of chromosome 3 at position 21.31Molecular Location: base pairs 48,564,073 to 48,595,302 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        alpha 1 type VII collagenCO7A1_HUMANcollagen type VII alpha 1collagen VII, alpha-1 polypeptidecollagen, type VII, alpha 1collagen, type VII, alpha 1 (epidermolysis bullosa, dystrophic, dominant and recessive)EBD1EBDCTEBR1LC collagenlong chain collagen     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (fourth edition, 2002): Collagens Are the Major Proteins of the Extracellular Matrix      Clinical Information from GeneReviews (1 link)     Dystrophic Epidermolysis Bullosa      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL7A1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     COLLAGEN, TYPE VII, ALPHA-1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Collagens Are the Major Proteins of the Extracellular Matrix  Dystrophic Epidermolysis Bullosa  Tests of COL7A1  PubMed  COLLAGEN, TYPE VII, ALPHA-1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bruckner-Tuderman L, Hpfner B, Hammami-Hauasli N. Biology of anchoring fibrils: lessons from dystrophic epidermolysis bullosa. Matrix Biol. 1999 Feb;18(1):43-54. Review. Citation on PubMedDang N, Klingberg S, Marr P, Murrell DF. Review of collagen VII sequence variants found in Australasian patients with dystrophic epidermolysis bullosa reveals nine novel COL7A1 variants. J Dermatol Sci. 2007 Jun;46(3):169-78. Epub 2007 Apr 10. Review. Citation on PubMedGardella R, Castiglia D, Posteraro P, Bernardini S, Zoppi N, Paradisi M, Tadini G, Barlati S, McGrath JA, Zambruno G, Colombi M. Genotype-phenotype correlation in italian patients with dystrophic epidermolysis bullosa. J Invest Dermatol. 2002 Dec;119(6):1456-62. Citation on PubMedJrvikallio A, Pulkkinen L, Uitto J. Molecular basis of dystrophic epidermolysis bullosa: mutations in the type VII collagen gene (COL7A1). Hum Mutat. 1997;10(5):338-47. Citation on PubMedKern JS, Kohlhase J, Bruckner-Tuderman L, Has C. Expanding the COL7A1 mutation database: novel and recurrent mutations and unusual genotype-phenotype constellations in 41 patients with dystrophic epidermolysis bullosa. J Invest Dermatol. 2006 May;126(5):1006-12. Citation on PubMedOrtiz-Urda S, Garcia J, Green CL, Chen L, Lin Q, Veitch DP, Sakai LY, Lee H, Marinkovich MP, Khavari PA. Type VII collagen is required for Ras-driven human epidermal tumorigenesis. Science. 2005 Mar 18;307(5716):1773-6. Citation on PubMedPfendner EG, Lucky AW. Dystrophic Epidermolysis Bullosa. 2006 Aug 21 [updated 2015 Feb 26]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1304/ Citation on PubMedPourreyron C, Cox G, Mao X, Volz A, Baksh N, Wong T, Fassihi H, Arita K, O'Toole EA, Ocampo-Candiani J, Chen M, Hart IR, Bruckner-Tuderman L, Salas-Alanis JC, McGrath JA, Leigh IM, South AP. Patients with recessive dystrophic epidermolysis bullosa develop squamous-cell carcinoma regardless of type VII collagen expression. J Invest Dermatol. 2007 Oct;127(10):2438-44. Epub 2007 May 10. Citation on PubMedSawamura D, Goto M, Yasukawa K, Sato-Matsumura K, Nakamura H, Ito K, Nakamura H, Tomita Y, Shimizu H. Genetic studies of 20 Japanese families of dystrophic epidermolysis bullosa. J Hum Genet. 2005;50(10):543-6. Epub 2005 Sep 28. Erratum in: J Hum Genet. 2006;51(9):839. Citation on PubMedVarki R, Sadowski S, Uitto J, Pfendner E. Epidermolysis bullosa. II. Type VII collagen mutations and phenotype-genotype correlations in the dystrophic subtypes. J Med Genet. 2007 Mar;44(3):181-92. Epub 2006 Sep 13. Citation on PubMed or Free article on PubMed Central \n",
            "SLC26A6\t\n",
            "USP19\t\n",
            "GPX1\t   From NCBI Gene:The protein encoded by this gene belongs to the glutathione peroxidase family, members of which catalyze the reduction of organic hydroperoxides and hydrogen peroxide (H2O2) by glutathione, and thereby protect cells against oxidative damage. Other studies indicate that H2O2 is also essential for growth-factor mediated signal transduction, mitochondrial function, and maintenance of thiol redox-balance; therefore, by limiting H2O2 accumulation, glutathione peroxidases are also involved in modulating these processes. Several isozymes of this gene family exist in vertebrates, which vary in cellular location and substrate specificity. This isozyme is the most abundant, is ubiquitously expressed and localized in the cytoplasm, and whose preferred substrate is hydrogen peroxide. It is also a selenoprotein, containing the rare amino acid selenocysteine (Sec) at its active site. Sec is encoded by the UGA codon, which normally signals translation termination. The 3' UTRs of selenoprotein mRNAs contain a conserved stem-loop structure, designated the Sec insertion sequence (SECIS) element, that is necessary for the recognition of UGA as a Sec codon, rather than as a stop signal. This gene contains an in-frame GCG trinucleotide repeat in the coding region, and three alleles with 4, 5 or 6 repeats have been found in the human population. The allele with 4 GCG repeats has been significantly associated with breast cancer risk in premenopausal women. Alternatively spliced transcript variants have been found for this gene. Pseudogenes of this locus have been identified on chromosomes X and 21. [provided by RefSeq, Aug 2017]From UniProt:Protects the hemoglobin in erythrocytes from oxidative breakdown.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Glutathione peroxidase deficiency     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p21.31, which is the short (p) arm of chromosome 3 at position 21.31Molecular Location: base pairs 49,357,171 to 49,358,600 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GPXDGSHPX1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GPX1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GLUTATHIONE PEROXIDASE 1OMIM: GLUTATHIONE PEROXIDASE DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GPX1  OMIM: GLUTATHIONE PEROXIDASE 1OMIM: GLUTATHIONE PEROXIDASE DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RNF123\t\n",
            "RBM5\t\n",
            "SLC38A3\t\n",
            "SEMA3B\t\n",
            "C3orf18\t\n",
            "GLYCTK\t   From NCBI Gene:This locus encodes a member of the glycerate kinase type-2 family. The encoded enzyme catalyzes the phosphorylation of (R)-glycerate and may be involved in serine degradation and fructose metabolism. Decreased activity of the encoded enzyme may be associated with the disease D-glyceric aciduria. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jan 2009]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Deficiency of glycerate kinaseFrom UniProt:D-glyceric aciduria (D-GA): A rare metabolic disease characterized by chronic metabolic acidosis and a highly variable clinical phenotype. Clinical features range from an encephalopathic presentation with seizures, microcephaly, severe mental retardation and early death, to milder manifestations with only speech delay or even normal development. [MIM:220120]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p21.2, which is the short (p) arm of chromosome 3 at position 21.2Molecular Location: base pairs 52,287,558 to 52,295,256 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HBeAgBP4AHBEBP2HBEBP4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GLYCTK      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: D-GLYCERIC ACIDURIAOMIM: GLYCERATE KINASE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GLYCTK  OMIM: D-GLYCERIC ACIDURIAOMIM: GLYCERATE KINASE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NEK4\t\n",
            "ITIH1\t\n",
            "ITIH4\t   From NCBI Gene:The protein encoded by this gene is secreted into the blood, where it is cleaved by plasma kallikrein into two smaller forms. Expression of this gene has been detected only in liver, and it seems to be upregulated during surgical trauma. This gene is part of a cluster of similar genes on chromosome 3. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2009]From UniProt:Type II acute-phase protein (APP) involved in inflammatory responses to trauma. May also play a role in liver development or regeneration.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Familial hypercholesterolemia     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p21.1, which is the short (p) arm of chromosome 3 at position 21.1Molecular Location: base pairs 52,812,990 to 52,830,701 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GP120H4PIHRPITI-HC4ITIHL1PK-120PK120     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ITIH4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HYPERCHOLESTEROLEMIA, FAMILIALOMIM: INTER-ALPHA-TRYPSIN INHIBITOR, HEAVY CHAIN 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ITIH4  OMIM: HYPERCHOLESTEROLEMIA, FAMILIALOMIM: INTER-ALPHA-TRYPSIN INHIBITOR, HEAVY CHAIN 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DCP1A\t\n",
            "CCDC66\t\n",
            "DNAH12\t\n",
            "FLNB\t   The FLNB gene provides instructions for making a protein called filamin B. This protein helps build the network of protein filaments (cytoskeleton) that gives structure to cells and allows them to change shape and move. Filamin B attaches (binds) to another protein called actin and helps the actin to form the branching network of filaments that makes up the cytoskeleton. It also links actin to many other proteins to perform various functions within the cell, including the cell signaling that helps determine how the cytoskeleton will change as tissues grow and take shape during development.Filamin B is involved in the development of the skeleton before birth. It is active (expressed) in many cells and tissues of the body, including cartilage-forming cells called chondrocytes. Cartilage is a tough, flexible tissue that makes up much of the skeleton during early development. Most cartilage is later converted to bone (a process called ossification), except for the cartilage that continues to cover and protect the ends of bones and is present in the nose, airways (trachea and bronchi), and external ears. Filamin B appears to be important for normal cell growth and division (proliferation) and maturation (differentiation) of chondrocytes and for the ossification of cartilage.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Atelosteogenesis type 1     At least seven FLNB gene mutations have been identified that cause atelosteogenesis type 1, a disorder that affects the development of bones throughout the body. The mutations change single protein building blocks (amino acids) in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of atelosteogenesis type 1. More About This Health Condition      Atelosteogenesis type 3     At least six FLNB gene mutations have been identified that cause atelosteogenesis type 3, a disorder that affects the development of bones throughout the body. These mutations change single amino acids in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of atelosteogenesis type 3. More About This Health Condition      Boomerang dysplasia     At least two FLNB gene mutations have been identified that cause boomerang dysplasia, a disorder that affects the development of bones throughout the body. These mutations change single amino acids in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of boomerang dysplasia. More About This Health Condition      Larsen syndrome     At least 13 FLNB gene mutations have been identified that cause Larsen syndrome, a disorder that affects the development of bones throughout the body. These mutations change single amino acids in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of Larsen syndrome.It is not clear why similar mutations in the FLNB gene can result in four different disorders: atelosteogenesis type 1, atelosteogenesis type 3, boomerang dysplasia, or Larsen syndrome. Certain mutations in regions of the FLNB gene known as exons 2 through 5 seem to have the most harmful effects, usually resulting in the more severe disorders, atelosteogenesis type 1 and boomerang dysplasia. Atelosteogenesis type 3 and Larsen syndrome, which are less severe, are usually caused by apparently milder mutations in exons 2 through 5 or by mutations in exons 27 through 33. In a few cases, the same mutation has been associated with more than one of these disorders in different people. More About This Health Condition      Spondylocarpotarsal synostosis syndrome     At least five FLNB gene mutations have been identified that cause spondylocarpotarsal synostosis syndrome, a disorder that affects the development of bones throughout the body. These mutations, which may occur in any region of the gene, result in the production of an abnormally short filamin B protein that is unstable and breaks down rapidly. Loss of the filamin B protein appears to result in out-of-place (ectopic) ossification, resulting in fusion of the bones in the spine, wrists, and ankles and other signs and symptoms of spondylocarpotarsal synostosis syndrome. More About This Health Condition      Other disorders     Several common variations (polymorphisms) in the FLNB gene have been associated with low bone mineral density (osteoporosis), which weakens the bones and makes them prone to fracture. FLNB gene variations may affect the maintenance of bone structure throughout the lifespan and result in differences in bone mineral density.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least seven FLNB gene mutations have been identified that cause atelosteogenesis type 1, a disorder that affects the development of bones throughout the body. The mutations change single protein building blocks (amino acids) in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of atelosteogenesis type 1. More About This Health Condition  At least six FLNB gene mutations have been identified that cause atelosteogenesis type 3, a disorder that affects the development of bones throughout the body. These mutations change single amino acids in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of atelosteogenesis type 3. More About This Health Condition  At least two FLNB gene mutations have been identified that cause boomerang dysplasia, a disorder that affects the development of bones throughout the body. These mutations change single amino acids in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of boomerang dysplasia. More About This Health Condition  At least 13 FLNB gene mutations have been identified that cause Larsen syndrome, a disorder that affects the development of bones throughout the body. These mutations change single amino acids in the filamin B protein or delete a small section of the protein sequence, resulting in an abnormal protein. This abnormal protein appears to have a new, atypical function that interferes with normal proliferation or differentiation of chondrocytes, impairing ossification and leading to the signs and symptoms of Larsen syndrome.It is not clear why similar mutations in the FLNB gene can result in four different disorders: atelosteogenesis type 1, atelosteogenesis type 3, boomerang dysplasia, or Larsen syndrome. Certain mutations in regions of the FLNB gene known as exons 2 through 5 seem to have the most harmful effects, usually resulting in the more severe disorders, atelosteogenesis type 1 and boomerang dysplasia. Atelosteogenesis type 3 and Larsen syndrome, which are less severe, are usually caused by apparently milder mutations in exons 2 through 5 or by mutations in exons 27 through 33. In a few cases, the same mutation has been associated with more than one of these disorders in different people. More About This Health Condition  At least five FLNB gene mutations have been identified that cause spondylocarpotarsal synostosis syndrome, a disorder that affects the development of bones throughout the body. These mutations, which may occur in any region of the gene, result in the production of an abnormally short filamin B protein that is unstable and breaks down rapidly. Loss of the filamin B protein appears to result in out-of-place (ectopic) ossification, resulting in fusion of the bones in the spine, wrists, and ankles and other signs and symptoms of spondylocarpotarsal synostosis syndrome. More About This Health Condition  Several common variations (polymorphisms) in the FLNB gene have been associated with low bone mineral density (osteoporosis), which weakens the bones and makes them prone to fracture. FLNB gene variations may affect the maintenance of bone structure throughout the lifespan and result in differences in bone mineral density.    Cytogenetic Location: 3p14.3, which is the short (p) arm of chromosome 3 at position 14.3Molecular Location: base pairs 58,008,400 to 58,172,255 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABP-278ABP-280 homologactin-binding-like proteinactin binding protein 278beta-filaminFH1filamin-3filamin-Bfilamin B, betafilamin homolog 1FLN-BFLN1LFLNB_HUMANLRS1TABPTAPthyroid autoantigentruncated ABPtruncated actin-binding protein     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Developmental Biology (6th edition, 2000): Osteogenesis: The Development of Bones      Clinical Information from GeneReviews (1 link)     FLNB-Related Disorders      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FLNB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FILAMIN B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (6th edition, 2000): Osteogenesis: The Development of Bones  FLNB-Related Disorders  Tests of FLNB  PubMed  FILAMIN B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bicknell LS, Farrington-Rock C, Shafeghati Y, Rump P, Alanay Y, Alembik Y, Al-Madani N, Firth H, Karimi-Nejad MH, Kim CA, Leask K, Maisenbacher M, Moran E, Pappas JG, Prontera P, de Ravel T, Fryns JP, Sweeney E, Fryer A, Unger S, Wilson LC, Lachman RS, Rimoin DL, Cohn DH, Krakow D, Robertson SP. A molecular and clinical study of Larsen syndrome caused by mutations in FLNB. J Med Genet. 2007 Feb;44(2):89-98. Epub 2006 Jun 26. Citation on PubMed or Free article on PubMed CentralBicknell LS, Morgan T, Bonaf L, Wessels MW, Bialer MG, Willems PJ, Cohn DH, Krakow D, Robertson SP. Mutations in FLNB cause boomerang dysplasia. J Med Genet. 2005 Jul;42(7):e43. Citation on PubMed or Free article on PubMed CentralOMIM: FILAMIN BFarrington-Rock C, Firestein MH, Bicknell LS, Superti-Furga A, Bacino CA, Cormier-Daire V, Le Merrer M, Baumann C, Roume J, Rump P, Verheij JB, Sweeney E, Rimoin DL, Lachman RS, Robertson SP, Cohn DH, Krakow D. Mutations in two regions of FLNB result in atelosteogenesis I and III. Hum Mutat. 2006 Jul;27(7):705-10. Citation on PubMedIsidor B, Cormier-Daire V, Le Merrer M, Lefrancois T, Hamel A, Le Caignec C, David A, Jacquemont S. Autosomal dominant spondylocarpotarsal synostosis syndrome: phenotypic homogeneity and genetic heterogeneity. Am J Med Genet A. 2008 Jun 15;146A(12):1593-7. doi: 10.1002/ajmg.a.32217. Citation on PubMedKrakow D, Robertson SP, King LM, Morgan T, Sebald ET, Bertolotto C, Wachsmann-Hogiu S, Acuna D, Shapiro SS, Takafuta T, Aftimos S, Kim CA, Firth H, Steiner CE, Cormier-Daire V, Superti-Furga A, Bonafe L, Graham JM Jr, Grix A, Bacino CA, Allanson J, Bialer MG, Lachman RS, Rimoin DL, Cohn DH. Mutations in the gene encoding filamin B disrupt vertebral segmentation, joint formation and skeletogenesis. Nat Genet. 2004 Apr;36(4):405-10. Epub 2004 Feb 29. Citation on PubMedLi GH, Kung AW, Huang QY. Common variants in FLNB/CRTAP, not ARHGEF3 at 3p, are associated with osteoporosis in southern Chinese women. Osteoporos Int. 2010 Jun;21(6):1009-20. doi: 10.1007/s00198-009-1043-6. Epub 2009 Sep 1. Citation on PubMed or Free article on PubMed CentralSawyer GM, Clark AR, Robertson SP, Sutherland-Smith AJ. Disease-associated substitutions in the filamin B actin binding domain confer enhanced actin binding affinity in the absence of major structural disturbance: Insights from the crystal structures of filamin B actin binding domains. J Mol Biol. 2009 Jul 31;390(5):1030-47. doi: 10.1016/j.jmb.2009.06.009. Epub 2009 Jun 6. Citation on PubMedWilson SG, Jones MR, Mullin BH, Dick IM, Richards JB, Pastinen TM, Grundberg E, Ljunggren O, Surdulescu GL, Dudbridge F, Elliott KS, Cervino AC, Spector TD, Prince RL. Common sequence variation in FLNB regulates bone structure in women in the general population and FLNB mRNA expression in osteoblasts in vitro. J Bone Miner Res. 2009 Dec;24(12):1989-97. doi: 10.1359/jbmr.090530. Citation on PubMedYang CF, Wang CH, Siong H'ng W, Chang CP, Lin WD, Chen YT, Wu JY, Tsai FJ. Filamin B Loss-of-Function Mutation in Dimerization Domain Causes Autosomal-Recessive Spondylocarpotarsal Synostosis Syndrome with Rib Anomalies. Hum Mutat. 2017 May;38(5):540-547. doi: 10.1002/humu.23186. Epub 2017 Feb 27. Citation on PubMedZhang D, Herring JA, Swaney SS, McClendon TB, Gao X, Browne RH, Rathjen KE, Johnston CE, Harris S, Cain NM, Wise CA. Mutations responsible for Larsen syndrome cluster in the FLNB protein. J Med Genet. 2006 May;43(5):e24. Citation on PubMed or Free article on PubMed Central \n",
            "FAM107A\t\n",
            "ATXN7\t   From NCBI Gene:The autosomal dominant cerebellar ataxias (ADCA) are a heterogeneous group of neurodegenerative disorders characterized by progressive degeneration of the cerebellum, brain stem and spinal cord. Clinically, ADCA has been divided into three groups: ADCA types I-III. ADCAI is genetically heterogeneous, with five genetic loci, designated spinocerebellar ataxia (SCA) 1, 2, 3, 4 and 6, being assigned to five different chromosomes. ADCAII, which always presents with retinal degeneration (SCA7), and ADCAIII often referred to as the 'pure' cerebellar syndrome (SCA5), are most likely homogeneous disorders. Several SCA genes have been cloned and shown to contain CAG repeats in their coding regions. ADCA is caused by the expansion of the CAG repeats, producing an elongated polyglutamine tract in the corresponding protein. The expanded repeats are variable in size and unstable, usually increasing in size when transmitted to successive generations. This locus has been mapped to chromosome 3, and it has been determined that the diseased allele associated with spinocerebellar ataxia-7 contains 37-306 CAG repeats (near the N-terminus), compared to 4-35 in the normal allele. The encoded protein is a component of the SPT3/TAF9/GCN5 acetyltransferase (STAGA) and TBP-free TAF-containing (TFTC) chromatin remodeling complexes, and it thus plays a role in transcriptional regulation. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2016]From UniProt:Acts as component of the STAGA transcription coactivator-HAT complex. Mediates the interaction of STAGA complex with the CRX and is involved in CRX-dependent gene activation. Necessary for microtubule cytoskeleton stabilization.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spinocerebellar ataxia 7From UniProt:Spinocerebellar ataxia 7 (SCA7): Spinocerebellar ataxia is a clinically and genetically heterogeneous group of cerebellar disorders. Patients show progressive incoordination of gait and often poor coordination of hands, speech and eye movements, due to degeneration of the cerebellum with variable involvement of the brainstem and spinal cord. SCA7 belongs to the autosomal dominant cerebellar ataxias type II (ADCA II) which are characterized by cerebellar ataxia with retinal degeneration and pigmentary macular dystrophy. [MIM:164500]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 3p14.1, which is the short (p) arm of chromosome 3 at position 14.1Molecular Location: base pairs 63,864,557 to 64,003,462 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADCAIIOPCA3SCA7SGF73     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATXN7      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ATAXIN 7OMIM: SPINOCEREBELLAR ATAXIA 7      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ATXN7  OMIM: ATAXIN 7OMIM: SPINOCEREBELLAR ATAXIA 7  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MAGI1\t\n",
            "SUCLG2\t\n",
            "FRMD4B\t\n",
            "ZNF717\t\n",
            "CRYBG3\t\n",
            "OR5AC2\t\n",
            "OR5H14\t\n",
            "OR5H15\t\n",
            "OR5H6\t\n",
            "OR5K4\t\n",
            "OR5K3\t\n",
            "CPOX\t   The CPOX gene provides instructions for making an enzyme known as coproporphyrinogen oxidase. This enzyme is involved in the production of a molecule called heme. Heme is vital for all of the body's organs, although it is most abundant in the blood, bone marrow, and liver. Heme is an essential component of iron-containing proteins called hemoproteins, including hemoglobin (the protein that carries oxygen in the blood).The production of heme is a multi-step process that requires eight different enzymes. Coproporphyrinogen oxidase is responsible for the sixth step in this process, the removal of carbon and oxygen atoms from coproporphyrinogen III (the product of the fifth step) to form protoporphyrinogen IX. In subsequent steps, two other enzymes modify protoporphyrinogen IX and incorporate an iron atom to produce heme.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Porphyria     At least 45 mutations in the CPOX gene have been found to cause porphyria.  Mutations in this gene can cause two types of porphyria: hereditary coproporphyria and a variant known as harderoporphyria.Most CPOX gene mutations change single protein building blocks (amino acids) in coproporphyrinogen oxidase. A single mutation appears to be responsible for harderoporphyria; this genetic change replaces the amino acid glycine with the amino acid glutamic acid at position 404 (written as Lys404Glu or K404E).  Mutations in the CPOX gene reduce the activity of coproporphyrinogen oxidase, allowing compounds called porphyrins to build up in the body. These compounds are formed during the normal process of heme production, but reduced activity of coproporphyrinogen oxidase allows them to accumulate to toxic levels. This buildup, in combination with nongenetic factors such as certain drugs, alcohol, and dieting, leads to the signs and symptoms of hereditary coproporphyria and harderoporphyria. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 45 mutations in the CPOX gene have been found to cause porphyria.  Mutations in this gene can cause two types of porphyria: hereditary coproporphyria and a variant known as harderoporphyria.Most CPOX gene mutations change single protein building blocks (amino acids) in coproporphyrinogen oxidase. A single mutation appears to be responsible for harderoporphyria; this genetic change replaces the amino acid glycine with the amino acid glutamic acid at position 404 (written as Lys404Glu or K404E).  Mutations in the CPOX gene reduce the activity of coproporphyrinogen oxidase, allowing compounds called porphyrins to build up in the body. These compounds are formed during the normal process of heme production, but reduced activity of coproporphyrinogen oxidase allows them to accumulate to toxic levels. This buildup, in combination with nongenetic factors such as certain drugs, alcohol, and dieting, leads to the signs and symptoms of hereditary coproporphyria and harderoporphyria. More About This Health Condition    Cytogenetic Location: 3q11.2, which is the long (q) arm of chromosome 3 at position 11.2Molecular Location: base pairs 98,569,837 to 98,593,684 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        Coprogen oxidaseCoproporphyrinogen III oxidase, mitochondrialCoproporphyrinogen III Oxidasescoproporphyrinogen oxidase (coproporphyria, harderoporphyria)Coproporphyrinogen:oxygen oxidoreductase (decarboxylating)CoproporphyrinogenaseCOXCPOCPXHCPHEM6_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Mammalian Porphyrins Are Synthesized from Glycine and Succinyl Coenzyme A      Clinical Information from GeneReviews (1 link)     Hereditary Coproporphyria      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CPOX      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     COPROPORPHYRINOGEN OXIDASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Mammalian Porphyrins Are Synthesized from Glycine and Succinyl Coenzyme A  Hereditary Coproporphyria  Tests of CPOX  PubMed  COPROPORPHYRINOGEN OXIDASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Badminton MN, Elder GH. Molecular mechanisms of dominant expression in porphyria. J Inherit Metab Dis. 2005;28(3):277-86. Review. Citation on PubMedElder GH. Genetic defects in the porphyrias: types and significance. Clin Dermatol. 1998 Mar-Apr;16(2):225-33. Review. Citation on PubMedGouya L, Puy H, Robreau AM, Lyoumi S, Lamoril J, Da Silva V, Grandchamp B, Deybach JC. Modulation of penetrance by the wild-type allele in dominantly inherited erythropoietic protoporphyria and acute hepatic porphyrias. Hum Genet. 2004 Feb;114(3):256-62. Epub 2003 Dec 11. Citation on PubMedKauppinen R. Porphyrias. Lancet. 2005 Jan 15-21;365(9455):241-52. Review. Citation on PubMedLamoril J, Puy H, Whatley SD, Martin C, Woolf JR, Da Silva V, Deybach JC, Elder GH. Characterization of mutations in the CPO gene in British patients demonstrates absence of genotype-phenotype correlation and identifies relationship between hereditary coproporphyria and harderoporphyria. Am J Hum Genet. 2001 May;68(5):1130-8. Epub 2001 Apr 16. Citation on PubMed or Free article on PubMed CentralLee DS, Flachsov E, Bodnrov M, Demeler B, Martsek P, Raman CS. Structural basis of hereditary coproporphyria. Proc Natl Acad Sci U S A. 2005 Oct 4;102(40):14232-7. Epub 2005 Sep 21. Citation on PubMed or Free article on PubMed CentralRosipal R, Lamoril J, Puy H, Da Silva V, Gouya L, De Rooij FW, Te Velde K, Nordmann Y, Martsek P, Deybach JC. Systematic analysis of coproporphyrinogen oxidase gene defects in hereditary coproporphyria and mutation update. Hum Mutat. 1999;13(1):44-53. Citation on PubMedSassa S, Kappas A. Molecular aspects of the inherited porphyrias. J Intern Med. 2000 Feb;247(2):169-78. Review. Citation on PubMedSchmitt C, Gouya L, Malonova E, Lamoril J, Camadro JM, Flamme M, Rose C, Lyoumi S, Da Silva V, Boileau C, Grandchamp B, Beaumont C, Deybach JC, Puy H. Mutations in human CPO gene predict clinical expression of either hepatic hereditary coproporphyria or erythropoietic harderoporphyria. Hum Mol Genet. 2005 Oct 15;14(20):3089-98. Epub 2005 Sep 13. Citation on PubMed \n",
            "TTLL3\t\n",
            "DCBLD2\t\n",
            "IL17RC\t   The IL17RC gene provides instructions for making a protein that is involved in immune system function, specifically in the body's defense against a fungus called Candida. When the immune system recognizes Candida, it generates cells called Th17 cells. These cells produce signaling molecules (cytokines) called the interleukin-17 (IL-17) family as part of an immune process called the IL-17 pathway. The IL-17 pathway creates inflammation, sending other cytokines and white blood cells that fight foreign invaders and promote tissue repair. In addition, the IL-17 pathway promotes the production of certain antimicrobial protein segments (peptides) that control growth of Candida on the surface of mucous membranes.The protein produced from the IL17RC gene is present in many tissue types in the body, and is involved in cell signaling as part of the IL-17 pathway. Together with the protein produced from the IL17RA gene, it forms one of several receptors for IL-17 cytokines. Receptor proteins have specific sites into which certain other proteins, called ligands, fit like keys into locks. Certain IL-17 cytokines attach to receptors containing the IL17RC and IL17RA proteins, triggering signals that promote inflammation and the defense against Candida infection.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Familial candidiasis     At least three mutations in the IL17RC gene have been identified in people with familial candidiasis, an inherited tendency to develop infections caused by the Candida fungus (commonly known as yeast infections). Most people with familial candidiasis have chronic yeast infections of the skin, nails, and mucous membranes (collectively called chronic mucocutaneous candidiasis) beginning in early childhood.IL17RC gene mutations result in an absence of IL17RC receptor protein. Loss of this protein impairs binding of IL-17 cytokines and IL-17 signaling. This impaired signaling decreases production of the antimicrobial peptides and inflammatory molecules that are critical for defense against Candida, and results in the chronic infection that occurs in familial candidiasis. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three mutations in the IL17RC gene have been identified in people with familial candidiasis, an inherited tendency to develop infections caused by the Candida fungus (commonly known as yeast infections). Most people with familial candidiasis have chronic yeast infections of the skin, nails, and mucous membranes (collectively called chronic mucocutaneous candidiasis) beginning in early childhood.IL17RC gene mutations result in an absence of IL17RC receptor protein. Loss of this protein impairs binding of IL-17 cytokines and IL-17 signaling. This impaired signaling decreases production of the antimicrobial peptides and inflammatory molecules that are critical for defense against Candida, and results in the chronic infection that occurs in familial candidiasis. More About This Health Condition    Cytogenetic Location: 3p25.3, which is the short (p) arm of chromosome 3 at position 25.3Molecular Location: base pairs 9,917,074 to 9,933,627 on chromosome 3 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IL-17 receptor CIL-17RLIL17F receptorinterleukin-17 receptor homologinterleukin-17 receptor-like protein     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Immunobiology: The Immune System in Health and Disease (fifth edition, 2001): The Front Line of Host Defense      Tests Listed in the Genetic Testing Registry (1 link)     Tests of IL17RC      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     INTERLEUKIN 17 RECEPTOR C      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology: The Immune System in Health and Disease (fifth edition, 2001): The Front Line of Host Defense  Tests of IL17RC  PubMed  INTERLEUKIN 17 RECEPTOR C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Ho AW, Shen F, Conti HR, Patel N, Childs EE, Peterson AC, Hernndez-Santos N, Kolls JK, Kane LP, Ouyang W, Gaffen SL. IL-17RC is required for immune signaling via an extended SEF/IL-17R signaling domain in the cytoplasmic tail. J Immunol. 2010 Jul 15;185(2):1063-70. doi: 10.4049/jimmunol.0903739. Epub 2010 Jun 16. Citation on PubMed or Free article on PubMed CentralHuppler AR, Bishu S, Gaffen SL. Mucocutaneous candidiasis: the IL-17 pathway and implications for targeted immunotherapy. Arthritis Res Ther. 2012 Jul 23;14(4):217. doi: 10.1186/ar3893. Review. Citation on PubMed or Free article on PubMed CentralOMIM: INTERLEUKIN 17 RECEPTOR CLing Y, Cypowyj S, Aytekin C, Galicchio M, Camcioglu Y, Nepesov S, Ikinciogullari A, Dogu F, Belkadi A, Levy R, Migaud M, Boisson B, Bolze A, Itan Y, Goudin N, Cottineau J, Picard C, Abel L, Bustamante J, Casanova JL, Puel A. Inherited IL-17RC deficiency in patients with chronic mucocutaneous candidiasis. J Exp Med. 2015 May 4;212(5):619-31. doi: 10.1084/jem.20141065. Epub 2015 Apr 27. Citation on PubMed or Free article on PubMed CentralPlantinga TS, Johnson MD, Scott WK, Joosten LA, van der Meer JW, Perfect JR, Kullberg BJ, Netea MG. Human genetic susceptibility to Candida infections. Med Mycol. 2012 Nov;50(8):785-94. doi: 10.3109/13693786.2012.690902. Epub 2012 Jun 4. Review. Citation on PubMedPuel A, Cypowyj S, Mardi L, Abel L, Picard C, Casanova JL. Inborn errors of human IL-17 immunity underlie chronic mucocutaneous candidiasis. Curr Opin Allergy Clin Immunol. 2012 Dec;12(6):616-22. doi: 10.1097/ACI.0b013e328358cc0b. Review. Citation on PubMed or Free article on PubMed CentralSmeekens SP, van de Veerdonk FL, Kullberg BJ, Netea MG. Genetic susceptibility to Candida infections. EMBO Mol Med. 2013 Jun;5(6):805-13. doi: 10.1002/emmm.201201678. Epub 2013 Apr 30. Review. Citation on PubMed or Free article on PubMed Central \n",
            "ADH1C\t   From NCBI Gene:This gene encodes class I alcohol dehydrogenase, gamma subunit, which is a member of the alcohol dehydrogenase family. Members of this enzyme family metabolize a wide variety of substrates, including ethanol, retinol, other aliphatic alcohols, hydroxysteroids, and lipid peroxidation products. Class I alcohol dehydrogenase, consisting of several homo- and heterodimers of alpha, beta, and gamma subunits, exhibits high activity for ethanol oxidation and plays a major role in ethanol catabolism. Three genes encoding alpha, beta and gamma subunits are tandemly organized in a genomic segment as a gene cluster. [provided by RefSeq, Jul 2008]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Alcohol use disorderFrom NCBI Gene:Parkinson disease, late-onsetAlcohol dependence     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q23, which is the long (q) arm of chromosome 4 at position 23Molecular Location: base pairs 99,336,492 to 99,353,045 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADH3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADH1C      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ALCOHOL DEHYDROGENASE 1C, GAMMA POLYPEPTIDEOMIM: ALCOHOL DEPENDENCEOMIM: PARKINSON DISEASE, LATE-ONSET      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADH1C  OMIM: ALCOHOL DEHYDROGENASE 1C, GAMMA POLYPEPTIDEOMIM: ALCOHOL DEPENDENCEOMIM: PARKINSON DISEASE, LATE-ONSET  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "WDR1\t\n",
            "DDIT4L\t\n",
            "FGFRL1\t   From NCBI Gene:The protein encoded by this gene is a member of the fibroblast growth factor receptor (FGFR) family, where amino acid sequence is highly conserved between members and throughout evolution. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein would consist of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. A marked difference between this gene product and the other family members is its lack of a cytoplasmic tyrosine kinase domain. The result is a transmembrane receptor that could interact with other family members and potentially inhibit signaling. Multiple alternatively spliced transcript variants encoding the same isoform have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Has a negative effect on cell proliferation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:4p partial monosomy syndrome     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4p16.3, which is the short (p) arm of chromosome 4 at position 16.3Molecular Location: base pairs 1,009,930 to 1,026,898 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FGFR-5FGFR5FHFR     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FGFRL1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FIBROBLAST GROWTH FACTOR RECEPTOR-LIKE 1OMIM: WOLF-HIRSCHHORN SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FGFRL1  OMIM: FIBROBLAST GROWTH FACTOR RECEPTOR-LIKE 1OMIM: WOLF-HIRSCHHORN SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLC39A8\t   From NCBI Gene:This gene encodes a member of the SLC39 family of solute-carrier genes, which show structural characteristics of zinc transporters. The encoded protein is glycosylated and found in the plasma membrane and mitochondria, and functions in the cellular import of zinc at the onset of inflammation. It is also thought to be the primary transporter of the toxic cation cadmium, which is found in cigarette smoke. Multiple transcript variants encoding different isoforms have been found for this gene. Additional alternatively spliced transcript variants of this gene have been described, but their full-length nature is not known. [provided by RefSeq, Oct 2008]From UniProt:Acts as a manganese and zinc influx transporter (PubMed:12504855, PubMed:26637978). Plays a role in manganese reabsorption in the proximal tubule of the kidney and in manganese uptake into the brain (PubMed:26637978).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IInFrom UniProt:Congenital disorder of glycosylation 2N (CDG2N): A form of congenital disorder of glycosylation, a genetically heterogeneous group of autosomal recessive, multisystem disorders caused by a defect in glycoprotein biosynthesis and characterized by under-glycosylated serum glycoproteins. Congenital disorders of glycosylation result in a wide variety of clinical features, such as defects in the nervous system development, psychomotor retardation, dysmorphic features, hypotonia, coagulation disorders, and immunodeficiency. The broad spectrum of features reflects the critical role of N-glycoproteins during embryonic development, differentiation, and maintenance of cell functions. [MIM:616721]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q24, which is the long (q) arm of chromosome 4 at position 24Molecular Location: base pairs 102,251,041 to 102,345,498 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BIGM103CDG2NLZT-Hs6PP3105ZIP8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC39A8      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IInOMIM: SOLUTE CARRIER FAMILY 39 (ZINC TRANSPORTER), MEMBER 8      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SLC39A8  OMIM: CONGENITAL DISORDER OF GLYCOSYLATION, TYPE IInOMIM: SOLUTE CARRIER FAMILY 39 (ZINC TRANSPORTER), MEMBER 8  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CISD2\t   The CISD2 gene provides instructions for making a protein that is found in the outer membrane of cell structures called mitochondria. Mitochondria are involved in a wide variety of cellular activities, including energy production, chemical signaling, and regulation of cell growth and division. The exact function of the CISD2 protein is unknown, but it is thought to help keep mitochondria functioning normally.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Wolfram syndrome     At least one mutation in the CISD2 gene has been found to cause Wolfram syndrome. This condition is characterized by a lack of insulin leading to increased blood sugar (diabetes mellitus), a degeneration of nerves that carry information from the eyes to the brain (optic atrophy), and a number of other features involving the urinary tract, the brain, and hearing. People with this CISD2 gene mutation also experience gastrointestinal ulcers and excessive bleeding after injury.The CISD2 gene mutation that causes Wolfram syndrome replaces the amino acid glutamic acid with the amino acid glutamine at position 37 in the CISD2 protein (written as Glu37Gln or E37Q). This mutation results in an abnormally small, nonfunctional CISD2 protein. As a result, the function of the mitochondria is impaired and they eventually break down. Since the mitochondria provide energy to cells, the loss of mitochondria leads to decreased energy for cells. Cells that do not have enough energy to function will eventually die. Cells with high energy demands, such as nerve cells in the brain, eyes, or gastrointestinal tract, are most susceptible to cell death due to reduced energy. The gradual loss of cells in various body systems likely causes the signs and symptoms of Wolfram syndrome. When Wolfram syndrome is caused by CISD2 gene mutations, it is sometimes referred to as Wolfram syndrome type 2. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least one mutation in the CISD2 gene has been found to cause Wolfram syndrome. This condition is characterized by a lack of insulin leading to increased blood sugar (diabetes mellitus), a degeneration of nerves that carry information from the eyes to the brain (optic atrophy), and a number of other features involving the urinary tract, the brain, and hearing. People with this CISD2 gene mutation also experience gastrointestinal ulcers and excessive bleeding after injury.The CISD2 gene mutation that causes Wolfram syndrome replaces the amino acid glutamic acid with the amino acid glutamine at position 37 in the CISD2 protein (written as Glu37Gln or E37Q). This mutation results in an abnormally small, nonfunctional CISD2 protein. As a result, the function of the mitochondria is impaired and they eventually break down. Since the mitochondria provide energy to cells, the loss of mitochondria leads to decreased energy for cells. Cells that do not have enough energy to function will eventually die. Cells with high energy demands, such as nerve cells in the brain, eyes, or gastrointestinal tract, are most susceptible to cell death due to reduced energy. The gradual loss of cells in various body systems likely causes the signs and symptoms of Wolfram syndrome. When Wolfram syndrome is caused by CISD2 gene mutations, it is sometimes referred to as Wolfram syndrome type 2. More About This Health Condition    Cytogenetic Location: 4q24, which is the long (q) arm of chromosome 4 at position 24Molecular Location: base pairs 102,868,978 to 102,892,807 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDGSH iron-sulfur domain-containing protein 2CISD2_HUMANendoplasmic reticulum intermembrane small proteinERISMiner1NAF-1nutrient-deprivation autophagy factor-1WFS2ZCD2zinc finger, CDGSH-type domain 2     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): Mitochondria Are Bounded by a Double MembraneMadame Curie Bioscience Database: Damage to Mitochondria      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CISD2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CDGSH IRON SULFUR DOMAIN PROTEIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Mitochondria Are Bounded by a Double MembraneMadame Curie Bioscience Database: Damage to Mitochondria  Tests of CISD2  PubMed  CDGSH IRON SULFUR DOMAIN PROTEIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Amr S, Heisey C, Zhang M, Xia XJ, Shows KH, Ajlouni K, Pandya A, Satin LS, El-Shanti H, Shiang R. A homozygous mutation in a novel zinc-finger protein, ERIS, is responsible for Wolfram syndrome 2. Am J Hum Genet. 2007 Oct;81(4):673-83. Epub 2007 Aug 20. Citation on PubMed or Free article on PubMed CentralOMIM: CDGSH IRON SULFUR DOMAIN PROTEIN 2Chen YF, Wu CY, Kirby R, Kao CH, Tsai TF. A role for the CISD2 gene in lifespan control and human disease. Ann N Y Acad Sci. 2010 Jul;1201:58-64. doi: 10.1111/j.1749-6632.2010.05619.x. Review. Citation on PubMedKanki T, Klionsky DJ. Mitochondrial abnormalities drive cell death in Wolfram syndrome 2. Cell Res. 2009 Aug;19(8):922-3. doi: 10.1038/cr.2009.94. Citation on PubMed or Free article on PubMed Central \n",
            "SLC9B1\t\n",
            "TET2\t   The TET2 gene provides instructions for making a protein whose function is unknown. Based on the function of similar proteins, researchers believe the TET2 protein is involved in regulating the process of transcription, which is the first step in protein production.  Although this protein is found throughout the body, it may play a particularly important role in the production of blood cells from hematopoietic stem cells. These stem cells are located within the bone marrow and have the potential to develop into red blood cells, white blood cells, and platelets. The TET2 protein appears to act as a tumor suppressor, which is a protein that prevents cells from growing and dividing in an uncontrolled way.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Essential thrombocythemia     Some gene mutations are acquired during a person's lifetime and are present only in certain cells. These changes, which are called somatic mutations, are not inherited. Somatic mutations in the TET2 gene have been identified in a small number of people with essential thrombocythemia, which is a condition characterized by high numbers of platelets in the blood. Platelets are the blood cell fragments involved in blood clotting.TET2 gene mutations alter the TET2 protein in different ways; however, all of them appear to result in a nonfunctional protein. The role these mutations play in the development of essential thrombocythemia is unknown. More About This Health Condition      Polycythemia vera     Somatic mutations in the TET2 gene are associated with polycythemia vera, a disorder characterized by uncontrolled blood cell production. These mutations are thought to result in a nonfunctional protein. Mutations in this gene have been found in approximately 16 percent of people with polycythemia vera. It is unclear what role these mutations play in the development of polycythemia vera. More About This Health Condition      Primary myelofibrosis     Somatic mutations in the TET2 gene are associated with primary myelofibrosis. This condition is characterized by scar tissue (fibrosis) in the bone marrow, the tissue that produces blood cells. It is unclear what role the TET2 gene mutations play in the development of primary myelofibrosis. More About This Health Condition      Systemic mastocytosis     Genetics Home Reference provides information about systemic mastocytosis. More About This Health Condition      Other disorders     Somatic TET2 gene mutations are also associated with certain types of cancer of blood-forming cells (leukemia) and a disease of the blood and bone marrow called myelodysplastic syndrome. These mutations are thought to result in a nonfunctional TET2 protein. A loss of TET2 protein in hematopoietic stem cells may lead to uncontrolled growth and division of these cells. Researchers are working to determine exactly what role TET2 gene mutations play in the development of bone marrow disorders.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Some gene mutations are acquired during a person's lifetime and are present only in certain cells. These changes, which are called somatic mutations, are not inherited. Somatic mutations in the TET2 gene have been identified in a small number of people with essential thrombocythemia, which is a condition characterized by high numbers of platelets in the blood. Platelets are the blood cell fragments involved in blood clotting.TET2 gene mutations alter the TET2 protein in different ways; however, all of them appear to result in a nonfunctional protein. The role these mutations play in the development of essential thrombocythemia is unknown. More About This Health Condition  Somatic mutations in the TET2 gene are associated with polycythemia vera, a disorder characterized by uncontrolled blood cell production. These mutations are thought to result in a nonfunctional protein. Mutations in this gene have been found in approximately 16 percent of people with polycythemia vera. It is unclear what role these mutations play in the development of polycythemia vera. More About This Health Condition  Somatic mutations in the TET2 gene are associated with primary myelofibrosis. This condition is characterized by scar tissue (fibrosis) in the bone marrow, the tissue that produces blood cells. It is unclear what role the TET2 gene mutations play in the development of primary myelofibrosis. More About This Health Condition  Genetics Home Reference provides information about systemic mastocytosis. More About This Health Condition  Somatic TET2 gene mutations are also associated with certain types of cancer of blood-forming cells (leukemia) and a disease of the blood and bone marrow called myelodysplastic syndrome. These mutations are thought to result in a nonfunctional TET2 protein. A loss of TET2 protein in hematopoietic stem cells may lead to uncontrolled growth and division of these cells. Researchers are working to determine exactly what role TET2 gene mutations play in the development of bone marrow disorders.    Cytogenetic Location: 4q24, which is the long (q) arm of chromosome 4 at position 24Molecular Location: base pairs 105,145,875 to 105,279,803 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FLJ20032KIAA1546MGC125715probable methylcytosine dioxygenase TET2probable methylcytosine dioxygenase TET2 isoform aprobable methylcytosine dioxygenase TET2 isoform btet oncogene family member 2TET2_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Cancer Medicine (sixth edition, 2003): Tumor Suppressor Genes      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TET2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TET ONCOGENE FAMILY, MEMBER 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Cancer Medicine (sixth edition, 2003): Tumor Suppressor Genes  Tests of TET2  PubMed  TET ONCOGENE FAMILY, MEMBER 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Abdel-Wahab O, Mullally A, Hedvat C, Garcia-Manero G, Patel J, Wadleigh M, Malinge S, Yao J, Kilpivaara O, Bhat R, Huberman K, Thomas S, Dolgalev I, Heguy A, Paietta E, Le Beau MM, Beran M, Tallman MS, Ebert BL, Kantarjian HM, Stone RM, Gilliland DG, Crispino JD, Levine RL. Genetic characterization of TET1, TET2, and TET3 alterations in myeloid malignancies. Blood. 2009 Jul 2;114(1):144-7. doi: 10.1182/blood-2009-03-210039. Epub 2009 May 6. Citation on PubMed or Free article on PubMed CentralBacher U, Haferlach C, Schnittger S, Kohlmann A, Kern W, Haferlach T. Mutations of the TET2 and CBL genes: novel molecular markers in myeloid malignancies. Ann Hematol. 2010 Jul;89(7):643-52. doi: 10.1007/s00277-010-0920-6. Epub 2010 Mar 2. Review. Citation on PubMedDelhommeau F, Dupont S, Della Valle V, James C, Trannoy S, Mass A, Kosmider O, Le Couedic JP, Robert F, Alberdi A, Lcluse Y, Plo I, Dreyfus FJ, Marzac C, Casadevall N, Lacombe C, Romana SP, Dessen P, Soulier J, Vigui F, Fontenay M, Vainchenker W, Bernard OA. Mutation in TET2 in myeloid cancers. N Engl J Med. 2009 May 28;360(22):2289-301. doi: 10.1056/NEJMoa0810069. Citation on PubMedLangemeijer SM, Kuiper RP, Berends M, Knops R, Aslanyan MG, Massop M, Stevens-Linders E, van Hoogen P, van Kessel AG, Raymakers RA, Kamping EJ, Verhoef GE, Verburgh E, Hagemeijer A, Vandenberghe P, de Witte T, van der Reijden BA, Jansen JH. Acquired mutations in TET2 are common in myelodysplastic syndromes. Nat Genet. 2009 Jul;41(7):838-42. doi: 10.1038/ng.391. Epub 2009 May 31. Citation on PubMedSaint-Martin C, Leroy G, Delhommeau F, Panelatti G, Dupont S, James C, Plo I, Bordessoule D, Chomienne C, Delannoy A, Devidas A, Gardembas-Pain M, Isnard F, Plumelle Y, Bernard O, Vainchenker W, Najman A, Bellann-Chantelot C; French Group of Familial Myeloproliferative Disorders. Analysis of the ten-eleven translocation 2 (TET2) gene in familial myeloproliferative neoplasms. Blood. 2009 Aug 20;114(8):1628-32. doi: 10.1182/blood-2009-01-197525. Epub 2009 Jun 29. Citation on PubMedSchaub FX, Looser R, Li S, Hao-Shen H, Lehmann T, Tichelli A, Skoda RC. Clonal analysis of TET2 and JAK2 mutations suggests that TET2 can be a late event in the progression of myeloproliferative neoplasms. Blood. 2010 Mar 11;115(10):2003-7. doi: 10.1182/blood-2009-09-245381. Epub 2010 Jan 8. Citation on PubMedOMIM: TET ONCOGENE FAMILY, MEMBER 2Tefferi A, Pardanani A, Lim KH, Abdel-Wahab O, Lasho TL, Patel J, Gangat N, Finke CM, Schwager S, Mullally A, Li CY, Hanson CA, Mesa R, Bernard O, Delhommeau F, Vainchenker W, Gilliland DG, Levine RL. TET2 mutations and their clinical correlates in polycythemia vera, essential thrombocythemia and myelofibrosis. Leukemia. 2009 May;23(5):905-11. doi: 10.1038/leu.2009.47. Epub 2009 Mar 5. Citation on PubMed or Free article on PubMed CentralTefferi A. Novel mutations and their functional and clinical relevance in myeloproliferative neoplasms: JAK2, MPL, TET2, ASXL1, CBL, IDH and IKZF1. Leukemia. 2010 Jun;24(6):1128-38. doi: 10.1038/leu.2010.69. Epub 2010 Apr 29. Review. Citation on PubMed or Free article on PubMed Central \n",
            "RNF212\t   From NCBI Gene:This gene encodes a RING finger protein that may function as a ubiquitin ligase. The encoded protein may be involved in meiotic recombination. This gene is located within a linkage disequilibrium block and polymorphisms in this gene may influence recombination rates. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Oct 2010]From UniProt:SUMO E3 ligase that acts as a regulator of crossing-over during meiosis: required to couple chromosome synapsis to the formation of crossover-specific recombination complexes. Localizes to recombination sites and stabilizes meiosis-specific recombination factors, such as MutS-gamma complex proteins (MSH4 and MSH5) and TEX11. May mediate sumoylation of target proteins MSH4 and/or MSH5, leading to enhance their binding to recombination sites. Acts as a limiting factor for crossover designation and/or reinforcement and plays an antagonist role with CCNB1IP1/HEI10 in the regulation of meiotic recombination (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Recombination rate quantitative trait locus 1     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4p16.3, which is the short (p) arm of chromosome 4 at position 16.3Molecular Location: base pairs 1,056,248 to 1,113,794 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ZHP3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RNF212      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: RECOMBINATION RATE QUANTITATIVE TRAIT LOCUS 1OMIM: RING FINGER PROTEIN 212      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RNF212  OMIM: RECOMBINATION RATE QUANTITATIVE TRAIT LOCUS 1OMIM: RING FINGER PROTEIN 212  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SEC24B\t\n",
            "LRIT3\t   From NCBI Gene:This gene encodes a protein that has a fibronectin type III domain and a C-terminal transmembrane domain, as well as a leucine-rich repeat domain and immunoglobulin-like domain near the N-terminus. The encoded protein may regulate fibroblast growth factor receptors and affect the modification of these receptors, which are glycosylated differently in the Golgi and endoplasmic reticulum. Mutations in this gene are associated with congenital stationary night blindness, type 1F. [provided by RefSeq, May 2013]From UniProt:Required in retinal ON-bipolar cells for normal localization of the cation channel TRPM1 at dendrite tips (By similarity). May also have a role in cone synapse formation (By similarity). Might facilitate FGFR1 exit from the endoplasmic reticulum to the Golgi (PubMed:22673519). Could be a regulator of the FGFRs (PubMed:22673519).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autosomal recessive congenital stationary night blindnessFrom NCBI Gene:Congenital stationary night blindness, type 1FFrom UniProt:Night blindness, congenital stationary, 1F (CSNB1F): An autosomal recessive form of congenital stationary night blindness, a non-progressive retinal disorder characterized by impaired night vision, often associated with nystagmus and myopia. [MIM:615058]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q25, which is the long (q) arm of chromosome 4 at position 25Molecular Location: base pairs 109,848,182 to 109,872,315 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CSNB1FFIGLER4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LRIT3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LEUCINE-RICH REPEAT, IMMUNOGLOBULIN-LIKE, AND TRANSMEMBRANE DOMAINS-CONTAINING PROTEIN 3OMIM: NIGHT BLINDNESS, CONGENITAL STATIONARY, TYPE 1F      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LRIT3  OMIM: LEUCINE-RICH REPEAT, IMMUNOGLOBULIN-LIKE, AND TRANSMEMBRANE DOMAINS-CONTAINING PROTEIN 3OMIM: NIGHT BLINDNESS, CONGENITAL STATIONARY, TYPE 1F  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ALPK1\t\n",
            "FAT4\t   The FAT4 gene provides instructions for making a protein that is found in most tissues. The protein spans the membrane surrounding cells so that part of the protein is outside the cell and part of the protein is inside the cell. The precise function of the FAT4 protein is largely unknown; however, research shows that the FAT4 protein is likely involved in determining the position of various components within cells (cell polarity). The FAT4 protein is also thought to function as a tumor suppressor, which means that it keeps cells from growing and dividing too rapidly or in an uncontrolled way.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hennekam syndrome     At least seven mutations in the FAT4 gene have been found to cause Hennekam syndrome, an inherited disorder resulting from malformation of the lymphatic system, which consists of a network of vessels that transport lymph fluid and immune cells throughout the body. The FAT4 gene mutations that cause Hennekam syndrome reduce the activity of the FAT4 protein, which seems to impair normal development of the lymphatic system. However, the mechanism is unknown. A poorly formed lymphatic system leads to lymphatic vessels that are abnormally expanded (lymphangiectasia) and are prone to break open (rupture), puffiness or swelling caused by a buildup of fluid (lymphedema), and other features of Hennekam syndrome. FAT4 gene mutations account for about 25 percent of all cases of Hennekam syndrome. More About This Health Condition      Other disorders     Mutations in the FAT4 gene have also been found in individuals who have van Maldergem syndrome, a condition characterized by intellectual disability, hearing loss, skeletal abnormalities, and a brain malformation called periventricular heterotopia. The FAT4 gene mutations that cause van Maldergem syndrome lead to a decrease in FAT4 protein function. It is thought that a decrease in FAT4 protein activity in the brain disrupts the polarity of nerve cells in the brain, which leads to periventricular heterotopia.It is unknown how mutations in the FAT4 gene can cause the signs and symptoms of van Maldergem syndrome in some people but lead to different signs and symptoms in those with Hennekam syndrome (described above). One mutation that replaces the protein building block (amino acid) glutamic acid with the amino acid lysine at position 2375 in the FAT4 protein (written as Glu2375Lys or E2375K) has been found in both individuals with van Maldergem syndrome and those with Hennekam syndrome.FAT4 gene mutations have also been found in many types of cancers, including a type of skin cancer called head and neck squamous cell carcinoma, an aggressive form of skin cancer called melanoma, a liver cancer called hepatocellular carcinoma, stomach (gastric) cancer, and pancreatic cancer. The FAT4 gene mutations involved in these cancers are different from the ones that cause van Maldergem syndrome and Hennekam syndrome (described above). The FAT4 gene mutations associated with cancers are called somatic mutations; they are found only in cells that become cancerous and are not inherited. It is likely that these mutations prevent the FAT4 protein from acting as a tumor suppressor, contributing to the uncontrollable growth and division of cells that is characteristic of cancer. People with van Maldergem syndrome and Hennekam syndrome are not thought to have an increased risk of developing cancer.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least seven mutations in the FAT4 gene have been found to cause Hennekam syndrome, an inherited disorder resulting from malformation of the lymphatic system, which consists of a network of vessels that transport lymph fluid and immune cells throughout the body. The FAT4 gene mutations that cause Hennekam syndrome reduce the activity of the FAT4 protein, which seems to impair normal development of the lymphatic system. However, the mechanism is unknown. A poorly formed lymphatic system leads to lymphatic vessels that are abnormally expanded (lymphangiectasia) and are prone to break open (rupture), puffiness or swelling caused by a buildup of fluid (lymphedema), and other features of Hennekam syndrome. FAT4 gene mutations account for about 25 percent of all cases of Hennekam syndrome. More About This Health Condition  Mutations in the FAT4 gene have also been found in individuals who have van Maldergem syndrome, a condition characterized by intellectual disability, hearing loss, skeletal abnormalities, and a brain malformation called periventricular heterotopia. The FAT4 gene mutations that cause van Maldergem syndrome lead to a decrease in FAT4 protein function. It is thought that a decrease in FAT4 protein activity in the brain disrupts the polarity of nerve cells in the brain, which leads to periventricular heterotopia.It is unknown how mutations in the FAT4 gene can cause the signs and symptoms of van Maldergem syndrome in some people but lead to different signs and symptoms in those with Hennekam syndrome (described above). One mutation that replaces the protein building block (amino acid) glutamic acid with the amino acid lysine at position 2375 in the FAT4 protein (written as Glu2375Lys or E2375K) has been found in both individuals with van Maldergem syndrome and those with Hennekam syndrome.FAT4 gene mutations have also been found in many types of cancers, including a type of skin cancer called head and neck squamous cell carcinoma, an aggressive form of skin cancer called melanoma, a liver cancer called hepatocellular carcinoma, stomach (gastric) cancer, and pancreatic cancer. The FAT4 gene mutations involved in these cancers are different from the ones that cause van Maldergem syndrome and Hennekam syndrome (described above). The FAT4 gene mutations associated with cancers are called somatic mutations; they are found only in cells that become cancerous and are not inherited. It is likely that these mutations prevent the FAT4 protein from acting as a tumor suppressor, contributing to the uncontrollable growth and division of cells that is characteristic of cancer. People with van Maldergem syndrome and Hennekam syndrome are not thought to have an increased risk of developing cancer.    Cytogenetic Location: 4q28.1, which is the long (q) arm of chromosome 4 at position 28.1Molecular Location: base pairs 125,314,913 to 125,492,932 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        cadherin family member 14cadherin-related family member 11CDHF14CDHR11FAT-Jfat-like cadherin protein FAT-JFAT tumor suppressor homolog 4FAT4_HUMANFATJNBLA00548protocadherin Fat 4VMLDS2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FAT4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FAT TUMOR SUPPRESSOR, DROSOPHILA, HOMOLOG OF, 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FAT4  PubMed  FAT TUMOR SUPPRESSOR, DROSOPHILA, HOMOLOG OF, 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Alders M, Al-Gazali L, Cordeiro I, Dallapiccola B, Garavelli L, Tuysuz B, Salehi F, Haagmans MA, Mook OR, Majoie CB, Mannens MM, Hennekam RC. Hennekam syndrome can be caused by FAT4 mutations and be allelic to Van Maldergem syndrome. Hum Genet. 2014 Sep;133(9):1161-7. doi: 10.1007/s00439-014-1456-y. Epub 2014 Jun 7. Citation on PubMedOMIM: FAT TUMOR SUPPRESSOR, DROSOPHILA, HOMOLOG OF, 4Katoh M. Function and cancer genomics of FAT family genes (review). Int J Oncol. 2012 Dec;41(6):1913-8. doi: 10.3892/ijo.2012.1669. Epub 2012 Oct 17. Review. Citation on PubMed or Free article on PubMed CentralKatoh Y, Katoh M. Comparative integromics on FAT1, FAT2, FAT3 and FAT4. Int J Mol Med. 2006 Sep;18(3):523-8. Citation on PubMed \n",
            "LARP1B\t\n",
            "SCLT1\t   From NCBI Gene:This gene encodes an adaptor protein. Studies of a related gene in rat suggest that the encoded protein functions to link clathrin to the sodium channel protein type 10 subunit alpha protein. The encoded protein has also been identified as a component of distal appendages of centrioles that is necessary for ciliogenesis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014]From UniProt:Adapter protein that links SCN10A to clathrin. Regulates SCN10A channel activity, possibly by promoting channel internalization (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Coloboma     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q28.2, which is the long (q) arm of chromosome 4 at position 28.2Molecular Location: base pairs 128,874,390 to 129,093,607 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAP-1ACAP1A     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SCLT1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: SODIUM CHANNEL AND CLATHRIN LINKER 1      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SCLT1  OMIM: SODIUM CHANNEL AND CLATHRIN LINKER 1  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CRIPAK\t\n",
            "ELMOD2\t   From NCBI Gene:This gene encodes one of six engulfment and motility (ELMO) domain-containing proteins. This gene is thought to play a role in antiviral responses. Mutations in this gene may be involved in the cause of familial idiopathic pulmonary fibrosis. [provided by RefSeq, Sep 2010]From UniProt:Acts as a GTPase-activating protein (GAP) toward guanine nucleotide exchange factors like ARL2, ARL3, ARF1 and ARF6, but not for GTPases outside the Arf family. Regulates IFN-related antiviral responses.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Idiopathic pulmonary fibrosis     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q31.1, which is the long (q) arm of chromosome 4 at position 31.1Molecular Location: base pairs 140,524,146 to 140,553,770 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        9830169G11Rik     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ELMOD2      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: ELMO/CED12 DOMAIN-CONTAINING PROTEIN 2      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ELMOD2  OMIM: ELMO/CED12 DOMAIN-CONTAINING PROTEIN 2  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "INPP4B\t\n",
            "GYPA\t   From NCBI Gene:Glycophorins A (GYPA) and B (GYPB) are major sialoglycoproteins of the human erythrocyte membrane which bear the antigenic determinants for the MN and Ss blood groups. In addition to the M or N and S or s antigens that commonly occur in all populations, about 40 related variant phenotypes have been identified. These variants include all the variants of the Miltenberger complex and several isoforms of Sta, as well as Dantu, Sat, He, Mg, and deletion variants Ena, S-s-U- and Mk. Most of the variants are the result of gene recombinations between GYPA and GYPB. [provided by RefSeq, Jul 2008]From UniProt:Glycophorin A is the major intrinsic membrane protein of the erythrocyte. The N-terminal glycosylated segment, which lies outside the erythrocyte membrane, has MN blood group receptors. Appears to be important for the function of SLC4A1 and is required for high activity of SLC4A1. May be involved in translocation of SLC4A1 to the plasma membrane. Is a receptor for influenza virus. Is a receptor for Plasmodium falciparum erythrocyte-binding antigen 175 (EBA-175); binding of EBA-175 is dependent on sialic acid residues of the O-linked glycans. Appears to be a receptor for Hepatitis A virus (HAV).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:BLOOD GROUP, MNSusceptibility to malaria     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q31.21, which is the long (q) arm of chromosome 4 at position 31.21Molecular Location: base pairs 144,109,303 to 144,140,854 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD235aGPAGPErikGPSATHGpMiVHGpMiXIHGpSta(C)MNMNSPAS-2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GYPA      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BLOOD GROUP, MNOMIM: GLYCOPHORIN AOMIM: MALARIA, SUSCEPTIBILITY TO      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GYPA  OMIM: BLOOD GROUP, MNOMIM: GLYCOPHORIN AOMIM: MALARIA, SUSCEPTIBILITY TO  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "POU4F2\t\n",
            "CPEB2\t\n",
            "LRBA\t   From NCBI Gene:The protein encoded by this gene is a member of the WDL-BEACH-WD (WBW) gene family. Its expression is induced in B cells and macrophages by bacterial lipopolysaccharides (LPS). The encoded protein associates with protein kinase A and may be involved in leading intracellular vesicles to activated receptor complexes, which aids in the secretion and/or membrane deposition of immune effector molecules. Defects in this gene are associated with the disorder common variable immunodeficiency-8 with autoimmunity. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2012]From UniProt:May be involved in coupling signal transduction and vesicle trafficking to enable polarized secretion and/or membrane deposition of immune effector molecules.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Common variable immune deficiencyFrom NCBI Gene:Common variable immunodeficiency 8, with autoimmunityFrom UniProt:Immunodeficiency, common variable, 8, with autoimmunity (CVID8): An autosomal recessive immunologic disorder associated with defective B-cell differentiation and decreased or absent antibody production. Affected individuals have early-childhood onset of recurrent infections, particularly respiratory infections, and also develop variable autoimmune disorders, including idiopathic thrombocytopenic purpura, autoimmune hemolytic anemia, and inflammatory bowel disease. [MIM:614700]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q31.3, which is the long (q) arm of chromosome 4 at position 31.3Molecular Location: base pairs 150,264,515 to 151,015,727 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BGLCDC4LCVID8LAB300LBA     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LRBA      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: IMMUNODEFICIENCY, COMMON VARIABLE, 8, WITH AUTOIMMUNITYOMIM: LIPOPOLYSACCHARIDE-RESPONSIVE, BEIGE-LIKE ANCHOR PROTEIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LRBA  OMIM: IMMUNODEFICIENCY, COMMON VARIABLE, 8, WITH AUTOIMMUNITYOMIM: LIPOPOLYSACCHARIDE-RESPONSIVE, BEIGE-LIKE ANCHOR PROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PRSS48\t\n",
            "FHDC1\t\n",
            "RNF175\t\n",
            "CC2D2A\t   From NCBI Gene:This gene encodes a coiled-coil and calcium binding domain protein that appears to play a critical role in cilia formation. Mutations in this gene cause Meckel syndrome type 6, as well as Joubert syndrome type 9. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2009]From UniProt:Component of the tectonic-like complex, a complex localized at the transition zone of primary cilia and acting as a barrier that prevents diffusion of transmembrane proteins between the cilia and plasma membranes. Required for ciliogenesis and sonic hedgehog/SHH signaling (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Meckel syndromeJoubert syndromeColobomaFrom NCBI Gene:COACH syndromeJoubert syndrome 9Meckel syndrome type 6From UniProt:Meckel syndrome 6 (MKS6): A disorder characterized by a combination of renal cysts and variably associated features including developmental anomalies of the central nervous system (typically encephalocele), hepatic ductal dysplasia and cysts, and polydactyly. [MIM:612284]Joubert syndrome 9 (JBTS9): A disorder presenting with cerebellar ataxia, oculomotor apraxia, hypotonia, neonatal breathing abnormalities and psychomotor delay. Neuroradiologically, it is characterized by cerebellar vermian hypoplasia/aplasia, thickened and reoriented superior cerebellar peduncles, and an abnormally large interpeduncular fossa, giving the appearance of a molar tooth on transaxial slices (molar tooth sign). Additional variable features include retinal dystrophy and renal disease. [MIM:612285]COACH syndrome (COACHS): A disorder characterized by mental retardation, ataxia due to cerebellar hypoplasia, and hepatic fibrosis. Patients present the molar tooth sign, a midbrain-hindbrain malformation pathognomonic for Joubert syndrome and related disorders. Other features, such as coloboma and renal cysts, may be variable. [MIM:216360]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4p15.32, which is the short (p) arm of chromosome 4 at position 15.32Molecular Location: base pairs 15,468,660 to 15,601,971 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        JBTS9MKS6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CC2D2A      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: COACH SYNDROMEOMIM: COILED-COIL AND C2 DOMAINS-CONTAINING PROTEIN 2AOMIM: JOUBERT SYNDROME 9OMIM: MECKEL SYNDROME, TYPE 6      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CC2D2A  OMIM: COACH SYNDROMEOMIM: COILED-COIL AND C2 DOMAINS-CONTAINING PROTEIN 2AOMIM: JOUBERT SYNDROME 9OMIM: MECKEL SYNDROME, TYPE 6  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DCHS2\t\n",
            "FNIP2\t\n",
            "ANP32C\t\n",
            "SPOCK3\t\n",
            "DDX60L\t\n",
            "VEGFC\t   From NCBI Gene:The protein encoded by this gene is a member of the platelet-derived growth factor/vascular endothelial growth factor (PDGF/VEGF) family. The encoded protein promotes angiogenesis and endothelial cell growth, and can also affect the permeability of blood vessels. The proprotein is further cleaved into a fully processed form that can bind and activate VEGFR-2 and VEGFR-3 receptors. [provided by RefSeq, Apr 2014]From UniProt:Growth factor active in angiogenesis, and endothelial cell growth, stimulating their proliferation and migration and also has effects on the permeability of blood vessels. May function in angiogenesis of the venous and lymphatic vascular systems during embryogenesis, and also in the maintenance of differentiated lymphatic endothelium in adults. Binds and activates KDR/VEGFR2 and FLT4/VEGFR3 receptors.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Lymphedema, hereditary, IDFrom UniProt:Lymphedema, hereditary, 1D (LMPH1D): A chronic disabling condition which results in swelling of the extremities due to altered lymphatic flow. Patients with lymphedema suffer from recurrent local infections and physical impairment. [MIM:615907]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q34.3, which is the long (q) arm of chromosome 4 at position 34.3Molecular Location: base pairs 176,683,534 to 176,792,745 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        Flt4-LLMPH1DLMPHM4VRP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of VEGFC      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LYMPHATIC MALFORMATION 4OMIM: VASCULAR ENDOTHELIAL GROWTH FACTOR C      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of VEGFC  OMIM: LYMPHATIC MALFORMATION 4OMIM: VASCULAR ENDOTHELIAL GROWTH FACTOR C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CDKN2AIP\t\n",
            "C4orf47\t\n",
            "TLR3\t   From NCBI Gene:The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns (PAMPs) that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor is most abundantly expressed in placenta and pancreas, and is restricted to the dendritic subpopulation of the leukocytes. It recognizes dsRNA associated with viral infection, and induces the activation of NF-kappaB and the production of type I interferons. It may thus play a role in host defense against viruses. Use of alternative polyadenylation sites to generate different length transcripts has been noted for this gene. [provided by RefSeq, Jul 2008]From UniProt:Key component of innate and adaptive immunity. TLRs (Toll-like receptors) control host immune response against pathogens through recognition of molecular patterns specific to microorganisms. TLR3 is a nucleotide-sensing TLR which is activated by double-stranded RNA, a sign of viral infection. Acts via the adapter TRIF/TICAM1, leading to NF-kappa-B activation, IRF3 nuclear translocation, cytokine secretion and the inflammatory response.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Human immunodeficiency virus type 1, susceptibility toHerpes simplex encephalitis 2From UniProt:Encephalopathy, acute, infection-induced, Herpes-specific, 2 (IIAE2): A rare complication of human herpesvirus 1 (HHV-1) infection, occurring in only a small minority of HHV-1 infected individuals. It is characterized by hemorrhagic necrosis of parts of the temporal and frontal lobes. Onset is over several days and involves fever, headache, seizures, stupor, and often coma, frequently with a fatal outcome. [MIM:613002]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q35.1, which is the long (q) arm of chromosome 4 at position 35.1Molecular Location: base pairs 186,069,155 to 186,085,098 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD283IIAE2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TLR3      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ENCEPHALOPATHY, ACUTE, INFECTION-INDUCED (HERPES-SPECIFIC), SUSCEPTIBILITY TO, 2OMIM: HUMAN IMMUNODEFICIENCY VIRUS TYPE 1, SUSCEPTIBILITY TOOMIM: TOLL-LIKE RECEPTOR 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TLR3  OMIM: ENCEPHALOPATHY, ACUTE, INFECTION-INDUCED (HERPES-SPECIFIC), SUSCEPTIBILITY TO, 2OMIM: HUMAN IMMUNODEFICIENCY VIRUS TYPE 1, SUSCEPTIBILITY TOOMIM: TOLL-LIKE RECEPTOR 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FRG1\t   From NCBI Gene:This gene maps to a location 100 kb centromeric of the repeat units on chromosome 4q35 which are deleted in facioscapulohumeral muscular dystrophy (FSHD). It is evolutionarily conserved and has related sequences on multiple human chromosomes but DNA sequence analysis did not reveal any homology to known genes. In vivo studies demonstrate the encoded protein is localized to the nucleolus. [provided by RefSeq, Jul 2008]From UniProt:Binds to mRNA in a sequence-independent manner. May play a role in regulation of pre-mRNA splicing or in the assembly of rRNA into ribosomal subunits. May be involved in mRNA transport. May be involved in epigenetic regulation of muscle differentiation through regulation of activity of the histone-lysine N-methyltransferase KMT5B.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Facioscapulohumeral muscular dystrophyFrom UniProt:Facioscapulohumeral muscular dystrophy 1 (FSHD1): A degenerative muscle disease characterized by slowly progressive weakness of the muscles of the face, upper-arm, and shoulder girdle. The onset of symptoms usually occurs in the first or second decade of life. Affected individuals usually present with impairment of upper extremity elevation. This tends to be followed by facial weakness, primarily involving the orbicularis oris and orbicularis oculi muscles. [MIM:158900]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q35.2, which is the long (q) arm of chromosome 4 at position 35.2Molecular Location: base pairs 189,940,819 to 189,963,204 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FRG1AFSG1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FRG1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FACIOSCAPULOHUMERAL MUSCULAR DYSTROPHY 1OMIM: FSHD REGION GENE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FRG1  OMIM: FACIOSCAPULOHUMERAL MUSCULAR DYSTROPHY 1OMIM: FSHD REGION GENE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DUX4L4\t\n",
            "C4orf48\t\n",
            "POLN\t\n",
            "STIM2\t\n",
            "ADD1\t   From NCBI Gene:Adducins are a family of cytoskeletal proteins encoded by three genes (alpha, beta, and gamma). Adducin acts as a heterodimer of the related alpha, beta, or gamma subunits. The protein encoded by this gene represents the alpha subunit. Alpha- and beta-adducin include a protease-resistant N-terminal region and a protease-sensitive, hydrophilic C-terminal region. Adducin binds with high affinity to Ca(2+)/calmodulin and is a substrate for protein kinases A and C. [provided by RefSeq, Aug 2017]From UniProt:Membrane-cytoskeleton-associated protein that promotes the assembly of the spectrin-actin network. Binds to calmodulin.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Essential hypertension     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4p16.3, which is the short (p) arm of chromosome 4 at position 16.3Molecular Location: base pairs 2,843,857 to 2,930,076 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADDA     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADD1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ADDUCIN 1OMIM: HYPERTENSION, ESSENTIAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADD1  OMIM: ADDUCIN 1OMIM: HYPERTENSION, ESSENTIAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HTT\t   The HTT gene provides instructions for making a protein called huntingtin.  Although the exact function of this protein is unknown, it appears to play an important role in nerve cells (neurons) in the brain and is essential for normal development before birth.  Huntingtin is found in many of the body's tissues, with the highest levels of activity in the brain.  Within cells, this protein may be involved in chemical signaling, transporting materials, attaching (binding) to proteins and other structures, and protecting the cell from self-destruction (apoptosis).One region of the HTT gene contains a particular DNA segment known as a CAG trinucleotide repeat.   This segment is made up of a series of three DNA building blocks (cytosine, adenine, and guanine) that appear multiple times in a row.  Normally, the CAG segment is repeated 10 to 35 times within the gene.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Huntington disease     The inherited mutation that causes Huntington disease is known as a CAG trinucleotide repeat expansion.  This mutation increases the size of the CAG segment in the HTT gene.  People with Huntington disease have 36 to more than 120 CAG repeats. People with 36 to 39 CAG repeats may or may not develop the signs and symptoms of Huntington disease, while people with 40 or more repeats almost always develop the disorder.The expanded CAG segment leads to the production of an abnormally long version of the huntingtin protein. The elongated protein is cut into smaller, toxic fragments that bind together and accumulate in neurons, disrupting the normal functions of these cells. This process particularly affects regions of the brain that help coordinate movement and control thinking and emotions (the striatum and cerebral cortex).  The dysfunction and eventual death of neurons in these areas of the brain underlie the signs and symptoms of Huntington disease.As the altered HTT gene is passed from one generation to the next, the size of the CAG trinucleotide repeat often increases in size. A larger number of repeats is usually associated with an earlier onset of signs and symptoms. This phenomenon is called anticipation. People with the adult-onset form of Huntington disease (which appears in mid-adulthood) typically have 40 to 50 CAG repeats in the HTT gene, while people with the less common, juvenile form of the disorder (which appears in childhood or adolescence) tend to have more than 60 CAG repeats.Individuals who have 27 to 35 CAG repeats in the HTT gene do not develop Huntington disease, but they are at risk of having children who will develop the disorder. As the gene is passed from parent to child, the size of the CAG trinucleotide repeat may lengthen into the range associated with Huntington disease (36 repeats or more). More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      The inherited mutation that causes Huntington disease is known as a CAG trinucleotide repeat expansion.  This mutation increases the size of the CAG segment in the HTT gene.  People with Huntington disease have 36 to more than 120 CAG repeats. People with 36 to 39 CAG repeats may or may not develop the signs and symptoms of Huntington disease, while people with 40 or more repeats almost always develop the disorder.The expanded CAG segment leads to the production of an abnormally long version of the huntingtin protein. The elongated protein is cut into smaller, toxic fragments that bind together and accumulate in neurons, disrupting the normal functions of these cells. This process particularly affects regions of the brain that help coordinate movement and control thinking and emotions (the striatum and cerebral cortex).  The dysfunction and eventual death of neurons in these areas of the brain underlie the signs and symptoms of Huntington disease.As the altered HTT gene is passed from one generation to the next, the size of the CAG trinucleotide repeat often increases in size. A larger number of repeats is usually associated with an earlier onset of signs and symptoms. This phenomenon is called anticipation. People with the adult-onset form of Huntington disease (which appears in mid-adulthood) typically have 40 to 50 CAG repeats in the HTT gene, while people with the less common, juvenile form of the disorder (which appears in childhood or adolescence) tend to have more than 60 CAG repeats.Individuals who have 27 to 35 CAG repeats in the HTT gene do not develop Huntington disease, but they are at risk of having children who will develop the disorder. As the gene is passed from parent to child, the size of the CAG trinucleotide repeat may lengthen into the range associated with Huntington disease (36 repeats or more). More About This Health Condition    Cytogenetic Location: 4p16.3, which is the short (p) arm of chromosome 4 at position 16.3Molecular Location: base pairs 3,074,681 to 3,243,960 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HDHD_HUMANhuntingtin (Huntington disease)Huntington's disease proteinIT15     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Some Genetic Diseases Are Caused by the Expansion of Repeats of Three Nucleotides      Clinical Information from GeneReviews (1 link)     Huntington Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HTT      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     HUNTINGTIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Some Genetic Diseases Are Caused by the Expansion of Repeats of Three Nucleotides  Huntington Disease  Tests of HTT  PubMed  HUNTINGTIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bates G. Huntingtin aggregation and toxicity in Huntington's disease. Lancet. 2003 May 10;361(9369):1642-4. Review. Citation on PubMedBorrell-Pags M, Zala D, Humbert S, Saudou F. Huntington's disease: from huntingtin function and dysfunction to therapeutic strategies. Cell Mol Life Sci. 2006 Nov;63(22):2642-60. Review. Citation on PubMedCattaneo E. Dysfunction of wild-type huntingtin in Huntington disease. News Physiol Sci. 2003 Feb;18:34-7. Review. Citation on PubMedGrdin G, Vcsei L. Huntington's disease: pathomechanism and therapeutic perspectives. J Neural Transm (Vienna). 2004 Oct;111(10-11):1485-94. Review. Citation on PubMedImarisio S, Carmichael J, Korolchuk V, Chen CW, Saiki S, Rose C, Krishna G, Davies JE, Ttofi E, Underwood BR, Rubinsztein DC. Huntington's disease: from pathology and genetics to potential therapies. Biochem J. 2008 Jun 1;412(2):191-209. doi: 10.1042/BJ20071619. Review. Citation on PubMedJones L, Hughes A. Pathogenic mechanisms in Huntington's disease. Int Rev Neurobiol. 2011;98:373-418. doi: 10.1016/B978-0-12-381328-2.00015-8. Review. Citation on PubMedLandles C, Bates GP. Huntingtin and the molecular pathogenesis of Huntington's disease. Fourth in molecular medicine review series. EMBO Rep. 2004 Oct;5(10):958-63. Review. Citation on PubMed or Free article on PubMed CentralLi SH, Li XJ. Huntingtin and its role in neuronal degeneration. Neuroscientist. 2004 Oct;10(5):467-75. Review. Citation on PubMedMacDonald ME. Huntingtin: alive and well and working in middle management. Sci STKE. 2003 Nov 4;2003(207):pe48. Review. Citation on PubMedRangone H, Humbert S, Saudou F. Huntington's disease: how does huntingtin, an anti-apoptotic protein, become toxic? Pathol Biol (Paris). 2004 Jul;52(6):338-42. Review. Citation on PubMedSlow EJ, Graham RK, Hayden MR. To be or not to be toxic: aggregations in Huntington and Alzheimer disease. Trends Genet. 2006 Aug;22(8):408-11. Epub 2006 Jun 27. Citation on PubMedYoung AB. Huntingtin in health and disease. J Clin Invest. 2003 Feb;111(3):299-302. Review. Citation on PubMed or Free article on PubMed Centralvan Dellen A, Grote HE, Hannan AJ. Gene-environment interactions, neuronal dysfunction and pathological plasticity in Huntington's disease. Clin Exp Pharmacol Physiol. 2005 Dec;32(12):1007-19. Review. Citation on PubMed \n",
            "ZNF141\t   From NCBI Gene:The protein encoded by this gene is a zinc finger protein that may be a tumor suppressor. Defects in this gene have been associated with autosomal recessive postaxial polydactyly type A. [provided by RefSeq, Jan 2017]From UniProt:May be involved in transcriptional regulation as a repressor. Plays a role in limb development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Postaxial polydactyly type A6From UniProt:Polydactyly, postaxial A6 (PAPA6): A condition characterized by the occurrence of supernumerary digits in the upper and/or lower extremities. In postaxial polydactyly type A, the extra digit is well-formed and articulates with the fifth or a sixth metacarpal/metatarsal. [MIM:615226]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4p16.3, which is the short (p) arm of chromosome 4 at position 16.3Molecular Location: base pairs 337,779 to 384,864 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        D4S90pHZ-44     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ZNF141      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: POLYDACTYLY, POSTAXIAL, TYPE A6OMIM: ZINC FINGER PROTEIN 141      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ZNF141  OMIM: POLYDACTYLY, POSTAXIAL, TYPE A6OMIM: ZINC FINGER PROTEIN 141  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NWD2\t\n",
            "KLHL5\t\n",
            "NFXL1\t\n",
            "NIPAL1\t\n",
            "CWH43\t\n",
            "LRRC66\t\n",
            "FIP1L1\t   The FIP1L1 gene provides instructions for making part of a protein complex named cleavage and polyadenylation specificity factor (CPSF). This complex of proteins plays an important role in processing molecules called messenger RNAs (mRNAs), which serve as the genetic blueprints for making proteins. The CPSF protein complex helps add a string of the RNA building block adenine to the mRNA, creating a polyadenine tail or poly(A) tail. The poly(A) tail is important for stability of the mRNA and for protein production from the blueprint.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           PDGFRA-associated chronic eosinophilic leukemia     A deletion of genetic material from chromosome 4 brings together part of the FIP1L1 gene and part of another gene called PDGFRA, creating the FIP1L1-PDGFRA fusion gene. This mutation is a somatic mutation, which means it is acquired during a person's lifetime and is present only in certain cells. This fusion gene causes PDGFRA-associated chronic eosinophilic leukemia, which is a type of blood cell cancer characterized by an increased number of eosinophils, a type of white blood cell involved in allergic reactions.The FIP1L1-PDGFRA protein produced from the fusion gene has the function of the normal PDGFRA protein, which stimulates signaling pathways inside the cell that control many important cellular processes, such as cell growth and division (proliferation) and cell survival. Unlike the normal PDGFRA protein, however, the FIP1L1-PDGFRA protein is constantly turned on (constitutively activated), which means the cells are always receiving signals to proliferate. When the FIP1L1-PDGFRA fusion gene occurs in blood cell precursors, the growth of eosinophils (and occasionally other blood cells) is poorly controlled, leading to PDGFRA-associated chronic eosinophilic leukemia. It is unclear why eosinophils are preferentially affected by this genetic change. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A deletion of genetic material from chromosome 4 brings together part of the FIP1L1 gene and part of another gene called PDGFRA, creating the FIP1L1-PDGFRA fusion gene. This mutation is a somatic mutation, which means it is acquired during a person's lifetime and is present only in certain cells. This fusion gene causes PDGFRA-associated chronic eosinophilic leukemia, which is a type of blood cell cancer characterized by an increased number of eosinophils, a type of white blood cell involved in allergic reactions.The FIP1L1-PDGFRA protein produced from the fusion gene has the function of the normal PDGFRA protein, which stimulates signaling pathways inside the cell that control many important cellular processes, such as cell growth and division (proliferation) and cell survival. Unlike the normal PDGFRA protein, however, the FIP1L1-PDGFRA protein is constantly turned on (constitutively activated), which means the cells are always receiving signals to proliferate. When the FIP1L1-PDGFRA fusion gene occurs in blood cell precursors, the growth of eosinophils (and occasionally other blood cells) is poorly controlled, leading to PDGFRA-associated chronic eosinophilic leukemia. It is unclear why eosinophils are preferentially affected by this genetic change. More About This Health Condition    Cytogenetic Location: 4q12, which is the long (q) arm of chromosome 4 at position 12Molecular Location: base pairs 53,377,572 to 53,462,611 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FIP1-like 1 proteinFIP1_HUMANhFip1pre-mRNA 3'-end-processing factor FIP1Rhe     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Cell: A Molecular Approach (second edition, 2000): Processing of mRNA in Eukaryotes      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FIP1L1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FIP1-LIKE 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Cell: A Molecular Approach (second edition, 2000): Processing of mRNA in Eukaryotes  Tests of FIP1L1  PubMed  FIP1-LIKE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bain BJ. Relationship between idiopathic hypereosinophilic syndrome, eosinophilic leukemia, and systemic mastocytosis. Am J Hematol. 2004 Sep;77(1):82-5. Review. Citation on PubMedBuitenhuis M, Verhagen LP, Cools J, Coffer PJ. Molecular mechanisms underlying FIP1L1-PDGFRA-mediated myeloproliferation. Cancer Res. 2007 Apr 15;67(8):3759-66. Citation on PubMedCools J, DeAngelo DJ, Gotlib J, Stover EH, Legare RD, Cortes J, Kutok J, Clark J, Galinsky I, Griffin JD, Cross NC, Tefferi A, Malone J, Alam R, Schrier SL, Schmid J, Rose M, Vandenberghe P, Verhoef G, Boogaerts M, Wlodarska I, Kantarjian H, Marynen P, Coutre SE, Stone R, Gilliland DG. A tyrosine kinase created by fusion of the PDGFRA and FIP1L1 genes as a therapeutic target of imatinib in idiopathic hypereosinophilic syndrome. N Engl J Med. 2003 Mar 27;348(13):1201-14. Citation on PubMedOMIM: FIP1-LIKE 1Fukushima K, Matsumura I, Ezoe S, Tokunaga M, Yasumi M, Satoh Y, Shibayama H, Tanaka H, Iwama A, Kanakura Y. FIP1L1-PDGFRalpha imposes eosinophil lineage commitment on hematopoietic stem/progenitor cells. J Biol Chem. 2009 Mar 20;284(12):7719-32. doi: 10.1074/jbc.M807489200. Epub 2009 Jan 14. Citation on PubMed or Free article on PubMed CentralKaufmann I, Martin G, Friedlein A, Langen H, Keller W. Human Fip1 is a subunit of CPSF that binds to U-rich RNA elements and stimulates poly(A) polymerase. EMBO J. 2004 Feb 11;23(3):616-26. Epub 2004 Jan 29. Citation on PubMed or Free article on PubMed CentralRoufosse FE, Goldman M, Cogan E. Hypereosinophilic syndromes. Orphanet J Rare Dis. 2007 Sep 11;2:37. Review. Citation on PubMed or Free article on PubMed Central \n",
            "EVC\t   The EVC gene provides instructions for making a protein whose function is unclear. However, it appears to be important for normal growth and development, particularly the development of bones and teeth. The EVC protein is found in primary cilia, which are microscopic, finger-like projections that stick out from the surface of cells and are involved in signaling pathways that transmit information between cells. In particular, the EVC protein is thought to help regulate a signaling pathway known as Sonic Hedgehog, which plays roles in cell growth, cell specialization, and the normal shaping (patterning) of many parts of the body.EVC and another gene, EVC2, are located very close together on chromosome 4. Researchers believe that the two genes may have related functions and that their activity may be coordinated.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ellis-van Creveld syndrome     More than 25 mutations in the EVC gene have been reported to cause Ellis-van Creveld syndrome, an inherited disorder characterized by dwarfism, abnormal nails and teeth, and heart defects. The mutations that cause this condition occur in both copies of the EVC gene in each cell. These genetic changes disrupt the normal function of the EVC protein or lead to the production of an abnormally small, nonfunctional version of the protein. Although it is unclear how the loss of this protein's function underlies the signs and symptoms of Ellis-van Creveld syndrome, researchers believe that it may prevent normal Sonic Hedgehog signaling in the developing embryo. Problems with this signaling pathway may ultimately lead to the abnormal bone growth and heart defects seen with this condition. More About This Health Condition      Weyers acrofacial dysostosis     At least one mutation in the EVC gene has been found to cause the characteristic features of Weyers acrofacial dysostosis, affecting the development of the teeth, nails, and bones. The signs and symptoms of Weyers acrofacial dysostosis are similar to, but typically milder than, those of Ellis-van Creveld syndrome.The EVC gene mutation that causes Weyers acrofacial dysostosis occurs in one copy of the gene in each cell. It changes a single protein building block (amino acid) in the EVC protein, replacing the amino acid serine with the amino acid proline at position 307 (written as Ser307Pro). It is unclear how this genetic change leads to the specific features of Weyers acrofacial dysostosis. Studies suggest that the abnormal protein interferes with Sonic Hedgehog signaling in the developing embryo, disrupting the normal formation and growth of the teeth, nails, and bones. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 25 mutations in the EVC gene have been reported to cause Ellis-van Creveld syndrome, an inherited disorder characterized by dwarfism, abnormal nails and teeth, and heart defects. The mutations that cause this condition occur in both copies of the EVC gene in each cell. These genetic changes disrupt the normal function of the EVC protein or lead to the production of an abnormally small, nonfunctional version of the protein. Although it is unclear how the loss of this protein's function underlies the signs and symptoms of Ellis-van Creveld syndrome, researchers believe that it may prevent normal Sonic Hedgehog signaling in the developing embryo. Problems with this signaling pathway may ultimately lead to the abnormal bone growth and heart defects seen with this condition. More About This Health Condition  At least one mutation in the EVC gene has been found to cause the characteristic features of Weyers acrofacial dysostosis, affecting the development of the teeth, nails, and bones. The signs and symptoms of Weyers acrofacial dysostosis are similar to, but typically milder than, those of Ellis-van Creveld syndrome.The EVC gene mutation that causes Weyers acrofacial dysostosis occurs in one copy of the gene in each cell. It changes a single protein building block (amino acid) in the EVC protein, replacing the amino acid serine with the amino acid proline at position 307 (written as Ser307Pro). It is unclear how this genetic change leads to the specific features of Weyers acrofacial dysostosis. Studies suggest that the abnormal protein interferes with Sonic Hedgehog signaling in the developing embryo, disrupting the normal formation and growth of the teeth, nails, and bones. More About This Health Condition    Cytogenetic Location: 4p16.2, which is the short (p) arm of chromosome 4 at position 16.2Molecular Location: base pairs 5,711,197 to 5,829,043 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DWF-1Ellis van Creveld proteinEllis van Creveld syndromeEllis van Creveld syndrome proteinEVC1EVC_HUMANEVCL     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Developmental Biology (sixth edition, 2000): The Hedgehog Pathway      Tests Listed in the Genetic Testing Registry (1 link)     Tests of EVC      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     EVC GENE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): The Hedgehog Pathway  Tests of EVC  PubMed  EVC GENE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Blair HJ, Tompson S, Liu YN, Campbell J, MacArthur K, Ponting CP, Ruiz-Perez VL, Goodship JA. Evc2 is a positive modulator of Hedgehog signalling that interacts with Evc at the cilia membrane and is also found in the nucleus. BMC Biol. 2011 Feb 28;9:14. doi: 10.1186/1741-7007-9-14. Citation on PubMed or Free article on PubMed CentralHoward TD, Guttmacher AE, McKinnon W, Sharma M, McKusick VA, Jabs EW. Autosomal dominant postaxial polydactyly, nail dystrophy, and dental abnormalities map to chromosome 4p16, in the region containing the Ellis-van Creveld syndrome locus. Am J Hum Genet. 1997 Dec;61(6):1405-12. Citation on PubMed or Free article on PubMed CentralMorrell CH, Brant LJ. Modelling hearing thresholds in the elderly. Stat Med. 1991 Sep;10(9):1453-64. Citation on PubMedRuiz-Perez VL, Blair HJ, Rodriguez-Andres ME, Blanco MJ, Wilson A, Liu YN, Miles C, Peters H, Goodship JA. Evc is a positive mediator of Ihh-regulated bone growth that localises at the base of chondrocyte cilia. Development. 2007 Aug;134(16):2903-12. Citation on PubMedRuiz-Perez VL, Goodship JA. Ellis-van Creveld syndrome and Weyers acrodental dysostosis are caused by cilia-mediated diminished response to hedgehog ligands. Am J Med Genet C Semin Med Genet. 2009 Nov 15;151C(4):341-51. doi: 10.1002/ajmg.c.30226. Review. Citation on PubMedRuiz-Perez VL, Ide SE, Strom TM, Lorenz B, Wilson D, Woods K, King L, Francomano C, Freisinger P, Spranger S, Marino B, Dallapiccola B, Wright M, Meitinger T, Polymeropoulos MH, Goodship J. Mutations in a new gene in Ellis-van Creveld syndrome and Weyers acrodental dysostosis. Nat Genet. 2000 Mar;24(3):283-6. Erratum in: Nat Genet 2000 May;25(1):125. Citation on PubMedRuiz-Perez VL, Tompson SW, Blair HJ, Espinoza-Valdez C, Lapunzina P, Silva EO, Hamel B, Gibbs JL, Young ID, Wright MJ, Goodship JA. Mutations in two nonhomologous genes in a head-to-head configuration cause Ellis-van Creveld syndrome. Am J Hum Genet. 2003 Mar;72(3):728-32. Epub 2003 Feb 4. Citation on PubMed or Free article on PubMed CentralSund KL, Roelker S, Ramachandran V, Durbin L, Benson DW. Analysis of Ellis van Creveld syndrome gene products: implications for cardiovascular development and disease. Hum Mol Genet. 2009 May 15;18(10):1813-24. doi: 10.1093/hmg/ddp098. Epub 2009 Feb 27. Citation on PubMed or Free article on PubMed CentralTompson SW, Ruiz-Perez VL, Blair HJ, Barton S, Navarro V, Robson JL, Wright MJ, Goodship JA. Sequencing EVC and EVC2 identifies mutations in two-thirds of Ellis-van Creveld syndrome patients. Hum Genet. 2007 Jan;120(5):663-70. Epub 2006 Sep 21. Citation on PubMedValencia M, Lapunzina P, Lim D, Zannolli R, Bartholdi D, Wollnik B, Al-Ajlouni O, Eid SS, Cox H, Buoni S, Hayek J, Martinez-Frias ML, Antonio PA, Temtamy S, Aglan M, Goodship JA, Ruiz-Perez VL. Widening the mutation spectrum of EVC and EVC2: ectopic expression of Weyer variants in NIH 3T3 fibroblasts disrupts Hedgehog signaling. Hum Mutat. 2009 Dec;30(12):1667-75. doi: 10.1002/humu.21117. Citation on PubMed \n",
            "MAN2B2\t\n",
            "TMPRSS11A\t\n",
            "YTHDC1\t\n",
            "UGT2A3\t\n",
            "CABS1\t\n",
            "AFP\t   From NCBI Gene:This gene encodes alpha-fetoprotein, a major plasma protein produced by the yolk sac and the liver during fetal life. Alpha-fetoprotein expression in adults is often associated with hepatoma or teratoma. However, hereditary persistance of alpha-fetoprotein may also be found in individuals with no obvious pathology. The protein is thought to be the fetal counterpart of serum albumin, and the alpha-fetoprotein and albumin genes are present in tandem in the same transcriptional orientation on chromosome 4. Alpha-fetoprotein is found in monomeric as well as dimeric and trimeric forms, and binds copper, nickel, fatty acids and bilirubin. The level of alpha-fetoprotein in amniotic fluid is used to measure renal loss of protein to screen for spina bifida and anencephaly. [provided by RefSeq, Jul 2008]From UniProt:Binds copper, nickel, and fatty acids as well as, and bilirubin less well than, serum albumin. Only a small percentage (less than 2%) of the human AFP shows estrogen-binding properties.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Alpha-fetoprotein deficiencyAlpha-fetoprotein, hereditary persistence ofFrom UniProt:Alpha-fetoprotein, hereditary persistence (HPAFP): A benign autosomal dominant condition characterized by continued expression of alpha-fetoprotein in adult life. [MIM:615970]Alpha-fetoprotein deficiency (AFPD): A benign condition characterized by undetectable AFP levels in the amniotic fluid. Affected individuals are asymptomatic and present normal development. [MIM:615969]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q13.3, which is the long (q) arm of chromosome 4 at position 13.3Molecular Location: base pairs 73,436,219 to 73,455,785 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AFPDFETAHPAFP     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AFP      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ALPHA-FETOPROTEINOMIM: ALPHA-FETOPROTEIN DEFICIENCYOMIM: ALPHA-FETOPROTEIN, HEREDITARY PERSISTENCE OF      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AFP  OMIM: ALPHA-FETOPROTEINOMIM: ALPHA-FETOPROTEIN DEFICIENCYOMIM: ALPHA-FETOPROTEIN, HEREDITARY PERSISTENCE OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PPEF2\t\n",
            "SDAD1\t\n",
            "CPLX1\t   From NCBI Gene:Proteins encoded by the complexin/synaphin gene family are cytosolic proteins that function in synaptic vesicle exocytosis.  These proteins bind syntaxin, part of the SNAP receptor.  The protein product of this gene binds to the SNAP receptor complex and disrupts it, allowing transmitter release. [provided by RefSeq, Jul 2008]From UniProt:Positively regulates a late step in exocytosis of various cytoplasmic vesicles, such as synaptic vesicles and other secretory vesicles (PubMed:21785414). Organizes the SNAREs into a cross-linked zigzag topology that, when interposed between the vesicle and plasma membranes, is incompatible with fusion, thereby preventing SNAREs from releasing neurotransmitters until an action potential arrives at the synapse (PubMed:21785414). Also involved in glucose-induced secretion of insulin by pancreatic beta-cells. Essential for motor behavior.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:4p partial monosomy syndromeEPILEPTIC ENCEPHALOPATHY, EARLY INFANTILE, 63From UniProt:Epileptic encephalopathy, early infantile, 63 (EIEE63): A form of epileptic encephalopathy, a heterogeneous group of severe childhood onset epilepsies characterized by refractory seizures, neurodevelopmental impairment, and poor prognosis. Development is normal prior to seizure onset, after which cognitive and motor delays become apparent. EIEE63 is an autosomal recessive disease with onset in infancy. [MIM:617976]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4p16.3, which is the short (p) arm of chromosome 4 at position 16.3Molecular Location: base pairs 784,957 to 826,157 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CPX-ICPX1EIEE63     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CPLX1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: COMPLEXIN 1OMIM: EPILEPTIC ENCEPHALOPATHY, EARLY INFANTILE, 63OMIM: WOLF-HIRSCHHORN SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CPLX1  OMIM: COMPLEXIN 1OMIM: EPILEPTIC ENCEPHALOPATHY, EARLY INFANTILE, 63OMIM: WOLF-HIRSCHHORN SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FRAS1\t   The FRAS1 gene provides instructions for making a protein that is part of a group of proteins called the FRAS/FREM complex. This complex is found in basement membranes, which are thin, sheet-like structures that separate and support cells in many tissues. The FRAS/FREM complex is particularly important during development before birth. One of its roles is to anchor the top layer of skin by connecting the basement membrane of the top layer to the layer of skin below. The FRAS/FREM complex is also involved in the proper development of certain other organs and tissues, including the kidneys, although the mechanism is unclear.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Fraser syndrome     At least 24 mutations in the FRAS1 gene cause a condition called Fraser syndrome; FRAS1 gene mutations account for about half of cases of this condition. Fraser syndrome affects development before birth and is characterized by eyes that are completely covered by skin (cryptophthalmos), fusion of the skin between the fingers and toes (cutaneous syndactyly), and abnormalities of the kidneys and other organs and tissues.Most mutations in the FRAS1 gene involved in Fraser syndrome lead to production of an abnormal protein with little or no function. The abnormal protein is likely broken down quickly, preventing formation of the FRAS/FREM complex. Lack of the FRAS/FREM complex in the basement membrane of the skin leads to detachment of the top layer, causing blisters to form during development. These blisters likely prevent the proper formation of certain structures before birth, leading to cryptophthalmos and cutaneous syndactyly. It is unknown how lack of the FRAS/FREM complex leads to kidney abnormalities and other problems in Fraser syndrome. More About This Health Condition      Coloboma     Genetics Home Reference provides information about coloboma. More About This Health Condition      Congenital anomalies of kidney and urinary tract     Genetics Home Reference provides information about congenital anomalies of kidney and urinary tract. More About This Health Condition      Other disorders     Mutations in the FRAS1 gene have also been found in people with abnormalities of the kidneys and urinary tract but no other signs and symptoms of Fraser syndrome (described above). Such abnormalities are grouped together as congenital anomalies of the kidney and urinary tract (CAKUT). A common abnormality in people with FRAS1 gene mutations is the absence of one of the kidneys (a condition called renal agenesis). The FRAS1 gene mutations involved in CAKUT typically change single protein building blocks (amino acids) in the FRAS1 protein. Researchers speculate that the effects of these mutations are milder than those of mutations that cause Fraser syndrome; some FRAS1 protein function may still remain. How these gene mutations affect the FRAS/FREM complex or lead to renal agenesis and other CAKUT is unknown.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 24 mutations in the FRAS1 gene cause a condition called Fraser syndrome; FRAS1 gene mutations account for about half of cases of this condition. Fraser syndrome affects development before birth and is characterized by eyes that are completely covered by skin (cryptophthalmos), fusion of the skin between the fingers and toes (cutaneous syndactyly), and abnormalities of the kidneys and other organs and tissues.Most mutations in the FRAS1 gene involved in Fraser syndrome lead to production of an abnormal protein with little or no function. The abnormal protein is likely broken down quickly, preventing formation of the FRAS/FREM complex. Lack of the FRAS/FREM complex in the basement membrane of the skin leads to detachment of the top layer, causing blisters to form during development. These blisters likely prevent the proper formation of certain structures before birth, leading to cryptophthalmos and cutaneous syndactyly. It is unknown how lack of the FRAS/FREM complex leads to kidney abnormalities and other problems in Fraser syndrome. More About This Health Condition  Genetics Home Reference provides information about coloboma. More About This Health Condition  Genetics Home Reference provides information about congenital anomalies of kidney and urinary tract. More About This Health Condition  Mutations in the FRAS1 gene have also been found in people with abnormalities of the kidneys and urinary tract but no other signs and symptoms of Fraser syndrome (described above). Such abnormalities are grouped together as congenital anomalies of the kidney and urinary tract (CAKUT). A common abnormality in people with FRAS1 gene mutations is the absence of one of the kidneys (a condition called renal agenesis). The FRAS1 gene mutations involved in CAKUT typically change single protein building blocks (amino acids) in the FRAS1 protein. Researchers speculate that the effects of these mutations are milder than those of mutations that cause Fraser syndrome; some FRAS1 protein function may still remain. How these gene mutations affect the FRAS/FREM complex or lead to renal agenesis and other CAKUT is unknown.    Cytogenetic Location: 4q21.21, which is the long (q) arm of chromosome 4 at position 21.21Molecular Location: base pairs 78,056,968 to 78,544,269 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        extracellular matrix protein FRAS1FLJ14927FLJ22031Fraser syndrome 1KIAA1500     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FRAS1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     CONGENITAL ANOMALIES OF KIDNEY AND URINARY TRACT 1FRAS1 GENE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FRAS1  PubMed  CONGENITAL ANOMALIES OF KIDNEY AND URINARY TRACT 1FRAS1 GENE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: FRAS1 GENEKohl S, Hwang DY, Dworschak GC, Hilger AC, Saisawat P, Vivante A, Stajic N, Bogdanovic R, Reutter HM, Kehinde EO, Tasic V, Hildebrandt F. Mild recessive mutations in six Fraser syndrome-related genes cause isolated congenital anomalies of the kidney and urinary tract. J Am Soc Nephrol. 2014 Sep;25(9):1917-22. doi: 10.1681/ASN.2013101103. Epub 2014 Apr 3. Citation on PubMed or Free article on PubMed CentralPavlakis E, Chiotaki R, Chalepakis G. The role of Fras1/Frem proteins in the structure and function of basement membrane. Int J Biochem Cell Biol. 2011 Apr;43(4):487-95. doi: 10.1016/j.biocel.2010.12.016. Epub 2010 Dec 21. Review. Citation on PubMedPetrou P, Makrygiannis AK, Chalepakis G. The Fras1/Frem family of extracellular matrix proteins: structure, function, and association with Fraser syndrome and the mouse bleb phenotype. Connect Tissue Res. 2008;49(3):277-82. doi: 10.1080/03008200802148025. Citation on PubMedShort K, Wiradjaja F, Smyth I. Let's stick together: the role of the Fras1 and Frem proteins in epidermal adhesion. IUBMB Life. 2007 Jul;59(7):427-35. Review. Citation on PubMedvan Haelst MM, Maiburg M, Baujat G, Jadeja S, Monti E, Bland E, Pearce K; Fraser Syndrome Collaboration Group, Hennekam RC, Scambler PJ. Molecular study of 33 families with Fraser syndrome new data and mutation review. Am J Med Genet A. 2008 Sep 1;146A(17):2252-7. doi: 10.1002/ajmg.a.32440. Citation on PubMed \n",
            "SH3TC1\t\n",
            "ACOX3\t\n",
            "SEC31A\t   From NCBI Gene:The protein encoded by this gene shares similarity with the yeast Sec31 protein, and is a component of the outer layer of the coat protein complex II (COPII). The encoded protein is involved in vesicle budding from the endoplasmic reticulum (ER) and contains multiple WD repeats near the N-terminus and a proline-rich region in the C-terminal half. It associates with the protein encoded by the SEC13 homolog, nuclear pore and COPII coat complex component (SEC13), and is required for ER-Golgi transport. Monoubiquitylation of this protein by CUL3-KLHL12 was found to regulate the size of COPII coats to accommodate unusually shaped cargo. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2015]From UniProt:Component of the coat protein complex II (COPII) which promotes the formation of transport vesicles from the endoplasmic reticulum (ER) (PubMed:10788476). The coat has two main functions, the physical deformation of the endoplasmic reticulum membrane into vesicles and the selection of cargo molecules (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving SEC31A is associated with inflammatory myofibroblastic tumors (IMTs). Translocation t(2;4)(p23;q21) with ALK.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q21.22, which is the long (q) arm of chromosome 4 at position 21.22Molecular Location: base pairs 82,818,509 to 82,900,571 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABP125ABP130HSPC275HSPC334SEC31L1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SEC31A      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: SEC31, YEAST, HOMOLOG OF, A      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SEC31A  OMIM: SEC31, YEAST, HOMOLOG OF, A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "THAP9\t\n",
            "ZNF595\t\n",
            "HSD17B13\t\n",
            "DSPP\t   The DSPP gene provides instructions for making a protein called dentin sialophosphoprotein. Soon after it is produced, this protein is cut into two smaller proteins: dentin sialoprotein and dentin phosphoprotein. These proteins are components of dentin, which is a bone-like substance that makes up the protective middle layer of each tooth. A third smaller protein produced from dentin sialophosphoprotein, called dentin glycoprotein, was identified in pigs but has not been found in humans.Although the exact functions of the DSPP-derived proteins are unknown, these proteins appear to be essential for normal tooth development. Dentin phosphoprotein is thought to be involved in the normal hardening of collagen, the most abundant protein in dentin. Specifically, dentin phosphoprotein may play a role in the deposition of mineral crystals among collagen fibers (mineralization).The DSPP gene is also active in the inner ear, although it is unclear whether it plays a role in normal hearing.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Dentinogenesis imperfecta     More than 20 mutations in the DSPP gene have been identified in people with dentinogenesis imperfecta. These genetic changes are responsible for two forms of this disorder, type II and type III. Mutations in this gene also cause dentin dysplasia type II, a disorder with signs and symptoms very similar to those of dentinogenesis imperfecta. However, dentin dysplasia type II affects the primary (baby) teeth much more than the permanent teeth. Some researchers believe that this type of dentin dysplasia and dentinogenesis imperfecta types II and III are actually forms of a single disorder.About half of DSPP gene mutations affect dentin sialoprotein, altering its transport in cells. The remaining mutations affect dentin phosphoprotein, interfering with its normal production and/or secretion. As a result of these abnormalities of DSPP-related proteins, teeth have abnormally soft dentin. Teeth with defective dentin are discolored, weak, and prone to breakage and decay.Although the DSPP gene is active in the inner ear, it is unclear whether DSPP gene mutations are related to the hearing loss found in a few older individuals with dentinogenesis imperfecta type II. More About This Health Condition      Nonsyndromic hearing loss     Genetics Home Reference provides information about nonsyndromic hearing loss. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 20 mutations in the DSPP gene have been identified in people with dentinogenesis imperfecta. These genetic changes are responsible for two forms of this disorder, type II and type III. Mutations in this gene also cause dentin dysplasia type II, a disorder with signs and symptoms very similar to those of dentinogenesis imperfecta. However, dentin dysplasia type II affects the primary (baby) teeth much more than the permanent teeth. Some researchers believe that this type of dentin dysplasia and dentinogenesis imperfecta types II and III are actually forms of a single disorder.About half of DSPP gene mutations affect dentin sialoprotein, altering its transport in cells. The remaining mutations affect dentin phosphoprotein, interfering with its normal production and/or secretion. As a result of these abnormalities of DSPP-related proteins, teeth have abnormally soft dentin. Teeth with defective dentin are discolored, weak, and prone to breakage and decay.Although the DSPP gene is active in the inner ear, it is unclear whether DSPP gene mutations are related to the hearing loss found in a few older individuals with dentinogenesis imperfecta type II. More About This Health Condition  Genetics Home Reference provides information about nonsyndromic hearing loss. More About This Health Condition    Cytogenetic Location: 4q22.1, which is the long (q) arm of chromosome 4 at position 22.1Molecular Location: base pairs 87,608,529 to 87,616,873 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        dentin glycoproteindentin phosphophoryndentin phosphoproteindentin phosphoryndentin sialoproteinDFNA39DGI1DGPDPPDSPDSPP_HUMANDTDP2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     School of Dentistry, University of North Carolina at Chapel Hill      Tests Listed in the Genetic Testing Registry (1 link)     Tests of DSPP      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     DENTIN SIALOPHOSPHOPROTEIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      School of Dentistry, University of North Carolina at Chapel Hill  Tests of DSPP  PubMed  DENTIN SIALOPHOSPHOPROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Beattie ML, Kim JW, Gong SG, Murdoch-Kinch CA, Simmer JP, Hu JC. Phenotypic variation in dentinogenesis imperfecta/dentin dysplasia linked to 4q21. J Dent Res. 2006 Apr;85(4):329-33. Citation on PubMed or Free article on PubMed CentralDong J, Gu T, Jeffords L, MacDougall M. Dentin phosphoprotein compound mutation in dentin sialophosphoprotein causes dentinogenesis imperfecta type III. Am J Med Genet A. 2005 Jan 30;132A(3):305-9. Citation on PubMedKim JW, Hu JC, Lee JI, Moon SK, Kim YJ, Jang KT, Lee SH, Kim CC, Hahn SH, Simmer JP. Mutational hot spot in the DSPP gene causing dentinogenesis imperfecta type II. Hum Genet. 2005 Feb;116(3):186-91. Epub 2004 Dec 8. Citation on PubMedKim JW, Nam SH, Jang KT, Lee SH, Kim CC, Hahn SH, Hu JC, Simmer JP. A novel splice acceptor mutation in the DSPP gene causing dentinogenesis imperfecta type II. Hum Genet. 2004 Aug;115(3):248-54. Epub 2004 Jul 6. Citation on PubMedMacDougall M, Dong J, Acevedo AC. Molecular basis of human dentin diseases. Am J Med Genet A. 2006 Dec 1;140(23):2536-46. Review. Citation on PubMedMalmgren B, Lindskog S, Elgadi A, Norgren S. Clinical, histopathologic, and genetic investigation in two large families with dentinogenesis imperfecta type II. Hum Genet. 2004 Apr;114(5):491-8. Epub 2004 Feb 3. Citation on PubMedMcKnight DA, Simmer JP, Hart PS, Hart TC, Fisher LW. Overlapping DSPP mutations cause dentin dysplasia and dentinogenesis imperfecta. J Dent Res. 2008 Dec;87(12):1108-11. Erratum in: J Dent Res. 2009 Jan;88(1):95. Citation on PubMed or Free article on PubMed CentralMcKnight DA, Suzanne Hart P, Hart TC, Hartsfield JK, Wilson A, Wright JT, Fisher LW. A comprehensive analysis of normal variation and disease-causing mutations in the human DSPP gene. Hum Mutat. 2008 Dec;29(12):1392-404. doi: 10.1002/humu.20783. Citation on PubMedSong Y, Wang C, Peng B, Ye X, Zhao G, Fan M, Fu Q, Bian Z. Phenotypes and genotypes in 2 DGI families with different DSPP mutations. Oral Surg Oral Med Oral Pathol Oral Radiol Endod. 2006 Sep;102(3):360-74. Epub 2006 Jun 16. Citation on PubMedSong YL, Wang CN, Fan MW, Su B, Bian Z. Dentin phosphoprotein frameshift mutations in hereditary dentin disorders and their variation patterns in normal human population. J Med Genet. 2008 Jul;45(7):457-64. doi: 10.1136/jmg.2007.056911. Epub 2008 May 2. Citation on PubMedXiao S, Yu C, Chou X, Yuan W, Wang Y, Bu L, Fu G, Qian M, Yang J, Shi Y, Hu L, Han B, Wang Z, Huang W, Liu J, Chen Z, Zhao G, Kong X. Dentinogenesis imperfecta 1 with or without progressive hearing loss is associated with distinct mutations in DSPP. Nat Genet. 2001 Feb;27(2):201-4. Erratum in: Nat Genet 2001 Mar;27(3):345. Citation on PubMedYamakoshi Y, Hu JC, Fukae M, Iwata T, Kim JW, Zhang H, Simmer JP. Porcine dentin sialoprotein is a proteoglycan with glycosaminoglycan chains containing chondroitin 6-sulfate. J Biol Chem. 2005 Jan 14;280(2):1552-60. Epub 2004 Nov 10. Citation on PubMedYamakoshi Y, Hu JC, Fukae M, Zhang H, Simmer JP. Dentin glycoprotein: the protein in the middle of the dentin sialophosphoprotein chimera. J Biol Chem. 2005 Apr 29;280(17):17472-9. Epub 2005 Feb 23. Citation on PubMedYamakoshi Y, Hu JC, Iwata T, Kobayashi K, Fukae M, Simmer JP. Dentin sialophosphoprotein is processed by MMP-2 and MMP-20 in vitro and in vivo. J Biol Chem. 2006 Dec 15;281(50):38235-43. Epub 2006 Oct 17. Citation on PubMedZhang X, Zhao J, Li C, Gao S, Qiu C, Liu P, Wu G, Qiang B, Lo WH, Shen Y. DSPP mutation in dentinogenesis imperfecta Shields type II. Nat Genet. 2001 Feb;27(2):151-2. Citation on PubMed \n",
            "DMP1\t   From NCBI Gene:Dentin matrix acidic phosphoprotein is an extracellular matrix protein and a member of the small integrin binding ligand N-linked glycoprotein family. This protein, which is critical for proper mineralization of bone and dentin, is present in diverse cells of bone and tooth tissues. The protein contains a large number of acidic domains, multiple phosphorylation sites, a functional arg-gly-asp cell attachment sequence, and a DNA binding domain. In undifferentiated osteoblasts it is primarily a nuclear protein that regulates the expression of osteoblast-specific genes. During osteoblast maturation the protein becomes phosphorylated and is exported to the extracellular matrix, where it orchestrates mineralized matrix formation. Mutations in the gene are known to cause autosomal recessive hypophosphatemia, a disease that manifests as rickets and osteomalacia. The gene structure is conserved in mammals. Two transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Jul 2008]From UniProt:May have a dual function during osteoblast differentiation. In the nucleus of undifferentiated osteoblasts, unphosphorylated form acts as a transcriptional component for activation of osteoblast-specific genes like osteocalcin. During the osteoblast to osteocyte transition phase it is phosphorylated and exported into the extracellular matrix, where it regulates nucleation of hydroxyapatite.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Hereditary hypophosphatemic ricketsFrom NCBI Gene:Autosomal recessive hypophosphatemic vitamin D refractory ricketsFrom UniProt:Hypophosphatemic rickets, autosomal recessive, 1 (ARHR1): A hereditary form of hypophosphatemic rickets, a disorder of proximal renal tubule function that causes phosphate loss, hypophosphatemia and skeletal deformities, including rickets and osteomalacia unresponsive to vitamin D. Symptoms are bone pain, fractures and growth abnormalities. [MIM:241520]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 4q22.1, which is the long (q) arm of chromosome 4 at position 22.1Molecular Location: base pairs 87,650,302 to 87,664,361 on chromosome 4 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARHPARHRDMP-1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DMP1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DENTIN MATRIX ACIDIC PHOSPHOPROTEIN 1OMIM: HYPOPHOSPHATEMIC RICKETS, AUTOSOMAL RECESSIVE, 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DMP1  OMIM: DENTIN MATRIX ACIDIC PHOSPHOPROTEIN 1OMIM: HYPOPHOSPHATEMIC RICKETS, AUTOSOMAL RECESSIVE, 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TIGD2\t\n",
            "USP17L11\t\n",
            "USP17L12\t\n",
            "USP17L17\t\n",
            "USP17L20\t\n",
            "PDLIM5\t\n",
            "DGKQ\t\n",
            "SLCO6A1\t\n",
            "PPIP5K2\t\n",
            "NKD2\t\n",
            "EPB41L4A\t\n",
            "MCC\t\n",
            "KCNN2\t\n",
            "FAM170A\t\n",
            "ZNF474\t\n",
            "CEP120\t   From NCBI Gene:This gene encodes a protein that functions in the microtubule-dependent coupling of the nucleus and the centrosome. A similar protein in mouse plays a role in both interkinetic nuclear migration, which is a characteristic pattern of nuclear movement in neural progenitors, and in neural progenitor self-renewal. Mutations in this gene are predicted to result in neurogenic defects. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2009]From UniProt:Plays a role in the microtubule-dependent coupling of the nucleus and the centrosome. Involved in the processes that regulate centrosome-mediated interkinetic nuclear migration (INM) of neural progenitors and for proper positioning of neurons during brain development. Also implicated in the migration and selfrenewal of neural progenitors. Required for centriole duplication and maturation during mitosis and subsequent ciliogenesis (By similarity). Required for the recruitment of CEP295 to the proximal end of new-born centrioles at the centriolar microtubule wall during early S phase in a PLK4-dependent manner (PubMed:27185865).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Joubert syndromeAsphyxiating thoracic dystrophyFrom NCBI Gene:Short-rib thoracic dysplasia 13 with or without polydactylyJOUBERT SYNDROME 31From UniProt:Joubert syndrome 31 (JBTS31): A form of Joubert syndrome, a disorder presenting with cerebellar ataxia, oculomotor apraxia, hypotonia, neonatal breathing abnormalities and psychomotor delay. Neuroradiologically, it is characterized by cerebellar vermian hypoplasia/aplasia, thickened and reoriented superior cerebellar peduncles, and an abnormally large interpeduncular fossa, giving the appearance of a molar tooth on transaxial slices (molar tooth sign). Additional variable features include retinal dystrophy, renal disease, liver fibrosis, and polydactyly. JBTS31 inheritance is autosomal recessive. [MIM:617761]Short-rib thoracic dysplasia 13 with or without polydactyly (SRTD13): A form of short-rib thoracic dysplasia, a group of autosomal recessive ciliopathies that are characterized by a constricted thoracic cage, short ribs, shortened tubular bones, and a 'trident' appearance of the acetabular roof. Polydactyly is variably present. Non-skeletal involvement can include cleft lip/palate as well as anomalies of major organs such as the brain, eye, heart, kidneys, liver, pancreas, intestines, and genitalia. Some forms of the disease are lethal in the neonatal period due to respiratory insufficiency secondary to a severely restricted thoracic cage, whereas others are compatible with life. Disease spectrum encompasses Ellis-van Creveld syndrome, asphyxiating thoracic dystrophy (Jeune syndrome), Mainzer-Saldino syndrome, and short rib-polydactyly syndrome. [MIM:616300]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q23.2, which is the long (q) arm of chromosome 5 at position 23.2Molecular Location: base pairs 123,344,885 to 123,423,592 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CCDC100JBTS31SRTD13     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CEP120      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CENTROSOMAL PROTEIN, 120-KDOMIM: JOUBERT SYNDROME 31OMIM: SHORT-RIB THORACIC DYSPLASIA 13 WITH OR WITHOUT POLYDACTYLY      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CEP120  OMIM: CENTROSOMAL PROTEIN, 120-KDOMIM: JOUBERT SYNDROME 31OMIM: SHORT-RIB THORACIC DYSPLASIA 13 WITH OR WITHOUT POLYDACTYLY  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLC12A2\t\n",
            "ACSL6\t   From NCBI Gene:The protein encoded by this gene catalyzes the formation of acyl-CoA from fatty acids, ATP, and CoA, using magnesium as a cofactor. The encoded protein plays a major role in fatty acid metabolism in the brain. Translocations with the ETV6 gene are causes of myelodysplastic syndrome with basophilia, acute myelogenous leukemia with eosinophilia, and acute eosinophilic leukemia. Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Apr 2011]From UniProt:Activation of long-chain fatty acids for both synthesis of cellular lipids, and degradation via beta-oxidation. Plays an important role in fatty acid metabolism in brain and the acyl-CoAs produced may be utilized exclusively for the synthesis of the brain lipid.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving ACSL6 may be a cause of acute myelogenous leukemia with eosinophilia. Translocation t(5;12)(q31;p13) with ETV6.A chromosomal aberration involving ACSL6 may be a cause of myelodysplastic syndrome with basophilia. Translocation t(5;12)(q31;p13) with ETV6.A chromosomal aberration involving ACSL6 may be a cause of acute eosinophilic leukemia (AEL). Translocation t(5;12)(q31;p13) with ETV6.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q31.1, which is the long (q) arm of chromosome 5 at position 31.1Molecular Location: base pairs 131,949,973 to 132,012,068 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACS2FACL6LACS 6LACS2LACS5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ACSL6      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: ACYL-CoA SYNTHETASE LONG CHAIN FAMILY, MEMBER 6      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ACSL6  OMIM: ACYL-CoA SYNTHETASE LONG CHAIN FAMILY, MEMBER 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FSTL4\t\n",
            "DCANP1\t\n",
            "SMAD5\t\n",
            "BRD8\t\n",
            "SLC23A1\t\n",
            "TMEM173\t   The TMEM173 gene provides instructions for making a protein called STING, which is involved in immune system function. STING helps produce beta-interferon, a member of a class of proteins called cytokines that promote inflammation. Inflammation normally occurs when the immune system sends signaling molecules and white blood cells to a site of injury or disease to fight microbial invaders and help with tissue repair.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           STING-associated vasculopathy with onset in infancy     At least three TMEM173 gene mutations have been identified in individuals with STING-associated vasculopathy with onset in infancy (SAVI), a potentially life-threatening disorder that damages the skin, lungs, blood vessels, and other tissues. The mutations that cause SAVI are described as \"gain-of-function\" mutations because they enhance the activity of the STING protein, leading to overproduction of beta-interferon. Abnormally high beta-interferon levels cause excessive inflammation, resulting in damage to the body's own cells and tissues and leading to the signs and symptoms of SAVI. Disorders such as SAVI that result from abnormally increased inflammation are known as autoinflammatory diseases. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three TMEM173 gene mutations have been identified in individuals with STING-associated vasculopathy with onset in infancy (SAVI), a potentially life-threatening disorder that damages the skin, lungs, blood vessels, and other tissues. The mutations that cause SAVI are described as \"gain-of-function\" mutations because they enhance the activity of the STING protein, leading to overproduction of beta-interferon. Abnormally high beta-interferon levels cause excessive inflammation, resulting in damage to the body's own cells and tissues and leading to the signs and symptoms of SAVI. Disorders such as SAVI that result from abnormally increased inflammation are known as autoinflammatory diseases. More About This Health Condition    Cytogenetic Location: 5q31.2, which is the long (q) arm of chromosome 5 at position 31.2Molecular Location: base pairs 139,475,528 to 139,482,790 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        endoplasmic reticulum IFN stimulatorendoplasmic reticulum interferon stimulatorERISFLJ38577hMITAhSTINGmediator of IRF3 activationMITAmitochondrial mediator of IRF3 activationMPYSN-terminal methionine-proline-tyrosine-serine plasma membrane tetraspannerNET23SAVIstimulator of interferon genes proteinSTING     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Immunobiology (fifth edition, 2001): Induced Innate Immune Responses to Infection      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TMEM173      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSMEMBRANE PROTEIN 173      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology (fifth edition, 2001): Induced Innate Immune Responses to Infection  Tests of TMEM173  PubMed  TRANSMEMBRANE PROTEIN 173  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Burdette DL, Monroe KM, Sotelo-Troha K, Iwig JS, Eckert B, Hyodo M, Hayakawa Y, Vance RE. STING is a direct innate immune sensor of cyclic di-GMP. Nature. 2011 Sep 25;478(7370):515-8. doi: 10.1038/nature10429. Citation on PubMed or Free article on PubMed CentralChen H, Sun H, You F, Sun W, Zhou X, Chen L, Yang J, Wang Y, Tang H, Guan Y, Xia W, Gu J, Ishikawa H, Gutman D, Barber G, Qin Z, Jiang Z. Activation of STAT6 by STING is critical for antiviral innate immunity. Cell. 2011 Oct 14;147(2):436-46. doi: 10.1016/j.cell.2011.09.022. Citation on PubMedChiliveru S, Rahbek SH, Jensen SK, Jrgensen SE, Nissen SK, Christiansen SH, Mogensen TH, Jakobsen MR, Iversen L, Johansen C, Paludan SR. Inflammatory cytokines break down intrinsic immunological tolerance of human primary keratinocytes to cytosolic DNA. J Immunol. 2014 Mar 1;192(5):2395-404. doi: 10.4049/jimmunol.1302120. Epub 2014 Jan 31. Citation on PubMedIshikawa H, Barber GN. STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling. Nature. 2008 Oct 2;455(7213):674-8. doi: 10.1038/nature07317. Epub 2008 Aug 24. Erratum in: Nature. 2008 Nov 13;456(7219):274. Citation on PubMed or Free article on PubMed CentralIshikawa H, Barber GN. The STING pathway and regulation of innate immune signaling in response to DNA pathogens. Cell Mol Life Sci. 2011 Apr;68(7):1157-65. doi: 10.1007/s00018-010-0605-2. Epub 2010 Dec 15. Review. Citation on PubMed or Free article on PubMed CentralIshikawa H, Ma Z, Barber GN. STING regulates intracellular DNA-mediated, type I interferon-dependent innate immunity. Nature. 2009 Oct 8;461(7265):788-92. doi: 10.1038/nature08476. Epub 2009 Sep 23. Citation on PubMed or Free article on PubMed CentralLiu Y, Jesus AA, Marrero B, Yang D, Ramsey SE, Montealegre Sanchez GA, Tenbrock K, Wittkowski H, Jones OY, Kuehn HS, Lee CC, DiMattia MA, Cowen EW, Gonzalez B, Palmer I, DiGiovanna JJ, Biancotto A, Kim H, Tsai WL, Trier AM, Huang Y, Stone DL, Hill S, Kim HJ, St Hilaire C, Gurprasad S, Plass N, Chapelle D, Horkayne-Szakaly I, Foell D, Barysenka A, Candotti F, Holland SM, Hughes JD, Mehmet H, Issekutz AC, Raffeld M, McElwee J, Fontana JR, Minniti CP, Moir S, Kastner DL, Gadina M, Steven AC, Wingfield PT, Brooks SR, Rosenzweig SD, Fleisher TA, Deng Z, Boehm M, Paller AS, Goldbach-Mansky R. Activated STING in a vascular and pulmonary syndrome. N Engl J Med. 2014 Aug 7;371(6):507-18. doi: 10.1056/NEJMoa1312625. Epub 2014 Jul 16. Citation on PubMed or Free article on PubMed CentralPaludan SR, Bowie AG. Immune sensing of DNA. Immunity. 2013 May 23;38(5):870-80. doi: 10.1016/j.immuni.2013.05.004. Review. Citation on PubMed or Free article on PubMed CentralOMIM: TRANSMEMBRANE PROTEIN 173Xiao TS, Fitzgerald KA. The cGAS-STING pathway for DNA sensing. Mol Cell. 2013 Jul 25;51(2):135-9. doi: 10.1016/j.molcel.2013.07.004. Review. Citation on PubMed or Free article on PubMed Central \n",
            "NRG2\t   From NCBI Gene:This gene encodes a novel member of the neuregulin family of growth and differentiation factors. Through interaction with the ERBB family of receptors, this protein induces the growth and differentiation of epithelial, neuronal, glial, and other types of cells. The gene consists of 12 exons and the genomic structure is similar to that of neuregulin 1, another member of the neuregulin family of ligands. The products of these genes mediate distinct biological processes by acting at different sites in tissues and eliciting different biological responses in cells. This gene is located close to the region for demyelinating Charcot-Marie-Tooth disease locus, but is not responsible for this disease. Alternative transcript variants encoding distinct isoforms have been described. [provided by RefSeq, May 2010]From UniProt:Direct ligand for ERBB3 and ERBB4 tyrosine kinase receptors. Concomitantly recruits ERBB1 and ERBB2 coreceptors, resulting in ligand-stimulated tyrosine phosphorylation and activation of the ERBB receptors. May also promote the heterodimerization with the EGF receptor.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:5q31.3 microdeletion syndrome     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q31.2, which is the long (q) arm of chromosome 5 at position 31.2Molecular Location: base pairs 139,846,779 to 140,043,675 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DON1HRG2NTAK     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NRG2      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: NEUREGULIN 2      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NRG2  OMIM: NEUREGULIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SRA1\t   From NCBI Gene:Both long non-coding and protein-coding RNAs are transcribed from this gene, and they represent alternatively spliced transcript variants. This gene was initially defined as a non-coding RNA, which is a coactivator for several nuclear receptors (NRs) and is associated with breast cancer. It has now been found that this gene is involved in the regulation of many NR and non-NR activities, including metabolism, adipogenesis and chromatin organization. The long non-coding RNA transcripts interact with a variety of proteins, including the protein encoded by this gene. The encoded protein acts as a transcriptional repressor by binding to the non-coding RNA. [provided by RefSeq, Mar 2012]From UniProt:Functional RNA which acts as a transcriptional coactivator that selectively enhances steroid receptor-mediated transactivation ligand-independently through a mechanism involving the modulating N-terminal domain (AF-1) of steroid receptors. Also mediates transcriptional coactivation of steroid receptors ligand-dependently through the steroid-binding domain (AF-2). Enhances cellular proliferation and differentiation and promotes apoptosis in vivo. May play a role in tumorigenesis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Hypogonadotropic hypogonadism 7 with or without anosmia     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q31.3, which is the long (q) arm of chromosome 5 at position 31.3Molecular Location: base pairs 140,550,067 to 140,558,093 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        pp7684SRASRAPSTRAA1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SRA1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HYPOGONADOTROPIC HYPOGONADISM 7 WITH OR WITHOUT ANOSMIAOMIM: STEROID RECEPTOR RNA ACTIVATOR 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SRA1  OMIM: HYPOGONADOTROPIC HYPOGONADISM 7 WITH OR WITHOUT ANOSMIAOMIM: STEROID RECEPTOR RNA ACTIVATOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "WDR55\t\n",
            "PCDHA1\t\n",
            "PCDHB5\t\n",
            "PCDHB8\t\n",
            "PCDHB16\t\n",
            "PCDHB9\t\n",
            "PCDHB10\t\n",
            "PCDHGA8\t\n",
            "PCDHGA12\t\n",
            "DIAPH1\t   From NCBI Gene:This gene is a homolog of the Drosophila diaphanous gene, and has been linked to autosomal dominant, fully penetrant, nonsyndromic sensorineural progressive low-frequency hearing loss. Actin polymerization involves proteins known to interact with diaphanous protein in Drosophila and mouse. It has therefore been speculated that this gene may have a role in the regulation of actin polymerization in hair cells of the inner ear. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Actin nucleation and elongation factor required for the assembly of F-actin structures, such as actin cables and stress fibers (By similarity). Binds to the barbed end of the actin filament and slows down actin polymerization and depolymerization (By similarity). Required for cytokinesis, and transcriptional activation of the serum response factor (By similarity). DFR proteins couple Rho and Src tyrosine kinase during signaling and the regulation of actin dynamics (By similarity). Functions as a scaffold protein for MAPRE1 and APC to stabilize microtubules and promote cell migration (By similarity). Has neurite outgrowth promoting activity. Acts in a Rho-dependent manner to recruit PFY1 to the membrane (By similarity). In hear cells, it may play a role in the regulation of actin polymerization in hair cells (PubMed:20937854, PubMed:21834987, PubMed:26912466). The MEMO1-RHOA-DIAPH1 signaling pathway plays an important role in ERBB2-dependent stabilization of microtubules at the cell cortex (PubMed:20937854, PubMed:21834987). It controls the localization of APC and CLASP2 to the cell membrane, via the regulation of GSK3B activity (PubMed:20937854, PubMed:21834987). In turn, membrane-bound APC allows the localization of the MACF1 to the cell membrane, which is required for microtubule capture and stabilization (PubMed:20937854, PubMed:21834987). Plays a role in the regulation of cell morphology and cytoskeletal organization. Required in the control of cell shape (PubMed:20937854, PubMed:21834987). Plays a role in brain development (PubMed:24781755). Also acts as an actin nucleation and elongation factor in the nucleus by promoting nuclear actin polymerization inside the nucleus to drive serum-dependent SRF-MRTFA activity (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal dominant 1Seizures, cortical blindness, and microcephaly syndromeFrom UniProt:Deafness, autosomal dominant, 1 (DFNA1): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. Patients may have mild thrombocytopenia and enlarged platelets, although most of DFNA1 affected individuals do not have significant bleeding tendencies. [MIM:124900]Seizures, cortical blindness, and microcephaly syndrome (SCBMS): A severe autosomal recessive neurodevelopmental disorder characterized by microcephaly, early-onset seizures, severely delayed psychomotor development, short stature, and cortical blindness. [MIM:616632]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q31.3, which is the long (q) arm of chromosome 5 at position 31.3Molecular Location: base pairs 141,515,016 to 141,619,055 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNA1DIA1DRF1hDIA1LFHL1SCBMS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DIAPH1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DEAFNESS, AUTOSOMAL DOMINANT 1OMIM: DIAPHANOUS, DROSOPHILA, HOMOLOG OF, 1OMIM: SEIZURES, CORTICAL BLINDNESS, AND MICROCEPHALY SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DIAPH1  OMIM: DEAFNESS, AUTOSOMAL DOMINANT 1OMIM: DIAPHANOUS, DROSOPHILA, HOMOLOG OF, 1OMIM: SEIZURES, CORTICAL BLINDNESS, AND MICROCEPHALY SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PCDH1\t\n",
            "KCTD16\t\n",
            "RBM27\t\n",
            "PPP2R2B\t   From NCBI Gene:The product of this gene belongs to the phosphatase 2 regulatory subunit B family. Protein phosphatase 2 is one of the four major Ser/Thr phosphatases, and it is implicated in the negative control of cell growth and division. It consists of a common heteromeric core enzyme, which is composed of a catalytic subunit and a constant regulatory subunit, that associates with a variety of regulatory subunits. The B regulatory subunit might modulate substrate selectivity and catalytic activity. This gene encodes a beta isoform of the regulatory subunit B55 subfamily. Defects in this gene cause autosomal dominant spinocerebellar ataxia 12 (SCA12), a disease caused by degeneration of the cerebellum, sometimes involving the brainstem and spinal cord, and in resulting in poor coordination of speech and body movements. Multiple alternatively spliced variants, which encode different isoforms, have been identified for this gene. The 5' UTR of some of these variants includes a CAG trinucleotide repeat sequence (7-28 copies) that can be expanded to 55-78 copies in cases of SCA12. [provided by RefSeq, Jul 2016]From UniProt:The B regulatory subunit might modulate substrate selectivity and catalytic activity, and also might direct the localization of the catalytic enzyme to a particular subcellular compartment. Within the PP2A holoenzyme complex, isoform 2 is required to promote proapoptotic activity (By similarity). Isoform 2 regulates neuronal survival through the mitochondrial fission and fusion balance (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spinocerebellar ataxia 12From UniProt:Spinocerebellar ataxia 12 (SCA12): Spinocerebellar ataxia is a clinically and genetically heterogeneous group of cerebellar disorders. Patients show progressive incoordination of gait and often poor coordination of hands, speech and eye movements, due to degeneration of the cerebellum with variable involvement of the brainstem and spinal cord. SCA12 is an autosomal dominant cerebellar ataxia (ADCA). [MIM:604326]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q32, which is the long (q) arm of chromosome 5 at position 32Molecular Location: base pairs 146,589,504 to 147,081,520 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        B55BETAPP2AB55BETAPP2ABBETAPP2APR55BPP2APR55BETAPR2AB55BETAPR2ABBETAPR2APR55BETAPR52BPR55-BETAPR55BETASCA12     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PPP2R2B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PROTEIN PHOSPHATASE 2, REGULATORY SUBUNIT B, BETAOMIM: SPINOCEREBELLAR ATAXIA 12      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PPP2R2B  OMIM: PROTEIN PHOSPHATASE 2, REGULATORY SUBUNIT B, BETAOMIM: SPINOCEREBELLAR ATAXIA 12  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "C5orf46\t\n",
            "SPINK6\t\n",
            "TIGD6\t\n",
            "CDX1\t\n",
            "TCOF1\t   The TCOF1 gene provides instructions for making a protein called treacle.  This protein is active during early embryonic development in structures that become bones and other tissues of the face, and it appears to play a critical role in the formation of these structures.Studies suggest that treacle is involved in the production of a molecule called ribosomal RNA (rRNA), a chemical cousin of DNA. Ribosomal RNA helps assemble protein building blocks (amino acids) into functioning proteins, which is essential for the normal functioning and survival of cells. Treacle is active in the nucleolus, which is a small region inside the nucleus where rRNA is produced.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Treacher Collins syndrome     About 200 mutations in the TCOF1 gene have been identified in people with Treacher Collins syndrome, a condition that affects the development of bones and other tissues of the face. Most of these mutations insert or delete a small number of DNA building blocks (base pairs) in the TCOF1 gene, which leads to a reduction in the amount of functional treacle in cells. As a result, the production of rRNA is reduced, which likely triggers the self-destruction (apoptosis) of certain cells involved in the early development of facial bones and tissues. Researchers believe that this abnormal cell death may lead to the specific problems with facial development found in Treacher Collins syndrome. However, it is unclear why the effects of a reduction in rRNA are limited to facial development. More About This Health Condition      Coloboma     Genetics Home Reference provides information about coloboma. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      About 200 mutations in the TCOF1 gene have been identified in people with Treacher Collins syndrome, a condition that affects the development of bones and other tissues of the face. Most of these mutations insert or delete a small number of DNA building blocks (base pairs) in the TCOF1 gene, which leads to a reduction in the amount of functional treacle in cells. As a result, the production of rRNA is reduced, which likely triggers the self-destruction (apoptosis) of certain cells involved in the early development of facial bones and tissues. Researchers believe that this abnormal cell death may lead to the specific problems with facial development found in Treacher Collins syndrome. However, it is unclear why the effects of a reduction in rRNA are limited to facial development. More About This Health Condition  Genetics Home Reference provides information about coloboma. More About This Health Condition    Cytogenetic Location: 5q32-q33.1, which is the long (q) arm of chromosome 5 between positions 32 and 33.1Molecular Location: base pairs 150,357,639 to 150,400,308 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        TCOF_HUMANTCSTreacher Collins-Franceschetti syndrome 1Treacher Collins syndrome proteintreacle     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): The Three Roles of RNA in Protein SynthesisThe Cell: A Molecular Approach (second edition, 2000): The Nucleolus      Clinical Information from GeneReviews (1 link)     Treacher Collins Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TCOF1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TCOF1 GENE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): The Three Roles of RNA in Protein SynthesisThe Cell: A Molecular Approach (second edition, 2000): The Nucleolus  Treacher Collins Syndrome  Tests of TCOF1  PubMed  TCOF1 GENE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Conte C, D'Apice MR, Rinaldi F, Gambardella S, Sangiuolo F, Novelli G. Novel mutations of TCOF1 gene in European patients with Treacher Collins syndrome. BMC Med Genet. 2011 Sep 27;12:125. doi: 10.1186/1471-2350-12-125. Citation on PubMed or Free article on PubMed CentralDixon J, Jones NC, Sandell LL, Jayasinghe SM, Crane J, Rey JP, Dixon MJ, Trainor PA. Tcof1/Treacle is required for neural crest cell formation and proliferation deficiencies that cause craniofacial abnormalities. Proc Natl Acad Sci U S A. 2006 Sep 5;103(36):13403-8. Epub 2006 Aug 28. Citation on PubMed or Free article on PubMed CentralGonzales B, Henning D, So RB, Dixon J, Dixon MJ, Valdez BC. The Treacher Collins syndrome (TCOF1) gene product is involved in pre-rRNA methylation. Hum Mol Genet. 2005 Jul 15;14(14):2035-43. Epub 2005 Jun 1. Citation on PubMedHoriuchi K, Ariga T, Fujioka H, Kawashima K, Yamamoto Y, Igawa H, Sugihara T, Sakiyama Y. Mutational analysis of the TCOF1 gene in 11 Japanese patients with Treacher Collins Syndrome and mechanism of mutagenesis. Am J Med Genet A. 2005 May 1;134(4):363-7. Citation on PubMedKatsanis SH, Jabs EW. Treacher Collins Syndrome. 2004 Jul 20 [updated 2012 Aug 30]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1532/ Citation on PubMedMarszaek B, Wjcicki P, Kobus K, Trzeciak WH. Clinical features, treatment and genetic background of Treacher Collins syndrome. J Appl Genet. 2002;43(2):223-33. Review. Citation on PubMedSakai D, Trainor PA. Treacher Collins syndrome: unmasking the role of Tcof1/treacle. Int J Biochem Cell Biol. 2009 Jun;41(6):1229-32. doi: 10.1016/j.biocel.2008.10.026. Epub 2008 Nov 5. Review. Citation on PubMed or Free article on PubMed CentralSplendore A, Fanganiello RD, Masotti C, Morganti LS, Passos-Bueno MR. TCOF1 mutation database: novel mutation in the alternatively spliced exon 6A and update in mutation nomenclature. Hum Mutat. 2005 May;25(5):429-34. Citation on PubMedSplendore A, Silva EO, Alonso LG, Richieri-Costa A, Alonso N, Rosa A, Carakushanky G, Cavalcanti DP, Brunoni D, Passos-Bueno MR. High mutation detection rate in TCOF1 among Treacher Collins syndrome patients reveals clustering of mutations and 16 novel pathogenic changes. Hum Mutat. 2000 Oct;16(4):315-22. Review. Citation on PubMedValdez BC, Henning D, So RB, Dixon J, Dixon MJ. The Treacher Collins syndrome (TCOF1) gene product is involved in ribosomal DNA gene transcription by interacting with upstream binding factor. Proc Natl Acad Sci U S A. 2004 Jul 20;101(29):10709-14. Epub 2004 Jul 12. Citation on PubMed or Free article on PubMed Central \n",
            "IRGM\t   The IRGM gene provides instructions for making a protein that plays an important role in the immune system. This protein is involved in a process called autophagy, which cells use to surround and destroy foreign invaders such as bacteria and viruses. Specifically, the IRGM protein helps trigger autophagy in cells infected with certain kinds of bacteria, including the type of bacteria that causes tuberculosis. In addition to protecting cells from infection, autophagy is used to recycle worn-out cell parts and break down certain proteins when they are no longer needed. This process also plays an important role in controlled cell death (apoptosis).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Crohn disease     Several variations in or near the IRGM gene have been associated with an increased risk of developing Crohn disease. This increased risk has been found primarily in people of northern European ancestry. IRGM variations change single DNA building blocks (nucleotides) in regions of DNA that may regulate when and how the IRGM protein is produced. Researchers suspect that changes involving the IRGM protein may disrupt the autophagy process, preventing the immune system from destroying harmful bacteria effectively. An abnormal immune response to bacteria in the intestinal walls may lead to chronic inflammation and the digestive problems characteristic of Crohn disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several variations in or near the IRGM gene have been associated with an increased risk of developing Crohn disease. This increased risk has been found primarily in people of northern European ancestry. IRGM variations change single DNA building blocks (nucleotides) in regions of DNA that may regulate when and how the IRGM protein is produced. Researchers suspect that changes involving the IRGM protein may disrupt the autophagy process, preventing the immune system from destroying harmful bacteria effectively. An abnormal immune response to bacteria in the intestinal walls may lead to chronic inflammation and the digestive problems characteristic of Crohn disease. More About This Health Condition    Cytogenetic Location: 5q33.1, which is the long (q) arm of chromosome 5 at position 33.1Molecular Location: base pairs 150,846,523 to 150,902,402 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        A1A4Y4_HUMANIFI1immunity-related GTPase family, Mimmunity-related GTPase family, M1immunity-related GTPase MIRGM1LRG-47LRG-47-like proteinLRG47MGC149263MGC149264     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Eurekah Bioscience Collection: Origin and Evolution of Self-Consumption: Autophagy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of IRGM      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     IMMUNITY-RELATED GTPase FAMILY, M      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Eurekah Bioscience Collection: Origin and Evolution of Self-Consumption: Autophagy  Tests of IRGM  PubMed  IMMUNITY-RELATED GTPase FAMILY, M  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Chauhan S, Mandell MA, Deretic V. Mechanism of action of the tuberculosis and Crohn disease risk factor IRGM in autophagy. Autophagy. 2016;12(2):429-31. doi: 10.1080/15548627.2015.1084457. Epub 2015 Aug 27. Citation on PubMed or Free article on PubMed CentralParkes M, Barrett JC, Prescott NJ, Tremelling M, Anderson CA, Fisher SA, Roberts RG, Nimmo ER, Cummings FR, Soars D, Drummond H, Lees CW, Khawaja SA, Bagnall R, Burke DA, Todhunter CE, Ahmad T, Onnie CM, McArdle W, Strachan D, Bethel G, Bryan C, Lewis CM, Deloukas P, Forbes A, Sanderson J, Jewell DP, Satsangi J, Mansfield JC; Wellcome Trust Case Control Consortium, Cardon L, Mathew CG. Sequence variants in the autophagy gene IRGM and multiple other replicating loci contribute to Crohn's disease susceptibility. Nat Genet. 2007 Jul;39(7):830-2. Epub 2007 Jun 6. Citation on PubMed or Free article on PubMed CentralRufini S, Ciccacci C, Di Fusco D, Ruffa A, Pallone F, Novelli G, Biancone L, Borgiani P. Autophagy and inflammatory bowel disease: Association between variants of the autophagy-related IRGM gene and susceptibility to Crohn's disease. Dig Liver Dis. 2015 Sep;47(9):744-50. doi: 10.1016/j.dld.2015.05.012. Epub 2015 May 21. Citation on PubMedSingh SB, Davis AS, Taylor GA, Deretic V. Human IRGM induces autophagy to eliminate intracellular mycobacteria. Science. 2006 Sep 8;313(5792):1438-41. Epub 2006 Aug 3. Citation on PubMed \n",
            "FAT2\t   From NCBI Gene:This gene is the second identified human homolog of the Drosophila fat gene, which encodes a tumor suppressor essential for controlling cell proliferation during Drosophila development. The gene product is a member of the cadherin superfamily, a group of integral membrane proteins characterized by the presence of cadherin-type repeats. In addition to containing 34 tandem cadherin-type repeats, the gene product has two epidermal growth factor (EGF)-like repeats and one laminin G domain. This protein most likely functions as a cell adhesion molecule, controlling cell proliferation and playing an important role in cerebellum development. [provided by RefSeq, Jul 2008]From UniProt:Involved in the regulation of cell migration (PubMed:18534823). May be involved in mediating the organization of the parallel fibers of granule cells during cerebellar development (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:SPINOCEREBELLAR ATAXIA 45From UniProt:Spinocerebellar ataxia 45 (SCA45): A form of spinocerebellar ataxia, a clinically and genetically heterogeneous group of cerebellar disorders. Patients show progressive incoordination of gait and often poor coordination of hands, speech and eye movements, due to degeneration of the cerebellum with variable involvement of the brainstem and spinal cord. SCA45 is a slowly progressive, autosomal dominant form with onset in adulthood. [MIM:617769]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q33.1, which is the long (q) arm of chromosome 5 at position 33.1Molecular Location: base pairs 151,504,092 to 151,594,822 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CDHF8CDHR9HFAT2MEGF1SCA45     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FAT2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FAT TUMOR SUPPRESSOR, DROSOPHILA, HOMOLOG OF, 2OMIM: SPINOCEREBELLAR ATAXIA 45      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FAT2  OMIM: FAT TUMOR SUPPRESSOR, DROSOPHILA, HOMOLOG OF, 2OMIM: SPINOCEREBELLAR ATAXIA 45  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MFAP3\t\n",
            "GEMIN5\t\n",
            "KIF4B\t\n",
            "HAVCR1\t\n",
            "CYFIP2\t\n",
            "ADAM19\t   From NCBI Gene:This gene encodes a member of the ADAM (a disintegrin and metalloprotease domain) family. Members of this family are membrane-anchored proteins structurally related to snake venom disintegrins and have been implicated in a variety of biological processes involving cell-cell and cell-matrix interactions, including fertilization, muscle development, and neurogenesis. This member is a type I transmembrane protein and serves as a marker for dendritic cell differentiation. It has been demonstrated to be an active metalloproteinase, which may be involved in normal physiological processes such as cell migration, cell adhesion, cell-cell and cell-matrix interactions, and signal transduction. It is proposed to play a role in pathological processes, such as cancer, inflammatory diseases, renal diseases, and Alzheimer's disease. [provided by RefSeq, May 2013]From UniProt:Participates in the proteolytic processing of beta-type neuregulin isoforms which are involved in neurogenesis and synaptogenesis, suggesting a regulatory role in glial cell. Also cleaves alpha-2 macroglobulin. May be involved in osteoblast differentiation and/or osteoblast activity in bone (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:C3 glomerulopathy     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q33.3, which is the long (q) arm of chromosome 5 at position 33.3Molecular Location: base pairs 157,477,304 to 157,575,823 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FKSG34MADDAMMLTNB     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADAM19      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: A DISINTEGRIN AND METALLOPROTEINASE DOMAIN 19      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADAM19  OMIM: A DISINTEGRIN AND METALLOPROTEINASE DOMAIN 19  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RNF145\t\n",
            "ATP10B\t\n",
            "HMMR\t   From NCBI Gene:The protein encoded by this gene is involved in cell motility. It is expressed in breast tissue and together with other proteins, it forms a complex with BRCA1 and BRCA2, thus is potentially associated with higher risk of breast cancer. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene. [provided by RefSeq, Dec 2008]From UniProt:Receptor for hyaluronic acid (HA) (By similarity). Involved in cell motility (By similarity). When hyaluronan binds to HMMR, the phosphorylation of a number of proteins, including PTK2/FAK1 occurs. May also be involved in cellular transformation and metastasis formation, and in regulating extracellular-regulated kinase (ERK) activity. May act as a regulator of adipogenisis (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Familial cancer of breast     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q34, which is the long (q) arm of chromosome 5 at position 34Molecular Location: base pairs 163,460,511 to 163,491,945 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD168IHABPRHAMM     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HMMR      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: BREAST CANCEROMIM: HYALURONAN-MEDIATED MOTILITY RECEPTOR      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HMMR  OMIM: BREAST CANCEROMIM: HYALURONAN-MEDIATED MOTILITY RECEPTOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYO10\t\n",
            "NOP16\t\n",
            "CDHR2\t\n",
            "GPRIN1\t\n",
            "UIMC1\t\n",
            "FGFR4\t   The FGFR4 gene provides instructions for making a protein called fibroblast growth factor receptor 4. This protein is part of a family of fibroblast growth factor receptors that share similar structures and functions. These receptor proteins play a role in important processes such as cell division, regulating cell growth and maturation, formation of blood vessels, wound healing, and embryo development.The FGFR4 protein interacts with specific growth factors to conduct signals from the environment outside the cell to the nucleus. The nucleus responds to these signals by switching on or off appropriate genes that help the cell adjust to changes in the environment. In response, the cell might divide, move, or mature to take on specialized functions. Although specific functions of FGFR4 remain unclear, studies indicate that the gene is involved in muscle development and the maturation of bone cells in the skull. The FGFR4 gene may also play a role in the development and maintenance of specialized cells (called foveal cones) in the light-sensitive layer (the retina) at the back of the eye.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Prostate cancer     Genetics Home Reference provides information about prostate cancer. More About This Health Condition      Cancers     A variation (polymorphism) in the FGFR4 gene that causes a switch in amino acids (the building blocks of proteins) is associated with several types of cancer, such as those that occur in the breast, colon, head and neck, and prostate.  In people with this polymorphism, glycine is replaced by arginine at position 388 in the protein's chain of amino acids.  This variation is common and appears to occur in about 50 percent of humans.  Although it produces no ill effects in healthy people, the mutation is associated with accelerated disease progression in certain cancers.The abnormal activation and increased activity of the FGFR4 gene are also implicated in the development of pituitary tumors and gastric, pancreatic, and ovarian cancers.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about prostate cancer. More About This Health Condition  A variation (polymorphism) in the FGFR4 gene that causes a switch in amino acids (the building blocks of proteins) is associated with several types of cancer, such as those that occur in the breast, colon, head and neck, and prostate.  In people with this polymorphism, glycine is replaced by arginine at position 388 in the protein's chain of amino acids.  This variation is common and appears to occur in about 50 percent of humans.  Although it produces no ill effects in healthy people, the mutation is associated with accelerated disease progression in certain cancers.The abnormal activation and increased activity of the FGFR4 gene are also implicated in the development of pituitary tumors and gastric, pancreatic, and ovarian cancers.    Cytogenetic Location: 5q35.2, which is the long (q) arm of chromosome 5 at position 35.2Molecular Location: base pairs 177,086,872 to 177,098,142 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CD334FGR4_HUMANhydroxyaryl-protein kinaseJTK2 Geneprotein-tyrosine kinaseTKF Genetyrosylprotein kinase     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FGFR4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FIBROBLAST GROWTH FACTOR RECEPTOR 4      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FGFR4  PubMed  FIBROBLAST GROWTH FACTOR RECEPTOR 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bange J, Prechtl D, Cheburkin Y, Specht K, Harbeck N, Schmitt M, Knyazeva T, Mller S, Grtner S, Sures I, Wang H, Imyanitov E, Hring HU, Knayzev P, Iacobelli S, Hfler H, Ullrich A. Cancer progression and tumor cell motility are associated with the FGFR4 Arg(388) allele. Cancer Res. 2002 Feb 1;62(3):840-7. Citation on PubMedCornish EE, Natoli RC, Hendrickson A, Provis JM. Differential distribution of fibroblast growth factor receptors (FGFRs) on foveal cones: FGFR-4 is an early marker of cone photoreceptors. Mol Vis. 2004 Jan 8;10:1-14. Citation on PubMedEswarakumar VP, Lax I, Schlessinger J. Cellular signaling by fibroblast growth factor receptors. Cytokine Growth Factor Rev. 2005 Apr;16(2):139-49. Epub 2005 Feb 1. Review. Citation on PubMedMarics I, Padilla F, Guillemot JF, Scaal M, Marcelle C. FGFR4 signaling is a necessary step in limb muscle differentiation. Development. 2002 Oct;129(19):4559-69. Citation on PubMedStreit S, Bange J, Fichtner A, Ihrler S, Issing W, Ullrich A. Involvement of the FGFR4 Arg388 allele in head and neck squamous cell carcinoma. Int J Cancer. 2004 Aug 20;111(2):213-7. Citation on PubMedWang J, Stockton DW, Ittmann M. The fibroblast growth factor receptor-4 Arg388 allele is associated with prostate cancer initiation and progression. Clin Cancer Res. 2004 Sep 15;10(18 Pt 1):6169-78. Citation on PubMed \n",
            "NSD1\t   The NSD1 gene provides instructions for making a protein that functions as a histone methyltransferase. Histone methyltransferases are enzymes that modify structural proteins called histones, which attach (bind) to DNA and give chromosomes their shape. By adding a molecule called a methyl group to histones (a process called methylation), histone methyltransferases control (regulate) the activity of certain genes and can turn them on and off as needed. The NSD1 enzyme controls the activity of genes involved in normal growth and development, although most of these genes have not been identified.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Sotos syndrome     More than 380 mutations in the NSD1 gene have been identified in people with Sotos syndrome. The most common mutation in the Japanese population deletes genetic material from the region of chromosome 5 that contains the NSD1 gene. In most other populations, mutations within the gene itself are more frequent. These mutations include insertions or deletions of a small amount of DNA and changes in single DNA building blocks (base pairs) that make up the gene. Most mutations prevent one copy of the NSD1 gene from making any enzyme or lead to the production of an abnormally small, nonfunctional version of the enzyme. Research suggests that a reduced amount of the NSD1 enzyme disrupts the normal activity of genes involved in growth and development. However, it remains unclear exactly how a shortage of this enzyme during development leads to overgrowth, learning disabilities, and the other signs and symptoms of Sotos syndrome. More About This Health Condition      Cancers     A change involving the NSD1 gene is associated with a blood cancer called childhood acute myeloid leukemia. This change occurs when part of chromosome 5 breaks off and reattaches to part of chromosome 11. This change is acquired during a person's lifetime and is present only in cancer cells. This type of genetic change, called a somatic mutation, is not inherited. The rearrangement of genetic material involved in childhood acute myeloid leukemia, known as a translocation, abnormally fuses the NSD1 gene on chromosome 5 with the NUP98 gene on chromosome 11. Research shows that the fused NUP98-NSD1 gene turns on genes that promote the growth of immature blood cells and blocks processes that would turn the genes off. The resulting overgrowth of these immature cells leads to development of acute myeloid leukemia.A different type of alteration involving the NSD1 gene is associated with a cancer of nerve tissue called neuroblastoma and a type of brain cancer called glioma. This alteration, known as promoter hypermethylation, turns off the production of the NSD1 enzyme. Researchers speculate that without NSD1, the activity of one or more genes involved in cell growth and division is uncontrolled. As a result, the cells can grow and divide unchecked, leading to the development of cancer.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 380 mutations in the NSD1 gene have been identified in people with Sotos syndrome. The most common mutation in the Japanese population deletes genetic material from the region of chromosome 5 that contains the NSD1 gene. In most other populations, mutations within the gene itself are more frequent. These mutations include insertions or deletions of a small amount of DNA and changes in single DNA building blocks (base pairs) that make up the gene. Most mutations prevent one copy of the NSD1 gene from making any enzyme or lead to the production of an abnormally small, nonfunctional version of the enzyme. Research suggests that a reduced amount of the NSD1 enzyme disrupts the normal activity of genes involved in growth and development. However, it remains unclear exactly how a shortage of this enzyme during development leads to overgrowth, learning disabilities, and the other signs and symptoms of Sotos syndrome. More About This Health Condition  A change involving the NSD1 gene is associated with a blood cancer called childhood acute myeloid leukemia. This change occurs when part of chromosome 5 breaks off and reattaches to part of chromosome 11. This change is acquired during a person's lifetime and is present only in cancer cells. This type of genetic change, called a somatic mutation, is not inherited. The rearrangement of genetic material involved in childhood acute myeloid leukemia, known as a translocation, abnormally fuses the NSD1 gene on chromosome 5 with the NUP98 gene on chromosome 11. Research shows that the fused NUP98-NSD1 gene turns on genes that promote the growth of immature blood cells and blocks processes that would turn the genes off. The resulting overgrowth of these immature cells leads to development of acute myeloid leukemia.A different type of alteration involving the NSD1 gene is associated with a cancer of nerve tissue called neuroblastoma and a type of brain cancer called glioma. This alteration, known as promoter hypermethylation, turns off the production of the NSD1 enzyme. Researchers speculate that without NSD1, the activity of one or more genes involved in cell growth and division is uncontrolled. As a result, the cells can grow and divide unchecked, leading to the development of cancer.    Cytogenetic Location: 5q35.3, which is the long (q) arm of chromosome 5 at position 35.3Molecular Location: base pairs 177,131,835 to 177,300,213 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        androgen receptor-associated coregulator 267ARA267histone-lysine N-methyltransferase, H3 lysine-36 and H4 lysine-20 specificNR-binding SET domain containing proteinNSD1_HUMANnuclear receptor-binding Su-var, Enhancer of zeste and Trithorax domain protein 1SOTOS1     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Sotos Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of NSD1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     LEUKEMIA, ACUTE MYELOIDNUCLEAR RECEPTOR-BINDING SET DOMAIN PROTEIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Sotos Syndrome  Tests of NSD1  PubMed  LEUKEMIA, ACUTE MYELOIDNUCLEAR RECEPTOR-BINDING SET DOMAIN PROTEIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Berdasco M, Ropero S, Setien F, Fraga MF, Lapunzina P, Losson R, Alaminos M, Cheung NK, Rahman N, Esteller M. Epigenetic inactivation of the Sotos overgrowth syndrome gene histone methyltransferase NSD1 in human neuroblastoma and glioma. Proc Natl Acad Sci U S A. 2009 Dec 22;106(51):21830-5. doi: 10.1073/pnas.0906831106. Epub 2009 Dec 14. Citation on PubMed or Free article on PubMed CentralFaravelli F. NSD1 mutations in Sotos syndrome. Am J Med Genet C Semin Med Genet. 2005 Aug 15;137C(1):24-31. Review. Citation on PubMedJaju RJ, Fidler C, Haas OA, Strickson AJ, Watkins F, Clark K, Cross NC, Cheng JF, Aplan PD, Kearney L, Boultwood J, Wainscoat JS. A novel gene, NSD1, is fused to NUP98 in the t(5;11)(q35;p15.5) in de novo childhood acute myeloid leukemia. Blood. 2001 Aug 15;98(4):1264-7. Citation on PubMedKudithipudi S, Lungu C, Rathert P, Happel N, Jeltsch A. Substrate specificity analysis and novel substrates of the protein lysine methyltransferase NSD1. Chem Biol. 2014 Feb 20;21(2):226-37. doi: 10.1016/j.chembiol.2013.10.016. Epub 2014 Jan 9. Citation on PubMedKurotaki N, Imaizumi K, Harada N, Masuno M, Kondoh T, Nagai T, Ohashi H, Naritomi K, Tsukahara M, Makita Y, Sugimoto T, Sonoda T, Hasegawa T, Chinen Y, Tomita Ha HA, Kinoshita A, Mizuguchi T, Yoshiura Ki K, Ohta T, Kishino T, Fukushima Y, Niikawa N, Matsumoto N. Haploinsufficiency of NSD1 causes Sotos syndrome. Nat Genet. 2002 Apr;30(4):365-6. Epub 2002 Mar 18. Citation on PubMedLucio-Eterovic AK, Singh MM, Gardner JE, Veerappan CS, Rice JC, Carpenter PB. Role for the nuclear receptor-binding SET domain protein 1 (NSD1) methyltransferase in coordinating lysine 36 methylation at histone 3 with RNA polymerase II function. Proc Natl Acad Sci U S A. 2010 Sep 28;107(39):16952-7. doi: 10.1073/pnas.1002653107. Epub 2010 Sep 13. Citation on PubMed or Free article on PubMed CentralPasillas MP, Shah M, Kamps MP. NSD1 PHD domains bind methylated H3K4 and H3K9 using interactions disrupted by point mutations in human sotos syndrome. Hum Mutat. 2011 Mar;32(3):292-8. doi: 10.1002/humu.21424. Citation on PubMedQiao Q, Li Y, Chen Z, Wang M, Reinberg D, Xu RM. The structure of NSD1 reveals an autoregulatory mechanism underlying histone H3K36 methylation. J Biol Chem. 2011 Mar 11;286(10):8361-8. doi: 10.1074/jbc.M110.204115. Epub 2010 Dec 31. Citation on PubMed or Free article on PubMed CentralTatton-Brown K, Cole TRP, Rahman N. Sotos Syndrome. 2004 Dec 17 [updated 2015 Nov 19]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1479/ Citation on PubMedTatton-Brown K, Douglas J, Coleman K, Baujat G, Cole TR, Das S, Horn D, Hughes HE, Temple IK, Faravelli F, Waggoner D, Turkmen S, Cormier-Daire V, Irrthum A, Rahman N; Childhood Overgrowth Collaboration. Genotype-phenotype associations in Sotos syndrome: an analysis of 266 individuals with NSD1 aberrations. Am J Hum Genet. 2005 Aug;77(2):193-204. Epub 2005 Jun 7. Citation on PubMed or Free article on PubMed CentralTatton-Brown K, Rahman N. The NSD1 and EZH2 overgrowth genes, similarities and differences. Am J Med Genet C Semin Med Genet. 2013 May;163C(2):86-91. doi: 10.1002/ajmg.c.31359. Epub 2013 Apr 16. Citation on PubMed or Free article on PubMed CentralWang GG, Cai L, Pasillas MP, Kamps MP. NUP98-NSD1 links H3K36 methylation to Hox-A gene activation and leukaemogenesis. Nat Cell Biol. 2007 Jul;9(7):804-12. Epub 2007 Jun 24. Citation on PubMed \n",
            "DOK3\t\n",
            "C5orf60\t\n",
            "OR2Y1\t\n",
            "ZFP62\t\n",
            "BTNL8\t\n",
            "TRIM7\t\n",
            "IRX2\t\n",
            "C5orf38\t\n",
            "C5orf22\t\n",
            "ADAMTS12\t\n",
            "SLC45A2\t   The SLC45A2 gene (also called MATP) provides instructions for making a protein that is located in specialized cells called melanocytes.  These cells produce a pigment called melanin, which is the substance that gives skin, hair, and eyes their color. Melanin is also found in the light-sensitive tissue at the back of the eye (the retina), where it plays a role in normal vision.Although the exact function of the SLC45A2 protein is unknown, it is likely involved in the production of melanin.  This protein probably transports molecules necessary for the normal function of melanosomes, which are the structures in melanocytes where melanin is produced. Studies suggest that certain common variations (polymorphisms) in the SLC45A2 gene may be associated with normal differences in skin, hair, and eye coloring.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Oculocutaneous albinism     More than 20 mutations in the SLC45A2 gene are responsible for oculocutaneous albinism type 4.  The most common SLC45A2 mutation in the Japanese population switches a single protein building block (amino acid) in the SLC45A2 protein.  Specifically, this mutation replaces the amino acid aspartic acid with the amino acid asparagine at protein position 157 (written as Asp157Asn or D157N).  Other mutations, including changes in single amino acids and deletions or insertions of genetic material in the SLC45A2 gene, have also been reported in several populations worldwide.  Mutations in this gene reduce or eliminate the function of the SLC45A2 protein in melanin production.  Because this protein is important for normal pigmentation, its loss leads to changes in skin, hair, and eye coloration and problems with vision that are characteristic of oculocutaneous albinism type 4. More About This Health Condition      Melanoma     Genetics Home Reference provides information about melanoma. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 20 mutations in the SLC45A2 gene are responsible for oculocutaneous albinism type 4.  The most common SLC45A2 mutation in the Japanese population switches a single protein building block (amino acid) in the SLC45A2 protein.  Specifically, this mutation replaces the amino acid aspartic acid with the amino acid asparagine at protein position 157 (written as Asp157Asn or D157N).  Other mutations, including changes in single amino acids and deletions or insertions of genetic material in the SLC45A2 gene, have also been reported in several populations worldwide.  Mutations in this gene reduce or eliminate the function of the SLC45A2 protein in melanin production.  Because this protein is important for normal pigmentation, its loss leads to changes in skin, hair, and eye coloration and problems with vision that are characteristic of oculocutaneous albinism type 4. More About This Health Condition  Genetics Home Reference provides information about melanoma. More About This Health Condition    Cytogenetic Location: 5p13.2, which is the short (p) arm of chromosome 5 at position 13.2Molecular Location: base pairs 33,944,616 to 33,984,675 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AIM-1AIM1MATPmelanoma antigen AIM1membrane associated transportermembrane-associated transporter proteinS45A2_HUMANsolute carrier family 45, member 2     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Oculocutaneous Albinism Type 4      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SLC45A2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SOLUTE CARRIER FAMILY 45, MEMBER 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Oculocutaneous Albinism Type 4  Tests of SLC45A2  PubMed  SOLUTE CARRIER FAMILY 45, MEMBER 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Graf J, Hodgson R, van Daal A. Single nucleotide polymorphisms in the MATP gene are associated with normal human pigmentation variation. Hum Mutat. 2005 Mar;25(3):278-84. Citation on PubMedInagaki K, Suzuki T, Ito S, Suzuki N, Adachi K, Okuyama T, Nakata Y, Shimizu H, Matsuura H, Oono T, Iwamatsu H, Kono M, Tomita Y. Oculocutaneous albinism type 4: six novel mutations in the membrane-associated transporter protein gene and their phenotypes. Pigment Cell Res. 2006 Oct;19(5):451-3. Citation on PubMedInagaki K, Suzuki T, Ito S, Suzuki N, Fukai K, Horiuchi T, Tanaka T, Manabe E, Tomita Y. OCA4: evidence for a founder effect for the p.D157N mutation of the MATP gene in Japanese and Korean. Pigment Cell Res. 2005 Oct;18(5):385-8. Citation on PubMedInagaki K, Suzuki T, Shimizu H, Ishii N, Umezawa Y, Tada J, Kikuchi N, Takata M, Takamori K, Kishibe M, Tanaka M, Miyamura Y, Ito S, Tomita Y. Oculocutaneous albinism type 4 is one of the most common types of albinism in Japan. Am J Hum Genet. 2004 Mar;74(3):466-71. Epub 2004 Feb 11. Citation on PubMed or Free article on PubMed CentralNewton JM, Cohen-Barak O, Hagiwara N, Gardner JM, Davisson MT, King RA, Brilliant MH. Mutations in the human orthologue of the mouse underwhite gene (uw) underlie a new form of oculocutaneous albinism, OCA4. Am J Hum Genet. 2001 Nov;69(5):981-8. Epub 2001 Sep 26. Citation on PubMed or Free article on PubMed CentralRundshagen U, Zhlke C, Opitz S, Schwinger E, Ksmann-Kellner B. Mutations in the MATP gene in five German patients affected by oculocutaneous albinism type 4. Hum Mutat. 2004 Feb;23(2):106-10. Citation on PubMedYuasa I, Umetsu K, Harihara S, Kido A, Miyoshi A, Saitou N, Dashnyam B, Jin F, Lucotte G, Chattopadhyay PK, Henke L, Henke J. Distribution of the F374 allele of the SLC45A2 (MATP) gene and founder-haplotype analysis. Ann Hum Genet. 2006 Nov;70(Pt 6):802-11. Citation on PubMed \n",
            "AMACR\t   The AMACR gene provides instructions for making an enzyme called alpha-methylacyl-CoA racemase (AMACR). This enzyme is found in the energy-producing centers in cells (mitochondria) and in cell structures called peroxisomes. Peroxisomes contain a variety of enzymes that break down many different substances, including fatty acids and certain toxic compounds. They are also important for the production (synthesis) of fats (lipids) used in digestion and in the nervous system.In peroxisomes, the AMACR enzyme plays a role in the breakdown of a fatty acid called pristanic acid, which comes from meat and dairy foods in the diet. In mitochondria, AMACR is thought to help further break down the molecules derived from pristanic acid.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alpha-methylacyl-CoA racemase deficiency     Alpha-methylacyl-CoA racemase (AMACR) deficiency is caused by mutations in the AMACR gene. This disorder leads to a variety of neurological problems that begin in adulthood, including gradual loss in intellectual functioning (cognitive decline), seizures, and weakness and loss of sensation in the limbs due to nerve damage (sensorimotor neuropathy). Most individuals with AMACR deficiency have an AMACR gene mutation that replaces a protein building block (amino acid) called serine with an amino acid called proline at position 52 in the enzyme sequence, written as Ser52Pro or S52P. This mutation results in a lack (deficiency) of functional enzyme. The enzyme deficiency leads to accumulation of pristanic acid in the blood. However, it is unclear how this accumulation is related to the specific signs and symptoms of AMACR deficiency. More About This Health Condition      Other disorders     AMACR gene mutations that result in a lack of functional AMACR enzyme have also been identified in infants with a life-threatening disorder called congenital bile acid synthesis defect type 4. Babies with this disorder have cholestasis, which is a reduced ability to produce and release a digestive fluid called bile. Cholestasis leads to an enlarged liver (hepatomegaly) and irreversible liver disease (cirrhosis) in the first few months of life.Some researchers consider congenital bile acid synthesis defect type 4 and AMACR deficiency (see above) to be variations of the same disorder. Because most individuals with congenital bile acid synthesis defect type 4 do not survive infancy, it is unclear whether they would have later developed the neurological symptoms seen in adults with AMACR deficiency.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Alpha-methylacyl-CoA racemase (AMACR) deficiency is caused by mutations in the AMACR gene. This disorder leads to a variety of neurological problems that begin in adulthood, including gradual loss in intellectual functioning (cognitive decline), seizures, and weakness and loss of sensation in the limbs due to nerve damage (sensorimotor neuropathy). Most individuals with AMACR deficiency have an AMACR gene mutation that replaces a protein building block (amino acid) called serine with an amino acid called proline at position 52 in the enzyme sequence, written as Ser52Pro or S52P. This mutation results in a lack (deficiency) of functional enzyme. The enzyme deficiency leads to accumulation of pristanic acid in the blood. However, it is unclear how this accumulation is related to the specific signs and symptoms of AMACR deficiency. More About This Health Condition  AMACR gene mutations that result in a lack of functional AMACR enzyme have also been identified in infants with a life-threatening disorder called congenital bile acid synthesis defect type 4. Babies with this disorder have cholestasis, which is a reduced ability to produce and release a digestive fluid called bile. Cholestasis leads to an enlarged liver (hepatomegaly) and irreversible liver disease (cirrhosis) in the first few months of life.Some researchers consider congenital bile acid synthesis defect type 4 and AMACR deficiency (see above) to be variations of the same disorder. Because most individuals with congenital bile acid synthesis defect type 4 do not survive infancy, it is unclear whether they would have later developed the neurological symptoms seen in adults with AMACR deficiency.    Cytogenetic Location: 5p13.2, which is the short (p) arm of chromosome 5 at position 13.2Molecular Location: base pairs 33,986,986 to 34,008,115 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        2-methylacyl-CoA racemaseAMACR_HUMANAMACRDCBAS4RACERM     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Moecular Biology of the Cell (fourth edition, 2002): Peroxisomes      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AMACR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     ALPHA-METHYLACYL-CoA RACEMASEBILE ACID SYNTHESIS DEFECT, CONGENITAL, 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Moecular Biology of the Cell (fourth edition, 2002): Peroxisomes  Tests of AMACR  PubMed  ALPHA-METHYLACYL-CoA RACEMASEBILE ACID SYNTHESIS DEFECT, CONGENITAL, 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ALPHA-METHYLACYL-CoA RACEMASEDick D, Horvath R, Chinnery PF. AMACR mutations cause late-onset autosomal recessive cerebellar ataxia. Neurology. 2011 May 17;76(20):1768-70. doi: 10.1212/WNL.0b013e31821a4484. Citation on PubMed or Free article on PubMed CentralFerdinandusse S, Denis S, Clayton PT, Graham A, Rees JE, Allen JT, McLean BN, Brown AY, Vreken P, Waterham HR, Wanders RJ. Mutations in the gene encoding peroxisomal alpha-methylacyl-CoA racemase cause adult-onset sensory motor neuropathy. Nat Genet. 2000 Feb;24(2):188-91. Citation on PubMedFerdinandusse S, Denis S, IJlst L, Dacremont G, Waterham HR, Wanders RJ. Subcellular localization and physiological role of alpha-methylacyl-CoA racemase. J Lipid Res. 2000 Nov;41(11):1890-6. Citation on PubMedHaugarvoll K, Johansson S, Tzoulis C, Haukanes BI, Bredrup C, Neckelmann G, Boman H, Knappskog PM, Bindoff LA. MRI characterisation of adult onset alpha-methylacyl-coA racemase deficiency diagnosed by exome sequencing. Orphanet J Rare Dis. 2013 Jan 3;8:1. doi: 10.1186/1750-1172-8-1. Citation on PubMed or Free article on PubMed CentralWierzbicki AS. Peroxisomal disorders affecting phytanic acid alpha-oxidation: a review. Biochem Soc Trans. 2007 Nov;35(Pt 5):881-6. Review. Citation on PubMed \n",
            "AGXT2\t   From NCBI Gene:The protein encoded by this gene is a class III pyridoxal-phosphate-dependent mitochondrial aminotransferase. It catalyzes the conversion of glyoxylate to glycine using L-alanine as the amino donor. It is an important regulator of methylarginines and is involved in the control of blood pressure in kidney. Polymorphisms in this gene affect methylarginine and beta-aminoisobutyrate metabolism, and are associated with carotid atherosclerosis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]From UniProt:Can metabolize asymmetric dimethylarginine (ADMA) via transamination to alpha-keto-delta-(NN-dimethylguanidino) valeric acid (DMGV). ADMA is a potent inhibitor of nitric-oxide (NO) synthase, and this activity provides mechanism through which the kidney regulates blood pressure.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Beta-aminoisobutyric aciduria     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5p13.2, which is the short (p) arm of chromosome 5 at position 13.2Molecular Location: base pairs 34,998,099 to 35,048,188 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AGT2BAIBADAIBAT     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGXT2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ALANINE-GLYOXYLATE AMINOTRANSFERASE 2OMIM: BETA-AMINOISOBUTYRIC ACIDURIA      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AGXT2  OMIM: ALANINE-GLYOXYLATE AMINOTRANSFERASE 2OMIM: BETA-AMINOISOBUTYRIC ACIDURIA  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "UGT3A1\t\n",
            "CARD6\t\n",
            "C7\t   From NCBI Gene:This gene encodes a serum glycoprotein that forms a membrane attack complex together with complement components C5b, C6, C8, and C9 as part of the terminal complement pathway of the innate immune system. The protein encoded by this gene contains a cholesterol-dependent cytolysin/membrane attack complex/perforin-like (CDC/MACPF) domain and belongs to a large family of structurally related molecules that form pores involved in host immunity and bacterial pathogenesis. This protein initiates membrane attack complex formation by binding the C5b-C6 subcomplex and inserts into the phospholipid bilayer, serving as a membrane anchor. Mutations in this gene are associated with a rare disorder called C7 deficiency. [provided by RefSeq, Nov 2016]From UniProt:Constituent of the membrane attack complex (MAC) that plays a key role in the innate and adaptive immune response by forming pores in the plasma membrane of target cells. C7 serves as a membrane anchor.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Complement component 7 deficiencyFrom UniProt:Complement component 7 deficiency (C7D): A rare defect of the complement classical pathway associated with susceptibility to severe recurrent infections, predominantly by Neisseria gonorrhoeae or Neisseria meningitidis. [MIM:610102]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5p13.1, which is the short (p) arm of chromosome 5 at position 13.1Molecular Location: base pairs 40,909,252 to 40,984,640 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of C7      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: COMPLEMENT COMPONENT 7OMIM: COMPLEMENT COMPONENT 7 DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of C7  OMIM: COMPLEMENT COMPONENT 7OMIM: COMPLEMENT COMPONENT 7 DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GHR\t   The GHR gene provides instructions for making a protein called the growth hormone receptor. This receptor is embedded in the outer membrane of cells throughout the body and is most abundant in liver cells.The growth hormone receptor has three major parts: An extracellular region that sticks out from the surface of the cell, a transmembrane region that anchors the receptor to the cell membrane, and an intracellular region that transmits signals to the interior of the cell. The extracellular region attaches (binds) to a substance called growth hormone, fitting together like a lock and its key. The binding of growth hormone triggers signaling via the intracellular region of the receptor that stimulates the growth and division of cells. This signaling also leads to the production, primarily by liver cells, of another important growth-promoting hormone called insulin-like growth factor I (IGF-I).Growth hormone and IGF-I have a wide variety of effects on the growth and function of many parts of the body. For example, these hormones stimulate the growth and division of cells called chondrocytes, which play a critical role in producing new bone tissue. Growth hormone and IGF-I also influence metabolism, including how the body uses and stores carbohydrates, proteins, and fats from food.Researchers have identified two major versions (isoforms) of the growth hormone receptor. The two isoforms differ by the presence or absence of a particular segment known as exon 3, which is located in the extracellular region of the receptor. The version of the receptor that includes exon 3 is known as the full-length isoform (fl-GHR), while the version that is missing exon 3 is known as the exon 3-deficient isoform (d3-GHR). Both isoforms are relatively common in most populations. Each individual can have fl-GHR only, d3-GHR only, or a  mix of both isoforms. The two isoforms bind to growth hormone in the same way on the surface of cells, but for reasons that are unclear, d3-GHR is associated with enhanced signaling within cells compared with fl-GHR.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Laron syndrome     At least 70 mutations in the GHR gene have been found to cause Laron syndrome, a rare form of short stature that is also characterized by obesity, a distinctive facial appearance, and other signs and symptoms affecting multiple body systems. All of the identified mutations impair the function of the growth hormone receptor. Most of the mutations affect the extracellular region of the receptor, preventing it from binding to growth hormone effectively. A few mutations affect the intracellular region of the receptor, reducing or eliminating its ability to trigger the signals that promote growth.Although people with GHR gene mutations produce growth hormone, the defective receptors prevent cells from responding to the hormone by producing IGF-I or triggering cell growth and division. The cells' inability to react to growth hormone, which is described as growth hormone insensitivity, disrupts the normal growth of many different tissues. Short stature results when growth hormone cannot adequately stimulate the growth of bones. Changes in metabolism caused by insensitivity to growth hormone and the resulting shortage of IGF-I cause many of the other features of the condition, including obesity.Studies suggest that people with Laron syndrome have a significantly reduced risk of cancer and type 2 diabetes. Affected individuals appear to develop these common diseases much less frequently than their unaffected relatives, despite having obesity (a risk factor for both cancer and type 2 diabetes). Researchers are working to determine how mutations in the GHR gene may protect people with Laron syndrome from developing these diseases. Studies suggest that insensitivity to growth hormone may help prevent the uncontrolled growth and division of cells that can lead to the development of cancerous tumors. Growth hormone insensitivity also appears to alter how the body responds to insulin, which is a hormone that regulates blood sugar levels. Resistance to the effects of insulin is a major risk factor for type 2 diabetes. People with Laron syndrome have the opposite situation, an increased sensitivity to insulin, which likely helps explain their reduced risk of this disease. More About This Health Condition      Other disorders     The two isoforms of the growth hormone receptor have been studied in several conditions involving problems with growth. These conditions include short stature resulting from a shortage (deficiency) of growth hormone, slow growth starting before birth (small for gestational age), and short stature of unknown cause (idiopathic short stature). Some studies have found that children with these disorders respond differently to treatment depending on which isoform of the growth hormone receptor they have. Specifically, d3-GHR has been associated with faster growth during treatment compared with fl-GHR. Other studies have not found a difference in growth, although all of the studies used different methodologies and included relatively few participants.The two isoforms of the growth hormone receptor have also been studied in adults with acromegaly, a disorder characterized by increased levels of growth hormone. The abnormally high levels are most often caused by a noncancerous tumor in the pituitary gland, which is the part of the brain where growth hormone is produced. In response to the excess growth hormone, the bones of the hands, feet, and face grow unusually large. Other signs and symptoms of this condition can include thickened skin, increased sweating and body odor, enlargement of certain organs, muscle weakness, and excessive tiredness (fatigue). In adults with acromegaly, studies suggest that having the d3-GHR isoform is associated with better response to treatment compared with having fl-GHR. However, having d3-GHR appears to increase the risk of certain complications of the disease compared with having fl-GHR. The reasons for these differences are not fully understood.Whether a person has fl-GHR or d3-GHR does not seem to affect adult height in people without a growth disorder. Many genetic and non-genetic factors, some of which are unknown, influence how tall a person will be.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 70 mutations in the GHR gene have been found to cause Laron syndrome, a rare form of short stature that is also characterized by obesity, a distinctive facial appearance, and other signs and symptoms affecting multiple body systems. All of the identified mutations impair the function of the growth hormone receptor. Most of the mutations affect the extracellular region of the receptor, preventing it from binding to growth hormone effectively. A few mutations affect the intracellular region of the receptor, reducing or eliminating its ability to trigger the signals that promote growth.Although people with GHR gene mutations produce growth hormone, the defective receptors prevent cells from responding to the hormone by producing IGF-I or triggering cell growth and division. The cells' inability to react to growth hormone, which is described as growth hormone insensitivity, disrupts the normal growth of many different tissues. Short stature results when growth hormone cannot adequately stimulate the growth of bones. Changes in metabolism caused by insensitivity to growth hormone and the resulting shortage of IGF-I cause many of the other features of the condition, including obesity.Studies suggest that people with Laron syndrome have a significantly reduced risk of cancer and type 2 diabetes. Affected individuals appear to develop these common diseases much less frequently than their unaffected relatives, despite having obesity (a risk factor for both cancer and type 2 diabetes). Researchers are working to determine how mutations in the GHR gene may protect people with Laron syndrome from developing these diseases. Studies suggest that insensitivity to growth hormone may help prevent the uncontrolled growth and division of cells that can lead to the development of cancerous tumors. Growth hormone insensitivity also appears to alter how the body responds to insulin, which is a hormone that regulates blood sugar levels. Resistance to the effects of insulin is a major risk factor for type 2 diabetes. People with Laron syndrome have the opposite situation, an increased sensitivity to insulin, which likely helps explain their reduced risk of this disease. More About This Health Condition  The two isoforms of the growth hormone receptor have been studied in several conditions involving problems with growth. These conditions include short stature resulting from a shortage (deficiency) of growth hormone, slow growth starting before birth (small for gestational age), and short stature of unknown cause (idiopathic short stature). Some studies have found that children with these disorders respond differently to treatment depending on which isoform of the growth hormone receptor they have. Specifically, d3-GHR has been associated with faster growth during treatment compared with fl-GHR. Other studies have not found a difference in growth, although all of the studies used different methodologies and included relatively few participants.The two isoforms of the growth hormone receptor have also been studied in adults with acromegaly, a disorder characterized by increased levels of growth hormone. The abnormally high levels are most often caused by a noncancerous tumor in the pituitary gland, which is the part of the brain where growth hormone is produced. In response to the excess growth hormone, the bones of the hands, feet, and face grow unusually large. Other signs and symptoms of this condition can include thickened skin, increased sweating and body odor, enlargement of certain organs, muscle weakness, and excessive tiredness (fatigue). In adults with acromegaly, studies suggest that having the d3-GHR isoform is associated with better response to treatment compared with having fl-GHR. However, having d3-GHR appears to increase the risk of certain complications of the disease compared with having fl-GHR. The reasons for these differences are not fully understood.Whether a person has fl-GHR or d3-GHR does not seem to affect adult height in people without a growth disorder. Many genetic and non-genetic factors, some of which are unknown, influence how tall a person will be.    Cytogenetic Location: 5p13.1-p12, which is the short (p) arm of chromosome 5 between positions 13.1 and 12Molecular Location: base pairs 42,423,775 to 42,721,878 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GH receptorGHBPGHR_HUMANgrowth hormone binding proteinserum binding proteinsomatotropin receptor     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Endocrinology: An Integrated Approach (2001): Actions of Growth Hormone and Insulin-Like Growth Factors      Tests Listed in the Genetic Testing Registry (1 link)     Tests of GHR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     GROWTH HORMONE RECEPTOR      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Endocrinology: An Integrated Approach (2001): Actions of Growth Hormone and Insulin-Like Growth Factors  Tests of GHR  PubMed  GROWTH HORMONE RECEPTOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Amselem S, Duquesnoy P, Attree O, Novelli G, Bousnina S, Postel-Vinay MC, Goossens M. Laron dwarfism and mutations of the growth hormone-receptor gene. N Engl J Med. 1989 Oct 12;321(15):989-95. Citation on PubMedBrooks AJ, Waters MJ. The growth hormone receptor: mechanism of activation and clinical implications. Nat Rev Endocrinol. 2010 Sep;6(9):515-25. doi: 10.1038/nrendo.2010.123. Epub 2010 Jul 27. Review. Citation on PubMedDos Santos C, Essioux L, Teinturier C, Tauber M, Goffin V, Bougnres P. A common polymorphism of the growth hormone receptor is associated with increased responsiveness to growth hormone. Nat Genet. 2004 Jul;36(7):720-4. Epub 2004 Jun 20. Citation on PubMedFilopanti M, Giavoli C, Grottoli S, Bianchi A, De Marinis L, Ghigo E, Spada A. The exon 3-deleted growth hormone receptor: molecular and functional characterization and impact on GH/IGF-I axis in physiological and pathological conditions. J Endocrinol Invest. 2011 Dec;34(11):861-8. Review. Citation on PubMedOMIM: GROWTH HORMONE RECEPTORGuevara-Aguirre J, Balasubramanian P, Guevara-Aguirre M, Wei M, Madia F, Cheng CW, Hwang D, Martin-Montalvo A, Saavedra J, Ingles S, de Cabo R, Cohen P, Longo VD. Growth hormone receptor deficiency is associated with a major reduction in pro-aging signaling, cancer, and diabetes in humans. Sci Transl Med. 2011 Feb 16;3(70):70ra13. doi: 10.1126/scitranslmed.3001845. Citation on PubMed or Free article on PubMed CentralGuevara-Aguirre J, Rosenbloom AL. Obesity, diabetes and cancer: insight into the relationship from a cohort with growth hormone receptor deficiency. Diabetologia. 2015 Jan;58(1):37-42. doi: 10.1007/s00125-014-3397-3. Epub 2014 Oct 15. Review. Citation on PubMedJorge AA, Arnhold IJ. Growth hormone receptor exon 3 isoforms and their implication in growth disorders and treatment. Horm Res. 2009 Apr;71 Suppl 2:55-63. doi: 10.1159/000192438. Epub 2009 Apr 29. Review. Citation on PubMedSavage MO, Woods KA, Johnston LB, Postel-Vinay MC, Amselem S, Clark AJ. Defects of the growth hormone receptor and their clinical implications. Growth Horm IGF Res. 1999 Apr;9 Suppl A:57-61. Review. Citation on PubMedWassenaar MJ, Biermasz NR, Pereira AM, van der Klaauw AA, Smit JW, Roelfsema F, van der Straaten T, Cazemier M, Hommes DW, Kroon HM, Kloppenburg M, Guchelaar HJ, Romijn JA. The exon-3 deleted growth hormone receptor polymorphism predisposes to long-term complications of acromegaly. J Clin Endocrinol Metab. 2009 Dec;94(12):4671-8. doi: 10.1210/jc.2009-1172. Epub 2009 Oct 28. Citation on PubMedWassenaar MJ, Dekkers OM, Pereira AM, Wit JM, Smit JW, Biermasz NR, Romijn JA. Impact of the exon 3-deleted growth hormone (GH) receptor polymorphism on baseline height and the growth response to recombinant human GH therapy in GH-deficient (GHD) and non-GHD children with short stature: a systematic review and meta-analysis. J Clin Endocrinol Metab. 2009 Oct;94(10):3721-30. doi: 10.1210/jc.2009-0425. Epub 2009 Jul 7. Review. Citation on PubMed \n",
            "ANXA2R\t\n",
            "PARP8\t\n",
            "MAP3K1\t   The MAP3K1 gene provides instructions for making a protein that helps regulate signaling pathways that control various processes in the body, including the processes of determining sexual characteristics before birth. The MAP3K1 protein attaches (binds) to other molecules called RHOA, MAP3K4, FRAT1, and AXIN1. The binding of MAP3K1 to these molecules, which are called cofactors, helps MAP3K1 control the activity of the signaling pathways.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Swyer syndrome     Mutations in the MAP3K1 gene are thought to account for up to 18 percent of cases of Swyer syndrome, a condition affecting sexual development also known as 46,XY complete gonadal dysgenesis or 46,XY pure gonadal dysgenesis.People usually have 46 chromosomes in each cell. Two of the 46 chromosomes, known as X and Y, are called sex chromosomes because they help determine whether a person will develop male or female sex characteristics. Girls and women typically have two X chromosomes (46,XX karyotype), while boys and men usually have one X chromosome and one Y chromosome (46,XY karyotype).The MAP3K1 gene mutations that cause Swyer syndrome increase cofactor binding, which decreases signaling that leads to male sexual differentiation and enhances signaling that leads to female sexual differentiation. As a result, affected individuals with a typically male 46,XY karyotype will not develop male gonads (testes) but will develop female reproductive structures (a uterus and fallopian tubes). More About This Health Condition      Breast cancer     Genetics Home Reference provides information about breast cancer. More About This Health Condition      Langerhans cell histiocytosis     Genetics Home Reference provides information about Langerhans cell histiocytosis. More About This Health Condition      Other disorders     MAP3K1 gene mutations have also been identified in people with 46,XY disorder of sex development, which is also known as partial gonadal dysgenesis. These mutations likely have similar but less pronounced effects on signaling related to sexual development than those that cause Swyer syndrome (described above). Affected individuals may have external genitalia that do not look clearly male or clearly female (ambiguous genitalia) or other abnormalities of the genitals and reproductive organs.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the MAP3K1 gene are thought to account for up to 18 percent of cases of Swyer syndrome, a condition affecting sexual development also known as 46,XY complete gonadal dysgenesis or 46,XY pure gonadal dysgenesis.People usually have 46 chromosomes in each cell. Two of the 46 chromosomes, known as X and Y, are called sex chromosomes because they help determine whether a person will develop male or female sex characteristics. Girls and women typically have two X chromosomes (46,XX karyotype), while boys and men usually have one X chromosome and one Y chromosome (46,XY karyotype).The MAP3K1 gene mutations that cause Swyer syndrome increase cofactor binding, which decreases signaling that leads to male sexual differentiation and enhances signaling that leads to female sexual differentiation. As a result, affected individuals with a typically male 46,XY karyotype will not develop male gonads (testes) but will develop female reproductive structures (a uterus and fallopian tubes). More About This Health Condition  Genetics Home Reference provides information about breast cancer. More About This Health Condition  Genetics Home Reference provides information about Langerhans cell histiocytosis. More About This Health Condition  MAP3K1 gene mutations have also been identified in people with 46,XY disorder of sex development, which is also known as partial gonadal dysgenesis. These mutations likely have similar but less pronounced effects on signaling related to sexual development than those that cause Swyer syndrome (described above). Affected individuals may have external genitalia that do not look clearly male or clearly female (ambiguous genitalia) or other abnormalities of the genitals and reproductive organs.    Cytogenetic Location: 5q11.2, which is the long (q) arm of chromosome 5 at position 11.2Molecular Location: base pairs 56,815,073 to 56,896,152 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        M3K1_HUMANMAP/ERK kinase kinase 1MAPK/ERK kinase kinase 1MAPKKK1MEK kinase 1MEKKMEKK 1MEKK1mitogen-activated protein kinase kinase kinase 1, E3 ubiquitin protein ligaseSRXY6     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Endotext: Sex Determination      Clinical Information from GeneReviews (1 link)     Nonsyndromic Disorders of Testicular Development      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MAP3K1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     46,XY SEX REVERSAL 6MITOGEN-ACTIVATED KINASE KINASE KINASE 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Endotext: Sex Determination  Nonsyndromic Disorders of Testicular Development  Tests of MAP3K1  PubMed  46,XY SEX REVERSAL 6MITOGEN-ACTIVATED KINASE KINASE KINASE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Jara L, Gonzalez-Hormazabal P, Cerceo K, Di Capua GA, Reyes JM, Blanco R, Bravo T, Peralta O, Gomez F, Waugh E, Margarit S, Ibaez G, Romero C, Pakomio J, Roizen G. Genetic variants in FGFR2 and MAP3K1 are associated with the risk of familial and early-onset breast cancer in a South-American population. Breast Cancer Res Treat. 2013 Jan;137(2):559-69. doi: 10.1007/s10549-012-2359-z. Epub 2012 Dec 7. Citation on PubMedLoke J, Pearlman A, Radi O, Zuffardi O, Giussani U, Pallotta R, Camerino G, Ostrer H. Mutations in MAP3K1 tilt the balance from SOX9/FGF9 to WNT/-catenin signaling. Hum Mol Genet. 2014 Feb 15;23(4):1073-83. doi: 10.1093/hmg/ddt502. Epub 2013 Oct 16. Citation on PubMedOMIM: MITOGEN-ACTIVATED KINASE KINASE KINASE 1Nelson DS, van Halteren A, Quispel WT, van den Bos C, Bove JV, Patel B, Badalian-Very G, van Hummelen P, Ducar M, Lin L, MacConaill LE, Egeler RM, Rollins BJ. MAP2K1 and MAP3K1 mutations in Langerhans cell histiocytosis. Genes Chromosomes Cancer. 2015 Jun;54(6):361-8. doi: 10.1002/gcc.22247. Epub 2015 Mar 31. Citation on PubMedPearlman A, Loke J, Le Caignec C, White S, Chin L, Friedman A, Warr N, Willan J, Brauer D, Farmer C, Brooks E, Oddoux C, Riley B, Shajahan S, Camerino G, Homfray T, Crosby AH, Couper J, David A, Greenfield A, Sinclair A, Ostrer H. Mutations in MAP3K1 cause 46,XY disorders of sex development and implicate a common signal transduction pathway in human testis determination. Am J Hum Genet. 2010 Dec 10;87(6):898-904. doi: 10.1016/j.ajhg.2010.11.003. Citation on PubMed or Free article on PubMed CentralZheng Q, Ye J, Wu H, Yu Q, Cao J. Association between mitogen-activated protein kinase kinase kinase 1 polymorphisms and breast cancer susceptibility: a meta-analysis of 20 case-control studies. PLoS One. 2014 Mar 4;9(3):e90771. doi: 10.1371/journal.pone.0090771. eCollection 2014. Citation on PubMed or Free article on PubMed Central \n",
            "PDE4D\t   From NCBI Gene:This gene encodes one of four mammalian counterparts to the fruit fly 'dunce' gene. The encoded protein has 3',5'-cyclic-AMP phosphodiesterase activity and degrades cAMP, which acts as a signal transduction molecule in multiple cell types. This gene uses different promoters to generate multiple alternatively spliced transcript variants that encode functional proteins.[provided by RefSeq, Sep 2009]From UniProt:Hydrolyzes the second messenger cAMP, which is a key regulator of many important physiological processes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Acrodysostosis 2, with or without hormone resistanceFrom UniProt:Genetic variations in PDE4D might be associated with susceptibility to stroke. PubMed:17006457 states that association with stroke has to be considered with caution.Acrodysostosis 2, with or without hormone resistance (ACRDYS2): A pleiotropic disorder characterized by skeletal, endocrine, and neurological abnormalities. Skeletal features include brachycephaly, midface hypoplasia with a small upturned nose, brachydactyly, and lumbar spinal stenosis. Endocrine abnormalities include hypothyroidism and hypogonadism in males and irregular menses in females. Developmental disability is a common finding but is variable in severity and can be associated with significant behavioral problems. [MIM:614613]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q11.2-q12.1, which is the long (q) arm of chromosome 5 between positions 11.2 and 12.1Molecular Location: base pairs 58,969,038 to 60,524,329 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACRDYS2DPDE3HSPDE4DPDE4DN2PDE43STRK1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PDE4D      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ACRODYSOSTOSIS 2 WITH OR WITHOUT HORMONE RESISTANCEOMIM: PHOSPHODIESTERASE 4D, cAMP-SPECIFIC      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PDE4D  OMIM: ACRODYSOSTOSIS 2 WITH OR WITHOUT HORMONE RESISTANCEOMIM: PHOSPHODIESTERASE 4D, cAMP-SPECIFIC  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZSWIM6\t   From NCBI Gene:The protein encoded by this gene contains a zinc finger SWI2/SNF2 and MuDR (SWIM) domain. Proteins with SWIM domains have been found in a diverse number of species and are predicted to interact with DNA or proteins. Mutations in this gene result in acromelic frontonasal dysostosis. [provided by RefSeq, Apr 2017]From UniProt:involved in nervous system development, important for striatal morphology and motor regulation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:NEURODEVELOPMENTAL DISORDER WITH MOVEMENT ABNORMALITIES, ABNORMAL GAIT, AND AUTISTIC FEATURESAcromelic frontonasal dysostosisFrom UniProt:Acromelic frontonasal dysostosis (AFND): A rare variant form of frontonasal dysplasia, an array of abnormalities affecting the eyes, forehead and nose and linked to midfacial dysraphia. The clinical picture is highly variable. Major findings include true ocular hypertelorism, broadening of the nasal root, median facial cleft affecting the nose and/or upper lip and palate, unilateral or bilateral clefting of the alae nasi, lack of formation of the nasal tip, anterior cranium bifidum occultum, a V-shaped or widow's peak frontal hairline. AFND is characterized by the association of frontonasal malformations with various combinations of polydactyly, tibial hypoplasia, epibulbar dermoid, encephalocoele, corpus callosum agenesis and Dandy-Walker malformation. [MIM:603671]Neurodevelopmental disorder with movement abnormalities, abnormal gait, and autistic features (NEDMAGA): An autosomal dominant neurodevelopmental disorder characterized by infantile-onset global developmental delay, severe to profound intellectual disability, mildly delayed walking with broad-based and unsteady gait, and absence of meaningful language. Patients have features of autism, with repetitive behaviors and poor communication, but usually are socially reactive and have a happy demeanor. More variable neurologic features include mild seizures, spasticity, and peripheral neuropathy. [MIM:617865]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q12.1, which is the long (q) arm of chromosome 5 at position 12.1Molecular Location: base pairs 61,332,273 to 61,546,172 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AFNDNEDMAGA     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ZSWIM6      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ACROMELIC FRONTONASAL DYSOSTOSISOMIM: NEURODEVELOPMENTAL DISORDER WITH MOVEMENT ABNORMALITIES, ABNORMAL GAIT, AND AUTISTIC FEATURESOMIM: ZINC FINGER SWIM DOMAIN-CONTAINING PROTEIN 6      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ZSWIM6  OMIM: ACROMELIC FRONTONASAL DYSOSTOSISOMIM: NEURODEVELOPMENTAL DISORDER WITH MOVEMENT ABNORMALITIES, ABNORMAL GAIT, AND AUTISTIC FEATURESOMIM: ZINC FINGER SWIM DOMAIN-CONTAINING PROTEIN 6  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "UBE2QL1\t\n",
            "MAST4\t\n",
            "CDK7\t\n",
            "NAIP\t\n",
            "ANKDD1B\t\n",
            "POC5\t\n",
            "IQGAP2\t\n",
            "S100Z\t\n",
            "SCAMP1\t\n",
            "ARSB\t   The ARSB gene provides instructions for producing an enzyme called arylsulfatase B, which is involved in the breakdown of large sugar molecules called glycosaminoglycans (GAGs). Specifically, arylsulfatase B removes a chemical group known as a sulfate from two GAGs called dermatan sulfate and chondroitin sulfate. Arylsulfatase B is located in lysosomes, compartments within cells that digest and recycle different types of molecules.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Mucopolysaccharidosis type VI     More than 130 mutations in the ARSB gene have been found to cause mucopolysaccharidosis type VI (MPS VI). Most of these mutations change single DNA building blocks (nucleotides) in the gene. All of the mutations that cause MPS VI reduce or eliminate the function of arylsulfatase B. It usually cannot be determined whether a certain mutation will cause severe or mild MPS VI; however, mutations known to result in the complete absence of arylsulfatase B activity cause severe signs and symptoms.The lack of arylsulfatase B activity leads to the accumulation of GAGs within lysosomes. Conditions such as MPS VI that cause molecules to build up inside the lysosomes are called lysosomal storage disorders. The accumulation of GAGs within lysosomes increases the size of cells, which is why many tissues and organs are enlarged in this disorder. Researchers believe that the buildup of GAGs may also  interfere with the functions of other proteins inside lysosomes, triggering inflammation and cell death. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 130 mutations in the ARSB gene have been found to cause mucopolysaccharidosis type VI (MPS VI). Most of these mutations change single DNA building blocks (nucleotides) in the gene. All of the mutations that cause MPS VI reduce or eliminate the function of arylsulfatase B. It usually cannot be determined whether a certain mutation will cause severe or mild MPS VI; however, mutations known to result in the complete absence of arylsulfatase B activity cause severe signs and symptoms.The lack of arylsulfatase B activity leads to the accumulation of GAGs within lysosomes. Conditions such as MPS VI that cause molecules to build up inside the lysosomes are called lysosomal storage disorders. The accumulation of GAGs within lysosomes increases the size of cells, which is why many tissues and organs are enlarged in this disorder. Researchers believe that the buildup of GAGs may also  interfere with the functions of other proteins inside lysosomes, triggering inflammation and cell death. More About This Health Condition    Cytogenetic Location: 5q14.1, which is the long (q) arm of chromosome 5 at position 14.1Molecular Location: base pairs 78,777,209 to 78,986,087 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARSB_HUMANarylsulfatase B isoform 1 precursorASBchondroitinasechondroitinsulfataseG4SMPS6N-acetylgalactosamine-4-sulfataseN-acetylgalactosamine 4-sulfate sulfohydrolase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database: Defects in Glycosaminoglycan Degradation (Mucopolysaccharidoses)      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ARSB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ARYLSULFATASE B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Defects in Glycosaminoglycan Degradation (Mucopolysaccharidoses)  Tests of ARSB  PubMed  ARYLSULFATASE B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ARYLSULFATASE BClarke LA. The mucopolysaccharidoses: a success of molecular medicine. Expert Rev Mol Med. 2008 Jan 18;10:e1. doi: 10.1017/S1462399408000550. Review. Citation on PubMedGarrido E, Chabs A, Coll MJ, Blanco M, Domnguez C, Grinberg D, Vilageliu L, Cormand B. Identification of the molecular defects in Spanish and Argentinian mucopolysaccharidosis VI (Maroteaux-Lamy syndrome) patients, including 9 novel mutations. Mol Genet Metab. 2007 Sep-Oct;92(1-2):122-30. Epub 2007 Jul 20. Citation on PubMedGarrido E, Cormand B, Hopwood JJ, Chabs A, Grinberg D, Vilageliu L. Maroteaux-Lamy syndrome: functional characterization of pathogenic mutations and polymorphisms in the arylsulfatase B gene. Mol Genet Metab. 2008 Jul;94(3):305-12. doi: 10.1016/j.ymgme.2008.02.012. Epub 2008 Apr 10. Citation on PubMedKarageorgos L, Brooks DA, Pollard A, Melville EL, Hein LK, Clements PR, Ketteridge D, Swiedler SJ, Beck M, Giugliani R, Harmatz P, Wraith JE, Guffon N, Leo Teles E, S Miranda MC, Hopwood JJ. Mutational analysis of 105 mucopolysaccharidosis type VI patients. Hum Mutat. 2007 Sep;28(9):897-903. Citation on PubMedLitjens T, Hopwood JJ. Mucopolysaccharidosis type VI: Structural and clinical implications of mutations in N-acetylgalactosamine-4-sulfatase. Hum Mutat. 2001 Oct;18(4):282-95. Review. Citation on PubMedTessitore A, Pirozzi M, Auricchio A. Abnormal autophagy, ubiquitination, inflammation and apoptosis are dependent upon lysosomal storage and are useful biomarkers of mucopolysaccharidosis VI. Pathogenetics. 2009 Jun 16;2(1):4. doi: 10.1186/1755-8417-2-4. Citation on PubMed or Free article on PubMed Central \n",
            "MTRR\t   The MTRR gene provides instructions for making an enzyme called methionine synthase reductase. This enzyme is required for the proper function of another enzyme called methionine synthase. Methionine synthase helps process amino acids, which are the building blocks of proteins. Specifically, it converts the amino acid homocysteine to another amino acid called methionine. After a period of being turned on (active), methionine synthase turns off (becomes inactive). Methionine synthase reductase reactivates methionine synthase so the enzyme can continue to produce methionine.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Homocystinuria     At least 20 mutations in the MTRR gene have been identified in people with homocystinuria. Some of these mutations change single amino acids in methionine synthase reductase. Other mutations lead to an abnormally small, nonfunctional version of the enzyme. All these mutations prevent the enzyme from functioning normally. Without methionine synthase reductase, methionine synthase cannot convert homocysteine to methionine. As a result, homocysteine builds up in the bloodstream, and the amount of methionine is reduced. Some of the excess homocysteine is excreted in urine. Researchers have not determined how altered levels of homocysteine and methionine lead to the health problems associated with homocystinuria. More About This Health Condition      Other disorders     A specific version (variant) of the MTRR gene may be associated with an increased risk of various health problems before birth. The variant replaces a building block of DNA (nucleotide) called adenine with the nucleotide guanine at position 66 of the MTRR gene (written as A66G). This variant is associated with birth defects that occur during the development of the brain and spinal cord (neural tube defects). This variant may also increase the risk of having a child with Down syndrome, a condition characterized by intellectual disability and associated health problems. Researchers have not determined why there may be a connection between the A66G variant of the MTRR gene and the risk of neural tube defects or Down syndrome. Many factors play a part in determining the risk of these disorders.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 20 mutations in the MTRR gene have been identified in people with homocystinuria. Some of these mutations change single amino acids in methionine synthase reductase. Other mutations lead to an abnormally small, nonfunctional version of the enzyme. All these mutations prevent the enzyme from functioning normally. Without methionine synthase reductase, methionine synthase cannot convert homocysteine to methionine. As a result, homocysteine builds up in the bloodstream, and the amount of methionine is reduced. Some of the excess homocysteine is excreted in urine. Researchers have not determined how altered levels of homocysteine and methionine lead to the health problems associated with homocystinuria. More About This Health Condition  A specific version (variant) of the MTRR gene may be associated with an increased risk of various health problems before birth. The variant replaces a building block of DNA (nucleotide) called adenine with the nucleotide guanine at position 66 of the MTRR gene (written as A66G). This variant is associated with birth defects that occur during the development of the brain and spinal cord (neural tube defects). This variant may also increase the risk of having a child with Down syndrome, a condition characterized by intellectual disability and associated health problems. Researchers have not determined why there may be a connection between the A66G variant of the MTRR gene and the risk of neural tube defects or Down syndrome. Many factors play a part in determining the risk of these disorders.    Cytogenetic Location: 5p15.31, which is the short (p) arm of chromosome 5 at position 15.31Molecular Location: base pairs 7,851,186 to 7,901,124 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        cblEmethionine synthase reductaseMSRMTRR_HUMAN     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Disorders of Intracellular Cobalamin Metabolism      Tests Listed in the Genetic Testing Registry (1 link)     Tests of MTRR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (3 links)     DOWN SYNDROMEMETHIONINE SYNTHASE REDUCTASENEURAL TUBE DEFECTS, SUSCEPTIBILITY TO      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Disorders of Intracellular Cobalamin Metabolism  Tests of MTRR  PubMed  DOWN SYNDROMEMETHIONINE SYNTHASE REDUCTASENEURAL TUBE DEFECTS, SUSCEPTIBILITY TO  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Carmel R, Green R, Rosenblatt DS, Watkins D. Update on cobalamin, folate, and homocysteine. Hematology Am Soc Hematol Educ Program. 2003:62-81. Review. Citation on PubMedDoolin MT, Barbaux S, McDonnell M, Hoess K, Whitehead AS, Mitchell LE. Maternal genetic effects, exerted by genes involved in homocysteine remethylation, influence the risk of spina bifida. Am J Hum Genet. 2002 Nov;71(5):1222-6. Epub 2002 Oct 9. Citation on PubMed or Free article on PubMed CentralGuant-Rodriguez RM, Rendeli C, Namour B, Venuti L, Romano A, Anello G, Bosco P, Debard R, Grard P, Viola M, Salvaggio E, Guant JL. Transcobalamin and methionine synthase reductase mutated polymorphisms aggravate the risk of neural tube defects in humans. Neurosci Lett. 2003 Jul 3;344(3):189-92. Citation on PubMedHobbs CA, Sherman SL, Yi P, Hopkins SE, Torfs CP, Hine RJ, Pogribna M, Rozen R, James SJ. Polymorphisms in genes involved in folate metabolism as maternal risk factors for Down syndrome. Am J Hum Genet. 2000 Sep;67(3):623-30. Epub 2000 Aug 7. Citation on PubMed or Free article on PubMed CentralVilaseca MA, Vilarinho L, Zavadakova P, Vela E, Cleto E, Pineda M, Coimbra E, Suormala T, Fowler B, Kozich V. CblE type of homocystinuria: mild clinical phenotype in two patients homozygous for a novel mutation in the MTRR gene. J Inherit Metab Dis. 2003;26(4):361-9. Citation on PubMedWilson A, Leclerc D, Rosenblatt DS, Gravel RA. Molecular basis for methionine synthase reductase deficiency in patients belonging to the cblE complementation group of disorders in folate/cobalamin metabolism. Hum Mol Genet. 1999 Oct;8(11):2009-16. Citation on PubMedZavadakova P, Fowler B, Zeman J, Suormala T, Pristoupilov K, Kozich V, Zavad'kov P. CblE type of homocystinuria due to methionine synthase reductase deficiency: clinical and molecular studies and prenatal diagnosis in two families. J Inherit Metab Dis. 2002 Oct;25(6):461-76. Erratum in: J Inherit Metab Dis. 2003;26(1):95. Citation on PubMed \n",
            "CMYA5\t\n",
            "ZFYVE16\t\n",
            "MSH3\t   From NCBI Gene:The protein encoded by this gene forms a heterodimer with MSH2 to form MutS beta, part of the post-replicative DNA mismatch repair system. MutS beta initiates mismatch repair by binding to a mismatch and then forming a complex with MutL alpha heterodimer. This gene contains a polymorphic 9 bp tandem repeat sequence in the first exon. The repeat is present 6 times in the reference genome sequence and 3-7 repeats have been reported. Defects in this gene are a cause of susceptibility to endometrial cancer. [provided by RefSeq, Mar 2011]From UniProt:Component of the post-replicative DNA mismatch repair system (MMR). Heterodimerizes with MSH2 to form MutS beta which binds to DNA mismatches thereby initiating DNA repair. When bound, the MutS beta heterodimer bends the DNA helix and shields approximately 20 base pairs. MutS beta recognizes large insertion-deletion loops (IDL) up to 13 nucleotides long. After mismatch binding, forms a ternary complex with the MutL alpha heterodimer, which is thought to be responsible for directing the downstream MMR events, including strand discrimination, excision, and resynthesis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Familial adenomatous polyposis 4Endometrial carcinomaFrom UniProt:Familial adenomatous polyposis 4 (FAP4): A form of familial adenomatous polyposis, a condition characterized by the development of multiple colorectal adenomatous polyps, benign neoplasms derived from glandular epithelium. Some affected individuals may develop colorectal carcinoma. FAP4 inheritance is autosomal recessive. [MIM:617100]Endometrial cancer (ENDMC): A malignancy of endometrium, the mucous lining of the uterus. Most endometrial cancers are adenocarcinomas, cancers that begin in cells that make and release mucus and other fluids. [MIM:608089]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q14.1, which is the long (q) arm of chromosome 5 at position 14.1Molecular Location: base pairs 80,654,648 to 80,876,815 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DUPFAP4MRP1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MSH3      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: ENDOMETRIAL CANCEROMIM: FAMILIAL ADENOMATOUS POLYPOSIS 4OMIM: MutS, E. COLI, HOMOLOG OF, 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MSH3  OMIM: ENDOMETRIAL CANCEROMIM: FAMILIAL ADENOMATOUS POLYPOSIS 4OMIM: MutS, E. COLI, HOMOLOG OF, 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MCTP1\t\n",
            "ERAP1\t   The ERAP1 gene (also known as ERAAP and ARTS1) provides instructions for making a protein called endoplasmic reticulum aminopeptidase 1. As its name suggests, this protein is active in a cellular structure called the endoplasmic reticulum, which is involved in protein processing and transport. This protein is an aminopeptidase, which is an enzyme that cuts (cleaves) other proteins into smaller fragments called peptides.Endoplasmic reticulum aminopeptidase 1 has two major functions, both of which are important for normal immune system function. First, endoplasmic reticulum aminopeptidase 1 cleaves several proteins called cytokine receptors on the surface of cells.  Cleaving these receptors reduces their ability to transmit chemical signals into the cell, which affects the process of inflammation.Second, endoplasmic reticulum aminopeptidase 1 cleaves many types of proteins into small peptides that can be recognized by the immune system.  These peptides are exported to the cell surface, where they attach to major histocompatibility complex (MHC) class I proteins.  MHC class I proteins display the peptides to the immune system. If the immune system recognizes the peptides as foreign (such as viral or bacterial peptides), it responds by triggering the infected cell to self-destruct.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ankylosing spondylitis     Several variations (polymorphisms) in the ERAP1 gene have been found to influence the risk of ankylosing spondylitis. Each of these variations changes a single protein building block (amino acid) in endoplasmic reticulum aminopeptidase 1.  Little is known about the effects of these variations, although researchers believe that changes in the protein's structure could alter either of its two major functions.  It is unclear how these changes contribute to a person's risk of ankylosing spondylitis. Other genetic and environmental factors, many of which are unknown, also affect the chance of developing this condition. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several variations (polymorphisms) in the ERAP1 gene have been found to influence the risk of ankylosing spondylitis. Each of these variations changes a single protein building block (amino acid) in endoplasmic reticulum aminopeptidase 1.  Little is known about the effects of these variations, although researchers believe that changes in the protein's structure could alter either of its two major functions.  It is unclear how these changes contribute to a person's risk of ankylosing spondylitis. Other genetic and environmental factors, many of which are unknown, also affect the chance of developing this condition. More About This Health Condition    Cytogenetic Location: 5q15, which is the long (q) arm of chromosome 5 at position 15Molecular Location: base pairs 96,760,273 to 96,935,983 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        A-LAPadipocyte-derived leucine aminopeptidaseALAPaminopeptidase PILSaminopeptidase regulator of TNFR1 sheddingAPPILSARTS-1ARTS1ERAAPERAAP1ERAP1_HUMANKIAA0525PILS-APPILSAPpuromycin-insensitive leucyl-specific aminopeptidasetype 1 tumor necrosis factor receptor shedding aminopeptidase regulator     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Immunobiology (fifth edition, 2001): The Major Histocompatibility Complex and Its Functions      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ERAP1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ENDOPLASMIC RETICULUM AMINOPEPTIDASE 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology (fifth edition, 2001): The Major Histocompatibility Complex and Its Functions  Tests of ERAP1  PubMed  ENDOPLASMIC RETICULUM AMINOPEPTIDASE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Brionez TF, Reveille JD. The contribution of genes outside the major histocompatibility complex to susceptibility to ankylosing spondylitis. Curr Opin Rheumatol. 2008 Jul;20(4):384-91. doi: 10.1097/BOR.0b013e32830460fe. Review. Citation on PubMedBrown MA. Breakthroughs in genetic studies of ankylosing spondylitis. Rheumatology (Oxford). 2008 Feb;47(2):132-7. Epub 2007 Nov 22. Review. Citation on PubMedChang SC, Momburg F, Bhutani N, Goldberg AL. The ER aminopeptidase, ERAP1, trims precursors to lengths of MHC class I peptides by a \"molecular ruler\" mechanism. Proc Natl Acad Sci U S A. 2005 Nov 22;102(47):17107-12. Epub 2005 Nov 14. Citation on PubMed or Free article on PubMed CentralHammer GE, Gonzalez F, Champsaur M, Cado D, Shastri N. The aminopeptidase ERAAP shapes the peptide repertoire displayed by major histocompatibility complex class I molecules. Nat Immunol. 2006 Jan;7(1):103-12. Epub 2005 Nov 20. Citation on PubMedSaric T, Chang SC, Hattori A, York IA, Markant S, Rock KL, Tsujimoto M, Goldberg AL. An IFN-gamma-induced aminopeptidase in the ER, ERAP1, trims precursors to MHC class I-presented peptides. Nat Immunol. 2002 Dec;3(12):1169-76. Epub 2002 Nov 18. Citation on PubMedWellcome Trust Case Control Consortium; Australo-Anglo-American Spondylitis Consortium (TASC), Burton PR, Clayton DG, Cardon LR, Craddock N, Deloukas P, Duncanson A, Kwiatkowski DP, McCarthy MI, Ouwehand WH, Samani NJ, Todd JA, Donnelly P, Barrett JC, Davison D, Easton D, Evans DM, Leung HT, Marchini JL, Morris AP, Spencer CC, Tobin MD, Attwood AP, Boorman JP, Cant B, Everson U, Hussey JM, Jolley JD, Knight AS, Koch K, Meech E, Nutland S, Prowse CV, Stevens HE, Taylor NC, Walters GR, Walker NM, Watkins NA, Winzer T, Jones RW, McArdle WL, Ring SM, Strachan DP, Pembrey M, Breen G, St Clair D, Caesar S, Gordon-Smith K, Jones L, Fraser C, Green EK, Grozeva D, Hamshere ML, Holmans PA, Jones IR, Kirov G, Moskivina V, Nikolov I, O'Donovan MC, Owen MJ, Collier DA, Elkin A, Farmer A, Williamson R, McGuffin P, Young AH, Ferrier IN, Ball SG, Balmforth AJ, Barrett JH, Bishop TD, Iles MM, Maqbool A, Yuldasheva N, Hall AS, Braund PS, Dixon RJ, Mangino M, Stevens S, Thompson JR, Bredin F, Tremelling M, Parkes M, Drummond H, Lees CW, Nimmo ER, Satsangi J, Fisher SA, Forbes A, Lewis CM, Onnie CM, Prescott NJ, Sanderson J, Matthew CG, Barbour J, Mohiuddin MK, Todhunter CE, Mansfield JC, Ahmad T, Cummings FR, Jewell DP, Webster J, Brown MJ, Lathrop MG, Connell J, Dominiczak A, Marcano CA, Burke B, Dobson R, Gungadoo J, Lee KL, Munroe PB, Newhouse SJ, Onipinla A, Wallace C, Xue M, Caulfield M, Farrall M, Barton A; Biologics in RA Genetics and Genomics Study Syndicate (BRAGGS) Steering Committee, Bruce IN, Donovan H, Eyre S, Gilbert PD, Hilder SL, Hinks AM, John SL, Potter C, Silman AJ, Symmons DP, Thomson W, Worthington J, Dunger DB, Widmer B, Frayling TM, Freathy RM, Lango H, Perry JR, Shields BM, Weedon MN, Hattersley AT, Hitman GA, Walker M, Elliott KS, Groves CJ, Lindgren CM, Rayner NW, Timpson NJ, Zeggini E, Newport M, Sirugo G, Lyons E, Vannberg F, Hill AV, Bradbury LA, Farrar C, Pointon JJ, Wordsworth P, Brown MA, Franklyn JA, Heward JM, Simmonds MJ, Gough SC, Seal S; Breast Cancer Susceptibility Collaboration (UK), Stratton MR, Rahman N, Ban M, Goris A, Sawcer SJ, Compston A, Conway D, Jallow M, Newport M, Sirugo G, Rockett KA, Bumpstead SJ, Chaney A, Downes K, Ghori MJ, Gwilliam R, Hunt SE, Inouye M, Keniry A, King E, McGinnis R, Potter S, Ravindrarajah R, Whittaker P, Widden C, Withers D, Cardin NJ, Davison D, Ferreira T, Pereira-Gale J, Hallgrimsdo'ttir IB, Howie BN, Su Z, Teo YY, Vukcevic D, Bentley D, Brown MA, Compston A, Farrall M, Hall AS, Hattersley AT, Hill AV, Parkes M, Pembrey M, Stratton MR, Mitchell SL, Newby PR, Brand OJ, Carr-Smith J, Pearce SH, McGinnis R, Keniry A, Deloukas P, Reveille JD, Zhou X, Sims AM, Dowling A, Taylor J, Doan T, Davis JC, Savage L, Ward MM, Learch TL, Weisman MH, Brown M. Association scan of 14,500 nonsynonymous SNPs in four diseases identifies autoimmunity variants. Nat Genet. 2007 Nov;39(11):1329-37. Epub 2007 Oct 21. Citation on PubMed or Free article on PubMed CentralYork IA, Brehm MA, Zendzian S, Towne CF, Rock KL. Endoplasmic reticulum aminopeptidase 1 (ERAP1) trims MHC class I-presented peptides in vivo and plays an important role in immunodominance. Proc Natl Acad Sci U S A. 2006 Jun 13;103(24):9202-7. Epub 2006 Jun 5. Citation on PubMed or Free article on PubMed CentralYork IA, Chang SC, Saric T, Keys JA, Favreau JM, Goldberg AL, Rock KL. The ER aminopeptidase ERAP1 enhances or limits antigen presentation by trimming epitopes to 8-9 residues. Nat Immunol. 2002 Dec;3(12):1177-84. Epub 2002 Nov 18. Citation on PubMed \n",
            "CHD1\t   From NCBI Gene:The CHD family of proteins is characterized by the presence of chromo (chromatin organization modifier) domains and SNF2-related helicase/ATPase domains.  CHD genes alter gene expression possibly by modification of chromatin structure thus altering access of the transcriptional apparatus to its chromosomal DNA template. [provided by RefSeq, Jul 2008]From UniProt:ATP-dependent chromatin-remodeling factor which functions as substrate recognition component of the transcription regulatory histone acetylation (HAT) complex SAGA. Regulates polymerase II transcription. Also required for efficient transcription by RNA polymerase I, and more specifically the polymerase I transcription termination step. Regulates negatively DNA replication. Not only involved in transcription-related chromatin-remodeling, but also required to maintain a specific chromatin configuration across the genome. Is also associated with histone deacetylase (HDAC) activity (By similarity). Required for the bridging of SNF2, the FACT complex, the PAF complex as well as the U2 snRNP complex to H3K4me3. Functions to modulate the efficiency of pre-mRNA splicing in part through physical bridging of spliceosomal components to H3K4me3 (PubMed:18042460, PubMed:28866611). Required for maintaining open chromatin and pluripotency in embryonic stem cells (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:PILAROWSKI-BJORNSSON SYNDROMEFrom UniProt:Pilarowski-Bjornsson syndrome (PILBOS): An autosomal dominant disorder characterized by developmental delay, speech apraxia, intellectual disability, autism, and facial dysmorphic features. Some patients may have seizures. [MIM:617682]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 5q15-q21.1, which is the long (q) arm of chromosome 5 between positions 15 and 21.1Molecular Location: base pairs 98,855,204 to 98,929,772 on chromosome 5 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CHD-1PILBOS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CHD1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CHROMODOMAIN HELICASE DNA-BINDING PROTEIN 1OMIM: PILAROWSKI-BJORNSSON SYNDROME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CHD1  OMIM: CHROMODOMAIN HELICASE DNA-BINDING PROTEIN 1OMIM: PILAROWSKI-BJORNSSON SYNDROME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GCNT2\t   From NCBI Gene:This gene encodes the enzyme responsible for formation of the blood group I antigen. The i and I antigens are distinguished by linear and branched poly-N-acetyllactosaminoglycans, respectively. The encoded protein is the I-branching enzyme, a beta-1,6-N-acetylglucosaminyltransferase responsible for the conversion of fetal i antigen to adult I antigen in erythrocytes during embryonic development. Mutations in this gene have been associated with adult i blood group phenotype. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jul 2008]From UniProt:Isoform C: Determines the expression of the blood group I antigen in erythrocytes.Branching enzyme that converts linear into branched poly-N-acetyllactosaminoglycans. Introduces the blood group I antigen during embryonic development. It is closely associated with the development and maturation of erythroid cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Cataract 13 with adult i phenotypeI blood group systemFrom UniProt:Cataract 13, with adult i phenotype (CTRCT13): An opacification of the crystalline lens of the eye that frequently results in visual impairment or blindness. Opacities vary in morphology, are often confined to a portion of the lens, and may be static or progressive. In general, the more posteriorly located and dense an opacity, the greater the impact on visual function. CTRCT13 is associated with the rare adult i phenotype, in which adult red blood cells are rich in i antigen and contain low levels of I antigen. [MIM:116700]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p24.3-p24.2, which is the short (p) arm of chromosome 6 between positions 24.3 and 24.2Molecular Location: base pairs 10,521,283 to 10,629,368 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA360O19.2bA421M1.1CCATCTRCT13GCNT2CGCNT5IGNTIINACGT1NAGCT1ULG3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GCNT2      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BLOOD GROUP, I SYSTEMOMIM: CATARACT 13 WITH ADULT i PHENOTYPEOMIM: GLUCOSAMINYL (N-ACETYL) TRANSFERASE 2, I-BRANCHING ENZYME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GCNT2  OMIM: BLOOD GROUP, I SYSTEMOMIM: CATARACT 13 WITH ADULT i PHENOTYPEOMIM: GLUCOSAMINYL (N-ACETYL) TRANSFERASE 2, I-BRANCHING ENZYME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SYCP2L\t\n",
            "AK9\t\n",
            "LAMA4\t   From NCBI Gene:Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution, presumably reflecting diverse functions in vivo. This gene encodes the alpha chain isoform laminin, alpha 4. The domain structure of alpha 4 is similar to that of alpha 3, both of which resemble truncated versions of alpha 1 and alpha 2, in that approximately 1,200 residues at the N-terminus (domains IV, V and VI) have been lost. Laminin, alpha 4 contains the C-terminal G domain which distinguishes all alpha chains from the beta and gamma chains. The RNA analysis from adult and fetal tissues revealed developmental regulation of expression, however, the exact function of laminin, alpha 4 is not known. Tissue-specific utilization of alternative polyA-signal has been described in literature. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2011]From UniProt:Binding to cells via a high affinity receptor, laminin is thought to mediate the attachment, migration and organization of cells into tissues during embryonic development by interacting with other extracellular matrix components.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Familial dilated cardiomyopathyFrom NCBI Gene:Dilated cardiomyopathy 1JJFrom UniProt:Cardiomyopathy, dilated 1JJ (CMD1JJ): A disorder characterized by ventricular dilation and impaired systolic function, resulting in congestive heart failure and arrhythmia. Patients are at risk of premature death. [MIM:615235]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q21, which is the long (q) arm of chromosome 6 at position 21Molecular Location: base pairs 112,107,931 to 112,254,722 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMD1JJLAMA3LAMA4*-1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LAMA4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CARDIOMYOPATHY, DILATED, 1JJOMIM: LAMININ, ALPHA-4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LAMA4  OMIM: CARDIOMYOPATHY, DILATED, 1JJOMIM: LAMININ, ALPHA-4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MARCKS\t\n",
            "TSPYL1\t   The TSPYL1 gene provides instructions for making a protein called TSPY-like 1. This protein is active in the brain, testes (in males), and other tissues, although its function is not well understood. TSPY-like 1 contains a region called a nucleosome assembly protein (NAP) domain, which is found in other proteins that help control cell division, copy (replicate) DNA, and regulate the activity of various genes. It is unknown whether TSPY-like 1 also has these functions.Based on its role in a condition called sudden infant death with dysgenesis of the testes syndrome, researchers propose that TSPY-like 1 is involved in the development of the male reproductive system and the brain, including the brainstem. The brainstem is a part of the brain that is connected to the spinal cord. It regulates many basic body functions, including heart rate, breathing, eating, and sleeping. It also relays information about movement and the senses between the brain and the rest of the body.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Sudden infant death with dysgenesis of the testes syndrome     A single mutation in the TSPYL1 gene has caused all identified cases of sudden infant death with dysgenesis of the testes syndrome (SIDDT), a condition that has been reported in an Old Order Amish community in Pennsylvania. The condition is fatal in the first year of life; its major features include abnormalities of the reproductive system in males, breathing problems, a slow or uneven heart rate, and feeding difficulties.The mutation that causes SIDDT inserts a single DNA building block (nucleotide) into the TSPYL1 gene. This mutation is written as 457_458insG. The extra nucleotide alters how the gene's instructions are used to make TSPY-like 1, which results in the production of an abnormally short, nonfunctional protein. A loss of TSPY-like 1 function appears to disrupt the normal development of the male reproductive system and the brain, including the brainstem. Abnormalities of brainstem function, particularly involving breathing and heart rate, are likely the cause of sudden death in affected infants.Research findings suggest that mutations in the TSPYL1 gene are not associated with sudden infant death syndrome (SIDS) in the general population. SIDS is a major cause of death in children younger than 1 year. More About This Health Condition      Other disorders     Changes in the TSPYL1 gene have been studied as a potential cause of infertility, which is the inability to have biological children. Variations in the gene have been identified in several infertile men and in at least one person who was genetically male (with one X chromosome and one Y chromosome in each cell) but had external genitalia that appeared female. However, it is unclear whether the TSPYL1 gene variations were related to these abnormalities or occurred by coincidence. Changes in the TSPYL1 gene are not thought to be a major cause of infertility.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A single mutation in the TSPYL1 gene has caused all identified cases of sudden infant death with dysgenesis of the testes syndrome (SIDDT), a condition that has been reported in an Old Order Amish community in Pennsylvania. The condition is fatal in the first year of life; its major features include abnormalities of the reproductive system in males, breathing problems, a slow or uneven heart rate, and feeding difficulties.The mutation that causes SIDDT inserts a single DNA building block (nucleotide) into the TSPYL1 gene. This mutation is written as 457_458insG. The extra nucleotide alters how the gene's instructions are used to make TSPY-like 1, which results in the production of an abnormally short, nonfunctional protein. A loss of TSPY-like 1 function appears to disrupt the normal development of the male reproductive system and the brain, including the brainstem. Abnormalities of brainstem function, particularly involving breathing and heart rate, are likely the cause of sudden death in affected infants.Research findings suggest that mutations in the TSPYL1 gene are not associated with sudden infant death syndrome (SIDS) in the general population. SIDS is a major cause of death in children younger than 1 year. More About This Health Condition  Changes in the TSPYL1 gene have been studied as a potential cause of infertility, which is the inability to have biological children. Variations in the gene have been identified in several infertile men and in at least one person who was genetically male (with one X chromosome and one Y chromosome in each cell) but had external genitalia that appeared female. However, it is unclear whether the TSPYL1 gene variations were related to these abnormalities or occurred by coincidence. Changes in the TSPYL1 gene are not thought to be a major cause of infertility.    Cytogenetic Location: 6q22.1, which is the long (q) arm of chromosome 6 at position 22.1Molecular Location: base pairs 116,274,859 to 116,280,117 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        testis-specific Y-encoded-like protein 1TSPY-like 1TSPY-like protein 1TSPYL     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TSPYL1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TSPY-LIKE 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TSPYL1  PubMed  TSPY-LIKE 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Hering R, Frade-Martinez R, Bajanowski T, Poets CF, Tschentscher F, Riess O. Genetic investigation of the TSPYL1 gene in sudden infant death syndrome. Genet Med. 2006 Jan;8(1):55-8. Citation on PubMedJavaher P, Stuhrmann M, Wilke C, Frenzel E, Manukjan G, Grosshenig A, Dechend F, Schwaab E, Schmidtke J, Schubert S. Should TSPYL1 mutation screening be included in routine diagnostics of male idiopathic infertility? Fertil Steril. 2012 Feb;97(2):402-6. doi: 10.1016/j.fertnstert.2011.11.002. Epub 2011 Dec 2. Citation on PubMedPuffenberger EG, Hu-Lince D, Parod JM, Craig DW, Dobrin SE, Conway AR, Donarum EA, Strauss KA, Dunckley T, Cardenas JF, Melmed KR, Wright CA, Liang W, Stafford P, Flynn CR, Morton DH, Stephan DA. Mapping of sudden infant death with dysgenesis of the testes syndrome (SIDDT) by a SNP genome scan and identification of TSPYL loss of function. Proc Natl Acad Sci U S A. 2004 Aug 10;101(32):11689-94. Epub 2004 Jul 23. Citation on PubMed or Free article on PubMed CentralSchubert S, Haas C, Bartsch C, Mirshekarnejad M, Kohrs S, Roettinger I, Grosshennig A, Stuhrmann M, Scholz C, Schmidtke J. Variants in TSPYL1 are not associated with sudden infant death syndrome in a cohort of deceased infants from Switzerland. Mol Cell Probes. 2015 Feb;29(1):31-4. doi: 10.1016/j.mcp.2014.10.006. Epub 2014 Nov 4. Citation on PubMedOMIM: TSPY-LIKE 1Vinci G, Brauner R, Tar A, Rouba H, Sheth J, Sheth F, Ravel C, McElreavey K, Bashamboo A. Mutations in the TSPYL1 gene associated with 46,XY disorder of sex development and male infertility. Fertil Steril. 2009 Oct;92(4):1347-50. doi: 10.1016/j.fertnstert.2009.04.009. Epub 2009 May 21. Citation on PubMedVogel T, Dittrich O, Mehraein Y, Dechend F, Schnieders F, Schmidtke J. Murine and human TSPYL genes: novel members of the TSPY-SET-NAP1L1 family. Cytogenet Cell Genet. 1998;81(3-4):265-70. Citation on PubMed \n",
            "FAM162B\t\n",
            "TRDN\t   From NCBI Gene:This gene encodes an integral membrane protein that contains a single transmembrane domain. As similar protein in rabbits plays a role in skeletal muscle excitation-contraction coupling as part of the calcium release complex in association with the ryanodine receptor. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and single nucleotide polymorphisms in this gene may be markers for IgA nephritis. [provided by RefSeq, Oct 2011]From UniProt:Contributes to the regulation of lumenal Ca2+ release via the sarcoplasmic reticulum calcium release channels RYR1 and RYR2, a key step in triggering skeletal and heart muscle contraction. Required for normal organization of the triad junction, where T-tubules and the sarcoplasmic reticulum terminal cisternae are in close contact (By similarity). Required for normal skeletal muscle strength. Plays a role in excitation-contraction coupling in the heart and in regulating the rate of heart beats.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Catecholaminergic polymorphic ventricular tachycardia type 1Ventricular tachycardia, catecholaminergic polymorphic, 5, with or without muscle weaknessFrom UniProt:Ventricular tachycardia, catecholaminergic polymorphic, 5, with or without muscle weakness (CPVT5): An arrhythmogenic disorder characterized by stress-induced, bidirectional ventricular tachycardia that may degenerate into cardiac arrest and cause sudden death. Patients present with recurrent syncope, or sudden death after physical activity or emotional stress. Some patients have muscle weakness. CPVT5 inheritance is autosomal recessive. [MIM:615441]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q22.31, which is the long (q) arm of chromosome 6 at position 22.31Molecular Location: base pairs 123,216,339 to 123,637,093 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CPVT5TDNTRISK     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of TRDN      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: TRIADINOMIM: VENTRICULAR TACHYCARDIA, CATECHOLAMINERGIC POLYMORPHIC, 1, WITH OR WITHOUT ATRIAL DYSFUNCTION AND/OR DILATED CARDIOMYOPATHYOMIM: VENTRICULAR TACHYCARDIA, CATECHOLAMINERGIC POLYMORPHIC, 5, WITH OR WITHOUT MUSCLE WEAKNESS      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of TRDN  OMIM: TRIADINOMIM: VENTRICULAR TACHYCARDIA, CATECHOLAMINERGIC POLYMORPHIC, 1, WITH OR WITHOUT ATRIAL DYSFUNCTION AND/OR DILATED CARDIOMYOPATHYOMIM: VENTRICULAR TACHYCARDIA, CATECHOLAMINERGIC POLYMORPHIC, 5, WITH OR WITHOUT MUSCLE WEAKNESS  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TMEM244\t\n",
            "SMLR1\t\n",
            "TAAR5\t\n",
            "TAAR2\t\n",
            "VNN3\t\n",
            "AHI1\t   From NCBI Gene:This gene is apparently required for both cerebellar and cortical development in humans. This gene mutations cause specific forms of Joubert syndrome-related disorders. Joubert syndrome (JS) is a recessively inherited developmental brain disorder with several identified causative chromosomal loci. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Oct 2008]From UniProt:Involved in vesicle trafficking and required for ciliogenesis, formation of primary non-motile cilium, and recruitment of RAB8A to the basal body of primary cilium. Component of the tectonic-like complex, a complex localized at the transition zone of primary cilia and acting as a barrier that prevents diffusion of transmembrane proteins between the cilia and plasma membranes. Involved in neuronal differentiation. As a positive modulator of classical Wnt signaling, may play a crucial role in ciliary signaling during cerebellum embryonic development (PubMed:21623382).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Joubert syndromeFrom NCBI Gene:Joubert syndrome 3From UniProt:Joubert syndrome 3 (JBTS3): A disorder presenting with cerebellar ataxia, oculomotor apraxia, hypotonia, neonatal breathing abnormalities and psychomotor delay. Neuroradiologically, it is characterized by cerebellar vermian hypoplasia/aplasia, thickened and reoriented superior cerebellar peduncles, and an abnormally large interpeduncular fossa, giving the appearance of a molar tooth on transaxial slices (molar tooth sign). Additional variable features include retinal dystrophy and renal disease. Joubert syndrome type 3 shows minimal extra central nervous system involvement and appears not to be associated with renal dysfunction. [MIM:608629]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q23.3, which is the long (q) arm of chromosome 6 at position 23.3Molecular Location: base pairs 135,283,972 to 135,497,771 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AHI-1dJ71N10.1JBTS3ORF1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AHI1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ABELSON HELPER INTEGRATION SITE 1OMIM: JOUBERT SYNDROME 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AHI1  OMIM: ABELSON HELPER INTEGRATION SITE 1OMIM: JOUBERT SYNDROME 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "BCLAF1\t\n",
            "MAP7\t\n",
            "FOXF2\t\n",
            "REPS1\t   From NCBI Gene:This gene encodes a signaling adaptor protein with two EH domains that interacts with proteins that participate in signaling, endocytosis and cytoskeletal changes. The encoded protein has been found in association with intersectin 1 and Src homology 3-domain growth factor receptor-bound 2-like (endophilin) interacting protein 1 when intersectin 1 was isolated from clathrin-coated pits. The encoded protein has also been shown to interact with amphiphysin, a cytoplasmic protein at the surface of synaptic vesicles. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Mar 2014]From UniProt:May coordinate the cellular actions of activated EGF receptors and Ral-GTPases.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:NEURODEGENERATION WITH BRAIN IRON ACCUMULATION 7From UniProt:Neurodegeneration with brain iron accumulation 7 (NBIA7): A neurodegenerative disorder associated with iron accumulation, primarily in the basal ganglia. Clinical features include speech and motor delay, truncal hypotonia, progressive cerebellar ataxia, and loss of ambulation. NBIA7 transmission pattern is consistent with autosomal recessive inheritance. [MIM:617916]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q24.1, which is the long (q) arm of chromosome 6 at position 24.1Molecular Location: base pairs 138,904,015 to 138,988,261 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        NBIA7RALBP1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of REPS1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: NEURODEGENERATION WITH BRAIN IRON ACCUMULATION 7OMIM: RALBP1-ASSOCIATED EPS DOMAIN-CONTAINING PROTEIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of REPS1  OMIM: NEURODEGENERATION WITH BRAIN IRON ACCUMULATION 7OMIM: RALBP1-ASSOCIATED EPS DOMAIN-CONTAINING PROTEIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NMBR\t\n",
            "FUCA2\t\n",
            "ZC2HC1B\t\n",
            "ADGB\t\n",
            "SUMO4\t   From NCBI Gene:This gene is a member of the SUMO gene family. This family of genes encode small ubiquitin-related modifiers that are attached to proteins and control the target proteins' subcellular localization, stability, or activity. The protein described in this record is located in the cytoplasm and specifically modifies IKBA, leading to negative regulation of NF-kappa-B-dependent transcription of the IL12B gene. A specific polymorphism in this SUMO gene, which leads to the M55V substitution, has been associated with type I diabetes. The RefSeq contains this polymorphism. [provided by RefSeq, Jul 2008]From UniProt:Ubiquitin-like protein which can be covalently attached to target lysines as a monomer. Does not seem to be involved in protein degradation and may modulate protein subcellular localization, stability or activity. Upon oxidative stress, conjugates to various anti-oxidant enzymes, chaperones, and stress defense proteins. May also conjugate to NFKBIA, TFAP2A and FOS, negatively regulating their transcriptional activity, and to NR3C1, positively regulating its transcriptional activity. Covalent attachment to its substrates requires prior activation by the E1 complex SAE1-SAE2 and linkage to the E2 enzyme UBE2I.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Type 1 diabetesFrom NCBI Gene:Diabetes mellitus, insulin-dependent, 5From UniProt:Diabetes mellitus, insulin-dependent, 5 (IDDM5): A form of diabetes mellitus, a multifactorial disorder of glucose homeostasis that is characterized by susceptibility to ketoacidosis in the absence of insulin therapy. Clinical features are polydipsia, polyphagia and polyuria which result from hyperglycemia-induced osmotic diuresis and secondary thirst. These derangements result in long-term complications that affect the eyes, kidneys, nerves, and blood vessels. [MIM:600320]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q25.1, which is the long (q) arm of chromosome 6 at position 25.1Molecular Location: base pairs 149,400,359 to 149,401,046 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        dJ281H8.4IDDM5SMT3H4SUMO-4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SUMO4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DIABETES MELLITUS, INSULIN-DEPENDENT, 5OMIM: SMALL UBIQUITIN-LIKE MODIFIER 4      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SUMO4  OMIM: DIABETES MELLITUS, INSULIN-DEPENDENT, 5OMIM: SMALL UBIQUITIN-LIKE MODIFIER 4  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZC3H12D\t\n",
            "AKAP12\t\n",
            "OPRM1\t   The OPRM1 gene provides instructions for making a protein called the mu () opioid receptor. Opioid receptors are part of the endogenous opioid system, which is the body's internal system for regulating pain, reward, and addictive behaviors. It consists of opioid substances produced naturally within the body (called endogenous opioids) and their receptors, into which opioids fit like keys into locks. Opioid receptors are found in the nervous system, where they are embedded in the outer membrane of nerve cells (neurons). When opioids attach (bind) to the receptors, the interaction triggers a series of chemical changes within and between neurons that lead to feelings of pleasure and pain relief.The  opioid receptor was the first opioid receptor to be discovered. It is the primary receptor for endogenous opioids called beta-endorphin and enkephalins, which help regulate the body's response to pain, among other functions. The  opioid receptor is also the binding site for many opioids introduced from outside the body (called exogenous opioids). These include commonly prescribed pain medications such as oxycodone, fentanyl, buprenorphine, methadone, oxymorphone, hydrocodone, codeine, and morphine, as well as illegal opioid drugs such as heroin.When endogenous or exogenous opioids bind to the  opioid receptor, the interaction triggers a cascade of chemical signals in the nervous system. These signals reduce the activity (excitability) of neurons in certain areas of the brain, which leads to pain relief and feelings of pleasure and intense happiness (euphoria). In addition, the chemical signaling ultimately increases the production of a chemical called dopamine. Dopamine is a chemical messenger (neurotransmitter) that helps regulate areas of the brain involved in reward-seeking behavior, attention, and mood.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alcohol use disorder     Genetics Home Reference provides information about alcohol use disorder. More About This Health Condition      Opioid addiction     Common variations (polymorphisms) in the OPRM1 gene have been studied as risk factors for opioid addiction. Opioid addiction is a long-lasting (chronic) disease characterized by a powerful, sometimes uncontrollable urge to use opioid drugs. Opioid addiction has major health, social, and economic effects.The best-studied OPRM1 gene polymorphism changes a single protein building block (amino acid) in a particular place in the  opioid receptor protein. Specifically, it replaces the amino acid alanine (A) with the amino acid glycine (G) at position 118, written as Ala118Gly or A118G. (This polymorphism is also identified with a unique number, rs1799971.) The A118G polymorphism likely has an effect on the amount of  opioid receptor present in the membrane surrounding neurons, and on the ability of the receptor to transmit chemical signals. Research into the association of this polymorphism with opioid addiction has had mixed results. Some studies suggest that having glycine (G) instead of alanine (A) increases the amount of an opioid medication needed to achieve pain relief and raises the risk of opioid addiction. However, other studies found no association between the polymorphism and opioid addiction, and still others reported a lower risk with the glycine (G) version of the polymorphism.The glycine (G) version of the A118G polymorphism is much more common in certain populations, such as people of Asian or European ancestry, than in others, such as people of African or African American ancestry. These differences may help explain why the results of studies examining its role in opioid addiction have had conflicting results. Researchers suggest that studies with many more people would be needed to confirm an association between this polymorphism and the risk of opioid addiction in any particular population.Common variations in the OPRM1 gene other than A118G have also been associated with opioid addiction in specific populations, such as Han Chinese, European Americans, and African Americans. The A118G polymorphism and other common variations are among many suspected risk factors for opioid addiction. It is likely that a combination of health, social, economic, and lifestyle factors interact with genetic factors to determine an individual's risk of developing this complex disease. More About This Health Condition      Other disorders     Variations in the OPRM1 gene have been associated with addiction to several additional substances. The  opioid receptor appears to play a critical role in regulating the pleasure and reward that come from the use of alcohol, nicotine, and certain other drugs of abuse. Although these substances do not interact directly with the  opioid receptor, they affect the levels of other neurotransmitters in the brain, triggering the body's own endogenous opioids to attach to the receptor. This interaction starts the cascade of chemical signaling in the brain that leads to pain relief and feelings of relaxation and pleasure.As with opioid addiction (described above), most of the research on the connection between the  opioid receptor and other addictions has focused on the A118G polymorphism. Studies of the effects of this polymorphism on nicotine and alcohol abuse have had inconsistent results, with different studies suggesting an increased risk, a decreased risk, or no effect of having glycine (G) versus alanine (A) at position 118. A person's geographic and ethnic background may be important, and larger studies would be necessary to determine whether a true association exists.Studies have also found associations between the A118G polymorphism and a variety of other traits, including perception of physical and psychological pain, sensitivity to social rejection, a preference for sweet and fatty foods, and how the body responds to stress. It is unclear how this genetic variation causes changes in the brain that influence these traits.Another polymorphism in the OPRM1 gene, usually written as rs540825, is associated with the effectiveness of a drug called citalopram in people with major depressive disorder. This polymorphism changes a single amino acid near one end of the  opioid receptor. People with major depressive disorder who have this polymorphism are more likely to experience periods without any symptoms (remission) when treated with citalopram than those who do not have the polymorphism.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about alcohol use disorder. More About This Health Condition  Common variations (polymorphisms) in the OPRM1 gene have been studied as risk factors for opioid addiction. Opioid addiction is a long-lasting (chronic) disease characterized by a powerful, sometimes uncontrollable urge to use opioid drugs. Opioid addiction has major health, social, and economic effects.The best-studied OPRM1 gene polymorphism changes a single protein building block (amino acid) in a particular place in the  opioid receptor protein. Specifically, it replaces the amino acid alanine (A) with the amino acid glycine (G) at position 118, written as Ala118Gly or A118G. (This polymorphism is also identified with a unique number, rs1799971.) The A118G polymorphism likely has an effect on the amount of  opioid receptor present in the membrane surrounding neurons, and on the ability of the receptor to transmit chemical signals. Research into the association of this polymorphism with opioid addiction has had mixed results. Some studies suggest that having glycine (G) instead of alanine (A) increases the amount of an opioid medication needed to achieve pain relief and raises the risk of opioid addiction. However, other studies found no association between the polymorphism and opioid addiction, and still others reported a lower risk with the glycine (G) version of the polymorphism.The glycine (G) version of the A118G polymorphism is much more common in certain populations, such as people of Asian or European ancestry, than in others, such as people of African or African American ancestry. These differences may help explain why the results of studies examining its role in opioid addiction have had conflicting results. Researchers suggest that studies with many more people would be needed to confirm an association between this polymorphism and the risk of opioid addiction in any particular population.Common variations in the OPRM1 gene other than A118G have also been associated with opioid addiction in specific populations, such as Han Chinese, European Americans, and African Americans. The A118G polymorphism and other common variations are among many suspected risk factors for opioid addiction. It is likely that a combination of health, social, economic, and lifestyle factors interact with genetic factors to determine an individual's risk of developing this complex disease. More About This Health Condition  Variations in the OPRM1 gene have been associated with addiction to several additional substances. The  opioid receptor appears to play a critical role in regulating the pleasure and reward that come from the use of alcohol, nicotine, and certain other drugs of abuse. Although these substances do not interact directly with the  opioid receptor, they affect the levels of other neurotransmitters in the brain, triggering the body's own endogenous opioids to attach to the receptor. This interaction starts the cascade of chemical signaling in the brain that leads to pain relief and feelings of relaxation and pleasure.As with opioid addiction (described above), most of the research on the connection between the  opioid receptor and other addictions has focused on the A118G polymorphism. Studies of the effects of this polymorphism on nicotine and alcohol abuse have had inconsistent results, with different studies suggesting an increased risk, a decreased risk, or no effect of having glycine (G) versus alanine (A) at position 118. A person's geographic and ethnic background may be important, and larger studies would be necessary to determine whether a true association exists.Studies have also found associations between the A118G polymorphism and a variety of other traits, including perception of physical and psychological pain, sensitivity to social rejection, a preference for sweet and fatty foods, and how the body responds to stress. It is unclear how this genetic variation causes changes in the brain that influence these traits.Another polymorphism in the OPRM1 gene, usually written as rs540825, is associated with the effectiveness of a drug called citalopram in people with major depressive disorder. This polymorphism changes a single amino acid near one end of the  opioid receptor. People with major depressive disorder who have this polymorphism are more likely to experience periods without any symptoms (remission) when treated with citalopram than those who do not have the polymorphism.    Cytogenetic Location: 6q25.2, which is the long (q) arm of chromosome 6 at position 25.2Molecular Location: base pairs 154,010,496 to 154,246,867 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LMORM-OR-1MOPMORMOR-1MOR1mu opiate receptormu opioid receptor hMOR-1aOPRM     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Facing Addiction in America (2016): The Neurobiology of Substance Use, Misuse, and AddictionPain Management and the Opioid Epidemic (2017): The Neurobiology of the Reward Pathway and the Intersection of Pain and Opioid Use Disorder      Tests Listed in the Genetic Testing Registry (1 link)     Tests of OPRM1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     OPIOID RECEPTOR, MU-1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Facing Addiction in America (2016): The Neurobiology of Substance Use, Misuse, and AddictionPain Management and the Opioid Epidemic (2017): The Neurobiology of the Reward Pathway and the Intersection of Pain and Opioid Use Disorder  Tests of OPRM1  PubMed  OPIOID RECEPTOR, MU-1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bonenberger M, Plener PL, Groschwitz RC, Grn G, Abler B. Polymorphism in the -opioid receptor gene (OPRM1) modulates neural processing of physical pain, social rejection and error processing. Exp Brain Res. 2015 Sep;233(9):2517-26. doi: 10.1007/s00221-015-4322-9. Epub 2015 May 28. Citation on PubMedCrist RC, Berrettini WH. Pharmacogenetics of OPRM1. Pharmacol Biochem Behav. 2014 Aug;123:25-33. doi: 10.1016/j.pbb.2013.10.018. Epub 2013 Nov 5. Review. Citation on PubMed or Free article on PubMed CentralDavis C, Zai C, Levitan RD, Kaplan AS, Carter JC, Reid-Westoby C, Curtis C, Wight K, Kennedy JL. Opiates, overeating and obesity: a psychogenetic analysis. Int J Obes (Lond). 2011 Oct;35(10):1347-54. doi: 10.1038/ijo.2010.276. Epub 2011 Jan 25. Citation on PubMedGarriock HA, Tanowitz M, Kraft JB, Dang VC, Peters EJ, Jenkins GD, Reinalda MS, McGrath PJ, von Zastrow M, Slager SL, Hamilton SP. Association of mu-opioid receptor variants and response to citalopram treatment in major depressive disorder. Am J Psychiatry. 2010 May;167(5):565-73. doi: 10.1176/appi.ajp.2009.08081167. Epub 2010 Mar 1. Citation on PubMed or Free article on PubMed CentralHaerian BS, Haerian MS. OPRM1 rs1799971 polymorphism and opioid dependence: evidence from a meta-analysis. Pharmacogenomics. 2013 May;14(7):813-24. doi: 10.2217/pgs.13.57. Review. Citation on PubMedOMIM: OPIOID RECEPTOR, MU-1Pasternak G, Pan YX. Mu opioid receptors in pain management. Acta Anaesthesiol Taiwan. 2011 Mar;49(1):21-5. doi: 10.1016/j.aat.2010.12.008. Epub 2011 Mar 17. Review. Citation on PubMed or Free article on PubMed CentralPasternak GW, Pan YX. Mu opioids and their receptors: evolution of a concept. Pharmacol Rev. 2013 Sep 27;65(4):1257-317. doi: 10.1124/pr.112.007138. Print 2013. Review. Citation on PubMed or Free article on PubMed CentralReed B, Butelman ER, Yuferov V, Randesi M, Kreek MJ. Genetics of opiate addiction. Curr Psychiatry Rep. 2014 Nov;16(11):504. doi: 10.1007/s11920-014-0504-6. Review. Citation on PubMedSchwantes-An TH, Zhang J, Chen LS, Hartz SM, Culverhouse RC, Chen X, Coon H, Frank J, Kamens HM, Konte B, Kovanen L, Latvala A, Legrand LN, Maher BS, Melroy WE, Nelson EC, Reid MW, Robinson JD, Shen PH, Yang BZ, Andrews JA, Aveyard P, Beltcheva O, Brown SA, Cannon DS, Cichon S, Corley RP, Dahmen N, Degenhardt L, Foroud T, Gaebel W, Giegling I, Glatt SJ, Grucza RA, Hardin J, Hartmann AM, Heath AC, Herms S, Hodgkinson CA, Hoffmann P, Hops H, Huizinga D, Ising M, Johnson EO, Johnstone E, Kaneva RP, Kendler KS, Kiefer F, Kranzler HR, Krauter KS, Levran O, Lucae S, Lynskey MT, Maier W, Mann K, Martin NG, Mattheisen M, Montgomery GW, Mller-Myhsok B, Murphy MF, Neale MC, Nikolov MA, Nishita D, Nthen MM, Nurnberger J, Partonen T, Pergadia ML, Reynolds M, Ridinger M, Rose RJ, Rouvinen-Lagerstrm N, Scherbaum N, Schml C, Soyka M, Stallings MC, Steffens M, Treutlein J, Tsuang M, Wall TL, Wodarz N, Yuferov V, Zill P, Bergen AW, Chen J, Cinciripini PM, Edenberg HJ, Ehringer MA, Ferrell RE, Gelernter J, Goldman D, Hewitt JK, Hopfer CJ, Iacono WG, Kaprio J, Kreek MJ, Kremensky IM, Madden PA, McGue M, Munaf MR, Philibert RA, Rietschel M, Roy A, Rujescu D, Saarikoski ST, Swan GE, Todorov AA, Vanyukov MM, Weiss RB, Bierut LJ, Saccone NL. Association of the OPRM1 Variant rs1799971 (A118G) with Non-Specific Liability to Substance Dependence in a Collaborative de novo Meta-Analysis of European-Ancestry Cohorts. Behav Genet. 2016 Mar;46(2):151-69. doi: 10.1007/s10519-015-9737-3. Epub 2015 Sep 21. Citation on PubMed or Free article on PubMed CentralWalter C, Doehring A, Oertel BG, Ltsch J. -opioid receptor gene variant OPRM1 118 A>G: a summary of its molecular and clinical consequences for pain. Pharmacogenomics. 2013 Nov;14(15):1915-25. doi: 10.2217/pgs.13.187. Review. Citation on PubMedXu J, Lu Z, Xu M, Pan L, Deng Y, Xie X, Liu H, Ding S, Hurd YL, Pasternak GW, Klein RJ, Cartegni L, Zhou W, Pan YX. A heroin addiction severity-associated intronic single nucleotide polymorphism modulates alternative pre-mRNA splicing of the  opioid receptor gene OPRM1 via hnRNPH interactions. J Neurosci. 2014 Aug 13;34(33):11048-66. doi: 10.1523/JNEUROSCI.3986-13.2014. Citation on PubMed or Free article on PubMed Central \n",
            "TIAM2\t\n",
            "ARID1B\t   The ARID1B gene provides instructions for making a protein that forms one piece (subunit) of several different SWI/SNF protein complexes. SWI/SNF complexes regulate gene activity (expression) by a process known as chromatin remodeling. Chromatin is the network of DNA and proteins that packages DNA into chromosomes. The structure of chromatin can be changed (remodeled) to alter how tightly DNA is packaged. Chromatin remodeling is one way gene expression is regulated during development; when DNA is tightly packed, gene expression is lower than when DNA is loosely packed.Through their ability to regulate gene activity, SWI/SNF complexes are involved in many processes, including repairing damaged DNA; copying (replicating) DNA; and controlling the growth, division, and maturation (differentiation) of cells. The ARID1B protein and other SWI/SNF subunits are thought to act as tumor suppressors, which keep cells from growing and dividing too rapidly or in an uncontrolled way.The ARID1B subunit is able to attach (bind) to DNA and is thought to help target SWI/SNF complexes to the chromatin location that needs to be remodeled.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Coffin-Siris syndrome     At least 69 mutations in the ARID1B gene have been found to cause Coffin-Siris syndrome. This condition is characterized by delayed development, abnormalities of the fifth (pinky) fingers or toes, and characteristic facial features that are described as coarse. Most ARID1B gene mutations involved in Coffin-Siris syndrome lead to an abnormally short, nonfunctional protein. As a result, affected individuals have half the normal amount of functional ARID1B protein. Although it is unclear how these changes affect SWI/SNF complexes, researchers suggest that ARID1B gene mutations result in abnormal chromatin remodeling. Disturbance of this process alters the activity of many genes and disrupts several cellular processes, which could explain the diverse signs and symptoms of Coffin-Siris syndrome. People with Coffin-Siris syndrome do not appear to have an increased risk of cancer (see below). More About This Health Condition      Autism spectrum disorder     At least 13 ARID1B gene mutations have been identified in people with autism spectrum disorder (ASD), a varied condition characterized by impaired social skills, communication problems, and repetitive behaviors. Some affected individuals also have other features, including intellectual disability, severe speech problems, and brain malformations. The ARID1B gene mutations associated with ASD result in a reduced amount of the ARID1B protein or impair the protein's function in chromatin remodeling. These changes likely affect the control of gene expression and interfere with normal brain development, but the specific relationship between the mutations and ASD is unknown. More About This Health Condition      Cancers     Mutations in the ARID1B gene are involved in several types of cancer, including breast cancer, a childhood cancer of nerve tissue called neuroblastoma, and a type of blood cancer called diffuse large B-cell lymphoma. These mutations are somatic, which means they are acquired during a person's lifetime and are present only in tumor cells. The mechanism by which mutations in the ARID1B gene contribute to cancer is unknown, although it is thought that changes in SWI/SNF complexes are involved. These changes may impair normal cell differentiation, which leads to the overgrowth of certain cell types, causing cancer. Alternatively, abnormal SWI/SNF complexes may disrupt the regulation of genes that help control the growth and division of cells, which leads to cancer. It is likely that other genetic changes in addition to ARID1B gene mutations are necessary for cancer development.      Other disorders     Mutations in the ARID1B gene can cause intellectual disability without other hallmark features of Coffin-Siris syndrome (described above). As in Coffin-Siris syndrome, the gene mutations that cause intellectual disability lead to an abnormally short, nonfunctional protein, and affected individuals have half the normal amount of ARID1B protein. It is unclear why some people with an ARID1B gene mutation develop intellectual disability and others have the additional features of Coffin-Siris syndrome.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 69 mutations in the ARID1B gene have been found to cause Coffin-Siris syndrome. This condition is characterized by delayed development, abnormalities of the fifth (pinky) fingers or toes, and characteristic facial features that are described as coarse. Most ARID1B gene mutations involved in Coffin-Siris syndrome lead to an abnormally short, nonfunctional protein. As a result, affected individuals have half the normal amount of functional ARID1B protein. Although it is unclear how these changes affect SWI/SNF complexes, researchers suggest that ARID1B gene mutations result in abnormal chromatin remodeling. Disturbance of this process alters the activity of many genes and disrupts several cellular processes, which could explain the diverse signs and symptoms of Coffin-Siris syndrome. People with Coffin-Siris syndrome do not appear to have an increased risk of cancer (see below). More About This Health Condition  At least 13 ARID1B gene mutations have been identified in people with autism spectrum disorder (ASD), a varied condition characterized by impaired social skills, communication problems, and repetitive behaviors. Some affected individuals also have other features, including intellectual disability, severe speech problems, and brain malformations. The ARID1B gene mutations associated with ASD result in a reduced amount of the ARID1B protein or impair the protein's function in chromatin remodeling. These changes likely affect the control of gene expression and interfere with normal brain development, but the specific relationship between the mutations and ASD is unknown. More About This Health Condition  Mutations in the ARID1B gene are involved in several types of cancer, including breast cancer, a childhood cancer of nerve tissue called neuroblastoma, and a type of blood cancer called diffuse large B-cell lymphoma. These mutations are somatic, which means they are acquired during a person's lifetime and are present only in tumor cells. The mechanism by which mutations in the ARID1B gene contribute to cancer is unknown, although it is thought that changes in SWI/SNF complexes are involved. These changes may impair normal cell differentiation, which leads to the overgrowth of certain cell types, causing cancer. Alternatively, abnormal SWI/SNF complexes may disrupt the regulation of genes that help control the growth and division of cells, which leads to cancer. It is likely that other genetic changes in addition to ARID1B gene mutations are necessary for cancer development.  Mutations in the ARID1B gene can cause intellectual disability without other hallmark features of Coffin-Siris syndrome (described above). As in Coffin-Siris syndrome, the gene mutations that cause intellectual disability lead to an abnormally short, nonfunctional protein, and affected individuals have half the normal amount of ARID1B protein. It is unclear why some people with an ARID1B gene mutation develop intellectual disability and others have the additional features of Coffin-Siris syndrome.    Cytogenetic Location: 6q25.3, which is the long (q) arm of chromosome 6 at position 25.3Molecular Location: base pairs 156,777,847 to 157,210,779 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        6A3-5ARI1B_HUMANARID domain-containing protein 1BAT rich interactive domain 1B (SWI1-like)AT-rich interactive domain-containing protein 1BBAF250BBRG1-associated factor 250bBRG1-binding protein ELD/OSA1BRIGHTDAN15ELD (eyelid)/OSA proteinELD/OSA1KIAA1235MRD12OSA2P250R     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Biology of the Cell (fourth edition, 2002): ATP-Driven Chromatin Remodeling Machines Change Nucleosome StructureMolecular Biology of the Cell (fourth edition, 2002): Chromosomal DNA and Its Packaging in the Chromatin FiberSimons Searchlight      Clinical Information from GeneReviews (1 link)     Coffin-Siris Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ARID1B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     AT-RICH INTERACTION DOMAIN-CONTAINING PROTEIN 1B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): ATP-Driven Chromatin Remodeling Machines Change Nucleosome StructureMolecular Biology of the Cell (fourth edition, 2002): Chromosomal DNA and Its Packaging in the Chromatin FiberSimons Searchlight  Coffin-Siris Syndrome  Tests of ARID1B  PubMed  AT-RICH INTERACTION DOMAIN-CONTAINING PROTEIN 1B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: AT-RICH INTERACTION DOMAIN-CONTAINING PROTEIN 1BD'Gama AM, Pochareddy S, Li M, Jamuar SS, Reiff RE, Lam AT, Sestan N, Walsh CA. Targeted DNA Sequencing from Autism Spectrum Disorder Brains Implicates Multiple Genetic Mechanisms. Neuron. 2015 Dec 2;88(5):910-7. doi: 10.1016/j.neuron.2015.11.009. Citation on PubMed or Free article on PubMed CentralHalgren C, Kjaergaard S, Bak M, Hansen C, El-Schich Z, Anderson CM, Henriksen KF, Hjalgrim H, Kirchhoff M, Bijlsma EK, Nielsen M, den Hollander NS, Ruivenkamp CA, Isidor B, Le Caignec C, Zannolli R, Mucciolo M, Renieri A, Mari F, Anderlid BM, Andrieux J, Dieux A, Tommerup N, Bache I. Corpus callosum abnormalities, intellectual disability, speech impairment, and autism in patients with haploinsufficiency of ARID1B. Clin Genet. 2012 Sep;82(3):248-55. doi: 10.1111/j.1399-0004.2011.01755.x. Epub 2011 Aug 24. Citation on PubMed or Free article on PubMed CentralHoyer J, Ekici AB, Endele S, Popp B, Zweier C, Wiesener A, Wohlleber E, Dufke A, Rossier E, Petsch C, Zweier M, Ghring I, Zink AM, Rappold G, Schrck E, Wieczorek D, Riess O, Engels H, Rauch A, Reis A. Haploinsufficiency of ARID1B, a member of the SWI/SNF-a chromatin-remodeling complex, is a frequent cause of intellectual disability. Am J Hum Genet. 2012 Mar 9;90(3):565-72. doi: 10.1016/j.ajhg.2012.02.007. Citation on PubMed or Free article on PubMed CentralNord AS, Roeb W, Dickel DE, Walsh T, Kusenda M, O'Connor KL, Malhotra D, McCarthy SE, Stray SM, Taylor SM, Sebat J; STAART Psychopharmacology Network., King B, King MC, McClellan JM. Reduced transcript expression of genes affected by inherited and de novo CNVs in autism. Eur J Hum Genet. 2011 Jun;19(6):727-31. doi: 10.1038/ejhg.2011.24. Epub 2011 Mar 30. Citation on PubMed or Free article on PubMed CentralSanten GW, Aten E, Sun Y, Almomani R, Gilissen C, Nielsen M, Kant SG, Snoeck IN, Peeters EA, Hilhorst-Hofstee Y, Wessels MW, den Hollander NS, Ruivenkamp CA, van Ommen GJ, Breuning MH, den Dunnen JT, van Haeringen A, Kriek M. Mutations in SWI/SNF chromatin remodeling complex gene ARID1B cause Coffin-Siris syndrome. Nat Genet. 2012 Mar 18;44(4):379-80. doi: 10.1038/ng.2217. Citation on PubMedSanten GW, Kriek M, van Attikum H. SWI/SNF complex in disorder: SWItching from malignancies to intellectual disability. Epigenetics. 2012 Nov;7(11):1219-24. doi: 10.4161/epi.22299. Epub 2012 Sep 25. Review. Citation on PubMed or Free article on PubMed CentralShain AH, Pollack JR. The spectrum of SWI/SNF mutations, ubiquitous in human cancers. PLoS One. 2013;8(1):e55119. doi: 10.1371/journal.pone.0055119. Epub 2013 Jan 23. Citation on PubMed or Free article on PubMed CentralTsurusaki Y, Okamoto N, Ohashi H, Kosho T, Imai Y, Hibi-Ko Y, Kaname T, Naritomi K, Kawame H, Wakui K, Fukushima Y, Homma T, Kato M, Hiraki Y, Yamagata T, Yano S, Mizuno S, Sakazume S, Ishii T, Nagai T, Shiina M, Ogata K, Ohta T, Niikawa N, Miyatake S, Okada I, Mizuguchi T, Doi H, Saitsu H, Miyake N, Matsumoto N. Mutations affecting components of the SWI/SNF complex cause Coffin-Siris syndrome. Nat Genet. 2012 Mar 18;44(4):376-8. doi: 10.1038/ng.2219. Citation on PubMedWang X, Nagl NG, Wilsker D, Van Scoy M, Pacchione S, Yaciuk P, Dallas PB, Moran E. Two related ARID family proteins are alternative subunits of human SWI/SNF complexes. Biochem J. 2004 Oct 15;383(Pt 2):319-25. Citation on PubMed or Free article on PubMed Central \n",
            "RSPH3\t   From NCBI Gene:The protein encoded by this gene acts as a protein kinase A anchoring protein. Mutations in this gene cause primary ciliary dyskinesia; a disorder characterized by defects of the axoneme in motile cilia and sperm flagella. The homolog of this gene was first identified in the blue-green algae Chlamydomonas as encoding a radial spoke protein that formed a structural component of motile cilia and flagella. Alternate splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Dec 2016]From UniProt:Functions as a protein kinase A-anchoring protein that scaffolds the cAMP-dependent protein kinase holoenzyme. May serve as a point of convergence for MAPK and PKA signaling in cilia.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Ciliary dyskinesia, primary, 32From UniProt:Ciliary dyskinesia, primary, 32 (CILD32): A disorder characterized by abnormalities of motile cilia. Respiratory infections leading to chronic inflammation and bronchiectasis are recurrent, due to defects in the respiratory cilia. [MIM:616481]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q25.3, which is the long (q) arm of chromosome 6 at position 25.3Molecular Location: base pairs 158,947,012 to 159,000,426 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CILD32dJ111C20.1RSHL2RSP3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RSPH3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CILIARY DYSKINESIA, PRIMARY, 32OMIM: RADIAL SPOKE HEAD 3, CHLAMYDOMONAS, HOMOLOG OF      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RSPH3  OMIM: CILIARY DYSKINESIA, PRIMARY, 32OMIM: RADIAL SPOKE HEAD 3, CHLAMYDOMONAS, HOMOLOG OF  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FNDC1\t\n",
            "ACAT2\t   From NCBI Gene:The product of this gene is an enzyme involved in lipid metabolism, and it encodes cytosolic acetoacetyl-CoA thiolase. This gene shows complementary overlapping with the 3-prime region of the TCP1 gene in both mouse and human. These genes are encoded on opposite strands of DNA, as well as in opposite transcriptional orientation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2014]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Acetyl-CoA acetyltransferase-2 deficiency     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q25.3, which is the long (q) arm of chromosome 6 at position 25.3Molecular Location: base pairs 159,761,957 to 159,779,112 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ACAT2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ACETYL-CoA ACETYLTRANSFERASE 2OMIM: ACETYL-CoA ACETYLTRANSFERASE-2 DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ACAT2  OMIM: ACETYL-CoA ACETYLTRANSFERASE 2OMIM: ACETYL-CoA ACETYLTRANSFERASE-2 DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MRPL18\t\n",
            "SLC22A1\t\n",
            "MAP3K4\t\n",
            "ATXN1\t   The ATXN1 gene provides instructions for making a protein called ataxin-1. This protein is found throughout the body, but its function is unknown. Within cells, ataxin-1 is located in the nucleus. Researchers believe that ataxin-1 may be involved in regulating various aspects of producing proteins, including the first stage of protein production (transcription) and processing RNA, a chemical cousin of DNA.One region of the ATXN1 gene contains a DNA segment known as a CAG trinucleotide repeat. This segment is made up of a series of three DNA building blocks (cytosine, adenine, and guanine) that appear multiple times in a row. Normally, the CAG segment is repeated 4 to 39 times within the gene.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Spinocerebellar ataxia type 1     Spinocerebellar ataxia type 1 (SCA1) is a condition characterized by progressive problems with movement. SCA1 results from a mutation in the ATXN1 gene known as a trinucleotide repeat expansion. This mutation increases the length of the repeated CAG segment in the ATXN1 gene. People with SCA1 have from 40 to more than 80 CAG repeats in most cells.The expanded CAG segment leads to the production of an abnormally long version of the ataxin-1 protein that folds into the wrong 3-dimensional shape. This abnormal protein clusters with other proteins to form clumps (aggregates) within the nucleus of cells. These aggregates prevent the ataxin-1 protein from functioning normally, which damages cells. For reasons that are unclear, aggregates of ataxin-1 are found only in the brain and spinal cord (central nervous system). Certain brain cells called Purkinje cells seem to be particularly sensitive to the accumulation of these aggregates. Purkinje cells are located in the part of the brain that coordinates movement (cerebellum) and are involved in chemical signaling between nerve cells (neurons). The accumulation of protein aggregates in the cells of the central nervous system, particularly in Purkinje cells, leads to cell death. Over time, the loss of these cells causes the movement problems characteristic of SCA1. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Spinocerebellar ataxia type 1 (SCA1) is a condition characterized by progressive problems with movement. SCA1 results from a mutation in the ATXN1 gene known as a trinucleotide repeat expansion. This mutation increases the length of the repeated CAG segment in the ATXN1 gene. People with SCA1 have from 40 to more than 80 CAG repeats in most cells.The expanded CAG segment leads to the production of an abnormally long version of the ataxin-1 protein that folds into the wrong 3-dimensional shape. This abnormal protein clusters with other proteins to form clumps (aggregates) within the nucleus of cells. These aggregates prevent the ataxin-1 protein from functioning normally, which damages cells. For reasons that are unclear, aggregates of ataxin-1 are found only in the brain and spinal cord (central nervous system). Certain brain cells called Purkinje cells seem to be particularly sensitive to the accumulation of these aggregates. Purkinje cells are located in the part of the brain that coordinates movement (cerebellum) and are involved in chemical signaling between nerve cells (neurons). The accumulation of protein aggregates in the cells of the central nervous system, particularly in Purkinje cells, leads to cell death. Over time, the loss of these cells causes the movement problems characteristic of SCA1. More About This Health Condition    Cytogenetic Location: 6p22.3, which is the short (p) arm of chromosome 6 at position 22.3Molecular Location: base pairs 16,299,112 to 16,761,490 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ataxin-1ATX1ATX1_HUMANSCA1     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Genes and Disease: Spinocerebellar AtaxiaWashington University, St. Louis: Neuromuscular Disease Center      Clinical Information from GeneReviews (1 link)     Spinocerebellar Ataxia Type 1      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATXN1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATAXIN 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Genes and Disease: Spinocerebellar AtaxiaWashington University, St. Louis: Neuromuscular Disease Center  Spinocerebellar Ataxia Type 1  Tests of ATXN1  PubMed  ATAXIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ATAXIN 1Kang S, Hong S. Molecular pathogenesis of spinocerebellar ataxia type 1 disease. Mol Cells. 2009 Jun 30;27(6):621-7. doi: 10.1007/s10059-009-0095-y. Epub 2009 Jun 22. Review. Citation on PubMedLim J, Crespo-Barreto J, Jafar-Nejad P, Bowman AB, Richman R, Hill DE, Orr HT, Zoghbi HY. Opposing effects of polyglutamine expansion on native protein complexes contribute to SCA1. Nature. 2008 Apr 10;452(7188):713-8. doi: 10.1038/nature06731. Epub 2008 Mar 12. Citation on PubMed or Free article on PubMed CentralZoghbi HY, Orr HT. Pathogenic mechanisms of a polyglutamine-mediated neurodegenerative disease, spinocerebellar ataxia type 1. J Biol Chem. 2009 Mar 20;284(12):7425-9. doi: 10.1074/jbc.R800041200. Epub 2008 Oct 28. Review. Citation on PubMed or Free article on PubMed Central \n",
            "TCP10L2\t\n",
            "TTLL2\t\n",
            "TCP10\t\n",
            "AFDN\t   From NCBI Gene:This gene encodes a multi-domain protein involved in signaling and organization of cell junctions during embryogenesis. It has also been identified as the fusion partner of acute lymphoblastic leukemia (ALL-1) gene, involved in acute myeloid leukemias with t(6;11)(q27;q23) translocation. Alternatively spliced transcript variants encoding different isoforms have been described for this gene, however, not all have been fully characterized.[provided by RefSeq, May 2011]From UniProt:Belongs to an adhesion system, probably together with the E-cadherin-catenin system, which plays a role in the organization of homotypic, interneuronal and heterotypic cell-cell adherens junctions (AJs). Nectin- and actin-filament-binding protein that connects nectin to the actin cytoskeleton.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving AFDN is associated with acute leukemias. Translocation t(6;11)(q27;q23) with KMT2A/MLL1. The result is a rogue activator protein.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q27, which is the long (q) arm of chromosome 6 at position 27Molecular Location: base pairs 167,826,872 to 167,972,023 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AF6l-afadinMLL-AF6MLLT4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AFDN      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: AFADIN      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AFDN  OMIM: AFADIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "DACT2\t\n",
            "TBP\t   The TBP gene provides instructions for making a protein called the TATA box binding protein.  This protein is active in cells and tissues throughout the body, where it plays an essential role in regulating the activity of most genes.The TATA box binding protein attaches (binds) to a particular sequence of DNA known as the TATA box. This sequence occurs in a regulatory region of DNA near the beginning of many genes. Once the protein is attached to the TATA box near a gene, it acts as a landmark to indicate where other enzymes should start reading the gene. The process of reading a gene's DNA and transferring the information to a similar molecule called mRNA is known as transcription.One region of the TBP gene contains a particular DNA segment known as a CAG/CAA trinucleotide repeat. This segment is made up of a series of three DNA building blocks (nucleotides) that appear multiple times in a row. Normally, the CAG/CAA segment is repeated 25 to 42 times within the gene.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Huntington disease-like syndrome     A particular type of mutation in the TBP gene has been found to cause a progressive brain disorder known as Huntington disease-like 4 (HDL4) or spinocerebellar ataxia type 17 (SCA17). The features of this disorder vary widely among affected individuals. The condition was first described as HDL4 in people whose signs and symptoms closely resembled those of Huntington disease, including uncontrolled movements, emotional problems, and loss of thinking ability. The disorder is now more commonly known as SCA17 because difficulty coordinating movements (ataxia) and other movement problems are the most frequent signs and symptoms.  It is unknown why some people with TBP mutations have a disorder resembling Huntington disease, while others have more prominent ataxia.The mutation associated with HDL4/SCA17 increases the size of the CAG/CAA trinucleotide repeat in the TBP gene. People with this condition have 43 to 66 CAG/CAA repeats. People with 43 to 48 CAG/CAA repeats may or may not have signs and symptoms, while people with 49 or more repeats almost always develop the disorder.An increased number of CAG/CAA repeats in the TBP gene leads to the production of an abnormally long version of the TATA box binding protein. The abnormal protein builds up in nerve cells (neurons) in the brain and disrupts the normal functions of these cells. The dysfunction and eventual death of neurons in certain areas of the brain underlie the signs and symptoms of HDL4/SCA17. Because the TBP gene is active throughout the body, it is unclear why the effects of a mutation in this gene are limited to the brain. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      A particular type of mutation in the TBP gene has been found to cause a progressive brain disorder known as Huntington disease-like 4 (HDL4) or spinocerebellar ataxia type 17 (SCA17). The features of this disorder vary widely among affected individuals. The condition was first described as HDL4 in people whose signs and symptoms closely resembled those of Huntington disease, including uncontrolled movements, emotional problems, and loss of thinking ability. The disorder is now more commonly known as SCA17 because difficulty coordinating movements (ataxia) and other movement problems are the most frequent signs and symptoms.  It is unknown why some people with TBP mutations have a disorder resembling Huntington disease, while others have more prominent ataxia.The mutation associated with HDL4/SCA17 increases the size of the CAG/CAA trinucleotide repeat in the TBP gene. People with this condition have 43 to 66 CAG/CAA repeats. People with 43 to 48 CAG/CAA repeats may or may not have signs and symptoms, while people with 49 or more repeats almost always develop the disorder.An increased number of CAG/CAA repeats in the TBP gene leads to the production of an abnormally long version of the TATA box binding protein. The abnormal protein builds up in nerve cells (neurons) in the brain and disrupts the normal functions of these cells. The dysfunction and eventual death of neurons in certain areas of the brain underlie the signs and symptoms of HDL4/SCA17. Because the TBP gene is active throughout the body, it is unclear why the effects of a mutation in this gene are limited to the brain. More About This Health Condition    Cytogenetic Location: 6q27, which is the long (q) arm of chromosome 6 at position 27Molecular Location: base pairs 170,554,333 to 170,572,870 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CCG1 ProteinCCGSCell Cycle Gene 1 ProteinDYT3 protein, humanGTF2DGTF2D1RNA Polymerase II TATA-Binding ProteinRNA Polymerase IIA 250kDSCA17TAF(II)250TAF1 RNA Polymerase II TATA Box Binding ProteinTAF2ATAFII250TATA-Binding ProteinTATA box binding proteinTATA-Box FactorTATA Sequence-Binding ProteinTBP_HUMANTF2DTFIIDTranscription Factor IIDTranscription Factor TBPTranscription Initiation Factor TFIID 250 kDa Subunit     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Biochemistry (fifth edition, 2002): The TATA-Box-Binding Protein Initiates the Assembly of the Active Transcription ComplexProtein Data Bank Molecule of the Month: TATA-Binding Protein      Clinical Information from GeneReviews (1 link)     Spinocerebellar Ataxia Type 17      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TBP      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TATA BOX-BINDING PROTEIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): The TATA-Box-Binding Protein Initiates the Assembly of the Active Transcription ComplexProtein Data Bank Molecule of the Month: TATA-Binding Protein  Spinocerebellar Ataxia Type 17  Tests of TBP  PubMed  TATA BOX-BINDING PROTEIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bauer P, Laccone F, Rolfs A, Wllner U, Bsch S, Peters H, Liebscher S, Scheible M, Epplen JT, Weber BH, Holinski-Feder E, Weirich-Schwaiger H, Morris-Rosendahl DJ, Andrich J, Riess O. Trinucleotide repeat expansion in SCA17/TBP in white patients with Huntington's disease-like phenotype. J Med Genet. 2004 Mar;41(3):230-2. Citation on PubMed or Free article on PubMed CentralGao R, Matsuura T, Coolbaugh M, Zhlke C, Nakamura K, Rasmussen A, Siciliano MJ, Ashizawa T, Lin X. Instability of expanded CAG/CAA repeats in spinocerebellar ataxia type 17. Eur J Hum Genet. 2008 Feb;16(2):215-22. Epub 2007 Nov 28. Citation on PubMedRasmussen A, De Biase I, Fragoso-Bentez M, Macas-Flores MA, Yescas P, Ochoa A, Ashizawa T, Alonso ME, Bidichandani SI. Anticipation and intergenerational repeat instability in spinocerebellar ataxia type 17. Ann Neurol. 2007 Jun;61(6):607-10. Citation on PubMedRolfs A, Koeppen AH, Bauer I, Bauer P, Buhlmann S, Topka H, Schls L, Riess O. Clinical features and neuropathology of autosomal dominant spinocerebellar ataxia (SCA17). Ann Neurol. 2003 Sep;54(3):367-75. Citation on PubMedSchneider SA, van de Warrenburg BP, Hughes TD, Davis M, Sweeney M, Wood N, Quinn NP, Bhatia KP. Phenotypic homogeneity of the Huntington disease-like presentation in a SCA17 family. Neurology. 2006 Nov 14;67(9):1701-3. Citation on PubMedZhlke C, Hellenbroich Y, Dalski A, Kononowa N, Hagenah J, Vieregge P, Riess O, Klein C, Schwinger E. Different types of repeat expansion in the TATA-binding protein gene are associated with a new form of inherited ataxia. Eur J Hum Genet. 2001 Mar;9(3):160-4. Citation on PubMedvan Roon-Mom WM, Reid SJ, Faull RL, Snell RG. TATA-binding protein in neurodegenerative disease. Neuroscience. 2005;133(4):863-72. Review. Citation on PubMed \n",
            "KIF13A\t\n",
            "BTN3A1\t\n",
            "BTN3A3\t\n",
            "PRSS16\t\n",
            "ZSCAN26\t\n",
            "ZNF311\t\n",
            "OR2J2\t\n",
            "OR12D2\t\n",
            "SERPINB6\t   From NCBI Gene:The protein encoded by this gene is a member of the serpin (serine proteinase inhibitor) superfamily, and ovalbumin(ov)-serpin subfamily. It was originally discovered as a placental thrombin inhibitor. The mouse homolog was found to be expressed in the hair cells of the inner ear. Mutations in this gene are associated with nonsyndromic progressive hearing loss, suggesting that this serpin plays an important role in the inner ear in the protection against leakage of lysosomal content during stress, and that loss of this protection results in cell death and sensorineural hearing loss. Alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Sep 2010]From UniProt:May be involved in the regulation of serine proteinases present in the brain or extravasated from the blood (By similarity). Inhibitor of cathepsin G, kallikrein-8 and thrombin. May play an important role in the inner ear in the protection against leakage of lysosomal content during stress and loss of this protection results in cell death and sensorineural hearing loss.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, autosomal recessive 91From UniProt:Deafness, autosomal recessive, 91 (DFNB91): A form of non-syndromic deafness characterized by progressive and age-dependent sensorineural hearing loss. Vestibular function is normal. [MIM:613453]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p25.2, which is the short (p) arm of chromosome 6 at position 25.2Molecular Location: base pairs 2,948,159 to 2,972,165 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CAPDFNB91MSTP057PI-6PI6PTISPI3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SERPINB6      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 91OMIM: SERPIN PEPTIDASE INHIBITOR, CLADE B (OVALBUMIN), MEMBER 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SERPINB6  OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 91OMIM: SERPIN PEPTIDASE INHIBITOR, CLADE B (OVALBUMIN), MEMBER 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TRIM31\t\n",
            "ABCF1\t\n",
            "C6orf15\t\n",
            "CDSN\t   From NCBI Gene:This gene encodes a protein found in corneodesmosomes, which localize to human epidermis and other cornified squamous epithelia. The encoded protein undergoes a series of cleavages during corneocyte maturation. This gene is highly polymorphic in human populations, and variation has been associated with skin diseases such as psoriasis, hypotrichosis and peeling skin syndrome. The gene is located in the major histocompatibility complex (MHC) class I region on chromosome 6. [provided by RefSeq, Dec 2014]From UniProt:Important for the epidermal barrier integrity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Peeling skin syndromeHypotrichosis 2From UniProt:Peeling skin syndrome 1 (PSS1): A genodermatosis characterized by generalized, continuous shedding of the outer layers of the epidermis. Two main PSS subtypes have been suggested. Patients with non-inflammatory PSS (type A) manifest white scaling, with painless and easy removal of the skin, irritation when in contact with water, dust and sand, and no history of erythema, pruritis or atopy. Inflammatory PSS (type B) is associated with generalized erythema, pruritus and atopy. It is an ichthyosiform erythroderma characterized by lifelong patchy peeling of the entire skin with onset at birth or shortly after. Several patients have been reported with high IgE levels. [MIM:270300]Hypotrichosis 2 (HYPT2): A condition characterized by the presence of less than the normal amount of hair. Affected individuals have normal hair in early childhood but experience progressive hair loss limited to the scalp beginning in the middle of the first decade and almost complete baldness by the third decade. Body hair, beard, eyebrows, axillary hair, teeth, and nails develop normally. [MIM:146520]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p21.33, which is the short (p) arm of chromosome 6 at position 21.33Molecular Location: base pairs 31,115,088 to 31,120,475 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HTSSHTSS1HYPT2PSSPSS1S     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CDSN      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: CORNEODESMOSINOMIM: HYPOTRICHOSIS 2OMIM: PEELING SKIN SYNDROME 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CDSN  OMIM: CORNEODESMOSINOMIM: HYPOTRICHOSIS 2OMIM: PEELING SKIN SYNDROME 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TCF19\t\n",
            "HLA-B\t   The HLA-B gene provides instructions for making a protein that plays a critical role in the immune system. HLA-B is part of a family of genes called the human leukocyte antigen (HLA) complex. The HLA complex helps the immune system distinguish the body's own proteins from proteins made by foreign invaders such as viruses and bacteria.HLA is the human version of the major histocompatibility complex (MHC), a gene family that occurs in many species. Genes in this complex are categorized into three basic groups:  class I, class II, and class III. In humans, the HLA-B gene and two related genes, HLA-A and HLA-C, are the main genes in MHC class I.MHC class I genes provide instructions for making proteins that are present on the surface of almost all cells. On the cell surface, these proteins are bound to protein fragments (peptides) that have been exported from within the cell.  MHC class I proteins display these peptides to the immune system. If the immune system recognizes the peptides as foreign (such as viral or bacterial peptides), it responds by triggering the infected cell to self-destruct.The HLA-B gene has many possible variations, allowing each person's immune system to react to a wide range of foreign invaders. Hundreds of versions (alleles) of the HLA-B gene are known, each of which is given a particular number (such as HLA-B27). Closely related alleles are categorized together; for example, more than 60 very similar alleles are subtypes of HLA-B27. These subtypes are designated as HLA-B*2701 to HLA-B*2763.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ankylosing spondylitis     Several variations of the HLA-B gene increase the risk of developing ankylosing spondylitis, particularly a version called HLA-B27. It is uncertain how this variation causes the increased risk. Researchers speculate that HLA-B27 may abnormally display peptides that trigger an immune reaction, resulting in the inflammatory process that causes arthritis.  Other research suggests that the joint inflammation characteristic of this disorder may result from improper folding of the HLA-B27 protein or the presence of abnormal forms of the protein on the cell surface. Although many people with ankylosing spondylitis have the HLA-B27 variation, most people with this version of the HLA-B gene never develop the disorder.  Additional genetic and environmental factors, many of which are unknown, affect the chances of developing ankylosing spondylitis and influence its progression. More About This Health Condition      Behet disease     Several versions of the HLA-B gene, particularly HLA-B51, are associated with an increased risk of developing Behet disease, a chronic inflammatory condition that affects many parts of the body. This association is strongest in people from Japan, the Middle East, and other parts of Asia. Researchers do not know how HLA-B51 increases the risk of this disorder. Although many people with Behet disease have the HLA-B51 variation, most people with this version of the HLA-B gene never develop the condition. It appears likely that other factors, such as viral or bacterial infections and changes in other genes, also influence the development of this complex disorder. More About This Health Condition      Juvenile idiopathic arthritis     Genetics Home Reference provides information about juvenile idiopathic arthritis. More About This Health Condition      Psoriatic arthritis     Genetics Home Reference provides information about psoriatic arthritis. More About This Health Condition      Rheumatoid arthritis     Genetics Home Reference provides information about rheumatoid arthritis. More About This Health Condition      Shingles     Genetics Home Reference provides information about shingles. More About This Health Condition      Stevens-Johnson syndrome/toxic epidermal necrolysis     Several variations of the HLA-B gene have been studied as risk factors for Stevens-Johnson syndrome/toxic epidermal necrolysis (SJS/TEN), a potentially life-threatening skin reaction most often triggered by medications. For example, the variation HLA-B*1502 increases the risk of SJS/TEN in people taking certain medications used to treat seizures, particularly a drug called carbamazepine. This version of the gene is most common among people of Han Chinese or southeast Asian descent. Another version of the gene, HLA-B*5801, increases the risk of SJS/TEN in people treated with allopurinol (a drug used to treat kidney stones and gout, which is a form of arthritis caused by a buildup of uric acid in the joints). This association has been confirmed in southeast Asians and in people of non-Asian ancestry, although HLA-B*5801 occurs less frequently in non-Asian populations.Studies suggest that the HLA-B gene variations associated with SJS/TEN cause the immune system to react abnormally to some medications. In a process that is not well understood, the triggering drug causes immune cells called cytotoxic T cells and natural killer (NK) cells to release a substance called granulysin. This substance destroys cells in the skin and mucous membranes, including the lining of the mouth and the airways. The death of these cells causes severe blistering and peeling that can have life-threatening effects.Most people who have variations in the HLA-B gene that are associated with an increased risk of SJS/TEN never develop the condition, even if they are exposed to drugs that can trigger it. Researchers believe that additional genetic and nongenetic factors, many of which are unknown, likely play a role in whether a particular individual develops SJS/TEN. More About This Health Condition      Other disorders     The HLA-B27 variant is associated with a group of inflammatory joint diseases related to ankylosing spondylitis. These conditions are known as spondyloarthropathies. Some of these disorders are associated with a common skin condition called psoriasis or with chronic disorders that cause inflammation of the intestinal walls (inflammatory bowel disease). One of the spondyloarthropathies, reactive arthritis, is typically triggered by bacterial infections of the gastrointestinal or genital tract. Following an infection, affected individuals may develop arthritis, back pain, and eye inflammation. Like ankylosing spondylitis, many factors probably contribute to the development of reactive arthritis and other spondyloarthropathies.Among people with human immunodeficiency virus (HIV) infection, a version of the HLA-B gene designated HLA-B*5701 increases the risk of an adverse reaction (hypersensitivity) to the drug abacavir. This medication slows the spread of the HIV-1 virus in the body. People with abacavir hypersensitivity often develop a fever, chills, rash, upset stomach, and other symptoms when treated with this drug.Several variations of the HLA-B gene appear to play a role in the progression of HIV infection to acquired immunodeficiency syndrome (AIDS). AIDS is a disease that damages the immune system, preventing it from effectively defending the body against infections. The signs and symptoms of AIDS may not appear until 10 or more years after infection with HIV. Studies suggest that people with HIV infection who have HLA-B27 or HLA-B57 tend to progress more slowly than usual to AIDS. On the other hand, researchers believe that HIV-positive individuals who have HLA-B35 tend to develop the signs and symptoms of AIDS more quickly than usual. Other factors also influence the progression of HIV infection to AIDS.Another version of the HLA-B gene, HLA-B53, has been shown to help protect against severe malaria, a disease caused by a parasite that is carried by mosquitoes. HLA-B53 is most common in West African populations, where malaria is a frequent cause of death in children. Studies suggest that this version of the HLA-B gene may help the immune system respond more effectively to the parasite that causes malaria.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several variations of the HLA-B gene increase the risk of developing ankylosing spondylitis, particularly a version called HLA-B27. It is uncertain how this variation causes the increased risk. Researchers speculate that HLA-B27 may abnormally display peptides that trigger an immune reaction, resulting in the inflammatory process that causes arthritis.  Other research suggests that the joint inflammation characteristic of this disorder may result from improper folding of the HLA-B27 protein or the presence of abnormal forms of the protein on the cell surface. Although many people with ankylosing spondylitis have the HLA-B27 variation, most people with this version of the HLA-B gene never develop the disorder.  Additional genetic and environmental factors, many of which are unknown, affect the chances of developing ankylosing spondylitis and influence its progression. More About This Health Condition  Several versions of the HLA-B gene, particularly HLA-B51, are associated with an increased risk of developing Behet disease, a chronic inflammatory condition that affects many parts of the body. This association is strongest in people from Japan, the Middle East, and other parts of Asia. Researchers do not know how HLA-B51 increases the risk of this disorder. Although many people with Behet disease have the HLA-B51 variation, most people with this version of the HLA-B gene never develop the condition. It appears likely that other factors, such as viral or bacterial infections and changes in other genes, also influence the development of this complex disorder. More About This Health Condition  Genetics Home Reference provides information about juvenile idiopathic arthritis. More About This Health Condition  Genetics Home Reference provides information about psoriatic arthritis. More About This Health Condition  Genetics Home Reference provides information about rheumatoid arthritis. More About This Health Condition  Genetics Home Reference provides information about shingles. More About This Health Condition  Several variations of the HLA-B gene have been studied as risk factors for Stevens-Johnson syndrome/toxic epidermal necrolysis (SJS/TEN), a potentially life-threatening skin reaction most often triggered by medications. For example, the variation HLA-B*1502 increases the risk of SJS/TEN in people taking certain medications used to treat seizures, particularly a drug called carbamazepine. This version of the gene is most common among people of Han Chinese or southeast Asian descent. Another version of the gene, HLA-B*5801, increases the risk of SJS/TEN in people treated with allopurinol (a drug used to treat kidney stones and gout, which is a form of arthritis caused by a buildup of uric acid in the joints). This association has been confirmed in southeast Asians and in people of non-Asian ancestry, although HLA-B*5801 occurs less frequently in non-Asian populations.Studies suggest that the HLA-B gene variations associated with SJS/TEN cause the immune system to react abnormally to some medications. In a process that is not well understood, the triggering drug causes immune cells called cytotoxic T cells and natural killer (NK) cells to release a substance called granulysin. This substance destroys cells in the skin and mucous membranes, including the lining of the mouth and the airways. The death of these cells causes severe blistering and peeling that can have life-threatening effects.Most people who have variations in the HLA-B gene that are associated with an increased risk of SJS/TEN never develop the condition, even if they are exposed to drugs that can trigger it. Researchers believe that additional genetic and nongenetic factors, many of which are unknown, likely play a role in whether a particular individual develops SJS/TEN. More About This Health Condition  The HLA-B27 variant is associated with a group of inflammatory joint diseases related to ankylosing spondylitis. These conditions are known as spondyloarthropathies. Some of these disorders are associated with a common skin condition called psoriasis or with chronic disorders that cause inflammation of the intestinal walls (inflammatory bowel disease). One of the spondyloarthropathies, reactive arthritis, is typically triggered by bacterial infections of the gastrointestinal or genital tract. Following an infection, affected individuals may develop arthritis, back pain, and eye inflammation. Like ankylosing spondylitis, many factors probably contribute to the development of reactive arthritis and other spondyloarthropathies.Among people with human immunodeficiency virus (HIV) infection, a version of the HLA-B gene designated HLA-B*5701 increases the risk of an adverse reaction (hypersensitivity) to the drug abacavir. This medication slows the spread of the HIV-1 virus in the body. People with abacavir hypersensitivity often develop a fever, chills, rash, upset stomach, and other symptoms when treated with this drug.Several variations of the HLA-B gene appear to play a role in the progression of HIV infection to acquired immunodeficiency syndrome (AIDS). AIDS is a disease that damages the immune system, preventing it from effectively defending the body against infections. The signs and symptoms of AIDS may not appear until 10 or more years after infection with HIV. Studies suggest that people with HIV infection who have HLA-B27 or HLA-B57 tend to progress more slowly than usual to AIDS. On the other hand, researchers believe that HIV-positive individuals who have HLA-B35 tend to develop the signs and symptoms of AIDS more quickly than usual. Other factors also influence the progression of HIV infection to AIDS.Another version of the HLA-B gene, HLA-B53, has been shown to help protect against severe malaria, a disease caused by a parasite that is carried by mosquitoes. HLA-B53 is most common in West African populations, where malaria is a frequent cause of death in children. Studies suggest that this version of the HLA-B gene may help the immune system respond more effectively to the parasite that causes malaria.    Cytogenetic Location: 6p21.33, which is the short (p) arm of chromosome 6 at position 21.33Molecular Location: base pairs 31,353,866 to 31,357,245 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        1B07_HUMANHLA class I histocompatibility antigen, B alpha chainleukocyte antigen BMHC class I HLA-B heavy chain     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Immunobiology (fifth edition, 2001): The Major Histocompatibility Complex and Its FunctionsInternational Society of Psychiatric Genetics: Genetic Testing and Psychiatric DisordersMedical Genetics Summaries: Allopurinol Therapy and HLA-B*5801 Genotype      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HLA-B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     HUMAN IMMUNODEFICIENCY VIRUS TYPE 1, SUSCEPTIBILITY TOMAJOR HISTOCOMPATIBILITY COMPLEX, CLASS I, B      Research Resources (7 links)     Anthony Nolan Research Institute: Nomenclature for Factors of the HLA SystemAtlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology (fifth edition, 2001): The Major Histocompatibility Complex and Its FunctionsInternational Society of Psychiatric Genetics: Genetic Testing and Psychiatric DisordersMedical Genetics Summaries: Allopurinol Therapy and HLA-B*5801 Genotype  Tests of HLA-B  PubMed  HUMAN IMMUNODEFICIENCY VIRUS TYPE 1, SUSCEPTIBILITY TOMAJOR HISTOCOMPATIBILITY COMPLEX, CLASS I, B  Anthony Nolan Research Institute: Nomenclature for Factors of the HLA SystemAtlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cheng CY, Su SC, Chen CH, Chen WL, Deng ST, Chung WH. HLA associations and clinical implications in T-cell mediated drug hypersensitivity reactions: an updated review. J Immunol Res. 2014;2014:565320. doi: 10.1155/2014/565320. Epub 2014 May 8. Review. Citation on PubMed or Free article on PubMed CentralChung WH, Hung SI. Recent advances in the genetics and immunology of Stevens-Johnson syndrome and toxic epidermal necrosis. J Dermatol Sci. 2012 Jun;66(3):190-6. doi: 10.1016/j.jdermsci.2012.04.002. Epub 2012 Apr 11. Review. Citation on PubMedColbert RA, Tran TM, Layh-Schmitt G. HLA-B27 misfolding and ankylosing spondylitis. Mol Immunol. 2014 Jan;57(1):44-51. doi: 10.1016/j.molimm.2013.07.013. Epub 2013 Aug 30. Review. Citation on PubMed or Free article on PubMed CentralKaur G, Mehra N. Genetic determinants of HIV-1 infection and progression to AIDS: immune response genes. Tissue Antigens. 2009 Nov;74(5):373-85. doi: 10.1111/j.1399-0039.2009.01337.x. Epub 2009 Sep 18. Review. Citation on PubMedLpez C, Saravia C, Gomez A, Hoebeke J, Patarroyo MA. Mechanisms of genetically-based resistance to malaria. Gene. 2010 Nov 1;467(1-2):1-12. doi: 10.1016/j.gene.2010.07.008. Epub 2010 Jul 22. Review. Citation on PubMedMeguro A, Inoko H, Ota M, Katsuyama Y, Oka A, Okada E, Yamakawa R, Yuasa T, Fujioka T, Ohno S, Bahram S, Mizuki N. Genetics of Behet disease inside and outside the MHC. Ann Rheum Dis. 2010 Apr;69(4):747-54. doi: 10.1136/ard.2009.108571. Epub 2009 Aug 13. Citation on PubMedReveille JD. An update on the contribution of the MHC to AS susceptibility. Clin Rheumatol. 2014 Jun;33(6):749-57. doi: 10.1007/s10067-014-2662-7. Epub 2014 May 18. Review. Citation on PubMed or Free article on PubMed CentralSorrentino R, Bckmann RA, Fiorillo MT. HLA-B27 and antigen presentation: at the crossroads between immune defense and autoimmunity. Mol Immunol. 2014 Jan;57(1):22-7. doi: 10.1016/j.molimm.2013.06.017. Epub 2013 Jul 31. Review. Citation on PubMedSousa-Pinto B, Pinto-Ramos J, Correia C, Gonalves-Costa G, Gomes L, Gil-Mata S, Arajo L, Delgado L. Pharmacogenetics of abacavir hypersensitivity: A systematic review and meta-analysis of the association with HLA-B*57:01. J Allergy Clin Immunol. 2015 Oct;136(4):1092-4.e3. doi: 10.1016/j.jaci.2015.03.019. Epub 2015 Apr 29. Review. Citation on PubMedWallace GR. HLA-B*51 the primary risk in Behet disease. Proc Natl Acad Sci U S A. 2014 Jun 17;111(24):8706-7. doi: 10.1073/pnas.1407307111. Epub 2014 May 29. Citation on PubMed or Free article on PubMed CentralYoung K, Frodsham A, Doumbo OK, Gupta S, Dolo A, Hu JT, Robson KJ, Crisanti A, Hill AV, Gilbert SC. Inverse associations of human leukocyte antigen and malaria parasite types in two West African populations. Infect Immun. 2005 Feb;73(2):953-5. Citation on PubMed or Free article on PubMed Centralde Menthon M, Lavalley MP, Maldini C, Guillevin L, Mahr A. HLA-B51/B5 and the risk of Behet's disease: a systematic review and meta-analysis of case-control genetic association studies. Arthritis Rheum. 2009 Oct 15;61(10):1287-96. doi: 10.1002/art.24642. Review. Citation on PubMed or Free article on PubMed Central \n",
            "MICA\t   From NCBI Gene:This gene encodes the highly polymorphic major histocompatability complex class I chain-related protein A. The protein product is expressed on the cell surface, although unlike canonical class I molecules it does not seem to associate with beta-2-microglobulin. It is a ligand for the NKG2-D type II integral membrane protein receptor. The protein functions as a stress-induced antigen that is broadly recognized by intestinal epithelial gamma delta T cells. Variations in this gene have been associated with susceptibility to psoriasis 1 and psoriatic arthritis, and the shedding of MICA-related antibodies and ligands is involved in the progression from monoclonal gammopathy of undetermined significance to multiple myeloma. Alternative splicing of this gene results in multiple transcript variants. [provided by RefSeq, Jan 2014]From UniProt:Seems to have no role in antigen presentation. Acts as a stress-induced self-antigen that is recognized by gamma delta T-cells. Ligand for the KLRK1/NKG2D receptor. Binding to KLRK1 leads to cell lysis.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autoimmune Addison diseaseAlopecia areataIdiopathic pulmonary fibrosisFrom UniProt:Psoriasis 1 (PSORS1): A common, chronic inflammatory disease of the skin with multifactorial etiology. It is characterized by red, scaly plaques usually found on the scalp, elbows and knees. These lesions are caused by abnormal keratinocyte proliferation and infiltration of inflammatory cells into the dermis and epidermis. [MIM:177900]Anti-MICA antibodies and ligand shedding are involved in the progression of monoclonal gammopathy of undetermined significance (MGUS)to multiple myeloma.Psoriatic arthritis (PSORAS): An inflammatory, seronegative arthritis associated with psoriasis. It is a heterogeneous disorder ranging from a mild, non-destructive disease to a severe, progressive, erosive arthropathy. Five types of psoriatic arthritis have been defined: asymmetrical oligoarthritis characterized by primary involvement of the small joints of the fingers or toes; asymmetrical arthritis which involves the joints of the extremities; symmetrical polyarthritis characterized by a rheumatoid like pattern that can involve hands, wrists, ankles, and feet; arthritis mutilans, which is a rare but deforming and destructive condition; arthritis of the sacroiliac joints and spine (psoriatic spondylitis). [MIM:607507]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p21.33, which is the short (p) arm of chromosome 6 at position 21.33Molecular Location: base pairs 31,399,784 to 31,415,315 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MIC-APERB11.1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MICA      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MAJOR HISTOCOMPATIBILITY COMPLEX CLASS I CHAIN-RELATED GENE A      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MICA  OMIM: MAJOR HISTOCOMPATIBILITY COMPLEX CLASS I CHAIN-RELATED GENE A  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HSPA1L\t\n",
            "CYP21A2\t   The CYP21A2 gene provides instructions for making an enzyme called 21-hydroxylase, which is part of the cytochrome P450 family of enzymes. Cytochrome P450 enzymes are involved in many processes in the body, such as assisting with reactions that break down drugs and helping to produce cholesterol, certain hormones, and fats (lipids).The 21-hydroxylase enzyme is found in the adrenal glands, which are located on top of the kidneys and produce a variety of hormones that regulate many essential functions in the body.  21-hydroxylase plays a role in producing hormones called cortisol and aldosterone.  Cortisol helps maintain blood sugar levels, protects the body from stress, and suppresses inflammation. Aldosterone is sometimes called the salt-retaining hormone because it regulates the amount of salt retained by the kidneys. The retention of salt affects fluid levels in the body and blood pressure.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           21-hydroxylase deficiency     More than 100 mutations in the CYP21A2 gene have been found to cause 21-hydroxylase deficiency.  Some of these mutations result from an exchange of genetic material between the CYP21A2 gene and a similar but nonfunctional piece of DNA called a pseudogene, which is located very close to the CYP21A2 gene on chromosome 6. This type of DNA exchange is called a gene conversion.  The genetic material from the pseudogene contains errors that, when introduced into the CYP21A2 gene, disrupt the way the gene's instructions are used to make a protein.  Other mutations that cause 21-hydroxylase deficiency change single protein building blocks (amino acids) in the 21-hydroxylase enzyme or delete or insert pieces of DNA in the CYP21A2 gene.Researchers have described three forms of 21-hydroxylase deficiency. Individuals with a form of the disorder called the salt-wasting type have CYP21A2 mutations that result in a completely nonfunctional enzyme.  People with the simple virilizing type of this condition have CYP21A2 gene mutations that allow the production of low levels of functional enzyme.  Individuals with the non-classic type of this disorder have CYP21A2 mutations that result in the production of reduced amounts of the enzyme, but more enzyme than any of the other types.  All types of 21-hydroxylase deficiency interfere with the production of cortisol and aldosterone.  The substances that are usually used to form these hormones instead build up in the adrenal glands and are converted to androgens, which are male sex hormones.  The excess production of androgens leads to abnormalities of sexual development in people with 21-hydroxylase deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 100 mutations in the CYP21A2 gene have been found to cause 21-hydroxylase deficiency.  Some of these mutations result from an exchange of genetic material between the CYP21A2 gene and a similar but nonfunctional piece of DNA called a pseudogene, which is located very close to the CYP21A2 gene on chromosome 6. This type of DNA exchange is called a gene conversion.  The genetic material from the pseudogene contains errors that, when introduced into the CYP21A2 gene, disrupt the way the gene's instructions are used to make a protein.  Other mutations that cause 21-hydroxylase deficiency change single protein building blocks (amino acids) in the 21-hydroxylase enzyme or delete or insert pieces of DNA in the CYP21A2 gene.Researchers have described three forms of 21-hydroxylase deficiency. Individuals with a form of the disorder called the salt-wasting type have CYP21A2 mutations that result in a completely nonfunctional enzyme.  People with the simple virilizing type of this condition have CYP21A2 gene mutations that allow the production of low levels of functional enzyme.  Individuals with the non-classic type of this disorder have CYP21A2 mutations that result in the production of reduced amounts of the enzyme, but more enzyme than any of the other types.  All types of 21-hydroxylase deficiency interfere with the production of cortisol and aldosterone.  The substances that are usually used to form these hormones instead build up in the adrenal glands and are converted to androgens, which are male sex hormones.  The excess production of androgens leads to abnormalities of sexual development in people with 21-hydroxylase deficiency. More About This Health Condition    Cytogenetic Location: 6p21.33, which is the short (p) arm of chromosome 6 at position 21.33Molecular Location: base pairs 32,038,316 to 32,041,670 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CA21HCAH1CP21A_HUMANCPS1CYP21CYP21BCytochrome P450 Family 21 Subfamily A Polypeptide 2Cytochrome P450 XXIcytochrome P450, family 21, subfamily A, polypeptide 2cytochrome P450, subfamily XXIA (steroid 21-hydroxylase, congenital adrenal hyperplasia), polypeptide 2Cytosteroid 21-MonooxygenaseP450c21Bsteroid 21-hydroxylasesteroid 21-monooxygenase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Endocrinology: An Integrated Approach (first edition, 2001): Congenital adrenal hyperplasia (CAH) - CYP21A2 deficiency      Clinical Information from GeneReviews (1 link)     21-Hydroxylase-Deficient Congenital Adrenal Hyperplasia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CYP21A2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ADRENAL HYPERPLASIA, CONGENITAL, DUE TO 21-HYDROXYLASE DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Endocrinology: An Integrated Approach (first edition, 2001): Congenital adrenal hyperplasia (CAH) - CYP21A2 deficiency  21-Hydroxylase-Deficient Congenital Adrenal Hyperplasia  Tests of CYP21A2  PubMed  ADRENAL HYPERPLASIA, CONGENITAL, DUE TO 21-HYDROXYLASE DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ADRENAL HYPERPLASIA, CONGENITAL, DUE TO 21-HYDROXYLASE DEFICIENCYHughes IA. Congenital adrenal hyperplasia: 21-hydroxylase deficiency in the newborn and during infancy. Semin Reprod Med. 2002 Aug;20(3):229-42. Review. Citation on PubMedHuynh T, McGown I, Cowley D, Nyunt O, Leong GM, Harris M, Cotterill AM. The clinical and biochemical spectrum of congenital adrenal hyperplasia secondary to 21-hydroxylase deficiency. Clin Biochem Rev. 2009 May;30(2):75-86. Citation on PubMed or Free article on PubMed CentralKeen-Kim D, Redman JB, Alanes RU, Eachus MM, Wilson RC, New MI, Nakamoto JM, Fenwick RG. Validation and clinical application of a locus-specific polymerase chain reaction- and minisequencing-based assay for congenital adrenal hyperplasia (21-hydroxylase deficiency). J Mol Diagn. 2005 May;7(2):236-46. Citation on PubMed or Free article on PubMed CentralKrone N, Riepe FG, Grtzinger J, Partsch CJ, Sippell WG. Functional characterization of two novel point mutations in the CYP21 gene causing simple virilizing forms of congenital adrenal hyperplasia due to 21-hydroxylase deficiency. J Clin Endocrinol Metab. 2005 Jan;90(1):445-54. Epub 2004 Oct 13. Citation on PubMedLee HH. Diversity of the CYP21P-like gene in CYP21 deficiency. DNA Cell Biol. 2005 Jan;24(1):1-9. Review. Citation on PubMedNew MI. An update of congenital adrenal hyperplasia. Ann N Y Acad Sci. 2004 Dec;1038:14-43. Citation on PubMedNimkarn S, Lin-Su K, New MI. Steroid 21 hydroxylase deficiency congenital adrenal hyperplasia. Endocrinol Metab Clin North Am. 2009 Dec;38(4):699-718. doi: 10.1016/j.ecl.2009.08.001. Review. Citation on PubMedTorres N, Mello MP, Germano CM, Elias LL, Moreira AC, Castro M. Phenotype and genotype correlation of the microconversion from the CYP21A1P to the CYP21A2 gene in congenital adrenal hyperplasia. Braz J Med Biol Res. 2003 Oct;36(10):1311-8. Epub 2003 Sep 16. Citation on PubMed \n",
            "TNXB\t   The TNXB gene provides instructions for making a protein called tenascin-X. This protein plays an important role in organizing and maintaining the structure of tissues that support the body's muscles, joints, organs, and skin (connective tissues). In particular, studies suggest that it helps to regulate the production and assembly of certain types of collagen. Collagens are a family of proteins that strengthen and support connective tissues throughout the body. Tenascin-X is also involved in regulating the structure and stability of elastic fibers, which provide flexibility and stretchiness (elasticity) to connective tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ehlers-Danlos syndrome     Mutations in the TNXB gene cause a very small percentage of all cases of a form of Ehlers-Danlos syndrome called the hypermobile type. Ehlers-Danlos syndrome is a group of disorders that affect the connective tissues that support the skin, bones, blood vessels, and many other organs and tissues. The hypermobile type is characterized by an unusually large range of joint movement (hypermobility). The mutations that cause this form of the disorder occur in one copy of the TNXB gene in each cell. These mutations reduce the amount of functional tenascin-X that cells produce, which decreases the ability of tenascin-X to interact with collagens and elastic fibers. These changes weaken connective tissues in many parts of the body, which results in the signs and symptoms of the hypermobile type of Ehlers-Danlos syndrome.Some people with a condition called benign joint hypermobility syndrome (BJHS) also make a reduced amount of tenascin-X protein, although no TNXB gene mutations have been identified in these individuals. This condition causes hypermobility and chronic joint pain. The signs and symptoms of benign joint hypermobility syndrome overlap significantly with those of the hypermobile type of Ehlers-Danlos syndrome. Studies suggest that they may be forms of the same condition.Some people with Ehlers-Danlos syndrome have mutations in two copies of the TNXB gene in each cell. These individuals have a form of the disorder that is often called the classical-like type. Its signs and symptoms are similar to the classical type of Ehlers-Danlos syndrome, including hypermobility and skin that is soft, highly stretchy (elastic), and fragile. However, affected individuals do not have the unusual scarring that is characteristic of that type. Mutations that occur in both copies of the TNXB gene prevent production of any tenascin-X protein. A loss of this protein severely disrupts the organization of collagen fibrils and elastic fibers, which significantly weakens connective tissues. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the TNXB gene cause a very small percentage of all cases of a form of Ehlers-Danlos syndrome called the hypermobile type. Ehlers-Danlos syndrome is a group of disorders that affect the connective tissues that support the skin, bones, blood vessels, and many other organs and tissues. The hypermobile type is characterized by an unusually large range of joint movement (hypermobility). The mutations that cause this form of the disorder occur in one copy of the TNXB gene in each cell. These mutations reduce the amount of functional tenascin-X that cells produce, which decreases the ability of tenascin-X to interact with collagens and elastic fibers. These changes weaken connective tissues in many parts of the body, which results in the signs and symptoms of the hypermobile type of Ehlers-Danlos syndrome.Some people with a condition called benign joint hypermobility syndrome (BJHS) also make a reduced amount of tenascin-X protein, although no TNXB gene mutations have been identified in these individuals. This condition causes hypermobility and chronic joint pain. The signs and symptoms of benign joint hypermobility syndrome overlap significantly with those of the hypermobile type of Ehlers-Danlos syndrome. Studies suggest that they may be forms of the same condition.Some people with Ehlers-Danlos syndrome have mutations in two copies of the TNXB gene in each cell. These individuals have a form of the disorder that is often called the classical-like type. Its signs and symptoms are similar to the classical type of Ehlers-Danlos syndrome, including hypermobility and skin that is soft, highly stretchy (elastic), and fragile. However, affected individuals do not have the unusual scarring that is characteristic of that type. Mutations that occur in both copies of the TNXB gene prevent production of any tenascin-X protein. A loss of this protein severely disrupts the organization of collagen fibrils and elastic fibers, which significantly weakens connective tissues. More About This Health Condition    Cytogenetic Location: 6p21.33-p21.32, which is the short (p) arm of chromosome 6 between positions 21.33 and 21.32Molecular Location: base pairs 32,041,155 to 32,109,374 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        hexabrachion-likeHXBLtenascin XB1tenascin XB2TENXTENX_HUMANTNXTNXB1TNXB2TNXBSXBXBS     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Hypermobile Ehlers-Danlos Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TNXB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TENASCIN XB      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarEhlers-Danlos Syndrome Variant DatabaseHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Hypermobile Ehlers-Danlos Syndrome  Tests of TNXB  PubMed  TENASCIN XB  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarEhlers-Danlos Syndrome Variant DatabaseHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Demirdas S, Dulfer E, Robert L, Kempers M, van Beek D, Micha D, van Engelen BG, Hamel B, Schalkwijk J, Loeys B, Maugeri A, Voermans NC. Recognizing the tenascin-X deficient type of Ehlers-Danlos syndrome: a cross-sectional study in 17 patients. Clin Genet. 2017 Mar;91(3):411-425. doi: 10.1111/cge.12853. Epub 2016 Nov 4. Citation on PubMedHendriks AG, Voermans NC, Schalkwijk J, Hamel BC, van Rossum MM. Well-defined clinical presentation of Ehlers-Danlos syndrome in patients with tenascin-X deficiency: a report of four cases. Clin Dysmorphol. 2012 Jan;21(1):15-8. doi: 10.1097/MCD.0b013e32834c4bb7. Citation on PubMedLindor NM, Bristow J. Tenascin-X deficiency in autosomal recessive Ehlers-Danlos syndrome. Am J Med Genet A. 2005 May 15;135(1):75-80. Citation on PubMedMalfait F, Francomano C, Byers P, Belmont J, Berglund B, Black J, Bloom L, Bowen JM, Brady AF, Burrows NP, Castori M, Cohen H, Colombi M, Demirdas S, De Backer J, De Paepe A, Fournel-Gigleux S, Frank M, Ghali N, Giunta C, Grahame R, Hakim A, Jeunemaitre X, Johnson D, Juul-Kristensen B, Kapferer-Seebacher I, Kazkaz H, Kosho T, Lavallee ME, Levy H, Mendoza-Londono R, Pepin M, Pope FM, Reinstein E, Robert L, Rohrbach M, Sanders L, Sobey GJ, Van Damme T, Vandersteen A, van Mourik C, Voermans N, Wheeldon N, Zschocke J, Tinkle B. The 2017 international classification of the Ehlers-Danlos syndromes. Am J Med Genet C Semin Med Genet. 2017 Mar;175(1):8-26. doi: 10.1002/ajmg.c.31552. Citation on PubMedSchalkwijk J, Zweers MC, Steijlen PM, Dean WB, Taylor G, van Vlijmen IM, van Haren B, Miller WL, Bristow J. A recessive form of the Ehlers-Danlos syndrome caused by tenascin-X deficiency. N Engl J Med. 2001 Oct 18;345(16):1167-75. Citation on PubMedTinkle B, Castori M, Berglund B, Cohen H, Grahame R, Kazkaz H, Levy H. Hypermobile Ehlers-Danlos syndrome (a.k.a. Ehlers-Danlos syndrome Type III and Ehlers-Danlos syndrome hypermobility type): Clinical description and natural history. Am J Med Genet C Semin Med Genet. 2017 Mar;175(1):48-69. doi: 10.1002/ajmg.c.31538. Epub 2017 Feb 1. Citation on PubMedValcourt U, Alcaraz LB, Exposito JY, Lethias C, Bartholin L. Tenascin-X: beyond the architectural function. Cell Adh Migr. 2015;9(1-2):154-65. doi: 10.4161/19336918.2014.994893. Review. Citation on PubMed or Free article on PubMed CentralVoermans NC, Jenniskens GJ, Hamel BC, Schalkwijk J, Guicheney P, van Engelen BG. Ehlers-Danlos syndrome due to tenascin-X deficiency: muscle weakness and contractures support overlap with collagen VI myopathies. Am J Med Genet A. 2007 Sep 15;143A(18):2215-9. Citation on PubMedZweers MC, Bristow J, Steijlen PM, Dean WB, Hamel BC, Otero M, Kucharekova M, Boezeman JB, Schalkwijk J. Haploinsufficiency of TNXB is associated with hypermobility type of Ehlers-Danlos syndrome. Am J Hum Genet. 2003 Jul;73(1):214-7. Citation on PubMed or Free article on PubMed CentralZweers MC, Dean WB, van Kuppevelt TH, Bristow J, Schalkwijk J. Elastic fiber abnormalities in hypermobility type Ehlers-Danlos syndrome patients with tenascin-X mutations. Clin Genet. 2005 Apr;67(4):330-4. Citation on PubMedZweers MC, Hakim AJ, Grahame R, Schalkwijk J. Joint hypermobility syndromes: the pathophysiologic role of tenascin-X gene defects. Arthritis Rheum. 2004 Sep;50(9):2742-9. Review. Citation on PubMed \n",
            "ATF6B\t\n",
            "NOTCH4\t   From NCBI Gene:This gene encodes a member of the NOTCH family of proteins. Members of this Type I transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple different domain types. Notch signaling is an evolutionarily conserved intercellular signaling pathway that regulates interactions between physically adjacent cells through binding of Notch family receptors to their cognate ligands. The encoded preproprotein is proteolytically processed in the trans-Golgi network to generate two polypeptide chains that heterodimerize to form the mature cell-surface receptor. This receptor may play a role in vascular, renal and hepatic development. Mutations in this gene may be associated with schizophrenia. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016]From UniProt:Functions as a receptor for membrane-bound ligands Jagged1, Jagged2 and Delta1 to regulate cell-fate determination. Upon ligand activation through the released notch intracellular domain (NICD) it forms a transcriptional activator complex with RBPJ/RBPSUH and activates genes of the enhancer of split locus. Affects the implementation of differentiation, proliferation and apoptotic programs. May regulate branching morphogenesis in the developing vascular system (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Alopecia areata     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 32,194,843 to 32,224,067 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        INT3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NOTCH4      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: NOTCH, DROSOPHILA, HOMOLOG OF, 4      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NOTCH4  OMIM: NOTCH, DROSOPHILA, HOMOLOG OF, 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HLA-DRA\t   From NCBI Gene:HLA-DRA is one of the HLA class II alpha chain paralogues. This class II molecule is a heterodimer consisting of an alpha and a beta chain, both anchored in the membrane. It plays a central role in the immune system by presenting peptides derived from extracellular proteins. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa and its gene contains 5 exons. Exon 1 encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, and exon 4 encodes the transmembrane domain and the cytoplasmic tail. DRA does not have polymorphisms in the peptide binding part and acts as the sole alpha chain for DRB1, DRB3, DRB4 and DRB5. [provided by RefSeq, Jul 2008]From UniProt:Binds peptides derived from antigens that access the endocytic route of antigen presenting cells (APC) and presents them on the cell surface for recognition by the CD4 T-cells. The peptide binding cleft accommodates peptides of 10-30 residues. The peptides presented by MHC class II molecules are generated mostly by degradation of proteins that access the endocytic route, where they are processed by lysosomal proteases and other hydrolases. Exogenous antigens that have been endocytosed by the APC are thus readily available for presentation via MHC II molecules, and for this reason this antigen presentation pathway is usually referred to as exogenous. As membrane proteins on their way to degradation in lysosomes as part of their normal turn-over are also contained in the endosomal/lysosomal compartments, exogenous antigens must compete with those derived from endogenous components. Autophagy is also a source of endogenous peptides, autophagosomes constitutively fuse with MHC class II loading compartments. In addition to APCs, other cells of the gastrointestinal tract, such as epithelial cells, express MHC class II molecules and CD74 and act as APCs, which is an unusual trait of the GI tract. To produce a MHC class II molecule that presents an antigen, three MHC class II molecules (heterodimers of an alpha and a beta chain) associate with a CD74 trimer in the ER to form a heterononamer. Soon after the entry of this complex into the endosomal/lysosomal system where antigen processing occurs, CD74 undergoes a sequential degradation by various proteases, including CTSS and CTSL, leaving a small fragment termed CLIP (class-II-associated invariant chain peptide). The removal of CLIP is facilitated by HLA-DM via direct binding to the alpha-beta-CLIP complex so that CLIP is released. HLA-DM stabilizes MHC class II molecules until primary high affinity antigenic peptides are bound. The MHC II molecule bound to a peptide is then transported to the cell membrane surface. In B-cells, the interaction between HLA-DM and MHC class II molecules is regulated by HLA-DO. Primary dendritic cells (DCs) also to express HLA-DO. Lysosomal microenvironment has been implicated in the regulation of antigen loading into MHC II molecules, increased acidification produces increased proteolysis and efficient peptide loading.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Alopecia areata     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 32,439,842 to 32,445,046 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HLA-DRA1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HLA-DRA      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR ALPHA      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HLA-DRA  OMIM: MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR ALPHA  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HLA-DRB5\t   From NCBI Gene:HLA-DRB5 belongs to the HLA class II beta chain paralogues. This class II molecule is a heterodimer consisting of an alpha (DRA) and a beta (DRB) chain, both anchored in the membrane. It plays a central role in the immune system by presenting peptides derived from extracellular proteins. Class II molecules are expressed in antigen presenting cells (APC: B lymphocytes, dendritic cells, macrophages). The beta chain is approximately 26-28 kDa and its gene contains 6 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, exon 4 encodes the transmembrane domain and exon 5 encodes the cytoplasmic tail. Within the DR molecule the beta chain contains all the polymorphisms specifying the peptide binding specificities. Typing for these polymorphisms is routinely done for bone marrow and kidney transplantation. DRB1 is expressed at a level five times higher than its paralogues DRB3, DRB4 and DRB5. The presence of DRB5 is linked with allelic variants of DRB1, otherwise it is omitted. There are 4 related pseudogenes: DRB2, DRB6, DRB7, DRB8 and DRB9. [provided by RefSeq, Jul 2008]From UniProt:Binds peptides derived from antigens that access the endocytic route of antigen presenting cells (APC) and presents them on the cell surface for recognition by the CD4 T-cells. The peptide binding cleft accommodates peptides of 10-30 residues. The peptides presented by MHC class II molecules are generated mostly by degradation of proteins that access the endocytic route, where they are processed by lysosomal proteases and other hydrolases. Exogenous antigens that have been endocytosed by the APC are thus readily available for presentation via MHC II molecules, and for this reason this antigen presentation pathway is usually referred to as exogenous. As membrane proteins on their way to degradation in lysosomes as part of their normal turn-over are also contained in the endosomal/lysosomal compartments, exogenous antigens must compete with those derived from endogenous components. Autophagy is also a source of endogenous peptides, autophagosomes constitutively fuse with MHC class II loading compartments. In addition to APCs, other cells of the gastrointestinal tract, such as epithelial cells, express MHC class II molecules and CD74 and act as APCs, which is an unusual trait of the GI tract. To produce a MHC class II molecule that presents an antigen, three MHC class II molecules (heterodimers of an alpha and a beta chain) associate with a CD74 trimer in the ER to form a heterononamer. Soon after the entry of this complex into the endosomal/lysosomal system where antigen processing occurs, CD74 undergoes a sequential degradation by various proteases, including CTSS and CTSL, leaving a small fragment termed CLIP (class-II-associated invariant chain peptide). The removal of CLIP is facilitated by HLA-DM via direct binding to the alpha-beta-CLIP complex so that CLIP is released. HLA-DM stabilizes MHC class II molecules until primary high affinity antigenic peptides are bound. The MHC II molecule bound to a peptide is then transported to the cell membrane surface. In B-cells, the interaction between HLA-DM and MHC class II molecules is regulated by HLA-DO. Primary dendritic cells (DCs) also to express HLA-DO. Lysosomal microenvironment has been implicated in the regulation of antigen loading into MHC II molecules, increased acidification produces increased proteolysis and efficient peptide loading.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Alopecia areata     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 32,517,343 to 32,530,316 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of HLA-DRB5      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR BETA-5      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of HLA-DRB5  OMIM: MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR BETA-5  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "HLA-DRB1\t   The HLA-DRB1 gene provides instructions for making a protein that plays a critical role in the immune system. The HLA-DRB1 gene is part of a family of genes called the human leukocyte antigen (HLA) complex. The HLA complex helps the immune system distinguish the body's own proteins from proteins made by foreign invaders such as viruses and bacteria.The HLA complex is the human version of the major histocompatibility complex (MHC), a gene family that occurs in many species. The HLA-DRB1 gene belongs to a group of MHC genes called MHC class II. MHC class II genes provide instructions for making proteins that are present on the surface of certain immune system cells. These proteins attach to protein fragments (peptides) outside the cell. MHC class II proteins display these peptides to the immune system. If the immune system recognizes the peptides as foreign (such as viral or bacterial peptides), it triggers a response to attack the invading viruses or bacteria.The protein produced from the HLA-DRB1 gene, called the beta chain, attaches (binds) to another protein called the alpha chain, which is produced from the HLA-DRA gene. Together, they form a functional protein complex called the HLA-DR antigen-binding heterodimer. This complex displays foreign peptides to the immune system to trigger the body's immune response.Each MHC class II gene has many possible variations, allowing the immune system to react to a wide range of foreign invaders. Researchers have identified hundreds of different versions (alleles) of the HLA-DRB1 gene, each of which is given a particular number (such as HLA-DRB1*04:01).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Alopecia areata     Genetics Home Reference provides information about alopecia areata. More About This Health Condition      Autoimmune Addison disease     Certain variations in the HLA-DRB1 gene have been linked to an increased risk of developing an autoimmune disorder called autoimmune Addison disease. Autoimmune disorders occur when the immune system malfunctions and attacks the body's tissues and organs. In autoimmune Addison disease, the immune system attacks the adrenal glands, which are small hormone-producing glands located on top of each kidney. Loss of hormones produced by the adrenal glands leads to the features of the condition, which include extreme tiredness (fatigue), nausea, low blood pressure (hypotension), and abnormally dark areas of skin (hyperpigmentation), especially in regions that experience a lot of friction such as the armpits, elbows, and knuckles. A particular HLA-DRB1 gene variant called HLA-DRB1*04:04 is the most well-known risk factor for autoimmune Addison disease.Normally, the immune system responds only to proteins made by foreign invaders, not to the body's own proteins. In autoimmune Addison disease, however, an immune response is triggered by a normal adrenal gland protein; in about 85 percent of people with autoimmune Addison disease, the protein is 21-hydroxylase. 21-hydroxylase is found in the adrenal glands where it plays a key role in producing a variety of hormones that regulate many essential functions in the body. The prolonged immune attack triggered by 21-hydroxylase damages the adrenal glands (specifically the outer layers of the glands known, collectively, as the adrenal cortex), preventing hormone production. A shortage of adrenal hormones (adrenal insufficiency) disrupts several normal functions in the body, leading to the diverse features of autoimmune Addison disease. It is not clear how HLA-DRB1*04:04 and other HLA-DRB1 variations are involved in the inappropriate immune response that causes autoimmune Addison disease. More About This Health Condition      Crohn disease     Genetics Home Reference provides information about Crohn disease. More About This Health Condition      Graves disease     Genetics Home Reference provides information about Graves disease. More About This Health Condition      Hashimoto thyroiditis     Genetics Home Reference provides information about Hashimoto thyroiditis. More About This Health Condition      Idiopathic inflammatory myopathy     Genetics Home Reference provides information about idiopathic inflammatory myopathy. More About This Health Condition      Juvenile idiopathic arthritis     Genetics Home Reference provides information about juvenile idiopathic arthritis. More About This Health Condition      Lyme disease     Genetics Home Reference provides information about Lyme disease. More About This Health Condition      Multiple sclerosis     Variations in the HLA-DRB1 gene have been associated with an increased risk of developing multiple sclerosis. This condition affects the brain and spinal cord (central nervous system), causing muscle weakness, poor coordination, numbness, and a variety of other health problems. One variant of this gene, called HLA-DRB1*15:01, is the most strongly linked genetic factor for the risk of multiple sclerosis.Because the HLA-DRB1 gene is involved in the immune system, changes in it might be related to the autoimmune response and inflammation that damage nerves and the protective coating surrounding them (the myelin sheath), leading to the signs and symptoms of multiple sclerosis. However, it is unclear exactly what role HLA-DRB1 gene variants play in development of multiple sclerosis. A combination of genetic and environmental factors is likely involved in this condition. More About This Health Condition      Narcolepsy     Genetics Home Reference provides information about narcolepsy. More About This Health Condition      Psoriatic arthritis     Genetics Home Reference provides information about psoriatic arthritis. More About This Health Condition      Rheumatoid arthritis     Several common variations of the HLA-DRB1 gene are associated with a person's risk of developing rheumatoid arthritis. This disease causes chronic abnormal inflammation that primarily affects the joints. HLA-DRB1 is one of several genes in the HLA complex that have been associated with rheumatoid arthritis; variations of this gene are the most significant known genetic risk factor for the disease.The HLA-DRB1 gene variations associated with an increased risk of rheumatoid arthritis affect single protein building blocks (amino acids) in the beta chain. These changes occur near the antigen-recognizing binding groove, which is the part of the protein that attaches (binds) to viral or bacterial peptides. This binding triggers the immune response that attacks foreign invaders. Although the mechanism by which HLA-DRB1 gene variations increase the risk of rheumatoid arthritis is unclear, researchers suspect it is related to changes in peptide binding that stimulate an abnormal immune response. However, many other genetic and environmental factors also contribute to a person's overall risk of developing rheumatoid arthritis.A few variations of the HLA-DRB1 gene appear to decrease the risk of developing rheumatoid arthritis. It is unclear why these particular changes may be protective. More About This Health Condition      Rosacea     Genetics Home Reference provides information about rosacea. More About This Health Condition      Type 1 diabetes     Combinations of variations in the HLA-DRB1 gene and other HLA genes affect the risk of type 1 diabetes. Type 1 diabetes is characterized by high blood sugar levels resulting from a shortage of the hormone insulin and is caused by autoimmune damage to insulin-producing cells in the pancreas.Type 1 diabetes risk is most increased by two specific combinations of variations of the HLA-DRB1 gene and other HLA genes called HLA-DQA1 and HLA-DQB1. Combinations of HLA gene variants are called HLA haplotypes. One haplotype, written as DRB1*03:01-DQA1*05:01-DQB1*02, is called DR3. The other haplotype, written as DRB1*04:01/02/04/05/08-DQA1*03:01-DQB1*02, is called DR4. People at highest risk of developing type 1 diabetes have one copy of the DR3 haplotype and one copy of the DR4 haplotype in each cell. Other HLA haplotypes only mildly increase the risk of type 1 diabetes, while some haplotypes seem to protect against developing this condition. Variations in other genes and environmental factors are also thought to affect the risk of this complex disorder. More About This Health Condition      Autoimmune disorders     Normal variations of the HLA-DRB1 gene have been associated with many other autoimmune disorders, including pemphigus, sarcoidosis, and others listed on this page. Pemphigus is a condition that causes severe blistering of the skin and mucous membranes (such as the moist lining of the mouth). Sarcoidosis is a disorder in which inflammation occurs in many organs and tissues of the body.It is unclear how different versions of the HLA-DRB1 gene influence the risk of developing autoimmune disorders. These disorders typically result from a combination of multiple environmental and genetic factors. Changes in other HLA and non-HLA genes, some of which remain unknown, also likely contribute to the risk of developing these complex conditions.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetics Home Reference provides information about alopecia areata. More About This Health Condition  Certain variations in the HLA-DRB1 gene have been linked to an increased risk of developing an autoimmune disorder called autoimmune Addison disease. Autoimmune disorders occur when the immune system malfunctions and attacks the body's tissues and organs. In autoimmune Addison disease, the immune system attacks the adrenal glands, which are small hormone-producing glands located on top of each kidney. Loss of hormones produced by the adrenal glands leads to the features of the condition, which include extreme tiredness (fatigue), nausea, low blood pressure (hypotension), and abnormally dark areas of skin (hyperpigmentation), especially in regions that experience a lot of friction such as the armpits, elbows, and knuckles. A particular HLA-DRB1 gene variant called HLA-DRB1*04:04 is the most well-known risk factor for autoimmune Addison disease.Normally, the immune system responds only to proteins made by foreign invaders, not to the body's own proteins. In autoimmune Addison disease, however, an immune response is triggered by a normal adrenal gland protein; in about 85 percent of people with autoimmune Addison disease, the protein is 21-hydroxylase. 21-hydroxylase is found in the adrenal glands where it plays a key role in producing a variety of hormones that regulate many essential functions in the body. The prolonged immune attack triggered by 21-hydroxylase damages the adrenal glands (specifically the outer layers of the glands known, collectively, as the adrenal cortex), preventing hormone production. A shortage of adrenal hormones (adrenal insufficiency) disrupts several normal functions in the body, leading to the diverse features of autoimmune Addison disease. It is not clear how HLA-DRB1*04:04 and other HLA-DRB1 variations are involved in the inappropriate immune response that causes autoimmune Addison disease. More About This Health Condition  Genetics Home Reference provides information about Crohn disease. More About This Health Condition  Genetics Home Reference provides information about Graves disease. More About This Health Condition  Genetics Home Reference provides information about Hashimoto thyroiditis. More About This Health Condition  Genetics Home Reference provides information about idiopathic inflammatory myopathy. More About This Health Condition  Genetics Home Reference provides information about juvenile idiopathic arthritis. More About This Health Condition  Genetics Home Reference provides information about Lyme disease. More About This Health Condition  Variations in the HLA-DRB1 gene have been associated with an increased risk of developing multiple sclerosis. This condition affects the brain and spinal cord (central nervous system), causing muscle weakness, poor coordination, numbness, and a variety of other health problems. One variant of this gene, called HLA-DRB1*15:01, is the most strongly linked genetic factor for the risk of multiple sclerosis.Because the HLA-DRB1 gene is involved in the immune system, changes in it might be related to the autoimmune response and inflammation that damage nerves and the protective coating surrounding them (the myelin sheath), leading to the signs and symptoms of multiple sclerosis. However, it is unclear exactly what role HLA-DRB1 gene variants play in development of multiple sclerosis. A combination of genetic and environmental factors is likely involved in this condition. More About This Health Condition  Genetics Home Reference provides information about narcolepsy. More About This Health Condition  Genetics Home Reference provides information about psoriatic arthritis. More About This Health Condition  Several common variations of the HLA-DRB1 gene are associated with a person's risk of developing rheumatoid arthritis. This disease causes chronic abnormal inflammation that primarily affects the joints. HLA-DRB1 is one of several genes in the HLA complex that have been associated with rheumatoid arthritis; variations of this gene are the most significant known genetic risk factor for the disease.The HLA-DRB1 gene variations associated with an increased risk of rheumatoid arthritis affect single protein building blocks (amino acids) in the beta chain. These changes occur near the antigen-recognizing binding groove, which is the part of the protein that attaches (binds) to viral or bacterial peptides. This binding triggers the immune response that attacks foreign invaders. Although the mechanism by which HLA-DRB1 gene variations increase the risk of rheumatoid arthritis is unclear, researchers suspect it is related to changes in peptide binding that stimulate an abnormal immune response. However, many other genetic and environmental factors also contribute to a person's overall risk of developing rheumatoid arthritis.A few variations of the HLA-DRB1 gene appear to decrease the risk of developing rheumatoid arthritis. It is unclear why these particular changes may be protective. More About This Health Condition  Genetics Home Reference provides information about rosacea. More About This Health Condition  Combinations of variations in the HLA-DRB1 gene and other HLA genes affect the risk of type 1 diabetes. Type 1 diabetes is characterized by high blood sugar levels resulting from a shortage of the hormone insulin and is caused by autoimmune damage to insulin-producing cells in the pancreas.Type 1 diabetes risk is most increased by two specific combinations of variations of the HLA-DRB1 gene and other HLA genes called HLA-DQA1 and HLA-DQB1. Combinations of HLA gene variants are called HLA haplotypes. One haplotype, written as DRB1*03:01-DQA1*05:01-DQB1*02, is called DR3. The other haplotype, written as DRB1*04:01/02/04/05/08-DQA1*03:01-DQB1*02, is called DR4. People at highest risk of developing type 1 diabetes have one copy of the DR3 haplotype and one copy of the DR4 haplotype in each cell. Other HLA haplotypes only mildly increase the risk of type 1 diabetes, while some haplotypes seem to protect against developing this condition. Variations in other genes and environmental factors are also thought to affect the risk of this complex disorder. More About This Health Condition  Normal variations of the HLA-DRB1 gene have been associated with many other autoimmune disorders, including pemphigus, sarcoidosis, and others listed on this page. Pemphigus is a condition that causes severe blistering of the skin and mucous membranes (such as the moist lining of the mouth). Sarcoidosis is a disorder in which inflammation occurs in many organs and tissues of the body.It is unclear how different versions of the HLA-DRB1 gene influence the risk of developing autoimmune disorders. These disorders typically result from a combination of multiple environmental and genetic factors. Changes in other HLA and non-HLA genes, some of which remain unknown, also likely contribute to the risk of developing these complex conditions.    Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 32,578,769 to 32,589,836 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        2B1F_HUMAN2B1G_HUMAN2B13_HUMANDRB1DRw10DW2.2/DR2.2HLA class II histocompatibility antigen, DR-1 beta chainHLA-DR1BHLA-DRBhuman leucocyte antigen DRB1lymphocyte antigen DRB1major histocompatibility complex, class II, DR beta 1 precursorMHC class II antigenMHC class II HLA-DR beta 1 chainMHC class II HLA-DR-beta cell surface glycoproteinMHC class II HLA-DRw10-betaSS1     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Anthony Nolan Research Institute: Nomenclature for Factors of the HLA SystemImmunobiology (fifth edition, 2001): The Major Histocompatibility Complex and its FunctionsThe Merck Manual for Health Professionals: Human Leukocyte Antigen (HLA) System      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HLA-DRB1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR BETA-1SARCOIDOSIS, SUSCEPTIBILITY TO, 1      Research Resources (8 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt: 2B1F_HUMANUniProt: 2B1G_HUMANUniProt: 2B13_HUMAN      Anthony Nolan Research Institute: Nomenclature for Factors of the HLA SystemImmunobiology (fifth edition, 2001): The Major Histocompatibility Complex and its FunctionsThe Merck Manual for Health Professionals: Human Leukocyte Antigen (HLA) System  Tests of HLA-DRB1  PubMed  MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR BETA-1SARCOIDOSIS, SUSCEPTIBILITY TO, 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt: 2B1F_HUMANUniProt: 2B1G_HUMANUniProt: 2B13_HUMAN  Alcina A, Abad-Grau Mdel M, Fedetz M, Izquierdo G, Lucas M, Fernndez O, Ndagire D, Catal-Rabasa A, Ruiz A, Gayn J, Delgado C, Arnal C, Matesanz F. Multiple sclerosis risk variant HLA-DRB1*1501 associates with high expression of DRB1 gene in different human populations. PLoS One. 2012;7(1):e29819. doi: 10.1371/journal.pone.0029819. Epub 2012 Jan 13. Citation on PubMed or Free article on PubMed CentralAly TA, Ide A, Jahromi MM, Barker JM, Fernando MS, Babu SR, Yu L, Miao D, Erlich HA, Fain PR, Barriga KJ, Norris JM, Rewers MJ, Eisenbarth GS. Extreme genetic risk for type 1A diabetes. Proc Natl Acad Sci U S A. 2006 Sep 19;103(38):14074-9. Epub 2006 Sep 11. Citation on PubMed or Free article on PubMed CentralChinoy H, Lamb JA, Ollier WE, Cooper RG. An update on the immunogenetics of idiopathic inflammatory myopathies: major histocompatibility complex and beyond. Curr Opin Rheumatol. 2009 Nov;21(6):588-93. doi: 10.1097/BOR.0b013e3283315a22. Review. Citation on PubMedGombos Z, Hermann R, Kiviniemi M, Nejentsev S, Reimand K, Fadeyev V, Peterson P, Uibo R, Ilonen J. Analysis of extended human leukocyte antigen haplotype association with Addison's disease in three populations. Eur J Endocrinol. 2007 Dec;157(6):757-61. Citation on PubMedHor H, Kutalik Z, Dauvilliers Y, Valsesia A, Lammers GJ, Donjacour CE, Iranzo A, Santamaria J, Peraita Adrados R, Vicario JL, Overeem S, Arnulf I, Theodorou I, Jennum P, Knudsen S, Bassetti C, Mathis J, Lecendreux M, Mayer G, Geisler P, Benet A, Petit B, Pfister C, Brki JV, Didelot G, Billiard M, Ercilla G, Verduijn W, Claas FH, Vollenweider P, Waeber G, Waterworth DM, Mooser V, Heinzer R, Beckmann JS, Bergmann S, Tafti M. Genome-wide association study identifies new HLA class II haplotypes strongly protective against narcolepsy. Nat Genet. 2010 Sep;42(9):786-9. doi: 10.1038/ng.647. Epub 2010 Aug 15. Erratum in: Nat Genet. 2011 Apr;43(4):388. Vollenwider, Peter [corrected to Vollenweider, Peter]. Citation on PubMedOMIM: MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DR BETA-1Noble JA, Valdes AM. Genetics of the HLA region in the prediction of type 1 diabetes. Curr Diab Rep. 2011 Dec;11(6):533-42. doi: 10.1007/s11892-011-0223-x. Review. Citation on PubMed or Free article on PubMed CentralPrahalad S, Glass DN. A comprehensive review of the genetics of juvenile idiopathic arthritis. Pediatr Rheumatol Online J. 2008 Jul 21;6:11. doi: 10.1186/1546-0096-6-11. Citation on PubMed or Free article on PubMed CentralRaychaudhuri S, Sandor C, Stahl EA, Freudenberg J, Lee HS, Jia X, Alfredsson L, Padyukov L, Klareskog L, Worthington J, Siminovitch KA, Bae SC, Plenge RM, Gregersen PK, de Bakker PI. Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis. Nat Genet. 2012 Jan 29;44(3):291-6. doi: 10.1038/ng.1076. Citation on PubMed or Free article on PubMed CentralRottembourg D, Deal C, Lambert M, Mallone R, Carel JC, Lacroix A, Caillat-Zucman S, le Deist F. 21-Hydroxylase epitopes are targeted by CD8 T cells in autoimmune Addison's disease. J Autoimmun. 2010 Dec;35(4):309-15. doi: 10.1016/j.jaut.2010.07.001. Epub 2010 Aug 3. Citation on PubMedSkinningsrud B, Lie BA, Lavant E, Carlson JA, Erlich H, Akselsen HE, Gervin K, Wolff AB, Erichsen MM, Lvs K, Husebye ES, Undlien DE. Multiple loci in the HLA complex are associated with Addison's disease. J Clin Endocrinol Metab. 2011 Oct;96(10):E1703-8. doi: 10.1210/jc.2011-0645. Epub 2011 Aug 3. Citation on PubMedSteck AK, Rewers MJ. Genetics of type 1 diabetes. Clin Chem. 2011 Feb;57(2):176-85. doi: 10.1373/clinchem.2010.148221. Epub 2011 Jan 4. Review. Citation on PubMed or Free article on PubMed CentralViatte S, Plant D, Raychaudhuri S. Genetics and epigenetics of rheumatoid arthritis. Nat Rev Rheumatol. 2013 Mar;9(3):141-53. doi: 10.1038/nrrheum.2012.237. Epub 2013 Feb 5. Review. Citation on PubMed or Free article on PubMed CentralYu L, Brewer KW, Gates S, Wu A, Wang T, Babu SR, Gottlieb PA, Freed BM, Noble J, Erlich HA, Rewers MJ, Eisenbarth GS. DRB1*04 and DQ alleles: expression of 21-hydroxylase autoantibodies and risk of progression to Addison's disease. J Clin Endocrinol Metab. 1999 Jan;84(1):328-35. Citation on PubMed \n",
            "HLA-DQA1\t   The HLA-DQA1 gene provides instructions for making a protein that plays a critical role in the immune system. The HLA-DQA1 gene is part of a family of genes called the human leukocyte antigen (HLA) complex. The HLA complex helps the immune system distinguish the body's own proteins from proteins made by foreign invaders such as viruses and bacteria.The HLA complex is the human version of the major histocompatibility complex (MHC), a gene family that occurs in many species. The HLA-DQA1 gene belongs to a group of MHC genes called MHC class II. MHC class II genes provide instructions for making proteins that are present on the surface of certain immune system cells. These proteins attach to protein fragments (peptides) outside the cell. MHC class II proteins display these peptides to the immune system. If the immune system recognizes the peptides as foreign (such as viral or bacterial peptides), it triggers a response to attack the invading viruses or bacteria.The protein produced from the HLA-DQA1 gene attaches (binds) to the protein produced from another MHC class II gene, HLA-DQB1. Together, they form a functional protein complex called an antigen-binding DQ heterodimer. This complex displays foreign peptides to the immune system to trigger the body's immune response.Each MHC class II gene has many possible variations, allowing the immune system to react to a wide range of foreign invaders. Researchers have identified hundreds of different versions (alleles) of the HLA-DQA1 gene, each of which is given a particular number (such as HLA-DQA1*05:01).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Celiac disease     At least two specific combinations of HLA gene variants (HLA haplotypes) have been found to increase the risk of developing celiac disease, a disorder in which inflammation damages the intestinal tract and other organs and tissues. One of these haplotypes, known as DQ2, is composed of the protein produced from HLA-DQA1 gene variants known as HLA-DQA1*05:01 or HLA-DQA1*05:05 bound to the protein produced from HLA-DQB1 gene variants known as HLA-DQB1*02:01 or HLA-DQB1*02:02. The other haplotype, known as DQ8, is composed of the protein produced from  HLA-DQA1 gene variants known as HLA-DQA1*03:01 or HLA-DQA1*03:02 bound to the protein produced from the HLA-DQB1 gene variant known as HLA-DQB1*03:02.The DQ2 and DQ8 haplotypes, which may occur separately or together, seem to increase the risk of an inappropriate immune response to the protein gluten, which is found in wheat, rye, and barley. This immune system malfunction results in the damage to the body's organs and tissues that occurs in celiac disease.  However, the DQ2 and DQ8 haplotypes are also found in 30 percent of the general population, and only 3 percent of individuals with these haplotypes develop celiac disease. More About This Health Condition      Alopecia areata     Genetics Home Reference provides information about alopecia areata. More About This Health Condition      Autoimmune Addison disease     Genetics Home Reference provides information about autoimmune Addison disease. More About This Health Condition      Idiopathic inflammatory myopathy     Genetics Home Reference provides information about idiopathic inflammatory myopathy. More About This Health Condition      Juvenile idiopathic arthritis     Genetics Home Reference provides information about juvenile idiopathic arthritis. More About This Health Condition      Narcolepsy     Genetics Home Reference provides information about narcolepsy. More About This Health Condition      Rosacea     Genetics Home Reference provides information about rosacea. More About This Health Condition      Type 1 diabetes     Combinations of variations in the HLA-DQA1 gene and other HLA genes affect the risk of type 1 diabetes. Type 1 diabetes is characterized by high blood sugar levels resulting from a shortage of the hormone insulin and is caused by autoimmune damage to insulin-producing cells in the pancreas.Type 1 diabetes risk is most increased by two HLA haplotypes involving variations of the HLA-DQA1 and HLA-DQB1 genes and another HLA gene called HLA-DRB1. One haplotype, written as DRB1*03:01-DQA1*05:01-DQB1*02, is called DR3. The other haplotype, written as DRB1*04:01/02/04/05/08-DQA1*03:01-DQB1*02, is called DR4. People at highest risk of developing type 1 diabetes have one copy of the DR3 haplotype and one copy of the DR4 haplotype in each cell. Other HLA haplotypes only mildly increase the risk of type 1 diabetes, while some haplotypes seem to protect against developing this condition. Variations in other genes and environmental factors are also thought to affect the risk of this complex disorder. More About This Health Condition      Autoimmune disorders     Certain normal variations of the HLA-DQA1 gene have been associated with increased risk of autoimmune disorders, which occur when the immune system malfunctions and attacks the body's own tissues and organs. It is unclear how different versions of the HLA-DQA1 gene influence the risk of developing autoimmune disorders. These conditions are thought to result from a combination of multiple environmental and genetic factors. Changes in other HLA and non-HLA genes, some of which remain unknown, also likely contribute to the risk of developing these complex conditions.      Other disorders     Normal variations in the HLA-DQA1 gene can affect the body's ability to recognize and react to foreign invaders (pathogens). For example, variations of this gene have been shown to increase or decrease a person's chance of getting infections such as hepatitis B and leprosy or may affect the severity of illness if infection occurs.A particular variant of the HLA-DQA1 gene known as HLA-DQA1*02:01 increases the risk of liver damage in women with advanced breast cancer treated with a drug called lapatinib. Researchers suggest that the variant may increase immune system sensitivity to the drug, resulting in inflammation that damages the liver.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two specific combinations of HLA gene variants (HLA haplotypes) have been found to increase the risk of developing celiac disease, a disorder in which inflammation damages the intestinal tract and other organs and tissues. One of these haplotypes, known as DQ2, is composed of the protein produced from HLA-DQA1 gene variants known as HLA-DQA1*05:01 or HLA-DQA1*05:05 bound to the protein produced from HLA-DQB1 gene variants known as HLA-DQB1*02:01 or HLA-DQB1*02:02. The other haplotype, known as DQ8, is composed of the protein produced from  HLA-DQA1 gene variants known as HLA-DQA1*03:01 or HLA-DQA1*03:02 bound to the protein produced from the HLA-DQB1 gene variant known as HLA-DQB1*03:02.The DQ2 and DQ8 haplotypes, which may occur separately or together, seem to increase the risk of an inappropriate immune response to the protein gluten, which is found in wheat, rye, and barley. This immune system malfunction results in the damage to the body's organs and tissues that occurs in celiac disease.  However, the DQ2 and DQ8 haplotypes are also found in 30 percent of the general population, and only 3 percent of individuals with these haplotypes develop celiac disease. More About This Health Condition  Genetics Home Reference provides information about alopecia areata. More About This Health Condition  Genetics Home Reference provides information about autoimmune Addison disease. More About This Health Condition  Genetics Home Reference provides information about idiopathic inflammatory myopathy. More About This Health Condition  Genetics Home Reference provides information about juvenile idiopathic arthritis. More About This Health Condition  Genetics Home Reference provides information about narcolepsy. More About This Health Condition  Genetics Home Reference provides information about rosacea. More About This Health Condition  Combinations of variations in the HLA-DQA1 gene and other HLA genes affect the risk of type 1 diabetes. Type 1 diabetes is characterized by high blood sugar levels resulting from a shortage of the hormone insulin and is caused by autoimmune damage to insulin-producing cells in the pancreas.Type 1 diabetes risk is most increased by two HLA haplotypes involving variations of the HLA-DQA1 and HLA-DQB1 genes and another HLA gene called HLA-DRB1. One haplotype, written as DRB1*03:01-DQA1*05:01-DQB1*02, is called DR3. The other haplotype, written as DRB1*04:01/02/04/05/08-DQA1*03:01-DQB1*02, is called DR4. People at highest risk of developing type 1 diabetes have one copy of the DR3 haplotype and one copy of the DR4 haplotype in each cell. Other HLA haplotypes only mildly increase the risk of type 1 diabetes, while some haplotypes seem to protect against developing this condition. Variations in other genes and environmental factors are also thought to affect the risk of this complex disorder. More About This Health Condition  Certain normal variations of the HLA-DQA1 gene have been associated with increased risk of autoimmune disorders, which occur when the immune system malfunctions and attacks the body's own tissues and organs. It is unclear how different versions of the HLA-DQA1 gene influence the risk of developing autoimmune disorders. These conditions are thought to result from a combination of multiple environmental and genetic factors. Changes in other HLA and non-HLA genes, some of which remain unknown, also likely contribute to the risk of developing these complex conditions.  Normal variations in the HLA-DQA1 gene can affect the body's ability to recognize and react to foreign invaders (pathogens). For example, variations of this gene have been shown to increase or decrease a person's chance of getting infections such as hepatitis B and leprosy or may affect the severity of illness if infection occurs.A particular variant of the HLA-DQA1 gene known as HLA-DQA1*02:01 increases the risk of liver damage in women with advanced breast cancer treated with a drug called lapatinib. Researchers suggest that the variant may increase immune system sensitivity to the drug, resulting in inflammation that damages the liver.    Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 32,637,406 to 32,654,846 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DC-1 alpha chainDC-alphaDQ-A1FLJ27088FLJ27328GSEHLA class II histocompatibility antigen, DQ alpha 1 chainHLA class II histocompatibility antigen, DQ alpha 1 chain precursorHLA class II histocompatibility antigen, DQ(W3) alpha chainHLA-DCAHLA-DQAleucocyte antigen DQA1leukocyte antigen alpha chainMGC149527MHC class II antigenMHC class II DQA1MHC class II HLA-D alpha glycoproteinMHC class II HLA-DQ-alpha-1MHC class II surface glycoproteinMHC HLA-DQ alpha     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Immunobiology (fifth edition, 2001): The Major Histocompatibility Complex and Its FunctionsNational Center for Biotechnology Information (2004): The Genetic Landscape of DiabetesThe Merck Manual for Health Professionals: Human Leukocyte Antigen (HLA) System      Clinical Information from GeneReviews (1 link)     Celiac Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of HLA-DQA1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DQ ALPHA-1      Research Resources (7 links)     Anthony Nolan Research Institute: Nomenclature for Factors of the HLA SystemAtlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology (fifth edition, 2001): The Major Histocompatibility Complex and Its FunctionsNational Center for Biotechnology Information (2004): The Genetic Landscape of DiabetesThe Merck Manual for Health Professionals: Human Leukocyte Antigen (HLA) System  Celiac Disease  Tests of HLA-DQA1  PubMed  MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DQ ALPHA-1  Anthony Nolan Research Institute: Nomenclature for Factors of the HLA SystemAtlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Abadie V, Sollid LM, Barreiro LB, Jabri B. Integration of genetic and immunological insights into a model of celiac disease pathogenesis. Annu Rev Immunol. 2011;29:493-525. doi: 10.1146/annurev-immunol-040210-092915. Review. Citation on PubMedLipton RB, Drum M, Greeley SA, Danielson KK, Bell GI, Hagopian WA. HLA-DQ haplotypes differ by ethnicity in patients with childhood-onset diabetes. Pediatr Diabetes. 2011 Jun;12(4 Pt 2):388-95. doi: 10.1111/j.1399-5448.2010.00712.x. Epub 2011 Mar 21. Citation on PubMed or Free article on PubMed CentralOMIM: MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DQ ALPHA-1Noble JA, Valdes AM. Genetics of the HLA region in the prediction of type 1 diabetes. Curr Diab Rep. 2011 Dec;11(6):533-42. doi: 10.1007/s11892-011-0223-x. Review. Citation on PubMed or Free article on PubMed CentralSpraggs CF, Budde LR, Briley LP, Bing N, Cox CJ, King KS, Whittaker JC, Mooser VE, Preston AJ, Stein SH, Cardon LR. HLA-DQA1*02:01 is a major risk factor for lapatinib-induced hepatotoxicity in women with advanced breast cancer. J Clin Oncol. 2011 Feb 20;29(6):667-73. doi: 10.1200/JCO.2010.31.3197. Epub 2011 Jan 18. Citation on PubMedTaylor AK, Lebwohl B, Snyder CL, Green PHR. Celiac Disease. 2008 Jul 3 [updated 2015 Sep 17]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1727/ Citation on PubMedTjon JM, van Bergen J, Koning F. Celiac disease: how complicated can it get? Immunogenetics. 2010 Oct;62(10):641-51. doi: 10.1007/s00251-010-0465-9. Epub 2010 Jul 27. Review. Citation on PubMed or Free article on PubMed CentralUibo R, Tian Z, Gershwin ME. Celiac disease: a model disease for gene-environment interaction. Cell Mol Immunol. 2011 Mar;8(2):93-5. doi: 10.1038/cmi.2010.62. Epub 2011 Feb 14. Review. Citation on PubMed or Free article on PubMed CentralXun YH, Guo JC, Shi WZ, Shi JP, Liu CL. [Association between HLA-DQA1 gene polymorphism and the outcomes of hepatitis B virus infection]. Zhonghua Shi Yan He Lin Chuang Bing Du Xue Za Zhi. 2009 Dec;23(6):430-3. Chinese. Citation on PubMedda Silva SA, Mazini PS, Reis PG, Sell AM, Tsuneto LT, Peixoto PR, Visentainer JE. HLA-DR and HLA-DQ alleles in patients from the south of Brazil: markers for leprosy susceptibility and resistance. BMC Infect Dis. 2009 Aug 22;9:134. doi: 10.1186/1471-2334-9-134. Citation on PubMed or Free article on PubMed Central \n",
            "TAP2\t   The TAP2 gene provides instructions for making a protein that plays an important role in the immune system. The TAP2 protein assembles with another protein called TAP1 (produced from the TAP1 gene) to form a protein complex called the transporter associated with antigen processing (TAP) complex. This complex, which is found in the membrane of a cell structure called the endoplasmic reticulum, moves (transports) protein fragments (peptides) from foreign invaders into the endoplasmic reticulum. There, the peptides are attached to major histocompatibility complex (MHC) class I proteins. The peptide-bound MHC class I proteins are then moved to the surface of the cell so that specialized immune system cells can interact with them. When these immune system cells recognize the peptides as harmful, they launch an immune response to get rid of the foreign invaders.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Bare lymphocyte syndrome type I     At least seven mutations in the TAP2 gene have been found to cause bare lymphocyte syndrome type I (BLS I). This immune system disorder often causes recurrent bacterial infections in the respiratory tract and open sores (ulcers) on the skin, although some people with BLS I have no symptoms of the condition. TAP2 gene mutations involved in BLS I prevent production of functional TAP2 protein. Absence of functional TAP2 impairs the formation of the TAP complex, without which peptides from foreign invaders cannot be transported into the endoplasmic reticulum and attached to MHC class I proteins. Consequently, MHC class I proteins are broken down, which results in a shortage of these proteins on the surface of cells. A lack of MHC class I proteins impairs the body's immune response to bacteria, leading to recurrent bacterial infections. Researchers are unsure why people with BLS I do not also get viral infections, but they suspect that other immune processes are able to recognize and fight viruses. It is also not clear how TAP2 gene mutations are involved in the development of skin ulcers. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least seven mutations in the TAP2 gene have been found to cause bare lymphocyte syndrome type I (BLS I). This immune system disorder often causes recurrent bacterial infections in the respiratory tract and open sores (ulcers) on the skin, although some people with BLS I have no symptoms of the condition. TAP2 gene mutations involved in BLS I prevent production of functional TAP2 protein. Absence of functional TAP2 impairs the formation of the TAP complex, without which peptides from foreign invaders cannot be transported into the endoplasmic reticulum and attached to MHC class I proteins. Consequently, MHC class I proteins are broken down, which results in a shortage of these proteins on the surface of cells. A lack of MHC class I proteins impairs the body's immune response to bacteria, leading to recurrent bacterial infections. Researchers are unsure why people with BLS I do not also get viral infections, but they suspect that other immune processes are able to recognize and fight viruses. It is also not clear how TAP2 gene mutations are involved in the development of skin ulcers. More About This Health Condition    Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 32,821,833 to 32,838,823 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABC transporter, MHC 2ABC18ABCB3APT2ATP-binding cassette, sub-family B (MDR/TAP), member 3D6S217Epeptide supply factor 2peptide transporter involved in antigen processing 2peptide transporter PSF2PSF-2PSF2RING11transporter 2, ABC (ATP binding cassette)transporter 2, ATP-binding cassette, sub-family B (MDR/TAP)     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Immunobiology: The Immune System in Health and Disease (fifth edition, 2001): The Major Histocompatibility Complex and Its FunctionsMolecular Biology of the Cell (fourth edition, 2002): T Cells and MHC Proteins      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TAP2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSPORTER, ATP-BINDING CASSETTE, MAJOR HISTOCOMPATIBILITY COMPLEX, 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology: The Immune System in Health and Disease (fifth edition, 2001): The Major Histocompatibility Complex and Its FunctionsMolecular Biology of the Cell (fourth edition, 2002): T Cells and MHC Proteins  Tests of TAP2  PubMed  TRANSPORTER, ATP-BINDING CASSETTE, MAJOR HISTOCOMPATIBILITY COMPLEX, 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Eggensperger S, Tamp R. The transporter associated with antigen processing: a key player in adaptive immunity. Biol Chem. 2015 Sep;396(9-10):1059-72. doi: 10.1515/hsz-2014-0320. Review. Citation on PubMedLeonhardt RM, Keusekotten K, Bekpen C, Knittler MR. Critical role for the tapasin-docking site of TAP2 in the functional integrity of the MHC class I-peptide-loading complex. J Immunol. 2005 Oct 15;175(8):5104-14. Citation on PubMedNeefjes JJ, Momburg F, Hmmerling GJ. Selective and ATP-dependent translocation of peptides by the MHC-encoded transporter. Science. 1993 Aug 6;261(5122):769-71. Erratum in: Science 1994 Apr 1;264(5155):16. Citation on PubMedParcej D, Tamp R. ABC proteins in antigen translocation and viral inhibition. Nat Chem Biol. 2010 Aug;6(8):572-80. doi: 10.1038/nchembio.410. Review. Erratum in: Nat Chem Biol. 2010 Oct;6(10):782. Citation on PubMedOMIM: TRANSPORTER, ATP-BINDING CASSETTE, MAJOR HISTOCOMPATIBILITY COMPLEX, 2Zimmer J, Andrs E, Donato L, Hanau D, Hentges F, de la Salle H. Clinical and immunological aspects of HLA class I deficiency. QJM. 2005 Oct;98(10):719-27. Epub 2005 Aug 8. Review. Citation on PubMed \n",
            "HLA-DMA\t\n",
            "COL11A2\t   The COL11A2 gene provides instructions for making a component of type XI collagen called the pro-alpha2(XI) chain. Collagens are molecules that provide structure and strength to the connective tissues that support the body's muscles, joints, organs, and skin. Type XI collagen is normally found in cartilage, a tough but flexible tissue that makes up much of the skeleton during early development. Most cartilage is later converted to bone, except for the cartilage that continues to cover and protect the ends of bones and is present in the nose and external ears. Type XI collagen made with the pro-alpha2(XI) chain is also part of the inner ear and the nucleus pulposus, which is the center portion of the discs between the bones of the spine (vertebrae).Collagens begin as rope-like procollagen molecules that are each made up of three chains. The pro-alpha2(XI) chain combines with two other collagen chains, pro-alpha1(XI) and pro-alpha1(II), to form a triple-stranded procollagen molecule. Then the ropelike procollagen is processed by enzymes to create mature collagen. Mature collagen molecules arrange themselves into long, thin fibrils that form stable interactions (cross-links) with one another in the spaces between cells (the extracellular matrix). The cross-links result in the formation of very strong type XI collagen fibers.Type XI collagen also helps maintain the spacing and width (diameter) of another type of collagen molecule, type II collagen. Type II collagen is an important component of mature cartilage. The arrangement and size of type II collagen fibrils is essential for the normal structure of these tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Fibrochondrogenesis     At least two mutations in the COL11A2 gene have been identified in people with fibrochondrogenesis type 2, a disorder of bone growth characterized by severe skeletal abnormalities and hearing loss. Infants with fibrochondrogenesis type 2 have a very narrow chest that prevents the lungs from developing normally. Most infants with this condition are stillborn or die shortly after birth from respiratory failure, although some have lived into childhood.The COL11A2 gene mutations that cause fibrochondrogenesis type 2 lead to the production of an abnormal version of the pro-alpha2(XI) chain. When this abnormal chain is incorporated into collagen molecules, it creates defective type XI collagen. The abnormal collagen weakens connective tissues, impairing the formation of bones throughout the skeleton and causing changes in the inner ear that lead to hearing problems. More About This Health Condition      Nonsyndromic hearing loss     Mutations in the COL11A2 gene have been identified in people with nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene can cause two forms of nonsyndromic hearing loss: DFNA13 and DFNB53.DFNA13 is inherited in an autosomal dominant pattern, which means only one mutated copy of the COL11A2 gene in each cell is sufficient to cause the condition. This type of hearing loss begins in childhood or adolescence. It is classified as postlingual because it starts after a child learns to speak.At least two COL11A2 gene mutations have been identified in people with DFNA13. Both of these mutations change a single protein building block (amino acid) in the pro-alpha2(XI) chain of type XI collagen. These mutations are thought to change the structure of type XI collagen, which plays an important role in the structure and function of the inner ear.DFNB53 is inherited in an autosomal recessive pattern, which means both copies of the COL11A2 gene are mutated in each cell. It is characterized by profound hearing loss that is present before a child learns to speak (prelingual).At least three mutations in the COL11A2 gene have been found to cause DFNB53. Each of these mutations changes a single amino acid in the pro-alpha2(XI) chain of type XI collagen. Studies suggest that the altered protein causes hearing loss by changing the structure of type XI collagen and impairing its ability to interact with other proteins. More About This Health Condition      Otospondylomegaepiphyseal dysplasia     At least 14 mutations in the COL11A2 gene have been found to cause otospondylomegaepiphyseal dysplasia (OSMED), a disorder characterized by skeletal abnormalities, distinctive facial features, and severe hearing loss. These signs and symptoms are similar to those of Weissenbacher-Zweymller syndrome (described below) and to a form of Stickler syndrome classified as type III. In some cases, it can be difficult to tell these conditions apart. Some researchers believe they represent a single disorder with a range of signs and symptoms.Most of the reported COL11A2 gene mutations that cause OSMED lead to the production of an abnormally short version of the pro-alpha2(XI) chain that is probably not incorporated into type XI collagen molecules. The defective collagen impairs the normal development of several tissues, including bones and the inner ear. More About This Health Condition      Stickler syndrome     Genetics Home Reference provides information about Stickler syndrome. More About This Health Condition      Weissenbacher-Zweymller syndrome     At least one mutation in the COL11A2 gene has been associated with Weissenbacher-Zweymller syndrome, a disorder of bone growth characterized by skeletal abnormalities, hearing loss, and distinctive facial features. These signs and symptoms are similar to those of OSMED (described above) and to a form of Stickler syndrome classified as type III.Like the other mutations that cause OSMED, this mutation leads to defective type XI collagen. The abnormal collagen impairs the normal development of several tissues, including bones and the inner ear. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least two mutations in the COL11A2 gene have been identified in people with fibrochondrogenesis type 2, a disorder of bone growth characterized by severe skeletal abnormalities and hearing loss. Infants with fibrochondrogenesis type 2 have a very narrow chest that prevents the lungs from developing normally. Most infants with this condition are stillborn or die shortly after birth from respiratory failure, although some have lived into childhood.The COL11A2 gene mutations that cause fibrochondrogenesis type 2 lead to the production of an abnormal version of the pro-alpha2(XI) chain. When this abnormal chain is incorporated into collagen molecules, it creates defective type XI collagen. The abnormal collagen weakens connective tissues, impairing the formation of bones throughout the skeleton and causing changes in the inner ear that lead to hearing problems. More About This Health Condition  Mutations in the COL11A2 gene have been identified in people with nonsyndromic hearing loss, which is loss of hearing that is not associated with other signs and symptoms. Mutations in this gene can cause two forms of nonsyndromic hearing loss: DFNA13 and DFNB53.DFNA13 is inherited in an autosomal dominant pattern, which means only one mutated copy of the COL11A2 gene in each cell is sufficient to cause the condition. This type of hearing loss begins in childhood or adolescence. It is classified as postlingual because it starts after a child learns to speak.At least two COL11A2 gene mutations have been identified in people with DFNA13. Both of these mutations change a single protein building block (amino acid) in the pro-alpha2(XI) chain of type XI collagen. These mutations are thought to change the structure of type XI collagen, which plays an important role in the structure and function of the inner ear.DFNB53 is inherited in an autosomal recessive pattern, which means both copies of the COL11A2 gene are mutated in each cell. It is characterized by profound hearing loss that is present before a child learns to speak (prelingual).At least three mutations in the COL11A2 gene have been found to cause DFNB53. Each of these mutations changes a single amino acid in the pro-alpha2(XI) chain of type XI collagen. Studies suggest that the altered protein causes hearing loss by changing the structure of type XI collagen and impairing its ability to interact with other proteins. More About This Health Condition  At least 14 mutations in the COL11A2 gene have been found to cause otospondylomegaepiphyseal dysplasia (OSMED), a disorder characterized by skeletal abnormalities, distinctive facial features, and severe hearing loss. These signs and symptoms are similar to those of Weissenbacher-Zweymller syndrome (described below) and to a form of Stickler syndrome classified as type III. In some cases, it can be difficult to tell these conditions apart. Some researchers believe they represent a single disorder with a range of signs and symptoms.Most of the reported COL11A2 gene mutations that cause OSMED lead to the production of an abnormally short version of the pro-alpha2(XI) chain that is probably not incorporated into type XI collagen molecules. The defective collagen impairs the normal development of several tissues, including bones and the inner ear. More About This Health Condition  Genetics Home Reference provides information about Stickler syndrome. More About This Health Condition  At least one mutation in the COL11A2 gene has been associated with Weissenbacher-Zweymller syndrome, a disorder of bone growth characterized by skeletal abnormalities, hearing loss, and distinctive facial features. These signs and symptoms are similar to those of OSMED (described above) and to a form of Stickler syndrome classified as type III.Like the other mutations that cause OSMED, this mutation leads to defective type XI collagen. The abnormal collagen impairs the normal development of several tissues, including bones and the inner ear. More About This Health Condition    Cytogenetic Location: 6p21.32, which is the short (p) arm of chromosome 6 at position 21.32Molecular Location: base pairs 33,162,692 to 33,193,009 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        collagen type XI alpha 2collagen, type XI, alpha 2HKE5PARPSTL3     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Biology of the Cell (fourth edition, 2002): Collagens Are the Major Proteins of the Extracellular MatrixMolecular Biology of the Cell (fourth edition, 2002): Some Types of Collagen and Their PropertiesMolecular Cell Biology (fourth edition, 2000): Collagen: The Fibrous Proteins of the Matrix      Clinical Information from GeneReviews (2 links)     Hereditary Hearing Loss and Deafness OverviewStickler Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL11A2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     COLLAGEN, TYPE XI, ALPHA-2OSTEOARTHRITIS SUSCEPTIBILITY 1      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHereditary Hearing Loss HomepageHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Collagens Are the Major Proteins of the Extracellular MatrixMolecular Biology of the Cell (fourth edition, 2002): Some Types of Collagen and Their PropertiesMolecular Cell Biology (fourth edition, 2000): Collagen: The Fibrous Proteins of the Matrix  Hereditary Hearing Loss and Deafness OverviewStickler Syndrome  Tests of COL11A2  PubMed  COLLAGEN, TYPE XI, ALPHA-2OSTEOARTHRITIS SUSCEPTIBILITY 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHereditary Hearing Loss HomepageHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: COLLAGEN, TYPE XI, ALPHA-2Chakchouk I, Grati M, Bademci G, Bensaid M, Ma Q, Chakroun A, Foster J 2nd, Yan D, Duman D, Diaz-Horta O, Ghorbel A, Mittal R, Farooq A, Tekin M, Masmoudi S, Liu XZ. Novel mutations confirm that COL11A2 is responsible for autosomal recessive non-syndromic hearing loss DFNB53. Mol Genet Genomics. 2015 Aug;290(4):1327-34. doi: 10.1007/s00438-015-0995-9. Epub 2015 Jan 30. Citation on PubMed or Free article on PubMed CentralChen W, Kahrizi K, Meyer NC, Riazalhosseini Y, Van Camp G, Najmabadi H, Smith RJ. Mutation of COL11A2 causes autosomal recessive non-syndromic hearing loss at the DFNB53 locus. J Med Genet. 2005 Oct;42(10):e61. Epub 2005 Jul 20. Citation on PubMed or Free article on PubMed CentralDe Leenheer EM, Kunst HH, McGuirt WT, Prasad SD, Brown MR, Huygen PL, Smith RJ, Cremers CW. Autosomal dominant inherited hearing impairment caused by a missense mutation in COL11A2 (DFNA13). Arch Otolaryngol Head Neck Surg. 2001 Jan;127(1):13-7. Citation on PubMedHarel T, Rabinowitz R, Hendler N, Galil A, Flusser H, Chemke J, Gradstein L, Lifshitz T, Ofir R, Elbedour K, Birk OS. COL11A2 mutation associated with autosomal recessive Weissenbacher-Zweymuller syndrome: molecular and clinical overlap with otospondylomegaepiphyseal dysplasia (OSMED). Am J Med Genet A. 2005 Jan 1;132A(1):33-5. Citation on PubMedJakkula E, Melkoniemi M, Kiviranta I, Lohiniva J, Rin SS, Perl M, Warman ML, Ahonen K, Krger H, Gring HH, Ala-Kokko L. The role of sequence variations within the genes encoding collagen II, IX and XI in non-syndromic, early-onset osteoarthritis. Osteoarthritis Cartilage. 2005 Jun;13(6):497-507. Citation on PubMedMcGuirt WT, Prasad SD, Griffith AJ, Kunst HP, Green GE, Shpargel KB, Runge C, Huybrechts C, Mueller RF, Lynch E, King MC, Brunner HG, Cremers CW, Takanosu M, Li SW, Arita M, Mayne R, Prockop DJ, Van Camp G, Smith RJ. Mutations in COL11A2 cause non-syndromic hearing loss (DFNA13). Nat Genet. 1999 Dec;23(4):413-9. Citation on PubMedMelkoniemi M, Brunner HG, Manouvrier S, Hennekam R, Superti-Furga A, Kriinen H, Pauli RM, van Essen T, Warman ML, Bonaventure J, Miny P, Ala-Kokko L. Autosomal recessive disorder otospondylomegaepiphyseal dysplasia is associated with loss-of-function mutations in the COL11A2 gene. Am J Hum Genet. 2000 Feb;66(2):368-77. Citation on PubMed or Free article on PubMed CentralPihlajamaa T, Prockop DJ, Faber J, Winterpacht A, Zabel B, Giedion A, Wiesbauer P, Spranger J, Ala-Kokko L. Heterozygous glycine substitution in the COL11A2 gene in the original patient with the Weissenbacher-Zweymller syndrome demonstrates its identity with heterozygous OSMED (nonocular Stickler syndrome). Am J Med Genet. 1998 Nov 2;80(2):115-20. Citation on PubMedRyder JJ, Garrison K, Song F, Hooper L, Skinner J, Loke Y, Loughlin J, Higgins JP, MacGregor AJ. Genetic associations in peripheral joint osteoarthritis and spinal degenerative disease: a systematic review. Ann Rheum Dis. 2008 May;67(5):584-91. Epub 2007 Aug 24. Review. Citation on PubMedTompson SW, Faqeih EA, Ala-Kokko L, Hecht JT, Miki R, Funari T, Funari VA, Nevarez L, Krakow D, Cohn DH. Dominant and recessive forms of fibrochondrogenesis resulting from mutations at a second locus, COL11A2. Am J Med Genet A. 2012 Feb;158A(2):309-14. doi: 10.1002/ajmg.a.34406. Epub 2012 Jan 13. Citation on PubMed or Free article on PubMed Central \n",
            "APOBEC2\t\n",
            "TRERF1\t\n",
            "PTCRA\t\n",
            "PEX6\t   From NCBI Gene:This gene encodes a member of the AAA (ATPases associated with diverse cellular activities) family of ATPases. This member is a predominantly cytoplasmic protein, which plays a direct role in peroxisomal protein import and is required for PTS1 (peroxisomal targeting signal 1, a C-terminal tripeptide of the sequence ser-lys-leu) receptor activity. Mutations in this gene cause peroxisome biogenesis disorders of complementation group 4 and complementation group 6. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Oct 2015]From UniProt:Involved in peroxisome biosynthesis. Required for stability of the PTS1 receptor. Anchored by PEX26 to peroxisome membranes, possibly to form heteromeric AAA ATPase complexes required for the import of proteins into peroxisomes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Zellweger spectrum disorderFrom NCBI Gene:Peroxisome biogenesis disorder 4a (zellweger)Heimler syndrome 2Peroxisome biogenesis disorder 4BFrom UniProt:Heimler syndrome 2 (HMLR2): A form of Heimler syndrome, a very mild peroxisome biogenesis disorder characterized by sensorineural hearing loss, amelogenesis imperfecta resulting in enamel hyoplasia of the secondary dentition, nail defects, and occasional or late-onset retinal pigmentation abnormalities. [MIM:616617]Peroxisome biogenesis disorder complementation group 4 (PBD-CG4): A peroxisomal disorder arising from a failure of protein import into the peroxisomal membrane or matrix. The peroxisome biogenesis disorders (PBD group) are genetically heterogeneous with at least 14 distinct genetic groups as concluded from complementation studies. Include disorders are: Zellweger syndrome (ZWS), neonatal adrenoleukodystrophy (NALD), infantile Refsum disease (IRD), and classical rhizomelic chondrodysplasia punctata (RCDP). ZWS, NALD and IRD are distinct from RCDP and constitute a clinical continuum of overlapping phenotypes known as the Zellweger spectrum (PBD-ZSS). [MIM:614862]Peroxisome biogenesis disorder 4A (PBD4A): A fatal peroxisome biogenesis disorder belonging to the Zellweger disease spectrum and clinically characterized by severe neurologic dysfunction with profound psychomotor retardation, severe hypotonia and neonatal seizures, craniofacial abnormalities, liver dysfunction, and biochemically by the absence of peroxisomes. Additional features include cardiovascular and skeletal defects, renal cysts, ocular abnormalities, and hearing impairment. Most severely affected individuals with the classic form of the disease (classic Zellweger syndrome) die within the first year of life. [MIM:614862]Peroxisome biogenesis disorder 4B (PBD4B): A peroxisome biogenesis disorder that includes neonatal adrenoleukodystrophy (NALD) and infantile Refsum disease (IRD), two milder manifestations of the Zellweger disease spectrum. The clinical course of patients with the NALD and IRD presentation is variable and may include developmental delay, hypotonia, liver dysfunction, sensorineural hearing loss, retinal dystrophy and vision impairment. Children with the NALD presentation may reach their teens, while patients with the IRD presentation may reach adulthood. The clinical conditions are often slowly progressive in particular with respect to loss of hearing and vision. The biochemical abnormalities include accumulation of phytanic acid, very long chain fatty acids (VLCFA), di- and trihydroxycholestanoic acid and pipecolic acid. [MIM:614863]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p21.1, which is the short (p) arm of chromosome 6 at position 21.1Molecular Location: base pairs 42,963,873 to 42,980,224 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HMLR2PAF-2PAF2PBD4APDB4BPXAAA1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PEX6      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: HEIMLER SYNDROME 2OMIM: PEROXISOME BIOGENESIS DISORDER 4A (ZELLWEGER)OMIM: PEROXISOME BIOGENESIS DISORDER 4BOMIM: PEROXISOME BIOGENESIS FACTOR 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PEX6  OMIM: HEIMLER SYNDROME 2OMIM: PEROXISOME BIOGENESIS DISORDER 4A (ZELLWEGER)OMIM: PEROXISOME BIOGENESIS DISORDER 4BOMIM: PEROXISOME BIOGENESIS FACTOR 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RRP36\t\n",
            "TTBK1\t\n",
            "TCTE1\t\n",
            "ENPP5\t\n",
            "CYP39A1\t\n",
            "ADGRF2\t\n",
            "CRISP1\t\n",
            "TFAP2D\t\n",
            "MLIP\t\n",
            "COL21A1\t\n",
            "DST\t   From NCBI Gene:This gene encodes a member of the plakin protein family of adhesion junction plaque proteins. Multiple alternatively spliced transcript variants encoding distinct isoforms have been found for this gene, but the full-length nature of some variants has not been defined. It has been reported that some isoforms are expressed in neural and muscle tissue, anchoring neural intermediate filaments to the actin cytoskeleton, and some isoforms are expressed in epithelial tissue, anchoring keratin-containing intermediate filaments to hemidesmosomes. Consistent with the expression, mice defective for this gene show skin blistering and neurodegeneration. [provided by RefSeq, Mar 2010]From UniProt:Isoform 3: plays a structural role in the assembly of hemidesmosomes of epithelial cells; anchors keratin-containing intermediate filaments to the inner plaque of hemidesmosomes. Required for the regulation of keratinocyte polarity and motility; mediates integrin ITGB4 regulation of RAC1 activity.Isoform 7: regulates the organization and stability of the microtubule network of sensory neurons to allow axonal transport.Cytoskeletal linker protein. Acts as an integrator of intermediate filaments, actin and microtubule cytoskeleton networks. Required for anchoring either intermediate filaments to the actin cytoskeleton in neural and muscle cells or keratin-containing intermediate filaments to hemidesmosomes in epithelial cells. The proteins may self-aggregate to form filaments or a two-dimensional mesh. Regulates the organization and stability of the microtubule network of sensory neurons to allow axonal transport. Mediates docking of the dynein/dynactin motor complex to vesicle cargos for retrograde axonal transport through its interaction with TMEM108 and DCTN1 (By similarity).Isoform 6: required for bundling actin filaments around the nucleus.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Epidermolysis bullosa simplex, autosomal recessive 2Neuropathy, hereditary sensory and autonomic, type VIFrom UniProt:Neuropathy, hereditary sensory and autonomic, 6 (HSAN6): A form of hereditary sensory and autonomic neuropathy, a genetically and clinically heterogeneous group of disorders characterized by degeneration of dorsal root and autonomic ganglion cells, and by sensory and/or autonomic abnormalities. HSAN6 is a severe autosomal recessive disorder characterized by neonatal hypotonia, respiratory and feeding difficulties, lack of psychomotor development, and autonomic abnormalities including labile cardiovascular function, lack of corneal reflexes leading to corneal scarring, areflexia, and absent axonal flare response after intradermal histamine injection. [MIM:614653]Epidermolysis bullosa simplex, autosomal recessive 2 (EBSB2): A form of epidermolysis bullosa, a dermatologic disorder characterized by localized blistering on the dorsal, lateral and plantar surfaces of the feet. EBSB2 is characterized by trauma-induced blistering mainly occurring on the feet and ankles. Ultrastructural analysis of skin biopsy shows abnormal hemidesmosomes with poorly formed inner plaques. [MIM:615425]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6p12.1, which is the short (p) arm of chromosome 6 at position 12.1Molecular Location: base pairs 56,457,987 to 56,954,671 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BP240BPABPAG1CATX-15CATX15D6S1101DMHDTEBSB2HSAN6MACF2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DST      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DYSTONINOMIM: EPIDERMOLYSIS BULLOSA SIMPLEX, AUTOSOMAL RECESSIVE 2OMIM: NEUROPATHY, HEREDITARY SENSORY AND AUTONOMIC, TYPE VI      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DST  OMIM: DYSTONINOMIM: EPIDERMOLYSIS BULLOSA SIMPLEX, AUTOSOMAL RECESSIVE 2OMIM: NEUROPATHY, HEREDITARY SENSORY AND AUTONOMIC, TYPE VI  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "F13A1\t   The F13A1 gene provides instructions for making one part, the A subunit, of a protein called factor XIII. This protein is part of a group of related proteins called coagulation factors that are essential for normal blood clotting. They work together as part of the coagulation cascade, which is a series of chemical reactions that forms blood clots in response to injury. After an injury, clots seal off blood vessels to stop bleeding and trigger blood vessel repair. Factor XIII acts at the end of the cascade to strengthen and stabilize newly formed clots, preventing further blood loss.Factor XIII in the bloodstream is made of two A subunits (produced from the F13A1 gene) and two B subunits (produced from the F13B gene). When a new blood clot forms, the A and B subunits separate from one another, and the A subunit is cut (cleaved) to produce the active form of factor XIII (factor XIIIa). The active protein links together molecules of fibrin, the material that forms the clot, which strengthens the clot and keeps other molecules from breaking it down.Studies suggest that factor XIII has additional functions, although these are less well understood than its role in blood clotting. Specifically, factor XIII is likely involved in other aspects of wound healing, immune system function, maintaining pregnancy, bone formation, and the growth of new blood vessels (angiogenesis).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Factor XIII deficiency     At least 140 mutations in the F13A1 gene have been found to cause inherited factor XIII deficiency, a rare bleeding disorder. Without treatment, affected individuals have a greatly increased risk of abnormal bleeding episodes, including life-threatening bleeding inside the skull (intracranial hemorrhage). F13A1 gene mutations severely reduce the amount or activity of the A subunit of factor XIII. In most people with these mutations, the level of functional factor XIII in the bloodstream is less than 5 percent of normal. This loss of factor XIII activity weakens new blood clots and prevents them from stopping blood loss effectively. More About This Health Condition      Other disorders     Several common variations (polymorphisms) in the F13A1 gene have been studied as possible risk factors for diseases involving abnormal blood clotting. The most common F13A1 gene polymorphism changes a single protein building block (amino acid) in a critical part of the A subunit, replacing the amino acid valine with the amino acid leucine at protein position 34 (written as Val34Leu or V34L). This genetic change speeds up the activation of factor XIII. The Val34Leu polymorphism has been studied in relation to heart disease, stroke, recurrent pregnancy loss, and several other conditions. However, the results of these studies have been conflicting, and it remains unclear whether the polymorphism represents a major risk factor for any of these conditions.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 140 mutations in the F13A1 gene have been found to cause inherited factor XIII deficiency, a rare bleeding disorder. Without treatment, affected individuals have a greatly increased risk of abnormal bleeding episodes, including life-threatening bleeding inside the skull (intracranial hemorrhage). F13A1 gene mutations severely reduce the amount or activity of the A subunit of factor XIII. In most people with these mutations, the level of functional factor XIII in the bloodstream is less than 5 percent of normal. This loss of factor XIII activity weakens new blood clots and prevents them from stopping blood loss effectively. More About This Health Condition  Several common variations (polymorphisms) in the F13A1 gene have been studied as possible risk factors for diseases involving abnormal blood clotting. The most common F13A1 gene polymorphism changes a single protein building block (amino acid) in a critical part of the A subunit, replacing the amino acid valine with the amino acid leucine at protein position 34 (written as Val34Leu or V34L). This genetic change speeds up the activation of factor XIII. The Val34Leu polymorphism has been studied in relation to heart disease, stroke, recurrent pregnancy loss, and several other conditions. However, the results of these studies have been conflicting, and it remains unclear whether the polymorphism represents a major risk factor for any of these conditions.    Cytogenetic Location: 6p25.1, which is the short (p) arm of chromosome 6 at position 25.1Molecular Location: base pairs 6,144,078 to 6,320,691 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA525O21.1 (coagulation factor XIII, A1 polypeptide)coagulation factor XIII A chain precursorcoagulation factor XIII, A polypeptidecoagulation factor XIII, A1 polypeptidecoagulation factor XIIIaF13Afactor XIIIafibrin stabilizing factor, A subunitfibrinoligaseFSF, A subunitprotein-glutamine gamma-glutamyltransferase A chainTGasetransglutaminase A chaintransglutaminase. plasma     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Diagram: Blood Clotting Cascade      Tests Listed in the Genetic Testing Registry (1 link)     Tests of F13A1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FACTOR XIII, A1 SUBUNIT      Research Resources (8 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarFactor XIII Registry Database: F13A GeneFactor XIII Registry Database: F13A MutationsHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Diagram: Blood Clotting Cascade  Tests of F13A1  PubMed  FACTOR XIII, A1 SUBUNIT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarFactor XIII Registry Database: F13A GeneFactor XIII Registry Database: F13A MutationsHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bagoly Z, Koncz Z, Hrsfalvi J, Muszbek L. Factor XIII, clot structure, thrombosis. Thromb Res. 2012 Mar;129(3):382-7. doi: 10.1016/j.thromres.2011.11.040. Epub 2011 Dec 24. Review. Citation on PubMedBiswas A, Ivaskevicius V, Seitz R, Thomas A, Oldenburg J. An update of the mutation profile of Factor 13 A and B genes. Blood Rev. 2011 Sep;25(5):193-204. doi: 10.1016/j.blre.2011.03.001. Epub 2011 Jun 2. Review. Citation on PubMedBiswas A, Ivaskevicius V, Thomas A, Oldenburg J. Coagulation factor XIII deficiency. Diagnosis, prevalence and management of inherited and acquired forms. Hamostaseologie. 2014;34(2):160-6. doi: 10.5482/HAMO-13-08-0046. Epub 2014 Feb 7. Review. Citation on PubMedOMIM: FACTOR XIII, A1 SUBUNITHethershaw EL, Cilia La Corte AL, Duval C, Ali M, Grant PJ, Arins RA, Philippou H. The effect of blood coagulation factor XIII on fibrin clot structure and fibrinolysis. J Thromb Haemost. 2014 Feb;12(2):197-205. doi: 10.1111/jth.12455. Citation on PubMedLevy JH, Greenberg C. Biology of Factor XIII and clinical manifestations of Factor XIII deficiency. Transfusion. 2013 May;53(5):1120-31. doi: 10.1111/j.1537-2995.2012.03865.x. Epub 2012 Aug 28. Review. Citation on PubMedMuszbek L, Bereczky Z, Bagoly Z, Komromi I, Katona . Factor XIII: a coagulation factor with multiple plasmatic and cellular functions. Physiol Rev. 2011 Jul;91(3):931-72. doi: 10.1152/physrev.00016.2010. Review. Citation on PubMedSchroeder V, Kohler HP. Factor XIII deficiency: an update. Semin Thromb Hemost. 2013 Sep;39(6):632-41. doi: 10.1055/s-0033-1353392. Epub 2013 Aug 8. Review. Citation on PubMed \n",
            "EYS\t   From NCBI Gene:The product of this gene contains multiple epidermal growth factor (EGF)-like and LamG domains. The protein is expressed in the photoreceptor layer of the retina, and the gene is mutated in autosomal recessive retinitis pigmentosa. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Dec 2008]From UniProt:Required to maintain the integrity of photoreceptor cells (PubMed:18836446). Specifically required for normal morphology of the photoreceptor ciliary pocket, and might thus facilitate protein trafficking between the photoreceptor inner and outer segments via the transition zone (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Cone-rod dystrophyRetinitis pigmentosaFrom NCBI Gene:Retinitis pigmentosa 25From UniProt:Retinitis pigmentosa 25 (RP25): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:602772]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q12, which is the long (q) arm of chromosome 6 at position 12Molecular Location: base pairs 63,719,980 to 65,707,225 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA74E24.1bA166P24.2bA307F22.3C6orf178C6orf179C6orf180dJ22I17.2dJ303F19.1dJ1018A4.2EGFL10EGFL11RP25SPAM     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of EYS      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: EYES SHUT, DROSOPHILA, HOMOLOG OFOMIM: RETINITIS PIGMENTOSA 25      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of EYS  OMIM: EYES SHUT, DROSOPHILA, HOMOLOG OFOMIM: RETINITIS PIGMENTOSA 25  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FAM135A\t\n",
            "RREB1\t\n",
            "KHDC3L\t   The KHDC3L gene provides instructions for making a protein whose role is not known. The KHDC3L protein is thought to be involved in regulating gene activity (expression) through a phenomenon known as genomic imprinting. Through genomic imprinting, certain genes are turned off (inactivated) based on which parent the copy of the gene came from. For most genes, both copies of the gene (one copy inherited from each parent) are active in all cells. However, for a small subset of genes, only one of the two copies is active and the other is turned off. For some of these genes, the copy from the father is normally active, while for others, the copy from the mother is normally active.It is likely that the KHDC3L protein has additional roles in egg cell (oocyte) and embryonic development; however, its exact functions are unclear.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Recurrent hydatidiform mole     At least six mutations in the KHDC3L gene have been found to cause a pregnancy-related condition known as recurrent hydatidiform mole. A hydatidiform mole is a mass that forms early in pregnancy and is made up of cells from an abnormally developed embryo and placenta. The placenta, a structure in the uterus that normally provides nutrients to a growing fetus, is dysfunctional and appears as numerous small sacs, often described as resembling a bunch of grapes. When a hydatidiform mole develops more than once, the condition is known as recurrent hydatidiform mole. KHDC3L gene mutations account for recurrent hydatidiform mole in about 5 percent of women with this condition.KHDC3L gene mutations result in the production of a protein with reduced function. As a result, oocytes do not develop normally. A pregnancy that results from an abnormal oocyte cannot develop properly, resulting in recurrent hydatidiform mole. KHDC3L gene mutations can also prevent proper imprinting of multiple genes that contribute to a developing embryo, leading to abnormal gene activity (expression). It is not clear if problems with imprinting also contribute to the development of a hydatidiform mole. In women with KHDC3L gene mutations, a hydatidiform mole will develop in every pregnancy that occurs with her egg cells. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least six mutations in the KHDC3L gene have been found to cause a pregnancy-related condition known as recurrent hydatidiform mole. A hydatidiform mole is a mass that forms early in pregnancy and is made up of cells from an abnormally developed embryo and placenta. The placenta, a structure in the uterus that normally provides nutrients to a growing fetus, is dysfunctional and appears as numerous small sacs, often described as resembling a bunch of grapes. When a hydatidiform mole develops more than once, the condition is known as recurrent hydatidiform mole. KHDC3L gene mutations account for recurrent hydatidiform mole in about 5 percent of women with this condition.KHDC3L gene mutations result in the production of a protein with reduced function. As a result, oocytes do not develop normally. A pregnancy that results from an abnormal oocyte cannot develop properly, resulting in recurrent hydatidiform mole. KHDC3L gene mutations can also prevent proper imprinting of multiple genes that contribute to a developing embryo, leading to abnormal gene activity (expression). It is not clear if problems with imprinting also contribute to the development of a hydatidiform mole. In women with KHDC3L gene mutations, a hydatidiform mole will develop in every pregnancy that occurs with her egg cells. More About This Health Condition    Cytogenetic Location: 6q13, which is the long (q) arm of chromosome 6 at position 13Molecular Location: base pairs 73,362,677 to 73,364,175 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C6orf221ECAT1ES cell-associated transcript 1 proteinHYDM2KH domain containing 3-like, subcortical maternal complex memberKHDC3-like protein     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Developmental Biology (sixth edition, 2000): DNA Methylation and Gene ActivityMadame Curie Bioscience Database: Genomic Imprinting As a Nonmutational Means to Modulate Dosage Dependent Genes      Tests Listed in the Genetic Testing Registry (1 link)     Tests of KHDC3L      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     KHDC3-LIKE PROTEIN, SUBCORTICAL MATERNAL COMPLEX MEMBER      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): DNA Methylation and Gene ActivityMadame Curie Bioscience Database: Genomic Imprinting As a Nonmutational Means to Modulate Dosage Dependent Genes  Tests of KHDC3L  PubMed  KHDC3-LIKE PROTEIN, SUBCORTICAL MATERNAL COMPLEX MEMBER  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Fallahian M, Sebire NJ, Savage PM, Seckl MJ, Fisher RA. Mutations in NLRP7 and KHDC3L confer a complete hydatidiform mole phenotype on digynic triploid conceptions. Hum Mutat. 2013 Feb;34(2):301-8. doi: 10.1002/humu.22228. Epub 2012 Nov 2. Citation on PubMedHui P, Buza N, Murphy KM, Ronnett BM. Hydatidiform Moles: Genetic Basis and Precision Diagnosis. Annu Rev Pathol. 2017 Jan 24;12:449-485. doi: 10.1146/annurev-pathol-052016-100237. Review. Citation on PubMedOMIM: KHDC3-LIKE PROTEIN, SUBCORTICAL MATERNAL COMPLEX MEMBERNguyen NM, Slim R. Genetics and Epigenetics of Recurrent Hydatidiform Moles: Basic Science and Genetic Counselling. Curr Obstet Gynecol Rep. 2014 Jan 21;3:55-64. eCollection 2014. Review. Citation on PubMed or Free article on PubMed CentralParry DA, Logan CV, Hayward BE, Shires M, Landolsi H, Diggle C, Carr I, Rittore C, Touitou I, Philibert L, Fisher RA, Fallahian M, Huntriss JD, Picton HM, Malik S, Taylor GR, Johnson CA, Bonthron DT, Sheridan EG. Mutations causing familial biparental hydatidiform mole implicate c6orf221 as a possible regulator of genomic imprinting in the human oocyte. Am J Hum Genet. 2011 Sep 9;89(3):451-8. doi: 10.1016/j.ajhg.2011.08.002. Epub 2011 Sep 1. Citation on PubMed or Free article on PubMed CentralReddy R, Akoury E, Phuong Nguyen NM, Abdul-Rahman OA, Dery C, Gupta N, Daley WP, Ao A, Landolsi H, Ann Fisher R, Touitou I, Slim R. Report of four new patients with protein-truncating mutations in C6orf221/KHDC3L and colocalization with NLRP7. Eur J Hum Genet. 2013 Sep;21(9):957-64. doi: 10.1038/ejhg.2012.274. Epub 2012 Dec 12. Citation on PubMed or Free article on PubMed Central \n",
            "COL12A1\t   From NCBI Gene:This gene encodes the alpha chain of type XII collagen, a member of the FACIT (fibril-associated collagens with interrupted triple helices) collagen family. Type XII collagen is a homotrimer found in association with type I collagen, an association that is thought to modify the interactions between collagen I fibrils and the surrounding matrix. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]From UniProt:Type XII collagen interacts with type I collagen-containing fibrils, the COL1 domain could be associated with the surface of the fibrils, and the COL2 and NC3 domains may be localized in the perifibrillar matrix.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Ehlers-Danlos syndromeFrom NCBI Gene:Bethlem myopathy 2Ullrich congenital muscular dystrophy 2From UniProt:Bethlem myopathy 2 (BTHLM2): A form of Bethlem myopathy, a benign proximal myopathy characterized by early childhood onset and joint contractures most frequently affecting the elbows and ankles. BTHLM2 inheritance is autosomal dominant. [MIM:616471]Ullrich congenital muscular dystrophy 2 (UCMD2): A form of Ullrich muscular dystrophy, a congenital myopathy characterized by muscle weakness and multiple joint contractures, generally noted at birth or early infancy. The clinical course is more severe than in Bethlem myopathy. [MIM:616470]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q13-q14.1, which is the long (q) arm of chromosome 6 between positions 13 and 14.1Molecular Location: base pairs 75,084,326 to 75,206,262 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BA209D8.1BTHLM2COL12A1LDJ234P15.1EDSMYPUCMD2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL12A1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: BETHLEM MYOPATHY 2OMIM: COLLAGEN, TYPE XII, ALPHA-1OMIM: ULLRICH CONGENITAL MUSCULAR DYSTROPHY 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COL12A1  OMIM: BETHLEM MYOPATHY 2OMIM: COLLAGEN, TYPE XII, ALPHA-1OMIM: ULLRICH CONGENITAL MUSCULAR DYSTROPHY 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MYO6\t   From NCBI Gene:This gene encodes a reverse-direction motor protein that moves toward the minus end of actin filaments and plays a role in intracellular vesicle and organelle transport. The protein consists of a motor domain containing an ATP- and an actin-binding site and a globular tail which interacts with other proteins. This protein maintains the structural integrity of inner ear hair cells and mutations in this gene cause non-syndromic autosomal dominant and recessive hearing loss. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Jul 2014]From UniProt:Myosins are actin-based motor molecules with ATPase activity (By similarity). Unconventional myosins serve in intracellular movements (By similarity). Myosin 6 is a reverse-direction motor protein that moves towards the minus-end of actin filaments (PubMed:10519557). Has slow rate of actin-activated ADP release due to weak ATP binding (By similarity). Functions in a variety of intracellular processes such as vesicular membrane trafficking and cell migration (By similarity). Required for the structural integrity of the Golgi apparatus via the p53-dependent pro-survival pathway (PubMed:16507995). Appears to be involved in a very early step of clathrin-mediated endocytosis in polarized epithelial cells (PubMed:11447109). May act as a regulator of F-actin dynamics (By similarity). May play a role in transporting DAB2 from the plasma membrane to specific cellular targets (By similarity). May play a role in the extension and network organization of neurites (By similarity). Required for structural integrity of inner ear hair cells (By similarity). Modulates RNA polymerase II-dependent transcription (PubMed:16949370).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossAge-related hearing lossFrom NCBI Gene:Deafness, autosomal dominant 22Deafness, autosomal recessive 37From UniProt:Deafness, autosomal recessive, 37 (DFNB37): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. [MIM:607821]Deafness, autosomal dominant 22, with hypertrophic cardiomyopathy (DFNHCM): An autosomal dominant sensorineural deafness associated with hypertrophic cardiomyopathy. [MIM:606346]Deafness, autosomal dominant, 22 (DFNA22): A form of non-syndromic sensorineural hearing loss. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. DFNA22 is progressive and postlingual, with onset during childhood. By the age of approximately 50 years, affected individuals invariably have profound sensorineural deafness. [MIM:606346]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q14.1, which is the long (q) arm of chromosome 6 at position 14.1Molecular Location: base pairs 75,749,176 to 75,919,537 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DFNA22DFNB37     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MYO6      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: DEAFNESS, AUTOSOMAL DOMINANT 22OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 37OMIM: MYOSIN VI      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MYO6  OMIM: DEAFNESS, AUTOSOMAL DOMINANT 22OMIM: DEAFNESS, AUTOSOMAL RECESSIVE 37OMIM: MYOSIN VI  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IRAK1BP1\t\n",
            "CGA\t\n",
            "CFAP206\t\n",
            "GABRR1\t   From NCBI Gene:GABA is the major inhibitory neurotransmitter in the mammalian brain where it acts at GABA receptors, which are ligand-gated chloride channels. GABRR1 is a member of the rho subunit family. Several transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Feb 2012]From UniProt:GABA, the major inhibitory neurotransmitter in the vertebrate brain, mediates neuronal inhibition by binding to the GABA/benzodiazepine receptor and opening an integral chloride channel. Rho-1 GABA receptor could play a role in retinal neurotransmission.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Schizoaffective disorder     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 6q15, which is the long (q) arm of chromosome 6 at position 15Molecular Location: base pairs 89,177,504 to 89,231,288 on chromosome 6 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GABRR1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: GAMMA-AMINOBUTYRIC ACID RECEPTOR, RHO-1      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GABRR1  OMIM: GAMMA-AMINOBUTYRIC ACID RECEPTOR, RHO-1  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CASP8AP2\t\n",
            "MMS22L\t\n",
            "ZAN\t\n",
            "SRRT\t\n",
            "MUC3A\t\n",
            "MUC12\t\n",
            "MUC17\t\n",
            "COL26A1\t\n",
            "SPDYE2\t\n",
            "POLR2J3\t\n",
            "FBXL13\t\n",
            "LRRC17\t\n",
            "NFE4\t\n",
            "RELN\t   The RELN gene provides instructions for making a protein called reelin. This protein is produced in the brain both before and after birth. Reelin is released by certain brain cells; then it attaches (binds) to specific receptor proteins. In the developing brain, this binding turns on (activates) a signaling pathway that triggers nerve cells (neurons) to migrate to their proper locations.After birth, reelin likely plays a role in many brain processes, including the extension of axons and dendrites, which are specialized outgrowths from nerve cells that are essential for the transmission of nerve impulses. Reelin may also regulate synaptic plasticity, which is the ability of connections between neurons (synapses) to change and adapt over time in response to experience. Additionally, reelin controls the release of chemicals that relay signals in the nervous system (neurotransmitters).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Lissencephaly with cerebellar hypoplasia     At least six mutations in the RELN gene have been found to cause lissencephaly with cerebellar hypoplasia (LCH). This condition affects brain development, resulting in the brain having a smooth appearance (lissencephaly) instead of its normal folds and grooves. In addition, the brain region involved in coordinating movements is unusually small and underdeveloped (cerebellar hypoplasia). The RELN gene mutations that cause LCH lead to a complete lack of reelin. As a result, the signaling pathway that triggers neuronal migration is not activated. Without reelin, neurons are disorganized, the normal folds and grooves of the brain do not form, and brain structures do not develop properly.  This impairment of brain development leads to intellectual disability, delayed overall development, movement problems, and other signs and symptoms of LCH. More About This Health Condition      Autism spectrum disorder     Genetics Home Reference provides information about autism spectrum disorder. More About This Health Condition      Myoclonus-dystonia     Genetics Home Reference provides information about myoclonus-dystonia. More About This Health Condition      Other disorders     Studies have shown certain variations (polymorphisms) in the RELN gene to be associated with an increased risk of psychiatric disorders such as schizophrenia and bipolar disease. Women with these polymorphisms are at particular risk of developing bipolar disease. In addition, certain genetic changes that result in a decrease in production of reelin (but not a complete absence) may be a risk factor for autism spectrum disorder, which affects communication and social interaction. However, other studies have not supported these findings. Many genetic and environmental factors are believed to contribute to these complex conditions.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least six mutations in the RELN gene have been found to cause lissencephaly with cerebellar hypoplasia (LCH). This condition affects brain development, resulting in the brain having a smooth appearance (lissencephaly) instead of its normal folds and grooves. In addition, the brain region involved in coordinating movements is unusually small and underdeveloped (cerebellar hypoplasia). The RELN gene mutations that cause LCH lead to a complete lack of reelin. As a result, the signaling pathway that triggers neuronal migration is not activated. Without reelin, neurons are disorganized, the normal folds and grooves of the brain do not form, and brain structures do not develop properly.  This impairment of brain development leads to intellectual disability, delayed overall development, movement problems, and other signs and symptoms of LCH. More About This Health Condition  Genetics Home Reference provides information about autism spectrum disorder. More About This Health Condition  Genetics Home Reference provides information about myoclonus-dystonia. More About This Health Condition  Studies have shown certain variations (polymorphisms) in the RELN gene to be associated with an increased risk of psychiatric disorders such as schizophrenia and bipolar disease. Women with these polymorphisms are at particular risk of developing bipolar disease. In addition, certain genetic changes that result in a decrease in production of reelin (but not a complete absence) may be a risk factor for autism spectrum disorder, which affects communication and social interaction. However, other studies have not supported these findings. Many genetic and environmental factors are believed to contribute to these complex conditions.    Cytogenetic Location: 7q22.1, which is the long (q) arm of chromosome 7 at position 22.1Molecular Location: base pairs 103,471,784 to 103,989,516 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        LIS2PRO1598RELN_HUMANRL     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Jasper's Basic Mechanisms of the Epilepsies (fourth edition, 2012): Neuronal Migration Signaling Pathways (figure)      Tests Listed in the Genetic Testing Registry (1 link)     Tests of RELN      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     REELIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Jasper's Basic Mechanisms of the Epilepsies (fourth edition, 2012): Neuronal Migration Signaling Pathways (figure)  Tests of RELN  PubMed  REELIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Chang BS, Duzcan F, Kim S, Cinbis M, Aggarwal A, Apse KA, Ozdel O, Atmaca M, Zencir S, Bagci H, Walsh CA. The role of RELN in lissencephaly and neuropsychiatric disease. Am J Med Genet B Neuropsychiatr Genet. 2007 Jan 5;144B(1):58-63. Citation on PubMedGoes FS, Willour VL, Zandi PP, Belmonte PL, MacKinnon DF, Mondimore FM, Schweizer B; National Institute of Mental Health Genetics Initiative Bipolar Disorder Consortium, DePaulo JR Jr, Gershon ES, McMahon FJ, Potash JB. Sex-specific association of the Reelin gene with bipolar disorder. Am J Med Genet B Neuropsychiatr Genet. 2010 Mar 5;153B(2):549-53. doi: 10.1002/ajmg.b.31018. Citation on PubMed or Free article on PubMed CentralHong SE, Shugart YY, Huang DT, Shahwan SA, Grant PE, Hourihane JO, Martin ND, Walsh CA. Autosomal recessive lissencephaly with cerebellar hypoplasia is associated with human RELN mutations. Nat Genet. 2000 Sep;26(1):93-6. Erratum in: Nat Genet 2001 Feb;27(2):225. Citation on PubMedLakatosova S, Ostatnikova D. Reelin and its complex involvement in brain development and function. Int J Biochem Cell Biol. 2012 Sep;44(9):1501-4. doi: 10.1016/j.biocel.2012.06.002. Epub 2012 Jun 15. Review. Citation on PubMedOMIM: REELINWedenoja J, Tuulio-Henriksson A, Suvisaari J, Loukola A, Paunio T, Partonen T, Varilo T, Lnnqvist J, Peltonen L. Replication of association between working memory and Reelin, a potential modifier gene in schizophrenia. Biol Psychiatry. 2010 May 15;67(10):983-91. doi: 10.1016/j.biopsych.2009.09.026. Epub 2009 Nov 17. Citation on PubMed or Free article on PubMed CentralZaki M, Shehab M, El-Aleem AA, Abdel-Salam G, Koeller HB, Ilkin Y, Ross ME, Dobyns WB, Gleeson JG. Identification of a novel recessive RELN mutation using a homozygous balanced reciprocal translocation. Am J Med Genet A. 2007 May 1;143A(9):939-44. Citation on PubMed \n",
            "LHFPL3\t\n",
            "ATXN7L1\t\n",
            "CAV2\t\n",
            "PTPRZ1\t\n",
            "PAX4\t   From NCBI Gene:This gene is a member of the paired box (PAX) family of transcription factors. Members of this gene family typically contain a paired box domain, an octapeptide, and a paired-type homeodomain. These genes play critical roles during fetal development and cancer growth. The paired box 4 gene is involved in pancreatic islet development and mouse studies have demonstrated a role for this gene in differentiation of insulin-producing beta cells. [provided by RefSeq, Jul 2008]From UniProt:Plays an important role in the differentiation and development of pancreatic islet beta cells. Transcriptional repressor that binds to a common element in the glucagon, insulin and somatostatin promoters. Competes with PAX6 for this same promoter binding site. Isoform 2 appears to be a dominant negative form antagonizing PAX4 transcriptional activity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Diabetes mellitus, ketosis-proneDiabetes mellitus type 2Maturity-onset diabetes of the young, type 9From UniProt:Diabetes mellitus, ketosis-prone (KPD): An atypical form of diabetes mellitus characterized by an acute initial presentation with severe hyperglycemia and ketosis, as seen in classic type 1 diabetes, but after initiation of insulin therapy, prolonged remission is often possible with cessation of insulin therapy and maintenance of appropriate metabolic control. Metabolic studies show a markedly blunted insulin secretory response to glucose, partially reversible with the improvement of blood glucose control. Variable levels of insulin resistance are observed, especially in obese patients. Pancreatic beta-cell autoimmunity is a rare finding. [MIM:612227]Diabetes mellitus, non-insulin-dependent (NIDDM): A multifactorial disorder of glucose homeostasis caused by a lack of sensitivity to the body's own insulin. Affected individuals usually have an obese body habitus and manifestations of a metabolic syndrome characterized by diabetes, insulin resistance, hypertension and hypertriglyceridemia. The disease results in long-term complications that affect the eyes, kidneys, nerves, and blood vessels. [MIM:125853]Diabetes mellitus, insulin-dependent (IDDM): A multifactorial disorder of glucose homeostasis that is characterized by susceptibility to ketoacidosis in the absence of insulin therapy. Clinical features are polydipsia, polyphagia and polyuria which result from hyperglycemia-induced osmotic diuresis and secondary thirst. These derangements result in long-term complications that affect the eyes, kidneys, nerves, and blood vessels. [MIM:222100]Maturity-onset diabetes of the young 9 (MODY9): A form of diabetes that is characterized by an autosomal dominant mode of inheritance, onset in childhood or early adulthood (usually before 25 years of age), a primary defect in insulin secretion and frequent insulin-independence at the beginning of the disease. [MIM:612225]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q32.1, which is the long (q) arm of chromosome 7 at position 32.1Molecular Location: base pairs 127,610,292 to 127,618,192 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KPDMODY9     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PAX4      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: DIABETES MELLITUS, KETOSIS-PRONEOMIM: DIABETES MELLITUS, NONINSULIN-DEPENDENTOMIM: MATURITY-ONSET DIABETES OF THE YOUNG, TYPE 9OMIM: PAIRED BOX GENE 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PAX4  OMIM: DIABETES MELLITUS, KETOSIS-PRONEOMIM: DIABETES MELLITUS, NONINSULIN-DEPENDENTOMIM: MATURITY-ONSET DIABETES OF THE YOUNG, TYPE 9OMIM: PAIRED BOX GENE 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "FLNC\t   From NCBI Gene:This gene encodes one of three related filamin genes, specifically gamma filamin. These filamin proteins crosslink actin filaments into orthogonal networks in cortical cytoplasm and participate in the anchoring of membrane proteins for the actin cytoskeleton. Three functional domains exist in filamin: an N-terminal filamentous actin-binding domain, a C-terminal self-association domain, and a membrane glycoprotein-binding domain. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:Muscle-specific filamin, which plays a central role in muscle cells, probably by functioning as a large actin-cross-linking protein. May be involved in reorganizing the actin cytoskeleton in response to signaling events, and may also display structural functions at the Z lines in muscle cells. Critical for normal myogenesis and for maintaining the structural integrity of the muscle fibers.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Myofibrillar myopathyFrom NCBI Gene:Cardiomyopathy, familial hypertrophic, 26Myofibrillar myopathy, filamin C-relatedMyopathy, distal, 4From UniProt:Cardiomyopathy, familial restrictive 5 (RCM5): A heart disorder characterized by impaired filling of the ventricles with reduced diastolic volume, in the presence of normal or near normal wall thickness and systolic function. [MIM:617047]Cardiomyopathy, familial hypertrophic 26 (CMH26): A hereditary heart disorder characterized by ventricular hypertrophy, which is usually asymmetric and often involves the interventricular septum. The symptoms include dyspnea, syncope, collapse, palpitations, and chest pain. They can be readily provoked by exercise. The disorder has inter- and intrafamilial variability ranging from benign to malignant forms with high risk of cardiac failure and sudden cardiac death. [MIM:617047]Myopathy, myofibrillar, 5 (MFM5): A form of myofibrillar myopathy, a group of chronic neuromuscular disorders characterized at ultrastructural level by disintegration of the sarcomeric Z disk and myofibrils, and replacement of the normal myofibrillar markings by small dense granules, or larger hyaline masses, or amorphous material. MFM5 is characterized by onset in adulthood, clinical features of a limb-girdle myopathy, and focal myofibrillar destruction. [MIM:609524]Myopathy, distal, 4 (MPD4): A slowly progressive muscular disorder characterized by distal muscle weakness and atrophy affecting the upper and lower limbs. Onset occurs around the third to fourth decades of life, and patients remain ambulatory even after long disease duration. Muscle biopsy shows non-specific changes with no evidence of rods, necrosis, or inflammation. [MIM:614065]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q32.1, which is the long (q) arm of chromosome 7 at position 32.1Molecular Location: base pairs 128,830,429 to 128,859,274 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABP-280ABP280AABPAABPLCMH26FLN2MFM5MPD4RCM5     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FLNC      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: CARDIOMYOPATHY, FAMILIAL HYPERTROPHIC, 26OMIM: FILAMIN COMIM: MYOPATHY, DISTAL, 4OMIM: MYOPATHY, MYOFIBRILLAR, 5      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FLNC  OMIM: CARDIOMYOPATHY, FAMILIAL HYPERTROPHIC, 26OMIM: FILAMIN COMIM: MYOPATHY, DISTAL, 4OMIM: MYOPATHY, MYOFIBRILLAR, 5  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IRF5\t   The protein produced from the IRF5 gene, called interferon regulatory factor 5 (IRF5), acts as a transcription factor, which means that it attaches (binds) to specific regions of DNA and helps control the activity of certain genes. When a virus is recognized in the cell, the IRF5 gene is turned on (activated), which leads to the production of IRF5 protein. The protein binds to specific regions of DNA that regulate the activity of genes that produce interferons and other cytokines. Cytokines are proteins that help fight infection by promoting inflammation and regulating the activity of immune system cells. In particular, interferons control the activity of genes that help block the replication of viruses, and they stimulate the activity of certain immune system cells known as natural killer cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Systemic scleroderma     Several normal variations in the IRF5 gene have been associated with an increased risk of developing systemic scleroderma, which is an autoimmune disorder characterized by the buildup of scar tissue (fibrosis) in the skin and internal organs. Although the IRF5 gene is known to stimulate the immune system in response to viruses, it is unknown how the gene variations contribute to the increased risk of systemic scleroderma. Researchers believe that a combination of genetic and environmental factors may play a role in development of the condition. More About This Health Condition      Rheumatoid arthritis     Genetics Home Reference provides information about rheumatoid arthritis. More About This Health Condition      Systemic lupus erythematosus     Genetics Home Reference provides information about systemic lupus erythematosus. More About This Health Condition      Ulcerative colitis     Genetics Home Reference provides information about ulcerative colitis. More About This Health Condition      Autoimmune disorders     Studies have associated normal variations in the IRF5 gene with an increased risk of several autoimmune disorders. Autoimmune disorders occur when the immune system malfunctions and attacks the body's tissues and organs. These disorders include systemic lupus erythematosus, Sjgren syndrome, and rheumatoid arthritis.There is some evidence that certain variations of the IRF5 gene are associated with increased activity of the gene and elevated cytokines. However, it is unknown what role, if any, these effects play in the increased risk of autoimmune disorders. Researchers believe that a combination of genetic and environmental factors may contribute to the development of these conditions.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several normal variations in the IRF5 gene have been associated with an increased risk of developing systemic scleroderma, which is an autoimmune disorder characterized by the buildup of scar tissue (fibrosis) in the skin and internal organs. Although the IRF5 gene is known to stimulate the immune system in response to viruses, it is unknown how the gene variations contribute to the increased risk of systemic scleroderma. Researchers believe that a combination of genetic and environmental factors may play a role in development of the condition. More About This Health Condition  Genetics Home Reference provides information about rheumatoid arthritis. More About This Health Condition  Genetics Home Reference provides information about systemic lupus erythematosus. More About This Health Condition  Genetics Home Reference provides information about ulcerative colitis. More About This Health Condition  Studies have associated normal variations in the IRF5 gene with an increased risk of several autoimmune disorders. Autoimmune disorders occur when the immune system malfunctions and attacks the body's tissues and organs. These disorders include systemic lupus erythematosus, Sjgren syndrome, and rheumatoid arthritis.There is some evidence that certain variations of the IRF5 gene are associated with increased activity of the gene and elevated cytokines. However, it is unknown what role, if any, these effects play in the increased risk of autoimmune disorders. Researchers believe that a combination of genetic and environmental factors may contribute to the development of these conditions.    Cytogenetic Location: 7q32.1, which is the long (q) arm of chromosome 7 at position 32.1Molecular Location: base pairs 128,937,032 to 128,950,042 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        IRF-5IRF5_HUMANSLEB10     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Immunobiology: The Immune System in Health and Disease (5th edition, 2001): Interferons Induced by Viral Infections Make Several Contributions to Host Defense      Tests Listed in the Genetic Testing Registry (1 link)     Tests of IRF5      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (4 links)     INTERFERON REGULATORY FACTOR 5RHEUMATOID ARTHRITISSJOGREN SYNDROMESYSTEMIC LUPUS ERYTHEMATOSUS, SUSCEPTIBILITY TO, 10      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Immunobiology: The Immune System in Health and Disease (5th edition, 2001): Interferons Induced by Viral Infections Make Several Contributions to Host Defense  Tests of IRF5  PubMed  INTERFERON REGULATORY FACTOR 5RHEUMATOID ARTHRITISSJOGREN SYNDROMESYSTEMIC LUPUS ERYTHEMATOSUS, SUSCEPTIBILITY TO, 10  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Allanore Y, Dieude P, Boileau C. Updating the genetics of systemic sclerosis. Curr Opin Rheumatol. 2010 Nov;22(6):665-70. doi: 10.1097/BOR.0b013e32833d110a. Review. Citation on PubMedBarnes BJ, Moore PA, Pitha PM. Virus-specific activation of a novel interferon regulatory factor, IRF-5, results in the induction of distinct interferon alpha genes. J Biol Chem. 2001 Jun 29;276(26):23382-90. Epub 2001 Apr 12. Citation on PubMedDieud P, Guedj M, Wipff J, Avouac J, Fajardy I, Diot E, Granel B, Sibilia J, Cabane J, Mouthon L, Cracowski JL, Carpentier PH, Hachulla E, Meyer O, Kahan A, Boileau C, Allanore Y. Association between the IRF5 rs2004640 functional polymorphism and systemic sclerosis: a new perspective for pulmonary fibrosis. Arthritis Rheum. 2009 Jan;60(1):225-33. doi: 10.1002/art.24183. Citation on PubMedGraham RR, Kozyrev SV, Baechler EC, Reddy MV, Plenge RM, Bauer JW, Ortmann WA, Koeuth T, Gonzlez Escribano MF; Argentine and Spanish Collaborative Groups, Pons-Estel B, Petri M, Daly M, Gregersen PK, Martn J, Altshuler D, Behrens TW, Alarcn-Riquelme ME. A common haplotype of interferon regulatory factor 5 (IRF5) regulates splicing and expression and is associated with increased risk of systemic lupus erythematosus. Nat Genet. 2006 May;38(5):550-5. Epub 2006 Apr 16. Citation on PubMedOMIM: INTERFERON REGULATORY FACTOR 5Ito I, Kawaguchi Y, Kawasaki A, Hasegawa M, Ohashi J, Hikami K, Kawamoto M, Fujimoto M, Takehara K, Sato S, Hara M, Tsuchiya N. Association of a functional polymorphism in the IRF5 region with systemic sclerosis in a Japanese population. Arthritis Rheum. 2009 Jun;60(6):1845-50. doi: 10.1002/art.24600. Citation on PubMedRadstake TR, Gorlova O, Rueda B, Martin JE, Alizadeh BZ, Palomino-Morales R, Coenen MJ, Vonk MC, Voskuyl AE, Schuerwegh AJ, Broen JC, van Riel PL, van 't Slot R, Italiaander A, Ophoff RA, Riemekasten G, Hunzelmann N, Simeon CP, Ortego-Centeno N, Gonzlez-Gay MA, Gonzlez-Escribano MF; Spanish Scleroderma Group, Airo P, van Laar J, Herrick A, Worthington J, Hesselstrand R, Smith V, de Keyser F, Houssiau F, Chee MM, Madhok R, Shiels P, Westhovens R, Kreuter A, Kiener H, de Baere E, Witte T, Padykov L, Klareskog L, Beretta L, Scorza R, Lie BA, Hoffmann-Vold AM, Carreira P, Varga J, Hinchcliff M, Gregersen PK, Lee AT, Ying J, Han Y, Weng SF, Amos CI, Wigley FM, Hummers L, Nelson JL, Agarwal SK, Assassi S, Gourh P, Tan FK, Koeleman BP, Arnett FC, Martin J, Mayes MD. Genome-wide association study of systemic sclerosis identifies CD247 as a new susceptibility locus. Nat Genet. 2010 May;42(5):426-9. doi: 10.1038/ng.565. Epub 2010 Apr 11. Citation on PubMed or Free article on PubMed CentralRullo OJ, Woo JM, Wu H, Hoftman AD, Maranian P, Brahn BA, McCurdy D, Cantor RM, Tsao BP. Association of IRF5 polymorphisms with activation of the interferon alpha pathway. Ann Rheum Dis. 2010 Mar;69(3):611-7. doi: 10.1136/ard.2009.118315. Epub 2009 Oct 23. Citation on PubMed or Free article on PubMed CentralTakaoka A, Yanai H, Kondo S, Duncan G, Negishi H, Mizutani T, Kano S, Honda K, Ohba Y, Mak TW, Taniguchi T. Integral role of IRF-5 in the gene induction programme activated by Toll-like receptors. Nature. 2005 Mar 10;434(7030):243-9. Epub 2005 Jan 23. Citation on PubMed \n",
            "CPA5\t\n",
            "COPG2\t\n",
            "PODXL\t\n",
            "CALD1\t\n",
            "CREB3L2\t\n",
            "SVOPL\t\n",
            "KIAA1549\t\n",
            "TRBV10-1\t\n",
            "TRBV20-1\t\n",
            "TRBC2\t\n",
            "EPHB6\t\n",
            "CLCN1\t   The CLCN1 gene belongs to the CLC family of genes, which provide instructions for making chloride channels. These channels, which transport negatively charged chlorine atoms (chloride ions), play a key role in a cell's ability to generate and transmit electrical signals. Some CLC channels regulate the flow of chloride ions across cell membranes, while others transport chloride ions within cells.The CLCN1 gene provides instructions for making a chloride channel called ClC-1.  These channels are abundant in muscles used for movement (skeletal muscles). For the body to move normally, skeletal muscles must tense (contract) and relax in a coordinated way. Muscle contraction and relaxation are controlled by the flow of certain ions into and out of muscle cells. ClC-1 channels, which span the cell membrane, control the flow of chloride ions into these cells.  This influx stabilizes the cells' electrical charge, which prevents muscles from contracting abnormally.ClC-1 channels are made of two identical protein subunits, each produced from the CLCN1 gene. Although each subunit forms a separate opening (pore) that allows chloride ions to pass through, the two proteins work together to regulate the flow of chloride ions into skeletal muscle cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Myotonia congenita     More than 80 mutations in the CLCN1 gene have been identified in people with myotonia congenita. Most of these mutations cause the autosomal recessive form of the disorder, which is known as Becker disease. Autosomal recessive inheritance means two copies of the gene in each cell are altered. Becker disease results when CLCN1 mutations change the structure or function of both protein subunits that make up the ClC-1 channel. The altered channels greatly reduce the flow of chloride ions into skeletal muscle cells, which triggers prolonged muscle contractions.  Abnormally sustained muscle contractions are the hallmark of myotonia.CLCN1 mutations also cause the autosomal dominant form of myotonia congenita, which is known as Thomsen disease. Autosomal dominant inheritance means one copy of the altered gene in each cell is sufficient to cause the disorder. Studies suggest that the CLCN1 mutations responsible for Thomsen disease change one of the two protein subunits that make up the ClC-1 channel. The altered protein takes on new, but harmful, properties that disrupt the ability of both subunits to regulate chloride ion flow. Reduced movement of chloride ions into skeletal muscle cells leads to myotonia, which underlies the stiffness and other muscle problems in people with myotonia congenita.Because several CLCN1 mutations can cause either Becker disease or Thomsen disease, doctors usually rely on characteristic signs and symptoms to distinguish the two forms of myotonia congenita. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 80 mutations in the CLCN1 gene have been identified in people with myotonia congenita. Most of these mutations cause the autosomal recessive form of the disorder, which is known as Becker disease. Autosomal recessive inheritance means two copies of the gene in each cell are altered. Becker disease results when CLCN1 mutations change the structure or function of both protein subunits that make up the ClC-1 channel. The altered channels greatly reduce the flow of chloride ions into skeletal muscle cells, which triggers prolonged muscle contractions.  Abnormally sustained muscle contractions are the hallmark of myotonia.CLCN1 mutations also cause the autosomal dominant form of myotonia congenita, which is known as Thomsen disease. Autosomal dominant inheritance means one copy of the altered gene in each cell is sufficient to cause the disorder. Studies suggest that the CLCN1 mutations responsible for Thomsen disease change one of the two protein subunits that make up the ClC-1 channel. The altered protein takes on new, but harmful, properties that disrupt the ability of both subunits to regulate chloride ion flow. Reduced movement of chloride ions into skeletal muscle cells leads to myotonia, which underlies the stiffness and other muscle problems in people with myotonia congenita.Because several CLCN1 mutations can cause either Becker disease or Thomsen disease, doctors usually rely on characteristic signs and symptoms to distinguish the two forms of myotonia congenita. More About This Health Condition    Cytogenetic Location: 7q34, which is the long (q) arm of chromosome 7 at position 34Molecular Location: base pairs 143,316,126 to 143,352,008 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        chloride channel 1, skeletal muscleChloride channel protein 1Chloride channel protein, skeletal musclechloride channel, voltage-sensitive 1ClC-1CLC1CLCN1_HUMANMGC138361MGC142055skeletal muscle chloride channel 1     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Basic Neurochemistry (sixth edition, 1999): Mutations in the ClC1 muscle Cl- channel produce human diseaseNeuromuscular Disease Center, Washington University: Becker DiseaseNeuromuscular Disease Center, Washington University: Thomsen Disease      Clinical Information from GeneReviews (1 link)     Myotonia Congenita      Tests Listed in the Genetic Testing Registry (1 link)     Tests of CLCN1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     CHLORIDE CHANNEL 1, SKELETAL MUSCLE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry (sixth edition, 1999): Mutations in the ClC1 muscle Cl- channel produce human diseaseNeuromuscular Disease Center, Washington University: Becker DiseaseNeuromuscular Disease Center, Washington University: Thomsen Disease  Myotonia Congenita  Tests of CLCN1  PubMed  CHLORIDE CHANNEL 1, SKELETAL MUSCLE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Colding-Jrgensen E. Phenotypic variability in myotonia congenita. Muscle Nerve. 2005 Jul;32(1):19-34. Review. Citation on PubMedDun M, Colding-Jrgensen E, Grunnet M, Jespersen T, Vissing J, Schwartz M. Difference in allelic expression of the CLCN1 gene and the possible influence on the myotonia congenita phenotype. Eur J Hum Genet. 2004 Sep;12(9):738-43. Citation on PubMedDun M, Colding-Jrgensen E. Myotonia Congenita. 2005 Aug 3 [updated 2015 Aug 6]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1355/ Citation on PubMedGrunnet M, Jespersen T, Colding-Jrgensen E, Schwartz M, Klaerke DA, Vissing J, Olesen SP, Dun M. Characterization of two new dominant ClC-1 channel mutations associated with myotonia. Muscle Nerve. 2003 Dec;28(6):722-32. Citation on PubMedJentsch TJ, Stein V, Weinreich F, Zdebik AA. Molecular structure and physiological function of chloride channels. Physiol Rev. 2002 Apr;82(2):503-68. Review. Erratum in: Physiol Rev. 2003 Apr;83(2):following table of contents. Citation on PubMedPusch M. Myotonia caused by mutations in the muscle chloride channel gene CLCN1. Hum Mutat. 2002 Apr;19(4):423-34. Review. Citation on PubMedZhang J, Bendahhou S, Sanguinetti MC, Ptcek LJ. Functional consequences of chloride channel gene (CLCN1) mutations causing myotonia congenita. Neurology. 2000 Feb 22;54(4):937-42. Citation on PubMedZhang J, George AL Jr, Griggs RC, Fouad GT, Roberts J, Kwieciski H, Connolly AM, Ptcek LJ. Mutations in the human skeletal muscle chloride channel gene (CLCN1) associated with dominant and recessive myotonia congenita. Neurology. 1996 Oct;47(4):993-8. Citation on PubMed \n",
            "FAM131B\t\n",
            "CTAGE15\t\n",
            "TCAF2\t\n",
            "OR2F2\t\n",
            "OR2A5\t\n",
            "OR2A14\t\n",
            "CTAGE8\t\n",
            "ARHGEF5\t\n",
            "NOBOX\t\n",
            "KRBA1\t\n",
            "SSPO\t\n",
            "REPIN1\t\n",
            "GIMAP7\t\n",
            "GIMAP6\t\n",
            "TMEM176B\t\n",
            "ATG9B\t\n",
            "AGAP3\t\n",
            "ASB10\t   From NCBI Gene:The protein encoded by this gene is a member of the ankyrin repeat and SOCS box-containing (ASB) family of proteins. The SOCS box serves to couple suppressor of cytokine signaling (SOCS) proteins and their binding partners with the elongin B and C complex, possibly targeting them for degradation. Multiple alternatively spliced transcript variants have been described for this gene. [provided by RefSeq, Dec 2008]From UniProt:May be a substrate-recognition component of a SCF-like ECS (Elongin-Cullin-SOCS-box protein) E3 ubiquitin-protein ligase complex which mediates the ubiquitination and subsequent proteasomal degradation of target proteins.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Glaucoma 1, open angle, FFrom UniProt:Glaucoma 1, open angle, F (GLC1F): A form of primary open angle glaucoma (POAG). POAG is characterized by a specific pattern of optic nerve and visual field defects. The angle of the anterior chamber of the eye is open, and usually the intraocular pressure is increased. However, glaucoma can occur at any intraocular pressure. The disease is generally asymptomatic until the late stages, by which time significant and irreversible optic nerve damage has already taken place. [MIM:603383]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q36.1, which is the long (q) arm of chromosome 7 at position 36.1Molecular Location: base pairs 151,175,698 to 151,187,832 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GLC1F     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ASB10      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ANKYRIN REPEAT- AND SOCS BOX-CONTAINING PROTEIN 10OMIM: GLAUCOMA 1, OPEN ANGLE, F      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ASB10  OMIM: ANKYRIN REPEAT- AND SOCS BOX-CONTAINING PROTEIN 10OMIM: GLAUCOMA 1, OPEN ANGLE, F  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "IQCA1L\t\n",
            "GALNTL5\t\n",
            "GALNT11\t\n",
            "KMT2C\t   From NCBI Gene:This gene is a member of the myeloid/lymphoid or mixed-lineage leukemia (MLL) family and encodes a nuclear protein with an AT hook DNA-binding domain, a DHHC-type zinc finger, six PHD-type zinc fingers, a SET domain, a post-SET domain and a RING-type zinc finger. This protein is a member of the ASC-2/NCOA6 complex (ASCOM), which possesses histone methylation activity and is involved in transcriptional coactivation. [provided by RefSeq, Jul 2008]From UniProt:Histone methyltransferase. Methylates 'Lys-4' of histone H3. H3 'Lys-4' methylation represents a specific tag for epigenetic transcriptional activation. Central component of the MLL2/3 complex, a coactivator complex of nuclear receptors, involved in transcriptional coactivation. KMT2C/MLL3 may be a catalytic subunit of this complex. May be involved in leukemogenesis and developmental disorder.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:CholangiocarcinomaFrom NCBI Gene:KLEEFSTRA SYNDROME 2From UniProt:Kleefstra syndrome 2 (KLEFS2): A form of Kleefstra syndrome, an autosomal dominant disease characterized by variable mental retardation, psychomotor developmental delay, seizures, behavioral abnormalities, and facial dysmorphisms. [MIM:617768]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q36.1, which is the long (q) arm of chromosome 7 at position 36.1Molecular Location: base pairs 152,134,925 to 152,436,642 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HALRKLEFS2MLL3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of KMT2C      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: KLEEFSTRA SYNDROME 2OMIM: LYSINE-SPECIFIC METHYLTRANSFERASE 2C      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of KMT2C  OMIM: KLEEFSTRA SYNDROME 2OMIM: LYSINE-SPECIFIC METHYLTRANSFERASE 2C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RNF32\t\n",
            "MNX1\t   From NCBI Gene:This gene encodes a nuclear protein, which contains a homeobox domain and is a transcription factor. Mutations in this gene result in Currarino syndrome, an autosomic dominant congenital malformation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2009]From UniProt:Putative transcription factor involved in pancreas development and function.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Currarino triadFrom UniProt:Currarino syndrome (CURRAS): The triad of a presacral tumor, sacral agenesis and anorectal malformation constitutes the Currarino syndrome which is caused by dorsal-ventral patterning defects during embryonic development. The syndrome occurs in the majority of patients as an autosomal dominant trait. [MIM:176450]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q36.3, which is the long (q) arm of chromosome 7 at position 36.3Molecular Location: base pairs 157,004,853 to 157,010,653 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        HB9HLXB9HOXHB9SCRA1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MNX1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CURRARINO SYNDROMEOMIM: MOTOR NEURON AND PANCREAS HOMEOBOX 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MNX1  OMIM: CURRARINO SYNDROMEOMIM: MOTOR NEURON AND PANCREAS HOMEOBOX 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MEOX2\t\n",
            "FERD3L\t\n",
            "MACC1\t\n",
            "STEAP1B\t\n",
            "GRIFIN\t\n",
            "C7orf31\t\n",
            "NPVF\t\n",
            "LFNG\t   From NCBI Gene:This gene is a member of the glycosyltransferase 31 gene family. Members of this gene family, which also includes the MFNG (GeneID: 4242) and RFNG (GeneID: 5986) genes, encode evolutionarily conserved glycosyltransferases that act in the Notch signaling pathway to define boundaries during embryonic development. While their genomic structure is distinct from other glycosyltransferases, these proteins have a fucose-specific beta-1,3-N-acetylglucosaminyltransferase activity that leads to elongation of O-linked fucose residues on Notch, which alters Notch signaling. The protein encoded by this gene is predicted to be a single-pass type II Golgi membrane protein but it may also be secreted and proteolytically processed like the related proteins in mouse and Drosophila (PMID: 9187150). Mutations in this gene have been associated with autosomal recessive spondylocostal dysostosis 3. [provided by RefSeq, May 2018]From UniProt:Glycosyltransferase that initiates the elongation of O-linked fucose residues attached to EGF-like repeats in the extracellular domain of Notch molecules. Modulates NOTCH1 activity by modifying O-fucose residues at specific EGF-like domains resulting in inhibition of NOTCH1 activation by JAG1 and enhancement of NOTCH1 activation by DLL1 via an increase in its binding to DLL1 (By similarity). Decreases the binding of JAG1 to NOTCH2 but not that of DLL1 (PubMed:11346656). Essential mediator of somite segmentation and patterning (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Spondylocostal dysostosisFrom NCBI Gene:Spondylocostal dysostosis 3From UniProt:Spondylocostal dysostosis 3, autosomal recessive (SCDO3): A condition of variable severity associated with vertebral and rib segmentation defects. The main skeletal malformations include fusion of vertebrae, hemivertebrae, fusion of certain ribs, and other rib malformations. Deformity of the chest and spine (severe scoliosis, kyphoscoliosis and lordosis) is a natural consequence of the malformation and leads to a dwarf-like appearance. As the thorax is small, infants frequently have respiratory insufficiency and repeated respiratory infections resulting in life-threatening complications in the first year of life. [MIM:609813]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7p22.3, which is the short (p) arm of chromosome 7 at position 22.3Molecular Location: base pairs 2,512,529 to 2,529,177 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        SCDO3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of LFNG      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: LUNATIC FRINGEOMIM: SPONDYLOCOSTAL DYSOSTOSIS 3, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of LFNG  OMIM: LUNATIC FRINGEOMIM: SPONDYLOCOSTAL DYSOSTOSIS 3, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TAX1BP1\t\n",
            "FAM20C\t   From NCBI Gene:This gene encodes a member of the family of secreted protein kinases. The encoded protein binds calcium and phosphorylates proteins involved in bone mineralization. Mutations in this gene are associated with the autosomal recessive disorder Raine syndrome. [provided by RefSeq, Apr 2014]From UniProt:Golgi serine/threonine protein kinase that phosphorylates secretory pathway proteins within Ser-x-Glu/pSer motifs and plays a key role in biomineralization of bones and teeth (PubMed:22582013, PubMed:23754375, PubMed:25789606). Constitutes the main protein kinase for extracellular proteins, generating the majority of the extracellular phosphoproteome (PubMed:26091039). Mainly phosphorylates proteins within the Ser-x-Glu/pSer motif, but also displays a broader substrate specificity (PubMed:26091039). Phosphorylates casein as well as a number of proteins involved in biomineralization such as AMELX, AMTN, ENAM and SPP1 (PubMed:22582013, PubMed:25789606). In addition to its role in biomineralization, also plays a role in lipid homeostasis, wound healing and cell migration and adhesion (PubMed:26091039).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Raine syndromeFrom UniProt:Raine syndrome (RNS): Autosomal recessive osteosclerotic bone dysplasia with neonatal lethal outcome. Clinical features include generalized osteosclerosis, craniofacial dysplasia and microcephaly. [MIM:259775]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7p22.3, which is the short (p) arm of chromosome 7 at position 22.3Molecular Location: base pairs 192,939 to 260,774 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DMP-4DMP4G-CKGEF-CKRNS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FAM20C      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: FAMILY WITH SEQUENCE SIMILARITY 20, MEMBER COMIM: RAINE SYNDROME      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FAM20C  OMIM: FAMILY WITH SEQUENCE SIMILARITY 20, MEMBER COMIM: RAINE SYNDROME  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TRIL\t\n",
            "INMT\t\n",
            "NPSR1\t   From NCBI Gene:This gene encodes a member of the vasopressin/oxytocin subfamily of G protein-coupled receptors. The encoded membrane protein acts as a receptor for neuropeptide S and affects a variety of cellular processes through its signaling. Increased expression of this gene in ciliated cells of the respiratory epithelium and in bronchial smooth muscle cells is associated with asthma. Polymorphisms in this gene have also been associated with asthma susceptibility, panic disorders, inflammatory bowel disease, and rheumatoid arthritis. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2014]From UniProt:G-protein coupled receptor for neuropeptide S (NPS) (PubMed:16790440). Promotes mobilization of intracellular Ca(2+) stores (PubMed:16790440). Inhibits cell growth in response to NPS binding (PubMed:15947423). Involved in pathogenesis of asthma and other IgE-mediated diseases.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Asthma-related traits, susceptibility to, 2From UniProt:Asthma-related traits 2 (ASRT2): Asthma-related traits include clinical symptoms of asthma, such as coughing, wheezing, dyspnea, bronchial hyperresponsiveness as assessed by methacholine challenge test, serum IgE levels, atopy and atopic dermatitis. [MIM:608584]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7p14.3, which is the short (p) arm of chromosome 7 at position 14.3Molecular Location: base pairs 34,658,239 to 34,878,332 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ASRT2GPR154GPRANPSRPGR14VRR1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NPSR1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ASTHMA-RELATED TRAITS, SUSCEPTIBILITY TO, 2OMIM: NEUROPEPTIDE S RECEPTOR 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NPSR1  OMIM: ASTHMA-RELATED TRAITS, SUSCEPTIBILITY TO, 2OMIM: NEUROPEPTIDE S RECEPTOR 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ANLN\t   From NCBI Gene:This gene encodes an actin-binding protein that plays a role in cell growth and migration, and in cytokinesis. The encoded protein is thought to regulate actin cytoskeletal dynamics in podocytes, components of the glomerulus. Mutations in this gene are associated with focal segmental glomerulosclerosis 8. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Oct 2014]From UniProt:Required for cytokinesis (PubMed:16040610). Essential for the structural integrity of the cleavage furrow and for completion of cleavage furrow ingression. Plays a role in bleb assembly during metaphase and anaphase of mitosis (PubMed:23870127). May play a significant role in podocyte cell migration (PubMed:24676636).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Focal segmental glomerulosclerosis 8From UniProt:Focal segmental glomerulosclerosis 8 (FSGS8): A renal pathology defined by the presence of segmental sclerosis in glomeruli and resulting in proteinuria, reduced glomerular filtration rate and progressive decline in renal function. Renal insufficiency often progresses to end-stage renal disease, a highly morbid state requiring either dialysis therapy or kidney transplantation. [MIM:616032]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7p14.2, which is the short (p) arm of chromosome 7 at position 14.2Molecular Location: base pairs 36,389,803 to 36,453,791 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FSGS8scraScraps     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ANLN      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ACTIN-BINDING PROTEIN ANILLINOMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 8      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ANLN  OMIM: ACTIN-BINDING PROTEIN ANILLINOMIM: FOCAL SEGMENTAL GLOMERULOSCLEROSIS 8  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "AOAH\t\n",
            "TRGC2\t\n",
            "TRGC1\t\n",
            "TRGV11\t\n",
            "VPS41\t\n",
            "POU6F2\t   From NCBI Gene:This gene encodes a member of the POU protein family characterized by the presence of a bipartite DNA binding domain, consisting of a POU-specific domain and a homeodomain, separated by a variable polylinker. The DNA binding domain may bind to DNA as monomers or as homo- and/or heterodimers, in a sequence-specific manner. The POU family members are transcriptional regulators, many of which are known to control cell type-specific differentiation pathways. This gene is a tumor suppressor involved in Wilms tumor (WT) predisposition. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene.[provided by RefSeq, Oct 2009]From UniProt:Probable transcription factor likely to be involved in early steps in the differentiation of amacrine and ganglion cells. Recognizes and binds to the DNA sequence 5'-ATGCAAAT-3'. Isoform 1 does not bind DNA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Wilms tumorFrom NCBI Gene:Wilms tumor and radial bilateral aplasiaFrom UniProt:Hereditary susceptibility to Wilms tumor 5 (WT5): Pediatric malignancy of kidney and one of the most common solid cancers in childhood. [MIM:601583]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7p14.1, which is the short (p) arm of chromosome 7 at position 14.1Molecular Location: base pairs 38,977,878 to 39,468,601 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        RPF-1WT5WTSL     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of POU6F2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: POU DOMAIN, CLASS 6, TRANSCRIPTION FACTOR 2OMIM: WILMS TUMOR 5      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of POU6F2  OMIM: POU DOMAIN, CLASS 6, TRANSCRIPTION FACTOR 2OMIM: WILMS TUMOR 5  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "COA1\t\n",
            "TNS3\t\n",
            "AP5Z1\t   From NCBI Gene:This gene was identified by genome-wide screen for genes involved in homologous recombination DNA double-strand break repair (HR-DSBR). The encoded protein was found in a complex with other proteins that have a role in HR-DSBR. Knockdown of this gene reduced homologous recombination, and mutations in this gene were found in patients with spastic paraplegia. It was concluded that this gene likely encodes a helicase (PMID:20613862). [provided by RefSeq, Jan 2011]From UniProt:As part of AP-5, a probable fifth adaptor protein complex it may be involved in endosomal transport. According to PubMed:20613862 it is a putative helicase required for efficient homologous recombination DNA double-strand break repair.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Spastic paraplegia 48, autosomal recessiveFrom UniProt:Spastic paraplegia 48, autosomal recessive (SPG48): A form of spastic paraplegia, a neurodegenerative disorder characterized by a slow, gradual, progressive weakness and spasticity of the lower limbs. Rate of progression and the severity of symptoms are quite variable. Initial symptoms may include difficulty with balance, weakness and stiffness in the legs, muscle spasms, and dragging the toes when walking. In some forms of the disorder, bladder symptoms (such as incontinence) may appear, or the weakness and stiffness may spread to other parts of the body. [MIM:613647]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7p22.1, which is the short (p) arm of chromosome 7 at position 22.1Molecular Location: base pairs 4,775,617 to 4,794,395 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        KIAA0415SPG48zeta     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AP5Z1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ADAPTOR-RELATED PROTEIN COMPLEX 5, ZETA-1 SUBUNITOMIM: SPASTIC PARAPLEGIA 48, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AP5Z1  OMIM: ADAPTOR-RELATED PROTEIN COMPLEX 5, ZETA-1 SUBUNITOMIM: SPASTIC PARAPLEGIA 48, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TNRC18\t\n",
            "PMS2\t   The PMS2 gene provides instructions for making a protein that plays an essential role in repairing DNA. This protein helps fix errors that are made when DNA is copied (DNA replication) in preparation for cell division. The PMS2 protein joins with another protein called MLH1 (produced from the MLH1 gene) to form a protein complex. This complex coordinates the activities of other proteins that repair errors made during DNA replication. Repairs are made by removing the section of DNA that contains errors and replacing it with a corrected DNA sequence. The PMS2 gene is a member of a set of genes known as the mismatch repair (MMR) genes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Lynch syndrome     Mutations in the PMS2 gene have been reported in about 2 percent of families with Lynch syndrome that have an identified gene mutation. Lynch syndrome increases the risk of many types of cancer, particularly cancers of the colon (large intestine) and rectum, which are collectively referred to as colorectal cancer. People with Lynch syndrome also have an increased risk of cancers of the endometrium (lining of the uterus), ovaries, stomach, small intestine, liver, gallbladder duct, upper urinary tract, and brain. PMS2 gene mutations involved in this condition lead to the production of an abnormally short or inactive PMS2 protein that cannot efficiently repair errors made during DNA replication. The errors accumulate as the cells continue to divide, which may cause the cells to function abnormally, increasing the risk of tumor formation in the colon or another part of the body.Some mutations in the PMS2 gene can cause a variant of Lynch syndrome called Turcot syndrome. In addition to colorectal cancer, people with Turcot syndrome tend to develop a particular type of brain tumor called a glioblastoma. More About This Health Condition      Alopecia areata     Genetics Home Reference provides information about alopecia areata. More About This Health Condition      Ovarian cancer     Genetics Home Reference provides information about ovarian cancer. More About This Health Condition      Other cancers     While Lynch syndrome is associated with a mutation in one copy of the PMS2 gene, very rarely, individuals in affected families inherit two PMS2 gene mutations, one from each parent. Most often in these cases, the same mutation occurs in both copies of the gene (a homozygous mutation). People with a homozygous PMS2 gene mutation have a syndrome distinct from Lynch syndrome. In addition to colorectal cancer, these individuals may develop cancers of the blood (leukemia or lymphoma). Some of these individuals will also develop characteristic features of a condition known as neurofibromatosis, including noncancerous tumors that grow along nerves (neurofibromas) and light brown patches of skin called caf-au-lait spots. The onset of colon cancer in these individuals is extremely early, often occurring during childhood. This syndrome involving colon cancer, leukemia or lymphoma, and neurofibromatosis is sometimes called CoLoN.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the PMS2 gene have been reported in about 2 percent of families with Lynch syndrome that have an identified gene mutation. Lynch syndrome increases the risk of many types of cancer, particularly cancers of the colon (large intestine) and rectum, which are collectively referred to as colorectal cancer. People with Lynch syndrome also have an increased risk of cancers of the endometrium (lining of the uterus), ovaries, stomach, small intestine, liver, gallbladder duct, upper urinary tract, and brain. PMS2 gene mutations involved in this condition lead to the production of an abnormally short or inactive PMS2 protein that cannot efficiently repair errors made during DNA replication. The errors accumulate as the cells continue to divide, which may cause the cells to function abnormally, increasing the risk of tumor formation in the colon or another part of the body.Some mutations in the PMS2 gene can cause a variant of Lynch syndrome called Turcot syndrome. In addition to colorectal cancer, people with Turcot syndrome tend to develop a particular type of brain tumor called a glioblastoma. More About This Health Condition  Genetics Home Reference provides information about alopecia areata. More About This Health Condition  Genetics Home Reference provides information about ovarian cancer. More About This Health Condition  While Lynch syndrome is associated with a mutation in one copy of the PMS2 gene, very rarely, individuals in affected families inherit two PMS2 gene mutations, one from each parent. Most often in these cases, the same mutation occurs in both copies of the gene (a homozygous mutation). People with a homozygous PMS2 gene mutation have a syndrome distinct from Lynch syndrome. In addition to colorectal cancer, these individuals may develop cancers of the blood (leukemia or lymphoma). Some of these individuals will also develop characteristic features of a condition known as neurofibromatosis, including noncancerous tumors that grow along nerves (neurofibromas) and light brown patches of skin called caf-au-lait spots. The onset of colon cancer in these individuals is extremely early, often occurring during childhood. This syndrome involving colon cancer, leukemia or lymphoma, and neurofibromatosis is sometimes called CoLoN.    Cytogenetic Location: 7p22.1, which is the short (p) arm of chromosome 7 at position 22.1Molecular Location: base pairs 5,970,925 to 6,009,106 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DNA mismatch repair gene homologueHNPCC4MLH4PMS2-C terminal -likePMS2 postmeiotic segregation increased 2 (S. cerevisiae)PMS2_HUMANPMS2CLPMSL2postmeiotic segregation increased (S. cerevisiae) 2     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Cancer Medicine (sixth edition, 2003): DNA Mismatch Repair Gene Defects and HNPCCMolecular Biology of the Cell (fourth edition, 2002): Defects in DNA Mismatch Repair Provide an Alternative Route to Colorectal Cancer      Clinical Information from GeneReviews (1 link)     Lynch Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PMS2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (3 links)     MISMATCH REPAIR CANCER SYNDROMENEUROFIBROMATOSIS, TYPE IPOSTMEIOTIC SEGREGATION INCREASED, S. CEREVISIAE, 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Cancer Medicine (sixth edition, 2003): DNA Mismatch Repair Gene Defects and HNPCCMolecular Biology of the Cell (fourth edition, 2002): Defects in DNA Mismatch Repair Provide an Alternative Route to Colorectal Cancer  Lynch Syndrome  Tests of PMS2  PubMed  MISMATCH REPAIR CANCER SYNDROMENEUROFIBROMATOSIS, TYPE IPOSTMEIOTIC SEGREGATION INCREASED, S. CEREVISIAE, 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bandipalliam P. Syndrome of early onset colon cancers, hematologic malignancies & features of neurofibromatosis in HNPCC families with homozygous mismatch repair gene mutations. Fam Cancer. 2005;4(4):323-33. Review. Citation on PubMedGill S, Lindor NM, Burgart LJ, Smalley R, Leontovich O, French AJ, Goldberg RM, Sargent DJ, Jass JR, Hopper JL, Jenkins MA, Young J, Barker MA, Walsh MD, Ruszkiewicz AR, Thibodeau SN. Isolated loss of PMS2 expression in colorectal cancers: frequency, patient age, and familial aggregation. Clin Cancer Res. 2005 Sep 15;11(18):6466-71. Citation on PubMedHendriks YM, Jagmohan-Changur S, van der Klift HM, Morreau H, van Puijenbroek M, Tops C, van Os T, Wagner A, Ausems MG, Gomez E, Breuning MH, Brcker-Vriends AH, Vasen HF, Wijnen JT. Heterozygous mutations in PMS2 cause hereditary nonpolyposis colorectal carcinoma (Lynch syndrome). Gastroenterology. 2006 Feb;130(2):312-22. Citation on PubMedKohlmann W, Gruber SB. Lynch Syndrome. 2004 Feb 5 [updated 2014 May 22]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1211/ Citation on PubMedLynch HT, de la Chapelle A. Hereditary colorectal cancer. N Engl J Med. 2003 Mar 6;348(10):919-32. Review. Citation on PubMedMarti TM, Kunz C, Fleck O. DNA mismatch repair and mutation avoidance pathways. J Cell Physiol. 2002 Apr;191(1):28-41. Review. Citation on PubMedNakagawa H, Lockman JC, Frankel WL, Hampel H, Steenblock K, Burgart LJ, Thibodeau SN, de la Chapelle A. Mismatch repair gene PMS2: disease-causing germline mutations are frequent in patients whose tumors stain negative for PMS2 protein, but paralogous genes obscure mutation detection and interpretation. Cancer Res. 2004 Jul 15;64(14):4721-7. Citation on PubMedOMIM: POSTMEIOTIC SEGREGATION INCREASED, S. CEREVISIAE, 2Peltomki P. Lynch syndrome genes. Fam Cancer. 2005;4(3):227-32. Review. Citation on PubMedSantucci-Darmanin S, Paquis-Flucklinger V. [Homologs of MutS and MutL during mammalian meiosis]. Med Sci (Paris). 2003 Jan;19(1):85-91. Review. French. Citation on PubMedSenter L, Clendenning M, Sotamaa K, Hampel H, Green J, Potter JD, Lindblom A, Lagerstedt K, Thibodeau SN, Lindor NM, Young J, Winship I, Dowty JG, White DM, Hopper JL, Baglietto L, Jenkins MA, de la Chapelle A. The clinical phenotype of Lynch syndrome due to germ-line PMS2 mutations. Gastroenterology. 2008 Aug;135(2):419-28. doi: 10.1053/j.gastro.2008.04.026. Epub 2008 May 2. Citation on PubMed or Free article on PubMed CentralTruninger K, Menigatti M, Luz J, Russell A, Haider R, Gebbers JO, Bannwart F, Yurtsever H, Neuweiler J, Riehle HM, Cattaruzza MS, Heinimann K, Schr P, Jiricny J, Marra G. Immunohistochemical analysis reveals high frequency of PMS2 defects in colorectal cancer. Gastroenterology. 2005 May;128(5):1160-71. Citation on PubMedVaughn CP, Baker CL, Samowitz WS, Swensen JJ. The frequency of previously undetectable deletions involving 3' Exons of the PMS2 gene. Genes Chromosomes Cancer. 2013 Jan;52(1):107-12. doi: 10.1002/gcc.22011. Epub 2012 Sep 25. Citation on PubMed \n",
            "FAM220A\t\n",
            "ZNF736\t\n",
            "ZNF680\t\n",
            "ZNF117\t\n",
            "ERV3-1\t\n",
            "VKORC1L1\t\n",
            "POM121\t\n",
            "GTF2IRD2\t   From NCBI Gene:This gene is one of several closely related genes on chromosome 7 encoding proteins containing helix-loop-helix motifs. These proteins may function as regulators of transcription. The encoded protein is unique in that its C-terminus is derived from CHARLIE8 transposable element sequence. This gene is located in a region of chromosome 7 that is deleted in Williams-Beuren syndrome, and loss of this locus may contribute to the cognitive phenotypes observed in this disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Williams syndromeFrom UniProt:GTF2IRD2 is located in the Williams-Beuren syndrome (WBS) critical region. WBS results from a hemizygous deletion of several genes on chromosome 7q11.23, thought to arise as a consequence of unequal crossing over between highly homologous low-copy repeat sequences flanking the deleted region (PubMed:16532385).     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q11.23, which is the long (q) arm of chromosome 7 at position 11.23Molecular Location: base pairs 74,796,144 to 74,851,576 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FP630GTF2IRD2 alphaGTF2IRD2A     Related Information   How are genetic conditions and genes named?          Catalog of Genes and Diseases from OMIM (1 link)     OMIM: GTF2I REPEAT DOMAIN-CONTAINING PROTEIN 2, ALPHA      Research Resources (4 links)     HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      OMIM: GTF2I REPEAT DOMAIN-CONTAINING PROTEIN 2, ALPHA  HGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TRIM73\t\n",
            "SPDYE5\t\n",
            "STYXL1\t\n",
            "SRRM3\t\n",
            "CD36\t   From NCBI Gene:The protein encoded by this gene is the fourth major glycoprotein of the platelet surface and serves as a receptor for thrombospondin in platelets and various cell lines. Since thrombospondins are widely distributed proteins involved in a variety of adhesive processes, this protein may have important functions as a cell adhesion molecule. It binds to collagen, thrombospondin, anionic phospholipids and oxidized LDL. It directly mediates cytoadherence of Plasmodium falciparum parasitized erythrocytes and it binds long chain fatty acids and may function in the transport and/or as a regulator of fatty acid transport. Mutations in this gene cause platelet glycoprotein deficiency. Multiple alternatively spliced transcript variants have been found for this gene. [provided by RefSeq, Feb 2014]From UniProt:Multifunctional glycoprotein that acts as receptor for a broad range of ligands. Ligands can be of proteinaceous nature like thrombospondin, fibronectin, collagen or amyloid-beta as well as of lipidic nature such as oxidized low-density lipoprotein (oxLDL), anionic phospholipids, long-chain fatty acids and bacterial diacylated lipopeptides. They are generally multivalent and can therefore engage multiple receptors simultaneously, the resulting formation of CD36 clusters initiates signal transduction and internalization of receptor-ligand complexes. The dependency on coreceptor signaling is strongly ligand specific. Cellular responses to these ligands are involved in angiogenesis, inflammatory response, fatty acid metabolism, taste and dietary fat processing in the intestine (Probable). Binds long-chain fatty acids and facilitates their transport into cells, thus participating in muscle lipid utilization, adipose energy storage, and gut fat absorption (By similarity) (PubMed:18353783, PubMed:21610069). In the small intestine, plays a role in proximal absorption of dietary fatty acid and cholesterol for optimal chylomicron formation, possibly through the activation of MAPK1/3 (ERK1/2) signaling pathway (By similarity) (PubMed:18753675). Involved in oral fat perception and preferences (PubMed:22240721, PubMed:25822988). Detection into the tongue of long-chain fatty acids leads to a rapid and sustained rise in flux and protein content of pancreatobiliary secretions (By similarity). In taste receptor cells, mediates the induction of an increase in intracellular calcium levels by long-chain fatty acids, leading to the activation of the gustatory neurons in the nucleus of the solitary tract (By similarity). Important factor in both ventromedial hypothalamus neuronal sensing of long-chain fatty acid and the regulation of energy and glucose homeostasis (By similarity). Receptor for thombospondins, THBS1 and THBS2, mediating their antiangiogenic effects (By similarity). As a coreceptor for TLR4:TLR6 heterodimer, promotes inflammation in monocytes/macrophages. Upon ligand binding, such as oxLDL or amyloid-beta 42, interacts with the heterodimer TLR4:TLR6, the complex is internalized and triggers inflammatory response, leading to NF-kappa-B-dependent production of CXCL1, CXCL2 and CCL9 cytokines, via MYD88 signaling pathway, and CCL5 cytokine, via TICAM1 signaling pathway, as well as IL1B secretion, through the priming and activation of the NLRP3 inflammasome (By similarity) (PubMed:20037584). Selective and nonredundant sensor of microbial diacylated lipopeptide that signal via TLR2:TLR6 heterodimer, this cluster triggers signaling from the cell surface, leading to the NF-kappa-B-dependent production of TNF, via MYD88 signaling pathway and subsequently is targeted to the Golgi in a lipid-raft dependent pathway (By similarity) (PubMed:16880211).(Microbial infection) Directly mediates cytoadherence of Plasmodium falciparum parasitized erythrocytes and the internalization of particles independently of TLR signaling.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Platelet glycoprotein IV deficiencyCoronary heart disease 7Susceptibility to malariaFrom UniProt:Platelet glycoprotein IV deficiency (PG4D): A disorder characterized by macrothrombocytopenia without notable hemostatic problems and bleeding tendency. Platelet glycoprotein IV deficiency can be divided into 2 subgroups. The type I phenotype is characterized by platelets and monocytes/macrophages exhibiting complete CD36 deficiency. The type II phenotype lacks the surface expression of CD36 in platelets, but expression in monocytes/macrophages is near normal. [MIM:608404]Coronary heart disease 7 (CHDS7): A multifactorial disease characterized by an imbalance between myocardial functional requirements and the capacity of the coronary vessels to supply sufficient blood flow. Decreased capacity of the coronary vessels is often associated with thickening and loss of elasticity of the coronary arteries. [MIM:610938]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q21.11, which is the long (q) arm of chromosome 7 at position 21.11Molecular Location: base pairs 80,602,188 to 80,679,277 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BDPLT10CHDS7FATGP3BGP4GPIVPASIVSCARB3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CD36      Catalog of Genes and Diseases from OMIM (4 links)     OMIM: CD36 ANTIGENOMIM: CORONARY HEART DISEASE, SUSCEPTIBILITY TO, 7OMIM: MALARIA, SUSCEPTIBILITY TOOMIM: PLATELET GLYCOPROTEIN IV DEFICIENCY      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CD36  OMIM: CD36 ANTIGENOMIM: CORONARY HEART DISEASE, SUSCEPTIBILITY TO, 7OMIM: MALARIA, SUSCEPTIBILITY TOOMIM: PLATELET GLYCOPROTEIN IV DEFICIENCY  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PCLO\t   From NCBI Gene:The protein encoded by this gene is part of the presynaptic cytoskeletal matrix, which is involved in establishing active synaptic zones and in synaptic vesicle trafficking. Variations in this gene have been associated with bipolar disorder and major depressive disorder. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2011]From UniProt:Scaffold protein of the presynaptic cytomatrix at the active zone (CAZ) which is the place in the synapse where neurotransmitter is released (By similarity). After synthesis, participates in the formation of Golgi-derived membranous organelles termed Piccolo-Bassoon transport vesicles (PTVs) that are transported along axons to sites of nascent synaptic contacts (By similarity). At the presynaptic active zone, regulates the spatial organization of synaptic vesicle cluster, the protein complexes that execute membrane fusion and compensatory endocytosis (By similarity). Organizes as well the readily releasable pool of synaptic vesicles and safeguards a fraction of them to be not immediately available for action potential-induced release (By similarity). Functions also in processes other than assembly such as the regulation of specific presynaptic protein ubiquitination by interacting with SIAH1 or the regulation of presynaptic autophagy (By similarity). Mediates also synapse to nucleus communication leading to reconfiguration of gene expression by associating with the transcriptional corepressor CTBP1 and by subsequently reducing the size of its pool available for nuclear import (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Pontocerebellar hypoplasia type 3From UniProt:Pontocerebellar hypoplasia 3 (PCH3): A form of pontocerebellar hypoplasia, a disorder characterized by structural defects of the pons and cerebellum. Brain MRI shows an abnormally small cerebellum and brainstem, decreased cerebral white matter, and a thin corpus callosum. PCH3 features include seizures, short stature, optic atrophy, progressive microcephaly, severe developmental delay. [MIM:608027]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q21.11, which is the long (q) arm of chromosome 7 at position 21.11Molecular Location: base pairs 82,754,005 to 83,164,161 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ACZPCH3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PCLO      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: PICCOLO, MOUSE, HOMOLOG OFOMIM: PONTOCEREBELLAR HYPOPLASIA, TYPE 3      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PCLO  OMIM: PICCOLO, MOUSE, HOMOLOG OFOMIM: PONTOCEREBELLAR HYPOPLASIA, TYPE 3  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SUN1\t\n",
            "STEAP2\t\n",
            "CFAP69\t\n",
            "GTPBP10\t\n",
            "AKAP9\t   From NCBI Gene:The A-kinase anchor proteins (AKAPs) are a group of structurally diverse proteins which have the common function of binding to the regulatory subunit of protein kinase A (PKA) and confining the holoenzyme to discrete locations within the cell. This gene encodes a member of the AKAP family. Alternate splicing of this gene results in at least two isoforms that localize to the centrosome and the Golgi apparatus, and interact with numerous signaling proteins from multiple signal transduction pathways. These signaling proteins include type II protein kinase A, serine/threonine kinase protein kinase N, protein phosphatase 1, protein phosphatase 2a, protein kinase C-epsilon and phosphodiesterase 4D3. [provided by RefSeq, Aug 2008]From UniProt:Scaffolding protein that assembles several protein kinases and phosphatases on the centrosome and Golgi apparatus. Required to maintain the integrity of the Golgi apparatus (PubMed:10202149, PubMed:15047863). Required for microtubule nucleation at the cis-side of the Golgi apparatus (PubMed:15047863, PubMed:19242490). Required for association of the centrosomes with the poles of the bipolar mitotic spindle during metaphase (PubMed:25657325). In complex with PDE4DIP isoform 13/MMG8/SMYLE, recruits CAMSAP2 to the Golgi apparatus and tethers non-centrosomal minus-end microtubules to the Golgi, an important step for polarized cell movement (PubMed:27666745, PubMed:28814570). In complex with PDE4DIP isoform 13/MMG8/SMYLE, EB1/MAPRE1 and CDK5RAP2, contributes to microtubules nucleation and extension also from the centrosome to the cell periphery (PubMed:29162697).Isoform 4: Associated with the N-methyl-D-aspartate receptor and is specifically found in the neuromuscular junction (NMJ) as well as in neuronal synapses, suggesting a role in the organization of postsynaptic specializations.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Romano-Ward syndromeFrom NCBI Gene:Long QT syndrome 11From UniProt:Long QT syndrome 11 (LQT11): A heart disorder characterized by a prolonged QT interval on the ECG and polymorphic ventricular arrhythmias. They cause syncope and sudden death in response to exercise or emotional stress, and can present with a sentinel event of sudden cardiac death in infancy. [MIM:611820]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 7q21.2, which is the long (q) arm of chromosome 7 at position 21.2Molecular Location: base pairs 91,940,867 to 92,112,908 on chromosome 7 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AKAP-9AKAP350AKAP450CG-NAPHYPERIONLQT11MU-RMS-40.16APPP1R45PRKA9YOTIAO     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AKAP9      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: A-KINASE ANCHOR PROTEIN 9OMIM: LONG QT SYNDROME 11      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AKAP9  OMIM: A-KINASE ANCHOR PROTEIN 9OMIM: LONG QT SYNDROME 11  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZNF3\t\n",
            "PVRIG\t\n",
            "VPS13B\t   Researchers are still working to determine the exact role of the VPS13B gene (also called the COH1 gene) in the human body. Studies show that the protein produced from this gene is a part of the Golgi apparatus, which is a cell structure in which newly produced proteins are modified so they can carry out their functions. In particular, the VPS13B protein is involved in a modification called glycosylation, which is the attachment of sugar molecules to proteins. The VPS13B protein also appears to be involved in the sorting and transporting of proteins inside the cell.Studies suggest several functions for the VPS13B protein in the body. The protein appears to play an important role in the normal growth and development of nerve cells (neurons). It may also be involved in the growth and development of adipocytes, which are cells that store fats for energy, and may play a role in the storage and distribution of fats in the body.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Cohen syndrome     More than 150 different mutations in the VPS13B gene have been found in individuals with Cohen syndrome. Individuals with this condition typically have intellectual disability, eye problems, and distinctive facial features. Another common feature of the condition is obesity, particularly around the torso but not the arms and legs (truncal obesity). Most of the mutations that cause Cohen syndrome result in a premature stop signal in the instructions for making the VPS13B protein. Researchers believe that this genetic change leads to the production of an abnormally short, nonfunctional version of the protein. Studies suggest that a loss of this protein's function disrupts the organization of the Golgi apparatus and impairs normal glycosylation. However, it is not known how the absence of functional VPS13B protein or these cellular changes lead to the signs and symptoms of Cohen syndrome. Researchers speculate that problems with neuron development underlie intellectual disability, eye problems and other features of Cohen syndrome. Abnormal fat storage may cause truncal obesity in people with the condition.In the Finnish population, 75 percent of individuals with Cohen syndrome have a mutation in both copies of the VPS13B gene that deletes two DNA building blocks (base pairs). This mutation is sometimes written as 3348_3349delCT. The deletion causes a premature stop signal in the instructions for making the VPS13B protein.Two common mutations occur in the Old Order Amish population. The first mutation adds one base pair and is sometimes written as 9258_9259insT. This mutation creates a premature stop signal in the instructions for making the VPS13B protein. The second mutation changes a single protein building block (amino acid) in the VPS13B protein. Specifically, this mutation replaces the amino acid isoleucine with the amino acid threonine at position 2820 (written as Ile2820Thr or I2820T). Outside the Finnish and Amish populations, nearly all mutations in the VPS13B gene occur in only one or a small number of families. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 150 different mutations in the VPS13B gene have been found in individuals with Cohen syndrome. Individuals with this condition typically have intellectual disability, eye problems, and distinctive facial features. Another common feature of the condition is obesity, particularly around the torso but not the arms and legs (truncal obesity). Most of the mutations that cause Cohen syndrome result in a premature stop signal in the instructions for making the VPS13B protein. Researchers believe that this genetic change leads to the production of an abnormally short, nonfunctional version of the protein. Studies suggest that a loss of this protein's function disrupts the organization of the Golgi apparatus and impairs normal glycosylation. However, it is not known how the absence of functional VPS13B protein or these cellular changes lead to the signs and symptoms of Cohen syndrome. Researchers speculate that problems with neuron development underlie intellectual disability, eye problems and other features of Cohen syndrome. Abnormal fat storage may cause truncal obesity in people with the condition.In the Finnish population, 75 percent of individuals with Cohen syndrome have a mutation in both copies of the VPS13B gene that deletes two DNA building blocks (base pairs). This mutation is sometimes written as 3348_3349delCT. The deletion causes a premature stop signal in the instructions for making the VPS13B protein.Two common mutations occur in the Old Order Amish population. The first mutation adds one base pair and is sometimes written as 9258_9259insT. This mutation creates a premature stop signal in the instructions for making the VPS13B protein. The second mutation changes a single protein building block (amino acid) in the VPS13B protein. Specifically, this mutation replaces the amino acid isoleucine with the amino acid threonine at position 2820 (written as Ile2820Thr or I2820T). Outside the Finnish and Amish populations, nearly all mutations in the VPS13B gene occur in only one or a small number of families. More About This Health Condition    Cytogenetic Location: 8q22.2, which is the long (q) arm of chromosome 8 at position 22.2Molecular Location: base pairs 99,013,085 to 99,877,586 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CHS1COH1Cohen syndrome 1DKFZp313I0811KIAA0532vacuolar protein sorting 13 homolog B (yeast)vacuolar protein sorting 13BVP13B_HUMAN     Related Information   How are genetic conditions and genes named?          Clinical Information from GeneReviews (1 link)     Cohen Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of VPS13B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     VACUOLAR PROTEIN SORTING 13, YEAST, HOMOLOG OF, B      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Cohen Syndrome  Tests of VPS13B  PubMed  VACUOLAR PROTEIN SORTING 13, YEAST, HOMOLOG OF, B  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Duplomb L, Duvet S, Picot D, Jego G, El Chehadeh-Djebbar S, Marle N, Gigot N, Aral B, Carmignac V, Thevenon J, Lopez E, Rivire JB, Klein A, Philippe C, Droin N, Blair E, Girodon F, Donadieu J, Bellann-Chantelot C, Delva L, Michalski JC, Solary E, Faivre L, Foulquier F, Thauvin-Robinet C. Cohen syndrome is associated with major glycosylation defects. Hum Mol Genet. 2014 May 1;23(9):2391-9. doi: 10.1093/hmg/ddt630. Epub 2013 Dec 13. Citation on PubMedFalk MJ, Feiler HS, Neilson DE, Maxwell K, Lee JV, Segall SK, Robin NH, Wilhelmsen KC, Trskelin AL, Kolehmainen J, Lehesjoki AE, Wiznitzer M, Warman ML. Cohen syndrome in the Ohio Amish. Am J Med Genet A. 2004 Jul 1;128A(1):23-8. Citation on PubMedKolehmainen J, Black GC, Saarinen A, Chandler K, Clayton-Smith J, Trskelin AL, Perveen R, Kivitie-Kallio S, Norio R, Warburg M, Fryns JP, de la Chapelle A, Lehesjoki AE. Cohen syndrome is caused by mutations in a novel gene, COH1, encoding a transmembrane protein with a presumed role in vesicle-mediated sorting and intracellular protein transport. Am J Hum Genet. 2003 Jun;72(6):1359-69. Epub 2003 May 2. Citation on PubMed or Free article on PubMed CentralLimoge F, Faivre L, Gautier T, Petit JM, Gautier E, Masson D, Jego G, El Chehadeh-Djebbar S, Marle N, Carmignac V, Deckert V, Brindisi MC, Edery P, Ghoumid J, Blair E, Lagrost L, Thauvin-Robinet C, Duplomb L. Insulin response dysregulation explains abnormal fat storage and increased risk of diabetes mellitus type 2 in Cohen Syndrome. Hum Mol Genet. 2015 Dec 1;24(23):6603-13. doi: 10.1093/hmg/ddv366. Epub 2015 Sep 10. Citation on PubMedSeifert W, Holder-Espinasse M, Spranger S, Hoeltzenbein M, Rossier E, Dollfus H, Lacombe D, Verloes A, Chrzanowska KH, Maegawa GH, Chitayat D, Kotzot D, Huhle D, Meinecke P, Albrecht B, Mathijssen I, Leheup B, Raile K, Hennies HC, Horn D. Mutational spectrum of COH1 and clinical heterogeneity in Cohen syndrome. J Med Genet. 2006 May;43(5):e22. Citation on PubMed or Free article on PubMed CentralSeifert W, Khnisch J, Maritzen T, Horn D, Haucke V, Hennies HC. Cohen syndrome-associated protein, COH1, is a novel, giant Golgi matrix protein required for Golgi integrity. J Biol Chem. 2011 Oct 28;286(43):37665-75. doi: 10.1074/jbc.M111.267971. Epub 2011 Aug 24. Citation on PubMed or Free article on PubMed CentralSeifert W, Khnisch J, Maritzen T, Lommatzsch S, Hennies HC, Bachmann S, Horn D, Haucke V. Cohen syndrome-associated protein COH1 physically and functionally interacts with the small GTPase RAB6 at the Golgi complex and directs neurite outgrowth. J Biol Chem. 2015 Feb 6;290(6):3349-58. doi: 10.1074/jbc.M114.608174. Epub 2014 Dec 9. Citation on PubMed or Free article on PubMed CentralWang H, Falk MJ, Wensel C, Traboulsi EI. Cohen Syndrome. 2006 Aug 29 [updated 2016 Jul 21]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1482/ Citation on PubMed \n",
            "SNX31\t\n",
            "RRM2B\t   The RRM2B gene provides instructions for making one piece, called the p53 inducible small subunit (p53R2), of a protein called ribonucleotide reductase (RNR). Two copies of the p53R2 subunit are attached to two copies of another protein called R1 to form RNR. (R1 can also attach to another small subunit, called R2, to make another form of RNR). Whether made with p53R2 or R2, RNR helps produce DNA building blocks (nucleotides), which are joined to one another in a particular order to form DNA.RNRs containing p53R2 make nucleotides that are used for the formation of DNA in specialized cell structures called mitochondria. Although most DNA is packaged in chromosomes within the cell's nucleus (nuclear DNA), mitochondria also have a small amount of their own DNA (mitochondrial DNA or mtDNA). Mitochondria are the energy-producing centers in cells, and the DNA in these structures contains genes essential for the process of energy production (called oxidative phosphorylation). The production of nucleotides by p53R2 also helps maintain a normal amount of mtDNA in cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Progressive external ophthalmoplegia     At least 17 mutations in the RRM2B gene have been identified in people with an eye condition called progressive external ophthalmoplegia. This disorder weakens the muscles that control eye movement and causes the eyelids to droop (ptosis). Some affected individuals have additional signs and symptoms, such as weakness of other muscles, extreme tiredness (fatigue), hearing loss caused by problems with the inner ear (sensorineural hearing loss), and digestive problems.Typically, mutations that cause progressive external ophthalmoplegia occur in one copy of the RRM2B gene, although rarely both copies of the gene are altered. RRM2B gene mutations associated with progressive external ophthalmoplegia lead to impaired RNR activity. These mutations result in large deletions of genetic material from mtDNA in muscle tissue, possibly because impairment of RNR activity leads to a shortage of nucleotides, although the mechanism is unclear. Researchers have not determined how deletions of mtDNA lead to the specific signs and symptoms of progressive external ophthalmoplegia, although the features of the condition may be related to impaired oxidative phosphorylation. It has been suggested that eye muscles are commonly affected by mitochondrial defects because they are especially dependent on oxidative phosphorylation for energy. More About This Health Condition      RRM2B-related mitochondrial DNA depletion syndrome, encephalomyopathic form with renal tubulopathy     More than a dozen mutations in the RRM2B gene can cause RRM2B-related mitochondrial DNA depletion syndrome, encephalomyopathic form with renal tubulopathy (RRM2B-MDS), a severe condition that affects multiple body systems. It typically leads to brain dysfunction combined with muscle weakness (encephalomyopathy) and a problem with kidney function known as renal tubulopathy. The mutations that cause this disorder occur in both copies of the RRM2B gene. They reduce the activity or amount of RNR, which likely impairs production of mtDNA nucleotides. A shortage of nucleotides available for the production of mtDNA molecules leads to a reduction in the amount of mtDNA (known as mtDNA depletion) and impairs mitochondrial function in many different types of cells.Impairment of oxidative phosphorylation is thought to underlie the signs and symptoms of mitochondrial DNA depletion syndrome. It is unclear why some RRM2B gene mutations result in deletions of genetic material from mtDNA (as in progressive external ophthalmoplegia, described above) and others reduce the overall amount of mtDNA (as in RRM2B-MDS). More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 17 mutations in the RRM2B gene have been identified in people with an eye condition called progressive external ophthalmoplegia. This disorder weakens the muscles that control eye movement and causes the eyelids to droop (ptosis). Some affected individuals have additional signs and symptoms, such as weakness of other muscles, extreme tiredness (fatigue), hearing loss caused by problems with the inner ear (sensorineural hearing loss), and digestive problems.Typically, mutations that cause progressive external ophthalmoplegia occur in one copy of the RRM2B gene, although rarely both copies of the gene are altered. RRM2B gene mutations associated with progressive external ophthalmoplegia lead to impaired RNR activity. These mutations result in large deletions of genetic material from mtDNA in muscle tissue, possibly because impairment of RNR activity leads to a shortage of nucleotides, although the mechanism is unclear. Researchers have not determined how deletions of mtDNA lead to the specific signs and symptoms of progressive external ophthalmoplegia, although the features of the condition may be related to impaired oxidative phosphorylation. It has been suggested that eye muscles are commonly affected by mitochondrial defects because they are especially dependent on oxidative phosphorylation for energy. More About This Health Condition  More than a dozen mutations in the RRM2B gene can cause RRM2B-related mitochondrial DNA depletion syndrome, encephalomyopathic form with renal tubulopathy (RRM2B-MDS), a severe condition that affects multiple body systems. It typically leads to brain dysfunction combined with muscle weakness (encephalomyopathy) and a problem with kidney function known as renal tubulopathy. The mutations that cause this disorder occur in both copies of the RRM2B gene. They reduce the activity or amount of RNR, which likely impairs production of mtDNA nucleotides. A shortage of nucleotides available for the production of mtDNA molecules leads to a reduction in the amount of mtDNA (known as mtDNA depletion) and impairs mitochondrial function in many different types of cells.Impairment of oxidative phosphorylation is thought to underlie the signs and symptoms of mitochondrial DNA depletion syndrome. It is unclear why some RRM2B gene mutations result in deletions of genetic material from mtDNA (as in progressive external ophthalmoplegia, described above) and others reduce the overall amount of mtDNA (as in RRM2B-MDS). More About This Health Condition    Cytogenetic Location: 8q22.3, which is the long (q) arm of chromosome 8 at position 22.3Molecular Location: base pairs 102,204,501 to 102,239,118 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        MTDPS8AMTDPS8Bp53-inducible ribonucleotide reductase small subunit 2 homologp53-inducible ribonucleotide reductase small subunit 2-like proteinp53-inducible ribonucleotide reductase small subunit 2 short form betaP53R2ribonucleoside-diphosphate reductase subunit M2 B isoform 1ribonucleoside-diphosphate reductase subunit M2 B isoform 2ribonucleoside-diphosphate reductase subunit M2 B isoform 3ribonucleotide reductase M2 B (TP53 inducible)TP53-inducible ribonucleotide reductase M2 B     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Food Molecules Are Broken Down in Three Stages to Produce ATPMolecular Cell Biology (fourth edition, 2000): Mitochondria Are the Principal Sites of ATP Production in Aerobic Cells      Clinical Information from GeneReviews (2 links)     Mitochondrial DNA Maintenance Defects OverviewRRM2B-Related Mitochondrial Disease      Tests Listed in the Genetic Testing Registry (1 link)     Tests of RRM2B      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     RIBONUCLEOTIDE REDUCTASE, M2 B      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Food Molecules Are Broken Down in Three Stages to Produce ATPMolecular Cell Biology (fourth edition, 2000): Mitochondria Are the Principal Sites of ATP Production in Aerobic Cells  Mitochondrial DNA Maintenance Defects OverviewRRM2B-Related Mitochondrial Disease  Tests of RRM2B  PubMed  RIBONUCLEOTIDE REDUCTASE, M2 B  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bourdon A, Minai L, Serre V, Jais JP, Sarzi E, Aubert S, Chrtien D, de Lonlay P, Paquis-Flucklinger V, Arakawa H, Nakamura Y, Munnich A, Rtig A. Mutation of RRM2B, encoding p53-controlled ribonucleotide reductase (p53R2), causes severe mitochondrial DNA depletion. Nat Genet. 2007 Jun;39(6):776-80. Epub 2007 May 7. Citation on PubMedFratter C, Raman P, Alston CL, Blakely EL, Craig K, Smith C, Evans J, Seller A, Czermin B, Hanna MG, Poulton J, Brierley C, Staunton TG, Turnpenny PD, Schaefer AM, Chinnery PF, Horvath R, Turnbull DM, Gorman GS, Taylor RW. RRM2B mutations are frequent in familial PEO with multiple mtDNA deletions. Neurology. 2011 Jun 7;76(23):2032-4. doi: 10.1212/WNL.0b013e31821e558b. Citation on PubMed or Free article on PubMed CentralGorman GS, Taylor RW. RRM2B-Related Mitochondrial Disease. 2014 Apr 17. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK195854/ Citation on PubMedPitceathly RD, Smith C, Fratter C, Alston CL, He L, Craig K, Blakely EL, Evans JC, Taylor J, Shabbir Z, Deschauer M, Pohl U, Roberts ME, Jackson MC, Halfpenny CA, Turnpenny PD, Lunt PW, Hanna MG, Schaefer AM, McFarland R, Horvath R, Chinnery PF, Turnbull DM, Poulton J, Taylor RW, Gorman GS. Adults with RRM2B-related mitochondrial disease have distinct clinical and molecular characteristics. Brain. 2012 Nov;135(Pt 11):3392-403. doi: 10.1093/brain/aws231. Epub 2012 Oct 29. Citation on PubMed or Free article on PubMed CentralPontarin G, Ferraro P, Bee L, Reichard P, Bianchi V. Mammalian ribonucleotide reductase subunit p53R2 is required for mitochondrial DNA replication and DNA repair in quiescent cells. Proc Natl Acad Sci U S A. 2012 Aug 14;109(33):13302-7. doi: 10.1073/pnas.1211289109. Citation on PubMed or Free article on PubMed CentralPontarin G, Fijolek A, Pizzo P, Ferraro P, Rampazzo C, Pozzan T, Thelander L, Reichard PA, Bianchi V. Ribonucleotide reduction is a cytosolic process in mammalian cells independently of DNA damage. Proc Natl Acad Sci U S A. 2008 Nov 18;105(46):17801-6. doi: 10.1073/pnas.0808198105. Epub 2008 Nov 7. Citation on PubMed or Free article on PubMed CentralOMIM: RIBONUCLEOTIDE REDUCTASE, M2 BTyynismaa H, Ylikallio E, Patel M, Molnar MJ, Haller RG, Suomalainen A. A heterozygous truncating mutation in RRM2B causes autosomal-dominant progressive external ophthalmoplegia with multiple mtDNA deletions. Am J Hum Genet. 2009 Aug;85(2):290-5. doi: 10.1016/j.ajhg.2009.07.009. Epub 2009 Aug 6. Citation on PubMed or Free article on PubMed Central \n",
            "FZD6\t   The FZD6 gene provides instructions for making a protein called frizzled-6. This protein is embedded in the outer membrane of cells, where it is involved in transmitting chemical signals from outside the cell to the cell's nucleus.Specifically, frizzled-6 participates in the Wnt signaling pathway, a series of steps that affect the way cells and tissues develop. Wnt signaling is important for cell division (proliferation), attachment of cells to one another (adhesion), cell movement (migration), and many other cellular activities. At the cell surface, frizzled-6 attaches (binds) to certain proteins, which triggers the frizzled-6 protein to send signals into the cell and initiate the Wnt signaling pathway.Frizzled-6 is active in many tissues. During early development it plays a critical role in the growth and development of nails, particularly the attachment of the nail to the nail bed. Studies suggest that frizzled-6 is also involved in hair growth.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Nonsyndromic congenital nail disorder 10     At least three mutations in the FZD6 gene have been found to cause nonsyndromic congenital nail disorder 10. This condition is characterized by abnormally thick fingernails and toenails that may appear claw-like. The mutations lead to the production of a frizzled-6 protein that cannot get to the cell membrane where it is needed or that cannot transmit signals into the cell. As a result, Wnt signaling is disrupted and the growth and development of nails is poorly regulated, which leads to separation of the nail from the underlying nail bed (onycholysis) and other abnormalities of the fingernails and toenails. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three mutations in the FZD6 gene have been found to cause nonsyndromic congenital nail disorder 10. This condition is characterized by abnormally thick fingernails and toenails that may appear claw-like. The mutations lead to the production of a frizzled-6 protein that cannot get to the cell membrane where it is needed or that cannot transmit signals into the cell. As a result, Wnt signaling is disrupted and the growth and development of nails is poorly regulated, which leads to separation of the nail from the underlying nail bed (onycholysis) and other abnormalities of the fingernails and toenails. More About This Health Condition    Cytogenetic Location: 8q22.3, which is the long (q) arm of chromosome 8 at position 22.3Molecular Location: base pairs 103,298,433 to 103,332,866 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        frizzled 6, seven transmembrane spanning receptorfrizzled family receptor 6frizzled homolog 6FZ-6FZ6HFZ6     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Developmental Biology (sixth edition, 2000): The Wnt PathwayMadame Curie Bioscience: Secreted Antagonists/Modulators of Wnt SignalingMolecular Biology of the Cell (fourth edition, 2002): Wnt Proteins Bind to Frizzled Receptors and Inhibit the Degradation of -Catenin      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FZD6      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FRIZZLED, DROSOPHILA, HOMOLOG OF, 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): The Wnt PathwayMadame Curie Bioscience: Secreted Antagonists/Modulators of Wnt SignalingMolecular Biology of the Cell (fourth edition, 2002): Wnt Proteins Bind to Frizzled Receptors and Inhibit the Degradation of -Catenin  Tests of FZD6  PubMed  FRIZZLED, DROSOPHILA, HOMOLOG OF, 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: FRIZZLED, DROSOPHILA, HOMOLOG OF, 6Frjmark AS, Schuster J, Sobol M, Entesarian M, Kilander MB, Gabrikova D, Nawaz S, Baig SM, Schulte G, Klar J, Dahl N. Mutations in Frizzled 6 cause isolated autosomal-recessive nail dysplasia. Am J Hum Genet. 2011 Jun 10;88(6):852-60. doi: 10.1016/j.ajhg.2011.05.013. Citation on PubMed or Free article on PubMed CentralGuo N, Hawkins C, Nathans J. Frizzled6 controls hair patterning in mice. Proc Natl Acad Sci U S A. 2004 Jun 22;101(25):9277-81. Citation on PubMed or Free article on PubMed CentralNaz G, Pasternack SM, Perrin C, Mattheisen M, Refke M, Khan S, Gul A, Simons M, Ahmad W, Betz RC. FZD6 encoding the Wnt receptor frizzled 6 is mutated in autosomal-recessive nail dysplasia. Br J Dermatol. 2012 May;166(5):1088-94. doi: 10.1111/j.1365-2133.2011.10800.x. Citation on PubMedRaza SI, Muhammad N, Khan S, Ahmad W. A novel missense mutation in the gene FZD6 underlies autosomal recessive nail dysplasia. Br J Dermatol. 2013 Feb;168(2):422-5. doi: 10.1111/j.1365-2133.2012.11203.x. Citation on PubMed \n",
            "RP1L1\t   From NCBI Gene:This gene encodes a member of the doublecortin family. The protein encoded by this gene contains two N-terminal doublecortin domains, which bind microtubules and regulate microtubule polymerization, and two C-terminal large repetitive regions, both of which contain a high percentage of glutamine and glutamic acid residues. This protein is a retinal-specific protein. Its exact length varies among individuals due to the presence of a 16aa repeat in the first C-terminal repetitive region. The 16aa repeat is encoded by the highly polymorphic 48-bp repeat, and 1-6 copies of the 16aa repeat have been identified in normal individuals. The current reference sequence shown here has a single copy of the 16aa repeat. This protein and the RP1 protein, another retinal-specific protein, play essential and synergistic roles in affecting photosensitivity and outer segment morphogenesis of rod photoreceptors. Mutations in this gene cause occult macular dystrophy (OMD). [provided by RefSeq, Sep 2010]From UniProt:Required for the differentiation of photoreceptor cells. Plays a role in the organization of outer segment of rod and cone photoreceptors (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Occult macular dystrophyFrom UniProt:Occult macular dystrophy (OCMD): An inherited macular dystrophy characterized by progressive loss of macular function but normal ophthalmoscopic appearance. It is typically characterized by a central cone dysfunction leading to a loss of vision despite normal ophthalmoscopic appearance, normal fluorescein angiography, and normal full-field electroretinogram (ERGs), but the amplitudes of the focal macular ERGs and multifocal ERGs are significantly reduced at the central retina. [MIM:613587]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p23.1, which is the short (p) arm of chromosome 8 at position 23.1Molecular Location: base pairs 10,606,349 to 10,655,107 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DCDC4B     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RP1L1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: OCCULT MACULAR DYSTROPHYOMIM: RP1-LIKE PROTEIN 1      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RP1L1  OMIM: OCCULT MACULAR DYSTROPHYOMIM: RP1-LIKE PROTEIN 1  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SLC35G5\t\n",
            "CSMD3\t\n",
            "FDFT1\t\n",
            "USP17L7\t\n",
            "USP17L2\t\n",
            "MAL2\t\n",
            "FAM86B1\t\n",
            "FAM86B2\t\n",
            "TBC1D31\t\n",
            "WDYHV1\t\n",
            "MTSS1\t\n",
            "NDUFB9\t   From NCBI Gene:The protein encoded by this gene is a subunit of the mitochondrial oxidative phosphorylation complex I (nicotinamide adenine dinucleotide: ubiquinone oxidoreductase). Complex I is localized to the inner mitochondrial membrane and functions to dehydrogenate nicotinamide adenine dinucleotide and to shuttle electrons to coenzyme Q. Complex I deficiency is the most common defect found in oxidative phosphorylation disorders and results in a range of conditions, including lethal neonatal disease, hypertrophic cardiomyopathy, liver disease, and adult-onset neurodegenerative disorders. Pseudogenes of this gene are found on chromosomes five, seven and eight. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2015]From UniProt:Accessory subunit of the mitochondrial membrane respiratory chain NADH dehydrogenase (Complex I), that is believed to be not involved in catalysis. Complex I functions in the transfer of electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Mitochondrial complex I deficiencyFrom NCBI Gene:MITOCHONDRIAL COMPLEX I DEFICIENCY, NUCLEAR TYPE 24From UniProt:Mitochondrial complex I deficiency, nuclear type 24 (MC1DN24): A form of mitochondrial complex I deficiency, the most common biochemical signature of mitochondrial disorders, a group of highly heterogeneous conditions characterized by defective oxidative phosphorylation, which collectively affects 1 in 5-10000 live births. Clinical disorders have variable severity, ranging from lethal neonatal disease to adult-onset neurodegenerative disorders. Phenotypes include macrocephaly with progressive leukodystrophy, non-specific encephalopathy, cardiomyopathy, myopathy, liver disease, Leigh syndrome, Leber hereditary optic neuropathy, and some forms of Parkinson disease. MC1DN24 transmission pattern is consistent with autosomal recessive inheritance. [MIM:618245]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8q24.13, which is the long (q) arm of chromosome 8 at position 24.13Molecular Location: base pairs 124,539,102 to 124,549,986 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        B22CI-B22LYRM3MC1DN24UQOR22     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NDUFB9      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MITOCHONDRIAL COMPLEX I DEFICIENCY, NUCLEAR TYPE 24OMIM: NADH-UBIQUINONE OXIDOREDUCTASE 1 BETA SUBCOMPLEX, 9      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NDUFB9  OMIM: MITOCHONDRIAL COMPLEX I DEFICIENCY, NUCLEAR TYPE 24OMIM: NADH-UBIQUINONE OXIDOREDUCTASE 1 BETA SUBCOMPLEX, 9  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "POU5F1B\t\n",
            "WISP1\t\n",
            "JRK\t   From NCBI Gene:This gene encodes a conserved protein that is similar to DNA-binding proteins, such as major centromere autoantigen B (CENPB). Inactivation of the related gene in mice resulted in epileptic seizures. Childhood Absence Epilepsy (CAE) has been mapped to the same chromosomal location (8q24.3) as this gene. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]From UniProt:May bind DNA.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Childhood absence epilepsy     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8q24.3, which is the long (q) arm of chromosome 8 at position 24.3Molecular Location: base pairs 142,651,501 to 142,669,994 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        jerkyJH8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of JRK      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: JERKY, MOUSE, HOMOLOG OF      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of JRK  OMIM: JERKY, MOUSE, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ZFP41\t\n",
            "RHPN1\t\n",
            "MAFA\t\n",
            "ZC3H3\t\n",
            "FAM83H\t   The FAM83H gene provides instructions for making a protein whose function is not well understood. The protein is found in several types of cells, including specialized cells called ameloblasts. Ameloblasts produce tooth enamel, which is the hard, calcium-rich material that forms the protective outer layer of each tooth. The FAM83H protein is thought to be involved in the formation of enamel, although its role in this process is unknown.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Amelogenesis imperfecta     At least 20 mutations in the FAM83H gene have been found in an autosomal dominant form of a disorder of tooth development called amelogenesis imperfecta. Autosomal dominant inheritance means that one copy of the FAM83H gene in each cell is altered. All of the known FAM83H gene mutations lead to production of an abnormally short protein. While the normal protein is found in the fluid-filled space inside the cell (the cytoplasm), the altered protein is found in the cell's nucleus. Researchers speculate that the altered protein produced from the mutated copy of the gene interferes with the function of the normal protein produced from the non-mutated copy of the gene (such mutations are described as \"dominant-negative\"). However, researchers do not know how the altered protein leads to teeth with unusually thin, rough, yellowish-brown enamel. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 20 mutations in the FAM83H gene have been found in an autosomal dominant form of a disorder of tooth development called amelogenesis imperfecta. Autosomal dominant inheritance means that one copy of the FAM83H gene in each cell is altered. All of the known FAM83H gene mutations lead to production of an abnormally short protein. While the normal protein is found in the fluid-filled space inside the cell (the cytoplasm), the altered protein is found in the cell's nucleus. Researchers speculate that the altered protein produced from the mutated copy of the gene interferes with the function of the normal protein produced from the non-mutated copy of the gene (such mutations are described as \"dominant-negative\"). However, researchers do not know how the altered protein leads to teeth with unusually thin, rough, yellowish-brown enamel. More About This Health Condition    Cytogenetic Location: 8q24.3, which is the long (q) arm of chromosome 8 at position 24.3Molecular Location: base pairs 143,723,933 to 143,741,626 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AI3FA83H_HUMANFAM83H variant 1family with sequence similarity 83, member HFLJ46072protein FAM83H     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     School of Dentistry, University of North Carolina at Chapel Hill      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FAM83H      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     FAMILY WITH SEQUENCE SIMILARITY 83, MEMBER H      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      School of Dentistry, University of North Carolina at Chapel Hill  Tests of FAM83H  PubMed  FAMILY WITH SEQUENCE SIMILARITY 83, MEMBER H  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Ding Y, Estrella MR, Hu YY, Chan HL, Zhang HD, Kim JW, Simmer JP, Hu JC. Fam83h is associated with intracellular vesicles and ADHCAI. J Dent Res. 2009 Nov;88(11):991-6. doi: 10.1177/0022034509349454. Citation on PubMed or Free article on PubMed CentralOMIM: FAMILY WITH SEQUENCE SIMILARITY 83, MEMBER HKim JW, Lee SK, Lee ZH, Park JC, Lee KE, Lee MH, Park JT, Seo BM, Hu JC, Simmer JP. FAM83H mutations in families with autosomal-dominant hypocalcified amelogenesis imperfecta. Am J Hum Genet. 2008 Feb;82(2):489-94. doi: 10.1016/j.ajhg.2007.09.020. Citation on PubMed or Free article on PubMed CentralLee SK, Hu JC, Bartlett JD, Lee KE, Lin BP, Simmer JP, Kim JW. Mutational spectrum of FAM83H: the C-terminal portion is required for tooth enamel calcification. Hum Mutat. 2008 Aug;29(8):E95-9. doi: 10.1002/humu.20789. Citation on PubMed or Free article on PubMed CentralLee SK, Lee KE, Jeong TS, Hwang YH, Kim S, Hu JC, Simmer JP, Kim JW. FAM83H mutations cause ADHCAI and alter intracellular protein localization. J Dent Res. 2011 Mar;90(3):377-81. doi: 10.1177/0022034510389177. Epub 2010 Nov 30. Citation on PubMed \n",
            "SCRIB\t   From NCBI Gene:This gene encodes a protein that was identified as being similar to the Drosophila scribble protein. The mammalian protein is involved in tumor suppression pathways. As a scaffold protein involved in cell polarization processes, this protein binds to many other proteins. The encoded protein binds to papillomavirus E6 protein via its PDZ domain and the C-terminus of E6. Two alternatively spliced transcript variants that encode different protein isoforms have been found for this gene. [provided by RefSeq, Nov 2011]From UniProt:Scaffold protein involved in different aspects of polarized cells differentiation regulating epithelial and neuronal morphogenesis. Most probably functions in the establishment of apico-basal cell polarity. May function in cell proliferation regulating progression from G1 to S phase and as a positive regulator of apoptosis for instance during acinar morphogenesis of the mammary epithelium. May also function in cell migration and adhesion and hence regulate cell invasion through MAPK signaling. May play a role in exocytosis and in the targeting synaptic vesicles to synapses. Functions as an activator of Rac GTPase activity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Neural tube defects (NTD): Congenital malformations of the central nervous system and adjacent structures related to defective neural tube closure during the first trimester of pregnancy. Failure of neural tube closure can occur at any level of the embryonic axis. Common NTD forms include anencephaly, myelomeningocele and spina bifida, which result from the failure of fusion in the cranial and spinal region of the neural tube. NTDs have a multifactorial etiology encompassing both genetic and environmental components. [MIM:182940]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8q24.3, which is the long (q) arm of chromosome 8 at position 24.3Molecular Location: base pairs 143,790,920 to 143,815,379 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CRIB1SCRB1SCRIB1Vartul     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SCRIB      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: SCRIBBLE, DROSOPHILA, HOMOLOG OF      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SCRIB  OMIM: SCRIBBLE, DROSOPHILA, HOMOLOG OF  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PLEC\t   The PLEC gene provides instructions for making a protein called plectin. This protein is produced in many different tissues in the body, including skin and muscle. Within cells, plectin interacts with several molecules that make up the cell's structural framework (the cytoskeleton). For example, plectin interacts with intermediate filaments, which form networks that provide support and strength to cells. Plectin attaches (cross-links) intermediate filaments to one another and to the cell membrane.The exact function of plectin in different tissues is unclear. In skin cells, this protein is an essential part of structures called hemidesmosomes, which attach the network of intermediate filaments to the cell membrane. It is also a component of desmosomes, which form junctions between neighboring cells. As part of these structures, plectin plays a critical role in anchoring the outer layer of the skin (the epidermis) to underlying layers.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Epidermolysis bullosa with pyloric atresia     At least nine mutations in the PLEC gene can cause epidermolysis bullosa with pyloric atresia (EB-PA). In addition to skin blistering, people with EB-PA are born with a life-threatening obstruction of the digestive tract called pyloric atresia. Mutations in the PLEC gene account for about 15 percent of all cases of EB-PA.The PLEC gene mutations responsible for EB-PA change the normal structure and function of plectin. Mutations that prevent the cell from making any functional plectin tend to cause more severe signs and symptoms, while mutations that reduce the amount of plectin or alter the protein's structure usually cause milder signs and symptoms. A shortage of functional plectin disrupts the attachment of the epidermis to underlying skin layers, making the skin less resistant to friction and minor trauma. It is less clear how mutations in the PLEC gene are related to pyloric atresia. More About This Health Condition      Congenital myasthenic syndrome     Genetics Home Reference provides information about congenital myasthenic syndrome. More About This Health Condition      Epidermolysis bullosa simplex     At least one mutation in the PLEC gene is associated with the features of epidermolysis bullosa simplex, a condition that causes the skin to be very fragile and to blister easily. This mutation has been found in a small number of families with a form of the disorder known as the Ogna type. Researchers are uncertain whether this rare condition is actually a subtype of epidermolysis bullosa simplex or represents a separate form of epidermolysis bullosa.The mutation responsible for the Ogna type of epidermolysis bullosa simplex changes a single protein building block (amino acid) in the plectin protein. Specifically, this genetic change replaces the amino acid arginine with the amino acid tryptophan at protein position 2000 (written as Arg2000Trp or R2000W). Studies suggest that this mutation may change the way the protein folds into a 3-dimensional shape, which could prevent it from interacting with molecules that make up the cytoskeleton. In the skin, these changes cause epidermal cells to become fragile and easily damaged. As a result, the skin is less resistant to friction and minor trauma and blisters easily. More About This Health Condition      Other disorders     Mutations in the PLEC gene also cause at least one other form of epidermolysis bullosa known as epidermolysis bullosa with muscular dystrophy. In addition to skin blistering, people with this disorder experience progressive muscle weakness and wasting (atrophy) later in life. More than 20 PLEC gene mutations have been found to cause this form of the disorder. These mutations alter the structure, function, or production of plectin. In the skin, a reduced amount of functional plectin disrupts the attachment of the epidermis to underlying skin layers, making the skin less resistant to friction. A shortage of functional plectin in muscle tissue affects critical structures needed for muscle tensing (contraction), which leads to muscle weakness and atrophy.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least nine mutations in the PLEC gene can cause epidermolysis bullosa with pyloric atresia (EB-PA). In addition to skin blistering, people with EB-PA are born with a life-threatening obstruction of the digestive tract called pyloric atresia. Mutations in the PLEC gene account for about 15 percent of all cases of EB-PA.The PLEC gene mutations responsible for EB-PA change the normal structure and function of plectin. Mutations that prevent the cell from making any functional plectin tend to cause more severe signs and symptoms, while mutations that reduce the amount of plectin or alter the protein's structure usually cause milder signs and symptoms. A shortage of functional plectin disrupts the attachment of the epidermis to underlying skin layers, making the skin less resistant to friction and minor trauma. It is less clear how mutations in the PLEC gene are related to pyloric atresia. More About This Health Condition  Genetics Home Reference provides information about congenital myasthenic syndrome. More About This Health Condition  At least one mutation in the PLEC gene is associated with the features of epidermolysis bullosa simplex, a condition that causes the skin to be very fragile and to blister easily. This mutation has been found in a small number of families with a form of the disorder known as the Ogna type. Researchers are uncertain whether this rare condition is actually a subtype of epidermolysis bullosa simplex or represents a separate form of epidermolysis bullosa.The mutation responsible for the Ogna type of epidermolysis bullosa simplex changes a single protein building block (amino acid) in the plectin protein. Specifically, this genetic change replaces the amino acid arginine with the amino acid tryptophan at protein position 2000 (written as Arg2000Trp or R2000W). Studies suggest that this mutation may change the way the protein folds into a 3-dimensional shape, which could prevent it from interacting with molecules that make up the cytoskeleton. In the skin, these changes cause epidermal cells to become fragile and easily damaged. As a result, the skin is less resistant to friction and minor trauma and blisters easily. More About This Health Condition  Mutations in the PLEC gene also cause at least one other form of epidermolysis bullosa known as epidermolysis bullosa with muscular dystrophy. In addition to skin blistering, people with this disorder experience progressive muscle weakness and wasting (atrophy) later in life. More than 20 PLEC gene mutations have been found to cause this form of the disorder. These mutations alter the structure, function, or production of plectin. In the skin, a reduced amount of functional plectin disrupts the attachment of the epidermis to underlying skin layers, making the skin less resistant to friction. A shortage of functional plectin in muscle tissue affects critical structures needed for muscle tensing (contraction), which leads to muscle weakness and atrophy.    Cytogenetic Location: 8q24.3, which is the long (q) arm of chromosome 8 at position 24.3Molecular Location: base pairs 143,915,147 to 143,976,800 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        EBS1EBSOHD1hemidesmosomal protein 1PCNPLEC1PLEC1_HUMANplectin 1plectin 1, intermediate filament binding protein 500kDaPlectin-11PLTN     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2002): Intermediate Filaments Are Cross-linked and Bundled Into Strong ArraysThe Cell: A Molecular Approach (second edition, 2000): Intracellular Organization of Intermediate Filaments      Clinical Information from GeneReviews (2 links)     Epidermolysis Bullosa SimplexEpidermolysis Bullosa with Pyloric Atresia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PLEC      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     EPIDERMOLYSIS BULLOSA SIMPLEX WITH MUSCULAR DYSTROPHYPLECTIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Intermediate Filaments Are Cross-linked and Bundled Into Strong ArraysThe Cell: A Molecular Approach (second edition, 2000): Intracellular Organization of Intermediate Filaments  Epidermolysis Bullosa SimplexEpidermolysis Bullosa with Pyloric Atresia  Tests of PLEC  PubMed  EPIDERMOLYSIS BULLOSA SIMPLEX WITH MUSCULAR DYSTROPHYPLECTIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Charlesworth A, Chiaverini C, Chevrant-Breton J, DelRio M, Diociaiuti A, Dupuis RP, El Hachem M, Le Fiblec B, Sankari-Ho AM, Valhquist A, Wierzbicka E, Lacour JP, Meneguzzi G. Epidermolysis bullosa simplex with PLEC mutations: new phenotypes and new mutations. Br J Dermatol. 2013 Apr;168(4):808-14. doi: 10.1111/bjd.12202. Citation on PubMedKiritsi D, Pigors M, Tantcheva-Poor I, Wessel C, Arin MJ, Kohlhase J, Bruckner-Tuderman L, Has C. Epidermolysis bullosa simplex ogna revisited. J Invest Dermatol. 2013 Jan;133(1):270-3. doi: 10.1038/jid.2012.248. Epub 2012 Aug 2. Citation on PubMedKoss-Harnes D, Hyheim B, Anton-Lamprecht I, Gjesti A, Jrgensen RS, Jahnsen FL, Olaisen B, Wiche G, Gedde-Dahl T Jr. A site-specific plectin mutation causes dominant epidermolysis bullosa simplex Ogna: two identical de novo mutations. J Invest Dermatol. 2002 Jan;118(1):87-93. Citation on PubMedKoss-Harnes D, Hyheim B, Jonkman MF, de Groot WP, de Weerdt CJ, Nikolic B, Wiche G, Gedde-Dahl T Jr. Life-long course and molecular characterization of the original Dutch family with epidermolysis bullosa simplex with muscular dystrophy due to a homozygous novel plectin point mutation. Acta Derm Venereol. 2004;84(2):124-31. Citation on PubMedMcMillan JR, Akiyama M, Rouan F, Mellerio JE, Lane EB, Leigh IM, Owaribe K, Wiche G, Fujii N, Uitto J, Eady RA, Shimizu H. Plectin defects in epidermolysis bullosa simplex with muscular dystrophy. Muscle Nerve. 2007 Jan;35(1):24-35. Citation on PubMedNakamura H, Sawamura D, Goto M, Nakamura H, McMillan JR, Park S, Kono S, Hasegawa S, Paku S, Nakamura T, Ogiso Y, Shimizu H. Epidermolysis bullosa simplex associated with pyloric atresia is a novel clinical subtype caused by mutations in the plectin gene (PLEC1). J Mol Diagn. 2005 Feb;7(1):28-35. Citation on PubMed or Free article on PubMed CentralPfendner E, Rouan F, Uitto J. Progress in epidermolysis bullosa: the phenotypic spectrum of plectin mutations. Exp Dermatol. 2005 Apr;14(4):241-9. Review. Citation on PubMedRezniczek GA, Walko G, Wiche G. Plectin gene defects lead to various forms of epidermolysis bullosa simplex. Dermatol Clin. 2010 Jan;28(1):33-41. doi: 10.1016/j.det.2009.10.004. Review. Erratum in: Dermatol Clin. 2010 Apr;28(2):439-41. Citation on PubMedUitto J, Pulkkinen L, Smith FJ, McLean WH. Plectin and human genetic disorders of the skin and muscle. The paradigm of epidermolysis bullosa with muscular dystrophy. Exp Dermatol. 1996 Oct;5(5):237-46. Review. Citation on PubMed \n",
            "OPLAH\t   From NCBI Gene:The protein encoded by this gene acts as a homodimer, using ATP hydrolysis to catalyze the conversion of 5-oxo-L-proline to L-glutamate. Defects in this gene are a cause of 5-oxoprolinase deficiency (OPLAHD). [provided by RefSeq, Jun 2012]From UniProt:Catalyzes the cleavage of 5-oxo-L-proline to form L-glutamate coupled to the hydrolysis of ATP to ADP and inorganic phosphate.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:5-Oxoprolinase deficiencyFrom UniProt:5-oxoprolinase deficiency (OPLAHD): A disorder characterized by calcium oxalate/carbonate urolithiasis, and excessive urinary 5-oxo-L-proline. Affected individuals have recurrent episodes of vomiting, diarrhea, and abdominal pain. [MIM:260005]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8q24.3, which is the long (q) arm of chromosome 8 at position 24.3Molecular Location: base pairs 144,051,266 to 144,064,026 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        5-OpaseOPLAOPLAHD     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of OPLAH      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: 5-OXOPROLINASE (ATP-HYDROLYZING)OMIM: 5-OXOPROLINASE DEFICIENCY      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of OPLAH  OMIM: 5-OXOPROLINASE (ATP-HYDROLYZING)OMIM: 5-OXOPROLINASE DEFICIENCY  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ADCK5\t\n",
            "RECQL4\t   The RECQL4 gene provides instructions for making one member of a protein family called RecQ helicases. Helicases are enzymes that bind to DNA and temporarily unwind the two spiral strands (double helix) of the DNA molecule.  This unwinding is necessary for copying (replicating) DNA in preparation for cell division, and for repairing damaged DNA. Because RecQ helicases maintain the structure and integrity of DNA, they are known as the \"caretakers of the genome.\"The RECQL4 protein is active in several types of cells before and after birth. Researchers believe that this protein is particularly important in cells of the developing bones and skin. It has also been found in enterocytes, which are cells that line the intestine and absorb nutrients.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Baller-Gerold syndrome     Several mutations in the RECQL4 gene have been identified in people with Baller-Gerold syndrome. Most of these mutations prevent the production of any RECQL4 protein or change the way the protein is pieced together, which disrupts its usual function. A shortage of this protein may prevent normal DNA replication and repair, causing widespread damage to a person's genetic information over time. It is unclear how these changes result in the varied signs and symptoms of Baller-Gerold syndrome, including the abnormal fusion of certain skull bones (craniosynostosis), small stature, missing thumbs or bones in the forearm (radial ray malformations), and a skin rash. More About This Health Condition      RAPADILINO syndrome     At least 10 mutations in the RECQL4 gene have been identified in people with RAPADILINO syndrome. This condition has many features, including radial ray malformations, malformed or missing kneecaps, diarrhea, and short stature. The condition was first identified in Finland, and the most common mutation in RAPADILINO syndrome is found in all affected individuals of Finnish descent as well as some people from other populations. This mutation, which is written as IVS7+2delT, is known as a splice-site mutation, and it causes the RECQL4 protein to be pieced together incorrectly. This genetic change results in the production of a protein that is missing a region called exon 7. The altered protein does not have helicase activity, which may prevent normal DNA replication and repair. These changes may result in the accumulation of DNA errors and cell death, although it is unclear exactly how RECQL4 gene mutations lead to the specific features of RAPADILINO syndrome. More About This Health Condition      Rothmund-Thomson syndrome     More than 40 mutations in the RECQL4 gene have been found in people with Rothmund-Thomson syndrome. These mutations likely prevent the production of any RECQL4 protein or lead to the production of an abnormally short, nonfunctional version of the protein. A shortage of this protein may prevent normal DNA replication and repair, causing widespread damage to a person's genetic information over time. Further study is needed to determine how these changes result in the characteristic features of Rothmund-Thomson syndrome, which include a skin rash, sparse hair, small stature, skeletal abnormalities, and an increased risk of certain cancers.Because Rothmund-Thomson syndrome, Baller-Gerold syndrome, and RAPADILINO syndrome have overlapping features and can be caused by mutations in the same gene, researchers are investigating whether they are separate disorders or part of a single syndrome with overlapping signs and symptoms. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several mutations in the RECQL4 gene have been identified in people with Baller-Gerold syndrome. Most of these mutations prevent the production of any RECQL4 protein or change the way the protein is pieced together, which disrupts its usual function. A shortage of this protein may prevent normal DNA replication and repair, causing widespread damage to a person's genetic information over time. It is unclear how these changes result in the varied signs and symptoms of Baller-Gerold syndrome, including the abnormal fusion of certain skull bones (craniosynostosis), small stature, missing thumbs or bones in the forearm (radial ray malformations), and a skin rash. More About This Health Condition  At least 10 mutations in the RECQL4 gene have been identified in people with RAPADILINO syndrome. This condition has many features, including radial ray malformations, malformed or missing kneecaps, diarrhea, and short stature. The condition was first identified in Finland, and the most common mutation in RAPADILINO syndrome is found in all affected individuals of Finnish descent as well as some people from other populations. This mutation, which is written as IVS7+2delT, is known as a splice-site mutation, and it causes the RECQL4 protein to be pieced together incorrectly. This genetic change results in the production of a protein that is missing a region called exon 7. The altered protein does not have helicase activity, which may prevent normal DNA replication and repair. These changes may result in the accumulation of DNA errors and cell death, although it is unclear exactly how RECQL4 gene mutations lead to the specific features of RAPADILINO syndrome. More About This Health Condition  More than 40 mutations in the RECQL4 gene have been found in people with Rothmund-Thomson syndrome. These mutations likely prevent the production of any RECQL4 protein or lead to the production of an abnormally short, nonfunctional version of the protein. A shortage of this protein may prevent normal DNA replication and repair, causing widespread damage to a person's genetic information over time. Further study is needed to determine how these changes result in the characteristic features of Rothmund-Thomson syndrome, which include a skin rash, sparse hair, small stature, skeletal abnormalities, and an increased risk of certain cancers.Because Rothmund-Thomson syndrome, Baller-Gerold syndrome, and RAPADILINO syndrome have overlapping features and can be caused by mutations in the same gene, researchers are investigating whether they are separate disorders or part of a single syndrome with overlapping signs and symptoms. More About This Health Condition    Cytogenetic Location: 8q24.3, which is the long (q) arm of chromosome 8 at position 24.3Molecular Location: base pairs 144,511,284 to 144,517,828 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATP-Dependent DNA Helicase Q4RecQ helicase-like 4RecQ protein 4RecQ protein-like 4RecQ Protein Like 4RECQ4RECQ4_HUMANRTS     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): DNA Replication, Recombination, and Repair      Clinical Information from GeneReviews (2 links)     Baller-Gerold SyndromeRothmund-Thomson Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of RECQL4      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     RECQ PROTEIN-LIKE 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): DNA Replication, Recombination, and Repair  Baller-Gerold SyndromeRothmund-Thomson Syndrome  Tests of RECQL4  PubMed  RECQ PROTEIN-LIKE 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Croteau DL, Rossi ML, Ross J, Dawut L, Dunn C, Kulikowicz T, Bohr VA. RAPADILINO RECQL4 mutant protein lacks helicase and ATPase activity. Biochim Biophys Acta. 2012 Nov;1822(11):1727-34. doi: 10.1016/j.bbadis.2012.07.014. Epub 2012 Jul 31. Citation on PubMed or Free article on PubMed CentralCroteau DL, Singh DK, Hoh Ferrarelli L, Lu H, Bohr VA. RECQL4 in genomic instability and aging. Trends Genet. 2012 Dec;28(12):624-31. doi: 10.1016/j.tig.2012.08.003. Epub 2012 Aug 30. Review. Citation on PubMed or Free article on PubMed CentralDietschy T, Shevelev I, Stagljar I. The molecular role of the Rothmund-Thomson-, RAPADILINO- and Baller-Gerold-gene product, RECQL4: recent progress. Cell Mol Life Sci. 2007 Apr;64(7-8):796-802. Review. Citation on PubMedKitao S, Shimamoto A, Goto M, Miller RW, Smithson WA, Lindor NM, Furuichi Y. Mutations in RECQL4 cause a subset of cases of Rothmund-Thomson syndrome. Nat Genet. 1999 May;22(1):82-4. Citation on PubMedLarizza L, Magnani I, Roversi G. Rothmund-Thomson syndrome and RECQL4 defect: splitting and lumping. Cancer Lett. 2006 Jan 28;232(1):107-20. Epub 2005 Nov 3. Review. Citation on PubMedPetkovic M, Dietschy T, Freire R, Jiao R, Stagljar I. The human Rothmund-Thomson syndrome gene product, RECQL4, localizes to distinct nuclear foci that coincide with proteins involved in the maintenance of genome stability. J Cell Sci. 2005 Sep 15;118(Pt 18):4261-9. Epub 2005 Sep 1. Erratum in: J Cell Sci. 2005 Oct 1;118(Pt 19):4587. Citation on PubMedSiitonen HA, Kopra O, Kriinen H, Haravuori H, Winter RM, Smnen AM, Peltonen L, Kestil M. Molecular defect of RAPADILINO syndrome expands the phenotype spectrum of RECQL diseases. Hum Mol Genet. 2003 Nov 1;12(21):2837-44. Epub 2003 Sep 2. Citation on PubMedSuzuki T, Kohno T, Ishimi Y. DNA helicase activity in purified human RECQL4 protein. J Biochem. 2009 Sep;146(3):327-35. doi: 10.1093/jb/mvp074. Epub 2009 May 18. Citation on PubMedVan Maldergem L, Siitonen HA, Jalkh N, Chouery E, De Roy M, Delague V, Muenke M, Jabs EW, Cai J, Wang LL, Plon SE, Fourneau C, Kestil M, Gillerot Y, Mgarban A, Verloes A. Revisiting the craniosynostosis-radial ray hypoplasia association: Baller-Gerold syndrome caused by mutations in the RECQL4 gene. J Med Genet. 2006 Feb;43(2):148-52. Epub 2005 Jun 17. Citation on PubMed or Free article on PubMed CentralWang LL, Gannavarapu A, Kozinetz CA, Levy ML, Lewis RA, Chintagumpala MM, Ruiz-Maldanado R, Contreras-Ruiz J, Cunniff C, Erickson RP, Lev D, Rogers M, Zackai EH, Plon SE. Association between osteosarcoma and deleterious mutations in the RECQL4 gene in Rothmund-Thomson syndrome. J Natl Cancer Inst. 2003 May 7;95(9):669-74. Citation on PubMedWerner SR, Prahalad AK, Yang J, Hock JM. RECQL4-deficient cells are hypersensitive to oxidative stress/damage: Insights for osteosarcoma prevalence and heterogeneity in Rothmund-Thomson syndrome. Biochem Biophys Res Commun. 2006 Jun 23;345(1):403-9. Epub 2006 Apr 27. Citation on PubMedYin J, Kwon YT, Varshavsky A, Wang W. RECQL4, mutated in the Rothmund-Thomson and RAPADILINO syndromes, interacts with ubiquitin ligases UBR1 and UBR2 of the N-end rule pathway. Hum Mol Genet. 2004 Oct 15;13(20):2421-30. Epub 2004 Aug 18. Citation on PubMed \n",
            "DLGAP2\t\n",
            "PDGFRL\t   From NCBI Gene:This gene encodes a protein with significant sequence similarity to the ligand binding domain of platelet-derived growth factor receptor beta. Mutations in this gene, or deletion of a chromosomal segment containing this gene, are associated with sporadic hepatocellular carcinomas, colorectal cancers, and non-small cell lung cancers. This suggests this gene product may function as a tumor suppressor. [provided by RefSeq, Jul 2008]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Hepatocellular carcinomaFrom UniProt:Colorectal cancer (CRC): A complex disease characterized by malignant lesions arising from the inner wall of the large intestine (the colon) and the rectum. Genetic alterations are often associated with progression from premalignant lesion (adenoma) to invasive adenocarcinoma. Risk factors for cancer of the colon and rectum include colon polyps, long-standing ulcerative colitis, and genetic family history. [MIM:114500]A polymorphism in PDGFRL has been reported to be associated with susceptibility to Behcet disease (PubMed:22926996). Behcet disease is a complex multiple-system disorder characterized by recurrent oral ulcerations, recurrent genital ulcerations, typical skin lesions, and uveitis. Behcet disease also involves joints, blood vessels, musculoskeletal, neurological systems, and the gastrointestinal tract.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p22, which is the short (p) arm of chromosome 8 at position 22Molecular Location: base pairs 17,576,433 to 17,643,133 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        PDGRLPRLTS     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of PDGFRL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: HEPATOCELLULAR CARCINOMAOMIM: PLATELET-DERIVED GROWTH FACTOR RECEPTOR-LIKE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of PDGFRL  OMIM: HEPATOCELLULAR CARCINOMAOMIM: PLATELET-DERIVED GROWTH FACTOR RECEPTOR-LIKE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MTUS1\t   From NCBI Gene:This gene encodes a protein which contains a C-terminal domain able to interact with the angiotension II (AT2) receptor and a large coiled-coil region allowing dimerization. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. One of the transcript variants has been shown to encode a mitochondrial protein that acts as a tumor suppressor and partcipates in AT2 signaling pathways. Other variants may encode nuclear or transmembrane proteins but it has not been determined whether they also participate in AT2 signaling pathways. [provided by RefSeq, Jul 2008]From UniProt:Cooperates with AGTR2 to inhibit ERK2 activation and cell proliferation. May be required for AGTR2 cell surface expression. Together with PTPN6, induces UBE2V2 expression upon angiotensin-II stimulation. Isoform 1 inhibits breast cancer cell proliferation, delays the progression of mitosis by prolonging metaphase and reduces tumor growth.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Hepatocellular carcinoma (HCC): A primary malignant neoplasm of epithelial liver cells. The major risk factors for HCC are chronic hepatitis B virus (HBV) infection, chronic hepatitis C virus (HCV) infection, prolonged dietary aflatoxin exposure, alcoholic cirrhosis, and cirrhosis due to other causes. [MIM:114550]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p22, which is the short (p) arm of chromosome 8 at position 22Molecular Location: base pairs 17,643,794 to 17,801,579 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATBPATIPATIP3ICISMP44MTSG1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MTUS1      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: MITOCHONDRIAL TUMOR SUPPRESSOR GENE 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MTUS1  OMIM: MITOCHONDRIAL TUMOR SUPPRESSOR GENE 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NAT2\t   From NCBI Gene:This gene encodes an enzyme that functions to both activate and deactivate arylamine and hydrazine drugs and carcinogens. Polymorphisms in this gene are responsible for the N-acetylation polymorphism in which human populations segregate into rapid, intermediate, and slow acetylator phenotypes. Polymorphisms in this gene are also associated with higher incidences of cancer and drug toxicity. A second arylamine N-acetyltransferase gene (NAT1) is located near this gene (NAT2). [provided by RefSeq, Jul 2008]From UniProt:Participates in the detoxification of a plethora of hydrazine and arylamine drugs. Catalyzes the N- or O-acetylation of various arylamine and heterocyclic amine substrates and is able to bioactivate several known carcinogens.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Age-related hearing lossFrom NCBI Gene:Slow acetylator due to N-acetyltransferase enzyme variant     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p22, which is the short (p) arm of chromosome 8 at position 22Molecular Location: base pairs 18,386,585 to 18,401,219 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AAC2NAT-2PNAT     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NAT2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ACETYLATION, SLOWOMIM: N-ACETYLTRANSFERASE 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NAT2  OMIM: ACETYLATION, SLOWOMIM: N-ACETYLTRANSFERASE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "ARHGEF10\t   From NCBI Gene:This gene encodes a Rho guanine nucleotide exchange factor (GEF). Rho GEFs regulate the activity of small Rho GTPases by stimulating the exchange of guanine diphosphate (GDP) for guanine triphosphate (GTP) and may play a role in neural morphogenesis. Mutations in this gene are associated with slowed nerve conduction velocity (SNCV). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2015]From UniProt:May play a role in developmental myelination of peripheral nerves.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Charcot-Marie-Tooth diseaseFrom NCBI Gene:Slowed nerve conduction velocity, autosomal dominantFrom UniProt:Slowed nerve conduction velocity (SNCV): Affected individuals present a reduction in nerve conduction velocities without any clinical signs of peripheral or central nervous system dysfunction. SNCV inheritance is autosomal dominant. [MIM:608236]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p23.3, which is the short (p) arm of chromosome 8 at position 23.3Molecular Location: base pairs 1,823,976 to 1,958,641 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GEF10SNCV     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ARHGEF10      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: RHO GUANINE NUCLEOTIDE EXCHANGE FACTOR 10OMIM: SLOWED NERVE CONDUCTION VELOCITY, AUTOSOMAL DOMINANT      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ARHGEF10  OMIM: RHO GUANINE NUCLEOTIDE EXCHANGE FACTOR 10OMIM: SLOWED NERVE CONDUCTION VELOCITY, AUTOSOMAL DOMINANT  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "LPL\t   The LPL gene provides instructions for making an enzyme called lipoprotein lipase. This enzyme is found primarily on the surface of cells that line tiny blood vessels (capillaries) within muscles and in fatty (adipose) tissue. Lipoprotein lipase plays a critical role in breaking down fat in the form of triglycerides, which are carried from various organs to the blood by molecules called lipoproteins.Lipoprotein lipase breaks down triglycerides carried by two different types of lipoproteins, which bring fat to the bloodstream from different organs. Fat from the intestine, which is taken in from the diet, is transported to the bloodstream by lipoproteins called chylomicrons. Another type of lipoprotein called very low density lipoprotein (VLDL) carries triglycerides from the liver to the bloodstream. When lipoprotein lipase breaks down triglycerides, the fat molecules are used by the body as energy or stored in fatty tissue for later use.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Familial lipoprotein lipase deficiency     More than 220 mutations in the LPL gene have been found to cause familial lipoprotein lipase deficiency. This condition disrupts the normal breakdown of triglycerides in the body, resulting in an increase of these fats. The most common mutation in people of European ancestry replaces the protein building block (amino acid) glycine with the amino acid glutamic acid at position 188 in the enzyme (written as Gly188Glu or G188E). Mutations that cause familial lipoprotein lipase deficiency reduce or eliminate lipoprotein lipase activity, which prevents the enzyme from effectively breaking down triglycerides in the bloodstream. As a result, triglycerides attached to lipoproteins accumulate in the blood and tissues, leading to inflammation of the pancreas (pancreatitis), enlarged liver and spleen (hepatosplenomegaly), fatty deposits in the skin (eruptive xanthomas), and the other signs and symptoms of familial lipoprotein lipase deficiency. More About This Health Condition      Other disorders     Certain variations in the LPL gene have been shown to influence the levels of fats in the bloodstream. The LPL gene variants likely result in the production of lipoprotein lipase enzymes with altered abilities to break down triglycerides. In some cases, the enzyme is overactive, resulting in low fat levels. In other cases, the enzyme is impaired, resulting in increased fat levels, a condition called hyperlipidemia. Individuals with hyperlipidemia are at greater than normal risk of developing atherosclerosis, a condition in which fatty deposits accumulate on artery walls. This fatty material hardens over time, eventually blocking the arteries and increasing the chance of having a heart attack or stroke. It is unclear how much of a role LPL gene variants play in the development of atherosclerosis, as a large number of genetic and environmental factors determine the risk of developing this complex condition.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 220 mutations in the LPL gene have been found to cause familial lipoprotein lipase deficiency. This condition disrupts the normal breakdown of triglycerides in the body, resulting in an increase of these fats. The most common mutation in people of European ancestry replaces the protein building block (amino acid) glycine with the amino acid glutamic acid at position 188 in the enzyme (written as Gly188Glu or G188E). Mutations that cause familial lipoprotein lipase deficiency reduce or eliminate lipoprotein lipase activity, which prevents the enzyme from effectively breaking down triglycerides in the bloodstream. As a result, triglycerides attached to lipoproteins accumulate in the blood and tissues, leading to inflammation of the pancreas (pancreatitis), enlarged liver and spleen (hepatosplenomegaly), fatty deposits in the skin (eruptive xanthomas), and the other signs and symptoms of familial lipoprotein lipase deficiency. More About This Health Condition  Certain variations in the LPL gene have been shown to influence the levels of fats in the bloodstream. The LPL gene variants likely result in the production of lipoprotein lipase enzymes with altered abilities to break down triglycerides. In some cases, the enzyme is overactive, resulting in low fat levels. In other cases, the enzyme is impaired, resulting in increased fat levels, a condition called hyperlipidemia. Individuals with hyperlipidemia are at greater than normal risk of developing atherosclerosis, a condition in which fatty deposits accumulate on artery walls. This fatty material hardens over time, eventually blocking the arteries and increasing the chance of having a heart attack or stroke. It is unclear how much of a role LPL gene variants play in the development of atherosclerosis, as a large number of genetic and environmental factors determine the risk of developing this complex condition.    Cytogenetic Location: 8p21.3, which is the short (p) arm of chromosome 8 at position 21.3Molecular Location: base pairs 19,939,071 to 19,967,259 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        clearing factor lipasediacylglycerol lipaseLIPDpostheparin lipasetriacylglycerol protein acylhydrolase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Genetic Landscape of Diabetes: The Enzyme Lipoprotein Lipase      Clinical Information from GeneReviews (1 link)     Familial Lipoprotein Lipase Deficiency      Tests Listed in the Genetic Testing Registry (1 link)     Tests of LPL      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     HYPERLIPIDEMIA, FAMILIAL COMBINEDLIPOPROTEIN LIPASE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Genetic Landscape of Diabetes: The Enzyme Lipoprotein Lipase  Familial Lipoprotein Lipase Deficiency  Tests of LPL  PubMed  HYPERLIPIDEMIA, FAMILIAL COMBINEDLIPOPROTEIN LIPASE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Benlian P, De Gennes JL, Foubert L, Zhang H, Gagn SE, Hayden M. Premature atherosclerosis in patients with familial chylomicronemia caused by mutations in the lipoprotein lipase gene. N Engl J Med. 1996 Sep 19;335(12):848-54. Erratum in: N Engl J Med 1997 Feb 6;336(6):451. Citation on PubMedGilbert B, Rouis M, Griglio S, de Lumley L, Laplaud P. Lipoprotein lipase (LPL) deficiency: a new patient homozygote for the preponderant mutation Gly188Glu in the human LPL gene and review of reported mutations: 75 % are clustered in exons 5 and 6. Ann Genet. 2001 Jan-Mar;44(1):25-32. Review. Citation on PubMedKersten S. Physiological regulation of lipoprotein lipase. Biochim Biophys Acta. 2014 Jul;1841(7):919-33. doi: 10.1016/j.bbalip.2014.03.013. Epub 2014 Apr 8. Review. Citation on PubMedMead JR, Irvine SA, Ramji DP. Lipoprotein lipase: structure, function, regulation, and role in disease. J Mol Med (Berl). 2002 Dec;80(12):753-69. Epub 2002 Oct 24. Review. Citation on PubMedMead JR, Ramji DP. The pivotal role of lipoprotein lipase in atherosclerosis. Cardiovasc Res. 2002 Aug 1;55(2):261-9. Review. Citation on PubMedPirim D, Wang X, Radwan ZH, Niemsiri V, Hokanson JE, Hamman RF, Barmada MM, Demirci FY, Kamboh MI. Lipoprotein lipase gene sequencing and plasma lipid profile. J Lipid Res. 2014 Jan;55(1):85-93. doi: 10.1194/jlr.M043265. Epub 2013 Nov 9. Citation on PubMed or Free article on PubMed CentralTang W, Apostol G, Schreiner PJ, Jacobs DR Jr, Boerwinkle E, Fornage M. Associations of lipoprotein lipase gene polymorphisms with longitudinal plasma lipid trends in young adults: The Coronary Artery Risk Development in Young Adults (CARDIA) study. Circ Cardiovasc Genet. 2010 Apr;3(2):179-86. doi: 10.1161/CIRCGENETICS.109.913426. Epub 2010 Feb 11. Citation on PubMed or Free article on PubMed Central \n",
            "GFRA2\t\n",
            "SFTPC\t   The SFTPC gene provides instructions for making a protein called surfactant protein-C (SP-C). This protein is one of four proteins (each produced from a different gene) in surfactant, a mixture of certain fats (called phospholipids) and proteins that lines the lung tissue and makes breathing easy. Without normal surfactant, the tissue surrounding the air sacs in the lungs (the alveoli) sticks together after exhalation (because of a force called surface tension), causing the alveoli to collapse. As a result, filling the lungs with air on each breath becomes very difficult, and the delivery of oxygen to the body is impaired. Surfactant lowers surface tension, easing breathing and avoiding lung collapse. The SP-C protein helps spread the surfactant across the surface of the lung tissue, aiding in the surface tension-lowering property of surfactant.The phospholipids and proteins that make up surfactant are packaged in cellular structures known as lamellar bodies, which are found in specialized lung cells. The surfactant proteins must go through several processing steps to mature and become functional; some of these steps occur in lamellar bodies.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Surfactant dysfunction     More than 35 mutations in the SFTPC gene have been identified in people with surfactant dysfunction. When this condition is caused by mutations in the SFTPC gene (sometimes called SP-C dysfunction), it can cause severe breathing problems in newborns or gradual onset of milder breathing problems in children or adults.SFTPC gene mutations associated with surfactant dysfunction affect the processing of the SP-C protein. Many of the mutations occur in a particular region of the gene called the BRICHOS domain, which appears to be involved in the processing and cellular placement of the SP-C protein.Mutations in the SFTPC gene result in a reduction or absence of mature SP-C and a buildup of abnormal forms of SP-C. It is unclear which of these outcomes causes the signs and symptoms of SP-C dysfunction. Lack of mature SP-C can lead to abnormal composition of surfactant and decreased surfactant function. The loss of functional surfactant would raise surface tension in the alveoli, causing difficulty breathing and collapse of the lungs. Alternatively, research suggests that abnormally processed SP-C proteins form the wrong three-dimensional shape and accumulate inside lung cells. These misfolded proteins may trigger a cellular response that results in cell damage and death. This damage may disrupt surfactant production and release, leading to the breathing problems associated with surfactant dysfunction. More About This Health Condition      Idiopathic pulmonary fibrosis     Genetics Home Reference provides information about idiopathic pulmonary fibrosis. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 35 mutations in the SFTPC gene have been identified in people with surfactant dysfunction. When this condition is caused by mutations in the SFTPC gene (sometimes called SP-C dysfunction), it can cause severe breathing problems in newborns or gradual onset of milder breathing problems in children or adults.SFTPC gene mutations associated with surfactant dysfunction affect the processing of the SP-C protein. Many of the mutations occur in a particular region of the gene called the BRICHOS domain, which appears to be involved in the processing and cellular placement of the SP-C protein.Mutations in the SFTPC gene result in a reduction or absence of mature SP-C and a buildup of abnormal forms of SP-C. It is unclear which of these outcomes causes the signs and symptoms of SP-C dysfunction. Lack of mature SP-C can lead to abnormal composition of surfactant and decreased surfactant function. The loss of functional surfactant would raise surface tension in the alveoli, causing difficulty breathing and collapse of the lungs. Alternatively, research suggests that abnormally processed SP-C proteins form the wrong three-dimensional shape and accumulate inside lung cells. These misfolded proteins may trigger a cellular response that results in cell damage and death. This damage may disrupt surfactant production and release, leading to the breathing problems associated with surfactant dysfunction. More About This Health Condition  Genetics Home Reference provides information about idiopathic pulmonary fibrosis. More About This Health Condition    Cytogenetic Location: 8p21.3, which is the short (p) arm of chromosome 8 at position 21.3Molecular Location: base pairs 22,161,671 to 22,164,479 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BRICD6PSP-CPSPC_HUMANpulmonary surfactant apoprotein-2 SP-Cpulmonary surfactant-associated protein Cpulmonary surfactant-associated proteolipid SPL(Val)SFTP2SMDP2SP-CSP5     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Biology of the Cell (fourth edition, 2002): Adjacent Cell Types Collaborate in the Alveoli of the Lungs      Clinical Information from GeneReviews (1 link)     Pulmonary Fibrosis, Familial      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SFTPC      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     PULMONARY FIBROSIS, IDIOPATHICSURFACTANT, PULMONARY-ASSOCIATED PROTEIN C      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): Adjacent Cell Types Collaborate in the Alveoli of the Lungs  Pulmonary Fibrosis, Familial  Tests of SFTPC  PubMed  PULMONARY FIBROSIS, IDIOPATHICSURFACTANT, PULMONARY-ASSOCIATED PROTEIN C  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Guillot L, Epaud R, Thouvenin G, Jonard L, Mohsni A, Couderc R, Counil F, de Blic J, Taam RA, Le Bourgeois M, Reix P, Flamein F, Clement A, Feldmann D. New surfactant protein C gene mutations associated with diffuse lung disease. J Med Genet. 2009 Jul;46(7):490-4. doi: 10.1136/jmg.2009.066829. Epub 2009 May 13. Erratum in: J Med Genet. 2010 Jul;47(7):485. Citation on PubMedHamvas A, Nogee LM, White FV, Schuler P, Hackett BP, Huddleston CB, Mendeloff EN, Hsu FF, Wert SE, Gonzales LW, Beers MF, Ballard PL. Progressive lung disease and surfactant dysfunction with a deletion in surfactant protein C gene. Am J Respir Cell Mol Biol. 2004 Jun;30(6):771-6. Epub 2003 Dec 4. Citation on PubMedMulugeta S, Maguire JA, Newitt JL, Russo SJ, Kotorashvili A, Beers MF. Misfolded BRICHOS SP-C mutant proteins induce apoptosis via caspase-4- and cytochrome c-related mechanisms. Am J Physiol Lung Cell Mol Physiol. 2007 Sep;293(3):L720-9. Epub 2007 Jun 22. Citation on PubMedOMIM: SURFACTANT, PULMONARY-ASSOCIATED PROTEIN CWeaver TE. Synthesis, processing and secretion of surfactant proteins B and C. Biochim Biophys Acta. 1998 Nov 19;1408(2-3):173-9. Review. Citation on PubMed \n",
            "PEBP4\t\n",
            "R3HCC1\t\n",
            "ADAM28\t\n",
            "ADAMDEC1\t\n",
            "ADAM7\t   From NCBI Gene:This gene encodes a member of the ADAMs family of zinc proteases. These transmembrane proteins play roles in multiple processes including cell signaling, adhesion and migration. The encoded protein lacks protease activity and may play roles in protein-protein interactions and cell adhesion processes including sperm-egg fusion. Mutations in this gene may be involved in the progression of melanoma. [provided by RefSeq, Oct 2011]From UniProt:May play an important role in male reproduction including sperm maturation and gonadotrope function. This is a non catalytic metalloprotease-like protein (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:Has been found to be frequently mutated in melanoma. ADAM7 mutations may play a role in melanoma progression and metastasis.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p21.2, which is the short (p) arm of chromosome 8 at position 21.2Molecular Location: base pairs 24,440,996 to 24,509,565 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ADAM 7ADAM-7EAPIGP-83GP83     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ADAM7      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: A DISINTEGRIN AND METALLOPROTEINASE DOMAIN 7      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ADAM7  OMIM: A DISINTEGRIN AND METALLOPROTEINASE DOMAIN 7  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "EPHX2\t   From NCBI Gene:This gene encodes a member of the epoxide hydrolase family. The protein, found in both the cytosol and peroxisomes, binds to specific epoxides and converts them to the corresponding dihydrodiols. Mutations in this gene have been associated with familial hypercholesterolemia. Alternatively spliced transcript variants have been described. [provided by RefSeq, Feb 2012]From UniProt:Bifunctional enzyme (PubMed:12574510). The C-terminal domain has epoxide hydrolase activity and acts on epoxides (alkene oxides, oxiranes) and arene oxides (PubMed:12869654, PubMed:12574510, PubMed:22798687). Plays a role in xenobiotic metabolism by degrading potentially toxic epoxides (By similarity). Also determines steady-state levels of physiological mediators (PubMed:12869654, PubMed:12574510, PubMed:22798687). The N-terminal domain has lipid phosphatase activity, with the highest activity towards threo-9,10-phosphonooxy-hydroxy-octadecanoic acid, followed by erythro-9,10-phosphonooxy-hydroxy-octadecanoic acid, 12-phosphonooxy-octadec-9Z-enoic acid and 12-phosphonooxy-octadec-9E-enoic acid (PubMed:12574510).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Familial hypercholesterolemia     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p21.2-p21.1, which is the short (p) arm of chromosome 8 between positions 21.2 and 21.1Molecular Location: base pairs 27,491,002 to 27,597,559 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ABHD20CEHSEH     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of EPHX2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: EPOXIDE HYDROLASE 2, CYTOSOLICOMIM: HYPERCHOLESTEROLEMIA, FAMILIAL      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of EPHX2  OMIM: EPOXIDE HYDROLASE 2, CYTOSOLICOMIM: HYPERCHOLESTEROLEMIA, FAMILIAL  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "CSMD1\t\n",
            "UBXN8\t\n",
            "NRG1\t   From NCBI Gene:The protein encoded by this gene is a membrane glycoprotein that mediates cell-cell signaling and plays a critical role in the growth and development of multiple organ systems. An extraordinary variety of different isoforms are produced from this gene through alternative promoter usage and splicing. These isoforms are expressed in a tissue-specific manner and differ significantly in their structure, and are classified as types I, II, III, IV, V and VI. Dysregulation of this gene has been linked to diseases such as cancer, schizophrenia, and bipolar disorder (BPD). [provided by RefSeq, Apr 2016]From UniProt:Direct ligand for ERBB3 and ERBB4 tyrosine kinase receptors. Concomitantly recruits ERBB1 and ERBB2 coreceptors, resulting in ligand-stimulated tyrosine phosphorylation and activation of the ERBB receptors. The multiple isoforms perform diverse functions such as inducing growth and differentiation of epithelial, glial, neuronal, and skeletal muscle cells; inducing expression of acetylcholine receptor in synaptic vesicles during the formation of the neuromuscular junction; stimulating lobuloalveolar budding and milk production in the mammary gland and inducing differentiation of mammary tumor cells; stimulating Schwann cell proliferation; implication in the development of the myocardium such as trabeculation of the developing heart. Isoform 10 may play a role in motor and sensory neuron development. Binds to ERBB4 (PubMed:10867024, PubMed:7902537). Binds to ERBB3 (PubMed:20682778). Acts as a ligand for integrins and binds (via EGF domain) to integrins ITGAV:ITGB3 or ITGA6:ITGB4. Its binding to integrins and subsequent ternary complex formation with integrins and ERRB3 are essential for NRG1-ERBB signaling. Induces the phosphorylation and activation of MAPK3/ERK1, MAPK1/ERK2 and AKT1 (PubMed:20682778). Ligand-dependent ERBB4 endocytosis is essential for the NRG1-mediated activation of these kinases in neurons (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Hirschsprung diseaseLung cancerFrom NCBI Gene:Schizophrenia 6From UniProt:A chromosomal aberration involving NRG1 produces gamma-heregulin. Translocation t(8;11) with TENM4. The translocation fuses the 5'-end of TENM4 to NRG1 (isoform 8). The product of this translocation was first thought to be an alternatively spliced isoform. Gamma-heregulin is a soluble activating ligand for the ERBB2-ERBB3 receptor complex and acts as an autocrine growth factor in a specific breast cancer cell line (MDA-MB-175). Not detected in breast carcinoma samples, including ductal, lobular, medullary, and mucinous histological types, neither in other breast cancer cell lines.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p12, which is the short (p) arm of chromosome 8 at position 12Molecular Location: base pairs 31,639,222 to 32,771,716 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ARIAGGFGGF2HGLHRGHRG1HRGAMST131MSTP131NDFNRG1-IT2SMDF     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of NRG1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: NEUREGULIN 1OMIM: SCHIZOPHRENIA 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of NRG1  OMIM: NEUREGULIN 1OMIM: SCHIZOPHRENIA 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RAB11FIP1\t\n",
            "PLEKHA2\t\n",
            "IDO2\t\n",
            "TDRP\t\n",
            "PXDNL\t\n",
            "RP1\t   From NCBI Gene:This gene encodes a member of the doublecortin family. The protein encoded by this gene contains two doublecortin domains, which bind microtubules and regulate microtubule polymerization. The encoded protein is a photoreceptor microtubule-associated protein and is required for correct stacking of outer segment disc. This protein and the RP1L1 protein, another retinal-specific protein, play essential and synergistic roles in affecting photosensitivity and outer segment morphogenesis of rod photoreceptors. Because of its response to in vivo retinal oxygen levels, this protein was initially named ORP1 (oxygen-regulated protein-1). This protein was subsequently designated RP1 (retinitis pigmentosa 1) when it was found that mutations in this gene cause autosomal dominant retinitis pigmentosa. Mutations in this gene also cause autosomal recessive retinitis pigmentosa. Transcript variants resulted from an alternative promoter and alternative splicings have been found, which overlap the current reference sequence and has several exons upstream and downstream of the current reference sequence. However, the biological validity and full-length nature of some variants cannot be determined at this time.[provided by RefSeq, Sep 2010]From UniProt:Microtubule-associated protein regulating the stability and length of the microtubule-based axoneme of photoreceptors. Required for the differentiation of photoreceptor cells, it plays a role in the organization of the outer segment of rod and cone photoreceptors ensuring the correct orientation and higher-order stacking of outer segment disks along the photoreceptor axoneme (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Retinitis pigmentosaFrom NCBI Gene:Retinitis pigmentosa 1From UniProt:Retinitis pigmentosa 1 (RP1): A retinal dystrophy belonging to the group of pigmentary retinopathies. Retinitis pigmentosa is characterized by retinal pigment deposits visible on fundus examination and primary loss of rod photoreceptor cells followed by secondary loss of cone photoreceptors. Patients typically have night vision blindness and loss of midperipheral visual field. As their condition progresses, they lose their far peripheral visual field and eventually central vision as well. [MIM:180100]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8q11.23-q12.1, which is the long (q) arm of chromosome 8 between positions 11.23 and 12.1Molecular Location: base pairs 54,509,448 to 54,630,834 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        DCDC4AORP1     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RP1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: RETINITIS PIGMENTOSA 1OMIM: RP1 GENE      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RP1  OMIM: RETINITIS PIGMENTOSA 1OMIM: RP1 GENE  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MCPH1\t   From NCBI Gene:This gene encodes a DNA damage response protein. The encoded protein may play a role in G2/M checkpoint arrest via maintenance of inhibitory phosphorylation of cyclin-dependent kinase 1. Mutations in this gene have been associated with primary autosomal recessive microcephaly 1 and premature chromosome condensation syndrome. Alternatively spliced transcript variants have been described. [provided by RefSeq, Feb 2010]From UniProt:Implicated in chromosome condensation and DNA damage induced cellular responses. May play a role in neurogenesis and regulation of the size of the cerebral cortex.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autosomal recessive primary microcephalyFrom NCBI Gene:Primary autosomal recessive microcephaly 1From UniProt:Microcephaly 1, primary, autosomal recessive (MCPH1): A disease defined as a head circumference more than 3 standard deviations below the age-related mean. Brain weight is markedly reduced and the cerebral cortex is disproportionately small. Despite this marked reduction in size, the gyral pattern is relatively well preserved, with no major abnormality in cortical architecture. Affected individuals are mentally retarded. Primary microcephaly is further defined by the absence of other syndromic features or significant neurological deficits due to degenerative brain disorder. Some MCHP1 patients also present growth retardation, short stature, and misregulated chromosome condensation as indicated by a high number of prophase-like cells detected in routine cytogenetic preparations and poor-quality metaphase G-banding. [MIM:251200]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 8p23.1, which is the short (p) arm of chromosome 8 at position 23.1Molecular Location: base pairs 6,406,592 to 6,648,505 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BRIT1MCT     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of MCPH1      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MCPH1 GENEOMIM: MICROCEPHALY 1, PRIMARY, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of MCPH1  OMIM: MCPH1 GENEOMIM: MICROCEPHALY 1, PRIMARY, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "BHLHE22\t\n",
            "MTFR1\t\n",
            "ZNF705G\t\n",
            "TERF1\t\n",
            "TMEM70\t   The TMEM70 gene provides instructions for making a protein called transmembrane protein 70. This protein is found in cell structures called mitochondria, which convert the energy from food into a form that cells can use. Transmembrane protein 70 is thought to play an important role in assembling and stabilizing a group of proteins called complex V. Complex V is the last of five complexes that carry out a multistep process called oxidative phosphorylation, through which cells derive much of their energy. Complex V is involved in the final step of oxidative phosphorylation. Specifically, one segment of complex V allows positively charged particles, called protons, to flow across a specialized membrane inside mitochondria. Another segment of complex V uses the energy created by this proton flow to convert a molecule called adenosine diphosphate (ADP) to adenosine triphosphate (ATP), which is used by the cell as energy.Transmembrane protein 70 is also thought to be involved in the assembly of complex I, which is the first mitochondrial complex involved in oxidative phosphorylation.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Mitochondrial complex V deficiency     At least 12 mutations in the TMEM70 gene have been identified in people who have mitochondrial complex V deficiency, a disorder with a wide variety of signs and symptoms. A few of these gene mutations are particular to people of Roma or Arab descent, and account for the majority of mitochondrial complex V deficiency cases caused by TMEM70 gene mutations. This disorder can also be caused by mutations in other genes.The signs and symptoms of mitochondrial complex V deficiency are most prominent in organs and tissues that require a large amount of energy, such as the brain and heart. Abnormal brain function (encephalopathy) and other neurological problems can occur. Another common feature of mitochondrial complex V deficiency, especially when caused by TMEM70 gene mutations, is hypertrophic cardiomyopathy. This condition is characterized by thickening (hypertrophy) of the heart (cardiac) muscle that can lead to heart failure. TMEM70 gene mutations alter transmembrane protein 70 and impair its ability to perform its function in complex V assembly. As a result, the amount of complex V in cells is reduced. The resulting impairment of oxidative phosphorylation and energy production leads to the signs and symptoms of mitochondrial complex V deficiency. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 12 mutations in the TMEM70 gene have been identified in people who have mitochondrial complex V deficiency, a disorder with a wide variety of signs and symptoms. A few of these gene mutations are particular to people of Roma or Arab descent, and account for the majority of mitochondrial complex V deficiency cases caused by TMEM70 gene mutations. This disorder can also be caused by mutations in other genes.The signs and symptoms of mitochondrial complex V deficiency are most prominent in organs and tissues that require a large amount of energy, such as the brain and heart. Abnormal brain function (encephalopathy) and other neurological problems can occur. Another common feature of mitochondrial complex V deficiency, especially when caused by TMEM70 gene mutations, is hypertrophic cardiomyopathy. This condition is characterized by thickening (hypertrophy) of the heart (cardiac) muscle that can lead to heart failure. TMEM70 gene mutations alter transmembrane protein 70 and impair its ability to perform its function in complex V assembly. As a result, the amount of complex V in cells is reduced. The resulting impairment of oxidative phosphorylation and energy production leads to the signs and symptoms of mitochondrial complex V deficiency. More About This Health Condition    Cytogenetic Location: 8q21.11, which is the long (q) arm of chromosome 8 at position 21.11Molecular Location: base pairs 73,976,142 to 73,982,783 on chromosome 8 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FLJ20533MC5DN2transmembrane protein 70, mitochondrial isoform atransmembrane protein 70, mitochondrial isoform b     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     The Cell -- A Molecular Approach (second edition, 2000): The Mechanism of Oxidative Phosphorylation      Clinical Information from GeneReviews (1 link)     Mitochondrial Disorders Overview      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TMEM70      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TRANSMEMBRANE PROTEIN 70      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      The Cell -- A Molecular Approach (second edition, 2000): The Mechanism of Oxidative Phosphorylation  Mitochondrial Disorders Overview  Tests of TMEM70  PubMed  TRANSMEMBRANE PROTEIN 70  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Diodato D, Invernizzi F, Lamantea E, Fagiolari G, Parini R, Menni F, Parenti G, Bollani L, Pasquini E, Donati MA, Cassandrini D, Santorelli FM, Haack TB, Prokisch H, Ghezzi D, Lamperti C, Zeviani M. Common and Novel TMEM70 Mutations in a Cohort of Italian Patients with Mitochondrial Encephalocardiomyopathy. JIMD Rep. 2015;15:71-8. doi: 10.1007/8904_2014_300. Epub 2014 Apr 17. Citation on PubMed or Free article on PubMed CentralGuerrero-Castillo S, Baertling F, Kownatzki D, Wessels HJ, Arnold S, Brandt U, Nijtmans L. The Assembly Pathway of Mitochondrial Respiratory Chain Complex I. Cell Metab. 2017 Jan 10;25(1):128-139. doi: 10.1016/j.cmet.2016.09.002. Epub 2016 Oct 6. Citation on PubMedHejzlarov K, Mrek T, Vrback M, Kaplanov V, Karbanov V, Nskov H, Pecina P, Houtk J. Nuclear genetic defects of mitochondrial ATP synthase. Physiol Res. 2014;63 Suppl 1:S57-71. Review. Citation on PubMedMagner M, Dvorakova V, Tesarova M, Mazurova S, Hansikova H, Zahorec M, Brennerova K, Bzduch V, Spiegel R, Horovitz Y, Mandel H, Eminolu FT, Mayr JA, Koch J, Martinelli D, Bertini E, Konstantopoulou V, Smet J, Rahman S, Broomfield A, Stojanovi V, Dionisi-Vici C, van Coster R, Morava E, Sperl W, Zeman J, Honzik T. TMEM70 deficiency: long-term outcome of 48 patients. J Inherit Metab Dis. 2015 May;38(3):417-26. doi: 10.1007/s10545-014-9774-8. Epub 2014 Oct 18. Erratum in: J Inherit Metab Dis. 2015 May;38(3):583-4. Morava-Kozicz, Eva [corrected to Morava, Eva]. Citation on PubMedSpiegel R, Khayat M, Shalev SA, Horovitz Y, Mandel H, Hershkovitz E, Barghuti F, Shaag A, Saada A, Korman SH, Elpeleg O, Yatsiv I. TMEM70 mutations are a common cause of nuclear encoded ATP synthase assembly defect: further delineation of a new syndrome. J Med Genet. 2011 Mar;48(3):177-82. doi: 10.1136/jmg.2010.084608. Epub 2010 Dec 8. Citation on PubMedOMIM: TRANSMEMBRANE PROTEIN 70Torraco A, Verrigni D, Rizza T, Meschini MC, Vazquez-Memije ME, Martinelli D, Bianchi M, Piemonte F, Dionisi-Vici C, Santorelli FM, Bertini E, Carrozzo R. TMEM70: a mutational hot spot in nuclear ATP synthase deficiency with a pivotal role in complex V biogenesis. Neurogenetics. 2012 Nov;13(4):375-86. doi: 10.1007/s10048-012-0343-8. Epub 2012 Sep 18. Citation on PubMed \n",
            "SPAG11A\t\n",
            "ZFHX4\t\n",
            "TPD52\t\n",
            "PRAG1\t\n",
            "LRRCC1\t\n",
            "SLC7A13\t\n",
            "ERI1\t\n",
            "CCDC180\t\n",
            "FOXE1\t   From NCBI Gene:This intronless gene encodes a protein that belongs to the forkhead family of transcription factors. Members of this family contain a conserved 100-amino acid DNA-binding 'forkhead' domain. The encoded protein functions as a thyroid transcription factor that plays a role in thyroid morphogenesis. Mutations in this gene are associated with the Bamforth-Lazarus syndrome, and with susceptibility to nonmedullary thyroid cancer-4. [provided by RefSeq, Nov 2016]From UniProt:Transcription factor that binds consensus sites on a variety of gene promoters and activate their transcription. Involved in proper palate formation, most probably through the expression of MSX1 and TGFB3 genes which are direct targets of this transcription factor. Also implicated in thyroid gland morphogenesis. May indirectly play a role in cell growth and migration through the regulation of WNT5A expression.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Thyroid cancer, nonmedullary, 4Bamforth syndromeFrom UniProt:Thyroid cancer, non-medullary, 4 (NMTC4): A form of non-medullary thyroid cancer (NMTC), a cancer characterized by tumors originating from the thyroid follicular cells. NMTCs represent approximately 95% of all cases of thyroid cancer and are classified into papillary, follicular, Hurthle cell, and anaplastic neoplasms. [MIM:616534]Bamforth-Lazarus syndrome (BLS): A disease characterized by thyroid agenesis, cleft palate and choanal atresia. [MIM:241850]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q22.33, which is the long (q) arm of chromosome 9 at position 22.33Molecular Location: base pairs 97,853,255 to 97,856,715 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        FKHL15FOXE2HFKH4HFKL5NMTC4TITF2TTF-2TTF2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of FOXE1      Catalog of Genes and Diseases from OMIM (3 links)     OMIM: FORKHEAD BOX E1OMIM: HYPOTHYROIDISM, THYROIDAL OR ATHYROIDAL, WITH SPIKY HAIR AND CLEFT PALATEOMIM: THYROID CANCER, NONMEDULLARY, 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of FOXE1  OMIM: FORKHEAD BOX E1OMIM: HYPOTHYROIDISM, THYROIDAL OR ATHYROIDAL, WITH SPIKY HAIR AND CLEFT PALATEOMIM: THYROID CANCER, NONMEDULLARY, 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TBC1D2\t\n",
            "MRPL50\t\n",
            "ALDOB\t   The ALDOB gene provides instructions for making the aldolase B enzyme. This enzyme is one of a group of three aldolase enzymes that are responsible for breaking down certain molecules in cells throughout the body. Four identical aldolase B enzymes need to be attached (bound) to each other in a four-enzyme unit called a tetramer to work.Aldolase B is found primarily in the liver, but it is also present at lower levels in kidney and intestinal cells. Aldolase B is involved in the breakdown (metabolism) of the simple sugar fructose, which is found mostly in fruits and is used in the body for energy. Aldolase B is responsible for the second step in the metabolism of fructose, which breaks down the molecule fructose-1-phosphate into glyceraldehyde and dihydroxyacetone phosphate. To a lesser degree, aldolase B is also involved in the breakdown of the simple sugar glucose.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Hereditary fructose intolerance     More than 50 mutations in the ALDOB gene have been found to cause hereditary fructose intolerance, a condition characterized by nausea and intestinal discomfort following ingestion of foods containing fructose. Most of these mutations replace single protein building blocks (amino acids) in the aldolase B enzyme and result in the production of an enzyme with reduced function. A mutation found in approximately half of people with hereditary fructose intolerance replaces the amino acid alanine with the amino acid proline at position 149 in the enzyme (written as Ala149Pro or A149P). This mutation alters the 3-dimensional shape of the enzyme. Alteration of the shape of the enzyme makes it difficult for the aldolase B enzymes to bind together and form tetramers. If it is not in a tetramer, aldolase B cannot metabolize fructose.A lack of functional aldolase B results in an accumulation of fructose-1-phosphate in liver cells. This buildup is toxic, resulting in the death of liver cells over time. Additionally, the breakdown products of fructose-1-phosphase are needed in the body to produce energy and to maintain blood sugar levels. The combination of decreased cellular energy, low blood sugar, and liver cell death leads to the features of hereditary fructose intolerance. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 50 mutations in the ALDOB gene have been found to cause hereditary fructose intolerance, a condition characterized by nausea and intestinal discomfort following ingestion of foods containing fructose. Most of these mutations replace single protein building blocks (amino acids) in the aldolase B enzyme and result in the production of an enzyme with reduced function. A mutation found in approximately half of people with hereditary fructose intolerance replaces the amino acid alanine with the amino acid proline at position 149 in the enzyme (written as Ala149Pro or A149P). This mutation alters the 3-dimensional shape of the enzyme. Alteration of the shape of the enzyme makes it difficult for the aldolase B enzymes to bind together and form tetramers. If it is not in a tetramer, aldolase B cannot metabolize fructose.A lack of functional aldolase B results in an accumulation of fructose-1-phosphate in liver cells. This buildup is toxic, resulting in the death of liver cells over time. Additionally, the breakdown products of fructose-1-phosphase are needed in the body to produce energy and to maintain blood sugar levels. The combination of decreased cellular energy, low blood sugar, and liver cell death leads to the features of hereditary fructose intolerance. More About This Health Condition    Cytogenetic Location: 9q31.1, which is the long (q) arm of chromosome 9 at position 31.1Molecular Location: base pairs 101,420,560 to 101,435,780 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ALDBALDO2ALDOB_HUMANaldolase 2aldolase B, fructose-bisphosphatasealdolase B, fructose-bisphosphatefructose-bisphosphate aldolase Bliver-type aldolase     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): The Entry of Fructose and Galactose into Glycolysis      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ALDOB      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ALDOLASE B, FRUCTOSE-BISPHOSPHATE      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyBoston University: Hereditary Fructose Intolerance Mutational DatabaseClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): The Entry of Fructose and Galactose into Glycolysis  Tests of ALDOB  PubMed  ALDOLASE B, FRUCTOSE-BISPHOSPHATE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyBoston University: Hereditary Fructose Intolerance Mutational DatabaseClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ALDOLASE B, FRUCTOSE-BISPHOSPHATEDalby AR, Tolan DR, Littlechild JA. The structure of human liver fructose-1,6-bisphosphate aldolase. Acta Crystallogr D Biol Crystallogr. 2001 Nov;57(Pt 11):1526-33. Epub 2001 Oct 25. Citation on PubMedEsposito G, Vitagliano L, Santamaria R, Viola A, Zagari A, Salvatore F. Structural and functional analysis of aldolase B mutants related to hereditary fructose intolerance. FEBS Lett. 2002 Nov 6;531(2):152-6. Citation on PubMedMalay AD, Allen KN, Tolan DR. Structure of the thermolabile mutant aldolase B, A149P: molecular basis of hereditary fructose intolerance. J Mol Biol. 2005 Mar 18;347(1):135-44. Epub 2005 Jan 20. Citation on PubMedSanter R, Rischewski J, von Weihe M, Niederhaus M, Schneppenheim S, Baerlocher K, Kohlschtter A, Muntau A, Posselt HG, Steinmann B, Schneppenheim R. The spectrum of aldolase B (ALDOB) mutations and the prevalence of hereditary fructose intolerance in Central Europe. Hum Mutat. 2005 Jun;25(6):594. Citation on PubMed \n",
            "GRIN3A\t\n",
            "OR13F1\t\n",
            "OR13C5\t\n",
            "FSD1L\t\n",
            "FKTN\t   The FKTN gene (formerly known as FCMD) provides instructions for making a protein called fukutin. This protein is present in many of the body's tissues but is particularly abundant in heart (cardiac) muscle, the brain, and the muscles used for movement (skeletal muscles). Within cells, fukutin is found in a specialized structure called the Golgi apparatus, where newly produced proteins are modified.The fukutin protein is involved in a protein modification process called glycosylation. Through this chemical process, sugar molecules are added to certain proteins. In particular, the fukutin protein adds a molecule called ribitol 5-phosphate to the chain of sugars attached to a protein called alpha ()-dystroglycan. Glycosylation is critical for the normal function of -dystroglycan.The -dystroglycan protein helps anchor the structural framework inside each cell (cytoskeleton) to the lattice of proteins and other molecules outside the cell (extracellular matrix). In skeletal muscles, glycosylated -dystroglycan helps stabilize and protect muscle fibers. In the brain, it helps direct the movement (migration) of nerve cells (neurons) during early development.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Fukuyama congenital muscular dystrophy     At least 18 mutations in the FKTN gene have been found to cause Fukuyama congenital muscular dystrophy, a condition that causes skeletal muscle weakness and brain and eye abnormalities. This form of congenital muscular dystrophy is seen almost exclusively in Japan. Virtually everyone with this condition has at least one copy of the same mutation, an insertion of about 3,000 extra DNA building blocks (3 kilobases [kb]) in the FKTN gene. This insertion occurs in a part of the gene known as the 3' untranslated region, which helps regulate the gene's activity. Researchers believe that the 3-kb insertion reduces the amount of fukutin protein that is produced from the gene.A shortage of fukutin prevents the normal glycosylation of -dystroglycan. As a result, -dystroglycan can no longer effectively anchor cells to the proteins and other molecules that surround them. Without functional -dystroglycan to stabilize the muscle fibers, they become damaged as they repeatedly contract and relax with use. The damaged fibers weaken and die over time, which affects the development, structure, and function of skeletal muscles in people with Fukuyama congenital muscular dystrophy.Defective -dystroglycan also affects the migration of neurons during the early development of the brain. Instead of stopping when they reach their intended destinations, some neurons migrate past the surface of the brain into the fluid-filled space that surrounds it. Researchers believe that this problem with neuronal migration causes a brain abnormality called cobblestone lissencephaly, in which the surface of the brain lacks the normal folds and grooves and instead appears bumpy and irregular. Less is known about the effects of FKTN mutations in other parts of the body. More About This Health Condition      Limb-girdle muscular dystrophy     Genetics Home Reference provides information about limb-girdle muscular dystrophy. More About This Health Condition      Walker-Warburg syndrome     At least 11 FKTN gene mutations have been identified in people with Walker-Warburg syndrome, the most severe form of congenital muscular dystrophy. This condition is found in populations worldwide. Like Fukuyama congenital muscular dystrophy (described above), Walker-Warburg syndrome is associated with skeletal muscle weakness and eye and brain abnormalities, including cobblestone lissencephaly; however, individuals with Walker-Warburg syndrome have more severe brain and eye abnormalities and live only into infancy or early childhood. The FKTN gene mutations associated with this condition prevent the production of any functional fukutin protein, which leads to the severe muscle, eye, and brain problems that develop in Walker-Warburg syndrome. More About This Health Condition      Other disorders     Mutations in the FKTN gene cause other disorders that affect skeletal muscles and the heart. Unlike Fukuyama congenital muscular dystrophy (described above), which is mostly limited to the Japanese population, these conditions have been described in several populations worldwide.Several people with a heart condition called dilated cardiomyopathy have been found to have mutations in the FKTN gene. This condition weakens and enlarges the heart, preventing it from pumping blood efficiently. When dilated cardiomyopathy is associated with FKTN gene mutations, it is known as type 1X (DCM1X). In addition to heart problems, some people with DCM1X have experienced mild skeletal muscle weakness beginning in adulthood.Changes in the FKTN gene that reduce but do not eliminate the production of fukutin lead to the somewhat less severe medical problems seen in DCM1X and limb-girdle muscular dystrophy (linked above).        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 18 mutations in the FKTN gene have been found to cause Fukuyama congenital muscular dystrophy, a condition that causes skeletal muscle weakness and brain and eye abnormalities. This form of congenital muscular dystrophy is seen almost exclusively in Japan. Virtually everyone with this condition has at least one copy of the same mutation, an insertion of about 3,000 extra DNA building blocks (3 kilobases [kb]) in the FKTN gene. This insertion occurs in a part of the gene known as the 3' untranslated region, which helps regulate the gene's activity. Researchers believe that the 3-kb insertion reduces the amount of fukutin protein that is produced from the gene.A shortage of fukutin prevents the normal glycosylation of -dystroglycan. As a result, -dystroglycan can no longer effectively anchor cells to the proteins and other molecules that surround them. Without functional -dystroglycan to stabilize the muscle fibers, they become damaged as they repeatedly contract and relax with use. The damaged fibers weaken and die over time, which affects the development, structure, and function of skeletal muscles in people with Fukuyama congenital muscular dystrophy.Defective -dystroglycan also affects the migration of neurons during the early development of the brain. Instead of stopping when they reach their intended destinations, some neurons migrate past the surface of the brain into the fluid-filled space that surrounds it. Researchers believe that this problem with neuronal migration causes a brain abnormality called cobblestone lissencephaly, in which the surface of the brain lacks the normal folds and grooves and instead appears bumpy and irregular. Less is known about the effects of FKTN mutations in other parts of the body. More About This Health Condition  Genetics Home Reference provides information about limb-girdle muscular dystrophy. More About This Health Condition  At least 11 FKTN gene mutations have been identified in people with Walker-Warburg syndrome, the most severe form of congenital muscular dystrophy. This condition is found in populations worldwide. Like Fukuyama congenital muscular dystrophy (described above), Walker-Warburg syndrome is associated with skeletal muscle weakness and eye and brain abnormalities, including cobblestone lissencephaly; however, individuals with Walker-Warburg syndrome have more severe brain and eye abnormalities and live only into infancy or early childhood. The FKTN gene mutations associated with this condition prevent the production of any functional fukutin protein, which leads to the severe muscle, eye, and brain problems that develop in Walker-Warburg syndrome. More About This Health Condition  Mutations in the FKTN gene cause other disorders that affect skeletal muscles and the heart. Unlike Fukuyama congenital muscular dystrophy (described above), which is mostly limited to the Japanese population, these conditions have been described in several populations worldwide.Several people with a heart condition called dilated cardiomyopathy have been found to have mutations in the FKTN gene. This condition weakens and enlarges the heart, preventing it from pumping blood efficiently. When dilated cardiomyopathy is associated with FKTN gene mutations, it is known as type 1X (DCM1X). In addition to heart problems, some people with DCM1X have experienced mild skeletal muscle weakness beginning in adulthood.Changes in the FKTN gene that reduce but do not eliminate the production of fukutin lead to the somewhat less severe medical problems seen in DCM1X and limb-girdle muscular dystrophy (linked above).    Cytogenetic Location: 9q31.2, which is the long (q) arm of chromosome 9 at position 31.2Molecular Location: base pairs 105,558,117 to 105,655,950 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CMD1XFCMDFKTN_HUMANFukuyama type congenital muscular dystrophy proteinLGMD2MMGC126857MGC134944MGC134945MGC138243     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): Protein Glycosylation in the ER and Golgi ComplexNeuromuscular Disease Center, Washington University      Clinical Information from GeneReviews (2 links)     Dilated Cardiomyopathy OverviewFukuyama Congenital Muscular Dystrophy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of FKTN      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (2 links)     CARDIOMYOPATHY, DILATED, 1XFUKUTIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Protein Glycosylation in the ER and Golgi ComplexNeuromuscular Disease Center, Washington University  Dilated Cardiomyopathy OverviewFukuyama Congenital Muscular Dystrophy  Tests of FKTN  PubMed  CARDIOMYOPATHY, DILATED, 1XFUKUTIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Cotarelo RP, Valero MC, Prados B, Pea A, Rodrguez L, Fano O, Marco JJ, Martnez-Fras ML, Cruces J. Two new patients bearing mutations in the fukutin gene confirm the relevance of this gene in Walker-Warburg syndrome. Clin Genet. 2008 Feb;73(2):139-45. doi: 10.1111/j.1399-0004.2007.00936.x. Epub 2007 Dec 19. Citation on PubMedGerin I, Ury B, Breloy I, Bouchet-Seraphin C, Bolse J, Halbout M, Graff J, Vertommen D, Muccioli GG, Seta N, Cuisset JM, Dabaj I, Quijano-Roy S, Grahn A, Van Schaftingen E, Bommer GT. ISPD produces CDP-ribitol used by FKTN and FKRP to transfer ribitol phosphate onto -dystroglycan. Nat Commun. 2016 May 19;7:11534. doi: 10.1038/ncomms11534. Citation on PubMed or Free article on PubMed CentralGodfrey C, Escolar D, Brockington M, Clement EM, Mein R, Jimenez-Mallebrera C, Torelli S, Feng L, Brown SC, Sewry CA, Rutherford M, Shapira Y, Abbs S, Muntoni F. Fukutin gene mutations in steroid-responsive limb girdle muscular dystrophy. Ann Neurol. 2006 Nov;60(5):603-10. Citation on PubMedHayashi YK, Ogawa M, Tagawa K, Noguchi S, Ishihara T, Nonaka I, Arahata K. Selective deficiency of alpha-dystroglycan in Fukuyama-type congenital muscular dystrophy. Neurology. 2001 Jul 10;57(1):115-21. Citation on PubMedKanagawa M, Kobayashi K, Tajiri M, Manya H, Kuga A, Yamaguchi Y, Akasaka-Manya K, Furukawa J, Mizuno M, Kawakami H, Shinohara Y, Wada Y, Endo T, Toda T. Identification of a Post-translational Modification with Ribitol-Phosphate and Its Defect in Muscular Dystrophy. Cell Rep. 2016 Mar 8;14(9):2209-23. doi: 10.1016/j.celrep.2016.02.017. Citation on PubMedKondo-Iida E, Kobayashi K, Watanabe M, Sasaki J, Kumagai T, Koide H, Saito K, Osawa M, Nakamura Y, Toda T. Novel mutations and genotype-phenotype relationships in 107 families with Fukuyama-type congenital muscular dystrophy (FCMD). Hum Mol Genet. 1999 Nov;8(12):2303-9. Citation on PubMedMurakami T, Hayashi YK, Noguchi S, Ogawa M, Nonaka I, Tanabe Y, Ogino M, Takada F, Eriguchi M, Kotooka N, Campbell KP, Osawa M, Nishino I. Fukutin gene mutations cause dilated cardiomyopathy with minimal muscle weakness. Ann Neurol. 2006 Nov;60(5):597-602. Citation on PubMedSaito Y, Yamamoto T, Mizuguchi M, Kobayashi M, Saito K, Ohno K, Osawa M. Altered glycosylation of alpha-dystroglycan in neurons of Fukuyama congenital muscular dystrophy brains. Brain Res. 2006 Feb 23;1075(1):223-8. Epub 2006 Feb 7. Citation on PubMedToda T, Kobayashi K, Takeda S, Sasaki J, Kurahashi H, Kano H, Tachikawa M, Wang F, Nagai Y, Taniguchi K, Taniguchi M, Sunada Y, Terashima T, Endo T, Matsumura K. Fukuyama-type congenital muscular dystrophy (FCMD) and alpha-dystroglycanopathy. Congenit Anom (Kyoto). 2003 Jun;43(2):97-104. Review. Citation on PubMedYis U, Uyanik G, Heck PB, Smitka M, Nobel H, Ebinger F, Dirik E, Feng L, Kurul SH, Brocke K, Unalp A, zer E, Cakmakci H, Sewry C, Cirak S, Muntoni F, Hehr U, Morris-Rosendahl DJ. Fukutin mutations in non-Japanese patients with congenital muscular dystrophy: less severe mutations predominate in patients with a non-Walker-Warburg phenotype. Neuromuscul Disord. 2011 Jan;21(1):20-30. doi: 10.1016/j.nmd.2010.08.007. Epub 2010 Oct 18. Citation on PubMedde Bernab DB, van Bokhoven H, van Beusekom E, Van den Akker W, Kant S, Dobyns WB, Cormand B, Currier S, Hamel B, Talim B, Topaloglu H, Brunner HG. A homozygous nonsense mutation in the fukutin gene causes a Walker-Warburg syndrome phenotype. J Med Genet. 2003 Nov;40(11):845-8. Citation on PubMed or Free article on PubMed Central \n",
            "TXNDC8\t\n",
            "SVEP1\t\n",
            "C9orf43\t\n",
            "RGS3\t\n",
            "CDK5RAP2\t   From NCBI Gene:This gene encodes a regulator of CDK5 (cyclin-dependent kinase 5) activity. The protein encoded by this gene is localized to the centrosome and Golgi complex, interacts with CDK5R1 and pericentrin (PCNT), plays a role in centriole engagement and microtubule nucleation, and has been linked to primary microcephaly and Alzheimer's disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2013]From UniProt:Potential regulator of CDK5 activity via its interaction with CDK5R1. Negative regulator of centriole disengagement (licensing) which maintains centriole engagement and cohesion. Involved in regulation of mitotic spindle orientation (By similarity). Plays a role in the spindle checkpoint activation by acting as a transcriptional regulator of both BUBR1 and MAD2 promoter. Together with EB1/MAPRE1, may promote microtubule polymerization, bundle formation, growth and dynamics at the plus ends. Regulates centrosomal maturation by recruitment of the gamma-tubulin ring complex (gamma-TuRC) onto centrosomes (PubMed:26485573). In complex with PDE4DIP isoform 13/MMG8/SMYLE, MAPRE1 and AKAP9, contributes to microtubules nucleation and extension from the centrosome to the cell periphery (PubMed:29162697). Required for the recruitment of AKAP9 to centrosomes (PubMed:29162697). Plays a role in neurogenesis (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Autosomal recessive primary microcephalyFrom NCBI Gene:Primary autosomal recessive microcephaly 3From UniProt:Microcephaly 3, primary, autosomal recessive (MCPH3): A disease defined as a head circumference more than 3 standard deviations below the age-related mean. Brain weight is markedly reduced and the cerebral cortex is disproportionately small. Despite this marked reduction in size, the gyral pattern is relatively well preserved, with no major abnormality in cortical architecture. Affected individuals are mentally retarded. Primary microcephaly is further defined by the absence of other syndromic features or significant neurological deficits due to degenerative brain disorder. [MIM:604804]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q33.2, which is the long (q) arm of chromosome 9 at position 33.2Molecular Location: base pairs 120,388,869 to 120,580,170 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        C48Cep215MCPH3     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CDK5RAP2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CDK5 REGULATORY SUBUNIT-ASSOCIATED PROTEIN 2OMIM: MICROCEPHALY 3, PRIMARY, AUTOSOMAL RECESSIVE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CDK5RAP2  OMIM: CDK5 REGULATORY SUBUNIT-ASSOCIATED PROTEIN 2OMIM: MICROCEPHALY 3, PRIMARY, AUTOSOMAL RECESSIVE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "MEGF9\t\n",
            "CNTRL\t   From NCBI Gene:This gene encodes a centrosomal protein required for the centrosome to function as a microtubule organizing center. The gene product is also associated with centrosome maturation. One version of stem cell myeloproliferative disorder is the result of a reciprocal translocation between chromosomes 8 and 9, with the breakpoint associated with fibroblast growth factor receptor 1 and centrosomal protein 1. [provided by RefSeq, Jul 2008]From UniProt:Involved in cell cycle progression and cytokinesis. During the late steps of cytokinesis, anchors exocyst and SNARE complexes at the midbody, thereby allowing secretory vesicle-mediated abscission.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From UniProt:A chromosomal aberration involving CEP110 may be a cause of stem cell myeloproliferative disorder (MPD). Translocation t(8;9)(p12;q33) with FGFR1. MPD is characterized by myeloid hyperplasia, eosinophilia and T-cell or B-cell lymphoblastic lymphoma. In general it progresses to acute myeloid leukemia. The fusion protein CEP110-FGFR1 is found in the cytoplasm, exhibits constitutive kinase activity and may be responsible for the transforming activity.     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q33.2, which is the long (q) arm of chromosome 9 at position 33.2Molecular Location: base pairs 121,074,773 to 121,177,610 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA165P4.1CEP1CEP110FAN     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CNTRL      Catalog of Genes and Diseases from OMIM (1 link)     OMIM: CENTROSOMAL PROTEIN 1      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CNTRL  OMIM: CENTROSOMAL PROTEIN 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "OR1J2\t\n",
            "OR1N2\t\n",
            "OR1B1\t\n",
            "OR1L6\t\n",
            "LURAP1L\t\n",
            "ST6GALNAC6\t\n",
            "USP20\t\n",
            "HMCN2\t\n",
            "CEL\t   From NCBI Gene:The protein encoded by this gene is a glycoprotein secreted from the pancreas into the digestive tract and from the lactating mammary gland into human milk. The physiological role of this protein is in cholesterol and lipid-soluble vitamin ester hydrolysis and absorption. This encoded protein promotes large chylomicron production in the intestine. Also its presence in plasma suggests its interactions with cholesterol and oxidized lipoproteins to modulate the progression of atherosclerosis. In pancreatic tumoral cells, this encoded protein is thought to be sequestrated within the Golgi compartment and is probably not secreted. This gene contains a variable number of tandem repeat (VNTR) polymorphism in the coding region that may influence the function of the encoded protein. [provided by RefSeq, Jul 2008]     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Maturity-onset diabetes of the young, type 8     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q34.13, which is the long (q) arm of chromosome 9 at position 34.13Molecular Location: base pairs 133,061,978 to 133,071,863 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BALBSDLBSSLCEaseCELLFAPFAPPLIPAMODY8     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CEL      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CARBOXYL-ESTER LIPASEOMIM: MATURITY-ONSET DIABETES OF THE YOUNG, TYPE 8, WITH EXOCRINE DYSFUNCTION      Research Resources (5 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene      Tests of CEL  OMIM: CARBOXYL-ESTER LIPASEOMIM: MATURITY-ONSET DIABETES OF THE YOUNG, TYPE 8, WITH EXOCRINE DYSFUNCTION  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI Gene \n",
            "ABO\t   From NCBI Gene:This gene encodes proteins related to the first discovered blood group system, ABO. Which allele is present in an individual determines the blood group. The 'O' blood group is caused by a deletion of guanine-258 near the N-terminus of the protein which results in a frameshift and translation of an almost entirely different protein. Individuals with the A, B, and AB alleles express glycosyltransferase activities that convert the H antigen into the A or B antigen. Other minor alleles have been found for this gene. [provided by RefSeq, Jul 2008]From UniProt:This protein is the basis of the ABO blood group system. The histo-blood group ABO involves three carbohydrate antigens: A, B, and H. A, B, and AB individuals express a glycosyltransferase activity that converts the H antigen to the A antigen (by addition of UDP-GalNAc) or to the B antigen (by addition of UDP-Gal), whereas O individuals lack such activity.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:ABO blood group system     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q34.2, which is the long (q) arm of chromosome 9 at position 34.2Molecular Location: base pairs 133,255,176 to 133,275,214 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        A3GALNTA3GALT1GTBNAGAT     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ABO      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: ABO GLYCOSYLTRANSFERASEOMIM: BLOOD GROUP, ABO SYSTEM      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ABO  OMIM: ABO GLYCOSYLTRANSFERASEOMIM: BLOOD GROUP, ABO SYSTEM  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SURF6\t\n",
            "COL5A1\t   The COL5A1 gene provides instructions for making a component of type V collagen. Collagens are a family of proteins that strengthen and support many tissues in the body, including skin, ligaments, bones, tendons, and muscles.A component of type V collagen called the pro-1(V) chain is produced from the COL5A1 gene. Collagens begin as rope-like procollagen molecules that are each made up of three chains. Two combinations of chains can produce type V collagen: three pro-1(V) chains or two pro-1(V) chains and one pro-2(V) chain (which is produced from the COL5A2 gene).The triple-stranded procollagen molecules are processed by enzymes outside the cell to create mature collagen. The collagen molecules then arrange themselves into long, thin fibrils with another form of collagen, type I. Type V collagen regulates the width (diameter) of those fibrils. Studies suggest that type V collagen also controls the assembly of other types of collagen into fibrils in several tissues.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Ehlers-Danlos syndrome     Mutations in the COL5A1 gene cause a form of Ehlers-Danlos syndrome called the classical type. Ehlers-Danlos syndrome is a group of disorders that affect the connective tissues that support the skin, bones, blood vessels, and many other organs and tissues. This form of the disorder is characterized by skin that is soft, highly stretchy (elastic), and fragile; abnormal scarring; and an unusually large range of joint movement (hypermobility). More than 100 COL5A1 gene mutations have been identified in people with classical Ehlers-Danlos syndrome. The mutations affect one copy of the gene in each cell, reducing the amount of pro-1(V) chains that cells produce. As a result, fibrils containing type V and type I collagens in the skin and other tissues are disorganized and larger than usual. Researchers believe that the abnormal collagen weakens connective tissues throughout the body, which causes the signs and symptoms of classical Ehlers-Danlos syndrome. More About This Health Condition      Carpal tunnel syndrome     Genetics Home Reference provides information about carpal tunnel syndrome. More About This Health Condition      Keratoconus     Genetics Home Reference provides information about keratoconus. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Mutations in the COL5A1 gene cause a form of Ehlers-Danlos syndrome called the classical type. Ehlers-Danlos syndrome is a group of disorders that affect the connective tissues that support the skin, bones, blood vessels, and many other organs and tissues. This form of the disorder is characterized by skin that is soft, highly stretchy (elastic), and fragile; abnormal scarring; and an unusually large range of joint movement (hypermobility). More than 100 COL5A1 gene mutations have been identified in people with classical Ehlers-Danlos syndrome. The mutations affect one copy of the gene in each cell, reducing the amount of pro-1(V) chains that cells produce. As a result, fibrils containing type V and type I collagens in the skin and other tissues are disorganized and larger than usual. Researchers believe that the abnormal collagen weakens connective tissues throughout the body, which causes the signs and symptoms of classical Ehlers-Danlos syndrome. More About This Health Condition  Genetics Home Reference provides information about carpal tunnel syndrome. More About This Health Condition  Genetics Home Reference provides information about keratoconus. More About This Health Condition    Cytogenetic Location: 9q34.3, which is the long (q) arm of chromosome 9 at position 34.3Molecular Location: base pairs 134,641,789 to 134,844,843 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        alpha 1 type V collagen preproproteinCO5A1_HUMANcollagen type V alpha 1collagen, type V, alpha 1     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Cell Biology (fourth edition, 2000): Collagen: The Fibrous Proteins of the Matrix      Clinical Information from GeneReviews (1 link)     Classic Ehlers-Danlos Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL5A1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     COLLAGEN, TYPE V, ALPHA-1      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarEhlers-Danlos Syndrome Variant DatabaseHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Collagen: The Fibrous Proteins of the Matrix  Classic Ehlers-Danlos Syndrome  Tests of COL5A1  PubMed  COLLAGEN, TYPE V, ALPHA-1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarEhlers-Danlos Syndrome Variant DatabaseHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Bowen JM, Sobey GJ, Burrows NP, Colombi M, Lavallee ME, Malfait F, Francomano CA. Ehlers-Danlos syndrome, classical type. Am J Med Genet C Semin Med Genet. 2017 Mar;175(1):27-39. doi: 10.1002/ajmg.c.31548. Epub 2017 Feb 13. Review. Citation on PubMedDe Paepe A, Nuytinck L, Hausser I, Anton-Lamprecht I, Naeyaert JM. Mutations in the COL5A1 gene are causal in the Ehlers-Danlos syndromes I and II. Am J Hum Genet. 1997 Mar;60(3):547-54. Citation on PubMed or Free article on PubMed CentralMalfait F, Francomano C, Byers P, Belmont J, Berglund B, Black J, Bloom L, Bowen JM, Brady AF, Burrows NP, Castori M, Cohen H, Colombi M, Demirdas S, De Backer J, De Paepe A, Fournel-Gigleux S, Frank M, Ghali N, Giunta C, Grahame R, Hakim A, Jeunemaitre X, Johnson D, Juul-Kristensen B, Kapferer-Seebacher I, Kazkaz H, Kosho T, Lavallee ME, Levy H, Mendoza-Londono R, Pepin M, Pope FM, Reinstein E, Robert L, Rohrbach M, Sanders L, Sobey GJ, Van Damme T, Vandersteen A, van Mourik C, Voermans N, Wheeldon N, Zschocke J, Tinkle B. The 2017 international classification of the Ehlers-Danlos syndromes. Am J Med Genet C Semin Med Genet. 2017 Mar;175(1):8-26. doi: 10.1002/ajmg.c.31552. Citation on PubMedMalfait F, Wenstrup RJ, De Paepe A. Clinical and genetic aspects of Ehlers-Danlos syndrome, classic type. Genet Med. 2010 Oct;12(10):597-605. doi: 10.1097/GIM.0b013e3181eed412. Review. Citation on PubMedRitelli M, Dordoni C, Venturini M, Chiarelli N, Quinzani S, Traversa M, Zoppi N, Vascellaro A, Wischmeijer A, Manfredini E, Garavelli L, Calzavara-Pinton P, Colombi M. Clinical and molecular characterization of 40 patients with classic Ehlers-Danlos syndrome: identification of 18 COL5A1 and 2 COL5A2 novel mutations. Orphanet J Rare Dis. 2013 Apr 12;8:58. doi: 10.1186/1750-1172-8-58. Citation on PubMed or Free article on PubMed CentralSchwarze U, Atkinson M, Hoffman GG, Greenspan DS, Byers PH. Null alleles of the COL5A1 gene of type V collagen are a cause of the classical forms of Ehlers-Danlos syndrome (types I and II). Am J Hum Genet. 2000 Jun;66(6):1757-65. Epub 2000 May 4. Citation on PubMed or Free article on PubMed CentralSymoens S, Malfait F, Renard M, Andr J, Hausser I, Loeys B, Coucke P, De Paepe A. COL5A1 signal peptide mutations interfere with protein secretion and cause classic Ehlers-Danlos syndrome. Hum Mutat. 2009 Feb;30(2):E395-403. doi: 10.1002/humu.20887. Citation on PubMedSymoens S, Syx D, Malfait F, Callewaert B, De Backer J, Vanakker O, Coucke P, De Paepe A. Comprehensive molecular analysis demonstrates type V collagen mutations in over 90% of patients with classic EDS and allows to refine diagnostic criteria. Hum Mutat. 2012 Oct;33(10):1485-93. doi: 10.1002/humu.22137. Epub 2012 Jul 5. Citation on PubMedWenstrup RJ, Florer JB, Willing MC, Giunta C, Steinmann B, Young F, Susic M, Cole WG. COL5A1 haploinsufficiency is a common molecular mechanism underlying the classical form of EDS. Am J Hum Genet. 2000 Jun;66(6):1766-76. Epub 2000 Apr 24. Citation on PubMed or Free article on PubMed Central \n",
            "OBP2A\t\n",
            "SNAPC4\t\n",
            "SEC16A\t\n",
            "AGPAT2\t   The AGPAT2 gene provides instructions for making an enzyme that is found in many of the body's cells and tissues. It plays a critical role in the growth and development of adipocytes, which are cells that store fats for energy. Adipocytes are the major component of the body's fatty (adipose) tissue.The AGPAT2 enzyme is part of a chemical pathway in many cells that produces two important types of fats (lipids): glycerophospholipids and triacylglycerols.  Glycerophospholipids are the major component of cell membranes and are involved in chemical signaling within cells. Triacylglycerols (also known as triglycerides) are fat molecules that are stored in adipocytes for later conversion to energy.The AGPAT2 enzyme is responsible for a particular chemical reaction in the production of these two types of lipids.  Specifically, the enzyme helps convert a molecule called lysophosphatidic acid (LPA) to another molecule, phosphatidic acid (PA).  Additional reactions convert phosphatidic acid to glycerophospholipids and triacylglycerols.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Congenital generalized lipodystrophy     At least 26 mutations in the AGPAT2 gene have been identified in people with congenital generalized lipodystrophy (also called Berardinelli-Seip congenital lipodystrophy) type 1. This rare condition is characterized by an almost total absence of adipose tissue and a very muscular appearance. A shortage of adipose tissue leads to multiple health problems, including high levels of triglycerides circulating in the bloodstream (hypertriglyceridemia) and diabetes mellitus.The AGPAT2 gene mutations that cause congenital generalized lipodystrophy type 1 greatly reduce or eliminate the activity of the AGPAT2 enzyme. Studies suggest that a loss of this enzyme's activity reduces the production and storage of triacylglycerols in adipocytes, which prevents these cells from storing fats. A lack of enzyme activity may also reduce the levels of glycerophospholipids in adipocytes, which changes the structure of the cell membrane and disrupts normal signaling within these cells. All of these abnormalities prevent the body from storing fats normally in adipose tissue. The resulting lack of body fat underlies the varied signs and symptoms of congenital generalized lipodystrophy type 1. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 26 mutations in the AGPAT2 gene have been identified in people with congenital generalized lipodystrophy (also called Berardinelli-Seip congenital lipodystrophy) type 1. This rare condition is characterized by an almost total absence of adipose tissue and a very muscular appearance. A shortage of adipose tissue leads to multiple health problems, including high levels of triglycerides circulating in the bloodstream (hypertriglyceridemia) and diabetes mellitus.The AGPAT2 gene mutations that cause congenital generalized lipodystrophy type 1 greatly reduce or eliminate the activity of the AGPAT2 enzyme. Studies suggest that a loss of this enzyme's activity reduces the production and storage of triacylglycerols in adipocytes, which prevents these cells from storing fats. A lack of enzyme activity may also reduce the levels of glycerophospholipids in adipocytes, which changes the structure of the cell membrane and disrupts normal signaling within these cells. All of these abnormalities prevent the body from storing fats normally in adipose tissue. The resulting lack of body fat underlies the varied signs and symptoms of congenital generalized lipodystrophy type 1. More About This Health Condition    Cytogenetic Location: 9q34.3, which is the long (q) arm of chromosome 9 at position 34.3Molecular Location: base pairs 136,673,143 to 136,687,459 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        1-acyl-sn-glycerol-3-phosphate acyltransferase beta1-acylglycerol-3-phosphate O-acyltransferase 2 (lysophosphatidic acid acyltransferase, beta)1-AGP acyltransferase 21-AGPAT2BSCL1LPAABLPAAT-betalysophosphatidic acid acyltransferase-betaPLCB_HUMAN     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Biochemistry (fifth edition, 2002): Phosphatidate Is a Common Intermediate in the Synthesis of Phospholipids and Triacylglycerols      Clinical Information from GeneReviews (1 link)     Berardinelli-Seip Congenital Lipodystrophy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AGPAT2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     1-ACYLGLYCEROL-3-PHOSPHATE O-ACYLTRANSFERASE 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Biochemistry (fifth edition, 2002): Phosphatidate Is a Common Intermediate in the Synthesis of Phospholipids and Triacylglycerols  Berardinelli-Seip Congenital Lipodystrophy  Tests of AGPAT2  PubMed  1-ACYLGLYCEROL-3-PHOSPHATE O-ACYLTRANSFERASE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Agarwal AK, Arioglu E, De Almeida S, Akkoc N, Taylor SI, Bowcock AM, Barnes RI, Garg A. AGPAT2 is mutated in congenital generalized lipodystrophy linked to chromosome 9q34. Nat Genet. 2002 May;31(1):21-3. Epub 2002 Apr 22. Citation on PubMedAgarwal AK. Lysophospholipid acyltransferases: 1-acylglycerol-3-phosphate O-acyltransferases. From discovery to disease. Curr Opin Lipidol. 2012 Aug;23(4):290-302. doi: 10.1097/MOL.0b013e328354fcf4. Review. Citation on PubMedGale SE, Frolov A, Han X, Bickel PE, Cao L, Bowcock A, Schaffer JE, Ory DS. A regulatory role for 1-acylglycerol-3-phosphate-O-acyltransferase 2 in adipocyte differentiation. J Biol Chem. 2006 Apr 21;281(16):11082-9. Epub 2006 Feb 22. Citation on PubMedMagr J, Delpine M, Van Maldergem L, Robert JJ, Maassen JA, Meier M, Panz VR, Kim CA, Tubiana-Rufi N, Czernichow P, Seemanova E, Buchanan CR, Lacombe D, Vigouroux C, Lascols O, Kahn CR, Capeau J, Lathrop M. Prevalence of mutations in AGPAT2 among human lipodystrophies. Diabetes. 2003 Jun;52(6):1573-8. Citation on PubMedMiranda DM, Wajchenberg BL, Calsolari MR, Aguiar MJ, Silva JM, Ribeiro MG, Fonseca C, Amaral D, Boson WL, Resende BA, De Marco L. Novel mutations of the BSCL2 and AGPAT2 genes in 10 families with Berardinelli-Seip congenital generalized lipodystrophy syndrome. Clin Endocrinol (Oxf). 2009 Oct;71(4):512-7. doi: 10.1111/j.1365-2265.2009.03532.x. Epub 2009 Feb 18. Citation on PubMedPatni N, Garg A. Congenital generalized lipodystrophies--new insights into metabolic dysfunction. Nat Rev Endocrinol. 2015 Sep;11(9):522-34. doi: 10.1038/nrendo.2015.123. Epub 2015 Aug 4. Review. Citation on PubMed \n",
            "RABL6\t\n",
            "MAMDC4\t\n",
            "FBXW5\t\n",
            "NDOR1\t\n",
            "FAM166A\t\n",
            "CACNA1B\t   From NCBI Gene:The protein encoded by this gene is the pore-forming subunit of an N-type voltage-dependent calcium channel, which controls neurotransmitter release from neurons. The encoded protein forms a complex with alpha-2, beta, and delta subunits to form the high-voltage activated channel. This channel is sensitive to omega-conotoxin-GVIA and omega-agatoxin-IIIA but insensitive to dihydropyridines. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Aug 2011]From UniProt:Voltage-sensitive calcium channels (VSCC) mediate the entry of calcium ions into excitable cells and are also involved in a variety of calcium-dependent processes, including muscle contraction, hormone or neurotransmitter release, gene expression, cell motility, cell division and cell death. The isoform alpha-1B gives rise to N-type calcium currents. N-type calcium channels belong to the 'high-voltage activated' (HVA) group and are specifically blocked by omega-conotoxin-GVIA (AC P01522) (AC P01522) (By similarity). They are however insensitive to dihydropyridines (DHP). Calcium channels containing alpha-1B subunit may play a role in directed migration of immature neurons.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Dystonia 23From UniProt:Dystonia 23 (DYT23): A form of dystonia, a disorder defined by the presence of sustained involuntary muscle contraction, often leading to abnormal postures. DYT23 is an autosomal dominant dystonia affecting the face, neck, limbs. Some DYT23 patients manifest generalized myoclonus in addition to progressive action-induced multifocal dystonia. [MIM:614860]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q34.3, which is the long (q) arm of chromosome 9 at position 34.3Molecular Location: base pairs 137,877,789 to 138,124,624 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BIIICACNL1A5CACNNCav2.2DYT23     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CACNA1B      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: CALCIUM CHANNEL, VOLTAGE-DEPENDENT, N TYPE, ALPHA-1B SUBUNITOMIM: DYSTONIA 23      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CACNA1B  OMIM: CALCIUM CHANNEL, VOLTAGE-DEPENDENT, N TYPE, ALPHA-1B SUBUNITOMIM: DYSTONIA 23  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "SMARCA2\t   The SMARCA2 gene provides instructions for making one piece (subunit) of a group of similar protein complexes known as SWI/SNF complexes. These complexes regulate gene activity (expression) by a process known as chromatin remodeling. Chromatin is the network of DNA and proteins that packages DNA into chromosomes. The structure of chromatin can be changed (remodeled) to alter how tightly DNA is packaged. Chromatin remodeling is one way gene expression is regulated during development; when DNA is tightly packed, gene expression is lower than when DNA is loosely packed. SWI/SNF complexes help with chromatin remodeling by moving parts of chromatin called nucleosomes, which makes DNA more accessible for gene expression. To provide energy for chromatin remodeling, the SMARCA2 protein uses a molecule called ATP.SWI/SNF complexes regulate genes that are involved in many processes, including repairing damaged DNA; copying (replicating) DNA; and controlling the growth, division, and maturation (differentiation) of cells. The SMARCA2 protein and other SWI/SNF subunits are thought to act as tumor suppressors, which keep cells from growing and dividing too rapidly or in an uncontrolled way.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Nicolaides-Baraitser syndrome     At least 50 mutations in the SMARCA2 gene have been found to cause Nicolaides-Baraitser syndrome. This condition is characterized by multiple abnormalities, primarily sparse scalp hair, small head size (microcephaly), distinctive facial features, short stature, abnormal fingers, recurrent seizures (epilepsy), and moderate to severe intellectual disability with impaired language development. Almost all SMARCA2 gene mutations that cause Nicolaides-Baraitser syndrome change single protein building blocks (amino acids) in the protein. These mutations are located within an area of the protein that attaches to ATP and is responsible for providing energy to the SWI/SNF complexes. These altered proteins are able to form SWI/SNF complexes, but these complexes are nonfunctional and cannot participate in chromatin remodeling. Disturbance of this regulatory process alters the activity of many genes, which likely explains the diverse signs and symptoms of Nicolaides-Baraitser syndrome. More About This Health Condition      Cancers     Common DNA variants (polymorphisms) that affect the SMARCA2 gene have been found in certain types of cancer, particularly lung cancer in tobacco smokers and head and neck cancer. The changes that are associated with these cancers are described as germline, meaning that they are present from birth and found in all of the body's cells. (While the mutations that cause Nicolaides-Baraitser syndrome, which is described above, are also germline, there is no increased risk for cancer in individuals with that condition.) Two specific SMARCA2 gene polymorphisms have been found to be associated with the development of these cancers. These changes occur in an area near the SMARCA2 gene called the promoter region, which controls the production of the SMARCA2 protein. One change adds seven building blocks of DNA (nucleotides) at position -741 (insTATTTTT); the other adds six nucleotides at position -1321 (insTTTTAA) of the promoter. The incidence of lung cancer and head and neck cancer is increased only when both of these polymorphisms are present in both copies of the SMARCA2 gene in each cell.These polymorphisms impair the function of the promoter and reduce the expression of the SMARCA2 gene. The reduced gene activity likely decreases or alters protein production, which would lead to changes in SWI/SNF complexes. These changes may impair normal cell differentiation, which leads to the overgrowth of certain cell types, causing cancer. Alternatively, abnormal SWI/SNF complexes may disrupt the regulation of genes that help control the growth and division of cells, which leads to cancer. It is unclear why these changes in the promoter region of the SMARCA2 gene seem to be associated only with head and neck cancer and lung cancer in smokers.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 50 mutations in the SMARCA2 gene have been found to cause Nicolaides-Baraitser syndrome. This condition is characterized by multiple abnormalities, primarily sparse scalp hair, small head size (microcephaly), distinctive facial features, short stature, abnormal fingers, recurrent seizures (epilepsy), and moderate to severe intellectual disability with impaired language development. Almost all SMARCA2 gene mutations that cause Nicolaides-Baraitser syndrome change single protein building blocks (amino acids) in the protein. These mutations are located within an area of the protein that attaches to ATP and is responsible for providing energy to the SWI/SNF complexes. These altered proteins are able to form SWI/SNF complexes, but these complexes are nonfunctional and cannot participate in chromatin remodeling. Disturbance of this regulatory process alters the activity of many genes, which likely explains the diverse signs and symptoms of Nicolaides-Baraitser syndrome. More About This Health Condition  Common DNA variants (polymorphisms) that affect the SMARCA2 gene have been found in certain types of cancer, particularly lung cancer in tobacco smokers and head and neck cancer. The changes that are associated with these cancers are described as germline, meaning that they are present from birth and found in all of the body's cells. (While the mutations that cause Nicolaides-Baraitser syndrome, which is described above, are also germline, there is no increased risk for cancer in individuals with that condition.) Two specific SMARCA2 gene polymorphisms have been found to be associated with the development of these cancers. These changes occur in an area near the SMARCA2 gene called the promoter region, which controls the production of the SMARCA2 protein. One change adds seven building blocks of DNA (nucleotides) at position -741 (insTATTTTT); the other adds six nucleotides at position -1321 (insTTTTAA) of the promoter. The incidence of lung cancer and head and neck cancer is increased only when both of these polymorphisms are present in both copies of the SMARCA2 gene in each cell.These polymorphisms impair the function of the promoter and reduce the expression of the SMARCA2 gene. The reduced gene activity likely decreases or alters protein production, which would lead to changes in SWI/SNF complexes. These changes may impair normal cell differentiation, which leads to the overgrowth of certain cell types, causing cancer. Alternatively, abnormal SWI/SNF complexes may disrupt the regulation of genes that help control the growth and division of cells, which leads to cancer. It is unclear why these changes in the promoter region of the SMARCA2 gene seem to be associated only with head and neck cancer and lung cancer in smokers.    Cytogenetic Location: 9p24.3, which is the short (p) arm of chromosome 9 at position 24.3Molecular Location: base pairs 2,015,219 to 2,193,624 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATP-dependent helicase SMARCA2BAF190BAF190BBRG1-associated factor 190BBRMglobal transcription activator homologous sequencehBRMhSNF2aprotein brahma homologSNF2SNF2-alphaSNF2/SWI2-like protein 2SNF2L2SNF2LASth1psucrose nonfermenting 2-like protein 2SWI/SNF-related matrix-associated actin-dependent regulator of chromatin a2SWI2     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Molecular Cell Biology (fourth edition, 2000): Chromatin-Remodeling Factors Participate in Activation at Some Promoters      Clinical Information from GeneReviews (1 link)     Nicolaides-Baraitser Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of SMARCA2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     SWI/SNF-RELATED, MATRIX-ASSOCIATED, ACTIN-DEPENDENT REGULATOR OF CHROMATIN, SUBFAMILY A, MEMBER 2      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportIUPHAR DatabaseMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): Chromatin-Remodeling Factors Participate in Activation at Some Promoters  Nicolaides-Baraitser Syndrome  Tests of SMARCA2  PubMed  SWI/SNF-RELATED, MATRIX-ASSOCIATED, ACTIN-DEPENDENT REGULATOR OF CHROMATIN, SUBFAMILY A, MEMBER 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportIUPHAR DatabaseMonarch InitiativeNCBI GeneUniProt  Liu G, Gramling S, Munoz D, Cheng D, Azad AK, Mirshams M, Chen Z, Xu W, Roberts H, Shepherd FA, Tsao MS, Reisman D. Two novel BRM insertion promoter sequence variants are associated with loss of BRM expression and lung cancer risk. Oncogene. 2011 Jul 21;30(29):3295-304. doi: 10.1038/onc.2011.81. Epub 2011 Apr 11. Citation on PubMed or Free article on PubMed CentralOMIM: SWI/SNF-RELATED, MATRIX-ASSOCIATED, ACTIN-DEPENDENT REGULATOR OF CHROMATIN, SUBFAMILY A, MEMBER 2Sousa SB, Hennekam RC; Nicolaides-Baraitser Syndrome International Consortium. Phenotype and genotype in Nicolaides-Baraitser syndrome. Am J Med Genet C Semin Med Genet. 2014 Sep;166C(3):302-14. doi: 10.1002/ajmg.c.31409. Epub 2014 Aug 28. Review. Citation on PubMedVan Houdt JK, Nowakowska BA, Sousa SB, van Schaik BD, Seuntjens E, Avonce N, Sifrim A, Abdul-Rahman OA, van den Boogaard MJ, Bottani A, Castori M, Cormier-Daire V, Deardorff MA, Filges I, Fryer A, Fryns JP, Gana S, Garavelli L, Gillessen-Kaesbach G, Hall BD, Horn D, Huylebroeck D, Klapecki J, Krajewska-Walasek M, Kuechler A, Lines MA, Maas S, Macdermot KD, McKee S, Magee A, de Man SA, Moreau Y, Morice-Picard F, Obersztyn E, Pilch J, Rosser E, Shannon N, Stolte-Dijkstra I, Van Dijck P, Vilain C, Vogels A, Wakeling E, Wieczorek D, Wilson L, Zuffardi O, van Kampen AH, Devriendt K, Hennekam R, Vermeesch JR. Heterozygous missense mutations in SMARCA2 cause Nicolaides-Baraitser syndrome. Nat Genet. 2012 Feb 26;44(4):445-9, S1. doi: 10.1038/ng.1105. Citation on PubMedWong KM, Qiu X, Cheng D, Azad AK, Habbous S, Palepu P, Mirshams M, Patel D, Chen Z, Roberts H, Knox J, Marquez S, Wong R, Darling G, Waldron J, Goldstein D, Leighl N, Shepherd FA, Tsao M, Der S, Reisman D, Liu G. Two BRM promoter insertion polymorphisms increase the risk of early-stage upper aerodigestive tract cancers. Cancer Med. 2014 Apr;3(2):426-33. doi: 10.1002/cam4.201. Epub 2014 Feb 12. Citation on PubMed or Free article on PubMed Central \n",
            "MLLT3\t\n",
            "DOCK8\t   The DOCK8 gene provides instructions for making a member of the DOCK family of proteins. The proteins in this family act as guanine nucleotide exchange factors (GEFs). GEFs turn on (activate) proteins called GTPases, which play an important role in chemical signaling within cells. Signaling stimulated by DOCK family proteins are typically involved in the arrangement of the structural framework inside cells (the cytoskeleton). By controlling the shape of the cytoskeleton, DOCK family proteins play a role in cell structure and movement (migration).The DOCK8 protein is found most abundantly in cells of the immune system. DOCK8 plays a critical role in the survival and function of several types of immune system cells, including T cells, NK cells, and B cells. T cells and NK cells recognize and attack foreign invaders, such as viruses, to prevent infection. B cells produce proteins called antibodies, which attach to foreign particles and germs and mark them for destruction.Through its function as a GEF, the DOCK8 protein helps maintain the structure and integrity of T cells and NK cells. It also aids in the migration of these immune system cells to sites of infection, particularly the skin. DOCK8 is also involved in chemical signaling pathways that stimulate B cells to mature and produce antibodies. DOCK8 is also involved in the normal development and survival of other types of immune system cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Autosomal recessive hyper-IgE syndrome     At least 110 mutations in the DOCK8 gene have been found to cause autosomal recessive hyper-IgE syndrome (AR-HIES), an immune system disorder that causes recurrent severe infections of the skin and respiratory tract. Most of the mutations involved in this disorder delete regions of DNA from the DOCK8 gene. These deletions and other DOCK8 gene mutations lead to an abnormally short protein, if any protein is produced. As a result, affected individuals have little or no functional DOCK8 protein.A shortage of DOCK8 protein impairs normal immune cell development and function. It is thought that T cells lacking DOCK8 cannot maintain their shape as they move through dense spaces, such as those found within the skin. The abnormal cells die, resulting in reduced numbers of these cells. A shortage of T cells impairs the immune response to foreign invaders, accounting for the severe skin infections common in AR-HIES. A lack of DOCK8 also impairs B cell maturation and the production of antibodies. Impairment of this type of immune response leads to recurrent respiratory tract infections in people with this disorder. More About This Health Condition      Other disorders     Mutations affecting the DOCK8 gene have been found in two individuals with a rare form of intellectual disability described in the medical literature as autosomal dominant mental retardation 2. These individuals have developmental delays, including delayed or absent speech. The mutations involved in this condition result in production of an abnormally short DOCK8 protein. Unlike in AR-HIES (described above), the mutations associated with autosomal dominant mental retardation 2 affect only one copy of the DOCK8 gene. Researchers are unsure how these genetic changes lead to intellectual disability.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 110 mutations in the DOCK8 gene have been found to cause autosomal recessive hyper-IgE syndrome (AR-HIES), an immune system disorder that causes recurrent severe infections of the skin and respiratory tract. Most of the mutations involved in this disorder delete regions of DNA from the DOCK8 gene. These deletions and other DOCK8 gene mutations lead to an abnormally short protein, if any protein is produced. As a result, affected individuals have little or no functional DOCK8 protein.A shortage of DOCK8 protein impairs normal immune cell development and function. It is thought that T cells lacking DOCK8 cannot maintain their shape as they move through dense spaces, such as those found within the skin. The abnormal cells die, resulting in reduced numbers of these cells. A shortage of T cells impairs the immune response to foreign invaders, accounting for the severe skin infections common in AR-HIES. A lack of DOCK8 also impairs B cell maturation and the production of antibodies. Impairment of this type of immune response leads to recurrent respiratory tract infections in people with this disorder. More About This Health Condition  Mutations affecting the DOCK8 gene have been found in two individuals with a rare form of intellectual disability described in the medical literature as autosomal dominant mental retardation 2. These individuals have developmental delays, including delayed or absent speech. The mutations involved in this condition result in production of an abnormally short DOCK8 protein. Unlike in AR-HIES (described above), the mutations associated with autosomal dominant mental retardation 2 affect only one copy of the DOCK8 gene. Researchers are unsure how these genetic changes lead to intellectual disability.    Cytogenetic Location: 9p24.3, which is the short (p) arm of chromosome 9 at position 24.3Molecular Location: base pairs 211,296 to 465,260 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        1200017A24Rikdedicator of cytokinesis protein 8 isoform 1dedicator of cytokinesis protein 8 isoform 2dedicator of cytokinesis protein 8 isoform 3epididymis luminal protein 205FLJ00026FLJ00152FLJ00346HEL-205MRD2ZIR8     Related Information   How are genetic conditions and genes named?          Educational Resources (3 links)     Molecular Biology of the Cell (fourth edition, 2002): B Cells and AntibodiesMolecular Biology of the Cell (fourth edition, 2002): Extracellular Signals Can Induce Major Cytoskeletal RearrangementsMolecular Biology of the Cell (fourth edition, 2002): T Cells and MHC Proteins      Tests Listed in the Genetic Testing Registry (1 link)     Tests of DOCK8      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     DEDICATOR OF CYTOKINESIS 8      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2002): B Cells and AntibodiesMolecular Biology of the Cell (fourth edition, 2002): Extracellular Signals Can Induce Major Cytoskeletal RearrangementsMolecular Biology of the Cell (fourth edition, 2002): T Cells and MHC Proteins  Tests of DOCK8  PubMed  DEDICATOR OF CYTOKINESIS 8  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Crawford G, Enders A, Gileadi U, Stankovic S, Zhang Q, Lambe T, Crockford TL, Lockstone HE, Freeman A, Arkwright PD, Smart JM, Ma CS, Tangye SG, Goodnow CC, Cerundolo V, Godfrey DI, Su HC, Randall KL, Cornall RJ. DOCK8 is critical for the survival and function of NKT cells. Blood. 2013 Sep 19;122(12):2052-61. doi: 10.1182/blood-2013-02-482331. Epub 2013 Aug 8. Citation on PubMed or Free article on PubMed CentralOMIM: DEDICATOR OF CYTOKINESIS 8Engelhardt KR, McGhee S, Winkler S, Sassi A, Woellner C, Lopez-Herrera G, Chen A, Kim HS, Lloret MG, Schulze I, Ehl S, Thiel J, Pfeifer D, Veelken H, Niehues T, Siepermann K, Weinspach S, Reisli I, Keles S, Genel F, Kutukculer N, Camciolu Y, Somer A, Karakoc-Aydiner E, Barlan I, Gennery A, Metin A, Degerliyurt A, Pietrogrande MC, Yeganeh M, Baz Z, Al-Tamemi S, Klein C, Puck JM, Holland SM, McCabe ER, Grimbacher B, Chatila TA. Large deletions and point mutations involving the dedicator of cytokinesis 8 (DOCK8) in the autosomal-recessive form of hyper-IgE syndrome. J Allergy Clin Immunol. 2009 Dec;124(6):1289-302.e4. doi: 10.1016/j.jaci.2009.10.038. Erratum in: J Allergy Clin Immunol. 2010 Mar;125(3):743. Kutuculer, Necil [corrected to Kutukculer, Necil]. Citation on PubMed or Free article on PubMed CentralGriggs BL, Ladd S, Saul RA, DuPont BR, Srivastava AK. Dedicator of cytokinesis 8 is disrupted in two patients with mental retardation and developmental disabilities. Genomics. 2008 Feb;91(2):195-202. Epub 2007 Dec 3. Citation on PubMed or Free article on PubMed CentralHarada Y, Tanaka Y, Terasawa M, Pieczyk M, Habiro K, Katakai T, Hanawa-Suetsugu K, Kukimoto-Niino M, Nishizaki T, Shirouzu M, Duan X, Uruno T, Nishikimi A, Sanematsu F, Yokoyama S, Stein JV, Kinashi T, Fukui Y. DOCK8 is a Cdc42 activator critical for interstitial dendritic cell migration during immune responses. Blood. 2012 May 10;119(19):4451-61. doi: 10.1182/blood-2012-01-407098. Epub 2012 Mar 28. Citation on PubMed or Free article on PubMed CentralJabara HH, McDonald DR, Janssen E, Massaad MJ, Ramesh N, Borzutzky A, Rauter I, Benson H, Schneider L, Baxi S, Recher M, Notarangelo LD, Wakim R, Dbaibo G, Dasouki M, Al-Herz W, Barlan I, Baris S, Kutukculer N, Ochs HD, Plebani A, Kanariou M, Lefranc G, Reisli I, Fitzgerald KA, Golenbock D, Manis J, Keles S, Ceja R, Chatila TA, Geha RS. DOCK8 functions as an adaptor that links TLR-MyD88 signaling to B cell activation. Nat Immunol. 2012 May 13;13(6):612-20. doi: 10.1038/ni.2305. Citation on PubMed or Free article on PubMed CentralLambe T, Crawford G, Johnson AL, Crockford TL, Bouriez-Jones T, Smyth AM, Pham TH, Zhang Q, Freeman AF, Cyster JG, Su HC, Cornall RJ. DOCK8 is essential for T-cell survival and the maintenance of CD8+ T-cell memory. Eur J Immunol. 2011 Dec;41(12):3423-35. doi: 10.1002/eji.201141759. Epub 2011 Nov 10. Citation on PubMed or Free article on PubMed CentralRandall KL, Lambe T, Johnson AL, Treanor B, Kucharska E, Domaschenz H, Whittle B, Tze LE, Enders A, Crockford TL, Bouriez-Jones T, Alston D, Cyster JG, Lenardo MJ, Mackay F, Deenick EK, Tangye SG, Chan TD, Camidge T, Brink R, Vinuesa CG, Batista FD, Cornall RJ, Goodnow CC. Dock8 mutations cripple B cell immunological synapses, germinal centers and long-lived antibody production. Nat Immunol. 2009 Dec;10(12):1283-91. doi: 10.1038/ni.1820. Epub 2009 Nov 8. Erratum in: Nat Immunol. 2010 Jul;11(7):644. Johnson, Andy [corrected to Johnson, Andy L]. Citation on PubMed or Free article on PubMed CentralZhang Q, Dove CG, Hor JL, Murdock HM, Strauss-Albee DM, Garcia JA, Mandl JN, Grodick RA, Jing H, Chandler-Brown DB, Lenardo TE, Crawford G, Matthews HF, Freeman AF, Cornall RJ, Germain RN, Mueller SN, Su HC. DOCK8 regulates lymphocyte shape integrity for skin antiviral immunity. J Exp Med. 2014 Dec 15;211(13):2549-66. doi: 10.1084/jem.20141307. Epub 2014 Nov 24. Citation on PubMed or Free article on PubMed Central \n",
            "DDX58\t   From NCBI Gene:DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases which are implicated in a number of cellular processes involving RNA binding and alteration of RNA secondary structure. This gene encodes a protein containing RNA helicase-DEAD box protein motifs and a caspase recruitment domain (CARD). It is involved in viral double-stranded (ds) RNA recognition and the regulation of immune response. [provided by RefSeq, Jul 2008]From UniProt:Innate immune receptor which acts as a cytoplasmic sensor of viral nucleic acids and plays a major role in sensing viral infection and in the activation of a cascade of antiviral responses including the induction of type I interferons and proinflammatory cytokines. Its ligands include: 5'-triphosphorylated ssRNA and dsRNA and short dsRNA (<1 kb in length). In addition to the 5'-triphosphate moiety, blunt-end base pairing at the 5'-end of the RNA is very essential. Overhangs at the non-triphosphorylated end of the dsRNA RNA have no major impact on its activity. A 3'overhang at the 5'triphosphate end decreases and any 5'overhang at the 5' triphosphate end abolishes its activity. Upon ligand binding it associates with mitochondria antiviral signaling protein (MAVS/IPS1) which activates the IKK-related kinases: TBK1 and IKBKE which phosphorylate interferon regulatory factors: IRF3 and IRF7 which in turn activate transcription of antiviral immunological genes, including interferons (IFNs); IFN-alpha and IFN-beta. Detects both positive and negative strand RNA viruses including members of the families Paramyxoviridae: Human respiratory syncytial virus and measles virus (MeV), Rhabdoviridae: vesicular stomatitis virus (VSV), Orthomyxoviridae: influenza A and B virus, Flaviviridae: Japanese encephalitis virus (JEV), hepatitis C virus (HCV), dengue virus (DENV) and west Nile virus (WNV). It also detects rotavirus and reovirus. Also involved in antiviral signaling in response to viruses containing a dsDNA genome such as Epstein-Barr virus (EBV). Detects dsRNA produced from non-self dsDNA by RNA polymerase III, such as Epstein-Barr virus-encoded RNAs (EBERs). May play important roles in granulocyte production and differentiation, bacterial phagocytosis and in the regulation of cell migration.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Singleton-Merten syndrome 2From UniProt:Singleton-Merten syndrome 2 (SGMRT2): A form of Singleton-Merten syndrome, an autosomal dominant disorder characterized by marked aortic calcification, dental anomalies, osteopenia, acro-osteolysis, and to a lesser extent glaucoma, psoriasis, muscle weakness, and joint laxity. Additional clinical manifestations include particular facial characteristics and abnormal joint and muscle ligaments. SGMRT2 is an atypical form characterized by variable expression of glaucoma, aortic calcification, and skeletal abnormalities, without dental anomalies. [MIM:616298]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9p21.1, which is the short (p) arm of chromosome 9 at position 21.1Molecular Location: base pairs 32,455,302 to 32,526,324 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        RIG-IRIG1RIGIRLR-1SGMRT2     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of DDX58      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: DEAD BOX POLYPEPTIDE 58OMIM: SINGLETON-MERTEN SYNDROME 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of DDX58  OMIM: DEAD BOX POLYPEPTIDE 58OMIM: SINGLETON-MERTEN SYNDROME 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TAF1L\t\n",
            "AQP7\t   From NCBI Gene:This gene encodes a member of the aquaporin family of water-selective membrane channels. The encoded protein localizes to the plasma membrane and allows movement of water, glycerol and urea across cell membranes. This gene is highly expressed in the adipose tissue where the encoded protein facilitates efflux of glycerol. In the proximal straight tubules of kidney, the encoded protein is localized to the apical membrane and prevents excretion of glycerol into urine. The encoded protein is present in spermatids, as well as in the testicular and epididymal spermatozoa suggesting an important role in late spermatogenesis. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms. This gene is located adjacent to a related aquaporin gene on chromosome 9. Multiple pseudogenes of this gene have been identified. [provided by RefSeq, Dec 2015]From UniProt:Forms a channel that mediates water and glycerol transport across cell membranes at neutral pH (PubMed:9405233, PubMed:11952783, PubMed:30423801, PubMed:30420639). The channel is also permeable to urea (PubMed:9405233). Plays an important role in body energy homeostasis under conditions that promote lipid catabolism, giving rise to glycerol and free fatty acids. Mediates glycerol export from adipocytes. After release into the blood stream, glycerol is used for gluconeogenesis in the liver to maintain normal blood glucose levels and prevent fasting hypoglycemia. Required for normal glycerol reabsorption in the kidney (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Glycerol release during exercise, defective     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9p13.3, which is the short (p) arm of chromosome 9 at position 13.3Molecular Location: base pairs 33,383,135 to 33,402,682 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AQP7LAQPapGLYCQTL     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of AQP7      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: AQUAPORIN 7OMIM: GLYCEROL QUANTITATIVE TRAIT LOCUS      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of AQP7  OMIM: AQUAPORIN 7OMIM: GLYCEROL QUANTITATIVE TRAIT LOCUS  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "PRSS3\t\n",
            "UBAP2\t\n",
            "FAM205A\t\n",
            "UNC13B\t\n",
            "CCDC107\t\n",
            "TPM2\t   The TPM2 gene provides instructions for making a protein called beta ()-tropomyosin, which is part of the tropomyosin protein family. Tropomyosin proteins regulate the tensing of muscle fibers (muscle contraction) by controlling the binding of two muscle proteins, myosin and actin. In non-muscle cells, tropomyosin proteins play a role in controlling cell shape.-tropomyosin is found primarily in skeletal muscles, which are the muscles used for movement. This protein helps regulate muscle contraction by interacting with other muscle proteins, particularly myosin and actin. These interactions are essential for stabilizing and maintaining structures called sarcomeres within muscle cells. Sarcomeres are the basic units of muscle contraction; they are made of proteins that generate the mechanical force needed for muscles to contract.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Cap myopathy     At least three TPM2 gene mutations have been identified in people with cap myopathy, a disorder that leads to muscle weakness (myopathy) and poor muscle tone (hypotonia). These mutations delete or duplicate genetic material in the TPM2 gene or replace single protein building blocks (amino acids) in the -tropomyosin protein sequence. The specific effects of these TPM2 gene mutations are unclear, but researchers suggest they may interfere with normal actin-myosin binding, impairing muscle contraction and resulting in the muscle weakness that occurs in cap myopathy. More About This Health Condition      Distal arthrogryposis type 1     At least three mutations in the TPM2 gene have been found to cause distal arthrogryposis type 1, a disorder characterized by joint deformities (contractures) in the hands and feet. It is unclear how these mutations lead to contractures in people with distal arthrogryposis type 1, or why the joint problems are typically limited to the hands and feet. However, researchers speculate that contractures may be related to problems with muscle contraction that limit the movement of joints before birth. More About This Health Condition      Congenital fiber-type disproportion     Genetics Home Reference provides information about congenital fiber-type disproportion. More About This Health Condition      Nemaline myopathy     Genetics Home Reference provides information about nemaline myopathy. More About This Health Condition      Sheldon-Hall syndrome     At least six TPM2 gene mutations have been identified in people with Sheldon-Hall syndrome, a muscle and skeletal disorder similar to distal arthrogryposis type 1 (described above) that impairs joint movement in the hands and feet. Mutations in the TPM2 gene may alter the structure of -tropomyosin and disrupt the protein's normal function in controlling muscle contractions, resulting in the contractures and other muscle and skeletal abnormalities associated with this condition. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least three TPM2 gene mutations have been identified in people with cap myopathy, a disorder that leads to muscle weakness (myopathy) and poor muscle tone (hypotonia). These mutations delete or duplicate genetic material in the TPM2 gene or replace single protein building blocks (amino acids) in the -tropomyosin protein sequence. The specific effects of these TPM2 gene mutations are unclear, but researchers suggest they may interfere with normal actin-myosin binding, impairing muscle contraction and resulting in the muscle weakness that occurs in cap myopathy. More About This Health Condition  At least three mutations in the TPM2 gene have been found to cause distal arthrogryposis type 1, a disorder characterized by joint deformities (contractures) in the hands and feet. It is unclear how these mutations lead to contractures in people with distal arthrogryposis type 1, or why the joint problems are typically limited to the hands and feet. However, researchers speculate that contractures may be related to problems with muscle contraction that limit the movement of joints before birth. More About This Health Condition  Genetics Home Reference provides information about congenital fiber-type disproportion. More About This Health Condition  Genetics Home Reference provides information about nemaline myopathy. More About This Health Condition  At least six TPM2 gene mutations have been identified in people with Sheldon-Hall syndrome, a muscle and skeletal disorder similar to distal arthrogryposis type 1 (described above) that impairs joint movement in the hands and feet. Mutations in the TPM2 gene may alter the structure of -tropomyosin and disrupt the protein's normal function in controlling muscle contractions, resulting in the contractures and other muscle and skeletal abnormalities associated with this condition. More About This Health Condition    Cytogenetic Location: 9p13.3, which is the short (p) arm of chromosome 9 at position 13.3Molecular Location: base pairs 35,681,993 to 35,690,056 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        beta-tropomyosinNEM4TMSBTPM2_HUMANtropomyosin-2tropomyosin 2 (beta)tropomyosin beta chaintropomyosin, skeletal muscle beta     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Biology of the Cell (fourth edition, 2000): Role of Tropomyosin and Troponin in Skeletal Muscle ContractionThe Cell: A Molecular Approach (second edition, 2000): Association of Tropomyosin and Troponins with Actin Filaments (Figure)      Clinical Information from GeneReviews (1 link)     Nemaline Myopathy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of TPM2      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     TROPOMYOSIN 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Biology of the Cell (fourth edition, 2000): Role of Tropomyosin and Troponin in Skeletal Muscle ContractionThe Cell: A Molecular Approach (second edition, 2000): Association of Tropomyosin and Troponins with Actin Filaments (Figure)  Nemaline Myopathy  Tests of TPM2  PubMed  TROPOMYOSIN 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Beck AE, McMillin MJ, Gildersleeve HI, Kezele PR, Shively KM, Carey JC, Regnier M, Bamshad MJ. Spectrum of mutations that cause distal arthrogryposis types 1 and 2B. Am J Med Genet A. 2013 Mar;161A(3):550-5. doi: 10.1002/ajmg.a.35809. Epub 2013 Feb 7. Citation on PubMed or Free article on PubMed CentralClarke NF, Domazetovska A, Waddell L, Kornberg A, McLean C, North KN. Cap disease due to mutation of the beta-tropomyosin gene (TPM2). Neuromuscul Disord. 2009 May;19(5):348-51. doi: 10.1016/j.nmd.2009.03.003. Epub 2009 Apr 3. Citation on PubMedDonner K, Ollikainen M, Ridanp M, Christen HJ, Goebel HH, de Visser M, Pelin K, Wallgren-Pettersson C. Mutations in the beta-tropomyosin (TPM2) gene--a rare cause of nemaline myopathy. Neuromuscul Disord. 2002 Feb;12(2):151-8. Citation on PubMedGoebel HH. Cap disease uncapped. Neuromuscul Disord. 2007 Jun;17(6):429-32. Epub 2007 Apr 16. Citation on PubMedLehtokari VL, Ceuterick-de Groote C, de Jonghe P, Marttila M, Laing NG, Pelin K, Wallgren-Pettersson C. Cap disease caused by heterozygous deletion of the beta-tropomyosin gene TPM2. Neuromuscul Disord. 2007 Jun;17(6):433-42. Epub 2007 Apr 16. Citation on PubMedMarttila M, Lemola E, Wallefeld W, Memo M, Donner K, Laing NG, Marston S, Grnholm M, Wallgren-Pettersson C. Abnormal actin binding of aberrant -tropomyosins is a molecular cause of muscle weakness in TPM2-related nemaline and cap myopathy. Biochem J. 2012 Feb 15;442(1):231-9. doi: 10.1042/BJ20111030. Citation on PubMedMonnier N, Lunardi J, Marty I, Mezin P, Labarre-Vila A, Dieterich K, Jouk PS. Absence of beta-tropomyosin is a new cause of Escobar syndrome associated with nemaline myopathy. Neuromuscul Disord. 2009 Feb;19(2):118-23. doi: 10.1016/j.nmd.2008.11.009. Epub 2009 Jan 19. Citation on PubMedOchala J. Thin filament proteins mutations associated with skeletal myopathies: defective regulation of muscle contraction. J Mol Med (Berl). 2008 Nov;86(11):1197-204. doi: 10.1007/s00109-008-0380-9. Epub 2008 Jun 24. Review. Citation on PubMedOhlsson M, Quijano-Roy S, Darin N, Brochier G, Lacne E, Avila-Smirnow D, Fardeau M, Oldfors A, Tajsharghi H. New morphologic and genetic findings in cap disease associated with beta-tropomyosin (TPM2) mutations. Neurology. 2008 Dec 2;71(23):1896-901. doi: 10.1212/01.wnl.0000336654.44814.b8. Citation on PubMedSung SS, Brassington AM, Grannatt K, Rutherford A, Whitby FG, Krakowiak PA, Jorde LB, Carey JC, Bamshad M. Mutations in genes encoding fast-twitch contractile proteins cause distal arthrogryposis syndromes. Am J Hum Genet. 2003 Mar;72(3):681-90. Citation on PubMed or Free article on PubMed CentralTajsharghi H, Kimber E, Holmgren D, Tulinius M, Oldfors A. Distal arthrogryposis and muscle weakness associated with a beta-tropomyosin mutation. Neurology. 2007 Mar 6;68(10):772-5. Citation on PubMedTajsharghi H, Ohlsson M, Lindberg C, Oldfors A. Congenital myopathy with nemaline rods and cap structures caused by a mutation in the beta-tropomyosin gene (TPM2). Arch Neurol. 2007 Sep;64(9):1334-8. Citation on PubMed \n",
            "TMEM8B\t\n",
            "HRCT1\t\n",
            "EXOSC3\t   The EXOSC3 gene provides instructions for making a protein known as exosome component 3. As its name suggests, this protein forms one part (subunit) of a large, multi-protein complex known as the RNA exosome. Within cells, this complex helps to process multiple types of RNA, which are chemical cousins of DNA, by cutting (cleaving) RNA molecules in certain places. The RNA exosome also breaks down (degrades) molecules of RNA when they are no longer needed. Appropriate processing and breakdown of RNA molecules is essential for the normal functioning of all cells.Studies suggest that the activity of exosome component 3 is necessary for the normal development and growth of certain areas of the brain, particularly the cerebellum, which is the part of the brain that coordinates movement. Exosome component 3 also appears to be important for the survival of specialized nerve cells in the spinal cord called motor neurons, which play an essential role in muscle movement.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Pontocerebellar hypoplasia     At least 16 mutations in the EXOSC3 gene have been identified in people with a disorder of brain development called pontocerebellar hypoplasia. The major features of this condition include delayed development overall, an unusually small head size (microcephaly), and intellectual disability. EXOSC3 gene mutations cause about half of all cases of a form of the disorder designated pontocerebellar hypoplasia type 1 (PCH1). When PCH1 results from EXOSC3 gene mutations, it is sometimes categorized more specifically as PCH1B. In addition to the features listed above, PCH1B causes problems with muscle movement resulting from a loss of specialized nerve cells called motor neurons in the spinal cord.The EXOSC3 gene mutations that cause PCH1B result in an exosome component 3 protein with reduced or no function. The most common mutation alters a single protein building block (amino acid) in exosome component 3; it replaces the amino acid aspartic acid with the amino acid alanine at protein position 132 (written as Asp132Ala or D132A). People with this mutation tend to have somewhat less severe brain abnormalities than people with other EXOSC3 gene mutations. Changes in this gene likely impair the activity of the RNA exosome, but it is unclear how these alterations lead to the problems with brain development and loss of motor neurons characteristic of PCH1B. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      At least 16 mutations in the EXOSC3 gene have been identified in people with a disorder of brain development called pontocerebellar hypoplasia. The major features of this condition include delayed development overall, an unusually small head size (microcephaly), and intellectual disability. EXOSC3 gene mutations cause about half of all cases of a form of the disorder designated pontocerebellar hypoplasia type 1 (PCH1). When PCH1 results from EXOSC3 gene mutations, it is sometimes categorized more specifically as PCH1B. In addition to the features listed above, PCH1B causes problems with muscle movement resulting from a loss of specialized nerve cells called motor neurons in the spinal cord.The EXOSC3 gene mutations that cause PCH1B result in an exosome component 3 protein with reduced or no function. The most common mutation alters a single protein building block (amino acid) in exosome component 3; it replaces the amino acid aspartic acid with the amino acid alanine at protein position 132 (written as Asp132Ala or D132A). People with this mutation tend to have somewhat less severe brain abnormalities than people with other EXOSC3 gene mutations. Changes in this gene likely impair the activity of the RNA exosome, but it is unclear how these alterations lead to the problems with brain development and loss of motor neurons characteristic of PCH1B. More About This Health Condition    Cytogenetic Location: 9p13.2, which is the short (p) arm of chromosome 9 at position 13.2Molecular Location: base pairs 37,779,714 to 37,785,092 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        bA3J10.7CGI-102exosome complex component RRP40exosome complex exonuclease RRP40hRrp-40hRrp40pp10PCH1Bribosomal RNA-processing protein 40RRP40Rrp40p     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Genomes (second edition, 2002): Degradation of mRNAsMarie Curie Bioscience Database: Structural Components and Architectures of RNA Exosomes      Clinical Information from GeneReviews (1 link)     EXOSC3-Related Pontocerebellar Hypoplasia      Tests Listed in the Genetic Testing Registry (1 link)     Tests of EXOSC3      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     EXOSOME COMPONENT 3      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Genomes (second edition, 2002): Degradation of mRNAsMarie Curie Bioscience Database: Structural Components and Architectures of RNA Exosomes  EXOSC3-Related Pontocerebellar Hypoplasia  Tests of EXOSC3  PubMed  EXOSOME COMPONENT 3  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Biancheri R, Cassandrini D, Pinto F, Trovato R, Di Rocco M, Mirabelli-Badenier M, Pedemonte M, Panicucci C, Trucks H, Sander T, Zara F, Rossi A, Striano P, Minetti C, Santorelli FM. EXOSC3 mutations in isolated cerebellar hypoplasia and spinal anterior horn involvement. J Neurol. 2013 Jul;260(7):1866-70. doi: 10.1007/s00415-013-6896-0. Epub 2013 Apr 7. Citation on PubMedOMIM: EXOSOME COMPONENT 3Eggens VR, Barth PG, Niermeijer JM, Berg JN, Darin N, Dixit A, Fluss J, Foulds N, Fowler D, Hortobgyi T, Jacques T, King MD, Makrythanasis P, Mt A, Nicoll JA, O'Rourke D, Price S, Williams AN, Wilson L, Suri M, Sztriha L, Dijns-de Wissel MB, van Meegen MT, van Ruissen F, Aronica E, Troost D, Majoie CB, Marquering HA, Poll-Th BT, Baas F. EXOSC3 mutations in pontocerebellar hypoplasia type 1: novel mutations and genotype-phenotype correlations. Orphanet J Rare Dis. 2014 Feb 13;9:23. doi: 10.1186/1750-1172-9-23. Citation on PubMed or Free article on PubMed CentralEggens VRC, Barth PG, Baas F. EXOSC3-Related Pontocerebellar Hypoplasia. 2014 Aug 21. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK236968/ Citation on PubMedRudnik-Schneborn S, Senderek J, Jen JC, Houge G, Seeman P, Puchmajerov A, Graul-Neumann L, Seidel U, Korinthenberg R, Kirschner J, Seeger J, Ryan MM, Muntoni F, Steinlin M, Sztriha L, Colomer J, Hbner C, Brockmann K, Van Maldergem L, Schiff M, Holzinger A, Barth P, Reardon W, Yourshaw M, Nelson SF, Eggermann T, Zerres K. Pontocerebellar hypoplasia type 1: clinical spectrum and relevance of EXOSC3 mutations. Neurology. 2013 Jan 29;80(5):438-46. doi: 10.1212/WNL.0b013e31827f0f66. Epub 2013 Jan 2. Citation on PubMed or Free article on PubMed CentralWan J, Yourshaw M, Mamsa H, Rudnik-Schneborn S, Menezes MP, Hong JE, Leong DW, Senderek J, Salman MS, Chitayat D, Seeman P, von Moers A, Graul-Neumann L, Kornberg AJ, Castro-Gago M, Sobrido MJ, Sanefuji M, Shieh PB, Salamon N, Kim RC, Vinters HV, Chen Z, Zerres K, Ryan MM, Nelson SF, Jen JC. Mutations in the RNA exosome component gene EXOSC3 cause pontocerebellar hypoplasia and spinal motor neuron degeneration. Nat Genet. 2012 Apr 29;44(6):704-8. doi: 10.1038/ng.2254. Citation on PubMed or Free article on PubMed CentralZanni G, Scotton C, Passarelli C, Fang M, Barresi S, Dallapiccola B, Wu B, Gualandi F, Ferlini A, Bertini E, Wei W. Exome sequencing in a family with intellectual disability, early onset spasticity, and cerebellar atrophy detects a novel mutation in EXOSC3. Neurogenetics. 2013 Nov;14(3-4):247-50. doi: 10.1007/s10048-013-0371-z. Epub 2013 Aug 24. Citation on PubMed \n",
            "CNTNAP3\t\n",
            "SPATA31A3\t\n",
            "SPATA31A6\t\n",
            "CNTNAP3B\t\n",
            "CBWD6\t\n",
            "FOXD4L5\t\n",
            "CBWD3\t\n",
            "FOXD4L3\t\n",
            "TRPM3\t\n",
            "GDA\t\n",
            "PRUNE2\t\n",
            "KIF27\t\n",
            "SLC28A3\t\n",
            "AGTPBP1\t\n",
            "SPATA31E1\t\n",
            "SPATA31C1\t\n",
            "NINJ1\t\n",
            "C9orf129\t\n",
            "PHF2\t\n",
            "BARX1\t\n",
            "PTCH1\t   The PTCH1 gene provides instructions for producing the patched-1 protein, which functions as a receptor. Receptor proteins have specific sites into which certain other proteins, called ligands, fit like keys into locks. A protein called Sonic Hedgehog is the ligand for the patched-1 receptor. Together, ligands and their receptors trigger signals that affect cell development and function.Patched-1 and Sonic Hedgehog function in a pathway that is essential for early development. This pathway plays a role in cell growth, cell specialization, and determining the shape (patterning) of many different parts of the developing body. When Sonic Hedgehog is not present, patched-1 prevents cells from growing and dividing (proliferating). When Sonic Hedgehog is attached, patched-1 stops suppressing cell proliferation.  Based on its role in preventing cells from proliferating in an uncontrolled way, PTCH1 is called a tumor suppressor gene.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Gorlin syndrome     More than 225 mutations in the PTCH1 gene have been found to cause Gorlin syndrome (also known as nevoid basal cell carcinoma syndrome), a condition that affects many areas of the body and increases the risk of developing various cancerous and noncancerous tumors. Mutations in this gene prevent the production of patched-1 or lead to the production of an abnormal version of the receptor. An altered or missing patched-1 receptor cannot effectively suppress cell growth and division. As a result, cells proliferate uncontrollably to form the tumors that are characteristic of Gorlin syndrome. It is less clear how PTCH1 gene mutations cause the other signs and symptoms related to this condition, including small depressions (pits) in the skin of the palms of the hands and soles of the feet, an unusually large head size (macrocephaly), and skeletal abnormalities. More About This Health Condition      Nonsyndromic holoprosencephaly     Genetics Home Reference provides information about nonsyndromic holoprosencephaly. More About This Health Condition      9q22.3 microdeletion     The PTCH1 gene is located in a region of chromosome 9 that is deleted in people with a 9q22.3 microdeletion. As a result of this deletion, affected individuals are missing one copy of the PTCH1 gene in each cell. Researchers believe that many of the features associated with 9q22.3 microdeletions, particularly the signs and symptoms of Gorlin syndrome (described above), result from a loss of the PTCH1 gene. When this gene is missing, patched-1 is not available to suppress cell proliferation. As a result, cells divide uncontrollably to form the tumors that are characteristic of Gorlin syndrome. Other signs and symptoms related to 9q22.3 microdeletions (such as delayed development, intellectual disability, overgrowth of the body, and other physical abnormalities) may result from the loss of additional genes in the deleted region of chromosome 9. More About This Health Condition      Coloboma     Genetics Home Reference provides information about coloboma. More About This Health Condition      Cancers     Some mutations are acquired during a person's lifetime and are present only in certain cells. These genetic changes, called somatic mutations, are not inherited. Somatic mutations in both copies of the PTCH1 gene are associated with a non-hereditary (sporadic) type of skin cancer called basal cell carcinoma. Other sporadic types of cancer may be associated with somatic mutations in the PTCH1 gene, including some forms of skin cancer, a childhood brain tumor called medulloblastoma, breast cancer, and colon cancer. A noncancerous (benign) jaw tumor called a keratocystic odontogenic tumor can also be associated with somatic PTCH1 gene mutations.      Other disorders     At least seven mutations in the PTCH1 gene have been found to cause nonsyndromic holoprosencephaly. This condition occurs when the brain fails to divide into two halves during early development. PTCH1 gene mutations are a rare cause of nonsyndromic holoprosencephaly. These mutations prevent the signaling that is necessary for normal brain cell patterning. The signs and symptoms of nonsyndromic holoprosencephaly are caused by abnormal development of the brain and face.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 225 mutations in the PTCH1 gene have been found to cause Gorlin syndrome (also known as nevoid basal cell carcinoma syndrome), a condition that affects many areas of the body and increases the risk of developing various cancerous and noncancerous tumors. Mutations in this gene prevent the production of patched-1 or lead to the production of an abnormal version of the receptor. An altered or missing patched-1 receptor cannot effectively suppress cell growth and division. As a result, cells proliferate uncontrollably to form the tumors that are characteristic of Gorlin syndrome. It is less clear how PTCH1 gene mutations cause the other signs and symptoms related to this condition, including small depressions (pits) in the skin of the palms of the hands and soles of the feet, an unusually large head size (macrocephaly), and skeletal abnormalities. More About This Health Condition  Genetics Home Reference provides information about nonsyndromic holoprosencephaly. More About This Health Condition  The PTCH1 gene is located in a region of chromosome 9 that is deleted in people with a 9q22.3 microdeletion. As a result of this deletion, affected individuals are missing one copy of the PTCH1 gene in each cell. Researchers believe that many of the features associated with 9q22.3 microdeletions, particularly the signs and symptoms of Gorlin syndrome (described above), result from a loss of the PTCH1 gene. When this gene is missing, patched-1 is not available to suppress cell proliferation. As a result, cells divide uncontrollably to form the tumors that are characteristic of Gorlin syndrome. Other signs and symptoms related to 9q22.3 microdeletions (such as delayed development, intellectual disability, overgrowth of the body, and other physical abnormalities) may result from the loss of additional genes in the deleted region of chromosome 9. More About This Health Condition  Genetics Home Reference provides information about coloboma. More About This Health Condition  Some mutations are acquired during a person's lifetime and are present only in certain cells. These genetic changes, called somatic mutations, are not inherited. Somatic mutations in both copies of the PTCH1 gene are associated with a non-hereditary (sporadic) type of skin cancer called basal cell carcinoma. Other sporadic types of cancer may be associated with somatic mutations in the PTCH1 gene, including some forms of skin cancer, a childhood brain tumor called medulloblastoma, breast cancer, and colon cancer. A noncancerous (benign) jaw tumor called a keratocystic odontogenic tumor can also be associated with somatic PTCH1 gene mutations.  At least seven mutations in the PTCH1 gene have been found to cause nonsyndromic holoprosencephaly. This condition occurs when the brain fails to divide into two halves during early development. PTCH1 gene mutations are a rare cause of nonsyndromic holoprosencephaly. These mutations prevent the signaling that is necessary for normal brain cell patterning. The signs and symptoms of nonsyndromic holoprosencephaly are caused by abnormal development of the brain and face.    Cytogenetic Location: 9q22.32, which is the long (q) arm of chromosome 9 at position 22.32Molecular Location: base pairs 95,442,980 to 95,517,057 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BCNSFLJ26746FLJ42602HPE7NBCCSpatchedpatched homolog 1 (Drosophila)PTCPTC1PTC1_HUMANPTCH     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Developmental Biology (sixth edition, 2000): The Hedgehog Signal Transduction PathwayEurekah Bioscience Collection: The Patched Receptor: Switching On/Off the Hedgehog Signaling Pathway      Clinical Information from GeneReviews (2 links)     Holoprosencephaly OverviewNevoid Basal Cell Carcinoma Syndrome      Tests Listed in the Genetic Testing Registry (1 link)     Tests of PTCH1      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     PATCHED, DROSOPHILA, HOMOLOG OF, 1      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Developmental Biology (sixth edition, 2000): The Hedgehog Signal Transduction PathwayEurekah Bioscience Collection: The Patched Receptor: Switching On/Off the Hedgehog Signaling Pathway  Holoprosencephaly OverviewNevoid Basal Cell Carcinoma Syndrome  Tests of PTCH1  PubMed  PATCHED, DROSOPHILA, HOMOLOG OF, 1  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Adolphe C, Hetherington R, Ellis T, Wainwright B. Patched1 functions as a gatekeeper by promoting cell cycle progression. Cancer Res. 2006 Feb 15;66(4):2081-8. Citation on PubMedBale AE, Yu KP. The hedgehog pathway and basal cell carcinomas. Hum Mol Genet. 2001 Apr;10(7):757-62. Review. Citation on PubMedGorlin RJ. Nevoid basal cell carcinoma (Gorlin) syndrome. Genet Med. 2004 Nov-Dec;6(6):530-9. Review. Citation on PubMedIwasaki JK, Srivastava D, Moy RL, Lin HJ, Kouba DJ. The molecular genetics underlying basal cell carcinoma pathogenesis and links to targeted therapeutics. J Am Acad Dermatol. 2012 May;66(5):e167-78. doi: 10.1016/j.jaad.2010.06.054. Epub 2010 Aug 30. Review. Citation on PubMedLindstrm E, Shimokawa T, Toftgrd R, Zaphiropoulos PG. PTCH mutations: distribution and analyses. Hum Mutat. 2006 Mar;27(3):215-9. Review. Citation on PubMedLing G, Ahmadian A, Persson A, Undn AB, Afink G, Williams C, Uhln M, Toftgrd R, Lundeberg J, Pontn F. PATCHED and p53 gene alterations in sporadic and hereditary basal cell cancer. Oncogene. 2001 Nov 22;20(53):7770-8. Citation on PubMedLupi O. Correlations between the Sonic Hedgehog pathway and basal cell carcinoma. Int J Dermatol. 2007 Nov;46(11):1113-7. Review. Citation on PubMedMing JE, Kaupas ME, Roessler E, Brunner HG, Golabi M, Tekin M, Stratton RF, Sujansky E, Bale SJ, Muenke M. Mutations in PATCHED-1, the receptor for SONIC HEDGEHOG, are associated with holoprosencephaly. Hum Genet. 2002 Apr;110(4):297-301. Epub 2002 Mar 2. Erratum in: Hum Genet 2002 Oct;111(4-5):464. Citation on PubMedMuller EA, Aradhya S, Atkin JF, Carmany EP, Elliott AM, Chudley AE, Clark RD, Everman DB, Garner S, Hall BD, Herman GE, Kivuva E, Ramanathan S, Stevenson DA, Stockton DW, Hudgins L. Microdeletion 9q22.3 syndrome includes metopic craniosynostosis, hydrocephalus, macrosomia, and developmental delay. Am J Med Genet A. 2012 Feb;158A(2):391-9. doi: 10.1002/ajmg.a.34216. Epub 2011 Dec 21. Citation on PubMedOMIM: PATCHED, DROSOPHILA, HOMOLOG OF, 1Redon R, Baujat G, Sanlaville D, Le Merrer M, Vekemans M, Munnich A, Carter NP, Cormier-Daire V, Colleaux L. Interstitial 9q22.3 microdeletion: clinical and molecular characterisation of a newly recognised overgrowth syndrome. Eur J Hum Genet. 2006 Jun;14(6):759-67. Citation on PubMed \n",
            "ERCC6L2\t   From NCBI Gene:This gene encodes a member of the Snf2 family of helicase-like proteins. The encoded protein may play a role in DNA repair and mitochondrial function. Mutations in this gene have been associated with bone marrow failure syndrome 2. Alternatively spliced transcript variants that encode different protein isoforms have been described. [provided by RefSeq, Apr 2014]From UniProt:May be involved in early DNA damage response.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Bone marrow failure syndrome 2From UniProt:Bone marrow failure syndrome 2 (BMFS2): An autosomal recessive disorder characterized by trilineage bone marrow failure, bone marrow hypocellularity, learning difficulties, and microcephaly. Insufficient hematopoiesis results in peripheral blood cytopenias, affecting myeloid, erythroid and megakaryocyte lines. Cutaneous features and increased chromosome breakage are not features. [MIM:615715]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: 9q22.32, which is the long (q) arm of chromosome 9 at position 22.32Molecular Location: base pairs 95,875,618 to 96,041,092 on chromosome 9 (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BMFS2C9orf102RAD26LSR278     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of ERCC6L2      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: BONE MARROW FAILURE SYNDROME 2OMIM: ERCC6-LIKE 2      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of ERCC6L2  OMIM: BONE MARROW FAILURE SYNDROME 2OMIM: ERCC6-LIKE 2  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TAF7L\t\n",
            "ARMCX4\t\n",
            "TCEAL6\t\n",
            "TEX13A\t\n",
            "FRMPD3\t\n",
            "VSIG1\t\n",
            "COL4A6\t   From NCBI Gene:This gene encodes one of the six subunits of type IV collagen, the major structural component of basement membranes. Like the other members of the type IV collagen gene family, this gene is organized in a head-to-head conformation with another type IV collagen gene, alpha 5 type IV collagen, so that the gene pair shares a common promoter. Deletions in the alpha 5 gene that extend into the alpha 6 gene result in diffuse leiomyomatosis accompanying the X-linked Alport syndrome caused by the deletion in the alpha 5 gene. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Dec 2013]From UniProt:Type IV collagen is the major structural component of glomerular basement membranes (GBM), forming a 'chicken-wire' meshwork together with laminins, proteoglycans and entactin/nidogen.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Nonsyndromic hearing lossFrom NCBI Gene:Deafness, X-linked 6From UniProt:Deletions covering the N-terminal regions of COL4A5 and COL4A6, which are localized in a head-to-head manner, are found in the chromosome Xq22.3 centromeric deletion syndrome. This results in a phenotype with features of diffuse leiomyomatosis and Alport syndrome (DL-ATS).Deafness, X-linked, 6 (DFNX6): A non-syndromic form of sensorineural hearing loss with prelingual onset. Sensorineural deafness results from damage to the neural receptors of the inner ear, the nerve pathways to the brain, or the area of the brain that receives sound information. [MIM:300914]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: Xq22.3, which is the long (q) arm of the X chromosome at position 22.3Molecular Location: base pairs 108,155,607 to 108,439,497 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        CXDELq22.3DELXq22.3DFNX6     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of COL4A6      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: COLLAGEN, TYPE IV, ALPHA-6OMIM: DEAFNESS, X-LINKED 6      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of COL4A6  OMIM: COLLAGEN, TYPE IV, ALPHA-6OMIM: DEAFNESS, X-LINKED 6  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "RBMXL3\t\n",
            "SLC25A5\t\n",
            "GRIA3\t   From NCBI Gene:Glutamate receptors are the predominant excitatory neurotransmitter receptors in the mammalian brain and are activated in a variety of normal neurophysiologic processes. These receptors are heteromeric protein complexes composed of multiple subunits, arranged to form ligand-gated ion channels. The classification of glutamate receptors is based on their activation by different pharmacologic agonists. The subunit encoded by this gene belongs to a family of AMPA (alpha-amino-3-hydroxy-5-methyl-4-isoxazole propionate)-sensitive glutamate receptors, and is subject to RNA editing (AGA->GGA; R->G). Alternative splicing at this locus results in different isoforms, which may vary in their signal transduction properties. [provided by RefSeq, Jul 2008]From UniProt:Receptor for glutamate that functions as ligand-gated ion channel in the central nervous system and plays an important role in excitatory synaptic transmission. L-glutamate acts as an excitatory neurotransmitter at many synapses in the central nervous system. Binding of the excitatory neurotransmitter L-glutamate induces a conformation change, leading to the opening of the cation channel, and thereby converts the chemical signal to an electrical impulse. The receptor then desensitizes rapidly and enters a transient inactive state, characterized by the presence of bound agonist. In the presence of CACNG4 or CACNG7 or CACNG8, shows resensitization which is characterized by a delayed accumulation of current flux upon continued application of glutamate.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mental retardation, X-linked, syndromic, wu typeFrom UniProt:Mental retardation, X-linked 94 (MRX94): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. Intellectual deficiency is the only primary symptom of non-syndromic X-linked mental retardation, while syndromic mental retardation presents with associated physical, neurological and/or psychiatric manifestations. MRX94 patients have moderate mental retardation. Other variable features are macrocephaly, seizures, myoclonic jerks, autistic behavior, asthenic body habitus, distal muscle weakness and hyporeflexia. [MIM:300699]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: Xq25, which is the long (q) arm of the X chromosome at position 25Molecular Location: base pairs 123,184,243 to 123,490,915 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GluA3GLUR-CGLUR-K3GLUR3GLURCMRX94     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of GRIA3      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: GLUTAMATE RECEPTOR, IONOTROPIC, AMPA 3OMIM: MENTAL RETARDATION, X-LINKED, SYNDROMIC, WU TYPE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of GRIA3  OMIM: GLUTAMATE RECEPTOR, IONOTROPIC, AMPA 3OMIM: MENTAL RETARDATION, X-LINKED, SYNDROMIC, WU TYPE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "TLR8\t\n",
            "CRLF2\t\n",
            "CT45A3\t\n",
            "CT45A10\t\n",
            "RBMX\t   From NCBI Gene:This gene belongs to the RBMY gene family which includes candidate Y chromosome spermatogenesis genes. This gene, an active X chromosome homolog of the Y chromosome RBMY gene, is widely expressed whereas the RBMY gene evolved a male-specific function in spermatogenesis. Pseudogenes of this gene, found on chromosomes 1, 4, 9, 11, and 6, were likely derived by retrotransposition from the original gene. Alternatively spliced transcript variants encoding different isoforms have been identified. A snoRNA gene (SNORD61) is found in one of its introns. [provided by RefSeq, Sep 2009]From UniProt:RNA-binding protein that plays several role in the regulation of pre- and post-transcriptional processes. Implicated in tissue-specific regulation of gene transcription and alternative splicing of several pre-mRNAs. Binds to and stimulates transcription from the tumor suppressor TXNIP gene promoter; may thus be involved in tumor suppression. When associated with SAFB, binds to and stimulates transcription from the SREBF1 promoter. Associates with nascent mRNAs transcribed by RNA polymerase II. Component of the supraspliceosome complex that regulates pre-mRNA alternative splice site selection. Can either activate or suppress exon inclusion; acts additively with TRA2B to promote exon 7 inclusion of the survival motor neuron SMN2. Represses the splicing of MAPT/Tau exon 10. Binds preferentially to single-stranded 5'-CC[A/C]-rich RNA sequence motifs localized in a single-stranded conformation; probably binds RNA as a homodimer. Binds non-specifically to pre-mRNAs. Plays also a role in the cytoplasmic TNFR1 trafficking pathways; promotes both the IL-1-beta-mediated inducible proteolytic cleavage of TNFR1 ectodomains and the release of TNFR1 exosome-like vesicles to the extracellular compartment.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Mental retardation X-linked syndromic 11From UniProt:Mental retardation, X-linked, syndromic, 11 (MRXS11): A disorder characterized by significantly below average general intellectual functioning associated with impairments in adaptive behavior and manifested during the developmental period. MRXS11 patients manifest moderate intellectual disability and craniofacial dysmorphism. [MIM:300238]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: Xq26.3, which is the long (q) arm of the X chromosome at position 26.3Molecular Location: base pairs 136,869,194 to 136,880,780 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        hnRNP-GHNRNPGHNRPGMRXS11RBMXP1RBMXRTRNMX     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of RBMX      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: MENTAL RETARDATION, X-LINKED, SYNDROMIC 11OMIM: RNA-BINDING MOTIF PROTEIN, X CHROMOSOME      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of RBMX  OMIM: MENTAL RETARDATION, X-LINKED, SYNDROMIC 11OMIM: RNA-BINDING MOTIF PROTEIN, X CHROMOSOME  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GPR101\t   The GPR101 gene provides instructions for making a type of protein called a G protein-coupled receptor. G protein-coupled receptors are embedded in the outer membrane of cells, where they relay chemical signals from outside the cell to the interior of the cell. However, the specific function of the GPR101 protein is unknown. The protein is found primarily in the brain, and studies suggest that it has a role in the pituitary gland. This gland, located at the base of the brain, produces hormones that control many important body functions. One of these hormones, called growth hormone, helps direct normal growth of the body's bones and tissues. The GPR101 protein is predominantly expressed in the pituitary gland during development before birth and again at adolescence, stages when the body grows the most. The protein is thought to be involved in the growth of cells in the pituitary gland, in the release of growth hormone from the gland, or in both processes.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           X-linked acrogigantism     Genetic changes involving the GPR101 gene cause a condition called X-linked acrogigantism (X-LAG), which is characterized by abnormally fast growth beginning in infancy or early childhood. Signs and symptoms of the condition result from enlargement (hyperplasia) of the gland or development of a noncancerous tumor in the gland (called a pituitary adenoma). The abnormal gland releases more growth hormone than normal, causing rapid growth in individuals with X-LAG.X-LAG occurs when a small amount of genetic material on the X chromosome is abnormally copied (duplicated). The duplication, often referred to as an Xq26.3 microduplication, occurs on the long (q) arm of the chromosome at a location designated q26.3. Although several genes can be duplicated, only an extra copy of the GPR101 gene is necessary to cause X-LAG. Duplication of the gene leads to an excess of GPR101 protein. It is unclear how extra GPR101 protein results in pituitary adenoma or hyperplasia or the release of excess growth hormone. More About This Health Condition      Other disorders     A variation in the GPR101 gene is found in a small percentage of people with sporadic acromegaly, a condition that begins in adulthood and is characterized by overgrowth of the hands and feet, \"coarse\" facial features, heart problems, and other abnormalities. Sporadic acromegaly occurs in people with no history of overgrowth in their family. This variant leads to production of an altered GPR101 protein in which the protein building block (amino acid) glutamate is replaced with the amino acid aspartic acid at position 308 (written as Glu308Asp or E308D). Studies in the lab show that this alteration can lead to a small increase in the release of growth hormone and promote the growth and division (proliferation) of cells, although researchers are unsure exactly how it is associated with acromegaly in affected individuals.While some studies found the E308D GPR101 variant in a small percentage of people with sporadic acromegaly, other studies were unable to replicate this result, suggesting that the genetic change may not play a major role in development of the condition.        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Genetic changes involving the GPR101 gene cause a condition called X-linked acrogigantism (X-LAG), which is characterized by abnormally fast growth beginning in infancy or early childhood. Signs and symptoms of the condition result from enlargement (hyperplasia) of the gland or development of a noncancerous tumor in the gland (called a pituitary adenoma). The abnormal gland releases more growth hormone than normal, causing rapid growth in individuals with X-LAG.X-LAG occurs when a small amount of genetic material on the X chromosome is abnormally copied (duplicated). The duplication, often referred to as an Xq26.3 microduplication, occurs on the long (q) arm of the chromosome at a location designated q26.3. Although several genes can be duplicated, only an extra copy of the GPR101 gene is necessary to cause X-LAG. Duplication of the gene leads to an excess of GPR101 protein. It is unclear how extra GPR101 protein results in pituitary adenoma or hyperplasia or the release of excess growth hormone. More About This Health Condition  A variation in the GPR101 gene is found in a small percentage of people with sporadic acromegaly, a condition that begins in adulthood and is characterized by overgrowth of the hands and feet, \"coarse\" facial features, heart problems, and other abnormalities. Sporadic acromegaly occurs in people with no history of overgrowth in their family. This variant leads to production of an altered GPR101 protein in which the protein building block (amino acid) glutamate is replaced with the amino acid aspartic acid at position 308 (written as Glu308Asp or E308D). Studies in the lab show that this alteration can lead to a small increase in the release of growth hormone and promote the growth and division (proliferation) of cells, although researchers are unsure exactly how it is associated with acromegaly in affected individuals.While some studies found the E308D GPR101 variant in a small percentage of people with sporadic acromegaly, other studies were unable to replicate this result, suggesting that the genetic change may not play a major role in development of the condition.    Cytogenetic Location: Xq26.3, which is the long (q) arm of the X chromosome at position 26.3Molecular Location: base pairs 137,030,148 to 137,031,674 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        GPCR6PAGH2     Related Information   How are genetic conditions and genes named?          Educational Resources (2 links)     Molecular Cell Biology (fourth edition, 2000): G Protein-Coupled Receptors and Their EffectorsThe Cell: A Molecular Approach (second edition, 2000): G Protein-Coupled Receptors      Clinical Information from GeneReviews (1 link)     X-Linked Acrogigantism      Tests Listed in the Genetic Testing Registry (1 link)     Tests of GPR101      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     G PROTEIN-COUPLED RECEPTOR 101      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Molecular Cell Biology (fourth edition, 2000): G Protein-Coupled Receptors and Their EffectorsThe Cell: A Molecular Approach (second edition, 2000): G Protein-Coupled Receptors  X-Linked Acrogigantism  Tests of GPR101  PubMed  G PROTEIN-COUPLED RECEPTOR 101  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Daly AF, Yuan B, Fina F, Caberg JH, Trivellin G, Rostomyan L, de Herder WW, Naves LA, Metzger D, Cuny T, Rabl W, Shah N, Jaffrain-Rea ML, Zatelli MC, Faucz FR, Castermans E, Nanni-Metellus I, Lodish M, Muhammad A, Palmeira L, Potorac I, Mantovani G, Neggers SJ, Klein M, Barlier A, Liu P, Ouafik L, Bours V, Lupski JR, Stratakis CA, Beckers A. Somatic mosaicism underlies X-linked acrogigantism syndrome in sporadic male subjects. Endocr Relat Cancer. 2016 Apr;23(4):221-33. doi: 10.1530/ERC-16-0082. Epub 2016 Mar 2. Citation on PubMed or Free article on PubMed CentralFerra F, Romeo PD, Puglisi S, Ragonese M, Torre ML, Scaroni C, Occhi G, De Menis E, Arnaldi G, Trimarchi F, Cannav S. Analysis of GPR101 and AIP genes mutations in acromegaly: a multicentric study. Endocrine. 2016 Dec;54(3):762-767. doi: 10.1007/s12020-016-0862-4. Epub 2016 Jan 27. Citation on PubMedOMIM: G PROTEIN-COUPLED RECEPTOR 101Iacovazzo D, Caswell R, Bunce B, Jose S, Yuan B, Hernndez-Ramrez LC, Kapur S, Caimari F, Evanson J, Ferra F, Dang MN, Gabrovska P, Larkin SJ, Ansorge O, Rodd C, Vance ML, Ramrez-Renteria C, Mercado M, Goldstone AP, Buchfelder M, Burren CP, Gurlek A, Dutta P, Choong CS, Cheetham T, Trivellin G, Stratakis CA, Lopes MB, Grossman AB, Trouillas J, Lupski JR, Ellard S, Sampson JR, Roncaroli F, Korbonits M. Germline or somatic GPR101 duplication leads to X-linked acrogigantism: a clinico-pathological and genetic study. Acta Neuropathol Commun. 2016 Jun 1;4(1):56. doi: 10.1186/s40478-016-0328-1. Citation on PubMed or Free article on PubMed CentralMatsumoto R, Izawa M, Fukuoka H, Iguchi G, Odake Y, Yoshida K, Bando H, Suda K, Nishizawa H, Takahashi M, Inoshita N, Yamada S, Ogawa W, Takahashi Y. Genetic and clinical characteristics of Japanese patients with sporadic somatotropinoma. Endocr J. 2016 Nov 30;63(11):953-963. Epub 2016 Aug 6. Citation on PubMedTrivellin G, Bjelobaba I, Daly AF, Larco DO, Palmeira L, Faucz FR, Thiry A, Leal LF, Rostomyan L, Quezado M, Schernthaner-Reiter MH, Janjic MM, Villa C, Wu TJ, Stojilkovic SS, Beckers A, Feldman B, Stratakis CA. Characterization of GPR101 transcript structure and expression patterns. J Mol Endocrinol. 2016 Aug;57(2):97-111. doi: 10.1530/JME-16-0045. Epub 2016 Jun 9. Citation on PubMed or Free article on PubMed CentralTrivellin G, Daly AF, Faucz FR, Yuan B, Rostomyan L, Larco DO, Schernthaner-Reiter MH, Szarek E, Leal LF, Caberg JH, Castermans E, Villa C, Dimopoulos A, Chittiboina P, Xekouki P, Shah N, Metzger D, Lysy PA, Ferrante E, Strebkova N, Mazerkina N, Zatelli MC, Lodish M, Horvath A, de Alexandre RB, Manning AD, Levy I, Keil MF, Sierra Mde L, Palmeira L, Coppieters W, Georges M, Naves LA, Jamar M, Bours V, Wu TJ, Choong CS, Bertherat J, Chanson P, Kamenick P, Farrell WE, Barlier A, Quezado M, Bjelobaba I, Stojilkovic SS, Wess J, Costanzi S, Liu P, Lupski JR, Beckers A, Stratakis CA. Gigantism and acromegaly due to Xq26 microduplications and GPR101 mutation. N Engl J Med. 2014 Dec 18;371(25):2363-74. doi: 10.1056/NEJMoa1408028. Epub 2014 Dec 3. Citation on PubMed or Free article on PubMed Central \n",
            "CSF2RA\t   From NCBI Gene:The protein encoded by this gene is the alpha subunit of the heterodimeric receptor for colony stimulating factor 2, a cytokine which controls the production, differentiation, and function of granulocytes and macrophages. The encoded protein is a member of the cytokine family of receptors. This gene is found in the pseudoautosomal region (PAR) of the X and Y chromosomes. Multiple transcript variants encoding different isoforms have been found for this gene, with some of the isoforms being membrane-bound and others being soluble. [provided by RefSeq, Jul 2008]From UniProt:Low affinity receptor for granulocyte-macrophage colony-stimulating factor. Transduces a signal that results in the proliferation, differentiation, and functional activation of hematopoietic cells.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        Covered on Genetics Home Reference:Surfactant dysfunctionFrom NCBI Gene:Surfactant metabolism dysfunction, pulmonary, 4From UniProt:Pulmonary surfactant metabolism dysfunction 4 (SMDP4): A rare lung disorder due to impaired surfactant homeostasis. It is characterized by alveolar filling with floccular material that stains positive using the periodic acid-Schiff method and is derived from surfactant phospholipids and protein components. Excessive lipoproteins accumulation in the alveoli results in severe respiratory distress. [MIM:300770]     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: Xp22.33 and Yp11.2, which is the short (p) arm of the X chromosome at position 22.33 and the short (p) arm of the Y chromosome at position 11.2Molecular Location: base pairs 1,268,800 to 1,325,097 on chromosomes X and Y (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        alphaGMRCD116CDw116CSF2RCSF2RAXCSF2RAYCSF2RXCSF2RYGM-CSF-R-alphaGMCSFRGMRSMDP4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of CSF2RA      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: COLONY-STIMULATING FACTOR 2 RECEPTOR, ALPHAOMIM: SURFACTANT METABOLISM DYSFUNCTION, PULMONARY, 4      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of CSF2RA  OMIM: COLONY-STIMULATING FACTOR 2 RECEPTOR, ALPHAOMIM: SURFACTANT METABOLISM DYSFUNCTION, PULMONARY, 4  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "GPR50\t\n",
            "ASMTL\t\n",
            "ARSD\t\n",
            "PPP2R3B\t\n",
            "DMD\t   DMD, the largest known human gene, provides instructions for making a protein called dystrophin. This protein is located primarily in muscles used for movement (skeletal muscles) and in heart (cardiac) muscle. Small amounts of dystrophin are present in nerve cells in the brain.In skeletal and cardiac muscles, dystrophin is part of a group of proteins (a protein complex) that work together to strengthen muscle fibers and protect them from injury as muscles contract and relax. The dystrophin complex acts as an anchor, connecting each muscle cell's structural framework (cytoskeleton) with the lattice of proteins and other molecules outside the cell (extracellular matrix). The dystrophin complex may also play a role in cell signaling by interacting with proteins that send and receive chemical signals.Little is known about the function of dystrophin in nerve cells. Research suggests that the protein is important for the normal structure and function of synapses, which are specialized connections between nerve cells where cell-to-cell communication occurs.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Duchenne and Becker muscular dystrophy     More than 2,000 mutations in the DMD gene have been identified in people with the Duchenne and Becker forms of muscular dystrophy. These conditions occur almost exclusively in males and are characterized by progressive muscle weakness and wasting (atrophy) and a heart condition called dilated cardiomyopathy. Most of the mutations that cause these conditions delete part of the DMD gene. Other mutations abnormally duplicate part of the gene or change a small number of DNA building blocks (nucleotides) in the gene.Mutations that cause Becker muscular dystrophy, which typically has milder features and appears at a later age than Duchenne muscular dystrophy, usually lead to an abnormal version of dystrophin that retains some function. Mutations that cause the more severe Duchenne muscular dystrophy typically prevent any functional dystrophin from being produced.Skeletal and cardiac muscle cells without enough functional dystrophin become damaged as the muscles repeatedly contract and relax with use. The damaged cells weaken and die over time, causing the characteristic muscle weakness and heart problems seen in Duchenne and Becker muscular dystrophy. More About This Health Condition      X-linked dilated cardiomyopathy     More than 30 mutations in the DMD gene can cause an X-linked form of familial dilated cardiomyopathy. This heart condition enlarges and weakens the cardiac muscle, preventing the heart from pumping blood efficiently. Although dilated cardiomyopathy is a sign of Duchenne and Becker muscular dystrophy (described above), X-linked dilated cardiomyopathy is typically not associated with weakness and wasting of skeletal muscles.The mutations that cause X-linked dilated cardiomyopathy preferentially affect the activity of dystrophin in cardiac muscle cells. As a result of these mutations, affected individuals typically have little or no functional dystrophin in the heart. Without enough of this protein, cardiac muscle cells become damaged as the heart muscle repeatedly contracts and relaxes. The damaged muscle cells weaken and die over time, leading to the heart problems characteristic of X-linked dilated cardiomyopathy.The mutations that cause X-linked dilated cardiomyopathy often lead to reduced amounts of dystrophin in skeletal muscle cells. However, enough of this protein is present to prevent weakness and wasting of the skeletal muscles. More About This Health Condition      Familial dilated cardiomyopathy     Genetics Home Reference provides information about familial dilated cardiomyopathy. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 2,000 mutations in the DMD gene have been identified in people with the Duchenne and Becker forms of muscular dystrophy. These conditions occur almost exclusively in males and are characterized by progressive muscle weakness and wasting (atrophy) and a heart condition called dilated cardiomyopathy. Most of the mutations that cause these conditions delete part of the DMD gene. Other mutations abnormally duplicate part of the gene or change a small number of DNA building blocks (nucleotides) in the gene.Mutations that cause Becker muscular dystrophy, which typically has milder features and appears at a later age than Duchenne muscular dystrophy, usually lead to an abnormal version of dystrophin that retains some function. Mutations that cause the more severe Duchenne muscular dystrophy typically prevent any functional dystrophin from being produced.Skeletal and cardiac muscle cells without enough functional dystrophin become damaged as the muscles repeatedly contract and relax with use. The damaged cells weaken and die over time, causing the characteristic muscle weakness and heart problems seen in Duchenne and Becker muscular dystrophy. More About This Health Condition  More than 30 mutations in the DMD gene can cause an X-linked form of familial dilated cardiomyopathy. This heart condition enlarges and weakens the cardiac muscle, preventing the heart from pumping blood efficiently. Although dilated cardiomyopathy is a sign of Duchenne and Becker muscular dystrophy (described above), X-linked dilated cardiomyopathy is typically not associated with weakness and wasting of skeletal muscles.The mutations that cause X-linked dilated cardiomyopathy preferentially affect the activity of dystrophin in cardiac muscle cells. As a result of these mutations, affected individuals typically have little or no functional dystrophin in the heart. Without enough of this protein, cardiac muscle cells become damaged as the heart muscle repeatedly contracts and relaxes. The damaged muscle cells weaken and die over time, leading to the heart problems characteristic of X-linked dilated cardiomyopathy.The mutations that cause X-linked dilated cardiomyopathy often lead to reduced amounts of dystrophin in skeletal muscle cells. However, enough of this protein is present to prevent weakness and wasting of the skeletal muscles. More About This Health Condition  Genetics Home Reference provides information about familial dilated cardiomyopathy. More About This Health Condition    Cytogenetic Location: Xp21.2-p21.1, which is the short (p) arm of the X chromosome between positions 21.2 and 21.1Molecular Location: base pairs 31,119,219 to 33,339,609 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        BMDDMD_HUMANdystrophin (muscular dystrophy, Duchenne and Becker types)     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Madame Curie Bioscience Database: Structure and Function of the Dystrophin-Glycoprotein Complex      Clinical Information from GeneReviews (2 links)     Dilated Cardiomyopathy OverviewDystrophinopathies      Tests Listed in the Genetic Testing Registry (1 link)     Tests of DMD      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     DYSTROPHIN      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Madame Curie Bioscience Database: Structure and Function of the Dystrophin-Glycoprotein Complex  Dilated Cardiomyopathy OverviewDystrophinopathies  Tests of DMD  PubMed  DYSTROPHIN  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Aartsma-Rus A, Van Deutekom JC, Fokkema IF, Van Ommen GJ, Den Dunnen JT. Entries in the Leiden Duchenne muscular dystrophy mutation database: an overview of mutation types and paradoxical cases that confirm the reading-frame rule. Muscle Nerve. 2006 Aug;34(2):135-44. Review. Citation on PubMedAnderson JL, Head SI, Rae C, Morley JW. Brain function in Duchenne muscular dystrophy. Brain. 2002 Jan;125(Pt 1):4-13. Review. Citation on PubMedBiggar WD, Klamut HJ, Demacio PC, Stevens DJ, Ray PN. Duchenne muscular dystrophy: current knowledge, treatment, and future prospects. Clin Orthop Relat Res. 2002 Aug;(401):88-106. Review. Citation on PubMedCohen N, Muntoni F. Multiple pathogenetic mechanisms in X linked dilated cardiomyopathy. Heart. 2004 Aug;90(8):835-41. Review. Citation on PubMed or Free article on PubMed CentralDarras BT, Miller DT, Urion DK. Dystrophinopathies. 2000 Sep 5 [updated 2014 Nov 26]. In: Pagon RA, Adam MP, Ardinger HH, Wallace SE, Amemiya A, Bean LJH, Bird TD, Ledbetter N, Mefford HC, Smith RJH, Stephens K, editors. GeneReviews [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2017. Available from http://www.ncbi.nlm.nih.gov/books/NBK1119/ Citation on PubMedEhmsen J, Poon E, Davies K. The dystrophin-associated protein complex. J Cell Sci. 2002 Jul 15;115(Pt 14):2801-3. Review. Citation on PubMedErvasti JM. Dystrophin, its interactions with other proteins, and implications for muscular dystrophy. Biochim Biophys Acta. 2007 Feb;1772(2):108-17. Epub 2006 Jun 7. Review. Citation on PubMedFerlini A, Sewry C, Melis MA, Mateddu A, Muntoni F. X-linked dilated cardiomyopathy and the dystrophin gene. Neuromuscul Disord. 1999 Jul;9(5):339-46. Review. Citation on PubMedLe Rumeur E, Winder SJ, Hubert JF. Dystrophin: more than just the sum of its parts. Biochim Biophys Acta. 2010 Sep;1804(9):1713-22. doi: 10.1016/j.bbapap.2010.05.001. Epub 2010 May 21. Review. Citation on PubMedMuntoni F, Torelli S, Ferlini A. Dystrophin and mutations: one gene, several proteins, multiple phenotypes. Lancet Neurol. 2003 Dec;2(12):731-40. Review. Citation on PubMedNakamura A. X-Linked Dilated Cardiomyopathy: A Cardiospecific Phenotype of Dystrophinopathy. Pharmaceuticals (Basel). 2015 Jun 9;8(2):303-20. doi: 10.3390/ph8020303. Review. Citation on PubMed or Free article on PubMed CentralNeri M, Valli E, Alfano G, Bovolenta M, Spitali P, Rapezzi C, Muntoni F, Banfi S, Perini G, Gualandi F, Ferlini A. The absence of dystrophin brain isoform expression in healthy human heart ventricles explains the pathogenesis of 5' X-linked dilated cardiomyopathy. BMC Med Genet. 2012 Mar 28;13:20. doi: 10.1186/1471-2350-13-20. Citation on PubMed or Free article on PubMed Central \n",
            "SSX5\t\n",
            "DGKK\t\n",
            "SHROOM4\t   From NCBI Gene:This gene encodes a member of the APX/Shroom family, which contain an N-terminal PDZ domain and a C-terminal ASD2 motif. The encoded protein may play a role in cytoskeletal architecture. Mutations in this gene have been linked to the X-linked Stocco dos Santos syndrome characterized by cognitive disabilities. Alternatively spliced transcript variants have been described. [provided by RefSeq, Jul 2017]From UniProt:Probable regulator of cytoskeletal architecture that plays an important role in development. May regulate cellular and cytoskeletal architecture by modulating the spatial distribution of myosin II (By similarity).     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work        From NCBI Gene:Stocco dos Santos syndromeFrom UniProt:Mental retardation, X-linked, syndromic, Stocco dos Santos type (SDSX): A syndrome characterized by severe mental retardation with hyperactivity, aggressive behavior, delayed or no speech, and seizures. Additional features include congenital bilateral hip luxation, short stature, and kyphosis. [MIM:300434]A chromosomal aberration involving SHROOM4 is a cause of X-linked mental retardation (XLMR). Translocation t(X;8)(p11.22;p23.3) with FBXO25.A chromosomal aberration involving SHROOM4 is a cause of X-linked mental retardation (XLMR). Translocation t(X;19).     Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health        Cytogenetic Location: Xp11.22, which is the short (p) arm of the X chromosome at position 11.22Molecular Location: base pairs 50,575,534 to 50,814,164 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        SHAPshrm4     Related Information   How are genetic conditions and genes named?          Tests Listed in the Genetic Testing Registry (1 link)     Tests of SHROOM4      Catalog of Genes and Diseases from OMIM (2 links)     OMIM: SHROOM FAMILY MEMBER 4OMIM: STOCCO DOS SANTOS X-LINKED MENTAL RETARDATION SYNDROME      Research Resources (5 links)     ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Tests of SHROOM4  OMIM: SHROOM FAMILY MEMBER 4OMIM: STOCCO DOS SANTOS X-LINKED MENTAL RETARDATION SYNDROME  ClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt \n",
            "NUDT11\t\n",
            "ZXDA\t\n",
            "VCX3A\t\n",
            "EDA2R\t\n",
            "AR\t   The AR gene provides instructions for making a protein called an androgen receptor.  Androgens are hormones (such as testosterone) that are important for normal male sexual development before birth and during puberty.  Androgen receptors allow the body to respond appropriately to these hormones.  The receptors are present in many of the body's tissues, where they attach (bind) to androgens.  The resulting androgen-receptor complex then binds to DNA and regulates the activity of androgen-responsive genes. By turning the genes on or off as necessary, the androgen receptor helps direct the development of male sexual characteristics.  Androgens and androgen receptors also have other important functions in both males and females, such as regulating hair growth and sex drive.In one region of the AR gene, a DNA segment known as CAG is repeated multiple times. This CAG segment is called a triplet or trinucleotide repeat. In most people, the number of CAG repeats in the AR gene ranges from fewer than 10 to about 36.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Androgen insensitivity syndrome     More than 600 different mutations in the AR gene have been identified in people with androgen insensitivity syndrome, a condition that affects sexual development before birth and during puberty. Most of these mutations are changes in single DNA building blocks (base pairs). Other mutations insert or delete multiple base pairs in the gene or affect how the gene is processed into a protein. Some mutations lead to an abnormally short version of the androgen receptor protein, while others result in the production of an abnormal receptor that cannot bind to androgens or to DNA.  As a result, cells that are sensitive to androgens become less responsive to these hormones or unable to use these hormones at all. People with this condition are genetically male, with one X chromosome and one Y chromosome in each cell. Because their bodies are unable to respond to androgens, they may have mostly female sex characteristics or signs of both male and female sexual development.Mutations that completely eliminate the function of the androgen receptor cause complete androgen insensitivity syndrome. Genetic changes that significantly reduce but do not eliminate the receptor's activity cause partial androgen insensitivity syndrome.  Mild androgen insensitivity syndrome results from changes that only slightly reduce the activity of the receptor. More About This Health Condition      Spinal and bulbar muscular atrophy     Spinal and bulbar muscular atrophy, a disorder of specialized nerve cells that control muscle movement (motor neurons), results from an expansion of the CAG trinucleotide repeat in the AR gene. In people with this disorder, CAG is abnormally repeated from 38 to more than 60 times. Although the extended CAG region changes the structure of the androgen receptor, it is unclear how the altered protein damages nerve cells. Researchers believe that a fragment of the androgen receptor protein containing the CAG repeats accumulates within these cells and interferes with normal cell functions. This buildup leads to the gradual loss of motor neurons, which results in muscle weakness and wasting (atrophy). More About This Health Condition      Androgenetic alopecia     Changes in the AR gene are associated with an increased risk of androgenetic alopecia, a form of hair loss also known as male-pattern baldness in men and female-pattern baldness in women. The variations result from small changes in the number or types of DNA building blocks (base pairs) that make up the AR gene. These genetic changes appear to be most frequent in men with hair loss that begins at an early age. Researchers believe that AR gene variations may increase the activity of androgen receptors in the scalp. Although androgenetic alopecia is related to the effects of androgens on hair growth, it remains unclear how changes in the AR gene increase the risk of hair loss in men and women with this condition. More About This Health Condition      Polycystic ovary syndrome     Genetics Home Reference provides information about polycystic ovary syndrome. More About This Health Condition      Prostate cancer     Genetics Home Reference provides information about prostate cancer. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      More than 600 different mutations in the AR gene have been identified in people with androgen insensitivity syndrome, a condition that affects sexual development before birth and during puberty. Most of these mutations are changes in single DNA building blocks (base pairs). Other mutations insert or delete multiple base pairs in the gene or affect how the gene is processed into a protein. Some mutations lead to an abnormally short version of the androgen receptor protein, while others result in the production of an abnormal receptor that cannot bind to androgens or to DNA.  As a result, cells that are sensitive to androgens become less responsive to these hormones or unable to use these hormones at all. People with this condition are genetically male, with one X chromosome and one Y chromosome in each cell. Because their bodies are unable to respond to androgens, they may have mostly female sex characteristics or signs of both male and female sexual development.Mutations that completely eliminate the function of the androgen receptor cause complete androgen insensitivity syndrome. Genetic changes that significantly reduce but do not eliminate the receptor's activity cause partial androgen insensitivity syndrome.  Mild androgen insensitivity syndrome results from changes that only slightly reduce the activity of the receptor. More About This Health Condition  Spinal and bulbar muscular atrophy, a disorder of specialized nerve cells that control muscle movement (motor neurons), results from an expansion of the CAG trinucleotide repeat in the AR gene. In people with this disorder, CAG is abnormally repeated from 38 to more than 60 times. Although the extended CAG region changes the structure of the androgen receptor, it is unclear how the altered protein damages nerve cells. Researchers believe that a fragment of the androgen receptor protein containing the CAG repeats accumulates within these cells and interferes with normal cell functions. This buildup leads to the gradual loss of motor neurons, which results in muscle weakness and wasting (atrophy). More About This Health Condition  Changes in the AR gene are associated with an increased risk of androgenetic alopecia, a form of hair loss also known as male-pattern baldness in men and female-pattern baldness in women. The variations result from small changes in the number or types of DNA building blocks (base pairs) that make up the AR gene. These genetic changes appear to be most frequent in men with hair loss that begins at an early age. Researchers believe that AR gene variations may increase the activity of androgen receptors in the scalp. Although androgenetic alopecia is related to the effects of androgens on hair growth, it remains unclear how changes in the AR gene increase the risk of hair loss in men and women with this condition. More About This Health Condition  Genetics Home Reference provides information about polycystic ovary syndrome. More About This Health Condition  Genetics Home Reference provides information about prostate cancer. More About This Health Condition    Cytogenetic Location: Xq12, which is the long (q) arm of the X chromosome at position 12Molecular Location: base pairs 67,544,623 to 67,730,619 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        AISANDR_HUMANandrogen receptor (dihydrotestosterone receptor; testicular feminization; spinal and bulbar muscular atrophy; Kennedy disease)DHTRHUMARAKDNR3C4SBMASMAX1TFM     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Endotext (2000): Androgen Physiology: Receptor and Metabolic Disorders      Clinical Information from GeneReviews (2 links)     Androgen Insensitivity SyndromeSpinal and Bulbar Muscular Atrophy      Tests Listed in the Genetic Testing Registry (1 link)     Tests of AR      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ANDROGEN RECEPTOR      Research Resources (7 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics WebClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Endotext (2000): Androgen Physiology: Receptor and Metabolic Disorders  Androgen Insensitivity SyndromeSpinal and Bulbar Muscular Atrophy  Tests of AR  PubMed  ANDROGEN RECEPTOR  Atlas of Genetics and Cytogenetics in Oncology and HaematologyCancer Genetics WebClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  Adachi H, Katsuno M, Minamiyama M, Waza M, Sang C, Nakagomi Y, Kobayashi Y, Tanaka F, Doyu M, Inukai A, Yoshida M, Hashizume Y, Sobue G. Widespread nuclear and cytoplasmic accumulation of mutant androgen receptor in SBMA patients. Brain. 2005 Mar;128(Pt 3):659-70. Epub 2005 Jan 19. Citation on PubMedBeitel LK, Scanlon T, Gottlieb B, Trifiro MA. Progress in Spinobulbar muscular atrophy research: insights into neuronal dysfunction caused by the polyglutamine-expanded androgen receptor. Neurotox Res. 2005;7(3):219-30. Review. Citation on PubMedBennett NC, Gardiner RA, Hooper JD, Johnson DW, Gobe GC. Molecular cell biology of androgen receptor signalling. Int J Biochem Cell Biol. 2010 Jun;42(6):813-27. doi: 10.1016/j.biocel.2009.11.013. Epub 2009 Nov 30. Review. Citation on PubMedGottlieb B, Beitel LK, Nadarajah A, Paliouras M, Trifiro M. The androgen receptor gene mutations database: 2012 update. Hum Mutat. 2012 May;33(5):887-94. doi: 10.1002/humu.22046. Epub 2012 Mar 13. Citation on PubMedGottlieb B, Beitel LK, Wu J, Elhaji YA, Trifiro M. Nuclear receptors and disease: androgen receptor. Essays Biochem. 2004;40:121-36. Review. Citation on PubMedHillmer AM, Hanneken S, Ritzmann S, Becker T, Freudenberg J, Brockschmidt FF, Flaquer A, Freudenberg-Hua Y, Jamra RA, Metzen C, Heyn U, Schweiger N, Betz RC, Blaumeiser B, Hampe J, Schreiber S, Schulze TG, Hennies HC, Schumacher J, Propping P, Ruzicka T, Cichon S, Wienker TF, Kruse R, Nothen MM. Genetic variation in the human androgen receptor gene is the major determinant of common early-onset androgenetic alopecia. Am J Hum Genet. 2005 Jul;77(1):140-8. Epub 2005 May 18. Citation on PubMed or Free article on PubMed CentralKatsuno M, Adachi H, Tanaka F, Sobue G. Spinal and bulbar muscular atrophy: ligand-dependent pathogenesis and therapeutic perspectives. J Mol Med (Berl). 2004 May;82(5):298-307. Epub 2004 Feb 27. Review. Citation on PubMedLevy-Nissenbaum E, Bar-Natan M, Frydman M, Pras E. Confirmation of the association between male pattern baldness and the androgen receptor gene. Eur J Dermatol. 2005 Sep-Oct;15(5):339-40. Citation on PubMedPoletti A, Negri-Cesi P, Martini L. Reflections on the diseases linked to mutations of the androgen receptor. Endocrine. 2005 Dec;28(3):243-62. Review. Citation on PubMedZajac JD, Fui MN. Kennedy's disease: clinical significance of tandem repeats in the androgen receptor. Adv Exp Med Biol. 2012;769:153-68. Review. Citation on PubMed \n",
            "FAM155B\t\n",
            "NAP1L2\t\n",
            "ATP7A\t   The ATP7A gene provides instructions for making a protein that is important for regulating copper levels in the body. Copper is necessary for many cellular functions, but it is toxic when present in excessive amounts. The ATP7A protein is found throughout the body, except in liver cells. In the small intestine, this protein helps control the absorption of copper from food. In other cells, the ATP7A protein has a dual role and shuttles between two cellular locations. The protein normally resides in a cell structure called the Golgi apparatus, which modifies newly produced proteins, including enzymes. In the Golgi apparatus, the ATP7A protein supplies copper to certain enzymes that are critical for the structure and function of bone, skin, hair, blood vessels, and the nervous system. If copper levels in the cell environment are elevated, however, the ATP7A protein moves to the cell membrane and eliminates excess copper from the cell.     Related Information   What is DNA?What is a gene?How do genes direct the production of proteins?More about How Genes Work           Cutis laxa     Several mutations in the ATP7A gene are responsible for a condition called occipital horn syndrome or X-linked cutis laxa, which is considered a mild form of Menkes syndrome. Occipital horn syndrome is characterized by loose and sagging skin, wedge-shaped calcium deposits in a bone at the base of the skull (the occipital bone), coarse hair, and loose joints.Most of the mutations that cause occipital horn syndrome reduce but do not eliminate the production of the ATP7A protein. A shortage of this protein impairs the absorption of copper from food and prevents its normal distribution to cells throughout the body. The decreased supply of copper can reduce the activity of numerous copper-containing enzymes, affecting the structure and function of bone, skin, hair, blood vessels, and the nervous system. The reduced activity of these enzymes underlies the characteristic features of occipital horn syndrome. More About This Health Condition      Menkes syndrome     Researchers have identified more than 150 mutations in the ATP7A gene that cause Menkes syndrome. Many of these mutations delete part of the gene and likely result in a shortened ATP7A protein. Other mutations insert additional DNA building blocks (nucleotides) into the gene or change single nucleotides. All of these mutations prevent the production of functional ATP7A protein. As a result, the absorption of copper from food is impaired, and copper is not supplied to certain enzymes. The abnormal protein may get stuck in the cell membrane and become unable to shuttle back and forth from the Golgi apparatus.The disrupted activity of the ATP7A protein causes copper to be poorly distributed to cells in the body. Copper accumulates in some tissues, such as the small intestine and kidneys, while the brain and other tissues have unusually low levels. The decreased supply of copper can reduce the activity of numerous copper-containing enzymes, affecting the structure and function of bone, skin, hair, blood vessels, and the nervous system. The signs and symptoms of Menkes syndrome are caused by the reduced activity of these copper-containing enzymes. More About This Health Condition      Charcot-Marie-Tooth disease     Genetics Home Reference provides information about Charcot-Marie-Tooth disease. More About This Health Condition        Related Information   What is a gene mutation and how do mutations occur?What kinds of gene mutations are possible?More about Mutations and Health      Several mutations in the ATP7A gene are responsible for a condition called occipital horn syndrome or X-linked cutis laxa, which is considered a mild form of Menkes syndrome. Occipital horn syndrome is characterized by loose and sagging skin, wedge-shaped calcium deposits in a bone at the base of the skull (the occipital bone), coarse hair, and loose joints.Most of the mutations that cause occipital horn syndrome reduce but do not eliminate the production of the ATP7A protein. A shortage of this protein impairs the absorption of copper from food and prevents its normal distribution to cells throughout the body. The decreased supply of copper can reduce the activity of numerous copper-containing enzymes, affecting the structure and function of bone, skin, hair, blood vessels, and the nervous system. The reduced activity of these enzymes underlies the characteristic features of occipital horn syndrome. More About This Health Condition  Researchers have identified more than 150 mutations in the ATP7A gene that cause Menkes syndrome. Many of these mutations delete part of the gene and likely result in a shortened ATP7A protein. Other mutations insert additional DNA building blocks (nucleotides) into the gene or change single nucleotides. All of these mutations prevent the production of functional ATP7A protein. As a result, the absorption of copper from food is impaired, and copper is not supplied to certain enzymes. The abnormal protein may get stuck in the cell membrane and become unable to shuttle back and forth from the Golgi apparatus.The disrupted activity of the ATP7A protein causes copper to be poorly distributed to cells in the body. Copper accumulates in some tissues, such as the small intestine and kidneys, while the brain and other tissues have unusually low levels. The decreased supply of copper can reduce the activity of numerous copper-containing enzymes, affecting the structure and function of bone, skin, hair, blood vessels, and the nervous system. The signs and symptoms of Menkes syndrome are caused by the reduced activity of these copper-containing enzymes. More About This Health Condition  Genetics Home Reference provides information about Charcot-Marie-Tooth disease. More About This Health Condition    Cytogenetic Location: Xq21.1, which is the long (q) arm of the X chromosome at position 21.1Molecular Location: base pairs 77,910,656 to 78,050,395 on the X chromosome (Homo sapiens Annotation Release 109, GRCh38.p12) (NCBI)Credit: Genome Decoration Page/NCBI     Related Information   How do geneticists indicate the location of a gene?        ATP7A_HUMANATPase, Cu++ transporting, alpha polypeptideATPase, Cu++ transporting, alpha polypeptide (Menkes syndrome)ATPP1copper pump 1MC1MKMNKOHS     Related Information   How are genetic conditions and genes named?          Educational Resources (1 link)     Basic Neurochemistry: Molecular, Cellular, and Medical Aspects (sixth edition, 1999): Linkage of Copper and Iron Metabolism in Basal Ganglia Disorders      Clinical Information from GeneReviews (1 link)     ATP7A-Related Copper Transport Disorders      Tests Listed in the Genetic Testing Registry (1 link)     Tests of ATP7A      Scientific Articles on PubMed (1 link)     PubMed      Catalog of Genes and Diseases from OMIM (1 link)     ATPase, Cu(2+)-TRANSPORTING, ALPHA POLYPEPTIDE      Research Resources (6 links)     Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt      Basic Neurochemistry: Molecular, Cellular, and Medical Aspects (sixth edition, 1999): Linkage of Copper and Iron Metabolism in Basal Ganglia Disorders  ATP7A-Related Copper Transport Disorders  Tests of ATP7A  PubMed  ATPase, Cu(2+)-TRANSPORTING, ALPHA POLYPEPTIDE  Atlas of Genetics and Cytogenetics in Oncology and HaematologyClinVarHGNC Gene Symbol ReportMonarch InitiativeNCBI GeneUniProt  OMIM: ATPase, Cu(2+)-TRANSPORTING, ALPHA POLYPEPTIDEBarnes N, Tsivkovskii R, Tsivkovskaia N, Lutsenko S. The copper-transporting ATPases, menkes and wilson disease proteins, have distinct roles in adult and developing cerebellum. J Biol Chem. 2005 Mar 11;280(10):9640-5. Epub 2005 Jan 5. Citation on PubMedBertini I, Rosato A. Menkes disease. Cell Mol Life Sci. 2008 Jan;65(1):89-91. Review. Citation on PubMedDonsante A, Tang J, Godwin SC, Holmes CS, Goldstein DS, Bassuk A, Kaler SG. Differences in ATP7A gene expression underlie intrafamilial variability in Menkes disease/occipital horn syndrome. J Med Genet. 2007 Aug;44(8):492-7. Epub 2007 May 11. Erratum in: J Med Genet. 2008 Jan;45(1):64. Citation on PubMed or Free article on PubMed CentralGreenough M, Pase L, Voskoboinik I, Petris MJ, O'Brien AW, Camakaris J. Signals regulating trafficking of Menkes (MNK; ATP7A) copper-translocating P-type ATPase in polarized MDCK cells. Am J Physiol Cell Physiol. 2004 Nov;287(5):C1463-71. Epub 2004 Jul 21. Citation on PubMedHarris ED. Basic and clinical aspects of copper. Crit Rev Clin Lab Sci. 2003 Oct;40(5):547-86. Review. Citation on PubMedKaler SG. Metabolic and molecular bases of Menkes disease and occipital horn syndrome. Pediatr Dev Pathol. 1998 Jan-Feb;1(1):85-98. Review. Citation on PubMedMller LB, Tmer Z, Lund C, Petersen C, Cole T, Hanusch R, Seidel J, Jensen LR, Horn N. Similar splice-site mutations of the ATP7A gene lead to different phenotypes: classical Menkes disease or occipital horn syndrome. Am J Hum Genet. 2000 Apr;66(4):1211-20. Epub 2000 Mar 17. Citation on PubMed or Free article on PubMed CentralProhaska JR. Role of copper transporters in copper homeostasis. Am J Clin Nutr. 2008 Sep;88(3):826S-9S. Citation on PubMed or Free article on PubMed CentralTang J, Robertson S, Lem KE, Godwin SC, Kaler SG. Functional copper transport explains neurologic sparing in occipital horn syndrome. Genet Med. 2006 Nov;8(11):711-8. Citation on PubMedVoskoboinik I, Camakaris J. Menkes copper-translocating P-type ATPase (ATP7A): biochemical and cell biology properties, and role in Menkes disease. J Bioenerg Biomembr. 2002 Oct;34(5):363-71. Review. Citation on PubMedde Bie P, Muller P, Wijmenga C, Klomp LW. Molecular pathogenesis of Wilson and Menkes disease: correlation of mutations with molecular defects and disease phenotypes. J Med Genet. 2007 Nov;44(11):673-88. Epub 2007 Aug 23. Review. Citation on PubMed or Free article on PubMed Central \n",
            "VCX\t\n",
            "VCX2\t\n"
          ],
          "name": "stdout"
        }
      ]
    }
  ]
}