
### What we did

#### Group Members

- [Alivia Blount, ML Engineer]()
- [Alex Sutherland, Data Scientist]()
- [Kennan Salinero, Biochemist]()
- [Masha Aseeva, Bioinformatics software engineer]()
- [Nikhita Damaraju, Biosciences Undergrad]()
- [Nitin Pasumarthy, Applied Machine Learning Engineer](https://www.linkedin.com/in/nitinpasumarthy/)

#### Deck [Link](https://docs.google.com/presentation/d/1ZXT5jUo6xXdOTiBgAHEMkdiA7NQvlT2sAEywak2mAVI/edit#slide=id.g5b3afa7024_3_0)

#### Generate pathway images with variant genes highlighted

We used the VCF file generated by Qiagen and applied additional filters to exclude common variants from 4 other databases.

1. A .vcf file filtering the Whole Genome Sequence by:
 1. Confidence: Call quality at least 20 (assay)
 2. Exclude "common" variants that are:
     1.  at least 1% in 1000 Genomes Project
     2.  at least 1% of all in the ExAC
     3.  at least 1% of all in the gnomAD
     4.  at least 1% of all NHLBI ESP exomes
     5. UNLESS it is confirmed Pathogenic
     
2. Next we used the Databse for Annotation, Visualisation and Integrated Discovery [(DAVID)](https://david-d.ncifcrf.gov/) to map gene names to generate a mapping file that maps gene name input from the above VCF to different ontologies / databases (gene names, GO, INTERPRO, KEGG). This produced a list of metabolic pathway names.

3. We generated a list of mutated genes for each KEGG pathway, and then fed it to R. The pathway images were generated using the ‘pathview’ package of Bioconductor in R. The package pulls in the KEGG pathway and labels the genes that I gave it.

#### Long Read Alignment/Variant Calling

Attempted to perform variant calling on Oxford Nanopore Long Read data

1. Perform sequence alighment using [minimap2](https://github.com/lh3/minimap2)
 * Used reference genome [GRCh37 (masked genomic DNA)](ftp://ftp.ensembl.org/pub/grch37/current/fasta/homo_sapiens/dna/Homo_sapiens.GRCh37.dna_rm.primary_assembly.fa.gz)
 * Used [ordinary minimizers as seeds](https://github.com/lh3/minimap2#map-long-noisy-genomic-reads) for optimum Oxford Nanopore alignment

2. TBD: Variant call results from step 1

#### Looking for genetic/metabolic pathways that may be inefficient/inactive and combinations of pathways that may additively cause multi-factorial impacts (‘pile-ups’) affecting John’s health outcomes.

	Components:
	
	•	Consulting blood, urine, and genetic analytic work from medical charts, create a list of metabolites, proteins, and antibody reactivities to identify possible pathways and disease states of interest.
	•	Create a Biological Context filter set to screen variants in John’s various genomic data sets to look for instances of variants mapping to these screens
	•	Add upstream and down-stream hops in data screening to find sets of associations – genes within a pathway – that may additively affect John’s ability to process particular metabolites, or to have an effective biological pathway (signaling, growth, response, immunity, etc).
	
	Outcome
	
	•	Compare sets of gene variants identified, with and without gene family/pathway association via the ‘hopping’ filter
	
	Background:
	
	•	By accident, Nikhita and I had searched the same vcf dataset and got opposite, almost non-overlapping results with respect to genes.  However, we found that I was getting genes that were part of the same pathway, but different components of the pathway.
	•	We investigated, after mapping out the dimensions of the non-overlap.  We found that the difference was solely due to my inclusion of the ‘hop’ feature in the biological context filter.  
	•	Therefore we were selecting, in our case, genes downstream of the genes in the biological context screen.  Therefore we were obtaining genes along the chromosome that were nearby, and that were part of functional groups co-localized in the genome.



